{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Tutorial: dynamic GRN inference from SHARE-seq dataset of mouse skin\n",
    "# Step 2: dynamic GRN inference and analysis with Dictys\n",
    "This is a [Dictys](https://github.com/pinellolab/dictys) tutorial to reconstruct and analyze dynamic GRNs from the SHARE-seq dataset of mouse skin. This notebook is step 2 that reconstructs and analyzes dynamic GRNs with Dictys. You should have completed running [main1.ipynb](main1.ipynb).\n",
    "\n",
    "This is the same dataset as the one used in the paper, but you should not expect it reproduce the figures. This is because the tutorial aims to demonstrate the reconstruction of dynamic GRNs not provided in the paper. Use tutorial [analysis-skin](https://github.com/pinellolab/dictys/tree/master/doc/tutorials/analysis-skin) if you want to redraw the figures.\n",
    "\n",
    "Before running, you need to **change the runtime configurations** for your machine at [2. Network inference configuration](#2.-Network-inference-configuration) and [3. Dynamic network inference](#3.-Dynamic-network-inference).\n",
    "\n",
    "If you face any issues or need any assistance, see [FAQ](https://github.com/pinellolab/dictys#faq) and [Issues](https://github.com/pinellolab/dictys#issues).\n",
    "\n",
    "## 1. Preparation of individual input files in `data` folder\n",
    "The following files/folders are already prepared. If you need to know how they are prepared, see the [short-multiome](https://github.com/pinellolab/dictys/tree/master/doc/tutorials/short-multiome) or [full-multiome](https://github.com/pinellolab/dictys/tree/master/doc/tutorials/full-multiome) tutorials.\n",
    "* expression.tsv.gz: Read count matrix of RNA-profiled cells in compressed tsv format.\n",
    "* bams: This folder contains one bam file for each cell with chromatin accessibility measurement. File name should be cell name.\n",
    "* subsets.txt: Names of cell subsets. For each cell subset, a static GRN is reconstructed.\n",
    "* subsets: Folder containing one subfolder for each cell subset as in `subsets.txt`. Each subfolder contains two files:\n",
    "    - names_rna.txt: Names of cells that belong to this subset and have transcriptome measurement\n",
    "    - names_atac.txt: Names of cells that belong to this subset and have chromatin accessibility measurement\n",
    "    - For joint measurements of RNA and ATAC, these two files should be identical in every folder.\n",
    "* whitelist.bed: Bed file containing potential genes each region may regulate expression. If absent, all region-gene regulations are allowed. Here, it is obtained from the [SHARE-seq paper](https://dx.doi.org/10.1016%2Fj.cell.2020.09.056).\n",
    "    \n",
    "### Prepare trajectory files\n",
    "Converting the inferred trajectory and each cell's location on the trajectory to h5 format"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Removes CPU usage limit by some jupyter versions\n",
    "import os\n",
    "os.environ['KMP_AFFINITY'] = ''\n",
    "# Configure matplotlib to enable large animations\n",
    "import matplotlib\n",
    "matplotlib.rcParams['animation.embed_limit'] = 2**128\n",
    "import matplotlib.pyplot as plt\n",
    "# Prepare trajectory files\n",
    "import pandas as pd\n",
    "import dictys\n",
    "\n",
    "# Load data\n",
    "dist=pd.read_csv('../tmp/dist.tsv.gz',header=0,index_col=0,sep='\\t')\n",
    "edge=pd.read_csv('../tmp/edge.tsv.gz',header=None,index_col=None,sep='\\t')\n",
    "branch=pd.read_csv('../tmp/branch.tsv.gz',header=None,index_col=None,sep='\\t')\n",
    "# Save data\n",
    "traj=dictys.traj.trajectory.fromdist(edge.values,dist.values)\n",
    "traj.to_file('../data/traj_node.h5')\n",
    "point=dictys.traj.point.fromdist(traj,branch.values,dist.values)\n",
    "point.to_file('../data/traj_cell_rna.h5',traj=False)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Prepare other files\n",
    "Motif, reference genome, gene transcriptional start site"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "cd ../data\n",
    "\n",
    "# Motifs (file motifs.motif)\n",
    "# Option 1: from HOCOMOCO (https://hocomoco11.autosome.org/)\n",
    "wget -q -o /dev/null -O - 'https://hocomoco11.autosome.org/final_bundle/hocomoco11/full/MOUSE/mono/HOCOMOCOv11_full_MOUSE_mono_homer_format_0.0001.motif' | awk -F \"\\t\" 'BEGIN { OFS = \"\\t\"} {if (substr($1,1,1) == \">\") $2=substr($2,1,1)tolower(substr($2,2)); print}' > motifs.motif\n",
    "# Option 2: from HOMER\n",
    "# dictys_helper motif_homer.sh -c 3 > motifs.motif\n",
    "# Option 3: provide your custom motifs\n",
    "\n",
    "# Reference genome (folder genome)\n",
    "# Note: You need the same reference genome version with chromatin accessibility reads\n",
    "# Option 1: download genome from HOMER\n",
    "dictys_helper genome_homer.sh mm10 genome\n",
    "# Option 2: provide your custom genome\n",
    "\n",
    "# Bed file for TSS (file gene.bed)\n",
    "# Download gtf file from ensembl\n",
    "wget -q -o /dev/null -O gene.gtf.gz http://ftp.ensembl.org/pub/release-102/gtf/mus_musculus/Mus_musculus.GRCm38.102.gtf.gz\n",
    "gunzip gene.gtf.gz\n",
    "# Convert to bed\n",
    "dictys_helper gene_gtf.sh gene.gtf gene.bed\n",
    "rm gene.gtf\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Optional: Prepare covariates\n",
    "The optional input file `covariate.tsv.gz` allows to account for technical covariates and other confounders of single-cell transcriptomic profiles. Covariate inclusion is automatic as long as `covariate.tsv.gz` is present in the `data` folder. To confirm your covariates are correctly recognized, you should find lines like `... python3 -m dictys  network reconstruct ... --fi_cov data/covariate.tsv.gz ...` in the output of step [3. Dynamic network inference](#3.-Dynamic-network-inference).\n",
    "\n",
    "As an example, the cell below creates `covariate.tsv.gz` to account for technical batch effects. To include these covariates, simply turn the cell below into code type (see Menu -> Run -> Cell Type -> Change To Code Cell Type) before running this notebook. This is disabled by default.\n",
    "\n",
    "You can put any discrete or continuous covariates from any source into `covariate.tsv.gz`, for both cell-type specific and dynamic GRN inference."
   ]
  },
  {
   "cell_type": "raw",
   "metadata": {},
   "source": [
    "import itertools\n",
    "from os import linesep\n",
    "import numpy as np\n",
    "#Number of technical rounds that can introduce batch effects\n",
    "nround=4\n",
    "\n",
    "#Load cell names\n",
    "cellnames=list(pd.read_csv('../data/expression.tsv.gz',header=0,index_col=0,sep='\\t',nrows=1).columns)\n",
    "print('Cell names: {}'.format(linesep+linesep.join(cellnames[:3]+['...'])))\n",
    "#Extract batch ID covariates from cell names\n",
    "covs=np.array([[int(y) for y in x.split('.')[-2*nround+1::2]] for x in cellnames]).T\n",
    "covnames=cellnames[0].split('.')[-2*nround::2]\n",
    "covid=[np.unique(x) for x in covs]\n",
    "#Exclude final batch in each round as redundant degrees of freedom\n",
    "covs=[covs[x]==np.repeat(covid[x][:-1].reshape(-1,1),covs.shape[1],axis=1) for x in range(nround)]\n",
    "covnames=[['{}={}'.format(covnames[x],y) for y in covid[x][:-1]] for x in range(nround)]\n",
    "assert all(len(covnames[x])==covs[x].shape[0] for x in range(nround))\n",
    "covnames=list(itertools.chain.from_iterable(covnames))\n",
    "#Save covariate file\n",
    "covs=pd.DataFrame(np.concatenate(covs,axis=0).T.astype(int),index=cellnames,columns=covnames)\n",
    "print('Covariates: '+linesep+str(covs.T.head().T.head())+linesep+'...')\n",
    "covs.to_csv('../data/covariate.tsv.gz',header=True,index=True,sep='\\t')"
   ]
  },  
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2. Network inference configuration\n",
    "### Generate configurations\n",
    "Please adjust them for your own machine and dataset"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "# Generate configuration template\n",
    "rm -Rf ../makefiles\n",
    "mkdir ../makefiles\n",
    "cd ../makefiles\n",
    "dictys_helper makefile_template.sh common.mk config.mk env_none.mk dynamic.mk\n",
    "\n",
    "# Update configurations, such as:\n",
    "# DEVICE: pytorch device, e.g. cpu, cuda:0. If you do not have a GPU, use 'cpu' and expect LONG computing time.\n",
    "# GENOME_MACS2: effective genome size for macs2. See https://deeptools.readthedocs.io/en/develop/content/feature/effectiveGenomeSize.html\n",
    "# JOINT: whether dataset is joint profiling of RNA and ATAC.\n",
    "# Other configurations include quality control thresholds, number of threads in each job, number of hidden confounders, etc.\n",
    "# They can be obtained in the full-multiome tutorial.\n",
    "dictys_helper makefile_update.py ../makefiles/config.mk '{\"DEVICE\": \"cuda:0\", \"GENOME_MACS2\": \"mm\", \"JOINT\": \"1\"}'\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Validate input data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Joint profile: True\n",
      "Found 6436 cells with RNA profile\n",
      "Found 19262 genes with RNA profile\n",
      "Found 6436 cells with ATAC profile\n",
      "Found 529 motifs\n",
      "Found 451 TFs\n",
      "Found 289 TFs in current dataset\n",
      "Missing 162 TFs in current dataset: Andr,Ap2a,Ap2b,Ap2c,Ap2d,Ari3a,Ari5b,Ascl1,Ascl2,Atoh1,Barx1,Bha15,Bhe40,Bhe41,Bmal1,Brac,Cebpd,Cebpe,Coe1,Cot1,Cot2,Crx,Dmrtb,Err1,Err2,Err3,Etv2,Evi1,Fev,Foxa1,Foxa2,Foxd1,Foxd3,Foxf2,Foxi1,Foxl2,Gata2,Gata5,Gcr,Hand1,Hen1,Hesx1,Hnf6,Htf4,Hxa1,Hxa10,Hxa13,Hxa5,Hxa7,Hxa9,Hxb1,Hxb4,Hxb6,Hxb7,Hxb8,Hxc6,Hxc8,Hxc9,Hxd10,Hxd13,Hxd4,Hxd9,Insm1,Itf2,Kaiso,Lhx3,Lyl1,Mafa,Mcr,Mlxpl,Msgn1,Msx3,Myba,Mybb,Myod1,Myog,Ndf1,Ndf2,Nf2l1,Nf2l2,Nfac1,Nfac2,Nfac3,Nfac4,Ngn2,Nkx21,Nkx22,Nkx25,Nkx28,Nkx31,Nkx32,Nkx61,Nobox,Nr0b1,Nr1i3,Nr2e3,Olig2,Onec2,P53,P63,P73,Pebb,Pit1,Pitx1,Pknx1,Po2f1,Po2f2,Po3f1,Po3f2,Po4f2,Po5f1,Po6f1,Prd14,Prd16,Prgr,Prop1,Ptf1a,Rorg,Sall1,Six2,Smca5,Sox17,Sox3,Sp5,Spib,Spz1,Srbp1,Srbp2,Sry,Sta5a,Sta5b,Stf1,Suh,Tf2l1,Tf65,Tf7l1,Tf7l2,Tfe2,Tha,Tha11,Thb,Tlx1,Twst1,Tyy1,Ubip1,Usf1,Usf2,Zbt17,Zbt18,Zbt7a,Zep1,Zep2,Zfp42,Zic3,Zksc1,Zn143,Zn148,Zn281,Zn322,Zn335,Zn423,Zn431\n",
      "Found 18125 genes with TSS information\n",
      "WARNING:root:Chromosomes not found in reference genome: chrGL456211.1,chrGL456221.1,chrGL456233.1,chrJH584295.1,chrJH584299.1,chrMT\n",
      "WARNING:root:Cannot find static.mk or subsets.txt. Skipping static network inference checks.\n"
     ]
    }
   ],
   "source": [
    "!cd .. && dictys_helper makefile_check.py"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3. Dynamic network inference\n",
    "* **This step can take a day or longer**. You are strongly suggested to use a GPU. If it takes over a day, see [FAQ](https://github.com/pinellolab/dictys#faq) to improve speed.\n",
    "* Please adjust `-j n_jobs` for your machine. The total thread count is n_jobs*n_threads. By default n_threads=4."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkdir -p tmp_dynamic\n",
      "if ! [ -e tmp_dynamic/subsets.txt ] || ! [ -e tmp_dynamic/subset_locs.h5 ] || ! [ -e tmp_dynamic/subset_edges.tsv.gz ]; then OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  dynamic subsets_rna data/traj_node.h5 data/traj_cell_rna.h5 data/coord_rna.tsv.gz tmp_dynamic/subsets.txt tmp_dynamic/subset_locs.h5 tmp_dynamic tmp_dynamic/subset_edges.tsv.gz 1000 875 0.05; fi\n",
      "cp tmp_dynamic/Subset1/names_rna.txt tmp_dynamic/Subset1/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset1/names_rna.txt tmp_dynamic/Subset1/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset2/names_rna.txt tmp_dynamic/Subset2/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset2/names_rna.txt tmp_dynamic/Subset2/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset3/names_rna.txt tmp_dynamic/Subset3/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset3/names_rna.txt tmp_dynamic/Subset3/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset4/names_rna.txt tmp_dynamic/Subset4/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset4/names_rna.txt tmp_dynamic/Subset4/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset5/names_rna.txt tmp_dynamic/Subset5/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset5/names_rna.txt tmp_dynamic/Subset5/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset6/names_rna.txt tmp_dynamic/Subset6/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset6/names_rna.txt tmp_dynamic/Subset6/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset7/names_rna.txt tmp_dynamic/Subset7/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset7/names_rna.txt tmp_dynamic/Subset7/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset8/names_rna.txt tmp_dynamic/Subset8/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset8/names_rna.txt tmp_dynamic/Subset8/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset9/names_rna.txt tmp_dynamic/Subset9/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset9/names_rna.txt tmp_dynamic/Subset9/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset10/names_rna.txt tmp_dynamic/Subset10/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset10/names_rna.txt tmp_dynamic/Subset10/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset11/names_rna.txt tmp_dynamic/Subset11/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset11/names_rna.txt tmp_dynamic/Subset11/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset12/names_rna.txt tmp_dynamic/Subset12/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset12/names_rna.txt tmp_dynamic/Subset12/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset13/names_rna.txt tmp_dynamic/Subset13/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset13/names_rna.txt tmp_dynamic/Subset13/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset14/names_rna.txt tmp_dynamic/Subset14/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset14/names_rna.txt tmp_dynamic/Subset14/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset15/names_rna.txt tmp_dynamic/Subset15/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset15/names_rna.txt tmp_dynamic/Subset15/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset16/names_rna.txt tmp_dynamic/Subset16/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset16/names_rna.txt tmp_dynamic/Subset16/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset17/names_rna.txt tmp_dynamic/Subset17/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset17/names_rna.txt tmp_dynamic/Subset17/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset18/names_rna.txt tmp_dynamic/Subset18/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset18/names_rna.txt tmp_dynamic/Subset18/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset19/names_rna.txt tmp_dynamic/Subset19/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset19/names_rna.txt tmp_dynamic/Subset19/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset20/names_rna.txt tmp_dynamic/Subset20/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset20/names_rna.txt tmp_dynamic/Subset20/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset21/names_rna.txt tmp_dynamic/Subset21/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset21/names_rna.txt tmp_dynamic/Subset21/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset22/names_rna.txt tmp_dynamic/Subset22/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset22/names_rna.txt tmp_dynamic/Subset22/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset23/names_rna.txt tmp_dynamic/Subset23/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset23/names_rna.txt tmp_dynamic/Subset23/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset24/names_rna.txt tmp_dynamic/Subset24/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset24/names_rna.txt tmp_dynamic/Subset24/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset25/names_rna.txt tmp_dynamic/Subset25/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset25/names_rna.txt tmp_dynamic/Subset25/expression0.tsv.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "cp tmp_dynamic/Subset26/names_rna.txt tmp_dynamic/Subset26/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset26/names_rna.txt tmp_dynamic/Subset26/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset27/names_rna.txt tmp_dynamic/Subset27/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset27/names_rna.txt tmp_dynamic/Subset27/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset28/names_rna.txt tmp_dynamic/Subset28/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset28/names_rna.txt tmp_dynamic/Subset28/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset29/names_rna.txt tmp_dynamic/Subset29/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset29/names_rna.txt tmp_dynamic/Subset29/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset30/names_rna.txt tmp_dynamic/Subset30/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset30/names_rna.txt tmp_dynamic/Subset30/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset31/names_rna.txt tmp_dynamic/Subset31/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset31/names_rna.txt tmp_dynamic/Subset31/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset32/names_rna.txt tmp_dynamic/Subset32/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset32/names_rna.txt tmp_dynamic/Subset32/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset33/names_rna.txt tmp_dynamic/Subset33/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset33/names_rna.txt tmp_dynamic/Subset33/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset34/names_rna.txt tmp_dynamic/Subset34/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset34/names_rna.txt tmp_dynamic/Subset34/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset35/names_rna.txt tmp_dynamic/Subset35/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset35/names_rna.txt tmp_dynamic/Subset35/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset36/names_rna.txt tmp_dynamic/Subset36/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset36/names_rna.txt tmp_dynamic/Subset36/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset37/names_rna.txt tmp_dynamic/Subset37/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset37/names_rna.txt tmp_dynamic/Subset37/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset38/names_rna.txt tmp_dynamic/Subset38/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset38/names_rna.txt tmp_dynamic/Subset38/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset39/names_rna.txt tmp_dynamic/Subset39/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset39/names_rna.txt tmp_dynamic/Subset39/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset40/names_rna.txt tmp_dynamic/Subset40/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset40/names_rna.txt tmp_dynamic/Subset40/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset41/names_rna.txt tmp_dynamic/Subset41/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset41/names_rna.txt tmp_dynamic/Subset41/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset42/names_rna.txt tmp_dynamic/Subset42/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset42/names_rna.txt tmp_dynamic/Subset42/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset43/names_rna.txt tmp_dynamic/Subset43/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset43/names_rna.txt tmp_dynamic/Subset43/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset44/names_rna.txt tmp_dynamic/Subset44/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset44/names_rna.txt tmp_dynamic/Subset44/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset45/names_rna.txt tmp_dynamic/Subset45/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset45/names_rna.txt tmp_dynamic/Subset45/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset46/names_rna.txt tmp_dynamic/Subset46/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset46/names_rna.txt tmp_dynamic/Subset46/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset47/names_rna.txt tmp_dynamic/Subset47/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset47/names_rna.txt tmp_dynamic/Subset47/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset48/names_rna.txt tmp_dynamic/Subset48/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset48/names_rna.txt tmp_dynamic/Subset48/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset49/names_rna.txt tmp_dynamic/Subset49/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset49/names_rna.txt tmp_dynamic/Subset49/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset50/names_rna.txt tmp_dynamic/Subset50/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset50/names_rna.txt tmp_dynamic/Subset50/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset51/names_rna.txt tmp_dynamic/Subset51/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset51/names_rna.txt tmp_dynamic/Subset51/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset52/names_rna.txt tmp_dynamic/Subset52/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset52/names_rna.txt tmp_dynamic/Subset52/expression0.tsv.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "cp tmp_dynamic/Subset53/names_rna.txt tmp_dynamic/Subset53/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset53/names_rna.txt tmp_dynamic/Subset53/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset54/names_rna.txt tmp_dynamic/Subset54/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset54/names_rna.txt tmp_dynamic/Subset54/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset55/names_rna.txt tmp_dynamic/Subset55/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset55/names_rna.txt tmp_dynamic/Subset55/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset56/names_rna.txt tmp_dynamic/Subset56/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset56/names_rna.txt tmp_dynamic/Subset56/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset57/names_rna.txt tmp_dynamic/Subset57/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset57/names_rna.txt tmp_dynamic/Subset57/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset58/names_rna.txt tmp_dynamic/Subset58/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset58/names_rna.txt tmp_dynamic/Subset58/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset59/names_rna.txt tmp_dynamic/Subset59/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset59/names_rna.txt tmp_dynamic/Subset59/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset60/names_rna.txt tmp_dynamic/Subset60/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset60/names_rna.txt tmp_dynamic/Subset60/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset61/names_rna.txt tmp_dynamic/Subset61/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset61/names_rna.txt tmp_dynamic/Subset61/expression0.tsv.gz\n",
      "cp tmp_dynamic/Subset62/names_rna.txt tmp_dynamic/Subset62/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_dynamic/Subset62/names_rna.txt tmp_dynamic/Subset62/expression0.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset1/expression0.tsv.gz tmp_dynamic/Subset1/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset2/expression0.tsv.gz tmp_dynamic/Subset2/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset3/expression0.tsv.gz tmp_dynamic/Subset3/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset4/expression0.tsv.gz tmp_dynamic/Subset4/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset5/expression0.tsv.gz tmp_dynamic/Subset5/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset6/expression0.tsv.gz tmp_dynamic/Subset6/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset7/expression0.tsv.gz tmp_dynamic/Subset7/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset8/expression0.tsv.gz tmp_dynamic/Subset8/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset9/expression0.tsv.gz tmp_dynamic/Subset9/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset10/expression0.tsv.gz tmp_dynamic/Subset10/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset11/expression0.tsv.gz tmp_dynamic/Subset11/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset12/expression0.tsv.gz tmp_dynamic/Subset12/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset13/expression0.tsv.gz tmp_dynamic/Subset13/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset14/expression0.tsv.gz tmp_dynamic/Subset14/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset15/expression0.tsv.gz tmp_dynamic/Subset15/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset16/expression0.tsv.gz tmp_dynamic/Subset16/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset17/expression0.tsv.gz tmp_dynamic/Subset17/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset18/expression0.tsv.gz tmp_dynamic/Subset18/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset19/expression0.tsv.gz tmp_dynamic/Subset19/expression.tsv.gz 50 10 0 200 100 0\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset20/expression0.tsv.gz tmp_dynamic/Subset20/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset21/expression0.tsv.gz tmp_dynamic/Subset21/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset22/expression0.tsv.gz tmp_dynamic/Subset22/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset23/expression0.tsv.gz tmp_dynamic/Subset23/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset24/expression0.tsv.gz tmp_dynamic/Subset24/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset25/expression0.tsv.gz tmp_dynamic/Subset25/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset26/expression0.tsv.gz tmp_dynamic/Subset26/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset27/expression0.tsv.gz tmp_dynamic/Subset27/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset28/expression0.tsv.gz tmp_dynamic/Subset28/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset29/expression0.tsv.gz tmp_dynamic/Subset29/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset30/expression0.tsv.gz tmp_dynamic/Subset30/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset31/expression0.tsv.gz tmp_dynamic/Subset31/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset32/expression0.tsv.gz tmp_dynamic/Subset32/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset33/expression0.tsv.gz tmp_dynamic/Subset33/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset34/expression0.tsv.gz tmp_dynamic/Subset34/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset35/expression0.tsv.gz tmp_dynamic/Subset35/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset36/expression0.tsv.gz tmp_dynamic/Subset36/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset37/expression0.tsv.gz tmp_dynamic/Subset37/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset38/expression0.tsv.gz tmp_dynamic/Subset38/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset39/expression0.tsv.gz tmp_dynamic/Subset39/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset40/expression0.tsv.gz tmp_dynamic/Subset40/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset41/expression0.tsv.gz tmp_dynamic/Subset41/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset42/expression0.tsv.gz tmp_dynamic/Subset42/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset43/expression0.tsv.gz tmp_dynamic/Subset43/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset44/expression0.tsv.gz tmp_dynamic/Subset44/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset45/expression0.tsv.gz tmp_dynamic/Subset45/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset46/expression0.tsv.gz tmp_dynamic/Subset46/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset47/expression0.tsv.gz tmp_dynamic/Subset47/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset48/expression0.tsv.gz tmp_dynamic/Subset48/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset49/expression0.tsv.gz tmp_dynamic/Subset49/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset50/expression0.tsv.gz tmp_dynamic/Subset50/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset51/expression0.tsv.gz tmp_dynamic/Subset51/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset52/expression0.tsv.gz tmp_dynamic/Subset52/expression.tsv.gz 50 10 0 200 100 0\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset53/expression0.tsv.gz tmp_dynamic/Subset53/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset54/expression0.tsv.gz tmp_dynamic/Subset54/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset55/expression0.tsv.gz tmp_dynamic/Subset55/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset56/expression0.tsv.gz tmp_dynamic/Subset56/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset57/expression0.tsv.gz tmp_dynamic/Subset57/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset58/expression0.tsv.gz tmp_dynamic/Subset58/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset59/expression0.tsv.gz tmp_dynamic/Subset59/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset60/expression0.tsv.gz tmp_dynamic/Subset60/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset61/expression0.tsv.gz tmp_dynamic/Subset61/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_dynamic/Subset62/expression0.tsv.gz tmp_dynamic/Subset62/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset1/expression.tsv.gz tmp_dynamic/Subset1/names_atac0.txt tmp_dynamic/Subset1/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset2/expression.tsv.gz tmp_dynamic/Subset2/names_atac0.txt tmp_dynamic/Subset2/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset3/expression.tsv.gz tmp_dynamic/Subset3/names_atac0.txt tmp_dynamic/Subset3/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset4/expression.tsv.gz tmp_dynamic/Subset4/names_atac0.txt tmp_dynamic/Subset4/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset5/expression.tsv.gz tmp_dynamic/Subset5/names_atac0.txt tmp_dynamic/Subset5/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset6/expression.tsv.gz tmp_dynamic/Subset6/names_atac0.txt tmp_dynamic/Subset6/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset7/expression.tsv.gz tmp_dynamic/Subset7/names_atac0.txt tmp_dynamic/Subset7/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset8/expression.tsv.gz tmp_dynamic/Subset8/names_atac0.txt tmp_dynamic/Subset8/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset9/expression.tsv.gz tmp_dynamic/Subset9/names_atac0.txt tmp_dynamic/Subset9/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset10/expression.tsv.gz tmp_dynamic/Subset10/names_atac0.txt tmp_dynamic/Subset10/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset11/expression.tsv.gz tmp_dynamic/Subset11/names_atac0.txt tmp_dynamic/Subset11/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset12/expression.tsv.gz tmp_dynamic/Subset12/names_atac0.txt tmp_dynamic/Subset12/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset13/expression.tsv.gz tmp_dynamic/Subset13/names_atac0.txt tmp_dynamic/Subset13/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset14/expression.tsv.gz tmp_dynamic/Subset14/names_atac0.txt tmp_dynamic/Subset14/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset15/expression.tsv.gz tmp_dynamic/Subset15/names_atac0.txt tmp_dynamic/Subset15/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset16/expression.tsv.gz tmp_dynamic/Subset16/names_atac0.txt tmp_dynamic/Subset16/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset17/expression.tsv.gz tmp_dynamic/Subset17/names_atac0.txt tmp_dynamic/Subset17/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset18/expression.tsv.gz tmp_dynamic/Subset18/names_atac0.txt tmp_dynamic/Subset18/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset19/expression.tsv.gz tmp_dynamic/Subset19/names_atac0.txt tmp_dynamic/Subset19/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset20/expression.tsv.gz tmp_dynamic/Subset20/names_atac0.txt tmp_dynamic/Subset20/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset21/expression.tsv.gz tmp_dynamic/Subset21/names_atac0.txt tmp_dynamic/Subset21/names_atac.txt\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset22/expression.tsv.gz tmp_dynamic/Subset22/names_atac0.txt tmp_dynamic/Subset22/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset23/expression.tsv.gz tmp_dynamic/Subset23/names_atac0.txt tmp_dynamic/Subset23/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset24/expression.tsv.gz tmp_dynamic/Subset24/names_atac0.txt tmp_dynamic/Subset24/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset25/expression.tsv.gz tmp_dynamic/Subset25/names_atac0.txt tmp_dynamic/Subset25/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset26/expression.tsv.gz tmp_dynamic/Subset26/names_atac0.txt tmp_dynamic/Subset26/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset27/expression.tsv.gz tmp_dynamic/Subset27/names_atac0.txt tmp_dynamic/Subset27/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset28/expression.tsv.gz tmp_dynamic/Subset28/names_atac0.txt tmp_dynamic/Subset28/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset29/expression.tsv.gz tmp_dynamic/Subset29/names_atac0.txt tmp_dynamic/Subset29/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset30/expression.tsv.gz tmp_dynamic/Subset30/names_atac0.txt tmp_dynamic/Subset30/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset31/expression.tsv.gz tmp_dynamic/Subset31/names_atac0.txt tmp_dynamic/Subset31/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset32/expression.tsv.gz tmp_dynamic/Subset32/names_atac0.txt tmp_dynamic/Subset32/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset33/expression.tsv.gz tmp_dynamic/Subset33/names_atac0.txt tmp_dynamic/Subset33/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset34/expression.tsv.gz tmp_dynamic/Subset34/names_atac0.txt tmp_dynamic/Subset34/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset35/expression.tsv.gz tmp_dynamic/Subset35/names_atac0.txt tmp_dynamic/Subset35/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset36/expression.tsv.gz tmp_dynamic/Subset36/names_atac0.txt tmp_dynamic/Subset36/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset37/expression.tsv.gz tmp_dynamic/Subset37/names_atac0.txt tmp_dynamic/Subset37/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset38/expression.tsv.gz tmp_dynamic/Subset38/names_atac0.txt tmp_dynamic/Subset38/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset39/expression.tsv.gz tmp_dynamic/Subset39/names_atac0.txt tmp_dynamic/Subset39/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset40/expression.tsv.gz tmp_dynamic/Subset40/names_atac0.txt tmp_dynamic/Subset40/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset41/expression.tsv.gz tmp_dynamic/Subset41/names_atac0.txt tmp_dynamic/Subset41/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset42/expression.tsv.gz tmp_dynamic/Subset42/names_atac0.txt tmp_dynamic/Subset42/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset43/expression.tsv.gz tmp_dynamic/Subset43/names_atac0.txt tmp_dynamic/Subset43/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset44/expression.tsv.gz tmp_dynamic/Subset44/names_atac0.txt tmp_dynamic/Subset44/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset45/expression.tsv.gz tmp_dynamic/Subset45/names_atac0.txt tmp_dynamic/Subset45/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset46/expression.tsv.gz tmp_dynamic/Subset46/names_atac0.txt tmp_dynamic/Subset46/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset47/expression.tsv.gz tmp_dynamic/Subset47/names_atac0.txt tmp_dynamic/Subset47/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset48/expression.tsv.gz tmp_dynamic/Subset48/names_atac0.txt tmp_dynamic/Subset48/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset49/expression.tsv.gz tmp_dynamic/Subset49/names_atac0.txt tmp_dynamic/Subset49/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset50/expression.tsv.gz tmp_dynamic/Subset50/names_atac0.txt tmp_dynamic/Subset50/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset51/expression.tsv.gz tmp_dynamic/Subset51/names_atac0.txt tmp_dynamic/Subset51/names_atac.txt\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset52/expression.tsv.gz tmp_dynamic/Subset52/names_atac0.txt tmp_dynamic/Subset52/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset53/expression.tsv.gz tmp_dynamic/Subset53/names_atac0.txt tmp_dynamic/Subset53/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset54/expression.tsv.gz tmp_dynamic/Subset54/names_atac0.txt tmp_dynamic/Subset54/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset55/expression.tsv.gz tmp_dynamic/Subset55/names_atac0.txt tmp_dynamic/Subset55/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset56/expression.tsv.gz tmp_dynamic/Subset56/names_atac0.txt tmp_dynamic/Subset56/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset57/expression.tsv.gz tmp_dynamic/Subset57/names_atac0.txt tmp_dynamic/Subset57/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset58/expression.tsv.gz tmp_dynamic/Subset58/names_atac0.txt tmp_dynamic/Subset58/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset59/expression.tsv.gz tmp_dynamic/Subset59/names_atac0.txt tmp_dynamic/Subset59/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset60/expression.tsv.gz tmp_dynamic/Subset60/names_atac0.txt tmp_dynamic/Subset60/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset61/expression.tsv.gz tmp_dynamic/Subset61/names_atac0.txt tmp_dynamic/Subset61/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset1/names_atac.txt data/bams tmp_dynamic/Subset1/reads.bam tmp_dynamic/Subset1/reads.bai tmp_dynamic/Subset1/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset2/names_atac.txt data/bams tmp_dynamic/Subset2/reads.bam tmp_dynamic/Subset2/reads.bai tmp_dynamic/Subset2/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset3/names_atac.txt data/bams tmp_dynamic/Subset3/reads.bam tmp_dynamic/Subset3/reads.bai tmp_dynamic/Subset3/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset4/names_atac.txt data/bams tmp_dynamic/Subset4/reads.bam tmp_dynamic/Subset4/reads.bai tmp_dynamic/Subset4/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset5/names_atac.txt data/bams tmp_dynamic/Subset5/reads.bam tmp_dynamic/Subset5/reads.bai tmp_dynamic/Subset5/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset6/names_atac.txt data/bams tmp_dynamic/Subset6/reads.bam tmp_dynamic/Subset6/reads.bai tmp_dynamic/Subset6/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset7/names_atac.txt data/bams tmp_dynamic/Subset7/reads.bam tmp_dynamic/Subset7/reads.bai tmp_dynamic/Subset7/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset8/names_atac.txt data/bams tmp_dynamic/Subset8/reads.bam tmp_dynamic/Subset8/reads.bai tmp_dynamic/Subset8/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset9/names_atac.txt data/bams tmp_dynamic/Subset9/reads.bam tmp_dynamic/Subset9/reads.bai tmp_dynamic/Subset9/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset10/names_atac.txt data/bams tmp_dynamic/Subset10/reads.bam tmp_dynamic/Subset10/reads.bai tmp_dynamic/Subset10/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset11/names_atac.txt data/bams tmp_dynamic/Subset11/reads.bam tmp_dynamic/Subset11/reads.bai tmp_dynamic/Subset11/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset12/names_atac.txt data/bams tmp_dynamic/Subset12/reads.bam tmp_dynamic/Subset12/reads.bai tmp_dynamic/Subset12/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset13/names_atac.txt data/bams tmp_dynamic/Subset13/reads.bam tmp_dynamic/Subset13/reads.bai tmp_dynamic/Subset13/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset14/names_atac.txt data/bams tmp_dynamic/Subset14/reads.bam tmp_dynamic/Subset14/reads.bai tmp_dynamic/Subset14/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset15/names_atac.txt data/bams tmp_dynamic/Subset15/reads.bam tmp_dynamic/Subset15/reads.bai tmp_dynamic/Subset15/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset16/names_atac.txt data/bams tmp_dynamic/Subset16/reads.bam tmp_dynamic/Subset16/reads.bai tmp_dynamic/Subset16/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset17/names_atac.txt data/bams tmp_dynamic/Subset17/reads.bam tmp_dynamic/Subset17/reads.bai tmp_dynamic/Subset17/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset18/names_atac.txt data/bams tmp_dynamic/Subset18/reads.bam tmp_dynamic/Subset18/reads.bai tmp_dynamic/Subset18/peaks.bed mm\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset19/names_atac.txt data/bams tmp_dynamic/Subset19/reads.bam tmp_dynamic/Subset19/reads.bai tmp_dynamic/Subset19/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset20/names_atac.txt data/bams tmp_dynamic/Subset20/reads.bam tmp_dynamic/Subset20/reads.bai tmp_dynamic/Subset20/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset21/names_atac.txt data/bams tmp_dynamic/Subset21/reads.bam tmp_dynamic/Subset21/reads.bai tmp_dynamic/Subset21/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset22/names_atac.txt data/bams tmp_dynamic/Subset22/reads.bam tmp_dynamic/Subset22/reads.bai tmp_dynamic/Subset22/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset23/names_atac.txt data/bams tmp_dynamic/Subset23/reads.bam tmp_dynamic/Subset23/reads.bai tmp_dynamic/Subset23/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset24/names_atac.txt data/bams tmp_dynamic/Subset24/reads.bam tmp_dynamic/Subset24/reads.bai tmp_dynamic/Subset24/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset25/names_atac.txt data/bams tmp_dynamic/Subset25/reads.bam tmp_dynamic/Subset25/reads.bai tmp_dynamic/Subset25/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset26/names_atac.txt data/bams tmp_dynamic/Subset26/reads.bam tmp_dynamic/Subset26/reads.bai tmp_dynamic/Subset26/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset27/names_atac.txt data/bams tmp_dynamic/Subset27/reads.bam tmp_dynamic/Subset27/reads.bai tmp_dynamic/Subset27/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset28/names_atac.txt data/bams tmp_dynamic/Subset28/reads.bam tmp_dynamic/Subset28/reads.bai tmp_dynamic/Subset28/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset29/names_atac.txt data/bams tmp_dynamic/Subset29/reads.bam tmp_dynamic/Subset29/reads.bai tmp_dynamic/Subset29/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset30/names_atac.txt data/bams tmp_dynamic/Subset30/reads.bam tmp_dynamic/Subset30/reads.bai tmp_dynamic/Subset30/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset31/names_atac.txt data/bams tmp_dynamic/Subset31/reads.bam tmp_dynamic/Subset31/reads.bai tmp_dynamic/Subset31/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset32/names_atac.txt data/bams tmp_dynamic/Subset32/reads.bam tmp_dynamic/Subset32/reads.bai tmp_dynamic/Subset32/peaks.bed mm\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:09: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset12/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset12/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:09: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:09: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:09: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:09: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:09: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:12:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:18:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:21:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:24:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25: 5480130 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25: #1 tag size is determined as 48 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25: #1 tag size = 48.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25: #1  total tags in treatment: 5480130 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:25: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:36: access pq hash for 19033088 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:01: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:02: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:02: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:02: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset33/names_atac.txt data/bams tmp_dynamic/Subset33/reads.bam tmp_dynamic/Subset33/reads.bai tmp_dynamic/Subset33/peaks.bed mm\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:10: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset18/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset18/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:10: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:10: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:10: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:10: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:10: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:13:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:19:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:22:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27: 5646622 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27: #1  total tags in treatment: 5646622 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:27: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:38: access pq hash for 19464359 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:38: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:05: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:05: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:05: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:05: Done! \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset34/names_atac.txt data/bams tmp_dynamic/Subset34/reads.bam tmp_dynamic/Subset34/reads.bai tmp_dynamic/Subset34/peaks.bed mm\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:10: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset10/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset10/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:10: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:10: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:10: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:10: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:10: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:13:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:19:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:22:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: 5954426 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #1  total tags in treatment: 5954426 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:28: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:39: access pq hash for 20636960 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:39: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:07: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:07: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:07: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:07: Done! \n",
      "\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:11: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset9/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset9/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:11: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:11: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:11: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:11: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:11: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:17:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:20:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:22:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: 5997899 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #1 tag size is determined as 45 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #1 tag size = 45.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #1  total tags in treatment: 5997899 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:28: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:40: access pq hash for 20801696 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:40: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:07: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:07: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:07: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:07: Done! \n",
      "\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:10: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset21/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset21/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:10: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:10: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:10: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:11: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:11: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:17:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:20:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:23:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:26:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: 5776345 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #1 tag size is determined as 44 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #1 tag size = 44.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #1  total tags in treatment: 5776345 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:28: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:39: access pq hash for 19932247 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:39: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:07: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:07: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:08: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:08: Done! \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset35/names_atac.txt data/bams tmp_dynamic/Subset35/reads.bam tmp_dynamic/Subset35/reads.bai tmp_dynamic/Subset35/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset36/names_atac.txt data/bams tmp_dynamic/Subset36/reads.bam tmp_dynamic/Subset36/reads.bai tmp_dynamic/Subset36/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset37/names_atac.txt data/bams tmp_dynamic/Subset37/reads.bam tmp_dynamic/Subset37/reads.bai tmp_dynamic/Subset37/peaks.bed mm\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:10: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset8/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset8/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:10: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:10: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:10: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:10: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:10: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:13:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:19:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:22:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: 6060104 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #1 tag size is determined as 43 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #1 tag size = 43.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #1  total tags in treatment: 6060104 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:28: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:40: access pq hash for 21014955 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:40: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:08: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:08: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:08: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:08: Done! \n",
      "\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset15/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset15/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:14: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:17:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:20:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:23:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:26:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:29:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: 5592636 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #1  total tags in treatment: 5592636 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:31: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:42: access pq hash for 19302762 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:42: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:08: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:08: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:09: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:09: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset38/names_atac.txt data/bams tmp_dynamic/Subset38/reads.bam tmp_dynamic/Subset38/reads.bai tmp_dynamic/Subset38/peaks.bed mm\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:12: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset4/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset4/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:12: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:12: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:12: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:12: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:12: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:18:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:21:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:24:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: 5862625 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #1 tag size is determined as 46 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #1 tag size = 46.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #1  total tags in treatment: 5862625 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #2 Skipped... \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:30: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:41: access pq hash for 20604899 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:41: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:08: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:09: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:09: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:09: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset39/names_atac.txt data/bams tmp_dynamic/Subset39/reads.bam tmp_dynamic/Subset39/reads.bai tmp_dynamic/Subset39/peaks.bed mm\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset17/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset17/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:15: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:18:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:21:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:24:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: 5556060 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1  total tags in treatment: 5556060 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:32: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:43: access pq hash for 19170272 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:43: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:09: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:09: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:09: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:09: Done! \n",
      "\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset19/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset19/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:15: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:18:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:21:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:24:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: 5608527 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1  total tags in treatment: 5608527 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:32: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:43: access pq hash for 19356912 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:43: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:09: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:09: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:09: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:09: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset40/names_atac.txt data/bams tmp_dynamic/Subset40/reads.bam tmp_dynamic/Subset40/reads.bai tmp_dynamic/Subset40/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset41/names_atac.txt data/bams tmp_dynamic/Subset41/reads.bam tmp_dynamic/Subset41/reads.bai tmp_dynamic/Subset41/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset42/names_atac.txt data/bams tmp_dynamic/Subset42/reads.bam tmp_dynamic/Subset42/reads.bai tmp_dynamic/Subset42/peaks.bed mm\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:12: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset5/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset5/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:12: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:12: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:12: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:12: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:12: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:18:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:21:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:24:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: 5953047 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #1  total tags in treatment: 5953047 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:30: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:42: access pq hash for 20764610 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:42: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:09: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:09: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: Done! \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: \r\n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset13/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\r\n",
      "# ARGUMENTS LIST:\r\n",
      "# name = 04\r\n",
      "# format = AUTO\r\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset13/reads.bam']\r\n",
      "# control file = None\r\n",
      "# effective genome size = 1.87e+09\r\n",
      "# band width = 300\r\n",
      "# model fold = [5, 50]\r\n",
      "# qvalue cutoff = 5.00e-02\r\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\r\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\r\n",
      "# Larger dataset will be scaled towards smaller dataset.\r\n",
      "# Range for calculating regional lambda is: 10000 bps\r\n",
      "# Broad region calling is off\r\n",
      "# Paired-End mode is off\r\n",
      "# Searching for subpeak summits is on\r\n",
      " \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: #1 read tag files... \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: #1 read treatment tags... \r\n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:15: Testing format BAM \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: Detected format is: BAM \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: * Input file is gzipped. \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:18:  1000000 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:21:  2000000 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:24:  3000000 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27:  4000000 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30:  5000000 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: 5593670 reads have been read. \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 tag size is determined as 48 bps \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 tag size = 48.0 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1  total tags in treatment: 5593670 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 finished! \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Build Peak Model... \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Skipped... \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Use 150 as fragment length \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Call peaks... \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Going to call summits inside each peak ... \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Pre-compute pvalue-qvalue table... \r\n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:32: Start to calculate pvalue stat... \r\n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:43: access pq hash for 19368872 times \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:43: #3 Call peaks for each chromosome... \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:23:09: #4 Write output xls file... 04_peaks.xls \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: #4 Write summits bed file... 04_summits.bed \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: Done! \r\n",
      "\r\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: \r\n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset14/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\r\n",
      "# ARGUMENTS LIST:\r\n",
      "# name = 04\r\n",
      "# format = AUTO\r\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset14/reads.bam']\r\n",
      "# control file = None\r\n",
      "# effective genome size = 1.87e+09\r\n",
      "# band width = 300\r\n",
      "# model fold = [5, 50]\r\n",
      "# qvalue cutoff = 5.00e-02\r\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\r\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\r\n",
      "# Larger dataset will be scaled towards smaller dataset.\r\n",
      "# Range for calculating regional lambda is: 10000 bps\r\n",
      "# Broad region calling is off\r\n",
      "# Paired-End mode is off\r\n",
      "# Searching for subpeak summits is on\r\n",
      " \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: #1 read tag files... \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: #1 read treatment tags... \r\n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:16: Testing format BAM \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: Detected format is: BAM \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: * Input file is gzipped. \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:19:  1000000 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:22:  2000000 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25:  3000000 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27:  4000000 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30:  5000000 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: 5638322 reads have been read. \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 tag size is determined as 48 bps \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 tag size = 48.0 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1  total tags in treatment: 5638322 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 finished! \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Build Peak Model... \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Skipped... \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Use 150 as fragment length \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Call peaks... \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Going to call summits inside each peak ... \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Pre-compute pvalue-qvalue table... \r\n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:32: Start to calculate pvalue stat... \r\n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:43: access pq hash for 19476440 times \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:43: #3 Call peaks for each chromosome... \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:23:09: #4 Write output xls file... 04_peaks.xls \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: #4 Write summits bed file... 04_summits.bed \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: Done! \r\n",
      "\r\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: \r\n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset16/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\r\n",
      "# ARGUMENTS LIST:\r\n",
      "# name = 04\r\n",
      "# format = AUTO\r\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset16/reads.bam']\r\n",
      "# control file = None\r\n",
      "# effective genome size = 1.87e+09\r\n",
      "# band width = 300\r\n",
      "# model fold = [5, 50]\r\n",
      "# qvalue cutoff = 5.00e-02\r\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\r\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\r\n",
      "# Larger dataset will be scaled towards smaller dataset.\r\n",
      "# Range for calculating regional lambda is: 10000 bps\r\n",
      "# Broad region calling is off\r\n",
      "# Paired-End mode is off\r\n",
      "# Searching for subpeak summits is on\r\n",
      " \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: #1 read tag files... \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: #1 read treatment tags... \r\n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:15: Testing format BAM \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: Detected format is: BAM \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: * Input file is gzipped. \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:18:  1000000 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:21:  2000000 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:24:  3000000 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27:  4000000 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30:  5000000 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: 5609610 reads have been read. \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 tag size is determined as 47 bps \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 tag size = 47.0 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1  total tags in treatment: 5609610 \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 finished! \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Build Peak Model... \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Skipped... \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Use 150 as fragment length \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Call peaks... \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Going to call summits inside each peak ... \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Pre-compute pvalue-qvalue table... \r\n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:32: Start to calculate pvalue stat... \r\n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:43: access pq hash for 19349731 times \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:43: #3 Call peaks for each chromosome... \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:23:09: #4 Write output xls file... 04_peaks.xls \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: #4 Write summits bed file... 04_summits.bed \r\n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: Done! \r\n",
      "\r\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset43/names_atac.txt data/bams tmp_dynamic/Subset43/reads.bam tmp_dynamic/Subset43/reads.bai tmp_dynamic/Subset43/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset44/names_atac.txt data/bams tmp_dynamic/Subset44/reads.bam tmp_dynamic/Subset44/reads.bai tmp_dynamic/Subset44/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset45/names_atac.txt data/bams tmp_dynamic/Subset45/reads.bam tmp_dynamic/Subset45/reads.bai tmp_dynamic/Subset45/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset46/names_atac.txt data/bams tmp_dynamic/Subset46/reads.bam tmp_dynamic/Subset46/reads.bai tmp_dynamic/Subset46/peaks.bed mm\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset11/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset11/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:15: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:18:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:21:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:24:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: 5693406 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 tag size is determined as 48 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 tag size = 48.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1  total tags in treatment: 5693406 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:32: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:43: access pq hash for 19756505 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:43: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: Done! \n",
      "\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:13: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset22/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset22/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:13: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:13: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:13: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:13: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:13: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:19:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:22:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: 5831941 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #1 tag size is determined as 44 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #1 tag size = 44.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #1  total tags in treatment: 5831941 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:31: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:42: access pq hash for 20117311 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:42: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:11: Done! \n",
      "\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:12: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset24/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset24/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:12: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:12: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:12: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:12: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:12: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:18:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:21:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:24:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: 6073874 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #1  total tags in treatment: 6073874 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:30: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:42: access pq hash for 21166597 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:42: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:10: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:11: Done! \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset47/names_atac.txt data/bams tmp_dynamic/Subset47/reads.bam tmp_dynamic/Subset47/reads.bai tmp_dynamic/Subset47/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset48/names_atac.txt data/bams tmp_dynamic/Subset48/reads.bam tmp_dynamic/Subset48/reads.bai tmp_dynamic/Subset48/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset49/names_atac.txt data/bams tmp_dynamic/Subset49/reads.bam tmp_dynamic/Subset49/reads.bai tmp_dynamic/Subset49/peaks.bed mm\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset20/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset20/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:16: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:19:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:22:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: 5635939 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #1  total tags in treatment: 5635939 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:33: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:44: access pq hash for 19453570 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:44: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:11: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:11: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:11: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:11: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset50/names_atac.txt data/bams tmp_dynamic/Subset50/reads.bam tmp_dynamic/Subset50/reads.bai tmp_dynamic/Subset50/peaks.bed mm\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:13: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset26/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset26/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:13: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:13: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:13: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:13: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:13: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:19:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:22:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: 6176810 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1  total tags in treatment: 6176810 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:32: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:44: access pq hash for 21475728 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:44: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:12: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:12: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:13: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:13: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset51/names_atac.txt data/bams tmp_dynamic/Subset51/reads.bam tmp_dynamic/Subset51/reads.bai tmp_dynamic/Subset51/peaks.bed mm\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset25/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset25/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:14: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:17:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:20:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:23:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:26:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:29:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: 6088286 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1  total tags in treatment: 6088286 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:32: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:44: access pq hash for 21207841 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:44: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:13: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:13: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:14: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:14: Done! \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset7/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset7/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:16: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:19:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:22:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: 5995616 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #1  total tags in treatment: 5995616 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:34: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:45: access pq hash for 20848721 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:45: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:13: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:14: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:14: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:14: Done! \n",
      "\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset1/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset1/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:16: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:19:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:22:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: 5982163 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #1 tag size is determined as 43 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #1 tag size = 43.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #1  total tags in treatment: 5982163 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:34: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:46: access pq hash for 20877110 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:46: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:13: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:14: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:14: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:14: Done! \n",
      "\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:11: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset2/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset2/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:11: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:11: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:11: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:11: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:11: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:17:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:20:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:23:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:26:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:29:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: 6506198 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #1 tag size is determined as 46 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #1 tag size = 46.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #1  total tags in treatment: 6506198 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:30: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:43: access pq hash for 22366598 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:43: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:13: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:14: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:14: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:14: Done! \n",
      "\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset6/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset6/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:16: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:19:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:22:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: 6012802 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #1  total tags in treatment: 6012802 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:34: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:45: access pq hash for 20915335 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:45: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:13: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:14: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:14: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:14: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset52/names_atac.txt data/bams tmp_dynamic/Subset52/reads.bam tmp_dynamic/Subset52/reads.bai tmp_dynamic/Subset52/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset53/names_atac.txt data/bams tmp_dynamic/Subset53/reads.bam tmp_dynamic/Subset53/reads.bai tmp_dynamic/Subset53/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset54/names_atac.txt data/bams tmp_dynamic/Subset54/reads.bam tmp_dynamic/Subset54/reads.bai tmp_dynamic/Subset54/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset55/names_atac.txt data/bams tmp_dynamic/Subset55/reads.bam tmp_dynamic/Subset55/reads.bai tmp_dynamic/Subset55/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset56/names_atac.txt data/bams tmp_dynamic/Subset56/reads.bam tmp_dynamic/Subset56/reads.bai tmp_dynamic/Subset56/peaks.bed mm\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:13: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset28/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset28/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:13: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:13: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:13: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:13: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:13: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:19:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:22:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32: 6410961 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #1 tag size is determined as 46 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #1 tag size = 46.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #1  total tags in treatment: 6410961 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:33: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:45: access pq hash for 22174981 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:45: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:15: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:15: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:15: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:15: Done! \n",
      "\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset27/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset27/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:14: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:17:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:20:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:23:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:26:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:29:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: 6309447 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #1  total tags in treatment: 6309447 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #2 Build Peak Model... \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:33: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:46: access pq hash for 21874230 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:46: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:15: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:15: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:16: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:16: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset57/names_atac.txt data/bams tmp_dynamic/Subset57/reads.bam tmp_dynamic/Subset57/reads.bai tmp_dynamic/Subset57/peaks.bed mm\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset29/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset29/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:14: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:17:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:20:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:23:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:26:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:29:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: 6493360 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #1 tag size is determined as 46 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #1 tag size = 46.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #1  total tags in treatment: 6493360 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:34: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:47: access pq hash for 22398274 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:47: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:15: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:16: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:16: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:16: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset58/names_atac.txt data/bams tmp_dynamic/Subset58/reads.bam tmp_dynamic/Subset58/reads.bai tmp_dynamic/Subset58/peaks.bed mm\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset23/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset23/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:14: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:14: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:17:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:20:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:23:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:26:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:29:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: 6345600 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #1 tag size is determined as 44 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #1 tag size = 44.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #1  total tags in treatment: 6345600 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:33: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:46: access pq hash for 21813388 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:46: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:15: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:16: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:16: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:16: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset59/names_atac.txt data/bams tmp_dynamic/Subset59/reads.bam tmp_dynamic/Subset59/reads.bai tmp_dynamic/Subset59/peaks.bed mm\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset60/names_atac.txt data/bams tmp_dynamic/Subset60/reads.bam tmp_dynamic/Subset60/reads.bai tmp_dynamic/Subset60/peaks.bed mm\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset30/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset30/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:15: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:18:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:21:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:24:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: 6550440 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #1 tag size is determined as 46 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #1 tag size = 46.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #1  total tags in treatment: 6550440 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:34: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:47: access pq hash for 22580329 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:47: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:17: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:18: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:18: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:18: Done! \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset61/names_atac.txt data/bams tmp_dynamic/Subset61/reads.bam tmp_dynamic/Subset61/reads.bai tmp_dynamic/Subset61/peaks.bed mm\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset31/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset31/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:16: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:16: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:19:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:22:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:25:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:28:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:31:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:34:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: 6883805 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #1  total tags in treatment: 6883805 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:36: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:50: access pq hash for 23622342 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:50: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:21: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:21: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:21: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:21: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_dynamic/Subset62/expression.tsv.gz tmp_dynamic/Subset62/names_atac0.txt tmp_dynamic/Subset62/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset1/reads.bam tmp_dynamic/Subset1/reads.bai tmp_dynamic/Subset1/peaks.bed tmp_dynamic/Subset1/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset32/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset32/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:15: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:15: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:18:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:21:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:24:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:27:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:30:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:33:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: 7196917 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #1  total tags in treatment: 7196917 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:36: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:36: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:51: access pq hash for 24609091 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:51: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:23: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:23: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:23: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:24: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset2/reads.bam tmp_dynamic/Subset2/reads.bai tmp_dynamic/Subset2/peaks.bed tmp_dynamic/Subset2/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:22:17: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset3/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset3/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:17: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:17: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:17: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:17: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:17: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:20:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:23:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:26:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:29:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:32:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:35:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:38:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:38: 7263327 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:38: #1 tag size is determined as 48 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:38: #1 tag size = 48.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:38: #1  total tags in treatment: 7263327 \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:38: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:38: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:38: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:38: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:38: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:38: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:38: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:38: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:38: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:22:53: access pq hash for 25248804 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:22:53: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:28: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:28: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:29: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:23:29: Done! \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset4/reads.bam tmp_dynamic/Subset4/reads.bai tmp_dynamic/Subset4/peaks.bed tmp_dynamic/Subset4/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:29: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset33/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset33/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:29: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:29: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:29: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:29: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:29: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:32:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:35:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:38:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:41:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:47:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:50:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:51: 7323259 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:51: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:51: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:51: #1  total tags in treatment: 7323259 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:51: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:51: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:51: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:51: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:51: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:51: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:51: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:51: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:51: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:05: access pq hash for 25005054 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:38: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:38: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:38: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:38: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset5/reads.bam tmp_dynamic/Subset5/reads.bai tmp_dynamic/Subset5/peaks.bed tmp_dynamic/Subset5/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset43/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset43/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:37: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:40:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:43:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:49:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:52:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:55:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: 6577764 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #1 tag size is determined as 48 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #1 tag size = 48.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #1  total tags in treatment: 6577764 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:57: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:10: access pq hash for 22562115 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:10: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:39: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:39: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:39: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:40: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset6/reads.bam tmp_dynamic/Subset6/reads.bai tmp_dynamic/Subset6/peaks.bed tmp_dynamic/Subset6/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset44/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset44/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:37: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:40:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:43:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:49:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:52:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:55:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: 6700588 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #1 tag size is determined as 48 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #1 tag size = 48.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #1  total tags in treatment: 6700588 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:57: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:11: access pq hash for 22985804 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:11: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:41: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:41: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:41: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:42: Done! \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset7/reads.bam tmp_dynamic/Subset7/reads.bai tmp_dynamic/Subset7/peaks.bed tmp_dynamic/Subset7/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset40/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset40/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:37: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:40:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:43:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:49:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:52:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:55:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: 6866846 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #1  total tags in treatment: 6866846 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:58: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:11: access pq hash for 23484069 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:11: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:41: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:42: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:42: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:42: Done! \n",
      "\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset41/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset41/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:37: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:40:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:43:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:49:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:52:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:55:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: 6791722 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #1  total tags in treatment: 6791722 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:57: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:11: access pq hash for 23241666 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:11: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:41: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:42: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:42: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:42: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset8/reads.bam tmp_dynamic/Subset8/reads.bai tmp_dynamic/Subset8/peaks.bed tmp_dynamic/Subset8/footprints.bed\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset9/reads.bam tmp_dynamic/Subset9/reads.bai tmp_dynamic/Subset9/peaks.bed tmp_dynamic/Subset9/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:36: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset36/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset36/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:36: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:36: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:36: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:36: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:36: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:39:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:42:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:45:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:48:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:51:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:54:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: 7056997 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #1 tag size is determined as 48 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #1 tag size = 48.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #1  total tags in treatment: 7056997 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:57: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:11: access pq hash for 24064997 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:11: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:42: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:43: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:43: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:43: Done! \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset10/reads.bam tmp_dynamic/Subset10/reads.bai tmp_dynamic/Subset10/peaks.bed tmp_dynamic/Subset10/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset39/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset39/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:37: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:40:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:43:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:49:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:52:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:55:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: 6897847 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #1  total tags in treatment: 6897847 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:58: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:12: access pq hash for 23586534 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:12: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:43: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:44: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:44: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:44: Done! \n",
      "\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:38: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset48/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset48/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:38: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:38: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:38: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:38: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:38: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:41:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:47:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:50:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:53:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:56:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: 6913962 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #1  total tags in treatment: 6913962 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:58: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:12: access pq hash for 23724055 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:12: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:43: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:44: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:44: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:44: Done! \n",
      "\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:45: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset61/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset61/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:45: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:45: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:45: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:45: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:45: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:48:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:51:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:54:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:03: 5882505 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:03: #1 tag size is determined as 43 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:03: #1 tag size = 43.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:03: #1  total tags in treatment: 5882505 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:03: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:03: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:03: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:03: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:03: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:03: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:03: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:03: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:03: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:16: access pq hash for 20605268 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:16: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:43: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:44: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:44: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:44: Done! \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset46/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset46/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:37: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:37: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:40:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:43:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:49:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:52:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:55:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: 6940853 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #1 tag size is determined as 48 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #1 tag size = 48.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #1  total tags in treatment: 6940853 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:58: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:12: access pq hash for 23797011 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:12: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:44: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:44: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:44: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:44: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset11/reads.bam tmp_dynamic/Subset11/reads.bai tmp_dynamic/Subset11/peaks.bed tmp_dynamic/Subset11/footprints.bed\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset12/reads.bam tmp_dynamic/Subset12/reads.bai tmp_dynamic/Subset12/peaks.bed tmp_dynamic/Subset12/footprints.bed\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset13/reads.bam tmp_dynamic/Subset13/reads.bai tmp_dynamic/Subset13/peaks.bed tmp_dynamic/Subset13/footprints.bed\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset14/reads.bam tmp_dynamic/Subset14/reads.bai tmp_dynamic/Subset14/peaks.bed tmp_dynamic/Subset14/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:36: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset34/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset34/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:36: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:36: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:36: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:36: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:36: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:39:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:42:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:45:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:48:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:51:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:54:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: 7228127 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #1 tag size is determined as 48 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #1 tag size = 48.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #1  total tags in treatment: 7228127 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:58: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:12: access pq hash for 24692368 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:12: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:44: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:45: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:45: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:45: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset15/reads.bam tmp_dynamic/Subset15/reads.bai tmp_dynamic/Subset15/peaks.bed tmp_dynamic/Subset15/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:40: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset37/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset37/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:40: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:40: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:40: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:40: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:40: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:43:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:49:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:52:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:55:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01: 6999390 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01: #1 tag size is determined as 48 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01: #1 tag size = 48.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01: #1  total tags in treatment: 6999390 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01: #2 Build Peak Model... \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO  @ Fri, 28 Oct 2022 20:25:01: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:01: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:15: access pq hash for 23896909 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:15: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:47: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:47: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:47: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:47: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset16/reads.bam tmp_dynamic/Subset16/reads.bai tmp_dynamic/Subset16/peaks.bed tmp_dynamic/Subset16/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset42/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset42/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:46: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:49:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:52:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:55:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:04:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: 6687004 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #1  total tags in treatment: 6687004 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:06: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:19: access pq hash for 22920070 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:19: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:50: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:50: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:50: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:50: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset17/reads.bam tmp_dynamic/Subset17/reads.bai tmp_dynamic/Subset17/peaks.bed tmp_dynamic/Subset17/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset47/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset47/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:46: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:49:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:52:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:55:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:04:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: 6888574 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #1 tag size is determined as 48 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #1 tag size = 48.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #1  total tags in treatment: 6888574 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:06: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:20: access pq hash for 23641959 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:20: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:50: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:51: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:51: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:51: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset18/reads.bam tmp_dynamic/Subset18/reads.bai tmp_dynamic/Subset18/peaks.bed tmp_dynamic/Subset18/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset38/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset38/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:46: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:49:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:52:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:55:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:04:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: 6927058 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #1  total tags in treatment: 6927058 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:07: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:20: access pq hash for 23664818 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:20: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:51: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:51: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:51: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:51: Done! \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset52/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset52/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:44: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:47:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:50:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:53:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:56:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:59:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:02:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: 6919179 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #1  total tags in treatment: 6919179 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:05: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:19: access pq hash for 23826646 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:19: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:51: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:51: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:52: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:52: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset19/reads.bam tmp_dynamic/Subset19/reads.bai tmp_dynamic/Subset19/peaks.bed tmp_dynamic/Subset19/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset50/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset50/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:44: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:47:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:50:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:53:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:56:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:59:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:02:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: 7057830 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #1  total tags in treatment: 7057830 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:05: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:19: access pq hash for 24220368 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:19: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:51: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:52: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:52: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:52: Done! \n",
      "\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset45/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset45/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:46: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:49:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:52:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:55:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:04:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: 6908267 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #1 tag size is determined as 48 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #1 tag size = 48.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #1  total tags in treatment: 6908267 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:06: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:20: access pq hash for 23653217 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:20: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:51: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:52: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:52: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:52: Done! \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset20/reads.bam tmp_dynamic/Subset20/reads.bai tmp_dynamic/Subset20/peaks.bed tmp_dynamic/Subset20/footprints.bed\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset21/reads.bam tmp_dynamic/Subset21/reads.bai tmp_dynamic/Subset21/peaks.bed tmp_dynamic/Subset21/footprints.bed\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset22/reads.bam tmp_dynamic/Subset22/reads.bai tmp_dynamic/Subset22/peaks.bed tmp_dynamic/Subset22/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset51/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset51/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:44: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:47:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:50:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:53:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:56:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:59:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:02:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: 7066690 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #1  total tags in treatment: 7066690 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:05: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:19: access pq hash for 24259039 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:19: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:52: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:52: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:52: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:53: Done! \n",
      "\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset35/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset35/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:46: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:49:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:52:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:55:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:04:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: 6965265 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #1 tag size is determined as 48 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #1 tag size = 48.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #1  total tags in treatment: 6965265 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:07: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:21: access pq hash for 23804739 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:21: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:52: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:52: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:52: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:53: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset23/reads.bam tmp_dynamic/Subset23/reads.bai tmp_dynamic/Subset23/peaks.bed tmp_dynamic/Subset23/footprints.bed\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset24/reads.bam tmp_dynamic/Subset24/reads.bai tmp_dynamic/Subset24/peaks.bed tmp_dynamic/Subset24/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:43: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset57/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset57/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:43: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:43: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:43: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:43: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:43: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:50:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:53:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:56:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:59:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:02:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: 7053920 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #1  total tags in treatment: 7053920 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:05: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:18: access pq hash for 24391733 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:19: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:53: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:53: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:53: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:53: Done! \n",
      "\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset54/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset54/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:46: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:49:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:52:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:55:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:00:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:03:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: 6956083 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #1  total tags in treatment: 6956083 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #2 Build Peak Model... \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:06: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:20: access pq hash for 23981554 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:20: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:53: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:53: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:53: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:53: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset25/reads.bam tmp_dynamic/Subset25/reads.bai tmp_dynamic/Subset25/peaks.bed tmp_dynamic/Subset25/footprints.bed\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset26/reads.bam tmp_dynamic/Subset26/reads.bai tmp_dynamic/Subset26/peaks.bed tmp_dynamic/Subset26/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset49/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset49/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:46: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:49:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:52:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:55:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:04:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: 7027069 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #1  total tags in treatment: 7027069 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:07: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:21: access pq hash for 24099675 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:21: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:55: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:55: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:55: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:55: Done! \n",
      "\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:45: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset55/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset55/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:45: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:45: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:45: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:45: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:45: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:48:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:51:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:54:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:57:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:00:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:03:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: 7097428 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #1  total tags in treatment: 7097428 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:07: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:21: access pq hash for 24479490 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:21: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:55: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:55: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:55: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:55: Done! \n",
      "\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset53/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset53/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:46: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:49:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:52:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:55:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:04:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: 6967226 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #1  total tags in treatment: 6967226 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #2 Build Peak Model... \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:07: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:21: access pq hash for 23984044 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:21: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:55: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:55: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:55: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:55: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset27/reads.bam tmp_dynamic/Subset27/reads.bai tmp_dynamic/Subset27/peaks.bed tmp_dynamic/Subset27/footprints.bed\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset28/reads.bam tmp_dynamic/Subset28/reads.bai tmp_dynamic/Subset28/peaks.bed tmp_dynamic/Subset28/footprints.bed\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset29/reads.bam tmp_dynamic/Subset29/reads.bai tmp_dynamic/Subset29/peaks.bed tmp_dynamic/Subset29/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:43: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset60/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset60/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:43: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:43: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:43: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:43: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:43: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:49:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:52:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:55:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:04:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: 7237905 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #1  total tags in treatment: 7237905 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:05: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:20: access pq hash for 25098079 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:20: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:55: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:56: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:56: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:56: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset30/reads.bam tmp_dynamic/Subset30/reads.bai tmp_dynamic/Subset30/peaks.bed tmp_dynamic/Subset30/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset56/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset56/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:46: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:46: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:49:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:52:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:55:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:58:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:01:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:04:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:07: 7058668 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #1  total tags in treatment: 7058668 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:08: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:21: access pq hash for 24396273 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:21: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:56: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:57: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:57: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:57: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset31/reads.bam tmp_dynamic/Subset31/reads.bai tmp_dynamic/Subset31/peaks.bed tmp_dynamic/Subset31/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset59/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset59/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:44: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:44: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:47:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:50:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:53:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:56:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:59:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:03:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: 7272088 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #1 tag size is determined as 48 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #1 tag size = 48.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #1  total tags in treatment: 7272088 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:06: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:06: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:20: access pq hash for 25153620 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:20: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:57: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:57: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:57: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:57: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset32/reads.bam tmp_dynamic/Subset32/reads.bai tmp_dynamic/Subset32/peaks.bed tmp_dynamic/Subset32/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 20:24:47: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset58/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset58/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:47: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:47: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:24:47: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:47: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:47: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:50:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:53:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:56:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:24:59:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:02:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:05:  6000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08:  7000000 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: 7208662 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #1  total tags in treatment: 7208662 \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:08: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:08: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 20:25:23: access pq hash for 24910965 times \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:23: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:59: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:59: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:59: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 20:25:59: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset33/reads.bam tmp_dynamic/Subset33/reads.bai tmp_dynamic/Subset33/peaks.bed tmp_dynamic/Subset33/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset34/reads.bam tmp_dynamic/Subset34/reads.bai tmp_dynamic/Subset34/peaks.bed tmp_dynamic/Subset34/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset35/reads.bam tmp_dynamic/Subset35/reads.bai tmp_dynamic/Subset35/peaks.bed tmp_dynamic/Subset35/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset36/reads.bam tmp_dynamic/Subset36/reads.bai tmp_dynamic/Subset36/peaks.bed tmp_dynamic/Subset36/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset37/reads.bam tmp_dynamic/Subset37/reads.bai tmp_dynamic/Subset37/peaks.bed tmp_dynamic/Subset37/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset38/reads.bam tmp_dynamic/Subset38/reads.bai tmp_dynamic/Subset38/peaks.bed tmp_dynamic/Subset38/footprints.bed\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset39/reads.bam tmp_dynamic/Subset39/reads.bai tmp_dynamic/Subset39/peaks.bed tmp_dynamic/Subset39/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset40/reads.bam tmp_dynamic/Subset40/reads.bai tmp_dynamic/Subset40/peaks.bed tmp_dynamic/Subset40/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset41/reads.bam tmp_dynamic/Subset41/reads.bai tmp_dynamic/Subset41/peaks.bed tmp_dynamic/Subset41/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset42/reads.bam tmp_dynamic/Subset42/reads.bai tmp_dynamic/Subset42/peaks.bed tmp_dynamic/Subset42/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset43/reads.bam tmp_dynamic/Subset43/reads.bai tmp_dynamic/Subset43/peaks.bed tmp_dynamic/Subset43/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset44/reads.bam tmp_dynamic/Subset44/reads.bai tmp_dynamic/Subset44/peaks.bed tmp_dynamic/Subset44/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset45/reads.bam tmp_dynamic/Subset45/reads.bai tmp_dynamic/Subset45/peaks.bed tmp_dynamic/Subset45/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset46/reads.bam tmp_dynamic/Subset46/reads.bai tmp_dynamic/Subset46/peaks.bed tmp_dynamic/Subset46/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset47/reads.bam tmp_dynamic/Subset47/reads.bai tmp_dynamic/Subset47/peaks.bed tmp_dynamic/Subset47/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset48/reads.bam tmp_dynamic/Subset48/reads.bai tmp_dynamic/Subset48/peaks.bed tmp_dynamic/Subset48/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset49/reads.bam tmp_dynamic/Subset49/reads.bai tmp_dynamic/Subset49/peaks.bed tmp_dynamic/Subset49/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset50/reads.bam tmp_dynamic/Subset50/reads.bai tmp_dynamic/Subset50/peaks.bed tmp_dynamic/Subset50/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset51/reads.bam tmp_dynamic/Subset51/reads.bai tmp_dynamic/Subset51/peaks.bed tmp_dynamic/Subset51/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset52/reads.bam tmp_dynamic/Subset52/reads.bai tmp_dynamic/Subset52/peaks.bed tmp_dynamic/Subset52/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset53/reads.bam tmp_dynamic/Subset53/reads.bai tmp_dynamic/Subset53/peaks.bed tmp_dynamic/Subset53/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset54/reads.bam tmp_dynamic/Subset54/reads.bai tmp_dynamic/Subset54/peaks.bed tmp_dynamic/Subset54/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset55/reads.bam tmp_dynamic/Subset55/reads.bai tmp_dynamic/Subset55/peaks.bed tmp_dynamic/Subset55/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset56/reads.bam tmp_dynamic/Subset56/reads.bai tmp_dynamic/Subset56/peaks.bed tmp_dynamic/Subset56/footprints.bed\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset57/reads.bam tmp_dynamic/Subset57/reads.bai tmp_dynamic/Subset57/peaks.bed tmp_dynamic/Subset57/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset58/reads.bam tmp_dynamic/Subset58/reads.bai tmp_dynamic/Subset58/peaks.bed tmp_dynamic/Subset58/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset59/reads.bam tmp_dynamic/Subset59/reads.bai tmp_dynamic/Subset59/peaks.bed tmp_dynamic/Subset59/footprints.bed\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset60/reads.bam tmp_dynamic/Subset60/reads.bai tmp_dynamic/Subset60/peaks.bed tmp_dynamic/Subset60/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset61/reads.bam tmp_dynamic/Subset61/reads.bai tmp_dynamic/Subset61/peaks.bed tmp_dynamic/Subset61/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_dynamic/Subset62/names_atac.txt data/bams tmp_dynamic/Subset62/reads.bam tmp_dynamic/Subset62/reads.bai tmp_dynamic/Subset62/peaks.bed mm\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset1/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset1/expression.tsv.gz tmp_dynamic/Subset1/motifs.bed tmp_dynamic/Subset1/wellington.tsv.gz tmp_dynamic/Subset1/homer.tsv.gz\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 28 Oct 2022 21:25:38: \n",
      "# Command line: callpeak -t ../tmp_dynamic/Subset62/reads.bam -n 04 -g mm --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['../tmp_dynamic/Subset62/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 1.87e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:38: #1 read tag files... \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:38: #1 read treatment tags... \n",
      "DEBUG @ Fri, 28 Oct 2022 21:25:38: Testing format BAM \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:38: Detected format is: BAM \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:38: * Input file is gzipped. \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:42:  1000000 \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:45:  2000000 \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:48:  3000000 \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:51:  4000000 \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:54:  5000000 \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:57: 5898715 reads have been read. \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:57: #1 tag size is determined as 43 bps \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:57: #1 tag size = 43.0 \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:57: #1  total tags in treatment: 5898715 \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:57: #1 finished! \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:57: #2 Build Peak Model... \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:57: #2 Skipped... \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:57: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:57: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:57: #3 Call peaks... \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:57: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 28 Oct 2022 21:25:57: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 28 Oct 2022 21:25:57: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 28 Oct 2022 21:26:09: access pq hash for 20694968 times \n",
      "INFO  @ Fri, 28 Oct 2022 21:26:09: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 28 Oct 2022 21:26:38: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 28 Oct 2022 21:26:38: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 28 Oct 2022 21:26:38: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 28 Oct 2022 21:26:39: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset2/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset2/expression.tsv.gz tmp_dynamic/Subset2/motifs.bed tmp_dynamic/Subset2/wellington.tsv.gz tmp_dynamic/Subset2/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4368\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4368\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "ls: cannot access ../data/genome/preparsed//genomer.*.cgbins: No such file or directory\n",
      "\tCould not find background files for 21 bp fragments\n",
      "\tBelow are the sizes that are already available prepared.\n",
      "\tHOMER will now create background files for 21 bp fragments\n",
      "\tTo CANCEL and rerun with a differet \"-size <#>\", hit <CTRL+C> now!\n",
      "\t\t5\n",
      "\t\t4\n",
      "\t\t3\n",
      "\t\t2\n",
      "\t\t1\n",
      "\tPreparsing genome for 21 bp fragments...(will probably take 1-5 min)\n",
      "\tUsing Custom Genome\n",
      "\tpreparse size set to 21\n",
      "\n",
      "\tNo reference file found - using random positions...\n",
      "\tOutput files will be placed in ../data/genome/preparsed//genomer.21.* ...\n",
      "\tBy default, HOMER will set the the preparsed Directory to be group read/write\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tChecking sequences from directory: ../data/genome\n",
      "\tExtracting sequences\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 301062 sequences from chr1\n",
      "\tExtracting 280617 sequences from chr2\n",
      "\tExtracting 248925 sequences from chr3\n",
      "\tExtracting 241274 sequences from chr4\n",
      "\tExtracting 234072 sequences from chr5\n",
      "\tExtracting 232558 sequences from chr6\n",
      "\tExtracting 222785 sequences from chr7\n",
      "\tExtracting 199043 sequences from chr8\n",
      "\tExtracting 193289 sequences from chr9\n",
      "\tExtracting 203113 sequences from chr10\n",
      "\tExtracting 184961 sequences from chr11\n",
      "\tExtracting 182689 sequences from chr12\n",
      "\tExtracting 185253 sequences from chr13\n",
      "\tExtracting 194226 sequences from chr14\n",
      "\tExtracting 161618 sequences from chr15\n",
      "\tExtracting 147286 sequences from chr16\n",
      "\tExtracting 146815 sequences from chr17\n",
      "\tExtracting 140402 sequences from chr18\n",
      "\tExtracting 97060 sequences from chr19\n",
      "\tExtracting 267558 sequences from chrX\n",
      "\tExtracting 142319 sequences from chrY\n",
      "\tExtracting 1109 sequences from chr5_JH584299_random\n",
      "\tExtracting 507 sequences from chrX_GL456233_random\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\tPotential Problem with Sequences (ignore if preparsing genome):\n",
      "\t\t1924251 of 4208157 removed because they had >70.00% Ns (i.e. masked repeats)\n",
      "\tDue to \thigh repeat content, consider using a non-repeat masked genome\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\n",
      "\tSequences processed:\n",
      "\t\tAuto detected maximum sequence length of 21 bp\n",
      "\t\t100000\n",
      "\t\t200000\n",
      "\t\t300000\n",
      "\t\t400000\n",
      "\t\t500000\n",
      "\t\t600000\n",
      "\t\t700000\n",
      "\t\t800000\n",
      "\t\t900000\n",
      "\t\t1000000\n",
      "\t\t1100000\n",
      "\t\t1200000\n",
      "\t\t1300000\n",
      "\t\t1400000\n",
      "\t\t1500000\n",
      "\t\t1600000\n",
      "\t\t1700000\n",
      "\t\t1800000\n",
      "\t\t1900000\n",
      "\t\t2000000\n",
      "\t\t2100000\n",
      "\t\t2200000\n",
      "\t\t2283906 total\n",
      "\n",
      "\tFrequency Bins: 0.01 0.02 0.03 0.04 0.05 0.06 0.08 0.1 0.12 0.14 0.16\n",
      "\tFreq\tBin\tCount\n",
      "\t0.01\t0\t1939111\n",
      "\t0.06\t5\t291460\n",
      "\t0.08\t6\t3559\n",
      "\t0.1\t7\t1597\n",
      "\t0.12\t8\t37287\n",
      "\t0.14\t9\t791\n",
      "\t0.16\t10\t6924\n",
      "\t11\t11\t3177\n",
      "\tFrequency Bins: 0.2 0.25 0.3 0.35 0.4 0.45 0.5 0.6 0.7 0.8\n",
      "\tFreq\tBin\tCount\n",
      "\t0.2\t0\t161200\n",
      "\t0.25\t1\t151131\n",
      "\t0.3\t2\t219109\n",
      "\t0.35\t3\t268835\n",
      "\t0.4\t4\t301829\n",
      "\t0.45\t5\t311540\n",
      "\t0.5\t6\t276211\n",
      "\t0.6\t7\t3\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4364\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4364\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "ls: cannot access ../data/genome/preparsed//genomer.*.cgbins: No such file or directory\n",
      "\tCould not find background files for 20 bp fragments\n",
      "\tBelow are the sizes that are already available prepared.\n",
      "\tHOMER will now create background files for 20 bp fragments\n",
      "\tTo CANCEL and rerun with a differet \"-size <#>\", hit <CTRL+C> now!\n",
      "\t\t5\n",
      "\t\t4\n",
      "\t\t3\n",
      "\t\t2\n",
      "\t\t1\n",
      "\tPreparsing genome for 20 bp fragments...(will probably take 1-5 min)\n",
      "\tUsing Custom Genome\n",
      "\tpreparse size set to 20\n",
      "\n",
      "\tNo reference file found - using random positions...\n",
      "\tOutput files will be placed in ../data/genome/preparsed//genomer.20.* ...\n",
      "\tBy default, HOMER will set the the preparsed Directory to be group read/write\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tChecking sequences from directory: ../data/genome\n",
      "\tExtracting sequences\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 307988 sequences from chr1\n",
      "\tExtracting 280596 sequences from chr2\n",
      "\tExtracting 255191 sequences from chr3\n",
      "\tExtracting 244083 sequences from chr4\n",
      "\tExtracting 238328 sequences from chr5\n",
      "\tExtracting 234446 sequences from chr6\n",
      "\tExtracting 227768 sequences from chr7\n",
      "\tExtracting 201366 sequences from chr8\n",
      "\tExtracting 192480 sequences from chr9\n",
      "\tExtracting 204978 sequences from chr10\n",
      "\tExtracting 203178 sequences from chr11\n",
      "\tExtracting 189910 sequences from chr12\n",
      "\tExtracting 191706 sequences from chr13\n",
      "\tExtracting 196232 sequences from chr14\n",
      "\tExtracting 160803 sequences from chr15\n",
      "\tExtracting 152976 sequences from chr16\n",
      "\tExtracting 148458 sequences from chr17\n",
      "\tExtracting 140779 sequences from chr18\n",
      "\tExtracting 96716 sequences from chr19\n",
      "\tExtracting 266491 sequences from chrX\n",
      "\tExtracting 144737 sequences from chrY\n",
      "\t!!Could not open file for 115702246 (.fa or .fa.masked)\n",
      "\tExtracting 1495 sequences from chr5_JH584299_random\n",
      "\tExtracting 520 sequences from chrX_GL456233_random\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\tPotential Problem with Sequences (ignore if preparsing genome):\n",
      "\t\t1962804 of 4280792 removed because they had >70.00% Ns (i.e. masked repeats)\n",
      "\tDue to \thigh repeat content, consider using a non-repeat masked genome\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\n",
      "\tSequences processed:\n",
      "\t\tAuto detected maximum sequence length of 21 bp\n",
      "\t\t100000\n",
      "\t\t200000\n",
      "\t\t300000\n",
      "\t\t400000\n",
      "\t\t500000\n",
      "\t\t600000\n",
      "\t\t700000\n",
      "\t\t800000\n",
      "\t\t900000\n",
      "\t\t1000000\n",
      "\t\t1100000\n",
      "\t\t1200000\n",
      "\t\t1300000\n",
      "\t\t1400000\n",
      "\t\t1500000\n",
      "\t\t1600000\n",
      "\t\t1700000\n",
      "\t\t1800000\n",
      "\t\t1900000\n",
      "\t\t2000000\n",
      "\t\t2100000\n",
      "\t\t2200000\n",
      "\t\t2300000\n",
      "\t\t2317988 total\n",
      "\n",
      "\tFrequency Bins: 0.01 0.02 0.03 0.04 0.05 0.06 0.08 0.1 0.12 0.14 0.16\n",
      "\tFreq\tBin\tCount\n",
      "\t0.01\t0\t1968065\n",
      "\t0.06\t5\t296310\n",
      "\t0.08\t6\t3708\n",
      "\t0.1\t7\t1648\n",
      "\t0.12\t8\t37551\n",
      "\t0.14\t9\t808\n",
      "\t0.16\t10\t6737\n",
      "\t11\t11\t3161\n",
      "\tFrequency Bins: 0.2 0.25 0.3 0.35 0.4 0.45 0.5 0.6 0.7 0.8\n",
      "\tFreq\tBin\tCount\n",
      "\t0.2\t0\t162558\n",
      "\t0.25\t1\t152479\n",
      "\t0.3\t2\t221391\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4368\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4368\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "ls: cannot access ../data/genome/preparsed//genomer.*.cgbins: No such file or directory\n",
      "\tCould not find background files for 20 bp fragments\n",
      "\tBelow are the sizes that are already available prepared.\n",
      "\tHOMER will now create background files for 20 bp fragments\n",
      "\tTo CANCEL and rerun with a differet \"-size <#>\", hit <CTRL+C> now!\n",
      "\t\t5\n",
      "\t\t4\n",
      "\t\t3\n",
      "\t\t2\n",
      "\t\t1\n",
      "\tPreparsing genome for 20 bp fragments...(will probably take 1-5 min)\n",
      "\tUsing Custom Genome\n",
      "\tpreparse size set to 20\n",
      "\n",
      "\tNo reference file found - using random positions...\n",
      "\tOutput files will be placed in ../data/genome/preparsed//genomer.20.* ...\n",
      "\tBy default, HOMER will set the the preparsed Directory to be group read/write\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tChecking sequences from directory: ../data/genome\n",
      "\tExtracting sequences\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 307988 sequences from chr1\n",
      "\tExtracting 280596 sequences from chr2\n",
      "\tExtracting 255191 sequences from chr3\n",
      "\tExtracting 244083 sequences from chr4\n",
      "\tExtracting 238328 sequences from chr5\n",
      "\tExtracting 234446 sequences from chr6\n",
      "\tExtracting 227768 sequences from chr7\n",
      "\tExtracting 201366 sequences from chr8\n",
      "\tExtracting 192480 sequences from chr9\n",
      "\tExtracting 204978 sequences from chr10\n",
      "\tExtracting 203178 sequences from chr11\n",
      "\tExtracting 189910 sequences from chr12\n",
      "\tExtracting 191706 sequences from chr13\n",
      "\tExtracting 196232 sequences from chr14\n",
      "\tExtracting 160803 sequences from chr15\n",
      "\tExtracting 152976 sequences from chr16\n",
      "\tExtracting 148458 sequences from chr17\n",
      "\tExtracting 140779 sequences from chr18\n",
      "\tExtracting 96716 sequences from chr19\n",
      "\tExtracting 266491 sequences from chrX\n",
      "\tExtracting 144737 sequences from chrY\n",
      "\t!!Could not open file for 115702246 (.fa or .fa.masked)\n",
      "\tExtracting 1495 sequences from chr5_JH584299_random\n",
      "\tExtracting 520 sequences from chrX_GL456233_random\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\tPotential Problem with Sequences (ignore if preparsing genome):\n",
      "\t\t1962804 of 4280792 removed because they had >70.00% Ns (i.e. masked repeats)\n",
      "\tDue to \thigh repeat content, consider using a non-repeat masked genome\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\n",
      "\tSequences processed:\n",
      "\t\tAuto detected maximum sequence length of 21 bp\n",
      "\t\t100000\n",
      "\t\t200000\n",
      "\t\t300000\n",
      "\t\t400000\n",
      "\t\t500000\n",
      "\t\t600000\n",
      "\t\t700000\n",
      "\t\t800000\n",
      "\t\t900000\n",
      "\t\t1000000\n",
      "\t\t1100000\n",
      "\t\t1200000\n",
      "\t\t1300000\n",
      "\t\t1400000\n",
      "\t\t1500000\n",
      "\t\t1600000\n",
      "\t\t1700000\n",
      "\t\t1800000\n",
      "\t\t1900000\n",
      "\t\t2000000\n",
      "\t\t2100000\n",
      "\t\t2200000\n",
      "\t\t2300000\n",
      "\t\t2317988 total\n",
      "\n",
      "\tFrequency Bins: 0.01 0.02 0.03 0.04 0.05 0.06 0.08 0.1 0.12 0.14 0.16\n",
      "\tFreq\tBin\tCount\n",
      "\t0.01\t0\t1968065\n",
      "\t0.06\t5\t296310\n",
      "\t0.08\t6\t3708\n",
      "\t0.1\t7\t1648\n",
      "\t0.12\t8\t37551\n",
      "\t0.14\t9\t808\n",
      "\t0.16\t10\t6737\n",
      "\t11\t11\t3161\n",
      "\tFrequency Bins: 0.2 0.25 0.3 0.35 0.4 0.45 0.5 0.6 0.7 0.8\n",
      "\tFreq\tBin\tCount\n",
      "\t0.2\t0\t162558\n",
      "\t0.25\t1\t152479\n",
      "\t0.3\t2\t221391\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4368\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4368\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "ls: cannot access ../data/genome/preparsed//genomer.*.cgbins: No such file or directory\n",
      "\tCould not find background files for 20 bp fragments\n",
      "\tBelow are the sizes that are already available prepared.\n",
      "\tHOMER will now create background files for 20 bp fragments\n",
      "\tTo CANCEL and rerun with a differet \"-size <#>\", hit <CTRL+C> now!\n",
      "\t\t5\n",
      "\t\t4\n",
      "\t\t3\n",
      "\t\t2\n",
      "\t\t1\n",
      "\tPreparsing genome for 20 bp fragments...(will probably take 1-5 min)\n",
      "\tUsing Custom Genome\n",
      "\tpreparse size set to 20\n",
      "\n",
      "\tNo reference file found - using random positions...\n",
      "\tOutput files will be placed in ../data/genome/preparsed//genomer.20.* ...\n",
      "\tBy default, HOMER will set the the preparsed Directory to be group read/write\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tChecking sequences from directory: ../data/genome\n",
      "\tExtracting sequences\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 307988 sequences from chr1\n",
      "\tExtracting 280596 sequences from chr2\n",
      "\tExtracting 255191 sequences from chr3\n",
      "\tExtracting 244083 sequences from chr4\n",
      "\tExtracting 238328 sequences from chr5\n",
      "\tExtracting 234446 sequences from chr6\n",
      "\tExtracting 227768 sequences from chr7\n",
      "\tExtracting 201366 sequences from chr8\n",
      "\tExtracting 192480 sequences from chr9\n",
      "\tExtracting 204978 sequences from chr10\n",
      "\tExtracting 203178 sequences from chr11\n",
      "\tExtracting 189910 sequences from chr12\n",
      "\tExtracting 191706 sequences from chr13\n",
      "\tExtracting 196232 sequences from chr14\n",
      "\tExtracting 160803 sequences from chr15\n",
      "\tExtracting 152976 sequences from chr16\n",
      "\tExtracting 148458 sequences from chr17\n",
      "\tExtracting 140779 sequences from chr18\n",
      "\tExtracting 96716 sequences from chr19\n",
      "\tExtracting 266491 sequences from chrX\n",
      "\tExtracting 144737 sequences from chrY\n",
      "\t!!Could not open file for 115702246 (.fa or .fa.masked)\n",
      "\tExtracting 1495 sequences from chr5_JH584299_random\n",
      "\tExtracting 520 sequences from chrX_GL456233_random\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\tPotential Problem with Sequences (ignore if preparsing genome):\n",
      "\t\t1962804 of 4280792 removed because they had >70.00% Ns (i.e. masked repeats)\n",
      "\tDue to \thigh repeat content, consider using a non-repeat masked genome\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\n",
      "\tSequences processed:\n",
      "\t\tAuto detected maximum sequence length of 21 bp\n",
      "\t\t100000\n",
      "\t\t200000\n",
      "\t\t300000\n",
      "\t\t400000\n",
      "\t\t500000\n",
      "\t\t600000\n",
      "\t\t700000\n",
      "\t\t800000\n",
      "\t\t900000\n",
      "\t\t1000000\n",
      "\t\t1100000\n",
      "\t\t1200000\n",
      "\t\t1300000\n",
      "\t\t1400000\n",
      "\t\t1500000\n",
      "\t\t1600000\n",
      "\t\t1700000\n",
      "\t\t1800000\n",
      "\t\t1900000\n",
      "\t\t2000000\n",
      "\t\t2100000\n",
      "\t\t2200000\n",
      "\t\t2300000\n",
      "\t\t2317988 total\n",
      "\n",
      "\tFrequency Bins: 0.01 0.02 0.03 0.04 0.05 0.06 0.08 0.1 0.12 0.14 0.16\n",
      "\tFreq\tBin\tCount\n",
      "\t0.01\t0\t1968065\n",
      "\t0.06\t5\t296310\n",
      "\t0.08\t6\t3708\n",
      "\t0.1\t7\t1648\n",
      "\t0.12\t8\t37551\n",
      "\t0.14\t9\t808\n",
      "\t0.16\t10\t6737\n",
      "\t11\t11\t3161\n",
      "\tFrequency Bins: 0.2 0.25 0.3 0.35 0.4 0.45 0.5 0.6 0.7 0.8\n",
      "\tFreq\tBin\tCount\n",
      "\t0.2\t0\t162558\n",
      "\t0.25\t1\t152479\n",
      "\t0.3\t2\t221391\n",
      "\t0.35\t3\t272418\n",
      "\t0.4\t4\t306720\n",
      "\t0.45\t5\t316194\n",
      "\t0.5\t6\t280697\n",
      "\t0.6\t7\t407794\n",
      "\t0.7\t8\t156691\n",
      "\t0.8\t9\t34506\n",
      "\t10\t10\t6540\n",
      "\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 277 sequences from chr1\n",
      "\tExtracting 329 sequences from chr2\n",
      "\tExtracting 251 sequences from chr3\n",
      "\tExtracting 287 sequences from chr4\n",
      "\tExtracting 308 sequences from chr5\n",
      "\tExtracting 233 sequences from chr6\n",
      "\tExtracting 321 sequences from chr7\n",
      "\tExtracting 219 sequences from chr8\n",
      "\tExtracting 223 sequences from chr9\n",
      "\tExtracting 252 sequences from chr10\n",
      "\tExtracting 286 sequences from chr11\n",
      "\tExtracting 173 sequences from chr12\n",
      "\tExtracting 196 sequences from chr13\n",
      "\tExtracting 165 sequences from chr14\n",
      "\tExtracting 181 sequences from chr15\n",
      "\tExtracting 163 sequences from chr16\n",
      "\tExtracting 180 sequences from chr17\n",
      "\tExtracting 122 sequences from chr18\n",
      "\tExtracting 150 sequences from chr19\n",
      "\tExtracting 48 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4364 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\t0.35\t3\t272418\n",
      "\t0.4\t4\t306720\n",
      "\t0.45\t5\t316194\n",
      "\t0.5\t6\t280697\n",
      "\t0.6\t7\t407794\n",
      "\t0.7\t8\t156691\n",
      "\t0.8\t9\t34506\n",
      "\t10\t10\t6540\n",
      "\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 281 sequences from chr1\n",
      "\tExtracting 321 sequences from chr2\n",
      "\tExtracting 218 sequences from chr3\n",
      "\tExtracting 302 sequences from chr4\n",
      "\tExtracting 278 sequences from chr5\n",
      "\tExtracting 216 sequences from chr6\n",
      "\tExtracting 349 sequences from chr7\n",
      "\tExtracting 227 sequences from chr8\n",
      "\tExtracting 225 sequences from chr9\n",
      "\tExtracting 253 sequences from chr10\n",
      "\tExtracting 319 sequences from chr11\n",
      "\tExtracting 185 sequences from chr12\n",
      "\tExtracting 206 sequences from chr13\n",
      "\tExtracting 152 sequences from chr14\n",
      "\tExtracting 182 sequences from chr15\n",
      "\tExtracting 154 sequences from chr16\n",
      "\tExtracting 202 sequences from chr17\n",
      "\tExtracting 111 sequences from chr18\n",
      "\tExtracting 146 sequences from chr19\n",
      "\tExtracting 41 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4368 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "99698\n",
      "\t0.7\t8\t154257\n",
      "\t0.8\t9\t33459\n",
      "\t10\t10\t6637\n",
      "\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 323 sequences from chr1\n",
      "\tExtracting 318 sequences from chr2\n",
      "\tExtracting 238 sequences from chr3\n",
      "\tExtracting 258 sequences from chr4\n",
      "\tExtracting 274 sequences from chr5\n",
      "\tExtracting 222 sequences from chr6\n",
      "\tExtracting 338 sequences from chr7\n",
      "\tExtracting 245 sequences from chr8\n",
      "\tExtracting 257 sequences from chr9\n",
      "\tExtracting 267 sequences from chr10\n",
      "\tExtracting 296 sequences from chr11\n",
      "\tExtracting 197 sequences from chr12\n",
      "\tExtracting 178 sequences from chr13\n",
      "\tExtracting 154 sequences from chr14\n",
      "\tExtracting 154 sequences from chr15\n",
      "\tExtracting 173 sequences from chr16\n",
      "\tExtracting 231 sequences from chr17\n",
      "\tExtracting 128 sequences from chr18\n",
      "\tExtracting 103 sequences from chr19\n",
      "\tExtracting 14 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4368 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\t0.35\t3\t272418\n",
      "\t0.4\t4\t306720\n",
      "\t0.45\t5\t316194\n",
      "\t0.5\t6\t280697\n",
      "\t0.6\t7\t407794\n",
      "\t0.7\t8\t156691\n",
      "\t0.8\t9\t34506\n",
      "\t10\t10\t6540\n",
      "\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 313 sequences from chr1\n",
      "\tExtracting 334 sequences from chr2\n",
      "\tExtracting 236 sequences from chr3\n",
      "\tExtracting 321 sequences from chr4\n",
      "\tExtracting 260 sequences from chr5\n",
      "\tExtracting 215 sequences from chr6\n",
      "\tExtracting 334 sequences from chr7\n",
      "\tExtracting 224 sequences from chr8\n",
      "\tExtracting 208 sequences from chr9\n",
      "\tExtracting 240 sequences from chr10\n",
      "\tExtracting 309 sequences from chr11\n",
      "\tExtracting 184 sequences from chr12\n",
      "\tExtracting 204 sequences from chr13\n",
      "\tExtracting 152 sequences from chr14\n",
      "\tExtracting 188 sequences from chr15\n",
      "\tExtracting 175 sequences from chr16\n",
      "\tExtracting 179 sequences from chr17\n",
      "\tExtracting 126 sequences from chr18\n",
      "\tExtracting 129 sequences from chr19\n",
      "\tExtracting 37 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4368 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset3/reads.bam tmp_dynamic/Subset3/reads.bai tmp_dynamic/Subset3/peaks.bed tmp_dynamic/Subset3/footprints.bed\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5418\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5418\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "ls: cannot access ../data/genome/preparsed//genomer.*.cgbins: No such file or directory\n",
      "\tCould not find background files for 20 bp fragments\n",
      "\tBelow are the sizes that are already available prepared.\n",
      "\tHOMER will now create background files for 20 bp fragments\n",
      "\tTo CANCEL and rerun with a differet \"-size <#>\", hit <CTRL+C> now!\n",
      "\t\t5\n",
      "\t\t4\n",
      "\t\t3\n",
      "\t\t2\n",
      "\t\t1\n",
      "\tPreparsing genome for 20 bp fragments...(will probably take 1-5 min)\n",
      "\tUsing Custom Genome\n",
      "\tpreparse size set to 20\n",
      "\n",
      "\tNo reference file found - using random positions...\n",
      "\tOutput files will be placed in ../data/genome/preparsed//genomer.20.* ...\n",
      "\tBy default, HOMER will set the the preparsed Directory to be group read/write\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tChecking sequences from directory: ../data/genome\n",
      "\tExtracting sequences\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 308032 sequences from chr1\n",
      "\tExtracting 505588 sequences from chr2\n",
      "\tExtracting 490960 sequences from chr4\n",
      "\tExtracting 367040 sequences from chr5\n",
      "\tExtracting 236273 sequences from chr6\n",
      "\tExtracting 455780 sequences from chr7\n",
      "\tExtracting 385593 sequences from chr9\n",
      "\tExtracting 190429 sequences from chr11\n",
      "\tExtracting 186903 sequences from chr12\n",
      "\tExtracting 122556 sequences from chr13\n",
      "\tExtracting 29480 sequences from chr14\n",
      "sh: line 1: 259192 Segmentation fault      homerTools extract \"../data/genome/preparsed//genomer.20.pos\" \"../data/genome\" -mask > \"0.801112258647667.tmp\"\n",
      "Use of uninitialized value $len in numeric lt (<) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 36, <IN> line 348850.\n",
      "Use of uninitialized value $len in numeric gt (>) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 40, <IN> line 348850.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 44, <IN> line 348850.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 45, <IN> line 348850.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 46, <IN> line 348850.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 47, <IN> line 348850.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 48, <IN> line 348850.\n",
      "Use of uninitialized value $line[1] in pattern match (m//) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 50, <IN> line 348\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5416\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5416\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "ls: cannot access ../data/genome/preparsed//genomer.*.cgbins: No such file or directory\n",
      "\tCould not find background files for 20 bp fragments\n",
      "\tBelow are the sizes that are already available prepared.\n",
      "\tHOMER will now create background files for 20 bp fragments\n",
      "\tTo CANCEL and rerun with a differet \"-size <#>\", hit <CTRL+C> now!\n",
      "\t\t5\n",
      "\t\t4\n",
      "\t\t3\n",
      "\t\t2\n",
      "\t\t1\n",
      "\tPreparsing genome for 20 bp fragments...(will probably take 1-5 min)\n",
      "\tUsing Custom Genome\n",
      "\tpreparse size set to 20\n",
      "\n",
      "\tNo reference file found - using random positions...\n",
      "\tOutput files will be placed in ../data/genome/preparsed//genomer.20.* ...\n",
      "\tBy default, HOMER will set the the preparsed Directory to be group read/write\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tChecking sequences from directory: ../data/genome\n",
      "\tExtracting sequences\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 308032 sequences from chr1\n",
      "\tExtracting 505588 sequences from chr2\n",
      "\tExtracting 490960 sequences from chr4\n",
      "\tExtracting 367040 sequences from chr5\n",
      "\tExtracting 236273 sequences from chr6\n",
      "\tExtracting 455780 sequences from chr7\n",
      "\tExtracting 385593 sequences from chr9\n",
      "\tExtracting 190429 sequences from chr11\n",
      "\tExtracting 186903 sequences from chr12\n",
      "\tExtracting 122556 sequences from chr13\n",
      "\tExtracting 29480 sequences from chr14\n",
      "sh: line 1: 259190 Segmentation fault      homerTools extract \"../data/genome/preparsed//genomer.20.pos\" \"../data/genome\" -mask > \"0.84980354512027.tmp\"\n",
      "Use of uninitialized value $len in numeric lt (<) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 36, <IN> line 348850.\n",
      "Use of uninitialized value $len in numeric gt (>) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 40, <IN> line 348850.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 44, <IN> line 348850.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 45, <IN> line 348850.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 46, <IN> line 348850.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 47, <IN> line 348850.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 48, <IN> line 348850.\n",
      "Use of uninitialized value $line[1] in pattern match (m//) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 50, <IN> line 3488\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5418\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5418\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "ls: cannot access ../data/genome/preparsed//genomer.*.cgbins: No such file or directory\n",
      "\tCould not find background files for 20 bp fragments\n",
      "\tBelow are the sizes that are already available prepared.\n",
      "\tHOMER will now create background files for 20 bp fragments\n",
      "\tTo CANCEL and rerun with a differet \"-size <#>\", hit <CTRL+C> now!\n",
      "\t\t5\n",
      "\t\t4\n",
      "\t\t3\n",
      "\t\t2\n",
      "\t\t1\n",
      "\tPreparsing genome for 20 bp fragments...(will probably take 1-5 min)\n",
      "\tUsing Custom Genome\n",
      "\tpreparse size set to 20\n",
      "\n",
      "\tNo reference file found - using random positions...\n",
      "\tOutput files will be placed in ../data/genome/preparsed//genomer.20.* ...\n",
      "\tBy default, HOMER will set the the preparsed Directory to be group read/write\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tChecking sequences from directory: ../data/genome\n",
      "\tExtracting sequences\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 308032 sequences from chr1\n",
      "\tExtracting 505588 sequences from chr2\n",
      "\tExtracting 490960 sequences from chr4\n",
      "\tExtracting 367040 sequences from chr5\n",
      "\tExtracting 236273 sequences from chr6\n",
      "\tExtracting 455780 sequences from chr7\n",
      "\tExtracting 385593 sequences from chr9\n",
      "\tExtracting 190429 sequences from chr11\n",
      "\tExtracting 186903 sequences from chr12\n",
      "\tExtracting 122556 sequences from chr13\n",
      "\tExtracting 29480 sequences from chr14\n",
      "sh: line 1: 259195 Segmentation fault      homerTools extract \"../data/genome/preparsed//genomer.20.pos\" \"../data/genome\" -mask > \"0.000633633070723505.tmp\"\n",
      "Use of uninitialized value $len in numeric lt (<) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 36, <IN> line 348850.\n",
      "Use of uninitialized value $len in numeric gt (>) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 40, <IN> line 348850.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 44, <IN> line 348850.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 45, <IN> line 348850.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 46, <IN> line 348850.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 47, <IN> line 348850.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 48, <IN> line 348850.\n",
      "Use of uninitialized value $line[1] in pattern match (m//) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 50, <IN> line \n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5418\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5418\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "ls: cannot access ../data/genome/preparsed//genomer.*.cgbins: No such file or directory\n",
      "\tCould not find background files for 21 bp fragments\n",
      "\tBelow are the sizes that are already available prepared.\n",
      "\tHOMER will now create background files for 21 bp fragments\n",
      "\tTo CANCEL and rerun with a differet \"-size <#>\", hit <CTRL+C> now!\n",
      "\t\t5\n",
      "\t\t4\n",
      "\t\t3\n",
      "\t\t2\n",
      "\t\t1\n",
      "\tPreparsing genome for 21 bp fragments...(will probably take 1-5 min)\n",
      "\tUsing Custom Genome\n",
      "\tpreparse size set to 21\n",
      "\n",
      "\tNo reference file found - using random positions...\n",
      "\tOutput files will be placed in ../data/genome/preparsed//genomer.21.* ...\n",
      "\tBy default, HOMER will set the the preparsed Directory to be group read/write\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tChecking sequences from directory: ../data/genome\n",
      "\tExtracting sequences\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 299831 sequences from chr1\n",
      "\tExtracting 280541 sequences from chr2\n",
      "\tExtracting 245812 sequences from chr3\n",
      "\tExtracting 248411 sequences from chr4\n",
      "\tExtracting 235376 sequences from chr5\n",
      "\tExtracting 233304 sequences from chr6\n",
      "\tExtracting 225147 sequences from chr7\n",
      "\tExtracting 202446 sequences from chr8\n",
      "\tExtracting 190651 sequences from chr9\n",
      "\tExtracting 205791 sequences from chr10\n",
      "\tExtracting 189196 sequences from chr11\n",
      "\tExtracting 185113 sequences from chr12\n",
      "\tExtracting 188601 sequences from chr13\n",
      "\tExtracting 197765 sequences from chr14\n",
      "\tExtracting 162712 sequences from chr15\n",
      "\tExtracting 150988 sequences from chr16\n",
      "\tExtracting 149895 sequences from chr17\n",
      "\tExtracting 138239 sequences from chr18\n",
      "\tExtracting 91912 sequences from chr19\n",
      "\tExtracting 263876 sequences from chrX\n",
      "\tExtracting 139526 sequences from chrY\n",
      "\tExtracting 1444 sequences from chr5_JH584299_random\n",
      "\tExtracting 466 sequences from chrX_GL456233_random\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\tPotential Problem with Sequences (ignore if preparsing genome):\n",
      "\t\t1933293 of 4225975 removed because they had >70.00% Ns (i.e. masked repeats)\n",
      "\tDue to \thigh repeat content, consider using a non-repeat masked genome\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\n",
      "\tSequences processed:\n",
      "\t\tAuto detected maximum sequence length of 21 bp\n",
      "\t\t100000\n",
      "\t\t200000\n",
      "\t\t300000\n",
      "\t\t400000\n",
      "\t\t500000\n",
      "\t\t600000\n",
      "\t\t700000\n",
      "\t\t800000\n",
      "\t\t900000\n",
      "\t\t1000000\n",
      "\t\t1100000\n",
      "\t\t1200000\n",
      "\t\t1300000\n",
      "\t\t1400000\n",
      "\t\t1500000\n",
      "\t\t1600000\n",
      "\t\t1700000\n",
      "\t\t1800000\n",
      "\t\t1900000\n",
      "\t\t2000000\n",
      "\t\t2100000\n",
      "\t\t2200000\n",
      "\t\t2292682 total\n",
      "\n",
      "\tFrequency Bins: 0.01 0.02 0.03 0.04 0.05 0.06 0.08 0.1 0.12 0.14 0.16\n",
      "\tFreq\tBin\tCount\n",
      "\t0.01\t0\t1944416\n",
      "\t0.06\t5\t294947\n",
      "\t0.08\t6\t3670\n",
      "\t0.1\t7\t1665\n",
      "\t0.12\t8\t37226\n",
      "\t0.14\t9\t858\n",
      "\t0.16\t10\t6709\n",
      "\t11\t11\t3191\n",
      "\tFrequency Bins: 0.2 0.25 0.3 0.35 0.4 0.45 0.5 0.6 0.7 0.8\n",
      "\tFreq\tBin\tCount\n",
      "\t0.2\t0\t159702\n",
      "\t0.25\t1\t149499\n",
      "\t0.3\t2\t217254\n",
      "\t0.35\t3\t266943\n",
      "\t0.4\t4\t301024\n",
      "\t0.45\t5\t312587\n",
      "\t0.5\t6\t277589\n",
      "\t0.6\t7\t407046\n",
      "\t0.7\t8\t159784\n",
      "\t0.8\t9\t34688\n",
      "\t10\t10\t6566\n",
      "\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 355 sequences from chr1\n",
      "\tExtracting 395 sequences from chr2\n",
      "\tExtracting 266 sequences from chr3\n",
      "\tExtracting 357 sequences from chr4\n",
      "\tExtracting 371 sequences from chr5\n",
      "\tExtracting 251 sequences from chr6\n",
      "\tExtracting 431 sequences from chr7\n",
      "\tExtracting 324 sequences from chr8\n",
      "\tExtracting 296 sequences from chr9\n",
      "\tExtracting 301 sequences from chr10\n",
      "\tExtracting 410 sequences from chr11\n",
      "\tExtracting 240 sequences from chr12\n",
      "\tExtracting 209 sequences from chr13\n",
      "\tExtracting 186 sequences from chr14\n",
      "\tExtracting 233 sequences from chr15\n",
      "\tExtracting 178 sequences from chr16\n",
      "\tExtracting 253 sequences from chr17\n",
      "\tExtracting 160 sequences from chr18\n",
      "\tExtracting 175 sequences from chr19\n",
      "\tExtracting 27 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5418 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "850.\n",
      "Use of uninitialized value $line[1] in concatenation (.) or string at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 55, <IN> line 348850.\n",
      "Use of uninitialized value $len in numeric lt (<) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 36, <IN> line 3248699.\n",
      "Use of uninitialized value $len in numeric gt (>) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 40, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 44, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 45, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 46, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 47, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 48, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in pattern match (m//) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 50, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in concatenation (.) or string at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 55, <IN> line 3248699.\n",
      "Use of uninitialized value $L in numeric lt (<) at ~/.conda/envs/dictys/bin/removePoorSeq.pl line 38, <IN> line 348850.\n",
      "Use of uninitialized value $L in numeric lt (<) at ~/.conda/envs/dictys/bin/removePoorSeq.pl line 38, <IN> line 3248699.\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\tPotential Problem with Sequences (ignore if preparsing genome):\n",
      "\t2 of 3248699 removed because they were shorter than 5 bp\n",
      "\t\t1440449 of 3248699 removed because they had >70.00% Ns (i.e. masked repeats)\n",
      "\tDue to \thigh repeat content, consider using a non-repeat masked genome\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\n",
      "\tSequences processed:\n",
      "\t\tAuto detected maximum sequence length of 21 bp\n",
      "\t\t100000\n",
      "\t\t200000\n",
      "\t\t300000\n",
      "\t\t400000\n",
      "\t\t500000\n",
      "\t\t600000\n",
      "\t\t700000\n",
      "\t\t800000\n",
      "\t\t900000\n",
      "\t\t1000000\n",
      "\t\t1100000\n",
      "\t\t1200000\n",
      "\t\t1300000\n",
      "\t\t1400000\n",
      "\t\t1500000\n",
      "\t\t1600000\n",
      "\t\t1700000\n",
      "\t\t1800000\n",
      "\t\t1808248 total\n",
      "\n",
      "\tFrequency Bins: 0.01 0.02 0.03 0.04 0.05 0.06 0.08 0.1 0.12 0.14 0.16\n",
      "\tFreq\tBin\tCount\n",
      "\t0.01\t0\t1274855\n",
      "\t0.06\t5\t201797\n",
      "\t0.08\t6\t2565\n",
      "\t0.1\t7\t1142\n",
      "\t0.12\t8\t26313\n",
      "\t0.14\t9\t568\n",
      "\t0.16\t10\t4845\n",
      "\t11\t11\t2241\n",
      "\tFrequency Bins: 0.2 0.25 0.3 0.35 0.4 0.45 0.5 0.6 0.7 0.8\n",
      "\tFreq\tBin\tCount\n",
      "\t0.2\t0\t117840\n",
      "\t0.25\t1\t110851\n",
      "\t0.3\t2\t163328\n",
      "\t0.35\t3\t202742\n",
      "\t0.4\t4\t232434\n",
      "\t0.45\t5\t245382\n",
      "\t0.5\t6\t223863\n",
      "\t0.6\t7\t337652\n",
      "\t0.7\t8\t137605\n",
      "\t0.8\t9\t30776\n",
      "\t10\t10\t5775\n",
      "\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 321 sequences from chr1\n",
      "\tExtracting 428 sequences from chr2\n",
      "\tExtracting 275 sequences from chr3\n",
      "\tExtracting 369 sequences from chr4\n",
      "\tExtracting 384 sequences from chr5\n",
      "\tExtracting 258 sequences from chr6\n",
      "\tExtracting 423 sequences from chr7\n",
      "\tExtracting 303 sequences from chr8\n",
      "\tExtracting 300 sequences from chr9\n",
      "\tExtracting 266 sequences from chr10\n",
      "\tExtracting 431 sequences from chr11\n",
      "\tExtracting 211 sequences from chr12\n",
      "\tExtracting 226 sequences from chr13\n",
      "\tExtracting 189 sequences from chr14\n",
      "\tExtracting 219 sequences from chr15\n",
      "\tExtracting 180 sequences from chr16\n",
      "\tExtracting 268 sequences from chr17\n",
      "\tExtracting 153 sequences from chr18\n",
      "\tExtracting 174 sequences from chr19\n",
      "\tExtracting 40 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5418 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%              50.\n",
      "Use of uninitialized value $line[1] in concatenation (.) or string at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 55, <IN> line 348850.\n",
      "Use of uninitialized value $len in numeric lt (<) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 36, <IN> line 3248699.\n",
      "Use of uninitialized value $len in numeric gt (>) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 40, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 44, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 45, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 46, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 47, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 48, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in pattern match (m//) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 50, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in concatenation (.) or string at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 55, <IN> line 3248699.\n",
      "Use of uninitialized value $L in numeric lt (<) at ~/.conda/envs/dictys/bin/removePoorSeq.pl line 38, <IN> line 348850.\n",
      "Use of uninitialized value $L in numeric lt (<) at ~/.conda/envs/dictys/bin/removePoorSeq.pl line 38, <IN> line 3248699.\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\tPotential Problem with Sequences (ignore if preparsing genome):\n",
      "\t2 of 3248699 removed because they were shorter than 5 bp\n",
      "\t\t1440449 of 3248699 removed because they had >70.00% Ns (i.e. masked repeats)\n",
      "\tDue to \thigh repeat content, consider using a non-repeat masked genome\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\n",
      "\tSequences processed:\n",
      "\t\tAuto detected maximum sequence length of 21 bp\n",
      "\t\t100000\n",
      "\t\t200000\n",
      "\t\t300000\n",
      "\t\t400000\n",
      "\t\t500000\n",
      "\t\t600000\n",
      "\t\t700000\n",
      "\t\t800000\n",
      "\t\t900000\n",
      "\t\t1000000\n",
      "\t\t1100000\n",
      "\t\t1200000\n",
      "\t\t1300000\n",
      "\t\t1400000\n",
      "\t\t1500000\n",
      "\t\t1600000\n",
      "\t\t1700000\n",
      "\t\t1800000\n",
      "\t\t1808248 total\n",
      "\n",
      "\tFrequency Bins: 0.01 0.02 0.03 0.04 0.05 0.06 0.08 0.1 0.12 0.14 0.16\n",
      "\tFreq\tBin\tCount\n",
      "\t0.01\t0\t1524227\n",
      "\t0.06\t5\t239672\n",
      "\t0.08\t6\t3021\n",
      "\t0.1\t7\t1333\n",
      "\t0.12\t8\t31039\n",
      "\t0.14\t9\t678\n",
      "\t0.16\t10\t5652\n",
      "\t11\t11\t2626\n",
      "\tFrequency Bins: 0.2 0.25 0.3 0.35 0.4 0.45 0.5 0.6 0.7 0.8\n",
      "\tFreq\tBin\tCount\n",
      "\t0.2\t0\t117840\n",
      "\t0.25\t1\t110851\n",
      "\t0.3\t2\t163328\n",
      "\t0.35\t3\t202742\n",
      "\t0.4\t4\t232434\n",
      "\t0.45\t5\t245382\n",
      "\t0.5\t6\t223863\n",
      "\t0.6\t7\t337652\n",
      "\t0.7\t8\t137605\n",
      "\t0.8\t9\t30776\n",
      "\t10\t10\t5775\n",
      "\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 298 sequences from chr1\n",
      "\tExtracting 365 sequences from chr2\n",
      "\tExtracting 292 sequences from chr3\n",
      "\tExtracting 344 sequences from chr4\n",
      "\tExtracting 367 sequences from chr5\n",
      "\tExtracting 252 sequences from chr6\n",
      "\tExtracting 458 sequences from chr7\n",
      "\tExtracting 279 sequences from chr8\n",
      "\tExtracting 294 sequences from chr9\n",
      "\tExtracting 322 sequences from chr10\n",
      "\tExtracting 421 sequences from chr11\n",
      "\tExtracting 210 sequences from chr12\n",
      "\tExtracting 261 sequences from chr13\n",
      "\tExtracting 183 sequences from chr14\n",
      "\tExtracting 221 sequences from chr15\n",
      "\tExtracting 203 sequences from chr16\n",
      "\tExtracting 281 sequences from chr17\n",
      "\tExtracting 140 sequences from chr18\n",
      "\tExtracting 179 sequences from chr19\n",
      "\tExtracting 46 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5416 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%               348850.\n",
      "Use of uninitialized value $line[1] in concatenation (.) or string at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 55, <IN> line 348850.\n",
      "Use of uninitialized value $len in numeric lt (<) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 36, <IN> line 3248699.\n",
      "Use of uninitialized value $len in numeric gt (>) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 40, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 44, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 45, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 46, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 47, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in substitution (s///) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 48, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in pattern match (m//) at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 50, <IN> line 3248699.\n",
      "Use of uninitialized value $line[1] in concatenation (.) or string at ~/.conda/envs/dictys/bin/cleanUpSequences.pl line 55, <IN> line 3248699.\n",
      "Use of uninitialized value $L in numeric lt (<) at ~/.conda/envs/dictys/bin/removePoorSeq.pl line 38, <IN> line 348850.\n",
      "Use of uninitialized value $L in numeric lt (<) at ~/.conda/envs/dictys/bin/removePoorSeq.pl line 38, <IN> line 3248699.\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\tPotential Problem with Sequences (ignore if preparsing genome):\n",
      "\t2 of 3248699 removed because they were shorter than 5 bp\n",
      "\t\t1440449 of 3248699 removed because they had >70.00% Ns (i.e. masked repeats)\n",
      "\tDue to \thigh repeat content, consider using a non-repeat masked genome\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\n",
      "\tSequences processed:\n",
      "\t\tAuto detected maximum sequence length of 21 bp\n",
      "\t\t100000\n",
      "\t\t200000\n",
      "\t\t300000\n",
      "\t\t400000\n",
      "\t\t500000\n",
      "\t\t600000\n",
      "\t\t700000\n",
      "\t\t800000\n",
      "\t\t900000\n",
      "\t\t1000000\n",
      "\t\t1100000\n",
      "\t\t1200000\n",
      "\t\t1300000\n",
      "\t\t1400000\n",
      "\t\t1500000\n",
      "\t\t1600000\n",
      "\t\t1700000\n",
      "\t\t1800000\n",
      "\t\t1808248 total\n",
      "\n",
      "\tFrequency Bins: 0.01 0.02 0.03 0.04 0.05 0.06 0.08 0.1 0.12 0.14 0.16\n",
      "\tFreq\tBin\tCount\n",
      "\t0.01\t0\t1524227\n",
      "\t0.06\t5\t239672\n",
      "\t0.08\t6\t3021\n",
      "\t0.1\t7\t1333\n",
      "\t0.12\t8\t31039\n",
      "\t0.14\t9\t678\n",
      "\t0.16\t10\t5652\n",
      "\t11\t11\t2626\n",
      "\tFrequency Bins: 0.2 0.25 0.3 0.35 0.4 0.45 0.5 0.6 0.7 0.8\n",
      "\tFreq\tBin\tCount\n",
      "\t0.2\t0\t117840\n",
      "\t0.25\t1\t110851\n",
      "\t0.3\t2\t163328\n",
      "\t0.35\t3\t202742\n",
      "\t0.4\t4\t232434\n",
      "\t0.45\t5\t245382\n",
      "\t0.5\t6\t223863\n",
      "\t0.6\t7\t337652\n",
      "\t0.7\t8\t137605\n",
      "\t0.8\t9\t30776\n",
      "\t10\t10\t5775\n",
      "\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 344 sequences from chr1\n",
      "\tExtracting 418 sequences from chr2\n",
      "\tExtracting 253 sequences from chr3\n",
      "\tExtracting 360 sequences from chr4\n",
      "\tExtracting 346 sequences from chr5\n",
      "\tExtracting 233 sequences from chr6\n",
      "\tExtracting 434 sequences from chr7\n",
      "\tExtracting 267 sequences from chr8\n",
      "\tExtracting 312 sequences from chr9\n",
      "\tExtracting 304 sequences from chr10\n",
      "\tExtracting 468 sequences from chr11\n",
      "\tExtracting 209 sequences from chr12\n",
      "\tExtracting 217 sequences from chr13\n",
      "\tExtracting 189 sequences from chr14\n",
      "\tExtracting 226 sequences from chr15\n",
      "\tExtracting 225 sequences from chr16\n",
      "\tExtracting 274 sequences from chr17\n",
      "\tExtracting 127 sequences from chr18\n",
      "\tExtracting 175 sequences from chr19\n",
      "\tExtracting 37 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5418 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                               100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "                   100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "                       100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset4/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset4/expression.tsv.gz tmp_dynamic/Subset4/motifs.bed tmp_dynamic/Subset4/wellington.tsv.gz tmp_dynamic/Subset4/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 3941\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 3941\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 240 sequences from chr1\n",
      "\tExtracting 315 sequences from chr2\n",
      "\tExtracting 205 sequences from chr3\n",
      "\tExtracting 269 sequences from chr4\n",
      "\tExtracting 260 sequences from chr5\n",
      "\tExtracting 184 sequences from chr6\n",
      "\tExtracting 307 sequences from chr7\n",
      "\tExtracting 231 sequences from chr8\n",
      "\tExtracting 187 sequences from chr9\n",
      "\tExtracting 220 sequences from chr10\n",
      "\tExtracting 248 sequences from chr11\n",
      "\tExtracting 167 sequences from chr12\n",
      "\tExtracting 183 sequences from chr13\n",
      "\tExtracting 147 sequences from chr14\n",
      "\tExtracting 185 sequences from chr15\n",
      "\tExtracting 165 sequences from chr16\n",
      "\tExtracting 162 sequences from chr17\n",
      "\tExtracting 121 sequences from chr18\n",
      "\tExtracting 119 sequences from chr19\n",
      "\tExtracting 26 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t3941 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 3937\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 3937\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 269 sequences from chr1\n",
      "\tExtracting 304 sequences from chr2\n",
      "\tExtracting 234 sequences from chr3\n",
      "\tExtracting 256 sequences from chr4\n",
      "\tExtracting 247 sequences from chr5\n",
      "\tExtracting 212 sequences from chr6\n",
      "\tExtracting 290 sequences from chr7\n",
      "\tExtracting 204 sequences from chr8\n",
      "\tExtracting 184 sequences from chr9\n",
      "\tExtracting 233 sequences from chr10\n",
      "\tExtracting 263 sequences from chr11\n",
      "\tExtracting 166 sequences from chr12\n",
      "\tExtracting 184 sequences from chr13\n",
      "\tExtracting 136 sequences from chr14\n",
      "\tExtracting 151 sequences from chr15\n",
      "\tExtracting 156 sequences from chr16\n",
      "\tExtracting 181 sequences from chr17\n",
      "\tExtracting 116 sequences from chr18\n",
      "\tExtracting 115 sequences from chr19\n",
      "\tExtracting 36 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t3937 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 3941\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 3941\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 284 sequences from chr1\n",
      "\tExtracting 285 sequences from chr2\n",
      "\tExtracting 243 sequences from chr3\n",
      "\tExtracting 266 sequences from chr4\n",
      "\tExtracting 236 sequences from chr5\n",
      "\tExtracting 192 sequences from chr6\n",
      "\tExtracting 313 sequences from chr7\n",
      "\tExtracting 214 sequences from chr8\n",
      "\tExtracting 220 sequences from chr9\n",
      "\tExtracting 224 sequences from chr10\n",
      "\tExtracting 259 sequences from chr11\n",
      "\tExtracting 152 sequences from chr12\n",
      "\tExtracting 173 sequences from chr13\n",
      "\tExtracting 118 sequences from chr14\n",
      "\tExtracting 162 sequences from chr15\n",
      "\tExtracting 150 sequences from chr16\n",
      "\tExtracting 171 sequences from chr17\n",
      "\tExtracting 123 sequences from chr18\n",
      "\tExtracting 124 sequences from chr19\n",
      "\tExtracting 32 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t3941 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 3941\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 3941\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 306 sequences from chr1\n",
      "\tExtracting 289 sequences from chr2\n",
      "\tExtracting 194 sequences from chr3\n",
      "\tExtracting 234 sequences from chr4\n",
      "\tExtracting 263 sequences from chr5\n",
      "\tExtracting 205 sequences from chr6\n",
      "\tExtracting 299 sequences from chr7\n",
      "\tExtracting 218 sequences from chr8\n",
      "\tExtracting 220 sequences from chr9\n",
      "\tExtracting 237 sequences from chr10\n",
      "\tExtracting 246 sequences from chr11\n",
      "\tExtracting 184 sequences from chr12\n",
      "\tExtracting 161 sequences from chr13\n",
      "\tExtracting 154 sequences from chr14\n",
      "\tExtracting 128 sequences from chr15\n",
      "\tExtracting 162 sequences from chr16\n",
      "\tExtracting 187 sequences from chr17\n",
      "\tExtracting 122 sequences from chr18\n",
      "\tExtracting 113 sequences from chr19\n",
      "\tExtracting 19 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t3941 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset5/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset5/expression.tsv.gz tmp_dynamic/Subset5/motifs.bed tmp_dynamic/Subset5/wellington.tsv.gz tmp_dynamic/Subset5/homer.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset6/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset6/expression.tsv.gz tmp_dynamic/Subset6/motifs.bed tmp_dynamic/Subset6/wellington.tsv.gz tmp_dynamic/Subset6/homer.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset7/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset7/expression.tsv.gz tmp_dynamic/Subset7/motifs.bed tmp_dynamic/Subset7/wellington.tsv.gz tmp_dynamic/Subset7/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4386\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4386\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 312 sequences from chr1\n",
      "\tExtracting 317 sequences from chr2\n",
      "\tExtracting 226 sequences from chr3\n",
      "\tExtracting 295 sequences from chr4\n",
      "\tExtracting 263 sequences from chr5\n",
      "\tExtracting 217 sequences from chr6\n",
      "\tExtracting 332 sequences from chr7\n",
      "\tExtracting 220 sequences from chr8\n",
      "\tExtracting 223 sequences from chr9\n",
      "\tExtracting 252 sequences from chr10\n",
      "\tExtracting 309 sequences from chr11\n",
      "\tExtracting 166 sequences from chr12\n",
      "\tExtracting 188 sequences from chr13\n",
      "\tExtracting 177 sequences from chr14\n",
      "\tExtracting 165 sequences from chr15\n",
      "\tExtracting 179 sequences from chr16\n",
      "\tExtracting 219 sequences from chr17\n",
      "\tExtracting 133 sequences from chr18\n",
      "\tExtracting 152 sequences from chr19\n",
      "\tExtracting 41 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4386 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4387\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4387\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 290 sequences from chr1\n",
      "\tExtracting 341 sequences from chr2\n",
      "\tExtracting 230 sequences from chr3\n",
      "\tExtracting 276 sequences from chr4\n",
      "\tExtracting 294 sequences from chr5\n",
      "\tExtracting 242 sequences from chr6\n",
      "\tExtracting 357 sequences from chr7\n",
      "\tExtracting 237 sequences from chr8\n",
      "\tExtracting 209 sequences from chr9\n",
      "\tExtracting 248 sequences from chr10\n",
      "\tExtracting 303 sequences from chr11\n",
      "\tExtracting 161 sequences from chr12\n",
      "\tExtracting 213 sequences from chr13\n",
      "\tExtracting 134 sequences from chr14\n",
      "\tExtracting 177 sequences from chr15\n",
      "\tExtracting 167 sequences from chr16\n",
      "\tExtracting 193 sequences from chr17\n",
      "\tExtracting 123 sequences from chr18\n",
      "\tExtracting 148 sequences from chr19\n",
      "\tExtracting 44 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4387 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4387\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4387\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 304 sequences from chr1\n",
      "\tExtracting 352 sequences from chr2\n",
      "\tExtracting 257 sequences from chr3\n",
      "\tExtracting 333 sequences from chr4\n",
      "\tExtracting 270 sequences from chr5\n",
      "\tExtracting 201 sequences from chr6\n",
      "\tExtracting 321 sequences from chr7\n",
      "\tExtracting 236 sequences from chr8\n",
      "\tExtracting 237 sequences from chr9\n",
      "\tExtracting 243 sequences from chr10\n",
      "\tExtracting 303 sequences from chr11\n",
      "\tExtracting 191 sequences from chr12\n",
      "\tExtracting 187 sequences from chr13\n",
      "\tExtracting 155 sequences from chr14\n",
      "\tExtracting 200 sequences from chr15\n",
      "\tExtracting 158 sequences from chr16\n",
      "\tExtracting 172 sequences from chr17\n",
      "\tExtracting 117 sequences from chr18\n",
      "\tExtracting 117 sequences from chr19\n",
      "\tExtracting 33 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4387 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4387\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4387\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 333 sequences from chr1\n",
      "\tExtracting 294 sequences from chr2\n",
      "\tExtracting 220 sequences from chr3\n",
      "\tExtracting 269 sequences from chr4\n",
      "\tExtracting 279 sequences from chr5\n",
      "\tExtracting 212 sequences from chr6\n",
      "\tExtracting 344 sequences from chr7\n",
      "\tExtracting 241 sequences from chr8\n",
      "\tExtracting 245 sequences from chr9\n",
      "\tExtracting 269 sequences from chr10\n",
      "\tExtracting 311 sequences from chr11\n",
      "\tExtracting 201 sequences from chr12\n",
      "\tExtracting 196 sequences from chr13\n",
      "\tExtracting 152 sequences from chr14\n",
      "\tExtracting 156 sequences from chr15\n",
      "\tExtracting 176 sequences from chr16\n",
      "\tExtracting 235 sequences from chr17\n",
      "\tExtracting 129 sequences from chr18\n",
      "\tExtracting 107 sequences from chr19\n",
      "\tExtracting 18 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4387 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset8/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset8/expression.tsv.gz tmp_dynamic/Subset8/motifs.bed tmp_dynamic/Subset8/wellington.tsv.gz tmp_dynamic/Subset8/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4589\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4589\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 312 sequences from chr1\n",
      "\tExtracting 377 sequences from chr2\n",
      "\tExtracting 265 sequences from chr3\n",
      "\tExtracting 294 sequences from chr4\n",
      "\tExtracting 314 sequences from chr5\n",
      "\tExtracting 219 sequences from chr6\n",
      "\tExtracting 346 sequences from chr7\n",
      "\tExtracting 224 sequences from chr8\n",
      "\tExtracting 241 sequences from chr9\n",
      "\tExtracting 249 sequences from chr10\n",
      "\tExtracting 313 sequences from chr11\n",
      "\tExtracting 186 sequences from chr12\n",
      "\tExtracting 197 sequences from chr13\n",
      "\tExtracting 149 sequences from chr14\n",
      "\tExtracting 197 sequences from chr15\n",
      "\tExtracting 172 sequences from chr16\n",
      "\tExtracting 232 sequences from chr17\n",
      "\tExtracting 121 sequences from chr18\n",
      "\tExtracting 142 sequences from chr19\n",
      "\tExtracting 39 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4589 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4589\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4589\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 323 sequences from chr1\n",
      "\tExtracting 328 sequences from chr2\n",
      "\tExtracting 251 sequences from chr3\n",
      "\tExtracting 341 sequences from chr4\n",
      "\tExtracting 275 sequences from chr5\n",
      "\tExtracting 205 sequences from chr6\n",
      "\tExtracting 365 sequences from chr7\n",
      "\tExtracting 255 sequences from chr8\n",
      "\tExtracting 244 sequences from chr9\n",
      "\tExtracting 273 sequences from chr10\n",
      "\tExtracting 315 sequences from chr11\n",
      "\tExtracting 196 sequences from chr12\n",
      "\tExtracting 205 sequences from chr13\n",
      "\tExtracting 162 sequences from chr14\n",
      "\tExtracting 194 sequences from chr15\n",
      "\tExtracting 159 sequences from chr16\n",
      "\tExtracting 204 sequences from chr17\n",
      "\tExtracting 122 sequences from chr18\n",
      "\tExtracting 131 sequences from chr19\n",
      "\tExtracting 41 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4589 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4587\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4587\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 294 sequences from chr1\n",
      "\tExtracting 340 sequences from chr2\n",
      "\tExtracting 247 sequences from chr3\n",
      "\tExtracting 304 sequences from chr4\n",
      "\tExtracting 294 sequences from chr5\n",
      "\tExtracting 254 sequences from chr6\n",
      "\tExtracting 389 sequences from chr7\n",
      "\tExtracting 235 sequences from chr8\n",
      "\tExtracting 234 sequences from chr9\n",
      "\tExtracting 280 sequences from chr10\n",
      "\tExtracting 328 sequences from chr11\n",
      "\tExtracting 176 sequences from chr12\n",
      "\tExtracting 186 sequences from chr13\n",
      "\tExtracting 139 sequences from chr14\n",
      "\tExtracting 192 sequences from chr15\n",
      "\tExtracting 161 sequences from chr16\n",
      "\tExtracting 197 sequences from chr17\n",
      "\tExtracting 138 sequences from chr18\n",
      "\tExtracting 150 sequences from chr19\n",
      "\tExtracting 49 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4587 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4589\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4589\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 349 sequences from chr1\n",
      "\tExtracting 325 sequences from chr2\n",
      "\tExtracting 223 sequences from chr3\n",
      "\tExtracting 286 sequences from chr4\n",
      "\tExtracting 283 sequences from chr5\n",
      "\tExtracting 233 sequences from chr6\n",
      "\tExtracting 339 sequences from chr7\n",
      "\tExtracting 248 sequences from chr8\n",
      "\tExtracting 265 sequences from chr9\n",
      "\tExtracting 272 sequences from chr10\n",
      "\tExtracting 330 sequences from chr11\n",
      "\tExtracting 198 sequences from chr12\n",
      "\tExtracting 205 sequences from chr13\n",
      "\tExtracting 168 sequences from chr14\n",
      "\tExtracting 167 sequences from chr15\n",
      "\tExtracting 185 sequences from chr16\n",
      "\tExtracting 225 sequences from chr17\n",
      "\tExtracting 141 sequences from chr18\n",
      "\tExtracting 127 sequences from chr19\n",
      "\tExtracting 20 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4589 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4584\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4584\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 307 sequences from chr1\n",
      "\tExtracting 331 sequences from chr2\n",
      "\tExtracting 237 sequences from chr3\n",
      "\tExtracting 302 sequences from chr4\n",
      "\tExtracting 320 sequences from chr5\n",
      "\tExtracting 246 sequences from chr6\n",
      "\tExtracting 392 sequences from chr7\n",
      "\tExtracting 235 sequences from chr8\n",
      "\tExtracting 230 sequences from chr9\n",
      "\tExtracting 241 sequences from chr10\n",
      "\tExtracting 330 sequences from chr11\n",
      "\tExtracting 169 sequences from chr12\n",
      "\tExtracting 183 sequences from chr13\n",
      "\tExtracting 170 sequences from chr14\n",
      "\tExtracting 191 sequences from chr15\n",
      "\tExtracting 162 sequences from chr16\n",
      "\tExtracting 198 sequences from chr17\n",
      "\tExtracting 136 sequences from chr18\n",
      "\tExtracting 151 sequences from chr19\n",
      "\tExtracting 53 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4584 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4586\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4586\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 316 sequences from chr1\n",
      "\tExtracting 359 sequences from chr2\n",
      "\tExtracting 275 sequences from chr3\n",
      "\tExtracting 301 sequences from chr4\n",
      "\tExtracting 298 sequences from chr5\n",
      "\tExtracting 215 sequences from chr6\n",
      "\tExtracting 351 sequences from chr7\n",
      "\tExtracting 218 sequences from chr8\n",
      "\tExtracting 235 sequences from chr9\n",
      "\tExtracting 266 sequences from chr10\n",
      "\tExtracting 342 sequences from chr11\n",
      "\tExtracting 178 sequences from chr12\n",
      "\tExtracting 222 sequences from chr13\n",
      "\tExtracting 137 sequences from chr14\n",
      "\tExtracting 183 sequences from chr15\n",
      "\tExtracting 161 sequences from chr16\n",
      "\tExtracting 227 sequences from chr17\n",
      "\tExtracting 120 sequences from chr18\n",
      "\tExtracting 140 sequences from chr19\n",
      "\tExtracting 42 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4586 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4586\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4586\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 339 sequences from chr1\n",
      "\tExtracting 343 sequences from chr2\n",
      "\tExtracting 242 sequences from chr3\n",
      "\tExtracting 322 sequences from chr4\n",
      "\tExtracting 267 sequences from chr5\n",
      "\tExtracting 216 sequences from chr6\n",
      "\tExtracting 355 sequences from chr7\n",
      "\tExtracting 247 sequences from chr8\n",
      "\tExtracting 250 sequences from chr9\n",
      "\tExtracting 280 sequences from chr10\n",
      "\tExtracting 315 sequences from chr11\n",
      "\tExtracting 191 sequences from chr12\n",
      "\tExtracting 201 sequences from chr13\n",
      "\tExtracting 166 sequences from chr14\n",
      "\tExtracting 190 sequences from chr15\n",
      "\tExtracting 165 sequences from chr16\n",
      "\tExtracting 197 sequences from chr17\n",
      "\tExtracting 127 sequences from chr18\n",
      "\tExtracting 134 sequences from chr19\n",
      "\tExtracting 39 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4586 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4586\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4586\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 326 sequences from chr1\n",
      "\tExtracting 321 sequences from chr2\n",
      "\tExtracting 227 sequences from chr3\n",
      "\tExtracting 293 sequences from chr4\n",
      "\tExtracting 295 sequences from chr5\n",
      "\tExtracting 222 sequences from chr6\n",
      "\tExtracting 348 sequences from chr7\n",
      "\tExtracting 254 sequences from chr8\n",
      "\tExtracting 263 sequences from chr9\n",
      "\tExtracting 274 sequences from chr10\n",
      "\tExtracting 325 sequences from chr11\n",
      "\tExtracting 210 sequences from chr12\n",
      "\tExtracting 190 sequences from chr13\n",
      "\tExtracting 170 sequences from chr14\n",
      "\tExtracting 176 sequences from chr15\n",
      "\tExtracting 174 sequences from chr16\n",
      "\tExtracting 223 sequences from chr17\n",
      "\tExtracting 142 sequences from chr18\n",
      "\tExtracting 130 sequences from chr19\n",
      "\tExtracting 23 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4586 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset9/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset9/expression.tsv.gz tmp_dynamic/Subset9/motifs.bed tmp_dynamic/Subset9/wellington.tsv.gz tmp_dynamic/Subset9/homer.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset10/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset10/expression.tsv.gz tmp_dynamic/Subset10/motifs.bed tmp_dynamic/Subset10/wellington.tsv.gz tmp_dynamic/Subset10/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4773\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4773\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 301 sequences from chr1\n",
      "\tExtracting 381 sequences from chr2\n",
      "\tExtracting 254 sequences from chr3\n",
      "\tExtracting 341 sequences from chr4\n",
      "\tExtracting 331 sequences from chr5\n",
      "\tExtracting 222 sequences from chr6\n",
      "\tExtracting 347 sequences from chr7\n",
      "\tExtracting 232 sequences from chr8\n",
      "\tExtracting 245 sequences from chr9\n",
      "\tExtracting 260 sequences from chr10\n",
      "\tExtracting 353 sequences from chr11\n",
      "\tExtracting 217 sequences from chr12\n",
      "\tExtracting 209 sequences from chr13\n",
      "\tExtracting 145 sequences from chr14\n",
      "\tExtracting 197 sequences from chr15\n",
      "\tExtracting 175 sequences from chr16\n",
      "\tExtracting 216 sequences from chr17\n",
      "\tExtracting 142 sequences from chr18\n",
      "\tExtracting 157 sequences from chr19\n",
      "\tExtracting 48 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4773 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4770\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4770\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 294 sequences from chr1\n",
      "\tExtracting 367 sequences from chr2\n",
      "\tExtracting 270 sequences from chr3\n",
      "\tExtracting 298 sequences from chr4\n",
      "\tExtracting 322 sequences from chr5\n",
      "\tExtracting 220 sequences from chr6\n",
      "\tExtracting 378 sequences from chr7\n",
      "\tExtracting 259 sequences from chr8\n",
      "\tExtracting 236 sequences from chr9\n",
      "\tExtracting 297 sequences from chr10\n",
      "\tExtracting 352 sequences from chr11\n",
      "\tExtracting 200 sequences from chr12\n",
      "\tExtracting 215 sequences from chr13\n",
      "\tExtracting 172 sequences from chr14\n",
      "\tExtracting 196 sequences from chr15\n",
      "\tExtracting 167 sequences from chr16\n",
      "\tExtracting 213 sequences from chr17\n",
      "\tExtracting 138 sequences from chr18\n",
      "\tExtracting 131 sequences from chr19\n",
      "\tExtracting 45 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4770 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4773\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4773\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 324 sequences from chr1\n",
      "\tExtracting 349 sequences from chr2\n",
      "\tExtracting 264 sequences from chr3\n",
      "\tExtracting 356 sequences from chr4\n",
      "\tExtracting 296 sequences from chr5\n",
      "\tExtracting 234 sequences from chr6\n",
      "\tExtracting 373 sequences from chr7\n",
      "\tExtracting 249 sequences from chr8\n",
      "\tExtracting 239 sequences from chr9\n",
      "\tExtracting 296 sequences from chr10\n",
      "\tExtracting 340 sequences from chr11\n",
      "\tExtracting 198 sequences from chr12\n",
      "\tExtracting 191 sequences from chr13\n",
      "\tExtracting 169 sequences from chr14\n",
      "\tExtracting 208 sequences from chr15\n",
      "\tExtracting 160 sequences from chr16\n",
      "\tExtracting 207 sequences from chr17\n",
      "\tExtracting 116 sequences from chr18\n",
      "\tExtracting 156 sequences from chr19\n",
      "\tExtracting 48 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4773 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4773\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4773\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 336 sequences from chr1\n",
      "\tExtracting 337 sequences from chr2\n",
      "\tExtracting 245 sequences from chr3\n",
      "\tExtracting 300 sequences from chr4\n",
      "\tExtracting 297 sequences from chr5\n",
      "\tExtracting 238 sequences from chr6\n",
      "\tExtracting 379 sequences from chr7\n",
      "\tExtracting 260 sequences from chr8\n",
      "\tExtracting 263 sequences from chr9\n",
      "\tExtracting 278 sequences from chr10\n",
      "\tExtracting 360 sequences from chr11\n",
      "\tExtracting 211 sequences from chr12\n",
      "\tExtracting 203 sequences from chr13\n",
      "\tExtracting 168 sequences from chr14\n",
      "\tExtracting 185 sequences from chr15\n",
      "\tExtracting 194 sequences from chr16\n",
      "\tExtracting 239 sequences from chr17\n",
      "\tExtracting 136 sequences from chr18\n",
      "\tExtracting 126 sequences from chr19\n",
      "\tExtracting 18 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4773 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset11/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset11/expression.tsv.gz tmp_dynamic/Subset11/motifs.bed tmp_dynamic/Subset11/wellington.tsv.gz tmp_dynamic/Subset11/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4755\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4755\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 315 sequences from chr1\n",
      "\tExtracting 384 sequences from chr2\n",
      "\tExtracting 257 sequences from chr3\n",
      "\tExtracting 370 sequences from chr4\n",
      "\tExtracting 305 sequences from chr5\n",
      "\tExtracting 215 sequences from chr6\n",
      "\tExtracting 356 sequences from chr7\n",
      "\tExtracting 235 sequences from chr8\n",
      "\tExtracting 266 sequences from chr9\n",
      "\tExtracting 262 sequences from chr10\n",
      "\tExtracting 338 sequences from chr11\n",
      "\tExtracting 205 sequences from chr12\n",
      "\tExtracting 193 sequences from chr13\n",
      "\tExtracting 155 sequences from chr14\n",
      "\tExtracting 206 sequences from chr15\n",
      "\tExtracting 159 sequences from chr16\n",
      "\tExtracting 209 sequences from chr17\n",
      "\tExtracting 139 sequences from chr18\n",
      "\tExtracting 147 sequences from chr19\n",
      "\tExtracting 39 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4755 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4755\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4755\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 319 sequences from chr1\n",
      "\tExtracting 331 sequences from chr2\n",
      "\tExtracting 272 sequences from chr3\n",
      "\tExtracting 345 sequences from chr4\n",
      "\tExtracting 288 sequences from chr5\n",
      "\tExtracting 232 sequences from chr6\n",
      "\tExtracting 393 sequences from chr7\n",
      "\tExtracting 245 sequences from chr8\n",
      "\tExtracting 245 sequences from chr9\n",
      "\tExtracting 288 sequences from chr10\n",
      "\tExtracting 326 sequences from chr11\n",
      "\tExtracting 191 sequences from chr12\n",
      "\tExtracting 193 sequences from chr13\n",
      "\tExtracting 171 sequences from chr14\n",
      "\tExtracting 214 sequences from chr15\n",
      "\tExtracting 177 sequences from chr16\n",
      "\tExtracting 213 sequences from chr17\n",
      "\tExtracting 127 sequences from chr18\n",
      "\tExtracting 140 sequences from chr19\n",
      "\tExtracting 45 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4755 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4754\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4754\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 285 sequences from chr1\n",
      "\tExtracting 361 sequences from chr2\n",
      "\tExtracting 270 sequences from chr3\n",
      "\tExtracting 315 sequences from chr4\n",
      "\tExtracting 336 sequences from chr5\n",
      "\tExtracting 231 sequences from chr6\n",
      "\tExtracting 357 sequences from chr7\n",
      "\tExtracting 264 sequences from chr8\n",
      "\tExtracting 253 sequences from chr9\n",
      "\tExtracting 276 sequences from chr10\n",
      "\tExtracting 356 sequences from chr11\n",
      "\tExtracting 167 sequences from chr12\n",
      "\tExtracting 207 sequences from chr13\n",
      "\tExtracting 160 sequences from chr14\n",
      "\tExtracting 215 sequences from chr15\n",
      "\tExtracting 159 sequences from chr16\n",
      "\tExtracting 203 sequences from chr17\n",
      "\tExtracting 136 sequences from chr18\n",
      "\tExtracting 146 sequences from chr19\n",
      "\tExtracting 57 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4754 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4755\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4755\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 346 sequences from chr1\n",
      "\tExtracting 346 sequences from chr2\n",
      "\tExtracting 246 sequences from chr3\n",
      "\tExtracting 295 sequences from chr4\n",
      "\tExtracting 297 sequences from chr5\n",
      "\tExtracting 237 sequences from chr6\n",
      "\tExtracting 369 sequences from chr7\n",
      "\tExtracting 259 sequences from chr8\n",
      "\tExtracting 258 sequences from chr9\n",
      "\tExtracting 281 sequences from chr10\n",
      "\tExtracting 354 sequences from chr11\n",
      "\tExtracting 217 sequences from chr12\n",
      "\tExtracting 211 sequences from chr13\n",
      "\tExtracting 166 sequences from chr14\n",
      "\tExtracting 180 sequences from chr15\n",
      "\tExtracting 183 sequences from chr16\n",
      "\tExtracting 225 sequences from chr17\n",
      "\tExtracting 134 sequences from chr18\n",
      "\tExtracting 131 sequences from chr19\n",
      "\tExtracting 20 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4755 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4751\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4751\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 308 sequences from chr1\n",
      "\tExtracting 366 sequences from chr2\n",
      "\tExtracting 264 sequences from chr3\n",
      "\tExtracting 315 sequences from chr4\n",
      "\tExtracting 328 sequences from chr5\n",
      "\tExtracting 211 sequences from chr6\n",
      "\tExtracting 363 sequences from chr7\n",
      "\tExtracting 237 sequences from chr8\n",
      "\tExtracting 262 sequences from chr9\n",
      "\tExtracting 246 sequences from chr10\n",
      "\tExtracting 395 sequences from chr11\n",
      "\tExtracting 172 sequences from chr12\n",
      "\tExtracting 226 sequences from chr13\n",
      "\tExtracting 162 sequences from chr14\n",
      "\tExtracting 178 sequences from chr15\n",
      "\tExtracting 161 sequences from chr16\n",
      "\tExtracting 226 sequences from chr17\n",
      "\tExtracting 138 sequences from chr18\n",
      "\tExtracting 153 sequences from chr19\n",
      "\tExtracting 40 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4751 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4747\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4747\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 310 sequences from chr1\n",
      "\tExtracting 377 sequences from chr2\n",
      "\tExtracting 263 sequences from chr3\n",
      "\tExtracting 316 sequences from chr4\n",
      "\tExtracting 314 sequences from chr5\n",
      "\tExtracting 239 sequences from chr6\n",
      "\tExtracting 342 sequences from chr7\n",
      "\tExtracting 243 sequences from chr8\n",
      "\tExtracting 264 sequences from chr9\n",
      "\tExtracting 279 sequences from chr10\n",
      "\tExtracting 343 sequences from chr11\n",
      "\tExtracting 168 sequences from chr12\n",
      "\tExtracting 197 sequences from chr13\n",
      "\tExtracting 158 sequences from chr14\n",
      "\tExtracting 189 sequences from chr15\n",
      "\tExtracting 174 sequences from chr16\n",
      "\tExtracting 229 sequences from chr17\n",
      "\tExtracting 122 sequences from chr18\n",
      "\tExtracting 160 sequences from chr19\n",
      "\tExtracting 60 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4747 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4751\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4751\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 349 sequences from chr1\n",
      "\tExtracting 328 sequences from chr2\n",
      "\tExtracting 238 sequences from chr3\n",
      "\tExtracting 361 sequences from chr4\n",
      "\tExtracting 292 sequences from chr5\n",
      "\tExtracting 217 sequences from chr6\n",
      "\tExtracting 404 sequences from chr7\n",
      "\tExtracting 249 sequences from chr8\n",
      "\tExtracting 235 sequences from chr9\n",
      "\tExtracting 284 sequences from chr10\n",
      "\tExtracting 333 sequences from chr11\n",
      "\tExtracting 209 sequences from chr12\n",
      "\tExtracting 178 sequences from chr13\n",
      "\tExtracting 161 sequences from chr14\n",
      "\tExtracting 227 sequences from chr15\n",
      "\tExtracting 177 sequences from chr16\n",
      "\tExtracting 211 sequences from chr17\n",
      "\tExtracting 125 sequences from chr18\n",
      "\tExtracting 135 sequences from chr19\n",
      "\tExtracting 38 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4751 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4751\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4751\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 323 sequences from chr1\n",
      "\tExtracting 350 sequences from chr2\n",
      "\tExtracting 251 sequences from chr3\n",
      "\tExtracting 295 sequences from chr4\n",
      "\tExtracting 303 sequences from chr5\n",
      "\tExtracting 234 sequences from chr6\n",
      "\tExtracting 352 sequences from chr7\n",
      "\tExtracting 270 sequences from chr8\n",
      "\tExtracting 265 sequences from chr9\n",
      "\tExtracting 274 sequences from chr10\n",
      "\tExtracting 359 sequences from chr11\n",
      "\tExtracting 220 sequences from chr12\n",
      "\tExtracting 202 sequences from chr13\n",
      "\tExtracting 165 sequences from chr14\n",
      "\tExtracting 193 sequences from chr15\n",
      "\tExtracting 183 sequences from chr16\n",
      "\tExtracting 216 sequences from chr17\n",
      "\tExtracting 135 sequences from chr18\n",
      "\tExtracting 142 sequences from chr19\n",
      "\tExtracting 19 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4751 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset12/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset12/expression.tsv.gz tmp_dynamic/Subset12/motifs.bed tmp_dynamic/Subset12/wellington.tsv.gz tmp_dynamic/Subset12/homer.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset13/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset13/expression.tsv.gz tmp_dynamic/Subset13/motifs.bed tmp_dynamic/Subset13/wellington.tsv.gz tmp_dynamic/Subset13/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4476\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4476\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 296 sequences from chr1\n",
      "\tExtracting 323 sequences from chr2\n",
      "\tExtracting 259 sequences from chr3\n",
      "\tExtracting 307 sequences from chr4\n",
      "\tExtracting 297 sequences from chr5\n",
      "\tExtracting 211 sequences from chr6\n",
      "\tExtracting 344 sequences from chr7\n",
      "\tExtracting 231 sequences from chr8\n",
      "\tExtracting 227 sequences from chr9\n",
      "\tExtracting 233 sequences from chr10\n",
      "\tExtracting 326 sequences from chr11\n",
      "\tExtracting 180 sequences from chr12\n",
      "\tExtracting 188 sequences from chr13\n",
      "\tExtracting 163 sequences from chr14\n",
      "\tExtracting 190 sequences from chr15\n",
      "\tExtracting 168 sequences from chr16\n",
      "\tExtracting 218 sequences from chr17\n",
      "\tExtracting 133 sequences from chr18\n",
      "\tExtracting 142 sequences from chr19\n",
      "\tExtracting 40 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4476 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4477\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4477\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 298 sequences from chr1\n",
      "\tExtracting 304 sequences from chr2\n",
      "\tExtracting 241 sequences from chr3\n",
      "\tExtracting 325 sequences from chr4\n",
      "\tExtracting 309 sequences from chr5\n",
      "\tExtracting 196 sequences from chr6\n",
      "\tExtracting 346 sequences from chr7\n",
      "\tExtracting 240 sequences from chr8\n",
      "\tExtracting 227 sequences from chr9\n",
      "\tExtracting 275 sequences from chr10\n",
      "\tExtracting 347 sequences from chr11\n",
      "\tExtracting 194 sequences from chr12\n",
      "\tExtracting 193 sequences from chr13\n",
      "\tExtracting 145 sequences from chr14\n",
      "\tExtracting 205 sequences from chr15\n",
      "\tExtracting 153 sequences from chr16\n",
      "\tExtracting 207 sequences from chr17\n",
      "\tExtracting 107 sequences from chr18\n",
      "\tExtracting 139 sequences from chr19\n",
      "\tExtracting 26 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4477 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4477\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4477\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 294 sequences from chr1\n",
      "\tExtracting 362 sequences from chr2\n",
      "\tExtracting 260 sequences from chr3\n",
      "\tExtracting 311 sequences from chr4\n",
      "\tExtracting 291 sequences from chr5\n",
      "\tExtracting 221 sequences from chr6\n",
      "\tExtracting 335 sequences from chr7\n",
      "\tExtracting 227 sequences from chr8\n",
      "\tExtracting 227 sequences from chr9\n",
      "\tExtracting 243 sequences from chr10\n",
      "\tExtracting 338 sequences from chr11\n",
      "\tExtracting 166 sequences from chr12\n",
      "\tExtracting 188 sequences from chr13\n",
      "\tExtracting 154 sequences from chr14\n",
      "\tExtracting 187 sequences from chr15\n",
      "\tExtracting 163 sequences from chr16\n",
      "\tExtracting 186 sequences from chr17\n",
      "\tExtracting 126 sequences from chr18\n",
      "\tExtracting 147 sequences from chr19\n",
      "\tExtracting 51 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4477 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4477\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4477\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 298 sequences from chr1\n",
      "\tExtracting 339 sequences from chr2\n",
      "\tExtracting 210 sequences from chr3\n",
      "\tExtracting 268 sequences from chr4\n",
      "\tExtracting 284 sequences from chr5\n",
      "\tExtracting 214 sequences from chr6\n",
      "\tExtracting 361 sequences from chr7\n",
      "\tExtracting 248 sequences from chr8\n",
      "\tExtracting 257 sequences from chr9\n",
      "\tExtracting 261 sequences from chr10\n",
      "\tExtracting 339 sequences from chr11\n",
      "\tExtracting 198 sequences from chr12\n",
      "\tExtracting 181 sequences from chr13\n",
      "\tExtracting 148 sequences from chr14\n",
      "\tExtracting 182 sequences from chr15\n",
      "\tExtracting 180 sequences from chr16\n",
      "\tExtracting 215 sequences from chr17\n",
      "\tExtracting 131 sequences from chr18\n",
      "\tExtracting 139 sequences from chr19\n",
      "\tExtracting 24 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4477 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset14/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset14/expression.tsv.gz tmp_dynamic/Subset14/motifs.bed tmp_dynamic/Subset14/wellington.tsv.gz tmp_dynamic/Subset14/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4294\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4294\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 281 sequences from chr1\n",
      "\tExtracting 308 sequences from chr2\n",
      "\tExtracting 209 sequences from chr3\n",
      "\tExtracting 261 sequences from chr4\n",
      "\tExtracting 281 sequences from chr5\n",
      "\tExtracting 191 sequences from chr6\n",
      "\tExtracting 353 sequences from chr7\n",
      "\tExtracting 228 sequences from chr8\n",
      "\tExtracting 242 sequences from chr9\n",
      "\tExtracting 261 sequences from chr10\n",
      "\tExtracting 337 sequences from chr11\n",
      "\tExtracting 190 sequences from chr12\n",
      "\tExtracting 165 sequences from chr13\n",
      "\tExtracting 149 sequences from chr14\n",
      "\tExtracting 184 sequences from chr15\n",
      "\tExtracting 175 sequences from chr16\n",
      "\tExtracting 205 sequences from chr17\n",
      "\tExtracting 113 sequences from chr18\n",
      "\tExtracting 140 sequences from chr19\n",
      "\tExtracting 21 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4294 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4294\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4294\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 284 sequences from chr1\n",
      "\tExtracting 323 sequences from chr2\n",
      "\tExtracting 247 sequences from chr3\n",
      "\tExtracting 306 sequences from chr4\n",
      "\tExtracting 254 sequences from chr5\n",
      "\tExtracting 212 sequences from chr6\n",
      "\tExtracting 316 sequences from chr7\n",
      "\tExtracting 218 sequences from chr8\n",
      "\tExtracting 215 sequences from chr9\n",
      "\tExtracting 271 sequences from chr10\n",
      "\tExtracting 342 sequences from chr11\n",
      "\tExtracting 167 sequences from chr12\n",
      "\tExtracting 179 sequences from chr13\n",
      "\tExtracting 127 sequences from chr14\n",
      "\tExtracting 195 sequences from chr15\n",
      "\tExtracting 134 sequences from chr16\n",
      "\tExtracting 212 sequences from chr17\n",
      "\tExtracting 115 sequences from chr18\n",
      "\tExtracting 147 sequences from chr19\n",
      "\tExtracting 30 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4294 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4290\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4290\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 256 sequences from chr1\n",
      "\tExtracting 334 sequences from chr2\n",
      "\tExtracting 208 sequences from chr3\n",
      "\tExtracting 303 sequences from chr4\n",
      "\tExtracting 305 sequences from chr5\n",
      "\tExtracting 222 sequences from chr6\n",
      "\tExtracting 338 sequences from chr7\n",
      "\tExtracting 226 sequences from chr8\n",
      "\tExtracting 234 sequences from chr9\n",
      "\tExtracting 233 sequences from chr10\n",
      "\tExtracting 347 sequences from chr11\n",
      "\tExtracting 143 sequences from chr12\n",
      "\tExtracting 164 sequences from chr13\n",
      "\tExtracting 150 sequences from chr14\n",
      "\tExtracting 166 sequences from chr15\n",
      "\tExtracting 152 sequences from chr16\n",
      "\tExtracting 217 sequences from chr17\n",
      "\tExtracting 110 sequences from chr18\n",
      "\tExtracting 131 sequences from chr19\n",
      "\tExtracting 51 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4290 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4294\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4294\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 256 sequences from chr1\n",
      "\tExtracting 323 sequences from chr2\n",
      "\tExtracting 222 sequences from chr3\n",
      "\tExtracting 297 sequences from chr4\n",
      "\tExtracting 294 sequences from chr5\n",
      "\tExtracting 194 sequences from chr6\n",
      "\tExtracting 336 sequences from chr7\n",
      "\tExtracting 223 sequences from chr8\n",
      "\tExtracting 222 sequences from chr9\n",
      "\tExtracting 218 sequences from chr10\n",
      "\tExtracting 339 sequences from chr11\n",
      "\tExtracting 190 sequences from chr12\n",
      "\tExtracting 194 sequences from chr13\n",
      "\tExtracting 145 sequences from chr14\n",
      "\tExtracting 183 sequences from chr15\n",
      "\tExtracting 153 sequences from chr16\n",
      "\tExtracting 185 sequences from chr17\n",
      "\tExtracting 124 sequences from chr18\n",
      "\tExtracting 153 sequences from chr19\n",
      "\tExtracting 43 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4294 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4497\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4497\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 304 sequences from chr1\n",
      "\tExtracting 322 sequences from chr2\n",
      "\tExtracting 262 sequences from chr3\n",
      "\tExtracting 306 sequences from chr4\n",
      "\tExtracting 278 sequences from chr5\n",
      "\tExtracting 216 sequences from chr6\n",
      "\tExtracting 364 sequences from chr7\n",
      "\tExtracting 242 sequences from chr8\n",
      "\tExtracting 236 sequences from chr9\n",
      "\tExtracting 271 sequences from chr10\n",
      "\tExtracting 381 sequences from chr11\n",
      "\tExtracting 178 sequences from chr12\n",
      "\tExtracting 163 sequences from chr13\n",
      "\tExtracting 145 sequences from chr14\n",
      "\tExtracting 194 sequences from chr15\n",
      "\tExtracting 125 sequences from chr16\n",
      "\tExtracting 195 sequences from chr17\n",
      "\tExtracting 118 sequences from chr18\n",
      "\tExtracting 164 sequences from chr19\n",
      "\tExtracting 33 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4497 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4497\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4497\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 276 sequences from chr1\n",
      "\tExtracting 350 sequences from chr2\n",
      "\tExtracting 248 sequences from chr3\n",
      "\tExtracting 303 sequences from chr4\n",
      "\tExtracting 316 sequences from chr5\n",
      "\tExtracting 203 sequences from chr6\n",
      "\tExtracting 376 sequences from chr7\n",
      "\tExtracting 248 sequences from chr8\n",
      "\tExtracting 244 sequences from chr9\n",
      "\tExtracting 221 sequences from chr10\n",
      "\tExtracting 335 sequences from chr11\n",
      "\tExtracting 185 sequences from chr12\n",
      "\tExtracting 192 sequences from chr13\n",
      "\tExtracting 155 sequences from chr14\n",
      "\tExtracting 184 sequences from chr15\n",
      "\tExtracting 161 sequences from chr16\n",
      "\tExtracting 194 sequences from chr17\n",
      "\tExtracting 113 sequences from chr18\n",
      "\tExtracting 151 sequences from chr19\n",
      "\tExtracting 42 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4497 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4496\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4496\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 277 sequences from chr1\n",
      "\tExtracting 324 sequences from chr2\n",
      "\tExtracting 226 sequences from chr3\n",
      "\tExtracting 312 sequences from chr4\n",
      "\tExtracting 326 sequences from chr5\n",
      "\tExtracting 228 sequences from chr6\n",
      "\tExtracting 338 sequences from chr7\n",
      "\tExtracting 214 sequences from chr8\n",
      "\tExtracting 242 sequences from chr9\n",
      "\tExtracting 244 sequences from chr10\n",
      "\tExtracting 341 sequences from chr11\n",
      "\tExtracting 172 sequences from chr12\n",
      "\tExtracting 196 sequences from chr13\n",
      "\tExtracting 157 sequences from chr14\n",
      "\tExtracting 185 sequences from chr15\n",
      "\tExtracting 170 sequences from chr16\n",
      "\tExtracting 220 sequences from chr17\n",
      "\tExtracting 135 sequences from chr18\n",
      "\tExtracting 139 sequences from chr19\n",
      "\tExtracting 50 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4496 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4497\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4497\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 284 sequences from chr1\n",
      "\tExtracting 333 sequences from chr2\n",
      "\tExtracting 213 sequences from chr3\n",
      "\tExtracting 279 sequences from chr4\n",
      "\tExtracting 284 sequences from chr5\n",
      "\tExtracting 214 sequences from chr6\n",
      "\tExtracting 346 sequences from chr7\n",
      "\tExtracting 236 sequences from chr8\n",
      "\tExtracting 259 sequences from chr9\n",
      "\tExtracting 264 sequences from chr10\n",
      "\tExtracting 367 sequences from chr11\n",
      "\tExtracting 198 sequences from chr12\n",
      "\tExtracting 190 sequences from chr13\n",
      "\tExtracting 147 sequences from chr14\n",
      "\tExtracting 197 sequences from chr15\n",
      "\tExtracting 185 sequences from chr16\n",
      "\tExtracting 228 sequences from chr17\n",
      "\tExtracting 116 sequences from chr18\n",
      "\tExtracting 139 sequences from chr19\n",
      "\tExtracting 18 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4497 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset15/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset15/expression.tsv.gz tmp_dynamic/Subset15/motifs.bed tmp_dynamic/Subset15/wellington.tsv.gz tmp_dynamic/Subset15/homer.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset16/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset16/expression.tsv.gz tmp_dynamic/Subset16/motifs.bed tmp_dynamic/Subset16/wellington.tsv.gz tmp_dynamic/Subset16/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4658\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4658\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 289 sequences from chr1\n",
      "\tExtracting 347 sequences from chr2\n",
      "\tExtracting 258 sequences from chr3\n",
      "\tExtracting 336 sequences from chr4\n",
      "\tExtracting 319 sequences from chr5\n",
      "\tExtracting 208 sequences from chr6\n",
      "\tExtracting 377 sequences from chr7\n",
      "\tExtracting 239 sequences from chr8\n",
      "\tExtracting 249 sequences from chr9\n",
      "\tExtracting 279 sequences from chr10\n",
      "\tExtracting 375 sequences from chr11\n",
      "\tExtracting 187 sequences from chr12\n",
      "\tExtracting 185 sequences from chr13\n",
      "\tExtracting 139 sequences from chr14\n",
      "\tExtracting 208 sequences from chr15\n",
      "\tExtracting 153 sequences from chr16\n",
      "\tExtracting 198 sequences from chr17\n",
      "\tExtracting 122 sequences from chr18\n",
      "\tExtracting 160 sequences from chr19\n",
      "\tExtracting 30 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4658 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4658\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4658\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 300 sequences from chr1\n",
      "\tExtracting 327 sequences from chr2\n",
      "\tExtracting 246 sequences from chr3\n",
      "\tExtracting 311 sequences from chr4\n",
      "\tExtracting 318 sequences from chr5\n",
      "\tExtracting 225 sequences from chr6\n",
      "\tExtracting 356 sequences from chr7\n",
      "\tExtracting 248 sequences from chr8\n",
      "\tExtracting 237 sequences from chr9\n",
      "\tExtracting 296 sequences from chr10\n",
      "\tExtracting 331 sequences from chr11\n",
      "\tExtracting 197 sequences from chr12\n",
      "\tExtracting 199 sequences from chr13\n",
      "\tExtracting 151 sequences from chr14\n",
      "\tExtracting 206 sequences from chr15\n",
      "\tExtracting 169 sequences from chr16\n",
      "\tExtracting 221 sequences from chr17\n",
      "\tExtracting 140 sequences from chr18\n",
      "\tExtracting 140 sequences from chr19\n",
      "\tExtracting 40 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4658 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4657\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4657\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 311 sequences from chr1\n",
      "\tExtracting 353 sequences from chr2\n",
      "\tExtracting 266 sequences from chr3\n",
      "\tExtracting 275 sequences from chr4\n",
      "\tExtracting 290 sequences from chr5\n",
      "\tExtracting 207 sequences from chr6\n",
      "\tExtracting 341 sequences from chr7\n",
      "\tExtracting 245 sequences from chr8\n",
      "\tExtracting 258 sequences from chr9\n",
      "\tExtracting 262 sequences from chr10\n",
      "\tExtracting 374 sequences from chr11\n",
      "\tExtracting 191 sequences from chr12\n",
      "\tExtracting 179 sequences from chr13\n",
      "\tExtracting 164 sequences from chr14\n",
      "\tExtracting 180 sequences from chr15\n",
      "\tExtracting 176 sequences from chr16\n",
      "\tExtracting 237 sequences from chr17\n",
      "\tExtracting 121 sequences from chr18\n",
      "\tExtracting 176 sequences from chr19\n",
      "\tExtracting 51 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4657 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4658\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4658\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 276 sequences from chr1\n",
      "\tExtracting 352 sequences from chr2\n",
      "\tExtracting 229 sequences from chr3\n",
      "\tExtracting 288 sequences from chr4\n",
      "\tExtracting 295 sequences from chr5\n",
      "\tExtracting 224 sequences from chr6\n",
      "\tExtracting 367 sequences from chr7\n",
      "\tExtracting 251 sequences from chr8\n",
      "\tExtracting 274 sequences from chr9\n",
      "\tExtracting 264 sequences from chr10\n",
      "\tExtracting 381 sequences from chr11\n",
      "\tExtracting 199 sequences from chr12\n",
      "\tExtracting 185 sequences from chr13\n",
      "\tExtracting 160 sequences from chr14\n",
      "\tExtracting 197 sequences from chr15\n",
      "\tExtracting 173 sequences from chr16\n",
      "\tExtracting 235 sequences from chr17\n",
      "\tExtracting 126 sequences from chr18\n",
      "\tExtracting 158 sequences from chr19\n",
      "\tExtracting 24 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4658 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset17/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset17/expression.tsv.gz tmp_dynamic/Subset17/motifs.bed tmp_dynamic/Subset17/wellington.tsv.gz tmp_dynamic/Subset17/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4649\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4649\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 308 sequences from chr1\n",
      "\tExtracting 340 sequences from chr2\n",
      "\tExtracting 247 sequences from chr3\n",
      "\tExtracting 301 sequences from chr4\n",
      "\tExtracting 289 sequences from chr5\n",
      "\tExtracting 250 sequences from chr6\n",
      "\tExtracting 371 sequences from chr7\n",
      "\tExtracting 232 sequences from chr8\n",
      "\tExtracting 261 sequences from chr9\n",
      "\tExtracting 248 sequences from chr10\n",
      "\tExtracting 352 sequences from chr11\n",
      "\tExtracting 174 sequences from chr12\n",
      "\tExtracting 185 sequences from chr13\n",
      "\tExtracting 181 sequences from chr14\n",
      "\tExtracting 197 sequences from chr15\n",
      "\tExtracting 159 sequences from chr16\n",
      "\tExtracting 220 sequences from chr17\n",
      "\tExtracting 130 sequences from chr18\n",
      "\tExtracting 160 sequences from chr19\n",
      "\tExtracting 44 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4649 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4653\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4653\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 293 sequences from chr1\n",
      "\tExtracting 328 sequences from chr2\n",
      "\tExtracting 268 sequences from chr3\n",
      "\tExtracting 330 sequences from chr4\n",
      "\tExtracting 317 sequences from chr5\n",
      "\tExtracting 229 sequences from chr6\n",
      "\tExtracting 355 sequences from chr7\n",
      "\tExtracting 243 sequences from chr8\n",
      "\tExtracting 238 sequences from chr9\n",
      "\tExtracting 256 sequences from chr10\n",
      "\tExtracting 353 sequences from chr11\n",
      "\tExtracting 189 sequences from chr12\n",
      "\tExtracting 182 sequences from chr13\n",
      "\tExtracting 149 sequences from chr14\n",
      "\tExtracting 203 sequences from chr15\n",
      "\tExtracting 160 sequences from chr16\n",
      "\tExtracting 230 sequences from chr17\n",
      "\tExtracting 132 sequences from chr18\n",
      "\tExtracting 155 sequences from chr19\n",
      "\tExtracting 43 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4653 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4653\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4653\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 286 sequences from chr1\n",
      "\tExtracting 389 sequences from chr2\n",
      "\tExtracting 224 sequences from chr3\n",
      "\tExtracting 318 sequences from chr4\n",
      "\tExtracting 322 sequences from chr5\n",
      "\tExtracting 181 sequences from chr6\n",
      "\tExtracting 350 sequences from chr7\n",
      "\tExtracting 248 sequences from chr8\n",
      "\tExtracting 271 sequences from chr9\n",
      "\tExtracting 266 sequences from chr10\n",
      "\tExtracting 371 sequences from chr11\n",
      "\tExtracting 190 sequences from chr12\n",
      "\tExtracting 196 sequences from chr13\n",
      "\tExtracting 152 sequences from chr14\n",
      "\tExtracting 207 sequences from chr15\n",
      "\tExtracting 134 sequences from chr16\n",
      "\tExtracting 220 sequences from chr17\n",
      "\tExtracting 136 sequences from chr18\n",
      "\tExtracting 165 sequences from chr19\n",
      "\tExtracting 27 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4653 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4653\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4653\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 294 sequences from chr1\n",
      "\tExtracting 319 sequences from chr2\n",
      "\tExtracting 229 sequences from chr3\n",
      "\tExtracting 283 sequences from chr4\n",
      "\tExtracting 288 sequences from chr5\n",
      "\tExtracting 224 sequences from chr6\n",
      "\tExtracting 378 sequences from chr7\n",
      "\tExtracting 253 sequences from chr8\n",
      "\tExtracting 256 sequences from chr9\n",
      "\tExtracting 275 sequences from chr10\n",
      "\tExtracting 381 sequences from chr11\n",
      "\tExtracting 190 sequences from chr12\n",
      "\tExtracting 187 sequences from chr13\n",
      "\tExtracting 161 sequences from chr14\n",
      "\tExtracting 212 sequences from chr15\n",
      "\tExtracting 187 sequences from chr16\n",
      "\tExtracting 230 sequences from chr17\n",
      "\tExtracting 115 sequences from chr18\n",
      "\tExtracting 167 sequences from chr19\n",
      "\tExtracting 24 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4653 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset18/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset18/expression.tsv.gz tmp_dynamic/Subset18/motifs.bed tmp_dynamic/Subset18/wellington.tsv.gz tmp_dynamic/Subset18/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4606\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4606\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 287 sequences from chr1\n",
      "\tExtracting 340 sequences from chr2\n",
      "\tExtracting 266 sequences from chr3\n",
      "\tExtracting 295 sequences from chr4\n",
      "\tExtracting 321 sequences from chr5\n",
      "\tExtracting 224 sequences from chr6\n",
      "\tExtracting 379 sequences from chr7\n",
      "\tExtracting 243 sequences from chr8\n",
      "\tExtracting 254 sequences from chr9\n",
      "\tExtracting 232 sequences from chr10\n",
      "\tExtracting 365 sequences from chr11\n",
      "\tExtracting 180 sequences from chr12\n",
      "\tExtracting 179 sequences from chr13\n",
      "\tExtracting 157 sequences from chr14\n",
      "\tExtracting 198 sequences from chr15\n",
      "\tExtracting 161 sequences from chr16\n",
      "\tExtracting 206 sequences from chr17\n",
      "\tExtracting 127 sequences from chr18\n",
      "\tExtracting 144 sequences from chr19\n",
      "\tExtracting 48 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4606 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4608\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4608\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 283 sequences from chr1\n",
      "\tExtracting 349 sequences from chr2\n",
      "\tExtracting 235 sequences from chr3\n",
      "\tExtracting 311 sequences from chr4\n",
      "\tExtracting 302 sequences from chr5\n",
      "\tExtracting 177 sequences from chr6\n",
      "\tExtracting 349 sequences from chr7\n",
      "\tExtracting 256 sequences from chr8\n",
      "\tExtracting 247 sequences from chr9\n",
      "\tExtracting 264 sequences from chr10\n",
      "\tExtracting 380 sequences from chr11\n",
      "\tExtracting 184 sequences from chr12\n",
      "\tExtracting 212 sequences from chr13\n",
      "\tExtracting 145 sequences from chr14\n",
      "\tExtracting 222 sequences from chr15\n",
      "\tExtracting 164 sequences from chr16\n",
      "\tExtracting 213 sequences from chr17\n",
      "\tExtracting 119 sequences from chr18\n",
      "\tExtracting 169 sequences from chr19\n",
      "\tExtracting 27 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4608 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4608\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4608\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 290 sequences from chr1\n",
      "\tExtracting 316 sequences from chr2\n",
      "\tExtracting 219 sequences from chr3\n",
      "\tExtracting 295 sequences from chr4\n",
      "\tExtracting 317 sequences from chr5\n",
      "\tExtracting 218 sequences from chr6\n",
      "\tExtracting 364 sequences from chr7\n",
      "\tExtracting 249 sequences from chr8\n",
      "\tExtracting 273 sequences from chr9\n",
      "\tExtracting 273 sequences from chr10\n",
      "\tExtracting 359 sequences from chr11\n",
      "\tExtracting 203 sequences from chr12\n",
      "\tExtracting 168 sequences from chr13\n",
      "\tExtracting 157 sequences from chr14\n",
      "\tExtracting 187 sequences from chr15\n",
      "\tExtracting 174 sequences from chr16\n",
      "\tExtracting 237 sequences from chr17\n",
      "\tExtracting 122 sequences from chr18\n",
      "\tExtracting 165 sequences from chr19\n",
      "\tExtracting 22 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4608 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4608\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4608\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 284 sequences from chr1\n",
      "\tExtracting 330 sequences from chr2\n",
      "\tExtracting 240 sequences from chr3\n",
      "\tExtracting 326 sequences from chr4\n",
      "\tExtracting 292 sequences from chr5\n",
      "\tExtracting 241 sequences from chr6\n",
      "\tExtracting 360 sequences from chr7\n",
      "\tExtracting 245 sequences from chr8\n",
      "\tExtracting 240 sequences from chr9\n",
      "\tExtracting 249 sequences from chr10\n",
      "\tExtracting 354 sequences from chr11\n",
      "\tExtracting 182 sequences from chr12\n",
      "\tExtracting 186 sequences from chr13\n",
      "\tExtracting 164 sequences from chr14\n",
      "\tExtracting 210 sequences from chr15\n",
      "\tExtracting 140 sequences from chr16\n",
      "\tExtracting 229 sequences from chr17\n",
      "\tExtracting 138 sequences from chr18\n",
      "\tExtracting 153 sequences from chr19\n",
      "\tExtracting 45 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4608 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset19/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset19/expression.tsv.gz tmp_dynamic/Subset19/motifs.bed tmp_dynamic/Subset19/wellington.tsv.gz tmp_dynamic/Subset19/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4736\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4736\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 294 sequences from chr1\n",
      "\tExtracting 363 sequences from chr2\n",
      "\tExtracting 262 sequences from chr3\n",
      "\tExtracting 339 sequences from chr4\n",
      "\tExtracting 313 sequences from chr5\n",
      "\tExtracting 234 sequences from chr6\n",
      "\tExtracting 378 sequences from chr7\n",
      "\tExtracting 257 sequences from chr8\n",
      "\tExtracting 223 sequences from chr9\n",
      "\tExtracting 256 sequences from chr10\n",
      "\tExtracting 380 sequences from chr11\n",
      "\tExtracting 190 sequences from chr12\n",
      "\tExtracting 178 sequences from chr13\n",
      "\tExtracting 165 sequences from chr14\n",
      "\tExtracting 202 sequences from chr15\n",
      "\tExtracting 171 sequences from chr16\n",
      "\tExtracting 205 sequences from chr17\n",
      "\tExtracting 122 sequences from chr18\n",
      "\tExtracting 163 sequences from chr19\n",
      "\tExtracting 41 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4736 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4733\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4733\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 290 sequences from chr1\n",
      "\tExtracting 350 sequences from chr2\n",
      "\tExtracting 265 sequences from chr3\n",
      "\tExtracting 302 sequences from chr4\n",
      "\tExtracting 334 sequences from chr5\n",
      "\tExtracting 219 sequences from chr6\n",
      "\tExtracting 374 sequences from chr7\n",
      "\tExtracting 252 sequences from chr8\n",
      "\tExtracting 254 sequences from chr9\n",
      "\tExtracting 257 sequences from chr10\n",
      "\tExtracting 380 sequences from chr11\n",
      "\tExtracting 203 sequences from chr12\n",
      "\tExtracting 178 sequences from chr13\n",
      "\tExtracting 144 sequences from chr14\n",
      "\tExtracting 199 sequences from chr15\n",
      "\tExtracting 166 sequences from chr16\n",
      "\tExtracting 229 sequences from chr17\n",
      "\tExtracting 134 sequences from chr18\n",
      "\tExtracting 159 sequences from chr19\n",
      "\tExtracting 44 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4733 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4736\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4736\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 290 sequences from chr1\n",
      "\tExtracting 346 sequences from chr2\n",
      "\tExtracting 208 sequences from chr3\n",
      "\tExtracting 297 sequences from chr4\n",
      "\tExtracting 332 sequences from chr5\n",
      "\tExtracting 226 sequences from chr6\n",
      "\tExtracting 380 sequences from chr7\n",
      "\tExtracting 265 sequences from chr8\n",
      "\tExtracting 257 sequences from chr9\n",
      "\tExtracting 271 sequences from chr10\n",
      "\tExtracting 376 sequences from chr11\n",
      "\tExtracting 197 sequences from chr12\n",
      "\tExtracting 184 sequences from chr13\n",
      "\tExtracting 155 sequences from chr14\n",
      "\tExtracting 196 sequences from chr15\n",
      "\tExtracting 176 sequences from chr16\n",
      "\tExtracting 248 sequences from chr17\n",
      "\tExtracting 138 sequences from chr18\n",
      "\tExtracting 174 sequences from chr19\n",
      "\tExtracting 20 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4736 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4736\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4736\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 301 sequences from chr1\n",
      "\tExtracting 353 sequences from chr2\n",
      "\tExtracting 229 sequences from chr3\n",
      "\tExtracting 324 sequences from chr4\n",
      "\tExtracting 311 sequences from chr5\n",
      "\tExtracting 176 sequences from chr6\n",
      "\tExtracting 386 sequences from chr7\n",
      "\tExtracting 257 sequences from chr8\n",
      "\tExtracting 294 sequences from chr9\n",
      "\tExtracting 260 sequences from chr10\n",
      "\tExtracting 395 sequences from chr11\n",
      "\tExtracting 172 sequences from chr12\n",
      "\tExtracting 210 sequences from chr13\n",
      "\tExtracting 170 sequences from chr14\n",
      "\tExtracting 219 sequences from chr15\n",
      "\tExtracting 152 sequences from chr16\n",
      "\tExtracting 216 sequences from chr17\n",
      "\tExtracting 123 sequences from chr18\n",
      "\tExtracting 154 sequences from chr19\n",
      "\tExtracting 34 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4736 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset20/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset20/expression.tsv.gz tmp_dynamic/Subset20/motifs.bed tmp_dynamic/Subset20/wellington.tsv.gz tmp_dynamic/Subset20/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4672\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4672\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 293 sequences from chr1\n",
      "\tExtracting 340 sequences from chr2\n",
      "\tExtracting 250 sequences from chr3\n",
      "\tExtracting 294 sequences from chr4\n",
      "\tExtracting 302 sequences from chr5\n",
      "\tExtracting 221 sequences from chr6\n",
      "\tExtracting 333 sequences from chr7\n",
      "\tExtracting 258 sequences from chr8\n",
      "\tExtracting 252 sequences from chr9\n",
      "\tExtracting 268 sequences from chr10\n",
      "\tExtracting 352 sequences from chr11\n",
      "\tExtracting 198 sequences from chr12\n",
      "\tExtracting 192 sequences from chr13\n",
      "\tExtracting 185 sequences from chr14\n",
      "\tExtracting 219 sequences from chr15\n",
      "\tExtracting 161 sequences from chr16\n",
      "\tExtracting 225 sequences from chr17\n",
      "\tExtracting 119 sequences from chr18\n",
      "\tExtracting 162 sequences from chr19\n",
      "\tExtracting 48 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4672 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4673\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4673\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 281 sequences from chr1\n",
      "\tExtracting 394 sequences from chr2\n",
      "\tExtracting 228 sequences from chr3\n",
      "\tExtracting 306 sequences from chr4\n",
      "\tExtracting 308 sequences from chr5\n",
      "\tExtracting 196 sequences from chr6\n",
      "\tExtracting 348 sequences from chr7\n",
      "\tExtracting 245 sequences from chr8\n",
      "\tExtracting 254 sequences from chr9\n",
      "\tExtracting 285 sequences from chr10\n",
      "\tExtracting 357 sequences from chr11\n",
      "\tExtracting 201 sequences from chr12\n",
      "\tExtracting 212 sequences from chr13\n",
      "\tExtracting 145 sequences from chr14\n",
      "\tExtracting 205 sequences from chr15\n",
      "\tExtracting 143 sequences from chr16\n",
      "\tExtracting 224 sequences from chr17\n",
      "\tExtracting 133 sequences from chr18\n",
      "\tExtracting 182 sequences from chr19\n",
      "\tExtracting 26 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4673 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4673\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4673\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 293 sequences from chr1\n",
      "\tExtracting 316 sequences from chr2\n",
      "\tExtracting 229 sequences from chr3\n",
      "\tExtracting 295 sequences from chr4\n",
      "\tExtracting 308 sequences from chr5\n",
      "\tExtracting 219 sequences from chr6\n",
      "\tExtracting 380 sequences from chr7\n",
      "\tExtracting 251 sequences from chr8\n",
      "\tExtracting 265 sequences from chr9\n",
      "\tExtracting 277 sequences from chr10\n",
      "\tExtracting 376 sequences from chr11\n",
      "\tExtracting 190 sequences from chr12\n",
      "\tExtracting 192 sequences from chr13\n",
      "\tExtracting 156 sequences from chr14\n",
      "\tExtracting 221 sequences from chr15\n",
      "\tExtracting 187 sequences from chr16\n",
      "\tExtracting 224 sequences from chr17\n",
      "\tExtracting 115 sequences from chr18\n",
      "\tExtracting 158 sequences from chr19\n",
      "\tExtracting 21 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4673 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4673\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4673\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tCould not find background files for 19 bp fragments\n",
      "\tBelow are the sizes that are already available prepared.\n",
      "\t\t20\n",
      "\t\t21\n",
      "\tHOMER will now create background files for 19 bp fragments\n",
      "\tTo CANCEL and rerun with a differet \"-size <#>\", hit <CTRL+C> now!\n",
      "\t\t5\n",
      "\t\t4\n",
      "\t\t3\n",
      "\t\t2\n",
      "\t\t1\n",
      "\tPreparsing genome for 19 bp fragments...(will probably take 1-5 min)\n",
      "\tUsing Custom Genome\n",
      "\tpreparse size set to 19\n",
      "\n",
      "\tNo reference file found - using random positions...\n",
      "\tOutput files will be placed in ../data/genome/preparsed//genomer.19.* ...\n",
      "\tBy default, HOMER will set the the preparsed Directory to be group read/write\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tChecking sequences from directory: ../data/genome\n",
      "\tExtracting sequences\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 306815 sequences from chr1\n",
      "\tExtracting 287590 sequences from chr2\n",
      "\tExtracting 253162 sequences from chr3\n",
      "\tExtracting 246684 sequences from chr4\n",
      "\tExtracting 241649 sequences from chr5\n",
      "\tExtracting 233748 sequences from chr6\n",
      "\tExtracting 234779 sequences from chr7\n",
      "\tExtracting 202054 sequences from chr8\n",
      "\tExtracting 196538 sequences from chr9\n",
      "\tExtracting 205688 sequences from chr10\n",
      "\tExtracting 194271 sequences from chr11\n",
      "\tExtracting 193157 sequences from chr12\n",
      "\tExtracting 188285 sequences from chr13\n",
      "\tExtracting 195754 sequences from chr14\n",
      "\tExtracting 166200 sequences from chr15\n",
      "\tExtracting 159174 sequences from chr16\n",
      "\tExtracting 149074 sequences from chr17\n",
      "\tExtracting 145949 sequences from chr18\n",
      "\tExtracting 97539 sequences from chr19\n",
      "\tExtracting 267883 sequences from chrX\n",
      "\tExtracting 142395 sequences from chrY\n",
      "\tExtracting 1472 sequences from chr5_JH584299_random\n",
      "\tExtracting 498 sequences from chrX_GL456233_random\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\tPotential Problem with Sequences (ignore if preparsing genome):\n",
      "\t\t1967726 of 4309939 removed because they had >70.00% Ns (i.e. masked repeats)\n",
      "\tDue to \thigh repeat content, consider using a non-repeat masked genome\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\n",
      "\tSequences processed:\n",
      "\t\tAuto detected maximum sequence length of 19 bp\n",
      "\t\t100000\n",
      "\t\t200000\n",
      "\t\t300000\n",
      "\t\t400000\n",
      "\t\t500000\n",
      "\t\t600000\n",
      "\t\t700000\n",
      "\t\t800000\n",
      "\t\t900000\n",
      "\t\t1000000\n",
      "\t\t1100000\n",
      "\t\t1200000\n",
      "\t\t1300000\n",
      "\t\t1400000\n",
      "\t\t1500000\n",
      "\t\t1600000\n",
      "\t\t1700000\n",
      "\t\t1800000\n",
      "\t\t1900000\n",
      "\t\t2000000\n",
      "\t\t2100000\n",
      "\t\t2200000\n",
      "\t\t2300000\n",
      "\t\t2342213 total\n",
      "\n",
      "\tFrequency Bins: 0.01 0.02 0.03 0.04 0.05 0.06 0.08 0.1 0.12 0.14 0.16\n",
      "\tFreq\tBin\tCount\n",
      "\t0.01\t0\t2016124\n",
      "\t0.06\t5\t278437\n",
      "\t0.08\t6\t3750\n",
      "\t0.1\t7\t1651\n",
      "\t0.12\t8\t32691\n",
      "\t0.14\t9\t779\n",
      "\t0.16\t10\t721\n",
      "\t11\t11\t8060\n",
      "\tFrequency Bins: 0.2 0.25 0.3 0.35 0.4 0.45 0.5 0.6 0.7 0.8\n",
      "\tFreq\tBin\tCount\n",
      "\t0.2\t0\t117241\n",
      "\t0.25\t1\t133355\n",
      "\t0.3\t2\t211858\n",
      "\t0.35\t3\t280570\n",
      "\t0.4\t4\t322901\n",
      "\t0.45\t5\t341226\n",
      "\t0.5\t6\t305949\n",
      "\t0.6\t7\t438971\n",
      "\t0.7\t8\t156149\n",
      "\t0.8\t9\t28890\n",
      "\t10\t10\t5103\n",
      "\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 301 sequences from chr1\n",
      "\tExtracting 318 sequences from chr2\n",
      "\tExtracting 255 sequences from chr3\n",
      "\tExtracting 334 sequences from chr4\n",
      "\tExtracting 313 sequences from chr5\n",
      "\tExtracting 226 sequences from chr6\n",
      "\tExtracting 381 sequences from chr7\n",
      "\tExtracting 255 sequences from chr8\n",
      "\tExtracting 245 sequences from chr9\n",
      "\tExtracting 243 sequences from chr10\n",
      "\tExtracting 370 sequences from chr11\n",
      "\tExtracting 163 sequences from chr12\n",
      "\tExtracting 167 sequences from chr13\n",
      "\tExtracting 172 sequences from chr14\n",
      "\tExtracting 189 sequences from chr15\n",
      "\tExtracting 173 sequences from chr16\n",
      "\tExtracting 237 sequences from chr17\n",
      "\tExtracting 136 sequences from chr18\n",
      "\tExtracting 154 sequences from chr19\n",
      "\tExtracting 41 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4673 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset21/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset21/expression.tsv.gz tmp_dynamic/Subset21/motifs.bed tmp_dynamic/Subset21/wellington.tsv.gz tmp_dynamic/Subset21/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4712\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4712\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 274 sequences from chr1\n",
      "\tExtracting 343 sequences from chr2\n",
      "\tExtracting 226 sequences from chr3\n",
      "\tExtracting 337 sequences from chr4\n",
      "\tExtracting 325 sequences from chr5\n",
      "\tExtracting 241 sequences from chr6\n",
      "\tExtracting 379 sequences from chr7\n",
      "\tExtracting 246 sequences from chr8\n",
      "\tExtracting 281 sequences from chr9\n",
      "\tExtracting 262 sequences from chr10\n",
      "\tExtracting 365 sequences from chr11\n",
      "\tExtracting 182 sequences from chr12\n",
      "\tExtracting 204 sequences from chr13\n",
      "\tExtracting 172 sequences from chr14\n",
      "\tExtracting 188 sequences from chr15\n",
      "\tExtracting 176 sequences from chr16\n",
      "\tExtracting 228 sequences from chr17\n",
      "\tExtracting 115 sequences from chr18\n",
      "\tExtracting 126 sequences from chr19\n",
      "\tExtracting 42 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4712 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4708\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4708\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 296 sequences from chr1\n",
      "\tExtracting 327 sequences from chr2\n",
      "\tExtracting 241 sequences from chr3\n",
      "\tExtracting 302 sequences from chr4\n",
      "\tExtracting 326 sequences from chr5\n",
      "\tExtracting 198 sequences from chr6\n",
      "\tExtracting 357 sequences from chr7\n",
      "\tExtracting 229 sequences from chr8\n",
      "\tExtracting 270 sequences from chr9\n",
      "\tExtracting 266 sequences from chr10\n",
      "\tExtracting 402 sequences from chr11\n",
      "\tExtracting 179 sequences from chr12\n",
      "\tExtracting 208 sequences from chr13\n",
      "\tExtracting 172 sequences from chr14\n",
      "\tExtracting 211 sequences from chr15\n",
      "\tExtracting 150 sequences from chr16\n",
      "\tExtracting 234 sequences from chr17\n",
      "\tExtracting 131 sequences from chr18\n",
      "\tExtracting 177 sequences from chr19\n",
      "\tExtracting 32 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4708 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4712\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4712\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 293 sequences from chr1\n",
      "\tExtracting 363 sequences from chr2\n",
      "\tExtracting 230 sequences from chr3\n",
      "\tExtracting 303 sequences from chr4\n",
      "\tExtracting 305 sequences from chr5\n",
      "\tExtracting 207 sequences from chr6\n",
      "\tExtracting 385 sequences from chr7\n",
      "\tExtracting 277 sequences from chr8\n",
      "\tExtracting 244 sequences from chr9\n",
      "\tExtracting 253 sequences from chr10\n",
      "\tExtracting 391 sequences from chr11\n",
      "\tExtracting 192 sequences from chr12\n",
      "\tExtracting 191 sequences from chr13\n",
      "\tExtracting 138 sequences from chr14\n",
      "\tExtracting 206 sequences from chr15\n",
      "\tExtracting 170 sequences from chr16\n",
      "\tExtracting 229 sequences from chr17\n",
      "\tExtracting 135 sequences from chr18\n",
      "\tExtracting 172 sequences from chr19\n",
      "\tExtracting 28 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4712 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4712\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4712\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 307 sequences from chr1\n",
      "\tExtracting 347 sequences from chr2\n",
      "\tExtracting 242 sequences from chr3\n",
      "\tExtracting 307 sequences from chr4\n",
      "\tExtracting 310 sequences from chr5\n",
      "\tExtracting 210 sequences from chr6\n",
      "\tExtracting 367 sequences from chr7\n",
      "\tExtracting 259 sequences from chr8\n",
      "\tExtracting 259 sequences from chr9\n",
      "\tExtracting 261 sequences from chr10\n",
      "\tExtracting 378 sequences from chr11\n",
      "\tExtracting 201 sequences from chr12\n",
      "\tExtracting 184 sequences from chr13\n",
      "\tExtracting 161 sequences from chr14\n",
      "\tExtracting 213 sequences from chr15\n",
      "\tExtracting 157 sequences from chr16\n",
      "\tExtracting 220 sequences from chr17\n",
      "\tExtracting 138 sequences from chr18\n",
      "\tExtracting 167 sequences from chr19\n",
      "\tExtracting 24 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4712 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset22/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset22/expression.tsv.gz tmp_dynamic/Subset22/motifs.bed tmp_dynamic/Subset22/wellington.tsv.gz tmp_dynamic/Subset22/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4619\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4619\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 286 sequences from chr1\n",
      "\tExtracting 337 sequences from chr2\n",
      "\tExtracting 232 sequences from chr3\n",
      "\tExtracting 286 sequences from chr4\n",
      "\tExtracting 312 sequences from chr5\n",
      "\tExtracting 208 sequences from chr6\n",
      "\tExtracting 370 sequences from chr7\n",
      "\tExtracting 228 sequences from chr8\n",
      "\tExtracting 262 sequences from chr9\n",
      "\tExtracting 269 sequences from chr10\n",
      "\tExtracting 379 sequences from chr11\n",
      "\tExtracting 178 sequences from chr12\n",
      "\tExtracting 203 sequences from chr13\n",
      "\tExtracting 172 sequences from chr14\n",
      "\tExtracting 204 sequences from chr15\n",
      "\tExtracting 164 sequences from chr16\n",
      "\tExtracting 221 sequences from chr17\n",
      "\tExtracting 118 sequences from chr18\n",
      "\tExtracting 152 sequences from chr19\n",
      "\tExtracting 38 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4619 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4619\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4619\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 288 sequences from chr1\n",
      "\tExtracting 333 sequences from chr2\n",
      "\tExtracting 214 sequences from chr3\n",
      "\tExtracting 303 sequences from chr4\n",
      "\tExtracting 311 sequences from chr5\n",
      "\tExtracting 205 sequences from chr6\n",
      "\tExtracting 357 sequences from chr7\n",
      "\tExtracting 254 sequences from chr8\n",
      "\tExtracting 257 sequences from chr9\n",
      "\tExtracting 257 sequences from chr10\n",
      "\tExtracting 375 sequences from chr11\n",
      "\tExtracting 198 sequences from chr12\n",
      "\tExtracting 189 sequences from chr13\n",
      "\tExtracting 150 sequences from chr14\n",
      "\tExtracting 199 sequences from chr15\n",
      "\tExtracting 166 sequences from chr16\n",
      "\tExtracting 229 sequences from chr17\n",
      "\tExtracting 142 sequences from chr18\n",
      "\tExtracting 173 sequences from chr19\n",
      "\tExtracting 19 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4619 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4619\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4619\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 308 sequences from chr1\n",
      "\tExtracting 341 sequences from chr2\n",
      "\tExtracting 247 sequences from chr3\n",
      "\tExtracting 312 sequences from chr4\n",
      "\tExtracting 297 sequences from chr5\n",
      "\tExtracting 208 sequences from chr6\n",
      "\tExtracting 355 sequences from chr7\n",
      "\tExtracting 265 sequences from chr8\n",
      "\tExtracting 260 sequences from chr9\n",
      "\tExtracting 261 sequences from chr10\n",
      "\tExtracting 371 sequences from chr11\n",
      "\tExtracting 192 sequences from chr12\n",
      "\tExtracting 166 sequences from chr13\n",
      "\tExtracting 173 sequences from chr14\n",
      "\tExtracting 197 sequences from chr15\n",
      "\tExtracting 157 sequences from chr16\n",
      "\tExtracting 218 sequences from chr17\n",
      "\tExtracting 109 sequences from chr18\n",
      "\tExtracting 141 sequences from chr19\n",
      "\tExtracting 41 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4619 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4618\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4618\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tCould not find background files for 19 bp fragments\n",
      "\tBelow are the sizes that are already available prepared.\n",
      "\t\t20\n",
      "\t\t21\n",
      "\tHOMER will now create background files for 19 bp fragments\n",
      "\tTo CANCEL and rerun with a differet \"-size <#>\", hit <CTRL+C> now!\n",
      "\t\t5\n",
      "\t\t4\n",
      "\t\t3\n",
      "\t\t2\n",
      "\t\t1\n",
      "\tPreparsing genome for 19 bp fragments...(will probably take 1-5 min)\n",
      "\tUsing Custom Genome\n",
      "\tpreparse size set to 19\n",
      "\n",
      "\tNo reference file found - using random positions...\n",
      "\tOutput files will be placed in ../data/genome/preparsed//genomer.19.* ...\n",
      "\tBy default, HOMER will set the the preparsed Directory to be group read/write\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tChecking sequences from directory: ../data/genome\n",
      "\tExtracting sequences\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 310030 sequences from chr1\n",
      "\tExtracting 287346 sequences from chr2\n",
      "\tExtracting 254057 sequences from chr3\n",
      "\tExtracting 248010 sequences from chr4\n",
      "\tExtracting 240795 sequences from chr5\n",
      "\tExtracting 236172 sequences from chr6\n",
      "\tExtracting 228593 sequences from chr7\n",
      "\tExtracting 208855 sequences from chr8\n",
      "\tExtracting 194647 sequences from chr9\n",
      "\tExtracting 204287 sequences from chr10\n",
      "\tExtracting 196647 sequences from chr11\n",
      "\tExtracting 192882 sequences from chr12\n",
      "\tExtracting 187575 sequences from chr13\n",
      "\tExtracting 197110 sequences from chr14\n",
      "\tExtracting 163862 sequences from chr15\n",
      "\tExtracting 151532 sequences from chr16\n",
      "\tExtracting 149834 sequences from chr17\n",
      "\tExtracting 141683 sequences from chr18\n",
      "\tExtracting 96484 sequences from chr19\n",
      "\tExtracting 272401 sequences from chrX\n",
      "\tExtracting 146471 sequences from chrY\n",
      "\tExtracting 1498 sequences from chr5_JH584299_random\n",
      "\tExtracting 510 sequences from chrX_GL456233_random\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\tPotential Problem with Sequences (ignore if preparsing genome):\n",
      "\t1 of 4310378 removed because they were shorter than 5 bp\n",
      "\t\t1955616 of 4310378 removed because they had >70.00% Ns (i.e. masked repeats)\n",
      "\tDue to \thigh repeat content, consider using a non-repeat masked genome\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\n",
      "\tSequences processed:\n",
      "\t\tAuto detected maximum sequence length of 19 bp\n",
      "\t\t100000\n",
      "\t\t200000\n",
      "\t\t300000\n",
      "\t\t400000\n",
      "\t\t500000\n",
      "\t\t600000\n",
      "\t\t700000\n",
      "\t\t800000\n",
      "\t\t900000\n",
      "\t\t1000000\n",
      "\t\t1100000\n",
      "\t\t1200000\n",
      "\t\t1300000\n",
      "\t\t1400000\n",
      "\t\t1500000\n",
      "\t\t1600000\n",
      "\t\t1700000\n",
      "\t\t1800000\n",
      "\t\t1900000\n",
      "\t\t2000000\n",
      "\t\t2100000\n",
      "\t\t2200000\n",
      "\t\t2300000\n",
      "\t\t2354761 total\n",
      "\n",
      "\tFrequency Bins: 0.01 0.02 0.03 0.04 0.05 0.06 0.08 0.1 0.12 0.14 0.16\n",
      "\tFreq\tBin\tCount\n",
      "\t0.01\t0\t2026141\n",
      "\t0.06\t5\t280490\n",
      "\t0.08\t6\t3700\n",
      "\t0.1\t7\t1649\n",
      "\t0.12\t8\t33063\n",
      "\t0.14\t9\t809\n",
      "\t0.16\t10\t725\n",
      "\t11\t11\t8184\n",
      "\tFrequency Bins: 0.2 0.25 0.3 0.35 0.4 0.45 0.5 0.6 0.7 0.8\n",
      "\tFreq\tBin\tCount\n",
      "\t0.2\t0\t116241\n",
      "\t0.25\t1\t133787\n",
      "\t0.3\t2\t211019\n",
      "\t0.35\t3\t280097\n",
      "\t0.4\t4\t324491\n",
      "\t0.45\t5\t343756\n",
      "\t0.5\t6\t309811\n",
      "\t0.6\t7\t442996\n",
      "\t0.7\t8\t157838\n",
      "\t0.8\t9\t29615\n",
      "\t10\t10\t5110\n",
      "\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 274 sequences from chr1\n",
      "\tExtracting 340 sequences from chr2\n",
      "\tExtracting 214 sequences from chr3\n",
      "\tExtracting 314 sequences from chr4\n",
      "\tExtracting 346 sequences from chr5\n",
      "\tExtracting 207 sequences from chr6\n",
      "\tExtracting 407 sequences from chr7\n",
      "\tExtracting 235 sequences from chr8\n",
      "\tExtracting 255 sequences from chr9\n",
      "\tExtracting 235 sequences from chr10\n",
      "\tExtracting 389 sequences from chr11\n",
      "\tExtracting 168 sequences from chr12\n",
      "\tExtracting 213 sequences from chr13\n",
      "\tExtracting 147 sequences from chr14\n",
      "\tExtracting 175 sequences from chr15\n",
      "\tExtracting 150 sequences from chr16\n",
      "\tExtracting 215 sequences from chr17\n",
      "\tExtracting 144 sequences from chr18\n",
      "\tExtracting 150 sequences from chr19\n",
      "\tExtracting 40 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4618 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset23/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset23/expression.tsv.gz tmp_dynamic/Subset23/motifs.bed tmp_dynamic/Subset23/wellington.tsv.gz tmp_dynamic/Subset23/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4620\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4620\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 319 sequences from chr1\n",
      "\tExtracting 349 sequences from chr2\n",
      "\tExtracting 242 sequences from chr3\n",
      "\tExtracting 323 sequences from chr4\n",
      "\tExtracting 310 sequences from chr5\n",
      "\tExtracting 204 sequences from chr6\n",
      "\tExtracting 366 sequences from chr7\n",
      "\tExtracting 258 sequences from chr8\n",
      "\tExtracting 252 sequences from chr9\n",
      "\tExtracting 252 sequences from chr10\n",
      "\tExtracting 367 sequences from chr11\n",
      "\tExtracting 196 sequences from chr12\n",
      "\tExtracting 173 sequences from chr13\n",
      "\tExtracting 166 sequences from chr14\n",
      "\tExtracting 184 sequences from chr15\n",
      "\tExtracting 147 sequences from chr16\n",
      "\tExtracting 225 sequences from chr17\n",
      "\tExtracting 107 sequences from chr18\n",
      "\tExtracting 141 sequences from chr19\n",
      "\tExtracting 39 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4620 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4620\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4620\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 280 sequences from chr1\n",
      "\tExtracting 333 sequences from chr2\n",
      "\tExtracting 236 sequences from chr3\n",
      "\tExtracting 288 sequences from chr4\n",
      "\tExtracting 320 sequences from chr5\n",
      "\tExtracting 201 sequences from chr6\n",
      "\tExtracting 393 sequences from chr7\n",
      "\tExtracting 225 sequences from chr8\n",
      "\tExtracting 275 sequences from chr9\n",
      "\tExtracting 260 sequences from chr10\n",
      "\tExtracting 391 sequences from chr11\n",
      "\tExtracting 151 sequences from chr12\n",
      "\tExtracting 198 sequences from chr13\n",
      "\tExtracting 184 sequences from chr14\n",
      "\tExtracting 212 sequences from chr15\n",
      "\tExtracting 167 sequences from chr16\n",
      "\tExtracting 214 sequences from chr17\n",
      "\tExtracting 110 sequences from chr18\n",
      "\tExtracting 148 sequences from chr19\n",
      "\tExtracting 34 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4620 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4620\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4620\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 278 sequences from chr1\n",
      "\tExtracting 344 sequences from chr2\n",
      "\tExtracting 214 sequences from chr3\n",
      "\tExtracting 289 sequences from chr4\n",
      "\tExtracting 300 sequences from chr5\n",
      "\tExtracting 213 sequences from chr6\n",
      "\tExtracting 352 sequences from chr7\n",
      "\tExtracting 260 sequences from chr8\n",
      "\tExtracting 256 sequences from chr9\n",
      "\tExtracting 263 sequences from chr10\n",
      "\tExtracting 370 sequences from chr11\n",
      "\tExtracting 197 sequences from chr12\n",
      "\tExtracting 188 sequences from chr13\n",
      "\tExtracting 147 sequences from chr14\n",
      "\tExtracting 209 sequences from chr15\n",
      "\tExtracting 168 sequences from chr16\n",
      "\tExtracting 234 sequences from chr17\n",
      "\tExtracting 144 sequences from chr18\n",
      "\tExtracting 174 sequences from chr19\n",
      "\tExtracting 20 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4620 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4619\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4619\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tCould not find background files for 19 bp fragments\n",
      "\tBelow are the sizes that are already available prepared.\n",
      "\t\t20\n",
      "\t\t21\n",
      "\tHOMER will now create background files for 19 bp fragments\n",
      "\tTo CANCEL and rerun with a differet \"-size <#>\", hit <CTRL+C> now!\n",
      "\t\t5\n",
      "\t\t4\n",
      "\t\t3\n",
      "\t\t2\n",
      "\t\t1\n",
      "\tPreparsing genome for 19 bp fragments...(will probably take 1-5 min)\n",
      "\tUsing Custom Genome\n",
      "\tpreparse size set to 19\n",
      "\n",
      "\tNo reference file found - using random positions...\n",
      "\tOutput files will be placed in ../data/genome/preparsed//genomer.19.* ...\n",
      "\tBy default, HOMER will set the the preparsed Directory to be group read/write\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tChecking sequences from directory: ../data/genome\n",
      "\tExtracting sequences\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 312027 sequences from chr1\n",
      "\tExtracting 290428 sequences from chr2\n",
      "\tExtracting 251423 sequences from chr3\n",
      "\tExtracting 246634 sequences from chr4\n",
      "\tExtracting 239483 sequences from chr5\n",
      "\tExtracting 237884 sequences from chr6\n",
      "\tExtracting 230598 sequences from chr7\n",
      "\tExtracting 201958 sequences from chr8\n",
      "\tExtracting 196306 sequences from chr9\n",
      "\tExtracting 208090 sequences from chr10\n",
      "\tExtracting 195775 sequences from chr11\n",
      "\tExtracting 190614 sequences from chr12\n",
      "\tExtracting 188471 sequences from chr13\n",
      "\tExtracting 196331 sequences from chr14\n",
      "\tExtracting 166122 sequences from chr15\n",
      "\tExtracting 153264 sequences from chr16\n",
      "\tExtracting 147703 sequences from chr17\n",
      "\tExtracting 139168 sequences from chr18\n",
      "\tExtracting 96472 sequences from chr19\n",
      "\tExtracting 266883 sequences from chrX\n",
      "\tExtracting 146266 sequences from chrY\n",
      "\tExtracting 1328 sequences from chr5_JH584299_random\n",
      "\tExtracting 521 sequences from chrX_GL456233_random\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\tPotential Problem with Sequences (ignore if preparsing genome):\n",
      "\t\t1969646 of 4303213 removed because they had >70.00% Ns (i.e. masked repeats)\n",
      "\tDue to \thigh repeat content, consider using a non-repeat masked genome\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\n",
      "\tSequences processed:\n",
      "\t\tAuto detected maximum sequence length of 19 bp\n",
      "\t\t100000\n",
      "\t\t200000\n",
      "\t\t300000\n",
      "\t\t400000\n",
      "\t\t500000\n",
      "\t\t600000\n",
      "\t\t700000\n",
      "\t\t800000\n",
      "\t\t900000\n",
      "\t\t1000000\n",
      "\t\t1100000\n",
      "\t\t1200000\n",
      "\t\t1300000\n",
      "\t\t1400000\n",
      "\t\t1500000\n",
      "\t\t1600000\n",
      "\t\t1700000\n",
      "\t\t1800000\n",
      "\t\t1900000\n",
      "\t\t2000000\n",
      "\t\t2100000\n",
      "\t\t2200000\n",
      "\t\t2300000\n",
      "\t\t2333567 total\n",
      "\n",
      "\tFrequency Bins: 0.01 0.02 0.03 0.04 0.05 0.06 0.08 0.1 0.12 0.14 0.16\n",
      "\tFreq\tBin\tCount\n",
      "\t0.01\t0\t2008691\n",
      "\t0.06\t5\t277027\n",
      "\t0.08\t6\t3757\n",
      "\t0.1\t7\t1608\n",
      "\t0.12\t8\t32722\n",
      "\t0.14\t9\t759\n",
      "\t0.16\t10\t759\n",
      "\t11\t11\t8244\n",
      "\tFrequency Bins: 0.2 0.25 0.3 0.35 0.4 0.45 0.5 0.6 0.7 0.8\n",
      "\tFreq\tBin\tCount\n",
      "\t0.2\t0\t116208\n",
      "\t0.25\t1\t132834\n",
      "\t0.3\t2\t209900\n",
      "\t0.35\t3\t277750\n",
      "\t0.4\t4\t321230\n",
      "\t0.45\t5\t341412\n",
      "\t0.5\t6\t305752\n",
      "\t0.6\t7\t438295\n",
      "\t0.7\t8\t155950\n",
      "\t0.8\t9\t28948\n",
      "\t10\t10\t5288\n",
      "\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 257 sequences from chr1\n",
      "\tExtracting 325 sequences from chr2\n",
      "\tExtracting 223 sequences from chr3\n",
      "\tExtracting 310 sequences from chr4\n",
      "\tExtracting 327 sequences from chr5\n",
      "\tExtracting 218 sequences from chr6\n",
      "\tExtracting 396 sequences from chr7\n",
      "\tExtracting 242 sequences from chr8\n",
      "\tExtracting 250 sequences from chr9\n",
      "\tExtracting 237 sequences from chr10\n",
      "\tExtracting 399 sequences from chr11\n",
      "\tExtracting 187 sequences from chr12\n",
      "\tExtracting 196 sequences from chr13\n",
      "\tExtracting 170 sequences from chr14\n",
      "\tExtracting 170 sequences from chr15\n",
      "\tExtracting 155 sequences from chr16\n",
      "\tExtracting 210 sequences from chr17\n",
      "\tExtracting 148 sequences from chr18\n",
      "\tExtracting 153 sequences from chr19\n",
      "\tExtracting 46 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4619 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4761\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4761\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 317 sequences from chr1\n",
      "\tExtracting 343 sequences from chr2\n",
      "\tExtracting 224 sequences from chr3\n",
      "\tExtracting 309 sequences from chr4\n",
      "\tExtracting 319 sequences from chr5\n",
      "\tExtracting 234 sequences from chr6\n",
      "\tExtracting 374 sequences from chr7\n",
      "\tExtracting 247 sequences from chr8\n",
      "\tExtracting 275 sequences from chr9\n",
      "\tExtracting 272 sequences from chr10\n",
      "\tExtracting 374 sequences from chr11\n",
      "\tExtracting 190 sequences from chr12\n",
      "\tExtracting 196 sequences from chr13\n",
      "\tExtracting 172 sequences from chr14\n",
      "\tExtracting 209 sequences from chr15\n",
      "\tExtracting 176 sequences from chr16\n",
      "\tExtracting 229 sequences from chr17\n",
      "\tExtracting 134 sequences from chr18\n",
      "\tExtracting 129 sequences from chr19\n",
      "\tExtracting 38 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4761 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4761\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4761\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 293 sequences from chr1\n",
      "\tExtracting 360 sequences from chr2\n",
      "\tExtracting 239 sequences from chr3\n",
      "\tExtracting 321 sequences from chr4\n",
      "\tExtracting 315 sequences from chr5\n",
      "\tExtracting 203 sequences from chr6\n",
      "\tExtracting 378 sequences from chr7\n",
      "\tExtracting 262 sequences from chr8\n",
      "\tExtracting 255 sequences from chr9\n",
      "\tExtracting 253 sequences from chr10\n",
      "\tExtracting 411 sequences from chr11\n",
      "\tExtracting 190 sequences from chr12\n",
      "\tExtracting 198 sequences from chr13\n",
      "\tExtracting 142 sequences from chr14\n",
      "\tExtracting 203 sequences from chr15\n",
      "\tExtracting 169 sequences from chr16\n",
      "\tExtracting 230 sequences from chr17\n",
      "\tExtracting 133 sequences from chr18\n",
      "\tExtracting 175 sequences from chr19\n",
      "\tExtracting 31 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4761 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4761\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4761\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 311 sequences from chr1\n",
      "\tExtracting 349 sequences from chr2\n",
      "\tExtracting 238 sequences from chr3\n",
      "\tExtracting 309 sequences from chr4\n",
      "\tExtracting 314 sequences from chr5\n",
      "\tExtracting 217 sequences from chr6\n",
      "\tExtracting 369 sequences from chr7\n",
      "\tExtracting 274 sequences from chr8\n",
      "\tExtracting 257 sequences from chr9\n",
      "\tExtracting 267 sequences from chr10\n",
      "\tExtracting 377 sequences from chr11\n",
      "\tExtracting 205 sequences from chr12\n",
      "\tExtracting 181 sequences from chr13\n",
      "\tExtracting 170 sequences from chr14\n",
      "\tExtracting 210 sequences from chr15\n",
      "\tExtracting 165 sequences from chr16\n",
      "\tExtracting 226 sequences from chr17\n",
      "\tExtracting 134 sequences from chr18\n",
      "\tExtracting 164 sequences from chr19\n",
      "\tExtracting 24 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4761 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4758\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4758\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 268 sequences from chr1\n",
      "\tExtracting 340 sequences from chr2\n",
      "\tExtracting 241 sequences from chr3\n",
      "\tExtracting 314 sequences from chr4\n",
      "\tExtracting 312 sequences from chr5\n",
      "\tExtracting 214 sequences from chr6\n",
      "\tExtracting 400 sequences from chr7\n",
      "\tExtracting 235 sequences from chr8\n",
      "\tExtracting 281 sequences from chr9\n",
      "\tExtracting 261 sequences from chr10\n",
      "\tExtracting 407 sequences from chr11\n",
      "\tExtracting 187 sequences from chr12\n",
      "\tExtracting 219 sequences from chr13\n",
      "\tExtracting 180 sequences from chr14\n",
      "\tExtracting 185 sequences from chr15\n",
      "\tExtracting 152 sequences from chr16\n",
      "\tExtracting 238 sequences from chr17\n",
      "\tExtracting 120 sequences from chr18\n",
      "\tExtracting 169 sequences from chr19\n",
      "\tExtracting 35 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4758 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset24/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset24/expression.tsv.gz tmp_dynamic/Subset24/motifs.bed tmp_dynamic/Subset24/wellington.tsv.gz tmp_dynamic/Subset24/homer.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset25/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset25/expression.tsv.gz tmp_dynamic/Subset25/motifs.bed tmp_dynamic/Subset25/wellington.tsv.gz tmp_dynamic/Subset25/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5277\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5277\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 307 sequences from chr1\n",
      "\tExtracting 379 sequences from chr2\n",
      "\tExtracting 283 sequences from chr3\n",
      "\tExtracting 337 sequences from chr4\n",
      "\tExtracting 363 sequences from chr5\n",
      "\tExtracting 252 sequences from chr6\n",
      "\tExtracting 442 sequences from chr7\n",
      "\tExtracting 298 sequences from chr8\n",
      "\tExtracting 322 sequences from chr9\n",
      "\tExtracting 273 sequences from chr10\n",
      "\tExtracting 425 sequences from chr11\n",
      "\tExtracting 220 sequences from chr12\n",
      "\tExtracting 232 sequences from chr13\n",
      "\tExtracting 185 sequences from chr14\n",
      "\tExtracting 204 sequences from chr15\n",
      "\tExtracting 174 sequences from chr16\n",
      "\tExtracting 278 sequences from chr17\n",
      "\tExtracting 126 sequences from chr18\n",
      "\tExtracting 148 sequences from chr19\n",
      "\tExtracting 29 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5277 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5277\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5277\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 357 sequences from chr1\n",
      "\tExtracting 412 sequences from chr2\n",
      "\tExtracting 253 sequences from chr3\n",
      "\tExtracting 363 sequences from chr4\n",
      "\tExtracting 346 sequences from chr5\n",
      "\tExtracting 230 sequences from chr6\n",
      "\tExtracting 418 sequences from chr7\n",
      "\tExtracting 251 sequences from chr8\n",
      "\tExtracting 282 sequences from chr9\n",
      "\tExtracting 288 sequences from chr10\n",
      "\tExtracting 435 sequences from chr11\n",
      "\tExtracting 198 sequences from chr12\n",
      "\tExtracting 204 sequences from chr13\n",
      "\tExtracting 181 sequences from chr14\n",
      "\tExtracting 236 sequences from chr15\n",
      "\tExtracting 205 sequences from chr16\n",
      "\tExtracting 255 sequences from chr17\n",
      "\tExtracting 149 sequences from chr18\n",
      "\tExtracting 174 sequences from chr19\n",
      "\tExtracting 40 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5277 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5277\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5277\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 346 sequences from chr1\n",
      "\tExtracting 385 sequences from chr2\n",
      "\tExtracting 261 sequences from chr3\n",
      "\tExtracting 341 sequences from chr4\n",
      "\tExtracting 352 sequences from chr5\n",
      "\tExtracting 242 sequences from chr6\n",
      "\tExtracting 411 sequences from chr7\n",
      "\tExtracting 317 sequences from chr8\n",
      "\tExtracting 284 sequences from chr9\n",
      "\tExtracting 293 sequences from chr10\n",
      "\tExtracting 414 sequences from chr11\n",
      "\tExtracting 236 sequences from chr12\n",
      "\tExtracting 207 sequences from chr13\n",
      "\tExtracting 179 sequences from chr14\n",
      "\tExtracting 230 sequences from chr15\n",
      "\tExtracting 183 sequences from chr16\n",
      "\tExtracting 250 sequences from chr17\n",
      "\tExtracting 145 sequences from chr18\n",
      "\tExtracting 174 sequences from chr19\n",
      "\tExtracting 27 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5277 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5273\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5273\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 306 sequences from chr1\n",
      "\tExtracting 399 sequences from chr2\n",
      "\tExtracting 263 sequences from chr3\n",
      "\tExtracting 351 sequences from chr4\n",
      "\tExtracting 365 sequences from chr5\n",
      "\tExtracting 230 sequences from chr6\n",
      "\tExtracting 441 sequences from chr7\n",
      "\tExtracting 268 sequences from chr8\n",
      "\tExtracting 296 sequences from chr9\n",
      "\tExtracting 291 sequences from chr10\n",
      "\tExtracting 421 sequences from chr11\n",
      "\tExtracting 202 sequences from chr12\n",
      "\tExtracting 234 sequences from chr13\n",
      "\tExtracting 174 sequences from chr14\n",
      "\tExtracting 217 sequences from chr15\n",
      "\tExtracting 196 sequences from chr16\n",
      "\tExtracting 253 sequences from chr17\n",
      "\tExtracting 134 sequences from chr18\n",
      "\tExtracting 183 sequences from chr19\n",
      "\tExtracting 49 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5273 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset26/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset26/expression.tsv.gz tmp_dynamic/Subset26/motifs.bed tmp_dynamic/Subset26/wellington.tsv.gz tmp_dynamic/Subset26/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4515\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4515\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 291 sequences from chr1\n",
      "\tExtracting 329 sequences from chr2\n",
      "\tExtracting 248 sequences from chr3\n",
      "\tExtracting 282 sequences from chr4\n",
      "\tExtracting 326 sequences from chr5\n",
      "\tExtracting 225 sequences from chr6\n",
      "\tExtracting 360 sequences from chr7\n",
      "\tExtracting 239 sequences from chr8\n",
      "\tExtracting 222 sequences from chr9\n",
      "\tExtracting 272 sequences from chr10\n",
      "\tExtracting 308 sequences from chr11\n",
      "\tExtracting 190 sequences from chr12\n",
      "\tExtracting 203 sequences from chr13\n",
      "\tExtracting 163 sequences from chr14\n",
      "\tExtracting 187 sequences from chr15\n",
      "\tExtracting 166 sequences from chr16\n",
      "\tExtracting 202 sequences from chr17\n",
      "\tExtracting 118 sequences from chr18\n",
      "\tExtracting 139 sequences from chr19\n",
      "\tExtracting 45 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4515 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4515\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4515\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 339 sequences from chr1\n",
      "\tExtracting 307 sequences from chr2\n",
      "\tExtracting 252 sequences from chr3\n",
      "\tExtracting 266 sequences from chr4\n",
      "\tExtracting 279 sequences from chr5\n",
      "\tExtracting 231 sequences from chr6\n",
      "\tExtracting 350 sequences from chr7\n",
      "\tExtracting 269 sequences from chr8\n",
      "\tExtracting 259 sequences from chr9\n",
      "\tExtracting 278 sequences from chr10\n",
      "\tExtracting 303 sequences from chr11\n",
      "\tExtracting 205 sequences from chr12\n",
      "\tExtracting 186 sequences from chr13\n",
      "\tExtracting 157 sequences from chr14\n",
      "\tExtracting 167 sequences from chr15\n",
      "\tExtracting 173 sequences from chr16\n",
      "\tExtracting 223 sequences from chr17\n",
      "\tExtracting 134 sequences from chr18\n",
      "\tExtracting 118 sequences from chr19\n",
      "\tExtracting 19 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4515 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4515\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4515\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 325 sequences from chr1\n",
      "\tExtracting 352 sequences from chr2\n",
      "\tExtracting 240 sequences from chr3\n",
      "\tExtracting 335 sequences from chr4\n",
      "\tExtracting 264 sequences from chr5\n",
      "\tExtracting 218 sequences from chr6\n",
      "\tExtracting 338 sequences from chr7\n",
      "\tExtracting 207 sequences from chr8\n",
      "\tExtracting 230 sequences from chr9\n",
      "\tExtracting 258 sequences from chr10\n",
      "\tExtracting 322 sequences from chr11\n",
      "\tExtracting 179 sequences from chr12\n",
      "\tExtracting 221 sequences from chr13\n",
      "\tExtracting 165 sequences from chr14\n",
      "\tExtracting 184 sequences from chr15\n",
      "\tExtracting 185 sequences from chr16\n",
      "\tExtracting 196 sequences from chr17\n",
      "\tExtracting 124 sequences from chr18\n",
      "\tExtracting 143 sequences from chr19\n",
      "\tExtracting 29 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4515 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4511\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4511\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 278 sequences from chr1\n",
      "\tExtracting 382 sequences from chr2\n",
      "\tExtracting 239 sequences from chr3\n",
      "\tExtracting 291 sequences from chr4\n",
      "\tExtracting 273 sequences from chr5\n",
      "\tExtracting 222 sequences from chr6\n",
      "\tExtracting 357 sequences from chr7\n",
      "\tExtracting 259 sequences from chr8\n",
      "\tExtracting 257 sequences from chr9\n",
      "\tExtracting 252 sequences from chr10\n",
      "\tExtracting 319 sequences from chr11\n",
      "\tExtracting 160 sequences from chr12\n",
      "\tExtracting 213 sequences from chr13\n",
      "\tExtracting 145 sequences from chr14\n",
      "\tExtracting 189 sequences from chr15\n",
      "\tExtracting 167 sequences from chr16\n",
      "\tExtracting 189 sequences from chr17\n",
      "\tExtracting 127 sequences from chr18\n",
      "\tExtracting 143 sequences from chr19\n",
      "\tExtracting 49 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4511 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4552\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4552\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 306 sequences from chr1\n",
      "\tExtracting 341 sequences from chr2\n",
      "\tExtracting 256 sequences from chr3\n",
      "\tExtracting 284 sequences from chr4\n",
      "\tExtracting 304 sequences from chr5\n",
      "\tExtracting 232 sequences from chr6\n",
      "\tExtracting 341 sequences from chr7\n",
      "\tExtracting 239 sequences from chr8\n",
      "\tExtracting 237 sequences from chr9\n",
      "\tExtracting 263 sequences from chr10\n",
      "\tExtracting 318 sequences from chr11\n",
      "\tExtracting 188 sequences from chr12\n",
      "\tExtracting 189 sequences from chr13\n",
      "\tExtracting 156 sequences from chr14\n",
      "\tExtracting 187 sequences from chr15\n",
      "\tExtracting 173 sequences from chr16\n",
      "\tExtracting 214 sequences from chr17\n",
      "\tExtracting 121 sequences from chr18\n",
      "\tExtracting 149 sequences from chr19\n",
      "\tExtracting 54 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4552 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4553\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4553\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 285 sequences from chr1\n",
      "\tExtracting 355 sequences from chr2\n",
      "\tExtracting 234 sequences from chr3\n",
      "\tExtracting 313 sequences from chr4\n",
      "\tExtracting 301 sequences from chr5\n",
      "\tExtracting 224 sequences from chr6\n",
      "\tExtracting 367 sequences from chr7\n",
      "\tExtracting 232 sequences from chr8\n",
      "\tExtracting 252 sequences from chr9\n",
      "\tExtracting 272 sequences from chr10\n",
      "\tExtracting 310 sequences from chr11\n",
      "\tExtracting 194 sequences from chr12\n",
      "\tExtracting 223 sequences from chr13\n",
      "\tExtracting 156 sequences from chr14\n",
      "\tExtracting 158 sequences from chr15\n",
      "\tExtracting 173 sequences from chr16\n",
      "\tExtracting 198 sequences from chr17\n",
      "\tExtracting 129 sequences from chr18\n",
      "\tExtracting 138 sequences from chr19\n",
      "\tExtracting 39 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4553 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4553\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4553\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 328 sequences from chr1\n",
      "\tExtracting 344 sequences from chr2\n",
      "\tExtracting 247 sequences from chr3\n",
      "\tExtracting 285 sequences from chr4\n",
      "\tExtracting 265 sequences from chr5\n",
      "\tExtracting 193 sequences from chr6\n",
      "\tExtracting 331 sequences from chr7\n",
      "\tExtracting 238 sequences from chr8\n",
      "\tExtracting 253 sequences from chr9\n",
      "\tExtracting 270 sequences from chr10\n",
      "\tExtracting 336 sequences from chr11\n",
      "\tExtracting 176 sequences from chr12\n",
      "\tExtracting 231 sequences from chr13\n",
      "\tExtracting 175 sequences from chr14\n",
      "\tExtracting 204 sequences from chr15\n",
      "\tExtracting 176 sequences from chr16\n",
      "\tExtracting 207 sequences from chr17\n",
      "\tExtracting 121 sequences from chr18\n",
      "\tExtracting 140 sequences from chr19\n",
      "\tExtracting 33 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4553 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4553\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4553\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 354 sequences from chr1\n",
      "\tExtracting 321 sequences from chr2\n",
      "\tExtracting 258 sequences from chr3\n",
      "\tExtracting 272 sequences from chr4\n",
      "\tExtracting 296 sequences from chr5\n",
      "\tExtracting 244 sequences from chr6\n",
      "\tExtracting 360 sequences from chr7\n",
      "\tExtracting 249 sequences from chr8\n",
      "\tExtracting 248 sequences from chr9\n",
      "\tExtracting 256 sequences from chr10\n",
      "\tExtracting 304 sequences from chr11\n",
      "\tExtracting 202 sequences from chr12\n",
      "\tExtracting 194 sequences from chr13\n",
      "\tExtracting 163 sequences from chr14\n",
      "\tExtracting 164 sequences from chr15\n",
      "\tExtracting 184 sequences from chr16\n",
      "\tExtracting 218 sequences from chr17\n",
      "\tExtracting 125 sequences from chr18\n",
      "\tExtracting 122 sequences from chr19\n",
      "\tExtracting 19 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4553 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset27/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset27/expression.tsv.gz tmp_dynamic/Subset27/motifs.bed tmp_dynamic/Subset27/wellington.tsv.gz tmp_dynamic/Subset27/homer.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset28/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset28/expression.tsv.gz tmp_dynamic/Subset28/motifs.bed tmp_dynamic/Subset28/wellington.tsv.gz tmp_dynamic/Subset28/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4723\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4723\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 311 sequences from chr1\n",
      "\tExtracting 378 sequences from chr2\n",
      "\tExtracting 256 sequences from chr3\n",
      "\tExtracting 289 sequences from chr4\n",
      "\tExtracting 310 sequences from chr5\n",
      "\tExtracting 217 sequences from chr6\n",
      "\tExtracting 391 sequences from chr7\n",
      "\tExtracting 254 sequences from chr8\n",
      "\tExtracting 250 sequences from chr9\n",
      "\tExtracting 287 sequences from chr10\n",
      "\tExtracting 371 sequences from chr11\n",
      "\tExtracting 183 sequences from chr12\n",
      "\tExtracting 191 sequences from chr13\n",
      "\tExtracting 165 sequences from chr14\n",
      "\tExtracting 182 sequences from chr15\n",
      "\tExtracting 170 sequences from chr16\n",
      "\tExtracting 202 sequences from chr17\n",
      "\tExtracting 131 sequences from chr18\n",
      "\tExtracting 139 sequences from chr19\n",
      "\tExtracting 46 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4723 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4723\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4723\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 330 sequences from chr1\n",
      "\tExtracting 351 sequences from chr2\n",
      "\tExtracting 255 sequences from chr3\n",
      "\tExtracting 332 sequences from chr4\n",
      "\tExtracting 313 sequences from chr5\n",
      "\tExtracting 204 sequences from chr6\n",
      "\tExtracting 339 sequences from chr7\n",
      "\tExtracting 236 sequences from chr8\n",
      "\tExtracting 256 sequences from chr9\n",
      "\tExtracting 269 sequences from chr10\n",
      "\tExtracting 355 sequences from chr11\n",
      "\tExtracting 195 sequences from chr12\n",
      "\tExtracting 236 sequences from chr13\n",
      "\tExtracting 164 sequences from chr14\n",
      "\tExtracting 182 sequences from chr15\n",
      "\tExtracting 175 sequences from chr16\n",
      "\tExtracting 208 sequences from chr17\n",
      "\tExtracting 127 sequences from chr18\n",
      "\tExtracting 159 sequences from chr19\n",
      "\tExtracting 37 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4723 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4721\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4721\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 319 sequences from chr1\n",
      "\tExtracting 335 sequences from chr2\n",
      "\tExtracting 285 sequences from chr3\n",
      "\tExtracting 332 sequences from chr4\n",
      "\tExtracting 303 sequences from chr5\n",
      "\tExtracting 228 sequences from chr6\n",
      "\tExtracting 361 sequences from chr7\n",
      "\tExtracting 253 sequences from chr8\n",
      "\tExtracting 256 sequences from chr9\n",
      "\tExtracting 276 sequences from chr10\n",
      "\tExtracting 307 sequences from chr11\n",
      "\tExtracting 196 sequences from chr12\n",
      "\tExtracting 225 sequences from chr13\n",
      "\tExtracting 145 sequences from chr14\n",
      "\tExtracting 217 sequences from chr15\n",
      "\tExtracting 147 sequences from chr16\n",
      "\tExtracting 212 sequences from chr17\n",
      "\tExtracting 124 sequences from chr18\n",
      "\tExtracting 151 sequences from chr19\n",
      "\tExtracting 49 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4721 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4723\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4723\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 359 sequences from chr1\n",
      "\tExtracting 327 sequences from chr2\n",
      "\tExtracting 269 sequences from chr3\n",
      "\tExtracting 262 sequences from chr4\n",
      "\tExtracting 293 sequences from chr5\n",
      "\tExtracting 262 sequences from chr6\n",
      "\tExtracting 364 sequences from chr7\n",
      "\tExtracting 257 sequences from chr8\n",
      "\tExtracting 282 sequences from chr9\n",
      "\tExtracting 268 sequences from chr10\n",
      "\tExtracting 317 sequences from chr11\n",
      "\tExtracting 203 sequences from chr12\n",
      "\tExtracting 222 sequences from chr13\n",
      "\tExtracting 175 sequences from chr14\n",
      "\tExtracting 164 sequences from chr15\n",
      "\tExtracting 194 sequences from chr16\n",
      "\tExtracting 220 sequences from chr17\n",
      "\tExtracting 142 sequences from chr18\n",
      "\tExtracting 124 sequences from chr19\n",
      "\tExtracting 19 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4723 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset29/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset29/expression.tsv.gz tmp_dynamic/Subset29/motifs.bed tmp_dynamic/Subset29/wellington.tsv.gz tmp_dynamic/Subset29/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5013\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5013\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 315 sequences from chr1\n",
      "\tExtracting 378 sequences from chr2\n",
      "\tExtracting 277 sequences from chr3\n",
      "\tExtracting 319 sequences from chr4\n",
      "\tExtracting 292 sequences from chr5\n",
      "\tExtracting 262 sequences from chr6\n",
      "\tExtracting 413 sequences from chr7\n",
      "\tExtracting 282 sequences from chr8\n",
      "\tExtracting 246 sequences from chr9\n",
      "\tExtracting 300 sequences from chr10\n",
      "\tExtracting 358 sequences from chr11\n",
      "\tExtracting 195 sequences from chr12\n",
      "\tExtracting 209 sequences from chr13\n",
      "\tExtracting 185 sequences from chr14\n",
      "\tExtracting 222 sequences from chr15\n",
      "\tExtracting 172 sequences from chr16\n",
      "\tExtracting 245 sequences from chr17\n",
      "\tExtracting 134 sequences from chr18\n",
      "\tExtracting 161 sequences from chr19\n",
      "\tExtracting 48 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5013 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5017\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5017\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 344 sequences from chr1\n",
      "\tExtracting 402 sequences from chr2\n",
      "\tExtracting 272 sequences from chr3\n",
      "\tExtracting 339 sequences from chr4\n",
      "\tExtracting 300 sequences from chr5\n",
      "\tExtracting 244 sequences from chr6\n",
      "\tExtracting 392 sequences from chr7\n",
      "\tExtracting 274 sequences from chr8\n",
      "\tExtracting 272 sequences from chr9\n",
      "\tExtracting 278 sequences from chr10\n",
      "\tExtracting 389 sequences from chr11\n",
      "\tExtracting 220 sequences from chr12\n",
      "\tExtracting 213 sequences from chr13\n",
      "\tExtracting 182 sequences from chr14\n",
      "\tExtracting 205 sequences from chr15\n",
      "\tExtracting 159 sequences from chr16\n",
      "\tExtracting 204 sequences from chr17\n",
      "\tExtracting 132 sequences from chr18\n",
      "\tExtracting 154 sequences from chr19\n",
      "\tExtracting 42 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5017 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5017\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5017\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 350 sequences from chr1\n",
      "\tExtracting 383 sequences from chr2\n",
      "\tExtracting 264 sequences from chr3\n",
      "\tExtracting 362 sequences from chr4\n",
      "\tExtracting 336 sequences from chr5\n",
      "\tExtracting 235 sequences from chr6\n",
      "\tExtracting 363 sequences from chr7\n",
      "\tExtracting 275 sequences from chr8\n",
      "\tExtracting 272 sequences from chr9\n",
      "\tExtracting 284 sequences from chr10\n",
      "\tExtracting 344 sequences from chr11\n",
      "\tExtracting 180 sequences from chr12\n",
      "\tExtracting 246 sequences from chr13\n",
      "\tExtracting 166 sequences from chr14\n",
      "\tExtracting 192 sequences from chr15\n",
      "\tExtracting 200 sequences from chr16\n",
      "\tExtracting 227 sequences from chr17\n",
      "\tExtracting 142 sequences from chr18\n",
      "\tExtracting 161 sequences from chr19\n",
      "\tExtracting 35 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5017 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5017\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5017\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 369 sequences from chr1\n",
      "\tExtracting 347 sequences from chr2\n",
      "\tExtracting 283 sequences from chr3\n",
      "\tExtracting 273 sequences from chr4\n",
      "\tExtracting 346 sequences from chr5\n",
      "\tExtracting 265 sequences from chr6\n",
      "\tExtracting 384 sequences from chr7\n",
      "\tExtracting 253 sequences from chr8\n",
      "\tExtracting 302 sequences from chr9\n",
      "\tExtracting 308 sequences from chr10\n",
      "\tExtracting 355 sequences from chr11\n",
      "\tExtracting 226 sequences from chr12\n",
      "\tExtracting 216 sequences from chr13\n",
      "\tExtracting 181 sequences from chr14\n",
      "\tExtracting 185 sequences from chr15\n",
      "\tExtracting 199 sequences from chr16\n",
      "\tExtracting 223 sequences from chr17\n",
      "\tExtracting 143 sequences from chr18\n",
      "\tExtracting 134 sequences from chr19\n",
      "\tExtracting 25 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5017 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5226\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5226\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 355 sequences from chr1\n",
      "\tExtracting 386 sequences from chr2\n",
      "\tExtracting 275 sequences from chr3\n",
      "\tExtracting 385 sequences from chr4\n",
      "\tExtracting 290 sequences from chr5\n",
      "\tExtracting 283 sequences from chr6\n",
      "\tExtracting 351 sequences from chr7\n",
      "\tExtracting 303 sequences from chr8\n",
      "\tExtracting 285 sequences from chr9\n",
      "\tExtracting 293 sequences from chr10\n",
      "\tExtracting 413 sequences from chr11\n",
      "\tExtracting 239 sequences from chr12\n",
      "\tExtracting 220 sequences from chr13\n",
      "\tExtracting 181 sequences from chr14\n",
      "\tExtracting 218 sequences from chr15\n",
      "\tExtracting 178 sequences from chr16\n",
      "\tExtracting 228 sequences from chr17\n",
      "\tExtracting 120 sequences from chr18\n",
      "\tExtracting 172 sequences from chr19\n",
      "\tExtracting 51 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5226 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5225\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5225\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 320 sequences from chr1\n",
      "\tExtracting 409 sequences from chr2\n",
      "\tExtracting 310 sequences from chr3\n",
      "\tExtracting 312 sequences from chr4\n",
      "\tExtracting 342 sequences from chr5\n",
      "\tExtracting 248 sequences from chr6\n",
      "\tExtracting 433 sequences from chr7\n",
      "\tExtracting 251 sequences from chr8\n",
      "\tExtracting 279 sequences from chr9\n",
      "\tExtracting 291 sequences from chr10\n",
      "\tExtracting 381 sequences from chr11\n",
      "\tExtracting 199 sequences from chr12\n",
      "\tExtracting 244 sequences from chr13\n",
      "\tExtracting 188 sequences from chr14\n",
      "\tExtracting 196 sequences from chr15\n",
      "\tExtracting 208 sequences from chr16\n",
      "\tExtracting 263 sequences from chr17\n",
      "\tExtracting 137 sequences from chr18\n",
      "\tExtracting 160 sequences from chr19\n",
      "\tExtracting 54 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5225 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5226\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5226\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 347 sequences from chr1\n",
      "\tExtracting 387 sequences from chr2\n",
      "\tExtracting 264 sequences from chr3\n",
      "\tExtracting 373 sequences from chr4\n",
      "\tExtracting 364 sequences from chr5\n",
      "\tExtracting 242 sequences from chr6\n",
      "\tExtracting 393 sequences from chr7\n",
      "\tExtracting 296 sequences from chr8\n",
      "\tExtracting 273 sequences from chr9\n",
      "\tExtracting 294 sequences from chr10\n",
      "\tExtracting 363 sequences from chr11\n",
      "\tExtracting 189 sequences from chr12\n",
      "\tExtracting 233 sequences from chr13\n",
      "\tExtracting 193 sequences from chr14\n",
      "\tExtracting 213 sequences from chr15\n",
      "\tExtracting 205 sequences from chr16\n",
      "\tExtracting 238 sequences from chr17\n",
      "\tExtracting 148 sequences from chr18\n",
      "\tExtracting 179 sequences from chr19\n",
      "\tExtracting 32 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5226 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5226\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5226\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 372 sequences from chr1\n",
      "\tExtracting 369 sequences from chr2\n",
      "\tExtracting 297 sequences from chr3\n",
      "\tExtracting 296 sequences from chr4\n",
      "\tExtracting 341 sequences from chr5\n",
      "\tExtracting 271 sequences from chr6\n",
      "\tExtracting 420 sequences from chr7\n",
      "\tExtracting 266 sequences from chr8\n",
      "\tExtracting 308 sequences from chr9\n",
      "\tExtracting 319 sequences from chr10\n",
      "\tExtracting 378 sequences from chr11\n",
      "\tExtracting 237 sequences from chr12\n",
      "\tExtracting 240 sequences from chr13\n",
      "\tExtracting 183 sequences from chr14\n",
      "\tExtracting 184 sequences from chr15\n",
      "\tExtracting 202 sequences from chr16\n",
      "\tExtracting 237 sequences from chr17\n",
      "\tExtracting 149 sequences from chr18\n",
      "\tExtracting 134 sequences from chr19\n",
      "\tExtracting 23 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5226 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset30/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset30/expression.tsv.gz tmp_dynamic/Subset30/motifs.bed tmp_dynamic/Subset30/wellington.tsv.gz tmp_dynamic/Subset30/homer.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset31/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset31/expression.tsv.gz tmp_dynamic/Subset31/motifs.bed tmp_dynamic/Subset31/wellington.tsv.gz tmp_dynamic/Subset31/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5390\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5390\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 360 sequences from chr1\n",
      "\tExtracting 396 sequences from chr2\n",
      "\tExtracting 281 sequences from chr3\n",
      "\tExtracting 352 sequences from chr4\n",
      "\tExtracting 349 sequences from chr5\n",
      "\tExtracting 267 sequences from chr6\n",
      "\tExtracting 413 sequences from chr7\n",
      "\tExtracting 297 sequences from chr8\n",
      "\tExtracting 302 sequences from chr9\n",
      "\tExtracting 304 sequences from chr10\n",
      "\tExtracting 401 sequences from chr11\n",
      "\tExtracting 212 sequences from chr12\n",
      "\tExtracting 243 sequences from chr13\n",
      "\tExtracting 209 sequences from chr14\n",
      "\tExtracting 217 sequences from chr15\n",
      "\tExtracting 190 sequences from chr16\n",
      "\tExtracting 242 sequences from chr17\n",
      "\tExtracting 148 sequences from chr18\n",
      "\tExtracting 161 sequences from chr19\n",
      "\tExtracting 46 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5390 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5390\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5390\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 391 sequences from chr1\n",
      "\tExtracting 386 sequences from chr2\n",
      "\tExtracting 289 sequences from chr3\n",
      "\tExtracting 336 sequences from chr4\n",
      "\tExtracting 353 sequences from chr5\n",
      "\tExtracting 281 sequences from chr6\n",
      "\tExtracting 409 sequences from chr7\n",
      "\tExtracting 293 sequences from chr8\n",
      "\tExtracting 315 sequences from chr9\n",
      "\tExtracting 309 sequences from chr10\n",
      "\tExtracting 393 sequences from chr11\n",
      "\tExtracting 250 sequences from chr12\n",
      "\tExtracting 249 sequences from chr13\n",
      "\tExtracting 193 sequences from chr14\n",
      "\tExtracting 198 sequences from chr15\n",
      "\tExtracting 195 sequences from chr16\n",
      "\tExtracting 236 sequences from chr17\n",
      "\tExtracting 142 sequences from chr18\n",
      "\tExtracting 149 sequences from chr19\n",
      "\tExtracting 23 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5390 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5386\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5386\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 335 sequences from chr1\n",
      "\tExtracting 399 sequences from chr2\n",
      "\tExtracting 276 sequences from chr3\n",
      "\tExtracting 345 sequences from chr4\n",
      "\tExtracting 341 sequences from chr5\n",
      "\tExtracting 285 sequences from chr6\n",
      "\tExtracting 396 sequences from chr7\n",
      "\tExtracting 290 sequences from chr8\n",
      "\tExtracting 308 sequences from chr9\n",
      "\tExtracting 316 sequences from chr10\n",
      "\tExtracting 399 sequences from chr11\n",
      "\tExtracting 204 sequences from chr12\n",
      "\tExtracting 232 sequences from chr13\n",
      "\tExtracting 199 sequences from chr14\n",
      "\tExtracting 212 sequences from chr15\n",
      "\tExtracting 201 sequences from chr16\n",
      "\tExtracting 274 sequences from chr17\n",
      "\tExtracting 128 sequences from chr18\n",
      "\tExtracting 180 sequences from chr19\n",
      "\tExtracting 66 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5386 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5390\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5390\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 364 sequences from chr1\n",
      "\tExtracting 397 sequences from chr2\n",
      "\tExtracting 307 sequences from chr3\n",
      "\tExtracting 355 sequences from chr4\n",
      "\tExtracting 373 sequences from chr5\n",
      "\tExtracting 255 sequences from chr6\n",
      "\tExtracting 408 sequences from chr7\n",
      "\tExtracting 276 sequences from chr8\n",
      "\tExtracting 294 sequences from chr9\n",
      "\tExtracting 314 sequences from chr10\n",
      "\tExtracting 391 sequences from chr11\n",
      "\tExtracting 204 sequences from chr12\n",
      "\tExtracting 222 sequences from chr13\n",
      "\tExtracting 182 sequences from chr14\n",
      "\tExtracting 218 sequences from chr15\n",
      "\tExtracting 205 sequences from chr16\n",
      "\tExtracting 236 sequences from chr17\n",
      "\tExtracting 162 sequences from chr18\n",
      "\tExtracting 184 sequences from chr19\n",
      "\tExtracting 43 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5390 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset32/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset32/expression.tsv.gz tmp_dynamic/Subset32/motifs.bed tmp_dynamic/Subset32/wellington.tsv.gz tmp_dynamic/Subset32/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5484\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5484\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 348 sequences from chr1\n",
      "\tExtracting 417 sequences from chr2\n",
      "\tExtracting 284 sequences from chr3\n",
      "\tExtracting 346 sequences from chr4\n",
      "\tExtracting 349 sequences from chr5\n",
      "\tExtracting 254 sequences from chr6\n",
      "\tExtracting 432 sequences from chr7\n",
      "\tExtracting 293 sequences from chr8\n",
      "\tExtracting 301 sequences from chr9\n",
      "\tExtracting 334 sequences from chr10\n",
      "\tExtracting 403 sequences from chr11\n",
      "\tExtracting 226 sequences from chr12\n",
      "\tExtracting 219 sequences from chr13\n",
      "\tExtracting 211 sequences from chr14\n",
      "\tExtracting 211 sequences from chr15\n",
      "\tExtracting 193 sequences from chr16\n",
      "\tExtracting 272 sequences from chr17\n",
      "\tExtracting 151 sequences from chr18\n",
      "\tExtracting 175 sequences from chr19\n",
      "\tExtracting 65 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5484 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5487\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5487\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 361 sequences from chr1\n",
      "\tExtracting 402 sequences from chr2\n",
      "\tExtracting 292 sequences from chr3\n",
      "\tExtracting 357 sequences from chr4\n",
      "\tExtracting 357 sequences from chr5\n",
      "\tExtracting 283 sequences from chr6\n",
      "\tExtracting 412 sequences from chr7\n",
      "\tExtracting 299 sequences from chr8\n",
      "\tExtracting 302 sequences from chr9\n",
      "\tExtracting 319 sequences from chr10\n",
      "\tExtracting 417 sequences from chr11\n",
      "\tExtracting 231 sequences from chr12\n",
      "\tExtracting 252 sequences from chr13\n",
      "\tExtracting 205 sequences from chr14\n",
      "\tExtracting 224 sequences from chr15\n",
      "\tExtracting 188 sequences from chr16\n",
      "\tExtracting 218 sequences from chr17\n",
      "\tExtracting 151 sequences from chr18\n",
      "\tExtracting 177 sequences from chr19\n",
      "\tExtracting 40 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5487 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5487\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5487\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 377 sequences from chr1\n",
      "\tExtracting 418 sequences from chr2\n",
      "\tExtracting 291 sequences from chr3\n",
      "\tExtracting 353 sequences from chr4\n",
      "\tExtracting 367 sequences from chr5\n",
      "\tExtracting 271 sequences from chr6\n",
      "\tExtracting 417 sequences from chr7\n",
      "\tExtracting 288 sequences from chr8\n",
      "\tExtracting 329 sequences from chr9\n",
      "\tExtracting 299 sequences from chr10\n",
      "\tExtracting 386 sequences from chr11\n",
      "\tExtracting 206 sequences from chr12\n",
      "\tExtracting 248 sequences from chr13\n",
      "\tExtracting 174 sequences from chr14\n",
      "\tExtracting 228 sequences from chr15\n",
      "\tExtracting 215 sequences from chr16\n",
      "\tExtracting 263 sequences from chr17\n",
      "\tExtracting 147 sequences from chr18\n",
      "\tExtracting 166 sequences from chr19\n",
      "\tExtracting 44 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5487 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5487\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5487\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 400 sequences from chr1\n",
      "\tExtracting 377 sequences from chr2\n",
      "\tExtracting 297 sequences from chr3\n",
      "\tExtracting 344 sequences from chr4\n",
      "\tExtracting 362 sequences from chr5\n",
      "\tExtracting 283 sequences from chr6\n",
      "\tExtracting 432 sequences from chr7\n",
      "\tExtracting 288 sequences from chr8\n",
      "\tExtracting 304 sequences from chr9\n",
      "\tExtracting 316 sequences from chr10\n",
      "\tExtracting 391 sequences from chr11\n",
      "\tExtracting 252 sequences from chr12\n",
      "\tExtracting 247 sequences from chr13\n",
      "\tExtracting 203 sequences from chr14\n",
      "\tExtracting 197 sequences from chr15\n",
      "\tExtracting 202 sequences from chr16\n",
      "\tExtracting 244 sequences from chr17\n",
      "\tExtracting 155 sequences from chr18\n",
      "\tExtracting 166 sequences from chr19\n",
      "\tExtracting 27 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5487 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5984\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5984\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 360 sequences from chr1\n",
      "\tExtracting 480 sequences from chr2\n",
      "\tExtracting 336 sequences from chr3\n",
      "\tExtracting 395 sequences from chr4\n",
      "\tExtracting 380 sequences from chr5\n",
      "\tExtracting 275 sequences from chr6\n",
      "\tExtracting 423 sequences from chr7\n",
      "\tExtracting 301 sequences from chr8\n",
      "\tExtracting 371 sequences from chr9\n",
      "\tExtracting 314 sequences from chr10\n",
      "\tExtracting 453 sequences from chr11\n",
      "\tExtracting 273 sequences from chr12\n",
      "\tExtracting 290 sequences from chr13\n",
      "\tExtracting 229 sequences from chr14\n",
      "\tExtracting 242 sequences from chr15\n",
      "\tExtracting 215 sequences from chr16\n",
      "\tExtracting 245 sequences from chr17\n",
      "\tExtracting 160 sequences from chr18\n",
      "\tExtracting 177 sequences from chr19\n",
      "\tExtracting 65 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5984 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5988\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5988\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 408 sequences from chr1\n",
      "\tExtracting 435 sequences from chr2\n",
      "\tExtracting 326 sequences from chr3\n",
      "\tExtracting 386 sequences from chr4\n",
      "\tExtracting 406 sequences from chr5\n",
      "\tExtracting 292 sequences from chr6\n",
      "\tExtracting 479 sequences from chr7\n",
      "\tExtracting 312 sequences from chr8\n",
      "\tExtracting 338 sequences from chr9\n",
      "\tExtracting 303 sequences from chr10\n",
      "\tExtracting 434 sequences from chr11\n",
      "\tExtracting 228 sequences from chr12\n",
      "\tExtracting 270 sequences from chr13\n",
      "\tExtracting 217 sequences from chr14\n",
      "\tExtracting 230 sequences from chr15\n",
      "\tExtracting 222 sequences from chr16\n",
      "\tExtracting 308 sequences from chr17\n",
      "\tExtracting 160 sequences from chr18\n",
      "\tExtracting 194 sequences from chr19\n",
      "\tExtracting 40 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5988 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5988\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5988\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 365 sequences from chr1\n",
      "\tExtracting 441 sequences from chr2\n",
      "\tExtracting 306 sequences from chr3\n",
      "\tExtracting 378 sequences from chr4\n",
      "\tExtracting 390 sequences from chr5\n",
      "\tExtracting 312 sequences from chr6\n",
      "\tExtracting 439 sequences from chr7\n",
      "\tExtracting 342 sequences from chr8\n",
      "\tExtracting 329 sequences from chr9\n",
      "\tExtracting 362 sequences from chr10\n",
      "\tExtracting 435 sequences from chr11\n",
      "\tExtracting 285 sequences from chr12\n",
      "\tExtracting 246 sequences from chr13\n",
      "\tExtracting 232 sequences from chr14\n",
      "\tExtracting 227 sequences from chr15\n",
      "\tExtracting 228 sequences from chr16\n",
      "\tExtracting 278 sequences from chr17\n",
      "\tExtracting 152 sequences from chr18\n",
      "\tExtracting 187 sequences from chr19\n",
      "\tExtracting 54 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5988 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5988\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5988\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 432 sequences from chr1\n",
      "\tExtracting 406 sequences from chr2\n",
      "\tExtracting 320 sequences from chr3\n",
      "\tExtracting 373 sequences from chr4\n",
      "\tExtracting 384 sequences from chr5\n",
      "\tExtracting 315 sequences from chr6\n",
      "\tExtracting 462 sequences from chr7\n",
      "\tExtracting 311 sequences from chr8\n",
      "\tExtracting 334 sequences from chr9\n",
      "\tExtracting 377 sequences from chr10\n",
      "\tExtracting 431 sequences from chr11\n",
      "\tExtracting 260 sequences from chr12\n",
      "\tExtracting 274 sequences from chr13\n",
      "\tExtracting 206 sequences from chr14\n",
      "\tExtracting 229 sequences from chr15\n",
      "\tExtracting 223 sequences from chr16\n",
      "\tExtracting 265 sequences from chr17\n",
      "\tExtracting 174 sequences from chr18\n",
      "\tExtracting 171 sequences from chr19\n",
      "\tExtracting 41 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5988 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset33/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset33/expression.tsv.gz tmp_dynamic/Subset33/motifs.bed tmp_dynamic/Subset33/wellington.tsv.gz tmp_dynamic/Subset33/homer.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_dynamic/Subset62/reads.bam tmp_dynamic/Subset62/reads.bai tmp_dynamic/Subset62/peaks.bed tmp_dynamic/Subset62/footprints.bed\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6408\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6408\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 412 sequences from chr1\n",
      "\tExtracting 499 sequences from chr2\n",
      "\tExtracting 374 sequences from chr3\n",
      "\tExtracting 385 sequences from chr4\n",
      "\tExtracting 415 sequences from chr5\n",
      "\tExtracting 303 sequences from chr6\n",
      "\tExtracting 452 sequences from chr7\n",
      "\tExtracting 368 sequences from chr8\n",
      "\tExtracting 373 sequences from chr9\n",
      "\tExtracting 365 sequences from chr10\n",
      "\tExtracting 476 sequences from chr11\n",
      "\tExtracting 268 sequences from chr12\n",
      "\tExtracting 264 sequences from chr13\n",
      "\tExtracting 236 sequences from chr14\n",
      "\tExtracting 279 sequences from chr15\n",
      "\tExtracting 209 sequences from chr16\n",
      "\tExtracting 290 sequences from chr17\n",
      "\tExtracting 165 sequences from chr18\n",
      "\tExtracting 191 sequences from chr19\n",
      "\tExtracting 84 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6408 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6409\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6409\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 435 sequences from chr1\n",
      "\tExtracting 505 sequences from chr2\n",
      "\tExtracting 317 sequences from chr3\n",
      "\tExtracting 386 sequences from chr4\n",
      "\tExtracting 429 sequences from chr5\n",
      "\tExtracting 316 sequences from chr6\n",
      "\tExtracting 466 sequences from chr7\n",
      "\tExtracting 325 sequences from chr8\n",
      "\tExtracting 362 sequences from chr9\n",
      "\tExtracting 374 sequences from chr10\n",
      "\tExtracting 479 sequences from chr11\n",
      "\tExtracting 271 sequences from chr12\n",
      "\tExtracting 285 sequences from chr13\n",
      "\tExtracting 236 sequences from chr14\n",
      "\tExtracting 257 sequences from chr15\n",
      "\tExtracting 258 sequences from chr16\n",
      "\tExtracting 321 sequences from chr17\n",
      "\tExtracting 147 sequences from chr18\n",
      "\tExtracting 195 sequences from chr19\n",
      "\tExtracting 45 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6409 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6409\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6409\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 373 sequences from chr1\n",
      "\tExtracting 457 sequences from chr2\n",
      "\tExtracting 316 sequences from chr3\n",
      "\tExtracting 409 sequences from chr4\n",
      "\tExtracting 445 sequences from chr5\n",
      "\tExtracting 327 sequences from chr6\n",
      "\tExtracting 503 sequences from chr7\n",
      "\tExtracting 349 sequences from chr8\n",
      "\tExtracting 337 sequences from chr9\n",
      "\tExtracting 345 sequences from chr10\n",
      "\tExtracting 467 sequences from chr11\n",
      "\tExtracting 287 sequences from chr12\n",
      "\tExtracting 279 sequences from chr13\n",
      "\tExtracting 265 sequences from chr14\n",
      "\tExtracting 232 sequences from chr15\n",
      "\tExtracting 243 sequences from chr16\n",
      "\tExtracting 309 sequences from chr17\n",
      "\tExtracting 187 sequences from chr18\n",
      "\tExtracting 206 sequences from chr19\n",
      "\tExtracting 73 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6409 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6409\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6409\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 452 sequences from chr1\n",
      "\tExtracting 427 sequences from chr2\n",
      "\tExtracting 368 sequences from chr3\n",
      "\tExtracting 411 sequences from chr4\n",
      "\tExtracting 421 sequences from chr5\n",
      "\tExtracting 322 sequences from chr6\n",
      "\tExtracting 491 sequences from chr7\n",
      "\tExtracting 327 sequences from chr8\n",
      "\tExtracting 362 sequences from chr9\n",
      "\tExtracting 385 sequences from chr10\n",
      "\tExtracting 467 sequences from chr11\n",
      "\tExtracting 274 sequences from chr12\n",
      "\tExtracting 301 sequences from chr13\n",
      "\tExtracting 225 sequences from chr14\n",
      "\tExtracting 232 sequences from chr15\n",
      "\tExtracting 242 sequences from chr16\n",
      "\tExtracting 286 sequences from chr17\n",
      "\tExtracting 189 sequences from chr18\n",
      "\tExtracting 189 sequences from chr19\n",
      "\tExtracting 38 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6409 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset1/wellington.tsv.gz tmp_dynamic/Subset1/homer.tsv.gz tmp_dynamic/Subset1/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset1/expression.tsv.gz tmp_dynamic/Subset1/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset1/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset2/wellington.tsv.gz tmp_dynamic/Subset2/homer.tsv.gz tmp_dynamic/Subset2/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset2/expression.tsv.gz tmp_dynamic/Subset2/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset2/tssdist.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6597\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6597\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 399 sequences from chr1\n",
      "\tExtracting 492 sequences from chr2\n",
      "\tExtracting 392 sequences from chr3\n",
      "\tExtracting 440 sequences from chr4\n",
      "\tExtracting 437 sequences from chr5\n",
      "\tExtracting 315 sequences from chr6\n",
      "\tExtracting 491 sequences from chr7\n",
      "\tExtracting 345 sequences from chr8\n",
      "\tExtracting 360 sequences from chr9\n",
      "\tExtracting 385 sequences from chr10\n",
      "\tExtracting 498 sequences from chr11\n",
      "\tExtracting 265 sequences from chr12\n",
      "\tExtracting 307 sequences from chr13\n",
      "\tExtracting 229 sequences from chr14\n",
      "\tExtracting 271 sequences from chr15\n",
      "\tExtracting 227 sequences from chr16\n",
      "\tExtracting 286 sequences from chr17\n",
      "\tExtracting 164 sequences from chr18\n",
      "\tExtracting 210 sequences from chr19\n",
      "\tExtracting 84 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6597 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6598\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6598\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 424 sequences from chr1\n",
      "\tExtracting 506 sequences from chr2\n",
      "\tExtracting 334 sequences from chr3\n",
      "\tExtracting 418 sequences from chr4\n",
      "\tExtracting 420 sequences from chr5\n",
      "\tExtracting 329 sequences from chr6\n",
      "\tExtracting 500 sequences from chr7\n",
      "\tExtracting 352 sequences from chr8\n",
      "\tExtracting 359 sequences from chr9\n",
      "\tExtracting 380 sequences from chr10\n",
      "\tExtracting 475 sequences from chr11\n",
      "\tExtracting 272 sequences from chr12\n",
      "\tExtracting 306 sequences from chr13\n",
      "\tExtracting 262 sequences from chr14\n",
      "\tExtracting 265 sequences from chr15\n",
      "\tExtracting 250 sequences from chr16\n",
      "\tExtracting 309 sequences from chr17\n",
      "\tExtracting 162 sequences from chr18\n",
      "\tExtracting 223 sequences from chr19\n",
      "\tExtracting 52 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6598 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6598\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6598\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 411 sequences from chr1\n",
      "\tExtracting 510 sequences from chr2\n",
      "\tExtracting 336 sequences from chr3\n",
      "\tExtracting 420 sequences from chr4\n",
      "\tExtracting 439 sequences from chr5\n",
      "\tExtracting 326 sequences from chr6\n",
      "\tExtracting 471 sequences from chr7\n",
      "\tExtracting 382 sequences from chr8\n",
      "\tExtracting 375 sequences from chr9\n",
      "\tExtracting 348 sequences from chr10\n",
      "\tExtracting 482 sequences from chr11\n",
      "\tExtracting 286 sequences from chr12\n",
      "\tExtracting 286 sequences from chr13\n",
      "\tExtracting 259 sequences from chr14\n",
      "\tExtracting 249 sequences from chr15\n",
      "\tExtracting 252 sequences from chr16\n",
      "\tExtracting 317 sequences from chr17\n",
      "\tExtracting 180 sequences from chr18\n",
      "\tExtracting 195 sequences from chr19\n",
      "\tExtracting 74 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6598 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6598\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6598\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 477 sequences from chr1\n",
      "\tExtracting 448 sequences from chr2\n",
      "\tExtracting 371 sequences from chr3\n",
      "\tExtracting 419 sequences from chr4\n",
      "\tExtracting 440 sequences from chr5\n",
      "\tExtracting 332 sequences from chr6\n",
      "\tExtracting 510 sequences from chr7\n",
      "\tExtracting 334 sequences from chr8\n",
      "\tExtracting 371 sequences from chr9\n",
      "\tExtracting 401 sequences from chr10\n",
      "\tExtracting 480 sequences from chr11\n",
      "\tExtracting 275 sequences from chr12\n",
      "\tExtracting 281 sequences from chr13\n",
      "\tExtracting 222 sequences from chr14\n",
      "\tExtracting 250 sequences from chr15\n",
      "\tExtracting 251 sequences from chr16\n",
      "\tExtracting 307 sequences from chr17\n",
      "\tExtracting 196 sequences from chr18\n",
      "\tExtracting 194 sequences from chr19\n",
      "\tExtracting 39 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6598 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset4/wellington.tsv.gz tmp_dynamic/Subset4/homer.tsv.gz tmp_dynamic/Subset4/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset4/expression.tsv.gz tmp_dynamic/Subset4/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset4/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset5/wellington.tsv.gz tmp_dynamic/Subset5/homer.tsv.gz tmp_dynamic/Subset5/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset5/expression.tsv.gz tmp_dynamic/Subset5/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset5/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset6/wellington.tsv.gz tmp_dynamic/Subset6/homer.tsv.gz tmp_dynamic/Subset6/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset6/expression.tsv.gz tmp_dynamic/Subset6/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset6/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset7/wellington.tsv.gz tmp_dynamic/Subset7/homer.tsv.gz tmp_dynamic/Subset7/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset7/expression.tsv.gz tmp_dynamic/Subset7/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset7/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset8/wellington.tsv.gz tmp_dynamic/Subset8/homer.tsv.gz tmp_dynamic/Subset8/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset8/expression.tsv.gz tmp_dynamic/Subset8/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset8/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset9/wellington.tsv.gz tmp_dynamic/Subset9/homer.tsv.gz tmp_dynamic/Subset9/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset9/expression.tsv.gz tmp_dynamic/Subset9/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset9/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset10/wellington.tsv.gz tmp_dynamic/Subset10/homer.tsv.gz tmp_dynamic/Subset10/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset10/expression.tsv.gz tmp_dynamic/Subset10/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset10/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset11/wellington.tsv.gz tmp_dynamic/Subset11/homer.tsv.gz tmp_dynamic/Subset11/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset11/expression.tsv.gz tmp_dynamic/Subset11/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset11/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset12/wellington.tsv.gz tmp_dynamic/Subset12/homer.tsv.gz tmp_dynamic/Subset12/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset12/expression.tsv.gz tmp_dynamic/Subset12/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset12/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset13/wellington.tsv.gz tmp_dynamic/Subset13/homer.tsv.gz tmp_dynamic/Subset13/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset13/expression.tsv.gz tmp_dynamic/Subset13/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset13/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset14/wellington.tsv.gz tmp_dynamic/Subset14/homer.tsv.gz tmp_dynamic/Subset14/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset14/expression.tsv.gz tmp_dynamic/Subset14/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset14/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset15/wellington.tsv.gz tmp_dynamic/Subset15/homer.tsv.gz tmp_dynamic/Subset15/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset15/expression.tsv.gz tmp_dynamic/Subset15/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset15/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset16/wellington.tsv.gz tmp_dynamic/Subset16/homer.tsv.gz tmp_dynamic/Subset16/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset16/expression.tsv.gz tmp_dynamic/Subset16/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset16/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset17/wellington.tsv.gz tmp_dynamic/Subset17/homer.tsv.gz tmp_dynamic/Subset17/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset17/expression.tsv.gz tmp_dynamic/Subset17/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset17/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset18/wellington.tsv.gz tmp_dynamic/Subset18/homer.tsv.gz tmp_dynamic/Subset18/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset18/expression.tsv.gz tmp_dynamic/Subset18/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset18/tssdist.tsv.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset19/wellington.tsv.gz tmp_dynamic/Subset19/homer.tsv.gz tmp_dynamic/Subset19/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset19/expression.tsv.gz tmp_dynamic/Subset19/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset19/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset20/wellington.tsv.gz tmp_dynamic/Subset20/homer.tsv.gz tmp_dynamic/Subset20/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset20/expression.tsv.gz tmp_dynamic/Subset20/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset20/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset21/wellington.tsv.gz tmp_dynamic/Subset21/homer.tsv.gz tmp_dynamic/Subset21/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset21/expression.tsv.gz tmp_dynamic/Subset21/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset21/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset22/wellington.tsv.gz tmp_dynamic/Subset22/homer.tsv.gz tmp_dynamic/Subset22/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset22/expression.tsv.gz tmp_dynamic/Subset22/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset22/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset23/wellington.tsv.gz tmp_dynamic/Subset23/homer.tsv.gz tmp_dynamic/Subset23/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset23/expression.tsv.gz tmp_dynamic/Subset23/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset23/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset24/wellington.tsv.gz tmp_dynamic/Subset24/homer.tsv.gz tmp_dynamic/Subset24/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset24/expression.tsv.gz tmp_dynamic/Subset24/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset24/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset25/wellington.tsv.gz tmp_dynamic/Subset25/homer.tsv.gz tmp_dynamic/Subset25/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset25/expression.tsv.gz tmp_dynamic/Subset25/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset25/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset26/wellington.tsv.gz tmp_dynamic/Subset26/homer.tsv.gz tmp_dynamic/Subset26/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset26/expression.tsv.gz tmp_dynamic/Subset26/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset26/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset27/wellington.tsv.gz tmp_dynamic/Subset27/homer.tsv.gz tmp_dynamic/Subset27/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset27/expression.tsv.gz tmp_dynamic/Subset27/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset27/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset28/wellington.tsv.gz tmp_dynamic/Subset28/homer.tsv.gz tmp_dynamic/Subset28/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset28/expression.tsv.gz tmp_dynamic/Subset28/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset28/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset29/wellington.tsv.gz tmp_dynamic/Subset29/homer.tsv.gz tmp_dynamic/Subset29/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset29/expression.tsv.gz tmp_dynamic/Subset29/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset29/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset30/wellington.tsv.gz tmp_dynamic/Subset30/homer.tsv.gz tmp_dynamic/Subset30/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset30/expression.tsv.gz tmp_dynamic/Subset30/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset30/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset31/wellington.tsv.gz tmp_dynamic/Subset31/homer.tsv.gz tmp_dynamic/Subset31/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset31/expression.tsv.gz tmp_dynamic/Subset31/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset31/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset32/wellington.tsv.gz tmp_dynamic/Subset32/homer.tsv.gz tmp_dynamic/Subset32/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset32/expression.tsv.gz tmp_dynamic/Subset32/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset32/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset33/wellington.tsv.gz tmp_dynamic/Subset33/homer.tsv.gz tmp_dynamic/Subset33/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset33/expression.tsv.gz tmp_dynamic/Subset33/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset33/tssdist.tsv.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset1/binding.tsv.gz tmp_dynamic/Subset1/tssdist.tsv.gz tmp_dynamic/Subset1/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset2/binding.tsv.gz tmp_dynamic/Subset2/tssdist.tsv.gz tmp_dynamic/Subset2/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset4/binding.tsv.gz tmp_dynamic/Subset4/tssdist.tsv.gz tmp_dynamic/Subset4/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset5/binding.tsv.gz tmp_dynamic/Subset5/tssdist.tsv.gz tmp_dynamic/Subset5/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset6/binding.tsv.gz tmp_dynamic/Subset6/tssdist.tsv.gz tmp_dynamic/Subset6/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset7/binding.tsv.gz tmp_dynamic/Subset7/tssdist.tsv.gz tmp_dynamic/Subset7/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset8/binding.tsv.gz tmp_dynamic/Subset8/tssdist.tsv.gz tmp_dynamic/Subset8/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset9/binding.tsv.gz tmp_dynamic/Subset9/tssdist.tsv.gz tmp_dynamic/Subset9/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset10/binding.tsv.gz tmp_dynamic/Subset10/tssdist.tsv.gz tmp_dynamic/Subset10/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset11/binding.tsv.gz tmp_dynamic/Subset11/tssdist.tsv.gz tmp_dynamic/Subset11/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset12/binding.tsv.gz tmp_dynamic/Subset12/tssdist.tsv.gz tmp_dynamic/Subset12/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset13/binding.tsv.gz tmp_dynamic/Subset13/tssdist.tsv.gz tmp_dynamic/Subset13/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset14/binding.tsv.gz tmp_dynamic/Subset14/tssdist.tsv.gz tmp_dynamic/Subset14/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset15/binding.tsv.gz tmp_dynamic/Subset15/tssdist.tsv.gz tmp_dynamic/Subset15/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset16/binding.tsv.gz tmp_dynamic/Subset16/tssdist.tsv.gz tmp_dynamic/Subset16/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset17/binding.tsv.gz tmp_dynamic/Subset17/tssdist.tsv.gz tmp_dynamic/Subset17/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset18/binding.tsv.gz tmp_dynamic/Subset18/tssdist.tsv.gz tmp_dynamic/Subset18/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset19/binding.tsv.gz tmp_dynamic/Subset19/tssdist.tsv.gz tmp_dynamic/Subset19/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset20/binding.tsv.gz tmp_dynamic/Subset20/tssdist.tsv.gz tmp_dynamic/Subset20/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset21/binding.tsv.gz tmp_dynamic/Subset21/tssdist.tsv.gz tmp_dynamic/Subset21/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset22/binding.tsv.gz tmp_dynamic/Subset22/tssdist.tsv.gz tmp_dynamic/Subset22/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset23/binding.tsv.gz tmp_dynamic/Subset23/tssdist.tsv.gz tmp_dynamic/Subset23/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset24/binding.tsv.gz tmp_dynamic/Subset24/tssdist.tsv.gz tmp_dynamic/Subset24/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset25/binding.tsv.gz tmp_dynamic/Subset25/tssdist.tsv.gz tmp_dynamic/Subset25/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset26/binding.tsv.gz tmp_dynamic/Subset26/tssdist.tsv.gz tmp_dynamic/Subset26/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset27/binding.tsv.gz tmp_dynamic/Subset27/tssdist.tsv.gz tmp_dynamic/Subset27/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset28/binding.tsv.gz tmp_dynamic/Subset28/tssdist.tsv.gz tmp_dynamic/Subset28/linking.tsv.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset29/binding.tsv.gz tmp_dynamic/Subset29/tssdist.tsv.gz tmp_dynamic/Subset29/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset30/binding.tsv.gz tmp_dynamic/Subset30/tssdist.tsv.gz tmp_dynamic/Subset30/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset31/binding.tsv.gz tmp_dynamic/Subset31/tssdist.tsv.gz tmp_dynamic/Subset31/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset32/binding.tsv.gz tmp_dynamic/Subset32/tssdist.tsv.gz tmp_dynamic/Subset32/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset33/binding.tsv.gz tmp_dynamic/Subset33/tssdist.tsv.gz tmp_dynamic/Subset33/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset1/linking.tsv.gz tmp_dynamic/Subset1/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset2/linking.tsv.gz tmp_dynamic/Subset2/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset4/linking.tsv.gz tmp_dynamic/Subset4/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset5/linking.tsv.gz tmp_dynamic/Subset5/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset6/linking.tsv.gz tmp_dynamic/Subset6/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset7/linking.tsv.gz tmp_dynamic/Subset7/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset8/linking.tsv.gz tmp_dynamic/Subset8/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset9/linking.tsv.gz tmp_dynamic/Subset9/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset10/linking.tsv.gz tmp_dynamic/Subset10/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset11/linking.tsv.gz tmp_dynamic/Subset11/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset12/linking.tsv.gz tmp_dynamic/Subset12/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset13/linking.tsv.gz tmp_dynamic/Subset13/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset14/linking.tsv.gz tmp_dynamic/Subset14/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset15/linking.tsv.gz tmp_dynamic/Subset15/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset16/linking.tsv.gz tmp_dynamic/Subset16/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset17/linking.tsv.gz tmp_dynamic/Subset17/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset18/linking.tsv.gz tmp_dynamic/Subset18/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset19/linking.tsv.gz tmp_dynamic/Subset19/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset20/linking.tsv.gz tmp_dynamic/Subset20/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset21/linking.tsv.gz tmp_dynamic/Subset21/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset22/linking.tsv.gz tmp_dynamic/Subset22/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset23/linking.tsv.gz tmp_dynamic/Subset23/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset24/linking.tsv.gz tmp_dynamic/Subset24/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset25/linking.tsv.gz tmp_dynamic/Subset25/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset26/linking.tsv.gz tmp_dynamic/Subset26/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset27/linking.tsv.gz tmp_dynamic/Subset27/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset28/linking.tsv.gz tmp_dynamic/Subset28/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset29/linking.tsv.gz tmp_dynamic/Subset29/binlinking.tsv.gz 20\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset30/linking.tsv.gz tmp_dynamic/Subset30/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset31/linking.tsv.gz tmp_dynamic/Subset31/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset32/linking.tsv.gz tmp_dynamic/Subset32/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset33/linking.tsv.gz tmp_dynamic/Subset33/binlinking.tsv.gz 20\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset61/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset61/expression.tsv.gz tmp_dynamic/Subset61/motifs.bed tmp_dynamic/Subset61/wellington.tsv.gz tmp_dynamic/Subset61/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4125\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4125\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 284 sequences from chr1\n",
      "\tExtracting 337 sequences from chr2\n",
      "\tExtracting 238 sequences from chr3\n",
      "\tExtracting 258 sequences from chr4\n",
      "\tExtracting 260 sequences from chr5\n",
      "\tExtracting 187 sequences from chr6\n",
      "\tExtracting 329 sequences from chr7\n",
      "\tExtracting 198 sequences from chr8\n",
      "\tExtracting 233 sequences from chr9\n",
      "\tExtracting 239 sequences from chr10\n",
      "\tExtracting 297 sequences from chr11\n",
      "\tExtracting 170 sequences from chr12\n",
      "\tExtracting 173 sequences from chr13\n",
      "\tExtracting 135 sequences from chr14\n",
      "\tExtracting 161 sequences from chr15\n",
      "\tExtracting 171 sequences from chr16\n",
      "\tExtracting 174 sequences from chr17\n",
      "\tExtracting 120 sequences from chr18\n",
      "\tExtracting 133 sequences from chr19\n",
      "\tExtracting 28 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4125 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4121\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4121\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 287 sequences from chr1\n",
      "\tExtracting 331 sequences from chr2\n",
      "\tExtracting 235 sequences from chr3\n",
      "\tExtracting 260 sequences from chr4\n",
      "\tExtracting 274 sequences from chr5\n",
      "\tExtracting 196 sequences from chr6\n",
      "\tExtracting 309 sequences from chr7\n",
      "\tExtracting 215 sequences from chr8\n",
      "\tExtracting 230 sequences from chr9\n",
      "\tExtracting 238 sequences from chr10\n",
      "\tExtracting 278 sequences from chr11\n",
      "\tExtracting 170 sequences from chr12\n",
      "\tExtracting 175 sequences from chr13\n",
      "\tExtracting 138 sequences from chr14\n",
      "\tExtracting 174 sequences from chr15\n",
      "\tExtracting 147 sequences from chr16\n",
      "\tExtracting 175 sequences from chr17\n",
      "\tExtracting 112 sequences from chr18\n",
      "\tExtracting 133 sequences from chr19\n",
      "\tExtracting 44 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4121 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4125\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4125\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 318 sequences from chr1\n",
      "\tExtracting 285 sequences from chr2\n",
      "\tExtracting 211 sequences from chr3\n",
      "\tExtracting 265 sequences from chr4\n",
      "\tExtracting 259 sequences from chr5\n",
      "\tExtracting 211 sequences from chr6\n",
      "\tExtracting 309 sequences from chr7\n",
      "\tExtracting 229 sequences from chr8\n",
      "\tExtracting 233 sequences from chr9\n",
      "\tExtracting 236 sequences from chr10\n",
      "\tExtracting 275 sequences from chr11\n",
      "\tExtracting 190 sequences from chr12\n",
      "\tExtracting 192 sequences from chr13\n",
      "\tExtracting 154 sequences from chr14\n",
      "\tExtracting 149 sequences from chr15\n",
      "\tExtracting 164 sequences from chr16\n",
      "\tExtracting 203 sequences from chr17\n",
      "\tExtracting 124 sequences from chr18\n",
      "\tExtracting 102 sequences from chr19\n",
      "\tExtracting 16 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4125 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4125\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4125\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 280 sequences from chr1\n",
      "\tExtracting 294 sequences from chr2\n",
      "\tExtracting 208 sequences from chr3\n",
      "\tExtracting 291 sequences from chr4\n",
      "\tExtracting 257 sequences from chr5\n",
      "\tExtracting 221 sequences from chr6\n",
      "\tExtracting 323 sequences from chr7\n",
      "\tExtracting 227 sequences from chr8\n",
      "\tExtracting 198 sequences from chr9\n",
      "\tExtracting 238 sequences from chr10\n",
      "\tExtracting 268 sequences from chr11\n",
      "\tExtracting 174 sequences from chr12\n",
      "\tExtracting 193 sequences from chr13\n",
      "\tExtracting 137 sequences from chr14\n",
      "\tExtracting 172 sequences from chr15\n",
      "\tExtracting 166 sequences from chr16\n",
      "\tExtracting 190 sequences from chr17\n",
      "\tExtracting 110 sequences from chr18\n",
      "\tExtracting 129 sequences from chr19\n",
      "\tExtracting 49 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4125 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset38/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset38/expression.tsv.gz tmp_dynamic/Subset38/motifs.bed tmp_dynamic/Subset38/wellington.tsv.gz tmp_dynamic/Subset38/homer.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset61/wellington.tsv.gz tmp_dynamic/Subset61/homer.tsv.gz tmp_dynamic/Subset61/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset61/expression.tsv.gz tmp_dynamic/Subset61/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset61/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset61/binding.tsv.gz tmp_dynamic/Subset61/tssdist.tsv.gz tmp_dynamic/Subset61/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset61/linking.tsv.gz tmp_dynamic/Subset61/binlinking.tsv.gz 20\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6152\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6152\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 368 sequences from chr1\n",
      "\tExtracting 494 sequences from chr2\n",
      "\tExtracting 317 sequences from chr3\n",
      "\tExtracting 406 sequences from chr4\n",
      "\tExtracting 410 sequences from chr5\n",
      "\tExtracting 290 sequences from chr6\n",
      "\tExtracting 421 sequences from chr7\n",
      "\tExtracting 347 sequences from chr8\n",
      "\tExtracting 360 sequences from chr9\n",
      "\tExtracting 348 sequences from chr10\n",
      "\tExtracting 451 sequences from chr11\n",
      "\tExtracting 237 sequences from chr12\n",
      "\tExtracting 279 sequences from chr13\n",
      "\tExtracting 237 sequences from chr14\n",
      "\tExtracting 239 sequences from chr15\n",
      "\tExtracting 236 sequences from chr16\n",
      "\tExtracting 288 sequences from chr17\n",
      "\tExtracting 176 sequences from chr18\n",
      "\tExtracting 180 sequences from chr19\n",
      "\tExtracting 68 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6152 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6155\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6155\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 376 sequences from chr1\n",
      "\tExtracting 410 sequences from chr2\n",
      "\tExtracting 326 sequences from chr3\n",
      "\tExtracting 378 sequences from chr4\n",
      "\tExtracting 422 sequences from chr5\n",
      "\tExtracting 320 sequences from chr6\n",
      "\tExtracting 491 sequences from chr7\n",
      "\tExtracting 329 sequences from chr8\n",
      "\tExtracting 329 sequences from chr9\n",
      "\tExtracting 367 sequences from chr10\n",
      "\tExtracting 460 sequences from chr11\n",
      "\tExtracting 270 sequences from chr12\n",
      "\tExtracting 266 sequences from chr13\n",
      "\tExtracting 217 sequences from chr14\n",
      "\tExtracting 282 sequences from chr15\n",
      "\tExtracting 225 sequences from chr16\n",
      "\tExtracting 290 sequences from chr17\n",
      "\tExtracting 153 sequences from chr18\n",
      "\tExtracting 190 sequences from chr19\n",
      "\tExtracting 54 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6155 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6155\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6155\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 397 sequences from chr1\n",
      "\tExtracting 488 sequences from chr2\n",
      "\tExtracting 337 sequences from chr3\n",
      "\tExtracting 420 sequences from chr4\n",
      "\tExtracting 366 sequences from chr5\n",
      "\tExtracting 295 sequences from chr6\n",
      "\tExtracting 513 sequences from chr7\n",
      "\tExtracting 325 sequences from chr8\n",
      "\tExtracting 331 sequences from chr9\n",
      "\tExtracting 356 sequences from chr10\n",
      "\tExtracting 473 sequences from chr11\n",
      "\tExtracting 228 sequences from chr12\n",
      "\tExtracting 250 sequences from chr13\n",
      "\tExtracting 217 sequences from chr14\n",
      "\tExtracting 248 sequences from chr15\n",
      "\tExtracting 194 sequences from chr16\n",
      "\tExtracting 296 sequences from chr17\n",
      "\tExtracting 163 sequences from chr18\n",
      "\tExtracting 194 sequences from chr19\n",
      "\tExtracting 64 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6155 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6155\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6155\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 421 sequences from chr1\n",
      "\tExtracting 444 sequences from chr2\n",
      "\tExtracting 326 sequences from chr3\n",
      "\tExtracting 390 sequences from chr4\n",
      "\tExtracting 398 sequences from chr5\n",
      "\tExtracting 304 sequences from chr6\n",
      "\tExtracting 448 sequences from chr7\n",
      "\tExtracting 326 sequences from chr8\n",
      "\tExtracting 338 sequences from chr9\n",
      "\tExtracting 376 sequences from chr10\n",
      "\tExtracting 457 sequences from chr11\n",
      "\tExtracting 288 sequences from chr12\n",
      "\tExtracting 245 sequences from chr13\n",
      "\tExtracting 229 sequences from chr14\n",
      "\tExtracting 223 sequences from chr15\n",
      "\tExtracting 236 sequences from chr16\n",
      "\tExtracting 306 sequences from chr17\n",
      "\tExtracting 173 sequences from chr18\n",
      "\tExtracting 193 sequences from chr19\n",
      "\tExtracting 34 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6155 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset37/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset37/expression.tsv.gz tmp_dynamic/Subset37/motifs.bed tmp_dynamic/Subset37/wellington.tsv.gz tmp_dynamic/Subset37/homer.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset38/wellington.tsv.gz tmp_dynamic/Subset38/homer.tsv.gz tmp_dynamic/Subset38/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset38/expression.tsv.gz tmp_dynamic/Subset38/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset38/tssdist.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset39/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset39/expression.tsv.gz tmp_dynamic/Subset39/motifs.bed tmp_dynamic/Subset39/wellington.tsv.gz tmp_dynamic/Subset39/homer.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset38/binding.tsv.gz tmp_dynamic/Subset38/tssdist.tsv.gz tmp_dynamic/Subset38/linking.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset42/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset42/expression.tsv.gz tmp_dynamic/Subset42/motifs.bed tmp_dynamic/Subset42/wellington.tsv.gz tmp_dynamic/Subset42/homer.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset38/linking.tsv.gz tmp_dynamic/Subset38/binlinking.tsv.gz 20\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset43/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset43/expression.tsv.gz tmp_dynamic/Subset43/motifs.bed tmp_dynamic/Subset43/wellington.tsv.gz tmp_dynamic/Subset43/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6232\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6232\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 388 sequences from chr1\n",
      "\tExtracting 461 sequences from chr2\n",
      "\tExtracting 368 sequences from chr3\n",
      "\tExtracting 420 sequences from chr4\n",
      "\tExtracting 402 sequences from chr5\n",
      "\tExtracting 310 sequences from chr6\n",
      "\tExtracting 431 sequences from chr7\n",
      "\tExtracting 317 sequences from chr8\n",
      "\tExtracting 317 sequences from chr9\n",
      "\tExtracting 346 sequences from chr10\n",
      "\tExtracting 482 sequences from chr11\n",
      "\tExtracting 257 sequences from chr12\n",
      "\tExtracting 280 sequences from chr13\n",
      "\tExtracting 227 sequences from chr14\n",
      "\tExtracting 263 sequences from chr15\n",
      "\tExtracting 233 sequences from chr16\n",
      "\tExtracting 297 sequences from chr17\n",
      "\tExtracting 171 sequences from chr18\n",
      "\tExtracting 189 sequences from chr19\n",
      "\tExtracting 73 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6232 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6236\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6236\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 385 sequences from chr1\n",
      "\tExtracting 513 sequences from chr2\n",
      "\tExtracting 352 sequences from chr3\n",
      "\tExtracting 412 sequences from chr4\n",
      "\tExtracting 400 sequences from chr5\n",
      "\tExtracting 278 sequences from chr6\n",
      "\tExtracting 487 sequences from chr7\n",
      "\tExtracting 332 sequences from chr8\n",
      "\tExtracting 353 sequences from chr9\n",
      "\tExtracting 356 sequences from chr10\n",
      "\tExtracting 464 sequences from chr11\n",
      "\tExtracting 246 sequences from chr12\n",
      "\tExtracting 252 sequences from chr13\n",
      "\tExtracting 229 sequences from chr14\n",
      "\tExtracting 256 sequences from chr15\n",
      "\tExtracting 204 sequences from chr16\n",
      "\tExtracting 292 sequences from chr17\n",
      "\tExtracting 168 sequences from chr18\n",
      "\tExtracting 191 sequences from chr19\n",
      "\tExtracting 66 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6236 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6236\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6236\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 366 sequences from chr1\n",
      "\tExtracting 433 sequences from chr2\n",
      "\tExtracting 316 sequences from chr3\n",
      "\tExtracting 406 sequences from chr4\n",
      "\tExtracting 412 sequences from chr5\n",
      "\tExtracting 304 sequences from chr6\n",
      "\tExtracting 511 sequences from chr7\n",
      "\tExtracting 353 sequences from chr8\n",
      "\tExtracting 353 sequences from chr9\n",
      "\tExtracting 382 sequences from chr10\n",
      "\tExtracting 475 sequences from chr11\n",
      "\tExtracting 277 sequences from chr12\n",
      "\tExtracting 256 sequences from chr13\n",
      "\tExtracting 211 sequences from chr14\n",
      "\tExtracting 252 sequences from chr15\n",
      "\tExtracting 241 sequences from chr16\n",
      "\tExtracting 278 sequences from chr17\n",
      "\tExtracting 162 sequences from chr18\n",
      "\tExtracting 197 sequences from chr19\n",
      "\tExtracting 51 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6236 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6236\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6236\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 445 sequences from chr1\n",
      "\tExtracting 440 sequences from chr2\n",
      "\tExtracting 324 sequences from chr3\n",
      "\tExtracting 386 sequences from chr4\n",
      "\tExtracting 414 sequences from chr5\n",
      "\tExtracting 316 sequences from chr6\n",
      "\tExtracting 467 sequences from chr7\n",
      "\tExtracting 320 sequences from chr8\n",
      "\tExtracting 337 sequences from chr9\n",
      "\tExtracting 362 sequences from chr10\n",
      "\tExtracting 458 sequences from chr11\n",
      "\tExtracting 291 sequences from chr12\n",
      "\tExtracting 262 sequences from chr13\n",
      "\tExtracting 229 sequences from chr14\n",
      "\tExtracting 233 sequences from chr15\n",
      "\tExtracting 237 sequences from chr16\n",
      "\tExtracting 317 sequences from chr17\n",
      "\tExtracting 168 sequences from chr18\n",
      "\tExtracting 193 sequences from chr19\n",
      "\tExtracting 37 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6236 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset35/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset35/expression.tsv.gz tmp_dynamic/Subset35/motifs.bed tmp_dynamic/Subset35/wellington.tsv.gz tmp_dynamic/Subset35/homer.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset41/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset41/expression.tsv.gz tmp_dynamic/Subset41/motifs.bed tmp_dynamic/Subset41/wellington.tsv.gz tmp_dynamic/Subset41/homer.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset37/wellington.tsv.gz tmp_dynamic/Subset37/homer.tsv.gz tmp_dynamic/Subset37/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset37/expression.tsv.gz tmp_dynamic/Subset37/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset37/tssdist.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6087\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6087\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 389 sequences from chr1\n",
      "\tExtracting 451 sequences from chr2\n",
      "\tExtracting 320 sequences from chr3\n",
      "\tExtracting 404 sequences from chr4\n",
      "\tExtracting 392 sequences from chr5\n",
      "\tExtracting 302 sequences from chr6\n",
      "\tExtracting 457 sequences from chr7\n",
      "\tExtracting 322 sequences from chr8\n",
      "\tExtracting 346 sequences from chr9\n",
      "\tExtracting 375 sequences from chr10\n",
      "\tExtracting 459 sequences from chr11\n",
      "\tExtracting 247 sequences from chr12\n",
      "\tExtracting 268 sequences from chr13\n",
      "\tExtracting 214 sequences from chr14\n",
      "\tExtracting 240 sequences from chr15\n",
      "\tExtracting 236 sequences from chr16\n",
      "\tExtracting 283 sequences from chr17\n",
      "\tExtracting 141 sequences from chr18\n",
      "\tExtracting 175 sequences from chr19\n",
      "\tExtracting 66 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6087 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6083\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6083\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 370 sequences from chr1\n",
      "\tExtracting 465 sequences from chr2\n",
      "\tExtracting 352 sequences from chr3\n",
      "\tExtracting 398 sequences from chr4\n",
      "\tExtracting 373 sequences from chr5\n",
      "\tExtracting 263 sequences from chr6\n",
      "\tExtracting 436 sequences from chr7\n",
      "\tExtracting 353 sequences from chr8\n",
      "\tExtracting 330 sequences from chr9\n",
      "\tExtracting 335 sequences from chr10\n",
      "\tExtracting 493 sequences from chr11\n",
      "\tExtracting 233 sequences from chr12\n",
      "\tExtracting 276 sequences from chr13\n",
      "\tExtracting 220 sequences from chr14\n",
      "\tExtracting 248 sequences from chr15\n",
      "\tExtracting 234 sequences from chr16\n",
      "\tExtracting 291 sequences from chr17\n",
      "\tExtracting 175 sequences from chr18\n",
      "\tExtracting 176 sequences from chr19\n",
      "\tExtracting 62 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6083 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6087\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6087\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 389 sequences from chr1\n",
      "\tExtracting 437 sequences from chr2\n",
      "\tExtracting 306 sequences from chr3\n",
      "\tExtracting 394 sequences from chr4\n",
      "\tExtracting 407 sequences from chr5\n",
      "\tExtracting 315 sequences from chr6\n",
      "\tExtracting 497 sequences from chr7\n",
      "\tExtracting 346 sequences from chr8\n",
      "\tExtracting 300 sequences from chr9\n",
      "\tExtracting 358 sequences from chr10\n",
      "\tExtracting 433 sequences from chr11\n",
      "\tExtracting 260 sequences from chr12\n",
      "\tExtracting 238 sequences from chr13\n",
      "\tExtracting 207 sequences from chr14\n",
      "\tExtracting 292 sequences from chr15\n",
      "\tExtracting 210 sequences from chr16\n",
      "\tExtracting 291 sequences from chr17\n",
      "\tExtracting 148 sequences from chr18\n",
      "\tExtracting 205 sequences from chr19\n",
      "\tExtracting 54 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6087 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6087\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6087\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 410 sequences from chr1\n",
      "\tExtracting 434 sequences from chr2\n",
      "\tExtracting 324 sequences from chr3\n",
      "\tExtracting 374 sequences from chr4\n",
      "\tExtracting 394 sequences from chr5\n",
      "\tExtracting 317 sequences from chr6\n",
      "\tExtracting 450 sequences from chr7\n",
      "\tExtracting 300 sequences from chr8\n",
      "\tExtracting 341 sequences from chr9\n",
      "\tExtracting 372 sequences from chr10\n",
      "\tExtracting 470 sequences from chr11\n",
      "\tExtracting 279 sequences from chr12\n",
      "\tExtracting 239 sequences from chr13\n",
      "\tExtracting 227 sequences from chr14\n",
      "\tExtracting 208 sequences from chr15\n",
      "\tExtracting 225 sequences from chr16\n",
      "\tExtracting 314 sequences from chr17\n",
      "\tExtracting 183 sequences from chr18\n",
      "\tExtracting 192 sequences from chr19\n",
      "\tExtracting 34 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6087 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset37/binding.tsv.gz tmp_dynamic/Subset37/tssdist.tsv.gz tmp_dynamic/Subset37/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset39/wellington.tsv.gz tmp_dynamic/Subset39/homer.tsv.gz tmp_dynamic/Subset39/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset39/expression.tsv.gz tmp_dynamic/Subset39/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset39/tssdist.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5747\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5747\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 386 sequences from chr1\n",
      "\tExtracting 430 sequences from chr2\n",
      "\tExtracting 306 sequences from chr3\n",
      "\tExtracting 384 sequences from chr4\n",
      "\tExtracting 368 sequences from chr5\n",
      "\tExtracting 254 sequences from chr6\n",
      "\tExtracting 389 sequences from chr7\n",
      "\tExtracting 308 sequences from chr8\n",
      "\tExtracting 329 sequences from chr9\n",
      "\tExtracting 335 sequences from chr10\n",
      "\tExtracting 410 sequences from chr11\n",
      "\tExtracting 222 sequences from chr12\n",
      "\tExtracting 237 sequences from chr13\n",
      "\tExtracting 223 sequences from chr14\n",
      "\tExtracting 231 sequences from chr15\n",
      "\tExtracting 219 sequences from chr16\n",
      "\tExtracting 271 sequences from chr17\n",
      "\tExtracting 176 sequences from chr18\n",
      "\tExtracting 193 sequences from chr19\n",
      "\tExtracting 76 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5747 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5749\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5749\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 330 sequences from chr1\n",
      "\tExtracting 441 sequences from chr2\n",
      "\tExtracting 291 sequences from chr3\n",
      "\tExtracting 400 sequences from chr4\n",
      "\tExtracting 381 sequences from chr5\n",
      "\tExtracting 287 sequences from chr6\n",
      "\tExtracting 432 sequences from chr7\n",
      "\tExtracting 320 sequences from chr8\n",
      "\tExtracting 299 sequences from chr9\n",
      "\tExtracting 344 sequences from chr10\n",
      "\tExtracting 444 sequences from chr11\n",
      "\tExtracting 211 sequences from chr12\n",
      "\tExtracting 261 sequences from chr13\n",
      "\tExtracting 223 sequences from chr14\n",
      "\tExtracting 227 sequences from chr15\n",
      "\tExtracting 217 sequences from chr16\n",
      "\tExtracting 244 sequences from chr17\n",
      "\tExtracting 150 sequences from chr18\n",
      "\tExtracting 196 sequences from chr19\n",
      "\tExtracting 51 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5749 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5749\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5749\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 380 sequences from chr1\n",
      "\tExtracting 438 sequences from chr2\n",
      "\tExtracting 307 sequences from chr3\n",
      "\tExtracting 367 sequences from chr4\n",
      "\tExtracting 355 sequences from chr5\n",
      "\tExtracting 276 sequences from chr6\n",
      "\tExtracting 444 sequences from chr7\n",
      "\tExtracting 306 sequences from chr8\n",
      "\tExtracting 302 sequences from chr9\n",
      "\tExtracting 359 sequences from chr10\n",
      "\tExtracting 435 sequences from chr11\n",
      "\tExtracting 266 sequences from chr12\n",
      "\tExtracting 231 sequences from chr13\n",
      "\tExtracting 215 sequences from chr14\n",
      "\tExtracting 239 sequences from chr15\n",
      "\tExtracting 208 sequences from chr16\n",
      "\tExtracting 247 sequences from chr17\n",
      "\tExtracting 145 sequences from chr18\n",
      "\tExtracting 181 sequences from chr19\n",
      "\tExtracting 48 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5749 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5749\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5749\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 399 sequences from chr1\n",
      "\tExtracting 397 sequences from chr2\n",
      "\tExtracting 311 sequences from chr3\n",
      "\tExtracting 347 sequences from chr4\n",
      "\tExtracting 374 sequences from chr5\n",
      "\tExtracting 313 sequences from chr6\n",
      "\tExtracting 442 sequences from chr7\n",
      "\tExtracting 308 sequences from chr8\n",
      "\tExtracting 334 sequences from chr9\n",
      "\tExtracting 343 sequences from chr10\n",
      "\tExtracting 424 sequences from chr11\n",
      "\tExtracting 255 sequences from chr12\n",
      "\tExtracting 214 sequences from chr13\n",
      "\tExtracting 179 sequences from chr14\n",
      "\tExtracting 211 sequences from chr15\n",
      "\tExtracting 227 sequences from chr16\n",
      "\tExtracting 294 sequences from chr17\n",
      "\tExtracting 163 sequences from chr18\n",
      "\tExtracting 180 sequences from chr19\n",
      "\tExtracting 34 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5749 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset37/linking.tsv.gz tmp_dynamic/Subset37/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset40/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset40/expression.tsv.gz tmp_dynamic/Subset40/motifs.bed tmp_dynamic/Subset40/wellington.tsv.gz tmp_dynamic/Subset40/homer.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5578\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5578\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 328 sequences from chr1\n",
      "\tExtracting 419 sequences from chr2\n",
      "\tExtracting 306 sequences from chr3\n",
      "\tExtracting 370 sequences from chr4\n",
      "\tExtracting 351 sequences from chr5\n",
      "\tExtracting 242 sequences from chr6\n",
      "\tExtracting 413 sequences from chr7\n",
      "\tExtracting 322 sequences from chr8\n",
      "\tExtracting 294 sequences from chr9\n",
      "\tExtracting 307 sequences from chr10\n",
      "\tExtracting 445 sequences from chr11\n",
      "\tExtracting 245 sequences from chr12\n",
      "\tExtracting 234 sequences from chr13\n",
      "\tExtracting 207 sequences from chr14\n",
      "\tExtracting 209 sequences from chr15\n",
      "\tExtracting 250 sequences from chr16\n",
      "\tExtracting 250 sequences from chr17\n",
      "\tExtracting 134 sequences from chr18\n",
      "\tExtracting 197 sequences from chr19\n",
      "\tExtracting 55 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5578 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5574\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5574\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 378 sequences from chr1\n",
      "\tExtracting 400 sequences from chr2\n",
      "\tExtracting 275 sequences from chr3\n",
      "\tExtracting 358 sequences from chr4\n",
      "\tExtracting 375 sequences from chr5\n",
      "\tExtracting 264 sequences from chr6\n",
      "\tExtracting 414 sequences from chr7\n",
      "\tExtracting 290 sequences from chr8\n",
      "\tExtracting 291 sequences from chr9\n",
      "\tExtracting 333 sequences from chr10\n",
      "\tExtracting 428 sequences from chr11\n",
      "\tExtracting 223 sequences from chr12\n",
      "\tExtracting 240 sequences from chr13\n",
      "\tExtracting 177 sequences from chr14\n",
      "\tExtracting 235 sequences from chr15\n",
      "\tExtracting 216 sequences from chr16\n",
      "\tExtracting 279 sequences from chr17\n",
      "\tExtracting 152 sequences from chr18\n",
      "\tExtracting 158 sequences from chr19\n",
      "\tExtracting 88 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5574 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5578\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5578\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 408 sequences from chr1\n",
      "\tExtracting 392 sequences from chr2\n",
      "\tExtracting 305 sequences from chr3\n",
      "\tExtracting 331 sequences from chr4\n",
      "\tExtracting 357 sequences from chr5\n",
      "\tExtracting 296 sequences from chr6\n",
      "\tExtracting 417 sequences from chr7\n",
      "\tExtracting 303 sequences from chr8\n",
      "\tExtracting 307 sequences from chr9\n",
      "\tExtracting 350 sequences from chr10\n",
      "\tExtracting 418 sequences from chr11\n",
      "\tExtracting 249 sequences from chr12\n",
      "\tExtracting 210 sequences from chr13\n",
      "\tExtracting 198 sequences from chr14\n",
      "\tExtracting 203 sequences from chr15\n",
      "\tExtracting 211 sequences from chr16\n",
      "\tExtracting 274 sequences from chr17\n",
      "\tExtracting 148 sequences from chr18\n",
      "\tExtracting 174 sequences from chr19\n",
      "\tExtracting 27 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5578 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5578\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5578\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 339 sequences from chr1\n",
      "\tExtracting 408 sequences from chr2\n",
      "\tExtracting 276 sequences from chr3\n",
      "\tExtracting 338 sequences from chr4\n",
      "\tExtracting 375 sequences from chr5\n",
      "\tExtracting 305 sequences from chr6\n",
      "\tExtracting 423 sequences from chr7\n",
      "\tExtracting 302 sequences from chr8\n",
      "\tExtracting 313 sequences from chr9\n",
      "\tExtracting 344 sequences from chr10\n",
      "\tExtracting 400 sequences from chr11\n",
      "\tExtracting 224 sequences from chr12\n",
      "\tExtracting 243 sequences from chr13\n",
      "\tExtracting 210 sequences from chr14\n",
      "\tExtracting 239 sequences from chr15\n",
      "\tExtracting 208 sequences from chr16\n",
      "\tExtracting 248 sequences from chr17\n",
      "\tExtracting 161 sequences from chr18\n",
      "\tExtracting 174 sequences from chr19\n",
      "\tExtracting 48 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5578 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset44/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset44/expression.tsv.gz tmp_dynamic/Subset44/motifs.bed tmp_dynamic/Subset44/wellington.tsv.gz tmp_dynamic/Subset44/homer.tsv.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset39/binding.tsv.gz tmp_dynamic/Subset39/tssdist.tsv.gz tmp_dynamic/Subset39/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset42/wellington.tsv.gz tmp_dynamic/Subset42/homer.tsv.gz tmp_dynamic/Subset42/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset42/expression.tsv.gz tmp_dynamic/Subset42/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset42/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset43/wellington.tsv.gz tmp_dynamic/Subset43/homer.tsv.gz tmp_dynamic/Subset43/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset43/expression.tsv.gz tmp_dynamic/Subset43/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset43/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset39/linking.tsv.gz tmp_dynamic/Subset39/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset42/binding.tsv.gz tmp_dynamic/Subset42/tssdist.tsv.gz tmp_dynamic/Subset42/linking.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6201\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6201\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 399 sequences from chr1\n",
      "\tExtracting 476 sequences from chr2\n",
      "\tExtracting 352 sequences from chr3\n",
      "\tExtracting 414 sequences from chr4\n",
      "\tExtracting 375 sequences from chr5\n",
      "\tExtracting 311 sequences from chr6\n",
      "\tExtracting 464 sequences from chr7\n",
      "\tExtracting 335 sequences from chr8\n",
      "\tExtracting 342 sequences from chr9\n",
      "\tExtracting 313 sequences from chr10\n",
      "\tExtracting 463 sequences from chr11\n",
      "\tExtracting 254 sequences from chr12\n",
      "\tExtracting 260 sequences from chr13\n",
      "\tExtracting 224 sequences from chr14\n",
      "\tExtracting 242 sequences from chr15\n",
      "\tExtracting 254 sequences from chr16\n",
      "\tExtracting 269 sequences from chr17\n",
      "\tExtracting 165 sequences from chr18\n",
      "\tExtracting 209 sequences from chr19\n",
      "\tExtracting 80 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6201 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6203\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6203\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 391 sequences from chr1\n",
      "\tExtracting 482 sequences from chr2\n",
      "\tExtracting 307 sequences from chr3\n",
      "\tExtracting 418 sequences from chr4\n",
      "\tExtracting 420 sequences from chr5\n",
      "\tExtracting 282 sequences from chr6\n",
      "\tExtracting 473 sequences from chr7\n",
      "\tExtracting 338 sequences from chr8\n",
      "\tExtracting 317 sequences from chr9\n",
      "\tExtracting 354 sequences from chr10\n",
      "\tExtracting 479 sequences from chr11\n",
      "\tExtracting 266 sequences from chr12\n",
      "\tExtracting 265 sequences from chr13\n",
      "\tExtracting 211 sequences from chr14\n",
      "\tExtracting 265 sequences from chr15\n",
      "\tExtracting 224 sequences from chr16\n",
      "\tExtracting 283 sequences from chr17\n",
      "\tExtracting 170 sequences from chr18\n",
      "\tExtracting 191 sequences from chr19\n",
      "\tExtracting 67 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6203 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6203\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6203\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 374 sequences from chr1\n",
      "\tExtracting 461 sequences from chr2\n",
      "\tExtracting 341 sequences from chr3\n",
      "\tExtracting 367 sequences from chr4\n",
      "\tExtracting 418 sequences from chr5\n",
      "\tExtracting 309 sequences from chr6\n",
      "\tExtracting 467 sequences from chr7\n",
      "\tExtracting 361 sequences from chr8\n",
      "\tExtracting 344 sequences from chr9\n",
      "\tExtracting 355 sequences from chr10\n",
      "\tExtracting 454 sequences from chr11\n",
      "\tExtracting 246 sequences from chr12\n",
      "\tExtracting 266 sequences from chr13\n",
      "\tExtracting 245 sequences from chr14\n",
      "\tExtracting 249 sequences from chr15\n",
      "\tExtracting 224 sequences from chr16\n",
      "\tExtracting 294 sequences from chr17\n",
      "\tExtracting 163 sequences from chr18\n",
      "\tExtracting 216 sequences from chr19\n",
      "\tExtracting 49 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6203 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6203\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6203\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 432 sequences from chr1\n",
      "\tExtracting 416 sequences from chr2\n",
      "\tExtracting 324 sequences from chr3\n",
      "\tExtracting 396 sequences from chr4\n",
      "\tExtracting 397 sequences from chr5\n",
      "\tExtracting 314 sequences from chr6\n",
      "\tExtracting 484 sequences from chr7\n",
      "\tExtracting 314 sequences from chr8\n",
      "\tExtracting 366 sequences from chr9\n",
      "\tExtracting 379 sequences from chr10\n",
      "\tExtracting 462 sequences from chr11\n",
      "\tExtracting 277 sequences from chr12\n",
      "\tExtracting 264 sequences from chr13\n",
      "\tExtracting 219 sequences from chr14\n",
      "\tExtracting 238 sequences from chr15\n",
      "\tExtracting 235 sequences from chr16\n",
      "\tExtracting 303 sequences from chr17\n",
      "\tExtracting 171 sequences from chr18\n",
      "\tExtracting 179 sequences from chr19\n",
      "\tExtracting 33 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6203 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5935\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5935\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 346 sequences from chr1\n",
      "\tExtracting 449 sequences from chr2\n",
      "\tExtracting 304 sequences from chr3\n",
      "\tExtracting 384 sequences from chr4\n",
      "\tExtracting 389 sequences from chr5\n",
      "\tExtracting 271 sequences from chr6\n",
      "\tExtracting 492 sequences from chr7\n",
      "\tExtracting 323 sequences from chr8\n",
      "\tExtracting 318 sequences from chr9\n",
      "\tExtracting 352 sequences from chr10\n",
      "\tExtracting 446 sequences from chr11\n",
      "\tExtracting 260 sequences from chr12\n",
      "\tExtracting 255 sequences from chr13\n",
      "\tExtracting 216 sequences from chr14\n",
      "\tExtracting 241 sequences from chr15\n",
      "\tExtracting 208 sequences from chr16\n",
      "\tExtracting 284 sequences from chr17\n",
      "\tExtracting 147 sequences from chr18\n",
      "\tExtracting 178 sequences from chr19\n",
      "\tExtracting 72 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5935 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5935\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5935\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 377 sequences from chr1\n",
      "\tExtracting 410 sequences from chr2\n",
      "\tExtracting 305 sequences from chr3\n",
      "\tExtracting 393 sequences from chr4\n",
      "\tExtracting 390 sequences from chr5\n",
      "\tExtracting 304 sequences from chr6\n",
      "\tExtracting 443 sequences from chr7\n",
      "\tExtracting 319 sequences from chr8\n",
      "\tExtracting 291 sequences from chr9\n",
      "\tExtracting 359 sequences from chr10\n",
      "\tExtracting 467 sequences from chr11\n",
      "\tExtracting 246 sequences from chr12\n",
      "\tExtracting 263 sequences from chr13\n",
      "\tExtracting 215 sequences from chr14\n",
      "\tExtracting 256 sequences from chr15\n",
      "\tExtracting 226 sequences from chr16\n",
      "\tExtracting 263 sequences from chr17\n",
      "\tExtracting 153 sequences from chr18\n",
      "\tExtracting 199 sequences from chr19\n",
      "\tExtracting 56 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5935 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5932\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5932\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 359 sequences from chr1\n",
      "\tExtracting 440 sequences from chr2\n",
      "\tExtracting 332 sequences from chr3\n",
      "\tExtracting 408 sequences from chr4\n",
      "\tExtracting 369 sequences from chr5\n",
      "\tExtracting 287 sequences from chr6\n",
      "\tExtracting 416 sequences from chr7\n",
      "\tExtracting 321 sequences from chr8\n",
      "\tExtracting 324 sequences from chr9\n",
      "\tExtracting 366 sequences from chr10\n",
      "\tExtracting 457 sequences from chr11\n",
      "\tExtracting 235 sequences from chr12\n",
      "\tExtracting 268 sequences from chr13\n",
      "\tExtracting 221 sequences from chr14\n",
      "\tExtracting 249 sequences from chr15\n",
      "\tExtracting 226 sequences from chr16\n",
      "\tExtracting 258 sequences from chr17\n",
      "\tExtracting 176 sequences from chr18\n",
      "\tExtracting 165 sequences from chr19\n",
      "\tExtracting 55 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5932 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5935\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5935\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 413 sequences from chr1\n",
      "\tExtracting 430 sequences from chr2\n",
      "\tExtracting 305 sequences from chr3\n",
      "\tExtracting 367 sequences from chr4\n",
      "\tExtracting 398 sequences from chr5\n",
      "\tExtracting 295 sequences from chr6\n",
      "\tExtracting 450 sequences from chr7\n",
      "\tExtracting 308 sequences from chr8\n",
      "\tExtracting 343 sequences from chr9\n",
      "\tExtracting 358 sequences from chr10\n",
      "\tExtracting 436 sequences from chr11\n",
      "\tExtracting 261 sequences from chr12\n",
      "\tExtracting 214 sequences from chr13\n",
      "\tExtracting 209 sequences from chr14\n",
      "\tExtracting 226 sequences from chr15\n",
      "\tExtracting 225 sequences from chr16\n",
      "\tExtracting 304 sequences from chr17\n",
      "\tExtracting 174 sequences from chr18\n",
      "\tExtracting 189 sequences from chr19\n",
      "\tExtracting 30 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5935 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset43/binding.tsv.gz tmp_dynamic/Subset43/tssdist.tsv.gz tmp_dynamic/Subset43/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset42/linking.tsv.gz tmp_dynamic/Subset42/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset35/wellington.tsv.gz tmp_dynamic/Subset35/homer.tsv.gz tmp_dynamic/Subset35/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset35/expression.tsv.gz tmp_dynamic/Subset35/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset35/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset41/wellington.tsv.gz tmp_dynamic/Subset41/homer.tsv.gz tmp_dynamic/Subset41/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset41/expression.tsv.gz tmp_dynamic/Subset41/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset41/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset43/linking.tsv.gz tmp_dynamic/Subset43/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset41/binding.tsv.gz tmp_dynamic/Subset41/tssdist.tsv.gz tmp_dynamic/Subset41/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset35/binding.tsv.gz tmp_dynamic/Subset35/tssdist.tsv.gz tmp_dynamic/Subset35/linking.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5639\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5639\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 327 sequences from chr1\n",
      "\tExtracting 438 sequences from chr2\n",
      "\tExtracting 299 sequences from chr3\n",
      "\tExtracting 360 sequences from chr4\n",
      "\tExtracting 372 sequences from chr5\n",
      "\tExtracting 255 sequences from chr6\n",
      "\tExtracting 427 sequences from chr7\n",
      "\tExtracting 315 sequences from chr8\n",
      "\tExtracting 306 sequences from chr9\n",
      "\tExtracting 323 sequences from chr10\n",
      "\tExtracting 395 sequences from chr11\n",
      "\tExtracting 239 sequences from chr12\n",
      "\tExtracting 250 sequences from chr13\n",
      "\tExtracting 219 sequences from chr14\n",
      "\tExtracting 217 sequences from chr15\n",
      "\tExtracting 214 sequences from chr16\n",
      "\tExtracting 273 sequences from chr17\n",
      "\tExtracting 149 sequences from chr18\n",
      "\tExtracting 204 sequences from chr19\n",
      "\tExtracting 57 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5639 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5637\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5637\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 365 sequences from chr1\n",
      "\tExtracting 409 sequences from chr2\n",
      "\tExtracting 302 sequences from chr3\n",
      "\tExtracting 352 sequences from chr4\n",
      "\tExtracting 355 sequences from chr5\n",
      "\tExtracting 252 sequences from chr6\n",
      "\tExtracting 418 sequences from chr7\n",
      "\tExtracting 318 sequences from chr8\n",
      "\tExtracting 305 sequences from chr9\n",
      "\tExtracting 311 sequences from chr10\n",
      "\tExtracting 420 sequences from chr11\n",
      "\tExtracting 236 sequences from chr12\n",
      "\tExtracting 236 sequences from chr13\n",
      "\tExtracting 195 sequences from chr14\n",
      "\tExtracting 259 sequences from chr15\n",
      "\tExtracting 213 sequences from chr16\n",
      "\tExtracting 292 sequences from chr17\n",
      "\tExtracting 144 sequences from chr18\n",
      "\tExtracting 181 sequences from chr19\n",
      "\tExtracting 74 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5637 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5639\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5639\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 388 sequences from chr1\n",
      "\tExtracting 385 sequences from chr2\n",
      "\tExtracting 300 sequences from chr3\n",
      "\tExtracting 323 sequences from chr4\n",
      "\tExtracting 372 sequences from chr5\n",
      "\tExtracting 311 sequences from chr6\n",
      "\tExtracting 413 sequences from chr7\n",
      "\tExtracting 296 sequences from chr8\n",
      "\tExtracting 315 sequences from chr9\n",
      "\tExtracting 339 sequences from chr10\n",
      "\tExtracting 434 sequences from chr11\n",
      "\tExtracting 261 sequences from chr12\n",
      "\tExtracting 231 sequences from chr13\n",
      "\tExtracting 191 sequences from chr14\n",
      "\tExtracting 212 sequences from chr15\n",
      "\tExtracting 219 sequences from chr16\n",
      "\tExtracting 281 sequences from chr17\n",
      "\tExtracting 174 sequences from chr18\n",
      "\tExtracting 166 sequences from chr19\n",
      "\tExtracting 28 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5639 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5639\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5639\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 384 sequences from chr1\n",
      "\tExtracting 419 sequences from chr2\n",
      "\tExtracting 287 sequences from chr3\n",
      "\tExtracting 364 sequences from chr4\n",
      "\tExtracting 344 sequences from chr5\n",
      "\tExtracting 282 sequences from chr6\n",
      "\tExtracting 415 sequences from chr7\n",
      "\tExtracting 319 sequences from chr8\n",
      "\tExtracting 302 sequences from chr9\n",
      "\tExtracting 352 sequences from chr10\n",
      "\tExtracting 420 sequences from chr11\n",
      "\tExtracting 232 sequences from chr12\n",
      "\tExtracting 232 sequences from chr13\n",
      "\tExtracting 215 sequences from chr14\n",
      "\tExtracting 235 sequences from chr15\n",
      "\tExtracting 218 sequences from chr16\n",
      "\tExtracting 256 sequences from chr17\n",
      "\tExtracting 146 sequences from chr18\n",
      "\tExtracting 171 sequences from chr19\n",
      "\tExtracting 46 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5639 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6051\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6051\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 395 sequences from chr1\n",
      "\tExtracting 465 sequences from chr2\n",
      "\tExtracting 313 sequences from chr3\n",
      "\tExtracting 420 sequences from chr4\n",
      "\tExtracting 411 sequences from chr5\n",
      "\tExtracting 304 sequences from chr6\n",
      "\tExtracting 485 sequences from chr7\n",
      "\tExtracting 300 sequences from chr8\n",
      "\tExtracting 335 sequences from chr9\n",
      "\tExtracting 335 sequences from chr10\n",
      "\tExtracting 471 sequences from chr11\n",
      "\tExtracting 252 sequences from chr12\n",
      "\tExtracting 253 sequences from chr13\n",
      "\tExtracting 203 sequences from chr14\n",
      "\tExtracting 231 sequences from chr15\n",
      "\tExtracting 204 sequences from chr16\n",
      "\tExtracting 275 sequences from chr17\n",
      "\tExtracting 166 sequences from chr18\n",
      "\tExtracting 170 sequences from chr19\n",
      "\tExtracting 63 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6051 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6053\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6053\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 381 sequences from chr1\n",
      "\tExtracting 454 sequences from chr2\n",
      "\tExtracting 305 sequences from chr3\n",
      "\tExtracting 389 sequences from chr4\n",
      "\tExtracting 405 sequences from chr5\n",
      "\tExtracting 301 sequences from chr6\n",
      "\tExtracting 443 sequences from chr7\n",
      "\tExtracting 359 sequences from chr8\n",
      "\tExtracting 342 sequences from chr9\n",
      "\tExtracting 372 sequences from chr10\n",
      "\tExtracting 472 sequences from chr11\n",
      "\tExtracting 216 sequences from chr12\n",
      "\tExtracting 264 sequences from chr13\n",
      "\tExtracting 214 sequences from chr14\n",
      "\tExtracting 242 sequences from chr15\n",
      "\tExtracting 245 sequences from chr16\n",
      "\tExtracting 272 sequences from chr17\n",
      "\tExtracting 139 sequences from chr18\n",
      "\tExtracting 180 sequences from chr19\n",
      "\tExtracting 58 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6053 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6053\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6053\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 371 sequences from chr1\n",
      "\tExtracting 426 sequences from chr2\n",
      "\tExtracting 332 sequences from chr3\n",
      "\tExtracting 392 sequences from chr4\n",
      "\tExtracting 391 sequences from chr5\n",
      "\tExtracting 293 sequences from chr6\n",
      "\tExtracting 470 sequences from chr7\n",
      "\tExtracting 316 sequences from chr8\n",
      "\tExtracting 298 sequences from chr9\n",
      "\tExtracting 366 sequences from chr10\n",
      "\tExtracting 430 sequences from chr11\n",
      "\tExtracting 283 sequences from chr12\n",
      "\tExtracting 248 sequences from chr13\n",
      "\tExtracting 218 sequences from chr14\n",
      "\tExtracting 286 sequences from chr15\n",
      "\tExtracting 221 sequences from chr16\n",
      "\tExtracting 294 sequences from chr17\n",
      "\tExtracting 155 sequences from chr18\n",
      "\tExtracting 201 sequences from chr19\n",
      "\tExtracting 62 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6053 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6053\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6053\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 407 sequences from chr1\n",
      "\tExtracting 434 sequences from chr2\n",
      "\tExtracting 314 sequences from chr3\n",
      "\tExtracting 376 sequences from chr4\n",
      "\tExtracting 387 sequences from chr5\n",
      "\tExtracting 316 sequences from chr6\n",
      "\tExtracting 461 sequences from chr7\n",
      "\tExtracting 304 sequences from chr8\n",
      "\tExtracting 340 sequences from chr9\n",
      "\tExtracting 366 sequences from chr10\n",
      "\tExtracting 457 sequences from chr11\n",
      "\tExtracting 277 sequences from chr12\n",
      "\tExtracting 238 sequences from chr13\n",
      "\tExtracting 222 sequences from chr14\n",
      "\tExtracting 211 sequences from chr15\n",
      "\tExtracting 221 sequences from chr16\n",
      "\tExtracting 315 sequences from chr17\n",
      "\tExtracting 179 sequences from chr18\n",
      "\tExtracting 195 sequences from chr19\n",
      "\tExtracting 33 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6053 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset41/linking.tsv.gz tmp_dynamic/Subset41/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset34/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset34/expression.tsv.gz tmp_dynamic/Subset34/motifs.bed tmp_dynamic/Subset34/wellington.tsv.gz tmp_dynamic/Subset34/homer.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset35/linking.tsv.gz tmp_dynamic/Subset35/binlinking.tsv.gz 20\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset36/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset36/expression.tsv.gz tmp_dynamic/Subset36/motifs.bed tmp_dynamic/Subset36/wellington.tsv.gz tmp_dynamic/Subset36/homer.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset44/wellington.tsv.gz tmp_dynamic/Subset44/homer.tsv.gz tmp_dynamic/Subset44/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset44/expression.tsv.gz tmp_dynamic/Subset44/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset44/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset40/wellington.tsv.gz tmp_dynamic/Subset40/homer.tsv.gz tmp_dynamic/Subset40/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset40/expression.tsv.gz tmp_dynamic/Subset40/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset40/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset44/binding.tsv.gz tmp_dynamic/Subset44/tssdist.tsv.gz tmp_dynamic/Subset44/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset40/binding.tsv.gz tmp_dynamic/Subset40/tssdist.tsv.gz tmp_dynamic/Subset40/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset44/linking.tsv.gz tmp_dynamic/Subset44/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset40/linking.tsv.gz tmp_dynamic/Subset40/binlinking.tsv.gz 20\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6457\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6457\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 386 sequences from chr1\n",
      "\tExtracting 510 sequences from chr2\n",
      "\tExtracting 341 sequences from chr3\n",
      "\tExtracting 433 sequences from chr4\n",
      "\tExtracting 418 sequences from chr5\n",
      "\tExtracting 319 sequences from chr6\n",
      "\tExtracting 486 sequences from chr7\n",
      "\tExtracting 368 sequences from chr8\n",
      "\tExtracting 350 sequences from chr9\n",
      "\tExtracting 368 sequences from chr10\n",
      "\tExtracting 467 sequences from chr11\n",
      "\tExtracting 269 sequences from chr12\n",
      "\tExtracting 295 sequences from chr13\n",
      "\tExtracting 253 sequences from chr14\n",
      "\tExtracting 273 sequences from chr15\n",
      "\tExtracting 233 sequences from chr16\n",
      "\tExtracting 270 sequences from chr17\n",
      "\tExtracting 157 sequences from chr18\n",
      "\tExtracting 198 sequences from chr19\n",
      "\tExtracting 63 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6457 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6457\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6457\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 421 sequences from chr1\n",
      "\tExtracting 475 sequences from chr2\n",
      "\tExtracting 333 sequences from chr3\n",
      "\tExtracting 418 sequences from chr4\n",
      "\tExtracting 416 sequences from chr5\n",
      "\tExtracting 318 sequences from chr6\n",
      "\tExtracting 465 sequences from chr7\n",
      "\tExtracting 350 sequences from chr8\n",
      "\tExtracting 355 sequences from chr9\n",
      "\tExtracting 367 sequences from chr10\n",
      "\tExtracting 477 sequences from chr11\n",
      "\tExtracting 270 sequences from chr12\n",
      "\tExtracting 273 sequences from chr13\n",
      "\tExtracting 248 sequences from chr14\n",
      "\tExtracting 261 sequences from chr15\n",
      "\tExtracting 234 sequences from chr16\n",
      "\tExtracting 320 sequences from chr17\n",
      "\tExtracting 184 sequences from chr18\n",
      "\tExtracting 221 sequences from chr19\n",
      "\tExtracting 51 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6457 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6454\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6454\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 406 sequences from chr1\n",
      "\tExtracting 494 sequences from chr2\n",
      "\tExtracting 347 sequences from chr3\n",
      "\tExtracting 402 sequences from chr4\n",
      "\tExtracting 448 sequences from chr5\n",
      "\tExtracting 315 sequences from chr6\n",
      "\tExtracting 477 sequences from chr7\n",
      "\tExtracting 344 sequences from chr8\n",
      "\tExtracting 361 sequences from chr9\n",
      "\tExtracting 355 sequences from chr10\n",
      "\tExtracting 467 sequences from chr11\n",
      "\tExtracting 268 sequences from chr12\n",
      "\tExtracting 258 sequences from chr13\n",
      "\tExtracting 234 sequences from chr14\n",
      "\tExtracting 256 sequences from chr15\n",
      "\tExtracting 231 sequences from chr16\n",
      "\tExtracting 318 sequences from chr17\n",
      "\tExtracting 183 sequences from chr18\n",
      "\tExtracting 202 sequences from chr19\n",
      "\tExtracting 88 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6454 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6457\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6457\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 450 sequences from chr1\n",
      "\tExtracting 454 sequences from chr2\n",
      "\tExtracting 360 sequences from chr3\n",
      "\tExtracting 408 sequences from chr4\n",
      "\tExtracting 427 sequences from chr5\n",
      "\tExtracting 335 sequences from chr6\n",
      "\tExtracting 503 sequences from chr7\n",
      "\tExtracting 333 sequences from chr8\n",
      "\tExtracting 377 sequences from chr9\n",
      "\tExtracting 401 sequences from chr10\n",
      "\tExtracting 458 sequences from chr11\n",
      "\tExtracting 266 sequences from chr12\n",
      "\tExtracting 282 sequences from chr13\n",
      "\tExtracting 227 sequences from chr14\n",
      "\tExtracting 232 sequences from chr15\n",
      "\tExtracting 249 sequences from chr16\n",
      "\tExtracting 296 sequences from chr17\n",
      "\tExtracting 173 sequences from chr18\n",
      "\tExtracting 189 sequences from chr19\n",
      "\tExtracting 37 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6457 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6370\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6370\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 404 sequences from chr1\n",
      "\tExtracting 475 sequences from chr2\n",
      "\tExtracting 341 sequences from chr3\n",
      "\tExtracting 403 sequences from chr4\n",
      "\tExtracting 414 sequences from chr5\n",
      "\tExtracting 305 sequences from chr6\n",
      "\tExtracting 493 sequences from chr7\n",
      "\tExtracting 349 sequences from chr8\n",
      "\tExtracting 341 sequences from chr9\n",
      "\tExtracting 344 sequences from chr10\n",
      "\tExtracting 487 sequences from chr11\n",
      "\tExtracting 268 sequences from chr12\n",
      "\tExtracting 289 sequences from chr13\n",
      "\tExtracting 235 sequences from chr14\n",
      "\tExtracting 272 sequences from chr15\n",
      "\tExtracting 218 sequences from chr16\n",
      "\tExtracting 288 sequences from chr17\n",
      "\tExtracting 161 sequences from chr18\n",
      "\tExtracting 211 sequences from chr19\n",
      "\tExtracting 72 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6370 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6373\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6373\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 386 sequences from chr1\n",
      "\tExtracting 464 sequences from chr2\n",
      "\tExtracting 340 sequences from chr3\n",
      "\tExtracting 436 sequences from chr4\n",
      "\tExtracting 446 sequences from chr5\n",
      "\tExtracting 299 sequences from chr6\n",
      "\tExtracting 488 sequences from chr7\n",
      "\tExtracting 360 sequences from chr8\n",
      "\tExtracting 348 sequences from chr9\n",
      "\tExtracting 373 sequences from chr10\n",
      "\tExtracting 474 sequences from chr11\n",
      "\tExtracting 275 sequences from chr12\n",
      "\tExtracting 242 sequences from chr13\n",
      "\tExtracting 232 sequences from chr14\n",
      "\tExtracting 264 sequences from chr15\n",
      "\tExtracting 241 sequences from chr16\n",
      "\tExtracting 291 sequences from chr17\n",
      "\tExtracting 161 sequences from chr18\n",
      "\tExtracting 209 sequences from chr19\n",
      "\tExtracting 44 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6373 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6373\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6373\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 410 sequences from chr1\n",
      "\tExtracting 473 sequences from chr2\n",
      "\tExtracting 351 sequences from chr3\n",
      "\tExtracting 406 sequences from chr4\n",
      "\tExtracting 406 sequences from chr5\n",
      "\tExtracting 279 sequences from chr6\n",
      "\tExtracting 461 sequences from chr7\n",
      "\tExtracting 345 sequences from chr8\n",
      "\tExtracting 338 sequences from chr9\n",
      "\tExtracting 366 sequences from chr10\n",
      "\tExtracting 507 sequences from chr11\n",
      "\tExtracting 268 sequences from chr12\n",
      "\tExtracting 273 sequences from chr13\n",
      "\tExtracting 222 sequences from chr14\n",
      "\tExtracting 278 sequences from chr15\n",
      "\tExtracting 235 sequences from chr16\n",
      "\tExtracting 309 sequences from chr17\n",
      "\tExtracting 190 sequences from chr18\n",
      "\tExtracting 175 sequences from chr19\n",
      "\tExtracting 81 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6373 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6373\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6373\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 435 sequences from chr1\n",
      "\tExtracting 437 sequences from chr2\n",
      "\tExtracting 330 sequences from chr3\n",
      "\tExtracting 403 sequences from chr4\n",
      "\tExtracting 406 sequences from chr5\n",
      "\tExtracting 331 sequences from chr6\n",
      "\tExtracting 494 sequences from chr7\n",
      "\tExtracting 328 sequences from chr8\n",
      "\tExtracting 372 sequences from chr9\n",
      "\tExtracting 383 sequences from chr10\n",
      "\tExtracting 452 sequences from chr11\n",
      "\tExtracting 286 sequences from chr12\n",
      "\tExtracting 281 sequences from chr13\n",
      "\tExtracting 232 sequences from chr14\n",
      "\tExtracting 244 sequences from chr15\n",
      "\tExtracting 229 sequences from chr16\n",
      "\tExtracting 319 sequences from chr17\n",
      "\tExtracting 173 sequences from chr18\n",
      "\tExtracting 199 sequences from chr19\n",
      "\tExtracting 39 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6373 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset34/wellington.tsv.gz tmp_dynamic/Subset34/homer.tsv.gz tmp_dynamic/Subset34/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset34/expression.tsv.gz tmp_dynamic/Subset34/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset34/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset36/wellington.tsv.gz tmp_dynamic/Subset36/homer.tsv.gz tmp_dynamic/Subset36/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset36/expression.tsv.gz tmp_dynamic/Subset36/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset36/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset34/binding.tsv.gz tmp_dynamic/Subset34/tssdist.tsv.gz tmp_dynamic/Subset34/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset36/binding.tsv.gz tmp_dynamic/Subset36/tssdist.tsv.gz tmp_dynamic/Subset36/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset34/linking.tsv.gz tmp_dynamic/Subset34/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset36/linking.tsv.gz tmp_dynamic/Subset36/binlinking.tsv.gz 20\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset45/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset45/expression.tsv.gz tmp_dynamic/Subset45/motifs.bed tmp_dynamic/Subset45/wellington.tsv.gz tmp_dynamic/Subset45/homer.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset46/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset46/expression.tsv.gz tmp_dynamic/Subset46/motifs.bed tmp_dynamic/Subset46/wellington.tsv.gz tmp_dynamic/Subset46/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5897\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5897\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 396 sequences from chr1\n",
      "\tExtracting 442 sequences from chr2\n",
      "\tExtracting 323 sequences from chr3\n",
      "\tExtracting 376 sequences from chr4\n",
      "\tExtracting 359 sequences from chr5\n",
      "\tExtracting 274 sequences from chr6\n",
      "\tExtracting 431 sequences from chr7\n",
      "\tExtracting 314 sequences from chr8\n",
      "\tExtracting 327 sequences from chr9\n",
      "\tExtracting 335 sequences from chr10\n",
      "\tExtracting 443 sequences from chr11\n",
      "\tExtracting 241 sequences from chr12\n",
      "\tExtracting 247 sequences from chr13\n",
      "\tExtracting 221 sequences from chr14\n",
      "\tExtracting 237 sequences from chr15\n",
      "\tExtracting 223 sequences from chr16\n",
      "\tExtracting 311 sequences from chr17\n",
      "\tExtracting 153 sequences from chr18\n",
      "\tExtracting 183 sequences from chr19\n",
      "\tExtracting 61 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5897 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5896\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5896\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 393 sequences from chr1\n",
      "\tExtracting 438 sequences from chr2\n",
      "\tExtracting 302 sequences from chr3\n",
      "\tExtracting 397 sequences from chr4\n",
      "\tExtracting 386 sequences from chr5\n",
      "\tExtracting 289 sequences from chr6\n",
      "\tExtracting 409 sequences from chr7\n",
      "\tExtracting 326 sequences from chr8\n",
      "\tExtracting 308 sequences from chr9\n",
      "\tExtracting 331 sequences from chr10\n",
      "\tExtracting 413 sequences from chr11\n",
      "\tExtracting 230 sequences from chr12\n",
      "\tExtracting 248 sequences from chr13\n",
      "\tExtracting 230 sequences from chr14\n",
      "\tExtracting 262 sequences from chr15\n",
      "\tExtracting 260 sequences from chr16\n",
      "\tExtracting 261 sequences from chr17\n",
      "\tExtracting 149 sequences from chr18\n",
      "\tExtracting 188 sequences from chr19\n",
      "\tExtracting 76 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5896 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5897\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5897\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 366 sequences from chr1\n",
      "\tExtracting 449 sequences from chr2\n",
      "\tExtracting 317 sequences from chr3\n",
      "\tExtracting 385 sequences from chr4\n",
      "\tExtracting 376 sequences from chr5\n",
      "\tExtracting 280 sequences from chr6\n",
      "\tExtracting 437 sequences from chr7\n",
      "\tExtracting 310 sequences from chr8\n",
      "\tExtracting 323 sequences from chr9\n",
      "\tExtracting 347 sequences from chr10\n",
      "\tExtracting 430 sequences from chr11\n",
      "\tExtracting 255 sequences from chr12\n",
      "\tExtracting 256 sequences from chr13\n",
      "\tExtracting 228 sequences from chr14\n",
      "\tExtracting 252 sequences from chr15\n",
      "\tExtracting 216 sequences from chr16\n",
      "\tExtracting 285 sequences from chr17\n",
      "\tExtracting 143 sequences from chr18\n",
      "\tExtracting 193 sequences from chr19\n",
      "\tExtracting 49 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5897 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5897\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5897\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 407 sequences from chr1\n",
      "\tExtracting 425 sequences from chr2\n",
      "\tExtracting 309 sequences from chr3\n",
      "\tExtracting 344 sequences from chr4\n",
      "\tExtracting 374 sequences from chr5\n",
      "\tExtracting 340 sequences from chr6\n",
      "\tExtracting 431 sequences from chr7\n",
      "\tExtracting 310 sequences from chr8\n",
      "\tExtracting 327 sequences from chr9\n",
      "\tExtracting 358 sequences from chr10\n",
      "\tExtracting 427 sequences from chr11\n",
      "\tExtracting 277 sequences from chr12\n",
      "\tExtracting 234 sequences from chr13\n",
      "\tExtracting 211 sequences from chr14\n",
      "\tExtracting 208 sequences from chr15\n",
      "\tExtracting 236 sequences from chr16\n",
      "\tExtracting 288 sequences from chr17\n",
      "\tExtracting 188 sequences from chr18\n",
      "\tExtracting 169 sequences from chr19\n",
      "\tExtracting 34 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5897 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5759\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5759\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 362 sequences from chr1\n",
      "\tExtracting 449 sequences from chr2\n",
      "\tExtracting 302 sequences from chr3\n",
      "\tExtracting 367 sequences from chr4\n",
      "\tExtracting 374 sequences from chr5\n",
      "\tExtracting 283 sequences from chr6\n",
      "\tExtracting 431 sequences from chr7\n",
      "\tExtracting 304 sequences from chr8\n",
      "\tExtracting 331 sequences from chr9\n",
      "\tExtracting 334 sequences from chr10\n",
      "\tExtracting 378 sequences from chr11\n",
      "\tExtracting 216 sequences from chr12\n",
      "\tExtracting 232 sequences from chr13\n",
      "\tExtracting 222 sequences from chr14\n",
      "\tExtracting 238 sequences from chr15\n",
      "\tExtracting 237 sequences from chr16\n",
      "\tExtracting 297 sequences from chr17\n",
      "\tExtracting 140 sequences from chr18\n",
      "\tExtracting 200 sequences from chr19\n",
      "\tExtracting 62 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5759 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5755\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5755\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 374 sequences from chr1\n",
      "\tExtracting 382 sequences from chr2\n",
      "\tExtracting 307 sequences from chr3\n",
      "\tExtracting 399 sequences from chr4\n",
      "\tExtracting 340 sequences from chr5\n",
      "\tExtracting 263 sequences from chr6\n",
      "\tExtracting 424 sequences from chr7\n",
      "\tExtracting 309 sequences from chr8\n",
      "\tExtracting 324 sequences from chr9\n",
      "\tExtracting 328 sequences from chr10\n",
      "\tExtracting 427 sequences from chr11\n",
      "\tExtracting 270 sequences from chr12\n",
      "\tExtracting 234 sequences from chr13\n",
      "\tExtracting 208 sequences from chr14\n",
      "\tExtracting 246 sequences from chr15\n",
      "\tExtracting 231 sequences from chr16\n",
      "\tExtracting 286 sequences from chr17\n",
      "\tExtracting 155 sequences from chr18\n",
      "\tExtracting 184 sequences from chr19\n",
      "\tExtracting 64 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5755 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5759\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5759\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 357 sequences from chr1\n",
      "\tExtracting 452 sequences from chr2\n",
      "\tExtracting 306 sequences from chr3\n",
      "\tExtracting 345 sequences from chr4\n",
      "\tExtracting 391 sequences from chr5\n",
      "\tExtracting 271 sequences from chr6\n",
      "\tExtracting 416 sequences from chr7\n",
      "\tExtracting 300 sequences from chr8\n",
      "\tExtracting 308 sequences from chr9\n",
      "\tExtracting 356 sequences from chr10\n",
      "\tExtracting 452 sequences from chr11\n",
      "\tExtracting 235 sequences from chr12\n",
      "\tExtracting 249 sequences from chr13\n",
      "\tExtracting 202 sequences from chr14\n",
      "\tExtracting 228 sequences from chr15\n",
      "\tExtracting 219 sequences from chr16\n",
      "\tExtracting 273 sequences from chr17\n",
      "\tExtracting 159 sequences from chr18\n",
      "\tExtracting 188 sequences from chr19\n",
      "\tExtracting 52 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5759 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5759\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5759\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 392 sequences from chr1\n",
      "\tExtracting 437 sequences from chr2\n",
      "\tExtracting 311 sequences from chr3\n",
      "\tExtracting 334 sequences from chr4\n",
      "\tExtracting 362 sequences from chr5\n",
      "\tExtracting 331 sequences from chr6\n",
      "\tExtracting 433 sequences from chr7\n",
      "\tExtracting 307 sequences from chr8\n",
      "\tExtracting 318 sequences from chr9\n",
      "\tExtracting 344 sequences from chr10\n",
      "\tExtracting 394 sequences from chr11\n",
      "\tExtracting 270 sequences from chr12\n",
      "\tExtracting 232 sequences from chr13\n",
      "\tExtracting 222 sequences from chr14\n",
      "\tExtracting 195 sequences from chr15\n",
      "\tExtracting 234 sequences from chr16\n",
      "\tExtracting 281 sequences from chr17\n",
      "\tExtracting 174 sequences from chr18\n",
      "\tExtracting 161 sequences from chr19\n",
      "\tExtracting 27 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5759 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset45/wellington.tsv.gz tmp_dynamic/Subset45/homer.tsv.gz tmp_dynamic/Subset45/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset45/expression.tsv.gz tmp_dynamic/Subset45/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset45/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset46/wellington.tsv.gz tmp_dynamic/Subset46/homer.tsv.gz tmp_dynamic/Subset46/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset46/expression.tsv.gz tmp_dynamic/Subset46/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset46/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset45/binding.tsv.gz tmp_dynamic/Subset45/tssdist.tsv.gz tmp_dynamic/Subset45/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset46/binding.tsv.gz tmp_dynamic/Subset46/tssdist.tsv.gz tmp_dynamic/Subset46/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset45/linking.tsv.gz tmp_dynamic/Subset45/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset46/linking.tsv.gz tmp_dynamic/Subset46/binlinking.tsv.gz 20\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset47/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset47/expression.tsv.gz tmp_dynamic/Subset47/motifs.bed tmp_dynamic/Subset47/wellington.tsv.gz tmp_dynamic/Subset47/homer.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset48/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset48/expression.tsv.gz tmp_dynamic/Subset48/motifs.bed tmp_dynamic/Subset48/wellington.tsv.gz tmp_dynamic/Subset48/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5699\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5699\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 365 sequences from chr1\n",
      "\tExtracting 435 sequences from chr2\n",
      "\tExtracting 289 sequences from chr3\n",
      "\tExtracting 316 sequences from chr4\n",
      "\tExtracting 359 sequences from chr5\n",
      "\tExtracting 299 sequences from chr6\n",
      "\tExtracting 416 sequences from chr7\n",
      "\tExtracting 334 sequences from chr8\n",
      "\tExtracting 323 sequences from chr9\n",
      "\tExtracting 364 sequences from chr10\n",
      "\tExtracting 425 sequences from chr11\n",
      "\tExtracting 232 sequences from chr12\n",
      "\tExtracting 218 sequences from chr13\n",
      "\tExtracting 214 sequences from chr14\n",
      "\tExtracting 235 sequences from chr15\n",
      "\tExtracting 220 sequences from chr16\n",
      "\tExtracting 275 sequences from chr17\n",
      "\tExtracting 155 sequences from chr18\n",
      "\tExtracting 171 sequences from chr19\n",
      "\tExtracting 54 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5699 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5695\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5695\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 358 sequences from chr1\n",
      "\tExtracting 418 sequences from chr2\n",
      "\tExtracting 311 sequences from chr3\n",
      "\tExtracting 359 sequences from chr4\n",
      "\tExtracting 386 sequences from chr5\n",
      "\tExtracting 249 sequences from chr6\n",
      "\tExtracting 438 sequences from chr7\n",
      "\tExtracting 286 sequences from chr8\n",
      "\tExtracting 312 sequences from chr9\n",
      "\tExtracting 304 sequences from chr10\n",
      "\tExtracting 414 sequences from chr11\n",
      "\tExtracting 249 sequences from chr12\n",
      "\tExtracting 239 sequences from chr13\n",
      "\tExtracting 245 sequences from chr14\n",
      "\tExtracting 256 sequences from chr15\n",
      "\tExtracting 207 sequences from chr16\n",
      "\tExtracting 265 sequences from chr17\n",
      "\tExtracting 157 sequences from chr18\n",
      "\tExtracting 180 sequences from chr19\n",
      "\tExtracting 62 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5695 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5699\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5699\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 356 sequences from chr1\n",
      "\tExtracting 420 sequences from chr2\n",
      "\tExtracting 303 sequences from chr3\n",
      "\tExtracting 354 sequences from chr4\n",
      "\tExtracting 368 sequences from chr5\n",
      "\tExtracting 275 sequences from chr6\n",
      "\tExtracting 387 sequences from chr7\n",
      "\tExtracting 294 sequences from chr8\n",
      "\tExtracting 314 sequences from chr9\n",
      "\tExtracting 331 sequences from chr10\n",
      "\tExtracting 401 sequences from chr11\n",
      "\tExtracting 247 sequences from chr12\n",
      "\tExtracting 246 sequences from chr13\n",
      "\tExtracting 213 sequences from chr14\n",
      "\tExtracting 235 sequences from chr15\n",
      "\tExtracting 241 sequences from chr16\n",
      "\tExtracting 296 sequences from chr17\n",
      "\tExtracting 141 sequences from chr18\n",
      "\tExtracting 215 sequences from chr19\n",
      "\tExtracting 62 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5699 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5699\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5699\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 387 sequences from chr1\n",
      "\tExtracting 428 sequences from chr2\n",
      "\tExtracting 327 sequences from chr3\n",
      "\tExtracting 341 sequences from chr4\n",
      "\tExtracting 367 sequences from chr5\n",
      "\tExtracting 309 sequences from chr6\n",
      "\tExtracting 422 sequences from chr7\n",
      "\tExtracting 293 sequences from chr8\n",
      "\tExtracting 311 sequences from chr9\n",
      "\tExtracting 338 sequences from chr10\n",
      "\tExtracting 389 sequences from chr11\n",
      "\tExtracting 270 sequences from chr12\n",
      "\tExtracting 245 sequences from chr13\n",
      "\tExtracting 218 sequences from chr14\n",
      "\tExtracting 190 sequences from chr15\n",
      "\tExtracting 222 sequences from chr16\n",
      "\tExtracting 279 sequences from chr17\n",
      "\tExtracting 180 sequences from chr18\n",
      "\tExtracting 159 sequences from chr19\n",
      "\tExtracting 24 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5699 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset47/wellington.tsv.gz tmp_dynamic/Subset47/homer.tsv.gz tmp_dynamic/Subset47/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset47/expression.tsv.gz tmp_dynamic/Subset47/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset47/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset47/binding.tsv.gz tmp_dynamic/Subset47/tssdist.tsv.gz tmp_dynamic/Subset47/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset47/linking.tsv.gz tmp_dynamic/Subset47/binlinking.tsv.gz 20\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5709\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5709\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 407 sequences from chr1\n",
      "\tExtracting 429 sequences from chr2\n",
      "\tExtracting 313 sequences from chr3\n",
      "\tExtracting 373 sequences from chr4\n",
      "\tExtracting 362 sequences from chr5\n",
      "\tExtracting 252 sequences from chr6\n",
      "\tExtracting 401 sequences from chr7\n",
      "\tExtracting 301 sequences from chr8\n",
      "\tExtracting 297 sequences from chr9\n",
      "\tExtracting 381 sequences from chr10\n",
      "\tExtracting 415 sequences from chr11\n",
      "\tExtracting 230 sequences from chr12\n",
      "\tExtracting 228 sequences from chr13\n",
      "\tExtracting 194 sequences from chr14\n",
      "\tExtracting 252 sequences from chr15\n",
      "\tExtracting 217 sequences from chr16\n",
      "\tExtracting 261 sequences from chr17\n",
      "\tExtracting 155 sequences from chr18\n",
      "\tExtracting 180 sequences from chr19\n",
      "\tExtracting 61 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5709 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5711\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5711\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 364 sequences from chr1\n",
      "\tExtracting 401 sequences from chr2\n",
      "\tExtracting 322 sequences from chr3\n",
      "\tExtracting 339 sequences from chr4\n",
      "\tExtracting 371 sequences from chr5\n",
      "\tExtracting 266 sequences from chr6\n",
      "\tExtracting 405 sequences from chr7\n",
      "\tExtracting 317 sequences from chr8\n",
      "\tExtracting 326 sequences from chr9\n",
      "\tExtracting 300 sequences from chr10\n",
      "\tExtracting 404 sequences from chr11\n",
      "\tExtracting 254 sequences from chr12\n",
      "\tExtracting 256 sequences from chr13\n",
      "\tExtracting 232 sequences from chr14\n",
      "\tExtracting 236 sequences from chr15\n",
      "\tExtracting 234 sequences from chr16\n",
      "\tExtracting 275 sequences from chr17\n",
      "\tExtracting 147 sequences from chr18\n",
      "\tExtracting 198 sequences from chr19\n",
      "\tExtracting 64 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5711 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5711\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5711\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 345 sequences from chr1\n",
      "\tExtracting 450 sequences from chr2\n",
      "\tExtracting 324 sequences from chr3\n",
      "\tExtracting 333 sequences from chr4\n",
      "\tExtracting 394 sequences from chr5\n",
      "\tExtracting 278 sequences from chr6\n",
      "\tExtracting 417 sequences from chr7\n",
      "\tExtracting 311 sequences from chr8\n",
      "\tExtracting 314 sequences from chr9\n",
      "\tExtracting 345 sequences from chr10\n",
      "\tExtracting 404 sequences from chr11\n",
      "\tExtracting 232 sequences from chr12\n",
      "\tExtracting 237 sequences from chr13\n",
      "\tExtracting 204 sequences from chr14\n",
      "\tExtracting 237 sequences from chr15\n",
      "\tExtracting 214 sequences from chr16\n",
      "\tExtracting 287 sequences from chr17\n",
      "\tExtracting 164 sequences from chr18\n",
      "\tExtracting 171 sequences from chr19\n",
      "\tExtracting 50 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5711 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5711\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5711\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 386 sequences from chr1\n",
      "\tExtracting 422 sequences from chr2\n",
      "\tExtracting 294 sequences from chr3\n",
      "\tExtracting 338 sequences from chr4\n",
      "\tExtracting 362 sequences from chr5\n",
      "\tExtracting 323 sequences from chr6\n",
      "\tExtracting 431 sequences from chr7\n",
      "\tExtracting 311 sequences from chr8\n",
      "\tExtracting 313 sequences from chr9\n",
      "\tExtracting 346 sequences from chr10\n",
      "\tExtracting 392 sequences from chr11\n",
      "\tExtracting 275 sequences from chr12\n",
      "\tExtracting 231 sequences from chr13\n",
      "\tExtracting 225 sequences from chr14\n",
      "\tExtracting 190 sequences from chr15\n",
      "\tExtracting 233 sequences from chr16\n",
      "\tExtracting 282 sequences from chr17\n",
      "\tExtracting 172 sequences from chr18\n",
      "\tExtracting 160 sequences from chr19\n",
      "\tExtracting 25 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5711 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset48/wellington.tsv.gz tmp_dynamic/Subset48/homer.tsv.gz tmp_dynamic/Subset48/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset48/expression.tsv.gz tmp_dynamic/Subset48/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset48/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset48/binding.tsv.gz tmp_dynamic/Subset48/tssdist.tsv.gz tmp_dynamic/Subset48/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset48/linking.tsv.gz tmp_dynamic/Subset48/binlinking.tsv.gz 20\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset49/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset49/expression.tsv.gz tmp_dynamic/Subset49/motifs.bed tmp_dynamic/Subset49/wellington.tsv.gz tmp_dynamic/Subset49/homer.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset50/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset50/expression.tsv.gz tmp_dynamic/Subset50/motifs.bed tmp_dynamic/Subset50/wellington.tsv.gz tmp_dynamic/Subset50/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5856\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5856\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 410 sequences from chr1\n",
      "\tExtracting 440 sequences from chr2\n",
      "\tExtracting 312 sequences from chr3\n",
      "\tExtracting 371 sequences from chr4\n",
      "\tExtracting 389 sequences from chr5\n",
      "\tExtracting 283 sequences from chr6\n",
      "\tExtracting 397 sequences from chr7\n",
      "\tExtracting 336 sequences from chr8\n",
      "\tExtracting 318 sequences from chr9\n",
      "\tExtracting 319 sequences from chr10\n",
      "\tExtracting 424 sequences from chr11\n",
      "\tExtracting 238 sequences from chr12\n",
      "\tExtracting 265 sequences from chr13\n",
      "\tExtracting 212 sequences from chr14\n",
      "\tExtracting 248 sequences from chr15\n",
      "\tExtracting 235 sequences from chr16\n",
      "\tExtracting 271 sequences from chr17\n",
      "\tExtracting 163 sequences from chr18\n",
      "\tExtracting 171 sequences from chr19\n",
      "\tExtracting 54 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5856 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5852\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5852\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 377 sequences from chr1\n",
      "\tExtracting 455 sequences from chr2\n",
      "\tExtracting 311 sequences from chr3\n",
      "\tExtracting 383 sequences from chr4\n",
      "\tExtracting 354 sequences from chr5\n",
      "\tExtracting 260 sequences from chr6\n",
      "\tExtracting 392 sequences from chr7\n",
      "\tExtracting 295 sequences from chr8\n",
      "\tExtracting 346 sequences from chr9\n",
      "\tExtracting 350 sequences from chr10\n",
      "\tExtracting 398 sequences from chr11\n",
      "\tExtracting 261 sequences from chr12\n",
      "\tExtracting 239 sequences from chr13\n",
      "\tExtracting 232 sequences from chr14\n",
      "\tExtracting 274 sequences from chr15\n",
      "\tExtracting 226 sequences from chr16\n",
      "\tExtracting 270 sequences from chr17\n",
      "\tExtracting 167 sequences from chr18\n",
      "\tExtracting 196 sequences from chr19\n",
      "\tExtracting 66 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5852 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5856\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5856\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 363 sequences from chr1\n",
      "\tExtracting 416 sequences from chr2\n",
      "\tExtracting 305 sequences from chr3\n",
      "\tExtracting 349 sequences from chr4\n",
      "\tExtracting 419 sequences from chr5\n",
      "\tExtracting 297 sequences from chr6\n",
      "\tExtracting 428 sequences from chr7\n",
      "\tExtracting 328 sequences from chr8\n",
      "\tExtracting 317 sequences from chr9\n",
      "\tExtracting 353 sequences from chr10\n",
      "\tExtracting 422 sequences from chr11\n",
      "\tExtracting 255 sequences from chr12\n",
      "\tExtracting 245 sequences from chr13\n",
      "\tExtracting 206 sequences from chr14\n",
      "\tExtracting 224 sequences from chr15\n",
      "\tExtracting 221 sequences from chr16\n",
      "\tExtracting 298 sequences from chr17\n",
      "\tExtracting 164 sequences from chr18\n",
      "\tExtracting 190 sequences from chr19\n",
      "\tExtracting 56 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5856 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5856\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5856\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 395 sequences from chr1\n",
      "\tExtracting 430 sequences from chr2\n",
      "\tExtracting 322 sequences from chr3\n",
      "\tExtracting 348 sequences from chr4\n",
      "\tExtracting 370 sequences from chr5\n",
      "\tExtracting 315 sequences from chr6\n",
      "\tExtracting 436 sequences from chr7\n",
      "\tExtracting 313 sequences from chr8\n",
      "\tExtracting 314 sequences from chr9\n",
      "\tExtracting 367 sequences from chr10\n",
      "\tExtracting 410 sequences from chr11\n",
      "\tExtracting 277 sequences from chr12\n",
      "\tExtracting 238 sequences from chr13\n",
      "\tExtracting 238 sequences from chr14\n",
      "\tExtracting 197 sequences from chr15\n",
      "\tExtracting 242 sequences from chr16\n",
      "\tExtracting 288 sequences from chr17\n",
      "\tExtracting 170 sequences from chr18\n",
      "\tExtracting 164 sequences from chr19\n",
      "\tExtracting 22 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5856 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset49/wellington.tsv.gz tmp_dynamic/Subset49/homer.tsv.gz tmp_dynamic/Subset49/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset49/expression.tsv.gz tmp_dynamic/Subset49/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset49/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset49/binding.tsv.gz tmp_dynamic/Subset49/tssdist.tsv.gz tmp_dynamic/Subset49/linking.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5836\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5836\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 394 sequences from chr1\n",
      "\tExtracting 402 sequences from chr2\n",
      "\tExtracting 319 sequences from chr3\n",
      "\tExtracting 339 sequences from chr4\n",
      "\tExtracting 390 sequences from chr5\n",
      "\tExtracting 324 sequences from chr6\n",
      "\tExtracting 444 sequences from chr7\n",
      "\tExtracting 306 sequences from chr8\n",
      "\tExtracting 307 sequences from chr9\n",
      "\tExtracting 361 sequences from chr10\n",
      "\tExtracting 437 sequences from chr11\n",
      "\tExtracting 254 sequences from chr12\n",
      "\tExtracting 257 sequences from chr13\n",
      "\tExtracting 197 sequences from chr14\n",
      "\tExtracting 226 sequences from chr15\n",
      "\tExtracting 206 sequences from chr16\n",
      "\tExtracting 279 sequences from chr17\n",
      "\tExtracting 160 sequences from chr18\n",
      "\tExtracting 192 sequences from chr19\n",
      "\tExtracting 42 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5836 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5836\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5836\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 395 sequences from chr1\n",
      "\tExtracting 459 sequences from chr2\n",
      "\tExtracting 304 sequences from chr3\n",
      "\tExtracting 372 sequences from chr4\n",
      "\tExtracting 387 sequences from chr5\n",
      "\tExtracting 297 sequences from chr6\n",
      "\tExtracting 397 sequences from chr7\n",
      "\tExtracting 331 sequences from chr8\n",
      "\tExtracting 343 sequences from chr9\n",
      "\tExtracting 311 sequences from chr10\n",
      "\tExtracting 424 sequences from chr11\n",
      "\tExtracting 217 sequences from chr12\n",
      "\tExtracting 232 sequences from chr13\n",
      "\tExtracting 215 sequences from chr14\n",
      "\tExtracting 230 sequences from chr15\n",
      "\tExtracting 240 sequences from chr16\n",
      "\tExtracting 279 sequences from chr17\n",
      "\tExtracting 158 sequences from chr18\n",
      "\tExtracting 175 sequences from chr19\n",
      "\tExtracting 70 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5836 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5834\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5834\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 358 sequences from chr1\n",
      "\tExtracting 421 sequences from chr2\n",
      "\tExtracting 320 sequences from chr3\n",
      "\tExtracting 372 sequences from chr4\n",
      "\tExtracting 379 sequences from chr5\n",
      "\tExtracting 254 sequences from chr6\n",
      "\tExtracting 413 sequences from chr7\n",
      "\tExtracting 307 sequences from chr8\n",
      "\tExtracting 324 sequences from chr9\n",
      "\tExtracting 316 sequences from chr10\n",
      "\tExtracting 419 sequences from chr11\n",
      "\tExtracting 251 sequences from chr12\n",
      "\tExtracting 265 sequences from chr13\n",
      "\tExtracting 220 sequences from chr14\n",
      "\tExtracting 278 sequences from chr15\n",
      "\tExtracting 230 sequences from chr16\n",
      "\tExtracting 289 sequences from chr17\n",
      "\tExtracting 169 sequences from chr18\n",
      "\tExtracting 189 sequences from chr19\n",
      "\tExtracting 60 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5834 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5836\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5836\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 391 sequences from chr1\n",
      "\tExtracting 432 sequences from chr2\n",
      "\tExtracting 309 sequences from chr3\n",
      "\tExtracting 352 sequences from chr4\n",
      "\tExtracting 374 sequences from chr5\n",
      "\tExtracting 297 sequences from chr6\n",
      "\tExtracting 444 sequences from chr7\n",
      "\tExtracting 314 sequences from chr8\n",
      "\tExtracting 322 sequences from chr9\n",
      "\tExtracting 356 sequences from chr10\n",
      "\tExtracting 397 sequences from chr11\n",
      "\tExtracting 292 sequences from chr12\n",
      "\tExtracting 238 sequences from chr13\n",
      "\tExtracting 234 sequences from chr14\n",
      "\tExtracting 203 sequences from chr15\n",
      "\tExtracting 236 sequences from chr16\n",
      "\tExtracting 275 sequences from chr17\n",
      "\tExtracting 167 sequences from chr18\n",
      "\tExtracting 173 sequences from chr19\n",
      "\tExtracting 30 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5836 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset49/linking.tsv.gz tmp_dynamic/Subset49/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset50/wellington.tsv.gz tmp_dynamic/Subset50/homer.tsv.gz tmp_dynamic/Subset50/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset50/expression.tsv.gz tmp_dynamic/Subset50/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset50/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset50/binding.tsv.gz tmp_dynamic/Subset50/tssdist.tsv.gz tmp_dynamic/Subset50/linking.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset52/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset52/expression.tsv.gz tmp_dynamic/Subset52/motifs.bed tmp_dynamic/Subset52/wellington.tsv.gz tmp_dynamic/Subset52/homer.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset50/linking.tsv.gz tmp_dynamic/Subset50/binlinking.tsv.gz 20\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset55/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset55/expression.tsv.gz tmp_dynamic/Subset55/motifs.bed tmp_dynamic/Subset55/wellington.tsv.gz tmp_dynamic/Subset55/homer.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset51/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset51/expression.tsv.gz tmp_dynamic/Subset51/motifs.bed tmp_dynamic/Subset51/wellington.tsv.gz tmp_dynamic/Subset51/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5546\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5546\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 346 sequences from chr1\n",
      "\tExtracting 410 sequences from chr2\n",
      "\tExtracting 284 sequences from chr3\n",
      "\tExtracting 335 sequences from chr4\n",
      "\tExtracting 352 sequences from chr5\n",
      "\tExtracting 275 sequences from chr6\n",
      "\tExtracting 415 sequences from chr7\n",
      "\tExtracting 288 sequences from chr8\n",
      "\tExtracting 317 sequences from chr9\n",
      "\tExtracting 323 sequences from chr10\n",
      "\tExtracting 392 sequences from chr11\n",
      "\tExtracting 218 sequences from chr12\n",
      "\tExtracting 260 sequences from chr13\n",
      "\tExtracting 234 sequences from chr14\n",
      "\tExtracting 229 sequences from chr15\n",
      "\tExtracting 187 sequences from chr16\n",
      "\tExtracting 273 sequences from chr17\n",
      "\tExtracting 160 sequences from chr18\n",
      "\tExtracting 183 sequences from chr19\n",
      "\tExtracting 65 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5546 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5549\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5549\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 357 sequences from chr1\n",
      "\tExtracting 388 sequences from chr2\n",
      "\tExtracting 308 sequences from chr3\n",
      "\tExtracting 353 sequences from chr4\n",
      "\tExtracting 382 sequences from chr5\n",
      "\tExtracting 264 sequences from chr6\n",
      "\tExtracting 347 sequences from chr7\n",
      "\tExtracting 305 sequences from chr8\n",
      "\tExtracting 299 sequences from chr9\n",
      "\tExtracting 327 sequences from chr10\n",
      "\tExtracting 441 sequences from chr11\n",
      "\tExtracting 245 sequences from chr12\n",
      "\tExtracting 224 sequences from chr13\n",
      "\tExtracting 206 sequences from chr14\n",
      "\tExtracting 216 sequences from chr15\n",
      "\tExtracting 232 sequences from chr16\n",
      "\tExtracting 271 sequences from chr17\n",
      "\tExtracting 167 sequences from chr18\n",
      "\tExtracting 168 sequences from chr19\n",
      "\tExtracting 49 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5549 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5549\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5549\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 361 sequences from chr1\n",
      "\tExtracting 396 sequences from chr2\n",
      "\tExtracting 301 sequences from chr3\n",
      "\tExtracting 311 sequences from chr4\n",
      "\tExtracting 364 sequences from chr5\n",
      "\tExtracting 283 sequences from chr6\n",
      "\tExtracting 413 sequences from chr7\n",
      "\tExtracting 313 sequences from chr8\n",
      "\tExtracting 307 sequences from chr9\n",
      "\tExtracting 336 sequences from chr10\n",
      "\tExtracting 445 sequences from chr11\n",
      "\tExtracting 221 sequences from chr12\n",
      "\tExtracting 227 sequences from chr13\n",
      "\tExtracting 193 sequences from chr14\n",
      "\tExtracting 232 sequences from chr15\n",
      "\tExtracting 197 sequences from chr16\n",
      "\tExtracting 268 sequences from chr17\n",
      "\tExtracting 157 sequences from chr18\n",
      "\tExtracting 187 sequences from chr19\n",
      "\tExtracting 37 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5549 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5549\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5549\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 350 sequences from chr1\n",
      "\tExtracting 396 sequences from chr2\n",
      "\tExtracting 295 sequences from chr3\n",
      "\tExtracting 355 sequences from chr4\n",
      "\tExtracting 340 sequences from chr5\n",
      "\tExtracting 284 sequences from chr6\n",
      "\tExtracting 433 sequences from chr7\n",
      "\tExtracting 288 sequences from chr8\n",
      "\tExtracting 302 sequences from chr9\n",
      "\tExtracting 330 sequences from chr10\n",
      "\tExtracting 379 sequences from chr11\n",
      "\tExtracting 284 sequences from chr12\n",
      "\tExtracting 234 sequences from chr13\n",
      "\tExtracting 215 sequences from chr14\n",
      "\tExtracting 206 sequences from chr15\n",
      "\tExtracting 217 sequences from chr16\n",
      "\tExtracting 265 sequences from chr17\n",
      "\tExtracting 164 sequences from chr18\n",
      "\tExtracting 185 sequences from chr19\n",
      "\tExtracting 27 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5549 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset54/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset54/expression.tsv.gz tmp_dynamic/Subset54/motifs.bed tmp_dynamic/Subset54/wellington.tsv.gz tmp_dynamic/Subset54/homer.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset52/wellington.tsv.gz tmp_dynamic/Subset52/homer.tsv.gz tmp_dynamic/Subset52/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset52/expression.tsv.gz tmp_dynamic/Subset52/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset52/tssdist.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset53/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset53/expression.tsv.gz tmp_dynamic/Subset53/motifs.bed tmp_dynamic/Subset53/wellington.tsv.gz tmp_dynamic/Subset53/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5617\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5617\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 357 sequences from chr1\n",
      "\tExtracting 406 sequences from chr2\n",
      "\tExtracting 293 sequences from chr3\n",
      "\tExtracting 366 sequences from chr4\n",
      "\tExtracting 372 sequences from chr5\n",
      "\tExtracting 260 sequences from chr6\n",
      "\tExtracting 440 sequences from chr7\n",
      "\tExtracting 339 sequences from chr8\n",
      "\tExtracting 287 sequences from chr9\n",
      "\tExtracting 334 sequences from chr10\n",
      "\tExtracting 363 sequences from chr11\n",
      "\tExtracting 262 sequences from chr12\n",
      "\tExtracting 223 sequences from chr13\n",
      "\tExtracting 203 sequences from chr14\n",
      "\tExtracting 243 sequences from chr15\n",
      "\tExtracting 204 sequences from chr16\n",
      "\tExtracting 262 sequences from chr17\n",
      "\tExtracting 167 sequences from chr18\n",
      "\tExtracting 192 sequences from chr19\n",
      "\tExtracting 44 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5617 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5615\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5615\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 364 sequences from chr1\n",
      "\tExtracting 416 sequences from chr2\n",
      "\tExtracting 284 sequences from chr3\n",
      "\tExtracting 361 sequences from chr4\n",
      "\tExtracting 360 sequences from chr5\n",
      "\tExtracting 263 sequences from chr6\n",
      "\tExtracting 403 sequences from chr7\n",
      "\tExtracting 288 sequences from chr8\n",
      "\tExtracting 323 sequences from chr9\n",
      "\tExtracting 305 sequences from chr10\n",
      "\tExtracting 426 sequences from chr11\n",
      "\tExtracting 233 sequences from chr12\n",
      "\tExtracting 236 sequences from chr13\n",
      "\tExtracting 210 sequences from chr14\n",
      "\tExtracting 250 sequences from chr15\n",
      "\tExtracting 233 sequences from chr16\n",
      "\tExtracting 274 sequences from chr17\n",
      "\tExtracting 159 sequences from chr18\n",
      "\tExtracting 185 sequences from chr19\n",
      "\tExtracting 42 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5615 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5617\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5617\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 326 sequences from chr1\n",
      "\tExtracting 402 sequences from chr2\n",
      "\tExtracting 286 sequences from chr3\n",
      "\tExtracting 356 sequences from chr4\n",
      "\tExtracting 378 sequences from chr5\n",
      "\tExtracting 274 sequences from chr6\n",
      "\tExtracting 435 sequences from chr7\n",
      "\tExtracting 291 sequences from chr8\n",
      "\tExtracting 308 sequences from chr9\n",
      "\tExtracting 343 sequences from chr10\n",
      "\tExtracting 447 sequences from chr11\n",
      "\tExtracting 249 sequences from chr12\n",
      "\tExtracting 233 sequences from chr13\n",
      "\tExtracting 198 sequences from chr14\n",
      "\tExtracting 219 sequences from chr15\n",
      "\tExtracting 214 sequences from chr16\n",
      "\tExtracting 257 sequences from chr17\n",
      "\tExtracting 170 sequences from chr18\n",
      "\tExtracting 194 sequences from chr19\n",
      "\tExtracting 37 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5617 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5617\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5617\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 387 sequences from chr1\n",
      "\tExtracting 413 sequences from chr2\n",
      "\tExtracting 294 sequences from chr3\n",
      "\tExtracting 338 sequences from chr4\n",
      "\tExtracting 341 sequences from chr5\n",
      "\tExtracting 283 sequences from chr6\n",
      "\tExtracting 425 sequences from chr7\n",
      "\tExtracting 309 sequences from chr8\n",
      "\tExtracting 333 sequences from chr9\n",
      "\tExtracting 325 sequences from chr10\n",
      "\tExtracting 413 sequences from chr11\n",
      "\tExtracting 257 sequences from chr12\n",
      "\tExtracting 235 sequences from chr13\n",
      "\tExtracting 210 sequences from chr14\n",
      "\tExtracting 219 sequences from chr15\n",
      "\tExtracting 204 sequences from chr16\n",
      "\tExtracting 262 sequences from chr17\n",
      "\tExtracting 162 sequences from chr18\n",
      "\tExtracting 177 sequences from chr19\n",
      "\tExtracting 30 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5617 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset52/binding.tsv.gz tmp_dynamic/Subset52/tssdist.tsv.gz tmp_dynamic/Subset52/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset55/wellington.tsv.gz tmp_dynamic/Subset55/homer.tsv.gz tmp_dynamic/Subset55/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset55/expression.tsv.gz tmp_dynamic/Subset55/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset55/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset52/linking.tsv.gz tmp_dynamic/Subset52/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset55/binding.tsv.gz tmp_dynamic/Subset55/tssdist.tsv.gz tmp_dynamic/Subset55/linking.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5821\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5821\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 335 sequences from chr1\n",
      "\tExtracting 428 sequences from chr2\n",
      "\tExtracting 279 sequences from chr3\n",
      "\tExtracting 349 sequences from chr4\n",
      "\tExtracting 398 sequences from chr5\n",
      "\tExtracting 292 sequences from chr6\n",
      "\tExtracting 426 sequences from chr7\n",
      "\tExtracting 325 sequences from chr8\n",
      "\tExtracting 327 sequences from chr9\n",
      "\tExtracting 317 sequences from chr10\n",
      "\tExtracting 436 sequences from chr11\n",
      "\tExtracting 253 sequences from chr12\n",
      "\tExtracting 243 sequences from chr13\n",
      "\tExtracting 200 sequences from chr14\n",
      "\tExtracting 262 sequences from chr15\n",
      "\tExtracting 224 sequences from chr16\n",
      "\tExtracting 305 sequences from chr17\n",
      "\tExtracting 173 sequences from chr18\n",
      "\tExtracting 183 sequences from chr19\n",
      "\tExtracting 66 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5821 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5825\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5825\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 385 sequences from chr1\n",
      "\tExtracting 420 sequences from chr2\n",
      "\tExtracting 322 sequences from chr3\n",
      "\tExtracting 351 sequences from chr4\n",
      "\tExtracting 383 sequences from chr5\n",
      "\tExtracting 294 sequences from chr6\n",
      "\tExtracting 430 sequences from chr7\n",
      "\tExtracting 332 sequences from chr8\n",
      "\tExtracting 329 sequences from chr9\n",
      "\tExtracting 341 sequences from chr10\n",
      "\tExtracting 442 sequences from chr11\n",
      "\tExtracting 257 sequences from chr12\n",
      "\tExtracting 243 sequences from chr13\n",
      "\tExtracting 204 sequences from chr14\n",
      "\tExtracting 218 sequences from chr15\n",
      "\tExtracting 208 sequences from chr16\n",
      "\tExtracting 274 sequences from chr17\n",
      "\tExtracting 165 sequences from chr18\n",
      "\tExtracting 191 sequences from chr19\n",
      "\tExtracting 36 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5825 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5825\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5825\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 416 sequences from chr1\n",
      "\tExtracting 452 sequences from chr2\n",
      "\tExtracting 311 sequences from chr3\n",
      "\tExtracting 361 sequences from chr4\n",
      "\tExtracting 384 sequences from chr5\n",
      "\tExtracting 285 sequences from chr6\n",
      "\tExtracting 377 sequences from chr7\n",
      "\tExtracting 312 sequences from chr8\n",
      "\tExtracting 319 sequences from chr9\n",
      "\tExtracting 329 sequences from chr10\n",
      "\tExtracting 431 sequences from chr11\n",
      "\tExtracting 239 sequences from chr12\n",
      "\tExtracting 244 sequences from chr13\n",
      "\tExtracting 225 sequences from chr14\n",
      "\tExtracting 223 sequences from chr15\n",
      "\tExtracting 231 sequences from chr16\n",
      "\tExtracting 260 sequences from chr17\n",
      "\tExtracting 182 sequences from chr18\n",
      "\tExtracting 183 sequences from chr19\n",
      "\tExtracting 61 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5825 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5825\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5825\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 382 sequences from chr1\n",
      "\tExtracting 423 sequences from chr2\n",
      "\tExtracting 300 sequences from chr3\n",
      "\tExtracting 359 sequences from chr4\n",
      "\tExtracting 365 sequences from chr5\n",
      "\tExtracting 302 sequences from chr6\n",
      "\tExtracting 447 sequences from chr7\n",
      "\tExtracting 305 sequences from chr8\n",
      "\tExtracting 330 sequences from chr9\n",
      "\tExtracting 352 sequences from chr10\n",
      "\tExtracting 394 sequences from chr11\n",
      "\tExtracting 288 sequences from chr12\n",
      "\tExtracting 245 sequences from chr13\n",
      "\tExtracting 227 sequences from chr14\n",
      "\tExtracting 213 sequences from chr15\n",
      "\tExtracting 232 sequences from chr16\n",
      "\tExtracting 281 sequences from chr17\n",
      "\tExtracting 163 sequences from chr18\n",
      "\tExtracting 182 sequences from chr19\n",
      "\tExtracting 35 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5825 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset55/linking.tsv.gz tmp_dynamic/Subset55/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset51/wellington.tsv.gz tmp_dynamic/Subset51/homer.tsv.gz tmp_dynamic/Subset51/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset51/expression.tsv.gz tmp_dynamic/Subset51/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset51/tssdist.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5519\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5519\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 334 sequences from chr1\n",
      "\tExtracting 384 sequences from chr2\n",
      "\tExtracting 302 sequences from chr3\n",
      "\tExtracting 341 sequences from chr4\n",
      "\tExtracting 360 sequences from chr5\n",
      "\tExtracting 281 sequences from chr6\n",
      "\tExtracting 421 sequences from chr7\n",
      "\tExtracting 298 sequences from chr8\n",
      "\tExtracting 325 sequences from chr9\n",
      "\tExtracting 316 sequences from chr10\n",
      "\tExtracting 407 sequences from chr11\n",
      "\tExtracting 251 sequences from chr12\n",
      "\tExtracting 228 sequences from chr13\n",
      "\tExtracting 197 sequences from chr14\n",
      "\tExtracting 225 sequences from chr15\n",
      "\tExtracting 186 sequences from chr16\n",
      "\tExtracting 258 sequences from chr17\n",
      "\tExtracting 160 sequences from chr18\n",
      "\tExtracting 193 sequences from chr19\n",
      "\tExtracting 52 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5519 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5519\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5519\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 371 sequences from chr1\n",
      "\tExtracting 395 sequences from chr2\n",
      "\tExtracting 289 sequences from chr3\n",
      "\tExtracting 359 sequences from chr4\n",
      "\tExtracting 376 sequences from chr5\n",
      "\tExtracting 248 sequences from chr6\n",
      "\tExtracting 404 sequences from chr7\n",
      "\tExtracting 338 sequences from chr8\n",
      "\tExtracting 264 sequences from chr9\n",
      "\tExtracting 344 sequences from chr10\n",
      "\tExtracting 405 sequences from chr11\n",
      "\tExtracting 222 sequences from chr12\n",
      "\tExtracting 216 sequences from chr13\n",
      "\tExtracting 213 sequences from chr14\n",
      "\tExtracting 220 sequences from chr15\n",
      "\tExtracting 229 sequences from chr16\n",
      "\tExtracting 250 sequences from chr17\n",
      "\tExtracting 162 sequences from chr18\n",
      "\tExtracting 181 sequences from chr19\n",
      "\tExtracting 33 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5519 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5517\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5517\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 348 sequences from chr1\n",
      "\tExtracting 407 sequences from chr2\n",
      "\tExtracting 304 sequences from chr3\n",
      "\tExtracting 343 sequences from chr4\n",
      "\tExtracting 359 sequences from chr5\n",
      "\tExtracting 262 sequences from chr6\n",
      "\tExtracting 386 sequences from chr7\n",
      "\tExtracting 303 sequences from chr8\n",
      "\tExtracting 298 sequences from chr9\n",
      "\tExtracting 295 sequences from chr10\n",
      "\tExtracting 408 sequences from chr11\n",
      "\tExtracting 236 sequences from chr12\n",
      "\tExtracting 250 sequences from chr13\n",
      "\tExtracting 236 sequences from chr14\n",
      "\tExtracting 218 sequences from chr15\n",
      "\tExtracting 205 sequences from chr16\n",
      "\tExtracting 268 sequences from chr17\n",
      "\tExtracting 167 sequences from chr18\n",
      "\tExtracting 173 sequences from chr19\n",
      "\tExtracting 51 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5517 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5519\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5519\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 349 sequences from chr1\n",
      "\tExtracting 398 sequences from chr2\n",
      "\tExtracting 288 sequences from chr3\n",
      "\tExtracting 336 sequences from chr4\n",
      "\tExtracting 338 sequences from chr5\n",
      "\tExtracting 285 sequences from chr6\n",
      "\tExtracting 404 sequences from chr7\n",
      "\tExtracting 291 sequences from chr8\n",
      "\tExtracting 316 sequences from chr9\n",
      "\tExtracting 336 sequences from chr10\n",
      "\tExtracting 412 sequences from chr11\n",
      "\tExtracting 250 sequences from chr12\n",
      "\tExtracting 232 sequences from chr13\n",
      "\tExtracting 209 sequences from chr14\n",
      "\tExtracting 226 sequences from chr15\n",
      "\tExtracting 216 sequences from chr16\n",
      "\tExtracting 261 sequences from chr17\n",
      "\tExtracting 161 sequences from chr18\n",
      "\tExtracting 183 sequences from chr19\n",
      "\tExtracting 28 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5519 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5587\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5587\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 360 sequences from chr1\n",
      "\tExtracting 398 sequences from chr2\n",
      "\tExtracting 291 sequences from chr3\n",
      "\tExtracting 334 sequences from chr4\n",
      "\tExtracting 399 sequences from chr5\n",
      "\tExtracting 276 sequences from chr6\n",
      "\tExtracting 365 sequences from chr7\n",
      "\tExtracting 293 sequences from chr8\n",
      "\tExtracting 306 sequences from chr9\n",
      "\tExtracting 332 sequences from chr10\n",
      "\tExtracting 385 sequences from chr11\n",
      "\tExtracting 217 sequences from chr12\n",
      "\tExtracting 243 sequences from chr13\n",
      "\tExtracting 234 sequences from chr14\n",
      "\tExtracting 216 sequences from chr15\n",
      "\tExtracting 243 sequences from chr16\n",
      "\tExtracting 270 sequences from chr17\n",
      "\tExtracting 189 sequences from chr18\n",
      "\tExtracting 182 sequences from chr19\n",
      "\tExtracting 54 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5587 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5590\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5590\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 361 sequences from chr1\n",
      "\tExtracting 426 sequences from chr2\n",
      "\tExtracting 300 sequences from chr3\n",
      "\tExtracting 363 sequences from chr4\n",
      "\tExtracting 370 sequences from chr5\n",
      "\tExtracting 277 sequences from chr6\n",
      "\tExtracting 369 sequences from chr7\n",
      "\tExtracting 308 sequences from chr8\n",
      "\tExtracting 307 sequences from chr9\n",
      "\tExtracting 318 sequences from chr10\n",
      "\tExtracting 419 sequences from chr11\n",
      "\tExtracting 246 sequences from chr12\n",
      "\tExtracting 231 sequences from chr13\n",
      "\tExtracting 204 sequences from chr14\n",
      "\tExtracting 225 sequences from chr15\n",
      "\tExtracting 224 sequences from chr16\n",
      "\tExtracting 260 sequences from chr17\n",
      "\tExtracting 157 sequences from chr18\n",
      "\tExtracting 178 sequences from chr19\n",
      "\tExtracting 47 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5590 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5590\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5590\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 359 sequences from chr1\n",
      "\tExtracting 385 sequences from chr2\n",
      "\tExtracting 290 sequences from chr3\n",
      "\tExtracting 340 sequences from chr4\n",
      "\tExtracting 367 sequences from chr5\n",
      "\tExtracting 273 sequences from chr6\n",
      "\tExtracting 430 sequences from chr7\n",
      "\tExtracting 333 sequences from chr8\n",
      "\tExtracting 291 sequences from chr9\n",
      "\tExtracting 355 sequences from chr10\n",
      "\tExtracting 436 sequences from chr11\n",
      "\tExtracting 225 sequences from chr12\n",
      "\tExtracting 220 sequences from chr13\n",
      "\tExtracting 217 sequences from chr14\n",
      "\tExtracting 235 sequences from chr15\n",
      "\tExtracting 191 sequences from chr16\n",
      "\tExtracting 276 sequences from chr17\n",
      "\tExtracting 152 sequences from chr18\n",
      "\tExtracting 177 sequences from chr19\n",
      "\tExtracting 38 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5590 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5590\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5590\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 349 sequences from chr1\n",
      "\tExtracting 404 sequences from chr2\n",
      "\tExtracting 316 sequences from chr3\n",
      "\tExtracting 338 sequences from chr4\n",
      "\tExtracting 346 sequences from chr5\n",
      "\tExtracting 279 sequences from chr6\n",
      "\tExtracting 427 sequences from chr7\n",
      "\tExtracting 290 sequences from chr8\n",
      "\tExtracting 309 sequences from chr9\n",
      "\tExtracting 327 sequences from chr10\n",
      "\tExtracting 393 sequences from chr11\n",
      "\tExtracting 287 sequences from chr12\n",
      "\tExtracting 234 sequences from chr13\n",
      "\tExtracting 215 sequences from chr14\n",
      "\tExtracting 207 sequences from chr15\n",
      "\tExtracting 220 sequences from chr16\n",
      "\tExtracting 256 sequences from chr17\n",
      "\tExtracting 170 sequences from chr18\n",
      "\tExtracting 194 sequences from chr19\n",
      "\tExtracting 29 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5590 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset51/binding.tsv.gz tmp_dynamic/Subset51/tssdist.tsv.gz tmp_dynamic/Subset51/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset54/wellington.tsv.gz tmp_dynamic/Subset54/homer.tsv.gz tmp_dynamic/Subset54/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset54/expression.tsv.gz tmp_dynamic/Subset54/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset54/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset53/wellington.tsv.gz tmp_dynamic/Subset53/homer.tsv.gz tmp_dynamic/Subset53/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset53/expression.tsv.gz tmp_dynamic/Subset53/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset53/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset51/linking.tsv.gz tmp_dynamic/Subset51/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset54/binding.tsv.gz tmp_dynamic/Subset54/tssdist.tsv.gz tmp_dynamic/Subset54/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset53/binding.tsv.gz tmp_dynamic/Subset53/tssdist.tsv.gz tmp_dynamic/Subset53/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset54/linking.tsv.gz tmp_dynamic/Subset54/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset53/linking.tsv.gz tmp_dynamic/Subset53/binlinking.tsv.gz 20\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset57/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset57/expression.tsv.gz tmp_dynamic/Subset57/motifs.bed tmp_dynamic/Subset57/wellington.tsv.gz tmp_dynamic/Subset57/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5438\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5438\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 327 sequences from chr1\n",
      "\tExtracting 409 sequences from chr2\n",
      "\tExtracting 274 sequences from chr3\n",
      "\tExtracting 359 sequences from chr4\n",
      "\tExtracting 349 sequences from chr5\n",
      "\tExtracting 273 sequences from chr6\n",
      "\tExtracting 392 sequences from chr7\n",
      "\tExtracting 293 sequences from chr8\n",
      "\tExtracting 303 sequences from chr9\n",
      "\tExtracting 312 sequences from chr10\n",
      "\tExtracting 461 sequences from chr11\n",
      "\tExtracting 244 sequences from chr12\n",
      "\tExtracting 205 sequences from chr13\n",
      "\tExtracting 207 sequences from chr14\n",
      "\tExtracting 197 sequences from chr15\n",
      "\tExtracting 203 sequences from chr16\n",
      "\tExtracting 250 sequences from chr17\n",
      "\tExtracting 156 sequences from chr18\n",
      "\tExtracting 181 sequences from chr19\n",
      "\tExtracting 43 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5438 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5438\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5438\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 318 sequences from chr1\n",
      "\tExtracting 386 sequences from chr2\n",
      "\tExtracting 274 sequences from chr3\n",
      "\tExtracting 350 sequences from chr4\n",
      "\tExtracting 360 sequences from chr5\n",
      "\tExtracting 245 sequences from chr6\n",
      "\tExtracting 427 sequences from chr7\n",
      "\tExtracting 313 sequences from chr8\n",
      "\tExtracting 304 sequences from chr9\n",
      "\tExtracting 334 sequences from chr10\n",
      "\tExtracting 402 sequences from chr11\n",
      "\tExtracting 243 sequences from chr12\n",
      "\tExtracting 228 sequences from chr13\n",
      "\tExtracting 186 sequences from chr14\n",
      "\tExtracting 239 sequences from chr15\n",
      "\tExtracting 213 sequences from chr16\n",
      "\tExtracting 258 sequences from chr17\n",
      "\tExtracting 141 sequences from chr18\n",
      "\tExtracting 181 sequences from chr19\n",
      "\tExtracting 36 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5438 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5437\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5437\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 374 sequences from chr1\n",
      "\tExtracting 397 sequences from chr2\n",
      "\tExtracting 312 sequences from chr3\n",
      "\tExtracting 344 sequences from chr4\n",
      "\tExtracting 326 sequences from chr5\n",
      "\tExtracting 265 sequences from chr6\n",
      "\tExtracting 390 sequences from chr7\n",
      "\tExtracting 317 sequences from chr8\n",
      "\tExtracting 296 sequences from chr9\n",
      "\tExtracting 324 sequences from chr10\n",
      "\tExtracting 393 sequences from chr11\n",
      "\tExtracting 207 sequences from chr12\n",
      "\tExtracting 236 sequences from chr13\n",
      "\tExtracting 210 sequences from chr14\n",
      "\tExtracting 222 sequences from chr15\n",
      "\tExtracting 199 sequences from chr16\n",
      "\tExtracting 233 sequences from chr17\n",
      "\tExtracting 165 sequences from chr18\n",
      "\tExtracting 179 sequences from chr19\n",
      "\tExtracting 48 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5437 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5438\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5438\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 360 sequences from chr1\n",
      "\tExtracting 391 sequences from chr2\n",
      "\tExtracting 291 sequences from chr3\n",
      "\tExtracting 334 sequences from chr4\n",
      "\tExtracting 316 sequences from chr5\n",
      "\tExtracting 289 sequences from chr6\n",
      "\tExtracting 423 sequences from chr7\n",
      "\tExtracting 304 sequences from chr8\n",
      "\tExtracting 317 sequences from chr9\n",
      "\tExtracting 296 sequences from chr10\n",
      "\tExtracting 391 sequences from chr11\n",
      "\tExtracting 247 sequences from chr12\n",
      "\tExtracting 223 sequences from chr13\n",
      "\tExtracting 203 sequences from chr14\n",
      "\tExtracting 224 sequences from chr15\n",
      "\tExtracting 199 sequences from chr16\n",
      "\tExtracting 258 sequences from chr17\n",
      "\tExtracting 164 sequences from chr18\n",
      "\tExtracting 182 sequences from chr19\n",
      "\tExtracting 26 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5438 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset57/wellington.tsv.gz tmp_dynamic/Subset57/homer.tsv.gz tmp_dynamic/Subset57/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset57/expression.tsv.gz tmp_dynamic/Subset57/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset57/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset57/binding.tsv.gz tmp_dynamic/Subset57/tssdist.tsv.gz tmp_dynamic/Subset57/linking.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset56/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset56/expression.tsv.gz tmp_dynamic/Subset56/motifs.bed tmp_dynamic/Subset56/wellington.tsv.gz tmp_dynamic/Subset56/homer.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset57/linking.tsv.gz tmp_dynamic/Subset57/binlinking.tsv.gz 20\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5470\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5470\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 343 sequences from chr1\n",
      "\tExtracting 377 sequences from chr2\n",
      "\tExtracting 267 sequences from chr3\n",
      "\tExtracting 360 sequences from chr4\n",
      "\tExtracting 338 sequences from chr5\n",
      "\tExtracting 281 sequences from chr6\n",
      "\tExtracting 401 sequences from chr7\n",
      "\tExtracting 298 sequences from chr8\n",
      "\tExtracting 273 sequences from chr9\n",
      "\tExtracting 353 sequences from chr10\n",
      "\tExtracting 431 sequences from chr11\n",
      "\tExtracting 232 sequences from chr12\n",
      "\tExtracting 236 sequences from chr13\n",
      "\tExtracting 191 sequences from chr14\n",
      "\tExtracting 229 sequences from chr15\n",
      "\tExtracting 184 sequences from chr16\n",
      "\tExtracting 264 sequences from chr17\n",
      "\tExtracting 174 sequences from chr18\n",
      "\tExtracting 198 sequences from chr19\n",
      "\tExtracting 40 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5470 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5467\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5467\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 356 sequences from chr1\n",
      "\tExtracting 428 sequences from chr2\n",
      "\tExtracting 283 sequences from chr3\n",
      "\tExtracting 369 sequences from chr4\n",
      "\tExtracting 352 sequences from chr5\n",
      "\tExtracting 253 sequences from chr6\n",
      "\tExtracting 415 sequences from chr7\n",
      "\tExtracting 313 sequences from chr8\n",
      "\tExtracting 288 sequences from chr9\n",
      "\tExtracting 307 sequences from chr10\n",
      "\tExtracting 389 sequences from chr11\n",
      "\tExtracting 231 sequences from chr12\n",
      "\tExtracting 201 sequences from chr13\n",
      "\tExtracting 204 sequences from chr14\n",
      "\tExtracting 231 sequences from chr15\n",
      "\tExtracting 228 sequences from chr16\n",
      "\tExtracting 241 sequences from chr17\n",
      "\tExtracting 148 sequences from chr18\n",
      "\tExtracting 189 sequences from chr19\n",
      "\tExtracting 41 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5467 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5470\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5470\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 333 sequences from chr1\n",
      "\tExtracting 415 sequences from chr2\n",
      "\tExtracting 291 sequences from chr3\n",
      "\tExtracting 350 sequences from chr4\n",
      "\tExtracting 368 sequences from chr5\n",
      "\tExtracting 270 sequences from chr6\n",
      "\tExtracting 417 sequences from chr7\n",
      "\tExtracting 303 sequences from chr8\n",
      "\tExtracting 317 sequences from chr9\n",
      "\tExtracting 311 sequences from chr10\n",
      "\tExtracting 410 sequences from chr11\n",
      "\tExtracting 243 sequences from chr12\n",
      "\tExtracting 210 sequences from chr13\n",
      "\tExtracting 193 sequences from chr14\n",
      "\tExtracting 237 sequences from chr15\n",
      "\tExtracting 187 sequences from chr16\n",
      "\tExtracting 260 sequences from chr17\n",
      "\tExtracting 153 sequences from chr18\n",
      "\tExtracting 165 sequences from chr19\n",
      "\tExtracting 37 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5470 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5470\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5470\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 351 sequences from chr1\n",
      "\tExtracting 389 sequences from chr2\n",
      "\tExtracting 294 sequences from chr3\n",
      "\tExtracting 331 sequences from chr4\n",
      "\tExtracting 330 sequences from chr5\n",
      "\tExtracting 275 sequences from chr6\n",
      "\tExtracting 412 sequences from chr7\n",
      "\tExtracting 303 sequences from chr8\n",
      "\tExtracting 328 sequences from chr9\n",
      "\tExtracting 318 sequences from chr10\n",
      "\tExtracting 389 sequences from chr11\n",
      "\tExtracting 256 sequences from chr12\n",
      "\tExtracting 234 sequences from chr13\n",
      "\tExtracting 206 sequences from chr14\n",
      "\tExtracting 216 sequences from chr15\n",
      "\tExtracting 216 sequences from chr16\n",
      "\tExtracting 251 sequences from chr17\n",
      "\tExtracting 163 sequences from chr18\n",
      "\tExtracting 183 sequences from chr19\n",
      "\tExtracting 25 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5470 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset56/wellington.tsv.gz tmp_dynamic/Subset56/homer.tsv.gz tmp_dynamic/Subset56/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset56/expression.tsv.gz tmp_dynamic/Subset56/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset56/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset56/binding.tsv.gz tmp_dynamic/Subset56/tssdist.tsv.gz tmp_dynamic/Subset56/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset56/linking.tsv.gz tmp_dynamic/Subset56/binlinking.tsv.gz 20\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset59/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset59/expression.tsv.gz tmp_dynamic/Subset59/motifs.bed tmp_dynamic/Subset59/wellington.tsv.gz tmp_dynamic/Subset59/homer.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset58/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset58/expression.tsv.gz tmp_dynamic/Subset58/motifs.bed tmp_dynamic/Subset58/wellington.tsv.gz tmp_dynamic/Subset58/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5567\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5567\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 351 sequences from chr1\n",
      "\tExtracting 395 sequences from chr2\n",
      "\tExtracting 278 sequences from chr3\n",
      "\tExtracting 393 sequences from chr4\n",
      "\tExtracting 378 sequences from chr5\n",
      "\tExtracting 258 sequences from chr6\n",
      "\tExtracting 429 sequences from chr7\n",
      "\tExtracting 310 sequences from chr8\n",
      "\tExtracting 315 sequences from chr9\n",
      "\tExtracting 324 sequences from chr10\n",
      "\tExtracting 438 sequences from chr11\n",
      "\tExtracting 223 sequences from chr12\n",
      "\tExtracting 220 sequences from chr13\n",
      "\tExtracting 179 sequences from chr14\n",
      "\tExtracting 237 sequences from chr15\n",
      "\tExtracting 198 sequences from chr16\n",
      "\tExtracting 246 sequences from chr17\n",
      "\tExtracting 166 sequences from chr18\n",
      "\tExtracting 189 sequences from chr19\n",
      "\tExtracting 40 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5567 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5567\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5567\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 373 sequences from chr1\n",
      "\tExtracting 418 sequences from chr2\n",
      "\tExtracting 292 sequences from chr3\n",
      "\tExtracting 348 sequences from chr4\n",
      "\tExtracting 353 sequences from chr5\n",
      "\tExtracting 254 sequences from chr6\n",
      "\tExtracting 408 sequences from chr7\n",
      "\tExtracting 309 sequences from chr8\n",
      "\tExtracting 309 sequences from chr9\n",
      "\tExtracting 301 sequences from chr10\n",
      "\tExtracting 449 sequences from chr11\n",
      "\tExtracting 234 sequences from chr12\n",
      "\tExtracting 243 sequences from chr13\n",
      "\tExtracting 214 sequences from chr14\n",
      "\tExtracting 228 sequences from chr15\n",
      "\tExtracting 192 sequences from chr16\n",
      "\tExtracting 268 sequences from chr17\n",
      "\tExtracting 148 sequences from chr18\n",
      "\tExtracting 174 sequences from chr19\n",
      "\tExtracting 52 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5567 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5563\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5563\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 351 sequences from chr1\n",
      "\tExtracting 425 sequences from chr2\n",
      "\tExtracting 275 sequences from chr3\n",
      "\tExtracting 376 sequences from chr4\n",
      "\tExtracting 389 sequences from chr5\n",
      "\tExtracting 264 sequences from chr6\n",
      "\tExtracting 380 sequences from chr7\n",
      "\tExtracting 338 sequences from chr8\n",
      "\tExtracting 290 sequences from chr9\n",
      "\tExtracting 336 sequences from chr10\n",
      "\tExtracting 395 sequences from chr11\n",
      "\tExtracting 232 sequences from chr12\n",
      "\tExtracting 207 sequences from chr13\n",
      "\tExtracting 196 sequences from chr14\n",
      "\tExtracting 225 sequences from chr15\n",
      "\tExtracting 210 sequences from chr16\n",
      "\tExtracting 251 sequences from chr17\n",
      "\tExtracting 176 sequences from chr18\n",
      "\tExtracting 192 sequences from chr19\n",
      "\tExtracting 55 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5563 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5567\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5567\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 351 sequences from chr1\n",
      "\tExtracting 404 sequences from chr2\n",
      "\tExtracting 292 sequences from chr3\n",
      "\tExtracting 323 sequences from chr4\n",
      "\tExtracting 341 sequences from chr5\n",
      "\tExtracting 280 sequences from chr6\n",
      "\tExtracting 427 sequences from chr7\n",
      "\tExtracting 319 sequences from chr8\n",
      "\tExtracting 301 sequences from chr9\n",
      "\tExtracting 306 sequences from chr10\n",
      "\tExtracting 411 sequences from chr11\n",
      "\tExtracting 274 sequences from chr12\n",
      "\tExtracting 225 sequences from chr13\n",
      "\tExtracting 194 sequences from chr14\n",
      "\tExtracting 252 sequences from chr15\n",
      "\tExtracting 211 sequences from chr16\n",
      "\tExtracting 271 sequences from chr17\n",
      "\tExtracting 157 sequences from chr18\n",
      "\tExtracting 206 sequences from chr19\n",
      "\tExtracting 22 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5567 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset59/wellington.tsv.gz tmp_dynamic/Subset59/homer.tsv.gz tmp_dynamic/Subset59/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset59/expression.tsv.gz tmp_dynamic/Subset59/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset59/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset59/binding.tsv.gz tmp_dynamic/Subset59/tssdist.tsv.gz tmp_dynamic/Subset59/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset59/linking.tsv.gz tmp_dynamic/Subset59/binlinking.tsv.gz 20\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5565\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5565\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 363 sequences from chr1\n",
      "\tExtracting 404 sequences from chr2\n",
      "\tExtracting 281 sequences from chr3\n",
      "\tExtracting 376 sequences from chr4\n",
      "\tExtracting 341 sequences from chr5\n",
      "\tExtracting 252 sequences from chr6\n",
      "\tExtracting 383 sequences from chr7\n",
      "\tExtracting 290 sequences from chr8\n",
      "\tExtracting 305 sequences from chr9\n",
      "\tExtracting 307 sequences from chr10\n",
      "\tExtracting 435 sequences from chr11\n",
      "\tExtracting 207 sequences from chr12\n",
      "\tExtracting 234 sequences from chr13\n",
      "\tExtracting 211 sequences from chr14\n",
      "\tExtracting 261 sequences from chr15\n",
      "\tExtracting 215 sequences from chr16\n",
      "\tExtracting 277 sequences from chr17\n",
      "\tExtracting 176 sequences from chr18\n",
      "\tExtracting 195 sequences from chr19\n",
      "\tExtracting 52 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5565 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5567\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5567\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 351 sequences from chr1\n",
      "\tExtracting 422 sequences from chr2\n",
      "\tExtracting 287 sequences from chr3\n",
      "\tExtracting 353 sequences from chr4\n",
      "\tExtracting 366 sequences from chr5\n",
      "\tExtracting 289 sequences from chr6\n",
      "\tExtracting 386 sequences from chr7\n",
      "\tExtracting 307 sequences from chr8\n",
      "\tExtracting 297 sequences from chr9\n",
      "\tExtracting 327 sequences from chr10\n",
      "\tExtracting 449 sequences from chr11\n",
      "\tExtracting 235 sequences from chr12\n",
      "\tExtracting 230 sequences from chr13\n",
      "\tExtracting 199 sequences from chr14\n",
      "\tExtracting 212 sequences from chr15\n",
      "\tExtracting 223 sequences from chr16\n",
      "\tExtracting 258 sequences from chr17\n",
      "\tExtracting 158 sequences from chr18\n",
      "\tExtracting 177 sequences from chr19\n",
      "\tExtracting 41 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5567 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5567\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5567\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 339 sequences from chr1\n",
      "\tExtracting 390 sequences from chr2\n",
      "\tExtracting 279 sequences from chr3\n",
      "\tExtracting 385 sequences from chr4\n",
      "\tExtracting 378 sequences from chr5\n",
      "\tExtracting 267 sequences from chr6\n",
      "\tExtracting 466 sequences from chr7\n",
      "\tExtracting 301 sequences from chr8\n",
      "\tExtracting 304 sequences from chr9\n",
      "\tExtracting 313 sequences from chr10\n",
      "\tExtracting 429 sequences from chr11\n",
      "\tExtracting 252 sequences from chr12\n",
      "\tExtracting 219 sequences from chr13\n",
      "\tExtracting 215 sequences from chr14\n",
      "\tExtracting 222 sequences from chr15\n",
      "\tExtracting 199 sequences from chr16\n",
      "\tExtracting 254 sequences from chr17\n",
      "\tExtracting 147 sequences from chr18\n",
      "\tExtracting 170 sequences from chr19\n",
      "\tExtracting 38 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5567 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5567\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5567\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 384 sequences from chr1\n",
      "\tExtracting 401 sequences from chr2\n",
      "\tExtracting 284 sequences from chr3\n",
      "\tExtracting 330 sequences from chr4\n",
      "\tExtracting 333 sequences from chr5\n",
      "\tExtracting 287 sequences from chr6\n",
      "\tExtracting 416 sequences from chr7\n",
      "\tExtracting 328 sequences from chr8\n",
      "\tExtracting 315 sequences from chr9\n",
      "\tExtracting 317 sequences from chr10\n",
      "\tExtracting 409 sequences from chr11\n",
      "\tExtracting 253 sequences from chr12\n",
      "\tExtracting 227 sequences from chr13\n",
      "\tExtracting 184 sequences from chr14\n",
      "\tExtracting 243 sequences from chr15\n",
      "\tExtracting 200 sequences from chr16\n",
      "\tExtracting 262 sequences from chr17\n",
      "\tExtracting 166 sequences from chr18\n",
      "\tExtracting 202 sequences from chr19\n",
      "\tExtracting 26 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5567 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset58/wellington.tsv.gz tmp_dynamic/Subset58/homer.tsv.gz tmp_dynamic/Subset58/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset58/expression.tsv.gz tmp_dynamic/Subset58/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset58/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset58/binding.tsv.gz tmp_dynamic/Subset58/tssdist.tsv.gz tmp_dynamic/Subset58/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset58/linking.tsv.gz tmp_dynamic/Subset58/binlinking.tsv.gz 20\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset60/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset60/expression.tsv.gz tmp_dynamic/Subset60/motifs.bed tmp_dynamic/Subset60/wellington.tsv.gz tmp_dynamic/Subset60/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5437\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5437\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 327 sequences from chr1\n",
      "\tExtracting 390 sequences from chr2\n",
      "\tExtracting 283 sequences from chr3\n",
      "\tExtracting 366 sequences from chr4\n",
      "\tExtracting 349 sequences from chr5\n",
      "\tExtracting 268 sequences from chr6\n",
      "\tExtracting 390 sequences from chr7\n",
      "\tExtracting 285 sequences from chr8\n",
      "\tExtracting 309 sequences from chr9\n",
      "\tExtracting 327 sequences from chr10\n",
      "\tExtracting 441 sequences from chr11\n",
      "\tExtracting 219 sequences from chr12\n",
      "\tExtracting 222 sequences from chr13\n",
      "\tExtracting 190 sequences from chr14\n",
      "\tExtracting 233 sequences from chr15\n",
      "\tExtracting 179 sequences from chr16\n",
      "\tExtracting 264 sequences from chr17\n",
      "\tExtracting 161 sequences from chr18\n",
      "\tExtracting 192 sequences from chr19\n",
      "\tExtracting 42 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5437 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5435\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5435\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 346 sequences from chr1\n",
      "\tExtracting 398 sequences from chr2\n",
      "\tExtracting 275 sequences from chr3\n",
      "\tExtracting 358 sequences from chr4\n",
      "\tExtracting 355 sequences from chr5\n",
      "\tExtracting 255 sequences from chr6\n",
      "\tExtracting 381 sequences from chr7\n",
      "\tExtracting 334 sequences from chr8\n",
      "\tExtracting 315 sequences from chr9\n",
      "\tExtracting 312 sequences from chr10\n",
      "\tExtracting 387 sequences from chr11\n",
      "\tExtracting 237 sequences from chr12\n",
      "\tExtracting 233 sequences from chr13\n",
      "\tExtracting 209 sequences from chr14\n",
      "\tExtracting 232 sequences from chr15\n",
      "\tExtracting 183 sequences from chr16\n",
      "\tExtracting 225 sequences from chr17\n",
      "\tExtracting 171 sequences from chr18\n",
      "\tExtracting 174 sequences from chr19\n",
      "\tExtracting 55 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5435 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5437\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5437\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 319 sequences from chr1\n",
      "\tExtracting 415 sequences from chr2\n",
      "\tExtracting 276 sequences from chr3\n",
      "\tExtracting 357 sequences from chr4\n",
      "\tExtracting 384 sequences from chr5\n",
      "\tExtracting 264 sequences from chr6\n",
      "\tExtracting 437 sequences from chr7\n",
      "\tExtracting 306 sequences from chr8\n",
      "\tExtracting 292 sequences from chr9\n",
      "\tExtracting 286 sequences from chr10\n",
      "\tExtracting 435 sequences from chr11\n",
      "\tExtracting 223 sequences from chr12\n",
      "\tExtracting 223 sequences from chr13\n",
      "\tExtracting 192 sequences from chr14\n",
      "\tExtracting 230 sequences from chr15\n",
      "\tExtracting 183 sequences from chr16\n",
      "\tExtracting 236 sequences from chr17\n",
      "\tExtracting 149 sequences from chr18\n",
      "\tExtracting 190 sequences from chr19\n",
      "\tExtracting 40 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5437 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5437\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5437\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 359 sequences from chr1\n",
      "\tExtracting 374 sequences from chr2\n",
      "\tExtracting 286 sequences from chr3\n",
      "\tExtracting 321 sequences from chr4\n",
      "\tExtracting 346 sequences from chr5\n",
      "\tExtracting 258 sequences from chr6\n",
      "\tExtracting 410 sequences from chr7\n",
      "\tExtracting 315 sequences from chr8\n",
      "\tExtracting 301 sequences from chr9\n",
      "\tExtracting 310 sequences from chr10\n",
      "\tExtracting 401 sequences from chr11\n",
      "\tExtracting 252 sequences from chr12\n",
      "\tExtracting 208 sequences from chr13\n",
      "\tExtracting 188 sequences from chr14\n",
      "\tExtracting 242 sequences from chr15\n",
      "\tExtracting 209 sequences from chr16\n",
      "\tExtracting 270 sequences from chr17\n",
      "\tExtracting 166 sequences from chr18\n",
      "\tExtracting 203 sequences from chr19\n",
      "\tExtracting 18 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5437 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset60/wellington.tsv.gz tmp_dynamic/Subset60/homer.tsv.gz tmp_dynamic/Subset60/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset60/expression.tsv.gz tmp_dynamic/Subset60/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset60/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset60/binding.tsv.gz tmp_dynamic/Subset60/tssdist.tsv.gz tmp_dynamic/Subset60/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset60/linking.tsv.gz tmp_dynamic/Subset60/binlinking.tsv.gz 20\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset62/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset62/expression.tsv.gz tmp_dynamic/Subset62/motifs.bed tmp_dynamic/Subset62/wellington.tsv.gz tmp_dynamic/Subset62/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4053\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4053\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 288 sequences from chr1\n",
      "\tExtracting 286 sequences from chr2\n",
      "\tExtracting 238 sequences from chr3\n",
      "\tExtracting 275 sequences from chr4\n",
      "\tExtracting 228 sequences from chr5\n",
      "\tExtracting 192 sequences from chr6\n",
      "\tExtracting 333 sequences from chr7\n",
      "\tExtracting 219 sequences from chr8\n",
      "\tExtracting 225 sequences from chr9\n",
      "\tExtracting 241 sequences from chr10\n",
      "\tExtracting 275 sequences from chr11\n",
      "\tExtracting 164 sequences from chr12\n",
      "\tExtracting 183 sequences from chr13\n",
      "\tExtracting 128 sequences from chr14\n",
      "\tExtracting 148 sequences from chr15\n",
      "\tExtracting 172 sequences from chr16\n",
      "\tExtracting 169 sequences from chr17\n",
      "\tExtracting 125 sequences from chr18\n",
      "\tExtracting 142 sequences from chr19\n",
      "\tExtracting 22 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4053 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4053\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4053\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 269 sequences from chr1\n",
      "\tExtracting 325 sequences from chr2\n",
      "\tExtracting 229 sequences from chr3\n",
      "\tExtracting 252 sequences from chr4\n",
      "\tExtracting 273 sequences from chr5\n",
      "\tExtracting 204 sequences from chr6\n",
      "\tExtracting 304 sequences from chr7\n",
      "\tExtracting 245 sequences from chr8\n",
      "\tExtracting 195 sequences from chr9\n",
      "\tExtracting 241 sequences from chr10\n",
      "\tExtracting 266 sequences from chr11\n",
      "\tExtracting 159 sequences from chr12\n",
      "\tExtracting 190 sequences from chr13\n",
      "\tExtracting 145 sequences from chr14\n",
      "\tExtracting 174 sequences from chr15\n",
      "\tExtracting 148 sequences from chr16\n",
      "\tExtracting 174 sequences from chr17\n",
      "\tExtracting 123 sequences from chr18\n",
      "\tExtracting 110 sequences from chr19\n",
      "\tExtracting 27 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4053 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4049\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4049\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 258 sequences from chr1\n",
      "\tExtracting 291 sequences from chr2\n",
      "\tExtracting 236 sequences from chr3\n",
      "\tExtracting 307 sequences from chr4\n",
      "\tExtracting 256 sequences from chr5\n",
      "\tExtracting 203 sequences from chr6\n",
      "\tExtracting 324 sequences from chr7\n",
      "\tExtracting 208 sequences from chr8\n",
      "\tExtracting 223 sequences from chr9\n",
      "\tExtracting 215 sequences from chr10\n",
      "\tExtracting 288 sequences from chr11\n",
      "\tExtracting 147 sequences from chr12\n",
      "\tExtracting 165 sequences from chr13\n",
      "\tExtracting 153 sequences from chr14\n",
      "\tExtracting 164 sequences from chr15\n",
      "\tExtracting 142 sequences from chr16\n",
      "\tExtracting 179 sequences from chr17\n",
      "\tExtracting 124 sequences from chr18\n",
      "\tExtracting 113 sequences from chr19\n",
      "\tExtracting 53 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4049 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 4053\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 4053\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 313 sequences from chr1\n",
      "\tExtracting 300 sequences from chr2\n",
      "\tExtracting 209 sequences from chr3\n",
      "\tExtracting 241 sequences from chr4\n",
      "\tExtracting 278 sequences from chr5\n",
      "\tExtracting 216 sequences from chr6\n",
      "\tExtracting 306 sequences from chr7\n",
      "\tExtracting 224 sequences from chr8\n",
      "\tExtracting 225 sequences from chr9\n",
      "\tExtracting 241 sequences from chr10\n",
      "\tExtracting 242 sequences from chr11\n",
      "\tExtracting 185 sequences from chr12\n",
      "\tExtracting 178 sequences from chr13\n",
      "\tExtracting 152 sequences from chr14\n",
      "\tExtracting 139 sequences from chr15\n",
      "\tExtracting 158 sequences from chr16\n",
      "\tExtracting 201 sequences from chr17\n",
      "\tExtracting 121 sequences from chr18\n",
      "\tExtracting 106 sequences from chr19\n",
      "\tExtracting 18 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t4053 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset62/wellington.tsv.gz tmp_dynamic/Subset62/homer.tsv.gz tmp_dynamic/Subset62/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset62/expression.tsv.gz tmp_dynamic/Subset62/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset62/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset62/binding.tsv.gz tmp_dynamic/Subset62/tssdist.tsv.gz tmp_dynamic/Subset62/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset62/linking.tsv.gz tmp_dynamic/Subset62/binlinking.tsv.gz 20\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_dynamic/Subset3/footprints.bed data/motifs.motif data/genome tmp_dynamic/Subset3/expression.tsv.gz tmp_dynamic/Subset3/motifs.bed tmp_dynamic/Subset3/wellington.tsv.gz tmp_dynamic/Subset3/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5310\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5310\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 353 sequences from chr1\n",
      "\tExtracting 380 sequences from chr2\n",
      "\tExtracting 286 sequences from chr3\n",
      "\tExtracting 344 sequences from chr4\n",
      "\tExtracting 356 sequences from chr5\n",
      "\tExtracting 254 sequences from chr6\n",
      "\tExtracting 355 sequences from chr7\n",
      "\tExtracting 267 sequences from chr8\n",
      "\tExtracting 309 sequences from chr9\n",
      "\tExtracting 323 sequences from chr10\n",
      "\tExtracting 422 sequences from chr11\n",
      "\tExtracting 221 sequences from chr12\n",
      "\tExtracting 205 sequences from chr13\n",
      "\tExtracting 193 sequences from chr14\n",
      "\tExtracting 227 sequences from chr15\n",
      "\tExtracting 179 sequences from chr16\n",
      "\tExtracting 240 sequences from chr17\n",
      "\tExtracting 155 sequences from chr18\n",
      "\tExtracting 180 sequences from chr19\n",
      "\tExtracting 61 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5310 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5313\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5313\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 311 sequences from chr1\n",
      "\tExtracting 404 sequences from chr2\n",
      "\tExtracting 269 sequences from chr3\n",
      "\tExtracting 351 sequences from chr4\n",
      "\tExtracting 329 sequences from chr5\n",
      "\tExtracting 269 sequences from chr6\n",
      "\tExtracting 396 sequences from chr7\n",
      "\tExtracting 313 sequences from chr8\n",
      "\tExtracting 292 sequences from chr9\n",
      "\tExtracting 312 sequences from chr10\n",
      "\tExtracting 400 sequences from chr11\n",
      "\tExtracting 227 sequences from chr12\n",
      "\tExtracting 238 sequences from chr13\n",
      "\tExtracting 188 sequences from chr14\n",
      "\tExtracting 217 sequences from chr15\n",
      "\tExtracting 171 sequences from chr16\n",
      "\tExtracting 253 sequences from chr17\n",
      "\tExtracting 154 sequences from chr18\n",
      "\tExtracting 184 sequences from chr19\n",
      "\tExtracting 35 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5313 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5313\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5313\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 311 sequences from chr1\n",
      "\tExtracting 377 sequences from chr2\n",
      "\tExtracting 266 sequences from chr3\n",
      "\tExtracting 350 sequences from chr4\n",
      "\tExtracting 366 sequences from chr5\n",
      "\tExtracting 234 sequences from chr6\n",
      "\tExtracting 418 sequences from chr7\n",
      "\tExtracting 315 sequences from chr8\n",
      "\tExtracting 288 sequences from chr9\n",
      "\tExtracting 312 sequences from chr10\n",
      "\tExtracting 413 sequences from chr11\n",
      "\tExtracting 233 sequences from chr12\n",
      "\tExtracting 207 sequences from chr13\n",
      "\tExtracting 184 sequences from chr14\n",
      "\tExtracting 233 sequences from chr15\n",
      "\tExtracting 189 sequences from chr16\n",
      "\tExtracting 244 sequences from chr17\n",
      "\tExtracting 165 sequences from chr18\n",
      "\tExtracting 172 sequences from chr19\n",
      "\tExtracting 36 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5313 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = ../data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in ../data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5313\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5313\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: ../data/genome\n",
      "\n",
      "\tExtracting sequences from directory: ../data/genome\n",
      "\tExtracting 353 sequences from chr1\n",
      "\tExtracting 391 sequences from chr2\n",
      "\tExtracting 277 sequences from chr3\n",
      "\tExtracting 325 sequences from chr4\n",
      "\tExtracting 337 sequences from chr5\n",
      "\tExtracting 259 sequences from chr6\n",
      "\tExtracting 405 sequences from chr7\n",
      "\tExtracting 312 sequences from chr8\n",
      "\tExtracting 277 sequences from chr9\n",
      "\tExtracting 287 sequences from chr10\n",
      "\tExtracting 402 sequences from chr11\n",
      "\tExtracting 233 sequences from chr12\n",
      "\tExtracting 216 sequences from chr13\n",
      "\tExtracting 180 sequences from chr14\n",
      "\tExtracting 231 sequences from chr15\n",
      "\tExtracting 208 sequences from chr16\n",
      "\tExtracting 254 sequences from chr17\n",
      "\tExtracting 148 sequences from chr18\n",
      "\tExtracting 197 sequences from chr19\n",
      "\tExtracting 21 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5313 total sequences read\n",
      "\t529 motifs loaded\n",
      "\tFinding instances of 529 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_dynamic/Subset3/wellington.tsv.gz tmp_dynamic/Subset3/homer.tsv.gz tmp_dynamic/Subset3/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_dynamic/Subset3/expression.tsv.gz tmp_dynamic/Subset3/wellington.tsv.gz data/gene.bed tmp_dynamic/Subset3/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  --fi_whitelist data/whitelist.bed tmp_dynamic/Subset3/binding.tsv.gz tmp_dynamic/Subset3/tssdist.tsv.gz tmp_dynamic/Subset3/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_dynamic/Subset3/linking.tsv.gz tmp_dynamic/Subset3/binlinking.tsv.gz 20\n",
      "make: Target `cpu' not remade because of errors.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset1/expression.tsv.gz tmp_dynamic/Subset1/binlinking.tsv.gz tmp_dynamic/Subset1/net_weight.tsv.gz tmp_dynamic/Subset1/net_meanvar.tsv.gz tmp_dynamic/Subset1/net_covfactor.tsv.gz tmp_dynamic/Subset1/net_loss.tsv.gz tmp_dynamic/Subset1/net_stats.tsv.gz\n",
      "WARNING:14602:2022-10-28 22:32:38,295:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset2/expression.tsv.gz tmp_dynamic/Subset2/binlinking.tsv.gz tmp_dynamic/Subset2/net_weight.tsv.gz tmp_dynamic/Subset2/net_meanvar.tsv.gz tmp_dynamic/Subset2/net_covfactor.tsv.gz tmp_dynamic/Subset2/net_loss.tsv.gz tmp_dynamic/Subset2/net_stats.tsv.gz\n",
      "WARNING:14615:2022-10-28 22:33:19,636:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset3/expression.tsv.gz tmp_dynamic/Subset3/binlinking.tsv.gz tmp_dynamic/Subset3/net_weight.tsv.gz tmp_dynamic/Subset3/net_meanvar.tsv.gz tmp_dynamic/Subset3/net_covfactor.tsv.gz tmp_dynamic/Subset3/net_loss.tsv.gz tmp_dynamic/Subset3/net_stats.tsv.gz\n",
      "WARNING:14628:2022-10-28 22:34:00,021:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset4/expression.tsv.gz tmp_dynamic/Subset4/binlinking.tsv.gz tmp_dynamic/Subset4/net_weight.tsv.gz tmp_dynamic/Subset4/net_meanvar.tsv.gz tmp_dynamic/Subset4/net_covfactor.tsv.gz tmp_dynamic/Subset4/net_loss.tsv.gz tmp_dynamic/Subset4/net_stats.tsv.gz\n",
      "WARNING:14640:2022-10-28 22:34:40,830:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset5/expression.tsv.gz tmp_dynamic/Subset5/binlinking.tsv.gz tmp_dynamic/Subset5/net_weight.tsv.gz tmp_dynamic/Subset5/net_meanvar.tsv.gz tmp_dynamic/Subset5/net_covfactor.tsv.gz tmp_dynamic/Subset5/net_loss.tsv.gz tmp_dynamic/Subset5/net_stats.tsv.gz\n",
      "WARNING:14655:2022-10-28 22:35:21,809:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset6/expression.tsv.gz tmp_dynamic/Subset6/binlinking.tsv.gz tmp_dynamic/Subset6/net_weight.tsv.gz tmp_dynamic/Subset6/net_meanvar.tsv.gz tmp_dynamic/Subset6/net_covfactor.tsv.gz tmp_dynamic/Subset6/net_loss.tsv.gz tmp_dynamic/Subset6/net_stats.tsv.gz\n",
      "WARNING:14667:2022-10-28 22:36:02,823:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset7/expression.tsv.gz tmp_dynamic/Subset7/binlinking.tsv.gz tmp_dynamic/Subset7/net_weight.tsv.gz tmp_dynamic/Subset7/net_meanvar.tsv.gz tmp_dynamic/Subset7/net_covfactor.tsv.gz tmp_dynamic/Subset7/net_loss.tsv.gz tmp_dynamic/Subset7/net_stats.tsv.gz\n",
      "WARNING:14682:2022-10-28 22:36:43,779:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset8/expression.tsv.gz tmp_dynamic/Subset8/binlinking.tsv.gz tmp_dynamic/Subset8/net_weight.tsv.gz tmp_dynamic/Subset8/net_meanvar.tsv.gz tmp_dynamic/Subset8/net_covfactor.tsv.gz tmp_dynamic/Subset8/net_loss.tsv.gz tmp_dynamic/Subset8/net_stats.tsv.gz\n",
      "WARNING:14695:2022-10-28 22:37:24,756:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset9/expression.tsv.gz tmp_dynamic/Subset9/binlinking.tsv.gz tmp_dynamic/Subset9/net_weight.tsv.gz tmp_dynamic/Subset9/net_meanvar.tsv.gz tmp_dynamic/Subset9/net_covfactor.tsv.gz tmp_dynamic/Subset9/net_loss.tsv.gz tmp_dynamic/Subset9/net_stats.tsv.gz\n",
      "WARNING:14709:2022-10-28 22:38:05,419:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset10/expression.tsv.gz tmp_dynamic/Subset10/binlinking.tsv.gz tmp_dynamic/Subset10/net_weight.tsv.gz tmp_dynamic/Subset10/net_meanvar.tsv.gz tmp_dynamic/Subset10/net_covfactor.tsv.gz tmp_dynamic/Subset10/net_loss.tsv.gz tmp_dynamic/Subset10/net_stats.tsv.gz\n",
      "WARNING:14721:2022-10-28 22:38:46,214:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset11/expression.tsv.gz tmp_dynamic/Subset11/binlinking.tsv.gz tmp_dynamic/Subset11/net_weight.tsv.gz tmp_dynamic/Subset11/net_meanvar.tsv.gz tmp_dynamic/Subset11/net_covfactor.tsv.gz tmp_dynamic/Subset11/net_loss.tsv.gz tmp_dynamic/Subset11/net_stats.tsv.gz\n",
      "WARNING:14733:2022-10-28 22:39:26,953:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset12/expression.tsv.gz tmp_dynamic/Subset12/binlinking.tsv.gz tmp_dynamic/Subset12/net_weight.tsv.gz tmp_dynamic/Subset12/net_meanvar.tsv.gz tmp_dynamic/Subset12/net_covfactor.tsv.gz tmp_dynamic/Subset12/net_loss.tsv.gz tmp_dynamic/Subset12/net_stats.tsv.gz\n",
      "WARNING:14745:2022-10-28 22:40:07,808:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset13/expression.tsv.gz tmp_dynamic/Subset13/binlinking.tsv.gz tmp_dynamic/Subset13/net_weight.tsv.gz tmp_dynamic/Subset13/net_meanvar.tsv.gz tmp_dynamic/Subset13/net_covfactor.tsv.gz tmp_dynamic/Subset13/net_loss.tsv.gz tmp_dynamic/Subset13/net_stats.tsv.gz\n",
      "WARNING:14759:2022-10-28 22:40:48,797:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset14/expression.tsv.gz tmp_dynamic/Subset14/binlinking.tsv.gz tmp_dynamic/Subset14/net_weight.tsv.gz tmp_dynamic/Subset14/net_meanvar.tsv.gz tmp_dynamic/Subset14/net_covfactor.tsv.gz tmp_dynamic/Subset14/net_loss.tsv.gz tmp_dynamic/Subset14/net_stats.tsv.gz\n",
      "WARNING:14772:2022-10-28 22:41:29,369:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset15/expression.tsv.gz tmp_dynamic/Subset15/binlinking.tsv.gz tmp_dynamic/Subset15/net_weight.tsv.gz tmp_dynamic/Subset15/net_meanvar.tsv.gz tmp_dynamic/Subset15/net_covfactor.tsv.gz tmp_dynamic/Subset15/net_loss.tsv.gz tmp_dynamic/Subset15/net_stats.tsv.gz\n",
      "WARNING:14783:2022-10-28 22:42:10,149:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset16/expression.tsv.gz tmp_dynamic/Subset16/binlinking.tsv.gz tmp_dynamic/Subset16/net_weight.tsv.gz tmp_dynamic/Subset16/net_meanvar.tsv.gz tmp_dynamic/Subset16/net_covfactor.tsv.gz tmp_dynamic/Subset16/net_loss.tsv.gz tmp_dynamic/Subset16/net_stats.tsv.gz\n",
      "WARNING:14798:2022-10-28 22:42:51,124:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset17/expression.tsv.gz tmp_dynamic/Subset17/binlinking.tsv.gz tmp_dynamic/Subset17/net_weight.tsv.gz tmp_dynamic/Subset17/net_meanvar.tsv.gz tmp_dynamic/Subset17/net_covfactor.tsv.gz tmp_dynamic/Subset17/net_loss.tsv.gz tmp_dynamic/Subset17/net_stats.tsv.gz\n",
      "WARNING:14811:2022-10-28 22:43:32,023:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset18/expression.tsv.gz tmp_dynamic/Subset18/binlinking.tsv.gz tmp_dynamic/Subset18/net_weight.tsv.gz tmp_dynamic/Subset18/net_meanvar.tsv.gz tmp_dynamic/Subset18/net_covfactor.tsv.gz tmp_dynamic/Subset18/net_loss.tsv.gz tmp_dynamic/Subset18/net_stats.tsv.gz\n",
      "WARNING:14823:2022-10-28 22:44:13,362:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset19/expression.tsv.gz tmp_dynamic/Subset19/binlinking.tsv.gz tmp_dynamic/Subset19/net_weight.tsv.gz tmp_dynamic/Subset19/net_meanvar.tsv.gz tmp_dynamic/Subset19/net_covfactor.tsv.gz tmp_dynamic/Subset19/net_loss.tsv.gz tmp_dynamic/Subset19/net_stats.tsv.gz\n",
      "WARNING:14835:2022-10-28 22:44:54,131:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset20/expression.tsv.gz tmp_dynamic/Subset20/binlinking.tsv.gz tmp_dynamic/Subset20/net_weight.tsv.gz tmp_dynamic/Subset20/net_meanvar.tsv.gz tmp_dynamic/Subset20/net_covfactor.tsv.gz tmp_dynamic/Subset20/net_loss.tsv.gz tmp_dynamic/Subset20/net_stats.tsv.gz\n",
      "WARNING:14847:2022-10-28 22:45:35,249:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset21/expression.tsv.gz tmp_dynamic/Subset21/binlinking.tsv.gz tmp_dynamic/Subset21/net_weight.tsv.gz tmp_dynamic/Subset21/net_meanvar.tsv.gz tmp_dynamic/Subset21/net_covfactor.tsv.gz tmp_dynamic/Subset21/net_loss.tsv.gz tmp_dynamic/Subset21/net_stats.tsv.gz\n",
      "WARNING:14859:2022-10-28 22:46:16,135:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset22/expression.tsv.gz tmp_dynamic/Subset22/binlinking.tsv.gz tmp_dynamic/Subset22/net_weight.tsv.gz tmp_dynamic/Subset22/net_meanvar.tsv.gz tmp_dynamic/Subset22/net_covfactor.tsv.gz tmp_dynamic/Subset22/net_loss.tsv.gz tmp_dynamic/Subset22/net_stats.tsv.gz\n",
      "WARNING:14871:2022-10-28 22:46:57,316:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset23/expression.tsv.gz tmp_dynamic/Subset23/binlinking.tsv.gz tmp_dynamic/Subset23/net_weight.tsv.gz tmp_dynamic/Subset23/net_meanvar.tsv.gz tmp_dynamic/Subset23/net_covfactor.tsv.gz tmp_dynamic/Subset23/net_loss.tsv.gz tmp_dynamic/Subset23/net_stats.tsv.gz\n",
      "WARNING:14883:2022-10-28 22:47:38,611:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset24/expression.tsv.gz tmp_dynamic/Subset24/binlinking.tsv.gz tmp_dynamic/Subset24/net_weight.tsv.gz tmp_dynamic/Subset24/net_meanvar.tsv.gz tmp_dynamic/Subset24/net_covfactor.tsv.gz tmp_dynamic/Subset24/net_loss.tsv.gz tmp_dynamic/Subset24/net_stats.tsv.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WARNING:14961:2022-10-28 22:48:19,715:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset25/expression.tsv.gz tmp_dynamic/Subset25/binlinking.tsv.gz tmp_dynamic/Subset25/net_weight.tsv.gz tmp_dynamic/Subset25/net_meanvar.tsv.gz tmp_dynamic/Subset25/net_covfactor.tsv.gz tmp_dynamic/Subset25/net_loss.tsv.gz tmp_dynamic/Subset25/net_stats.tsv.gz\n",
      "WARNING:14975:2022-10-28 22:49:00,941:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset26/expression.tsv.gz tmp_dynamic/Subset26/binlinking.tsv.gz tmp_dynamic/Subset26/net_weight.tsv.gz tmp_dynamic/Subset26/net_meanvar.tsv.gz tmp_dynamic/Subset26/net_covfactor.tsv.gz tmp_dynamic/Subset26/net_loss.tsv.gz tmp_dynamic/Subset26/net_stats.tsv.gz\n",
      "WARNING:14988:2022-10-28 22:49:41,839:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset27/expression.tsv.gz tmp_dynamic/Subset27/binlinking.tsv.gz tmp_dynamic/Subset27/net_weight.tsv.gz tmp_dynamic/Subset27/net_meanvar.tsv.gz tmp_dynamic/Subset27/net_covfactor.tsv.gz tmp_dynamic/Subset27/net_loss.tsv.gz tmp_dynamic/Subset27/net_stats.tsv.gz\n",
      "WARNING:15001:2022-10-28 22:50:23,061:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset28/expression.tsv.gz tmp_dynamic/Subset28/binlinking.tsv.gz tmp_dynamic/Subset28/net_weight.tsv.gz tmp_dynamic/Subset28/net_meanvar.tsv.gz tmp_dynamic/Subset28/net_covfactor.tsv.gz tmp_dynamic/Subset28/net_loss.tsv.gz tmp_dynamic/Subset28/net_stats.tsv.gz\n",
      "WARNING:15013:2022-10-28 22:51:04,283:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset29/expression.tsv.gz tmp_dynamic/Subset29/binlinking.tsv.gz tmp_dynamic/Subset29/net_weight.tsv.gz tmp_dynamic/Subset29/net_meanvar.tsv.gz tmp_dynamic/Subset29/net_covfactor.tsv.gz tmp_dynamic/Subset29/net_loss.tsv.gz tmp_dynamic/Subset29/net_stats.tsv.gz\n",
      "WARNING:15026:2022-10-28 22:51:45,023:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset30/expression.tsv.gz tmp_dynamic/Subset30/binlinking.tsv.gz tmp_dynamic/Subset30/net_weight.tsv.gz tmp_dynamic/Subset30/net_meanvar.tsv.gz tmp_dynamic/Subset30/net_covfactor.tsv.gz tmp_dynamic/Subset30/net_loss.tsv.gz tmp_dynamic/Subset30/net_stats.tsv.gz\n",
      "WARNING:15038:2022-10-28 22:52:25,963:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset31/expression.tsv.gz tmp_dynamic/Subset31/binlinking.tsv.gz tmp_dynamic/Subset31/net_weight.tsv.gz tmp_dynamic/Subset31/net_meanvar.tsv.gz tmp_dynamic/Subset31/net_covfactor.tsv.gz tmp_dynamic/Subset31/net_loss.tsv.gz tmp_dynamic/Subset31/net_stats.tsv.gz\n",
      "WARNING:15052:2022-10-28 22:53:06,916:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset32/expression.tsv.gz tmp_dynamic/Subset32/binlinking.tsv.gz tmp_dynamic/Subset32/net_weight.tsv.gz tmp_dynamic/Subset32/net_meanvar.tsv.gz tmp_dynamic/Subset32/net_covfactor.tsv.gz tmp_dynamic/Subset32/net_loss.tsv.gz tmp_dynamic/Subset32/net_stats.tsv.gz\n",
      "WARNING:15064:2022-10-28 22:53:48,017:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset33/expression.tsv.gz tmp_dynamic/Subset33/binlinking.tsv.gz tmp_dynamic/Subset33/net_weight.tsv.gz tmp_dynamic/Subset33/net_meanvar.tsv.gz tmp_dynamic/Subset33/net_covfactor.tsv.gz tmp_dynamic/Subset33/net_loss.tsv.gz tmp_dynamic/Subset33/net_stats.tsv.gz\n",
      "WARNING:15076:2022-10-28 22:54:29,079:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset34/expression.tsv.gz tmp_dynamic/Subset34/binlinking.tsv.gz tmp_dynamic/Subset34/net_weight.tsv.gz tmp_dynamic/Subset34/net_meanvar.tsv.gz tmp_dynamic/Subset34/net_covfactor.tsv.gz tmp_dynamic/Subset34/net_loss.tsv.gz tmp_dynamic/Subset34/net_stats.tsv.gz\n",
      "WARNING:15088:2022-10-28 22:55:10,161:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset35/expression.tsv.gz tmp_dynamic/Subset35/binlinking.tsv.gz tmp_dynamic/Subset35/net_weight.tsv.gz tmp_dynamic/Subset35/net_meanvar.tsv.gz tmp_dynamic/Subset35/net_covfactor.tsv.gz tmp_dynamic/Subset35/net_loss.tsv.gz tmp_dynamic/Subset35/net_stats.tsv.gz\n",
      "WARNING:15101:2022-10-28 22:55:51,602:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset36/expression.tsv.gz tmp_dynamic/Subset36/binlinking.tsv.gz tmp_dynamic/Subset36/net_weight.tsv.gz tmp_dynamic/Subset36/net_meanvar.tsv.gz tmp_dynamic/Subset36/net_covfactor.tsv.gz tmp_dynamic/Subset36/net_loss.tsv.gz tmp_dynamic/Subset36/net_stats.tsv.gz\n",
      "WARNING:15114:2022-10-28 22:56:32,597:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset37/expression.tsv.gz tmp_dynamic/Subset37/binlinking.tsv.gz tmp_dynamic/Subset37/net_weight.tsv.gz tmp_dynamic/Subset37/net_meanvar.tsv.gz tmp_dynamic/Subset37/net_covfactor.tsv.gz tmp_dynamic/Subset37/net_loss.tsv.gz tmp_dynamic/Subset37/net_stats.tsv.gz\n",
      "WARNING:15128:2022-10-28 22:57:13,711:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset38/expression.tsv.gz tmp_dynamic/Subset38/binlinking.tsv.gz tmp_dynamic/Subset38/net_weight.tsv.gz tmp_dynamic/Subset38/net_meanvar.tsv.gz tmp_dynamic/Subset38/net_covfactor.tsv.gz tmp_dynamic/Subset38/net_loss.tsv.gz tmp_dynamic/Subset38/net_stats.tsv.gz\n",
      "WARNING:15141:2022-10-28 22:57:54,830:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset39/expression.tsv.gz tmp_dynamic/Subset39/binlinking.tsv.gz tmp_dynamic/Subset39/net_weight.tsv.gz tmp_dynamic/Subset39/net_meanvar.tsv.gz tmp_dynamic/Subset39/net_covfactor.tsv.gz tmp_dynamic/Subset39/net_loss.tsv.gz tmp_dynamic/Subset39/net_stats.tsv.gz\n",
      "WARNING:15154:2022-10-28 22:58:38,061:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset40/expression.tsv.gz tmp_dynamic/Subset40/binlinking.tsv.gz tmp_dynamic/Subset40/net_weight.tsv.gz tmp_dynamic/Subset40/net_meanvar.tsv.gz tmp_dynamic/Subset40/net_covfactor.tsv.gz tmp_dynamic/Subset40/net_loss.tsv.gz tmp_dynamic/Subset40/net_stats.tsv.gz\n",
      "WARNING:15166:2022-10-28 22:59:19,663:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset41/expression.tsv.gz tmp_dynamic/Subset41/binlinking.tsv.gz tmp_dynamic/Subset41/net_weight.tsv.gz tmp_dynamic/Subset41/net_meanvar.tsv.gz tmp_dynamic/Subset41/net_covfactor.tsv.gz tmp_dynamic/Subset41/net_loss.tsv.gz tmp_dynamic/Subset41/net_stats.tsv.gz\n",
      "WARNING:15178:2022-10-28 23:00:01,011:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset42/expression.tsv.gz tmp_dynamic/Subset42/binlinking.tsv.gz tmp_dynamic/Subset42/net_weight.tsv.gz tmp_dynamic/Subset42/net_meanvar.tsv.gz tmp_dynamic/Subset42/net_covfactor.tsv.gz tmp_dynamic/Subset42/net_loss.tsv.gz tmp_dynamic/Subset42/net_stats.tsv.gz\n",
      "WARNING:15190:2022-10-28 23:00:42,361:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset43/expression.tsv.gz tmp_dynamic/Subset43/binlinking.tsv.gz tmp_dynamic/Subset43/net_weight.tsv.gz tmp_dynamic/Subset43/net_meanvar.tsv.gz tmp_dynamic/Subset43/net_covfactor.tsv.gz tmp_dynamic/Subset43/net_loss.tsv.gz tmp_dynamic/Subset43/net_stats.tsv.gz\n",
      "WARNING:15202:2022-10-28 23:01:23,167:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset44/expression.tsv.gz tmp_dynamic/Subset44/binlinking.tsv.gz tmp_dynamic/Subset44/net_weight.tsv.gz tmp_dynamic/Subset44/net_meanvar.tsv.gz tmp_dynamic/Subset44/net_covfactor.tsv.gz tmp_dynamic/Subset44/net_loss.tsv.gz tmp_dynamic/Subset44/net_stats.tsv.gz\n",
      "WARNING:15229:2022-10-28 23:02:04,524:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset45/expression.tsv.gz tmp_dynamic/Subset45/binlinking.tsv.gz tmp_dynamic/Subset45/net_weight.tsv.gz tmp_dynamic/Subset45/net_meanvar.tsv.gz tmp_dynamic/Subset45/net_covfactor.tsv.gz tmp_dynamic/Subset45/net_loss.tsv.gz tmp_dynamic/Subset45/net_stats.tsv.gz\n",
      "WARNING:15243:2022-10-28 23:02:45,750:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset46/expression.tsv.gz tmp_dynamic/Subset46/binlinking.tsv.gz tmp_dynamic/Subset46/net_weight.tsv.gz tmp_dynamic/Subset46/net_meanvar.tsv.gz tmp_dynamic/Subset46/net_covfactor.tsv.gz tmp_dynamic/Subset46/net_loss.tsv.gz tmp_dynamic/Subset46/net_stats.tsv.gz\n",
      "WARNING:15258:2022-10-28 23:03:26,896:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset47/expression.tsv.gz tmp_dynamic/Subset47/binlinking.tsv.gz tmp_dynamic/Subset47/net_weight.tsv.gz tmp_dynamic/Subset47/net_meanvar.tsv.gz tmp_dynamic/Subset47/net_covfactor.tsv.gz tmp_dynamic/Subset47/net_loss.tsv.gz tmp_dynamic/Subset47/net_stats.tsv.gz\n",
      "WARNING:15270:2022-10-28 23:04:07,778:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset48/expression.tsv.gz tmp_dynamic/Subset48/binlinking.tsv.gz tmp_dynamic/Subset48/net_weight.tsv.gz tmp_dynamic/Subset48/net_meanvar.tsv.gz tmp_dynamic/Subset48/net_covfactor.tsv.gz tmp_dynamic/Subset48/net_loss.tsv.gz tmp_dynamic/Subset48/net_stats.tsv.gz\n",
      "WARNING:15282:2022-10-28 23:04:48,626:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset49/expression.tsv.gz tmp_dynamic/Subset49/binlinking.tsv.gz tmp_dynamic/Subset49/net_weight.tsv.gz tmp_dynamic/Subset49/net_meanvar.tsv.gz tmp_dynamic/Subset49/net_covfactor.tsv.gz tmp_dynamic/Subset49/net_loss.tsv.gz tmp_dynamic/Subset49/net_stats.tsv.gz\n",
      "WARNING:15294:2022-10-28 23:05:29,716:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset50/expression.tsv.gz tmp_dynamic/Subset50/binlinking.tsv.gz tmp_dynamic/Subset50/net_weight.tsv.gz tmp_dynamic/Subset50/net_meanvar.tsv.gz tmp_dynamic/Subset50/net_covfactor.tsv.gz tmp_dynamic/Subset50/net_loss.tsv.gz tmp_dynamic/Subset50/net_stats.tsv.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WARNING:15306:2022-10-28 23:06:10,752:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset51/expression.tsv.gz tmp_dynamic/Subset51/binlinking.tsv.gz tmp_dynamic/Subset51/net_weight.tsv.gz tmp_dynamic/Subset51/net_meanvar.tsv.gz tmp_dynamic/Subset51/net_covfactor.tsv.gz tmp_dynamic/Subset51/net_loss.tsv.gz tmp_dynamic/Subset51/net_stats.tsv.gz\n",
      "WARNING:15318:2022-10-28 23:06:51,843:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset52/expression.tsv.gz tmp_dynamic/Subset52/binlinking.tsv.gz tmp_dynamic/Subset52/net_weight.tsv.gz tmp_dynamic/Subset52/net_meanvar.tsv.gz tmp_dynamic/Subset52/net_covfactor.tsv.gz tmp_dynamic/Subset52/net_loss.tsv.gz tmp_dynamic/Subset52/net_stats.tsv.gz\n",
      "WARNING:15331:2022-10-28 23:07:32,895:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset53/expression.tsv.gz tmp_dynamic/Subset53/binlinking.tsv.gz tmp_dynamic/Subset53/net_weight.tsv.gz tmp_dynamic/Subset53/net_meanvar.tsv.gz tmp_dynamic/Subset53/net_covfactor.tsv.gz tmp_dynamic/Subset53/net_loss.tsv.gz tmp_dynamic/Subset53/net_stats.tsv.gz\n",
      "WARNING:15344:2022-10-28 23:08:13,654:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset54/expression.tsv.gz tmp_dynamic/Subset54/binlinking.tsv.gz tmp_dynamic/Subset54/net_weight.tsv.gz tmp_dynamic/Subset54/net_meanvar.tsv.gz tmp_dynamic/Subset54/net_covfactor.tsv.gz tmp_dynamic/Subset54/net_loss.tsv.gz tmp_dynamic/Subset54/net_stats.tsv.gz\n",
      "WARNING:15357:2022-10-28 23:08:54,417:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset55/expression.tsv.gz tmp_dynamic/Subset55/binlinking.tsv.gz tmp_dynamic/Subset55/net_weight.tsv.gz tmp_dynamic/Subset55/net_meanvar.tsv.gz tmp_dynamic/Subset55/net_covfactor.tsv.gz tmp_dynamic/Subset55/net_loss.tsv.gz tmp_dynamic/Subset55/net_stats.tsv.gz\n",
      "WARNING:15370:2022-10-28 23:09:35,243:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset56/expression.tsv.gz tmp_dynamic/Subset56/binlinking.tsv.gz tmp_dynamic/Subset56/net_weight.tsv.gz tmp_dynamic/Subset56/net_meanvar.tsv.gz tmp_dynamic/Subset56/net_covfactor.tsv.gz tmp_dynamic/Subset56/net_loss.tsv.gz tmp_dynamic/Subset56/net_stats.tsv.gz\n",
      "WARNING:15383:2022-10-28 23:10:16,068:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset57/expression.tsv.gz tmp_dynamic/Subset57/binlinking.tsv.gz tmp_dynamic/Subset57/net_weight.tsv.gz tmp_dynamic/Subset57/net_meanvar.tsv.gz tmp_dynamic/Subset57/net_covfactor.tsv.gz tmp_dynamic/Subset57/net_loss.tsv.gz tmp_dynamic/Subset57/net_stats.tsv.gz\n",
      "WARNING:15396:2022-10-28 23:10:57,317:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset58/expression.tsv.gz tmp_dynamic/Subset58/binlinking.tsv.gz tmp_dynamic/Subset58/net_weight.tsv.gz tmp_dynamic/Subset58/net_meanvar.tsv.gz tmp_dynamic/Subset58/net_covfactor.tsv.gz tmp_dynamic/Subset58/net_loss.tsv.gz tmp_dynamic/Subset58/net_stats.tsv.gz\n",
      "WARNING:15408:2022-10-28 23:11:37,892:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset59/expression.tsv.gz tmp_dynamic/Subset59/binlinking.tsv.gz tmp_dynamic/Subset59/net_weight.tsv.gz tmp_dynamic/Subset59/net_meanvar.tsv.gz tmp_dynamic/Subset59/net_covfactor.tsv.gz tmp_dynamic/Subset59/net_loss.tsv.gz tmp_dynamic/Subset59/net_stats.tsv.gz\n",
      "WARNING:15420:2022-10-28 23:12:18,989:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset60/expression.tsv.gz tmp_dynamic/Subset60/binlinking.tsv.gz tmp_dynamic/Subset60/net_weight.tsv.gz tmp_dynamic/Subset60/net_meanvar.tsv.gz tmp_dynamic/Subset60/net_covfactor.tsv.gz tmp_dynamic/Subset60/net_loss.tsv.gz tmp_dynamic/Subset60/net_stats.tsv.gz\n",
      "WARNING:15433:2022-10-28 23:12:59,984:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset61/expression.tsv.gz tmp_dynamic/Subset61/binlinking.tsv.gz tmp_dynamic/Subset61/net_weight.tsv.gz tmp_dynamic/Subset61/net_meanvar.tsv.gz tmp_dynamic/Subset61/net_covfactor.tsv.gz tmp_dynamic/Subset61/net_loss.tsv.gz tmp_dynamic/Subset61/net_stats.tsv.gz\n",
      "WARNING:15447:2022-10-28 23:13:40,637:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cuda:0 --nth 4 tmp_dynamic/Subset62/expression.tsv.gz tmp_dynamic/Subset62/binlinking.tsv.gz tmp_dynamic/Subset62/net_weight.tsv.gz tmp_dynamic/Subset62/net_meanvar.tsv.gz tmp_dynamic/Subset62/net_covfactor.tsv.gz tmp_dynamic/Subset62/net_loss.tsv.gz tmp_dynamic/Subset62/net_stats.tsv.gz\n",
      "WARNING:15459:2022-10-28 23:14:21,413:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset1/net_weight.tsv.gz tmp_dynamic/Subset1/net_meanvar.tsv.gz tmp_dynamic/Subset1/net_covfactor.tsv.gz tmp_dynamic/Subset1/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset1/net_meanvar.tsv.gz tmp_dynamic/Subset1/net_weight.tsv.gz tmp_dynamic/Subset1/net_covfactor.tsv.gz tmp_dynamic/Subset1/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset2/net_weight.tsv.gz tmp_dynamic/Subset2/net_meanvar.tsv.gz tmp_dynamic/Subset2/net_covfactor.tsv.gz tmp_dynamic/Subset2/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset2/net_meanvar.tsv.gz tmp_dynamic/Subset2/net_weight.tsv.gz tmp_dynamic/Subset2/net_covfactor.tsv.gz tmp_dynamic/Subset2/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset3/net_weight.tsv.gz tmp_dynamic/Subset3/net_meanvar.tsv.gz tmp_dynamic/Subset3/net_covfactor.tsv.gz tmp_dynamic/Subset3/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset3/net_meanvar.tsv.gz tmp_dynamic/Subset3/net_weight.tsv.gz tmp_dynamic/Subset3/net_covfactor.tsv.gz tmp_dynamic/Subset3/net_iweight.tsv.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset4/net_weight.tsv.gz tmp_dynamic/Subset4/net_meanvar.tsv.gz tmp_dynamic/Subset4/net_covfactor.tsv.gz tmp_dynamic/Subset4/net_nweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset4/net_meanvar.tsv.gz tmp_dynamic/Subset4/net_weight.tsv.gz tmp_dynamic/Subset4/net_covfactor.tsv.gz tmp_dynamic/Subset4/net_iweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset5/net_weight.tsv.gz tmp_dynamic/Subset5/net_meanvar.tsv.gz tmp_dynamic/Subset5/net_covfactor.tsv.gz tmp_dynamic/Subset5/net_nweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset5/net_meanvar.tsv.gz tmp_dynamic/Subset5/net_weight.tsv.gz tmp_dynamic/Subset5/net_covfactor.tsv.gz tmp_dynamic/Subset5/net_iweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset6/net_weight.tsv.gz tmp_dynamic/Subset6/net_meanvar.tsv.gz tmp_dynamic/Subset6/net_covfactor.tsv.gz tmp_dynamic/Subset6/net_nweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset6/net_meanvar.tsv.gz tmp_dynamic/Subset6/net_weight.tsv.gz tmp_dynamic/Subset6/net_covfactor.tsv.gz tmp_dynamic/Subset6/net_iweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset7/net_weight.tsv.gz tmp_dynamic/Subset7/net_meanvar.tsv.gz tmp_dynamic/Subset7/net_covfactor.tsv.gz tmp_dynamic/Subset7/net_nweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset7/net_meanvar.tsv.gz tmp_dynamic/Subset7/net_weight.tsv.gz tmp_dynamic/Subset7/net_covfactor.tsv.gz tmp_dynamic/Subset7/net_iweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset8/net_weight.tsv.gz tmp_dynamic/Subset8/net_meanvar.tsv.gz tmp_dynamic/Subset8/net_covfactor.tsv.gz tmp_dynamic/Subset8/net_nweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset8/net_meanvar.tsv.gz tmp_dynamic/Subset8/net_weight.tsv.gz tmp_dynamic/Subset8/net_covfactor.tsv.gz tmp_dynamic/Subset8/net_iweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset9/net_weight.tsv.gz tmp_dynamic/Subset9/net_meanvar.tsv.gz tmp_dynamic/Subset9/net_covfactor.tsv.gz tmp_dynamic/Subset9/net_nweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset9/net_meanvar.tsv.gz tmp_dynamic/Subset9/net_weight.tsv.gz tmp_dynamic/Subset9/net_covfactor.tsv.gz tmp_dynamic/Subset9/net_iweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset10/net_weight.tsv.gz tmp_dynamic/Subset10/net_meanvar.tsv.gz tmp_dynamic/Subset10/net_covfactor.tsv.gz tmp_dynamic/Subset10/net_nweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset10/net_meanvar.tsv.gz tmp_dynamic/Subset10/net_weight.tsv.gz tmp_dynamic/Subset10/net_covfactor.tsv.gz tmp_dynamic/Subset10/net_iweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset11/net_weight.tsv.gz tmp_dynamic/Subset11/net_meanvar.tsv.gz tmp_dynamic/Subset11/net_covfactor.tsv.gz tmp_dynamic/Subset11/net_nweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset11/net_meanvar.tsv.gz tmp_dynamic/Subset11/net_weight.tsv.gz tmp_dynamic/Subset11/net_covfactor.tsv.gz tmp_dynamic/Subset11/net_iweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset12/net_weight.tsv.gz tmp_dynamic/Subset12/net_meanvar.tsv.gz tmp_dynamic/Subset12/net_covfactor.tsv.gz tmp_dynamic/Subset12/net_nweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset12/net_meanvar.tsv.gz tmp_dynamic/Subset12/net_weight.tsv.gz tmp_dynamic/Subset12/net_covfactor.tsv.gz tmp_dynamic/Subset12/net_iweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset13/net_weight.tsv.gz tmp_dynamic/Subset13/net_meanvar.tsv.gz tmp_dynamic/Subset13/net_covfactor.tsv.gz tmp_dynamic/Subset13/net_nweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset13/net_meanvar.tsv.gz tmp_dynamic/Subset13/net_weight.tsv.gz tmp_dynamic/Subset13/net_covfactor.tsv.gz tmp_dynamic/Subset13/net_iweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset14/net_weight.tsv.gz tmp_dynamic/Subset14/net_meanvar.tsv.gz tmp_dynamic/Subset14/net_covfactor.tsv.gz tmp_dynamic/Subset14/net_nweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset14/net_meanvar.tsv.gz tmp_dynamic/Subset14/net_weight.tsv.gz tmp_dynamic/Subset14/net_covfactor.tsv.gz tmp_dynamic/Subset14/net_iweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset15/net_weight.tsv.gz tmp_dynamic/Subset15/net_meanvar.tsv.gz tmp_dynamic/Subset15/net_covfactor.tsv.gz tmp_dynamic/Subset15/net_nweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset15/net_meanvar.tsv.gz tmp_dynamic/Subset15/net_weight.tsv.gz tmp_dynamic/Subset15/net_covfactor.tsv.gz tmp_dynamic/Subset15/net_iweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset16/net_weight.tsv.gz tmp_dynamic/Subset16/net_meanvar.tsv.gz tmp_dynamic/Subset16/net_covfactor.tsv.gz tmp_dynamic/Subset16/net_nweight.tsv.gz\r\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset16/net_meanvar.tsv.gz tmp_dynamic/Subset16/net_weight.tsv.gz tmp_dynamic/Subset16/net_covfactor.tsv.gz tmp_dynamic/Subset16/net_iweight.tsv.gz\r\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WARNING:15485:2022-10-28 23:14:24,135:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15505:2022-10-28 23:14:24,165:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15507:2022-10-28 23:14:24,172:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15484:2022-10-28 23:14:24,174:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15495:2022-10-28 23:14:24,176:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15499:2022-10-28 23:14:24,179:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15504:2022-10-28 23:14:24,182:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15502:2022-10-28 23:14:24,202:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15512:2022-10-28 23:14:24,206:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15511:2022-10-28 23:14:24,218:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15510:2022-10-28 23:14:24,220:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15494:2022-10-28 23:14:24,234:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15481:2022-10-28 23:14:24,235:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15493:2022-10-28 23:14:24,253:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15487:2022-10-28 23:14:24,263:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15506:2022-10-28 23:14:24,272:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15508:2022-10-28 23:14:24,284:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15482:2022-10-28 23:14:24,285:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15501:2022-10-28 23:14:24,292:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15489:2022-10-28 23:14:24,293:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15497:2022-10-28 23:14:24,304:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15509:2022-10-28 23:14:24,310:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15496:2022-10-28 23:14:24,322:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15488:2022-10-28 23:14:24,339:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15483:2022-10-28 23:14:24,343:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15503:2022-10-28 23:14:24,350:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15490:2022-10-28 23:14:24,358:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15491:2022-10-28 23:14:24,362:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15498:2022-10-28 23:14:24,367:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15500:2022-10-28 23:14:24,431:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15492:2022-10-28 23:14:24,460:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset17/net_weight.tsv.gz tmp_dynamic/Subset17/net_meanvar.tsv.gz tmp_dynamic/Subset17/net_covfactor.tsv.gz tmp_dynamic/Subset17/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset17/net_meanvar.tsv.gz tmp_dynamic/Subset17/net_weight.tsv.gz tmp_dynamic/Subset17/net_covfactor.tsv.gz tmp_dynamic/Subset17/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset18/net_weight.tsv.gz tmp_dynamic/Subset18/net_meanvar.tsv.gz tmp_dynamic/Subset18/net_covfactor.tsv.gz tmp_dynamic/Subset18/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset18/net_meanvar.tsv.gz tmp_dynamic/Subset18/net_weight.tsv.gz tmp_dynamic/Subset18/net_covfactor.tsv.gz tmp_dynamic/Subset18/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset19/net_weight.tsv.gz tmp_dynamic/Subset19/net_meanvar.tsv.gz tmp_dynamic/Subset19/net_covfactor.tsv.gz tmp_dynamic/Subset19/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset19/net_meanvar.tsv.gz tmp_dynamic/Subset19/net_weight.tsv.gz tmp_dynamic/Subset19/net_covfactor.tsv.gz tmp_dynamic/Subset19/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset20/net_weight.tsv.gz tmp_dynamic/Subset20/net_meanvar.tsv.gz tmp_dynamic/Subset20/net_covfactor.tsv.gz tmp_dynamic/Subset20/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset20/net_meanvar.tsv.gz tmp_dynamic/Subset20/net_weight.tsv.gz tmp_dynamic/Subset20/net_covfactor.tsv.gz tmp_dynamic/Subset20/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset21/net_weight.tsv.gz tmp_dynamic/Subset21/net_meanvar.tsv.gz tmp_dynamic/Subset21/net_covfactor.tsv.gz tmp_dynamic/Subset21/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset21/net_meanvar.tsv.gz tmp_dynamic/Subset21/net_weight.tsv.gz tmp_dynamic/Subset21/net_covfactor.tsv.gz tmp_dynamic/Subset21/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset22/net_weight.tsv.gz tmp_dynamic/Subset22/net_meanvar.tsv.gz tmp_dynamic/Subset22/net_covfactor.tsv.gz tmp_dynamic/Subset22/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset22/net_meanvar.tsv.gz tmp_dynamic/Subset22/net_weight.tsv.gz tmp_dynamic/Subset22/net_covfactor.tsv.gz tmp_dynamic/Subset22/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset23/net_weight.tsv.gz tmp_dynamic/Subset23/net_meanvar.tsv.gz tmp_dynamic/Subset23/net_covfactor.tsv.gz tmp_dynamic/Subset23/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset23/net_meanvar.tsv.gz tmp_dynamic/Subset23/net_weight.tsv.gz tmp_dynamic/Subset23/net_covfactor.tsv.gz tmp_dynamic/Subset23/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset24/net_weight.tsv.gz tmp_dynamic/Subset24/net_meanvar.tsv.gz tmp_dynamic/Subset24/net_covfactor.tsv.gz tmp_dynamic/Subset24/net_nweight.tsv.gz\n",
      "WARNING:15486:2022-10-28 23:14:24,792:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset24/net_meanvar.tsv.gz tmp_dynamic/Subset24/net_weight.tsv.gz tmp_dynamic/Subset24/net_covfactor.tsv.gz tmp_dynamic/Subset24/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset25/net_weight.tsv.gz tmp_dynamic/Subset25/net_meanvar.tsv.gz tmp_dynamic/Subset25/net_covfactor.tsv.gz tmp_dynamic/Subset25/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset25/net_meanvar.tsv.gz tmp_dynamic/Subset25/net_weight.tsv.gz tmp_dynamic/Subset25/net_covfactor.tsv.gz tmp_dynamic/Subset25/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset26/net_weight.tsv.gz tmp_dynamic/Subset26/net_meanvar.tsv.gz tmp_dynamic/Subset26/net_covfactor.tsv.gz tmp_dynamic/Subset26/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset26/net_meanvar.tsv.gz tmp_dynamic/Subset26/net_weight.tsv.gz tmp_dynamic/Subset26/net_covfactor.tsv.gz tmp_dynamic/Subset26/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset27/net_weight.tsv.gz tmp_dynamic/Subset27/net_meanvar.tsv.gz tmp_dynamic/Subset27/net_covfactor.tsv.gz tmp_dynamic/Subset27/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset27/net_meanvar.tsv.gz tmp_dynamic/Subset27/net_weight.tsv.gz tmp_dynamic/Subset27/net_covfactor.tsv.gz tmp_dynamic/Subset27/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset28/net_weight.tsv.gz tmp_dynamic/Subset28/net_meanvar.tsv.gz tmp_dynamic/Subset28/net_covfactor.tsv.gz tmp_dynamic/Subset28/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset28/net_meanvar.tsv.gz tmp_dynamic/Subset28/net_weight.tsv.gz tmp_dynamic/Subset28/net_covfactor.tsv.gz tmp_dynamic/Subset28/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset29/net_weight.tsv.gz tmp_dynamic/Subset29/net_meanvar.tsv.gz tmp_dynamic/Subset29/net_covfactor.tsv.gz tmp_dynamic/Subset29/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset29/net_meanvar.tsv.gz tmp_dynamic/Subset29/net_weight.tsv.gz tmp_dynamic/Subset29/net_covfactor.tsv.gz tmp_dynamic/Subset29/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset30/net_weight.tsv.gz tmp_dynamic/Subset30/net_meanvar.tsv.gz tmp_dynamic/Subset30/net_covfactor.tsv.gz tmp_dynamic/Subset30/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset30/net_meanvar.tsv.gz tmp_dynamic/Subset30/net_weight.tsv.gz tmp_dynamic/Subset30/net_covfactor.tsv.gz tmp_dynamic/Subset30/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset31/net_weight.tsv.gz tmp_dynamic/Subset31/net_meanvar.tsv.gz tmp_dynamic/Subset31/net_covfactor.tsv.gz tmp_dynamic/Subset31/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset31/net_meanvar.tsv.gz tmp_dynamic/Subset31/net_weight.tsv.gz tmp_dynamic/Subset31/net_covfactor.tsv.gz tmp_dynamic/Subset31/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset32/net_weight.tsv.gz tmp_dynamic/Subset32/net_meanvar.tsv.gz tmp_dynamic/Subset32/net_covfactor.tsv.gz tmp_dynamic/Subset32/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset32/net_meanvar.tsv.gz tmp_dynamic/Subset32/net_weight.tsv.gz tmp_dynamic/Subset32/net_covfactor.tsv.gz tmp_dynamic/Subset32/net_iweight.tsv.gz\n",
      "WARNING:15645:2022-10-28 23:14:26,102:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15648:2022-10-28 23:14:26,129:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15643:2022-10-28 23:14:26,134:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WARNING:15642:2022-10-28 23:14:26,161:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15649:2022-10-28 23:14:26,165:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15647:2022-10-28 23:14:26,167:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15644:2022-10-28 23:14:26,178:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15652:2022-10-28 23:14:26,190:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15646:2022-10-28 23:14:26,191:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15650:2022-10-28 23:14:26,213:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15654:2022-10-28 23:14:26,259:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15656:2022-10-28 23:14:26,285:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15658:2022-10-28 23:14:26,399:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15660:2022-10-28 23:14:26,409:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15653:2022-10-28 23:14:26,419:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15651:2022-10-28 23:14:26,486:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15662:2022-10-28 23:14:26,488:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15661:2022-10-28 23:14:26,498:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15655:2022-10-28 23:14:26,515:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15657:2022-10-28 23:14:26,528:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset33/net_weight.tsv.gz tmp_dynamic/Subset33/net_meanvar.tsv.gz tmp_dynamic/Subset33/net_covfactor.tsv.gz tmp_dynamic/Subset33/net_nweight.tsv.gz\n",
      "WARNING:15659:2022-10-28 23:14:26,550:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset33/net_meanvar.tsv.gz tmp_dynamic/Subset33/net_weight.tsv.gz tmp_dynamic/Subset33/net_covfactor.tsv.gz tmp_dynamic/Subset33/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset34/net_weight.tsv.gz tmp_dynamic/Subset34/net_meanvar.tsv.gz tmp_dynamic/Subset34/net_covfactor.tsv.gz tmp_dynamic/Subset34/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset34/net_meanvar.tsv.gz tmp_dynamic/Subset34/net_weight.tsv.gz tmp_dynamic/Subset34/net_covfactor.tsv.gz tmp_dynamic/Subset34/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset35/net_weight.tsv.gz tmp_dynamic/Subset35/net_meanvar.tsv.gz tmp_dynamic/Subset35/net_covfactor.tsv.gz tmp_dynamic/Subset35/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset35/net_meanvar.tsv.gz tmp_dynamic/Subset35/net_weight.tsv.gz tmp_dynamic/Subset35/net_covfactor.tsv.gz tmp_dynamic/Subset35/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset36/net_weight.tsv.gz tmp_dynamic/Subset36/net_meanvar.tsv.gz tmp_dynamic/Subset36/net_covfactor.tsv.gz tmp_dynamic/Subset36/net_nweight.tsv.gz\n",
      "WARNING:15670:2022-10-28 23:14:26,805:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15664:2022-10-28 23:14:26,820:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15666:2022-10-28 23:14:26,823:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15665:2022-10-28 23:14:26,834:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset36/net_meanvar.tsv.gz tmp_dynamic/Subset36/net_weight.tsv.gz tmp_dynamic/Subset36/net_covfactor.tsv.gz tmp_dynamic/Subset36/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset37/net_weight.tsv.gz tmp_dynamic/Subset37/net_meanvar.tsv.gz tmp_dynamic/Subset37/net_covfactor.tsv.gz tmp_dynamic/Subset37/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset37/net_meanvar.tsv.gz tmp_dynamic/Subset37/net_weight.tsv.gz tmp_dynamic/Subset37/net_covfactor.tsv.gz tmp_dynamic/Subset37/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset38/net_weight.tsv.gz tmp_dynamic/Subset38/net_meanvar.tsv.gz tmp_dynamic/Subset38/net_covfactor.tsv.gz tmp_dynamic/Subset38/net_nweight.tsv.gz\n",
      "WARNING:15668:2022-10-28 23:14:26,912:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15663:2022-10-28 23:14:26,914:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset38/net_meanvar.tsv.gz tmp_dynamic/Subset38/net_weight.tsv.gz tmp_dynamic/Subset38/net_covfactor.tsv.gz tmp_dynamic/Subset38/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset39/net_weight.tsv.gz tmp_dynamic/Subset39/net_meanvar.tsv.gz tmp_dynamic/Subset39/net_covfactor.tsv.gz tmp_dynamic/Subset39/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset39/net_meanvar.tsv.gz tmp_dynamic/Subset39/net_weight.tsv.gz tmp_dynamic/Subset39/net_covfactor.tsv.gz tmp_dynamic/Subset39/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset40/net_weight.tsv.gz tmp_dynamic/Subset40/net_meanvar.tsv.gz tmp_dynamic/Subset40/net_covfactor.tsv.gz tmp_dynamic/Subset40/net_nweight.tsv.gz\n",
      "WARNING:15672:2022-10-28 23:14:27,074:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15671:2022-10-28 23:14:27,142:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset40/net_meanvar.tsv.gz tmp_dynamic/Subset40/net_weight.tsv.gz tmp_dynamic/Subset40/net_covfactor.tsv.gz tmp_dynamic/Subset40/net_iweight.tsv.gz\n",
      "WARNING:15669:2022-10-28 23:14:27,208:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset41/net_weight.tsv.gz tmp_dynamic/Subset41/net_meanvar.tsv.gz tmp_dynamic/Subset41/net_covfactor.tsv.gz tmp_dynamic/Subset41/net_nweight.tsv.gz\n",
      "WARNING:15667:2022-10-28 23:14:27,238:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset41/net_meanvar.tsv.gz tmp_dynamic/Subset41/net_weight.tsv.gz tmp_dynamic/Subset41/net_covfactor.tsv.gz tmp_dynamic/Subset41/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset42/net_weight.tsv.gz tmp_dynamic/Subset42/net_meanvar.tsv.gz tmp_dynamic/Subset42/net_covfactor.tsv.gz tmp_dynamic/Subset42/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset42/net_meanvar.tsv.gz tmp_dynamic/Subset42/net_weight.tsv.gz tmp_dynamic/Subset42/net_covfactor.tsv.gz tmp_dynamic/Subset42/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset43/net_weight.tsv.gz tmp_dynamic/Subset43/net_meanvar.tsv.gz tmp_dynamic/Subset43/net_covfactor.tsv.gz tmp_dynamic/Subset43/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset43/net_meanvar.tsv.gz tmp_dynamic/Subset43/net_weight.tsv.gz tmp_dynamic/Subset43/net_covfactor.tsv.gz tmp_dynamic/Subset43/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset44/net_weight.tsv.gz tmp_dynamic/Subset44/net_meanvar.tsv.gz tmp_dynamic/Subset44/net_covfactor.tsv.gz tmp_dynamic/Subset44/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset44/net_meanvar.tsv.gz tmp_dynamic/Subset44/net_weight.tsv.gz tmp_dynamic/Subset44/net_covfactor.tsv.gz tmp_dynamic/Subset44/net_iweight.tsv.gz\n",
      "WARNING:15673:2022-10-28 23:14:27,579:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset45/net_weight.tsv.gz tmp_dynamic/Subset45/net_meanvar.tsv.gz tmp_dynamic/Subset45/net_covfactor.tsv.gz tmp_dynamic/Subset45/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset45/net_meanvar.tsv.gz tmp_dynamic/Subset45/net_weight.tsv.gz tmp_dynamic/Subset45/net_covfactor.tsv.gz tmp_dynamic/Subset45/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset46/net_weight.tsv.gz tmp_dynamic/Subset46/net_meanvar.tsv.gz tmp_dynamic/Subset46/net_covfactor.tsv.gz tmp_dynamic/Subset46/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset46/net_meanvar.tsv.gz tmp_dynamic/Subset46/net_weight.tsv.gz tmp_dynamic/Subset46/net_covfactor.tsv.gz tmp_dynamic/Subset46/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset47/net_weight.tsv.gz tmp_dynamic/Subset47/net_meanvar.tsv.gz tmp_dynamic/Subset47/net_covfactor.tsv.gz tmp_dynamic/Subset47/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset47/net_meanvar.tsv.gz tmp_dynamic/Subset47/net_weight.tsv.gz tmp_dynamic/Subset47/net_covfactor.tsv.gz tmp_dynamic/Subset47/net_iweight.tsv.gz\n",
      "WARNING:15758:2022-10-28 23:14:28,218:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset48/net_weight.tsv.gz tmp_dynamic/Subset48/net_meanvar.tsv.gz tmp_dynamic/Subset48/net_covfactor.tsv.gz tmp_dynamic/Subset48/net_nweight.tsv.gz\n",
      "WARNING:15759:2022-10-28 23:14:28,293:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15769:2022-10-28 23:14:28,374:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15801:2022-10-28 23:14:28,401:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15795:2022-10-28 23:14:28,402:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WARNING:15760:2022-10-28 23:14:28,456:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15762:2022-10-28 23:14:28,466:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15797:2022-10-28 23:14:28,511:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15806:2022-10-28 23:14:28,514:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15808:2022-10-28 23:14:28,541:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset48/net_meanvar.tsv.gz tmp_dynamic/Subset48/net_weight.tsv.gz tmp_dynamic/Subset48/net_covfactor.tsv.gz tmp_dynamic/Subset48/net_iweight.tsv.gz\n",
      "WARNING:15793:2022-10-28 23:14:28,637:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15805:2022-10-28 23:14:28,660:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15761:2022-10-28 23:14:28,706:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset49/net_weight.tsv.gz tmp_dynamic/Subset49/net_meanvar.tsv.gz tmp_dynamic/Subset49/net_covfactor.tsv.gz tmp_dynamic/Subset49/net_nweight.tsv.gz\n",
      "WARNING:15807:2022-10-28 23:14:28,756:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15818:2022-10-28 23:14:28,758:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15816:2022-10-28 23:14:28,761:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15764:2022-10-28 23:14:28,814:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15817:2022-10-28 23:14:28,837:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset49/net_meanvar.tsv.gz tmp_dynamic/Subset49/net_weight.tsv.gz tmp_dynamic/Subset49/net_covfactor.tsv.gz tmp_dynamic/Subset49/net_iweight.tsv.gz\n",
      "WARNING:15814:2022-10-28 23:14:28,843:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset50/net_weight.tsv.gz tmp_dynamic/Subset50/net_meanvar.tsv.gz tmp_dynamic/Subset50/net_covfactor.tsv.gz tmp_dynamic/Subset50/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset50/net_meanvar.tsv.gz tmp_dynamic/Subset50/net_weight.tsv.gz tmp_dynamic/Subset50/net_covfactor.tsv.gz tmp_dynamic/Subset50/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset51/net_weight.tsv.gz tmp_dynamic/Subset51/net_meanvar.tsv.gz tmp_dynamic/Subset51/net_covfactor.tsv.gz tmp_dynamic/Subset51/net_nweight.tsv.gz\n",
      "WARNING:15815:2022-10-28 23:14:28,971:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset51/net_meanvar.tsv.gz tmp_dynamic/Subset51/net_weight.tsv.gz tmp_dynamic/Subset51/net_covfactor.tsv.gz tmp_dynamic/Subset51/net_iweight.tsv.gz\n",
      "WARNING:15813:2022-10-28 23:14:29,030:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset52/net_weight.tsv.gz tmp_dynamic/Subset52/net_meanvar.tsv.gz tmp_dynamic/Subset52/net_covfactor.tsv.gz tmp_dynamic/Subset52/net_nweight.tsv.gz\n",
      "WARNING:15820:2022-10-28 23:14:29,033:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset52/net_meanvar.tsv.gz tmp_dynamic/Subset52/net_weight.tsv.gz tmp_dynamic/Subset52/net_covfactor.tsv.gz tmp_dynamic/Subset52/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset53/net_weight.tsv.gz tmp_dynamic/Subset53/net_meanvar.tsv.gz tmp_dynamic/Subset53/net_covfactor.tsv.gz tmp_dynamic/Subset53/net_nweight.tsv.gz\n",
      "WARNING:15826:2022-10-28 23:14:29,248:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset53/net_meanvar.tsv.gz tmp_dynamic/Subset53/net_weight.tsv.gz tmp_dynamic/Subset53/net_covfactor.tsv.gz tmp_dynamic/Subset53/net_iweight.tsv.gz\n",
      "WARNING:15819:2022-10-28 23:14:29,288:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15828:2022-10-28 23:14:29,292:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15825:2022-10-28 23:14:29,330:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset54/net_weight.tsv.gz tmp_dynamic/Subset54/net_meanvar.tsv.gz tmp_dynamic/Subset54/net_covfactor.tsv.gz tmp_dynamic/Subset54/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset54/net_meanvar.tsv.gz tmp_dynamic/Subset54/net_weight.tsv.gz tmp_dynamic/Subset54/net_covfactor.tsv.gz tmp_dynamic/Subset54/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset55/net_weight.tsv.gz tmp_dynamic/Subset55/net_meanvar.tsv.gz tmp_dynamic/Subset55/net_covfactor.tsv.gz tmp_dynamic/Subset55/net_nweight.tsv.gz\n",
      "WARNING:15827:2022-10-28 23:14:29,492:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset55/net_meanvar.tsv.gz tmp_dynamic/Subset55/net_weight.tsv.gz tmp_dynamic/Subset55/net_covfactor.tsv.gz tmp_dynamic/Subset55/net_iweight.tsv.gz\n",
      "WARNING:15829:2022-10-28 23:14:29,657:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset56/net_weight.tsv.gz tmp_dynamic/Subset56/net_meanvar.tsv.gz tmp_dynamic/Subset56/net_covfactor.tsv.gz tmp_dynamic/Subset56/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset56/net_meanvar.tsv.gz tmp_dynamic/Subset56/net_weight.tsv.gz tmp_dynamic/Subset56/net_covfactor.tsv.gz tmp_dynamic/Subset56/net_iweight.tsv.gz\n",
      "WARNING:15831:2022-10-28 23:14:29,705:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset57/net_weight.tsv.gz tmp_dynamic/Subset57/net_meanvar.tsv.gz tmp_dynamic/Subset57/net_covfactor.tsv.gz tmp_dynamic/Subset57/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset57/net_meanvar.tsv.gz tmp_dynamic/Subset57/net_weight.tsv.gz tmp_dynamic/Subset57/net_covfactor.tsv.gz tmp_dynamic/Subset57/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset58/net_weight.tsv.gz tmp_dynamic/Subset58/net_meanvar.tsv.gz tmp_dynamic/Subset58/net_covfactor.tsv.gz tmp_dynamic/Subset58/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset58/net_meanvar.tsv.gz tmp_dynamic/Subset58/net_weight.tsv.gz tmp_dynamic/Subset58/net_covfactor.tsv.gz tmp_dynamic/Subset58/net_iweight.tsv.gz\n",
      "WARNING:15832:2022-10-28 23:14:29,846:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset59/net_weight.tsv.gz tmp_dynamic/Subset59/net_meanvar.tsv.gz tmp_dynamic/Subset59/net_covfactor.tsv.gz tmp_dynamic/Subset59/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset59/net_meanvar.tsv.gz tmp_dynamic/Subset59/net_weight.tsv.gz tmp_dynamic/Subset59/net_covfactor.tsv.gz tmp_dynamic/Subset59/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset60/net_weight.tsv.gz tmp_dynamic/Subset60/net_meanvar.tsv.gz tmp_dynamic/Subset60/net_covfactor.tsv.gz tmp_dynamic/Subset60/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset60/net_meanvar.tsv.gz tmp_dynamic/Subset60/net_weight.tsv.gz tmp_dynamic/Subset60/net_covfactor.tsv.gz tmp_dynamic/Subset60/net_iweight.tsv.gz\n",
      "WARNING:15840:2022-10-28 23:14:29,996:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset61/net_weight.tsv.gz tmp_dynamic/Subset61/net_meanvar.tsv.gz tmp_dynamic/Subset61/net_covfactor.tsv.gz tmp_dynamic/Subset61/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset61/net_meanvar.tsv.gz tmp_dynamic/Subset61/net_weight.tsv.gz tmp_dynamic/Subset61/net_covfactor.tsv.gz tmp_dynamic/Subset61/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset62/net_weight.tsv.gz tmp_dynamic/Subset62/net_meanvar.tsv.gz tmp_dynamic/Subset62/net_covfactor.tsv.gz tmp_dynamic/Subset62/net_nweight.tsv.gz\n",
      "WARNING:15893:2022-10-28 23:14:30,301:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15895:2022-10-28 23:14:30,391:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_dynamic/Subset62/net_meanvar.tsv.gz tmp_dynamic/Subset62/net_weight.tsv.gz tmp_dynamic/Subset62/net_covfactor.tsv.gz tmp_dynamic/Subset62/net_iweight.tsv.gz\n",
      "WARNING:15930:2022-10-28 23:14:30,540:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15913:2022-10-28 23:14:30,552:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15915:2022-10-28 23:14:30,573:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset1/net_iweight.tsv.gz tmp_dynamic/Subset1/net_meanvar.tsv.gz tmp_dynamic/Subset1/net_covfactor.tsv.gz tmp_dynamic/Subset1/net_inweight.tsv.gz\n",
      "WARNING:15924:2022-10-28 23:14:30,654:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15912:2022-10-28 23:14:30,664:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15939:2022-10-28 23:14:30,673:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset2/net_iweight.tsv.gz tmp_dynamic/Subset2/net_meanvar.tsv.gz tmp_dynamic/Subset2/net_covfactor.tsv.gz tmp_dynamic/Subset2/net_inweight.tsv.gz\n",
      "WARNING:15916:2022-10-28 23:14:30,725:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset3/net_iweight.tsv.gz tmp_dynamic/Subset3/net_meanvar.tsv.gz tmp_dynamic/Subset3/net_covfactor.tsv.gz tmp_dynamic/Subset3/net_inweight.tsv.gz\n",
      "WARNING:15934:2022-10-28 23:14:30,900:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset4/net_iweight.tsv.gz tmp_dynamic/Subset4/net_meanvar.tsv.gz tmp_dynamic/Subset4/net_covfactor.tsv.gz tmp_dynamic/Subset4/net_inweight.tsv.gz\n",
      "WARNING:15955:2022-10-28 23:14:30,960:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset5/net_iweight.tsv.gz tmp_dynamic/Subset5/net_meanvar.tsv.gz tmp_dynamic/Subset5/net_covfactor.tsv.gz tmp_dynamic/Subset5/net_inweight.tsv.gz\n",
      "WARNING:15959:2022-10-28 23:14:31,020:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset6/net_iweight.tsv.gz tmp_dynamic/Subset6/net_meanvar.tsv.gz tmp_dynamic/Subset6/net_covfactor.tsv.gz tmp_dynamic/Subset6/net_inweight.tsv.gz\n",
      "WARNING:15953:2022-10-28 23:14:31,042:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset7/net_iweight.tsv.gz tmp_dynamic/Subset7/net_meanvar.tsv.gz tmp_dynamic/Subset7/net_covfactor.tsv.gz tmp_dynamic/Subset7/net_inweight.tsv.gz\n",
      "WARNING:15949:2022-10-28 23:14:31,132:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15961:2022-10-28 23:14:31,194:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15960:2022-10-28 23:14:31,217:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15969:2022-10-28 23:14:31,230:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15967:2022-10-28 23:14:31,259:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15963:2022-10-28 23:14:31,271:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15968:2022-10-28 23:14:31,282:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset8/net_iweight.tsv.gz tmp_dynamic/Subset8/net_meanvar.tsv.gz tmp_dynamic/Subset8/net_covfactor.tsv.gz tmp_dynamic/Subset8/net_inweight.tsv.gz\n",
      "WARNING:15970:2022-10-28 23:14:31,364:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset9/net_iweight.tsv.gz tmp_dynamic/Subset9/net_meanvar.tsv.gz tmp_dynamic/Subset9/net_covfactor.tsv.gz tmp_dynamic/Subset9/net_inweight.tsv.gz\n",
      "WARNING:15975:2022-10-28 23:14:31,423:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:15977:2022-10-28 23:14:31,484:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset10/net_iweight.tsv.gz tmp_dynamic/Subset10/net_meanvar.tsv.gz tmp_dynamic/Subset10/net_covfactor.tsv.gz tmp_dynamic/Subset10/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset11/net_iweight.tsv.gz tmp_dynamic/Subset11/net_meanvar.tsv.gz tmp_dynamic/Subset11/net_covfactor.tsv.gz tmp_dynamic/Subset11/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset12/net_iweight.tsv.gz tmp_dynamic/Subset12/net_meanvar.tsv.gz tmp_dynamic/Subset12/net_covfactor.tsv.gz tmp_dynamic/Subset12/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset13/net_iweight.tsv.gz tmp_dynamic/Subset13/net_meanvar.tsv.gz tmp_dynamic/Subset13/net_covfactor.tsv.gz tmp_dynamic/Subset13/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset14/net_iweight.tsv.gz tmp_dynamic/Subset14/net_meanvar.tsv.gz tmp_dynamic/Subset14/net_covfactor.tsv.gz tmp_dynamic/Subset14/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset15/net_iweight.tsv.gz tmp_dynamic/Subset15/net_meanvar.tsv.gz tmp_dynamic/Subset15/net_covfactor.tsv.gz tmp_dynamic/Subset15/net_inweight.tsv.gz\n",
      "WARNING:15983:2022-10-28 23:14:31,638:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset16/net_iweight.tsv.gz tmp_dynamic/Subset16/net_meanvar.tsv.gz tmp_dynamic/Subset16/net_covfactor.tsv.gz tmp_dynamic/Subset16/net_inweight.tsv.gz\n",
      "WARNING:15976:2022-10-28 23:14:31,755:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset17/net_iweight.tsv.gz tmp_dynamic/Subset17/net_meanvar.tsv.gz tmp_dynamic/Subset17/net_covfactor.tsv.gz tmp_dynamic/Subset17/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset18/net_iweight.tsv.gz tmp_dynamic/Subset18/net_meanvar.tsv.gz tmp_dynamic/Subset18/net_covfactor.tsv.gz tmp_dynamic/Subset18/net_inweight.tsv.gz\n",
      "WARNING:15989:2022-10-28 23:14:31,784:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset19/net_iweight.tsv.gz tmp_dynamic/Subset19/net_meanvar.tsv.gz tmp_dynamic/Subset19/net_covfactor.tsv.gz tmp_dynamic/Subset19/net_inweight.tsv.gz\n",
      "WARNING:15979:2022-10-28 23:14:31,913:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset20/net_iweight.tsv.gz tmp_dynamic/Subset20/net_meanvar.tsv.gz tmp_dynamic/Subset20/net_covfactor.tsv.gz tmp_dynamic/Subset20/net_inweight.tsv.gz\n",
      "WARNING:15985:2022-10-28 23:14:31,956:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset21/net_iweight.tsv.gz tmp_dynamic/Subset21/net_meanvar.tsv.gz tmp_dynamic/Subset21/net_covfactor.tsv.gz tmp_dynamic/Subset21/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset22/net_iweight.tsv.gz tmp_dynamic/Subset22/net_meanvar.tsv.gz tmp_dynamic/Subset22/net_covfactor.tsv.gz tmp_dynamic/Subset22/net_inweight.tsv.gz\n",
      "WARNING:16000:2022-10-28 23:14:32,065:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset23/net_iweight.tsv.gz tmp_dynamic/Subset23/net_meanvar.tsv.gz tmp_dynamic/Subset23/net_covfactor.tsv.gz tmp_dynamic/Subset23/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset24/net_iweight.tsv.gz tmp_dynamic/Subset24/net_meanvar.tsv.gz tmp_dynamic/Subset24/net_covfactor.tsv.gz tmp_dynamic/Subset24/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset25/net_iweight.tsv.gz tmp_dynamic/Subset25/net_meanvar.tsv.gz tmp_dynamic/Subset25/net_covfactor.tsv.gz tmp_dynamic/Subset25/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset26/net_iweight.tsv.gz tmp_dynamic/Subset26/net_meanvar.tsv.gz tmp_dynamic/Subset26/net_covfactor.tsv.gz tmp_dynamic/Subset26/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset27/net_iweight.tsv.gz tmp_dynamic/Subset27/net_meanvar.tsv.gz tmp_dynamic/Subset27/net_covfactor.tsv.gz tmp_dynamic/Subset27/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset28/net_iweight.tsv.gz tmp_dynamic/Subset28/net_meanvar.tsv.gz tmp_dynamic/Subset28/net_covfactor.tsv.gz tmp_dynamic/Subset28/net_inweight.tsv.gz\n",
      "WARNING:16016:2022-10-28 23:14:32,255:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16029:2022-10-28 23:14:32,370:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16025:2022-10-28 23:14:32,428:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16039:2022-10-28 23:14:32,468:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16035:2022-10-28 23:14:32,574:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16048:2022-10-28 23:14:32,593:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset29/net_iweight.tsv.gz tmp_dynamic/Subset29/net_meanvar.tsv.gz tmp_dynamic/Subset29/net_covfactor.tsv.gz tmp_dynamic/Subset29/net_inweight.tsv.gz\n",
      "WARNING:16050:2022-10-28 23:14:32,717:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset30/net_iweight.tsv.gz tmp_dynamic/Subset30/net_meanvar.tsv.gz tmp_dynamic/Subset30/net_covfactor.tsv.gz tmp_dynamic/Subset30/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset31/net_iweight.tsv.gz tmp_dynamic/Subset31/net_meanvar.tsv.gz tmp_dynamic/Subset31/net_covfactor.tsv.gz tmp_dynamic/Subset31/net_inweight.tsv.gz\n",
      "WARNING:16078:2022-10-28 23:14:32,955:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset32/net_iweight.tsv.gz tmp_dynamic/Subset32/net_meanvar.tsv.gz tmp_dynamic/Subset32/net_covfactor.tsv.gz tmp_dynamic/Subset32/net_inweight.tsv.gz\n",
      "WARNING:16089:2022-10-28 23:14:33,074:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16094:2022-10-28 23:14:33,100:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset33/net_iweight.tsv.gz tmp_dynamic/Subset33/net_meanvar.tsv.gz tmp_dynamic/Subset33/net_covfactor.tsv.gz tmp_dynamic/Subset33/net_inweight.tsv.gz\n",
      "WARNING:16083:2022-10-28 23:14:33,159:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16096:2022-10-28 23:14:33,159:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16088:2022-10-28 23:14:33,167:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16093:2022-10-28 23:14:33,169:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset34/net_iweight.tsv.gz tmp_dynamic/Subset34/net_meanvar.tsv.gz tmp_dynamic/Subset34/net_covfactor.tsv.gz tmp_dynamic/Subset34/net_inweight.tsv.gz\n",
      "WARNING:16095:2022-10-28 23:14:33,208:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset35/net_iweight.tsv.gz tmp_dynamic/Subset35/net_meanvar.tsv.gz tmp_dynamic/Subset35/net_covfactor.tsv.gz tmp_dynamic/Subset35/net_inweight.tsv.gz\n",
      "WARNING:16108:2022-10-28 23:14:33,273:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset36/net_iweight.tsv.gz tmp_dynamic/Subset36/net_meanvar.tsv.gz tmp_dynamic/Subset36/net_covfactor.tsv.gz tmp_dynamic/Subset36/net_inweight.tsv.gz\n",
      "WARNING:16106:2022-10-28 23:14:33,322:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16112:2022-10-28 23:14:33,323:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset37/net_iweight.tsv.gz tmp_dynamic/Subset37/net_meanvar.tsv.gz tmp_dynamic/Subset37/net_covfactor.tsv.gz tmp_dynamic/Subset37/net_inweight.tsv.gz\n",
      "WARNING:16113:2022-10-28 23:14:33,441:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset38/net_iweight.tsv.gz tmp_dynamic/Subset38/net_meanvar.tsv.gz tmp_dynamic/Subset38/net_covfactor.tsv.gz tmp_dynamic/Subset38/net_inweight.tsv.gz\n",
      "WARNING:16118:2022-10-28 23:14:33,505:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16119:2022-10-28 23:14:33,528:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16124:2022-10-28 23:14:33,587:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16125:2022-10-28 23:14:33,653:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset39/net_iweight.tsv.gz tmp_dynamic/Subset39/net_meanvar.tsv.gz tmp_dynamic/Subset39/net_covfactor.tsv.gz tmp_dynamic/Subset39/net_inweight.tsv.gz\n",
      "WARNING:16126:2022-10-28 23:14:33,692:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16127:2022-10-28 23:14:33,693:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16128:2022-10-28 23:14:33,714:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16129:2022-10-28 23:14:33,795:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset40/net_iweight.tsv.gz tmp_dynamic/Subset40/net_meanvar.tsv.gz tmp_dynamic/Subset40/net_covfactor.tsv.gz tmp_dynamic/Subset40/net_inweight.tsv.gz\n",
      "WARNING:16131:2022-10-28 23:14:33,833:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset41/net_iweight.tsv.gz tmp_dynamic/Subset41/net_meanvar.tsv.gz tmp_dynamic/Subset41/net_covfactor.tsv.gz tmp_dynamic/Subset41/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset42/net_iweight.tsv.gz tmp_dynamic/Subset42/net_meanvar.tsv.gz tmp_dynamic/Subset42/net_covfactor.tsv.gz tmp_dynamic/Subset42/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset43/net_iweight.tsv.gz tmp_dynamic/Subset43/net_meanvar.tsv.gz tmp_dynamic/Subset43/net_covfactor.tsv.gz tmp_dynamic/Subset43/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset44/net_iweight.tsv.gz tmp_dynamic/Subset44/net_meanvar.tsv.gz tmp_dynamic/Subset44/net_covfactor.tsv.gz tmp_dynamic/Subset44/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset45/net_iweight.tsv.gz tmp_dynamic/Subset45/net_meanvar.tsv.gz tmp_dynamic/Subset45/net_covfactor.tsv.gz tmp_dynamic/Subset45/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset46/net_iweight.tsv.gz tmp_dynamic/Subset46/net_meanvar.tsv.gz tmp_dynamic/Subset46/net_covfactor.tsv.gz tmp_dynamic/Subset46/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset47/net_iweight.tsv.gz tmp_dynamic/Subset47/net_meanvar.tsv.gz tmp_dynamic/Subset47/net_covfactor.tsv.gz tmp_dynamic/Subset47/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset48/net_iweight.tsv.gz tmp_dynamic/Subset48/net_meanvar.tsv.gz tmp_dynamic/Subset48/net_covfactor.tsv.gz tmp_dynamic/Subset48/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset49/net_iweight.tsv.gz tmp_dynamic/Subset49/net_meanvar.tsv.gz tmp_dynamic/Subset49/net_covfactor.tsv.gz tmp_dynamic/Subset49/net_inweight.tsv.gz\n",
      "WARNING:16176:2022-10-28 23:14:34,142:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset50/net_iweight.tsv.gz tmp_dynamic/Subset50/net_meanvar.tsv.gz tmp_dynamic/Subset50/net_covfactor.tsv.gz tmp_dynamic/Subset50/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset51/net_iweight.tsv.gz tmp_dynamic/Subset51/net_meanvar.tsv.gz tmp_dynamic/Subset51/net_covfactor.tsv.gz tmp_dynamic/Subset51/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset52/net_iweight.tsv.gz tmp_dynamic/Subset52/net_meanvar.tsv.gz tmp_dynamic/Subset52/net_covfactor.tsv.gz tmp_dynamic/Subset52/net_inweight.tsv.gz\n",
      "WARNING:16181:2022-10-28 23:14:34,266:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset53/net_iweight.tsv.gz tmp_dynamic/Subset53/net_meanvar.tsv.gz tmp_dynamic/Subset53/net_covfactor.tsv.gz tmp_dynamic/Subset53/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset54/net_iweight.tsv.gz tmp_dynamic/Subset54/net_meanvar.tsv.gz tmp_dynamic/Subset54/net_covfactor.tsv.gz tmp_dynamic/Subset54/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset55/net_iweight.tsv.gz tmp_dynamic/Subset55/net_meanvar.tsv.gz tmp_dynamic/Subset55/net_covfactor.tsv.gz tmp_dynamic/Subset55/net_inweight.tsv.gz\n",
      "WARNING:16182:2022-10-28 23:14:34,509:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset56/net_iweight.tsv.gz tmp_dynamic/Subset56/net_meanvar.tsv.gz tmp_dynamic/Subset56/net_covfactor.tsv.gz tmp_dynamic/Subset56/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset57/net_iweight.tsv.gz tmp_dynamic/Subset57/net_meanvar.tsv.gz tmp_dynamic/Subset57/net_covfactor.tsv.gz tmp_dynamic/Subset57/net_inweight.tsv.gz\n",
      "WARNING:16187:2022-10-28 23:14:34,555:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset58/net_iweight.tsv.gz tmp_dynamic/Subset58/net_meanvar.tsv.gz tmp_dynamic/Subset58/net_covfactor.tsv.gz tmp_dynamic/Subset58/net_inweight.tsv.gz\n",
      "WARNING:16200:2022-10-28 23:14:34,664:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset59/net_iweight.tsv.gz tmp_dynamic/Subset59/net_meanvar.tsv.gz tmp_dynamic/Subset59/net_covfactor.tsv.gz tmp_dynamic/Subset59/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset60/net_iweight.tsv.gz tmp_dynamic/Subset60/net_meanvar.tsv.gz tmp_dynamic/Subset60/net_covfactor.tsv.gz tmp_dynamic/Subset60/net_inweight.tsv.gz\n",
      "WARNING:16229:2022-10-28 23:14:34,882:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16217:2022-10-28 23:14:34,927:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16222:2022-10-28 23:14:35,004:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16242:2022-10-28 23:14:35,034:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16237:2022-10-28 23:14:35,053:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset61/net_iweight.tsv.gz tmp_dynamic/Subset61/net_meanvar.tsv.gz tmp_dynamic/Subset61/net_covfactor.tsv.gz tmp_dynamic/Subset61/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_dynamic/Subset62/net_iweight.tsv.gz tmp_dynamic/Subset62/net_meanvar.tsv.gz tmp_dynamic/Subset62/net_covfactor.tsv.gz tmp_dynamic/Subset62/net_inweight.tsv.gz\n",
      "WARNING:16266:2022-10-28 23:14:35,314:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16276:2022-10-28 23:14:35,327:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16282:2022-10-28 23:14:35,467:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16281:2022-10-28 23:14:35,472:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16280:2022-10-28 23:14:35,479:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16284:2022-10-28 23:14:35,521:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16286:2022-10-28 23:14:35,550:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16283:2022-10-28 23:14:35,617:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16288:2022-10-28 23:14:35,669:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16293:2022-10-28 23:14:35,691:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16294:2022-10-28 23:14:35,745:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16285:2022-10-28 23:14:35,776:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16295:2022-10-28 23:14:35,818:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16292:2022-10-28 23:14:35,854:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16301:2022-10-28 23:14:35,864:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16302:2022-10-28 23:14:35,949:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WARNING:16300:2022-10-28 23:14:35,966:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16309:2022-10-28 23:14:36,174:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16314:2022-10-28 23:14:36,203:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16308:2022-10-28 23:14:36,206:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16320:2022-10-28 23:14:36,350:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16324:2022-10-28 23:14:36,423:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16344:2022-10-28 23:14:36,688:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "WARNING:16345:2022-10-28 23:14:36,692:~/.conda/envs/dictys/lib/python3.9/site-packages/dictys/utils/parallel.py:63:Environmental variables have different values.\n",
      "mkdir -p output/\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network tofile --dynamic data tmp_dynamic tmp_dynamic/subsets.txt output/dynamic.h5\n"
     ]
    }
   ],
   "source": [
    "!cd ..; dictys_helper network_inference.sh -j 32 -J 1 dynamic "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 4. Dynamic network analysis\n",
    "Note: this is a demonstration of software function and not intended as a biological investigation.\n",
    "### Load data\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "from dictys.net import dynamic_network\n",
    "d0=dynamic_network.from_file('../output/dynamic.h5')\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### TF discovery based on 4 patterns of highly variable regulatory activity over developmental trajectory"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IRS\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAeIAAABECAYAAABd2zbkAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjYuMCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy89olMNAAAACXBIWXMAAAsTAAALEwEAmpwYAAAZEUlEQVR4nO2de3hU1bn/P28CQkEuBQSJCIIRix5AlNoiHAgCiqhoUVSK2HKRqvDj1x+04v2AD6IcTz31eEEEQW76YJUKNAgIEhDCpRGwAqdULgoaMQQSyMVAknl/f+w1YQiTTJKZZAZ8P88zz8zee631fmfttfa712WvLaqKYRiGYRjRIS7aAgzDMAzjx4w5YsMwDMOIIuaIDcMwDCOKmCM2DMMwjChijtgwDMMwoog5YsMwDMOIIuaIDcMwDCOKmCM2DMMwjChijtgwDMMwoog54mpARJJE5GsRWSsiH4tI0yBhLhORBRG0lyMijd322yKSGCTctSKiIlKrnLSWiEi2iPStgo6GIpIsIikisllEuorIoBBxRgT8fk9E1onIBhG5srL2y7HxpYjcF8TeABHZLiIjRWSZiHwqImtEpFWY9vznP0VENopIh0rEm1JqX4KIbBORgvLOWyU0lVkmqwMRaeTyIUVEjrvvOZVMI8V9v+zKxxYR6R6mrpK8FpG7ROQDEXnHbY8SkTQRuVVE1juba0SkeTg2A+z6y8YSEakbgfQmhavLpRVYT1LCKW+VtBuYJyki0qgCcTZUo5aQ5cJth7yeVhRzxNXHfFXtDcwFhtSAvUPAqGAHRMR/nh8BtoVI5yHgz1XU8ACwWFWTgB7ASaBcRwyMCPg9VFV7AU8C46qo4QxEpDOwAbg9iL07gSGq+hYwTlX/HXgB+H8RMD3f5cMf8fK0qhwD+gCbI6SpQmUyoMyEhaoeV9UklxdfuN/Dq5jcH1z5uAd4IhL6nEMfAzwF+Nzue4AeqpoM9HE25wG/iYRNTpeNVODuykaO1LkplWbpelKj9nF54j7HqyH9SlGBcgEVu55WCHPE1U9jABF5xt3tfSIil1WDnSXA7SIS77aTRGSRiCQDnUTkauAbIMcfIaAFukpEGgKo6ndhaMgHuolIM1UtwrvY93P/+6LS9kRkNNDRHe+oqoUunQuBf4ShI5BBwOtAPRG5I8De08BAYK6I9FDVAy58IVAcIdsADYETIvJ4QGuuC3iV3bWYU0TkXhf+Otc63ygiF6pqgapmRVAPeGUy3p2HS0TkZhH5s9P0uXg9NY+KyHCnLU1EboqEYRG5zbUyU0Wkv9vnb+l+KiKt3b7R4vWq/Lc/bqny8XkE5FwJ/BcwGPjB2b0buB5YKSKJATZ/AuyKgM1AdgCzRORdEVktIm85DdcElJUn3L7flqrPs0VkNWXcfFeBwHpSx+2bJiJ/F5GRTkOKiPwn3k1JtSEiE135/0REWotIE2d7rYj8jwtWS7yevzQRuS3CEkKWi2DX07BQVftE+AMkAV8DacBuoDMwwx3rAMwALgMWRNDeFOAPrvC8jVdBZwSEmQk0A1KAWm5fPfc9CngwIOwkoG8VdNQGngG+AFYDvwz8j8HsARsCjl+Ad1e+H+gcobxZ6r5/B9xayt7bQGLAdrzTfXmEzv964DugY8B/TwQWut/rgWbud5yLt8RtPwncEZBmyXmLUJlsBFwNLHVp13fhjgb89mtuBKyKwLlI9f8Pd67XlLLTF3jOHd/ivm8AUgLS+CvwLXBjBM7RcWCS277MX1ZL1ZHWwCZgJ3BpBPIgCZjifj+Hd8P5pNue7urMTwBx+9a67d9y+hpyPTDT/X7C/x8iXE9SgC5AHWB9QL50C9dWOWUzBZgDrHT7e7g86RNwnvz5ss+dm/rAxghrqUi5OOt6Gs7HWsTVx3xV7QpsxesCTRJvrGs6XiupOpgFPBiw/RmAiFwBnFDVTP8B13J+UUTWA2OBhHCNq2qhqj6rqh2Bt4DfV8aeqp5S1R54NxPPhqtHvHHyjiKyArgPrwVcHn8C5qnqvnBt453/nsA1wFRgmPvvszj938V/TlTV3/21031/i+tNiSCBZbK1qu7Cu8ivVNU8F2ZPwO+bXZldClwaAfuN8W5EVwOrgJYiInit70/xbiYT8C5wX6vXq/JZYAKq+ivgF3h5Gi7Tge7+lnkwVPWgqnYD/gPvRjcSDBORtXj5sQTY7vbvwLtRawssF5F1ePnlH5v250W7gDhn5E9VKKee7FTVk5zumo2IvTLwd9fP4HRvWBpefqwH4kRkIXC/O3bUnZs8ItuDBSHKRbDrabiYI65+nse7y12lp8fKHqgOQ6qaDezBu2OG0xWoI/BzV9E6AW/gOYj6zlm8Bki49kWkjYjUdpsZeN02/q7ysuypiysBcU/guoTCZBAwSlX7qzc22tJvL4j2kYCqaqS73XLwbrwewbvbfpCA/y5u0pScHncL1Bf2OSmD54EnROQW4H+BviLSzB0LvOg+DtwC3FFqf1XJxust6ePqQWegCZCk3vj803j/ORNo427euvgjB3SZ5gJ5hE8RcC/eDcBZk6ZEpLa7UYDIlUlwY/WqOgbPiXR2+zvjtfQeBqapNza9l9PlwH8ODgTEKcmfMAhWT+IIXlciUQ7K4ytO/7euePkRr6rPqOpQYII71kREWolIPU5fYyJFueWC4NfTsKiRWXE/ZlR1j3Mwh13rQoF38VoE1cH/4E0yCNSwGFgMJTNQH8IrYImuMB3Ca4HhxmBuAwaKyBuq+mYlbF8DvCciP+CNtY4C3hSR94HxwewBh0TkA7yu2DdERPHyaEzpxKvArcArAdu7gV+IyId4rd9AXge2uvxZp6r/EabtYSLSAy+fp+A5s/Xu4+dxYJmInMSrzN+XTsSVnY/wLk4rReQJVd0SjjBXJpsDj+Ll0dV4+VF6MtLfnN6teE40XHzAS8Aad553A/8HyBWRT3AtIVUtEm9mdSqwLiD+IvGeDIjHy7uwUdVjIjIMb0hka6nDLYH5IuLDm3j420jYDEIHEVkDHFTVTeLNGn5VRHYDp4Jo3iIiD7s4XwMHw7QfrJ5MDDPNKqGqh91YcCref/8NcL2ITMUb+lrtgmbiDaFdQwR6z4LoKLNclHE9DQt/f7thGIZRw4j36NEGVV0dKqxx/mJd04ZhGIYRRaxFbBiGYRhRxFrElaB///7+8ctofGJKR7Q0nDp1KlbyIhY0xIqOWNAQKzpiQUOs6IgFDbGio1zMEVeCzMyIzVYPi1jQURMa9u7dy5AhQ2jdujWtWrWiVatW/PSnP61xHaGIBQ0QGzpiQQPUvI7k5GSeeuqpqGooi1jQEQsaIHZ0lMZmTRshKSoq4ujRo6SlpXHrrbdGNO3c3Fy+//57tm/fDsD69ev529/+xqFDh7jgggsYM2YMH3/8MfXr1wco+TaMaOPz+cjMzGTlypU89thj3H///aEjGUYQzBH/SPD5fGzYsIGCggIAjh49ypYtp5+CKS4uZtOmTfzww9mPSh45coTCwkI6deoUEUd84sQJpk6dyrJlyzhw4AANGjSgS5cuXHjhhSQkJLBo0SI6depErVq1iI+P9COChhE+OTk5DBw4kG3bttGwYUMWLFhA7969oy3LOEcxR/wj4Pjx4yQlJXH8+HESE72XMokI3bt3p2HD04t83XTTTbRr1+6s+LVr1+aKK67g9NoGlWPv3r18+eWXrFmzhvT0dJYuXUqvXr2YO3curVu3pnnzsF9qYxjVTnFxMU8//TRvvfUWGRkZ3HnnnWRmZlK7du3QkQ2jHMwRn8ecOHGCN954g5dffpk77riDV199lbi4mpsWsHDhQlasWMGiRYvo1asXl156Kf3792f8+PF07dq1xnSc6+zdu5f333+/ZPvjjz9m9+7dxMXF8e2335YT04gUq1atYuLEidSrV48PPviAn//859SpUyd0xGpGVZk9ezapqaksX768pH5bufBYvHgxgwaFegFc9DFHfJ6RkZHBhAkTWLDAe9XxTTfdxAsvvMD9999f5RZtRdi2bRtjx45l06ZNJfuaN29ORkYGaWlpXHfdddVmOxzS09OZO3cuxcXFTJs2jRYtWjBixIizwh09epSXXnqpZLs6H/vLy8ujQ4cOHDp0qGTfsGHDSEjwlqju168fc+bMsW77aqKoqIgZM2Zw/PhxvvrqK2bOnEndunWZNGkSEyZMoFat6Fw2d+zYwfLlyzlx4gQvvvgiPp+32uQVV1zB4MGDWb58ufUuOT766CMGDBhAnTp1SobjYhlzxOcRq1ev5vbbb6dnz54cOnSIVq3Cer99SHw+H59//jk+n48xY8Zw8cUXU1BQEBMthWDs27eP7OxskpOTmT59OrVr1+bw4cP069ePzp078/rrr5OVlcXhw4fPilunTh02bNhA9+5hvY/+DLKzs9m3z3u/REFBAY888gjHjh0jKyuLgQMH8s4770TMllExDh06xJw5c5g7dy733nsvzZo1Y/fu3XTo0CGqurZs2ULPnj0ZMmQICQkJrFq1ij59+kRVU2mysrK48cYbyczMREQ4eDDclTcrTl5eHlu3bmXYsGFcfvnlrF+/nuTkZAYMGFBjGsLBHPF5wsaNG+nXrx/Tp0/nd7/7XbW2fsF7DOC+++7jo48+on379ogIM2bMiFknXFRURGJiIp07d6ZOnTrMmzePK6+8kri4OC655JJqz69g9O/fn8OHD9O0aVN8Ph/dunXjiSe89937W79GaNauXcv+/fsZOXJkyLAffvghR48eDXosPz+fcePGce211zJ//nxuuOGGSEutFPv372ft2rWoKmPHjuX5559n/PjxUdVUmhUrVpR0g7/22mskJCSwZMmSGtVQVFRE//79+de//kVGRgYdO3bkm2++4ZJLLqlRHeFgjjgGWL16NQcOHCg3zIMPnn67YWZmJjNnzizZ9vl8PPTQQ0yZMoWHHgp7/fEKkZubS8OGDfnss89o3759jdgMh3379tG2bVt27NgRbSkAzJkzhz179nDkyJGodXVWhpycHN57772S7tBAAstmNLjrrrvo0aNHSEecn5/Pr371K4YPH17mjdeMGTMYPXp0dcgEvAlfCxcu5OTJkwDceeedXHTRRWeF27dvHzfffDMtWrTgZz/7Gc8880xMOeE1a9awZcsWnnzyyZL87NGjB8899xwNGjSoEQ1//etfyczMZNOmTRw8eJC9e/fWmO3y+PLLL0lJSTljX6g6EvtXgHOInJwcsrOzKxWnuLiYfv368etf/5p69eqVGS7wRPq7YQKZNm0ajz76aKVsh8Nll13GrFmzasxeOBQWFrJ06VK6dInEG+OCk5+fX2ZLK1jYESNGMH/+/Jh2wkePHiU/Px+A+fPn8/LLLzNw4NmvdK5uR5ydnU1OTk7QYwUFBWRlZbF48eIz9p86deqMMXaAnTt3kpiYyOzZs6tNaygNqampPPjggzzwwAMlk+6COdgJEybQsmVLVq5cWe51obrJy8vj2LFjZ+zz+Xz07duXIUOGsGDBAoYOHVpjenJzc8nKyuLIkSMMGjSIkSNHEh8fz7vvvhsTThhg8uTJ7Nq164wJqeaIa5Bu3bpx4MABmjRpUuE4p06dok+fPixcuLDCcdq0aXNGi9gon6lTpzJp0qSSCWzVwd13383KlSsr1KVcUFDA4MGDY3oBiIKCApo1a1bSbZ+fn8/06dO55557alxLQkIC9evXp27ds18Ne/LkSfr06XPWDc0///nPs7qWc3JyatRpBNOQn5/P5MmTeeyxx1i7di2DBw8+Y0a8n1OnTrFx48aoOmGA2267jY0bN9KiRYuSfYWFhSQlJUVlDkPv3r3ZuXMn9evXZ9SoUTF5Hdy0aRPLli3jqquuqnAcc8RhcvDgQbZv305RURG7du0iLy8v6pXnx0R2dnbIMal33nmHv/zlL9x9993VpmH79u3s3buXtm3bVouNcFm3bl2lemu++OIL2rdvz549eyKqY/PmzXz//VmvXS6TjIwMfD4fGRkZlRrH79SpE2lpaVWRGDFCaejdu3eNLLlYkTrip1mzZnTv3p2tW7eSnp5OSkoK6enptGzZssY0lIXP5yMtLY2srCwaN24cVlqRJjU1lSNHjpCXl8f+/fsrPVxnjjhMnn32WZKTk+natSvjxo0zJ1zDZGZmhuxqvOqqqyI62zmYhqFDh9KmTZtqsxEOeXl5JCUlBe1WLgufz8fYsWMjrqVbt2707du3wvXE5/Px+OOPR2Uy3flCReoIeJOeli9fXjJxsGfPngwfPpyLL764xjSUh6ry8MMPx5wTVlW6d+/OLbfcQnx8PBMnTqz0kJM54jBJT09n1qxZEV+D2agYiYmJNT5LM5iGefPmRVVDeXz33Xe0a9cu6vmUm5tL3bp1WbVqlTnWGqQydaRp06akp6cTHx/PmjVrIrYAUCzU0+rC30Jfvnx5ldMwR1wJcnNzSU1NPWPf1q1bmTJlSpQUGcZpgpVPgE8//TTozNya1ADebNLGjRubE45hmjRpwiuvvEKDBg1qdBW+mqK88llVdu7cWal5QcGQ6lwh6HxDRI4AX0fJfKaq9o8VHbGgAaKaF7GgIVZ0xIKGWNERCxpiRUcsaIgVHSUagmGO2DAMwzCiyPnX9xDDTJ48Oep3PbGgAWJDRyxogNjQEQsaIDZ0xIIGiA0dpuE01anDHLFhGIZhRBFzxIZhGIYRRcwRG4ZhGEYUMUdsGIZhGFHEZk0bhmEYRhQpd0EPEWkIvAvUB+oCY1W1zMVTReQxYD5wERCnqtvKCJcC9FXVooB9fwYmqGpxJf8DIjJCVWeHm0553Hjj81G/Y8lpHRvv+i1oEgMLMsSABICin0RbAZxqGPWiCUDDqyv29qnqJC5GysVF9XOjLYG7Lg56+a1x+tXfG20JtIyPgYoK1Gm5P2gJDdU1/QCwWFWTgB5AuSvAq+oLqvotcA1wbWUEqurvw3CeIyKUjmEYhmHUKKEccT7QTUSaqWqRquaIyBgRuV1ErhCRY+IxWUSuF5G3RSQRGA38UUQWikiciMwSkXUi8lFA2tNE5O8iMhK8VrKI1BKRSSIyT0RWi8gsd+xyEdkiIktEZI2IXOZPREQGAh1d/H5lpDNXRJ4Wkc0i8oyLd5GILBWRtSLyegTz1DAMwzAqTChHPB84CKx1Du1iIBW4AegO/B24CugCbA+I9ybwoqoOBe4AMlS1FxD4ZoQFeK3s3wSxu11V+wKtRaQx8Afg/wKDgOaBAVV1KfCFqiap6sdlpNMS2KmqvwT8r6B5DHheVXsDOSLSrbQIERktImkikpaevrXsXDIMwzCMKlKuI1bVQlV9VlU7Am8Bvwf+AXQErgdeAv4dbzy4sIxk2uM5b1TVF7B/p6qeBHxB4ux03+lAI6At8A/X5bwzSPiyCEzH/ztXROKBDsALbry6D3DWG91V9U1V7aqqXRMSrq+EWcMwDMOoGOU6YhFpIyK13WYGnsMtxnOejYEUYDjwRamohUC8+70H+KVLL9BeeTNMAo8JcACv+zkeuDpE+LL2l05zDzDetaS7AufnO7oMwzCMmCbUaxCvAd4TkR/wnOtwt3870FhVT4pIEa7FG8Bm4G0R+Te8LuXbRWQ9kAsMqILO/wIW4t0MZDktgWwVkQ+BP1UizanAmyLSCO/GYhTwVVmBP/nk8RiZj2kYhmGcT5wTzxGLSC1VLXIt4o1Aj8BHnwzDMAzjXOVcWVmrnWtRbwXmmRM2DMMwzhdCdU3jum7946f+2dEHVHV42bHKTe8xYL573rgi4X8L1FLVngH73gamqGqFnhQXkSRgLt5YM3gzuQuBP+NNJqsNvKmqc8tK44eF0V/Qw5efF20JABTnRl9HfKOG0ZYAQHFO9BduqN2sWbQlAFB8PDvaEjjxZbTePX8mP+12XbQlQFx86DA1QHHG99GWQHyjxtGWAMBPhgYf4gzpiFX1OJAEICIb3OIeQRGRuFIzo4Ol90Iom9XEfFV9yr8hItOAdao6WkQEb/a3YRiGYdQoIR1xaUTkNuBRF/dZVV0hIpvxWso/iEgT4AegE7AWb3Z1N+A1VZ3tb83iPUN8O+Bfe+xWoDPwMt5ymktUdWoILY3wJnE1BHao6jgRaQq87zQUAv9dRvQbVHUigHoD5euDpD8ab3ESXhlxJyNvtEeYDMMwjMhS2THiOLzFNW7EayX/0e1vBjynquPd9ipV7Q4Mxnv++AZgZJD0vlHVAcC3eI57D5Ckqr8A+olIqAVCRwOLXLd1PRH5Bd7s5xku3QsCwg5zq27NqeifDXyO2JywYRiGUR1UtkXcGGgKrHbbzV23boaqfhMQzr94xnd4C3cUikiwMVZ/uG9d2m2BP4lIPeBKSq2iFYTLgeXudxqQ6NJY5vbtCAh7Rte0YRiGYcQClXXE2XirVN2sqsUiUltVVURKjwuXtZBGaUovsvEwME1VU0RkA6HfsbMPuA7YBXQFZgGt8Fb+2o3Xyl5ZRtxUERmqqgvdzUQPVf20LENlDbIbhhE7XBhtAYZRBSrbNe3DW9ZyjYisxZt1HEmSgVdF5D3gVAXCzwTuE5FPgZOquhnPGT8sIitcmLKW3pwM9HJLXG4E2oWl3DAMwzCqwDmxoEdl8C+jqao+EUkGRlf0USnDMAzDqGkqPWv6HOBCIFlELgBWmxM2DMMwYpnzrkVsGIZhGOcS58oSl4ZhGIZxXmKO2DAMwzCiiDliwzAMw4gi5ogNwzAMI4qYIzYMwzCMKPL/Aam1o7vW2MflAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<Figure size 518.4x50.4 with 15 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAeIAAABECAYAAABd2zbkAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjYuMCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy89olMNAAAACXBIWXMAAAsTAAALEwEAmpwYAAAawklEQVR4nO2deXRVRbq3n18IEQgJIQwiCGIzBgydkNiKCKYFRQKCM4LiCNjdV5ao3TatXm0VHC4O6CdcUNqhUbTttlFQ4aooERQRULQdLjYR9YoIOJAwyBB8vz92nXAIGU9Ocg5Yz1pnJbt27arfrl1Vb027tswMj8fj8Xg8sSEh1gI8Ho/H4/k54w2xx+PxeDwxxBtij8fj8XhiiDfEHo/H4/HEEG+IPR6Px+OJId4Qezwej8cTQ7wh9ng8Ho8nhnhD7PF4PB5PDPGG2OPxeDyeGOINcRSRlCepRFJrd3ysJJPUsRy/j0nqHOX4UyW9KGmxpLcl5dbgWklaLWlMFPWcJOk1p2eRpL6SlrpzUyU1kDRY0v+G3KMYd56kSWHHVYYv6WhJSyS9IWmOpAa1jP8Ld+9vSsqIMJznJW2RNDBSLWXCG+c0LZZUIGm3pCGS3pN0eTTiCIsrPA2el9SoAn9/lpRXjfA6Strowns5Qj3VKp91QaR5omxedm7HSXpL0lJJ90VB0yKna6RL5ycq8B+VMiJpgIvvDUlzJbUox0/U68gKtJStKx6TdKWkIe54lkvndpLelbRTUmI0NXhDHH1WA8Pd/2cCK+sx7ouAf5pZHnAisKYG154ObI6WEEktgVuAM5yeM4AdofNmNsHM9gJvA7+MVry1ZAsw1Mz6A+uA/FqGN9vd+x+A34QcJdWk3P0GmFpLHaWY2UNmlud0/R24iyC/jjSzv0QrnjBCafAWcA7U+P7L8orTf2qE168mduUTKsgTEfAFcLKZnQi0lpRZS00DgMHABUB6JX63UMsyIqkVcBNwugvnj0BSTcOpYxaa2Yvu/24unb8DBhDUWVHFG+Lo8xrBwwLoCXwE3C+pOZT2BHPc+T+5FuGtUYp7B9BHUkszKzGzrZIecHG8IKmZa+ldKamJa5GmuGtHAU9HSQcEBfQJMysGMLOtZvZe6KSLO9HMfjCzXVGMtyISJT3sev2nSWrqNDSVdIWkq52WIud/D7A3SnGnAsUuvv8C/iqpjaQFzu0OAEnPuF7qy5JSAcxsQ5Q07Iekowkq3VeAYcDjkk4smyZRjHI1MEvSg8BCSemSXpe0APiV05Qn6SVJ8yQtk3SJ66m9KEkunF+7Hlmk2qpVPiV1176Rg6tcL3GJpGclrZJ0ZKQJ4QjliT+5Z75cUrbT0Nf1mBdLGuH850ia79ybmtk3ZrbTnYtKXjWzH4F7CBrl1GEZyScw/ltdvJ8CyS7fF0i6MczvfnWkgtGTpyS9Kukvzu0mp/MdSYsiu/sDyJM0xvWUe0l6wcx2mtkP4Z4ktXX5eKmk6ZFG5g1x9NkN7JR0PPCJc1sBnO16Ar80s1XO/TXXIuwtqV0U4p4NfAm87jJqPyDZxfE0QQt8GjAEeBiY4oz1qUAB0TM8AG2BDQCSRrmMencUw68p6cANBPd+hZltA24nSIezgftDHiW1BU4Bajz8WYbRkt4AHgWecW5zzexC4E/Afa53dIM7d4mZneT8jigbWLRwRm0mcKWZvQEsBC4ws6VUkCZRoD9B2XjT9WbHALPMbDAQPry5x8yGAfOBbNdTWw9kE+SnrsCvgYGSekWgo7rl8w7gN+75/D/nrylwLnAvQfpEQtk8cb975hcAv3d+7gCGh41aAOw2s9OBl9jXkMClQSsz+zhCPWX5GjgeoA7LyBG4uiGMycDlLi16hjV0yqsjPzSzgcBuSceb2a0urdYAkXZqRocaXsBpIUczuxH4l5kNreC6b4FTXI85VVKXSCL3hrhueAmYAfzTHT9H0MrsB7wR5i/UQ/wXcHRtIzWzPS5TZgJ/IShA77rTK4HOFnxu6wkgN2zoZQxBxRBNNhAYY8xsDnAh0DLKcdSEzWa2yczWA2nO7RUgB3jUzH4CkHQY8Dgw1sxKahnnbFeJZBFUaAChRlhXguFazOwnBXNtU1wlfSUu7eqI3wIrwhqE4RyQJrVktKTXCdL8efbd/y/Yl//fDfP/ofv7dZn/m5vZLjPb7p7LC8AxEWqqTvlsaWb/C8HzcW4fu//D81BNKZsnQoZ5Fvueuczs2zJxh9KiNG5J6cCDQDTn9tsBy8KO66KMlNYNYXQDZjtDmOF0QPl1ZMhtNdDZaboGeNfMCiLQA27KwBn0hTW4rgXwD6f7RCIst94Q1w0vEVQ4K9zxNqAYuAp4KsxfaG70GODz2kYq6ShJDd3hJoIMHxoGzwUKJSUTGN5nJF3sznUlqIyuBSZI6l5bLQRpMFpSM3cc1cUNERD+vc/QMOdvCEYRxrrKBeAhYFoUexgAWwmGIgFCFesaXM/D9cSy2Dd6MS1MY1RRsDBpNMH8fXmUlya1YbaZ/drM/oNgxCV0/+vYl/+zw/xbBf9L+6ZRAPoChRFqqk753Cypq4s4VE+Wl4ciJZQnfgfkAWPDwjS5xUsVxa1gsdATwO/N7JtaagkF2giYAMwLc66LMvIScGHoeSpYkLWRYJ1CHkGdFXo25dWRvwz7W6hgoV+WmUW8aK0WjAKec7rfJMJ8EevK8ZDEDelcDrBvaoungUmhVrbjJEm/AwrM7KsoRJ1FYGB/JJi/GU0wx7KEoOCPIhgCuhNYRDBXt9DMspzWS4DEMhojwsw2S/oz8Lykn4ASF+9+Q0cKVnbfCRwj6VWChSA7y4YXIRe4IcgDkNSeYG50CME84U2SXgDOAo6SNIFg2HBuLeIfLelEoBEwiX1DjxDc8+NuPuwtgt5RZ0kLgf8j6Pkg6QFgKDBM0gwze6gWegCuA1oBL4flzVBP54A0Yd+webSZBTwr6SKgumsE+km6zflfYmbLI4m4muXzeuBhSQbMJejNR4OyeWI4QS88fKTsT8B8SbsIeu4bywnnXOBY4L/cPfzJzJaV46+6mvoQTBE8RLAgq87KiKsbbgNecNMk3xPktUecsd/DvqH//epId68Zbi74SzNb5v5Pcb3S1WY2IcJ0qBTXyVlA0AD4H0nXE6w5+KukM2oVdjBS6alrJOUDPc1sSqy1eDye/fHl8+DANe6XmtmrsdYSTXyPuB6QdDZwNftem/B4PHGCL5+eWON7xB6Px+PxxBC/WKsGnHbaaUawaCIWv7jSEQ8aYqwjHjTEi4540BAvOuJBQ7zoiAcN++kYNGhQzDWUhx+argHffvttrCUA8aEjHjRAfOiIBw0QHzriQQPEh4540ADxoSMeNAC88sortG/fnubNm9OkSRMaNGjA4YcfToMGDWjUqBEpKSmkpaWRkJBA48aNadWqFc2bNyc1NZVmzZqRmppKx44dadKkSVR1eUPs8Xg8np8FWVlZzJ07lx9++IEff/yRkpISvvkmePtrx44dbN26lS1btmBmbN++ncLCQoqKiiguLqaoqIiioiK++uorMjIyyM3NJTs7m7PPPpuWLWu3RYI3xB6Px+P5WZCQkMBRRx3FUUcdFXEY27dvZ9WqVaxcuZJFixZxww03kJeXx4UXXkjv3r3p0KFDjcP0htjj8Xg8nmqSnJxM//796d+/PwBffvklCxcu5K677qKwsJAWLVowduxYhg8fTqdOnaoVpjfEHs8hwt690dwq3OOJHmvWrCErK4u2bdvSsmVL2rdvT1ZWFj179uTII48kNTW16kDilA4dOjBu3DjGjRtHSUkJy5cv5/bbb2fKlClIok+fPjz77LOVhuENscdziPD+++/TtWtXOnXqRNeuXcnIyKB9+/bk5+eH7yDlqSd++OEHvvvuO1q0aMG6devYsmXLfucbN25M9+7R2E02/mnXrh3Tpk1j48aNbNmyhcLCQh555BE++eQTNm3aRHJyMgkJCWRkZNC/f3+aNGmCmdG6dWsOP/xwhgwZclDk4cTERPr27cuLLwbb+K9Zs4aVK6v+0qY3xB5PLVm1ahWJiYmkpKQwYsQIhg8fTlJSEmlpaeTk5FQdQJTIzs5m5syZrFu3jsLCQlasWMH06dMZP348xx13HBs2bCA5OZlly5ZRVFREdnZ26erPzMxMBgwYQLNmzcjMzKR169b1pvtQZePGjbRp0wYzIyUlhY4dO+53fsOGDRx22GH06tWLzMxMJJGcnMyxxx5LYWEhI0eOpFmzZrERH2WaNm1K3759yz23e/dutmzZwo4dO/jggw9YvHgx3377LZJYu3YtK1eu5JxzziE3N5emTZuSmppKly5dOPLIIxkyZEhEc7L1Rbdu3ejWrVuV/rwhPghZtWoVCQkJnHHGGYwcOfKA8wMHDqR58+YxUFb/FBYWcs4553DRRRcxbNiwmGjo3bs3y5cvZ+3atTz88MPcfXfwtcePPvqIzMzM0so0ISGBAQMGkJ6eTr9+/WjTpk1UdUgiNzeX3NzcUjczY8GCBXz66ad07NiRb775hvvuu482bdqwevVqAHbu3ElBQQHTp09n586dLFu2jISEYIuBlJQUrr76arp37063bt3o1SuSLw/WP0VFRZSUlJCYGLsqrnv37ixfvhwzIyEhoTRNQ5gZ7777LqtXry5duVtYWMiiRYt47bXXmDlzJhdffDHt2rWjTZs29OvXLxa3UeckJSWVNvw6dux4QDk2M9avX89nn33G2rVrKSwspHHjxixYsIDx48dzzDHH0LlzZ3r27MnNN9/MwoULSU9P5/jjy91mPi7xO2vVgCOOOMIuu+yy0uNBgwbRp08fABo2bFjRZdGidFwmJyfHCgoKuOGGG/j666/387Rp0yaWLFlCcnIyJSUlXHXVVdx22237ApFqWzkJoEOHDlZYWPnHbxITE+tqOKk00E6dOtn111/PhAkTAEqHs84//3wuuOACunbtWmVgET67Ug25ublW3vDT5s2bKSjY91W2b775hoKCArZs2cKiRYtITk7GzLjooouYNGkS6enpAPz0009Iqm7aVamjppSUlBCqF0K96h07djB//nxGjRrFbbfdxhFHHFHqv2HDhqUaGjRoYE2aNEESo0aNokWLFkBgGJ988klKSkoOiGvIkCFcfvnlDB48uLbSS3U0btzYSkpKaNSo0QHx5ebm0r9/f4YPH052dvYBgZSlhvkjKs9j165dPP300yxYsIC9e/fy8ssv07dvX26//XYyMzNp0KBBVUGU6ujRo4e9//77NdaQkJBQnXiqpSFaebMsu3fv5uWXX2bHjh1MmTKFlStX0qlTJzZu3Ej37t1ZvHgxycnJda6jGlRemM3M/6r5a9u2rU2aNMkmTZpk11xzjSUlJVliYqIBNmPGDKtjSnXk5ORU6nHr1q1WXFxsX3zxhbVr184aNGhgiYmJpVrvvPNO27t3b610NGzYcL9wy/4AGzZsmC1dujTSeKrUYGFpsXPnTisuLrbi4mJ74oknbOzYsQZUqC9c52WXXWbz5s2rtYaasG3bNisuLrbVq1dbfn6+paWl2dSpU23q1KmWnp5uJ598st1///12//332/Tp023r1q11oqMmrFu3zkaOHHlAuoZryM7OtuLiYnvnnXdKy0rot2TJktJnFPqtW7fObr75ZktJSbE5c+bUVuJ+aVE2ruLiYvv+++9t6tSpNn78+CrzRih/nHfeebZixYqINESL77//3iZMmGAJCQmWmJhokydPtk8++aRaOqpTDsr7SbJbb73VPvjgg0hl11veNDMrKSmx4uJiKykpsV27dpXm1frWUQGV2paYG7eD6VfRQ1y5cqUlJSVZfn7+fr/hw4fb559/XunTqQFRyUyvv/66tWzZ0nJycmz27NkR66hKw+bNm+2KK66wtLQ027lzZyTxVKmhOjqq4tNPP7Xx48dbUlJSTZ9VVAt3yDiMHz/ebrzxRrvuuutKj3v27GmTJ0+uFx0RUmsNf/vb30ySnXzyyZafn2+DBw+2Rx99tN51lGX9+vV28cUXW7t27arbeK3z5/GPf/zDhg4daikpKXbrrbfWmY558+bZmWeeaU2aNLH8/HwbNWqUbdu2rSZBxDxvPvzww3GhI1xDeT8/RxwFcnJyKCgo4LvvvtvPfcaMGfTu3bvS4VEzY/bs2XTp0qWuZQKQl5fH6tWr+fvf/87o0aOZNm1aja5ftqx6nzxt2bIlM2bM4KOPPqJ79+60adOGp5566oAFK7GmS5cuPPDAA+zZs4fs7GyuuOIK7rjjjnrXcdVVV1V4btmyZfTt25f58+eXe+5Q4LzzziMrK4t///vfAHz22WdceumlzJw5cz9/oemWCy+8sF50tW3blscee4xjjjmGjIyM0umDcD1//OMfOeecc+pFD8DZZ5/NWWedxcKFC8nPz+eFF144YP45Gvni9NNPZ+jQoSxdupTi4mImTpxI9+7dycnJYe7cuQfFKuYxY8bEWkK18IY4SpS3MOCUU05h1apVlV530003MW7cOAYNGlSpv4kTJ9ZKXzjt2rVjwoQJnHrqqRQVFUUt3PJ4/vnnWbNmDRMnTmTs2LEMGDCgymvOPffcar8IHy3uvfdeBg0axJlnnklqamqVlUw0n0dV9OnTh/fff59t27bVW5yxoGvXrvs1WgcOHHjAKz8LFy7kuuuu46uvvjrg+rp8Jq+99hrlrYl47rnnuPbaa1m7dm2dawhHEoMHD2bdunVs2LChTuMJLRLr27cvH3/8Mf369eOaa67h8MMPB4K55DFjxhzQSPFUH2+I65CkpKTSxVwVcc899zBnzpwDKpz6oEePHnUeR3p6On369OG+++7jmWeeqfI+586dy48//sgtt9xS59rCady4MWeccQb33nsvGzdurNe4q0NmZmasJdQ7GRkZB7hlZmayd+/eei8vrVu3LveVroyMDBITE2NSfiFYZVxfo0xpaWmccMIJzJkzh/fee6/0nmfNmkWHDh04//zz60XHoYg3xDGmV69eB80rIbWhd+/e9O7du0p/PXr0YNq0aZWOEJxwwgnRlLYfV199dZ2F7ak9TZs2ZdKkSbGWUUpaWhqTJ0+OtYx6ZcSIEYwYMaL0WBJz58494H3euiynhxr+9aUaIGkz8EWMov/WzE6LFx3xoAFimhbxoCFedMSDhnjREQ8a4kVHPGiIFx2lGsrDG2KPx+PxeGJIQtVePNHilltuiXmrJx40QHzoiAcNEB864kEDxIeOeNAA8aHDa9hHXerwhtjj8Xg8nhjiDbHH4/F4PDHEG2KPx+PxeGKIN8Qej8fj8cQQv2ra4/F4PJ4YUumGHpJSgaeAZKARcKWZVfgNKUkTgdlAKyDBzN6twN9iYKCZlYS5TQWuNbO9NbwHJF1mZo/UNpzK+OWV98W8xWJxsv3KrjjYyW5vUswfBwAlTX+KtQSsSVSzesSktYr9FpxNDtsdawkANEyIfb5o2Tj2zwOgxWHbYy2BVknxkRa39/pnuXvnVjU0fRHwTzPLA04E1lTm2czuNLP1QBZQ9TZK+187oRbGs/QjwbUMx+PxeDyeeqUqQ7wD6COppZmVmNlWSf8h6XRJXSR9r4BbJP1K0mOSOgPjgD9IelJSgqRZkgokLQgL+y5JKyRdDkEvWVKipD9L+qukVyXNcuc6SVou6XlJiyR1DAUiaRiQ6a4/pYJwHpf0n5LelnSTu66VpHmSXpc0PYpp6vF4PB5PtanKEM8GvgRedwatDfAWcALQF1gB9ACygffCrnsImGJmFwDDgU1mdhIwJMzPEwS97IvLifc9MxsIdJCUBvweuAo4C9hv53Uzmwf8y8zyzOyVCsI5AvjQzI4HhrlzE4E7zOzXwFZJB3ydQdI4SSslrfzuo0Pjc3Mej8fjiS8qNcRmtsfMbjWzTOAvwATgAyAT+BVwL9CPYD54TwXBdCUw3phZ+MTJh2a2CyhvMuVD9/droBlwNPCBG3L+sBz/FREeTuj/bZIaABnAnW6+egDQtuzFZvaQmeWaWW6LnpV/Rcnj8Xg8nkio1BBLOkpSQ3e4icDg7iUwnmnAYuBS4F9lLt0DNHD/rwGOd+GFx1fZapvwcwLWEQw/NwB6VuG/IveyYa4BrnE96Vzg+Ur0eDwej8dTJ1S1DjcLeEbSjwTG9VLn/h6QZma7JJXgerxhvA08JukYgiHl0yW9AWwD8iPQeTfwJEFj4AenJZx3JD0H3FODMG8HHpLUjKBhMQb4vCLP7z94deVfivd4PB6PJwIOiveIJSWaWYnrEb8JnBj+6pPH4/F4PAcrB8vOWr9wPep3gL96I+zxeDyeQ4Uqt4hwQ7eh+dPQ6uh1ZnZpxVdVGt5EYLZ737g6/i8BEs2sf5jbY8AkM1tbzTDygMcJ5pohWMm9B5hKsJisIfCQmT1eURiPvBb7oYOfYr9HAABr1+2MtQTS05NiLSFuaNk8PtrTuytarlmP7IqP/Two3hr7rQy2b4+DBwKkpcW+rKamxEcZ+d1pKneKs0pDbGZFQB6ApKVuc49ykZRQZmV0eeHdWVWcdcRsM7sxdCDpLqDAzMZJEsHqb4/H4/F46pUaNxMkDZX0hqS3JJ3m3N6W9N/A3W5Tj/+W9KakSZIelLRK0mXO72OSOku6RNKzkl5yP0nKcht/LJd0fTW0NJP0gtPzgHNr4TbpeMltAJJXweUnmNmTABbwRjnhl75HXPDCQzVNKo/H4/F4qqSmhjiBYHONkwl6yX9w7i2ByWZ2jTt+2cz6AucSvH98AnB5OeF9ZWb5wHqgF8ErRXlmdhxwiqTGVegZB/zNDVs3kXQcwernmS7c8DGR0W7XrUere7Ph7xGfNHRcdS/zeDwej6fa1PQzAmlAC+BVd9zaDetuMrOvwvyFNs/YQLBxxx5J5c2xhvytd2EfDdwjqQnQjTK7aJVDJ+Al9/9KoLMLY75zWx3md7+haY/H4/F44oGaGuItBLtUDTKzvZIamplJKjsvXNFGGmUpu8nGb4G7zGyxpKXOrTIKgRzgIyAXmAUcSbDz18cEvez/qeDatyRdYGZPusbEiWa2pKKILju5/En2nydVDVR4PD934uFTafGgwVMdajo0/RPBtpaLJL1OsOo4mrwIPCjpGaA66x8fBs6XtATYZWZvExjj30pa6PxUtHTwFuAkt8Xlm8AvaqXc4/F4PJ4IOCg29KgJoW00zewnSS8C46r7qpTH4/F4PPXNoTh20RR4UVIS8Ko3wh6Px+OJZw65HrHH4/F4PAcT8bHdiMfj8Xg8P1O8IfZ4PB6PJ4Z4Q+zxeDweTwzxhtjj8Xg8nhjiDbHH4/F4PDHk/wOJ8xiMxu4XPAAAAABJRU5ErkJggg==\n",
      "text/plain": [
       "<Figure size 518.4x50.4 with 15 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAJUAAABECAYAAACIwHfFAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjYuMCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy89olMNAAAACXBIWXMAAAsTAAALEwEAmpwYAAAJkklEQVR4nO2df5BVZRnHP98VkBDvMrkgLDsrCeRYw49yySCLTbGopJqoFGj9se4gGGbjVEY5FYYGEzYlJAOaIGD+SBnEEDETBGXU0YkNyREhEmX5sdvU7iVd4S5Pf5z3wmXZe/fs2bNc2H0/M2f23Pe853mfvefZ57zne84+R2aGxxMnBfl2wNP58EHliR0fVJ7Y8UHliR0fVJ7Y8UHliR0fVJ7Y8UHliR0fVJ7Y8UEVE5ISktZI2iDpJUllkr7Ryj6VGev3SKqVVBWTP+WSZreh/wtxjAs+qOLkamClmZUDlwAfADmDCqjMWP8l8MOOce3k4oMqPt4DRksqMrMUMAm43GWuvpIelfS8pGdcVpsKDHPbh5nZ3g7waZCkFXA0c/3CrW+V9EdJ1ZJGur59JP1J0muSRrVn0G7tctmTyXKgBFgvaT9wG1BqZt8BkHStmb3nTm9XmtliSVe7zHay6UeQJS8CrgG2AMXAp4FCYBFwRVTjPlPFhJkdNrPbzWwY8Afg++ltks4Afi1pIzCD4ACeDA5nrCtjfYeZNQJ7gD4ZbQfNbA9BYEXGB1VMSDpPUnf38QCQBM5wn0cCZ5nZ54Dfc+wAd/RzR/8D+rv1YRntmeOmfRki6SxJxUBDewb1p7/4GAk8Kul9ggxRBSyW9BhwC8FBexp4hyBDALwj6XHgp8BEYDIgScVmdns7/RHwb2C3pGeBHcC+HP3fAe4HhgA3tmtg/5Be50RSBdDbzBae7LH96a8TIulKYCrweF7G95nKEzc+U0Vg/PjxRjDZzccSix9mZslkMuvS1NQUyoeW8EEVgbq6uny7ALTPjzlz5pBIJCguLj5hSSQSVFVFv1vkg6qL8sYbb7BkyRKSyeQJy8aNG9m+fXtk215S6II0NjaycuVKrrvuuha3Dxo0iM2bNzN9+vSjbQUFBcycOZOSkpJW7ftM1QXZvn07qVSK0aNHt7i9pKSEZcuWMXz48KPL6tWr2bRpUyj7PlN1Qfbs2cPYsWPp2bNni9slUVFRcVzbW2+9RU1NTSj7PqhOYxobG5kxY0bo/j179uTOO++kpqaGAQMGtGms4uJi5s2bx65du1iwYEHOvj6oTmMaGhpYt24dN998c6j+N910E9OmTWP//v3079+/9R0ymDRpUtbM1hwfVKcxqVSKysrK0NlqxYoV1NbWUldXx8CBA9s01sCBA0OP4yfqpzGpVIq+ffuG7l9UVERdXR21tbUUFRV1mF8+U0Vk1apVLFq0iG7d4vkKm5qaGDx4MPPnzw+9T21tbZuDqqqqigMHDjB58uQobobC3/uLQFlZmY0aNYp9+/ZRWVnZ+g4hSCaTTJkyhRDH4+jDdkOHDrXq6mp69eoVaozdu3dTXV1NQUEB48aN48wzz4zqrnJt9JkqIg0NDUycOJEJEybEYu/IkSNUVFSQSqVCZ7/CwsLQAQVQWlpKaWlpVBdD4+dUEWloaKCwsF1P3R5HQUEBZ599NslkMjab+cIHVUTq6+tJJBKx2kwkEtTX18dqMx/4018EDh48yM6dO2PNVAA9evRg7dq1jBgxImufMWPGxDpmR+An6hGQVAu8nafh68xsfJ79OOpDS/ig8sSOn1OdZGbNmnVK/BV3pB8+qDyx44PKEzs+qDyx44PKEzv+6s8TOznFT0mFwBPu4yeAvwG7zKzlJ+ZDIKk/cL2Z3dGGfSrN7P5mbeXAODO7rQ123gTSdaAWmtkjkiYB0wlukr4NVJrZoWw2Dm3dn/e/QvX7UL5dAKD7uYkWbyznDCozqwfKISjfl66lJKnAzI5EccTM9gGhA8pRSVA8or3UZtaDkvRxgqIY48zskKRPcaxSiycibZpTSVoqaQHwtKRiSeslvSDpHre9XNJaSU9KelFSb0ljJL3s+l4vKbO62xWSNkraLCmtEr8k6V5JWySNl/RVjlWcuzyLX7e68Z6TVOraZjvb8yUtzfIrfRP4XTozmdkrZvZ+W74Tz4lEmai/aGZfAOqAy83sEiAhaajbfsjMJgBPAZcBXwJuNbPPk5FtJBUAPwAuJciG6XqXHyYorfMV4AYzWw1sNbNyM/tLc2fc6fRSM/sM8DNgpqQBwCddPajMAql9XXBukHQBMIBjp8OsSJoq6VVJr9732PJQX1JXJsoN5dfcz3OAhZL6AIM4Vh3udfczXaVtIXCbK0t4N0FBMIAi4ELgWfe5nyQRnKIOADjbrTEI+LtbfxX4OXBehh9bCAIbTjz97XV+b8s1gJktBhbDqTGnOtWJkqnSc6nJwCp3kF6k5epwAv5jZjcCPwJmZWyrA7YClzkbIyy4FG2pyluuA/kvIH1bvwzYSTDh/phrG55j38eA76Ur4Lky1afGLPg0pj2PvjwHLJP09Vb63eDqifcG5qYbzeyIpN8Af5VkwD+A72ax8YqkVcBdZnbcv8ma2T43X9sMHAKuMbO9bk62ydk9fKJJMLNtkh5yPqSv/nJe2fYYdm7OR2k9nVinktTNzFKuANj5ZvarfPvUVejMD+ndIWk00AR8O9/OdCVCZaq4RVBJPwaWu/LKYfpfC3Qzs/sy2pYCs81sR0gb5cADwC7X9DWC0+JvgY8C3YHFZvZALjv/3Lkj76m916FT45Hj/hde1HbxM002EbQlwgijZjYnzLgdwPJMBV7SXOB5M5vq5lSfzZNfnYpIN5RziJYLgXlOJF3oBMnZkha411NUur5LJQ2RdK2kxyU95RZJGqngdRsvS/pJCF8KJf3Z+XO3azvHTd6fkvSEy1ItMcbMHgRXr9BsY5YxjupUDz38cNu/sC5GlKDKJloWAXeY2S3u8zNOkPwWwRsQxgDXt2DvXTP7MoGuNRx4Eyg3s4sJ3u3S2iX+VOARJ3T2knQxQQ3zRc5uj4y+FU74XNKWX9jMFptZmZmVTbrqqrbs2iWJMlHvQyB8NhctD5jZuxn90uLjXuB1MzvspIPmNBdLPwLcJakXcAHBe1RyMZhAvYdA/BzibDzp2rZk9D3u9OfpGKIE1X+BGuCLZtYkqbuZmaTm8yjLst6c5mLndGCumW1Q8A661nShnQQv7tlGIH7eR/DioWEEGtVwYF2WfTdLmmJmD7o/jEua62DNOX/wEK9TtUJURT0tWq4nuHqKkzXAAkmPEoiZrXEvcJUTOj8ws5cIAmu6gtd2QBbxk0DhHytpA8FdgfPb5bkH6KTip7tZnVbt1wBTw8oXnvbTWcXP3sAaST2AZ31AnVw6Zaby5Bf/jw+e2PFB5YkdH1Se2PFB5YkdH1Se2Pk/EWmhtzDAiFkAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 86.4x50.4 with 5 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAJUAAABECAYAAACIwHfFAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjYuMCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy89olMNAAAACXBIWXMAAAsTAAALEwEAmpwYAAAJWUlEQVR4nO2df2xV5RnHP19WnbJb1GmJduusyNDixCEVmXOuglbGpGzoBGaMDgjGufQPsk032CIZZBDmwjYyNlDEiQEJLO34UetqRYTqHG44yxYXDTbRYW4ZYG4NrCjP/jhv6+Vyf/e0t7bvJzm557znPe/73Hu/95z3fM+5z5GZ4fGEyZBCB+AZeHhReULHi8oTOl5UntDxovKEjheVJ3S8qDyh40XlCR0vKk/oeFGFjKQqSR9IGu6Wr5FkksqT1H1Q0mckjZX0mqS3Qo6jTdJOSfWSzgqr7Ux4UfUO+4Bpbv6bwN5klcxsqZm9A7wBTADeDjmOJ8ysCmgBbk9VSVKoOvCi6h2agUlu/gpgP3CrpO9JGur2HsWS1kkaaWYxM3u/F+PZBzwiqQhA0k73uk7SSuBpSaWSnpO0W9Jve9KZF1Xv0AkclzQB+Jcr2w58HVgDLDezWB/GcwPw7xTr9phZNXAIuNnMrgeGSfp8vp15UfUeO4DfAX90ywasByrNbHsfxXCXpOeAc4H/xpUrbv4V93o+sNntxa4HSvPt1Iuq99hB8IX91S1/CpgLbJJ0dx/F8ISZ3Whm9wPvARdJugC4MK7OSff6baDOjcH2cKrwcqIo3w096TGzDmAOgCSAJcBS4FmCMczTXXUllQGPAV+Q1ATMNbO3Qg5pNbAV2A20J1nfDPxB0jd62pH8TXqesPGHP0/oeFF5QseLKg8mT55sBGdzvT6ZmcVise6pUHEkTGnxosqDQ4cO9VlfixcvZtiwYZSWllJaeupZfl/GkQteVP2cV199lY0bNxKLxYjF+tIvzR8vqn7M2rVr2bJlC2PGjCl0KDnhRdWPaW1tZfny5VRUVBQ6lJzw5mc/5ciRI9TV1bFs2bKUdQ4fPsymTZuoqanhrLPCvbNl69atdHR0JF03a9astNt6UfVTNm/eTDQaZcKECSnrHD16lBkzZtDY2Eh1dXVofUejUWpqapg5c2bS9V5UH1Oi0Si1tbWUlZWlrDNixAhGjRpFe3uyqy4963v06NFs2LAhr+39mKqf0t7eTklJScZ6w4cPJxqNFqTvVPg9VR60tbUxd+5cAMyMsrIyHnrooVDabm9vZ8GCBaxfv57Vq1dnrF9SUsLSpUvZv39/Vu2bGVOmTOG2227rLuvs7GT+/PkcP34cgBdffJHLL788vzeAv6CcF+Xl5bZw4UIAjh07Rm1tLWF9js3NzUybNo0VK1Ywffp0zjvvvMQq3bekVFZWWn19PQ0NDVm3v23bNgDq6uq6y9ra2igvL2fNmjXdZePHj09nZaS/LcbM/JTjNG7cOOvi5MmTNmTIEDtx4oSFQX19vU2dOjVdlaRxZEtDQ4NVV1efUtba2moVFRW5NJP28/Fjqh4iiUgkwvvvh3OLeSwWIxKJhNJWMoqLi0+zCjo6OiguLg6tDy+qECguLg7tEkrYX3AikUjktFjDFrIXVQhEIpGURmGuDIQ9lT/7C4GioiIaGxs5fPhwj9vatWsXY8eODSGq5EQiEQ4cOEBLS0t3WVNTE0OHDg2tD3/2lweS2oG2AnV/yMwmFziO7hiS4UXlCR0/pupjFi1a1C9+xb0ZhxeVJ3S8qDyh40XlCR0vKk/o+LM/T+ikNT8lnQPUu8WxwN+BA2b2nXw7lHQhMMfMluSwzWwzW5tQVgXcZGYLc2jndeCgW1xlZk9JmgXcR3DlvQ2YbWadqdqof+MnBf8VRj552p0LBWFS2fykdyukFZWZvQdUAUjabUFGECQNMbOTaTZN1+a7BMkqcmE2sDZjrcy0d70HAElXEGQ7ucnMOiWNBz4RQj+DmpzGVJkyr7k8kw2StkraIyki6TpJf3F150gql7Te1b9V0i5JLZK6XOKXJK2RtE/SZEk1wJUu+9zNKeJ6wPXXLOlzrmyxa/s3ktaleEu3A7/q2jOZ2ctmdiyXz8RzOvkM1DNlXus0s6kE+ZkmAV8DHjCzG4nb27g8k98HJhLsDX/gVn0aWECQde5eM/sT8JqZVZnZnxODcYfTiWb2ZeCnwI8kXQRcbWY3EKTO6aLEiXOnpMuAi/jocJgSSfMk7ZW0t3Hj37L6kAYz+YgqU+a1Vvf6DkEGt1XAHW7vdE1cOxcAFUAT8AxBQi4RHKKiFiRYPTeLeMqBf7j5vcBI4OK4OPbF1W134qwys67xVcaMcWa22swqzazylplXZxHS4CYfUWXKvBY/kBVwxMy+C/wQWBS37hDwGjDJtXGVBaeiidsntpnIW8BVbr4SeJNgwD3alaX7e+9moFbSGQCSKiWdnaa+Jwt6cutLtpnX7pU0HYgA3f+MNLOTkn4JPCvJgH8C96do42VJdcDDZvZC/Aoze9eN11oIErjebWYH3ZjsBdfuiWSNmtl+SRtcDF1nf2nPbKeN/FneaQsHCwPWp5JUZGYfSJoBjDCznxc6psHCQL5Jb4mkLwEfAncUOpjBRFZ7qrBNUEkPEmTOfSfL+vcARWb2SFzZOmCxmb2RZRtVwOPAAVc0jeCwuAIYBZwBrDazx9O1c6z5yYLv2g9efG2hQwBgxKUjczc/u0hlgiYjG2PUzJZm028v8ES8Ay9pGfC8mc1zY6qvFCiuAUVeF5TTmJargF84k3SVMyQXS1op6RVJs13ddZJGSrpH0hZJO9wkSV+U9LwzTH+cRSznSNrm4vm1KzvfDd53KHjYT1WKza8zsyfB5UE025Wij26f6tFtzbl/YIOMfESVyrS8AFhiZvPd8jPOkPwW8ChwHS6veAJvm9kUAl9rDPA6UGVm1wI3Z3GKPw94yhmdQyVdS5AE//eu3TPj6t7ljM/HcnnD8T7VnFsn5rLpoCSfgfq5BMZnk1se7g4dUTOLfwpUl/l4EGg1sxPOOkgk0Sy9BHhY0lDgMmB4hnguJXDv4SPz8xKCRPRwqvl5yuHP0zvkI6qjwH+AW8zsQ0lnmJlJShxHWYr5RBLNzvuAZWa2U9JuMj/O4k1gHMGTqiqBR4DPAlcSeFRjgMYU27ZIutPMnnQ/jOsTfbBEzp54Z8F9qhGFDiAD+TrqXablcwRnT2GyHVgpaROBmZmJNcBMZ3T+z8xeIhDWffroUR1JzU8Ch/+r7lLTHvr/9/WxYECan+5idZdrvx2Yl6194ek5A9X8jADbJZ0JNHlB9S0Dck/lKSz+jw+e0PGi8oSOF5UndLyoPKHjReUJnf8DuzaE8zC8dCYAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 86.4x50.4 with 5 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Hair-Shaft-Medulla\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAeIAAABECAYAAABd2zbkAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjYuMCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy89olMNAAAACXBIWXMAAAsTAAALEwEAmpwYAAAb80lEQVR4nO2deZhUxbmH349NCYsECAOoICghIiSyXAgIgkFUQJBAiIkbuJHHHTUGUCNIuG4Rl2CuIRcTEIleEeKGERcGEJhBRlllFUeQQTYFR2YYZvvdP87psRlmprune+Y0Sb3Pc57urlOnvl/XqTpfbV1tknA4HA6HwxEMNYIW4HA4HA7HfzLOETscDofDESDOETscDofDESDOETscDofDESDOETscDofDESDOETscDofDESDOETscDofDESDOETscDofDESDOETscDofDESDOEceBmfU3s8VmttTM/mlmTcqIM9PMzooirdFmdkMCNPUzsx2+rsVmdruZ/SyK6542sxfitV+OhqFRXDPdzJab2TIz+3GCNBSaWTP/83+ZmczsjDLi1vNtz43XbhkawvPhlCiuWea/TjSzNP/oH6ft18zs5HLiTTKzflGk19LMPjazPDOrFaOWnmY2y3/f1MyW+HUmYjpmNtDMNofyJVH4+TPFfz/CzOaZ2T/8zzeYWYaZDfZ1LjGz90NlKYH2Yy6f/v26MEH2w8vHI7Gka2a1/bJ5OJrnW4x6FpvZ0LD7s6xU3I5mtsLMPjCzv5uZJdh2LPX0OjPLjPfZ6RxxJTGzHwAPAEMknQ+MA+oEq6qE2ZL6SeoHrAMqdMRmlgK0qSoNkl6PYL8G8Iik84BrgYkJ0rAGuMx//3Mgo5x4PwE+kDQyQXbDCc+Hb2K47nlJPYGBVD4/ZvtlYAXwCyjJ68rwNdAfSI/1QklpQB0z6wxMAqYAxVFeno53f6oEMzsPuAW4P0zTL4HekhYA/SX1BZ4HRiXY/BqCLZ/HlY8YKASGAa8kWo+vKbuCeFsk9ZLUx//cLZG2Y6ynrwMD4jXuHHHlGYR3874FkLQVqGdm7/gt6PvD4k7wW9aToaQX8qKZvWdmz4XFG2JmC/0WaqKc+hjgar9Ff6+ZDfQ1DDGz3/lxxgLTEmTvOMzsJDN73czeNrOXzOv9n2FmqWb2CjBaUqYfvQAoSpDpRXjOA+Ac4BOgkX9/VprZvf65x4Bfm9kUMzvTP/ean2dnJEgLZjbO79ksMrNWZtbYb4GnmtmfwuOG5cdRIN4N4dcAM8zsGeBt326qmf0L6O5r62dmb/n3Kc2/R++b2QIzM0l5kg7GoeFe4FmglaR3/bAnzSzdzMb4Gmaa2XPhdUXSQUlH47BbEe2Bx4GRwBFfwy/w8mShmZ0lqcCPWxev/CSSmMtn6EK/V/iamTVIgI41wGl+ui39srHMzP7HDzuu/spjbwJsR4V5/MnMfhV2T8CrH18k2FYs9fQAXqMkLpwjrjwtgC9Lhf03cL3fgj7HzE7zwxf5veYuZnaqH7ZB0oVAvpn91A/bJ+livBbq8Di0XR0aZsGrZLMl9cdrvY7w44wAXjGzxsAPgG1x2IukYSywQtIlQPjDvBlwuaS/hYU9DBxT2OMgH8jz83eTH3YU6CepBzDAzOri9YZmS7of+C1wB17+J2IoMpQPfwd+5vf6HwAmAJ2BxZIu8G2WxSRgepwazsfLi+WSLgJuAGZIGgjUDItXIGko8AbQ2S8zWb7OePkcaAj8MyzsJeA8YHRYw7OsulJVXAQslPRVKEDSK3h1pr+kT/0HcRpwK7A+wfYrUz7Bc9oPAVeHOgJxcj6w1X9/ABggqTfQ0Mza4fV8y6q/iSb8mdG91LmngDRJLwGYN3S9AUgBviJ+4q2nceEcceX5EmhZKqw9MNsvSGcDoQfJav91Pd8NAYfC1gBnVRBWGcKHeD4MBfq99rZ+5T5N0md4BevPcdiKRoPhDZGD991CrJVU0vs1s7HARkmJnA98C/gLMD9kBnjLzJbg3aPSzrYNsM7XtSEB9kPDf9P5Lg8y8O7vUqCGmc0Brip9oZn9HGgi6R+VtH21maUCjYDXgI/88LZ8V9Y+Dosf+r67S73/fiXth3MV8CZwvX03N7zaz+cdfHcfyqorVcWzwHlmdkl5ESTt9KcIJuI10hJNrOUTvGmwJyVVNHwbDaXLB0ATvAb6YqA33jOuDWXX30RT5nML+CFe4+OlUICk1yV1BHYBlybKNpWop4nAOeLK8xZwVWhoyLwFC3uBX/s3tCuwyo8bmuPqiNczCA/7CbC9grB4KeDYXs9iYDLesBh4lexhYBbwMzP7ZYLshpMJdPLfhy/EKpknNLOLgF5484eJ5C08BxS6F48Cj/qjFp/iPfiO02pmNfEqf6L4nO/ubze8+1tT0gOSrgTuDo9s3oK1W/yjssyWdIGkW/CG+0P5nRmmJby3q3Lex7sYpi5wE54zm4vXIwf4iZ/PrYF9oTD/NbyuVBWFwOV4Ze64xWzmLUgKffds/OHrBBNr+QS4HbjX4l8kVbp8AFwBvOo/w5b79surv9XFVuBF4I/gDZWHnUv0ffmcGOppoohp9aPjOyTtN7M/AG/6lfVrvKGMv/kFpYDvhoH7mtnNwBJJu/y6fbaZvQ/slJRmZu2BJmb2DpCHN29VWa42s97++7/jtfr/T9LleMPT64AO/ve4BsCfC50i6eU47EbSsBA4jJc3pZmGV6lSzWyLpN8kQoSkw8D1AH6+LwCeMbONeEODpXkcmIPnGA6Wo7UyOvb4c0wrfLujgO5m9hBQG3iv1CV/xBt2W2hm30i6jMQxA5hnZtfgDYVGxMxqA//Ce0gtNLN7Ja2M0t7dwHRJR8zsWSAVOAmvjD8F/F1Svn9/SteVbsAjQEczew+4VFJe1N80ApK+NrOrgWUc2wsDb/pptpkV4+XT6ETZDbMfa/kEOARcA7xgZldK2pNASYuA581sWFjYq8Dc0vXXzF7G6zW3M7PHJL1GfIQ/M8LXziDpOTObYGbjgM1mdpd/ahvwTpx2w+3EVE/N7FJgPHCmmc2TNOK4RKPApHjXgThixcwmAcsklX74/ttiZrUkFfoP4uf9lbRJR5jOmng9gt6S4l6M4YiMmc3Eawx+GrQWx7GcKPX3RMX1iB3VxQIzqw98muSVuK2ZzQDqAc85J+xwACdO/T0hcT1ih8PhcDgCxC3WioFLLrlEeItYgjiSSkcyaAhYRzJoSBYdyaAhcB0q1av5T86LJNKQLDoqxDniGDhw4EDQEoDk0JEMGiA5dCSDBkgOHcmgAapfxyuvvEKPHj0C1VAeyaAjGTRA8ugojXPEDkcMSCIZpnPWr1/PxIkTg5bxH8/u3bsZNmwYd911F4MGDQpajuMExTnigEmWB7ujYubPn8+ll15KzZo1qVEjmGrz7rvvkpaWxjXXXEP//v3ZsyeRv1qJnTVr1jB3bkL/J+OEYtGiRQwaNIg6deqwatUqJk2aFLQkxwmKWzUdEM8++yyvvvoqGRkZfPPNN3Tu3JnatWvTpk0bTj/9dLZu3cqePXswM7p168bTTz9d7RolMWPGDPbv309xcTHdu3fnoosuqlKb69evZ/v27Xz00UfcdNNN5OTkUFhYyOrVq7niiiuq1HY4X3/9NWlpaaxYsYIPP/yQtWvX8uijjzJnzhzq1q1bbTokkZ6ezlNPPcXLL7+MmTFp0iQWLlxI586J2HkyNvLy8pg4cSL79u3jhRdeoEOHDowcWRX/lZG87Nq1iylTpjB9+nRmzJjBddddF/oNsMNRKZwjrkYKCwt55plnWLBgAampqTz11FNMmjSJpk2blji7jIwMDh06RNOmTRk7diw5OTmsWbOmyrV9/fXXvP322yW98zfffJOXXnqJLl260K5dOwoLC0lNTa0yR/zmm28yfPhw6tevT+fOnZHElCnfbbJVv359rrzySgYNGsSCBQuqRAPAhg0bmDRpEmlpabRq1YqePXsyatQoLrzwQpo3b15ldssiPT2dUaNGceTIEfr27cuOHTto1apVtdieNm0a999/P7m5uRQVFXHSSSfRo0cPli9fjiQefPBBVq1axbnnnlsteoImNzeXG264gc2bN7N27Vq6dOnCP//5T4YNGxa0NEcFfPDBB/Tp0ydyxKAJDY26I/LRtWtXVYbi4mL17dtXgLp27apZs2Zp06ZNsSYTt47SZGZmauTIkRo6dKhq166tCy+8UFdccYV+9atfCdDWrVtVXFxcpo5EacjIyNDgwYNVu3ZtzZ07V0VFRZKkoqIiffjhhyosLFRBQYGKioqUm5urESNGlGhIpA5JuvvuuwVo7NixWrp0aaToVaIhRFZWllJSUjRjxoySPKkOHfv27dPUqVPVtm1bZWRkKD8/X99++63mz5+vW265RYsXLy63TCRKQxxUmY7U1FQ1b95cb7zxhvLy8gLRECPJoCMwDUVFRbrppptUq1atQHWEUaFvCdy5nUhHrDdx8+bNGjlypOrXr6/WrVtHeqBGolI6iouLtXXrVm3atKnk2Lx5sw4dOqSUlBT96Ec/0u233660tLSYdCSiQH/88ceqW7euJkyYoP3790d1zZEjR0o0JEqH5DVKmjRpoqysrGgvqZLKnZWVpY0bN6pfv3668cYbq1XHxo0b1bhxY3Xp0kXr1q2L5dLAHnQFBQUl5TqSjr1792rLli1lNSQqJD8/X126dNF9990XTfTAH/oPPfRQoDoOHjyoX//614FqWLRokc4880zt3r07UB2SQs9954gTdcR6Ezt27KjLL79cPXr00KpVq2K6tgxi1vHRRx9pyJAhOvnkk9W+ffuSo169egLUq1evyjQO4irQu3fv1gMPPKBt27apQYMGmj59emWSiati5eTk6PHHH9e3334rSdqyZYvOPfdcTZw4sdo0hLNmzRqNHz9eN998swD98Ic/1IABA3To0KFq0fHcc8+V2I4xDxKmoTLk5+frkksuUcOGDdW+ffsKdSxbtkyA6tatqwEDBmj8+PGaM2dORBvFxcW644471Lx580g94RCBPfSzs7N17733qn79+oHpSE1NVZ8+fTRo0KBANHz22WeaMGGCunTpottvvz0UHNg9ycvL09VXX32MhrION0dcRUyZMoUNGzaQkZHBSSedFPmCBLN371769+/P0KFD2bx5M61bty45t3//ft544w2uvfbaal1kUlxczMCBA1m7di2TJ09m+PDhjBkzpsrsTZ06lRdffJGaNWseE56VlUVWVhZPPPEEZ555Jps2bWLAgAH89rdV8S935ZOTk8Pw4cNZunQpI0aM4JxzzmHJkiWcf/751WL/3Xff5Z577mH9+vVMnjy5Wm1HYvny5dx5550Vls/9+/dTq1Ytdu7cySmnnFJhen/5y18YO3Ysv/vd75g1axb5+flcf/319O/fn5SUlDKv2bFjB8OGDWP79u28/vrrgdTjaMjNzWX48OGsW7eOlJSUwFayz5o1i9GjRzN27Fjuu+++QDT8/ve/59ChQ/ziF7/g5z//eSAawFuMu2fPHi6++GLmz5/P888/X/EFkTy1OyrXmurYsaP+8Y9/RB0/CqLSsXTpUjVv3lx16tQJzacmmkq3LLt3766WLVsqLy9PK1euLOmRVlZDRTpWrlwpQPPmzVN6evpxx969e5Wenq5WrVrp7LPPjnm4siINK1euVIsWLdS6dWu1aNFCQ4YMOeb8tGnT1KRJE9WsWVPdu3fXmjVrKmM/oo6KOHLkiABNmTJFu3btqqztuDSU5vPPP9dpp52m1q1bC9D48ePLvHfhx8GDB8vU0bRp05J7EEpv7dq1x9i79tprBejOO+8ssd2sWTPdfPPNevHFFwXol7/8pfbu3VvteREtBQUFat68ubp27ar09HTl5ORUu465c+cqJSVFgF5++eXwU9WmoaioSJ07dxagffv2lT5dbTquuuqqkrwIHb/5zW+O0VDWEbhzO5GO0jdx9+7dGjJkiAYPHnzcAUQ7tBgtURWmzp0769Zbb1VmZqby8/MTaf8YHeVpmDZtmgYOHHhcflxwwQUyM2VnZydMQ1k63n//fQ0cOFApKSm66667IiaUm5sbmndOiIYJEyaoWbNmGjNmjDIzM7V161YBGjBgQEle1KpVq6SSVtJ2RB2R2LJli5o3bx6v7bg0hPPkk0+qdevWGjx4sDIzM7Vjx464dAAaNWqUMjMzlZmZWWZjIz8/X0uWLBGgNm3aaPDgwUpPTxegevXq6bHHHlNhYWGlNVTlQ//gwYO67LLL1K1bN5122mlllaNq0ZGfn68WLVrooYce0pdffhmIBkkaPXq0GjdurAMHDpR1usp1FBUVlSxyXb9+vfbu3asDBw4oMzPzuHUtZR1uaDoOFi5cyKeffspjjz123Llx48ZFHC6Lh4KCAvLyjv1b1q+++orVq1ezaNEiGjVqVGW2yyI/P5+jR4/yxBNPcOONN9KpU6fj4jz55JM0aNCgyjTk5eUxa9YsGjRowIwZM+jdu3fEaxL5m+CcnBwefvhh5syZw6BBg0ruQUZGBl9++WVJvNtuu4327dtz9OhRTj75uP+jr3KOHj3K5s2badeuXbXbDufIkSMUFnp/bvX4449zxx13cNVVV9GiRYu4027Xrh1Tp06lSZMm5capXbs2559/PkuWLCE7O5uuXbvSokULPvjgA7Kzs+nfv/9x0xrxcvjw4VBDgVq1alWq/OXl5fHee++xevVq/vznP3PWWWdVazkKf/Zs2rSJ3Nxc7rzzzmrTUFxcTE5OTsnnvLw8Zs6cyccff1zh/a5KPvnkE+bNm0d6ejodO3YsCY9aTyRP7Y6yW1NffPGFAP3pT3+KrskUP8fo6NChgwDVr1+/5AB06aWXVouO0i3L008/vURPlIta4tYQriM/P7+kl5mAhXExazh48KAA/fjHP64O2+XqiIbQYr2xY8cGpiErK6ukvNSrV08NGzZUQUFBteuoAsrVMHfu3GPqLKBvvvkmpsTDy/kf/vCHSumIlz59+pR8D0CDBw+uVg2TJ08+Jh9r1KihXr16VXRJlZcLQNdcc01UGso6XI+4knz22Wf07t2b2267LRD7u3fv5quvvqJx48aB2A9HEvv27SM3N7dad50KZ8+ePZx66qns2rUrEPs7duygU6dOrF27NhD70ZKXl0dBQQFFRUWBbdUJXv3p2bMnK1asCExDdbNt2zbGjRvHI488AkCHDh3YuXPnMT2oSARdzgF27tzJ9u3badu2LWZGrVrV60a2bdvGzJkzGTVqVLXaLY/s7Gzq1avHzJkzK52Gc8QxcPjw4ZIHx/z58zn11FMD0ZGdnc2hQ4f4/ve/H4h9ODYv9u/fj6RAnHBIR1paWrmrX6tDw9y5c2nZsmUg9sN1RHJsn332GY0aNaoyJxyNBoB58+YFVn+qi9J58c477zBixIiSzy1btmTu3LlkZ2dHnWZlynm09yQaiouL2bFjxzFlXYq8V34iNSxbtqzSTjiROkJs3LiRpk2bxvULFIsmEx0eZrYf2BGQ+QOSLkkWHcmgAQLNi2TQkCw6kkFDsuhIBg3JoiMZNCSLjhINZeEcscPhcDgcAeL+BrEaefDBBwNv9SSDBkgOHcmgAZJDRzJogOTQkQwaIDl0OA3fUZU6nCN2OBwOhyNAnCN2OBwOhyNAnCN2OBwOhyNAnCN2OBwOhyNA3Kpph8PhcDgCpMINPcysIfAiUA84GbhVUkYF8ccDs4EfADUkfVxOvMXAhZIKw8KeAu6WVBTjd8DMrpP0t3jTqYh+lzwaeIslv0Fi972tNNX3z4nl8u3pyZEXeY0DLxYUNCoOWgIAOjl4Hc9eEOHv5qqJnicfCloCp9QIZpc7R/nUaL6tzKdnpKHpa4D5kvoBvYEtFUWW9IikLOBcoEssAiWNjcN5XpegdBwOh8PhqFYiOeJcoKeZNZVUKOlbM7vFzIaYWTsz+9o8HjSz7mY208zOAsYA95jZHDOrYWYzzGyJmf0rLO1HzWyVmV0PXi/ZzGqZ2SQze97M3jOzGf65M81spZm9Zmbvm9kZoUTMbCjQyb9+QDnpzDKz35tZupk94F/3AzN73cxSzex/EpinDofD4XBETSRHPBvYCaT6Dq05sALoBZwHrAI6AJ2B1WHX/RX4o6QrgcuAfZL6AoPD4ryA18sua9PQ1ZIuBFqZWSPgt8AdwHCgWXhESa8D6yX1k/RuOem0ADZI+ikw1D83HnhY0gXAt2bWs7QIMxtjZhlmlrH7i5Xl55LD4XA4HJWkQkcsqUDSZEmdgOeAscA6oBPQHXgC6IM3H1xQTjI/xHPeSAqfRNog6ShQ1sTSBv91N3AK0AZY5w85bygjfnmEpxN6f9jMagJnA4/489X9geN27Jf0V0ndJHVreXqPGMw6HA6HwxEdFTpiM2ttZrX9j/vwHG4RnvNsBCwGrgXWl7q0AAitptkC/NRPL9xeRatcws8ZkIk3/FwTOCdC/PLCS6e5BbjL70l3A16rQI/D4XA4HFVCpL9BPBd42cyO4DnXa/3w1UAjSUfNrBC/xxtGOjDTzDriDSkPMbOlwGFgUCV0Pg7MwWsMHPS1hPOhmb0KTI0hzYeAv5rZKXgNixuAz8uLvPjtcUmwVtjhcFTM74IW4HDEzAnxO2IzqyWp0O8RLwd6h//0yeFwOByOE5UTZWettn6P+kPgeeeEHQ6Hw/HvQqShafyh29D8aWh1dKaka8u/qsL0xgOz/d8bRxN/NFBL0vlhYTOBKZI+jTKNfsAsvLlm8FZyFwBP4S0mqw38VdKs8tJY2bN74EMHNWonxyYWjU5vFLQEts3PjBypGiguCLxYUO+Mk4KWAECnq7oHLYGcL78KWgIAdRp+L2gJScM3n+8LWgJNz2kdtAQAmk6aUeYUZ0RHLOkboB+AmS3zN/coEzOrUWpldFnpPRLJZhUxW9L9oQ9m9iiwRNIYMzO81d8Oh8PhcFQrER1xaczsUrwVEbWAyZLeNrN0vJ7yETNrDBwBfgyk4q2u7gn8WdLfQr1ZvN8QDwFC+7ANBn4CPI23neZrkh6KoOUUvEVcDYE1km43sybAK76GAuDJci7vJWkcgLyJ8qVlpD8Gb3MSxrdpzbCUZqWjOBwOh8MRF7HOEdfA21zjZ3i95Hv88KbAf0u6y//8jqTzgJF4vz/uBVxfRnq7JA0CsvAc9xagn6QewAAzi7RZ6hjg//xh6++ZWQ+81c/T/XTrhMW92t916+/Rftnw3xE7J+xwOByOqiDWHnEjoAnwnv+5mT+su0/SrrB4oc0zvsTbuKPAzMqaTAvFy/LTbgNMNbPvAe0ptYtWGZwJvOW/zwDO8tN4ww9bExb3mKFph8PhcDiSgVgd8SG8XaoullRkZrUlycxKzwuXt5FGaUpvsnET8KikxWa2jMj/87Md6Ap8AnQDZgCn4e38tRGvl72wnGtXmNmVkub4jYnekj4oz1CPtA/d74iTiHZBC3AkJU2DFuA4jhZBCzgBiHVouhhvW8v3zSwVb9VxIlkAPGNmLwP5UcT/X+BXZvYBcFRSOp4zvsnM3vbjlLf15oNAX3+Ly+VA27iUOxwOh8NRCU6IDT1iIbSNpqRiM1sAjIn2p1IOh8PhcFQ3Ma+aPgGoDywwszrAe84JOxwOhyOZ+bfrETscDofDcSJxomxx6XA4HA7HvyXOETscDofDESDOETscDofDESDOETscDofDESDOETscDofDESD/DzoBuP3JGmB9AAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<Figure size 518.4x50.4 with 15 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAeIAAABECAYAAABd2zbkAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjYuMCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy89olMNAAAACXBIWXMAAAsTAAALEwEAmpwYAAAd8ElEQVR4nO2deXhVxf3/Xx9WAVlkCxikKKCBQMOSyhpAlgqEgAL+qm2xVASt9flacC0qKq1bhVJ9rIq4sVgRVAgCYggYpC5FQBCqtai4QdmqyA4Jef/+OOeGS9Z7k5t7r3Zez3Oe3DuZM/O+nzMzn5k5c+aYJBwOh8PhcMSGKrEW4HA4HA7H/zLOETscDofDEUOcI3Y4HA6HI4Y4R+xwOBwORwxxjtjhcDgcjhjiHLHD4XA4HDHEOWKHw+FwOGKIc8QOh8PhcMQQ54gdDofD4YghzhFXADMbYGY5ZvammS0ys0bFxHnOzNrEQFtfM1vt61tlZr3M7O/+//5iZlXNbIiZ/SsQHoE8+5nZF36eb5lZu3Kmk2lm+81sYAXyzzSzM0qId7eZ9QshvVZmtttPLyscLYU0/THoe5m2NrNzzWytX67+ZmZVy5N3IQ1fmNkbZrayuHJamRS6LjlmVj+Ec+4ufP3N7HYz2xlszwroOWhmDfzvz5lZil9mF5rZWDPrGnzt/Guxxq9LTSuSfyEdeYH0zOwnZiYza1VM3EprR0K1R2XkHZT/F75tc8zsCr/uzSshfkTrR5CGEm1QwjkRaTfBOeJyY2ZNgClAhqQ+wK1Ajdiq8jCzxsA9wCWS+gGXAEcC/5f0O0kngXeBlAhnP9fP82bg2iBN4ZS1a4G/VDD/t4HR5ci7MCsl9ZP00wqkES77gWF+udoODI1AmnMlXQTMBq4oLWIF7VVa/v3847typvEU8IsI6fkKuDroe31graTLJD0naUOh+AMk9QXmAL+KkAaATcAI//OlwPoIph0O4doj0syVNAAYgneNG5YSdz+Rrx9Qig0ilH6JOEdcfobiFZ6DAJL+DdQxsyy/53xHUNzf+723qVDQ23/BzLLN7Gk/rJN/3j/MbHIEtM2TdMDXdlDS+4F/+r3OapK+lXS8gnmVRD3ggJ/Xn4A5ZtbMzF7zw+73tSzwf3eWmdXz9f4nAvlvAp4ys0eBFWbW0B8RvgZc6Ofdz8yWm9kSM3vH7/mvMrNlZmZ+Ohf5ve+JEdAEUM3MZpnZJjMbbGZn+vY408yuMbOJ/nUJOKtc4GSE8gZoAGBmD/t2X2tmLf2wd83scWCamXUzs41+Od0Ywfwxs1v9kcZqM2tpZsPM7CEzq2JmKwJ6/LgdzJvdqCtpNxCpzfEzgYyg0dTfgCvM7I9WzGhcUq7/sRbwzwhpAFgNDPA/J/tpP2xmZ0HB7FVgNHpaOxJhyrSHf32e8svNa5WgAUlHgelABkCU60dJNrjfzFYGIvltRHW8uvycma03s2EVydg54vLTHCjsMO4Fxvk952Qza+GHr/Z7b13MLNEP2yppIHDCzLoDHwP9JHUDBplZrQpoOzugzcx+bmZ/N7NpFUgvHMaY2ZvAs8ACP2yRpF8Cvwdm+CPW2/3/jfXttQD4WQR19AFOAG/5o9mrgackDQGCp7JyJQ0HXgU6+73yHUBnPBueD1wEDDSzH0dAV0O8354OXCPpEHAfMAsYBTwciGhmZwODgHJNixdijJmtB64D5gK/9+1+D3CNH6cxcK+kScCdwHA8u7UsJr3y5J9jZs8C/SX1wptR+r2kpUAC8CTwqqQv/XOS8WwzJtDhjSAn8a75SP/73Xgd6zuKi+x3GN4Brge2RFDHCeCY3wZ85Ie9B4zyZyZSgkajxbUjkSIUe4wA9vjlJj3C+QezE+gOEMX6ASXb4PfAF2bWxswuAD71O2aBWdG+eG1buXGOuPz8B8/hBXMBMNfMcoB2QKCyBEajW4BzC4VtAtr44cvNbI1/bkXuQxVok/Q34Jd4jWw0mOs3Fp3wKhBAoCE5H2/KGEn5fs/zId9xX09Re5aHMWb2Bt7ILzMo7/M4ZfPgEd5W/+/OQp/PknRc0mFJecBSoEME9O2VtEfSDl8jwEqgK/CspHwAM6uJN4083s+/osyVlAqsw3Ost5jZWuCPnLL7Hklf+5/rSfpa0mFgW4Ty7wfMBD7ww9bjlX388P+HN/0c4Fa8jtuBCORfHE8B40OJKOlLST2Au4CbIqxjOfAE8Ir/fTHeiDANeDMoXnHtSCQpyx6n1d9KyD9AIvBO0Pdo1I8AJdngeeBy/3jBD/uvXy4OU8FRuXPE5Wc58Eszqwtg3kKK3cAVfoPTFa9nC6fuw3YAPi8UlgJ8CvwGeNDvbX4CBKZGy6ttjJ1aFFOtAmmVl4N409MAgUr7MX5P1+/tdwLq+I77r1TsNweYK+kiSb/FqxyBvLdzyuadg+KrhM8WuLY+vfCuU0U5LQ//77V4o9TxfgMD3ujwr5I+jECewdwP/Alv9iUNb+Qb0BHcuB4ws7PNrDannGUk+JxT1yEV+NQvC3fijc5vC4r7f8Bkq6RFSpL245XJC0uLZ2bVg25VHACORljKcrwOY6C9OOTncwOnGn0ovh2JGCHYo3D9jTjmLbD8HbAkKDhq9aMUG6zB6xil+Z8BGppZC7+OVGjBWCwa6B8Ekvaa2R+ApX4l/QZvmuIZv7Dk4k2lAPQ1s+uANZK+9ut0OzNbBXwp6R3faT5qZh/iTVdVVNvdQKaZ5QN5wAPAafeWzCzVD+9gZtl4CyCOVSRvvA5Ab+AMvNFW8OjhAWC2effP38YbMbcxsxV4CyV2+LoeAYYBw83sCUlPVlATeD3dl83sSiDU++Jp/jU+jrdo4x/lzPsX/tRjEczsHLwp4HS8+4NTzGwp3vTYj8zsd8DDkhaVM+/TkPSxXz7zzGw1p0anhfkD3jTdJ3jXJiJI2mXevfq38cr5r/Ac7iJJM83sJTNL9qPvB64E5pnZL/BsdB1eA3iW39mqKI8AZaXTHG+mKx+vLIyNQL4F+NOv4wBO+XvmA3+U9K+gqKe1I5HUEERp9liCdw/1TbzOQqQWSYHXbvTAc2hP4l37qNcPnyI28GfwPgCqBc0G7MObvu5EobY1XEyK1NoHR6j4TvLvkrJjrcXhKA7zFvPlmVkdIMu/p+uIEmY2FEiW9FCstTg8zFt0ulDSe2VGDhM3InY4HMXRy1+dW5cK9vYd4WFmo4CJnHqsyRFj/Lrwo8pwwuBGxA6Hw+FwxBS3WCsMBg8eLLzFNrE44krHoEGDYq4hxraIBw3xoiMeNMSLjnjQEC864kFDvOgoFTc1HQb79u2LtQQgPnRkZ2czatQoqlevTq1atTh48CBJSUmMHDmSLl26RE1HPNgiHjQA7Nq1i/vuu4+BAwdy4YWlLgSuNOLFFvGgIx40QHzoiAcNED86CuMcsaNcpKSk0KNHD3Jzc8nPz2fnzp0cOHCArl27smvXLhISEmIt8X+OLVu20KJFC26//XY2bNgQ1Q6Rw1EaW7dupV27dlx88cVceOGFzJs3j3379pGenk69evVo0KABzZs3Z/DgwbGWGhOcI3aUi2rVqnHTTUX3NThw4ACjR49m8uTJ/PSnP6Vq1Qrvx+4IkeTkZJYuXcrdd99N165dyc7OZsCAAWWf+APk3//+92m/feDAgdxyyy1RLY/bt2/nyy+/pGXLlnz00UckJSUFP54UNYJtUaNGDW677Tb69u0bVQ1t27blz3/+M6+99hpTp07lkksuISkpiaysLObPn09SUhLbtm3j+eef58wzzyQ/P5+LLrqIOnXqRFVnzJDkjhCPNm3aaNmyZXrnnXcUAwp0dO3aNRb5n6ajJA2HDx/WpEmTdM455ygxMVEzZszQ0aNHK0VDQMeqVau0f//+SOcRloYYUqyOGTNmqE6dOtq4caMk6dChQ1q5cqVyc3OLTeTjjz9Wr1691LVrVyUnJ+vOO+8Mx6ZxZ4vzzz9f2dnZys7O1ksvvaS2bduqRYsW6tixo5KSkvTwww9XuoYGDRoIUPXq1QXo2WefLRJ56dKl6tatm2666Sb985//rBQd5513XoEtpk2bpqpVq6pDhw7q2LGjOnTooPT0dL366qtatmyZXn/9dZ04cSLiGkIpFw899JAAmZmSk5OVlJSkAwcOVKqOjRs3atmyZVq/fn0k8glJQ3FHzJ3b9+moV6+ehgwZopo1a+qiiy5SRkaGLr/8cu3Zsyfci1Ie4qqxK0vDkSNH9OKLLwpQu3btSnQAFdEgidq1awtQixYtNHbsWB08eFDHjx+PdH6lamjSpIkyMjI0evRoXXbZZcrIyFBGRoauvvpqHT9+PGo6Cl+Txx9/XAkJCbr33ns1btw4AXrwwQeL2ObFF1/UmWeeqSuvvFI///nPNXPmTDVr1kxnn322JkyYEErDXKChU6dOuuWWW6Jh/1J1FLbFkSNHtHnzZm3evFmzZs1Su3btlJ+fX+kaXnjhBc2YMUPz589XSkqKjh8/ruPHj+vkyZP66quvBGjixInq27evmjdvruPHj0fKEZZoi507dxbYYtOmTZowYYKGDBmiIUOGKDExUR06dCgowxkZGRo1apS2bdsWUQ0lcfToUR09elQnT57UiBEjNHjwYB07duy08pSXl3eaHcPV8eabb+q3v/2ttmzZoho1amjw4MGqUqWKrrvuOh07diycdMPBOeJIHYHC9NlnnykzM1OZmZkaOXKkgGiMyL5XjjjA+++/r/bt22vKlCnavn27JGnr1q3KycnRunXrytsYFtiidevW2rFjh5YvX660tLQiI5Cvv/5an3zySXnyCFlDzZo1tXjxYrVt21YjRowoKBupqamqVq2asrKylJOTo127dlWqjsLXJC8vT5mZmWrYsKEaNGigVatWFdhm9uzZysnJ0cqVK1W/fn3NmTPntGuRlZUlQK1atVKDBg3Kcg6nGhR/leiUKVOUk5OjTZs2SfLKwaeffhrhn16yjtLK58mTJ9W6dWtt2LBBe/fuVU5OjnJycrR582Z9+eWXlaLh2LFjOuOMM1StWjUBat68ubp06aLhw4cXaGrcuLGqVKlS0GHasGFDxHWUxf79+wvKb+AIdOJuvfXWqGgI8N1336l9+/YFK4/nzZun1atXKyEhoSAsLS2t4PoFylppOgL2DRwPPfSQJOmjjz5S69atZWYC1Lt3b+Xk5ERqRH6ahuKOmDu379NRUmH6yU9+oueee67UqxABCnScd955WrhwoZYsWVJZvfoydYRTsVavXq1u3boJ0IwZMwSoT58+ql69um699Vbt27evXBpK0/H000+rbdu2BRVuzpw5WrhwYZFjy5Yt4eZdREOXLl1KjHTzzTcrLS1NSUlJSkxM1MGDB8ubX5k6Qr0ms2bNUlpamvr06aO0tDRNnDix1Pg9e/bUpEmTtHDhQr300ks6fPhwqRpWr16tPn36qE+fPgI0fvz4guvwxRdfhP8LQydkW1x55ZW6/PLL1aZNG7Vp00ZNmzYVoKpVq1akTISkIT8/XxMmTNDw4cOLLQ9ZWVlKS0uTmemuu+46rbwuXbo01Dof0Y77tm3bVK1aNd1///165ZVXQr3dFDENTzzxhNLS0pSWlqZrr71W+fn5Onz4sC655JLTytoDDzygNWvWlKijfv36ZeZ15MgRXXrppUpMTFT37t2LbTf27t0b7k8o1be4DT3CIDU1VevXF31v9x/+8AeeeeYZXn75ZTp16kSVKpWzH3rgw1lnnaUBAwawePFiXn75ZUaMGAHA559/zt69ewNxaNOmUvbKNyjZFqVx1113sXXrVtLT07nqqqt4/fXXufrqq0lLS+Phhx+mSZMmYWkoS8eJEyeYNGkSkti1a1eRhTK7du1i27ZtfPXVV9SoUSOs3xKqhgB5eXmkpKQwYMAAxowZU2yc5ORkateuXak6ysOqVat47LHHMDPeeOMNxo0bxwMPPBBczkvUMH/+fBYuXMjPfvYzFixYwLFjx5g1axZVqlQpsrJeEps3byY3N5dzzjmHZs2ahSs1ZFtkZWUxc+ZMGjVqxCOPPMKRI0d4+umn+eCDD8jOzmbJklPvHEhMTOTss0N+MVjErkdmZiZz5sw5rdy+/PLLLFiwgMsuK/Nd9REvFwsWLODFF1/klVdeYdmyZQwdWuZW05VeNoOZP38+CxYsYNGiRXz33XfUqxd458wpHe3atdNHH31UfAKF+O9//8vEiRM5cuTIaeHr1q2jZ8+e3HjjjVSrVo2UlJRQ2vzSV+mV5andUXav7ujRowU9spUrV5bcJ6oYRXRMmTJFHTt2lOT1smvXrq3k5GR16dJFQGXdn4xYL1vypiwbNWqk9PT0sDVUVEdeXp4aNmyop556qjynh60hOztbnTt3VmpqapGjZs2amjp1alR0VIQVK1YUV85D0rBt2zYlJiaqRYsWAnTkyJHT/r9u3ToBOvvss9WjR4/yyKuwLXJzc9W/f/+C63LOOeeoc+fOUdVQGtOnTxegb775JmY6xo8fr8ceeyyUqFEtmwH69++vZcuWVZqOTZs2qUuXLkpNTRWgVatWhXJaqb4l5s7t+3SUdRFvvvlm1a9fX/379y9u+q6iFNFx+PDhgvsZnTt3Vr169QqmrZKSknTBBRcUTOcEju7du2vFihUV1hHJihVYtNK9e3e99NJLIWuIhI5HH31U1atX11VXXRXuqRGt3K+88ooATZs2LaY6QiFQzgNlKhwNBw4cKJimbtu27Wlls1GjRrrmmmu0f//+gnId+N+gQYM0ZcoUpaam6vrrry8p+YjbIlDHevTooTFjxoRySqVfj969e6tVq1ZKS0srbd1BpemYOnWqUlJSQokaE0c8efJk3XXXXVHRceONN6pBgwbq0aOH3nrrrdKilupb3HPEEeT2229n2LBhjBw5kn/961+VvqFC7dq1+eCDD/j2228BaNasWcE01vLly/nqq6Jvr5s5cybLly/n4osvrlRt4dCiRQvWr1/Pk08+yauvvsqoUaPKPilCXHXVVSQlJTFw4EAef/zx8kxRR4SMjAymT5/OvHnzuPHGG2OiIVQC5bw81K1bl82bN3P48GFyc3OL/D85OZn69euzadMmvvvuu4Lw9PR0Vq5cyX333cfkyZN55JFHovJMbqCO7d27lwEDBjBz5kxq1apV6fmWxgsvvMBnn33G+PHjee+998p9LcrLNddcw5QpUzh58mRc7hPQoUMHFi1aFJW87rjjDoYPH860adNYuXIlPXv2LF9CZXlqd4Tfmxo9erQSExM1bty4kOKHSER6dUuXLlXNmjUrMm1daT3ctWvXCtDgwYNLPRQhWwTTtGnTYp/zLIWIa/jmm29Ut27dcBfgxWTUEW0N6enpBdPZderUUc+ePXXZZZcVtlWl6mjVqpU6depUpDwOGzZMhw4dioqGYG644QY1btxYI0eOLO5xsUrV0bhxY3Xt2lVr164tLVpMyuaGDRtOa0eioWPOnDmqWrWqPvzww5KiuBFxtJk+fTqbN29m+PDhzJw5M656jYMGDSIvL48dO3Zw7rnnxlrOafTo0YOsrCzy8vKinvcVV1zBxo0bGTt2bNTzDnDWWWdx8uRJDh06RN26dWOmIx559NFHueOOO6hVqxZvvfUWO3fuZOjQoTzxxBM0bNgwKhqWLVvGF198UST80ksvZfv27XTo0CEqOgJMnjyZiy++mKFDh7Jnz55wFpRVmNWrV3PDDTewZs0aevfuHbV8Q6Fz585kZ2dz4sSJqOU5evRopk2bxrp162jXrl3Y5ztHXAm0bNmSli1b0rhxY/bt2xdX+y7XqFGDLl26sHv37rhzxFWrVmXQoEExybtbt24sXrw4JnkHk5CQwO7du50jLkSrVq1o1aoV4O1znpKSwvnnn8/u3buj5ojbt29P+/bti4R3796d3bt3R90RN23alCFDhtCxY0d2794dVUfcsWNHhg8fzmeffRa1PEPFzKK+tWutWrUYNGgQu3fvLtf5zhFXIvXr12fRokX8+Mc/rnBa5b73UAxNmjQhMzOT/Pz8mOqIJxISEnj33Xd5++23Q4pfWXZo2LAhixYtolevXjHV8X2gUaNGLFq0qGCNRKxskZCQwJIlS6hVq1ZMNDRp0oTFixdz9OjRgrBo6EhISGD27Nkl1pn/tbKZkJBAdnZ2sTMEZdnCPUccBma2Fyg6NxUd9kkaHC864kEDxNQW8aAhXnTEg4Z40REPGuJFRzxoiBcdBRqKwzlih8PhcDhiSKVsAeUonnvuuSfmvZ540ADxoSMeNEB86IgHDRAfOuJBA8SHDqfhFJWpwzlih8PhcDhiiHPEDofD4XDEEOeIHQ6Hw+GIIc4ROxwOh8MRQ9yqaYfD4XA4YkipG3qYWT3gBaAOcAZwvaQSXyppZrcBc4EmQBVJG0uIlwMMlJQXFPYX4EZJJ8P8DZjZVZKeqWg6pfHTblNj3mP5T6/42G0pL+xX5kaeo4kRvbzlRtXD3xQl4lT+uw9C4qyEA7GWgFnMqykADWsfLTtSJdPkjEOxlgBAYq39sZZAev3NsZYAQP9zPy62tpY1NX0l8IqkfkBv4OPSIkt6QNIOoBMQ1quHJP2uAs7zqgil43A4HA5HVCnLER8BephZY0l5kg6a2W/NLMPM2prZN+Zxj5ldaGbPmVkbYAJws5k9b2ZVzOwpM1tjZq8Fpf2gmb1nZuPAGyWbWTUzu9vM5phZtpk95f+vtZn9w8wyzWyVmbUKJGJmw4GO/vmDSkhntpndaWbvmtkU/7wmZrbEzN4ws8ciaFOHw+FwOEKmLEc8F/gSeMN3aM2At4GeQC/gPaA90Bl4P+i8J4GHJP0CGAHskdQXSA+KMw9vlP2rYvJ9X9JAoKWZNQBuAm4ARgJNgyNKWgJskdRP0soS0mkObJXUHRju/+824H5JFwEHzaxHYRFmNsHM1pvZ+q/3lDgj73A4HA5HuSnVEUvKlTRVUkfgaeB3wAdAR+BC4M9AGt794KJv+fY4H895Iyn4htpWSceB4m6ybfX/7gTqA+cCH/hTzluLiV8SwekEPh8ys6pAO+AB/371AKDIq0skPSkpVVJqi6apYWTrcDgcDkdolOqIzexHZlbd/7oHz+GexHOeDYAc4NfAlkKn5gKBl/B+DHT30wvOr7RVFcH/M2A73vRzVSC5jPglhRdO82Ngkj+STgUyS9HjcDgcDkelUNZrEDsBC8zsKJ5z/bUf/j7QQNJxM8vDH/EG8S7wnJl1wJtSzjCzN4FDwNBy6JwGPI/XGfjW1xLMOjNbDEwPI837gCfNrD5ex+Jq4POSImf9Y0qcrE11OBwOxw+J78VzxGZWTVKePyJ+C+gd/OiTw+FwOBzfV74vO2ud54+o1wFznBN2OBwOxw+Fsqam8aduA/dPA6ujt0v6dclnlZrebcBc/3njUOKPBapJ6hMU9hzwR0mfhJhGP2A23r1m8FZy5wJ/wVtMVh14UtLsktL4zZ++jfnUwbmt68VaAgCf/Ht/rCVQu06NWEsA4IxaZVahSqdhw/iwxTffnIi1BL767L+xlhA3VK1etexIUaBps9i3W00Tzoi1BABuGVW12FucZbYikr4D+gGY2d/9zT2KxcyqFFoZXVx6D5SVZyUxV9IdgS9m9iCwRtIEMzO81d8Oh8PhcESVsLvzZjYMuMU/d6qkFWb2Lt5I+aiZNQSOAj8G3sBbXd0D+KukZwKjWbxniDOAWn7S6UAK8DDedpqZku4rQ0t9vEVc9YBNkv7PzBoBL/kacoEZJZzeU9KtAPJulL9ZTPoT8DYnoc/IP9O+29hSbeNwOBwOR7iEe4+4Ct7mGv3xRsk3++GNgXslTfK/Z0nqBVyG9/xxT2BcMel9LWkosAPPcX8M9JPUDRhkZrWKOSeYCcCL/rR1bTPrhrf6eaafbvCc3Rh/161nQ/2xwc8ROyfscDgcjsog3BFxA6ARkO1/b+pP6+6R9HVQvMDmGf/B27gj14rfjT0Qb4ef9rnAdDOrDVxAoV20iqE1sNz/vB5o46fxqh+2KSjuaVPTDofD4XDEA+E64v14u1RdLOmkmVWXJDMrfF+4pI00ClN4k43fAA9KyjGzv1P2e2U+BboC/wRSgaeAFng7f32IN8p+vYRz3zazX0h63u9M9Ja0tqSMHr/lLPcccQGNYi3AEZeUNYEVDVrEWoDDETbhTk3n421rucrM3sBbdRxJlgGPmtkCIJQlmLOAy81sLXBc0rt4zvg3ZrbCj1PS1pv3AH39LS7fAs6rkHKHw+FwOMrB92JDj3AIbKMpKd/MlgETQn1UyuFwOByOaBP7hyAjz5nAMjOrAWQ7J+xwOByOeOYHNyJ2OBwOh+P7xPdli0uHw+FwOH6QOEfscDgcDkcMcY7Y4XA4HI4Y4hyxw+FwOBwxxDlih8PhcDhiyP8H4iLGPv2tfZEAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 518.4x50.4 with 15 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAJUAAABECAYAAACIwHfFAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjYuMCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy89olMNAAAACXBIWXMAAAsTAAALEwEAmpwYAAALAUlEQVR4nO2dfXBU1RmHn1+IDCAmfEoTmMhXiy3jB0iLAdqmCi1SPhwKgnS0FjFWqs60E7SlHaYQMtahlha0DCgaQMQwmlFASxEQVAhBnDoF6sioQJWPwZTCBKmQyNs/7tmYrLvZzeYusXCemTPZvfc95/x2895zzn3vve/KzPB4wiSjpQV4Ljy8U3lCxzuVJ3S8U3lCxzuVJ3S8U3lCxzuVJ3S8U3lCxzuVJ3S8U4WMpAJJByW9KukVSZ1j2PSU9LR7/UaadWxx5X5JNyRpv7w5fXunSg8rzOx7wDLg1hbWUWBmBcA/gLhOFWV/e3M69U6VXjoASJrlRoDNknpG2WRKKpW0S9LoNGopBG6TtEnSTEk3OW1jJD3gbG6V9Lqk5h0IZuZLiAUoAA4Cu4B/AtcAi92+rwOLgZ7A027b+0AecCmwLQ06trjyADDX7fsa8IR7XQr0BtoDlzgdFUDXVPv2I1V6WGFmg4CdwM1AgaQtwCIgK8r232b2LzP7BPgsDToi09/OyEYz2wf0ltQW6GFmH5jZKTOrcTpeA76aaqfeqdLLQ8D1wIZ6/9zo9UonST0ktQNapVFLTVT7W4A5wGYASVnubyvgm8CBVDvKTLWiJzFm9q6kS4CjbqQyYBWwoZ5ZFfA74FqCf3KY3CZpmHv9FDBUUpmZTQKeI1i8f8Ptv0VSIXAOWGVmh1PtVOZv0vOEjJ/+PKHjncoTOt6pUmDkyJFGsD6KW8zMqqurv1BqamoS1k1QmqQjutTW1tqZM2cMsNOnT9fpimyL1h5Hb6N4p0qBqqqqhDZz5swhKyuL3NzcupKVlcXAgQPPq45oJkyYQJs2baioqODSSy8lNzeXbt26cdlll/HZZ0FEY9myZWRlZZGdnU3//v2bLqylg4X/j+W6666zeGzcuNG6du1qgG3durXBvpqaGsvIyLBOnTrZzJkz47aRgKR0RPPYY49Zly5dDLArrrjCACsqKqrb369fP+vQoYPl5eUZYMuXL7fa2lpr3bq1dezY0TZs2BBTQ6ziQwohsG/fPsrKygBYvXo1U6ZMoaioiO7duzewy8zM5NixY+zYsYNx48bRpk2bun35+fkMHz48dG0HDhxgxYoVLFy4kOLiYsaPH0/btm05fvw4OTk5dXaVlZWcOHECAEn06NGDjIwMjhw5QnFxMSUlJYwYMSKpPn1IIQUGDRpku3btqntfXFxMaWkpU6ZMQRLTpk0jLy8vbv1z584xb948Tp06BcCePXs4fPgwlZWVyXSveDpiMXfuXJ555hkmT55MUVER7dq1S6aPBnz44Yfk5eVRVVVF586dG2iISaKhzJfE0999991n8+fPj55xkub48eMG2Pjx45Mxj6ljz549NmDAABswYIBNmjSpbvvEiRNt5cqVKWuLcP3111u3bt1s//79DTTEKn6hHgJVVVV07do15fodO3Zk9+7dlJeXc/r06ZTa2L59O+3bt2fhwoWUlZVx9uxZIJiar7zyypS1RSgvLycnJ4c333wzsXEir/Ml/ki1Zs0au+uuuywjI8PWr1+f7EEfl549e9rYsWOturq6MbMGOj755BO79957rXv37lZSUmJmZjk5OTZ69GgrLCw0wE6cONFsbWZmM2bMsF69ejXQEKv4hXozWLp0KZJYvHgxw4YNS1whAaWlpYwcOZK9e/cyePDgpOq88847LFmyhAULFjBmzBgAVq5cyb59+wAoKysjOzu72doApk+fTp8+fRIbJvI6X+KPVIMHD7Zt27YlfaQnw9ixY628vLwxkwY61qxZY6NGjQpVQxI0+v34NVUKHD16lFmzZlFZWdngtDwMcnJyWLt2bdL269atC11Dc/FOlQKHDh0iMzOTefPmNRo6SIWhQ4dSUVGRtH1FRQX5+fmhamgufk2VAm3btmXWrFlpaTs/P7/uzC0Zzpw5w5AhQ9KiJVX8SJUCrVql7wbN7OxsTp48mbT9yZMnQ1uIh4V3qhQ4H05lltyVji+jU/npLwXS6VStW7emtraWTZs2xbykUn+qq66u5tNPP03p0ks68df+UkDSxwSPP7UEVWY2soV11GmIhXcqT+j4NdV5Zvbs2V+KozidOrxTeULHO5UndLxTeULHO5UndPzZnyd0Gg1+SsoGXnRvBwB/B/ab2U9T7VDSV4A7zaykCXWmmtmTUdsKgOFm9tsmtPMucMS9XWRmZS4X0z0E910fBKaaWdyLb/MzftTiR+Htq9Jz3bGpdJ50Tcx71Rt1KjM7SZDnCElvWJC1BEkZZnYuFSFmdhRI2qEcU4EnE1ol5uPIZwCQ1B+YQuCcZyV9i/RmXrkoaNKaymV8exRYLynX5bV8Q9Jf3P4CSX+VtFbSNkntJQ2RVOls74zKdzla0muStkuKRIl3SHpc0tuSRkoaC1zlMtHFfEZI0oOuv82S8ty2ua7thZJK43ykCcCfIyOTme00s/825TvxfJFUFurbzOz7BClwRpjZMCBLUiRJ1lkzGwO8DNwI3AQ8aEEOzLrRRlIGUESQh7IAmOF2dQJ+A/wQuNvM1gC7Lcjv9Eq0GDed3mBmQ4FZwK8l5QADzew7QP1ErV3rJVbtB+Tw+XQYF0mFLn3irgrbn9SXdDGTilO95f52Bp5zeZeGAblu+x739xBBzstFBLmPniZIphWhC0G6wo0E+ZpyJIlgijpmZpH6iehJkGcJgpSIfYEr6ul4u57tx845C8wssr7KJQFmtsTMBpnZoHz1SkLSxU0qThVZS00BXnBrlG18/oBh/YWsgP+Y2XSCnJOz6+2rAnYDN7o2rrHgVDS6fnSb0RwgyKsJMIggh+ZBPk/mdXUjdZ8D7neJyZA0yKUs9DSD5tz6shlYLunmBHZ3SxpPkKj04chGMzsn6Y/AJklGkHT153Ha2CnpBeARM3u9/g4zO+rWa9uBs8BPzOyIW5O97tqtidWome2VtMppiJz9NXpm+4tzzzf+dK7nwo1TSco0s1pJk4DeZvZQS2u6WLiQb9IrkZRPkPH3lpYWczGR1EgVdhBU0q8I0jEfStL+DiDTzJ6ot62UIC/4e0m2UUDwCwyR07dxBNPinwjyil8CLDGzZY2188H777X40J77zsaWlgBAm9E/a3rwM0K8IGgskgmMmtnvk+k3DayoH4GX9DCw1cwK3Zrq2y2k64IipQvKjQQtFwF/cEHSRS4gOVfSo5LekjTV2ZZK6ivpDknPS3rZFUm6VtJWFzCdmYSWbEnrnJ4Fbltnt3h/WdKLbpSKxRAzWwkuJ6HZa3H6qItTrXr22aZ/YRcZqThVvKBlF6DEzH7p3m9wAcmJwFJgCHBnjPY+MrNRBHGtq4F3gQIzGwyMSOIUvxAoc4HOdpIGA9MIfrpjFNC6nu1tLvD5VFM+cP041a2TJzel6kVJKgv1DgSBz8jEfrmbOo6Z2Uf17CLBxyPAHjOrcaGDaKKDpb2ARxT8AkI/4PIEevoQRO/h8+BnLyDy7Pjb9WwbTH+e9JCKU50ADgM/MLPPJF1iZiYpeh1lcV5HEx3svAd42My2KPgtvERxofeB64C9BMHPJ4AewFUEMaqrgb/Fqbtd0o/NbKU7MIZFx8Gi6d2nb8vHqfr0bWkFjZJqRD0StHyV4OwpTF4CHpW0miCYmYjHgcku0HnGzHYQONY9ktY7m5jBT4II/3fdpaZtBL8m5WkmF2Tw012sjkTtXwIKkw1feJrPhRr8bA+8JKk1sNE71PnlghypPC2Lf/DBEzreqTyh453KEzreqTyh453KEzr/A121rWljUKYGAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<Figure size 86.4x50.4 with 5 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAJUAAABECAYAAACIwHfFAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjYuMCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy89olMNAAAACXBIWXMAAAsTAAALEwEAmpwYAAAKQ0lEQVR4nO2de3BU1R3HP99FQhocZZKAhaltKqJQJdQSmkbQpj4i2Arig4qZthQcolSxaCm0QGeY4rS2UKGFYEUs2DKtLZlRqGkIIs9kVHCEqlBj5THDwzE8KjJICOTXP/ZsWEI22ezeJQLnM3Nn7z333HN+u/e755773bPnyszweIIk1N4BeM4/vKg8geNF5QkcLypP4HhReQLHi8oTOF5UnsDxovIEjheVJ3C8qFKEpEJJJyR1c9sDJJmknGbydpZUJekfKYjjfUn3ufXRUem3S3pL0hi33V3Sp5KuTLZOL6rUshkY5taHA5ti5OsHrDeze4OsXFI/YANwh0saHbX7TmCkmS102z8GXg+iXi+q1PIqcLNbvwZ4F+giaa2k1yX93O37DTBS0oyA678LKAUyJA0D+kpaI2kaMBRYLGmQpGzgEmBnEJV6UaWW48AxSd8Atrm0OqDQzPKBWyV9DpgK/NnMpgZc/3VmthGoAE4Ab5tZoZn90qUVm9kGwq3U3KAqvSiogjwxKQeeBsYC4wAB5ZIygKuBbqmo1PWN+kqqADoBNTHydQEuN7N3JQVStxdV6ikHbgM2uu0ngSfNbI2kDYRFlgruAh4ws1UAkpYBzY1zuhq4yomvL/AF4JZkKvaXvxRjZkfMbIydGrj2MjBX0t8JXx5TxbeB6qjtrcAJSS9KuiEqvtfNrMDMBgMrgQeTrVh+kJ4naHxL5QkcLypP4HhRJcDgwYONcKe3PZZ2iePo0aNWX19vn3zySav9JS+qBNi/f397hwCcnTg2b95McXExnTt3Ji0tja5du7Z6jBeVp1kmTJhAbm4uAwYMIBQKUVFRwbx58zh27Firx3qf6hymvr6+cb2mpobdu3dz7bXX0q1bcn5qRUUFCxcuZMWKFWRnZ9OrV682He9FdQ6zdetWli1bxtGjRxk5ciS9evXi5MmTPPXUU6SlpVFUVEQo1LaL0Z49exgyZAjz58+noKAgobi8T5UAeXl5tmlTrAEHKafRge/atasVFBRgZhQUFPD444+Tnp4OQIcOHSgtLWXo0KEcPHiQ3r17tyqwhoYGpk6dyq5du1iyZElcMTSLmfmljUv//v2tHWkxjsWLF9ukSZNs3rx5p929VVRUtFpwZWWlAVZeXh53DM0t7X6CzsUlIyPD+vfvb/n5+XbkyJHWTkDQxC3uRYsWGWCPPPKIpaen2969e2PmffTRR61Dhw42ZcqUNsXQ3NLuJ+hcXPr06WMbN260Hj162Nq1a62hoSGeE5E0dXV1Fh1Ha6I6fvy41dTU2JEjRyw3N9fKysrOyFNfX291dXWWnp5uy5cvj/dL4kUV9BI5mePHjzfASkpK4jkRSVNUVGTNxREPEydOtJ49e56WVl1d3Xh5zM7ObsuXo8XPx/tUSTBnzhxWrVrFtm3bWs8cADt37kz42AkTJnD48OHT0rZu3cqoUaMwM2pra/HjqT4j5OTksG7dOqZNm9aYFgqFePjhh+Nyn+Ohurqa8vJyamqaHWcXF5dddhm1tbWUlZVx9913A1BWVkZ+fn4gMUbjW6okycnJYdasWaSlpTUus2fPpqqqKrA6FixYwMqVKyktLU24jFAoxLhx45g5c2ZjWmVlJUVFRUGEeBq+pUqSUCjEY489dlra9u3bOXjwYGB1HDp0iMmTJzN8+PCkyikpKaG4uBgIe1IAeXl5ScfXFN9SpYCsrCwOHDgQWHkHDhwgKysr6XIyMzMb4zp8+DAZGRl07Ngx6XKb4luqFJCZmcnq1asZOHBgIOVt2bKFzMzMpMvJyspi3759VFVVsWPHDjp16hRAdGfif6ZJAEm1wK52qn6/hceTt2ccjTE0hxeVJ3B8n+osM3369M/EtziVcXhReQLHi8oTOF5UnsDxovIEjr/78wROi+anpEuBl9zmdcBbwA4z+2GiFUr6PDDGzJ5owzGjzey5JmmFwC3Whul3JL0H7HOb883sBUkjgYcID5HdBYw2s5hzHNzz9FXt/i0cUTis9UxngRG9f9vssIYWRWVmHwOFAJI2mFlkPWRmDYkEYmYfAnELyjEaeK7VXK1TG3kPAJKuAe4nLM7jkr4OdAignguaNvWpJC2SNBeokNRD0mpJGySVuv2Fkv4labmbw/JiSde7WeNWSxojKUfSX1z+70haJ6laUsQlfk3SAkmbJQ2WNJRTM8DdGiOuSa6+VyV90aXNcGX/QdKiGG/pHmBOpGUyszfM7NO2fCaeM0mko15lZkXAfuBWMxsEXCIp8uew42Z2B+F5mW4GhgCTzOxbRLU2kkLAT4CbCLeGE92uTGAK4alwSsxsGadmgFvZNBh3Ob3JzAYCvwB+Jqk78DUzu5HwnJcRujpxrpF0NdCdU5fDmEgaK2mTpE3b138c14d0IZOIqN50r1nAUklrgEFAD5f+jnvdA3QB5gMjXOs0IKqcbKAP8ApQCXRXeOhhrZl9ZGaR41sjB/i3W98EXAl8KSqOzVF5a504C80s0r/qQSuY2TNmlmdmeVfccGkcIV3YJCKqSF/qfuBF10ep4tR/waI7sgIOmdk44KfA9Kh9+4G3gZtdGf0sfCva9PimZTZlJ+HZfQHygA8Id7i/4tJyWzh2KTBeUkcASXluDk5PEiQz9OVV4HlJd7aSr0TSXcDFhKcmBMDMGiT9DlglyQjP9PajGGW8IelFYJaZrY/eYWYfuv5aNeGZ6X5gZvtcn2y9K7f+zCLBwvNc/tXFELn7a/HOdumDNamaTvG84bz1qSRdZGYnJH0XuMLMftXeMV0onM+D9J6QVACcBEa0dzAXEnG1VEGboJImE543fE+c+UcBF5nZs1Fpi4AZZvbfOMsoBBYDO1zSMMKXxdnAVUBH4BkzW9xSOVve/6jdm/bLj/6nvUMAILPfjW03PyPEMkGbIx5j1Mx+HU+9KeC0CfAlPQmsNbOxrk91Q+xDPfGS0A/KLZiW84GZziSd7wzJGZLmSnpT7oE7bv+VkkZJKpNU7hZJ+qrOfMxGS7FcKumfLp7fu7Qs13kvl/SSa6Wa43ozWwLg/nm7LkYdjT7V0r893/YP7AIjEVHFMi2zgSfMLPJ/pUpnSN4LLASuB8Y0U95uM7udsK+VC7zHmY/ZaImxwAvO6MyQlA88APzRlZsWlfd7zvj8U1vecLRPdc9932/LoRckiXTUuxA2Pl9x293cpeMjM9sdlS9iPu4D3jGzemcdNKWpWfplYJbif8xGT8LuPZwyP78MLHdpm6PypuL5L54mJCKq/wF7gdvM7KSkjmZmkpr2oyzGelOamp0P0bbHbHwA9Cf8hKo84FnCj8LoS9ijygVWxDi2WlKxmS1xX4xBTX2wpvTr1e0z4FOl5HE2gZGoox4xLVcTvnsKkrY+ZmMBcJ8zOuvM7DXCwnpI4eetQAzzk7DD/033U1MVcEVSkXuA89T8dD9WR1z7l4Gx8doXnuQ5X83Pi4GXJaUBr3hBnV3Oy5bK0774Pz54AseLyhM4XlSewPGi8gSOF5UncP4PHCUbapx5NT4AAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 86.4x50.4 with 5 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "#Trajectory branch defined as (starting node, ending node) from trajectory inference\n",
    "#See main1.ipynb\n",
    "branches={\n",
    "\t'IRS':(3,1),\n",
    "\t'Hair-Shaft-Medulla':(3,2)\n",
    "}\n",
    "#Value range for coloring\n",
    "vrange={\n",
    "\t'Terminal logFC':[-4.5,4.5],\n",
    "\t'Transient logFC':[-1.5,1.5],\n",
    "\t'Switching time':[0.2,0.8],\n",
    "}\n",
    "\n",
    "for branchname in branches:\n",
    "\tprint(branchname)\n",
    "\tfigs=d0.draw_discover(*branches[branchname],ntops=(12,12,2,2),vrange=vrange)\n",
    "\tplt.show()\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Colorbar"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAFYAAAAzCAYAAAANb/8eAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjYuMCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy89olMNAAAACXBIWXMAAAsTAAALEwEAmpwYAAAFiElEQVR4nO2aa6hUVRiGn3fOXI5SaSoZFEQXMMpMumCZlXQB6UK/gi5EQWH5I8HuUASVRf6wkm50tysVQYgYmXb12IWihPxTGCEpWVZ0FDwzs+fstx97TY3TnDNnzsw2qv3AYn3r22t9a6337Fln9p5PtsnoPbl/egH/VTJhUyITNiUyYVMiEzYlMmFToithJU2S9EEog6F+rot4t0k6pIP+V0m6psm3UtJRHcSYL2lrwz4mSZoo6cnQ3ijpyk72AZDvdEAjtgeB+WGBA7bnj9RXUs523Cbe/d2spwtetH1HvSFpGfCh7YWSBJzeacCuhG1G0gXALSHu3bbflvQp8BUwJGkKMATMAt4HJgOnAo/aflbSSmApMA+4EJgQQp8PHA+sAPqBVbbva7OWScDLwAHAJtuLJU0F3ghriIAHRxg+1/atAE6eoD7qUIqenrE54CbgLJK7+Obgnwbca/uG0H7H9mnAxcAzwFzg6hbxttk+D9hO8of4Bphvew5wrqQJLcY0shB4zfYZwERJc4BrgCdC3GJD3yu6Pcaa6eUdOxmYCqwP7YPCx+hn29sa+m0O9Y/AZtuRpFbP1fV+20Psw4HlkiYCM4CD2qznSOCtYH8BHBVirA6+TQ199zoKekEv79jfga+Bs8NZe3z4GDWfqx7BbqbxmoBFwDLbZwJbgm80vgNODPZJof09cFzwzRpl7MeSLgdQQsdnbC+FjYEHgHclvQ881MPYAGuARyS9DlTH0P8p4BJJG4CK7U+Bp4FFkt4OfaIRxt4FnCnpA2AjcESni9X/6e2WpByA7VjSGmCh7e1pzNXTbwX/AvYD1kgqAuvTEhX+Z3fsviR7pE2JTNiUyIRNiZ7885oyfY5r0W6UE8qJnJJaCiUncn+2QRK5HMFOSq5uA5KTNkby3nW9OKlxnNiOAUMcJ/44BhtsXLfjGNsQJz7buG7HbihJO64ZR2YLlbW2F+xzYaPqIHMWvEShVKQ4oUSxv0ihVKDYX6BQzFPqT0qh0Eep1EexmKNUFKWSKORFqQiFvCkVTKHPlPIxxfwwhdwwxb5aUuciCorIE1FwlXxcpVCrkB8uk6tV6asltapJTaUM1TJUq7hSxtUqcbnM8FCFuFKlNlRmuBJRG6oQ7akQDVWJ9lSJhiIqu6tEeyLKv0ZUfqpyQe3baZ1qkh0FKZEJmxKZsCmRCZsSmbApkQmbEj15VyBpM1DufjljZhrwyz6cr9/2zE4G9OrtVtn2ST2K1RZJX+zr+Todkx0FKZEJmxK9EvbJHsX5z8yXvehOiewoSImuhZW0StLSFv5vGvKhjuki/oOSNkha0eSfKWkg5FaN9lP2eOZcImmgybdS0mdhP5e1i9HV162woZEyUnaOlss1xvgnAPvZPl3S45JOtv15uHwPcCnJz+6PARd1M1fDnCVg9giXL7e9ZSxxur1jF5NsqhVTJH0k6QlJ/eOMfwqwLtjrSfK86hxo+4fwS+vkccZvxdXA8y38Bl6QtFrSYe2CjFtYSUcDO0kyYFoxL+RNbSXJoxoPk4FdwR5kbwEb194uK2ZMSCqQ5Ie91+LyjbbnAsuA5e1itT0KJB0MvNrk3kGy4TuBo1uNs/1bMN8ElrSbZwQGSbIFCfXvjVM02KOmh3bAFcArrS7U92N7QFLbdNO2wtreQciBbUTSWmAlMAWYKmmd7Q/DtSLJV7kKcBpJ3tR4+AS4FngdOCfMV+c3SYeSiLrr70PHxQxgtqTrgGMlXW/7YQBJB9jeJWkGI39K/8J2V4VE9KXBXkCSyzod+JIkr3QVsH8X8VcAG4CHgYOB24N/Fkle1UZgdrf7aDHvQNN8q4NvAzCz3fjsASElsgeElMiETYlM2JTIhE2JTNiUyIRNiUzYlMiETYk/AO28vbp6PZtDAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<Figure size 72x10.8 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAFIAAAAzCAYAAAAEhF9kAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjYuMCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy89olMNAAAACXBIWXMAAAsTAAALEwEAmpwYAAAE7klEQVR4nO2aXYhVVRiGn3f/nNGZUefCogwqK4NKw8RCy6jIIgoTgq66KK27giS6ECKpiAz6T0i0EczqJohKqKgMx/xJLZrQIfAiFC+68KIyLWfmzNlfF2uNbCbPOfscl3mU/cBmrb1+vv2t96y19tqHT2ZGyekTnW0HzhdKIQNRChmIUshAlEIGohQyEC0LKWmqpM8lDUjaLWl+k/YrJV0iaa6keQ3aDUhKJpS9KSlu1Uffd3kIO4Wf1+o5UtITwAkz2+AHPtnMjhXo9wiQmFl/nfoBYLGZjbXkUP3n7TCzRSFsFcLMWrqA5UA/MD1X9jiwBJgF/A4IeB64CdgIXAXsAg4AH+JWQj+wDfjS2xgAXgN+AB7NlSXAc8AmYAvQ7+uuBPYAnwHfApfn/LkfOOr731XHznvAs8BuYJXvdwGwGdgKvNOKLu3ske8Dh4GtkrZIusiLdDNwixfiWuAGYDDXbz3wipk9BCwFjpjZbcB9uTYfAIuAh0/x3EEzWwxcKqkPeBp4EngAuDDf0Mw2A/vN7HYz+6aOnYuBITNbgBMeYCWw2szuAI5JWlhUlJaFNLOqmb1gZnOADcAKYB8wBzcDXwduBSIzq9YxczVOfMwsy5UPmdkIkJ2iz5BPfwOmATOBfWZWy9UVIW9nPH/c76HXAC/7beZOYEZRo+28bC6TlPrbIzjBarjB9+GW0TJg/4SuVWB8wz8ALPD28j402rDzdQIOAnO8ANc1aV+vfKLNA8BTfibPx20bhUiaN/kPc4GPJJ3AibPMlw8CfWY2ImkMP+Ny7AY2SpqNW5JLJH0HHAfubcOPV3H77RHgD+9Lnr2SPsXtu0V5CVgvaRpuYjwGHCrSseW3dqcgKTGzMT8jdwKLQr3x2+FcPpBf4Wf0XmDT2RQRzuEZ2WmcyzOyoyiFDEQpZCDaOf4AMD2aYVWNQiRQ5E5hitxPI2GSLxMmQJwsG7/P503ecJ16V2cny5EhgcZTDMmI8hf5fEYkI86nGJEyYowYI5KIEIP7Rr8ys3v+FyGrjLCwdymqVKCrAmkClQrWlWBpjFUSsjQiS2OyVGSVyKWJqFVEluDuU1zeX5biymKXWmJYamQxkGaQGsQZUZoRJxlJUqOSjlGJa3SlY3TFY3QnVbqTKpPiKj3JCN1xld54mO5olN54mCnRCXqiEaZEwz4dZYpq9EQRvarQPePQ9Fb1KJd2IEohA1EKGYhSyECUQgaiFDIQbX9rSxoChsO60zFMMrPZrXRo+xwJDPs/P887JP3Yap9yaQeiFDIQpyPk+mBedB4tj638YzcQ5dIORCEhJb0habuktyaUr5O0U9IOSdefGRfPPA3G96CkvZL2SFra0EiBEJV5wLs+vxa4MVc306ezgI9bDX/phKvJ+HYB3cBkYNvphqwsAMbDPrYAJ8M4zOygz1aBWgFbnUjd8QG/Aj1AL/BXIyNFhOzLGTnq7yeyGni7gK1OpI/64/sEF/jwM7CmkZEiXzZHgak+PxX4M18paQXwi5ntKGCrE2k0vlW4gDCAL4Cv6xkpMiO/xwUUASzGhZ4AIOluXBTai0U87lDqjg8YAf4B/gYqjYw0FdLMfgKGJW3H7YOHJT3jq9fgosK2SlrXkvsdQpPxrcWFw+yiySG9PJAHojyQB6IUMhClkIEohQxEKWQgSiEDUQoZiFLIQPwLPPK504SclXEAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 72x10.8 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAFYAAAAzCAYAAAANb/8eAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjYuMCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy89olMNAAAACXBIWXMAAAsTAAALEwEAmpwYAAAFn0lEQVR4nO2abYhUVRjHf/8767y466olplYmYYgvacQaaVILFWwUFRGKEkQFCQV9KKgPfoqM6IOFGFn5oSWEIiSVhcpSW9xdKzGRssggKCqULF/Wl53ZnblPH+7Z5brt7uzszNlC7w8u55nnnPOcc/5zOHfunUdmRkLtCf7rCVyqJMJ6IhHWE4mwnkiE9UQirCfGLKykyZLa3XXGle9WMxlJMyStq7DP40P4miWtrzDO0dh6Vjnfakn7JHVI2iopPeqAZlb1BXTG7KAWMccydszXDKyvJg6wEGgD0u7zLUButPFqdhRIapX0BvCppFmSvpDUKelNV98s6RNJbZK6JDVIWi7pa9f2CUlzJG117e9zu2W/pBbn+0rSFkmHJbVIuh+40e2yu4eZ1wtuvL2SZjvfehd7k6TWYZb0MLDRzHoBzOyAmfWMWpBa7RqgFVjtPqeBOmdvBW4g2kU7nW8d8ADwEtDsfALmuPYB0A7UuVh7XJufgOnA1cD2cjsWmAHscr4VwGZgJvCx860CWp191I3ZDswD3gIWjlWTWt+8vnHllcA2Se1uQbOc/4gr/wCmuIWudLt0aSzONGA+sBv4DJgpScAJM/vTzPr7l2MO8K2zDwJzgeti8zgca3vCzJrddRQ4Fpt3xdRa2NCVa4AdZtYMdBHtRoD4iwkBp8zsKeB54MVY3V/Ad8CdLsYSi7bV4P6DYw7mF2CJs5uAn4FfgQXOt3iEvtuAZyRNAJDUJCk3QvuLqBttwwrZC7wn6cEy7dZKeghoAF7td5pZKOk1YI8kA34Anh4mxgFJO4ANZtYRrzCz4+783g/0Ao+a2TF3Rne4uH1DBTWz7yW97+Ygoi/ksTLrGUB2Gb7dklRnZkX3s+p6M3ul1mP42rH/d16WtAwoASt9DHBZ7tjxIHmk9UQirCcSYT1Rk5tXw+yshYWQIAAFIhAogEAiCISCYMAOFERXkCKlAClFKojKgBRSQKAUgVKIACFwpRQAQv2/aN0lDMLoKRIzLLSoLoxsC8PIVwoHbCuG0ediiJVCrGSEpRDDKFmIWVSGFvI7p3eZWcu4C1vKh8x/ZDrpjMjmAjK5gHRGTMymyWYmkMumB+yJ2Ry5dD0NmUbqM5PIpeupzzSSqasnVzeZbCoqc3WNpIN6UmRIKU2KLCllSZEmKAkVDRVD1GeoFKLekLC3RClfJCwUCfNFwp4ipXwfxbM9lHoKlM7mKZ7roXShQN/JCxS7C/T9fWHALpzroadY4HxfnnyxwPm+Hs735XnWPppWqSbJUeCJRFhPJMJ6IhHWE4mwnkiE9URN3hVIOgLkq5/OqJlG9M52vMia2aJKOtTq7VbezJpqFKsskg6O93iV9kmOAk8kwnqiVsK+U6M4l8x4yYtuTyRHgSeqyd26R9KPkjqHqY/nQi0Yqs0ox3nd5U5tHORf5DJtuiSN9Dd2JWPNknRIUl5S3aC6Vpe10y5pTdlgVWS/TAUyDJGJMlyGyhjGuBnY4uzNwNJY3XbgWqKsmJ01yujJunW14zJ5YnWtwFzvmTBmdsrMCiM0ucLlR70tKTvGYW4FPnf2bmBZrG6qmf1WQVZMWcwsb2anhqsmypVok3RduVg+z9gVZnY7UaLDk2OMMQXodvYZLhYwPnfhn+fMbDlRYsmGco3LCutyVtsHXR+U62dmJ525HajocTDGGaDR2Y3A6fgQMTvEM/3rMbNOomS7ESn7SGtmx4my90aNS9CVOypuI8qZGgtfAmuBD4G7iM65fk5KuoZI1O5/d60tkhrNrFvSPC7+goemioO+iejcO+3KLNAC3AtcBRwC9gE7gUlVjLMR6AA2Ee2Udc6/mCjhrgu4qUY3rwluLaeAPcAdsfHaiNJVO4BF5WIlDwieSB4QPJEI64lEWE8kwnoiEdYTibCeSIT1RCKsJ/4Ba0a2kB+enKYAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 72x10.8 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "from operator import or_\n",
    "from functools import reduce\n",
    "from dictys.plot import colorbar\n",
    "\n",
    "#Color bar font size\n",
    "fs=8\n",
    "\n",
    "colormaps=reduce(or_,[x[2] for x in figs])\n",
    "for xi0 in range(len(colormaps)):\n",
    "\txi=list(colormaps)[xi0]\n",
    "\tcolormap=colormaps[xi]\n",
    "\tans2=colorbar(colormap.cmap.name,colormap.norm.vmin,colormap.norm.vmax,orientation='horizontal',figsize=(1,0.15),title=xi,title_fontsize=fs)\n",
    "\txticks=[colormap.norm.vmin,0,colormap.norm.vmax] if colormap.norm.vmin*colormap.norm.vmax<0 else [colormap.norm.vmin,colormap.norm.vmax]\n",
    "\tans2[1].set_xticks(xticks)\n",
    "\tans2[1].tick_params(axis='x',labelsize=fs)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Animation visualization of dynamic networks\n",
    "#### Animation layout\n",
    "\n",
    "Here we use the 'notch' layout. Each row for c,d,e,f has their own set of select TFs. The layout is arbitrary and you can design your own custom layout.\n",
    "\n",
    "```\n",
    "+---+       +---+\n",
    "| a |       | b |\n",
    "+---+---+---+---+\n",
    "| c | d | e | f |\n",
    "+---+---+---+---+\n",
    "| c | d | e | f |\n",
    "+---+---+---+---+\n",
    "```\n",
    "\n",
    "* a: Dynamic tracking of cells used for GRN inference\n",
    "* b: Dynamic scatter plot for differential regulation v.s. differential expression logFCs\n",
    "* c: Dynamic plot for expression level (log CPM) of select TFs as a function of pseudo-time\n",
    "* d: Dynamic plot for regulatory activity (log target count) of select TFs as a function of pseudo-time\n",
    "* e: Dynamic heatmap for regulation strength from select TFs to select target genes\n",
    "* f: Dynamic subnetwork graph from select TF to its targets\n",
    "\n",
    "#### Branch independent parameters"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "from IPython.display import FileLink\n",
    "from dictys.plot import layout,panel\n",
    "# Number of frames (interpolated time points), use 100 or higher for finer resolution\n",
    "nframe=100\n",
    "# Animation FPS for saving. Determines speed of play\n",
    "fps=0.10*nframe\n",
    "# DPI for animation\n",
    "dpi=100\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### IRS branch\n",
    "##### Branch dependent customizations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "branchname='IRS'\n",
    "# Select TFs for each row's dynamic subnetwork graph\n",
    "tfs_subnet=[\n",
    "\t['Lhx2'],\n",
    "\t['Stat3'],\n",
    "]\n",
    "# Select TFs for each row's other plots\n",
    "tfs_ann=[\n",
    "\t['Lhx2','Dlx2','Sox4'],\n",
    "\t['Stat3','Klf4', 'Jun', 'Fosb'],\n",
    "]\n",
    "# Select genes to annotate as targets in all rows\n",
    "target_ann=['Krt75','Lgr5','Mgst1','Mki67', 'Scd1']\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "##### Branch independent lines\n",
    "\n",
    "Note: this step may take **> 1 hour**. Use `nframe=10` above to speed up exploratory analysis and `nframe=100` or higher for the final high-quality movie."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<a href='../output/animation-IRS.mp4' target='_blank'>../output/animation-IRS.mp4</a><br>"
      ],
      "text/plain": [
       "../output/animation-IRS.mp4"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAB2EAAAPlCAYAAABb9JZzAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjYuMCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy89olMNAAAACXBIWXMAAA9hAAAPYQGoP6dpAAEAAElEQVR4nOzdeXhU5dn48e+ZfSbLZCcJARLWmcAoIiK4YtxQxLW1trWittXY1GoXW+yvfRnt+7bY1qqt0XRT09LWuitGcYu7IiiLgUxYs7BkXybL7DPn98eZYAgBAgZZvD/XxWXmnOc85z5nYmauuee+H0VVVYQQQgghhBBCCCGEEEIIIYQQQowM3ZEOQAghhBBCCCGEEEIIIYQQQgghjieShBVCCCGEEEIIIYQQQgghhBBCiBEkSVghhBBCCCGEEEIIIYQQQgghhBhBkoQVQgghhBBCCCGEEEIIIYQQQogRJElYIYQQQgghhBBCCCGEEEIIIYQYQZKEFUIIIYQQQgghhBBCCCGEEEKIESRJWCGEEEIIIYQQQgghhBBCCCGEGEGShBVCCCGEEEIIIYQQQgghxJeeoklWFEU50rEIIY59hiMdgBBCCCHECFOPdABCCCEOmXzYJeR1XAghjk3yGi6OF0mA1+v1Huk4xHEmGlNZWdtBS0+ArCQLswrS0OvkT+cxbFhPniRhhRBCCCGEEEIIIYQQQgghhDgMlq9v5K5l1TR6A7u35dgtLF5QyLxpOUcwMnG4STtiIYQQQgghhBBCCCGEEEIIIUbY8vWN3LJ09R4JWIAmb4Bblq5m+frGIxSZ+CJIElYIIYQQQgghhBBCCCGEEEKIERSNqdy1rHrI9Tb6t921rJpoTFbkOF5JElYIIYQQQgghhBBCCCGEEMc0RVFuURTlU0VRuuP/PlQU5aIjHZf48lpZ27FXBexAKtDoDbCytuOLC0p8oSQJK4QQQgghhBBCCCGEEEKIY90OYBFwMjATqASeVxRl6hGNSnxptfTsOwF7KOPEscdwpAMQQgghhBBCCCGEEEIIIYT4PFRVXTZo0/9TFOUWYDaw4QiEJL7kspIsIzpOHHskCSuEEEIIIYQQQgghhBBCiOOGoih64KtAAvDhfsaZAfOATUmHOTTxJTKrII0cu4Umb2DIdWEVINtuYVZB2hcdmviCSDtiIYQQQgghhBBCCCGEEEIc8xRFcSmK0gsEgTLgClVVq/dzyJ2Ad8C/HYc/SvFlodcpLF5QCGgJ14H6Hy9eUIheN3ivOF4oqjpU/l0IIYQQ4pglb26EEOLYJZ8+CHkdF0KIY5O8houjgqIoJmAsYAe+AnwHOHtfidh9VMLu8Hq9JCcnH+5wxZfE8vWN3LWsmkbvZ2u/5tgtLF5QyLxpOUcwMvE5DOt1T5KwQgghhDjeyJsbIYQ4dskHuEJex4UQ4tgkr+HiqKQoyuvAVlVVbx7m+GTAK0lYMdKiMZWVtR209ATIStJaEEsF7DFtWE+erAkrhBBCCCGEEEIIIYQQQojjkY49K12FOCL0OoU5E9KPdBjiCyZJWCGEEEIIIYQQQgghhBBCHNMURfkN8DLQgNZW+BvAXODCIxiWEOJLTJKwQgghhBBCCCGEEEIIIYQ41mUB/wByAC/wKXChqqqvHdGohBBfWpKEFUIIIYQQQgghhBBCCCHEMU1V1W8f6RiEEGIg3ZEOQAghhBBCCCGEEEIIIYQQQgghjieShBVCCCGEEEIIIYQQQgghhBBCiBEkSVghhBBCCCGEEEIIIYQQQgghhBhBkoQVQgghhBBCCCGEEEIIIYQQQogRJElYIYQQQgghhBBCCCGEEEIIIYQYQZKEFUIIIYQQQgghhBBCCCGEEOJzaG1t5ZZbbmHs2LGYzWays7O58MILef/99wFQFIXnnnvuoOfNz8/n/vvv32Pbxo0bOeeccxg1ahQWi4Xx48fzi1/8gnA4PAJXIkaK4UgHIIQQQgghhBBCCCGEEEIIIcSx7KqrriIUClFeXs748eNpbm7mjTfeoL29fcTPZTQaue6665gxYwYpKSmsW7eO7373u8RiMX7961+P+PnEoVFUVT3SMQghhBBCjCR5cyOEEMcu5UgHII44eR0XQohjk7yGi+OCoijJgNfr9ZKcnHykwxHHkK6uLlJTU3nrrbc4++yz99qfn59PfX397sfjxo2jrq6OrVu38qMf/YgVK1bQ19eH0+nkN7/5Deeddx4Ac+fO5e23395jrn3l9X70ox+xatUq3n333RG8MrEPw3rdk3bEQgghhBBCCCGEEEIIIYQQQhyixMREEhMTee655wgGg3vtX7VqFQCPPvoojY2Nux/39vZy8cUX88Ybb7BmzRrmzZvHggULaGhoAOCZZ54hLy+Pu+++m8bGRhobG4c8/5YtW1i+fPmQCWBx5EgSVghxXMpfVPFW/qKK+w/DvNfnL6roGoF5svMXVbyWv6iibyTmGwn5iyrU/EUVlw9zrDt/UcXawxuREEIIIYQQQgghhBBCHJ2iMZUPt7bz/NqdrKr38sgjj1JeXk5KSgqnn346P//5z/n0008ByMzMBCAlJYXs7Ozdj0888URuvvlmpk2bxqRJk/jVr37FhAkTeOGFFwBIS0tDr9eTlJREdnY22dnZe8Rw2mmnYbFYmDRpEmeeeSZ33333F3gHxIHImrBCiKNK/qKKx4CUuiXzLx+0fS7wJpBat2R+1zCmuhI46FXI8xdVnA0sBqYDFmAn8AHw3bol80MHO99+/BDIiZ/HewjXd8jyF1W4gcvrlsyfPmhXDtB5OM/9RchfVFEH3F+3ZP79RzgUIYQQQgghhBBCCCHEMSgaU1lZ20FLT4CsJAuzCtLQ6z7rQLt8fSN3Laum0RvYvS3Hnspjr32CtWMLK1as4OWXX+a3v/0tf/vb37j++uuHPE9vby9ut5uKigoaGxuJRCL4/f7dlbAH8t///peenh7WrVvHHXfcwe9//3t++tOffq5rFyNHkrBCHANc5S4jkAl0Vi2s8h/peI4FdUvmd+xvf/6iCtPgpGr+oopCYDnwJ+AHgB+YBFwF6Ec4xAnAJ3VL5m8ecO4jqm7J/KYjHcPnMdRzKoQQQgghhBBCCCGEEAdj6ASrhcULCpk3LYfl6xu5ZelqBq/M2uQNcNuT1Tx87Qy2bv0XWVlZXH/99SxevHifSdif/OQnvPbaa/z+979n4sSJWK1WvvKVrxAKDe9jzjFjxgBQWFhINBrlpptu4sc//jF6/Uh/nC0OhSRhhTg2GAEr4ENLDH7hPA6nEUgHup01Ht+RiKFf/qKKdOBB4CwgFdgK/Lpuyfz/DBjzFrC2bsn82+OP64C/oyVVLweeAa4fNPUFQFPdkvkDvyq0FS0xOziGC4H7gTHAe8ANdUvmN8b3nQL8GjgJ7blbC/ywbsn81QNiGRf/+TqgHFgYn7ozf1EFQHndkvmD4+s/txm4G/gGkAVsB35Tt2T+3/MXVVyPVgWaMmD85cCzdUvmK/H9i+Pb+98n3FC3ZP5j8cdX1C2Z/1x8fx7wO+BCwAx4gJK6JfM/2kdc3wF+DBQAdcAf65bMf2iosfHxOuAnwE3x+9gM/Lluyfz/i+93AQ8Ac9B+958GflS3ZH5vfP9jQAqwCigBgvmLKurR7u19+Ysq7gOoWzJfiY8/A/gNMBNoA54F7qxbMr8v/jw8BJw0IDH+EFAEzKhbMv+I/s4LIYQQQgghhBBCCCFG3vXXX095efle23Nv+gvG1Nzdj5u8AW5ZuprSb5zEryo8eyVgAVRAAe5aVs3k+IDCwkKee+45AIxGI9FodI9j3n//fa6//nquuOIKQKuMraur22OMyWTa67ihxGIxwuEwsVhMkrBHCVkTVohjQNXCKh9QX7Wwqmuk5nSVu4yucle6q9xlOsA4i6vclfWPIp0eiAKxkYrhc7AAnwDzgWnAX4B/5i+qmHWA434CrENLjv5qiP1NQE7+ooqzDjCPLT7Xt9ASwWOB3w/Yn4SWWD0DmA1sBl7KX1SRFN9/Clpi9wm0FsC3oVXbAkwZsG1f/gF8Ha1a1wncDPQeIOZ+/wXuBTbEz5MT37aH/EUVicDbwGjgUuBE4Lfs43Ujf1HFN9ESw/8vHtPPgV/lL6pYONT4uN8Ai9Cei0K0pHJzfL4E4BW09sinAF8FzkNLvg90Lto9Ox+4BK0N9Q7gfwZcH/mLKiag3fOngROAr6E9Pw8C1C2Z/w/gJeBf+YsqDPmLKuYD3wG+KQlYIYQQQgghhBBCCCGOX/PmzaOxsZEdO3cx484nyCv5Jwb7qD3G9Cddf/H8+j0qZPtF/d00/efn9Gx4k/rNHhqa29m5cye//e1vueyyywDIz8/njTfeoKmpic5ObVW4SZMm8cwzz7B27VrWrVvHN77xDWKxPT+Cz8/P55133mHnzp20tbUB8K9//YsnnngCj8fDtm3beOKJJ7jzzjv52te+htFoHOE7JA6VVMIKcRRylbvSgUSgoWphlQpQtbAqchDHmxNisRyDqra9d2P1vpJzerQqTQOwv94GCqC8eKou+puHNrQON4bP6ZL8RRWD49791Z26JfN3smfS80/xytSrgZX7mbeybsn8e/ez/0m0qs+38xdVNAErgDeAf9Qtmd89YJwRKK5bMn8rQP6iigfRkn798VUOnDR/UcVNQBdwNvBi3ZL5rfmLKoKAv78FcP6iiv72yS37WxM2f1HF5Ph1nl+3ZP7r8c3b9nNNe6hbMt8fv7eRA7Qf/gZaC+xTBrR23rKf8XcBP65bMv+Z+OPaeIvlm9ES0oOvIwkt0fz9uiXz+/dvRasq7j+/Bbiubsn8vvgx3weW5S+q+FndkvnN8XF9wHcGtiHOX1QRBXoGXd+dwL8GrBO7OX9RxQ/Qnutb6pbMD8Rj/RT4I1oy1123ZP4n+7lmIYQQQgghhBBCCCHEMc5sNpOdnc2HW9tpj9nQJ9oACDRU0fnWI4RaatFbkkiYdi6xs76FotM+qu6reQ/v+/8h0tWIYjChmGx0r3yGiLeF5nAAm83KjBkzeOGFF0hOTub000/n1Vdf5a9//SujR4+mrq6OP/zhD9x4442cdtppZGRk8LOf/Yzu7u494rv77ru5+eabmTBhAsFgEFVVMRgM3HPPPWzatAlVVRk3bhzf//73+eEPf/iF3z+xb5KEFeJwcdv1aBV6u3B7uw7y6GwgGdgFhPc30FXuslvCiZHrP/4/taSsyAdwXp/PMCYcGTcmEtGVFlfagUhJWVHz/d97Of2jscuS1ue821a1sKrXVe5qqVpYtVdl69OX/8bWadNPaknL8BVHHthZUlbUvNeJD683gVsGbTsVWAqQv6hCj1ZpeTVapaYJrV3ugSoWP+7/IX9RRRlwbf/juiXzE+uWzI8CN+QvqvgFWhvaU+Pn+Vn+oopZ/e2GAV9/AjauEa0tcP/co4D/BebGt+vRqmfHHujCB4pXl/55wKaL0Co7o2hVqofTdGDNgdbWhd1VqxOAv+cvqvjrgF0GwLuPw5xoz9kb+9m/rj8BG/c+WiXuFOIVs0DVMNeBPRE4IX5P+ynx+QoAT92S+Z35iyq+jVaB+wGwZBjzCiGEEEIIIYQQQgghjgMtPZ9VuEZ62mh5yk3CtPPImP8jwu07aF/+JxSDkZQzvkmkt4O2Zb8jde4N2CbPIRbyE9y+gYRpRehMVqZuXso7r75ITk4ODz30EHV1ddxwww1897vf5f/+7/92nyc/P5/Kyj1qeigpKdnj8ezZs1m3bt0e2772ta/xta997TDcBTGSJAkrxOGTCkxEq5rsGrjDVe5KRUsebkJr73sWWivcDrRWtpsBY9XCqn0mYF3lrkS0BO2Uk3aeawH8pcWVNWVzbjMnWC2p5/e1d81pCnc962tO9dtG+U/++6zk09OvHG+KWs1AEOgt/vAB+811v8z8IP+5wLc+vjvQZ+oa15xQ33NCUtZoVW+9Whcx1gcJv3T57cur11qi4bol84dqdX849NUtmb9H1WV8fdJ+d6BVUd4OVKFVQ96Plozd77wDfv4f9qym3S1eaftPtBbHv0R7noqJr6XK3onx/nb//crR1s+9DahHu98fDiO+wV4ABq6/uhOtJe/+xAbFAtrv4ME6mLWHE+P//S57xgtawvjzzr8/fQceAmgx/hmtynWwhgE/n4UWcw6QAPR8ruiEEEIIIYQQQgghhBBHlWhMZWVtBy09AVp7grzy4oskJiYSUyEQjmIdfzKGtNHokzJJO78YRVEwpo8h2ttB59uPUXD+QpqbOiAWxTb5NAx2rT7HlJmPAmTbLWQkmjGZTDzyyCPYbDamTp2K230Xd/z0DmZeVUy23casgjT0usEf5YrjiSRhhThc3N423PaVaInVwTKAgnP7fCdf7fUqP83KPMdrMDQYYrE3z/L5cy/t9cXuyUiNvXXPqJa5/sApQA1u7+7WqvEErAutjbBqjFiCOiKRdEOdH3DoVTW/1mjyZreYTDNX391+8/f1hAzKmI/zXvYao+auRW0dKR2/mKprtf4sQx81TrIGk9aCqvToO880B2wZ7QkpubaYyWrEurkb5TSTqqYCn5QWVwYBXUlZ0Ugl0A7V6cDzdUvm91fG6oDJQPVwJ6hbMr8FaBnGuM78RRWNaAm5g4nve3VL5r8Uj28M2nO+P/3VnAPbLvcwKAmYv6iiCq1682zgdfbWCiTlL6pIGFBFOn2Icx1oZfZPge/kL6pIO1A1bN2S+c35iyp2AePrlsz/1wHm7bcZLRF7LvC3IfZ7gOsHXcfpaEnmjQeYe6jrWw0UDk7uD5S/qOI04GfAAuAetPVi97emrRBCCCGEEEIIIYQQ4hiyfH0jdy2r3r2ua9umVhILTuQPD/yJ0ydmcs1fPqQtoND+2sOYcx0oymdJUkueEzXk57bZabj7CrCMO5Fdj5RgLZiBJf8kEhxnoLcksnhBIY9vhBNPPBGbzbb7vGU1enx9fdz619cx2LPIsVtYvKCQedNyjsi9EIefJGGFOJwGJE4H6bqwt8+XEotNsqoknrYjkrFVr4vUj8aUGY2O6tMp5/+ytT27IBxpBhJfs1mVH2nVs5vja8P6gdbLe3p9V3f3Tpyo/4PBNIp0vRKN/Yqx1dZQant9LPPk9pxEJWO6JXhqYOcJuq4xqdVm1bDLvqV3TTAvlB44pdVn7jIlBtMMC9Z/37glbc3oZF/GnKRQmt1CYpuiMyw3Y1nVo0THRBSa0ao5k9GSW0c6CbsZ+Eo8adYJ/AgYxUEkYYeSv6jiZrSE5bNo65NagOuAqcCtBxnft/IXVXyMds9+x4HvWT1aRe0l+YsqXkJbL3av9Xzrlsyvy19UUQ48El/TdB0wDsiqWzL/CbRKVB/w6/xFFX9Ea6l8/eBpgIL8RRXTgR1o66cGB435D1or5ufyF1XcidZy+SRgV92S+R8OEf9i4I/5iyq8wHK0VsMzgdS6JfP/AJC/qOIN4Nm6JfMfrFsyP5C/qOIe4Lf5iypCaK2GM4GpdUvm/x34F9o6s+X5iyrc8X1/Av45YD3YfakDzspfVPE4EKxbMr8NLam6Ir5+79/QKmgL0dbW/X58jdp/An+sWzL/5fxFFTuAVfmLKpbVLZn/1AHOJ4QQQgghhBBCCCGEOMotX9/ILUtXM7jdY0gx8at3Onl4bAG/uf58blm6eq9WgwMfFzlHkT06D3fK76mrXkOgdg09q1+k+72l/PWp5cyblsPjQ5w32LvnR7BN3gC3LF3Nw9fOkETscUp3pAMQ4rjltg/5/5er3GUdFwrdaIvGbkiLRBw/GpWZ3hsytN+609vyyK6WEy7s6dOd6PP1TAqF/bnRaBew7p701F2A/eZObwJue05VbYOzqrYh8Ku2Dt/EgFlnJWrsimQ7G4KFjsJ6Ne1kz5gZ7T2nWBoSphtrpySf5/Lazx7Tkzo3uzf3qsktp8yKNF1lXRWak2sK2SKJ/tSIJZZwliWcMFWJ6QK2cPIaG4n/7E5oXV4257ZNmap++TP3z6uOJ+m6gO7S4krLF3cjh/S/aJWNrwBvobVyfm4E5l2J1ra2DNiAtu7qbODyuiXzD2YN1m+jtaNeTTyxxwGqbuMtkBejrUPajFaFuS+3AE8BDwE1wF+JV+rGq1avBS5Ga9X8dcA96Pin0RKlb6JVzn59iHhCwAXxuF+Kz7WIfbQXrlsy/2/Ad4Ab4mPfRkv+1g4YNoE9K4J/BdwL3I1W+fpf4mvr1i2Z7wMuBNKAVfHrfQP4/j7uyUD/A+SjJdJb4/N9ilY9PBl4F1gTP++u+DEPoCVmfx4fXxX/+c/5iypGD+OcQgghhBBCCCGEEEKIo1Q0pnLXsuq9ErAD3bWsmvMLs3n42hmk5BYQ3FWDqmpHZNstXJHbS1JSEnl5ecyblsM7Py3i/26+ih/89OeUPvEqo1IS6Kh+f/d869ato7fPt/u8wV0bUUxW9MnaR6TqgPNGY1/USoDii6T0/wIJIUaQ265HS4S1oSWjQq6CsUlA1++aW8estpj/NyEWywijbO3Q6615/nDs6gZlUk2mcXSiwb9yetj3OuAFunzRZNuzXT9fnZvn9p/jD5yKtlalBXintOnZTrPSW5BvXrWlwPLRt5t0JuPzpm1Be0vmmRE1c8M7zq77cnqzr5/YFZlsD+Tt7E5IzO+D6pReZ6Ki6vPUmNoYNPb5FHSBpHBaW6+pQ7GF7Sts0aTUtblvRFaMe6GpamHVtoGXVlpcmY62xmhLSVlR7Iu8rUIIMUzy5kYIIY5dsiCSkNdxIYQ4NslruDguKIqSDHi9Xi/JyclHOhwxgj7c2s7X/7pir+1tFfcRC/aRdeUvAPjPd2czZ0I6Ddt3MGXKFIou/SrX3liMpa+Jm276LiUlJbjdbu7/dwV/KH+GcLYLfYKd4K5NtL94L3f96VF+ecs3uf7663n66aeZc84FrM86j6i3mbaXHyDRdR6pZ1+/Vxz95xXHjGG97kk7YiEODxXoBgLAlR9azJGpwWDmdd4eZZ7Pv35qKFxri8U+sMViNVZwBb36lA5/QsHmWFKnX5+5I9wxzTsj8TmvXokY2sLjJrnMb46b0GtpiShhi06Jbe7V0X7BmFxTZvLfU+dt+nbHxsA5nQ7rG486dK25HYbQqTvM44IbkrpT04wtk8Z0Xlrpi3g6LDq99cQeW31VcvvrfmPfGdm+lGbwzwwF05NaEna+blatnxhVi19RlDDQY4xa+q9hsG7AIAlYIYQQQgghhBBCCCGEEMeClp7AQY0bOyaP5S+/xB133MH1C84mLS2Nb3/72/ziF79g+fpGfle5nY7Nawm99zSxoA+DPYvUc27kkfoUTlnfCMC5556LZdRYmv+5CDUaJsF5Fimnf/NzxSeOLZKEFWKkue3JQB5u77O1p0+wpE7ouyRzSmRzZiT6tfRA0gmNwTG5Y9hY7YPAKqv50uxwVJmUFNmVYextvYTud7t6Jls2RiYm7wo5V48xV12VY9qYYNb1rTcqAd224Oy0TYGz1905Y+kWfSx2ZV36p6FVeS+9csqOi13Pd/5vc0n2FWtHd10Wa1Fn9Prsb1yQEbKVpPt2/NPasq3TkH3JDJTo5nfGvpvh3KVOVoyJEaPSlxEM5ah9pmBXY+qWlm5Le9Zo7+TRjtZTTVObT9/40F2/7Bx8eSVlRWEg/MXfWCGEEEIIIYQQQgghhBDi4GUlDb3CXsb8H+5z3Nlnn83KlSv32N/f1tiYMYZRV9895Jx3LavmvUceRa9T+HBrOytSzzvk+MSxTdaEFeLzcNvH4bafOmjrLcBvcNvPBLVIVbkqezsTkqLR1lb/yRnvRec4/pKcbNxqMkVet9kmLLUnWxUd/zBYYs+tiaamr0tu7j3Btvz9MeYqM5Bi1IWaRpm2vdwRyXu1JzqqtiE4I+n+tZeNevCTbuMPP/FF7vB9bNIR0Vp2ub2RHjV5fCh1xUS9rrsyyZ/1sa1vtC3fW5hu8aXUhlTLxrMazr50fKz3dKMSSt2Y1F67IevTXT22NuspOy5WN2d+XGeOWpsBM/tY+1MIIYQQQgghhBBCCCGEOJbMKkgjx27Zbw/ZHLuFWQVp+51nZW0Hjd59V62qQKM3wMrajmGdVxnmecWxSZKwQnw+ZwBFuO12ANx2px/0O3W6idv1+tvHnt3+A2tG0GHLCn/lKz19l21Ora5pTPto60kB/1WuUGhSWjT22Fy/731fq+nMjW/kZ/Z9Ostev3leS55+Xde2lzP9sbCyGmgGsOh6R/dG0xx2feP05tCk9In1RC/ZEbBPNa35yi3ZXw2VlBVtB7AkfvzJ9tSq6jqzcVVy0zWfrilozXniDCWhKSGhrzUy2ehXjFvTlcaNYywfb63K+eip7enVK7alffop0FS1sKqzoOOEXcAGYMeRuaVCCCGEEEIIIYQQQgghxMjR6xQWLygE9r2Ypz8c5bXqpv3Oc7Btjfd33v7HixcUotftf4nRuXPncvvtt+9+7PP5uOqqq0hOTkZRFLq6uoYVl/hiSRJWiP1x21Nw20/HbbftY8TLwJO4vV7c9slbDYZ/1hmNN7Xqdb6YqqboTaRY7LF6nY5JE8KhzAlq86qbAh7fSaGwBTBe/pThvRNWGYLBbr0v1Km0JfbYV14U2tbjbTD/PGms/6e9TSYH4Nv8/CjLp5vPTGqPjMMXS1nRGR2zvnNz4tNqbviNlWaz4Z9JiQpAaXFl1sfNv+5J2/Sbty799GdJUaK0JNR1bMxe5QkY+mqATcvHv/xce1rVew6alK+3W/qCpr41HYm7wv8+4QcZHoczgc/+LuSUFleaD/s9FkIIIYQQQgghhBBCCCEOs3nTcnj42hnYbcYh93t9YW5Zuprl8TVdhzLctsEDx/WfN9u+57HZdgsPXzuDedNyuP7667n88sv32P/UU09hsVi4995795q/vLycd999lw8++IDGxkbs8TqxfsXFxSiKwv333z+seMXhIWvCCrF/4wAHWjXqlr32ur0dQAdu+0Sgo1dR2rv1uoQTA8GdidAF5AMNnTpd+1+SMidvVkZb5vrXuCyqWhMJ6P7mbzPdpOjUU3NO8dYUXlr/7xOWrF1dd+b4U0KjzOf4jekTu3W2zmTWr4349SdbN2+7NKHXtvOcE0szTF0d13EOr88+JbsZWAME5vzsIt0Uy0mndUdzOxvDhX15pnWjpuu2f7z4J0uX94frcTgNauCaQm+yaXK7oXPcbC/BD/W7HhvT6Vhw0lZlYtjw9KYz37tjy7tn/K4FyAN8pcWVIYCSsiL18N7q/ntqfwxIwe29PP7zwvieCFp17pPA/+Ae0PPBbT8bWAxMByzATuAD4Lu4vaEvJG4hhBBCCCGEEEIIIYQQR7XzC7Nxv7BhyH0qWnXqXcuqOb8we8jq1P72wk3eAEN9YK6gJVcHtxeeNy2H8wuzWVnbQUtPgKwkbcy+KmD/9re/UVJSQllZGTfccAPLli3bY//WrVtxOp1MmzZtr2OfffZZVqxYQW5u7pBziy+OVMIKsX91aMlU61573PZs3HYzbnsusDAW4WdTe8OJc7rC9lit9YS6lqnb/NHENCAVFas3XGCIqKYrPrBajMAmvSl2mSkpYo5FlJ3GxGiLwRJLB8h09azuDqT/oU49pa3XnKMC2UBVqnFnldP8erfV1zY1Fla+Eg3qvl61sCp480vR6t/9LbJjRe835+iU2NjTkh7tsijduSb7a+evG/vkTa5y16j+kMN2+8xzdyy7aHpth9rXlBNuacoNNCZtG9+cWD+jK3lUQs24UZFbvhfM3Zq21grEgDCQC4wrLa7c69XAVe4yuspdh7tadjmQA4wHfgjcDNy1e6/bXhgf8zFwFuACbgVCgP4wxyaEEEIIIYQQQgghhBDiGLGytoOm7uA+9w9e03Wwz9NeWK9TmDMhncumj2bOhPR9JmB/+9vfcuutt/L4449zww037LV/7ty53HvvvbzzzjsoisLcuXN379u5cye33nor//rXvzAah674FV8cqYQVYv++DVwMvIrbvgm3NwjgvWHUjTqj6eSk3NCzwGbAGuoxXNrbaDa3ZcZ2dXlt5Jm6F4RiZpNZ6Z1kU81jv90eq/13cuLKP2fZzaPDka5Cf9ipN0f7/G2WD9UoKAbsuO2mhCz+b2x6Y6il19AZSdlwzapg3hxf0Rml4zIrx+3qPYfNQfvjJ/X9Y6vBEq0COHedehoQ2xrNrTeFfatXRa+u6zCQ5zXT26HXW2b7/HlA83fv/n/OM08pPHmOf5XPHAr7W8JdrojSluE39Vg2Zn3k6bF0vLtzzg4/KAUfj3mpd0LH9FbAB5iB6D4qYUcBJle5q65qYVXsMD0HQdze/kb823HbXwfOB34W33YB0ITb+9MBx2xFS8wKIYQQQgghhBBCCCGEEMDBr+k6lP72wnctq6ZxQMPGbLuFxQsKmTct55Dj+9nPfsZDDz3Eiy++yLnnnjvkmGeeeYZFixaxfv16nnnmGUwmEwCxWIxvfetb3HHHHUydOvWQYxAjR5Kw4svNbdfj9kb3M+IDtC+/PAuMw23PBCw6g/linV7NUFW2KQoLgK5oWHnMYI2d3pEXcwTTA136qLfLGouZFAMNoBpt+i7f1a2b9fottpjZZiyImGITUyb4Q0Zb9D2dgcloVbdKiz+/oKszLS2qN+yKBdNNwQbT5Kiu+Tsf56Zv72HMp/XmPN/tF2a8UvzhAwkUV552SmJeU1Lvjr6SsqLNULSxtLhyjGf0q5a6rOZ3HmztPSk/qD+htLiSjomN4/4zqc2yLjyqaWzt3KSdCRuaqkfVVwNrVZ26bYGnJCOqRDLLZ/5iQ6etuWdA0rVtP/enCzAexgTsntz2acBpQP2ArU1ADm77Wbi973whcQghhBBCCCGEEEIIIYQ45hzKmq5DOdj2wsPx8ssv8/zzz/PGG29QVFS0z3FpaWnYbDZMJhPZ2dm7t99zzz0YDAZ+8IMfHHIMYmRJElZ8ebnt+cAM3Pb3cXubhx7jXQGsiNxpn63o+IHeSBBIsGYE10WD+o8VhUsAJ/BqQlb4QQjXnBDiR13GSHPkg8R3fbmh85LG+BvNxkBjjmnTy4aaJNcV1WpyY3pyYsMpfbpTRvnCueP8s4FUwIvbG6y47t/PJHvrZmXnNHgTwhn1Wbp6Q6M1vOu2MX3vMObxDcAZwBSrvqlpVsKzFycXBd7oewGTx+HMd9Z46oApjtZZCQFTS8Pk8Ja+CEYboJvSManGPur1dmvfqRPNqqomhfMfrc94vANtzdua7faaCT5Td/qNq5asH+76r1ULq3o/35MwLJfgtvei/b0yo7VJ/v6A/U8CFwJv47Y3ASuAN4B/4PZ2fwHxCSGEEEIIIYQQQgghhDiCojF1WAnRA63pCpAzxJquQ+lvLzwSMbf2BDnhhBNoa2tj8eLFzJo1i8TExGHP9cknn/DAAw+wevVqFOXQE8FiZEkSVnyZ+YEeYM8G8G57avwnU39ytq/RMteaEZoNsXAkqGsnqijGhOhbaFWyesCO1rY4J1GlLzEazeg80dhV0frLdy7i/plp7DDuCE47Ia2wQR/q8/1pbb7+qikZwVnGSNSkaHG0oLX+xWfL/o/Plr3sqrKf98TjMY+DcPGHz04Bznt8+v/VZ/aOzR5j9JwyxrQ2OTHWmuPRje31Jo2zlhZXpgPtCaGUhNNrr536T91F9Sn6nfVAw/fUFyN5nd6cUOzd3p3JHVte6vp/W+d5vnt1feqGDk/2B5sqCh8OAD2rxrxkohgb4C0pK/piKlz3703gFiABbU3YCG7v07v3apXMN+C2/wIoAk4Ffg78DLd9Fm5v4xcfshBCCCGEEEIIIYQQ4oukKMqdwJVoRSd+tC6HP1NVdeMRDUwcdsvXN+7VGjhnH62B+9d0vWXp6n3Od+mJOZ+ronU4BsfctqkVU9TEfX9+gruKv8a8efN4+eWXSUpKGtZ87777Li0tLYwdO3b3tmg0yo9//GPuv/9+6urqDsdliAPQHekAhDhi3N5m3N7XcHu7Pttmz45Fub1js+3BXStSlrR9dbQTt13nazEvi0aUakXPzmhAp0SjzOzZYb18jclY8ZLNunWzXh8AZgJW4Af1Bv0HqzJi3/JltUztiuaO9kYyO171/sj8evcP20bP6XJ9Pac9Z1ok9IgNSoEH/R3GR3a8n7JJ/R/7NSfYXrwc+EppcWVePM5gadOzBmCyUfHl31J1Xd25W7718Y7QdE9PNOsxvUl9asdZV/k8J30nC5gAZAMrAYMvln7GrvAJXwd+91DkW9+/MeGCgo99C7bsCJ1QBySN7SpMPr32yqziDx8Yi7aOas21q912tDcqh964fmT14fZuwe1dB9wInIrb/u29Rrm9O3F7/4nb+31gKmABir/YUIUQQgghhBBCCCGEEEfI2Wift84GzgeMwKuKoiQc0ajEYbV8fSO3LF29RwIWoMkb4Jalq1m+fu8anXnTcrjprIJ9zvmXd2qHPG6k7CvmQCSGu7KFu/7yBE1NTcybN4+enp5hzfmtb32LTz/9lLVr1+7+l5ubyx133MErr7xyOC5DDINUworjj9ueCEwCNuH29h3MoQGvfnokoLsk2K0Lhn36qvTC3vOAsTmzvADPAatNidGLgl7DdDVGaEwkekqtyeRvMRra3kmwJay1mD/+0/e2bvzk97lPtxoiaRvznn/8O52bro1iqDErfVWzk5aehdZOONEEr+L2vgdQ53AWGmyRaxpXppyQm/HB67XJs3aclPCcHvcVl4VVU8ykPJIJauF5yffZCqyrrwb+AVgjEF4wOueMb/DRBWM61agvlvLBrvAJZwIXKITfnGJ5+4PG0GSzNzb2KyrMMfe62j/um52sVw0nAxuTaNqkUxRXBMtVxR8+8CAQRkvABoH2EXk+RpLbG8Nt/zXwB9z2f+P2+vcxrhO3vRGtelYIIYQQQgghhBBCCHGcU1V13sDHiqJcj9aB8GTgnSMRkzi8ojGVu5ZVD9lWWAUU4K5l1ZxfmL1HZWs0pvLCuv0nWYc67nDH3O+hVV7eqHyT884t4sILL2T58uUHnDc9PZ309D1bIxuNRrKzs5kyZcrnjFocKknCiuNROpALdAAHlYSNBnVnKiqKOTm6ITmv525zcmQqMAZt3ddcwGawqJkGS3gqhC3E+OeVvX21DQb95oq2NPO5a5TZnj8611x5TV8NcAvu7bG/3/JcS0BNnLAw86aORH27ClShVZ3OCv44LX1bxagAsKEnbFi/KWAYNTm4s/C6rJu7Q336cX3Npit1pti2yZZ3dmVFqnU2b2NOt5JcmGzpvhDI6IHgJb29Z2cZ6h12Q0Z9ri44uyc2ytcTHWUxK335EywfdE+yvNP8SvOPV+cHMuymAAXom1J8SXmtevznpxhb0/PNq7a3hCf3bg6eFbL4WxOC5rSoqtNvLykrCuzrPh1hTwK/A0qA3+O23wxMB55Fu68W4Dq0athbj1CMQgghhBBCCCGEEEKII8se/2/HvgYoimIGzAM2Da/3qzgqrKzt2KuadCAVaPQGWFnbscfarYd63EgY7rl3hW289dZbnHPOOVx44YUEg8F9HiOOXpKEFcejBqAb6Bq48b0f3T41opryNvjnvVpSVrTHF01c5a60Z3c0RsdnhCfFInTaRoUjisJ30Vr79gBPoCVjzwdQVXoVhSCwBZg9NhKd8aOGnu7WnUmh7JleJ9rapKuBFwNq8gRg6hPtv//gxqwb/r686ye72sPjZnw1/Y4TA+1KFtoLe/jX1+g3nrYherm53WrNa/dvalxl36kzqGstKeGtZ8/8yzRfu4FXW89Kq02PnXCV4f2msZFwairkzPYH08P61mgbmXU9/knjfdGUF4GugJpStHrb+eNTOjYbcvTvJbWn5E0yRaLOsVufC7VmnbyuKXu2qUeXGeuM5n1yQep97495/L8xYDLQ56zxtB62Z+fzcnsjuO0PAj/FbX8YrfXyGUAZWqK8F9gAXI7b+/aRC1QIIYQQQgghhBBCCHEkKIqiA+4H3ldVdf1+ht4JLP5CghIjrqVneHVEg8cd6nH7E42prKztoKUnQFaShVkFaUNW0e5rzoz5P9xr3JwJo9m0adOQ4++///4DxiTrwB55koQVxx+3VwU6B2+OqOazVJSJX0m7own3lnbgPCByX2rKmgk266kBhfN0Ohw6Ex8Do4AMtC+eJAHbgEwgs2eneVe4T/dW2mR/INBl8DWusrfZxwW6Q92GpvSpPafaC3xnoVXjGuOnXg68f+NDV7bDlW362//nfIe18saOyOi2UWO3vr9rRWo9cMnif0V37UxnW1aH3mKeGZk+fl7bORufzn4ocXRwdKDLkNez3bYyNdI1foc3JavDkJeQpba2A91PJpverjUYcye1nfBRZmTMfLRveG0FwkE1yRZSE6aj+g1bMjYHM/uybMkpU6Zk96w7My/Rk77FOv/tav+8nGr/PFtJTVGvx+HsBY5sBazbe/2QP+85ZgmwJP5oDfCtwxyVEEIIIYQQQgghhBDi2FEKTEMr3tif3wB/GPA4CdhxuIISIysryXJI4w71uH1Zvr6Ru5ZV71HhmmO3sHhBIfOm5RzWc4ujmyRhxfHPbTcCOQn6rz6dbdx0+ijTljy0330lFsV0TlMgkB3DNqpTNy6WRVBn4B1gF1oL4jBgBXQfWMw9MwLB7Shq2GyPpgHNOkPspMxpPdfEospHBr+y05wcPg2VELCZeLuLkuwrrgKSwPsXgDOT/27riWZ22XSdyxUdK9AqNz8yxGgd18p6IEVnVM8Bxk25qinZ83iuo3OzLTrhopbJLd6UqWOVU2NjwjU7XvT+P1Os0Zf4zeq/1ztrPCtc5a68tL7cZ67+9GedwGWGYPdMxUIgMjk3p6sv0+63Nb7SbrCtSfNltUV1aWqS3TupwPzRyvZIwVhAD+Cs8dR8cU+MEEIIIYQQQgghhBBCjCxFUR4ELgHOUlV1vwlVVVWDwO4+r4oysut/isNrVkEaOXYLTd7AkGusKkC2XatKHYnjhrJ8fSO3LF291zxN3gC3LF3Nw9fO2CMRO5LnFkc/ScKKL4NJwNRTEp/8GHgV7RtQ44E3t7wwypk9zr94bFLQFOo2r4wkRzymxGgL8BZaFewUYKurYGwPMPG+plblvFx/OlplbKUpMbbDYAmNVfTEknKDhmhI+UBnoA6tFXEkfv4AA/5fs+h6X7PoeqsBGzA6nvh8HwC3XQdMAP7s7zRc37XZtlhvif4l3dHbrTNyXXJiZ7vJH9r0eEfpP5N66ufkda6cAxS4yl0dwBkdCbs6S8qKXiktrqxM7nxvYpbuzZTMjKyGPt2V4wp6zswH/nnP/NtWn1F71eRpTZdsD/tDyc+7fv/GmyUv+Q/f7RdCCCGEEEIIIYQQQojDS9EyqH8CrgDmqqpae4RDEoeZXqeweEEhtyxdjQJ7JDX70+nXnDKWFz/dtUeL4OEct3hBIQAfbm3fZ4vhaEzlrmXVQyZT1fhcdy2r5vzC7N3H7e/c/cf9cr5zyFbG4tijqOpQvx5CHEfcdhta0nUTbm8Itz0VmA60Nq60TzTYovcZbVHVmhV8wJwYawVaIn5ls86s/lSnIxSFN4Gus8eODj6+s+mKvGjUCDyG21sVn1/fGRltUlXdxDTj9k24vcNbIdtt/yngw+19cMA2B3AxsKJzs21iX7P5kqDXePeE+S1htPVok4GncXs34rafGeg0TPW3m/6durSx21XumgD0VC2sanGVu05K9lkvH9s+bobdYNye2HXKuoKuE6cDm1bmVai95k6zq/Hs9dtTas5cnfdKIKIP31u1sKqnP4zS4kodYCwpK5LVvoUQxyJ5cyOEEMcu+aRByOu4EEIcm+Q1XBxxiqI8BHwDuAzYOGCXV1XVYRWhKIqSDHi9Xi/JycmHIUpxOAzVDjjVZkQFunzh3dsGtwjeXxth4IAthj/c2s7X/7rigPH957uzmTMh/YAx7+s84qg0rNc9ScKK45r/1oxJqkqPLSPsA2y4vU0AuO2JHZtst6qqMsmWGehG1WFND29FS4A21r6e+mb6FN+tkeTYxh9MSU1Ni8V2/vF7W2/Gbc9Ga0+RCaiex3MbnDWeYGlx5WnAxHzzyor5DyxqH1ZwbnseENkdk7bNAswCqnB7O0M/TptsSopOAFbt0uttj6Qk/7RLp1vx+5JtS3HbTwDGAK+6CsbagXbgrPhMqhLTTRnXOTW7y9LqyOkZ3zm9sWizLZTc2mVpGR3SBzKyewueXz/q3VNbkxrSRjV/vMTV/ScdoJSUFXWXFlfOBLKB10vKio7sGrFCCHHw5M2NEEIcu+QDXCGv40IIcWyS13BxxCmKsq/3ETeoqvrYMOeQJOwxKhpTWVnbQUtPgLq2Pu57ffNeY/r/UA1sETzwuP5q19eqm4ZsMTz4+OfX7uS2x9ceMLYHrpnOZdNH77X9pU8b+d6/Vw8rTnHUGdbrnrQjFsct//czLKE+/Q06gxpGa/drx21/IV6p2qc3x9r0xtgYsz3ao9NHq4A5aJWmH6ZP8Z/W12Lqqa23LG2fqv8mUTrja8vOBHYCrlCvPgEIehzOtcwtbc02erIvSLn3Uty/WYPbu3Z3IG57AVpy93nc3s/WIBj482fbAsA7rnKXgXLXaX/LNIw+NRCdAJy8uu3HM3t7UtV3nKUtuO0JuL2fAp9e+9C0E6/whq5cY1WerDMZTWj/87/2/PaWDb/SnX16QN/nz+wdw/pR77r6zF3LLth045+B0UDsxKZz6lNqNu+YsXblgtUnbjR2pU5pB15CS+jq0NbEPa64yl1z0aqbU6sWVnUN85jHgJSqhVWXxx+/BaytWlh1++GIUQghhBBCCCGEEEIIcXBUVZUvA3yJ6XUKcyakE42pnHFP5ZBjhmoR3H9cv4NpMZyVZBlWbEONi8ZUflVRPew4xbFJkrDi+OS2n4DOmOFvN+oTRgVjoR6dVWdUY40fp5zY63CunXylMsU+LtAM1AKhrlprvcEaOc+WFQ7rdBiTxwQ+SR4TeCeHntde3kEKYAL0QCpaJexyX6vJgtbWuKukrGgb7iu6gZsAFbd9G25vdzyaZCADSDiIK8gGZt8+KnPjh/U7moBcbyQvOrOhKXbrut4zYrOYoHPbl+L29lyz7fxLfIbgKZGct1q+3tlTP3XttDPfMReevia7YfTF7fopsSZXV9X4yA0dCeplQb2/Kl7pOglw+JXeQCwpcdKOnDNarYH2T7qgC6CkrKg2fm++cPGE50Lgz1ULq4oH7SsFvgeUVy2suv6Lj04IIYQQQgghhBBCCCHE0WplbceQLX77qUCjN8DK2o69WgQf7PGzCtLIsVto8gaGTNoqQLZdq64d6TjFsUGSsOK443E4E8adZxzd12Q6KRLQJQO2rlpb0JIamWrLDEV1OvW/O95Nm2awRZLUqBIyJUerUGJXhXrNo4y22FPm5OgMtCrQP6D9rVsG2IAQ8CRwNZCTUuDflFLgfxG3NxY/tQWtcnQiMBn4GAC3dx1uuwe3N7SvmF3lLj3wVWBn1cKqd6sWVu1wlbuWneHztwK3AQVXp9/x202VmblBu/GkWETZ+WTnvQmh7y29ONdknpCEz/N6osk3oy3z7HDAcG56b2hjV9qUnGg4MCE17Kn5xvLqqLPG8/f+8534aamlKeuUMWsnpE80RxMmbXBe+efbH7por74HrnKXFQhWLayKDd53mG0HrnGVu35YtbDKH4/FgramQ8MXHIsQQgghhBBCCCGEEEKIY0BLz/BW19vXuIM5Xq9TWLygkFuWrkZhz3U1+mtXFy8oHLKS9fPGKY4NkoQVx6MT61/PyAbU9Kk9HnNqKFVnjKX2tZqa9Xo1KWNqz4ltnsRR3XUJYy3pgdHWrOC11owwapg3u+ttD2e6em4G0oDzAZu3wazXG1WXNT38hN6kfgA0ApOAQqAK6APA7a3Hbf8LWrXsnonC/SRg4wxADhDFbc8EUqpgK25vDLf9YeBsoy64Q6fGVgW7jMsMv+mKbL79P79LDNrPU0Mn1I7u+mQZMdsrSb6THT1pUSW7oWPctvYZ4zpthq5IVvbTzpef7oTdyd6s/3RGTCldm7esmnR9TfWoD7IaU7bW/r38p0rVwqrdrxOuclcycDZQD3z6eZ6QQ7AamABcCfwrvu1KtPu6u0LXVe7SAT9Dq0DOBjYBv6paWPXUgDEXA/ejrZ+7AigfeCJXucsNXF61sGr6gG23A7dXLazKH06wrnLXt9CS5VPQfh8q48e3DOtqhRBCCCGEEEIIIYQQQnxun6dF8KEcP29aDg9fO4O7llXvUdmabbeweEHhPtd0/bxximODJGHF8WgzKPl6UzS5u976eJar9wK9KTK2e4elR7HGgmZ7dLItK9jRXZewxZoRbItFlTlqWAnpLdFP2zYknRfu0z+fO7vLCGQBBXqTej6qkhH0Gntspa0x4DXc9vfQ1lXNBrbuPrPb2wF04LZbcdtnAlW4vb37C9ZV7spHS+yWApEQlOjgYgM8AjyJ29sCPOlxOBOAm+vGXuCvLK6MtKftnBLWhQNpjaGqLn/6pT965Rp97vj3x0Xsxqnto5JyYtb6WMg48Xflpz/97kPlj+RVLazaATgA1x035a+5tPq2hLE9ptxNo9Yl9po7xwGrS4srI0ABsJ05BIAWoG0En5uD8QhwA58lYW8EHgXmDhhzJ3AtUAxsBs4ClrrKXa1VC6vedpW7xgDPoN3bv6Ct6XvvYYjVCPwS2Ij2e/MH4DG0tYCFEEIIIYQQQgghhDhmKIqiB64HzkX7rEs3cL+qqkVHICwhhuXztAg+1OPnTcvh/MJsVtZ20NITICtJ27+/tVw/b5zi2CBJWHF8cduNGVOTpnVusU6wZQen2ccFfuCtNftDfn1mNKAfR0wh5FdqYhGlAVANFvUv9vzAybGwoutusHmAMd46WzR3ef1OoA5Yaf15yvpwn0GxpIarcdsLge24vT247Q5gFG57k+fxXDNaG+K1zhpPKP7zLCCcv6jiU0CpWzI/6HE47cBJwApnjSfgKncVANcAH1ctrHoN4OcPFiRc2dObVBgMddoGXFru7E57NKSc1E5PPdAb0Yfu85m7dJntW7f1JuR9ozNlQsbHvbmzE3Sd1saM2FvbE1rzVo5+ZEOf2Xsy4Ad2ANt1MdV63Vu56TFr/bSdmaOa2m27FMBQtbAqWPph5QnACYC1amFVFfDR4X3C9msp8BtXuWtc/PHpaPdqLoCr3GUGfg6cV7Ww6sP4mG2uctcZwM3A28AtwNaqhVU/ju/f6Cp3udCqZ0dM1cKqRwY83OYqd/0AWOUqdyVWLazabxJeCCGEEEIIIYQQQoijzANoSdgKYD0MmSMS4qj0eVoEf57j9TrloNZu/bxximODJGHF8WZ04uhAdsBrqEvICjYbbdGE5o1JWYFOU45q0wdSxvbF/C1mNSEzTLqzZ1frupS6YJdvpjUzNMWSGq4CngeMHofzNGCLs8bTov911wa9224HTlNjOPqazF3qTVkVSbmMAoy4vX087iwEnMAutGRnNdr6sNvQ1pC1An9FS8DOBbqAtV9d91NTh7UpqSG1ur7/AtJisR1RRak1QxTAVe4yANGqfL8RqDmDp99e13Ttp3/61c/D/ceUFlf+AbjUF7Nu8sXSXicafi6U2JnfaWv6BC2ZHAF44jeRCJDfm1A3IaZsb9mVfOYLAUPvDSftOP/Uu3/4r43p5EwE9MCWw/LsHISqhVWtrnJXBdobPgWoqFpY1eYqd/UPmYi2Vu9rA7YBmIA18Z+d7J1I/pAR5ip3nQy4gRPR2lH3fztwLNrvghBCCCGEEEIIIYQQx4prgKtVVX3pUA5WFCUK5Kiq2jJoezrQoqqqfgRiFGKfDtQi+PzCbD7c2r7PqtV503Io/cZJ/OL59XT0hfc6fl8thqMxdb/VsIP3n1+YPWScaQkmLpuei91qIhpT95uIPdA5xZElSVhx1PE4nBZgBtDgrPHsOMjDtxtt0Qy9MTa3uyFhkxrzzVT06gx06o5dxpmNvW0tuuRN297LPtmbYEyIjkVRp3XXW9v7ms2xaEA3FS1x2gksQFtf9NFnF9w1J7PgW6edmvp4T2+twde2ISlr9Jyu76GtG/tm/LzrgMbd8bq9UaAG4PKf36nvwxKF+aCtSdqF9g0y0n2jR6f7RocmtZ8cw203A2f/RKvgrAWyW+5OPe9XNsstjyclPYvWstgINJSUFX32lx8oKSvyP3lr2Vh9l7dw3PqKDSeveacZLmwuLZ5lA9SSsqKO+NBoW2qhceuEy3LDhoR2my514dwtKS5F0dlS/JkJaO1020vKivwHed8Pl0eAB+M/lwzalxj/73xg56B9wYM4R4zPvlzUzzjcg13lrgTglfi/bwKtaMnXV9ASwkIIIYQQQgghhBBCHEtCfL4ijX1lgMzxuYU47PbVIvi16ibOuKdyj6RnzqDk6vL1jfyqwrNHAjYtwcQv5+87Abt8feNeydSB8+5v/3s/K2JlbQevVzfx7NqdtPeFeOT9Oh55v26v2A7mnOLIkySsOBrpgXFoSbYdAB6HMxmIOms8fbjtOtzeWHz7KCDirPG0A1ry8+f2tsTRgbTueuvkaEiXpOhUq94Ubk7p2eo1Jqg7bVmh6c2r7agxZZc1PXS53qS29+6y/BM40Zoemp081v968xr7q0Cjx+FMs+Wde2H3Vsu0Osa+qO7ofTYa1E8zWKM2wI/buwIg3oL4s4Sx2z4GmAy8e7+JhG1GQ7Kr/KmLudNQWbWwam18jP67Wdb2is6f/+2KP/1gB27ygZMBYyxKmneb7WJLWqhrVlZwUlo05ojfjzMBC27773B7faXFlQloScTgefaXZqsBX47qb53icTgVZ41HBc4GTrhj0R9f/N2SH2xw1niCpcWVH6IlLm8DsEQTNgT1gZ7mpFr7C9MerKpaWNV+WJ7VQ7McLZGpoiU1B6pGS7aOrVpY9fY+jvcAlw7aNnvQ41Yg21XuUqoWVvV3fZh+EDE6gHRgUdXCqu0ArnLXzIM4XgghhBBCCCGEEEKIo8m9wG2KonxfVdVhtyJWFOUH8R9V4DuKogxcpksPnEW8cEWIL8LgFsHL1zdyy9LVe/XXbvIGuGXpah6+dgbAkGM6+0KU/Hs1D+tm7JXgPNC8N51VwF/eqT3geR95v26/YwaedzjXIonYI0+SsOKo46zx9NWdPqHdYIuOjvwsddzm57O3AzOByM4L8tXUycZcm9v+Nm5vEzBNb46quO3bgQbP47mhtClJZ5lGxRw+khPUXq/VkhbKiPgtplRD0ynW9ODbyWMDYzuUhDpbWuRtxRCLBDqNSRlTe7bUdJ3RNO2EVdcn6AOnNq+x1wPbAf/oXe+ujOpNfZGwf4OzZlMf8BFu+zwg6nE484B84ANnjSc24DKmAFOBDcBT/5ueNgeYBKwGmuJjXD2G4LXGsb+phh88gttbh9v+H0AX8elv6Nll0dc2nhOzm3UvnpW4rAytErYv2K0/01tru2797XfWw7kloHvnnzMWv97Wmpt1cnNm86iEWKAx+zQzDmf6uPxLYmucJ2XXpq67ZfGPH2vL6hv7AVpScyNgB/QdlqbnsnsL8tflvnkG2hukFw7fs3twqhZWRV3lLmf/z4P29bjKXb8H7nOVu3TAe2jXdDrQXbWwqhwoA37sKnf9DvgbWpL7+kGneQvIBH7qKnc9BcwDLgK6hxlmA9o3+G51lbvKgGnALw/yUoUQQgghhBBCCCGEOFqcAZwDXKQoyga07oG7qap65T6O+2H8vwpQTHy5tbgQ2rJpxSMaqRDDFI2p3LWsesgFjlW0X1r3CxsAZb9j7lpWzfmF2btb/h5oXoC/vrt3AnbwnKqqDvu8w7mWwXGKI0OSsOLo4rZbYxHFmXmiITfYoytqWmc7E51aQUypBQr9bcaxtkxjky0j3N+2Yl3WiT3ZaInQENBgTIwa/b4En68zIVGNhnTW1FB94uhge8SvSzMkhhtjYSXFaI31mpIj00PdeqW7wbo9FtLnfjr3ptHe2IzIlPqnuqCnAK0l7Yf6WCikj4WqJ1/ZuAm33YJWfWlH+//nG2jrjjYDmwdcybvABtzeJkVRJhek95rOnmnd9ujy3qYBYzb9y54UW5aYMPnOcpe9amGVF7e3DsDktrdb04Ne/c5Anqf+nPrpTy6Ntzm2P962Pjk3FlOnjDZWnbw9dGJWVDVmXd2c/bXNlmD008SLPk476So1arCcryrKWcnddaFJPQ04A5YZjSRuQVurNAtoA+4O6QNbm+21347pI7bEsL1i0DUcFaoWVu0vGfpLtErWO4HxaK2eVwO/jh/b4Cp3XQXcB9wKrAR+jtbmuH9+j6vc9b349l8CTwO/B24aZnytrnLX9fFz/iB+/p9wFCWzhRBCCCGEEEIIIYQ4CF3Aswd7kKqqBQCKorwJXKmqaucIxyXEIVtZ27FH297BVKCpe/+r3KlAozfAytqO3RW2B5oXILafevL+OQ/mvMO5lsFxiiNDOYhuAkIcVh6HM82aERxrSQ2fGurVJ6Co34oGdamBdssTwIvGxIjDmBDd5ms2v4PWgjcKpCr6mC11Ul/qqOk9dUBGLEJr93ZLSsOqnClG/BOT8nz2hKxIorfeUutvMxt0xlhrLKTPScgNpHgtY1zRgH6LeVfTfZVzSy1AniNcsdax48lL+5os+mhYFzSYYzZzSvi1sWd3nNtVa81q/Cj1Xec1u0zA2C0vZnnCvYaJQB9gBV6MtwFGUZQ04F9olZUAGHW8Ho5xdf8bEFe5KxfIqlpYtdbjcKahVVF+6Lxm18W9u8wTGt7JaI3qjC8ZYqExQMhZ46n2OJw2e4EvPfFkk+t1b8mkBF3nNamGnTk2XdeL/1EvfDCmj8zJ805ZYwx1X2oK9vRMtH5QkK5unlCnnvZsa2DcSX5TyriIJdUGbAPu7jF1XG6KWFzmmO2/JWVFr38hT7YQQhxe8uZGCCGOXfI1bSGv40IIcWyS13BxXFAUJRnwer1ekpOTj3Q4YgQ9v3Yntz2+dkTmeuCa6Vw2ffSIzzvc8w73nAPjFCNuWK97Ugkrjiap/jazOdBpXKNGdT2WjIAu7NefBvQC4dGzOzssqeHp3garr/GjVCuKGgCy1KhuakdNUk8koPPqDbExlvSILxYleezMxtXe7RZHNGCY2bTa0kxMZ7SkhaqCXmM7uthpvY2WhvaJEzcrUX8ki6biordK/uqs8azCfUVedJRi7tkZ6vS3mZL15tiktMm9b6gxNvtbTZOS8vxzgHWAeeIlLVtxezd4HM6iXltO4spTfn5hZXHl6pKyoha0BOz5Ay8wHOMc4N+47dcBC6vgA9zeD+K7Z6GtV9oCFCTmBk2FNdX/jK+He7feEg3htj/pvIbluL3bge3Nxa/+xGV90Zqob+tY77vY1pxVe8VH45YFz9n8zbQpbbOSw6ZkX6162p2u0DOZfLiz3pw5445ua15nb/rES/tso6ITkz682BvIHNOonmBFIbW0uFIpKSuSDz2EEEIIIYQQQgghhPiSUxQlE23ZNYCNqqq2DvM4PdqSYOeideXTDdyvqmrRCIYpxLBkJVkOy1wjOe9wzzvcc36RsYmhSRJWHE1qgZ2ODRsDAB6H8xS0as12YEfAa+jxtRtn+dvMJwA1ubO7UlFV+64VqduBMU1WJWdLuin/ZJ+6Id0Qnqkzq/lqUJcZ9ht0xAyqISFsSZ3Ul+dvN84Mek3jokHdaxtGX1s2d8utFwGXArM9Dudyc0pm4ti57dWdmxOioOaZkqKd3Q3WvrR/N67O+Flqts6gZgDr+pOnpcWVWcwtfQvIQ1srIV1RlBQGVMAOoAfmvbw5/P0ZDstFZSnJJzxV7tpVVdsQypiWuLZtfXITsAlIR1u7FcAHLEse50tCa7mbhtveB6TFeLZnY+CcLUnh1sTWyERrrtfQDGyc0HbSBLSWyZXdyujQUvN/ohPGPHt206hTT9FFAm2jWtdsNdknhCwJnWeZ6XT09KS815ucvxW4srS48uWSsiLfiD6zQgghhBBCCCGEEEKIY4KiKAnAn4Dr+CyBGlUU5R/AraqqHuizwwfQkrAVwHqk04U4CswqSCPFZqTLF97nGLvVgNVooLk7MOQvrQJk2y3MKkjbY94cu4Um79DHAOgUUNVD/x9h8HkPdM6h4hRHhiRhxVHDWeOJAQMbmb8H9OjNUX/OrK7s3p2Wjq5tCe8DjfkXtG5RY+QHu4xWQDWnhTMD1tjEXdn66sIqo90X1FksqWFdUkHfK6Fuk7lnp9ViTQsHk8cF+rq22qKxiBJLyvNFSrKvmN49x7J85wdpKuBFUcebk8K3dW6xbQl0mLYD1kAH27rrbeE0wGiNfQjYcXv7AEqLKycaQ70X5ja+1+ZoffbZrHk0JhtaDN/X1qjdp39siHQnuwwNNSbTuDHhcD5wdua03obMp3Y+WlpceWJlE9nACyXafYkAf8VtV4DUj16creSc1n1tSnJbYY6xuq0xXLgrELGPAz4c1Zf/ZNXCqp7SDyszgCbgFLQ3SmMCppQzE3q2T+pKmVDTmnFCsCchryAUSn3M6m2aY4gF3wL88fsfG5EnVAghhBBCCCGEEEIIcSz6A3A2sAB4P77tDOCPwL3ALQc4/hrgalVVXzpsEQpxGOgUhf+5pJCSf69GYc+kaX/v2cULCtHrPutEq9cpLF5QyC1L933Md88s4C/v1B5STEOddzjnHBynODJ0Bx4ixJHhrPFsc9Z4np0wv+WDpNFBLGlhK9CdMqEPa1q4sKfBmtT0ccoMxaheYcsM2HP61Pq5LxlfM9eb3or0Gf4Z8JrW6Q1KBINhnD85q7ArmtsZ7jWsDHqNfwp2mR5Kd/h6VZVRvlZzKtrfKR+qsjMaVXz+NtMU4CRgubPGU+Gs8YRw23W4vZ24vXUDwmxO71gfyW5amaVGlXHJhpZMYDSwdX/Xdv4ptvFbTaaGLSbju7sMhrZYlA/r3kjP8Dic56K1+JgKJOxxkNurugrGZnnyRz3/SfMF36vquyhzkuXd2d/JuvY14EagrKSsqCc++m2gEi2h2mjra0wKmZLG27tr25N6d3V5kwsipnB3tqm5/YLkXR/4m4x/Usrm3FYDvAScXVpceSZAaXFl4ud4CoUQQgghhBBCCCGEEMeeq4Bvq6r6sqqq3fF/LwHfBb4yjONDwJbDGqEQwxSNqXy4tZ37Xtu03ypYgE5fmNQEEw9fO4Ns+56tfLPtFh6+dgbzpuXsddy8aTn7PebOiwu5/bzJw4o3LcE4rPMe6JxDxSm+eFIJK45ubntiJKDPrH8jjaDXcCKQ4W83rti10u6ypIS3p0zsXe+ts/lMCeq27u3mkMWnc0YhIaDGpqkdxkiw0/A4dvMpMYvNkpjdnRvoML4UC+vbnTWehpqpU6bp9GpeNKT3oLU9bnHWePo8DucfgK8D49DWo2XXz+eMzTSaZxnd9pW4vQ394ZWUFfV4HCWPAXkR9LXADiCsqmpEUZTlaGvC6gdcUSwrb3SnM99Un9/XVf+79NRNQM3GJ3M3oiV984D/AIklZUVd8XtwMRDB7X1VF1MzLP6W5JBxYrQ6cO5jl6f9z/l1wZknAR+WlBWFBpzndLRvq3UCoYA5NS2mM8Zs/pYHzeHuZWN2vOWy+lvbduae6erTpTlTeuuc337zTB0WUtAqYkOlxZWTgRmlxZUfl5QVyZsmIYQQQgghhBBCCCG+HGxA8xDbW+L7DuRe4DZFUb6vqqq0IhZHzPL1jdy1rJpGb+DAg+NaegJcNn005xdms7K2g5aeAFlJWmvf/VWWzpuWs99j8jOG878O/PKSqWQnW4Z13gOdUxx5koQVRy2Pw2nOmJZ4SmJOUAfoQfED9Ym5waxYREmJBnU92TO6ayJ+RQ+xsWG/7nWgFyU2R9EpEwwJUezjfRcmjup6IdvcbbSZe8c2fZw8E+j2OJxvg84QjZIFXAlUA3UAzhrPTo/DWYuWEHWVFld+nGm48YxphhcnTDK9875xUJzOGk+Qzypf/fHYDf8eO+7BH7T1jG7zdbj6xyYkJK0+9ytXvvfjjHXlH9y8rbmqf8dC8Dic9wPRkrKiCNDhcTgnAqdPukIZYzCr3cCr625Y/94/l99X4U8oOC2sJo3fHDjLY1T8P0vVb88Cvt8/XWLP9vYx2183Rw22TzdN/trJMYN+xWzrUlUX657zlvmnyWkd1VfYu7d+3JeU5zaqmSdnN0XHpkVSO3flYQPeQGtjfCqwEy2Re9h5HM4DvSG7y1njccfH1gAFwDhnjadpiLnOAe5AuwYr2nP7MvAHZ41n5z7On4P2BnEmMBH4o7PGc/uhXIsQQgghhBBCCCGEEMewD4G7FEW5TlXVAICiKFZgcXzfgZwBnANcpCjKBmCP8kNVVa8c4XiF2Mvy9Y3csnT1Qa/DmpVkIRpTh5XYHGrcnAnp+5x3OLKTLfucYyh6nXJQ48UXS5Kw4miW3e5JzO/ZYW0Neo1mW1YwX0Ud173dUp2QFfrY12n8ODJRd23qJP+pKNRPuqS1yfN47lpUXQhddL1tVCBqSYlYzcnRD9prVKM+V7+2c0uiD0gHYs4az1qPw9ncZZ94sTnQeak12D7e43D+NX5uG1qL4hAQtdesze7s06Vs6czOcv6WxmHEnjndanW+Oin37tfGWsa+uO7djncbIu/39nZvHnK0236y8xqsuL3vxdd+NUFuLlCwcYXz2VXOH43uK64cB3hJPbEUWA2smmp9JWtbYFZDUE0wlxZX5paUFe0CmPXJkm7AGFEM6qbJX9MbiIaSgo32FabrLwFla0tqanS5Szfz4/G3/bdqYdXj8Ps9wiktrtwBdJWUFa082CftcxjYH+FrwN1orZn79QJ4HM4z0BKrTwELgXsGTuJxOG8GHgLK0Vqn1AFjgeuAHwM/2sf5zUAr8L/ADz/XlQghhBBCCCGEEEIIcey6DXgF2KEoyrr4thOBAHDhMI7vAp49PKEJcWDRmMpdy6oPKgGroLXy7ewLccY9lXtUz+bYLSxeULhHi9+hqmyHGtdvVkEaOXYLTd7APuNKtRmZVZB2EFGLo50kYcVRw+NwKoDRWePpb6vrVaO65mCXzoyWFNUpCrZwryGvq8c4EUXNjUW80WCP4lOj+taeBqsNcAJF0T5jS9dm47ae+th70ZAuCJzesta+LWNat76v0TLF327yArXOGk/j3254pj2/rsI8uvGDKGBCS/zF0NprUFJWpHocJRsAL7ALt30sMBl4H7fXD4Dbng9kAytxe2POGk+jx+F8quCEbaGfjvef/9MZNgV4EsBV7jIBkaqFVbEBl38GkORxOvoypibelDa5rz3/iq57/7v9t/q0juozEsLNZydZmlPbIwUfqKHo5hTv1hKfddRSe9mm0g+KKxcB18Zjx+Nwnhgy2O7stE8Y25J96mbggZvLLo49cZXnvY6U/JSYwRztsJp6G9ODUeKtkkuLK2eiVf5WlJQVhUvKiqpH9tk9sIEVrR6H0wuoQ1W5At8G/o227u0DDEjCehzOPOCPaFWsAxOpdcA7HoczZT/nr0N7g4nH4bzxUK9DCCGEEEIIIYQQQohjmaqq6xVFmQR8E3DEN/8H+Jeqqv5hHH/D4YxPiANZWdtxUC2I+2tcLz0xh5J/71092+QNcMvS1bvXWt1Xle3gcQPpdQqLFxRSvHT1PuPo9IV5rbpJ1nM9juiOdABCDJAHTPc4nInxxyra2gO1wHO+FvNjvmbLc6hKPWBTdLHAtlcyX2nfYO8LdZlyQz2G0cBUtCrJGHB6NKSbgLa2azLQ/fZJyklVk9Q5EUP8z6PbrpuV8eQWXaLxXmApYAdGof3d7UrK89XhtludNZ5XgH8BmUGvIQfIiJ8HgK5a6xld26znM2BNBGeNpy7lH027gJfQWm78YvP/ZbjQkogXDbr2fwJ/RVWSQj0GJRrStb/St8jut2aeq6jRuYU9z/kd1rfqE3TtG7ObP7Zlta5RJtS+0J+s1qN9C+2y0uLKWZ4J5+Z4bWR2pkxJas2YfglwHsDVT/+4+8T1D/8iqafBPbZN1/zNFZe8VrWwqqa0uDIlft1TgNxDe+q+GB6HMwn4Ktpz9Rpg9zicZw4Y8lW0ZPRvhzreWePpOtwxCiGEEEIIIYQQQghxrFNV1aeq6l9VVf1x/N/fhpOAFeJo0NIz/AQswKhkM6XfOIkX1jUOWaXav+2uZdWEIrF9VtkOHBeN7T3i/MJsUmyDFzz8jLKfY8WxSSphxdGkF+gGQh6HMw2YBWQBPWhVpjsTsgPbrRmhcd3bzehNsZNiYX1asMtU5a0zrEX7fc5Bq2Ctj8+ZCqxHq2JVq5sSLaPCtI5VlXS05O6oExJeKuBktrLM2wP0eBzOHrTWtAl5Z3TNAiy47c9D7njgK9teztxuTIiuDvcZzuBx5+vOGo+vebW9V2dQ2xpXpoac7r2uay1wGeAaHYmsAZou7u2z4LafDbyL2xuLXyvOGs/yv97wzPZQb0pKSfYVGc2hibqG9vx7DRu210xxrApM/e1TgQ9OXaBvHnVKakeqs+Usbf46IAKcDcysGR2c1ORMSM7v6HvRrCizgenAqx6HMztJZ/rK9LV/qu5IdzojesvE0uL89wE3UBO/z9MH3Luj0TXAZmeNZwOAx+F8HC2p/W58/ySg21njGU7LaCGEEEIIIYQQQgghBKAoyqXAy6qqhuM/75Oqqi8cYK5a2HcnWFVVxx9alEIMz3DXX+1379XT0SkKjd4AbRX3EQv2kXXlL/YYowKN3gD//LBuv1W2/eNW1nbstVbrytoOunzhoQ88wLHi2CRJWHHUcF6zq9vzeK4PGI2WkN2FljydgNb+d1rEr1+dkBNINtrCdd7t1hY1RD2wBi15exWwEy0hZ0OrlIw5azxBoNvjcM5c8JZS35rMS6YoXQB1r2X4rRmhsC0ztDOpP44aj9fjcFqBbH+bscWaEY7h9qo1q3/VmNjT0Gr3bt2c1NcYQ6u41AHEwrrXY2HMA1opD9QQ9unuM1hiqk2n7qiqbXChVf2eCHziKnf5VinKRIuqxgBC5pTzgdzeaPrKUaYtvqSpbVsfzXo0//UmNpYAa0764RjgLLT/f58vKSvylhZX/hH4OjA1r8NkUNWMnjUTo7+e3UDeyQlPBXtvvmYypDfXjZuX252cH5245ZlHDFF/ninYNTFkTlGAJrR1ZjtH5Mk8fG5Eq4LttxR42+Nw3uqs8fSgfVnogF8T8jicvQPncNZ4ikc2TCGEEEIIIYQQQgghjinPoRXCtMR/3heV+BJn+3H/oMdG4CRgHvC7Q4pOiIPQv/7qcFsSt/UGhz13fYdvWOOGqsYdboXuwVbyiqOXJGHF0cFttwCuUTO8kebVdgsQBarRKluDQDowOeg1NkYD+jp/u3mar9GmAo1oidoMICE+WytaEtYQT8z1W2sJox/TjgUY63E4t4Ep099uinRsxInDORFYgbZwvC0hOzA9GtKFgGUAu3JPH2MOOLyGSKDt+qXfqgGq+id21nh60RLH4LbPAhTc3o8API/nJgBFwEbnNbv0wAzgI+B13N5eyl3XnTF2tPHf5f7/Rh1OC3NLXwRSEn+1rRq3/dVHWx89DbgXeAX4BZCkxKIJVn9rcv/5S8qKukuLKyuAXiOTnzxnXYPjyndfVZw1D6zHfcWN3Z2WK3Sm2Kv14y58AQgn9e0IAFNnr7w7+M6Zf/gd0FBSVjRwjdqjjsfhLARmA7M8Duc9A3bp0Spk/wpsQmtRnHOAatjpA37uHulYhRBCCCGEEEIIIYQ4lqiqqhvq50Oc64GhtiuKUgLM/DxzCzEcw1l/daB9Vc7uePhGkmdeRvIpl+3eVnbbVYTyTibljG8CUH/PJaTNuxX/1lUEategT0oj9ZzvkJU0e9jnOdRx4ugna8KKEeVxOPUehzPF43AqBx69h0jYpwv1tZhSUNT+r52kOms8m4EOIAVi0yE2v6068c3uetsnaMnWArRvUXWiJeCaAB9QOygBi7PGE4lXxaYBmWhJWx+QiJbszUOruJ0D9KUU+DwJ2cFewFhaXJmqxKKTc3e9X3DKJ/eMPsC1nAzcjNse7xZMKB5rB1pr5ApgFW5va3x/i9WndESbzAuBBSXZVxhKsq+IJ3S9kS5z8xyI2WPExpcWV7oAT1br6vfH1b+c9sbcG3e/aSnJviLlsi03Np64/s9Xm0OdJ8bvDcB7oV7Dy7GQbm1JWVFlSVnRuy0Z06uBtwzR4IaSsqI6YHppceV5pcWVB/u8fZG+DbyDVkE8fcC/P8T3ATyFdr9/OtQEHoczBcBZ49ky4F/L4QxaCCGEEEIIIYQQQohjiaIo1ymKYh5iu0lRlOs+x9Qvo3UzFOKwmzcth4e+MQPdfj7xVoAcu4VZBWnMKkgjO/nAyc+eQIREs4GB03rf/w8JjjPJufFPWMfPpP3F3zPRDtGYyodb23l+7U4+3NrOyeNSybFb2FdIA+MRxwephBUjLQstwVmPVqE6PG5vZIvDWWMbFZiX7ugdE/HrX/XW2VoAxpzVbmldn7Q5HMCl6NSJOkNsbiyi86FVMO4CGoBkIIxWGVvnrPHsryVtPdDqrPH0ehxOE9AObESrNDWjJWO9yY82r8Jt1+P2RimuNKk6/Q5LsLNdp0YbcNsnAzrP47n1zhrP4AXpnwh6DRGzPZKM2z7PeQ1GoA0tOZiI27tl4OCqhVXLPQ6nHq0dRx1wOaC+dvviLS1pK69rHRfIn9JhT+jzzZ7bGnIGfvLgZTd6HCUPbS245JsNY86/uqa4cm6irmXtWcmnXGIZt6ujs3dStH7sBcbu5AKvU7u3mzJgU0b8fKXFlWlM++63gY9Lyor62ydPQXve3onHeVTxOJxG4FvA/zhrPOsH7fsb8COPwznVWePZ4HE4fwg86HE4k4F/oN3TPOA6tGrlH+/nPNPjPyYCmfHHIWeNp3pkr0gIIYQQQgghhBBCiKPWo8BytNbEAyXF9/3jEOf9ClqhihBfiItPyOFBTuJ7/16z177+ROjiBYXo45nar88ay//7z/7njKoqoUgMw4A5EqedS0Lh2ShA6lkL6flkGX9+6hUqOkft0RI5x27h0hNz+Ms7tXutqzdUPOLYJ0lYMdI60P529B5o4GDOa3YpfS2mWLhPr/o7TDogAbe9LzGXxMTc4KdbX0636AzqebasULLeyBs9O6yJQIOzxhP1OJzjgISkPP+WaEhX4HE4O9AqXVvQ1oWNAngcznS01sbbAOJVkAPfTPgYuC6q2xsFiCcrV0ARuO164JfBbn0yilrtcTg/dNZ4NvQf4nk8NxsYN+qkLlvaFJ8RiKkqCUHV9j1V1W2wuu0XAhbc3r7d167FV6Gd0x4E1Nbw+OuqdVtdfv0mxWFsDO9I3aF/3XDSa3AZlXNLdwFPAFcCZ/bGsmbVBmbarPbutWunf+VeYKbV17KvCs8Q4DUFvXnLz781ZeuEK8YArwLhAUnZo82laM/bs4N3OGs8Ho/D6UGrhv2Rs8bzkMfh3AT8JD7eipaIfRGtanZ/Br4anwx8Ay1pn/854xdCCCGEEEIIIYQQ4lgxOD/UL49hFN4oirKGvfNL2WhFIN8biQCFGK6LT8ilTKdw17LqPRKi2XYLixcUMm9aDtGYysraDtr7hrc2bCgawwbYbUYAjFkFA+acwVV/TeKBFz8mYdq5exzX5A3wl3dquemsAl5Y17jPeMTxQ5Kw4nAwof1uRQ/yuFhCVmhj5xZrn6/ZfCrQ6nk8933nNbvWA7GQ1+wy28OtalRnjQZ1E4CPnDUeFbc9UWfMTo6Fdb1pzp6J3fW2Ql+rcROqLoLWZhiPw9mEVjmbBNjRFoOPHOyFeRxOxVnjjfpvzXgnElRAVRIZmLTVGIC01vVJH6dN8VUDM7t7Es8PtJrTozarw5Lde72icAlu+zrgL8DVaGuatqMlCstwe9XO4so/vp/WXOv0Jl7U0JwRXj5t07a1CSs+jp/jW2hJyQeA9/QRf+bU8AslFqvfNqfsoe3xCs5vexx3/XlwpW5JWVFvaXHlx2N2vPknUErQkr/vlZQVvXmw9+NwcdZ4HgMeG/D4abS1X/c1vnDQ49eB1w/hvPIVIyGEEEIIIYQQQgjxpTQgeaoCbyiKMvDzUz3a8mfLhzHVc4Mex9CWa3tLVdWaEQhViIMyb1oO5xdms7K2g5aeAFlJWstfvU5h+frGvRK0/RRliO8jRLW0hwJYDNqKn985eyKXXjp7dxvhYCSGVd37ewxq/LgX1jXy9h3n8El9517xiOOLJGHFIYmv+WoEwoNa/yYCqWgVpcP72kg/tzcEbOotKrAC2+PHOzyP5/qdNZ46Hne+EfQat0SDOmMkoI+vmWpPApzjL25p3/J89kZbeiQ50BHpQdXVxWPRo31D6xy0ZGkFsMtZ4wnFj43i9vo8DqcOOAHocdZ4ttZMm5KZNDqQOPqV+toB15wJnOVxOLdBZgqw3VnjqRziSqqAh1CUen+H4ZxIQHe1wejDoNDdV6NvbutIyE539HXo9JiAM4Er1BhTghGLfkf31L6JGZ+UAZSUFbWXUPRY97dHnYtK9pTHkm7948qq/oRvJ9qazrGSsqKVHodzXJ89XO036Nbatf3daN9K250Ij6/3mlxSVuQFDD2JeRF9LLgL+AAY3grlQgghhBBCCCGEEEKI49Vz8f9OR1u6bWC3wxBat7mnDzSJqqp3jXBcQnxuep3CnAnpe2xbvr6RW5auHrLsG0BnsxPt/ayDdizoI+JtBrSEalO3lgI5tSB999wfbm0ntp/FElWg0Rvgk/rOveIRxx9JwopDZUZr8zo42doRf9w31EEH5LanjDmLsQ1vppn7mi3m+PzdMETrYLc9ARgPRIzWWKuzxhPyOJxeYBxaBWyds8ajehzONrRkZGc8YRzCbdehrYMaREuagrambBggeUzgAoM1enLLlXkPZz2zY7PH4RyN1vrXh5bc3Ia2/iylxZWJaMnNXfE4Y7jtncA9PbtMuRGf3mmyq2vM1tA2o8FobtuQlNpWZX/Iec0uD5AL/C3kU/re7v7O7+ojp4ya6L5cjV9fEqDqDOa/RIO6nAkrt+yuuC0pK1o28LY5azz1Hofzz8DO0uLKROaWFrDnmq8As4G5pcWV/wVqW0bNfBB4q6SsqPVQniohhBBCCCGEEEIIIcTxoz95qihKHfBfVVX3Lg0cJkVR9MDlgDO+aQPwgqqqB9s9UYjDIhpTuWtZ9T4TsACWcSfQV/UG1omz0FkS6Xp3Keh0+523pWd4/9vsa1x/a2SpkD0+SBJWHKrQoP8CEE9yHvR6sAPnjQR0oVCPISv++HUGVdR6HM4UwGYfbw/lzvIGgCbc3u747lh8vB6Y6XE4tztrPE3A2j3O4vbGWq7IC0XDukiOFnestLhyGxDQ3hWo62IhXVZfi8mU5bbrbdlp43xNllRgpbPGs83jcJqAWR6H02SYe68r3Vjv+OSku9bZ/C0ZKRP7XsyZyZmAUY0qnWGf4f3ksYFHFL2qxCLKNF+LaVUsxvpYRCkJ+/RzzMmRm82/7+rYUlzZgGmPv/nFQCTxzy33AZQWV+qBi4CGkrKiTwffOGeNZzNAZXHlKGAs0MRnCWbQksZ1QEdJWVEX8GRpcWVhaXFlzlDzCSGEEEIIIYQQQgghvnxUVS3/PMcrijIReAkYDWyMb74T2K4oynxVVbd+zhCF+NxW1nYM2YIYNYaiaIlW++yriXQ10/LU3ejMCaScee3uSth9yUqyDOv8Q40bqjVyjqwVe0yTJKw4JM4aTww45G9C9fM4nLloFbXbnNfsMnoez/WjtdqtRavML0BL6m7zOJzJaBW2OcAE77aEVu+2BBOw0+neHVcUWO9xOK2AjXhl61DaPUl6wNrlcOoq55YmWX3Nrklbn0n0OEo+ddY0rPc4nFvj1zgleYz/xHCPoTvcZ9gRP9yCtpB87wX2e6MWXa89mNya7/ebleSx/iQgLRbliZ0fpN1gTQvrs07oqQL6kscENyePaT4J+E7nVpszFlYmglpsdtsfLMlmy7KOX/hKi3EBNSXZrGHPdXVtwAwgBdhn0rSkrKi5tLjyD2hVu/1tiHNLyorq0JKwA52HVtUsSVghhBBCCCGEEEIIIUR/FesPgavRij1MA/erqpp2gCn+CGwFZquq2hGfMx1YGt83f6RjFuJg7bMS1efFmKolPHVmG5mX/WyP/YmucwFtbddsu4Wt0dgelaqzCtI4dfHzNHkDQ1bZ9h/Xv35sv321Rm7yBrhl6WoevnbGiCdiper28JMkrDjSLIBl0uVN6ZGAbqw5NZQZ7DTZ0Co469HWcg16HM5utNa9rWiJxHa0BGIh4Ac+Hjips8bjx22vI56I3IdtaO0wxgE7FTXWYfG3x9D+DmpzAE2XjvHrzbEdsYjSFj8vzhpPt8fhfBkIqm/XX2/I8wd6u22xlAl9Odb0kBH4722pWbHb7OFZCTmBid0NlgsMlljIlhU6HRjTudV2QqDdsMpgi/3NaIsZgXzgq1Osb7XuDLmSQbXi9r4+MNiSsqKe0uLKh9lPq+fS4soFKqrl9ykBH9BeAiuAU4BzS4srXywpK6oadMiTaFXDQgghhBBCCCGEEEIIAbAY+A5wL/C/wP+hfX55OXD3MI4/mwEJWABVVdsVRVkEvD/SwQ6kKMpZwB3AyWjFPFeoqvrc4TynODbVte35MXs00EtwRzWBhiqSpl+0xz4F9kiO9qcpFy8o3CtpqdcpLF5QSPHS1UOeVx3iuP21Rlbj57trWTXnF2aPWJJUqm6/GJKEFUdaLYDBErN2brHpw336BCAJrToWtL8xk9BaDPvQ1nUNAH6Pw7kaLSG5dq9ZtfVi89HWqN0B4HE4RwHp9gJfQ+6pXX2Q2ws0A50lZUUhj8O5BTgNrY1xxFnjaQbo3JSYAjQ6azwfexxOXXx92P61aXXBLuP2YJdxW9IY//ZomNJdK1LGXnRt8q/ywuGSV68JdH13l69s60tZ5+hNsfyUCX2nWFIiq3u2WyoDHcZgNKRf1VGTyJSvNO0EJuSbVwUuTPltTmt4wtdKi80bS8qKvAMvaxjrt46OQiJadWsLWhK2FvD034dBstAqbIfa94UqLa7cX/t9gLtKyorc8bE1aFXS40rKipqGmOsctDdbp6L9LtUBLwN/KCkr2rmP818J3AJMR7t/GwB3SVnRKwd/NUIIIYQQQgghhBBCHLO+CXxXVdUKRVHcwH9UVd2qKMqnwGy0atb9CaJ9xjtYIoOWtzsMEoB1wCPAM4f5XOIYtXx9I/e9vnmPbe0v3U+ocTPJs67AOmk2oCUlfzm/kF9V7JmszB7hZOU+WyPHqUCjN8DK2g7mTEj/3Oc7ElW3X1aShBVHVHwNWQBfk8O5Di1htgtodtZ4wvFEqz/+b48vnDhrPDUeh3MjYKopdIy2ZoZax729rb/y1Y+WhByYxMzXGWKjrRmh7IhftxXY7qzx1A7YH0VrhRyiv9rUbU/InW1t37UiNRpP4hrQWgLXA+OBXc4az+sAHodTZy/oS9Ubo9m3vxOwemZEoycHgqlGW6xz9OyuUFt1Yo0a0Z2ogrrqtGhWR6shfMa7nB6L6PqAVcAjJl2wzRdLO7c6POPkVl1sFvDaQd7SvxlQFLQ3GyHYnbh9rrS40lBaXPm9zJY1Ka7qv/mAR5lbOgdIKi2uXFFSVnSgJOjhNvCv+tfQvlU3ZcC2XoDS4soz0H5PngIWAvcMnKS0uPJm4CGgHLgKLQE7FrgO+DHwo32c/yy0+/1zoAu4AVhWWlx5aklZ0ZpDvywhhBBCCCGEEEIIIY4p2WidCkH7TM4e//lF4FfDOP5F4C+KonwbWBnfdipQBrwwgnHuRVXVl9GKMVAUaasq9haNqSx+fv1e27Ou/MVe2/7fRU4uPiGHC6dlD7ttb39V674MVdW6r9bIgw133P4ciarbLzNJwoovnMfhtAFRZ40nOGiXDa3a1YCWBAPtG1M9aInZxMHHOGs8qsfhzLakh86xj/Ntw21/H7c3itsbAwZXSK7TGWO1SaMDiTs/SM0BzB6HczOQDPTE15NdN/AAf7vRYcsKmgzWaG/Er7cA69GStfOBcHphTxdu+3zgbWeNt3fnBePeiumUS85ep55z5seGf42fF27HSlpSXkCXlBd4PxbhcdXAmYUBfrfLGPVOuMTvVXRqA9rftndwe0P/+P5rVa9bw0u6rMFJ7oNMwpaUFUUASj67fwCUFlfq0f6G+nVq2AxE779Ml7kt7/6qyzfcvvkoSMAysKK1tLjSC6hDVbkC3wb+DbwNPMCAJGxpcWUe2jfx/lhSVvTDAcfUAe+UFlem7Of8tw/a9PPS4srLgAWAJGGFEEIIIYQQQgghxJfFDrSCiQa0tV0vAFajLXs2+DPdofwArUDiQyAc32ZAS8DeNtLBfh6KopjRuuL1G6qCVxxHVtZ20NwzvILs5njSU69Thl2BeihVrVlJlmHNPdxxIx2fOHSShBWHhcfhNKK9YHU7azyRAdsVYCJalWbNgO0mtLVZdXyWiA0B29GSqblAksfhTEBbR3Y0sM1Z4+mNj2s0JkTbgNi+Yoq3MQ4ALT6HMwxEgDRgDBDxOJxeYEd/da7H4TQaE1NtBmtUifj13UBD5dxSa2H1I8FRLZ9cpoDeYI28h9ZGAwCDJfrvWEQ3wZYZyvbWJhRtfj77DaDBmh7qCXQZ/I6vNp0V9itXjO2NZmX3kBlG12vLCPcAC0Ixa57vF46/3vNgTSh/UcVLM4L6lNLiylklZUUr+RxKiytnAfOAR0vKih6FImARH5S7vgG148rm3LahhMFLxR6dSosrk4Cvon1zrgawlxZXnllSVvRufMhXARPw26GOLykr6jqIc+nQfoc7DjRWCCGEEEIIIYQQQojjyLPAucBHwJ+ApfGq1rHAfQc6WFXVLuAyRVEmAs74Zo+qqlsOT7ify51oa+CKL4mDqSat7/AdeNAhzj9w3KyCNHLsFpq8gSErVBW0FsizCtIOOp6RiE8cOknCisPFiPYNIiNashPYXbnaxGffgMLjcFoBPdo3rPxo36YyehxOc7zyNepxOHe8Ol0pNEe46jSP2m6MspPPEq5N/lZze8LD2/b4+kp8XjWefB2sA62iNYpWgZuBVi25y+NwZgKKs8bT6HE4a8O9hgy0tVPbgZzqwhu9mW1VT+ljwYnNn6SuT/tXU2/8fFnoEsPOqxvvbq1KnJw2pffyaFD3krfOZvG3mxKArLbqxA2hPt08gzXSoOhUpd2T1JiYG9jFuDT1E903T9genF5UDMvrlsx/s7S48gbAWVpcWVVSVuQf6ibnL6rIB64Gnq9bMn/jPp6LPrTK2MFf76kEsqoWVnWVFlfOQWuv/ERJWVGYo9c1wOaSsqINAKXFlY+jVcb2J2EnAd0lZUWNI3Cun6Al2J8YgbmEEEIIIYQQQgghhDgmqKq6aMDP/1UUpQGYA2xWVXXZQcyzBTgaE68D/Qb4w4DHSWifU4vj1MFUk45Lsx22+QeO0+sUFi8o5Jalq/dckxEtAQuweEHhiLQH/iKrboUkYcVh4qzx+DwOZwxI8Dic4YHVsM4aT8ug4blo1YsbnTWeGIDH4ZyL9remMj4mbIxSAOSFDewyRtkeP4c+3kZ4qP4BE9GSrHs0YI9X6Z4NtDlrPCs9DqeKlqT0OGs8UY/DmYr2t63RWePZ4XE4W4EkZ42nt7K4citgnHzR9k+NCdFpHdEx2Uuv+fP4wpp/TrZCkJgS8jye+8yok7tONBmi+lhMsecur6/zOJwPAcHWT5N1o2Z4+0zJkc5Qt9GgN6nOsM+QvXHdjI/qCmeBlnTti1d2LgOS9pWAjdOhJbr1+xoQT1huGLy9amFVE5+1bM5CW+vByIAE+VHoRmDpgMdLgbdLiytvLSkr6oG9XqOGVFpc2TtwjpKyouJB+7+B9g24y0rKigb/vgohhBBCCCGEEEII8aWhquqHaK2Fh0VRlKeBlaqq3jNo+0+BU1RV/eoIh3jIVFUNMqDFsqwje/ybVZDGqCTTAVsS6xT41pz8Q5r/UKpa503L4eFrZ3DXsuo92gVn2y0sXlDIvGk5Bx3LSMYnDo0kYY9zHofTDMScNZ7DlliLtxg2DHEOE5CA1gI4steBn2mKHz+wlbAPSPU4nNnOGk8TYDhzg9oQNLLNFqTGWeMJehzOJGC8x+Hc4azxtA8xbyMQ8zicKYDOWePpbyurQ1tntiv+uB5tjdr++DcDisfhTAZ88WrcoMfhNBbFx22anjE1bXJfftNY58aIISErYE5JtAbaHWjVpNUta+xtyWP9v/HW2RpS2d0KGY/DGetrNr2mN0crYhElN9RjSPJ3mqf48nMv1oV9H8SMNgMws7S48r3fpfhTgaaS/dy4uiXztwH/N3h7aXGlCTCXlBX1lBZXJgOxkrKi3r0m+MwywFBSVjS8ZvhHQGlxZSEwG5hVWlw58A2cHq1C9q/AJrQWxTkHqIadPuDn7kHnuQb4G/DVkrKi10cidiGEEEIIIYQQQgghjmaKolw63LGqqr5wgCFnAe4htr8M/PggwhJixOl1CnddNo3ipav3O+67ZxZgMugOaf5DrWqdNy2H8wuzWVnbQUtPgKwkLRk6EhWwIxGfOHiShD2OxZOjJrRq0MNZ3WgDzB6Hs8dZ4wnHK03tgBcIDWwH7HE404Cws8bT07/NWePpG2LOj9DWC8gAmuLzVhuChJ01nkg8uRxBa18cil+rMjCR66zxtMXPWYj2u96fhE0EqvsTt84az66BJ47Pnwjkx4/Z8d+v3JdYqDdPMESDPcC2aEDf1/pp8oZpTzy58e3iygafLduf6t16OzAKSFVjynhvnW0KYPI4nCuA1fHrzOrdab2wd6f1E2eN55EMLb7f23vq+mItpnd2jj57HbBguz56MXAi4EFbg+FgXQ2MKy2uvA8oBiKlxZXlJWVF7QClxZWpwKiSsqIagJKyohhDVxMfTb4NvAMMzkvfEN/3V+ApYAnwU+CHgycoLa5MKSkr6iopKxqyDUppceXXgUeAa0rKiipGMHYhhBBCCCGEEEIIIY5mzw1znMp+uvLFJTL0Z41hIPkgYjpoiqIkonVI7FegKMp0oENV1YbDeW5x9InG1CETmvOm5VB27Qx+9MQ6fKHoHscoCtx0ZgF3Xlx4yOf9PFWtep3CnAnph3zuwx2fODiKqh6wc6c4hnkcTh3auqiH7Yn2OJwGtGSvP77mqxVIAbqcNR7/gHE6tMRmyFnjOWBf/XgyV3HWeAav9ZoO5AHb+6tbPQ7nRLQ1aKsHX2s8Hh1au90+IBOtOnivFr2DYs0CvJVzS2NKLDIlv/5lW0H98s3OGk97fP94tIraINqbD138mtV4jJnAXLSE9CpgHXABWnL1I6BG0cdONlhjq8O9hmzgncq5pQnAeXl1FVubQ9svWZM5+ZmH//u/u9sp5y+quGJqe+2Y379b+ud4he6QSosrZ8fv0bPAOWgVpHqgNB7zdcBo4KGjsd1uaXHl9cD9JWVFKfHHRmAn8D8lZUVlg8Y60VpOTyspK9pQWlz5PeBB4FHgH2hr/eahXXNvSVnRkN+2i7cgLgduA54ZsMtfUlbkHbGLE0J8EeTNjRBCHLvk69ZCXseFEOLYJK/hYjdFUVYCL6qqeveg7W5ggaqqJx/Gc88F3hxiV7mqqtcP4/hkwOv1eklOPqz5YnGYLV/fuFeSMWdQkjEaU/lgcxtPr9mBLxTllPw0Fp6Wf0gVsEPZVxL4aHG0x3eUG9aNkkrY45yzxhPzOJw6j8NpAYL9jxnBxGx8vdeB7YYNgIVBv4Txc+8EdB6H0zooQTsZbdHzDf2Vs/tpoexDSyT64pW1aWiJ0MgQCVgbWgKuESiIx7kdmOpxOBsHtCgefE0x4uulVhZXJqg6Q1dtwYK6i1+5r791rQ4t6RtEq361x+Oa5nE4m+OP30Zra3spMAO4EK269+/Aa8DJGVN7L7CP840yJsTux+2NOrXq4ac9jpJZ44xJhpkdO40D43J01I37yuY3L4yhNLCfb6eVlBWtGPDw9dLiyibAEb9vVwJTgNeBtn3NcZS5FEhniKrgkrIiT2lxpQetGvZHJWVFD5UWV24CfhIfb0VLxL4I/GE/57gJ7Xe3NP6vXzlw/ee/BCGEEEIIIYQQQgghvhR+BTyjKMoEoDK+7Vz4/+zdeXycZ3nv/889+6JdsizZji3HTjKPsydgEhIgFYQmuIGULkAPFLpwUI4o3Whxe1pC2nNOTZuW9pyq0e+wpiTAKW1ZUiWhgEiAkMQQZ7HjZ5LYkWzH1mLtmn27f3/c91hjWZJlW15kX+/XSy9LM89yzyNFo8x3ruvifcBpnQertX4MeVPABe/RXQPc9cCOY97dNziZ4a4HdnDf+6/jtita8XoUb7psBW+6bMWijz07uLx+XT3P7BufM8isrGo9FwPPM1F1e6GTStgLgK0oDWFaQHgwVat5J+6mluDYtZhAd2rWbWsxbYQPz7HPekxw+pwTd4s2KL0C06ZihxN3J4450adq/ZjK1BE+NZm1t7UO7qhpGn+5SmHmuGbmCGHrgHXAfkyIVw5OLweedOLuglWgXR29UWADMNrZ3X5w1rGVrXqtwTyx12AqTgtAK/AdTIuNzZiq01cwQfBny9W9hU/Ut/vCpRCfmny48tif/Y1vhIE/yQVr3c7u9q+Wb3/s2jcGGzOTt/t06UUn7r5i1xjGzHOdbtvaEwEi/du2zBuudnX0Xm3X8R+d3e0LzeoVQojlSv64EUKI5UteMBPyPC6EEMuTPIefR5RSn1zo/tkVrvMcYwvwp8A1mJFyLwD3aK0fX4o1ni5SCbv8FUuamz/de1QFbCWFabv740+0n3AIOld1rUdBqeIv2NnVtvPtN9d2YllZ1A+PhLAXAFv56q++KF3KJTxV2fFgARNY5mdtpwCPE3eLcx5o7mNXY0LYxKzb/ZiWv0U35gQrW+f233TxRq1YmRkJvggkgEswT8SHZrceBsoB7JWYisY4n5octbdfChTcr63ah2kNXHDibt8ca/RVzJG9DBgFxiorcefT1dHrbet/+MoVh5/zVCcPvmCrfhdzXdZjKl+jgB+4z4m7i25p29XR6wHuAAY6u9u3H2fbj2BC7b/+m7r0ezAB+P/p37Zlrlm7QghxIZA/boQQYvmSF3CFPI8LIcTyJM/h5xGl1LOzbvIz02Vwr9b6ujO/qjNDQtjl78m9o7zvs08dd7uvfviGE6oCna+6drbyL8Nyte18+83eTiw70o5YgG1D7AcIN2WVx++PlLLekXzKV7KhZK6iejQE+N2Yk1xsEOvE3ek5zqkwP1sZN+ZUAQ1uzBnHVKDWr3vrVKmY9SQPbfevKOU8QWAY0yp5rmHtYP4nfMpuV9k+eC8mAC65MSfF0S2RK9dYuPL+K9ua/puXf/qnYgoTQM+eVTtne+bO7vaiG+scwoScJ/LHZBrTnngNMHUiAaw9bwn41iI33w3Udna3F/9ma89OTMh83IBZCCGEEEIIIYQQQgghZtNaXzv7NhtOfok5xoUJcS4Znp67AvZEtpur5fA9D+1e1LsFNSZIuOeh3bTHVs67X+V2t25qOeuticXpISHs+c+H+e85mzgYDhXzSudTvhImmPVjgsty4Jq325ZO8ZwRTGveKSADJDGhYBAIjr9SNdZ0+fRQKeepApKzq2iP8anJArDHjTn1wBq+5hwEQrCqFhh2PgVO3D0I4MacAFCcI0QOj9SqohN3X6680Qawjl3fq7ayN1W5vxN3BzAzZRdkj9UCTDhxdxD4ZzfmvBXY6MacNzhx9+lFHCOKac38fHk27vF0drf/qPx5/7YtO4GdXR29VV0dvRuAnTbQFUIIIYQQQgghhBBCiJOitZ5SSt0NPAR8+WyvR4j5NFeHTmm7uVoHN0T9jCXzc24/Fw0MTGb48pP987ZFrtxue9+YzGY9T0kIe/470pI2NRzUwAQmeC1g5sIWy7NNbavdpZgRmsHMnk3bKtVJoMme+9DU/rCa2h+ux7QEXvxvLhPuRgEvUIeZszqFfYxuzPFi2hJngH57WwBo3LaSV7b+pm+u6l5tt8+4MacBE8juA147kQdcsb4mwOvGnDzgd+Lu992Ycxlwnb2t1om7P1jgGJswc2WTwK4TObkNgavsfN43AW/APLaXTvyhCCGEEEIIIYQQQgghxFFq7YcQ56zN6xtorQ0xOJmZswK1PBN28/qGY+6br3XwiQSwlfaNpRa13WKrd8XyIyHsecCNOT7M/NVjKh7LLXZtQKkwVaLl3yFFu2/EjTmZBdoBH+/cASfuVv42UcwEsApoxQSmhzGtcj1ANSYgzFccq9pu+9qs45UdArx2vmvJ7uupeKxFN+ZM2OOWVQN1Fw+R2vnBnce0BLbXYq89/wZMkLpwZe48nLibcGPOILDKPo6sG3N2Aw/YTX4RqHNjzg8XaPf8AqaN8aKC066O3gBwNfBiO7wRuMmNOZ/nlq7tmCD31ZN5LEIIIYQQQgghhBBCiAuTUupjs2/CvN75AeCRM78iIRbP61Hcfccm7npgBwqOCVQ1cNOGJr793EFaasNcv66eZ/aN89pYkk9++8VFtRxerHUNkUVtt9jqXbH8SAi7zNnqxzCmgnXeOaA2oEzZfytb9mpMO+LFt6z9VK0P8PKpySxmjmzAjTk5W0kL5gk56sacvZj2vBdhAthxYC0miD3E0WEpmCfz8sdcj0EzU6k7Zj9PztpmcNZuY5gZsEmObxAzv3ViEdvOZwJTpVteZ8FWpuLGnK9jqmPnnbfrxN0s8KIbc7xuzLkU6K8Mx7s6et8GXAn8387u9iSmcvdWTKvng5gK4KnO7vYp4Ien8DiEEEIIIYQQQgghhBAXpt+f9XUJ8/ru/cBfHW9npdQngXu11qlZt4eBP9Ja/8VSLVSIudx2RSv3vf+6Y9oKl/3rjtf41x2mGaZHQWkpk1dmqm0/cGMbn/tx30lV5Yrzg4Swy5ytNs2xQBthG7r6gZStTA1igtfy7NPFBJRHTB8M1hSznnDi1rZhCJcDwoAbc8rVuGl7Ph+mqnQY2G9vC2Na9o5VVOSWH8sUpr3w7PV7gWbMrNW03TYHjBxvrfYci3p8Nqg9alvbojjvxN3pRR4jB7zsxpyrgEswwfNBN+asBYaduHtMNe48YsDbgCeB7RW3e6io/gVcTBC+y67fBeBTtVWPT324ZlfqHW8DftLZ3b5nkecVQgghhBBCCCGEEEJcwLTW60/xEHcD3ZiOf5Ui9j4JYcVpd9sVrdy6qYV/7N3DZ7738rzbnY4AFuDuOzYR8Hnmrcqt3M7rmbMuTZwHJIQ9D9jqyYXMri4tAcdtPezGnCAmgCxV3la1OuIvFSmlBsNV9rgFTLAbcmNO1om7I27MGQNqgHpMsFnAtA9+CfNzl7XH8wNBJ+4u1AI4jKkuLboxJ4MJcZPllsU2WI7a2075V6YNfdcA05gq3nnnqtpzM8d5X8WEo1VuzFkF/Cpmxuuji1zGq8DPsKGqvU43tsMOJ+7+Z3mjzu72HPD0HPv/1lWRnrpdqXcoTAC+B6Cro3cz8HPAFzu724cXuRYhhBBCCCGEEEIIIYRYrLm6wIIZqzZ2htciLnBf++n+M3q+ltoQd9+xiduuaAXmr8qdvZ04P0kIe2HIl4NaO8PVA3hZuHrWhwkRPTb49AEFfzRfpUu0UlJJTEiZw4SrCvNOprytvA1iQthyq2SPbVc8MetUTZiZtLn5ZtLaWasJTFVtwO4TZKay9zJgAya0HJrn8azAzK49OP9lOsJr150F+uzjKweuzUCior2xA6xwY87TTtw98hvUrvmwvQYHgR3MCnLdmNMKbAG+78TdvlmPOQ08UXHTWuBmu67vzPH4FFDde0tXDbCqs4UX6n2HisALtzz+uw1urNDsxN1y6Fpi7j+ChBBCCCGEEEIIIYQQFzil1DeY+/VDjSlY2QN8RWv90qz9xu02GnhZKVV5DC9QhamQFeKM2N43Nmc74tPhoz+3kZs2NrF5fcMxla3lqtztfWMMT2dorg7NuZ04/3iOv4lYDtyYo2wF5+zbg5j5rF4AG4Smjlc9a7dLY4LIWsxs10DN2nRQgy837S1Xtxac9x7yO+89pDAtjiOYoPJizJNtH3CoYl7sbGPA6OwA1o05dW7MCVU8hiqgwa57P2Z+K8A6TCgbxgSz84nY63Dc32p2La84cXfQibtT5RbImFC6BVhRsbnXHrvOjTnRivWX377yKtCAme06OwAOYa5t+Hhrssf5JvCUG3M+5sacd8+6/03AH9Qk+n4FeFvX4Dee41OTP6wff6lqrP6yPwTK2z8L/DuLaOUshBBCCCGEEEIIIYS4IE0C7cB1zISq19rbfMB7gOeVUjfN2u/3gD/AFOzcjZktW/7oAG7WWneegfULAcDw9JkJYCvPt71vjOIcPY69HsWNGxp51zWruXFDowSwFwiphD1/hLBzWYGME3fzNnAMYJ70ShXblgDcmOOpbDU8mxN3c7YiNor5WckrD1ld8BwupP1TwFrlK+lcwjs+fSDkwwSSQUzF6gRmHmoGCNpK1BEn7uZnnSYHrHRjTtCJu0N2XQGgEdPGeNCJu1k35rxmj1We3Yobc67GtDvuA17D/HEwn/2Ap/eWrlBvR2+ws7t9ovJO+zjBVK5OAE1uzCk4cXe04nqk3ZjThwmny7ftdGPOCOb6r3NjziTmnWA1gM+Ju9NuzFmJmYc7+/r2uTHn7524W+zq6H0dZg7sv3Z2tx/zzGDbHe+26yzaj0oD3lBx+Bb1mesPDF0Z2l33K6/v6ujtpf6ym/O+iMr7q3Y4ZrubgbcAXwdeXOB6CSGEEEIIIYQQQgghLkyDwFeAj2qtSwBKKQ/wD5juiO/FVLR+GvN6IwBa6/vttn3AE1rreTsxCnEmNFeHzti5/vEHe4583hAN8D/edQXvuEpaDV/oJIRdZmyw6se0GK58O0U53KzGVDhPMtN7/8i25Za69t+UG3My87UBtor22B6gbuTFmnFMSOkFqnTBE97/WEME1CimMnYU80Rcbt1ba7/2Y1obBzCtiSuDxvK7qYAj4e8gFXNrnbg7fczCPP620YZNK9OhpoPrXvv+hBN35wxh3ZizGhMO9wErgXBXR2+ys7u93Gb4UqANeAUTphYwM2jz9vHgxpw6TAXx1Kxj+zDVsSG7fcR+jGPaLCsn7u6prMB1Y06T3WYlZk5sGhMmN2JC8znfntPV0buGW7quBr7Q2d2e7uro3Qi8GfhmZ9x9hU/VHhx+ofqSTL5mdcnjexsQBx5LVF/0suv8+qF2c5jdmFB931znEEIIIYQQQgghhBBCXPB+C7ipHMACaK1LSqn/A/xEa/2nSql/BH40185a68eVUhuUUr+BGSP3u1rrYaXU7cB+rbUUh4gzYvP6BlprQwxOZk5pPl9d2MdkurDoY4wlc/y3r+zgt/av421Oq7QgvoBJO+IT5MYcrw0Sz9T5lBtzQhWVmn5MoHhUgO7E3YJtmzsBZNyY47HbJmcFngrzfa/CBLZF+5jCle10K46rgcOYYDCACVVr7ed7gZF8wl/MJ3yTmNbF2om7JSfuFjHBqh9TeZsBLsWEhqvKoaQTd7UTdw9UzCstB8VHHqN9LMco+MKvJKKrBhrG3RuA29yYUz3r2q1wY07DrN0GgIPlANbKYwLfYcy7vKYx7X/32ePUAu8EZrfXONLe2e7fjwlfU/baXgVsdmOOd1ZgvtJeh3cBf+nGnNd3drd/F/inzu72o0Le2afDDK9fab9uwcyKrQPgU5Op0d3Vf5jQTb+R91d9CTiA+Xnwq1L+w19/97232LX9tLO7PbHAeYQQQgghhBBCCCGEEBcuH6Zr32wxTHEOmNd758yklFJvAXYCb8CMSauyd10N3LOkKxViAV6P4u47Np3ycXJFjcaEKyfi8z/ex/s++xS/+7XneN9nn+Kmbb08umvglNcjlg+phD1xAcBnW9WW2/r6mXny8WGegIqzgreTVQ5TwVRo5is+P0a5qtXOUQ1gKlkLFfeXbJVpFBOOasyTYADIuTGnCshVVsc6cVe7MWcaE4zm7PZ+TNvhVzEVrxrbDtiGliVMKJvDVHo22HMkMDNgj1wbWy1aea182MpQOxdWuTFnvxN3JyofazA39XLL0PZXIpnRDcAqTDvgV+x6Evacfvv4B5y4qx2zpqPm4Tpxtw9TJYsbc3L2WJWtk8sVvqPM7VVMMK0wFbFRTCC7DjM/dvbPwV7gIGYu60ZM6EtFZW55fu/slsOPA/HO7vYD9usngOc7u9uPVAk7cbfkmKB5oKuj18HMZ/hhy9BPQ239j7SnQw3XTte0VXd19P5NZ3d7GiGEEEIIIYQQQgghhDjal4HPK6X+F/BTe9vrgT8F/tl+/RbmH3e2DfgzrfXfKaUqOxz2Ah89DesVYl63XdHKfe+/jnse2s3A5MnNiE3lzEv1tRE/E6nZExcXb3AqQ8cDO+h+/3XcdoW0Kr4QKK2XIie8cNiqTI+tgCzfFsGEsDlMeAimItKDCeBy9vMQJqBVQGGxIa09pz6RUNdWk/rmmME617YhuzYFhDHB5SFm5skWMa1yS5iAr1xNqzEVn+VZsPV2nxWYULPFbr8PaMXMePXYbQ7ZcNeLGerux1Sh9mNC2LDdbhUmeH4NuAI47MTdI83V7bVvwVzXGky4uRqYwoScQUzl6FC5pbG9niuBqfJ82YrjrQauBHY6cfdgxe2+yu/5PNfRg2kznLSPrRbzszI+z/Zzhq02CMeJu6dUrdrV0bsauAN4pP2xTg/gPHHDXw5lQw0NwPc6u9vlP34hxPlKfr8JIcTyJb25hDyPCyHE8iTP4ecRpZQX2IoJTMtd+YaA/wN8WmtdVEqtBUpa69fm2D8BXKm17rMh7NVa61eVUm1AXGt95gZ1niClVA0wOTk5SU1NzdlejlhCxZJme98YAxNpPvntXSSys2ugFqaAlTVB/vZXr2EkkeWVoWn+8Qd7T2ot9RE/P/uzW6U18fK2qG+eVMKeIFv9Wpp1cxpQtsp0DBNohjHVkeXKy3LVYRXmuk8yU9W6mHOeKAUEbZXpUTNf3Zjjqaji9dh1+DBhcRJTyamw1bEc3Va4xj6GabtPtd1/hX3MChPKKmAMUz1aDmqrMFWllb/dyhW7q+w2Q5gQtQ54Hhh34m7RtoCuB/K2yjeCefdVKVib9+kSO3LT/pds+LnHnkMB6zGzXMsBbK19PHX2/EeFsPZxHcKE6EcsIoBVQBMV83fnm1Fbccz5fsuf3NtxjjUKPIANhYE+Z46NFqi8FUIIIYQQQgghhBBCXGC01kXgfwL/04aSaK2nZm2zf4FDTGCKcvpm3X4tpohGiDPO61HcuKGRJ/eOnnAACyZUGJzK4lGKd12zmif3jp50CDueyvPUq6PctLHppPYXy4eEsEvABlzl4C2HaevrwwSBOUyoWa4wLQeqRTgS3kUx4WS2HOBVzkyd77y2grUwqyq33Bo5Z/+NujFHMxN8+jDhbNruV40Ji3PAsA1nczb0LNnjZ92YM4SpOC1h2v1GMFWuI/ZxZTFh7Agm4BzBVKNq4AZMyPoSJuAcqwgqU27MeQZ4xq6lFRPMFjCtete4MWcCUyG7HxPsXmyPe7knWCyEGnKHPX590Yaf7hm2xyzYawEm/E5XXK9VmKC1n1ltie2+U8AL813z45groD8hNjRfMPA9AUd+Lo8jgln37EBaCCGEEEIIIYQQQghxAVJK+YBbgA3AV+xtq4AprfXxOvh9Dfi0UupXMK9PepRSNwH3MtPOWIizYnj61GqgyvtvXt9AVdBHIntyL+c/uVdC2AuBhLCniQ3SytWXRUxQ6WemHXEQEw5GgTZ725Qbc4YxYWY1oMtBog1CPYC/8nPMvNTK+bNBZkLRaUz4W8IEbQpIYQLZkA2Km7BVsBXVscreFrXbTWGqUMthbR/myTPNTDuK8hzYAUwFZsKJuwVbZVmw65iw/7YCB8uzYCseT51df799/Csx1bNBTACbwAS7HvvYflLKeYKFtNdH5th3Xtlrsr/i64wNk1NO3F2qatNyJWnJibsjJ7l/uf2ywlzDk2pDPDu47+xuzwJZN+YE3RilBVpTZ5CWX0IIIYQQQgghhBBCCEAptQ54FFiLeW32u5jXYz9hv+44ziH+FOgCDmAKhXbbf78C/I/Ts2ohFqe5+tS6YTdVBXly7yjf2z140gGsIS/JXwgkhD0DbCBbcGNOOfjzM1OZqjGtib3MtOwtYkK5oN3mMCZUbQZWuDFnLyYALdj9q9yYk7Qhqrb7FjBBaBLAjTkl7NxXW31ai/n+5+35tK0ULYemzZiAdtqub9reVz5OwK4hgAnxxrGhLWZ+bHnGaxATvr5kH8s6TEvhFNDgxpzynNhB+xh9mErXtfZcz1asu9+uMWDX/CJa1ScHQ1ey+NbOY4vZbrFsgLoCcy0mTvFwBU6tkjaK+f4fqWi1wWz5es15jRYzN1gIIYQQQgghhBBCCHHB+AfgZ8DVmIKbsm8Anz3ezlrrHPBhpdRfAldgXvN+Vmv9ymlYqxAnZPP6BlpqggxOHdMoc0EKqIv4+cN/ee6E953LjRdLFeyFQELYM6iiWjVXcVvSjTlZZsI3PyaULNjPfQBuzKnDVIpW2+0a7McQJlzTNlhdiwkDNSbkLGLCzUa7XdhW115styuHp+swQes0JlTNYMLXpF2D396Xx4SxYfsxBaTrL53OFNK+1sRAMKgLnmpMG+SC3b/F7r/LHquACXjLM3MDmErd8vzZRnvskF1jjpk5qV5MsOtz4u6EG3NaMb//GtyYM30W5poW7dpO+rduZdX0qZiuuqg4Vn+pt3Luq52Rm0TeViOEEEIIIYQQQgghhFicNwFv1FrnlFKVt/cDqxd7EDs3dqHZsUKcdsWSZnvfGMPTGZqrQ2xe38Cn3nk5HQ/sWPQxFOYF9vHU0tQz1UX83LChcUmOJc5tSmvJZs41dh5rEBNM+uxHuf1vFSY4jWAqMPsw7WurgTWYcHUMUz07aQ8ZwsxXHbLHABPWNgL7MEFn1B7nMCYUzdtzhuwHmKAxhQlWN9r9JoBouClb7wsVw6nh4FAx570OM2TdxbSaWAEMOXF3r318UXtbDtO+WNm11Ni1l4PVGmCnXcsqYMSJu9O2dXF5m0FM1W4tZqbuqyd4uZeVro5ehQmoc53d7YVZ94UwYXd69n32fg/gmeu+0+3K+68MAMWdH9x5pkNyIcSFSf64EUKI5UsdfxNxnpPncSGEWJ7kOfw8opQaB27SWu9WSk0DV2utX1VK3Qz8m9Z65XH2/7t57tKYgpY9wLe01kvatXApKKVqgMnJyUlqamrO9nLEKXp01wD3PLSbgcmZ6YSttSHuvmMTAFv/fScTs4LVaMCL3+c56vbW2hDpfPGYbU9W9/uv47YrWpfkWOKsWdTznlTCnoOcuJuz7YPLrXr9mCeoAqYS1IOpwExhwtcUJpgrz1GtYSZE1fZ2DzNthKvt5+VZrUMVxwhhqlTLAWfEHqM8Y7bc8jjDTFjrT48EU5iq2BHgVUwYvMau8xDQZ1vjejGBazVwwFYHazfmjNjHe6l9vC/bNWl7nnLAiP38UnvuQSfuHrLX61Ra+S4XHmZaVx8JU7s6eo+0lu7sbi90dfSGAd3Z3Z6x9yvMdfN2dfQmsfOEy/efTlfef6UH86aCAjNvAhBCCCGEEEIIIYQQQpyb/hP4PeC/2q+1UqoKuAd4eBH7Xwtch3kNszym7lLMa8Vx4L8Bf6uUullrvXsJ1y3EEY/uGuCuB3Yc8w6/wckMdz2wg/vefx3P/NmtPLV3lCdfHQEUN25o5IaLTYVqZfVsqaT5L59/+pTX1FIT5FPvvFwC2AuIhLDnKNuiFkyweCRsc2POECZMKzITjpbbAqeAvZg+/R5MhehrmAHoh+1+JUyFbBATbpaPnaq4P4t5gsxgKmRX2PMFMIFq+eemXCVbbiM8ianUzQLbMa0p9gPDwFswVa8vYapnQ8zMvC23zW2wa87ZY6yxj60I9DtxtxzgZTBP1tPl6+TE3cHFXdnlrbO7vdjV0Zvo7G4/8tzR1dHrZ6ZqOokJYz3Yd5BXVMimKm6vxnzvT3sIu/ODO0tX3n9lmgsjJBdCCCGEEEIIIYQQYrn7OPCoUmo35nXcrwCXYApw3reI/b+F6Xj4G1rrKQClVC3wOeDHmLmyXwE+A/z8kq9eXPCKJc09D+2es8WKxlSg3fPQbm7d1MJNlzRx0yXHzme9saJd8LeeO3jKa/r9t13KR9s34vVI44ALiYSwy4ytHE27MceHqSpMAvWYoDKLCTkHME+O9ZgQswYTwtXb+0bt1wHMvNY0JhDNYkLUCUwQWq5aXa8CxZwvUFqXT/oUWjVjQr9y2+QcJmAL2vNWY6prxzBBX95u68HMta3DzA84Urvvxpxq+/XLdr31dh21mDA3h2m9jBN3S/ZxLIptf7wSUzWbWux+56rKANYqh6nlWcJ0drcnK+4vB/klG+J6MN/rHGfIzg/uPOMtkIUQQgghhBBCCCGEECdOa31AKXU18B7gasxrxp8HHtRaL6bT3R8Bt5YDWHvMSaXUp4D/1Fr/g1LqLzAVt0Isue19Y0e1IJ5NAwOTGbb3jR0Vts6nuTp03G2O52s/3c9H2zee8nHE8iIh7PKVYqbndM5+PgCkK4LagL2/3L54ABO4ejFBabnVb3n+LJjwsx7Tl3+F3XfIFyxmvAEdzicIY35ucvZYNfZYDZhAuA9Tdeuz+0btsacwT9br7X0HgWo35jTb85YwgeuoXZfXibvjbsyZxoS5gwA2fG4DJpy4O7LIa6WZaaO87HV19AapaCVsQ9nsfNt3drcfFba2P9bpHV5xbfItP/qKVKYKIYQQQgghhBBCCCGOUEr5MV0If0Fr/SDw4EkcphbT8XB2q+EVmNeTwRTgBBDiNBieXlwDyMVut3l9A621IQYnMycdMpxI6CvOHxLCniU2TCzZqs4T5sTdYsWxEjZ4nc1r/y1igswcJixdb28/hGlVXJ49q4B9mCfCCKa6ci+g89OBQN48SbZhgtKXME+S9fZYQXuODKbKNmK3y9rjNdjjjwEjdu5tnT1GFhOyFjE/kweYaZPbiAly/cyEzR77cYQbc7yV12TWtUphKm9PWVdHbwAzd3XOcy01O0eXWd9fHxVDn21lq7ezu/24U8Htz12o+fCzec5AK2IhhBBCCCGEEEIIIcTyobXOK6VOtezvW8AXlFJ/CPzU3vZ64F7gm/brzZiuiEIsucVWro5MZymW9HFbBHs9irvv2MRdD+w4pXUtNvQV5w/P8TcRS82NOX5MoFm1FMebJ4AFE65WY4LSEibwbGAmxPRjgjiFCTjL1a1Je5vCBKBVmOAviKnAfdnuWwNM26/jmJkAXrt9ORBU9hwHMAFsFIjYFsFh4IATd/digtggkHXi7oQTdzNuzPHabRqBOhtIrgESTtwdLj9IN+Y0Aje6MWfFSVy+RbNhZxUzVcNnQtR+VEphvkdlASDU1dHr7eroVV0dvfO+ucLO0D2jrYiFEEIIIYQQQgghhBDLShfwCaXUyRZxfQT4PvA1TNHPPvv594EOu00c+O1TXKcQcypXrh5v+upf9rjc/OleHt11/OmHt13Ryn3vv45IwHvcbeezFG2NxfIilbBnRxETpC2mf/6i2SpHHzOtaesB31A16UMrlOfaV3UEmMS0Ao7YbUuYMDWKqVzNY9oJ12ACzzq73v2YQDSBCf3ywLg9Rsbuv8Eez8XMII1gWk80YNoRr8OEh6OYEHq1Xes0M/NiKyuDV2Iqb8shMJiQd/abB5Td74SqU21YqRdb1drZ3V7q6uhN2cd2phxT3TrHTNgcM/Neg0Cgq6M3U75t9v5O3JUAVgghhBBCCCGEEEIIMZ/XA28F3q6U2snRBSFord89345KKS9wHfDHwO8DF9u7XtVaJyqO8dwSr1mIIyorVxULzykcmMzQ8cAOfv9tl/DR9ksWrIq9dVMLIf8uUrkTb5TZWhti8/qGE95PLG8Swp4FtgXx5Gk4tA9ToZp34m7RjTkJIN17rQq2jB0JOKcxT5phTHiXZ6atcGWV5HjFMSP2toNAEyYYHcEEyR67vwcTkJb3H6nYNmS3DWCqYQv2uK9iAl/sbf12u7IQJvTdCxy0Fb97Zj9oJ+6OuDFn7CRaO7fYa3JosTuU57CeKU7cnXfWa1lnd3uJmcrWPMDq1x6rBeXp6uDwHKGtEEIIIYQQQgghhBBCzGcC+LeT2VFrXVRK/SfgaK37gBeWcmFCLFa5cvWeh3YzMHn8l/U/871X+Or2A3zqnZu47YrWObfZ3jfGWPLEa5wUcPcdm44KeIslzfa+MYanMzRXm4D2eG2RxfIjIez5JQvkymGkE3cTAC/9xeXe179cCmOC0jQmLA1gAkgf5o0gI8AwZmD6tP16GhPqTmAqV8uVswFm2vKOAxcxM8tVMzNcXWNC1QLwGqYaNmj3XWfvmwRSTtzVbswZA1a4MafoxN203eeAE3ePqQZ1Y46/8vaTnK2b4OjK22XPBrLZ79/yz/50sMkHBLs6enPY6uUzNctWCCGEEEIIIYQQQgixPGmtf+MUD7ELUwHbtwTLEWfAmQ4EiyXNU6+O8uTeUUBz48VN3LChEeCodVy/rp5n9o0fta5iSXP/T/r5af8okYCPd121ij0jSQ6Mp1jXEOEDN7YR8HkoljTVIT/vunoVj744SP9oauFFAYNTpir2suYoVSE/l6yswu/1cPWaOsZTebb3jZzwY40GPPzXN2/k1k0tR67z93YP8o3nDjKWnIk+GqJ+fvGa1bxtU4sEsucRpbUUyV0o3JjjKYeVdi5tkZmZsQlMoLrBbn4IE1A2YILbDDOVo9OYANYLDNnbPXabekyQm8KEtwVM6Fr+DVeDqWxdgwl2X3Ti7qBdUwRoxVTLJuYKX+12DfY8h2xYW749YI87btecXWBeLnYubf5Mt+e16yw6cfe0BaL/9ze/FQ2lRyLTtesLmGvhB3Kd3e3HrawVQojzgPxxI4QQy5e80iDkeVwIIZYneQ4XRyilbgP+Cvhz4BmObWc8dZrP3wn8EeZ16+eB39Fab1/kvjXA5OTkJDU1Ncfd/nzw6K6BY6pFW2tD3H3H/BWhp3q+rf++k4nU0S//RwJeAj7PUbd7FJT00dukc8UF/2D0KHir08xP+8ePOcfZVBfxAyxqTafz+osls6jnvdmzNcV5rLJa1Im7eSfulpy4O4kJXMfsXUPAACZQzTMTopYwlaxjTtwdwbQRfs2JuylM1WyCmarXctBXg/kZq8IEs8puGwVewbQ3Droxp9quKYWZPQuw1oayc8nYc83+baXt+YPAJZhAd05uzPGkQw31Ays3t3R19J6xPxLdmOOx6wvMut3rxpyoneu7aF/6wIP+J268M+LGHC9AV0evv6ujN7D2wHdV88jz3qqpfeV5uSlsy+Ir778ydOX9VwYWOKwQQgghhBBCCCGEEEKcrIeBq4FvY7odjtuPCWbG4J0WSqn3AH8H3IOZTfs88B2lVPPpPO9y9eiuAe56YMcx7XoHJzPc9cAOHt01sOTn63hgx5xBZCpXPOb2kj52m+O9Y6+k4bu7h8+pABZM+LrYNZ2u6y/OPGlHLJhVCToJR6o1qzHvFprEBHiTQNHeFwIy9vPyNOm83WYSE/zV2v32YKoxJzGzaJswQeReoBFoti2IU07czbsxJ2O3P6ZC1QazBSfuHvPbx1bO7rdVvhfZcx1VAVyxbelffulvs4mqNeVA9IxUiDpxt+TGnDTHtkFWmMB6zkDYBsV+zDVWnd3tpa6OXk9VqRBJh5oCr2y43N/b0ZvEzuddd+D7iVSoKXeo9Y1g2hBrgCvvv1Ix01b6jFYACyGEEEIIIYQQQgghLgg/dxbP/QfAZ7XWXwRQSnUAW4DfBLadxXWdc4olzT0P7Z4z1NSYF6rveWg3t25qWZLWuMWS5lPf3n3Kx7kQnI7rL84OCWHFfAqYitMRYMKJu0U35pSDwCxQh6l+DWIqW0cxIWI9pmI2h2lbXIedR4ppPxzEVN5egwlaX8b8PjnSVti2GD7ydZmtEm216zo438JtkLsbExiHMXNmx+y6Q07cnQY4vOLaMSB4plv0OnG3MNdtbsxJLNA+OWA/vICvq6M309ndnu/q6E0dXnFtfqzBiQC+zT/9nzrvi+TscXJPdfSGMGF0yo05amfc1Vfef2VynnMsqG1rj+rftkVagwkhhBBCCCGEEEIIIealtX78bJxXKRUArse0Qi6vpaSU+h5w4zz7BDGvWZdVn9ZFnkO2940dUwFbSQMDkxm2941xo53XeqrnG5ya/3ziaEt9/cXZISGsmJOtHE1ydL/+EOZnJo+ZCRsF9o4HqyYfX31N7s5Xf7wKWImpQj2EecLyAsPMVF2WMLNopzCh6CWYdsWNbsyZwIS8aeBweWaqbbVbgwl9RyqOhRtz/HPNjnXibtLeH7DbF4BVQMCNOTkn7mY7u9uLzMyqPesWml+LeQwaKDSO7vKNNl5RAjBBLLWAb8XwjlxV8lBkpPEKT1dHb85WvyrAYyuIPUBi5wd3HjmPDdZ9883fBRO+Yr7vobatPdn+bVuOW0HbtrXHA2gJbYUQQgghhBBCCCGEuDAppSLAWmaNZtNav3CaTtmEeT16aNbtQ0Bsnn3+BLj7NK3nnDY8vbhAdLHbnanjXGjkui1vEsKKRbEtfv1AxlbFvoYJYvm12z/FbX1PNmU9Pk+wVBjGBJtrAAd4ERh04u7UrOPtw7zDqNwW148JWr32uHk35uQwwetaYCOw34m7L9n9azFtkD1uzBly4m7C3u7BBMFpe6wgZsatBxP+Zpy4e0YrX5eCDVRz9vsA4AUTUgPTK4af9V65+/Ohp17/Z/lUtLX87q1MZ3d7GsCNEYQ5O0v4MXN5WSCIDdntShzbRvkYNoCNYoLvYyqahRBCCCGEEEIIIYQQ5xal1McWu63W+n8f51grgC8Ct8+zifcElna6/RVmhmxZNWaO7XmvuTq0pNudqeNcaOS6LW8SworFKlewFgCcuDvpxpwUUPj4M1+t8ZUKBEuFUUxY58W8u2kPZvB5zoamUxXVnrWY9sQv2Va8FwFtwCvMBKjN9nhBTGtj3JhTYwPd6orbZ1dmeu2Hth/Y4HiYuYPIc4obc1T5Orkxx7/n4jvD+9femuvsbs9grn/e/osbczzt5hqFM4G6gNKlRO3EXu9k3YZEV0evqp3cGw1lRoNDt3RN2Mrf2fKYatljWiRXyAGl/m1bFhte68o1CiGEEEIIIYQQQgghznm/v8jtNLBgCAv8PWZM3RuAx4BfxBTO/Bnwhye1usUZwbyGvXLW7SuBwbl20FpnMR0bAVDqwpm9uXl9A621IQYnM3O+aK6AltoQm9c3LNn5WmpC0pJ4kZb6+ouzQ0JYsSi2NfDUrJuLQPCtB56pxYSuA5ifKS9mRuwUJiBcjamOzdiKy6zdrglT8erBVM76MMHrYUwL41p7rGlgDLgU87tnyp5LzTFfVQMTmIB2fFaLXz9Q48accSfuntbf9PYxBYGsE3dLbsyJAiU77/Z4+0XdmJO3a/T7C8mwP5cowpGWxXOtPRXKTSTWHfiur3ayT4czI/Te0hVA64sC2clILP7lPdA+OXsne7wFw9X+bVuKmO/1cV15/5WBaocAkNr5wZ3HrZoVQgghhBBCCCGEEEKcfVrr9Ut4uHbgXVrrnymlSsA+rfV3lVJTmPa/PUt4riO01jml1DPAW4FvAiilPPbrfzwd51zOvB7F3Xds4q4HdqA4unqpHEXffccmvJ6lCaa9HsWn3rmJjgd2LMnxzmen4/qLs8NzthcglrUQpnWwB0g7cbfkxN2cDRpftfc1248W4HLgIjfmrMNUSr6GCWt9mLmwQ5gZtG8DXo8J/soVrwHgACaMxYm7xcoA1o05dW7MWQ+ss+drBdbaebJleUzgeCYqNH3MBNJQUZV7HBqzvnLomQ6lRw7nA1Vzzq61s3s1EAYyLUM/TYYzI2l7uycdahysnj5wIBFdTVdHb1VXR++S/MZ2Y46vojWyEEIIIYQQQgghhBBClEUxRTYA45iOiAA7getO87n/DviwUuqDSikHuM+u54un+bzL0m1XtHLf+6+jpfbolrcttSHue/913HZF65Kfr/v911EXOfal5UjAe8zts/PHaMDL8V7g9ii4ak0N51pRc13Ef8zja4wGaI+toCF61Njk03b9xZmntD7nu7OKc5St2gxgKykrq05t++GVmOrT8m/wKkzoGrb/DmPaUhQxlbA19va32X8ftPuWMIFmwIm7w/b45RmlV2GC2XFM5WwJUwnbYD9es1+XMKFwdo7q2dPCjTleW0G8JLo6eo9U13Z2tx+pMnVjTgTz2CZnz3Xt6uj1Y0LaACYQLgJpO2N2rjUrTIVxye7vW9/XU+xbvyWICbG9QL79sc4I4HHi7vRSPT4hhFhC8seNEEIsX+fYSyXiLJDncSGEWJ7kOfw8o5RaA7wTWIt5bfEIrfUfHGffnwJ/prX+jlLq25jXZ/8E+Bjwy1rrDadl0TPn/yjwR5jCoOeAj2mtn17kvjXA5OTkJDU1NadvkeeYYkmzvW+M4ekMzdWmBe7prMAsljRPvTrKk3tHAc2NFzdxw4ZGgKPWcf26ep7ZN37Uuoolzf0/6een/aNEAj7eddUq9owkOTCeYl1DhOaaEB/76rNL8kflDRfXc0lzNQDffO4Q05mZaKGlJsj7Nq9lbWOUsUSWukiAiVSOurCfiXSe+kiA8VSOhmiAltrwkbbCc13nM339xZJY1DdIQlhxwtyYE8AEdfO2sbUBrQ9YhQlaX7Zf12He+XQAU7G6HngaU03bDsQx1bB5++9lmOCvAFwEvIipmA0w06J4zIm7/bPOX67CBdPeuGDXMuXE3dGTf/RnVmVb492xD/gnai8JZsKN6fbHOj2YEDRtt/PNDpdtaFuFeadXAjPbNQCk5pkPixtzwnlfxOcvpBK9t3R5ayf2VLUMbfcfbrqmMNa4qXz87LXP/X2xfuIVdfud9xYxoboCdLWzNT3tbsOuOde/bYu0JBZCnA3yx40QQixf8kqDkOdxIYRYnuQ5/DyilHor8G1Mt8MYsAtow3yfd2it24+z//sBn9b6S0qp64FHMQUzOeBDWuv/dxqXf0ou1BD2fFEsaW7+dC8Dk0szjfDPtzh86Kb1CwalEqBesBb1TZaZsOJkhACvG3Nys2auHmErKXNuzJnCVF9mnbibshWsIcyM2AOYsHUCEw72YcLYAbtPMyZorQE22H8VkLb3Z5y4u8ONOR435qwAUk7cTdol5IGDmD8OVmD+R74FOGYuapkNl0unu1J2MSF2BS/mv9PipviXA5PVbfqGnz6Sd2NH2kADUF5zV0evDyh1dreXOrvbS10dvUXMDNl8Z3d7tqujNzdfFSzAq21byIQag4Mtb0gBRV8hla2feKWQC9Rmi15fIZBPBg+vuLb4+OUfrEl5PWlMhbEX8CjvdKmQXFcTbP3XYnbgl5W9L3dyV0kIIYQQQgghhBBCCHGW/BVwr9b6bqXUNPBLmK6GD2IC1QVprR+o+PwZpdQ6TJi7X2s9cprWLATb+8aWLIAF+Msel8/9uI+779jEbVe0cqOt1i17dNcA9zy0+6hzttaGjmwvhMyEFScjCUzPFcC6MafajTnRiptSmDa55crLAmZua4sTdyeduPuSnSObAPZiqlUvxoSyHkwwO4Cp6HwNE8pqe9wmN+b47HZhIOTGHGWrYJsw7YkjQCOmNXIGmHJjjt+23a1ct8due3Tz+9PDD/hteBxYaEPbXjjlxN0ckK2d7k/b21OY78MRtvI1zNGPIQVMd3a3ZwAWCmC7OnpVf9s7/IMtbygBurO7Xb/3X38vff2zjyfW73u4cP1z/1B35YufW1Ef/79VB/KHVh/M7qt/5Jsf9/7SK715YGrTyB618ZU1NcqTLQBpb9Uuf+z/vK/uyvuvlN8zQgghhBBCCCGEEEIsHw7wz/bzAhDWWieATwKfON7OSqlPKqUi5a+11imt9Q4gqZT65OlYsFheiiXNk3tH+dZzB3ly7yjF0tI0Q/ne7sElOU6lwckMdz2wg0d3DRx1+6O7BrjrgR3HhL7zbS8uTBKOiBPmxN3iAtWiQSBgA8ZGTMvcqYr7U5gwda6K1HF736gTd8eBfkz74osx4ethTFvdFCYwXYcJbRswfwxMYqpdL8G04K0HpjBh5T774QWqmRW22srdFJCeHdAej32sJ7JPOUANAEE35njn27Cro9fXe0tXuKujt6r3lq78rOuuKve1c2Iz2Bm99jbd2d1+pEq2q6N3wdAXU2Gcqpw5Wz4XJvzOrZno99RPvjz+crQm9/C6zc0eTR0QzUTTFzVOl5q+/P/tLgElb3h/nfJk6gFP29aeQNvWHqm8F0IIIYQQQgghhBDi3Fd+7RJMgUzlDNemRex/N6aoZraIvU9cwB7dNcDNn+7lfZ99it/92nO877NPcfOne085tCyWNN947uASrXJGOR6+56HdR8LiYklzz0O755yjMdf24sIlIaxYamOY9sIebMVn5Z1O3M07cdd14u5hADfm1LgxZ7Ubc7y26vNF4JDdvBET6rrAE5iAdtoGkSPAK5jWxFWYUDViPw8yE7ROAqN2OzBhYoY52uTaatMSUG2raY/LVtBGOYEKWifualtFnAXSFVXCcwnYj7l+W4eByKwAuAh4uzp65wqFA60DP6l//Ob31c91Ilslmy5Xzc6iMfN0hwKlQm48VJNZkZ6oynn89a2JkUxDejIfynhKG0YmAwONhbVXJ79XX5i+atoTGhiYdrcVMd+T4AKPUwghhBBCCCGEEEIIcW54CrjZfv4w8LdKqf8OfMHedzzlgo7Zrsa8fiwuUKezenR73xhjyfypLnFOGhiYzLC9b+zIuRZqezx7e3Hhkso0saRsRSlAwY05wxVfzyeACec8QHHWnNQ85veVxvys5mxQi/3ah6mAPYgJJC/ChKuHKj4PYkLhDOC3QSsAbswJ2zWXA1qcuKvdmKMxYeZi5rdqu84jFap2Lmtxoda/5XNV7jePDJDt7G6fK6jN2rVVnsfPTGg7+xkn4y3mQpO1F/u7Ono9s6tduzp6g0Cgq6M3Ncf5QoDPjTmJfKi2GCrkSkl/aNRfKuauHTvgnRzoWzWtS9nWiezYwQaPzlcfrKryq7H/9zeTeSe+Rbdt7Ukx9x9eQgghhBBCCCGEEEKIc8sfMFPJWq5qfQ+mKOYP5ttJKTXOzOu5LyulKl8P9NrjdJ+OBYtz3/GqRxWmevTWTS14PSfUrBKA4emTmwVbF/Hz7mtX84Un+hd9jsWe62TXJM4fEsKK02YRASxO3B1xY87YXNs6cXfcttutwrS5SLkxZwhTIVuDCRwnMFWuTcCV9vO99t8UJpwNA/srA1irXKV7JIR1Y46foyvEA5jf/3OGsDYAzdh9A7sve7+i9caA3f6YattKXR29fkxYO+91Wui+eVpC54BSZ3f7UQFsV0evF1C+QnISlK/yuLZFceG6HX/rH26+3vPamlvmOmfBnlO7Mcf74Rf/I+fE3aQbc5KjdbForbc2GSp6pm8afCn73ar10ZFITfUVo3ujmJbJhUcgu5ifByGEEEIIIYQQQgghxNmltX614vMk0LHIXX8P81rqFzDhbeVIuhzQr7V+comWKZaZE6kevXFD4wkfv7l60c0qAXjX1a386uvWcsOGRrb3jS0qhC2fY7HnOtE1ifOPhLDirDtOODeKaVExiHm3lMLMem0GDmBmvl6b9lWn/YW09lGoxYSoA3Y/H6ZN8VxvnUkA2ga92q6jyNGVranFPAbblji4oe/bhcHWG3McW4V6FBuKhux2J/V2mLkqbu3nR85t1wW3dIVC6cPei/sfDgFFN9aTc+KutusIBjPjnrqpV6mbejX3rv/4i2PejGQrkPP282RFC+Rw40Sc25+NHzYB7T+Hbt3TV7V6IpkK+oYTQHB3w7rAf6x/Y+AHW3um+7dtWaj1shBCCCGEEEIIIYQQYpnSWt8PoJTqA57QWh+vC6C4gJzu6tHN6xtorQ0tGPRWWt9UxU2XNB217+BkZs5KXQW01IbYvL7hpLYXFy6ZCSvOaXZ+aglTkZrHhKppzLuopoDqEqzIBWvWT1VfdBh4wd6XBAq21XARaHRjzpFh8LbNcDWmGjZqP3DibsmJu6nynFb79WIqektAOpibynZ2t2eP14rYtvs9Ui3b1dEbsBWpi9LV0evBVPiGKm47Jmh+pW5NVe+a62qAjL+QTmGuXbrcwtiuI5MN1eecuJuobM1cqW1rj/ql33s09L/+2/eOzHW1AW8Bc53LjzcPTG4aGU78wmNuwYm7ya9e9rbUDy66vgiE2rb2RNu29px4LwkhhBBCCCGEEEIIIcRpo5QaU0o12c/H7ddzfhzvWFrrxyWAFbOd7upRr0dx9x2bFr39l57sp1jSR/b98y3OvIEqwN13bDrSJrnyXLNf7J5re3HhUlrLmEZxbrOBaQumKnUCqAWmnbibc2POJcCGnC9CUfkPhvOTGbtNEhNQTgF1QANQAgaduJuw1a8hTBWqD1MJu2D74NOpq6O3CqCzuz1xAvsEMJWwRRvKRoF8Z3f7kbf6fOS9n6yZCFb5nm69fKJ/25ZFtwO282H9QLL9sU7ff659feSHsV8OrtHhwqV573j7Y511mHm7Q7Nm0s6pbWuPFxOke4BU/7Yt8otHCHE6ye8YIYRYvuRVCiHP40IIsTzJc/gyp5T6IPA1rXVWKfUhFnhOLle8no+UUjXA5OTkJDU1NWd7OeeVYklz86d7j1s9+uNPtJ9SePkP33uFz3zv5UVt+9UP38CNGxp5dNcA9zy0e84q2tbaEHffsYnbrmg95r659ltoezDXYXvfGMPTGZqrTbWshLXL0qK+adKOWJzzbNg6ipkrWsC0KC6bAF4NFFJg2gtXYSpcpzB/KFRh/mPIU1F5aitdk/YYS9Ii1405nvmqZm2VahQoVIakFY7b9tiNOSoVavJHMiMFJ+6WOrvbj4TGN/3kT4KvNV8XePCKOwudFft8p+0NCcBzIgFshfJzofeGgRdLj63/uamQf02+s7tdu7Ej11G7MSeMaRWdnCuQtZWvEUwIrjDh95wVt0IIIYQQQgghhBBCiDOvMljVWn/pLC5FnKfK1aN3PbADxdEp/1JWj360fSNfeOJVJtPHL8Yens7w6K4BOh7YMe82f75l/kD1titauXVTy6JD1ZMJbcXyJpWwYtmz1bAFTCC7BhgCpjGVmh7Mmw1CwMFym2G7n8K2OS6Hp/a2oL1t0eGsrdYNYsLFEuCrrKy1IWwEE8JmT+Zxfv3dfxONJg/VtAxuH7vu+SeOOoYbc0L/ufZ14c9c997CR+I/zF+VKdWkIs1TV+z+QtE+/sxiKlYXeHyqcv+2rT1q7dSg769/9E++2nxKAx4n7s4bJLdt7QlgrosfKPZv23LWqo6FEBcE+eNGCCGWL3kLuJDncSGEWJ7kOfw8opQqAq1a6+FZtzcCw1pr79lZ2eknlbCn35kIIv/hey/zme+9ctztHvztN9D5lR1MpPJz3r9U1blgHvddD+w45o/d8lHve/91EsQuL1IJKy4Y45hq1iJmHmwaWIEJZtOYQHbEibulyuCVmXAWTJUsmIrOAKDcmJPD/IfkdeLu8YLToj1feX6t3405JVu5i50Rm1xg/+Mabr4+3zxMcrDlhmMCTCfuZh75+Q8V3nTwOd91h18u6chFnpah7eX2v0X7OOZ9MaOro9eHuR5H5tm6MSeIbdNcDmBtC2R/XbXKr50ejI6Hqqtr86kpJ+5OzHXciirYQv+2LQXMNRJCCCGEEEIIIYQQQpy75gsXgthOg0KcrBOtHj0ZH22/hC/+pP+44er2vrF5twHzgvrAZIbtfWPcuKHxpNdTLGnueWj3nC/Qa7ueex7aza2bWqQ18XlGQlix7Dlxd6Tiyyk35oSYCR+PqnQFaoAwJpydsrdV/pb12g+N+e+jBATdmJOfr9WwG3N8NmxN26+zQLEcwNrbPID/1bZ35PrbtkSBXGU74dls5aju37blyNrM9vPvc/u+p0O373saJ+4mgEE35kTtNUjPt/YKPkyVao6ZsNZvP688pwJ8vzYdLH2zdnWyMTOVw8zVXYg8awghhBBCCCGEEEIIcY5TSn3MfqqB31ZKJSru9gJvBuKLOE4U2Aq8FWjGvFZ7hNb64iVZsFi2vB7F5vUNR4LY7X1jiw5iFzNT1etRbHv3lQtWnv75lk386Td2Lmq9T+w5zOBUhrFEloZogJba8AkFx9v7xuacN1u2VGGvOPdICCvOO07czbgx5zXMk3uEoytAA5hwMYOZF5uY1aa3hAllc5j/PoKAGm66JtDb0Zvv7G4/qkWxrRatdWPOZLla1h5v9ttnAoC/aWRn8bXVt1DwR4/32zlYXosbc/xAaRHtkUtU/EHjxN1FVd52dfQGMWFtrrO7vTKsTQHaBsgRIPs3d95bvGsylKvSyvdfCmuzm3fuWPCdb/3btmjMrF4hhBBCCCGEEEIIIcS57fftvwrowLxmWJYD+u3tx/M54C3Al4EBZNzABW2u0PS7uwdPqiXxibQyvu2KVu57/3XHbN9it68NB5hIz18FW+kff7D3mNtOpIXy8PTx6phObDuxfEgIK85L5SpUN+aACSfL1agrMAFnChNYVmMrYt2YU64GTTpxV7sxp4D5A6H40mXvK4e307NOVcL8MXK8PySyQKEmccD75if+WA2vuDYP7djz+gFv7y1decDT2d2e5+gANGTPMe/M1Qpq9vzWShWzafOd3e05+3UAKNrzltcTAlLlFs77Lro1vLuhzQ8UHqzO5j4yFUrNCmyFEEIIIYQQQgghhBDLmNZ6PYBS6gfAu7XW4yd5qNuBLVrrJ5ZscWJZmis0rYv452wBPDiZ4a4Hdsw7G3W+maoL7bdQ6+NvPXfwlB7bwHHWW6m5OrTg/Se6nVg+PMffRIhlLQREbJgJcAjz7qtxTAuN1W7MqXr01t+pyfqraosKH/a/C9vCNw8U8v6qzOUvfr7kxpw6O1cWu00eGHfi7oIVoU7c1TYYLh5qfaOKX/q+6MM//4d+N+ZUY0JRvz83FQRCXR29qn/blmL/ti0lu4YMx2/5C6YdcnK+ANZSFR+Vs2rTFdtoZoLrIBA91HpjOt9wdQbQUx5dnF0RvBhuzPGe6D5CCCGEEEIIIYQQQogzS2v9c6cQwIJ57XVsqdYjlqdyaDq7De98M1jLL2rf89BuiqWjX+I+3kzV+fYD05r4xg2NvOua1dy4ofFIC+GmaPBEHs68a57vvJU2r2+gtTY079w+hams3by+4ZTXJM4tSmvpBCDOX27MCWB+F4YB7cTdaXubH1NdetF0pHU0FWmu82UOhX7sHB5/8G2+8Z0f3JmxYWsVphVw0o05VUD9ZE1bdrfzoYnfvP89uYqq0YwNZOdagw9Q5fu7Onp9QGB9X092/b6HQ5hWHsVdm36LNQcfq66b3Jsptzae51gBzJxXbW/zYGbdJirn0C4F+/iCgHqqZVPhXy5pD1bnkoV/uf+PT6jFsA1zy+te0jUKIcQc5I8bIYRYvhY3VEmcz+R5XAghlid5Dj+PKKW8wIeYf6Zr+3H2fz/wLuCDWuvFdPc7ZyilaoDJyclJampqzvZylq1iSXPzp3sXnIO6kK9++IajZqM+uXeU9332qRPebz6P7hpg67/vnDcQPlGLOW85lIaj/+At//JcTEWtOKcs6nlP2hGL81q5QtVWYZbb54YxbYiLQLo6NRAIZifrVCmVrk9S/OOvF4PuXznaibtZN+ZkgNIz1745cOCSX1XNh5+dSoVXVuV9kSAmPC23I16oNW8Q8Lgxpwh42mdC2yL87czcVhP6Hq/C1Gs/PBXbBuxjygIFgK6O3jCgO7vbj/ssZ4PdctXtUZy4m7frDt0wuDsfLmQDTelJBX98vMPOVpi1ZiGEEEIIIYQQQgghxLnpHzAhbA+wixN/k9QfAhuAIaVUP6bb4BFa6+tOfYniXLa9b+ykA1g4djbqUs5Una+t8alYzHmPN6NWAtjzk4Sw4oLgxN3K34JJzB8OEUyA6QkUUykg+ebdjIIOY0LTEJBz4m7pmWvfHPEV0g3JcHOpcXz3VKnfl4U7cOJu0YaUfjfm+CorWN2Y40mFV/heeP2fsfbA9zyrBp+KskBbYTuHNlUOQ7s6eoOAf31fT/od3/nbot0m68ac3KyWw1lglKP/mKkMnY9Rnhtrq33DLDBz1q4n1dXR62sY213yDfzkhJ89TeB8VMtjIYQQQgghhBBCCCHEuem9wK9qrR8+yf2/uYRrEcvQYkPT+cyejbpUM1UXamt8Kha7voVm1Irzk4Sw4oLjxN2CG3PymErWNNBo/81i2vqCqV4tYKtcI+nDyUS0NYguecKZsdTbv39fHo60Ag7Y7TP2uCG7H95iLuorpEvj9ZflVg0+pYHCcdoWh92YkwWK3NKla6b61JqDj0Vs8JoF6L2lC4eZGas24Jw9kzYVSQ54uzp6lZ37WnkeBURteFyy+y6mRbBnrGGTZ6xhU+mWRWwshBBCCCGEEEIIIYRYlnLAnpPdWWt9zxKuRSxDiw0lZ1OYytDZs1HLM1UHJzNzBqjz7TfbqVbozuVEZ7mWZ9SKC4Pn+JsIcd6KYAJYMIPiJzB/YNRjZsH6MBWlAIQz41P72t4xWhmilqtEMZWoAcyMhLDdvxTMTUav3nlfcWjl5mzvLV0lJ+7qro7e+d7WUm5tHAAa2h/rLL5ux72JgZYbPNuv3xrs6uhVtjq2qquj12PPE559kK6OXl8oObSqafjZiy+Lf6UKzExWN+ZEujp6w723dJUrX5U9VznInZMbc5Qbc8Jt/Q+r1oGf+C9/8fO1bsyJ9Nz28bCdbyuEEEIIIYQQQgghhDh//C3wu0opKc8TJ6Ucmp7ID1B527vv2HRMZajXo7j7jk1HbbeY/WY71Qrd2dQizysuXBLCiguSE3fTwAgmPD2AqQRdg6loTWNCSg1o25a4+tI9X2+9+cd/HLTVrwC0be0J3n7nvaHb77wXTDvgIiaADWNC1am8P5r156YCNZOvhv/tzv9VzUyIOntNJSfuppiZMauAcC5Qk09UX1QOSUtA8QvVGe+XY28vAlk35nhsyOoBuPTlrwWbR54LhrITpabR58uBsQfwBLKTAMqJu2l7rjSmlXJw9npsxSx2Hb5VA0+omqn+YjR5qFhU3lDV9P41zUPbjwmBhRBCCCGEEEIIIYQQy9rNwH8B9iqlHlJK/Xvlx1w7KKXGlFJN9vNx+/WcH2fygYizYzGhaV3Ef9TtLbUh7nv/dfPORi3PVG2pDS16v2JJ8+TeUb713EGe3DtKU9UxL4OftNbjrFcIAKX1Une/FmL5cWNOMBluXjXY/Lrp5sPPRnzFTF04Oz6GqYbVQKHgDQWmq9bUlTy+kcbx+JATd3Ov+90HQ9cPvRT+1Ze/n1iVGitiQtwVmNmvaSDx8sZfrlOlYqhhzE0ECsmqoebrsvvXvn28sk1weUZrxddee94oZi5ttmK53PoHj9QXFPqfev5gyq4xCGSduJtzY05gqnptsGZ6fx4buC7wuNXQiutrtaLUMvxMEcjYObchwI+p8vVMVV0UeG31m70b934zGCgk80A2FV5Rl/VXj75xe8/sVshCCHG2yR83QgixfMlbyIU8jwshxPIkz+HnEaXUFxe6X2v9G3Ps80Hga1rrrP18of3vP8UlnjZKqRpgcnJykpqamuNuLxb26K4B7nlo91EtgFtrQ9x9x6aTno1aLOlF7TfXuVtqgmQKJSZT+VP6o/PPtzh86Kb1UgF7YVvUN19aiYplr1yxWRlinign7ma7OnoPAoFweiTiL2ay/nwy7Svl0kATQNHjHyh6fKVIZtQPNLsx5/CXZypWG5j576kEjGOqT70N43GdDdRkGifiSYDqxIHSxle/5QM3b9fuASJuzMlh5slmnbibsY8tMftxtW3tUR/OBIvBxGsKE9Lm7TF8mMA258acAlCb9VeX3JiTmevauDEnApRevPw3C3XjL3tD2Ql/3eTecthbPNh6c6B6en+0JrHfnwqv0OlwUzEbrCtlg3Wl/nW3Za/c/fnD1z/7w8XMkhVCCCGEEEIIIYQQQiwTc4Wsi9jn/rk+Fxe2265oXTBsPZnZqIuZqfrorgHuemDHMUHr0FT2yG2Kk3/3X1N1UAJYsSgSworzQRXmd+bUqRyks7s9B+S+93NfyQw2Xane/MT/y9sWv9NAMJifbgiOx8utihsws2M9zLQvDtp/S6lgI5HsaAloaBrdpe0xajEBbQkIuTGnvPZyC2KAGiAJZNyY46u4HYD7fv63q29sdXzP1F+efN/gSMGes4CZb+uxxwLwDjVd5xlc+TpGV1ztcWYdx/JgnmdSK4d/VjNeuzGciK4u9nb0prmlqwSo2ok93ste/urUQOtNVfXje7zpSPP4rst/OzriKYX/+NJ7c/1zHLRta48HUP3btsw7Z1YIIYQQQgghhBBCCHHuUkr5gFuADcBXtNbTSqlVwJTWOnFWFyeWlcWEpkspVyjxp9/YOWfAqjFBQm3ET8jnZXBqpko2GvCSzC3uJe3m6tDxNxICCWHF+SF//E0W720/+FwB4Lmr3hDORle3BXLT46H81BimzXAYSBU9/umJ2o213mI2XTf1Kvb2IjB5qOWGYCqysr5t3yN5VSqG0TrtpaQxgasfGMS0Kwaowwah5WNj5rz67Nc5N+bknLirH3xPt7feW1/jpDKlL7QUJ//93z9e6uq4LgdEYvEHc6sGf3LkGcKJu/n4O/9icnTF1f7LX/ycdmOdVUDeibtZWwGrnLibAOjt6PUWPT5vJDWsI5nDquKxaF8h6dEo/4qR5301U33hmsSBKSA7kHLDl+fwt23tmZ4jbA1jAt7ppfuuCCGEEEIIIYQQQgghzgSl1DrgUWAtpgjku5jX+j5hv+44e6sTYn6P7hrgT7+xi7Hk/JGBBiZSeR78revweNRRFbrf2TXIR7+6w7yaPweFmUG7eX3DaVm/OP9ICCuWvXLr3qWmlbdOe7xN2WB9LpSfmmKmzXCy4AsHCr5QVOlSAZjEhJYBIKmVzxPKjPm18hXGGjeRDdR6Lzr0eAkYttsU+ta9oxDKjtU1DT+b95eyHkyVrA8YxVS2aky47APCbszJTtzSlanyR8ZvfeVfPL/ywgMhtm0pV72W4rH/kn3rY58/qjXwL377kwWg4MY6FaCHmq4J9Hb0FtvNsYO2TXGxZqrPnw02BLOBupRSJOw5i0B21cBTperUgKc6NZCz6yp2drcXvn/dJ4ND0Qbfr33u23O9NSiLzAERQgghhBBCCCGEEGK5+gfgZ8DVmNcry74BfPasrEiI45ivBfF8RpJZ3nXN6qNue8dVrfwj1/LfvvLsMduXX/C++45N0opYLJqEsELMI5QdT5dQh0p48pjwtDoVbAi9eMWHU69/5tOllYefTWNmuIaYqWz1rh74cRITROZygRpvMDcZwsyVHccEq7X1Ey/lDrbcFB6v3ciaQz/M10zvzwEZJ+4eCVLdmOMFqjGBaLqzu10DKTf2V9VAtRtz8p1xNw8k3ZgTcmOEgGTvLV0+QHd2txfAzMp1Y04xFW2pqZnqA9M+2Wc/iq/bcW/2lYt/MV30hXwTtRt8ti0zbswJM1M9W4WZN6u7OnpV08W/pMO5qTnD7/5tW2ROrBBCCCGEEEIIIYQQy9ebgDdqrXNKHRU29QOr59xDiLOoWNLc89DuE5rxOl9L4XdctYpuj+Keh3YzMDnzEnhLbYi779jEbVe0nuJqxYVEQlgh5jc90nSVx5+brgtMZAOB/PShvrYt/mBmLIoJZaOYsLWECWKxn0fs7d41h344DWQzgZqGbKixoXaqbwoo1E3uzRa8wTAQzIQaM2/46XeO9Edo29qjfuPFnug7Pf5oqJTPYILbgBtzcnYTbY8fcmNO0Ym75TmznqInEPblk96CP1oCCl0dvQqgJfYB3Ty8YyIZXZ114m7JjTkpJ+4W/+Xd9/rX1qz3tg4+OVn0BuujqaFAV8fPeaPJgcIbwFtekxN3E10dvaq3o9cLlEaarp5zbnnb1h6Fablc6N+2pbQ03wYhhBBCCCGEEEIIIcQZ5KHitcEKa1hgBJlSygtcDryitU7Pui8CbAR2aa3ldUOxpLb3jR0VmC5kMS2Fb7uilVs3tbC9b+yodsVSAStOlISwQszDibvF3o7eyet3/I0vkJ/2Ar4VIzuzE3Ubspi5pwpT3aqBBkwwWikMlJy4m/zi+79cbBn66YpAbjpx3XM/GgdwY45pbewNBXvf8sFIJtRYAgpc+9bisysu8b/ltWcbQumJCcycWGUrWj32vOUZs4XH3vRrheya9rq6iT2Z4er91fnMF8aV/3cKXR29IWwgOthygy+cHs1csfvzHvh4qfeWrlJvR2+4wReqGq+7VGVCjdMb9/5bMe+LegD1s/X93geu9+a+93u7cm7MCQLhumt+Lz9Rd0kYyH5vwz+n9zQ/4+m+Hz8mgE7t/ODOIuaPs6Bd4+zrIYQQQgghhBBCCCGEOPf9J/B7wH+1X2ulVBVwD/DwAvt9APgo8IY57ssBXwD+HnhgqRYqBMDw9IlNLFxMS2GvR3HjhsZTWZYQeM72AoQ4V3V19PoAT+1UfxLTwrfUPPpc3aV7/y2ICRvTmKA1jA07mamCLWBaFGcB/IVUKpIeTk9Vr636wq9/LfR/f/ObkQOrbwkAqfHajZ5E1ZqgKuWbw+nDdUB+LFSbiBYyh4BJJ+7mnLibBbBVrwnMO86SgJqs3aiywTo12LSm9LNLPJ7n1r4c6uxuL2HaGBftGrLr9z3sx8yXVYCn8fDzofoxN5AONWR3r7nR33vpu7Lh7Fjqxqf+PNc08PVIJjDTj6GkvHWX7PnXcP3o7qrWgz9ubBl+oRoTtmoqKmJtK+IM5o8qIYQQQgghhBBCCCHE8vNx4Cal1G5MAcZXmGlF/IkF9vst4F6tdXH2HVrrAvDXzAS74gJSLGme3DvKt547yJN7RymWTqRx8PHN11p4toaon/vef92StRQ+3Y9LLH9SCSvE/IKAJxVqyhZ84dJ09drS6oEnqqfDKy/xlgrFSHY0jalSHQOGMS2KA6MNm3KNY7ujOX+V/vFNn06Pv+GO4GX5hK5KDY71rbs9sGrgierJmvWBvD9SBDwNEy/rRPXqRMvg0x5vMZd85Js/i2BC3oQTd4sAtgK2bqjp2syLt3SlO7vbS9+/5bdKoCOX7vn6WDrUMBHOjBV/ssFTeDrmyQN0drfngbwbc/x2bVnMjFftQPFn195TCmXHC6lwc+IrazZW/+yyzb7PXLY588g3P+69fPAtRF68kcfe9Gue4ct/21c38XKxcXR3pm5qL9HUUPA9P0oXf3yFr1z9mnBjjnL/ygkB+X4zp1YIIYQQQgghhBBCCLEMaa0PKKWuBt4DXI3pyPd54MHZbYZnuQx4aoH7fwo4S7bQWZRS/x3YAlwD5LTWdafrXGLxHt01cMx81dYlnq86nsziUbBQBloV9PHUn7yNgG9pahPPxOMSy5+EsELMLwOo65/7UaGrozcMBOrHX5rIBhv8/tx0uR1xeP/qW5is3ZC9cvfnPZPVbYGh5tfVZAN1mVSk2QtED6+41h/ITgb8uUSubuIV5c+M14TSw1or/xQQ9JZywfX7vjOGCXQzmFmzHkC7MceL+e9UZYL1TUVvMA8cArKvrv+FYtPoThXOjKj6iVfCQPZD3y/lP/T9UsT9KyfjHB2GBuyx6erojQCqPT2cBFLVyYO877l/5uqGNv3n/9Gt2bYl29vRO96YGPWsPPxsdTA74U9UXZTxFrOld3znMyM2EGbnB3dWzm7wYKqBNab6VgghhBBCCCGEEEIIscwopfxAHPgFrfWDwIMnsHsUqFng/mpMJ8HTJQB8HXgSU5UrzrJHdw1w1wM7mJ2NDk5muOuBHUtSlfrorgE6v/LsMeeYLZEt0BsfWpKA9Ew8LnF+kBBWiHl0drdXhok5oHj9cz/KuTEnjfmDIpjzRcOZUF11JH04B2SD2XFf3fjL+bHGy1PDzdd5AN9k7cXpvC/qbRl6WifDzSt8wdr6urGXBgOFZANQKEEjKL8HfQgTYqaAPKYSN4AJN8c9pdxrqWirx97Hdc99pujLp3yBQnIFpnI2Y/cBOBKQpsIrVN4f9UUTB+t8pXw2mjhYTFat1k7cLQC4MUdtGtuX3TS2Lw/Q1dHrAXzXPP+/fdPRVXVaeVNrDv1wunxM2xL5KE7cLboxJ1Wu3J2tbWtPCCj1b9sibYqFEEIIIYQQQgghhDhHaa3zSqnF9XY91ivAG4EX5rn/ZrvNaaG1vhtAKfWh03UOsXjFkuaeh3bPGY5qTIXTPQ/t5tZNLcedz3oy55htKc53vHMu1eMS5w8JYYVYBBvIFsGEkG7MSQIECsn0isPPByPpw/50qCFb8IaTq4ae0quGnvI7//7xrBtzipjA1gcUtcc7NFJ3ZcpbzCVXjO5cCZD2VId1INRUlTk8gakozWHC1OrRmo15ty0YefGicfW2Zw5NbHz1G8GNr34DcImkD+eBCbvEEOZ3fA2QqQxDn7+qk7a+nmp/LuHzZUb2f+iBDxzVMsSJuxoz37bMC/gm6i9VqfCKZCq8Ilc/uceHDX/nM18A29XRq6jDZ9d3xkPYtq09CqB/2xZpyC+EEEIIIYQQQgghxPF1AZ9QSv22neW6WF8B/odS6ida66OCWNve+C8wc2HPGUqpIDOFLWCqdcUS2N43dlSr3tk0MDCZYXvfGDduaDwt51jq8y3mnEt1HnF+kBBWiJNXBIr1U33VQL73pi4VnX6NDX3fLow0XVXs7egNtM8ElwrIT9RdmjzcfF1xuOX1hfbHOgeA9QFyk1kVnga8E7UbdSQ1GA7kE7lkuLmwf+2tpYn6RHGqale+OnXIgwlp1Y/f+EuB52/pKnZ2t6cBKtoWTwDVP7rxF/0vXP07XkATXuFRujQSyE2GAI8bczzlalY35ijs7Fu7znQ76N5bulJv/cHni10dvZ62/h6NaROiK/ddjK6O3hAQ/PWpQOKfa3KlWfd5ADq72xd9vJMUxTz3JU/zeYQQQgghhBBCCCGEOB+8Hngr8Hal1E5mva6mtX73PPt9BrgdeEYp9T1MW2OAGPA24Am7zbnkT4C7z/YizkfD04sLRxe73VLteyrnO5H9T/U84vywNBOIhbjAOHG35MTdctvgKWDSiX9ZX7bnXxrqJvaEDq26OQAEe2/pUpiw1j9VtTY0XX1RlX/oiUDLKw8WgRYg6i9lA1Xp4bq8JxTRutSWCdQ1A0ml9UgoP5laP6QH/uj/PVtqnqLh+2tvSD7wurtadqzavCaaeK3u3971PyLPXnVjuQq2iGldXKM9/qpQ+nCgefiZan8+4f25Hz0w6SvlxzFBbbQ819VuXw9U2a+DQLj9sU5lQ1Ld37ZF9d7S5Tmw+pbgrH3p6uj1d3X0LvRmjhIQXFnyhisrUdu29gT7fcUqTEB6uhXsx7LRtrVHtW3tCbdt7fGf7bUIIYQQQgghhBBCiAvOBPBvwHeAQ8DkrI85aa3zwNuB/w60Av8V+Ij9/L8Db7fbLJpSaptSSh/nI3biD/GIvwJqKz7WnMKxRIXm6sV1tV7sdku176mc70T2P9XziPODVMIKcQpsVWgKgJijgSzgueLFz+V3Xf7buc7u9pIbQwEFfyGpC8l93uncUDScH/GXUCrnCxcDhVSVB5S/lNmP8qQC+aki0BrJHPY6L30la4+vAdZPD/teXhXEGX8+VOsdmq6f3LPeU8olM14ye1er6Ybs6zM5f/WUr5BOX/1CVySaPpwdr92YdmOfUJj5suUg1PPs1TcGQuaNGNNA2s519QL6v779k7wrr6NhrYqYWbOlTKix3G5Yg2kz3Dz8TItWngy0H57r+nR2t+e6OnrHK/bxPBHKa0IEngsUaCt4s0v7HTlW/7Yty/UtR+Xfzyf0h6kQQgghhBBCCCGEEKdCa/0bp7BvHtNyeKnaDv8t8KXjbPPqyR5ca53FvKYLgFIyw3OpbF7fQGttiMHJzJzzUxXQUhti8/qG03aOxZyvWNJs7xtjeDpDc7W5f6FZrmficYnzh9JaxiQKsVRse1+fE3fzs25TALffea/ueP4b1deM7FFNhWI0E2yqD6VHCtHsyDiQy/mi2US0NdAwueciTAjnA4a+f2lT6GDkIv3z8b3DKzNT1UBjETWF8ka9upDc04L6xhs9E+sn35FaldgcvGL358M10/tKmCrdchVoGBPoqZH6TbWJaGtUUUrs3fjLhzu72yurVL1A5PUZn3rH6IhKRVuz7Y91gmlJnHDibg7g6+++14MurSt6A8VL9v77YCR9OG/ny86pq6NXYSpui39Tl84Cun/blpI9p+rftkXbys/C+TS/1Y05AUzFcWqh6zNb+ZqcvpUtDTtz1w/kl8N6xQVDfhaFEGL5kle9hDyPCyHE8iTP4ecZpZQPuAXYAHxFaz2tlFoFTGmtE/Ps85vAgzbYPGuUUh8C/l5rXXcS+9YAk5OTk9TU1Cz10i44j+4a4K4HdgBH/5FX/oVx3/uv47YrWk/LOSrNd75Hdw1wz0O7j5rx2lob4u47Ni24rjPxuMQ5b1HPe9KOWIgl5MRd7cTdfM9tf+j/1i/8ef2/vPtej72t1HtLl+7ftkVvfeRzU8XG61M7rv3D6YGWzeOjTVeOvbDpt3JZX1Vj0ePfOFW9bmUy2JTHVKjmcx4m+5trkn2rL88ebrqOEp4wMOxFD+SCNSPZQE3Cr5un3vpCrfdNz8UbXrfj08Wa6X1TwJQTd7N5XyQ82rCp2om75T+OmqOpwXDJ4y9p5fWu7+s5qiK+f9uW4od3ftvznu1/X2odejrUMvh0EBMGH9Ua91f+/eOl4ZWvO1T0hib9+WQQU2kLHAlc6ero9XZ19AYAbNCbw4R1xXIA+4EP/M/Qr7zc2/zu3/i7CCYoDs53fdu29nhs6LeceDiJ/xFZRoGmH/M9k84KQgghhBBCCCGEEOcBpdQ6YCfwLaALWGHv+gRw7wK7fhbT0rd8nENKqbbTtMxjKKXWKqWuAdYCXqXUNfaj6ji7itPktitaue/919FSe3Rr3pba0JIFlfOd43jne/iFAToe2HFUAAswOJnhrgd28OiugRM+51I+LnF+kEpYIU6D77Z/JJoL1NQUvcHxd/b8j4wNJaNAobO7PdPV0esFIoHMmK9ucm+44A0nN+z9t4a8v6p2vPbSfP3kntfywZr0ysM7QlPRNfWj9bGsQnmKypfIeD3hbN0libqJl1PB7MQ6SkVV9Eez/tx0JBusTeR90alIZiRaP/FK+vpnH088cuvv1IKK5v3RsVWDTxWjqcEgpiLWl/dFav2F1ISdbwuYlsGO+891jWMvZgP5ROlb1/9O4Tv1dZ43Hngq9z+/9b+1rexU2Hmv9lgloODEXd3V0RtmJpDzYSpxk53d7aW5rtWfvetjwepcqvbJ1ssLzzVfWgSmywFtpbatPZ7yNezftiW9NN+pc0Pb1p7yXF+NuZbF5RLC2lDc179ti7RNFueSZfHfjxBCiDkttzfciaUnz+NCCLE8yXP4eUQp9U1MgchvAaPA1VrrV5VStwCf1VpfMs9+JaBFaz1sv54u73uG1v0l4INz3PVzWuvHFnkMqYQ9DU605e+pnqMpGgQFI4nsnOd7+IVDfPSrz1Ka5y/PckvhH3+ifcF1nonHJc5Zi/pGSwgrxGny4zf+UqBxbHfeBpMKU+VZ6OxuzwG4Mcf7s2v/0D9Ve3Eomngtu2HvN9eqQj4SLExT9AYP77/orZnWgSdLkdRQdSbU4KtJ7AuMB6KTj132ruDK5GjqI/9x79A33vmX4arpfSqaGq4NJw/VlvyR8afe8KmJutEXW1aM7kxedOhHY+lQU1XeH60Zan5d9sBF7TQPPzN9xe4v+PdddKtuHN1VrEoNZF5te4cHTVPDuFvcs/GXEqlws//FSCiTVqX6mpIn/LK/mNwZLE70b9tScGNOFSZcLRa8Ie909UXpNz717SxA29ae6IacJ1BXUr4bsz5vtJAvXNz3H4m1r/VmK1vxdnX0egBPZ3d7AeADH/hfnh+tvjoE5Pq3bSm0be3xYSpr05VhpA0ri4sN/Lo6en2D3hJ/2fW2wvG3PnvatvZUYd4lWMLM4C2HzN7+bVuSZ21hQixf8seNEEIsX/KKhZDncSGEWJ7kOfw8opQaBd6otX6pMki1Va27tdaRefY7qyHsUpAQ9vz36C5TAbsYX/3wDdy4ofE0r0gsU4t63pP2kUKcBm7M8TSaKlEvkLateFOzNous7+vh+Wt+p9pTzIf8hdR0OH34cKCQ0plArQqlRxumatrIhBsTqweeGFSwMlTIpvt9+emW1P58V0evp9kX0msGnshgWsIGKCRTQH7F2IsFlR1v+Oa1/83TmhrN7mG0kGq9PrsB/KnISs9YfUwVPd6qZLRV97e9I/uMLxHcODXUXFtIZ5sHf4r2+Ar/se6SUrA6Rn2RpB+mHvnmx/Xv73m8dnf7fy+9M99YYuyF+v7a1Zm2wKriPX/waOmlQNEHqAGSxbWHd6upsDP1lh33BatSA7XAOJBzY45pNXxLl3fSU/Ld+IcPZwf8OtfQ0BZ8w8CL4Vfq1ui2rT1FTAAbBrJAsXzB+rdtyXACdgQKDRmlFTC0iO9ZeQZv9kRmty6RJObx+jA/JwVmtX8+V9k5vqp/25ac/dprv14w+G7b2hMFkJBZCCGEEEIIIYQQ4pzjwbyuOdsaTIXsfMqd3ub7WogjzkYVabGkueeh3Yvefnj6hF6OFuIYEsIKcRo4cbfkxpwcFQEigBtzFBDBhGyZxom4Bkq1U6/6a6f3KWAKKIVyk9XNYztTYzUbCnl/lbfgDQf8xXSmupDJfuHBT6bcmFNzqPjlaDZUFxirv2yyYfylBCawzLU/1ukZr93g2dN0ecNEpNH7UjiQrMkGfS2pyZLf58knqtYUGsbjhYLyrVGokD83VWzLjKmhcH1yZe3F+WgxHQ4nRtRfP/H9XNoXTu+99H2ldc3XF4DA5aN9vvWZfLghvL72YLixMVtKTJVGtu8bX7XaszKP/8+3f0m15PKkIi2sGP1qya7JB+Tvf8u7w1cFamtzgdrCd73DyclgrW/AryON6YnUZx7/PzzTfJnPq0vhn4SvLGD+OMsx6480G/CVFtOqt21rj7rU50mk1aLfiVmee3vMeW14XHTi7klX1Np2ynqutfdv26LbtvaMY8LLfHn9gKdta0+kf9uW2QH+aWfP7z1emIoJzD1tW3vy9rGFMH+oL/QHOZiqX/kjXAghhBBCCCGEEOLc85/A7wH/1X6t7VzVe4CHF9hPAS8rpcqv+VQBz9oK2SO01g1Lu1yx3Dy6a4B7Htp91DzW1toQd9+x6bTOU93eN3bMDNiFNFfPP2dWiMWQEFaI08SJu9nKr92YU/6NrQDlxN08gAOF7t/O1ExXt+l1B74bSIeaoo2jL2Rrp/rGc75INBNs9BW9wYi/mE4DWTfmeIBw08gLnom6jdMN4y8VMMFuzoa/vnB6JN+QHDqYKaXHH1u5Rm+eCOd/5blvrCr4q0oDrTdOAzVVycHCdPXqoYm6S5Kvf/Gz3onaDb5MqKkYGo+P1E296ntp468G/IVkFF3Kre/7DzVeu7FQW3t95rq+h3w5/85sbdWag4eaL55Yf+Dx0v27DpWeb9qQ3TB5qG609pLIZO2GpPb4is2Hnw0CmdvvvNf31vGp2umJoUC1ry7VH672TvhKRaB03fDLekVmktv2b5/+0qbbS5jgugB4+rdtKdngNYgJ7eowrXon57rmNugMYMLf6MuBku7ftiVpA8UQkF8gVMwCeSfuHvVHob3eAbum44awbszxAqXKalp7/qh9bHMGqnOsK4j5vqZte2Y/plWxz26ft1WohdM0PzYI+Nu29qQwoWpxrlm9mO+H1wbJIbvO44bG59tcXyGEEEIIIYQQQojzyB8C31FK7ca8pvYV4BJgBHjfAvv9xhlYm1jmHt01wF0P7DimOmNwMsNdD+zgvvdfd9qC2BOpbG2tNdW5QpwKCWGFOHO8AE7cTZRvcGOOd2jFtbp4+W+HJ+s25NOHd1DwhQLJaGv+5UveE2wYe9ETSo+nQrmJPCYALLcBmQwUU8Hm0RfKLWs1UOzq6A143vSZUMvg02OxV75W3DD4dOmWS38tVPQGg00T8cShlZtL4w2xSDpQl4tkRyaC2QnfypHniwCB7GTmhSs78v1tt6v2xzrzmXBjQGd92Yax3VUFb6g2UbU6lYqujEzUrPO2Hn42V5s4OPXhwafzmPA3f/vWHs9tB3ZOvmf/C9nq6f355sPPRoAmYAzI+AON+YmGFYlpUL+cQv1zdVblFOq76zZn/uDZfyndfue9GhNUKhvQlauIy+1PCpjAb6FnSr/9KAem5edyhfl9p5kVpNog08ed92b6t205qnLZfr9KbsxJYULgBdkANgLkK9dpA8rcXMewAS2zZt8GMI97vOJx+extQbtNuX1xom1rD5jrdlTwvxj2/D5smFvxdZ6Z6xcB6my1bhrTOlkDXDbar9qmhwKbf2dsimhDCVNJXK7m9dlt04uoqBVCCCGEEEIIIYQQ5wCt9WtKqauB9wBXYypaPw88qLWe9431Wuv7z9ASxTJVbgc8V0WJxryIe89Du7l1U8tpaU18IpWtd9+x6bS3RxbnPwlhhThDnLh71OzLcmvilYefLb4IY1c//3/8jeNx3XtL1yjgCWbGqlccfpbmsRdSmApN7cacKrt7GqjHBnLAQSfuFn74Ww/VRhKH1hW9vgngEBCuH3uxDuWJlPDqVLQ1BXh9pVwm66/2jtdd0nqw9SZvJDuWBrwT9ZeBCdH8V++8D6B4oPVN3tGGK4orR3ak6ib2VBd9ER+mbbLefgnRsbqVNZmrbxzn9j9TQ83XVH+v5spcyhvIvuNnE7U1yYFMTT5d+OKT/y86Vu+o/lC48OiqyyI1Wnlapgcnbxp6JfXY+jd7b7/z3iCmevKoGbDWkTC2f9uWOStgK+QwFZtHVa3aitrk7EpOGxLW2HNkObb9cRAT0KZmV5vOM/u0hAkg87MXtkBAGgJ8bVt7EhXnUHYtJWYqaJOY39lF+zjLYW/efq6AbNvWHs8cj1MtUC0bxFQYJzEthMOYP6ynMD9jYbuGZrvNFFCLub75KX+k+Fq0UU0HItqurbIy12ePf+SPc1utHAJyEswKIYQQQgghhBBCnBuUUjuAt2qtx5VSnwTu1Vo/CDx4lpcmziPHawesgYHJDNv7xrhxQ+OSn3/z+gZaa0MMTmbmnZPmUfCP7zt91bjiwiIhrBBniQ1Vs0Cps7u94MY6/ZiKTw8QbBrdqWun9+Uw/50G3JiTZKbNqwIKOW84WlK+UKgwXeXGnMTrgg2+webXpbXX55moWV+ldElFUoe1ojTmoRhsGHMz+9e+Pe0vpArJcHNxsu4SnQtU+SfrNnrw+IIUc6WrXvxcAMikwitUMDtR3Tr0dKZ+am/an5vOH268MpAM1OsD9Rk1UTVZ9UpDINWcWRnZt3aF/00Hn08GQ6v1C9HmaFUxVffyilhTMFKjbjr0XLFh9EUPWqv/aLsq+kqgVGpNjEx9offe8OGGy+s99a9L9NRHchNerfu3bcm5McfvfvPjoV+77ZOl8VBNOUwtcGw4ewwb/s2udPX0b9tSmqeVLpjwNTPP/cp+zCWEmYV6JDx14m654lWxtcc7V2XtHIrYWbE2oIwwU03qwVTUlmyQXK6GLQFHzmtbBkfbtvZEAG/b1p5s/7YtuYqKWn/5tjnOn8eEq1lbBavtGvyYALUZE1Sn7bGuxFQ3AwwPRBtCE8Ho4bQ/5MeEtUNtW3uqMD+31Zhgt7Fta89qIG6PFQSGWER7ZyGEEEIIIYQQQghxRjiY13bGgbuBbhYxcmouSqlxmLfYMQPsAb6ktf7iyS1VLFeLbQd8Im2DT4TXo7j7jk3c9cCOI1Uws/3j+67lHVdJACuWhoSwQpxFTtzNVXyeAujt6PUA6ba+nlCwkPQVPEHtK2VDmMBKYdr/Ft2YMzFZu8HrLeYi6VJjKVHTFrro4GOelSPPDY02XM6hlhvqs8H6bLJqTbJ5cHugdmqvCmdG/e2PdSacuFvo6uhVLYNPjV4e/3IYVP5nr9s6edHBHwarEgeqp6Kr1LPX/F70ogO9etXAEyl/PukJ5qezoyuuGXlNZWsGojXVl018T60a3BguUJd9Ojhd2NL3k4Y1xUDV7lWvzz2+Yu1IX01ruqEwFf7Bmmur100N5i4e/tlE/Kq35d+6/2fq4zu+FiziqSl6gtGcR5dKygSKbVt71Gfq13pXJUY8gWLBC2b2KWYu6lFhqJ0/6gOS81V52krWQNvWntTsQNS2IS5XmHratvaswISMCXud0/3btiz0bJ/BVOceObcbcxR33hux6wrZgDbxD2//QHAiEA188Yo7CpWzUN2Y438EPE7cnX0ehZ1DO2t2atpek/neqFXCtqa2j73Wfp2295VbHRfs8RUzYW+1PUYD5g/ugP03AfTZ61SPCWer7bFr8PoSaa8vZ+9LYALWFXaboqdYqIvmUmung1X1eDyNdi0H7DmFEEIIIYQQQgghxLnhOeCLSqkfY14z+rhSKjHXhlrrvzjOsf4C+O/AI8B2e9tm4DagC1gP3KeU8mmtP7sEaxfLxGLbAZ9I2+ATddsVrdz3/uu456HdR1XlttaGuPuOTVIBK5aUhLBCnGM6u9tLAG4sqXL+qvCB1W+JBHNTuZqpft9E3SVRbyE9mbtyczAfacnvvOqu4ZGEO5n0qOqLdDTvK6TSrUPbC0P+1+Unazc2pUON3oaJl0NTNetaJuo21gTTY/um6y6mt6O3GvANttzgzQbqdDZYWwSy2VBD/rXVt4SGm65VRX800H/xHVMlXzAYzEz4Ljr0eAoYeSHEdG2iKhgerK5rzk8GqrKHqnY1rffFGy/JXdb/tLps6IX8QO5A3eFQDfdfeWfmxoPPZKpzqczFUwOpL3xvW/D5xosjaW8gEy7mxvLBmvEna6p8U55SOZCr+/23fMz7+yPFqduKEd/9ZDNwJHANt23tyWLCUY0JF8uVoEFMODg7kC0wUzmK3VZhKo6rsEEn5ndhuaVvhsVV3RYrt3NjTgjwN6QnC2PhWo1p8TvVtrXHv7n1ysbm9IQ3kk9PtW3tKVZUpPr6aloDv/OR7sKe+ouK9vw5W8VaYO45tFF7jCPhrH3MR/4otkFrABN4HqmAtS2Ua+w1yVY89pL9PMhMi2swrYdLmMrX8jsVp+y/GWyADVxkP0/Y43uB1wAVzaUvqcqnvQWPN5sORmsxlbUKmGzb2pNeIFAWQgghhBBCCCGEEGfOh4B7gF/AvA50O3N3MdOYkHUhNwN/prXurrxRKfUR4O1a619SSr0AfAyQEPYCcrx2wApoqQ2xeX3DaV3HbVe0cuumFrb3jTE8naG52pxTZsCKpSYhrBDnkC5TBevp7G4vOHE3+dPrfi43VbM+PN7gpIFSKH3YE0kNlwr+qrpcoCrb/ljnxL9veDMX5XLRi0t+bzLamkiGVyYmazeUoskhTyQ1XJquWZeilE+pYlEHvdO+oi/sW7P/+/6q5EFvIrpKvbb2bRoTmqnh5utKwLQ/O1W67KWvTFZP78/u2fDu0urJ3vCUPxL8Bq+UvveZj6Tbtvbkd0Z+vvir/c8X2if3Ry4fD0Yp5fJonW0eeW4yWHNzZG02H/7wq8+uieZSgx/44TfH27b2eD+881sN1bl06O+uf+/B4UhD7t8e/SvfR/svK8bz+2rv/M2XIzRfqn0a7fNVhZpLlC7KezxtW3t8V2S92g0Uq4qKICb803bGahagbWsPzNE9wgal6Vk3RzDBYZqZPySzwIT9XM/Ttvd4SkAx4Q/XYQLeHCaQnHpm5WXZjROvTaX8YR8mRPX1b9uS+kj7xwtpXzB4OFJfg2nvojDzYQvlyl0bMJf6t23J22rhEseGyrWY8LbcIiZvj5Wf1Wa5HECX2w1P2WvhsetNAa32eClgP9CCCZRfA+ooFuqULo1oXyAJDNvHWrDbJ+znIUwI3DQdjDT4dGFP2hf0Yqpla4Fr7Bp+YM8rhBBCCCGEEEIIIc4irfVLwHsBlFIlzHzY4ZM83M8Dn5jj9u8Df2s/fxjYdpLHF8vUQu2Ay/Hn3XdsOq1haLGkjwpff+GqVRK+itNGQlghzhFdHb1eIAyoro7eZGd3e0l7/D5fPuVv639YX9zfk+i9pSuZCa8ITNZcnCr6w5lL9n4j+O69P/QBw8lIS/Xhpqsz0fRQcc1rjzcVPAEiqeF8IRBJ77ziI4lI4uB0oJAoNIy7+tJX/72QCK2sUcW8L5o4eLhpdFdhQ9+3I6lgY6Zv/TuCuUCdCqVHo5H0YVUY+I73pWDA99RV75raU7cm3La1p3Dd0Eu+39717eLIpR/KH2qtma4ffyla8EaCkzXrisOZQKD7qjv58J6fRTcPvVxflR4ed2OOhzvvDTy25pr02unh5OuG4t7RUG39zvq19ansgH+qbr32BRoKSpNsLKpyIJg94C8FfJraS/LenNZ5NZUbVgdqWo48N5fnvT7yzY/nnbibPd41rqgQzdhWw+VK23IlbRBT3TpnMGhnss45X9a2ls7lTNVu0R4vBCSLHl/wpYa2cvVpAFPVm6ampWhv05jgVFMRnNqANWCPlweoCFrLyj83ETv3tWjDWh+mFXO5WreACYXzwCG7tgwzM2Br7Nev2W3K7YhH7fWoAvSK9ER+ZXK89Gr96gOpQCTDTIAbxlS5TtvHvgJYicfrH4/UT2DmirQBr9j7L8I8B0kIK4QQQgghhBBCCHGWKaV2YILXcUxF7JytiBdpDLgD+Mys2++w94F53Wn6FM4hlqn52gG3nIF2wI/uGpA2xOKMkhBWiHNHEBOopTu720tuzFG+UIM/mJuq12b+QhqIeIq5aMNEPHPli5/zY8KyLJCPpgZ14NCUAmojiYP+iZqLI9HUIeVLFqLO7i8F/PlEyafzh392/Sd8r7ZtySYircXq6f01G1/9ZrpxbLfe2bA+PFp9UUtLcqiQrtkw8qNN7/UeDlUF173ytVLO6/P5i4Xsh158OOquvNIfKRT9VdqbP4zyJ6rW6PqJl/M10/uKVcnXGAjXeG7ve7JqlJKmMLGvLrFvHPB949tba59e6VRtb7187J17f1TXV7Mq8OOWy3J+FUo2hdcl3lhQ1YcKKjnu1Xlm2uNSUHh3B4qelZOHEu969fs63tgW/trN9xX7aloDXHUnf/DLf5z/MPjdmJMtz9h1Y44XG7Y6cbfyDVXl8NRnq1HLlbDldsQAubatParcJte2Qi6vJwx4bRvdbNvWHvVzB54Jv++l73kuShxOOXG3BIzb45SrTcstkcvhbDl09cxTqXsk4LVhapKKN4XZlsIBexyvPWbK7lf5lq28vb8eE7CW598W7VoK9nNl1xm2H+PAHkwFrJeZds5ZIL9+amhyVWqksGvlJVl7DjB/PE9hKmaLdh9dnZnen/aF8gWf3+8tFg4Ah4peXwl4HlN5PFe7ZSGEEEIIIYQQQghx5jmYYHQc+CRwH+Y1p5Pxl5iZrz/HzEzY1wPvADrs17cCj5/0asWydjbaAT+6a4C7HthxTDvFwckMdz2wg/vef50EsWLJSQgrxLkjC+Q7u9vzAE7c1Y+/6dcSE3Ub802jO0tO3C089f5/bqRUaPbmEvsHG65al440+SbqLt37a//6u+NPbt5CMrKyPhFuZX8kWuXB2zBed+low+Qr+Zy/KqRRkVDiQKqt/5FiODnoCaYOR6uSA1O1yQMRoJjyBTyNiUNVDbnsyKEN78qMeAuerIdwuPktg5fH7y/+Qv9TnoIn4HvF0+rZs+rG1IrEF9I7qy+qXzn4tLf58HMJfyGVzgTqGqi9vPribCE9rtPsqqmfXDNODaB2Nm0I1mcS+s0Hn1frEoc9tbl08TPXv2c05w14/8urz0Yum5qkI315+tcf/FDhT+/83eDja66Nwoo0MPZSoOgdrKnzDG18S7SvdpWvITNVuObwK6X6zFQunM/kn23a6PnSptv9L5s5qvoR87vNhwkDj8xt7d+2pdC2tScFNGLaAicw1Z8pTBiqgMoAthxQFu2c1iwmwPS1be3JA76ULxTaX72SixKH0/Yc5X3LVaLlytMgM2HmkXbDlSrOly8fp3/bllLb1h6/bVGs7eMqV61G7fFymOregj2Oz54vjfnDuYQJcpOYYNVnH2vRfq2YeedhORgtz7wdPHqtW2jb2tNgr8M0R1fTZoERoLY5ORa+aWCnHg9WJx676Lroe175wVTSF/R8a+ObFaYaNoFUwQohhBBCCCGEEEKcK54DvqiU+jHmtaKPK1MYcgyt9YIzYbXWn1VK7QY+Crzb3vwS8Bat9U/sNn873/7iwuD1KG7c0HhGzlUsae55aPecc2g15gf+nod2c+umFmlNLJaUhLBCnCM6u9vLodcRb/nRV0pvqaiULHmCk+BvGlr5urAvN+0refwtWpdKX3/XX2ViidfqpiMr1+Z9waInPTrqU56kX/sax+oui4bTwxOR9GA+kh0NrhzaPuEpFWqLyt8Yyo2NY4I03xuGX8qOhVceaEgP5QBva9E74i2i89UXe3NNmwtTamdVKtwUXDn0s2zNVJ924q7u7ehNTdRt9G+//hPRbLhJ1Y/Fxwv+cPhwOJRuONTLW157LggEp/zh7Cff+OFcfWYq+Sf7D+YHdTTVp1L+O/f+yPPIxrdlddVaFRj5Uf6TF60o/e+PPVCz2R+JxMb6OVi1ohzwqT945mulvXVr8i81tmXi9WtTv7TncZ79+/eV4H3l6tAqTDCYduJuyo05BSfuFm0LYmVnyGIrWEcxVZ9+wH/X89/IvbPviQKQc+Kubtvao5qTY4GvPf4Pvv6altzWm+/KtW3t8djty7NkI4B6uvXyqY2TB4u//YOvH9Um2X6qf3PXfwRuee3Z0q/f9uc+u45U29Ye1ba1x9+/bUvebh9iJjSOYipLs/Y4HqABSP/RR79bWB/01PT5Sw3AICZADdprFGjb2pO0bZaxx1P927akgfQvfeje0Gi4rr6/tnXMXoMQJswtt0LO2n3yNnDOAb4rR/bixpwQkHfibvnnM23XVQ50y3Nqy6HqWCIQrgZ0f01rHshH8+nST1fG6jAVsOU2xPIXjRBCCCGEEEIIIcS54UOYNsS/gHm96HZsp7pZNLBgCAugtX4CeGIJ1yeWidkzV093heti1vOlJ/qOakE8mwYGJjNs7xs7Y8GwuDBICCvEMvLG7Z+afO7KzhcyVSubRlo2H/JnpzK+fLraX0i3TEdamQqvbvbnp5vW5TOvTletGmkY+tmEt5BuLHm8zVl/XeRQ0+qp/g3vKq0++Ni0L58cKCRD6WhmOOgvpDfmveGJROPlo5n8+tD1z/xNad/at2dXDv1U71v388Vo4mC9p5gNT1S1Ta0aeaaweuAnBYD2xzoLPTdvK44EIs1j3nz4ubVtY3f99HOB3xhyQ5iKyOasx6d/ttLJbRw/kP1199Ggp+EG/fD6N2cGS8OF16qaCx1TIU1gTfgHzq+WIFc9Eqlv+PHqqyMNmampW/ufXjUYqU/sbL50evNw3HfN+MHUylW/mOn88p+W2rZerZhpKVyeB5vDVnLefue9YIJGL6Bs5aq3f9uW/Pq8J/yat0TeY9rtvlK3OvDIujcEVqQnEsScSe681+tBB92Gdb4bBncXML8ry+2i87ZNcB5bTfpg7O3eB83s1fJfE1FM9WzCjX08AvhaEyPZgaqmcpVp0G5bbuc7O8TMtW3t8WNaCRft+dVrvlKNXxPGBOdJ+7ijdr9qwGOrdcvtk33lc1w3/HLk1dpVoUQgXF7DkSpfTCVwuT1xeU2TgPrrH9+H3Q5m3iTgZab9ccZ+PVlxXyjlD6tvbHxLObCu/eyV7yzYbQpArf0YbtvaE7bXanZbZiGEEEIIIYQQQghxhmitXwLeC6CUKmHmww6f7PGUUl7gTkybY4AXgW9rrY/pDifOH+fazNW51rOQ4enFbSfEYnmOv4kQ4hwSvmZnlx8oBHJTvvqJl0dQet9U7brSvrbbMmONTp8/n54s5BNto6HahmR0ZRblnfBonc37ApO7W19XmAo3FPZu/KXMS86v53a87o8Dr6z/xZaUtzrkLWVD3kLGV/QGQ0BNy/BPa6KpwXDDeDxfN7lnylfKe7XONn970y9HX2nY6H/6dW/39a27vSk89lz1d6K58R+HC9PDPu1369dGMh5/CSDni+Rf2PiLua9f/X7VmJni9cMvlZr7vhGuGX2mKlTIZ//pB3+XC7z2qNrlL2T2+Irq4pzHBwz5dOlwsJgNtkwfXnfJ+GsrAZ8Td9M/vmkbQLSro/fIHFMbVgYxQV62XF2KCSD9FR8hIPSbH/vPwKV5r29N0ePHhJcTP1sZG//BmmsT+6tXlgAe+ebHPVW5VOqeG35z3Im75dmrUXvcQNvWnmD/ti05W10bwFTVVgFN9t8cM2FmGkg++Y8fzAHatgpOc/RMjRJAVS4ViuTT5d/LHnuslfb40WeChcy0R+8DhjAtjZPAaP+2LQlMe1/s4/TYc5TDaXoufmN+MNo4/rZCc6mrozdgQ89Ru9bydSvZfSk/Plv9mqo8FiZILWCC4AIzIXIJExyvBBL2+pSreRWQ6t+2JVuXmZ6ozUwfZiY0l9mwQgghhBBCCCGEEOcIrbXnFAPYjYAL/DOmHfG7gQeAF5VSG5ZmleJcU565OjvwLM9cfXTXwDmxnoU0V4dO44rEhUgqYYVYXsrPGEV/IeW76OBjxWB+OrO37ReapqrXefJVLaNKl16pzo7X1aeGDxb8Nflk9apEOHl4YsSjC0OhQPjKvm9f7A2uKI01bCLvj3qHmq+JeLMT2ZbDz6rqqVfz6UhLOpIeTtdO92dejP168IXVr/cfarkh77hf9qQCVY1BAuHx6MriuqlnPeH0YV9g1c2ZjVMj/jUju7xFPKmgvy7yn6/73dLGPd+uDhZS9S2JweS7X/1JfuXwjhyQX5GdYixQ7V2RGvUBa2vHdmYfueItU3+x8/HqpsRgaP3QT5M/XH118ZF1mxPJYGS/V+vEX//onzxu7ON+bunKA8XO7nb9N2b+q8KEgGmgZANZbKBYDkJrMCFgAQj9LFTIfWQqNBXQFIZ0MpoKhEMToZrSRKgmv3PFxtS79/7QDwQ+vuNrmY63/pGvbWuPvvng87nnmzZMTgerSphAt3J8QGU1q8ZUyh4JLG+/814Tqm7tKWGC3PIs1bydARsA6kKFbHHzoFs7FqrRzzVfUm5NfQhTMVoPlHKK1IBPh+xteVuNO2XbMZfDzggmNA3Z65IBOFS1onCoakVdKVesuijlnezq6C1QR9Y+noC9PmFMyJy1j6vQv21LsaINMfb6HvnLxW5btPN2Pfbcebt++rdtydv2zwGgqm1rT+qRR+9Rj67bHP6Ha3/V97Fnv56+fd/TJbZtWejnXgghhBBCCCGEEEKcRkqpdwKPaK3z9vN5aa2/fZzD/W9gL3CD1nrMHr8R/OwG3wABAABJREFUE8T+b0BeCDrPnGszVxdaz1wU0FJrWicLsZQkhBViGXHibgHAgUJXB54nbtoWDWVG/ev2PZpPVK32UMx7ErXrxhsLqX0q3FJYdeBrwWB2THt1MR/1BgrRYtbfmMt4RuuVqp3ck/WUCvl9rauyh1ovC3mUitRM9VdlQw0H/YWUF6j+4sXXJi8d2lGtElPegj+aqM0kxpsLhcZVeR0MFFJqxeHnSplA7Yp3BMPZ6MSLKl53kSfsLQT3NVQX9256d/D6fY+FV6RHvOsGDxUuHX81sH39rb6H11yVe7qxNXznnserR4NVnpJSCSB9Tf93ddHj82QCNZE3H3w+s7+qeeJB5+d9DelJz6/secwH+Dq724+0rO3ftqUEZN2Yo5y4W/iN//IXkadaLg+m/SFv29aeaRuEajfmTH5/zbWh3Q1tPHzxTZkpjy51drfn/7+3vrem2LSh6uH1b5zGBLZBIPq+2+7Of/XRezLfvvjmsLdYWFX0+qYj+Yx+w+Du0vfWbR7ChJXBNtsKubwOu6w8xwpjZrNO2zmrXszfHmG7XxAIZHxBPRht0HXZxITdL2DvT2OC1Slm5r9OAXX2GNMc/bu8ClMVG8QE0157vZJtW3umB7ylIpDu7G4vdQJtW3tSQKhta089M22JfUBgS99PfG/teK20t26NHxMuH9MyuH/bFs1MoBwFcv3btkzM3sauQ2FbJQeKhTTA1YdfCdvbknNcOyGEEEIIIYQQQghxZnwTaAGG7efz0ZjXtxbyFioCWACt9ahSaisyJ/a8tL1v7JyauXq89VQqR8J337HprM6uFecnaUcsxPKlgWIm1Fh62w8+P/7ef/3/2bvv+CrL+//jr+uck70ZIWHJ1AQIKkuQ6UbRVq2jrqK/Dldrx9dW7FLrt2q/tdZqbW3VVqx7j6K4kCFDZAcIiEAYIUAge59x/f6472ASAwQInBzyfj4e5wG55+ecQO6T+30+1/WTvXijdpWmZe3eOODiWm+wPrU6rmuc1wbrgOK4YH1l/+IvdqZWbv2irHP22qF5/97as+BDbN0MW+55c0/G7qXbE2p3h9J3LfHidEd6T9n0XqAuUF510ra5pZm7F1d23724PiEYKjbGlANVfm9spQn6o04o3xnKLt5S3Km2LDOmujB1whfvVJ+zeX4oua6ueEta34SVmack1sR2Cu1KzOyaHorpdNGm+dV7Y5NLnhzyjfInc74Reu/N271Rwdqa1UNurNydPjIaqPn9W48EAU9xXIovP6lbQ6BqAPpMm2H6TJsRPXPkpKia2C5JeVnZ0T0qi5KH71rnG1S0MeHkXV906zNtRjTA8i4DYnM798uujIob8Mjsh+sbhiueULCybsSu9eUZlXtqB5Zs65xSV2GBmNLYpORZPU6NnrJpfsrA0m0xQLAkJjGYn5zpA2JG1vpie/k9nYDUPs6csw31fO1DLW4wWYsTpIITbnbCCUjB6Ra1QClQuyGlZ2BbYtcQTherD6fj1Yczv24AJyyucYchLgNK8h+YEmoInN3jNby7KMIJaBsP91tW6rX2j6k1+2rNf2BKQw1R7vls/gNTKq5e90H9d9bO9GSVbI3DCYS97vNpkRvG1gO+PtNmJDRe5+7nxemstdnr8ur+OOLqGiDx/pHXxdJoqOM+02b4GjqaRURERERERETk2Gg8BLH79/09DhbAgnOvJ6mF5Q3TeMlxprVzqR6rOVcP5Typ8VH8/dphYZmzVo5/6oQViVC3Pn6mzcvKrgPi8rKw2evyArc+fmbwsZtm+QBbnZhZur3XGXQpWVuZvS7P9nGGw43B6WgM5mU9HEqt3LnTV7crNHi79SXVENrTaXDQHxXfqfPetbXRgarqa9d94MUdlhaw8bV74kaseKgEJ9QztQndkmsTupWUxHUtK+wxLlhRlbc3GKz2xMf09uVs+TjNV1ds/KY+pru/Jj6uttg3ZMfS2gHlhUlx/urQ8937+vfGJtX/cvF/4oB0oDZz52eVoUB5HeB97KZZnptNLLPi/FU5xfnROCFhwzC3UYn11TELTrzUjt21PiXgi/MP2726tDwqIa3SF5NUExUb6FGxO/6JSZf7PN7o+MF7NtfvSOpS8cgpl3HbtBleN3gMjdm5JvDywElJFtJ7VOyuL4tJCgBJL5x0dvlPl79cnV5TVrMOyj/PHNzw5jKQFDJJQWO8OywJQUMdTugZjTOMb41bX4z7ZyxOZ2hDyFiN8+GqTjjBp6/htQX843esrEvw1ySVxiRWV8QkVrrbNmwXk//AlIpG/wSqaToscjXusMzuaxVyg9Hm3au22X7kPzCl2h1WOAYI9pk2w0PWufXXrfvA/nDl61Uz+p5u3e2sO+SwJ/+BKQEANxj3ZK/LC+Y/MKXODVxNn2kzjHv+hv32dbrmZWUnvAeh8y9+MPBlWq9AQ4e3KwbnA0ItdRWLiIiIiIiIiEj791/gn8aY7wKL3WWnAY8DBxvKWCJQa+dSPVZzrh7KeUqqdRtSjh6FsCLHnxrA1sR1DdXEdSV7XV5D4NbQ4drQMWmA+snLbAiozl6XZ1+4/NG6tNL1yZ5QIIjz8yHD3bcWJ+QDJyCLAhJTyzb6y5NOKCnoOamu0961qSfWBev/0+Ok4m9tzY3xhIKb4oP1UQNKtvaOC/pjgbi+e9cklNeVR1dHJ9nlXc+tHlOUn5Sf3IN44mqjQrW+eYlRcSu6jqmvGJsZyor5NHbwrnGeb1RH1+N8es7/yoBJMYu+/1fo3Nd//dp3zeiifG9V2qC6tdEh38oBE2ozvfNtSfqQ6iGb/eVda2pT8lJP6OoxIV+36uId3auKA0FM/ICSbeAM5+uriIrz5SdnhmIDdYGq6LhYYBfg3Z7czfc/E39U4r5WKTjD/lYCyXPi/NE+iwkaavgqKPQDnL9pQfLazn3jtqRkwlehcVHDNyb/gSmhPtNmFOOEtg3jbpS53zNOKtlWsrTbSWkVMYkBnLA8BNDnq/lvG7NAQp9pM/z5D0ypawhF3e7cxoH1Pm4o2uKwv24wXe2GqJ0Af/a6vHIAvppz1g/EnOD3xN51y0d16SFPzZlu125eVnZV9rq8UP4DU2r7TJuRiDOEclWj44cana7h32S5W7Nxa/a4r4XG/RARERERERERCQNjjAe4HrgU6INzH2cz8CrwH2tta6bZvA2YDizkq/tnPpwA9sdtW7G0B6P6diIzJZadZbUtzsN6rOdcPVg9jR3r+WqlY1EIKxKhHrtpVjSTHrO3Pn5mZaNlMYC99fEzvzasR/4DUwJ9ps2odsM2cP7/x+FcZ6qBuqte+VF9XlZ2FU5gG+1uU4IzTEhDR2zDULkAsb23zwqe99FjtfNO/1ttQnVhcIy3JrnK1iYRqNgF1IfiMipKPR5WJSZGn1hWGMisKi5YHx8flVNSkFrQZXD8jJQBNbGbPiVj79pUP56ylekD6xIGvBpVFFXm2ZNQUPPyL/4RhDzysrJtTVRMSnTQHwUUT8lfVA147x5xZWBzckanoDcq5E0KVhtPKCquPL6+qMbWxvhrE0pjk+OqfDFp6TUlwSmbF/j7VuwM5GX9xQvUJvlrTG1UbHRtVGwBTtAbdF+LUGbVnoRJ25bHFiZ0rpzda1hMWk35gB5VRXZrYvqO8tikgLttvfvahvKysv0f9Rru8WLrt6RkBnDC2wCNhvF9783b46498Wzz7KDJBid8LXeP4wPMHe89Wd9n2oy97mu/7/3Bz0vjuPXxMxt3ix5IHW63cyu3x52vNRqodbtWoWkQmgh4+kybUQrU9/d7fekhT8Mcr34glL0uL+QOI+z71obZrE/rtd96s9flVbvnTMb5t9Ywr63Jf2DKsRmTREREREREREREmjDGGJyg9AJgJZCLc/8nG3gaJ5i9+GDHsdaWAt80xgwEstzFedbaL9u8aGkXvB7DXRcN4uZnl+0b2q9BOOZcbVxPg2B1GaXznqVm4xKC1SV4YhOJ7tqXlLFXUcigNp2v9sUXX+Sqq67im9/8Jm+++WabHFMik0JYkcgVgzP0rB/gsZtmGZwgLQjUu8PExuN0NDaEhY2DOT9Op6LBCQsbhNxlDXOZxgFx+Und6gsTOofG7FybWufx+f844uqyb2z8NHbo3k1ReVnZcV3Af/7FD9aeXPRF9OkFq+oTA7WJQMgX8m//ovNJIU99UbrxpXTanN4r3le/Z1u3isKYESW7qrem9SnvXzAvPrG+qn5olKfyeSj7v//sjdnSldAZq+dafuEUlb0uL5Rx7nXF83v1C+Et597JDweff/i8wJfOMMvVgAlWDAsBZetSqcp/fEqwz7QZmxNrK3sP37UuwRoTXxqbVDdoy2dF7nOqzl6XF2LajMSogD/2tF1ryz/tcXJD8FzVs2K3p39ZQfA7694Pzu41rNYXClb3LdtRfea2ZcWPnHp5HM5wvyGAvKzseMBz9ralxWdvW2r/228sfDXPqs/9PpiPeg4Luq9vGU7gGXK7RqOAuj7TZjQM87vv+/HYTbOigNjHbppV1zhcd7ta9wXwzZa3OoB1xbg1NgTtDV3Ajf9NND52xWM3zfLc+viZIcgLNarX16dsR/QVX8wKJfurjfOhx5blPzAl2GfajGr33DHuc2nNJylFREREREREROTouB6YAJxlrf2k8QpjzJnAm8aY71hrn2nNway1G4ANbV6lHFPBkGXx5mJ2V9SSnuR0s7YUpk4eksnfrx3GPe+spbDsqz6LjJRY7rpo0DGfc7Whnmmv5VJa46fojfuwwQBdpvwUX2oGwaoSaresJFTjzADXVvPV5ufnc/vttzN+/Pg2OZ5ENoWwIpGrybCytz5+pn3splk1gO+xm2aZM50gteHxNe4wxV8b8D57XV7D/KXkZWXX4HRBRhUkdI3ZG5scVWe89QHjrehdsSsBX1JqSXJfm1xZsMMbqu/y6ju/jHrupHMCvSqLYt3z7k6pyK8fXrU9NjoUSNiZ0L0+N61b0vw+Z8Z2qS33lwb2ek7eOo+PBp4fP3jH8vL1aT2ikuqruu7w5NSfsTrXAh63htDc7kPpXV1i45Nnp8RGj0+pqkqte+j/vbbnvQW/tm/1G1c+o8/o4HXrPrA9K4uS+pYXevPevD32rm7Z5oGR1+6JIrQrFDLRJ5ZusTiBXzRg8rKyq5MuuKdm2K4vkj2EugLF7lM3rzz985q8rOza8y9+MA6oK0pI2/VO//Ex7uvRDahwa4sBQp/0ODXt0+45iZtTuleOLlztjamvS9iS0i0mP7UnOB2z5k8jro51/17XaHhej1tPyH00udrPj/WHegU8od4Bb6DPtBlxQDD/gSlf63Q+QtatqyFsDTVb/7UhjJ0A9mtq81O61yb7q320Igh255AN4Xb+NswhKyIiIiIiIiIiYXEVcF/zABbAWjvLGPMAcA3wtRDWGPNQa09irf3ZEVUpx8zM1YVfC1UzDxCqTh6SyTmDMloV2h4Lk4dksq6wgodmLKdu+xq6XXU/sb1zAPClpBPT/aR924Yq9vDNb97Cxx9/jMfjYfLkyTz66KN069aNdevWMWzYMJ588kmuvvpqAF5++WWmTp3K0qVLGTRoEADBYJBrrrmGe+65h3nz5lFaWnrMn7O0LwphRSLUfkIwH+58oNnr8oK00Cl5KNygtiIvK9vjtUFPTvGmxBgb9MQEg6HzNy8sjvEkxYW8UfFgA0D09sT0Trvj02o+T8+qTPDXVp5YVuB/ecDEpPhAXdzZW5fEdK7e5c/eFQj2KNnSKz5Qv+PlrLOqhxQXdE4OBqPK47tE9ystDFz6xWzTvXJ3ACeMDLqPxMqouLj6RF/SObl7/P8aHFURXbsnbnfFhtRPM4d4PaFAWs/KoootyRnbx+9YFdja80xPp+K1jN6VF/Xmf39Vh9PRGwJ8H/YaHp9SV1U/oKyATnUVodyHrqj66WU/L92c3N1+L/ftqFdOPNOUxSSm9Zk2o4yLHwwAaW4NDd2ewZTaiqrk+uqGYZt9QN2S3uOjbVRM1JnbltYGjOlW74mKii+urUuvKfMvzhxc654/Dqh5783bvf9cPD3qvlFTo3CHQU4OGl+3oImnWQi7IDYQixPUWvd8R0MtTYPhJva3vIXtnBD1gSmtns0+39m21duLiIiIiIiIiMhRMxQaxqVr0Xvsf+izU1t5Dn0IP0LMXF3Izc8u+9o3bGdZLTc/u4y/XzusxSDW6zFtNqzvkZq5upC/fLwBEx2HiY6jesMiYrpnYXxR+7YxQLekaH532/UkJiYyZ84cAoEAt956K1deeSWzZ88mKyuLBx98kFtuuYVx48bh8Xi46aab+MMf/rAvgAX43e9+R3p6Ot/97neZN29eGJ6xtDemdfNoi0gkcIck9pw5+9aGwC/QMBRxW3CHOIavOmy74HRQVgExvx793YSlGdnRmZV7Sn6y4pXqoXs2Ri3vOjDu456nRp2z9XPvyXs3V9V6fN22JKZ3qohO2FMam2yzSrbEb+wyqGBG76Fct/bdmJy9m8pwgs8QsAtIKY5JSn69/4SKhPrqhC61ZambUnuUpdZV1nlsyC7MGBydVbwlOi5Yz8Cygp3Lcm6s61xTlmIT1oZiyl4ylyyyfsBTmAb/HNfPLvJ9PwUbFQ1UdQ2Y8usrY/1nzr7VLOqW7R29Ky/2uRPPtms6900evnt9TU1UbPFzWed2wQlaG4LIqIs3zPHcuOadkvMvfrDypOItMes7neDt6adrp6Cn8g8zf1bywolnp32Zkhl38u4NMcP3fFl69tJ5e9z5UmOB0OVfzEocuSuv9hfjbw3ihOZRY2t8/vSgx6yKCdRsjAoF3HCSPtNm+ABP/gNT6vtMm2HULSrSKvp/IiISucLzEXlpT3QdFxGJTLqGHweMMfXACdbawv2s7w5sttbGHNvKjh1jTDJQVlZWRnJycrjLCZtgyDLuD7OadMA2ZnCGGf70jjPD1uV6MPWBEKPv/5jiKuf2eNX6+RTPfBQbqCe6W39ieg0hIXsC0el9+UG/Cn57y7Vs3ryZXr16AbB27VoGDx7M4sWLGTlyJAAXXngh5eXlREdH4/V6mTlzJs5UyvDpp5/y7W9/mxUrVtClSxeuv/56SktLNSfs8atV//DVCStyHLn18TMtEMzLwuB0T3ra8vhuZyyAzcvK9uGErw1dnp6gx7sH8P7roweCOMP22lOLNlRuSerWzRcKpRbHJJasTzuhcEDptuKUQI3tV7EzuCOhc81nnbtX5RR9EdW/rMAAgVf6T6xIryru5LOBASUxSV4v1rcxtUfFuB0rd3euqwgl7tnkTa8uCf1zyEUVJbHJccbjKblw40LS6ioD94z2R12zc1Wix7/KXx9tKI+z/uQa6rekm+TxWzYn5mdui94Z1y+UvWdzzPiysjRf8tBKvy+e0bvygntjknwV0XExQCC1virw6xn/sM9Nm7EHiOlcXdIjob4mEB+qxxcKemeeMCoARK3vdEIUQMbevOrhu9fXZa/LC/4O9vSZNiO+ODYldUdyt2CPrOyo56MTfA+MvC60qusA/8LMIVXRQX+NOy9qHBDqHPJUpIaMd2NUqKHz1Q/gzhFLn2kzGr6nhzrfq4iIiIiIiIiISGt4+Wo0uJYEUabQISzeXLzfABacT80VltWyeHNxu+l6bWzm6kJ++UYuxVVfDcCXcNJY4vuPpHbbGup2rKN201LKP3uNq/7n9yTWxtOrV699ASzAoEGDSE1NJS8vb18I+69//YsTTzwRj8fDmjVr9gWwFRUVXHfddTzxxBN06dLl2D5Zadf0A1PkOOSGpZUAeVnZ8e6y6jY+R4Cmwx3Xvun8Gch783YfjT4JMr5gZXVCoNZ/z2k3eFakn2huWvVmxa74tJSU2sqoOl+UHb1zbfcBJdtKtiZ07b8zPq024PGYqpjYmM5VZd4dCV0DBrvj4o3z7LKuJ0a/2X9C1SOz/8K7A0/tvDW9Z5ezvvis+Htr/xsEKrPX5VmmzfDUhmpLzllbVhdvQp5X+14QtT2xa+yPc5+t3tE56P9m5cr6fwzt5//Jileil/YYkbwkJpRWMuDsmr571hR3ryiKPqFil62KiqvF2io3+Iw+ZfcXnrLo+JgTynbWl8Um7t2U2iP1vb6jk0fsXBuzI6FL+Y6k9Pq02oqolLrKfV3H+Q9Mqe4zbUb9us59QhdsXhi3skv/3gNLt1Ws6jpgx6eP3VAPN9Bn2oxooL5z0Hg+jfXHL/vjBZX/njZjfyFrHODtM21GpbphRURERERERETkKDDA08aYuv2sP247YKWp3RX7D2APZ7tjaX/DKAMYXzRxfU8lru+pMPYq9r73CB89+1dG33mgUbi/snLlSqqqqvB4PBQWFpKZ6QzHvHHjRvLz87nooov2bRsKOQMr+nw+1q9fT//+/Y/4uUnkUQgr0jG0eWjnDk3sdcPYJtxlpXlZ2V7A07muoh6oyqza491e3dXz+NCL/UDZPQueiO5bvjMhzl8b8nt88bvi0zwVCd1jor3RFSeWrP9yeUZmTJJ3R/J5X2yoWdl1QHStL6pr98o9gT+MuGanzdga6moLo8uSetXizKsa+8boc4O/jE+zP5j1UmVeVnYUEFjedaBnQsHKBE8gujp7e03ZpeuetNN/8WL84oyBXdNq9kSllm8N7YhLid3a/eTgj1a9Xnn+lsXBrJKt3ulZk9OBisF7NnkHlGxPLEjsUrus20mV5bFJ8UCoZ+mO6OG71qV+u2yHd2HmkKI1nfrYjJri6LysbM/5Fz9ocYYe9uc/MCV03XUra6p9MUW13uhAw/yqfabN8OK8cQ2cWueLCxoMUHmA+VfrAW/+A1Nso31rWztfq4iIiIiIiIiIyEFMb8U2zxz1KiTs0pNi23S7YyUYstzzztpW3wyP6tyL+m1LyM7OZtu2bWzbtq3JcMSlpaX75nwtLi7m+uuv51e/+hWFhYVcc801LFu2jLi4OLKyssjNzW1y7F//+tdUVFTwl7/8pUmHrXQsCmFFjnNt3QHbSAwQlZeVXZ29Lm9/3ZsxOMOYGCB6UPGWmJDx8n6f08x7b94edJfXAGUE60za3k2l/s7DOkV5EqouWvxx9cc/Gu7rsjfWvpo1MnFXdE9PeUzinimbF/iG7t0UW5DhL431r62MK/OH/jXogtSRu9Z5NiVnevfEp5qLvvdI4f+5868+OucvcSUxiaEkf011w3DKnZPf6JTk2dP19O1FiSYY3FDn9dZ0qqswbq2Je2NTvHviUkN9y3YEp2xe4A94vDF9ywqSYoP1cTsSOpNsC/3L+8d67Y7qHYOL8+sHF+fXrOzS39+lpizafd4e3Hlk+0yb4aHHydFAcePA1B2KuAYInlrvC3CQMeTdYYkbAm+PW6uHr+aqbbWc6TnRgMmdmru/TzUespzpOQbnOQfcY4fc5bGAP3dqbtDdJhaoz52aq2GVRURERERERETaEWvtDeGuQdqHUX07kZkSy86y2pY7SnHmhB3Vt9OxLu2A9jeMcrCmnKI3HyBx6DlEd+2DiY7Dv/NLKj9/nWsvv5izzz6bnJwcrrnmGh5++GECgQC33HILEydOZMSIEQDcdNNN9OrVi1//+tfU1dVx6qmncvvtt/PYY48RGxvLkCFDmpwzNTUV4GvLpWNRCCsih6seCB0ggAVnvlgPznXZDirOr+tZsdv8ZMUrMe66Gpy5JGJxfh5VnVCysXDSvOdDeVnZ3qviiNuSRk2BpzwhMa6gR26X/gULu+fUb0tKt5O3LK78bPCe1KouvdM2xWdG13l8URtTuu/eG5+WsDOhc/yG1J41A0u3h/5y8mWpe2OTo7NKtjJh+PjqNSfuqR/c20OX+lDl3phOtig+KbQxtVf5Lz//TzxOaFyVXbzF8//WzggNK9rQ0NFaWR7t824f8GXFpC8y4uO8O+IykuuqNg+iPPvtPD9AtvM86gDygT7TZlTlPzAl1GfajGjjrSK2+0u+nOnT/DjhaTB3aq5tmO8Vd/7XnOk5UTgB5r5hjVuS/8AUf59pMwL7G5Y4Lyt73+vZaB7fxqLc70ldo31M9ro8mzM9x+vWcKD5P1oSRaMhaXKm51S5X6cCdTnTcypxAmMfzmulEFZERERERERERKQd8noMd100iJufXebc2G20rqGT5K6LBuH1HLCv5ICCIcvizcXsrqglPckJdI/keLD/4ZE9UXHEdD+Ris/fxF+6E0IBvEldufzqqTz28H0YY3jrrbf40Y9+xIQJE/B4PEyePJlHH30UgGeeeYZ3332X5cuX4/P58Pl8PPvss4wbN44LL7yQ888//4jqluOXsVZTC4rIseMGhFE4AWHIXWYAT+NA1x3KOBYnKPQt6XriCe/3Hln6aa9T/UBo5M61dV2iP0urD6VGrY0dV5u9Nz8hJhQofa/vmGqc9wXB/AemBEbc9mxSvScqbmzhKiYVbYseuHdjbVFScVR+zElxG5L6BM7dvqyoV2VRAEjACUhL3BKMW2ccUPWnSzye6hji9iYZ+81FwdDYPGx0gED2uryagz1nN1yNxQkhPUBd86A1Lyvb8+pY02Vjpgk8/b9rig9yLJs7NTeQMz3H5E7N/doP8bys7Gi39uqWQli3I3Vft6q7TyIQuuJOn3FrrGzp2Aeoq+H1CgFJQIW7KgY3hM+dmluVMz3H29AF6wa+oUM5j0gr6d+UiEjkOrK7LnI80HVcRCQy6RouxwVjTDJQVlZWRnJycrjLCbuZqwu55521TbpLM1NiueuiQUwektnujrtw416uemLRQbfrnBDN7y8ZckTnkg6vVdc9hbAicsw1dF0e4j6JgD3/4gergejn3707yYP1/XTCj4r/9dED/g97DU//IrVX8L/9xxXjBJ7eOxc/Uzthx6quQP3fRv+o/LRaX3xs7d7A4HXP+G8+77r4tJraxN/Of7MsNuhPwenQLAHKAc/uFOLfPs3j61Ju6y9eZKvW9DaeGq9N3tzdxO7obCp//HaoDqeb01OYhvfMhXm1bsAcaJgn1w2XE3an4P/hLT6L0/EajTM8b6jZ8/MUdCJ+d6qpv/rdtfvthM2ZnpOEE3TW4ATH/typuS1/xOvQXt84nBA2AHhyp+b6GwWr/tYGpW6wGo8z5HCdu6xhqGKL2wWcMz3HA3QGanKn5lYe4HgtBs0iB6F/MyIikUs3cEXXcRGRyKRruISVMaYP8BvgTCAD2AE8C/zeWnvAUeeaHUchbDNt3bE6c3UhNz+77Gtv+hqO+Pdrhx12OBoMWcb9YdZ+h1EGSIjx8vjVwzl9YJcj7ryVDk0hrIgcP/Kysj0AjbpnE3A7NgFKYhLjVnfuG7pv1NR6nE5M/3tv3h4C0nA6XItmTXos2HX3slDO2qcSnjojKb7Qe1LSSfmJ5Zd/OTsGJ9AscE8XUxpP6Knz4oPp9VM4bT01J2x+JXVzNzqHDBWDCijLXpdXkZeV7dvSNSa9KCUhfcnAyrXXfFIfM2N01+iV/eIqpr24NWr6WZ6Y+Fqb2KWcmufP9O7B6aoNNHTBPnbTLM+Zs2+1gGl4XgfjhpzgBLGx7vH8R/La7i9sdeeOjQFqD+UczYNTd17YzjhdzQZnKOoATlgbwvkeWpxQO8atpdb9eyJQljs1t6bZObyaV1YOQG9uREQil+6CiK7jIiKRSddwCStjzGTgSuAF4EtgCPAE8B9r7e2HcByFsEdRQ0ja0ryt8NVcs5/eceZhB6QNIS8c+I1lW3TeSofWqn+gmhNWRI5Yn2kzDE6g5s9/YEqrP1l2KBqHlO5QxR7A36ijtvp04L5pMzw4AZ/f/XM3TpgXfebsW0PnX/xg7W/3Dg6dt2pN7fu9PcFZvYZXX/7l7DqcOW7rcDpV61Or8d88M8G3I7M+oT4qKWgNoco4qqMC1Ls1JNZ5SbKmiw35kmpP/7K779WJxdVFnQcnZlRGJ6RUb/Un1WCrYvHHBjB/fSzg+fuFnqg1J3hCAI/dNMsHxBV2G+XJ3LU4lJeVXdma7uCX7w/4gKC77b5g0h2m2Nu8K9btOLWNuk9j3ecabHiuNJ3PtfH3z4/zXuVQ54clZ3pOjLt/w7ywfrfeRPd8FqfzOAEnnLY439M4nO+DdR9RQEzO9Jx6d10NzrUrOmd6Ts1hzF0rIiIiIiIiIiLHGWvtTGBmo0WbjDEnATcD+w1hjTExfHVfDJzmDjlKFm8u3m8AC87NwMKyWhZvLmZM/86HdY7JQzL5+7XDvjbccXM7y2q5+dllR9R5K3IwCmFFpK0Yjt2nHkM4oV5L3ZkNQaY3e12eHydsrHaH2zVA7D+HfsP7yOxNoe8uX1z54LrptXlZf/Ly1fyvIaA2e11efV5WdnTn4rX1py96uy5n+otVuVNz9+RlZfsAX3U0cVu60i2/W1HhaRvjt910y5e1QOyYTdmlAQq8V9zpC+ROza10O3g9QOxdz4dC2evW1AKMXPJA1O6up0S9dtq2oM/viZ51iodcmoamzZ+YO7xxNE4oGnCDTuMGr1GAN2d6Tl3Dvm6Ha4K7fY37/BsCbI97rIZhkmn+errHOZwuW697bNxzl/NVZ2sZTsga79ZW5dYQzVdz5la6c95aYI9bQzxON20pznyzBud7KyIiIiIiIiIi0pIUoPgg29wJ3HUMahFgd0XrZlVr7Xb7M3lIJucMymDRpr3c+twySmu+fovT4txgvOedtZwzKENDE8tRoRBWRI5Y/gNTLO6wwMeC2wV6SFfi7HV5NXlZ2bH/N+9v3l+Mv6UmyV/jA3zu/LTBvKzsGpxQLwT43cDTm1a6gW/fMzhuQJSJe+XcQWaIExr64usxGCr676j3pJZtzPzdsxllv712d10lnyQM3WxtyG+qccLcEBDKy8qGRiF1UuU2T1Lltvq53/b5weN1O1UbQtMgUO1+7W3o9rziTp/35fsDVYDNmZ5jYutsbMizL4isueu5YPTgrTYqh5wgTsdrHc7PeS/O3KvBnOk5lY1C2lCjTtIWO5jd4Y9Nw3YNQxc3DKncEjdArcGd/xVomF/Ww1fvb6Ib7RKDM4dsKGd6Tmmj1yLdfR4NofEu3K7ctpgHV0REREREREREjk/GmAHAjzhAF6zrfuChRl8nAduPVl0dXXpSbJtudyBej8FjTIsBbIO26LwVORDNCSsiHUZeVnY84Mlel1eZl5UdjRNQ1jQMA5yXlR2FM9RvKC8r22xPI6U2ms6buhPa0tVw4WLrzyylCCfUtDifpjObMlIzlpyYUffBqaUlJliUePGiUOWZK21tnJ/q7HV5dY06WP3NhlU2APnpxL0/3GMf/vOaGre7NZQ7NdfvzqUahRM8BoHYpGrr71oGmzKNZ+S6UNppX9iqCWtsafa6PPvuxOxkgDv+n68GJ8wN4XSPBoHtuVNzvzbvbM70HB9O0Fvn1uTFmaO2IXRNADy5U3Mr3K9j3OfSEFon4IStNc2P3ew8Xr6aXzaUMz0nxX0NGzpk69x6U91je4HTqLOjCNk+GFNMrHkJ2Oiev1hzwsoB6M2NiEjk0sfPRddxEZHIpGu4HBXGmAeAOw6yWba1dl2jfXoAc4DZ1trvHeL5NCfsUdQwJ+zOstoW3/S1xZywjb21ooAfv7jioNv95dun8M1Tehzx+aRD0ZywIiKNnX/xg7VAQvqP/hMz3bm50ziA9c4bbJLX9DbMmp5TnLsuz778vUG2W7EN9N+BP2eTDS0cZLzWkPCtBbZhKOTqei+pa3uU19ZEhTqdsqGmZPkAAksHmpopS2wUkJqXlb0XYM4Qk/RldxN40BlOF3A6el89d5B3eT9SqmII5EzPaZjH1pczPSf68nnBYHwdCXNyPJ6SeOrKkoxn0Fab1msP7E2iJuilftiXtvaKO33RTM+x6Zd4SayxFudnu8UJN8vdY+57X+MOqWyvuNPXMAywN2d6Tv1l80K+C2KJS3TeA1W4m9fR9IJSjzNcckNIG8IJTw/G6z487vy1DR2/Ab4aWjgO6O/WbIDe+OmPpTNeUgnaiXhNMc6Qxl40HLGIiIiIiIiIyPHuT8DTB9lmU8NfjDHdgU+ABcAPjl5ZciDBkGXx5mJ2V9SSnhTLqL6d8HoMXo/hrosGcfOzyzA0/fRdww3Iuy4a1GZDAx/LzluRliiEFZGOxAKhqXkzfTgdpjU4ISPZ6/KCL31/cG1eb7PvCm+gvDbG1PbfZb0VsXirY4337dGe6m8tCDTMIVu3vTM19d5QqEtxeTA1isCJO4y/516biDN0bhzOMLo+T9B683qa+JH/HMKVc0Pegi4m+NGpnjKu8ZrkKltVnmAahult6BCtu/xTWw/UDygIenrsJeqFiZ4q66Eua5s1r47z1NwyI5iQWLcv3LQjNtj609aHYnem2fq/X+j14PyMLwb8zeaYjcvtjXHX1wDVgMfvo/Oqvsaenmf3NmzYKGz14nToWhoNXZw7Nbe6NS987tTc+pzpOQG3C9bghNEN4XBDV7IF9rr1dAKqiGIrls3AXrxmq7uurKWuXhEREREREREROb5Ya4uAotZs63bAfgIsBW6w1ur+URjMXF3IPe+spbDsq9nEMlNiueuiQUweksnkIZn8/dphX9smw93m/JzuvPHGG1x88cVHXMuovp3ITIk9aOftqL6djvhcIi1RCCsiHYY7d21VXtbtiTjhaJN5Te9+Yk1V469/++Ra63aNepJqqbr2k1DctZ+EooHgm6ONnZPjCf3ypWD5iTtNYre9tvrDYXgySm0ouYrEJf1Jum5GXrG7vy+xFo8vSHTAS8qWdAJ10dTizrWaMz2n1hu0cSO+CGVuTffEFHY2RT9+I+D5zySTPDfHU9Nnl/Wctt528oZsYOZwb8lHp2IAT3IN9ds7Y3ECYX9VLMwa6onb2tUEceborcXp2LXu0MZ+dwjfmqUDjReISaqyZsEtq4M503PMxkwqipKNefhibzC30euQMz3HgxMOh2jUydtovWkc8rpDHAebBb+4dXg7lVtvcbLxuMsaguosoC+wAyhx615OjFkOphbYjfuBOAWwIiIiIiIiIiLSmBvAzga24MwD29W4vRbW2p3hq6xjmbm6kJufXfa1wHNnWS03P7uMv187jMlDMnnxwTtJLynloYee+lq37JEqLi7mrrvu4oMPPmDr1q0kpXaitsdw0sZfi4lJ2Lfd0ei8FWlOIayIdES1QG3DUMQHUY8zR2ooLyvb4nSdmiUDPd6CLsZOWJxXlZeVHQB6T14GVbHs2tADX73PxOVlZSdkr8uruvnng0JRAWI6V7B3U3djFwzyxH/r02Bpxh6b+u38wVFXbLWVb472eH0BEzxhly3us9uGKuNMr/gaG2tDds/6HqbWF6C2PopglN/G+KMMQN0Vd/qigaQeu0Ox3hBFc4Z6EtLKQollCcTjDNlbiTMnbMh9HvznwuyohF7ExNeZypv/G6zpXEHM75YNqst9cm0wZ3pOgK+6UvfNWO92r4ZwQtSGdyQNwwFHA9E503Oq3fMk48796h6vJndqbsN+PfoV2oRxa0Klr47zxFXHmobu13qgO5DJV8PHlOJ26PLVHLwBIDZnek5U7tTcffWJiIiIiIiIiEiHdw4wwH1sb7ZOCdsxEAxZ7nlnbYsdpw1DAN7zzlrOGZQBgDEwpn/nNq9jx44d7NixgwcffJBBgwaxZcsWvvP/vk/lh38l6cKvphfOaNSdK3K0eA6+iYjI8SV7XV4ge11eoJXb+rPX5dW7f6/OXpdXDlT3KrKVf/5HwORlZUdlr8sLAns7VbHr3Pl51UUpnm0jvvxqSN+CLp5gXL2p+PnroXIg6oTdJKdV0HVnJ1Jj621KfA1x0UGi5+d4ap74w5q9u1JNdcDLzqo48qtjTWxyVahvvY/Egs4mYUi+Teu7I2Rwfn6neOttN0+IDF+A9Oja0MD4Gjr3LAqFzl4eSvnGopAHJ0ytc7dPen6Cp+snOZ7Ed04zCQVdTOjzE43vo1M80e5www3Bps2ZnhPvzt2KG6D6ccLrBCAJJ9xNxglirXt8j7s8BScw3XeNcbtiK4Dqz040SdVR9ALi3WP0xel+XQl8iRMCN3TzVuMEynV8NYesOmFFRERERERERGQfa+3T1lrT0iPctXUUizcXNxleuDkLFJbVsnhz8UGPtWfPHi655BLi4+MZOHAgb7/99r51v/vd7+jevTt79+67/cqUKVM444wzCIVCDBkyhNdee42LLrqI/v37c+aZZ/LnPz5A5YbP+M8NI/jLt0/hhe+P5tM7zlQAK0edOmFFRA5R9rq84I1Z2R6cD3AZd1l5w/q7nlxjcQJLAN6+Y3UNztyrMD2n+MLPgvTfCc+c7Sk0ltitXUmojDMGqMmZnpNIpqnelOmtwh2qtyba1BZ0IlQbTTzYap81dZcsCMUvHWDYmm42FCeSXh1L0BpCJUk2dOIOAqd+GYqaM8STjhNuVgOJQGJRqvGml9mqaD++t0d7fDjBKDjhahROMJqOE3juAvzJVTahPMEk4nSnNsxS78cZ3jjgDlfcsL7EXV8DpAKpOdNzgjghatWeBDpXxpgReE03INc9vwdn7tpqt4ZKoM4Nbht3vNp9r6OIiIiIiIiIiIi0G7sr9h/AHup299xzD//3f//HH//4Rx599FGuueYatmzZQqdOnfjVr37FzJkz+d73vscbb7zBY489xoIFC1i5ciUeT8t9h2VlZSQnJzP+pG6H9JxEjpRCWBER9nV7JuAEi3UH2z57XV6IrwLMVsudmhvMuz+7EmDhzatrcqbneCoSiMUJF6txhvIFd/jdM5aHiirjCX1+kicIeJcP9AZ7FlkGbwmZohRCW9NNoCrR7MbpPt1SnWBiv+hpbXQgVL50oOkCdMqdmlvmBqFVSdU2NsZvY71BYsB0xwlbQ+75vO5xduCEpr1+fuug3efV0C2vp+Xkrab+5bEe64820UDtSdtCUaMfH5JAnEnDCWG7uccoxAlzTwZMcplNiK+3u3Z29exNqbYZvgC1xTEmH/gCqHL3rcIJXy3OfLLqdhUREREREREREWlHgiHL4s3FLc7jmp4Ue5C9HXsq6rAHmSTu+uuv56qrrgLgvvvu45FHHmHx4sVMnjwZr9fLs88+yymnnMK0adN45JFHePLJJ+ndu3fL59uzh3vvvZcf/OAHrX+iIm1EIayISFOtmSf2a/KyshOBYPa6vIN2ajbeJndqbpU7n6pxg8eGzs96oD7v/myA0BV3emrd2sz2rsZWxVLzZaapcrf38lWImVYdZ6IWZ3sNTmdqnRswe4HaPWmesupYG++1pHbba2P2pJAa9JlYnDB2t3ucAPW2L9HGFqSZmlNLbSCl2rC8j+kV8lAFBONqQnElsbavCZganNC1J84wxTVAGrAHp5PXX++lu/FSDgRLEk2Bx7IRZ4jhhiGLY4DK3Km59Yfz2ouIiIiIiIiIiMjRNXN1Ife8s7bJkMOZjeZVHdW3E5kpsewsqz3gDdZ7Z+RRtbqQgSn7Hyl66NCh+/6ekJBAcnIyu3fv3resX79+PPjgg9x4441ceeWVXH311S0ep7y8nClTpjBo0CDuvvvuVj9XkbaiEFZEhH1zllYewSEaukkP99z727eq0Tbs224q9v85QS0503P8OCGqj6/mTq3JnZpb6673uPv5gWB1nPH0KLKBLuV2b02MKSpPpDOQgRPUbkstC0WZgPVWJHgqcwd6euT1sTYQZXZ4/baiS5mN7b0rFJXfhS41UQyo84X2grfIPW8cTjevcY8XAKpqE83WWsxWYG9lomkIjetyp+Zat7ZSmg47LCIiIiIiIiIiIu3EzNWF3Pzssq/dwNxZVsvNzy7j79cOY/KQTO66aBA3P7sMw4FvlNbUB1mxvYqZqwtbnJc1KiqqydfGGEKhpgPnzZ07F6/XS35+PoFAAJ+vadxVUVHB5MmTSUpK4o033vjaMQ/U1SvSVhTCioi0gex1edVH6bgHDXZzp+b63TAzAQjkTs2taLY+lDM9p8r90wPUFHQ11eUJlFTEGz+wHViDE/gGrdekekMQiMIkVIZ6dK7Av6ML9YEoE5VYY70+S6zxmNpANDtCXrMbJ3hdB/TGCV4bzh/A6XiNwglmvblTcyub10aj+XNFRERERERERESk/QiGLPe8s7bFUNXi3PS75521nDMog8lDMvn7tcO+1jG7Pw37HWr4+dJLL/H6668ze/ZsrrjiCu69917uueeefevLy8s577zziImJ4e233yY2tulQyQfr6hVpKwphRUSOA27AWo3TkdtEzvSchqGOG0LP6mbr/Y3nYM2ZnlMMpgxI7rfLJg8otP4b3w+V/Op6X3l+hknamm4C/mhTDSTjdL+mAUXAMpz5XZNxunQr3OGWo93tAkfjuYuIiIiIiIiIiMjRsXhz8QEDVQsUltWyeHMxY/p3ZvKQTM4ZlMHT8zdz74y8/e4XqqtiyxdreXaGl5N7pQLQuXPng9azfft2br75Zv7whz8wbtw4/v3vf3PhhRdy/vnnM3r0aMrLyzn33HOprq7m2Wefpby8nPLycgC6du3Kh3m7W9XVK9IWFMKKiBwncqfmBpsvy5me4wPicqbn1OZOzfW7yxIAmzs1t9rdL9TsOAEgkDM9Z29CPaXfWGw9CXXOcMFBn/EHoTZ3am7t7743qO6V8d5YnKGE/ThBayywCydwbQh+63GHThYREREREREREZHIsbuidYPYNd7O6zF0SYo54PZ1W3MpfPo2rn/6q2Xf/e53D7iPtZbrr7+eUaNG8cMf/hCA8847j5tvvplrr72WFStWsGzZMj777DMABgwY0GT/Lzdu4p53NrWqq1dDE0tbMNYe1hSGIiISAdzhh+Nw5mANuMvicULYmgPs1/AuIxbn/UcN4AFigNqX7w+AM/xxffa6vDp3n2h3+8rmwa7IMaY3NyIikUt3OkTXcRGRyKRruBwXjDHJQFlZWRnJycnhLqddWLhxL1c9seig273w/dGM6f9VJ+vh7nc0tceaJGK16rqnTlgRkeOYG4ZWNVvWmvlrE3AuJNHu13U4N8QaZrCvAYI0Gv5YHa8iIiIiIiIiIiLHl1F9O5GZEsvOstoWPy1ngIyUWEb17dQm+x1Nh9PVK3IkPOEuQERE2qUgTqBahdPZ2jDUcRTgzV6XZ7PX5VVnr8vzh61CEREREREREREROaq8HsNdFw0Cvt761/D1XRcN+trwvYe739GUnhTbqu32VNTx1ooCFm7cSzCkgVrk8Gk4YhER2S936GIvThBr3SGHQw1DG4u0U3pzIyISuTSUoeg6LiISmXQNl+OChiPev5mrC7nnnbUUln3VJZqZEstdFw1i8pDMNt/vaAiGLOP+MGu/3bkAHgONc9dw1SrtXquuewphRURkv9w5ZU2jTliRSKA3NyIikUs3cEXXcRGRyKRruBwXFMIeWDBkWby5mN0VtaQnOUMJt6aT9XD3Oxpmri7k5meXAa1749lQ5d+vHaYgVhpTCCsiIiIdkt7ciIhELt3AFV3HRUQik67hclxQCNsxtNSd27wDtrGG+Ws/vePMsIXH0u4ohBUREZEOSW9uREQil+5oiK7jIiKRSddwOS4ohO04Gnfn7qmo494ZeQfd54Xvj2ZM/87HoDqJAK267vmOdhUiIhIZcqbnxAGe3Km5VeGuRURERERERERERORo8XrMvkD1rRUFrdpnd0XtwTcSacQT7gJERKTdsKjzQERERERERERERDqQ9KTYNt1OpIFCWBERASB3am5t7tTc6nDXISIiIiIiIiIiInIkjDG8+eabrdp2VN9OZKbE7nd8WQNkpsQyqm+ntipPOgiFsCIiIiIiIiIiIiIiItLuXX/99RhjMMYQFRVFt27dOOecc/jXv/5FKBQ6rGOGggF6bnidgqduZetD32L7Y99hz3//RKBi775g9q6LBuH1aPprOTQKYUVERERERERERERERCQiTJ48mcLCQvLz83nvvfc444wz+PGPf8yFF15IIBA45ONVV1dTuu0Lbr/jTob+8HG6XvxL/MUFFL1+Lxkpsfz92mFMHpJ5FJ6JHO984S5AREREREREREREREREpDViYmLIyMgAoEePHgwbNozRo0dz1lln8fTTT/O9732vyfbPPPMMt9xyC8uXL2fgwIEA3HLLLcyaNYtly5aRkpLChx9+CMC9IcvizcXMn5zFz6+9kOevGkDfPgpg5fCoE1ZEREREREREREREREQi1plnnsnJJ5/M66+//rV13/nOd7jgggu45pprCAQCzJgxgyeffJLnnnuO+Pj4Jtt6PYYx/TtzSrcYjDF07pR2rJ6CHIfUCSsiIiIiIiIiIiIiIiLHTNDtON1dUUt6Uiyj+nZqcc7V5ttZu/9jZmVlsWrVqhbX/eMf/2Do0KHcdtttvP7669x9990MHz68xW1ra2u54447uOqqq0hOTj6s5ycCCmFFRERERERERERERETkGJm5upB73llLYVntvmWZKbHcddGgJnOvtrRd1epCBqZ8PawFsNZiTMvr0tLSeOqppzjvvPM4/fTTmTZtWovb+f1+rrjiCqy1/P3vfz+cpyeyj4YjFhERERERERERERERkaNu5upCbn52WZNgFWBnWS03P7uMmasLD7hdTX2QFdvL9m3XWF5eHn379t3vuefOnYvX66WwsJCqqqqvrW8IYLds2cKHH36oLlg5YgphRURERERERERERERE5KgKhiz3vLOWlkYUblh2zztrqQ+E9rsdjbYLhr7aYtasWeTm5vKtb32rxe0XLFjAH/7wB9555x0SExP54Q9/2GR9QwC7YcMGPvroIzp37nxoT06kBRqOWERERERERERERERERI6qxZuLv9bZ2pgFCstq+c/C/ANvF/CzvaCQGQvX0DOunpkzZ3L//fdz4YUX8p3vfOdr21dUVHDddddx2223cf7559OzZ09GjhzJRRddxGWXXYbf7+eyyy5j2bJl/Pe//yUYDLJz504AOnXqRHR09BE/d+mYFMKKiIiIiIiIiIiIiIjIUbW7Yv/BamNbiqsPuL5281K2P3Ydl/7DR6e0NE4++WQeeeQRpk6disfz9QFgf/zjH5OQkMB9990HQE5ODvfddx833ngjY8aMwe/38/bbbwNwyimnNNn3k08+YdKkSa2qW6Q5Y+2BGrpFREREIo7e3IiIRC4T7gIk7HQdFxGJTLqGy3HBGJMMlJWVlWk+0KNg4ca9XPXEooNu95sp2dw7I++g273w/dGM6a9hgyUsWnXd05ywIiIiIiIiIiIiIiIiclSN6tuJzJTY/aZXBshMieW6MX1atd2ovp2OTqEibUQhrIiIiIiIiIiIiIiIiBxVXo/hrosGAV9vI2z4+q6LBhHt87RqO69HTfjSvimEFRERERERERERERERkaNu8pBM/n7tMDJSYpssz0iJ5e/XDmPykMxD2k6kPdOcsCIiInK80ZsbEZHIpY+yi67jIiKRSddwCTtjzNvAKUA6UAJ8BNxhrd1xCMfQnLDHSDBkWby5mN0VtaQnOUMLt9TZ2trtRI6xVv0jVAgrIiIixxu9uRERiVy6myK6jouIRCZdwyXsjDE/BRYChUAP4EEAa+3ph3AMhbAi0hoKYUVERKRD0psbEZHIpRu4ouu4iEhk0jVc2h1jzDeAN4EYa61/P9vEADGNFiUB2xXCishBtOq6pzlhRURERERERERERETkuGGM6QRcAyzYXwDruhMoa/TYfgzKE5EOQiGsiIiIiIiIiIiIiIhEPGPMH4wxVcBeoDfwzYPscj+Q0ujR8+hWKCIdiUJYERERERERERERERFpd4wxDxhj7EEeWY12+SNwKnAuEASeMcbsd9hQa22dtba84QFUHN1nJCIdieaEFRERkeON3tyIiEQuzScnuo6LiEQmXcPlqDDGdAU6H2SzTdba+hb27QlsA0631i5s5fmSgTLNCSsiB9Gq657vaFchIiIiIiIiIiIiIiJyqKy1RUDRYe7eMBJoTBuVIyJySBTCioiIiIiIiIiIiIhIxDLGnAaMBD4FSoD+wL3ARqBVXbAiIm1Nc8KKiIiIiIiIiIiIiEgkqwYuBT4G1gNPAauAidbaunAWJiIdlzphRUREREREREREREQkYllrc4Ez2+p45eXlbXUoETkOpaSkJAMV1lp7oO3MQdaLiIiIiIiIiIiIiIgc94wxPYDt4a5DRCJCirX2gJ/YUAgrIiIiIiIiIiIiIiIdnjHGAN2BinDXcgSScILknkT282gv9Hq2rePp9TxoJ6yGIxYRERERERERERERkQ7PDVQKwl3HkXByZMAJiDSu8hHS69m2Otrr6Ql3ASIiIiIiIiIiIiIiIiIixxOFsCIiIiIiIiIiIiIiIiIibUghrIiIiIiIiIiIiIiIyPGhDrjH/VOOnF7PttWhXk9zkDljRURERERERERERERERETkEKgTVkRERERERERERERERESkDSmEFRERERERERERERERERFpQwphRURERERERERERERERETakEJYEREREREREREREREREZE2pBBWRERERERERERERETkOGKM6WOMecoYs9kYU2OM2WiMuccYEx3u2iKFMeZWY0y+MabWGPOZMWZUuGuKVMaYO40xnxtjKowxu40xbxpjTgp3XUebQlgREREREREREREREZHjSxZOBnQjMBj4KXATcF84i4oUxpgrgYeAe4BhwErgfWNMelgLi1wTgceA0cA5QBTwgTEmIaxVHWXGWhvuGkREREREREREREREROQoMsb8HLjZWtsv3LW0d8aYz4DPrbU/dL/2ANuAR621D4S1uOOAMaYrsBuYaK2dG+56jhZ1woqIiIiIiIiIiIiIiBz/UoDicBfR3rlDNg8HPmpYZq0NuV+PCVddx5kU98/j+t+jQlgREREREREREREREZHjmDFmAPAj4B/hriUCdAG8wK5my3cBGce+nOOL21X8MDDfWrs6zOUcVQphRUREREREREREREREIoAx5gFjjD3II6vZPj2AmcAr1tonwlO5yD6PAUOAb4e7kKPNF+4CREREREREREREREREpFX+BDx9kG02NfzFGNMd+ARYAPzg6JV1XNkDBIFuzZZ3A3Ye+3KOH8aYvwIXAhOstdvDXc/RphBWREREREREREREREQkAlhri4Ci1mzrdsB+AiwFbnDnNZWDsNbWG2OWAmcBb8K+IXTPAv4axtIiljHGAI8ClwCTrLWbw1zSMaEQVkRERERERERERERE5DjiBrCzgS3A7UBXJwcDa626OQ/uIWC6MWYJsBj4CZAA/DucRUWwx4CrgW8CFcaYhrl1y6y1NeEr6+gy1tpw1yAiIiIiIiIiIiIiIiJtxBhzPfsJDK215thWE5mMMT8Efg5kACuA26y1n4W1qAhljNlfGHmDtfbpY1nLsaQQVkRERERERERERERERESkDXnCXYCIiIiIiIiIiIiIiIiIyPFEIayIiIiIiIiIiIiIiIiISBtSCCsiIiIiIiIiIiIiIiIi0oYUwoqIiIiIiIiIiIiIiIiItCGFsCIiIiIiIiIiIiIiIiIibUghrIiIiIiIiIiIiIiIiIhIG1IIKyIiIiIiIiIiIiIiIiLShhTCioiIiIiIiIiIiIiIiIi0IYWwIiIiIiIiIiIiIiIi0oQxxhpjLm70dZYxZpExptYYs2J/y9obY8z1xpjSQ9wn3xjzk6NTUfvV/HsuR0YhrIiIiIiIiIiIiIiISAdgjHnaDdqsMcZvjNlljPnQGPP/jDHNM6NM4L1GX98DVAEnAWcdYFnY7Cc8fQk4MQzlRKLm33M5AgphRUREREREREREREREOo6ZOGFbH+B84BPgL8B/jTG+ho2stTuttXWN9usPfGqt3WKt3XuAZYfEGBN9OPu1lrW2xlq7+2ie41gwDt/Btzx8LXzP5QgohBUREREREREREREREek46tywrcBau8xaex/wTZxA9vqGjRoPTWuMscBw4Lfu8rtbWuZu28sY87IxptQYU2yMecsY06fRcZ82xrxpjPmVMWYHsP4Q97vdGFNojNlrjHnMGBPlrp8NnAD8uaHb113eZDhiY0x/99i7jDGVxpjPjTFnH+qLaIz5njEmzx2KeZ0x5pZG6/5ljFlljIlxv442xiw3xjzjft3HrfHbxpgF7jFWG2MmNjrGJHeb840xS4E6YJwxxmOMudMYs9kYU2OMWWmMuazRfmnGmOeMMUXu+g3GmBsa1fFX9/WrNcZsMcbc2dL33P06xxgzyz3OXmPMP40xia39nnR0CmFFREREREREREREREQ6MGvtLGAlcOl+NskE1gB/cv/+YEvL3PDtfaACGA+MBSqBmc06Xs/CGcL4HODCQ9jvDJzu2zOAqTih8fXuukuB7cBv3Xoy9/NcEoF33RpOxekMfscY03s/23+NMeYa4HfAr4Bs4JfAvcaYqe4mtwEJwAPu178HUoEfNjvUH3Fev1OBhW4dnZtt8wAwzT3PKuBO4DvATcBg4M/As40C3HuBQTihejZwM7CnUV3fAK7Aef2vAfL38xwTcL4nJcBI4HLgbOCvzTY90PekQzuqbcsiIiIiIiIiIiIiIiISEdYBQ1taYa3daYwJAJXW2p3u4srmy4wx1+I0AH7PWtvQiXoDUApMAj5w961yt6k/xP1KgB9aa4PAOmPMDJww9QlrbbExJghUNKqxpeeyEidwbvAbY8wlOOFk84Bxf+4B/sda+7r79WZjzCDgRmC6tbbSfU5zjDEVwE+AM6y15c2O81dr7Wvu870ZmAx8F/i/Rtv81lr7obtNDE7ge7a1dqG7fpMxZpx77jlAb2C5tXaJuz6/0bF6AxtwhpC2wJYDPMergVjgO9baKvf8P8QJiu+w1u5yt9vv9+QAx+4QFMKKiIiIiIiIiIiIiIiIAewRHuNkYABQYYxpvDwWp1uyQW5DAHuI+61xw74GhUDOoRToDqd7NzAFp1vWB8ThBJSt2T/BrekpY0zjoNEHlDV8Ya1daIx5EPgN8Adr7actHG5ho+0DxpglON2rjS1p9PcBQDzwYbPXKRpY7v7978BrxphhOOH1m9baBe66p4EPgfXGmJnAf621H9CybGBlQwDrmo8Tlp8ENISwR/w9OV4phBUREREREREREREREZFsYPMRHiMRWIozzG1zRY3+XtVsXWv38zdbZzn0qTcfxBkG+XbgS6AGeBUnyGyNhjlRvw981mzdvjDSGOPBGVY5iBOeHq7Gr1XDuacABc22qwOw1r5njDkBuADneX5sjHnMWnu7tXaZMaYvzlDFZwMvG2M+stZexuFri+/JcUkhrIiIiIiIiIiIiIiISAdmjDkTp3vxz0d4qGXAlcDuFobePRr7NVcPeA+yzVjgaWvtG7CvM7ZPa09grd1ljNkB9LPWPneATX8OZAETgfeNMTdYa//dbJvRwFy3Dh8wnAMPibwWJ2ztba2dc4Aai4DpwHRjzDycuWdvd9eVAy8BLxljXsWZd7eTtba42WHygOuNMQmNumHHAiFg/QFqFJeSaBERERERERERERERkY4jxhiTYYzpYYwZZoz5JfAW8F/gmSM89nPAHuAtY8x4Y0xfY8wkY8wjxpieR2G/5vKBCe5z67KfbTYAlxpjTjHGnAw8z6HnZXcBdxpjbjPGnGiMyTHG3GCM+RmAMeZU4Hc4c9zOB34G/MUY06/ZcW41xlxijMkCHgPSgH/t76TW2gqcTt4/G2OmGmP6u9/DHxljprrn/p0x5pvGmAHGmMHAhTiBKsaYnxljrjLGZBljTgQuB3bizL3b3HNALU6QO8QYcwbwKPCfRvPBygEohBUREREREREREREREek4JuPM25kPzATOAG4Dvtlsbs9DZq2tBiYAW4HXccK/p3Dmdt1vh+vh7teC3+J0tW6k6TDGjf0MKAEWAO8A7+N04raatfZJ4HvADUAuMAe4HthsjIkFnsXptn3H3f6fwCfAf4wxjTt1p7mPlcA44BvW2j0HOf1vgHuBO3Fep5k4wxM3DCVdD9wPrMLpsg0C33bXVQC/wJln9nOc1+oCa22ohedYDZwHdHK3fRX4GPjhQeoTl7H2SOdYFhEREREREREREREREZHWMMb0wQlNT7XWrghvNXK0qBNWRERERERERERERERERKQNKYQVEREREREREREREREREWlDGo5YRERERERERERERERERKQNqRNWRERERERERERERERERKQNKYQVERERERERERERERGRY8IYM9sY83C462gNY8z1xpjScNchkUkhrIiIiIiIiIiIiIiIiEQsNyy1LTy+12ibaGPML4wxK40x1caYPcaY+caYG4wxUfs59EvAiW1QnzHG/MAY85kxptIYU2qMWWKM+YkxJt7d5u5GdQeMMfnGmD8bYxLd9X3cdUFjTI9mx89097HGmD5HWq+0DV+4CxARERERERERERERERE5QuXASc2WlYETwALvAycDvwHmu9uPBm4HlgMrmh/QWlsD1LRBbf8BLgX+F/ghUOTW8hMgH3jT3W4NcDZOfjcW+BcQD9zY6FgFwHeA+xstm+ou790GtUobUSesiIiIiIiIiIiIiIiIhIUxJs0Y84wxpsTtUH3PGDOw2TbfN8Zsc9e/YYz5WQvDBFtr7c5mj4YA9SfABOAsa+1j1toV1tpN1trngdOADfuprclwxG636gpjzHVup2qZMeZFY0zSAZ7fFcA1wFXW2vustZ9ba/OttW8BZwKfNNo84Na93Vr7EvAc8I1mh5wO3NBs2Q3ucmlHFMKKiIiIiIiIiIiIiIhIuDwNjMAJG8cABni3YYhgY8xY4HHgL8ApwIfArw7xHNcAH1lrlzdfYa31W2urDuFY/YGLgQvdx0Rg2kHOvd4NXZuf21pryw6wbw0Q3WzZ20CaMWYcgPtnGvBOa5+AHBsKYUVEROSoMMbcbIxZZYwpdx8LjTHnH2D7lubuqD2WNYuIiIiIiIiIyLHjdrx+A/ietXaetXYlTmjZAyfoBPgR8J619kFr7RfW2r8B77VwuBR3vtWGx85G6wYC69qobA9wvbV2tbV2Hs5Qw2cdYPuBwPpDPYkxZjhwNTCr2So/8Czw/9yv/5/7tf9QzyFHl+aEbYExxgBJQIW11oa7HhERkQi1HedTgBtwPsE4FXjLGHOqtXbNfvZpPnfHIV2HdQ0XERGJXLqOi4iIiHRI2UAA+KxhgbV2rzFmvbsOnHtFbzTbbzFOF2pjFcCwRl+HGv3dtEm1jnxrbUWjrwuB9ANsfyjnzjHGVAJenA7YGThzyDb3L2CBMeaXwOU4HcTK/NoZfUNalgSUlZUdqANcRETaqbZ8QyVHwFrbfAiUXxljbgZGA/sLYa21dud+1rWGruEiIpFN1/GOLQkoW7F8GZs2fhnuWkRE5BD16z9A13ERCbeQtXZ/byS/ALLa6DzNO04tBx559lDOvR6nMzgA7LDW1re0kbU21xizDngByLPWrjbGnNLKc8gxouGIRURE5KgzxniNMd8GEoCFB9g00RizxRizzRjzljFm8EGOG2OMSW544Ny8FRERERERERGRyJCH0zB4WsMCY0xnnO7Xte6i9cDIZvs1//pgngfONsac2nyFMSbKGJNwiMc71HOfaIz5ZgvnNsaYlEaL6q21X1pr8/cXwDbyL2CS+6e0QwphRURE5KgxxjQMoVIHPA5cYq1du5/N1+PMYfFN4Fqc9ykLjDE9D3CKO4GyRo/tbVW7iIiIiIiIiIgcXdbaDcBbwBPGmHHGmJNx5jctcJcDPApcYIz5mTFmoDHmRuB8Dm0aq4eB+cDHxphbjTEnG2P6GWOuABbhzNt6tLwMvAS8YIz5pTFmhDHmBGPMhcBHwBmHedwngK7Ak21Up7QxhbCHSNPSiIiIHJL1wCk4n2b8OzDdGDOopQ2ttQuttc9Ya1dYa+cAlwJFwI0HOP79QEqjx4ECWxERERERERERaX9uAJYC/8UZQc0AF1hr/QDW2vnATcDPgJXAZODPQG1rT2CtrQPOAf4P517TIuBz4DbgEWB1Gz2Xls5tgatx6r8YmAOsAu7GCZrfP8zjBqy1e6y1gbapVNqaUaj4de5whmVlZWUkJyc3WTdt3jS2V2xnWPowTk0/lVPSTyEtNi08hYqISEs0B007Zoz5CNhorT1QsNp4+1eAgLX2qlZuv99r+JO5T/JB/geHWvIhi/HGcMeoOxjSZchRP5eIyHFI1/EOrOE6vmL5MpKSNMOAiEik0ZywInIsGWOeALKstePDXYvI/vjCXUAksdayuHAxRTVFrCxayb/X/BuAvil9GZY+jOHdhjOs2zC6J3THGL3nEBERaYEHiGnNhsYYL5ADvNsWJ95VtYu84ry2ONRBPb7ycf561l+PyblERERERERERI53xpjbgQ+BKpyhiKcCt4S1KJGDUCdsC/bXRWOtpaCygOW7l7Ns9zKW71rOxrKNX9s/IyFjXyg7vNtw+qX0UygrInLs6AduO2GMuR94D9gKJOEMu3IHcJ619kNjzDNAgbX2Tnf73+IMBfMlkAr8HGeIluEHmEe2+Tn32wm7sXQjO6t2tsEz27/i2mJ++ekv8Rkfn1zxCamxqUf1fCIixyFdxzswdcKKiEQ2dcKKyNFkjHkZmIRzj2kT8Ki19vGwFiVyEOqEPQTGGHom9aRnUk8u6n8RAKW1pftC2WW7lrF271p2Vu3k3c3v8u5mp3EnLSaN4d2GMyJjBCO6jWBg2kA8RtPxiojIcS8deAbIBMpw5ro4z1r7obu+NxBqtH0a8ASQAZTgzAVyemsD2IPpn9qf/qn92+JQB/TM2mdYV7yOD7Z8wBUnXXHUzyciIiIiIiIicryz1uomi0QcdcK24EBdNAdT7a9m1Z5VLN21lKW7lrKqaBV1wbom2yRHJzO823BGZoxkZMZITkw7UaGsiEjb0SdvO7AjuYa3lX+v/jcPLX2I4d2G8/Tkp8NSg4hIBNN1vANTJ6yISGRTJ6yIiEhTCmFb0JY3cOuD9azZu4alu5ayZOcSlu9eTnWgusk2CmVFRNqUfunrwNpDCLuzaifnvHoOAB9e9iEZCRlhqUNEJELpOt6BKYQVEYlsCmFFRESaUgjbgqN5AzcQCpC3N4/Pd33Okp1LWLZ7GVX+qibbpMSkMKLbCEZljOK0zNM0p6yIyKHRD8wOrD2EsADXz7yepbuW8rPhP+OGITeErQ4RkQik63gHphBWRCSyKYQVERFpSiFsC47lDdxAKMC64nUs3rmYz3d+zrJdy77WKds5tjOjMkdxWsZpnJZ5Gj2Teh7VmkREIpx+6evA2ksI+/L6l7l30b1kdcrilYteCVsdIiIRSNfxDkwhrIhIZFMIKyIi0pRC2BaE8wauP+Rn7d61LC5czOKdi1m+e/nX5pTtkdiD0zJPY3TmaEZljKJzXOdjWqOISDunX/o6sPYSwpbWlnLGy2cQsAHe/Oab9E/tH7ZaREQijK7jHZhCWBGRyKYQVkREpCmFsC1oLzdwwZlTdmXRShbvXMxnhZ+RW5RLwAaabHNi2omMzhzN6MzRDO82nPio+DBVKyLSLuiXvg6sPV3Df/jxD5mzfQ4/GPoDfnTqj8Jai4hIBNF1vANTCCsiEtkUwoqIiDSlELYF7ekGbnPV/mqW7lrKZ4WfsahwEetL1jdZ7/P4GNplKGO6j2FM9zEM7jwYn8cXpmpFRMJCv/R1YAe6hr+xfDvzv9x71GuIjfJw86QBrCz+hDvm3UHPxJ68e+m7mt9dRKR19MOyA1MIKyIS2RTCioiINKV0LsLER8Uzvud4xvccD0BxbTGLdy5m0Y5FLCpcREFlAct2L2PZ7mU8tuIxkqKSGJU5itO7n86Y7mPoldQrzM9AREQkPJZvLeXVpduPyblq/SF+d/Ek4nxxbK/cztq9axncZfAxObeIiIiIiIiIiIiEn0LYCNcpthOT+0xmcp/JAGwr38bCwoUsKnRC2Yr6Cj7e+jEfb/0YgJ6JPRnbYyxjuo9hVMYokqL16WIREekYzhnUje6pcUf1HEUVdTz16WY+ytvFA5fmMK7HOD7c8iEfb/1YIayIiIiIiIiIiEgHouGIW3CgoQyttREznGAwFGTt3rUsLFzIgh0LWLl7ZZP5ZL3Gy9CuQzm9++mM7T6WQZ0H4fV4w1ixiEibiIwf0nJUhHtKgUAwxKj7Pqa4qp7nv3caJZ7PmDZvGv1S+vHWxW8d83pERCKQruMdmIYjFhGJbBqOWEREpCmFsC040A3chz78go27K7lpYn9yeqaEp8DDVOWvYsnOJSzYsYAFOxaQX57fZH1KTApjMscwtsdYTu9+Ounx6eEpVETkyOiXvg4s3CEswC9eXcnLS7YzdcwJ3H7+CUx4aQKBUIC3Ln6Lfin9wlKTiEgE0XW8A1MIKyIS2RTCioiINKXhiA9BrT/I9AX5lNX4mZFbyOn9O3PTxP6MH9glIrpjE6ISmNhrIhN7TQSgoLLACWQLFvBZ4WeU1ZUxM38mM/NnAnBi2omM7TGWcd3HcWr6qUR5o8JZvoiISEQ4b3AGLy/ZzvtrdnHXRYM5LeM05u+Yz6yts+iXoxBWRERERERERESkI2j3nbDGmHzghBZW/c1ae+t+9rkcuBfoA2wA7rDWvnsI59xvF826neX8Y84m3l65g2DIee0Gd0/m5kn9OX9IJl5P+w9jWxIIBcjdk8unBZ+yoGABa/auwfLVv414XzynZZ7GuB7jGN9jPJmJmWGsVkTkgCLzB7G0ifbQCVvrDzL83g+pqg/y5q1j2VDzIb9b+DuGdB7CCxe+EJaaREQiiK7jHZg6YUVEIps6YUVERJqKhBC2K9B4otIhwIfAGdba2S1sfzowF7gT+C9wNXAHMMxau7qV5zzoDdztJdU89elmXly8jRp/EIC+XRK4aWI/Ljm1J9E+T2ufYrtUXFvMwh0LmV8wn/k75lNcW9xkff+U/k4g23M8w9KHqUtWRNoT/dLXgbWHEBbg1ueXMWNVITdP6s93J3bhzJfPxGL58LIPyUjICFtdIiIRQNfxDkwhrIhIZFMIKyIi0lS7D2GbM8Y8DFwIDLQtFG+MeQlIsNZe2GjZImCFtfam/RwzBohptCgJ2N6aG7glVfU8vSCfp91higG6p8Ry06T+XDGiF7FR3gPuHwlCNkRecR6fbv+U+Tvms7JoJSEb2rc+3hfP6MzRjO85nnE9xunmsoiEm37p68DaSwj7zsod/OiF5fTrksDH/zOR62dez7Ldy5g2ahrXZF8TtrpERCKAruMdmEJYEZHIphBWRESkqYgKYY0x0cAO4CFr7X372Waru/7hRsvuAS621p68n33uBu5qvvxQbuBW1QV4/rOtPDFvE7sr6gDomhTDD8b345rRvYmPPn6m3y2rK2Nh4cJ9oeyemj1N1g9MG8iEHhMY33M8J3c9GZ/n+HnuIhIR9EtfB9ZeQtiKWj/D7/2I+mCID386gQV7XufBJQ8yKmMUT533VNjqEhGJALqOd2AKYUVEIptCWBERkaYiLYS9Ange6G2t3bGfbeqBqdbaFxotuwW4y1rbbT/7HHYnbHO1/iCvLNnG43M2UVBaA0DnhGi+P6Ef140+gYSY4yuQDNkQ64rX8WnBp8zbPo9Ve1Y16ZJNik5ibPexTOg5gbE9xtIptlMYqxWRDkK/9HVg7SWEBbjh34v5ZH0Rt597IhePjOP818/Ha7x8csUnpMWmhbU2EZF2TNfxDkwhrIhIZFMIKyIi0lSkhbDvA/XW2osOsM0hh7AtHOOIb+DWB0K8sXw7j32yka3F1QB0Sojm++P78Z0xx18Y26C0tpT5O+Yzr2Ae8wvmU1pXum+dwZDTJYcJPScwoecEsjplYYzem4lIm9MPlg6sPYWwL32+lTteyyWnRwrv/Ggcl79zOeuK13F11tUM6TIkrLUdS3G+OCb0nEC0NzrcpYhIZNB1vANTCCsiEtkUwoqIiDQVMSGsMeYEYBNwqbX2rQNsd8jDEbdwjDa7gesPhnhrxQ7+OmsD+XudMLZzQjQ3TuzHdaP7EBcd+XPG7k8wFCR3Ty5zt89lXsE81hWva7I+PS6d8T3HM7HnRE7LPI34qPgwVSoixxn90teBtacQdm9lHSN//xEhCwumnclbW57mbyv+FtaawuW2U2/j+0O/H+4yRCQy6DregSmEFRGJbAphRUREmoqkEPZu4Eagl7U2cIDtXgLiG3fLGmMWAKustTe18lxtfgM34Iaxj8zawBY3jO2SGMOtZ/TnqlG9iY06fsPYBruqdvFpwafM2T6HRYWLqAnU7FsX7YlmZOZIJvacyMSeE+me2D2MlYpIhNMvfR1YewphAb719wUs3VLC/ZfmMHloEvcvvp/yuvJwl3XMlNSVsHbvWoZ0HsILF75w8B1ERHQd79AUwoqIRDaFsCIiIk1FRAhrjPEAm4EXrLXTmq17Biiw1t7pfn06MAeYBswAvg38EhhmrV3dyvMdtRu4gWCI15cX8OisDWwrdkLIzJRYfnTmQC4f0ZMor6dNz9de1Qfr+Xzn58zdPpc52+dQUFnQZP2A1AFM7DmRSb0mkdMlB6/n+A+pRaTN6Je+Dqy9hbCPfryBP334BecN7sY/rhsR7nKOuaLqIs585UwMhtlXztbc8CLSGrqOd2AKYUVEIptCWBERkaYiJYQ9F3gfOMla+0WzdbOBfGvt9Y2WXQ78L9AH2AD8wlr77iGc76jfwPUHQ7yyZDuPztpAYVktACd0judn55zIRUO74/F0nPcs1lo2lW3aF8gu372ckA3tW58ak8r4HuOZ2GsiY7uPJTE6MYzVikgE6Dg/QOVr2lsIu2p7Kd/463wSY3ws+805RPs6xoetGmuYC/e+cfdxUf+LDr6DiHR0uo53YAphRUQim0JYERGRpiIihD3WjuUN3Fp/kOc/28rfZn/Jnsp6ALIykrj93JM4KzsdYzree5eyurJ9wxZ/WvApFfUV+9b5PD6GdxvOpJ6TmNhrIr2SeoWxUhFppzreD07Zp72FsKGQZdR9H7Gnsp4Xvj+aMf07h7ukY+7hpQ/z1OqnmNJvCg+MfyDc5YhI+6freAemEFZEJLIphBUREWlKIWwLwnEDt6ouwL/nb+YfczdRUetMeTuyTxrTzs9i+Akdd+i+QCjA8t3Lmbt9LrO3zSa/PL/J+v4p/ZnYyxm2eGiXoRq2WERAN287tPYWwgL87KUVvL68gBsn9uPO87PDXc4xt2TnEm54/wbSYtKYfeVsPKbjdQOLyCHRdbwDUwgrIhLZFMKKiIg0pRC2BeG8gVtaXc/jczbx7/mbqQs4Q/KeM6gbd0w+iQHp+iV0S/kW5mybw5ztc1i6aylBG9y3Li0mjfE9xzOp1yRO7346CVEJYaxURMJIv/R1YO0xhH1rRQE/fnEFWRlJzPzJhHCXc8z5Q34mvDiBSn8lz1/wPDldc8Jdkoi0b7qOd2AKYUVEIptCWBERkaYUwragPdzA3VlWy8MffcHLS7YRsuAxcOXI3vz07IGkJ8eGpab2pqyujPkF85m9ffbXhi2O8kQxMmMkk3pNYlLPSWQmZoaxUhE5xvRLXwfWHq7hzRVX1TP8fz/EWlh051lkpHS86/hPP/kpH239iFtOuYWbT7453OWISPum63gHphBWRCSyKYQVERFpSiFsC9rTDdwvd1fwh5nr+XDtLgDiorx8f0I/bpzQj4QYX1hra0/8IT8rdq/gk22fMGfbHLZWbG2y/qS0k5jUaxJn9DqD7M7ZGgpR5PimX/o6sPZ0DW/s4sfms2JbKX/4Vg5Xjuwd7nKOude+eI27F97N0K5Dee6C58Jdjoi0b7qOd2AKYUVEIptCWBERkaYUwragPd7A/Ty/mPvezWP51lIAuibFcPu5J3LZ8F54PXp/05i1ls3lm5m9bTZzts1hRdEKQja0b33XuK5M7DWRM3qdwaiMUcT6Ol5HkshxTj8U2wljzM3AzUAfd9Ea4HfW2vcOsM/lwL3uPhuAO6y17x7COdvdNRzg4Y++4OGPNnD+kAz+fu3wcJdzzO2s2sk5r56DwTD3yrmkxqaGuyQRab90He/AFMKKiEQ2hbAiIiJNKYRtQXu9gWut5d3cnfzf++vYsrcagKyMJH49ZRDjBnYJc3XtV0ltCfMK5jF722zmF8ynOlC9b12cL44xmWOY1GsSE3pOoHNc5/AVKiJtRb/0tRPGmIuAIE6YaoCpwM+BU621a1rY/nRgLnAn8F/gauAOYJi1dnUrz9kur+ErtpVy8WPzSYrxsey35xDl7XgjMlz69qVsKNnAH8b/gQv6XRDuckSk/dJ1vANTCCsiEtkUwoqIiDSlELYF7fUGboO6QJD/LNzCIx9voLw2AMDZ2en88oJs+nVNDHN17Vt9sJ7Pd37OJ9s+Yfa22eyq3rVvncEwtOvQffPI9k/tjzF67ygSgfQftx0zxhQDP7fWPtXCupeABGvthY2WLQJWWGtvauXx2+U1PBiyjPjfDymp9vPyjWMY1bdTuEs65h5a8hD/XvNvvtH/G/x+3O/DXY6ItF+6jndgCmFFRCKbQlgREZGmFMK2oL3ewG2upKqeR2Zt4D8LtxAIWXwew3fG9OHHZw0kJT4q3OW1e9Za1hWvY/b22Xyy9RPyivOarO+V1GvfPLKnpp+Kz6M5eEUihH7pa4eMMV7gcmA6Tifs2ha22Qo8ZK19uNGye4CLrbUn7+e4MUBMo0VJwPb2eA3/8YvLeWvFDnqmxdE1KebgOxwlF5/Sg6mn9znm511cuJjvfvBd4nxxDEwbeMzP3yA9Lp3/Hfe/JEQlhK0GETkgXcc7MIWwIiKRTSGsiIhIUwphWxApIWyDjUWV3Dcjj4/X7QYgLT6K/zn3JK4a1VvzxR6CnVU7mbt9Lp9s+4TPCj/DH/LvW5ccncz4nuOZ1GsSY7uPJSlaNwRE2jH94GtHjDE5wEIgFqgErt7fHK/GmHpgqrX2hUbLbgHustZ2288+dwN3NV/eHq/hM1cXctOzy8JdBjE+D6vuPpcYn/eYntcf9HPOq+ewt3bvMT1vS353+u+4ZOAl4S5DRFqm63gHphBWRCSyKYQVERFpSiFsCyIthG0w94si7v3vWjbsrgQgOzOZuy4axOh+muf0UFX5q1i4YyGfbPuEedvnUVJXsm+dz+NjRLcRzrDFvSbRI7FHGCsVkRbol752xBgTDfQGUoDLgO8BE/fTCXs4IWzEdMJaa1mypYTSav/BNz5Kpr22ir1V9bxy0xhG9jn2QyLvrNpJ3t68g294lLy/5X1mbJqhIZFF2jddx8PAGGOBS6y1b4a5DoWwIiIRTCGsiIhIUwphWxCpISxAIBji2UVbeOjDL/bNF3vh0Ex+NSWbzJS4MFcXmYKhICuLVjJ7+2xmb5vN5rLNTdYPTBvIpJ7OsMWDuwzGYzzhKVREGuiXvnbMGPMRsNFae2ML6w55OOIWjhGx1/Bj4dbnlzFjVSH/c86J/Ois8A0JHC4Ldizgxg9vJDMhk/e/9b7mfhdpn/Qf8ygxxjwNpFprL25h3RGFsMaYTsA9wLk4H74qAt4EfmOtLTuE4yiEFRGJYAphRUREmlJadJzxeT1cP7Yvs39+Btec1htj4L+rCjnzwTk89smX1AWC4S4x4ng9XoZ1G8bPhv+Mty9+m3cufofbR9zO8G7D8RgPG0o28ETuE1z97tWc9cpZ3L3gbmZvm01NoCbcpYuItEcemnauNrYQOKvZsnPc5dIGGkbHWLgp/EMCh8MpXU/BZ3wUVhVSUFkQ7nJERI4n3d3H7cAQ4HpgMvBUGGsSEREREREJK1+4C5Cjo1NCNL+/JIerRvXm7rfXsGRLCX98fz2vLt3O3d8YzMQTu4a7xIjVJ6UPfVL6MHXwVEprS5lbMJfZ22Yzv2A+e2r28NqG13htw2vEemMZ3X00k3pOYmKviXSJ6xLu0kVEjiljzP3Ae8BWnGGCrwYmAee5658BCqy1d7q7/AWYY4z5H2AG8G1gBPCDY1v58WtMP2cI4qVbSqgLBI/5vLDhFh8Vz5AuQ1hRtILPd35Oz6Se4S5JRKQ96WKMeQPnOl0A/I+19m0AY8xvgZuAHGvtXnfZDCAeOMtauxr4VqNjbTTG/Ap41hjjs9YGjuUTERERERERaQ/UCXucG9IjhVduGsOfrzyZrkkxbN5TxdR/LebmZ5dSUKpOzSOVGpvKN/p/g4cmPcS8b8/j8bMf59snfZvMhExqg7XM3jabuxfezRkvn8HVM67mHyv/wfri9WgYcBHpINKBZ4D1wMfASOA8a+2H7vreQGbDxtbaBThB7Q+AlThzyF7s3tiVNtC/ayJdEmOoC4RYua3Vo0MeV0ZmjARgya4lYa5ERKTduQt4GRgKvAs85w4zDPB7IB94EsAYcytwOs5c7qH9HC8FKD9QAGuMiTHGJDc8cD60JSIiIiIiclzQnLAtOF7nk6uo9fPwRxt4ekE+wZAlLsrLbWcN5Lvj+hLtUx7flqy1fFHyBZ9s+4Q52+awem/T/CAzIZOJPScyqdckRmaMJNobHaZKRY5LmoOmAzter+Ft6YfPL+O/qwr52TknclsHnBd24Y6F/ODDH5CRkMEH3/pA88KKtD/6T3mUtGJO2P+11v7G/ToBqATOt9bOdJf1A1YAfwNuA75nrX1+P+fqAiwFnrXW/uoANd2NE/42oTlhRUQik+aEFRERaUohbAuO9xu463aW89s317A4vxiAgemJ3HvxkH3zxEnb2129m7nbnWGLFxUuoi5Yt29dvC+e07ufzsReE5nQcwKdYjvt/0Ai0hr6pa8DO96v4W3h2UVb+PWbqxnTrzMv/GB0uMs55qr91Yx9cSyBUIB3L32XXkm9wl2SiDSl6/hR0ooQ9gpr7SuNlpUBP7LWPtNo2Q+AfwAvWWu/vZ/zJAMfAsXAN6y1/gPUFEPTueKTgO0KYUVEIpNCWBERkaY0J2wHlJWRzEs3jub1ZQXc924eG3ZX8u1/LuLSYT341QXZdE6MOfhB5JCkx6dz2YmXcdmJl1ETqOGzws+YvW02c7fPpaimiI+2fsRHWz/CYBjadSgTe05kYq+JDEwdqA4dERFpUw0fulq2tYRaf5DYqI43L2xOlxyW717Okp1LFMKKiHyleVhq+foURhOAINCnpblejTFJwEygArjkQAEsgLW2Dtj3CVX97iMiIiIiIscTjUHbQRlj+Nbwnnz8PxO55rTeGAOvLyvgrIfm8NLnWwmF1CF9tMT54pjUaxJ3n343H13+ES9OeZGbTr6J7E7ZWCwri1byyPJH+Nbb32Lya5P5/aLfM79gPvXB+nCXLiIix4H+XRMazQtbGu5ywmJEtxEAfL7z8zBXIiISOYwxVwKXApNw5nX/TbP1ycAHQD1OB2ztsa5RRERERESkPVEI28Glxkfz+0tyeP3m08nOTKa02s8dr+Xy7X8uYsOuinCXd9zzGA+Duwzm1lNu5eWLXubDyz7kN6N/w4SeE4jxxrCjagcvrn+Rmz66iXEvjuMnn/yENza8wZ6aPeEuXUREIpQxhtH9nKHvF20qDnM14TEyYyQAn+/6HE3NISIdTIox5pRmj4MOCWCM6Qn8HbjDWvspcAPwS2PMaHd9QwCbAHwXSDbGZLiPjjXkgoiIiIiIiEtzwrago84nFwiG+Pf8fB768Atq/EGivIabJw3glkn9O9xQhe1Bw7DFc7bPYc62ORTVFDVZP6TzECb0nMCEXhPI7pSNx+gzFSIujWPXgXXUa/iheu6zLfzqjdWM7teJF38wJtzlHHM1gRpOf+F0zQsr0j7pOn6UuHPCTm1h1VM4wekl1to3G21fCvwEmI4zx2sAON+6NxGMMY8AFwCnACOAT/Zz6r7W2vxW1pgMlGlOWBGRyKQ5YUVERJpSCNuCjn4Dd3tJNb99aw2z1u0GoF/XBO6/JIfT3Dnk5Niz1pJXnLcvkF2zd02T9V3iujiBbI8JjO4+moSohDBVKtIu6Je+DqyjX8Nba2NRJWf9aQ7RPg+r7jq3Q37Yaup7U1m2exm/O/13XDLwknCXIyJf0XW8A1MIKyIS2RTCioiINKUQtgW6geuEfu/m7uSut9ewp7IOgKtG9ebOC7JIjo0Kc3Wyp2YP87bPY+72uSzYsYDqQPW+dT6PjxHdRjihbM8JnJB8QhgrFQkL/dLXgR3wGr5+JhQsbbqsxzA46fxjVl97Ya1l1H0fU1RRx1WjetE1MeaQ9jfGcOHQTAZ2i9wb5I8uf5R/rvonF/W7iPvG3xfuckTkK7qOd2AKYUVEIptCWBERkaYUwrZAIexXyqr9PDAzjxcWbwOgW3IM935zCOcOzghzZdKgPljPkl1LmLd9HnO2z2FbxbYm609IPoHxPcYzvud4RnQbQbQ3OkyVihwz+qWvAzvgNXzG7fD5E8128MDP8iCp413Xfvzict5aseOw9z+lVypv3jq2DSs6thYVLuL7H3yfzIRMPrjsg3CXIyJf0XW8A1MIKyIS2RTCioiINKUQtgUKYb9u0aa93Pl6Lpv3VAEwJSeTu78xmK5Jh9Y5I0eXtZb88nzmbp/LvO3zWLprKQEb2Lc+zhfHaZmnOaFsj/FkJmaGsVqRo0a/9HVgB7yGr3kDtiz46uu1b0PlTrjyWci+6JjW2R5sL6nmmYVbqPMHD2m/+qDlhcVb8XkMq+85L2KHMq7yV3H6C6cTsiFmXT6LrvFdw12SiDh0He/AFMKKiEQ2hbAiIiJNKYRtgULYltX6gzw6awOPz9lEMGRJjY/itxcO4pJTe2CM3mO1R5X1lSwqXMS8gnnM2z6PopqiJusHpA5gfI/xjOsxjlPTTyXKq6Gm5bigH0gd2CFdw9++DZZNh7E/gXPuORblHRestZx238fsrqjj1ZvGMKJPp3CXdNi+9fa3+KLkCx4+42HO6n1WuMsREYeu4x2YQlgRkcimEFZERKQpX7gLkMgRG+Xl5+dlcf6QTH7x6irWFpbzs5dX8s7KHdx3aQ6ZKXHhLlGaSYxO5OwTzubsE87GWsu64nV8WvAp8wrmsbJoJV+WfsmXpV/y7zX/JiEqgdMyTmNcz3GM7zGejISONzSniHQwPUc4Iez2JeGuJKIYYzilVyofrN3F8q2lER3C5nTJ4YuSL1hVtEohrIiIiIiIiIiItCmFsHLIhvRI4a0fjuWfczfxl4828Mn6Is59aC6/vjCbK0b0UldsO2WMIbtzNtmds/n+0O9TVlfGwh0LmVcwj08LPqW4tphZ22Yxa9sswOmSHddjHGN7jGVY+jDNJSsix5+eI50/dyyDYAC8elvUWqf2TnNC2G0l4S7liJzc9WRe2/AauXtyw12KiIiIiIiIiIgcZ3S3UQ5LlNfDrWcM4LzB3bj9lVWs2FbKHa/l8t9VhTzwraH0SFVXbHuXEpPC5L6Tmdx3MiEbIm9v3r5ANndP7r4u2afXPE2cL45RGaMY22MsY7uPpXdy73CXLyJy5LqcBDHJUFcORXmQkRPuiiLGKb1SAVixtTSsdRypnC7O93z1ntUEQgF8Hr01FhERERERERGRtqE7TXJEBqQn8drNp/PUp5v40wdfMG/DHs7781x+PSWbK0eqKzZSeIyHwV0GM7jLYG46+SZKa0tZVLiITws+Zf6O+eyp2cOc7XOYs30OAL2SenF699MZ230sozJHkRCVEOZnICJyGDwe6H4qbJ4D2z9XCHsIhvZMwWNgR1ktu8pr6ZYcG+6SDku/1H4kRiVS6a9kY+lGTup0UrhLEhERERERERGR44Qn3AVI5PN6DD+Y0J/3fjye4SekUVkXYNrruUz99+cUltWEuzw5DKmxqUzuO5n/Hfe/zLp8Fq9c9Ao/GfYTRmaMxGd8bKvYxkvrX+K2T25j3IvjuGHmDTyZ+yRr9q4hZEPhLl9EpPUahiTevjS8dUSYhBgfJ2UkA7A8grthPcbDkC5DAFi1Z1WYqxERERERERERkeOJQlhpM/26JvLyjWP41QXZRPs8zP2iiHP/PJdXl27HWhvu8uQwGWPI6pTFd3O+y7/O+xefXvUpj5zxCFeedCW9knoRCAVYsmsJf1n2F779328z6aVJ/GLOL3hjwxvsrNoZ7vJFRA5sXwj7eXjriEANQxJH+rywDUMSrypSCCsiIiIiIiIi0lb6TJsRFe4awk3DEUub8noM35/QjzOy0rn9lZWs2FbK7a+s5P01O7nvkhy6JsWEu0Q5QglRCZzR+wzO6H0GAFvLt7JgxwIW7FjA4p2LKakr4b3893gv/z0A+qX0Y0z3MYzJHMPIjJHER8WHs3wRkaZ6jnD+3LMeakohLjWc1USUU3un8sLirRE/L+zJXU8GILcoN8yViIiIiIiIiIgcH/pMm/Fv4Po+02Y8nf/AlBvCXU+4GHUofp0xJhkoKysrIzk5OdzlRKxAMMQ/5m7i4Y++wB+0pMVH8ftLcrggJzPcpclR4g/5WVW0ioU7FrJwx0JW713dZHhin8fH0C5DGdN9DKMzRzOkyxB8Hn0WRNqcJqPuwA7rGv6Xk6EkH657A/qfeTTLO65s2FXBOX+eS1yUl9y7z8XnjcwBVopri5n40kQA5l81n+RovfcTCTNdxzuwhuv4iuXLSEpKCnc5IiJyiPr1H6DruIiI0GfajKhQbWV9qL4aX3I6QHT+A1P84a4rHNr93TJjTA9jzLPGmL3GmBpjTK4xZsRB9rnGGLPSGFNtjCk0xvzLGNO5TQpSaN1qPq+HW88YwNs/HEd2ZjIl1X5ueW4ZP31pBWU1HfL/23EvyhPF8G7D+eGpP+S5Kc8x98q5/Gnin7jsxMvokdiDQCjAst3LeGzFY1z33nWMf3E8P5r1I57Le46NpRs1bLWIhMe+IYmXhLeOCNO/ayJJMT5q/EHW76oIdzmHrVNsJ3om9gRg9Z7VYa5GRERERERERCSyFb31hwR/yY7/ugHs0x01gIV2PhyxMSYNmA98ApwPFAEDgf1OPmaMGQs8A/wUeAfoATwOPAFcesRFvXs7FG+GwZdA1hSI73TEhzzeZWcm89atY3nk4w38bfaXvLG8gEWb9vLHy05m3MAu4S5PjqKUmBTO7XMu5/Y5F4Bt5dtYWLiQRYWL+KzwM8rry5m9bTazt80GoGtcV0ZljuK0jNMYnTmazER1TYvIMdBjBOS+ohD2EHk8hlN6pzJvwx5WbCtlcPeUcJd02IZ2Hcr2yu3kFuVyevfTw12OiIiIiIiIiEhESjr5vDhCwZMLp//0oj7TZkR15AAW2nkIC9wBbLPWNh4vevNB9hkD5FtrH2nY3hjzD/dYRyYUhDVvQPVe2Pgx/Pcn0O+MrwJZzSO3X9E+D7efdxJnZKXzPy+vIH9vNdc+9RnXn96HaednERvlDXeJcgz0Su5Fr+ReXHHSFQRDQdYVr2NR4SIWFS5i+e7lFNUUMWPTDGZsmgFA76Te+0LZkRkj6RzXNg3tIiJN7OuE/dwZ8cJoBK3WOqWXE8Iu31rKNaedEO5yDtvQrkN5d/O7rNqzKtyliIiIiIiIiIhEpISscR6sHVu1fv5HAB09gIX2H8J+A3jfGPMKMBEoAP5mrX3iAPssBO4zxlwAvAekA5cB7+5vB2NMDBDTaFHLk894vPD/PoC1b8CaN2HXavjyQ+fx32gYcDYMvhROOh9iEg/leXYYw09I490fj+f3M/J47rOtPL0gn/lf7uHPV57CkB6R20Ejh87r8TK4y2AGdxnMd3O+S12wjpW7Vzpdsjs/Y82eNWyt2MrWiq28+sWrAAxIHcDIjJGclnEaIzJGkBKjfzMi0gYycsAbAzXFULIZOvULd0UR49TeqQCs2FYa1jqO1NAuQwHILcrFWotREC8iIiIiIiIickhsKHSG8XhmhbuO9sS05zkYjTG17l8fAl4BRgJ/AW6y1k4/wH6XA/8CYnGC5neAb1lrW0zdjTF3A3c1X15WVkZycvL+C9yzwemMXf06FOV9tdwX5wSxOZc5wawvZv/H6MA+Wb+bX7y6iqKKOqK8hp+ecyI3TuiP16MbnwKV9ZUs271s39DFX5R80WS9wXBi2omMzBjJqIxRDOs2TKGsNNAPkQ7MGJMMlB30Gt7ck2c7nbDffMz5QJW0yt6qOsb+4RMA5v58EkmxUWGu6PDUB+s5461z8If8vHDOs/RI6B7ukhzeaOdDgK0Q4409rPA42ufRey9pb/QPsgNruI6vWL6MpKSWPxstIiLtV7/+A3QdFxHpoBJOGjsRYz6rWvdp7cG37jjaewhbDyyx1p7eaNkjwEhr7Zj97DMI+Aj4M/A+kAn8EfjcWvvd/ezTUifs9kO6gbtrLax5HVa/BsWbvloemwKDvgk5V8AJY8Hjad3xOojiqnrufH0V76/ZBcCovp3485Wn0CM1LsyVSXtTUlvCkl1LWFy4mMU7F7OpbFOT9QbDSZ1OYkS3EYzIGMHw9OGkxqaGp1gJN/3S14Eddgg7805Y9LejVpe0f9dkdmNVbOR+cC5QkU3N9qmHvF+XxGjevW086cmxR6EqkcOi63gHphBWRCSyKYQVEemY4k86/TTj8a2vyptbGu5a2pv2HsJuAT601n6v0bKbgV9ba3vsZ5//ALHW2ssbLRsHzAO6W2sLW3Hew7uBC85ccjuWO2Hs6tegotHpknvAkG/B0CshY8ihHfc4Zq3llaXbueftNVTVB0mK9fG/Fw/hm6e0+C0WAWBPzR6W7FzC4p2LWbJrCZvLvj5d9MC0gU4o220Ew7oNo0tclzBUKmGgX/o6sMO+hm/9DJ75JgRqjlpt0r69mJTI/Z3TCEXwUMQV6++G0KGHqQ9dcTKXDuvZ9gWJHJ7I/U8oR0whrIhIZFMIKyLS8SRkTzgFY3ZWrZ2zM9y1tEftPYR9HuhlrR3faNmfgdMad8c22+c1IGCtvbLRsjHAAqCHtXZHK857+CFsY6EgbJkPua/A2regtuyrdd2GwNArnA7Z5MzDP8dxZMveKn7y0gqWby0F4JJTe/C7bw6O2KEN5dhqTSjbJ7kPw7sN3/fonthOhpuUtqZf+jqwI7qGB+ohWH9U6jre1QaCBEPt9z1la9UG6wjZYLjLACD6o98QtfY16kffhn/s7Qfc9tvvXsruml08fuZTnJI+rNXnuO/dPJ77bCvfH9+XX00ZdKQli7QVXcc7MIWwIiKRTSGsiEjHkjBoYjbW1lTlzc0Pdy3tlS/cBRzEn4EFxphfAi8Do4AfuA8AjDH344Sr33EXvQM84XbMNgxH/DCwuDUBbJvyeKHvBOdxwYPwxfuQ+7Lz567V8OFq+Ohu6DcJTr4KsqZAdMIxLbE9OaFzAq/cOIZHZ33Jo7M28MbyApZsKebhK09l+Alp4S5P2rkucV2Y3Hcyk/tOBpxQdumupSzZuYRlu5fxRckX5Jfnk1+ez2sbXgMgMyGTYd2GMSzdefRL7YfHaMhwkbZijLkTuBTIAmpwPhB1h7V2/QH2uR74d7PFddbaoz9Wqi/aecghi+BRfJtoV+/Ceo2ANa8SX7wBEg/8gYKsziexe/sutlR+ydheo1p9ipwezlzqeYUVR1SqiIiIiIiIiHQsCVnj+gC2at2n+WEupV1r1yGstfZzY8wlwP3Ab4HNwE+stc812iwT6N1on6eNMUn8f/buOzqu6trj+HePmq3m3nDBXe7dFHfjAIY8wHQCAUyHUAKBhEBCSUKAkITeO4QSIIDp1RR3sHHvuOLeZMuWZKvu98cdGWHkLumq/D5rnSXNuW3Pe8EzuvvufeAq4N/AFuAL4MbyirtEsQnQ6cRgZKfD3FEw83X4YSIs/iIY8cnQaQT0+BW06Fct14+NjYlw3dHtGdiuPr/973RWpG/njCcmcs1R7bjqqLbERPRAneyb+jXrc2zLYzm25bEAZORkMHXdVKaun8rUdVOZu2kua7LW8MGSD/hgyQcApMan0rNhz52jc/3OJMRUkcyCSDgGA48Akwm+c9wJfGpmndw9aw/HbQXSir2u/CWWIvurUXTpirWz97prWp00xqwcw8LNC/frEh2bBMndeWu24u5YJW7FLCIiIiIiIiLlI6nzkEOAWlnzx80IO5aKrkK3Iw5LqbUj3hfpS4Jk7IxXYfOyH+drHwo9zg5G7Ra7Pbwq27ojj1tGzead6UEB82Gt6nL/mT04pHbNkCOTqiA7L5uZG2cGidl1U5m5cSbbd1kLMi4SR6d6nejVsBc9Gvage4Pu1KtZL6SIZT8oi1BBmVkDYD0w2N3H7GafkcD97l77AK9Rfp/hImVp+2b4R8vg9xuXQ83au93142Uf8/uvf0+Xel149f9e3edL7MgroNOtH1Po8O3Nw2iYWvYF5yL7QJ/j1ZjaEYuIVG5qRywiUvUldx5a1wvy22bNH/tt2LFUBkrCliCUG7juQVXs9FdgzijILWoLZ0E7456/ho4nQFz1S0C+PW0lf357Nlm5BdSqGcc/Tu3G8C6Nww5Lqpi8wjzmb5rP1PVTmb5+OlPXTyV9R/rP9muR0mJnQrZHwx60qdWGmEhMCBHLHuiPvgrKzNoC3wNd3b3E8r5oEvZpYBUQAaYCN7v7nN3snwAUL1lPAVYqCStVwr2dYetKGPkhtOy/292WZizlxFEnUiOmBpPOnrRfn0vD/v0Vizdk8fwFfRmS1rA0ohY5WPocr8aUhBURqdyUhBURqdpSuh9bs3BHZt+sBeNLLK6Qn1MStgShV9HkZsO892D6S7C02P+Wa9SCrqcHCdkmPaAatYxbtjGL3/53GjNWZgBwzuEtuOX/OlEjTskvKRvuzoptK5i2fhrT1k9j+vrpLM5Y/LP9kuKS6Fq/K90bdKd7g+50a9CNWgm1QohYiqk+/zhWImYWAd4Farv7gD3sdyTQDpgJ1AJuAAYBnd19ZQn73w7ctuu8krBSJbxyJiz8GI67Bw6/bLe7FRQWcMQrR7CjYAfvjHiH1rVa7/MlrnplKu/PXMONwztwxZA2pRG1yMHS53g1piSsiEjlpiSsiEjVldRhQMQLC4dlL5zwWdixVCYVek3Yais+EbqfGYzNy4NWxdNehowfYPLTwWjcFXqdHyRl99CerqpoWT+JNy7vx72fLeTxrxfz8jc/MGXZZh46uyftG+mPcyl9ZkaL1Ba0SG3BSW1PAoJ1ZWdumMm09dOYuWEmMzfOJCsvi0lrJjFpzaSdx7ZMbUm3Bt3o3qA7Xet3pV2ddsRG9M+tVHuPAF2A3SZgAdx9IjCx6LWZTQDmAZcBt5RwyF3AvcVepwA/S9aKVEqNuwZJ2LWz9rhbTCSG9nXaM3PjTBamL9yvJGzHJqm8P3MN89ZsPdhoRURERERERKQKMjOr2e6IYRaJGR12LJWNsgIVXZ1DYcgfYdAfYOnXMO2loEp27Sz48Ab49M/QaQT0Ph9aHFmlq2PjYyP88bgO9G9bj+tem8GCdds48eFx3HZCZ87q2xyrwu9dKoZaCbUY2GwgA5sNBCC/MJ9FWxYxY/0MZm6cyYwNM1i+dTnLti5j2dZlvLv4XQBqxNSgU71OdK3fla4NutKtfjcaJzXW/2alQjKzJUBfd9+0y3xtYKq773t258djHwb+DxhUUjXrnrh7nplNA9ruZnsOkFPsWvsbnkjF1ahL8HNdid27f6J93SAJu2DzAoa3Gr7Pl+jUJKgYVxJWREREREREREpSs/2RgyPxNcdkzhpdGHYslY2SsJVFJAJthgYjOx1mvg5TX4D1c2Hmf4NRv31QHdvjbEisG3bEZWZguwZ89NuBXP/GDMYs3MBNb81i3KKN3HVKV1JrxIUdnlQjsZFYOtTtQIe6HTiTMwHYvGMzszbOCiplN8xk9sbZbMvbxtT1U5m6furOY+vVqEfX+l3pXL9z8LNeZ2rXqB3SOxH5iZZASb3eE4Cm+3MiCzKiDwEnA0Pcfen+BmNmMUBX4MP9PVak0mvcNfi5fh4U5EPM7r+6p9VJA2BB+oL9ukTHaBJ2ycYsduQVaKkHEREREREREdkpqcOAIyJxNaZlzhqds/e9ZVdaE7YEoa8Ju6/cYdV38N3zMPstyMsK5mMSoNNJ0OeCKl0dW1joPDV2Cf/8ZAH5hU6Luok8fHZPujWrHXZoIjsVeiHLti5j1oYgMTtr4yy+3/w9+Z7/s32bJTejS/0udKnfhU71OtGpXieS4pJCiLrSq5r/6JUxMzsx+uso4Hwgo9jmGGAYcLS7p+3HOR8FzgZOAopnhjLcfXt0nxeBVe5+U/T1rcAkYBFQG/g9MALo7e5z9+GaleMzXGRfFBbCXc2C73hXfgsNdv+f37T10zjvo/NomNiQ0afve3cgd6fX3z5jc3Ye7101gK7NtK65hE6f49WY1oQVEanctCasiEjVkpTWv5fFxq/KnPPlurBjqayUhC1BpbyBu2MrzP4fTHkO1s78cb5BB+hzIXQ/C2pUzZtq037YzNWvTmPl5u3ExRg3Du/ARQNaqSWlVFg78ncwP30+szbOYvbG2czZNIflW5f/bD/DaFmrJZ3rdaZzvc50qteJDnU7kBiXGELUlYr+4z8AZlbUTsT5+f8N84BlwPXu/v5+nHN3XzIucPfno/t8BSxz95HR1/cBpwCNgc3Ad8Cf3X3aPl6z8n2Gi+zJ07+AlZPh1Geg62m73S0rL4sjXjkCgLFnjt2v7gpnPzWJCYs3cc+p3Tijb/ODjVjkYOlzvBpTElZEpHJTElZEpOpI6jiwE1h21rwxy8KOpTJTErYElf4G7qqpMOVZmP0m5GUHc3GJwY27vhdDk+7hxlcGMrbn8cc3Z/LR7LUA/KJjQ/51endqJ8aHHJnIvsnIyWDOpjnM3TSXORvnMHvTbNZmrf3ZfobRqlYrOtbrSKe6nehYryMd6nYgJV43qYrRH30HwcyWEqwJuzHsWA5Epf8MF9nVe9fCd8/BgOvgF7fvcdfj3jyOlZkrefqYpzm8yeH7fIm/vT+XZ8YtZWS/ltx+YueDi1fk4OlzvBpTElZEpHJTElZEpGpI6jSkFV4YnzVvzP6teSQ/ozVhq6KmvYJx7N9hxmsw5RnYMB+mvhiMZn2h7yVBy+K4GmFHWypq1Yzj0XN68dKk5fzt/Xl8Pm89xz8wlofO7kXvQ+uEHZ7IXtVKqEW/Q/rR75B+O+c2bt/I3E1zdyZm56bPZX32epZkLGFJxhI+WPLBzn2bpzSnY92OO5OyHep2oH7N+mG8Fank3L1V2DGISDGNuwQ/187e665pddNYmbmSBekL9isJW7Qu7Lw1Ww8oRBERERERERGpGpI6DmoKnpI1b+zMve8te6MkbFVWoxYcfikcdgn8MBEmPw1z3w1a2q2cDJ/cBL3OC9oV124RdrQHzcw498iW9GxRh6temcqyTdmc8cRE/nBsGpcMbE0koofxpHKpX7M+g5oNYlCzQTvnihKz8zbNC36mz2NN1hpWbFvBim0r+HT5pzv3bVCzwc6EbFrdNDrU7UDzlOZELBLG25FKxMyGEawB2xD4yf9g3P3CUIISqa4adQ1+rtuHJGydNEb/MJoFm/fvQdWOTYJqs3lrtuLuWtJBREREREREpBpK7jy0LoUFTbMWjP827FiqCiVhqwMzOLRfMLatg2kvwpTnYetKGHcfjH8A0o4PkrWtBgf7V2JdmtbivasHcPPbs3lvxmru+mg+3y5N599nqD2xVH4lJWY379jM/PT5zEufx7xN85ifPp/lW5ezYfsGNqzawNhVY3fuWzO2Ju3rtCetThppddNoX6c97eu0r7TrzG7YsIFbb72VDz74gHXr1lGnTh3WrVv3CfBXdx9fmtcys7OAV4F33H1EaZ67IjGz24BbgSnAGoI1YkUkLI06BT+3rYGsTZBUb7e7tq/bHoCFmxfu1yXaNkwmNmJs3ZHP6owdNK1d84DDFREREREREZHKJ7nz0ETPz+2StWD8mLBjqUq0JmwJqsV6cgX5sPBj+PZJWPr1j/MNOgTJ2G5nQUJyePGVAnfnlW9/4C/vzSU3v5CmtWvy0Nk96dVC7Yml6svOy2bh5oXMS5/HgvQFLEhfwPdbvienIKfE/ZunNCetThrt6rTbmZhtltKswlfNDho0iNzcXO666y5at27NunXrOPzww28G5rj7u6V1HTNrCYwDlgDpVTwJuwb4g7v/J+xYDkS1+AyX6ueBHrB5KZz3DrQestvdVm5byXFvHUdsJJZvz/6WuJi4fb7E8PvHMH/tNp46rw9Hd2p08DGLHLjK/USoHBStCSsiUrlpTVgRkcqpRouuMZEaKUdlL5zwWdixVDVKwpag2t3AXT8fJj8FM/4LuZnBXEIt6HVukJCt0zLU8A7W7FUZXPnKVJZvyiY2Ytx0fEcu7N9Srfak2skvzGf51uUsSF/A/M3zWZi+kIWbF7Jh+4YS968ZW5M2tdrQrk67naNt7bbUq1GvQvz3s2XLFurUqcNXX33F4MGDi2/aGZyZtQAeImitWwh8DFzt7uvMrAMwFbjY3V+J7n8G8ALQ293nRudigDHAs8BAoHYVT8JuAg5z98Vhx3Igqt1nuFQPr/0a5r0Hx/wd+l21293cnX6v9iMzL5P/nfA/0uqm7fMlrnttOm9PW8Xvjm7PNcPalUbUIgcq/C8ZEholYUVEKjclYUVEKh8zs5rtjvhFfMNWn28Z94oShqVMSdgSVNsbuDsyYPorQXVs+pLopAWtio+4HFoOrLStirftyOPGN2fy4ay1ABzXpTH/OK0bqTX2vUJEpKpK35HOws0Lg2rZzd/z/ZbvWbxl8W6rZusk1KFtnba0rf3jaFO7DbUSapVr3Pn5+dSpU4eLL76Yu+++m4SEhKJNBmBmEeA7IBO4lqAF/yNAprsPie7zG+BOoBtBknYmcLu7P7jzZGZ/Abq5+8lm9jxVPwn7D4L/G/0t7FgORLX9DJeq7at/wFd3BuvDthq0x13PTx/P1Lx0hsQ3onlsUjCZ1AAaBK2K4yJxnNruVFqktvjJcU+OWcydH86nXcNkBrVvsN8hdmicwul9mu/3cSIlqJx/cEipUBJWRKRyUxJWRKTySUrrP9jia0zKnDW65JvBclCUhC1Btb+BW1gIiz6Hbx6HxaN/nG/UFY64ArqcCnE1wovvALk7L0xYxt8/nEdegXNovUQePacXnQ8p38SRSGVQUFjAD9t+2JmU/X7z9yzasogftv6A72aJ0AY1G9CmdpsfR602ZZ6cffPNN7nkkkvYvn07vXr1YvDgwdx1113d3X2mmR0NfAS0cvcVAGbWCZhDUOk5OTr3PpAK5AIFwHCPfjia2QDgv0APd99YTZKwDwDnESSkZwJ5xbe7++/CiGtfVfvPcKmavv8MXj5tn3a9p25t/lNrz//bP/rQo7l3yL0/mZu0ZBNnPTnpgEME+Pr3Qzi0XtJBnUMEJWGrNSVhRUQqNyVhRUQql6S0/kdaXMLczNlfZIQdS1WlJGwJdAO3mA0LgmTsjP9CXnYwl9QA+l4MfS6C5P2vlAjbtB82c9Ur01i1ZTvxsRH+dlJnzuzbYu8Higjb87ezJGMJizYvYvGWxXy/JUjOrs1au9tj6tWoR5vabWhdqzWta7cOftZqTf2a9fe7rXFBYQHfrP2GdVnrSKubRqd6ndixYwdjx45l0qRJfPTRR0ycOLEAuJggsXqdu7cqfg4z2wz81t1fjL5uCCwkqITt7O5rovMpBEnI37j7R9G556n6Sdgv97DZ3f2ocgvmAOgzXKqkwkKY/DRsW73XXTcV7OCNrKXs8IJgYt0cyMuCLqeyLr4G7y95n5apLXnv5Pd+cpy78/I3P7Bqy/b9Du/tqatYu3WH1pOV0qKbt9WYkrAiIpWbkrAiIpVHUlr/XhYbvypzzpfrwo6lKlMStgS6gVuC7HSY+mLQqnjrqmAuJgG6nQFHXgkNO4Yb337akp3L716fwRfz1wNweu9m/PWkLtSMjwk5MpHKKTM3kyUZS1i8ZTGLtixiccZilmxZwpqsNbs9JiUuhVa1W9EqtRWtarWida3WtKzVkmYpzYiL/LxV+JItS7hi9BWszvwxCXFY48O4f+j9pMT/eJPOzJ4Bjgb+zb4lYY8GPiRIwh7p7lOj8z2AaQTVsUUi0Z+FQFplXTe1KtNnuMguXv0VLPgQjv8Xazr9H8e8eQyxFsvkX08mNhJbKpe4+tVpvDdjNTcf34FLB7UplXNKtaabt9WYkrAiIpWbkrAiIpVDUocBnTHbljVv7A9hx1LVlc6dF6n6EuvCgGuDhOvcd2DSo7DqO5j2n2C0GQb9robWQyrFurG1E+N5+rw+PPb1Yv796QLe+G4ls1Zl8Nive9Oqvtroieyv5PhkujXoRrcG3X4yn5WXxdKMpTsTtEu2LGFJxhJWZq5kW942Zm6YycwNM39yTKzF0iylGS1rtaRVrSBJ2zylOX8c+0c2bt/4k32nrJvC3yb+jXsG31N8ei4wApgHNDez5ru0I64d3Qczqws8D/wdaAK8bGa93H07MB/oustbvQNIAX4LrDig/2OJiJSnetGk6KbFNEpqREJMAjkFOazOXP2zdWEPVNF3p6Ubs0rlfCIiIiIiIiJSNpI6DGgDlqsEbPlQElb2T0wcdD0tWBd2xbcw8WGY/36wduzi0dCoCxx5VbA9Nj7saPcoEjGuHNqWni1qc82r05i/dhsnPjSOf53RnWM7Nw47PJEqISkuiS71u9ClfpefzOcU5LB863KWZizdmaRdlrGMZVuXsT1/O8u2Br9/teKrPZ6/0Av5eNnHnJRyEofUOYQpU6YA/AF4B/gcmEWQWL2W4DPvUeBrd58SPcXjBMnUO4AEgsrXfwFXuvsOYHbx65nZFgB3/8l8VRJtR7zbNhkVvR2xiOyiXtvg56bviViEFqkt+H7z9yzbuqzUkrCto0nYxRuUhBURERERERGpqJI6DmqKWVLWvLEz9763lAYlYeXAmEGLw4ORvhQmPQbTXoJ1s2HU5TD6r3DE5dB7JNSoFXa0e9SvTX0+uGYgV70ylcnLNnPZf77jssGt+f0xacTGRPZ+AhHZbwkxCbSv0572ddr/ZL7QC1mfvX5ncnbZ1mUsy1jG3E1zycgteX14xxlxzgiyf8imefPmAE8Bd7q7m9lJwEPAGIIWwh8DVwOY2XnA8UBPd88H8s3s18A4M3u/aB3Yamj6Lq/jgB5AF+CF8g5GRA5SvXbBz02LAGiZ2jJIwmYsY1CzQaVyidYNVAkrIiIiIiIiUpEldRxYD6dp1vyx34YdS3WiJKwcvLqt4Ph7YOhNMOVZ+OYJ2LYaPrsVvv4n9BkJh18BtZqGHeluNUqtwSuXHME9H8/nqbFLeeLrJUz/YQsPn92LBikJYYcnUm1ELELjpMY0TmrMkYccuXN++vrpnPvRuSUekxCTwPKZy0mMSyyauqXoF3f/ATippOOia8K+uMvct8Buy/jdfeS+vZPKy92vK2nezG4Hkss3GhE5aEWVsFt+gPwcWqa2BGD51uWldomidsQbtuWwbUceKTV+vq63iIiIiIiIiIQjKa1/knthp+yFE8eGHUt1ozI/KT0168DA6+HaWXDSI9CgA+RugwkPwQPd4e0rYP28sKPcrbiYCH/6ZScePacXyQmxfLM0nV8+OJYpy9LDDk2k2uveoDvdG3QnxmJ+Mm8YZ3c8u3gCVsrOS8CFYQchIvspuSEkpIIXQvpSWtZqCcCyrctK7RIpNeJ2PrSmalgRERERERGRiiOlx/BY4EglYMOhJKyUvtgE6PlruGIinP0GHDoACvNgxivw6BHwypmwfGLYUe7W8V2b8M5V/WnXMJn123I468lJPDtuKe67XSJRRMqYmfHwUQ8zqNkgDAMgPhLP+Z3P55qe14QcXbVxJLAj7CBEZD+ZQb02we+bvufQ1EOB0k3Cwo/VsEu0LqyIiIiIiIhIhWBmVpC9dWhcg0NHhx1LdaV2xFJ2IhFof0wwVk6B8Q/AvPdg4cfBaH44DLgO2h0b7FuBtGmQzKgr+/PHt2bx3ozV/PX9uUxfsYW7T+1KYrz+sxEJQ+0atXnwqAfZuH0jG7I30DylOcnx6o5b2szsrV2ngCZAH+Bv5R+RiBy0em1h9TTYtIiWrYN1YNdnryc7L7vUOgm0aZDEt0vTWaJKWBEREREREZEKIbF9v6GYfbVl3CuqMAtJxcp8SdXVrA+c+R+4agr0Oh9i4mHFN/DqWfB4f5jxGhTkhR3lTyQlxPLgWT247YROxEaMd2es5uRHJqjNnkjI6tesT8d6HZWALTsZu4x04CvgeHf/S4hxiciBqtcu+LlpEbUSalEnoQ5QNuvCLtmQWWrnFBEREREREZEDk5TWvz8wJWv+uIqVeKlmlISV8lW/LZz4YLBubP/fQnwKrJ8Lb18KD/aCb5+CvO1hR7mTmXFB/1a8eukRNEhJYMG6bZz40Dg+nbM27NBERMqEu1+wy7jI3f/o7p+GHZuIHKCidsQbFwHsbElcuknY4MEYPawmIiIiIiIiEq6kDgP6YJHvsxaM3xp2LNWdkrASjpTGcPRf4brZMOxWSGoAGT/AhzfA/V1h7L2wo+L8+9C3ZV0+uHoAfVvWYVtOPpf+5zv+9ckCCgpVxS8iVZOZ9TazX0dHz7DjEZGDUK9t8HNTkIRtWaslAEu3Li21S7RuEFTCLt2Yhbu+H4mIiIiIiIiEIanDwK6Yrc+aP3Z92LGIkrAStpq1YeD1QWXs8f+CWi0gawOM/gvc1wW+uAOyNoUdJQANU2vwyiVHcEH/lgA8/OUiLnh+MpuzcsMNTESkFJlZQzP7ApgMPBgd35nZaDNrEG50InJAipKw2Rth++YyqYRtXieRmIiRnVvAuq05pXZeEREREREREdk3SZ0Gt8ULc7Lmjf0h7FgkoCSsVAxxNeGwS+CaqTDicaifBjkZMOafcH8X+Phm2Lom7CiJi4lw2wmdeeCsHtSIizBm4QZOeHgcs1dlhB2aiEhpeQhIATq7e113rwt0AVIJErIiUtkkJENKk+D3TYtpldoKgOUZpZeEjY+N0KJuIgBLNmpdWJHSYmaxZnaemTUKOxYREREREam4kjoNbk5Bfs2sBeMXhh2L/EhJWKlYYuKgx6/gN5PgjBehcTfIy4ZJj8AD3eD938Hm0rtheKBO6tGUt3/TnxZ1E1m5eTunPjaBt6etDDssEZHSMBz4jbvPK5pw97nAlcBxoUUlIgenqBp24/c7K2GXbV1Wqq2DW9UPWhIv2aB1YUVKi7vnA48DNcKORUREREREKqbkLsPqU1jQKGvB+FlhxyI/pSSsVEyRCHQ6CS4bA+e8Cc2PgIJcmPIMPNQLRv0GNi0ONcSOTVJ576oBDE1rQE5+Ide9NoPb351DXkFhqHGJiBykCJBXwnwe+t4gUnkVWxe2RWoLDCMzL5NNO0pv2YeiJOzSjUrCipSyb4EeYQchIiIiIiIVT3LXXyQX5m3vkDV/3JSwY5Gf081UqdjMoN0v4MKPYeQH0HoIFObD9Jfh4T7w5sWwfn5o4dVKjOOZ8/tyzVHBjc3nJyzjnKe+YcM2rYUmIpXWF8ADZnZI0YSZNQXuA0aHFpWIHJxiSdj4mHgOSQ7+E1+WsazULtG6QVElrNoRi5SyR4F7zewqMzvSzLoVH2EHJyIiIiIi4UjpMTy2MCfriOwFE8aFHYuUTElYqRzMoOUAOO8duHg0tB8OXgiz3oBHj4DXz4O1s0MJLRIxfndMGk+e25vkhFi+XZbOCQ+NY9oPm0OJR0TkIF1FsP7rMjNbbGaLgaXRuatDjUxEDlz9dsHPTYsAaJnaEoDlW0tvmQdVwoqUmf8CrQjWZh8PTAemFfspIiIiIiLVjJlZQfbWofENW6loogKr8ElYM2tqZi+Z2SYz225ms8ysz16OSTCzv5vZcjPLMbNlZnZhecUsZaxZHzj7taBVcccTAIe578Dj/eG/58CaGaGEdUznxrxzVX/aNEhi7dYdnPnEJF6b/EMosYiIHCh3XwH0An4J3B8dx7t7L3fX4tcildXOStjFUFhIy1otgdJNwrZpkAzAis3byc3X8gwipahVCaN1sZ8iIiIiIlLNJLbvN9Qika+2jHvFw45Fdq9CJ2HNrA7Bk755wHFAJ+B6YG8lhq8Dw4CLgDTgV8CCsotUQtGkO5z5ElwxATqfAhjMfx+eGASvnAWry/+h8DYNkhl1ZX+O7dyI3IJCbnxzFje/PUs3IkWkUvHAZ+7+UHR8fiDnMbObzGyymW0zs/VmNsrM0vbhuNPNbL6Z7Yg+fHX8gVxfRIqp3QIisZC/Hbau4tDUQwFYunVpqV2iYUoCifExFBQ6P6Rnl9p5Rao7d1++pxF2fCIiIiIiUr4S0/oNICZ2ctb8cXlhxyJ7FlvaJzSzZ/dlP3ffl8rUG4EV7n5Bsbk93ikys+HAYKC1u6dHp5ftS0xSSTXqDKc/B0P+CGP+CbPfhIUfBaPdsTDkRmjau9zCSakRx2Pn9Oaxrxfzr08X8Mo3P7Bg7TYeO6cXDVNrlFscIiIHwsweBBa5+4O7zF8FtHX3a/fjdIOBR4DJBN857gQ+NbNO7l5iv1Iz6we8CtwEvA+cDYwys17uHk7feZGqICYO6rSCTd/DpkVl0o7YzGhVP4k5q7eydGMWbRsml9q5Rao7M2sDXAt0jE7NBR5w98WhBSUiIiIiIuUuqePA3hYTvyBr7lfbwo5F9q4sKmFHAkOB2kCdPYx9cSIwxczeiFbQTDOzS/blGOAPZrbKzBaa2b/MrObuDoi2L04tGkDKPsYnFUmDNDj1abjyW+h2JlgEvv8EnjoKXj4DVn1XbqFEIsaVQ9vy7Pl9SakRy3fLN3PCw+OYqnViRaTiO5WgC8WuJgCn7c+J3H24uz/v7nPcfQbBd4QWwJ6ejPkt8LG7/9Pd57n7LcBUgrVqReRg7GxJ/GMSdsW2FeQX5pfaJVpHWxIv2ZBZaucUqe7M7FiCpOthwMzoOByYY2ZHhxmbiIiIiIiUn6QOA7tikfVZc7/aEHYssm/MvXTbRZvZIwTtf5cDzwEvFatI3d9z7Yj+ei/wBtAXeAC43N1f2M0xHwNDgM+BvwL1gUeBL3epqC1+zO3AbbvOZ2RkkJqaeiChS0WwaXFQGTvzNfBoO+B2xwYVs017lVsYSzdmcemLU/h+fSbxMRH+elJnzjqsRbldX6QasrADqMyin71d3H3RLvNtgdnufsAl/dFzfA903V1Vq5n9ANzr7vcXm/sLMMLdu5ewfwKQUGwqBVipz3CREnz6Z5jwENRpSWHdNhxeuIgdOIdRk7j9/KezJhGujtSjtSX8ZH7pxiwWbcphSduRXHb++aUZvVQf+hzfhZlNAz5x9z/uMn83cIy7l98fN2Us+lB0xvRpU0lJ0bPRIiKVTes2bfU5LiJSRpI6DGwHkDV/7PdhxyL7rtSTsLDzhugpwIVAP+AD4BngU9+PC5pZLjDF3fsVm3sQ6OvuR+7mmE+BgUBjd8+Izp0C/A9Icvftu4lXN3Crqk2LYcy/YOZ/f0zGth8eJGMP6VkuIWTm5PP7N2bw0ey1AJxzeAtuO6Ez8bEVellmkcpKf/QdBDObDTzu7g/vMn81cIW7dzrA80aAd4Ha7j5gD/vlAue7+6vF5n4D3ObujUrY/3b0IJXIvpn9JvzvxxVBzm/SkKk1DnyphDO3buPPm0ru8jE7vgddbv76gM8t1Zo+x3cRfUCqq7t/v8t8e2DmwTwgVdEoCSsiUrkpCSsiUjaSOg1ujnutrHljtFRXJVPqa8ICuHsOwXpur5rZoQTtBx8FYs2ss7vva3+yNQRtl4qbR9AqcU/HrCpKwBY7xoBmBBU4JcWbU/TaTN8XqpR6beDkx2DQDT9Wxi78OBhpxwfJ2CY/K64qVckJsTx6Ti8e+XIR//5sIS9H14l99Ne9aJhSZe6ZiEjVcC/wsJk1AL6Izg0DridYi+5APQJ0AXabgD1AdxHEXCQFWFnK1xCpGjqNgF8lwfagSc0/87bxzbZlFLJ/D2VOy1rJm+kzWN20Bww84yfbVi2ZS9OZD1E7b20pBS0iwAagBz//W7YHsL68gxERERERkfKT1GlIAwoLGmXNHzcl7Fhk/5VJEnYXhYATJEFj9vPY8UDaLnPtCVod7+mY080suViyt300Dt2Urc7qtYGTH4eBN8CYe2DWG7Dgw2B0+D8YchM07lJmlzczrjqqHZ0OSeW3r05nyvLNnPDQOJ44tw89mtcus+uKiOwPd3822iHiT8At0ellBFWwLx7IOc3sYeD/gEHuvrfP4rXArhWvjaLzJcWrB6lE9lUkBtKG73zZEDjhAE5Tf9V43vz8ctbEGPQ4+6eXqLMAZj5Ew8KNFBYUEInZ36//IlKCp4Anzaw1wRrtAP2BG/npg0giIiIiIlKFJHUanOIFeWnZCyaMCzsWOTBl0gvVzBLM7Fdm9hmwEOgKXAW02I8qWID7gCPM7GYza2tmZwOXElTTFF3rLjMrflP4FWAT8JyZdTKzQcA/gWdLakUs1VD9tnDKk3Dlt9D1dMBg/vvweH94/XxYP79ML39Uh0a8c1V/2jRIYt3WHM54YiJvTFlRptcUEdkf7v6YuzcjSH6munvrA0nAWuBh4GTgKHdfug+HTSSovC3u6Oi8iFQATZKaALA26+fPRtRv0pICN+Itn/QNq8o7NJGq6m/AX4Grga+j4yrgduCO8MISEREREZGyktRhQBwF+X2VgK3cSj0Ja2aPErQE/iPwPtDc3U939w/dixbk3DfuPpngxu2vgNkEFTnXuvvLxXZrArQodkwmwc3a2sAU4GXgPeCaA31PUkXVbwenPg2/mQSdTw7m5o6CR4+ANy+GjYvK7NKtGyQz6sr+/KJjI3LzC/n9/2Zy+7tzyCvYr/9ERETKlLtv2M+Hp3b1CPBr4Gxgm5k1jo6aRTuY2YtmdlexYx4AhpvZ9WbWIbrmax/gJ2vUikh4Gic1BiAzL5Ntudt+si0uPoGNVheA9NVLyj02karIA/dFH5CqBdRy92bu/oC7718/cRERERERqfBqDzjb3AuHZi+c8GXYscjBKYtK2MuBrcASYDBB26S3dh37ejJ3f9/du7p7DXfv6O5P7bJ9pLsP2WVuvrsf7e6J7t7c3a9XFazsVsMOcPrzcMUE6HgC4EGr4kf6wqjfQPq+FG7tv5QacTx5bm9+O6wdAM9PWMa5z3zDpsycvRwpIlJpXEFws/grgge0isaZxfZpQfBAFQDuPoEgaXspMAM4DRjh7rPLJ2QR2ZvEuERqJdQCSq6G3RzbEIDM9cvKMyyRKsvMvjCz2gDuvs3dt0XnU83siz0eLCIiIiIilU7u+mVHxdRM/UIPXVZ+ZZGEfRH4EtgCZOxhiFQsjTrDmS/BZWOg/XDwQpj+MjzcB969BraUfsvgSMS47uj2PHFub5LiY5i0JJ0THx7PnNX6T0REKj93t92M54vtM8TdR+5y3BvunubuCe7exd0/LO/YRWTPGicG1bBrstb8bFtWzWBb7qYfyjUmkSpsCBBfwnwNYGD5hiIiIiIiImUpKa1//0hC4jfbpn+cH3YscvBiS/uEu95IFal0mnSHs1+DlVPgyzth8WiY+gLMeBV6XwADfwcpjUv1ksd2bszbV/bn0hensGxTNqc+NoF7TuvOid0PKdXriIiIiJSGJklNWLB5QYmVsHlJh8A2IGNl+QcmUoWYWbdiLzuZWfE/QmKA4YAWXxYRERERqSKS0vr3tbgaCzJnfX4wy4NJBVIWlbCYWUszu8TMrjSzzmVxDZEy16wPnPsWXPAxtBwIBbnw7RPwQA/49BbI2lSql2vfKIV3rhzAoPYN2JFXyDWvTuMfH8+noFAdB0Sk/JjZeWaWUMJ8vJmdF0ZMIlLxFK0LW1IlrNVuDkB81upyjUmkCpoOTAMc+CL6umh8B/wZ+GsokYmIiIiISKlKSuvflZjYtZmzR28MOxYpPaWehDWzocAc4AngIWCamf26tK8jUm4OPRLOfw/Oewea9YX87TDhQXige1Apu6P0WgfXSozjuZF9uWxQawAe+2oxF70wmYzteaV2DRGRvXiOYB3XXaVEt4mI0CQ5WMq5pErY+LpBEjYl5+fbRGS/tALaAAYcFn1dNJoCqe7+bHjhiYiIiIhIaUjqMLAdMbHbs+Z+XfprIkqoyqIS9m/AZwR/FNYDngLuKYPriJQfM2g9BC76DM5+HRp3hdxt8PU/4P5uMPZeyM0qlUvFRIybju/IA2f1ICE2wlcLNnDyI+NZtF4dCESkXBhBxc2umqE13UUkqklSkIQtqRI2pVErAOrkbyjXmESqGndf7u7L3D3i7lOir4vGGncvCDtGERERERE5OEkdB7bALCFr7teLwo5FSl+prwkLdAH6ufsaADP7PXCZmdVz99Lt3ypS3syg/bHQ9miY/15QCbthPoz+C0x6DAZeD30ugNifdfLcbyf1aEqbBslc+uIUlmzM4uRHxnP/WT0Y1rFRKbwREZGfMrOidocOjDaz/GKbYwiqbj4OIzYRqXiK2hGXVAlbv2mb4Cdb2LE9ixo1k8o1NpGqysw6AS2A+OLz7v5uOBGJiIiIiMjBSOowsCF4w6z5Y6eEHYuUjbKohE0FdvasdvdsYDsltzYUqZwiEeh0ElwxAU5+Euq0hKz18PGN8GAvmPoiFOTv9TR706VpLd69egCHtazLtpx8Ln5xCo98uQh3rRMrIqVuFPAOQSXsJ9Hfi8Z/gcsALS8gIsCPlbDrstZRUPjTYrxadRuS7cEDaRtXLS332ESqGjNrbWYzgNnABwSf2aOAt6NDREREREQqmZTux6bihe2y5o9TArYKK4tKWIBjzax4y8IIMMzMuhRN6GldqRIiMdD9TOhyCkx7Cb6+B7auhHevhnH3w9CbofMpQdL2ANVPTuCliw/nr+/P4aVJP/DPTxYwd/VW/nl6NxLjy+o/YRGpbtz9LwBmtgx4zd13hBuRiFRk9WvWJ8ZiyPd8Nu3YRMPEhju3WSTChpgGHFq4ki1rl9CsbZc9nElE9sEDwFJgWPTnYQRL//wbuCHEuERERERE5AAkdRgQh3ufrAXjvwg7FilbZVEJC/ACPz6dOwqoCTzBT5/YFak6YuKCNsTXTINj74TEepC+GN68CJ4YCAs+hoOoXo2PjXDHiK7ceXJX4mKMD2at4dTHJrIiPbsU34SICLj7C0ANM7vYzO4ys7oAZtbLzJqGHJ6IVBCxkdidideS1oXdGhdsy96wvFzjEqmijgRudfeNQCFQ6O7jgJuAB0ONTERERERE9kvtAWebe+HQ7IUTvgw7Fil7pZ6EdffIPoyY0r6uSIUQVwOOvBJ+OwOG/hkSUmHdbHj1THjmaFg65qBOf/bhLXjlkiOonxzPvDVbOemR8UxaoqWWRaT0mFk3YCFwI0F1Te3oplOAu0IKS0QqoKKWxCUlYbcnBtsKNq8o15hEqqgYYFv0943AIdHflwNpZXVRM3MzG1FW5xcRERERqY5y1y87KqZm6heuNQerhbKqhBWp3hJSYPDvg2Rs/2shtiasnAwvnAAvngQrvzvgU/dtWZd3rxpA16a1SM/K5ddPf8OLE5dpnVgRKS33Ac+7ezugeEviD4FB4YQkIhVRo6RGAKzNXPuzbQUpzQCI2baqXGMSqaJmA92jv38D/MHM+gO3Akv292Rm9nw0wepmlmdm68zsMzO70MwO6B6BmcWZ2T/MbJaZZZnZajN70cwO2fvRIiIiIiLVQ1Ja/0GRuISJ26Z/nB92LFI+Sj0Ja2a9zexLM0stYVut6LbuJR0rUuUk1oWj/wK/nQ59L4FIHCz5Cp4+Cv57Dqyfd0CnPaR2Td64/EhO6nEI+YXOre/M4ea3Z5GbX1iq4YtItdSHYAmBXa0CGpdzLCJSge2pEjamTnMAamz/+TYR2W938OPf7rcCrYCxwPHANQd4zo+BJkBL4DjgS4K1Z983s9gDOF8i0Av4W/TnKQRVuu8eYHwiIiIiIlVKUlr/w4hE5mTO+VJrDFYjZVEJez3whbtv3XWDu2cAnwG/L4PrilRcKY3hl/+Cq6dA97PBIjD/fXj0SHj7cti8bL9PWSMuhvvP7MFNx3UgYvDqtys4+6lJbNiWU/rxi0h1kgP87EEqoD2woZxjEZEKrCgJuzbr55WwiQ0OBaBW7rpyjUmkKnL3T9z9rejvi9y9A1AfaOjuXxzgaXPcfa27r3L3qe5+J3ASQUJ25K47m9l5ZpZpZu2KzT1qZvPNLNHdM9z9aHd/3d0XuPsk4Cqgt5m1OMAYRURERESqhKSOA7sRiVmVNW+s1hasZsoiCXs48M4etr8H9CuD64pUfHVawsmPwRUToeMJgMOMV+GhPvDBDbBt/25UmhmXDW7DsyP7klIjlinLN3Piw+OYuXJLWUQvItXDu8CtZhYXfe3Rm6f/AN4MLywRqWj2VAlbq3ErABoUbMAL1alDpLS5e3ppryEVTejOIKhi3XXbiwRLE7xsZrFm9kvgYuAcd9/dk/y1AAe27O6aZpZgZqlFA0g5yLchIiIiIlKhJHUclAaWmTVvjNbrqYbKIgnbFNi2h+2ZBG2PRKqvhh3gzJfgki+g9VAozIPJT8GDPeDzv8D2Lft1uiFpDXnnyv60aZDEmowdnP74REZN07/pInJArgeSgfVATeBrYBHBZ/ufQoxLRCqYxklBh/KSKmEbHBIkYRMth4z09eUal0hVYGZv7eso5UvPJ2hRXJLLCP6WfxB4Brjd3b/bTfw1CB7gerWkLlnF3ARkFBsrDyxsEREREZGKJ6njoJZ4YWzWvDFLwo5FwlEWSdgNBGu/7E4HYGMZXFek8mnaG84bBee9C037QF42jLsXHugGY++F3H1vD9+6QTJvX9mfozo0JCe/kGtfm85dH86joLBUH5AXkSquqJ0gcALBOnMPA8e7+2B3zwo3OhGpSJokB89Vbs7ZzPb87T/ZViMxmU3UAmDjav2tKXIAMvZjlCYjqF79GXffDFwEXAEsBu4u8QRBN43Xo+e6Yi/Xu4ugYrZoNDugqEVEREREKpjkzkMbUVhQN2v+uDlhxyLhKYsk7OfsplLGzCy67fMyuK5I5dV6MFz8OZz1CjToCDsyYPRf4MGeMPlpKMjbp9Ok1ojjqfP68JshbQB4YswSLnx+Mhnb9+14EZEi7j7O3R9193vcXZ/bIvIzKXEpJMYmArAu6+dLKqTHNgQgc92y8gxLKokNGzZwxRVX0KJFCxISEmjcuDHHHnssZta/tK9lZmeZmZvZqNI+d1lx9wv2dZTypTsCS/ewfRBQQFARm7TrxmIJ2EOBo/dSBYu757j71qLBnrtqiYiIiIhUCsldjqrlBXltshaMnxp2LBKuskjC3gF0NbNvzOwMM+seHWcC3wBdgL+XwXVFKjcz6PBLuGI8nPwE1G4BmWvhg+vh4T4w8w3YhzXVYiLGH4Z34KFf9aRGXISvF25gxCPjWbRe9zNEZO/M7JrdjKvN7BIzG2pmMWHHKSLhM7M9rgubmdAIgJxNy8s1LqkcTj31VKZNm8YLL7zAwoULeffddxkyZAhAvdK8jpm1BP4FjC3N81ZFZnYU0JXdrAFvZv2AGwm6ZWQSdMsovr0oAdsO+IW7byrTgEVEREREKqDkrsMSPC+nd9b8cRPCjkXCF1vaJ3T3xWb2C+B54L/82MrIgLkET8MuKu3rilQZkRjofhZ0PgW+ex7G/BM2L4O3Lobx98NRt0D7Y4Ok7R6c0P0QWtVP4rL/fMfSjVmMeGQCD5zVg2EdG5XHuxCRyus6oAGQCGyOztUBsgluuDYElpjZUHdfEU6IIlJRNE5uzOKMxSWuC5uTdAhkgW/RPxXyU1u2bGHs2LF89dVXDB48GIBDDz2Uww47jJtuuuldADNrATwEDAMKgY+Bq919nZl1AKYCF7v7K9H9zwBeAHq7+9zoXAzwMnAbMBCoXY5vs9SY2VJ20yIYwN1bH8BpE8ysMRADNAKGE6zP+j7wYgkxpAD/AR5094/MbCUw2czec/f/RROw/wN6Af8HxETPD5Du7rkHEKOIiIiISKVSe8DZVpiTPWj795PUVU6AsqmExd2nuHsXoDdwFvAroJe7d3H3yWVxTZEqJzYeDr8UfjsdjvozJKTCutnw6pnw7HBYvvcHabo0rcU7V/XnsJZ1yczJ5+IXp/DIl4tw1zqxIrJbNwOTgXbuXs/d6wHtCbpZ/BZoAawF7gsvRBGpKPZUCUutYGnHuKzV5RmSVALJyckkJyczatQocnJyfrbdzCLAO0BdYDBwNNAaeA3A3ecDNwCPmlkLM2sGPA7cWJSAjboVWO/uz5TpGyp79wMPFBuPAhMJ1lB98gDPORxYAywjSHAPJVgL/iR3Lyhh/weALILvCbj7rOjvT5hZU6ApcCLBmq7To+cuGv0OMEYRERERkUold/2yX1gkZrTrBrxEmf638HNmlgpkZGRkkJqaGnY4IoHs9KAS9psnIH9HMNfumKAytkm3PR6am1/IX9+fw0uTfgDgl92a8M/TupEYX+rF8CIVwZ7LxGWPzGwxcKq7T99lvifwpru3jrYjfNPdm4QR457oM1ykfD0580kemvYQI9qO4G/9//aTbVM/eo5e31zL/LiOdPjTpJAilIrqzTff5JJLLmH79u306tWLwYMHc9ZZZ9GtWzczs6OBj4BWRV0XzKwTMAc4rOjBXjN7H0gFcgnWKR1edLPDzAYQdGbq4e4bzex5oLa7jyjnt1pmzOxKoE8ZrAsbmqLP8enTppKSkhJ2OCIisp9at2mrv8dFpNpKbN9vUCQuYUrmnC+zw45FKo4yy8CY2b272eTADmAR8I67p5dVDCJVSmJdOPqvcPjl8PU9MPVF+P7TYHQ5DYbeDPXalHhofGyEO0Z0pWOTVG57Zw4fzFzD0g1ZPHleb5rVSSznNyIiFVwTSv5+EAsUtRVcDejOqIjQOCn4Z6GkdsTJjVoBUCdvfbnGJBXTqsxVvLbgNeanz6dJUhNOG3Qaq1evZuzYsUyaNImPPvqIe+65h4KCgpEEidUVxdveu/tcM9sCdCTo2ABwIbCQoF1x52IJ2KLWuZe4+8bye5fl7iPgLqDKJGFFRERERCqjpLT+h0XiEmYrASu7KpN2xFE9gYuASwlaSA0GLonODQPuBRZFn2gWkX2VegiccD9cNTlIvgLM/h88chi8fx1sLaEdYNQ5hx/KK5ccQb2keOau2cqJD4/nmyWbyiduEaksviRoLdizaCL6+2PAF9GprsDSEGITkQqmqB1xSUnYuo2DJGx9Tycv9+ctZ6X6mLlhJiNGjeDFOS8ycfVE3ln0Dmd/eDbv//A+Rx99NLfccgsTJkxg5MiRAH/Zj1N3B5Kio3h3hjZAS+A9M8s3s3zgPODE6OuSn1ysfE4D9FCziIiIiEiIkjoM6E4kZlXmnC/13Vx+piyTsO8AnwOHuHtvd+9NsD7MZ8CrBGvGjEFryokcmHpt4LRn4LKxQVviwnyY8iw82BM+uzVoX1yCw1rV5d2rB9D5kFTSs3I55+lv+M+k5VonVkSKXERwQ/c7M8sxsxxgSnTuoug+mcD1IcUnIhVIUSXsmqw1P/suUbdRM3I9hhhzNq5ZFkJ0UhG4O7dNuI3cwlwKokuNFv2885s72bJjy859O3XqBEFCdR7Q3MyaF22LPrxbG5gbfV0XeB74e/Tny2ZWM7r7fIIHhnoUG+8SPGjUA9hZYVsZmNk0M5tabEwzszXAndEhIiIiIiIhSOowoAMW2ZY1b8yqsGORiqnM1oQ1s1XA0e4+d5f5zsCn7t7UzHpFf69fJkEcIK0nJ5XS8gnw+V9gRXTNtYRa0P8aOOIKiE/62e7bcwv4w5szeW/GagB+dVgL/nJiZ+Jjy/LZDJFyoTVoDpCZGdAc2AC0ANKimxa4+4LQAtsP+gwXKV+5Bbn0eakPjhMbicV2/Se4ILcUr2YQE1eK55Py4O7ke/5ut19+6OUcf+jxTJkyhauvvpp169Y9C1wMTAW2AdcStMR/FMh09yEAZvY6wWfVACABmAZ85u5XlnSdyrwmrJndtstUIcFn9VfuPj+EkMqM1oQVEanctCasiFQnSR0HtQRPzJo3du5ed5ZqqyyTsJnA/7n7V7vMDwHec/cUM2sNTHf3CnWXVDdwpdJyh4WfwOi/wvo5wVxSQxj8B+h1PsTG77K78/jXS7jnk/m4Q59D6/DYr3vTICUhhOBFSo3+6DtAZhYhWLe9s7t/H3Y8B0Kf4SLl76JPLuLbtd+GHYZUUumvpLNl3BaaN2/O6aefzh133JHo7tvNrAXwEMFSNoXAx8DV7r7OzM4jSMr2LPq8MrPDgHHASe7+0a7XqcxJ2OpESVgRkcpNSVgRqS6SOg1uTGFhk6z5Y6eFHYtUbGWZhH0ZOJKgXeHk6HRf4F/ABHc/18zOAm5w9z5lEsQB0g1cqfQKC2D2m/DFHbBleTBX+1AY+ifoejpEflrt+uWC9Vzz6jS27cinSa0aPHFub7o1q13+cYuUDv3RdxDMbA5wkbtPCjuWA6HPcJHyV+iFrM9eX+I2L3S2bFhNQeHuKyFLMnHxJp4au5QezWvz15M6w6jfwIb5cOLD0Pao0ghbykmhF3Luh+eyYfsGnJ/+7RkhwienfbKzrXWUPsd3Ef1sK4kDOe5emiXnoVISVkSkclMSVkSqg6SOg2pTWNAxa8H4iWHHIhVfbBme+zKC9V7/W+w6+cALwHXR1/MJWk2JSGmKxEC3M6DTCJj6Anx9T5CMfftSGP8ADLsF2g8HC74bD01ryDtX9ueSF6eweEMWpz8+kbtP7crJPZuF+z5EJAx/BP5pZle4++ywgxGRii9ikV2TaD/RJKXJfp9zQ8F61n/prNyRSuPGPSClKaydAwX5sIdrScV0y5G38NsvfkvEIhR4ARGLUOiFXNzt4j3+b0d22gLs9ulpM1tJsC7uX9y9sJxiEhERERGpdpK7Dkvwgrye2Qsnfhl2LFI5lNnij+6e6e6XAPWAntFRz90vdfes6D7T3X16WcUgUu3FxsNhl8Bvp8OwW4N1YtfPgVfPgmePhWXjd+7aukEyb1/Zn2EdGpKTX8h1r83g7x/MJb9A93FEqpkXgcOAGWa23czSi4/9OZGZDTKz98xstZm5mY3Yy/5DovvtOnSHXqSaqZ8ULI2QnhUt8EtqEPzM2hhSRHIwhjQfwgvHvcCQ5kM4JOkQejbsyT8H/ZOrelwVdmiVxUhgNXAnMCI67gRWAVcATwLXEDxIJSIiIiIiZSCpw4BIYe72wdu/n/RV2LFI5VGWlbBAkIwtumnr7pllfT0RKUF8Egy8HvpcGFTCTnocVnwDzx8PbX8RJGibdCe1RhxPndeHez9byMNfLuKpsUuZv3YbD/2qJ7UT4/d+HRGpCq4txXMlATOAZ4G39uO4NGBrsdcl9zkVkSqrbnLwvSM9Kxd3xxLrBRuyN4UYlRyMHg17cH/D+8MOo7I6H7je3V8vNveemc0CLnP3YWb2A/AnguSsiIiIiIiUMi8sGGaRmM+9rNb4lCqpzJKwZhYB/kywJmxydG4b8G/g72qTJBKCmnXgF7fD4ZcHLYqnvgCLPg9G55Nh6J+J1G/LDcem0emQVK5/fQZjv9/IiQ+P56nz+pDWWOsyiVR17v5CKZ7rI+AjALP9Whpovbtv2ZcdzSwBSCg2pX+oRKqAeklBEja3oJBtOfmkJtUPNmRtCDEqkdD0Ay4vYX4acGT093FAi3KLSERERESkGkls329QTM3UcdtmfKK8luyXMmtHDPwduIqgJVJRO+KbgauBv5XhdUVkb1Iaw//dC1dNhq5nAAZz3oZHDoN3r4aMlRzftQlv/aYfzerU5If0bE5+dDwfz14TduQiUo7MrIaZpRYf5XTp6Wa2xsw+M7P+e9n3JiCj2FhZ5tGJSJmrERdDYnwMAOmZuZBYlIRVO2KpllYAF5Uwf1F0GwTLAG0ut4hERERERKqJxLR+h0fiEmZvm/HJ9rBjkcqnLJOw5wMXu/tj7j4zOh4FLiFY00ZEwla3NZz6FFw+DtoPBy+AqS/Cg73g45vpmJrLe1cNoF+bemTnFnD5S1O599MFFBaq44JIVWVmSWb2sJmtB7IIbugWH2VpDUGlz6nRsQL4ysx67eGYu4BaxUazMo5RRMpJ3Wg17KasXCiqhM1WElaqpRuA68xshpk9HR3TCZYQuD66T1/gtZDiExERERGpkhLT+vWw2PgVmXO+TA87FqmcyjIJWxeYX8L8/Og2EakoGneBs1+DCz+FQwdAQQ5MegQe6E6db//Ni+d04ML+rQB48ItFXPLiFLbuyAs5aBEpI/cARwFXADnAxcBtwGrgvLK8sLsvcPcn3P07d5/g7hcCE4Dr9nBMjrtvLRrAtrKMUUTKT1FL4vSs4pWwWhNWqh93fxfoQNDiv250fAR0cPf3o/s85u6/Cy9KEREREZGqJSmtf0eLxG7NmvPV6rBjkcqrLJOwMwjaEe/qKmDmvp7EzJqa2UtmtsnMtpvZLDPrs4/H9jez/OhTwiKyNy0Oh5Hvw6/fhCbdITcTvr6b2Id6cmu90dx3ShrxsRFGz1/PiEfGs3hDZtgRi0jpOwH4jbu/CeQDY939DoIlBc4JIZ5vgbYhXFdEQlZ3ZxI2B5LqBZPZG8HVkUOqH3df6u5/dPdTouMmd18WdlwiIiIiIlVRUocBLYlELGvemCVhxyKVW1kmYf8AXGhmc83smeiYS9CK+IZ9OYGZ1QHGA3nAcUAngnZLe22HaGa1gReB0QcUvUh1ZQZtfwGXfg2nvwD12sH2dPj0z5w89gRGD15Ks9RYlmzIYsTD4/l87rqwIxaR0lUXKPqCuZUfu1eMAwaFEE8PgjbFIlLN1E1KAKLtiIsqYfN3QG5WiFGJhMPMBkYfTp5gZk2jc+ea2YCwYxMRERERqUqSOg85BKxO1ryxc8OORSq/MkvCuvvXQHvgbaB2dLwFdAbO3cfT3AiscPcL3P3b6NO/n7r74n049nHgFWDi/sYuIgTJ2M4j4DeT4MSHoVZz2Laa5uNv4qvEG7mu0Qwyc3K5+MUpPDj6e60TK1J1LAFaRX+fD5wR/f0EYMv+nMjMks2sh5n1iE61ir5uEd1+l5m9WGz/a83sJDNra2ZdzOx+gtbIjxzwuxGRSqt+crQSNjMX4pMgtmawQevCSjVjZqcCnwDbgV5AQnRTLYJOFSIiIiIiUgqSOg6qTX7eoVnzx04LOxapGsqyEhZ3X+3uf3L3U6Pjz0Ad4KJ9PMWJwBQze8PM1pvZNDO7ZG8HmdkFQGvgL/tyETNLMLPUogGk7GN8IlVfTCz0Oheu/g6OuweSGhC7ZSm/zfgHE2vfxtGRKdz72QKuePk7MnPyw45WRA7ec0D36O93A1ea2Q7gPuCf+3muPsC06AC4N/r7X6OvmwAtiu0fD/wbmAV8HY3jF+6urhYi1VBRO+JNWbnBw2FJRevCKgkr1c6fgcvd/RKCLlFFxhMkZUVERERE5CCldD+2phfm98haMF6FfVJqYsMOYC9aA1cQ3LS9E+gLPGhmue7+QkkHmFk7gpvGA90938z25To3AbeVTsgiVVRsAhx+GfQ4B759AsY/QOMdi3kq/l5mFLbhnnlncPLDmTx5fl9a1U8KO1oROUDufl+x3z83sw5Ab2CRu+/zmu7R478CdvtB7O4jd3l9D3DP/lxDRKqunyRhARLrQcYKJWGlOkoDxpQwn0HQcUpERERERA5CUocBEdwHZC+Y8FnYsUjVUqaVsKUgAkx195vdfZq7Pwk8BVxe0s5mFkPQgvg2d1+4H9e5i6CVU9FodnBhi1RhCckw8Hr47YzgZ1wi3SOLeTn+Lv665SZuefgZvpy/PuwoReQAmdl5ZlbU5hB3X+7ubwHzzey8EEMTkWqmXlE74qycYKKoElbtiKX6WQu0LWF+AD+u4y4iIiIiIgfICwuGYaZObFLqKnoSdg2w6+LH8/hp68LiUghaHz5sZvlmlg/cCnSPvj6qpIPcPcfdtxYNYFspxS9SddWsA8NuDZKxh1+Ox8RzZMxcXuIWCl8+g/+++77WiRWpnJ4jeCBpVynRbSIi5aJuUvA8SHpmUSWs2hFLtfUU8ICZHQ44cIiZnUPQwv+xUCMTEREREankEtP6DbFIzNis+eMKw45Fqp5Sb0dsZm/tZZfa+3G68QStl4prDyzfzf5bga67zP0GOAo4DVi6H9cWkX2R3BCO+wd25FUUfHUPTH+ZYTHTYOo5TFkwmI5n30VS085hRyki+84IbvDuqhlB20MRkXJRr1g7YnfHVAkr1dfdBA9QjwYSCVoT5xCs1f50iHGJiIiIiFRqSWn9j7RI7PSseWN2hB2LVE1lUQmbsZexHHhxH891H3CEmd1sZm3N7GzgUuCRoh3M7C4zexHA3QvdfXbxAawHdkRfZ5XSexSRXdVuTsyIh4i5ejLLDzmOQjf6ZH1NjacGsO3ViyBdz0CIVGRmNs3MphIkYEeb2dRiYwYwFvg83ChFpDopWhM2J7+Q7NyCYE1YgKxNIUYlUv488HegLtAFOAJoQPD3tb5ki4iIiIgcgKS0/r2IjVueNW/MlrBjkaqr1Cth3f2CUjzXZDM7mWDN1lsJ/sC81t1fLrZbE3bfnlhEylu9Nhx66X+ZN30i69+5hcE+mZQF/6Nw4Sgivc6FQb+HWk3DjlJEfm5U9GcP4BMgs9i2XGAZ8Ga5RiQi1VpifAwJsRFy8gvZlJlLUlElbNaGcAMTKSfRNdpvB44mWvnq7qPM7ALgbaCA4MFlERERERHZD0mdBnfELD1rzlerw45FqrZST8KWNnd/H3h/D9tH7uX42wn+cBWRctSxx5HUa/M+f3z+VY5b/yyDY2bCd8/h01/B+l4EA64LWhmLSIXg7n8BMLNlwGvurjYsIhIqM6N+cgKrtmxnU1YOLZIaBBvUjliqj78ClxF0ougHvGFmzxFUwl4PvOHuBSHGJyIiIiJS6SR1HNQaL/Ss+eOWhR2LVH1l0Y5YRASAhik1+OsV5/NFn8c4PedWvinsgBXkwKRH4YHu8NltkJ0edpgiUoy7v6AErIhUFEUtidOzciGxqBJW7Yil2jgdOM/dTwOOAWIIHqTu7u7/VQJWRERERGT/JHUecgjutbLmj5sfdixSPSgJKyJlKj42wl9O6sKZp53JuYW38evcm5gbaQ952TD+fri/G3zxd9i+JexQRUREpIIpSsJuysqFpOiasKqEleqjGfAdgLvPJmhJfJ+7e6hRiYiIiIhUQsmdh9b1/NzmWfPHTgs7Fqk+lIQVkXJxWu9mvHVFf5amHsbx2bdxRcEfyKjVEXK3wZh74IFu8PU/IWdb2KGKiIhIBVGvpErYvGzIzQ4xKpFyE0OwLnuRfH66ZruIiIiIiOyDlO7H1vT83K7ZCyZ8E3YsUr0oCSsi5aZL01q8d/UA+retz0d5Peix7k/8r82deIMOsCMDvrwjqIwddx/kZoUdroiIiITsJ+2IE1IgJnitalipJgx43szeMrO3gBrA40Wvi82LiIiIiMhuJHUYECnYsW1g1oLxX4cdi1Q/SsKKSLmqmxTPCxccxmWDW+NEuGFOS86O/Tdbf/k41GsL29Ph89uDZOyEh1XpIiIiUo3VTY62I87MBbNi68IqCSvVwgvAeiAjOl4CVhd7XTRERERERKQEZmZeWDDMLPJ52LFI9RQbdgAiUv3ExkS46biO9GhWmxvemMHEpRkcm96QR3/1MT23fA5f3w2bl8Gnf4IJD8KA66D3BRBXI+zQRao8M7t3N5sc2AEsAt5x9/Tyi0pEqqt6O9eEzQkmkurBttVKwkq14O4XhB2DiIiIiEhllpjWf4jFJYzJnDW6MOxYpHpSJayIhOa4rk1456r+tG6QxJqMHZz51BRezumHXzkZTnwIareAzHXw8R/hwR7wzZOQtyPssEWqup7ARcClwODouCQ6Nwy4F1hkZp1Ci1BEqo26SQlAtB0x/FgJq3bEIiIiIiIisgdJaf2PxCLTMmeNzgk7Fqm+lIQVkVC1bZjCO1f2Z3jnxuQWFPKnt2fz+7fnsaPrOXDVd/B/90NqM9i2Bj76PTzYE759CvL12SlSRt4BPgcOcffe7t4baAZ8BrwKNAXGAPeFF6KIVBf1ircjBkhqEPxUJayIiIiIiIjsRlKHgT2JiV2aNW/MlrBjkepNSVgRCV1KjTge+3UvbjquAxGD/323klMfm8CKrfnQ5wK4Zir88t+Q2jRoQfjhDfBgL5j8jJKxIqXv98At7r61aMLdM4DbgT+4ezbwV6B3OOGJSHVS1I54ZyVskiphRUREREREZPeSOg7shNnmrLlfrw07FhElYUWkQjAzLhvchpcuOpx6SfHMWb2V/3toHF/OXw+xCdD3YrhmGhz/L0hpAltXwge/UzJWpPTVAhqWMN8ASI3+vgWIL6+ARKT6qhtNwm7PK2B7bgEk1gs2ZG0KMSoRERERERGpiJI7D22De2HWvDHLwo5FBJSEFZEKpl/b+rx39QB6NK9NxvY8LnxhMvd+tpCCQg+SsYddAtdMh+P+uUsytidMflrJWJGD9w7wrJmdbGbNouNk4BlgVHSfw4CFYQUoItVHckIs8THBnyybsnJUCSsiIiIiIiIlSuo4qKkX5CVnzR83P+xYRIooCSsiFc4htWvy2mVHcO4Rh+IOD47+nguen/xjK8K4GnD4pdFk7D3RZOwq+OD6H9eMzdsR6nsQqcQuA0YD/wWWR8d/o3OXR/eZD1wcSnQiUq2Y2c5q2PSsXEiMJmG1JqyIiIiIiIhEJXceWpfCgqZZ88fNCDsWkeKUhBWRCikhNoa/jejCfWd2p0ZchDELN3DCQ+OYsWLLjzvF1YDDLytWGXtIkIz98AZ4sAd88wTkbQ/pHYhUTu6e6e6XAPWAntFRz90vdfes6D7T3X16iGGKSDVSlITdlJn7YyVs1oYQIxIREREREZGKIrnz0MTCvJwuWQvGfxt2LCK7UhJWRCq0k3s2Y9SV/WlZL5FVW7Zz+uMTeWnSctz9x512VsZG14xNbQrb1sBHf4AHusPERyA3O7w3IVIJuXsmkA6kR38XEQlFveRoErZ4JWy21oStyszMzWxE2HGIiIiIiEjFVqNF15jC/JwB2QsnjAk7FpGSxIYdgIjI3nRonMq7Vw/g92/M4JM56/jzqNl8t3wzfz+5C4nxxf4Zi6sRrBnb6zyY9hKMuw8yVsAnNwe/97sa+lwECcnhvRmRCs7MIsCfgeuB5OjcNuDfwN/dvTDE8ESkGvqxHXEOJNULJnMzg6UH4mqEGJkcjJEjR7JlyxZGjRpV6uc2s7rAX4BjgBbABoJ1zW9x94xSv6CIiFR52Z4UdggiIrILM7OabQ8fFt+g5WdhxyKyO6qEFZFKIbVGHI//ujc3HdeBmIjx9rRVjHhkPIs3lFCgF5sAfS+Cq6fCCQ9C7UODtoWf3Qr3d4Ux/4Qduv8msht/B64C/siP7YhvBq4G/hZiXCJSTe1sR5yVCzVqQyT6AFa21oWV3TokOm4AugAjgeHAMyHGJCIiIiIipSixfb+hkRpJX28Z94rvfW+RcCgJKyKVhplx2eA2vHLx4TRISWDhukxOfGgc789cXfIBsfHQ+3y4+js46VGo2xq2p8MXdwTJ2C/vhOz08n0TIhXf+cDF7v6Yu8+MjkeBSwhuYouIlKv6yQkApGfmgtmPLYmzlISt4uqb2dtmlm1m35vZiUUbzOxWM1ttZvWKzX1gZl+aWcTdZ7v7qe7+nrsvdvcvgD8BJ5iZukGJiIiIiFRySWn9+1tcwneZs0bnhB2LyJ4oCSsilc7hrevxwTUDOKJ1XbJyC7jqlWnc/u4ccvN30yU1Jg56ngNXToZTnoL6aUEl7Nf/CJKxn90KmevL902IVFx1gfklzM+PbhMRKVc/tiPODSaSitaFVRK2irsNeB3oBnwIvBxtMwxB14ZlwNMAZnYl0A84fw9t82sBW909vyyDFhERERGRspXYvl9vi41flDn7C7U6lApPSVgRqZQaptTgpYsO54ohbQB4fsIyTn9iIis3Z+/+oJhY6HYG/GYSnP4CNOoarCk3/oEgGfvRjZCxqpzegUiFNYOgHfGuropuExEpVz9pRwyQGC1+zNoUUkRSTp5391fdfRFBW/xk4DAAdy8Afg0MM7O7gX8CV7r7DyWdyMzqA7cAT5ZL5CIiIiIiUiaSOgzsajGxmzLnfLku7FhE9oWSsCJSacXGRLhxeAeeOb8PtWrGMWPFFn754DhGz9vLZ3AkAp1HwOVj4VevQdM+kL8DvnkcHugO714NmxaXy3sQqYD+AFxoZnPN7JnomEvQivj34YYmItVRvZ1J2GiXqaJK2KwNIUUkB8Ldmb5iC69++wNfLVhPIba3Q2YWOzYL2Ao0LDa3hGDN1xuBd939lZJOYmapwAfAXOD2g3oTIiIiIiISmqQOA9pglps1b8yysGMR2VdKwopIpTesYyPev3oA3ZvVImN7Hhe9MIW7P5pPfsHuutFFmUHacLj4czh3FLQcCIV5MPVFeLgP/O8iWDenXN6DSEXh7l8D7YG3gdrR8RaQ5u5j9+dcZjbIzN6LrtvnZjZiH44ZYmZTzSzHzBaZ2cj9fQ8iUrXsbEecWVQJq3bElU3G9jzOfHISIx4Zz01vzWLkc5P5psFxbI+vs6fD8nZ57fz879dBQAHQsqS1Xs0sBfgY2Aac7O67nlNERERERCqBpLT+zYDkrHljFoQdi8j+UBJWRKqE5nUTef3yIxnZryUAj3+9mLOf+oa1GTv2frAZtBkKI9+HCz+FdseAF8Ls/8Fj/eCVs2DF5LJ9AyIViLuvdvc/ufup0fFnIGJm+9vGMYmghfGV+7KzmbUiqFb6EugB3A88bWbH7ud1RaQKqZeUAEBWbgE78gqKVcIqCVtZ3Pz2LL5btvknczmRmixpPnzvD83thpmdCZwCDAFaELQbLr49FfgUyAVOdPd9+FIoIiIiIiIVTXKXYfUdb5o1f5yWyZJKx9w97BgqnOgf7BkZGRmkpqaGHY6I7KcPZq7hxjdnkpmTT72keO47sweD2jfYv5OsmQnj7oU5owgKLwgqZQdcB22OChK3UlHp/zllwMy6A1PdPeYAj3eCKqRRe9jnH8Av3b1Lsbn/ArXdffhujkkAEopNpQAr9RkuUnW4O+3+9BH5hc6xnRtxVOb7nLnuPtJjG7KkZteww9tvm+Ma8UH9Cyn8eeFmlZSTX8Anc3a/VMSfB9WjzyE1dr7u2bNnC+AHdvnMMLMtwLXu/ryZNSNoV3ybuz8UfVjnfWCgu08qloBNBE4GsopdckN0TVmpgIr+Fp8+bSopKSlhhyMislO2J4UdQqXQpW0T/T0uIqUqKa1/krv3zl44YUzYsYgciOrxl7+IVCu/7NaEzoek8puXpzJ3zVbOf+5brhzSlmt/0Y7YmH1sANCkG5z+PAxdBOPug5n/hWVjg9Gke5CM7XgiRA4oHyUiJTsS+HyXuU8IKmJ35ybgtrIKSETCZ2a0qJfIkg1ZfDJnHVmRGpwZD3Xz11N32+iwwzsgz6xtzaTCTmGHUSFcd/PtZM74pPjUHv9NNzMDnge+BR4GcPdPzOwx4CUz6wH0Ag6PHrJol1O0ApYdbNwiIiIiIlK2UnoMj3X3ftkLJ3wWdiwiB0qVsCVQJaxI1bAjr4A7PpjLS5N+AOCwVnV56Fc9aZRaYy9HliBjJUx4GKa+AHnZwVzdNtD/t9D9LIhN2PPxUp705G0ZKKdK2IXAc+5+V7G54wlaFCe6+/YSjlElrEg1sGj9NsYsjLYf9kJarf+MmjmVrx1xpxWvkrp9JaO7/ZvlDYeFHU65yM0v5F+fLiC/sOS/O9/6TT96tfjJ2rD6HK/GVAkrIhWVKmH3jSphRaS0mJnVbHfEMfENW326ZdwrSmJJpaVKWBGpsmrExXDHiK4c3qoeN701i2+XpnPcA2O594zuDElruH8nq9UMjrsbBv0evn0CvnkC0hfDe9fAl3fCEVdAnwugRq2yeTMiUiJ3zwFyil6bWoWLVEltG6bQtmHxhEyb0GI5KP+ZCotXMqx1EvRoFXY05SZjRx6Pf7WY4ndOYszo2aI2PZvXDissERERERGpoBLb9xsak1z3SyVgpbJTElZEqrwTuh9Cl6a1uDLannjkc5O5Ykgbfnd0e+L2tT1xkaR6MPRm6HdNUBU78RHYugo+vw3G/hv6XBgkZFMal82bESkjZvbWXnapXQ5hrAUa7TLXCNhaUhWsiEilEx+tosnNDDeOcnbDMWkAPDtuKTn5hZjBMZ0bcdcpXfXwjIiIiIiI/ERiWr8Bhk3Z+t17uWHHInKwlIQVkWqhVf0k3vpNP/7+wTz+M2k5j321mG+XpvPgr3rStHbN/T9hQjIceSX0vQRmvQHjH4CNC2D8/TDpUeh2ZpCobdC+1N+LSBnJ2IftL5ZxDBOB43eZOzo6LyJS+cUnBz+rWRI2JmLcOLwDVw5ty4r0bBqkJFA/WUs5iIiIiIjITyWl9e8biU34PnPOl1vDjkWkNCgJKyLVRo24GP42ogtHtqnHjf+byXfLN3P8A2O557RuHNv5ACtXY+Oh5znQ/Vew8OMgCbviG5j2n2Ck/RL6XwMtjijV9yJS2tz9gtI+p5klA22LTbUysx5Aurv/YGZ3AU3d/bzo9seBq8zsHuBZ4CjgDOCXpR2biEgoEoqSsFnhxhGS5IRYOjbRet0iIiIiIvJzSR0HdbHY+PWZc75cF3YsIqVlP/twiohUfsd3bcIH1wyke7NaZGzP47L/fMft784hJ7/gwE8aiUCH4+GiT+HCT4LkK8CCD+DZY+Hpo2Huu1B4ENcQqXz6ANOiA+De6O9/jb5uArQo2tndlxIkXI8GZgDXAxe7+yflFbCISJkqakecU70qYUVERERERPYkqcPAdhQW5GbO+XJ52LGIlCZVwopItdSiXiJvXN6Pf326gCfHLOH5CcuYvCydh37Vk9YNkg/y5EcEY8NCmPgQzPgvrPwWXj8X6rYO2hh3PxviE0vnzYhUUO7+FbDbxf7cfeRujulZZkGJiISpmq4JKyIiIiIisjtJnQY398KCmtkLJ8wMOxaR0qZKWBGptuJjI9x8fEeeG9mXuknxzFm9lf97aBz/+24l7n7wF2jQHk58CK6dDQNvgBq1IX0JfHA93NcZvvg7ZG44+OuIiIhI5RCfEvyspu2IRUREREREikvqNKQBBflNlICVqkpJWBGp9oZ2aMhHvx3Ika3rkZ1bwA1vzOC616azbUde6VwgpREMuwWumwPH3QO1D4Xt6TDmniAZ+85VsH5+6VxLREREKi5VwoqIiIiIiACQ3PUXyRTmd8haMP7bsGMRKSsVPglrZk3N7CUz22Rm281slpn12cP+p5jZZ2a2wcy2mtlEMzu2PGMWkcqnUWoNXrr4cG44pj0xEWPU9NX88sFxTF+xpfQukpAMh18G10yD01+Apn2gIAem/QcePRxeOhUWfwGlUYUrIiIiFc/OJKwqYUVEREREpPpK6TE8tjAn+8is+ePGhh2LSFmq0ElYM6sDjAfygOOATsD1wOY9HDYI+Aw4HugNfAm8Z2ZaX05E9igmYlx1VDteu/QImtauyQ/p2Zz22AQe/WoRhYWlmBiNxEDnEXDx53Dhp9DxBMBg0efwn5PhsX4w9UXI21F61xQREZHwJUTbEedsCzcOERERERGRkJiZFWRnDItv2PLzsGMRKWtWKuselhEzuxvo7+4DD/I8c4DX3P2v+7h/KpCRkZFBamrqwVxaRCqpjOw8bn57Fh/MWgNA/7b1uPeMHjRKrVE2F0xfAt88AdNe+rFFYWJ96HsR9LkoaGks+8rCDkDCo89wEanQlk+A546Dum3gmqlhR1NR6XO8Giv6HJ8+bSopKSlhhyMislO2J4UdQqXQpW0TfY6LyF4ltu83zCKRMVnzx5XSWnAiFVeFroQFTgSmmNkbZrbezKaZ2SX7cwIziwApQPoe9kkws9SiEd1fRKqxWolxPHx2T/5xaldqxsUwftEmht8/hk/nrC2bC9ZtDcf9I1g39pg7oFZzyN4IX/8jWDf27cth9fSyubaIiIiUj/jk4KfaEYuIiIiISDWUmNZvgMXGfasErFQXFT0J2xq4AvgeOBZ4DHjQzM7fj3PcACQDr+9hn5uAjGJj5QFFKyJViplxZt8WvHf1ADofksrm7Dwu/c93/OntWWzPLSibi9asDf2uhmumw+nPQ/PDoTAPZrwKTw6GZ4fDnFFQkF821xcREZGys3NN2Mxw4xARERERESlnSR0H9bWY+AVZc7/W+ixSbVT0dsS5wBR371ds7kGgr7sfuQ/Hnw08BZzk7rvtL25mCUBCsakUYKVaGYpIkZz8Av796UKeHLMEgLYNk3ngrB50PqRW2V985Xcw6VGYOwoKo8nX1GZw2MXQ63xIrFv2MVQuan9UjakdsYhUaNvWwb/bAwa3pkOkoj8TGwp9jldjakcsIhWV2hHvG7UjFpHdSeowsCtGRta8sT+EHYtIearof/WvAebuMjcPaLG3A83sLOBp4Iw9JWAB3D3H3bcWDUBPYojITyTExnDz8R35z0WH0TAlgUXrMxnxyHieHLOYwsIyfpilWW847Rm4dhYMvAES68HWlfD57XBvJ3j3Glg3p2xjEBERkYOXEG1HjEP+9lBDERERERERKQ9JHQa2A3YoASvVUUVPwo4H0naZaw8s39NBZvYr4DngV+7+QRnFJiLV0MB2Dfj42kEc06kReQXOnR/O59xnv2FNRjncSE09BIbdEqwbe9Ij0LhrcAN36gvwWD947pcw9x21KhYREamoYmuys9AzRy2JRURERESkakvqOLAFXlgja/7Y78OORSQMFT0Jex9whJndbGZto+2FLwUeKdrBzO4ysxeLvT4beBG4HvjGzBpHRzn0DBWR6qBuUjxPnNubu0/pSs24GMYv2sTw+8fy/szV5RNAXE3o+Wu4bCxc8BF0OgksBpaPg9fPgwe6wZh/QuaG8olHRERE9k0konVhRURERESkWkjqNKQB7g2zFoyfFXYsImGp0ElYd58MnAz8CpgN3AJc6+4vF9utCT9tT3wpEEuQqF1TbDxQHjGLSPVgZpx1WAs+uGYA3ZvVImN7Hle9Mo3fvTadrTvyyisIOLQfnPFisVbF9WHrKvjiDrivE7x1KayYDBV4/W8REZFqJT7akjg3K9w4REREREREykhSp8Epnp/TIWv+uClhxyISJnPdmP8ZM0sFMjIyMkhNTQ07HBGp4PIKCnlo9Pc8/OUiCh2a1q7Jv8/ozhGt65V/MPk5MGcUfPskrCr2HadxNzjsEuhyGsQnln9c5cvCDkDCo89wEanwHuwF6YuDbhaH9gs7mopIn+PVWNHn+PRpU0lJSQk7HBGRnbI9KewQKoUubZvoc1xESOowIM4LC4dkL5zwWdixiIStQlfCiohUBnExEX53TBpvXH4kLeomsmrLdn711CTu/HAeOfkF5RtMbAJ0PxMuGQ2XfAHdz4aYBFg7E969Gu7tAB/fBBu1DIOIiEgoElQJKyIiIiIiVVPtAWebFxYctf37iZ+HHYtIRaAkrIhIKel9aF0+/O1AzurbHHd4cswSTnxoPHNXbw0noKa94eTH4Pr5cPTfoE5L2JEBkx6Fh/vACycEVbMF5dQ+WURERIq1I9aasCIiIiIiUrXkrl86LCax1mhXC1YRQElYEZFSlZwQy92nduOp8/pQPzmeBeu2cdIj43j0q0UUFIb03SOxLvS/Bq6eBue8Ce2PAwyWjoE3zof7OgdryG5ZEU58IiIi1Ul8tJ1hjpKwIiIiIiJSdSSl9R8USUiatG36x/lhxyJSUSgJKyJSBo7u1IiPrx3E0Z0akVfg3PPxAs54YiLLNobYejASgXa/gLP/C9fOhIHXQ1IDyFwHY/4JD3SDl8+ABR9DYTm3URYREaku4tWOWEREREREqpakDgMPIyZuXuasz/W0qUgxSsKKiJSR+skJPHlub/51endSEmL5bvlmjntgLP+ZtJzQO3LUbgHDboXr5sLpz0OrQeCF8P0n8OqZcH83+OofkLEq3DhFRESqmqJK2Nxt4cYhIiIiIiJSCpLS+nclElmTNferDWHHIlLRKAkrIlKGzIzTejfj4+sGcWTremzPK+CWUbM595lvWbVle9jhQWw8dD4Zzn8PrpoCR14FNevA1pXw1Z1wfxd45aygOrZAnUREREQOmiphRURERESkikjqOCgNi+zImvu11jkTKYGSsCIi5aBp7Zq8fPHh3H5CJ2rERRi3aCPD7xvD61NWhF8VW6R+Ozj27/C7+XDKU3Bo/6A6duFH0erYrvDF32Hz8rAjFRERqbwSlIQVEREREZHKL7nz0EOBuKz5Y78POxaRikpJWBGRchKJGCP7t+LDawbSs0VttuXk84f/zeTC5yezNmNH2OH9KK4GdDsDLvgQrpwcrY6tC9tWw5h74IHu8OIImP0W5OeEHa2IiEjlUtSOOEdLJYmIiIiISOWU3HloI8/PbZg1b8zssGMRqciUhBURKWetGyTzv8v7cePwDsTHRPhywQaOue9r3vxuZcWpii3SoH1QHXv9fDjtWWg9BHBY8iX87wL4dwf4+CZYNzfsSEVERCqHne2IlYQVEREREZHKJ6X7salZc79am71wQtOwYxGp6JSEFREJQUzEuGJIG96/ZgDdmtVi6458rn9jBhe/MIV1WytQVWyR2ATociqc9w5cMx0G/R5SmsD2dJj0KDx2JDx1FEx5FnZkhB2tiIhIxaU1YUVEREREpBIws+fNbFTxudTeJ8QXbN/WtxTOXdfMHjKzBWa23cx+MLMHzazWwZ5bpCJRElZEJETtG6Xw1hX9uOGY9sTFGKPnr+foe7/mfxWxKrZI3VZw1J/h2tlw9uvQ4f8gEgurvoP3r4N/pcFbl8LSMVBYGHa0AJiZm9mIsOMQERHZ2Y5YlbBSzsysgZk9Fr3BlWNma83sEzPrXwbXOiv6/WtUaZ9bRERERMJRe8DZlr9t49Dt30/8ohROd0h03AB0AUYCw4FnSuHcIhWGkrAiIiGLjYlw1VHteP/qgXRtGlTF3vDGDC58fjJrMraHHd7uxcRC+2PhrJfhd/PgmDugfhrkb4eZr8ELJ8CD3eHLu2DzsjIJYeTIkZgZZkZcXByNGjXCzD4zswvN7IA+48wszsz+YWazzCzLzFab2Ytmdkhpxy8iItVQgiphJTRvAj2B84H2wInAV0C90ryImbUE/gWMLc3zioiIiEi4ctcv+0VMYq3R/mPlSH0ze9vMss3sezM7sWhfM7s1ek+tXrG5D8zsSzOLuPtsdz/V3d9z98Xu/gXwJ+AEM4st57cmUmaUhBURqSDSGqfw9m/68YfhaT+uFXvvGF799oeKWxVbJLkh9LsarvwGLh4NvS+AhFTY8gN8fTc80B2e/z+Y/grklG7lz/Dhw1mzZg3Lli3jo48+AvgSeAB4/wC/tCUCvYC/RX+eAqQB75ZSyCIiUp1pTVgJgZnVBgYCN7r7l+6+3N2/dfe73P3d6D4tzOwdM8s0s61m9rqZNYpu6xC9uXZ2sXOeEW0d16nYXAzwMnAbsKQ836OIiIiIlJ3E9v0GRRISJ26b/nF+senbgNeBbsCHwMtmVje67e/AMuBpADO7EugHnO/uu2udVwvY6u75u9kuUukoCSsiUoHExkT4zZC2fHDNAHo0r822nHxuemsWv37mG37YlB12eHtnBs36wAn3w/UL4JSnoPVQwGDZWBh1BfyrPbx9OSz5ulTaFSckJNC4cWOaNm1Kr169cPc7gZOA4whamewSop0XvbnYrtjco2Y238wS3T3D3Y9299fdfYG7TwKuAnqbWYuDDrgaMrMrzWyZme0ws2/M7LA97Dsy2r6w+KiACyWLiBygonbEpfxQksheZEbHCDNL2HVjtIPIO0BdYDBwNNAaeA3A3ecTtIp7NJqsbQY8TpDUnVvsVLcC6919n9rImVmCmaUWDSDlgN+hiIiIiJSJpLT+h0Xia87NnPX5rn/EPO/ur7r7IuBmIBk4DMDdC4BfA8PM7G7gn8CV7v5DSdcws/rALcCTZfU+RMKgJKyISAXUrlEKb17Rjz//siMJsRHGL9rEsfeP4emxSygorOBVsUXiE6HbGXDeKLhuNhx1C9RtDXlZMONVePFEeKAbjP4rbFhYqpeOtjCZQVDFuuu2F/nx6bxYM/slcDFwjrvvLtNdC3BgS6kGWg2Y2ZnAvcBfCCqLZwCfmFnDPRy2FWhSbBxa1nGKiJSbeLUjlvIXrSYYSdCKeIuZjTezO82sW3SXYUBX4Gx3/87dvwHOAwabWd/oOR4FxgEvAc8Dk4GHiq5hZgOAi4BL9iO0m4CMYmPlgb5HERERESkDMbGpxMSuyZw9emMJW2cW/eLuWQT3cxoWm1tC8CDfjcC77v5KSZeIPoz3ATAXuL30ghcJn5KwIiIVVEzEuHhgaz65dhCHt6rL9rwC7vhgHqc+NoEFa7eFHd7+qdUMBt0AV0+FCz+F3iMhoRZkrICx/4ZH+sKTQ+GbJyCrpO90B2Q+0HI32y4jSO49CDwD3O7u35W0o5nVAP4BvOruW0sruGrkd8BT7v5ctFLmciAbuHAPx7i7ry021u1uR1XQiEilU5SEzd8OhQXhxiLViru/CRxCsBbsx8AQYKqZjQQ6AivcfUWx/ecSPIDWsdhpLiRoN9cLGFm0HpiZpQD/AS5x9/35MncXwcNuRaPZAbw1ERERESkLMXGpuBdkzf16xW72yNvltfPznNMgoABoWdKyYdHvkR8D24CT3X3Xc4pUakrCiohUcC3rJ/HqJUdw58ldSUmIZfqKLfzfQ2O597OF5ORXspu3ZtDicDjhAbhhAZz2HLQ7FiwGVk+Fj/4A/06DV86E2W9B3vafHr9tHYx/AD66Eab+h3jb4xIRRvDl72fcfTNBpcYVwGLg7pLDtTiCtS0suq/sBzOLB3oDnxfNRdf9+Bw4cg+HJpvZcjNbEV2brvMe9lUFjYhULkXtiEHrwkq5c/cd7v6Zu//N3fsRVLT+ZT9O0R1Iio4mxebbEDz89p6Z5ZtZPkEl7YnR1212E0+Ou28tGgQ330REREQkZEkdB7UEIhQWHHALn2h3tFMIHv5rQdBuuPj2VOBTIBc40d21HJVUOT978kBERCqeSMQ4+/AWHNWhIX8eNZvP563jwdHf88HM1dx9ajf6tqy795NUNHE1ocspwchcD7P+BzNfgzXTYeHHwYhPgU4nBm2N83Ph9V9DQV6QtC3M4+6mNbgpq8furtARWLqHCIqexGtCcCPxJzf9iiVgDwWOUhXsAakPxAC7VrKuAzrs5pgFBFU2MwkqYm4AJphZZ3cvKcF6F0G74yIpKBErIhVZbAJEYqEwP2hJXKNW2BFJ9TYXGAHMA5qbWfOialgz6wTUju6DmdUlSNr+neD708tm1svdtxN0IOm6y7nvIPhc/i2wu+oJEREREalgUg87uSleWJ+CvC3AoWbWY5ddNu3tHGbWDHgMuNHdx5nZBcD7ZvaRu08qloBNJFg7tqjDGcCG6JqyIpWekrAiIpVI41o1eOq83nw4ay23vTuHxRuyOP3xiZxzeAv+MLwDtWrGhR3igUluCEf+Jhjr5wfJ2FlvBO2Kp78cjOKFrV4IQK2YHK5uPu9npzOzowhuBN5X0uXMrB/BehQnELQafphgjbSi7UUJ2HbAUHff65dLKR3uPhGYWPTazCYQ3Bi+jF2emIzunwPkFNu/HKIUETkIZkE17I4MyFElrJQPM6sHvAE8S/Cg0zagD/AH4B2CLhWzCBKr1xLcK3gU+Nrdp0RP8zhBMvUOIAGYBvwLuDJatTB7l2tuAXD3n8yLiIiISMXV8o8fPJfad8TI2JT6zy//x/9BUMU6bZfdntnTOSy4OfM88C3BPTfc/RMzewx4KZrU7QUcHj1k0S6naAUsO+A3IVKBWHQJFykm+sRFRkZGBqmpqXvdX0QkDBnZedz54TxemxIUFjRMSeAvJ3ZmeJfGVSMRVVgIKyYFCdmZr0Ne9m53Xf/rr8mr2YB169bRu3fvPxG0qP0KGOHuBWbmBOtKjIquNTEdGOXu15tZV2Ay8Gt3/180Afs/gi+D/8dPqzjT3T23LN5uVRRtR5wNnObuo4rNvwDUdveT9vE8bwD57v6rfdhXn+EiUvHd2xm2roRLvoSmvcKOpqKpAl9iKh4zSwBuB44haB0cR5BQfQO40923m1kL4CFgGFBIsDbX1e6+zszOI0jK9nT376PnPAwYB5zk7h+VcM3nCT7vR+xHnKlAxvRpU0lJ0TLvIlJxZHvS3ncSurRtos9xkUqs5R8/iCNoDVwkftndv9QarSIHQUnYEugGrohUJhMXb+JPb89iycZgiYZfdGzEX07qTNPaNUOOrBR9+zR8eP1uN3d+NJOFmyPUqVOHDRs2fA68ArwQXX+UXZKwzxJUfvSNVlFiZr8D/gR0I7gpubs2xkPd/avSelvVgZl9A3zr7ldHX0eAH4CH3b3EtXh3OT4GmAN86O6/24f99RkuIhXfw4fBxgVw/vvQamDY0VQ0unlbjSkJKyIVlZKw+0ZJWJHKr+UfP3gOGAk8v+zuX14QcjgilV4k7ABEROTgHNmmHh/+diDXHNWWuBjj83nrOPrer3l67BLyCwrDDq907OkGdc26zFm9nby8PNavX4+7H+3uzxUlYAHc3YoqMd39QnfvVpSAjc7d6+713H2Vuy+L7l/S+Krs3mSVdS9wiZmdb2YdCdYDSQKeAzCzF83srqKdzexWMzvGzFqbWS/gJYJ1eZ8OIXYRkbIRH72Rm6t2xCIiIiIiUnFEE6/xSsCKlA4lYUVEqoAacTH87pg0PrxmIH1b1iE7t4A7PpjHiQ+PZ/qKLWGHd/AapEHX0/lpcUz096P+BLHxYUQl+8DdXwNuAP5K0Aa6BzDc3YvaPLcAmhQ7pA7wFME6sB8CqUA/d59bTiGLiJS9hOTgZ25WuHGIiIiIiIjsQi2IRUqPkrAiIlVIu0YpvHbpkfzj1K7UqhnH3DVbOfnR8fx51Cwytlfy708jHoMhN0Fi/eB1vbZw8pPQ9+Jw45K9cveH3f1Qd09w98Pd/Zti24a4+8hir68rtm9jd/+lu08LJXARkbISX5SEVSWsiIiIiIiISFUVG3YAIiJSuiIR48y+LRjWsRF3fjCPt6at4qVJP/Dx7LX86ZcdGdGjKWaVcJmWmDgYcmMwCgsgEhN2RCIiIgemqB1xjpKwIiIiIiIiIlWVKmFFRKqo+skJ3HtmD1655HDaNEhiY2Yu1702g189NYlF67eFHd7BUQJWREQqs3i1IxYRERERERGp6pSEFRGp4vq1qc9Hvx3E749No0ZchElL0hl+/1ju/mg+2bn5YYcnIiJS/RRVwqodsYiIiIiIiEiVpSSsiEg1EB8b4cqhbfnsusH8omMj8gudx79ezC/+/TUfzlqDu4cdooiISPWRkBL8VBJWREREREREpMpSElZEpBppXjeRp8/vw9Pn9aFZnZqsztjBb16eyrnPfMui9boRLCIiUi52VsKqHbGIiIiIiIhIVaUkrIhINfSLTo34/HeDuWZYO+JjI4xbtJHjHhjDXR/OIzNHLYpFRETKlJKwIiIiIiIiIlWekrAiItVUjbgYfnd0ez67bhDDOjQkr8B5YswSjvrXV4yatkotikVERMpKfLQdcc62cOMQERERERERkTKjJKyISDV3aL0knhnZl2dH9uHQeoms35bDta9N54wnJjJ7VUbY4YmIiFQ9qoQVERERERERqfKUhBUREQCO6tCIT64dxO+PTaNmXAyTl23mhIfHcdNbM9mUmRN2eCIiIlXHziSs1mMXERERERERqaqUhBURkZ1qxMVw5dC2fHHDYE7qcQju8Oq3Kxjyr694euwScvMLww5RRESk8ktIDn6qElZERERERESkyqrwSVgza2pmL5nZJjPbbmazzKzPXo4ZYmZTzSzHzBaZ2chyCldEpEpoUqsmD5zVkzcuP5LOh6SybUc+d3wwj+H3j2H0vHVaL1ZERORgxBclYVUJKyIiIiIiIlJVVegkrJnVAcYDecBxQCfgemDzHo5pBXwAfAn0AO4HnjazY8s4XBGRKqdvy7q8e9UA7j6lK/WT41myMYuLXpjCec9+y4K128IOT0REpHIqSsLmKAkrIiIiIiIiUlXFhh3AXtwIrHD3C4rNLd3LMZcDS939+ujreWY2ALgO+KSkA8wsAfh/9u47Pq6rzv//6zNF0oyae40dWcV2ihPHtqy4pJMlYMgmoWchOCwdlh9fqsMuEBZ2CbB0lg5xQgsLgVBMEkiPm9wSx4mb5N671WYkTTm/P2Ykj2XZVpnRSNb7+XjMY+aee++5n5kUSfO+55zclKbCHtYrInLB8XqMt86eyIIrxvK9p2r5+bIdPF9zlNd8+zneNnsiH7t5MsMLcs/fkYiIiCS0rQkbj0C0FXw52a1HRERERERERNKuX4+EBW4F1pjZ78zssJm9YGbvOc85c4AnOrQ9nmw/m3uAupTH3p4WLCJyoSrM83PPay/hiY9dxy2XjSHu4FfVu7n+a8/ww2e30RyJZbtEERGRgaFtJCxoSmIRERERERGRC1R/D2FLgQ8ANcCrgR8A3zGzd57jnDHAoQ5th4AiMwuc5ZwvA8Upj4t6U7SIyIXs4uH5/PAdM3novVdz+fgiGlqi3PfoZm76+rP8ef1+rRcrIiJyPl4f+PISrxXCioiIiIiIiFyQ+nsI6wHWOec+45x7wTn3Y+AnJKYcThvnXItzrr7tAWihQxGR87i6dDh//tB8/udNVzKmKI99J8N85DcvcNv3l7Nqx/FslyciItK/tU1J3NqU3TpEREREREREJCP6ewh7ANjYoW0TMPEc5xwERndoGw3UO+fCaaxNRGTQ83iMN868iKc/cT0fv3kywRwv6/ec5M0/WsF7HlxD7WGN7hEREemUQlgRERERERGRC1p/D2GXAVM6tE0Gdp3jnBXATR3abk62i4hIBgRyvPzbTRU888nrubNqIl6P8Y+Nh3j1t57j3/+4gcMNzdkuUUREpH/JKUw8t2gSHhEREREREZELUX8PYb8JXG1mnzGzcjO7E3gv8L9tB5jZl83swZRzfgiUmtlXzWyqmX0QeHOyLxERyaBRhXn89+3TePyj1/CqS0YTizt+Vb2b6776DN/4+xYamiPZLlFERKR/0EhYERERERERkQtavw5hnXOrgduBtwEvA58FPuqc+1XKYWNJmZ7YObcDWEBi9Ot64OPAu51zj/dV3SIig135qEJ++s5Z/Pa9V3PVxCGEIzG+81Qt13/tGe5ftoOWaCzjNZiZM7PbMn4hERGRnsgtSDwrhD0n/TwXERERERGRgapfh7AAzrm/OuemOefynHOXOOd+0mH/Qufc9R3annHOXeWcy3XOlTnnFvdlzSIiklBVOpw/fGAuP3z7DEpH5HOsqZUv/GUjN339Wf6wbi+xuOtV/wsXLuS2225LT7EdmNkwM/uumW0xs7CZ7Taz75hZcUYuKCIig0v7SNjBOR3xwoULMTPMDL/fz+jRo7n55psxs3eZWY/+TjUzv5l9xcw2mFmTme03swfNbFy66xcRERERERE5n34fwoqIyMBmZtxy+Vge/3/X8t+3T2NUYS57T4T52P+t57Xffp6/v3IQ53oXxmbIuOTjE8DlwELgFuBnWaxJREQuFDkaCXvLLbdw4MABdu7cyaOPPsoNN9wA8G3gr2bm60GXQWAG8MXk8x3AFODP6apZREREREREpKsUwoqISJ/wez3cWTWRZz95A5++ZSpFeT62HGrgvb9Yy+3fX87y2qOZuOwIM/ujmYXMrMbMbm3bYWafS46QGZ7StsTMnjYzj3PuZefcG5xzf3HObXPOPQX8O/D6Hn4xLCIickpbCNvSmN06sig3N5cxY8Ywfvx4ZsyYwWc+8xmAfwZeQ+Lmp9OY2V1m1mhmFSlt3zezzWYWdM7VOeduds79n3Nui3NuJfBhYKaZTezYn4iIiIiIiEgmKYQVEZE+Fcjx8oHry3j+UzfywevLCPi9vLjnJHf+tJo7f7KStbtOpPNynwf+D7gC+BvwKzMbltz3X8BO4KcAZvYhYC7wTudc/Cz9FQP1zrloOosUEZFBqH064sE7ErYzyZue1pMYxdpx34Oc+nnuM7MFwLuBf3HOhc7SZTHggJOZqVhERERERESkcwphRUQkK4qDfj51y1Se/dT1LJxbgt9rLN92jDf8YDnvWryal/fVpeMyi51zv3HO1QKfAQqA2QDOuRjwduAmM7sP+BrwIefc7s46MrMRwGeBH6ejMBERGeTapyMevCNhz2EzUHKWfe8DxgLfIbFEwL3OubWdHWhmecBXgN845+ozUKeIiIiIiIjIWWk6RRERyapRhXnce+tlvPuaSXz3yVp+v24vT20+zFObD3PLZWP46M0VTB1TBIBzjnW7T3KovpkpYwq70v1LbS+cc01mVg+MSmnbbmafAH4E/NY59+vOOjGzImAJsBG4t6fvVUREpF3uIAxhj9bAoVegaDyJwalnZWc7wDl3wsz+FXgcWA7c12kHZn4Ss2EY8IHelC0iIiIiIiLSEwphRUSkX7hoaJCvvPEK3n99Gd9+Yit/Wr+fx145yGOvHGTBFWN588yL+K+/bWLroVNfVg8fMpdxdX8/V7eRDtuOM2eBuBaIASVm5us41bCZFQKPAQ3A7c65jn3KeSSnev4kMIbEFJP/5pxbdY7j3wR8kcQoqBrg0865v/VBqSIifWcwTUfc2gQPvwe2LGlv+tK4Qr4UuuxsZ1wC7DhHj20/u8cC+SR+RrdLCWAvBm7UKFgRERERERHJBk1HLCIi/cqkEfl8661X8fePXsuCK8YCsOSlA7zz/tXUHDp9tNCx3LHsHT23x9cys7eQWHPuemAiiemGU/cXAX8HWoFbnXPNPb7YIJX8jL8BfAGYQSKEfdzMRp3l+LnAb0hMMXkV8AjwiJld3icFi4j0lcEUwv7tE7D1sdOaxvob+Wzpeoifvgy7md0ITAMe7qyr5M+JTwOvBxqB73XY3xbAVgCvcs4dS9O7EBEREREREekWjYQVEZF+qWJ0If975ww+fEM9//HIBtbuOnnmvITm4URxBc9Xr6MwN3Ff0VVXXTUdOO8XrmZ2EfADEqMsl5rZ3cBfzexR59zKlAA2SGLt2KJkG8CR5Jqycn4fA37inLsfwMzeDywA3kXnU0j+f8BjzrmvJbc/a2Y3Ax8G3t/xYDPLBXJTmro0T7WISNblJP93dWQzLPl4dmvJpGgzvPgbOs4u7DXH+Lwwx9f+kfC4qzl06BCPPfYYwJ+AvwIPduwqOTvFL4DvOOceNbO9wGoz+4tz7vfJAPb3JG76eR3gNbMxydOPO+daM/U2RURERERERDpSCCsiIv3aJWOLeMOMCazddbLzA8zDTa+7g8jRXW0tL5AYRXlWZmbAYmAVyRE0zrnHzewHwC/NbDqJL3CrkqfUduhiErCze+9k8DGzHGAm8OW2Nudc3MyeAOac5bQ5JEbOpnocuO0sx98DfL53lYqIZEHh6MRz0xFY/dPs1pJF93zgbfx8vWPo0KFceeWVAB8BHnDOxTs5/NtAE/AZAOfcBjP7DPAjM1sB+IFbk8e+2OHcG4Bn0v8ORERERERERDqnEFZERPq9spH5Z92X4/VwbE8NhXn+tiZLPv7fOTsAAOnESURBVL+747HOuSEpm6/qZP9HSHz5C4kvaq3jMdItIwAvcKhD+yFg6lnOGXOW48d0ciwkAt7U0LYQ2Nu9MkVEsmDMFXDbD+HEzmxXklnRFlj+HTjLBBI/+v3T/KhkXmrT/akbzjlLef2ujuc7577B6T8H9LNbRERERERE+gWFsCIi0u/NnjSMS8cWseVQA7H4qekMzeDOqompAawMMs65FqClbTsxyFlEZAAwg+lvy3YVfSMSgtU/gdTBreaF0ZfBxT1f211ERERERESkP/NkuwAREZHzMTMWv6uSqknD2tu8HuNtlRP5zGsvyWJlch5HgRgwukP7aODgWc452M3jRUSkv/unL8GMd4LHe6qtZB78y+8TYbSIiIiIiIjIBUgjYUVEZEAYVZjHr99zNbuONXGwrpmyUQWMKMjNdllyDs65VjNbC9wEPAJgZp7k9vfOctqK5P5vpbTdnGwXEZGByJcDr/8W3PDvcHQrFI2FYaXZrkpEREREREQkoxTCnkN9fX22SxARkQ6G+mHoCD/EW6ivbzljf3FxcRHQ4JxzZ54tWfAN4AEzWwOsAj4K5JNc88/MHgT2OefuSR7/beBZM/s4sAR4KzALeG93Lqqf4SIi/VEuDJuWeHmW/0/r57gANDY2ZrsEEZHThFOn1JezMhtn+hkuIiJyiunn4pnMbDywN9t1iIhIjxU755TC9RNm9mHgk8AY4EXgI8656uS+Z4CdzrmFKce/CfgSUALUAJ9yzv2ti9fSz3ARkYFPP8cHKf0cFxEZ8PQzXEREJIVC2E6YmQHjgIZOdheS+KPworPsl97R55t5+owzS59vZnX189UImkFKP8OzTp9xZunzzSx9vpnVnc9XP8cHqfP8HM8G/X+ha/Q5dY0+p/PTZ9Q1/flz0s9wERGRFJqOuBPJXxb2dbYv8TchkPilQnd2pZk+38zTZ5xZ+nwzS5+vnI9+hmeXPuPM0uebWfp8M0ufr3TFuX6OZ4P+ve0afU5do8/p/PQZdY0+JxERkYHDk+0CREREREREREREREREREQuJAphRURERERERERERERERETSSCFs97UAX0g+S/rp8808fcaZpc83s/T5Sm/o35/M02ecWfp8M0ufb2bp85WBSP/edo0+p67R53R++oy6Rp+TiIjIAGFaK11EREREREREREREREREJH00ElZEREREREREREREREREJI0UwoqIiIiIiIiIiIiIiIiIpJFCWBERERERERERERERERGRNFIIKyIiIiIiIiIiIiIiIiKSRgphRURERERERERERERERETSSCGsiIiIiIiIiIiIiIiIiEgaKYQVEREREREREREREREREUkjhbAiIiIiIiIiIiIiIiIiImmkEFZEREREREREREREREREJI0UwoqIiIiIiIiIiIiIiIiIpJFCWBERERERERERERERERGRNFIIKyIiIiIiIiIiIiIiIiKSRgphRURERERERERERERERETSSCGsiIiIiIiIiIiIiIiIiEgaKYQVEREREREREREREREREUkjhbAiIiIiIiIiIiIiIiIiImmkEFZEREREREREREREREREJI0UwoqIiIiIiIiIiIiIiIiIpJFCWBERERERERERERERERGRNFIIKyIiIiIiIiIiIiIiIiKSRgphRURERERERERERERERETSSCGsiIiIiIiIiIiIiIiIiEgaKYQVEREREREREREREREREUkjhbAiIiIiIiIiIiIiIiIiImmkEFZEREREREREREREREREJI0UwoqIiIiIiIiIiIiIiIiIpJFCWBERERERERERERERERGRNFIIKyIiIiIiIiIiIiIiIiKSRgphRURERERERERERERERETSSCGsiIiIiIiIiIiIiIiIiEgaKYQVEREREREREREREREREUkjhbAiIiIiIiIiIiIiIiIiImmkEFZEREREREREREREREREJI0UwoqIiIiIiIiIiIiIiIiIpJFCWBERERERERERERERERGRNFIIKyIiIiIiIiIiIiIiIiKSRgphRURERERERERERERERETSSCGsiIiIiIiIiIiIiIiIiEgaKYQVEREREREREREREREREUkjhbAiIiIiIiIiIiIiIiIiImmkEFZEREREREREREREREREJI0UwoqIiIiIiIiIiIiIiIiIpJFCWBEREREREelTZjbSzH5gZrvNrMXMDprZ42Y2L7nfmdltPeh3p5l9tEPbFDN72swOmVmzmW03sy+ZmT8970ZERERERETkTL5sFyAiIiIiIiKDzsNADvBOYDswGrgJGJ6Ba0WAB4F1wEngSuAnJG5K/kwGriciIiIiIiKCOeeyXYOIiIiIiIgMEmY2BDgBXO+ce7aT/TuBi1OadjnnSsysDPgGcDWQD2wC7nHOPZE87xngutS+nHN2lhq+AVQ6567p7fsRERERERER6YymI+6EJRSZWad/sIuIiEj/pJ/hIiIDQmPycZuZ5XayvzL5fDcwNmW7APgbiRGzVwGPAX8xs4nJ/XcAe4HPJc8b29nFzawcuAU4IwCW7NLPcRERERERuZBoJGwnzKwIqKurq6OoqCjb5YiISPfoS7tBTD/DRUQGhocffpj3vOc9hMNhZsyYwXXXXcdb3/pWrrjiCoPEmrDA7c65R87Vj5m9DPzQOfe95PZO4FvOuW91cuxyYAaQC/wY+IBzLp7O9yW90/Zz/IUXXqSwsDDb5YiISDeVlZXq73EREZEUWhNWREREREREMmrroQZ+tnQHL++rY9yQAO+4+lr279/P888/z8qVK3n00Uf56le/SiwWW+icW9xZH2ZWANwLLCAxytUHBICJnR3fibcAhSTWhP0a8Angq717ZyIiIiIiIiKdy+p0xGZ2rZn9xcz2m5kzs9s67L/DzP5uZseS+6d3oc+FyWNTH82Zeg8iIiIiIiJydstrj7LgO8/z+7V7eWV/PU9tPsxdP1/Fg6v2cfPNN/PZz36W5cuXs3DhQoAvnKOr/wFuBz4DXANMBzYAOV2pwzm3xzm30Tn3G2ARcK+ZeXvx1kRERERERETOKttrwuYD64EPnWP/UuDT3ey3nlNrAI0FLu5pgR2Fo2FCkRDhaJjmaDOtsVYisQiReIRoPEosHkNTPIuIiIiIiEA8HueeP2wgGnPE4om/k9qev/LYFo40tLQfe+mll0Lib0CACNAxIJ0HLHbO/dE5twE4CJR0OKa1k/M64wH8ZP9vYhERERER6SdKFi3xZ7sGubBkdTpi59yjwKMAZmcuGeCc+0VyX0n3u3YHe1tfZ+569C42H9/cpWMNw8zw4MHMMAyPnf21xzyntXvMgwdP+76zPvDg8Zw61uvxJp7t9Oe2117z4vGkvDYPPo+vfX9nr30eX/t26rPX4z2j3efx4TMfXs+pdr/Hf2pfyjFtr1P3e0zfg4iIiIiIDHRbDjbw/l+sZdfxUKf7Y3HHL59cx21XjmHNmjV89atfBfhTcvdO4CYzWwa0OOdOADXAHWb2F8ABX+TMEHUncK2ZPZQ876iZ/QuJUHcD0ALMAr4M/NY5F0nfOxYRERERkYGqZNGS+10ssrBk0ZLFO+9bcHe265ELw4W6JmyBme0i8Qf5OuAzzrlXznawmeUCuSlNhekowuFwzhEnnviKQLokNcxtC2f9Hn/7o33b6z+t3e8989i2Y3K8Oe1tba9Tn3M8Oe3Ht23nenMT254ccryntn3m6/SmARERERERSfjTi/tY9PAGwpHYOY/78pe/zH+8/CQTJkzgPe95D1/60pc+nNz1ceAbwHuAfSRGvH4M+DmwHDgKfAUo6tDl54AfAdtI/I1nQJTE7EqTk9u7gO8B3+zl2xQRERERkQtAcgTswmjdYfzDxi8sWbTkvTvvW6AbNqXXrL9MnWtmDrjdOfdIJ/tKgB3AVc65F8/TzxygAngJKAY+AVwLXOac23uWc+4FPt+xva6ujqKi0/+mb442E3dxAOIunghak2ErgHOJ7fZ9qdsdXsc59RyPx087L+Zi4CDmYu1tcRcn7uLEXGLK49TXbcd1tt3+HD99u/11PNa+HYsn2iLxyGnHReNR4i5ONB49/bU7czsWTxzfdl7qcWc8XPRc/zj7JcMSQW0yrG0LaM947mRfZ8d1fJ3rzSXXl0ueN6/Tdr9HMyKItFm4cCEnT57kkUceSW3WXRKDmJkVAXWd/QwXEblQ1Rxq4FhTa7bLaPfYywdZvHwnAPPKhrP7eIi9J8Jn3Jfq9Rgr77mJkYWp98Pq5/hg1vZz/IUXXqSwMC33RouISB8qKyvVz3ERGbBKFi25P3LiwEL/0LEaCStpc8GNhHXOrQBWtG2b2XJgE/A+4LNnOe3LJO6yblMIdBrY5vny0lOoAInQumNA27a+btvrSDxCJBZpP6411nraca3x5HYs0v66NdZKJJ7YbluzNxKPtLe3Hdv+OtZKa7z11Hmp28lrt9eMoyXWQkus5RzvLHO85iXHm0OeN689rG0LaNvC2tR9eb689rb212dpazsv4Au0tyn0FREREemfWqNx/vOvr/DLlbuzXUqnPnh9GR//pylUbz/GO+9fRdwlpiD2eoxY3PHpW6Z0DGBFRERERESyYud9C+7Onzr/f5s2L12T7VrkwnHBhbAdOeciZvYCUH6OY1pIrA0EdL4+rWSGmeE3f78P+mLxWHsomxrQpm63xFpojZ16bo210hxrJhKLJNrirbREW9rPbY41tx/fEm05fTvl0dbWXouLEY6GCUfDKf/WZo7XvO0B7WnPnbxODW8DvsBp+1O3c72JoLd9nzeAz6NpnqV3SkpK+OhHP8pHP/rR9jYzexF4xDl3b3LbkZjWcAHwahLTG37cOffnvq5XRESkN442tvDBX61j1Y7jmMGkEfn9Zghpfq6PD99Qzj9dNgaAueUjWPKRa/jZ0h28vK+O8UMDvL3qYq6dPDLLlYqIiIiIiJwSbwmdez0VkW664ENYM/MC04C/ZbsWGbi8Hi8BT4CAL5CV68dd/PSANhnatsRaaI42t4e6bdttx7W9DkfDZ7Q1R5tPO6c51tzeb3O0GZecMC7mYjRFmmiKNGX0PXYMewO+wKmgNhnwnivcTQ11g75gp+f5PX4FvQKJ6ec/BXwS+DfgV2Z2sXPueHbLEhER6VxrNM7S2iM0tsTat7/5j63sOxmmINfHt986nZsuGZ3lKs9t8uhCvvKGK7JdhoiIiIiIyKBSsmiJX+vbZk9WQ1gzK+D0EaqTzGw6cNw5t9vMhgETgXHJ/VOSAcpB59zBZB8PAvucc/cktz8HrARqgSEkvmS/GPhpxt+QSIZ4zNMeMPYF5xyt8dbTQtzU8DYcDbcHuOFIuD24PeM52kw4Fk6EuynntR0Tjobb11jui7DXY54zQ15vHgF/oNP21KA3ta1jQJzapqB3QFjsnPsNgJl9BvgIMBt4LKtViYiIdOJoYwvv+8Va1u46cca+SSPy+cldMykfpbUzRUREREREei0xi94Fo2TRkvuBhSWLlmid2yzJ9kjYWcDTKdtt67I+ACwEbgXuT9n/UPL5C8C9ydcTgXjKMUOBnwBjgBPAWmCuc25jGusWuaCZWfv6spnknCMajxKOhQlHwqcC32RA2x7kpoS3bVMxp4a9bdvhWGJfaugbjobb1/SNuzihaIhQNJSx9+QxT5cC27NN1dx+jvfUyN62KZvbXud6c/GYJ2PvoT+LuzhrD63lYNNBpgyb0tNuXmp74ZxrMrN6YFRaChQREUmjLQcb+NcHVrP3RJjCXB+Xjy9u31c6Mp9P3TKV4kD/XtZDRERERERE+l7JoiX+aN3hhb7iUZAIYt+rEbF9L6shrHPuGTj70kXOucXA4vP0cX2H7f8H/L9eFyciGWdm+L1+/F4/RTlFGbtOJB6hJXpqRG8oGuo06G2btvm0sDeWsi/ZRygaOjWNc/LY1KC3L6Zv7nRt3rZ1eX25Z4S4ZxyTXLv3fOv99qewd0fdDj785IfZ3bC7va3oiiKGLx8OgMfjwbkzblbr7Jvpjr9sOKD/vFERERHgmS2H+fCvX6CxJcrFw4P87J2VlI8qyHZZIiIiIiIiMgAceeTLsaKr3/gXX/Go1wOLFcBmR7ZHwoqIZJzf48ef46cgJ3NfXEbikfZAtiXa0j4q97TtyJlB72lTN3cY8ZsaFIejYSLxUz8nm2OJkcC0ZOwtAZDjyWkPdXN9idHRbQFtant7W3J/Z23trzsc19aW48k561TOsXiMDzzxAQ42HTytvX5YPfErE5MhjBw5kgMHDrTvM7MiYFLGPhwREZEM2XG0ifc+uJbWWJzZk4bxo7fPZGh+TrbLEhERERERkQHCxWMz6lf+/raRt93jVQCbPQphRUTSoC3oLczJ3JpssXiMllhLpyNx24Pd1CmZY82EIolj26Z6bju2fWRw7Mx1f1tip5Ld1ngrra2tNNCQsffVxkhOg50S9raFueFomH2N+zo7icaLGjnZfJIbb7yRxYsX87Wvfe0a4CTwn0As44WLiIik2X2PbqI1Fmdu2XAW3z2bHJ8mbBAREREREck083jjZuZ1zg387xRjUZq2rohz+nKe0scUwoqIDBBej5egJ0jQH8zodeIuflogmxrQpga+zbHm9rC3bf9przsc09Z22r5YC3GX+D3A4U6N8O0ODxwJH+Gee+5hx44d/Pa3v/0rUAd8Fo2EFRGRAaZ6+zEef+UQHoN7b71MAayIiIiIiEgfMa//eKB01nDgcLZr6Y3g5DkT8Pr3Z7sOUQgrIiIdeMxD0J/5sBfAOUc0Hj0zpE0Nb6MtbD6+me+v/36nfVjcGFcwjnx/Pg899BAPPfRQccruBzpc74z5jp1zQ9L6pkRERHooHnf89982AfCWyolMHp25GTZEREREREQyqWTREv9AmwY3cmzPId+w8VMZ4CEsLj4mVFO9OttliEJYERHJIjPD7/Xj9/op5OxfNF8/4Xqe2/ccm45tIpYyG4iLO+blzSPfn98X5YqIiGTUX17az/q9deTnePnYzZOzXY6IiIiIiEiPlCxacn+8tXlhyaIli3fet+DubNfTVa1Hd0cCZbP82a6jN/KnzveDDfzplC8QmttKRET6PTPjf2/6X+aOm9ve5qKO0rpSvvPm72SxMhERkfRojsT46mNbAPjA9WWMLMzNckUiIiIiIiLdV7JoiR9YGKs/ArAwuS19xMUi03NGl76Q7TokQSNhRURkQBiWN4zvv+r7HGw6yJHQESYWTaQ4t/j8J4qIiPRDtYcb+PjvXmLv8RAAkVic+uYoY4vz+Nf5pVmuTkREREREpGd23rcgUrJoyWJPsGhhtO7w/+39wd0DakpinItnu4TeOrn01y7bNUiCRsKKiMiAMiZ/DNNGTlMAKyIiA9Yr++t4y49Wsn7PSY41tXKsqZX65igA/77gEgI53ixXKCIiIiIi0nM771twtzdYnHPs7//7zWzXMpjkT51fgnl3Z7sOOUUjYUVERERERPrIC7tP8M6fr6K+Ocq08cV8+Y5p5PgS98bm5/oYPySQ5QpFRERERER6b+d9CyLBh/9zf7CianyopnpftusZDFy0dVSodtWqbNchpyiEFREREREROYc9x0P8ef1+IrHezUoVjTnuX7aDptYYsy4eys/vrqQoT8sjiYiIiIjIhSlUU707UDbramDghLBmA3IG2UDpzBzz+FqzXYecTiGsiIiIiIjIWWw6UM+//LSa403p+1t2fvkIfnzXTII5+nNMREREREQubOb1HwxOnjsutHX5/mzXciEzj3d6eNuq1dmuQ06nv/pFREREREQ6kRrATh1TyKySob3uc/yQIHfPKyHPr3VfRURERETkwhfaumJnsKyyChgYIaxzvZsCKYuccy7bNcjpFMKKiIiIiIh0kBrAXnFRMb94VxXFQU0dLCIiIiIi0m0ez2GNhs2c4OQ5pXh927Ndh5xJIayIiIiIiAw6S2uO8rk/vcyJUOfTDDe1xGiNxRXAioiIiIiI9FKopnpHcm3Y/h/CDsQ1YeOxEaHaVQph+yGFsCIiIiIiMqis2nGcdz+4mubIuWeZmj5hCA/cPVsBrIiIiIiISC8ZdihYUTUmVFN9MNu1XEjyp87PA5qzXYd0TiGsiIiIiIgMGuv3nORdixMB7A1TRnLPay/BOjnO4zEmDc/H4+lsr4iIiIiIiHRHaNvqttGw/TuEjccH1LqqLtJyeXj7mrXZrkM6N/CGVYuIiIiIiPTApgP13PXzVTS2RJlTOpwfvH0mk0cXUtHJo2xkgQJYGfDMbKSZ/cDMdptZi5kdNLPHzWxecr8zs9t60O9OM/toh7YpZva0mR0ys2Yz225mXzIzDSUXEREREQDM6z8cLKscle06zslsQIWwmHmccwOr5kFEI2FFREREROSCt+tYE+/42SrqwhFmTBzCT985izy/N9tliWTaw0AO8E5gOzAauAkYnoFrRYAHgXXASeBK4Cckbv7+TAauJyIiIiIDTGjriu3J0bCHs13LhSBYcXWF+XO3ZbsOOTuNhBURERERkQvakYYW3vGzVRxtbOHSsUXcf/ds8nN1P6pc2MxsCHAN8Gnn3NPOuV3OuVXOuS875/5sZjuTh/4xOSJ2Z/K8MjP7U3JEa6OZrTazV6X0+wxwMfDN5HkOwDm33Tl3v3NuffJafwZ+laxBRERERAQAw44GK6pGZruOszFfbjxnxMSBMZuLiw9t2vT8sWyXIWenEFZERERERC5YDc0RFt6/it3HQ0wcFmTxuyopDgyMv6dFeqkx+bjNzHI72V+ZfL4bGJuyXQD8jcSI2auAx4C/mNnE5P47gL3A55Lnje3s4mZWDtwCPHu2As0s18yK2h5AYdffnoiIiIgMRKFtq2tdNFKa7TrOysUO+4dPGJ3tMs4nf8q8fDzecLbrkHPT7d8iIiIiItLnnHNsPFBPcySeyavw9b9v5ZX99YwoyOHBd81mVGFeBq8n0n8456JmtpDElMDvN7N1JALRh5xzLznnjpgZwEnn3MGU89YD61O6+qyZ3Q7cCnzPOXfczGJAQ+p5bcxsOTADyAV+TCKsPZt7gM/35n2KiIiIyABknuPBKXNHhLYsP5rtUjoK1646EiirvIzEjYf9lotHLw3VVK/Odh1ybgphRURERESkT8Xijg/9ah2PvXJGfpMR+TleFt89m5IR+X1yPZH+wjn3sJktITEl8NXAa4BPmdm7nXOLOzvHzAqAe4EFJEa5+oAAMLGz4zvxFhIjWq8EvgZ8AvjqWY79MvCNlO1C+vmXXSIiIiLSe+Hta2qSa8P2uxDWORcLlM7s/7PIOmfZLkHOTyGsiIiIiIj0qf/+2yYee+Ugfq8xbkggo9cqDvj5zGsv4fLxxRm9jkh/5ZxrBv6RfHzRzH4KfAFYfJZT/ge4mUR4WguEgd8DOV283p7ky41m5gV+bGZfd87FOjm2BWhp206OzBURERGRQcDMezz/kmuG98c1Tc3j7de/mAYrrp5q/ryt2a5Dzi+rIayZXQt8EphJ4g7b251zj6TsvwN4f3L/MOAq59yLXej3TcAXgRKgBvi0c+5vaS5fRERERES66cEVO/nZ0h0AfOPN03n9leOyXJHIoLMRuC35OgJ4O+yfByx2zv0R2kfGlnQ4prWT8zrjAfzJ5zNCWBEREREZvEK11VuTo2H7XQjrXLxfh7DgCps2Pbc521XI+WV7JGw+ibVmfg784Sz7lwL/R2Idm/Mys7nAb0isLfNX4E7gETOb4Zx7OR1Fi4iIiIgMZuHWGK3R7q/lunLHMe798ysAfPLVUxTAimSQmQ0Hfkfi7+2XgAZgFvAp4E/Jw3YCN5nZMqDFOXeCxI3Md5jZXwBH4gbnjtOx7QSuNbOHkucdNbN/IRHqbiAxunUWiemGf+uci2TqfYqIiIjIwGUe38n8KfOGNm1ZdiLbtQwU+VPmFeFcY7brkK7JagjrnHsUeBQ6n3bIOfeL5L6SbnT7/wGPOee+ltz+rJndDHyYxKhaERERERHpoV+u3MV//nVjj0LYNm+ZNYEPXl+WxqpEpBONQDXw/4AyEiNS95C4wfm/k8d8nMSarO8B9pEY8foxEsHtchJrdH0FKOrQ9+eAHwHbgFzAgCjwaWBycnsX8D3gmxl4byIiIiJyAQjVrNycHA27Mtu1nMa5nv/Bm2EuFpkaql21Ktt1SNdkeyRsJswh8Udkqsc5Nd3SGcwsl8Qfjm0K01+WiIiIiMjAtrTmKJ/708vEXc/7eN0VY/nS7Zdr7UeRDEuut3pP8nG2Y/4C/KVD207gxg6H/m+HY1YCV3Zo+y3w255XLCIiIiKDkZmnLv+Sa4c0bXruZLZrGRDMOv2LvGTREv/O+xZoBpp+5kIMYccAhzq0HUq2n809wOczVpGIiIiIyAC382gTH/r1OuIO3jDjIu57wzR6EqP6vB1nNRUREREREZHBKlS7alNg0owqErO49A9m/fIP1+DkOZeaP++MtWBLFi25P9Z0YmHJoiWLd9634O5s1Cad65f/ImXBl4HilMdF2S1HRERERKT/aGyJ8p4H11AXjjB9whD+6/bL8Xs9+HrwEBEREREREUllHm9DsKKqONt19HvxWEHTxmcbUptKFi3xx8L1CxMrgrCwZNESf1Zqk05diCNhDwKjO7SNTrZ3KjlNU0vbtqZGExEREZELzZGGFr73VA2HG1rOf3AHO4+FqDncyKjCXH70jpnk+b0ZqFBEREREREQGo9C21Rv71dqw8TMX4cn2dL/5l1w7BOxkx/ad9y2IjH3H15fkjp+6AFisKYn7lwsxhF0B3AR8K6Xt5mS7iIiIiMigE4nF+cAv17Jm14ke95Hj8/Cjd8xkdFFeGisTERERERERAcPq86fMK2rasqw+27V0XHf14k/9+X6cy+p0vy7SPDlUu2pVx/bg5LmjwX1k1B3/cbsC2P4nqyGsmRUA5SlNk8xsOnDcObfbzIYBE4Fxyf1TkqNUDzrnDib7eBDY55y7J3nMt4FnzezjwBLgrcAs4L2Zfj8iIiKSXma2iMSyAd92zn00y+WIDFhffWwza3adoDDXx8f/aTLeHkwLPKd0OOWjCjJQnYiIiIiIiAx2/Wk0rHm87dOllixa4nexyMJ4uAFf0ciFJYuWvLc/hZ0u1jopvG1N1j8z6Vy2R8LOAp5O2f5G8vkBYCFwK3B/yv6Hks9fAO5Nvp4IxNsOcM4tN7M7gS8B/w3UALc5515Oc+0iIiKSQWZWCbwPeCnbtYgMZI+9fICfPL8DgK+96UpuuXxMlisSEREREREROZNhTcHyqoJQbXVjNutwLt4ewu68b0Fk4scefhDcXWRput9g+ezLPbn5Z2RcwSlzR5h5j/d1PdJ1WQ1hnXPPkFwt+Cz7FwOLz9PH9Z20/Q74Xa+KExERkaxJzpbxK+A9wH9kuRyRAWvH0SY++bvEfQzvvbZUAayIiIiIiIj0W6Ftqzf0l9GwqfZ8843vyb/8pm81bnjihSyVEGx85elQx0YXbS3XKNj+rfvzkImIiIhk3v8CS5xzT5zrIDPLNbOitgdQ2DflifR/zZEYH/zVOhpaolSWDOWTr56S7ZJERERERESkHyhZtMSf7RrOxswTyp8yLz+rRcTjrkOLi7eEYtkoJf+Sa4bj9Z0x2jVYUVVsHl9DNmqSrlMIKyIiIv2Kmb0VmAHcc75jk8fUpTz2ZrA0kQHlvkc3s+lAPcPzc/jenTPw92AdWBEREREREbmwlCxacj/Qmnzud0K1q16KR1umZbMGT26w1T90XE5KkyNliuK+5CItZaEty2vPaI/HLgnVrHwlGzVJ1+mbGBEREek3zGwC8G3gX5xzzV045ctAccrjogyWJzJgPLX5EIuX7wTgf950JaOL8rJbkIiIiIiIiGRdyaIl/nhLaGG07hDAwv46ItbMEyq47IZgtq4fC9cfyBldOi6lKY5zfR7Cmlmn18yfMi/fzHPG9MTS/yiEFRERkf5kJjAKWGdmUTOLAtcBH0lue1MPds61OOfq2x6ApmGRQe9wfXP7OrB3zyvhhqmjslyRiIiIiIiI9Ac771sQ8eQGF1tOgNZD2/+0874FkWzX1JlQ7aqXYs0NV2Tr+i27N5x00dahKU3OuVif52mBsspp5st5qWN7PNoyLVS76ox26X8UwoqIiEh/8iQwDZie8lgD/AqY7pzLyvobIv1ZLO5oaonS1BKlsSXKx3+3nmNNrVwytohFr5ma7fJERERERESkH9l534K7vYGinJNLf/n+wKQZV2W7nrOzcLC8MpCNKzvn4i5l+mHnXMc1YvtKXtPmpafNFFcw7aZcM09LluqRbvJluwARERGRNs65BuDl1DYzawKOOede7vwskcHrcEMzt31vGfvrTp+9O8/v4btvm06uz3uWM0VERERERGSw2nnfggj3LTgYnDI3GiibVRnetmZ1tmvqKLxt9fpA2ayrgZVZKcC5eFaum5Q/9ZpReLxHO7bHQnVXNW9fW52NmqT7NBJWRERERGQAcs7xmT9sOCOA9XqML/7z5ZSPKsxSZSIiIiIiIjIQhLYsP+rx5dYmw85+x8zTkj91fl5Wru3x9vkasKlctKUktHXF9tS2wum3+Mw88SyOzJVu0khYERER6decc9dnuwaR/ujhdft4YtNhcrwe/vDBuZSPKgASIazfq3stRURERERE5Pyatiw7kT9l3sZA6ay54e1rlme7nlThbatfDJTOmg30+cjP1OmI+5qZWaCs8oz2WMOxmTljK1ZloSTpIX07IyIiIiIywByoC/OFv7wCwEdvruDy8cXk+b3k+b0KYGVAWLhwIbfddlu2yxAREREREaBpy7J6jz93faB05jXZriWVc845XKRg2k252a6lLwUqrp5uvpwXUtvyp873AJxc+muNgh1A9A2NiIiIiMgA4pzj0w9voKE5ypUThvDea0qzXZKIiIiIiIgMcE1bljV584esCkyacb2ZZXUq3lTN29e+EAvVXdXnF47Hsxd2urivafPSyGlN0dZZ5s/td2v3yrkphBURERER6cfqQhH+7Tcv8LYfr+RtP17J7d9fznNbj5Dj8/D1N12JTyNfZYArKSnhW9/61mltZvaimd2bsu3M7N1m9kczC5lZjZnd2selioiIiIhc0Bo3PNniKx61NG/SjBuGzL+zXwSxyfVPW/Onzvf36YXNshLC5k+dPxbzHDm9FDPn4p6mzUvj2ahJek7f2IiIiIiI9GP//bdN/GX9flZsP8aK7cd4cc9JAD75T1Pa14EVGSQ+D/wfcAXwN+BXZjYsuyWJiIiIiFxYGl58LOrJCTzTcqDmxryJ07zZrgcSo2HjkeaZfXlN83izEkK7aOuE0NYVO1Pb8kpnXuXx563NRj3SO75sFyAiIiIiIp2r3n6M367ZA8C9r7+U4QWJZXCKA36uqRiRzdJEsmGxc+43AGb2GeAjwGzgsaxWJSIiIiJygWnavDQ+ZP6dT7Xs23xdoHTm8vD2ta3ZrMc554JlldH8qfP9Hafpzdw1430ewg6Zf6fh3BmjXQ3rs/ct6aUQVkRERESkH2qJxvjMHzcA8LbZE1k4b1KWKxLpuZoTNfxi4y/YdHwTY/PHUjesDk52u5uX2l4455rMrB4YlcYyRUREREQk6eTSXzszezZv0oxrguWVq0O1q8PZrCdnbMXalgNbq4CV2awjk1oPbpsRjzS/kNoWKKu80uzU30IysCiEFRERERHph374zHa2HWliREEui26Zmu1yRHqs+kA173/i/TjniLkYNSdqiF0ZY7hnOAAej4fEMk+n6Wy9p453fju0xI6IiIiISMYk12N9Llg+e16wvGp9qLa6MVu1nFz6axcomxXLmzjN27x7QyzjF+xkRGrmuU7emwuEatdkNQCXnlMI202HvvY1ogcO4ikoSDzyg3jbXyeevQX5p/YXFOAJBjGPvhsQERERGQyaIzHiZwZK3bL7eIj/fboWgM+9/lKKg53lUSKda442E0/5viDoD/a6z2g8Smus+zOQOef4woovEIvHcCT+u4i5xHcKxy47xtHwUUaOHMmBAwfazzGzIkBDv0VERERE+olQ7aplgdKZVwcrqjaFaqrrslVH7tjJa1r2b6kEVmX6Wubp2/gs/5Jrx+PxHkhtC5bPvtyTm/9Kuq5RsmiJf+d9CzStcR9SCNtNTUuX0bJlS/dOMsOT3xbM5uMtKEy8Liw483VhYeKYwkI8BYV4CwvwFBbiLSjAcnIy86ZEREREJC0+/6eXeWDFrrT1d93kkbz+irFp608ufF9Y8QV+v/X3p7Vdd9F1fPfG72LWsyWNNh/fzMLHFtIUaUpHiad4YOm+pdx4440sXryY17/+9Vx77bXTgP8EMn9nu4iIiIiIdFl4+9qVgdKZs4NT5m4PbVl+NBs1nFz6axcsq7T8qfM9TZuXZnikquvTNWFdpHl8qHbV6eGyc/lNG59tSEf/JYuW3B+tP7KwZNGSxTvvW3B3OvqU81MI200jPvB+oocOEWtsJN7YRLyxkXhTE7Gm5HZDA/GmRHusqQmiUXAucVxj70bqW25ueyDrKSpKPBcW4i1KBrapz4WFiSC37biiIjz5+RqRKyIiIpIh63afSGsAO6Igly/ddnmPgzMZfNYcXHNGAAvw7N5neXzn49wy6ZZu9+mc479W/lf6A9gkw7jnnnvYsWMHr3vd6wCWAJ9FI2FFRERERPqd8Pa1qwKlM2cGJ8/1hrYuP5SNGsyfu9pFWiqB6kxex7m4mZklp2TGzNO7Ka/OIX/qfA8e72mhcrB89iXmy9majv5LFi3xR47uWegbMgZgYcmiJe/ViNi+oRC2m4pu6foXF845XEtLIpBtaDgV2DY0EG9oJN7YcCrMbWgg1thw+uuGxkSoGwol+mtpIdbSQuxoD28yMTs9xG0LaQsL8RQV4i0sSgS4RcWngty2/cXFiRBXXwKKiIiInCEed3zhz4kZgt448yK++M+X97rPHJ8Hr0e/e0nXxOIx7lt1HwBvqHgDn579aQDuf/l+frD+B3x97de5bsJ1BHyBbvW7ZMcSXjzyIgFfgIdvfZgRgRHdOt85xxv/8kb2Nuxtn464jTlj/vj5FAWKeOihh9qaJyafH+jQzxn/MTjnhnSrGBERERER6bXw9rVrA2WVVyaD2P19ff2mzUvjwfLZLtOjYT2+3LB/5MUBIASJUDZT13KxyEzz+tee1ubixaEtyzalo//Dv7u3YtgtH/6t+fxvARYrgO07CmEzyMywvDw8eXn4RnTvy4pULhZLBrnJ4La+PhHoNjSktDUQb6hPPifD3fr6ROBbX4+LRBIjcuvridfXw/4e/L/R4zkV3BYVJYLbtsC2qCgR4hYX4Uk+e4tSXhcWajplERERuWD9ft1e1u+toyDXx6dumUIgx5vtkmSQebjmYbac2EJhTiEfmfGR9rD1XZe/i0dqH+FA0wF+/vLP+dD0D3W5z1AkxDfXfBOA90x7DxMKJ/Sots/P+TwfeOIDxF2cmIvhwUOcOK8OvJrhgeE96lNERNJj9cGSbJcwIFSO2ZntEkRE+pXwttXrg2WV04KT53hDW1fs6evrmy9njYu2ziKDa8NGTh7c5x9x8TigNlPXaBePe5pqTgXKwfKqMvN4t6Wj6+DkOROA2N7/fedbSxYteYcC2L6lEHYAMK8Xb3Ex3uLiHvcRb2khVld3KrxNCW1jDfUpgW0DsYYGYvV17a/jdXWJEDceJ1ZXR6yujp78V2qBAN7CwkRQW1ScGGVbVISnuCgZ5nYS4hYV4y0uwnJzNQpXRERE+qWG5ghffWwLAP92YzmjCvOyXJEMNnUtdXz3he8C8MErP8iwvGHt+/J8eXx81sf5xLOf4P6X7+f28tsZVzCuS/3+dMNPORw+zPiC8dx12V09rq9qbBX/97r/45ebfslfV/2Vhv0NXFt8LV/9zFd73KeIiIiIiGRXaNvqDcGyykuDk+eUhLau2NmX127avDQeKJvlyeRo2Mjh7Q2+whGlmeg7VXDynAl4PHtPa3TxEaHa1b0OYfOnzCty0dax4e1rVwEogO17CmEHCU9uLp5Ro2DUqG6f2zatcqy+PhHg1tUnA9zEI16fDHPr61Je1xOvq2sftQvgwmGi4TDRw4e7XYP5/XiKOwluk23to3I7hLfeoiIsEFCAKyIiIhnzvadqOdrYwqQR+dw9T8tYSt/74fofcrLlJGXFZbxl6lvO2P9PF/8Ts0bPYs2hNXx9zdf5+vVfP2+fexr28MAriRmBPznrk+R6c3tVY/nQcu6dey/3zr23V/2IiIiIiEj/Edq2emOwrHJKsLyqLFRbnZaRm13l8eetikeaZwKrM9G/c84Fy2dnPliIx8aGale1j+gNVlRNMn/u7t52O2T+neairTPC29c+09u+pOcUwsp5pU6r3KMQNxZLjr5NTpdcnwhnY3X1p0bc1idf19WfGolblwh5icdxkQixo0d7th6u398e3p4+8rZtauXi5OjcojOCXE9+UAGuiIjIBehkqJV3P7CGvSfCve7rcEMzAJ993SXk+Dy97k+y73DoMB956iMcCR/JdildcjSc+B35U7M/hd/jP2O/mbFo9iLe/Nc38/ddf+em/7sJzvMrbjgSpjXeStXYKm6ceGMmyhYRERERkQtAaNvqLcGyyvJgedXkUG311r66btPmpfFgWaVnyPw77eTSX7tMXCN1HVgzT9qvkTdxmteTV3DaSF4Xj40O1VSv7G3fLQe2Xuvx5z3X236kd7IawprZtcAngZnAWOB259wjKfsN+ALwHmAIsAz4gHOu5hx93gt8vkPzFufc1HTWLl1nXi/eIUPwDhnS7XOdc8SbmoidrEuMvm0Pbtte158W2KaOwI3V10MsBpEIsWPHiB071v3ivd7EOrgpUyafWgu3k3Vx2wLdwsLEOrg+3ecgMhCZ2WJgiHPutiyXIiIZ8r9P17Jm14m09feqS0Zz49TRaetPsut7L3yPV469ku0yuuXmi29m7ri5Z90/ZdgU7px6J7/c9EsOh7s2M02uN5dFlYt0U6KIiIiIiJxTaNvq2mBF1aRg+exLQrWrNvXVdeOR5jWtB2pmAmsyc4F4e/CaGsimiyc3OCNn1KT22oOT50wwr39fb/sNlM2a4fHlvpSpqZql67KdEOUD64GfA3/oZP+ngI8A7wR2AF8EHjezS51zzefo9xXgVSnb0fSUK33NzPAWFOAtKADGd+tc5xwuFGoPZGN15wpvU0fh1p9aBzcWI3byJLGTJ3u0Dq4nPz8ZzhYlwty25+IivIVFifC27bnDcZ78fMyj0TQi57Jw4UIeeOCBjs0OqHDO1WahJBEZAPadDPPAil0AfPUNV3DpuKJe9ecxo3xUQTpKk35g28lt/GnbnwD4n+v+hwmFE7Jc0fl5zUvpkPMvVfSJWZ/gjoo7iMS79pvtyMBIRgZH9rY8EREREREZBEI11TuCFVUTg2WV00LbVm/oi2s2794QC5TN8mVqNKx5MxyhOXd63bHouNC21dW96TJYXlVmXv/xpi3L0nfnufRYVkNY59yjwKPAGXdXJ0fBfhT4knPuT8m2u4BDwG3AQ+foOuqcO5j+imUgMTMsPx9Pfj7+sWO7da5zDtfcfJbpk1PWwG0LcRvqU6ZVrseFQgDEm5qINzURPXCgJ28AT0FBIpRNjqxNhLgFeAqL8BQWnApyCwqTgW7haed4cnu3dpfIQHDLLbdw//33t2+PHTt2LDAw5o8Ukaz41j+20hqNc3XpMN406yKN8pPTfGfdd4i7ODdOuJFXl7w62+WkldfjpWJoRbbLEBERERGRC1Sopnp3sKJqfLB89hWh2lUv9cU1Pf68tRkdDZshwclzSsyXs+fU9txxeDxdm7bobH1WVI108Vh+uLa6Tz57Ob9sj4Q9l0nAGOCJtgbnXJ2ZVQNzOHcIW2Fm+4FmYAVwj3PurAsZm1kukJpWFfamcBn4zAwLBPAEAjC6B+vgRiLEGhsTUyO3ja5taCBWV5+YVrktxG1oTOxrG63b2EC8vgHX0gLOEW9oIN7Q0PP34fcnwtjCArwFbWFuAZ6CtrC27XV+IrgtKEi0FxS0B7qeYFAjcqVfy83NZcyYMe3bbTfhmNl1wNeAK4HjwAPAfzjnosn9byQxfX05EAJeAP7ZOdfU1peZfR74MImfEb8GPuKca+2DtyUiGVJzqIGH1+0F4NO3TFUAK6dZf2Q9T+15Co95+MiMj2S7HBERERERkQEnVFO9L1hRFQuUzZoR3rZmXaav17R5aSRQOtNjZuacS+to2ExMQdwuHhvVtHXFqvZrxVonhret6fFasAXTbsp18fiU8PY1S9NToKRDfw5h275RP9Sh/VDKvs5UAwuBLSTWmf088LyZXe6cO1uadQ9nriMr0mPm9+MbOhSGDu3R+fGWlkRoW99AvDEZ3jYmt5MhbryxgVhDYyLAbWxMHJ8MceONjUAyDD5+nNjx4z2aTrmNJz+/PaD1FOTjzU++bmvPDyaC2/btfDzBxCjk0x7BgAJd6RNmNh74G7AYuAuYCvyExM0595rZWOA3JKa9/yOJm2+uAVJ/sbopefz1QAlwP3AM+Pc+eAsikiFffXwLcQevvmw0V03s2c9puTA55/jW2m8BcGvZrZQNKctuQdIjWtddRERERCT7QjXVB4NT5kYDpTNnh7evXXX+M3rHkxN4Ia905lVAxkJfM0/aAt78qfP9cOor+2BF1UjDjvamz1jjiTnNO194ttfFSVr15xC2R5JTHLd5KTlydhfwZuBnZznty8A3UrYLgb2ZqVDk/Dy5uXhyc/GNGNGj8108npgKuSEZ1DY2EEuOqo03Niba2kLbxqZTr5tCxNsD3UaIJpZTbptWmUMd74noJjM8gUBKKBs8/Tk/mAxvg4kRuMHEsycYxBMItrd7AoHE/kBQwa7w17/+lYKCU2sxNjU1/Q7YCuwBPpy8A26zmY0DvmJm/0niJh0f8Afn3K7kqR3XqmgF3uWcCwGvmNnngK+Z2Wedc1rUXmSA2H0sRF048XfNruNN/GPjITwGn3z1lCxXJv3N0n1LWXNoDTmeHD40/UPZLueCd5Z13QHKta67iIiIiMjAF9qy/Gj+lHmxYFnlnNC21Ssyea3kaNictI+GTfkOMJ2jYl209arwttWr2xvi8dLerAUbLKucY15vdbpHAkvv9ecQtm1N19FA6oKao4EXu9qJc+6kmW0lMd3k2Y5pAVratjUtnQx05vHgTa4j6+9hH845XGtrSijblHjd1JgIcpuaEgFuY2MipG1sJB5qSozKbQq1B7ftAW48nphiORQiHgrBkfQt2Wl5ee3BrCcYSISzgQCevDwsGEgEuIEAnkBeYprpvEDiuLxEmydw6nV7X3nJ13l5mL+nn6Kkm3OO8Lp1RA8dIndKIkC54YYb+MEPftB+TEVFxUeA/wVWdPjFYxlQAFwErAeeBDaY2ePA34HfO+dSF6xfnwxg26xInj+BxM09ItLPPbHxEO9+8MwlYd40cwLlo7T6hJzinOM7L3wHgLdNfRtj8s818Y6kS8d13QHGjh27I0vliIiIiIhImjVtWXYif8q8VwJllfPD21ZneprcdekeDWuezEVobd9b5k+ZNxSPt66n/QTLZ1+Ox7MjVFMdTl91ki79OYTdQSKIvYlk6GpmRUAV8IOzn3Y6MysAyoBfpL9EkQuXmWHJEbkMH96rvpxzuObmU6FsqENI27adDGjbQ9xw+NS+cJh4+NR+Fw5DMl9zzc3EmpuJnThxnkp6yOvFk5ubCHBzc7G8PCwvF09uXiK4zU1u5+Qmg9tcLCc30ZaXl3idm5M4Nzexz5N36rXl5ODJzUlu55z+0E0h7Vp37WLPBz5I6/bt7W1vGDKEX4waSXn5qftsnHMHzve5OediZnYzMBf4J+DfgP8ysyrnnL787SYz2w5UOueOdWgfAqxzzpVmpTAZ1JxzfP0fWwEYlp9Dni8xa8LIwlw+9k+Ts1ma9EPP7HmGzcc3E/QFefe0d2e7nEGj47ru0P4zWuu6i4iIiIhcIJq2LKvPnzLvhcCkGdeFd6zL2HS54e1rW9M/Gtal/cvZ4OQ5pebL3dm2HY+2TOnpWrDBiqqJYK2hmuqD5z9asiGrIWwyIE0doTrJzKYDx51zu83sW8B/mFkNiVD2i8B+4JGUPp4E/uic+15y+3+Av5AYpTQO+AIQI7H2n4hkgZklAsxAAHo4xXJH7cFuW1AbSgSz8VAoGdiGk9uJ1/HmMC4UJt7cTDwcwoWbT7WHm0+1N7cQb24+LeQlFkuM3g2FiKWl+q4zv//MYLb94cfjT9luOzb1OfWRc+o1Pt+pdl9be7LN50s8/H7wJdt9px74/JjPe2rb78e8Xszrzdjn4GIxdr/nPUT27T+tvfTkSW6Ndzoz8CbgDR1+6ZoHNJCcbj7ZvgxYlpyieBdwO6emp7/SzALOuba7yK4GGklMcyynKwE6+xcgFxjft6WIJDyx6TCbDtSTn+PlqY9fx5BgTrZLkn7KOcePXvoRAG+d+laG5A3JbkGDnNZ1FxERERG58DRtWdZUMO2mlYHSWTc071j7TKamzfXkBF4MlFVOJ3GjZq+lcwridvHYiKatK7YD5F96XaGZp7En3eRPmVdEPD4qtG31mVOASb+R7ZGws4CnU7bbvvh+AFgIfBXIB34MDAGWArc455pTzikDUlOdi0j8UT4cOJI852rnXPrmPhWRrDst2B02LO39O+dwkUgiyG1uwTUnn1uaiYebE8/NzbiWZGjbtq+5JdHW0oxraU0Exa0tiddt7a0RXEtLYl+ktX2fa2nBRSKn1xGJJNqams5SaT9ilgh3fb5ToWxqQNvW7vOC15ds82KelP0eD7Qd7/Umtn1eYseOE9l9ZvbpAabX1xM7eRLvkCGpu74PfBT4rpl9D5hC4qacbzjn4mZWReIL2r8Dh0nMsjCSRHjbJgf4mZl9icQXuF8Avqf1YE8xs1tTNl9tZqlTp3hJfMY7+7QoERL/D//uUzUAvGNOiQJYOafl+5fzyrFXyPPmcdeld2W7nEGl47rur3nNawA+iNZ1FxERERG54DRueLKlcPotz+dNmnlT/tT5TzVtXpr2382bNi9tDpTNyk13vwBmnl4Hx/lT5+eRuGEUANcSurQna8HmT53viUeaZ4S3r32mtzVJZmU1hHXOPcPpdyx33O+AzyUfZzumpMP2W9NUnogMYmaG5eRATg7e4r67rovHE8FrSwuutbX9EW9tTYS3ra241kRYm7q/LaxtO5Zo9NQxkSgu0tphO3l823HtzxGIRBLHRFMekWR7cpvORp86lzw3Ql+uAO8FIocPnxbCOuf2mdlrSUxluJ7EVIY/A76UPKQeuJZEUFtEYhTsx51zj6Z0/SRQAzxHYkTnb4B7M/dOBqRHks+OxA1UqSIkAtiP92E9IgA8u/UIL+2tI8/v4d3XTMp2OdKPpY6CfdOUNzE80LslGOTsYvX11D3yCM2bt+AfM5ri5uYz1nXPz8/n97///SVoXXcRERERkQtSw4uPRfOnzn8q3tp8Y97EaU83796Q9okHzWx9oKzyyvC21et739ep4DUdo2JdpOXK8PY1qwCC5ZUBzHq0jmu8NXytJyfwXG/rkczL9khYERFJYR4PlpsLuRm5YSttXDyeCHpjsVPBbMr2GftiMVw0lgh543FcLJZsi6a8jkE8hovFIRZNHB+PQSxOy84dnPzVrzutxXJy8I8bd2aNzj0LzO60fuc2Abec9f05tzBl8/Pd+WwGE+ecB8DMdpBYE/ZolksSSY6CrQXgX6ouZkRB//7/qWTX6oOreeHwC+R4clh42cJsl3PBatm2jV1vfwexkyfBk1if+T3xOL8eN/a0dd27Quu6i4iIiIgMbE2bl8aHzL/zyZb9W24omHbTssYNT7aks/9Q7epwYNKMvPT0Zi6ta8wmunIAzrkrw9tWd3st2EDpzJnm9b2QiZHEkn4KYUVEpNvM44GcnLNPZZBmzjmaX3iR5i1bIJZyg5wZQ976VrwpUxlK33POaaih9Bsrth1j7a4T5Pg8vO/a0myXI/1c2yjY2ytuZ1RwVJaruXDt/8y/E6uvT8yakfw57gHevP8A8aYmPPn5qYdrXXcRERERkQvcyaW/dmb2dN6kGdcVXHbDqsZXng6d/6yu8waLN6RjNKzl5DXmXnRZEVB33oPPIzhlbrn5c7cB5E+d7yexjEr3+qi4usLMczRUU93reqRvKIQVEZF+z8yY8OMfsf9Tn6Zp+fJEo8/HkDe9kdGf0Gy3/YGZ3URiDdhRJL5bb+ece1dWipIB63hTK+/7xRoO1DWf9ZhrJ4/kv2+fdkZ72yjYt1ZOYFRRmm58lYyLuziLnlvES0df6rNrOufY37Qfn8fHv17+r3123cEmsn8/zevP/N7DgFznaHjmGYoXLEjdpXXdRUREREQGgeTNlc8Eyirn51963fqmjc82pKvvxleeDgXKZgV620/k6K79/qHjLwbqer0mbCw6rGnL8loAF2mZ0bx97arunB6cPHc08VheaNvqml7VIX1KIayIiAwIvhEjmPjzn9G6dx/Rw4fImTQJ39Ch2S5LADP7PIn129cAB6BPlwWWC9DiZTtYvfPEOY/5dfVu3jDjImZefOr/Axv21rFi+zF8HuP915VlukxJo+f2PsejOx89/4EZ8MaKNzK2YGxWrj0YxEPnvqG9436t6y4iIiIiMriEt61eGpg0Y07+lHmbm7YsO/eXAd3g8eVuCJZVTgttW72hp31EjuwKe/OH9foO7/wp8/IxawbInzrfkzotcZfOnzo/z0Vby8Lb1yzvbS3StxTCiojIgJJz0XhyLhqf7TLkdO8HFjrnfpHtQmTga47E+GX1bgA++7pLTwtZ2/zk+e0seekAP1+647T9P1u6HYDXXTGWcUN6fcOr9KFfbEz87+ONk9/IHeV39Nl1fR4fFUMr+ux6g1HOxRfjHT6c2LFjne7Pn33m8u1a111EREREZHAJ71i3IlhWWRWcMndbaMvyo+nos2nLsqZA2az88x95ds45FyyfnXwd7/HKbC4WuSxUu2oVgIu0VJo/d3V3zo+3hK4O71j3TE+vL9mjEFZERER6KwfQnXiSFo+8sI/jTa2MHxLgnXMuxuf1nHHMv91YzpKXDvDoywfYeyLERUODHKpv5q8vHQDgX+drLdiBZPPxzaw6uAqveXnfFe9jTP6YbJckaWR+P6M+8QkO3HPPGfuGvOlN5Fx8cRaqEhERERGR/ia0bXV1sKxyVnDyXG9o6/JD6ejTk5v/Sm9Hw6bTkPl3msNZaPPSLi+VEiidOc+TG1yZybokc878VktERESke34K3JntImTgc87x82U7AFg4t6TTABZg6pgi5pUPJ+7ggeU7AXhwxU6iccfskmFMu6i4r0qWNGgbBftPF/+TAtgLVOHNr8JycxMbPh/+iy5i1KJPM+ZeDVQVEREREemukkVL/NmuIVNC21avIR4dF5w8d1w6+mva+GyDw/VyNGzPR8ACBMurJpvXXwPQenDbDBdp6fIo2GBZ5TTz5dY2bV7a3JsaJHsUwoqIiEhv5QEfM7Nnzey7ZvaN1Ee2i5OB4/mao2w91Eh+jpe3zJ5wzmP/df4kAB5atYejjS38KjmF8buS7TIwHA0f5dEdiaU8337p27NcjWRK3cMP41payCkrY+qGlyh/4h8MX7gQ83qzXZqIiIiIyIBSsmjJ/UBr8vmCFKpd9QLx6IhgRdXEdPTn8eVuDJbPvry3/Zh5uryG6+nckKYty06YmTkX8zfv3hDrylnBssqL8Xib0zUqWLJDIayIiIj01hXAi0AcuBy4KuUxPWtVyYDzs6WJUbBvmjWBorxz39h7/eRRlI7Mp6Elyr8uXs3JUISJw4LcfOnovihV0uShzQ8RiUe4cuSVXDHyimyXIxngYjGO/+KXAAy76y7MenUTuYiIiIjIoFWyaInfxWMLI8f2Ejmxf+HwV384kO2aMiVUu+ol4vGCYEVVr++0btqyrN65eEE66uqu/EuvK8Q8TQB5pTOv8gaLX+jKecGKqmLn4iNCNStrMluhZJpCWBEREekV59wN53jcmO36ZGCoPdzAs1uPYAZ3zys57/Eej3H3vMTfYuv31gGJKYy9HgU8A0VztJn/2/J/ALzj0ndkuRrJlIYnnySydy/eIUMo/udbs12OiIiIiMiAtfO+BRHzeBf7h1+Eb8iYxeHtayYHK6oqgxVVV9gFeLdjaNvqjcTjOcHyqrLe9mUe76ZgWeWlPTo3OQK2J9MSu9bwJaGala8kN3MaNzzZcr5z8qfO97hY9Krw9rVru3s96X982S5ARERERDKvNRrnnj9sYOexpmyX0qnDDYnlTW6+ZDQXD+/aci1vmDGe/3l8C3XhCIW5Pt5cee4pjKV/WbJ9CSdaTjAufxw3Tbwp2+XIOcTq6jjwH/9B9Oixbp/bumcPAEPe+hY8eXnpLk1EREREZFDZed+Cu0sWLXnvrq+8PsJXXg9A/tT5eYGy2TOCk+cYZsdCW5bvyHKZaRPatnpLsOLqimB51eRQbfXWHvdTU10XKJt1STpr645gRdUVHl/uhq4cG28JXevJDT6X6ZqkbyiEFRERkV4xs6eBs66LodGw/cOSDft5eN3ebJdxXu+9trTLxwZzfNw9r4RvPVHDO+ZcTEGufrUdSH675bcAvG3q2/B59M+uPzv+q1/R8I8neny+BQIMvfPONFYkIiIiIjJ47bxvQSR1u2nz0mZgLUBwytwRwfKqmQB4vbtCW5Yf7fsK0ytUs7ImWFE1KVhWeWlo2+qNPe3H48vdEiidNTW8fc3m7pzXkxGwAMHy2ZeY15+4VjweaKpZdt674gNlsyo9OYF1TZuXxntyTel/9G2HiIiI9NaLHbb9JNaCvRx4oK+Lkc79ZlViNNpbKydww9RRWa6mc2OL87jioiHdOucjN1ZwTcUIpk8YmpmiJCNeOfYKm45vwu/x88/l/5ztcuQcXDxO3e8fBmD4+95HYNrl3e4jp7QM/6j++f8dEREREZELSTJ0PQoQnDJ3UrCssgTA/LkvJ8PaASlUU70jWFE1MVhRdUWopvqlnvTRtGXZiUDZrCk9raFtWuIuc66oacuyTcGKqy/D4zlv8Bssr5psHu+hpi3L6ntao/Q/CmFFRESkV5xz/6+zdjO7Fyjo22qkM9uONLJqx3E8Bv/fqyoYWxzIdklp4/EYMy8elu0ypJse3poI9V518asYmqcAvT9rWracyP79eIqKGPGB92tKYRERERGRASI5LfEOM7NA+expwYqqXLDWcG31S8657gWK/UCopnp3sKIqFiyvmh6qrX6xJ32Yx7s1WFY5JbRt9ZYun9Pd8BXIv+TaIZjVA7h4tDC8bU3duY4PVlSNAZcTqlnV7SmXSxYt8XccHS39hyfbBYiIiMgF65fAu7JdhMBvVydGwd4wZdQFFcDKwBSKhPjbjr8B8MaKN2a5Gjmfk7/7HQDFr3+9AthBzMyeMbNvpWwHzexhM6s3M2dmQ7JXnYiIiIici3POhWqqXwrVVK92scimQNnsGcGyylnB8qqybNfWXaGa6n0YBwOlM2f28PzjDtejO4G7My2xizRPDtWu2hSsuLrCPN5zBqvB8soA8fikUO2ql7tbU8miJfcDrcln6YcUwoqIiEimzAEG7FQ3F4rWaJyH1ybWgn3r7IlZrkYEHt/5OE2RJiYWTqRyTGW2y5FziB49SsNTTwEw5M1vynI1kklmttjMHunQ9kYzazazj3dyyjuBa4C5wFigrsO5P0yGsx/NUMkiIiIi0gPh7WtbQ7XVa0PbVq8xf87JYHnVzGBZ5azg5Lmjs11bV4Vqqg+aP3dXsKyyqifne3ICNcHyqsldPb6na8ICEI8NC9VUHz/bbjMzF4teHdq2ekV3uy5ZtMQfC9UtjNYdBlhYsmiJv8d1XoD6y+ehEFZERER6xcz+0OHxRzNbCdwP/KibfX3AzF5KjqypN7MVZvaazFQ+OPxj4yGONbUyuiiXG6aMzHY5Ivy+5vcA3FFxB2Y9/1tWMq/ukUcgGiXviivIm9LjpZNkADKzdwO/Aj7gnPt6J4eUAZuccy875w6mTmdnZrcDVwP7+6ZaEREREemJpk3PH2sPZM3yguVVM4Pls2cXXHZDMNu1nU9oy/Kj5svZGiyrnNPdc5s2PX/MuVjG1jUKTp4zzbz+V4LlVWV4/TvPdWxe6ax5mGdZT65z6HefnxlvbnjYVzwKYLGmJD7l4k/+aXG04Vi/GCGsEFZERER6q67D4zjwDPBa59wXutnXXmARMBOYBTwF/MnMLktbtYPMQ6t3A/CmmRPwefWrn2RXzYkaXjryEj7z8c/l/5ztcuQcnHOc/F0iMB/yJk0bPZiY2aeA7wJvdc6d8aWFmT0DfBy4Njna9ZmUfeOT5/4LoC+BRERERAaIpi3LdoVqq9eGt61eHW9pKk2Mjp1z1ZD5d/bbO2ebtiw7Yb6cVwJllfO7e675cmq7OhrWzOPMzNPltWHjsUDTlmVNzsVGhrYuP3S2w4Lls68wj2dLePva1i6WDUD+lHlDA5NmXOfNK3xp34/f90YgZ+d9C+7uTh8XmmB5VUHiJoKqmfmXXDs72nD0nd78IdAPRgj7snlxERERGficc2n7Rc8595cOTf9uZh8gMaLmlXRdZ7DYczzE8zVHMYO3VE7IdjkiPFzzMADXT7ieEYERWa5GziW0ajWtu3bhCQYpfu1rs12O9BEz+wrwQeB1zrknz3LYHcB9wOXJ163Jcz3AL4CvOede6cpIdzPLBXJTmgp7Xr2IiIiI9FZyhpOXAQqn3+JrPbxjZrCiysBOhmpW1mS5vDM0bVlWX3DZDesCpTOvb96x7tnUGVrOJbRl+dFA2azyrhxr/rwTeaUzu7SObP4l1wzHPCeCFVUTzZuz+2zHBSuqJoE1hWqqj3Sl3zaB0lmXYeYL71j3bFvbYBwBmz91/lgXjYwDwMyZzx8KbVm2ru2ff8miJYuBhfSDEcIKYUVERCQtzGwmcEly8xXn3Au97M8LvAnIBzpdG0Nf3sLv1uxhyYYDne47WJdYknd++QgmDOv3swlJP+Wc43svfo+Nxzb2uq91h9YB8MbJGlnZ35383e8AKHrd6/Dk52e5GukjrwH+GbjJOffU2Q5yzh03sxDQ6pw7mLLr00AU+E43rnkP8PmeFCsiIiIimdXw4mNRYA1A/iXXDgmWV83ExQ2vf39o6/J+s/RE4ytPhwqm3bQib9LM683smYs//VdfV4I383i3Bcsqy0PbVtee67jWg7X7c8aUT463hs57l6GLtJSFaletCpZVVoW2VVd3dkz+lHlDcW5IqHZVl783y5863x9vCc01f97mc42uvVAFp8wtJxYrbm/weA6HaqvXnu34nfctuLtk0ZL3ZjuABYWwIiIi0ktmNgp4CLgeOJlsHmJmT5OYyrBbd/WZ2TQSoWse0Ajc7pw7W/ozqL+8jccdn//zK4RaY+c87u1XX9xHFcmFaP2R9fz4pR+nrb8JhROYM67by/ZIH4o1NtLwj38Amop4kHkJGAF8wcxWOecau3pi8kas/w+Y0dXRB0lfBr6Rsl1IYmkCEREREelHmjY9dxJYCxCcPGdCsLxqJoYnZ1Rp7cllvzmR3eqgccOTLYXTb3l+1Fu+9Khz8VeXLFqy+HxT9IZqqo8EymaVAecMYSMn9rcGymblnK8GM7NA6SwLVlSNBw52dkz+1PmeeLTlivC2Nc92tr8zwclzSlw0Mq555wvPdfN37QEpf+p8j4vHLycea59G2Ly+7U211ef859RRfwhgQSGsiIiI9N53SXxpeplzbhOAmV0KPEBiNMzbutnfFmA6UAy8EXjAzK47SxA7qL+83XcyTKg1Ro7Xw3/dfjmdTf04vCCH6yePzEJ1cqF4dMejAFSNreL1pa/vVV9mxqzRs/CY1ifuzxqffBLX0kJOSQl5l1+e7XIGHTPzAXcCjzvn+vIu930kfu4+DTxmZq9xzjV08dxrgFHA7pSfRV7g62b2UedcSWcnOedagJa27a5MYSwiIiIi2RXaumIPsGf8e3/8sMPdMfad3/xL3fLf3mte34tNm5fGs1XX8Fv+zaJNJ1/dvPNF8i6+cmFXRkKaN2dHsLyqLFRbve18/Z9vTdhAWeU08+Wsj0eap4e3rVnZ2THxltC1ueOndjmADZTOvNo8viPh7WuWd/WcgSZ/6vw8F49dTjye+HzNnItFXu7uWrn9lUJYERER6a1bgFe1BbAAzrmNZvYh4O/d7cw518qpuxDXmlklidE17+vk2EH95W3tkcQgpUkj8nnTLK35KukXjUd5fOfjANx16V1ce9G1Wa5I+kLdkiUAFC1YMOj+v9ofOOeiZvZDTk3x35fX3mVm13EqiL2li0HsL4AnOrQ9nmy/P81lioiIiEiWlSxa4vcPG39HtP4oDns95rkvHm2tCpZVRvB4G0M1Kzf3dU0771sQKVm0ZDEjJy1sfOWZpS2713u4b8E5zwltXX4oMGlGCXDeELYL8lw8XmxYpzdSBspmVXoChWtPLv31eUezBiuqhrlIy+WevIJVTZuXNqehtn4jOGXuCGKxU1O2eb0t4ZqVay/UUb5ZvQXdzK41s7+Y2X4zc2Z2W4f9Zmb/aWYHzCxsZk+YWUUX+v2Qme00s2Yzqzaz2Rl7EyIiIuIBOruzMEJ6ftfwcPq6r5JUeygRwpaPKshyJXKhWn1wNceaj1GcW8ycsZpCeDCIHj9O07LETdZFC879hYVk1CoSs0L0OefcHhJLDIwCHjezoi6cc8w593Lqg8TvAQedc1syW7GIiIiI9LXkCNPFvqIR+IeNXxyuWbHCkstZONzoQOnM2YGyWZXByXP69I7xnfctuNtXMDQntPGZt8aaTr6j4LIbguc7x3z+XcHJc0rPecx5RsEGp8wdAXaMWKQktG31jjP2l1VOMa//YNPGZ897g2OwrHKai8UuCu988bkLIYANTp5TEiyvmtn2AApDtdVr2x9blr98oQawkP2RsPnAeuDnwB862f8p4CPAO4EdwBdJ/BF4qXOu03/5zOwtJKYlfD9QDXw0ec4U59zhtL8DEREReQr4tpm9zTm3H8DMxgPfBJ7sTkdm9mXgUWA3iamF7yTxRfCr01nwhaL2sEJYyay2qYhvvvhm/F7/eY6WC0HD449DLEbepZeSWzop2+UMZt8HvmFmE0isvdWUutM591ImL+6c22tm15MYEfs4uhlKRERERDrYed+Cu9un/L3PAWyA5Jqezl3pXNznnLskWD57LID587Ym15bNdF0R7luwLzh57l9i4fq351963W/OFX6GaqoPBspmXQ1sP1e/zsXPPk1QLFqK17ed+Jlr7wQnzx0HeJPTOJ9VoHRmDs7NMZ9/Y7im+si5ju2vzMwCk+dcRizW/veD+XL2NG19fmcWy8qqrIawzrlHSXzResY0V5Zo+CjwJefcn5JtdwGHgNuAh87S7ceAnzjn7k+e835gAfAu4L50vwcRERHhw8CfgZ1m1vYL5QTgZeDt3exrFPAgMBaoA14CXu2c+0eaar2g1BxO/A2hEFYyoTXWyhO7ErOLvnbSa7NcjfSV1KmIJava/t79TkqbAyz57E3nxZxzCztp2wdMPsvxH+1CnyW9rUtERERE+rfO1lxNrgv7AkD+1Pl+F2m50uHMtYavDZTO3G8en4u3hl5s3r0hlsnaQluXH8qfMu938ZbQ2/KnzPtd05ZlJ852rGG7ghVVk0I11WeMYoVzB7BmZoGySly0tbzjWrD5U+blu2jLxPD2tZ2uEdsmWFE1CRjdvPOF5wbSqND8qfP9Lh6/gnjiH2WgbDYeX+6mxi1Ph7JcWr+R7ZGw5zIJGEPKujLOuTozqwbm0EkIa2Y5wEzgyynnxM3sieQ5nTKzXE6/s7ew19WLiIgMEs65PWY2A3gVMDXZvMk513FtuK709a9pLe4C5pzTSFjJqOf3PU9DpIFRwVHMHD0z2+VIH4gcOEB4zVoAil77mixXM+hpGLKIiIiIDHhNm5dGgDUABZfdEIw1N1wWj8c8npzArcGyyj14faHQ1hUbM3b9LctO5E+Z96t4pOUdgYqq34drqo92dlxo2+oDydGwnYawcPYpiQNls6/E491h8eiI0483yyuZPju844Wnz96nWV7JVVebz3/wfEFtf5B/ybVDXLS1FOcSobTHGzGP54WmrcvjWS6t3+rPIeyY5HPHRYwPpezraASJO4I7O2fqmYe3uwf4fHcLFBERkYTkXXr/SD6kDxxpbKG+OYrHYNKI/GyXIxegtqmIbym5Bc+ZMyrJBaj+b4l/5oFZM/GPHZvlagY359yubNcgIiIiIpJOja88HQJWAwQrqoqdi092seiIQPnsW3HxAx5f7uGmLcvS/ntw05ZlTQXTbro/1lT3rsDkOX8Mb11xsLPjzOPdE5w8pyS0dcXOM/ada01YF88lFi8NbVtdndqcN2nGfE9OYOnZTgtOmTsib9KMS4GVoZrq1i6/oT6UP2XeRS4WHd3eYFYfqlm5LoslDTj9OYTtS18msY5sm0Jgb5ZqERERGVDM7DtArXPuOx3aPwyUd2XKQum+2kOJUbAThwXJ86d1VkoRQpEQz+55FtBUxINJ3ZK/AlCsqYj7BTMrI7FEzyXJpo3At51z27JWlIiIiIhIGoRqqus4FciOdHE3KR6LXBosr7oc3BHz+mvONX1wdzVueLIlf+r8H8Vbm98TrLj60VDNyt2d1LQvORp2Z1f7DU6eOxqvvxEXO+2LmUBZ5ZXmz92UHAl8hkDprCvMzIW3r32uu+8lU8zMAhVzJhOPnZpuzeM9EKqtXpvFsga8boWwZvbzrhznnHtXz8o5TdvdCKOBAynto4EXz3LOUSCWPCbV6JT+zuCcawFa2rY7rk8rIiIi5/QG4NZO2pcDi0h8gSxpVntk8ExFfCR0hL/v+jvReDTbpfSpiwov4qaJN/Wqj5ePvszaQ93/e2lH3Q6aY81MLJzIpcMv7VUN0nvx1lYa/v4PCm+8AU8w2K1zG556itZdZ3y/cOY1wiFaNm4Cn4/CW27paamSJmb2ahLrrb8ILEs2zwNeMbPXa610EREREblQhGqqjwBHAIIVVeNdPD7ORVtvCJRVNpnZCfPlvHC2MLM7mjYvjZvZj/NKZ74rWFH1XKimuqbjMR5f7t5gRdXEUE31GX9EdboubDx6sXNxT+pasMGKqknm9TWEtiw/Y+rj/Knz8+ItoSrz+TeEaqqP9/Y99UbexGleT17BFcTjHkis52o+/9amjcsbslnXhaa7I2EXArtILKqc6aRyB4ng9CaSoauZFQFVwA86O8E512pma5PnPJI8x5Pc/l6G6xURERmshgN1nbTXk1gqQDKg5lBbCHvhL2X/pZVf4qk9T2W7jKz45Wt/yZUjr+zRueFomHf//d00RZp6fP3XTHqNblDsB4798Ecc/f73Gfr2tzPmP/69y+eFN7zM3g9+qFvXyp87B9/Qod0tUdLvPuCbzrlFqY1mdh/wFTT9v4iIiIhcgEI11fuAfcDq4OQ5JfFYdKRFWt4cLK/cj3mOhWtXbUguidUjyXN/FiirvCtQUZUXrqnekLq/acuyvYFJM6qA3aefd2YAO2T+nYZ5cnCuPbQMVlQNw7khoZoVL3Q8PlhWOcnFY6PCO9Y929P6eyN/yrx8F4u2L9vpCRTG4uGGl5p3b4hlo57Borsh7A+AtwGTgPuBXzrnepzWm1kBUJ7SNMnMpgPHnXO7zexbwH+YWQ2JUPaLwH6SAWuyjyeBPzrn2kLWbwAPmNkaYBWJ0Tf5yXpFREQk/WqBWzjzhqfXANv7vpzBofbw4BgJG46GWbY/MQjs5otvJtebm+WK+sbGYxvZXredJ3c/2eMQduX+lTRFmhiSO4T54+d3+/zCnELuuuyuHl1b0qv+748D0PCPfzD63z/T5WC84R+JnC6ntJS8yy877/GWk8Pwu+/ueaGSTpcAb+6k/edohgkRERERGQSSa7PuBFYHy6smx2PRUXmTZtwVKKvcbh7P3lBN9Y6e9h3etvrBQOmstwbKZ/vDtatOW+PUfP4DnY2G7bgubOvBbTOci7vm7WteAiicfovPxWPTwtvWPHv6eWZ5F0+fY17fvvCOdaetG5tJwYqqMTjGt9fh9YXC21at602ILd3XrRDWOfchM/sYcAfwLuDLZrYE+Bnw9x78w5sFPJ2y3bYu6wMkRt1+lUSA+mNgCLAUuMU515xyThkpo2ycc781s5HAfwJjSIyivcU5d6ibtYmIiEjXfAP4XvLnb9twxZuAj6MvijOmbTriigs8hF25fyUtsRbGF4zn69d9fdCMynx0x6N86rlP8eyeZ/nYzI/1qI9n955a0/WeqnvSWZ70odbdu2mtTSwBGj10iOaNGwlcdv5AFaDx6cSfWiM+8AGKX/+6jNUoGXEEmA50nCJtOnC4r4sREREREcmmUG31VmCrmS0LlFVe5uLxycHyymudo8Z8/q2dTf17PuHtax4KlFXeHqi4OhCuWdm2BAihmurdybVh20PYjgEsgHPxHPN4WttysWjd4fm546eeFsAGyypH5U2aMdWTE1iRjimVzyU4ZW45sVhxe4PHczhUs0LruWZZd0fCtq2f+hvgN2Z2MYmw9PuAz8wuc841dqOvZzjHtMbJf3k/l3yc7ZiSTtq+h6YfFhER6RPOuZ+bWS7w78Bnk807gQ845x7MWmEXsLpQhCMNieXsyy7wELYtSLzuousGTQALMG/8PHzmY3vddnbX72Zi0cRunR93cZ7Z8wwA1024Lv0FSp9pC1JPbT/TpRC2de8+WmpqwOul4NprMlWeZM5PgB+bWSmJNdYhsSbspzl187KIiIiIyKCSzIxeBl42M8srnXWFi7bOD5RVFhKPbfUWDH2xccOTLV3tL7xt9R8D5VWvDVZcfVOoZuWTbe3m8e4PVlSNT06PfBozG4nH+z8491pcvMjMTuDx7jSP7z/CO9Y5M3PA7XklV+0wjzcS3r72ua7UYmY7gW85576V0jYF+CFwKVBMYqbYX3uCQ/4zd/wlU4nH/O3Hen3bm2qra7v63qVveHp5fhxwJIJUb+/LERERkYHIOfcD59xFwGigyDlXqgA2c2qPJJYbGVucR0Fut++pGzAGc5BYlFPEjNEzANo/g+545egrHGs+Rr4/n8rRlektTvpUw9PPAJB7ySUAND7VtfWR28Lb4IwZeIuLz3O09ENfJDG7078BzyYfHwbuBb6UvbJERERERPoH55wLb1u9PrxtzSMu0vxrPN5oLFR3R6Bs9p2B0plXWBfv5A7XVv/NubgFyqte09YWqqne7eKxCe0HmSfuopG2DOxhnJvpzR/6WWCyt3D4RzDP0y7aEmw73BMccqknJ7A7tG31xl6+zQjwoPlyX+cfNemNnuCQ74J9wEWaf+CiLZtDtdVr2x5NW5ad6OW1JAO6HcKaWa6Zvc3M/gFsBaaR+GNwYndGwYqIiMiFxzl3RL8PZN5gWQ92sAeJ10+4Hjg1Grg7ntn7DABzx83F7/Wf+2Dpt2INDYTWrAFgzGf/A8xo3riRyKHzr7TS+HQirC244YaM1iiZ4RK+mbzBqRgods5d5Jz7ttZwEhERERE5XfPuDbHwttVrw9vW/MYbLHzYzJufN2nmXYGyyjcHJ88pPd/54dpVT4BrCJTNvrWtzbz+g8GKquSaqu4oXt9QMxsCXOMbOvbPueOn/jhQPjuaM6p0g4u23uOc+7OZZy9APHTyv0Jblx9Pjm7FzMrM7E9mdsjMGs1stZm9qv1aZs8AFwPfNDNnZi5YcfWMQHnVkEDZ7JfySmeEI4d3LIk1nfgmuAdcpHlyePva1vR9gpIp3Qphzez7wAFgEfBXYIJz7k3Oub855+KZKFBERERETldzaHCEsG1B4rxx8wZlkHj9RdcDsO7QOupb67t17rN7EsHtDRMUwA1kTc8/D9EoOaWlBGfMIHDFFUBiSuJziTU20rQ6Ed4W3qh/BwYiM3sq+QUPzrkG51xDsr3IzLo2HFpEREREZBBq3PBkS2jbqhXh7WseMPP8LR6PlgbKKt8ZKKu8Izh57uiznReuXbXUvN4DwbLZbwIIbV2xs200bKzh2CESv583AqFY/dFJ0RMHi4nHLwrVrto0ZP6dFiirnO8fMeF1ye7uBsYCbXeUFwB/A24CrgIeA/5iZhMB/KNL/z+wQ5YT+KF/dNmrfSMvnh2qWbkuVLNyXai2em1oy/KXnXPOzMqBW0jMlCMDQHfnr3s/icWItwPXAdd1NqLbOXdH70sTERERkc7UHhkkIeyeZ4BTI0IHmwlFEygtLmV73XaW7VvGaya95vwnAfsb97PlxBY85uGa8VoLdCBreCoxpXBbkFpwww2E16+n8emnGfrWt5z1vKalSyESIWfSJHJKSvqiVEm/64GcTtrzAP2HLSIiIiLSBaHa6kbgCYBgRdUwF2udHyirHIrZSW9u/j8aX3k6dNrxW1esDlRc3Room/X28LY1vzSP91Bw8txxLQdr9wfKKr3OuaivePRPY/WH39F6eNtBDrPGvP41vuJRL3gLhv2i9ciuWDIzO+mcO9jWr3NuPbAewMwsMHnOb8M11W+znMCHguVV/+cfMm5P5ND2Ztca3tJ6sPbvHd+HmS0HZgC5wI+Bz2XqM5P06m4I+yCJNWBFREREJEvaRsJWjCrMciWZs79xP1tPbB30QeL1E65ne912ntnzTJdD2Lbpi6ePnM6QvCGZK04yykWjND7/PHBqSuGCG27gyLe+RdPKlcRDITzBYKfntq0Hq6mIBx4zuyJl81IzG5Oy7SVx1/u+vq1KRERERGTgC9VUHwf+DBAomzUm1tx4a6B0Vp55vAdzxpQ9fnLprx1AuGbl+mDF1c2Bssq7w9tW3x8om3U1sD953gT/iIv/7ise9eeWvRt95su51UVbr42e2P+h6In9AIs7Xjd/6nx/LFRf1Xpg6/tcrHU+MCK8dYUBAdfSZKHa6rUA51nC9i1AIXAl8DXgE8BX0/PJSCZ1K4R1zi3MUB0iIiIyQJnZXcBvnXMtHdpzgLc65x7MTmUXplBrlH0nw8CFPRJWQWLC9ROu5+cv/5zn9z1PJB7B7zn/tMxtUxFfN+G6TJcnGRRat454XR3eIUMITJ8OQO7kCvzjxxPZt4+mFSsovOmmM85z0SiNzyT+HSi84fq+K1jS5UUSNz47oLNph8PAv/VlQSIiIiIiF5rwtjUHgYcAAhVzJrXs33pXoHQWeDybwrWrVoVqVm7JnzKvOVBe+X7DHg9WVI0BZ+CZj4tv8wSKDuSVTJ9kXt9nQjXVdWb2U+ALJENYT3BIWbC8aiYAHm+kZc+GtwNzgY8BtSR+r/89nc9+cwbn3J7ky41m5gV+bGZfd87F0vahSEZ0dyQsZlYC3EziX45nnHOvpLsoERERGVDuJ7GWxeEO7YXJfQph02j7kSYAhufnMCy/S7+rD0iDfSriNleMuIKhuUM50XKCFw+/SOWYynMe3xRpYtXBVcCpNWVlYGpMTkVccN21mNcLJO6MLrjhBk788pc0PP10pyFs+MUXidXV4SkuJnDVVX1as6TFJMBILAE0GziSsq8VOKwvWkRERERE0idcs2IHsAMgWF41I1BW+U6cw/w5y82X+0cizW9x0dZVeHP84EbhOOhaw6PDO9Y9B5A/Zd5FlhOsd62homTwGomHTu5sG+EKYGbzgMXOuT8mtwuAkg6ltJKY/eZ8PIA/+ay/Dfq5boWwZnYD8FcgkGyKmtm7nHO/THtlIiIiMlAYnS9XcBFQ18e1nNWTu57kid1PcFv5bVSNrUpbv7+u3s3L+/vube45nliupCyDo2BDkRC/3PRLXl/6esYWjM3Ydc6mKdLE6oOrAY3m9Hq8XHPRNfx525/51tpvMWXYlHMefyR8hEg8wsTCiUwqntRHVcrZxEMhjv7wR8Tquv//iIYnnwSg4IYbT2svuOH6RAj7jyc44D3zz7mWzZsTx113Lebr9j23kmXOuV3Jl56sFiIiIiIiMgiFaqvXAesAAmWVN+LcBCDiH3/pj53HN808tqrh+V/d6ckN/sE/dFyZ+fyhaP2Rsa41fCfwh1Bt9Voz2wncZGbLgBbn3AmgBrjDzP5C4ju0L3Lm7/w7gWvN7KHkeUfN7F+ACLABaAFmAV8mMSNdJMMfh6RBd/8q/yLwD+ADQDPwJRLzTiuEFRERGWTM7AVOTZn4pJlFU3Z7SYzmeSwbtXXm+X3P89ftf2VY3rC0hbC1hxv5zB83pKWv7rpsXFHG+n5g4wN8/8Xvs/n4Zr5x/Tcydp2zqT5QfSpILFKQeOPEG/nztj/z0tGXeOnoS10+5zzryUgfOPm733Hsxz/u8fmWm0v+/HmnteVXVuIpLiZeV8fJ3/72rOcW3vSqHl9X+g8zuxSYSIdpypxzf85ORSIiIiIig4N5/ZtdtKXO/Lle5wtMa9rxIsXTrpttvpxfxZsb74yH6ycAPswOmj/vL/7RpT8PlM2a4QkWfz8erv9/OPceYB+JEa8fA34OLAeOAl8BOn6x8zngR8A2IJfEoIco8GlgcnJ7F/A94JuZfv+SHt0NYS8H5jrnDgCY2SeB95nZcOfcsbRXJyIiIv3ZI8nn6cDjQGPKvlYSd/A93KcVncPsMbN5uObh9hGW6bB821EgsTbr668Yl7Z+zyfP7+FNsyZkrP+V+1cCsOrgKuIujsf6dkBW23S6V4+9WkEicOOEG7l3zr0cDnec8btzQV+Q2ytuz3BV0hVNKxL/LRXceCN5l13a7fODsyrxFpw+6t1ycpj4ox/SuGzZWc/zjRhJ4c0KYQcyMysF/ghMI3GzU9v/DNtmnujKNGUiIiIiInIOwSlzRxCNlAA4F3eYOZwzM4+ZL+cgvpwG19pyHeG6Lfnjp06JNxx9KXf8JT81j3dPqLZ621m6XecfOu77/pEXjwYbGiibNSOvdCbAJzoctwIgUDZrRuJ5dgR4l4snxjgESmfOzCudWQO8B8Djz4u5eOx466Fth2P1R1rT/VlIZnQ3hC0ikdID4JwLmVkYKAYUwoqIiAwizrkvACSnWfmtc645uxWdW9tampuPb6aupY7i3OJe97m8NvHrz23Tx/HhGyt63V9/EI6G20db1rXUsfXEVqYOm9qnNbSFsLPHzu7T6/ZXZsYbJr8h22VIN7lolNDqxE0fIz74QQKXX5a2vgPTpxOYPj1t/Um/9G0S61LdlHyeDQwHvs6ZX96IiIiIiMhZ5E+ZVxSPtpS3hatwKnA1j/dEePuatc6505bZyrv4imJz8Tfg8R4wf+6fo0e2X4V5fwnuSWAW5jkQKJt1tZn3eKi2emvHa0ZO7G8F9iQfaZEzYqLfWzRiWO64qRXxERNzguVVyfcSO1W7c4mbN806WzYMM+9p55h5HF7fycixvYejx/aEOn4O51OyaIl/530LNC3yOfRkkaBXm1nqokYeEvNbX97WoKmRREREBg/n3ANmNsTM3g6UAV9zzh03sxnAIefcviyXCMDI4EhKi0vZXredNQfXcNPFN/Wqv3jcsXJHIoSdUzY8HSX2Cy8cfoFo/NTM0qsOrOrTEPZ483FqTtQAp4JzkYGoeeNG4k1NeIqKyLukb29kkAvCHODG5DpQcSDunFtqZvcA3wGuym55IiIiIiL9R6BketD8eVOJxzwOF8c5w8xjmMM89c3b177onIufr5/8qfP98Ujr7Xj9OZ6cwC+aNi+N5JXO/KJ5fL9ysegM83pzzON90MUibwP+bDk5LcGyyirM6kO1qzZl8j22Ht0dAQ4lH2lhZt680plDckZNmugrHhUIlM3q5BjvmWEvMGT+2z/vKx51a8miJYt33rfg7nTVdKHpSQj7QCdtP0p57dDUSCIiIoOGmV0BPAHUkVjn4ifAceAOEuvY3ZW14jqoHFPJ9rrtrDq4qtch7OaDDZwMRQjmeLnioiHpKbAfaJuu2e/xE4lHWH1wNXdd1nf/CNuuXz6knGF5w/rsuiLp1lRdDUCwshLz6s8j6TYv0JB8fRQYB2whsQbUlGwVJSIiIiKSLYHSmTnm9V9CPOaHDtMH+/Oa4y1N65t3b4j1pG8zs0DZ7Buci5dgniXN29YeSl5zJs7lxMP16z2B4lziUcPDJZ6cvJ/FIy1vd5GWZ0LbVlfnX3LtkGQY2xSqXfVyOt93JjnnYiRmue3WTLcli5b4gVuTmwtLFi15r0bEdq5bIaxzrm8XBBMREZGB4JvAYufcp8ysIaX9b8Cvs1RTp2aPmc1vt/y2fbrb3mhbD7ayZBh+74XzK9KqA4nP5o2T38hvNv+GNYfWEI1H8Xl6cu9e97WFsFVjq/rkeiKZElqZCGHzqzSttvTIy8CVJKYirgY+ZWatwHuB7dksTEREREQkU/ImTvN68goudbGIH8Cw9i9czOtrjZ7YvzE5IjRtAmWzLs2bNHOO83jXhrdWP9XWnj9l3lCHXWte31Oe/LzSyLHdz/mKx7wZs5pYqC5uvpxfOXhToHRmXnj72k1Adf6l1xUmw9hwqHbVS+mssz/Zed+CSMmiJYuBhcBiBbBn1zffpomIiMiFbBaJL4U72geM6eNazqltetvak7UcCx9jeKDn0wiv3H7hTUXc2NrIK8deAeCuS+/ir9v+SkOkgc3HN3P5iMvPc3Z6tAXkmopYBjLX2kpo3ToAglW6oUB65EtAfvL154C/As+TuEP9LdkqSkRERESkt8zMEyivmkw8VgDgSEwTbJjHk5sfJx7dFN62JpzpOoJllRc73BzgmCcn7/6mzUtPm6441tp8Cbjh4do1jwfKZs2IHNsbDZTOImd02ZrWAzWzQ9tWVxdMu+n3saa6W4Pls/NCtateaNr4bANQXXDZDcHEmrGelvC21S92d63VgWDnfQvu1gjY8+tWCGtmM4H/Af7ZOVffYV8x8AjwUefc+rRVKCIiIv1dC1DUSftk4Egf13JOQ/OGMnnoZLae2MrqQ6u5peSWHvUTjcWp3n4cgLkXUAi77vA6Yi7GRQUXcVHhRcwcM5Nn9jzDqoOr+iSEPRw6zI66HRjGrNFnrkMiMlCEN2zAhcN4hw4lt6Ii2+XIAOScezzl9f/f3n2HyXVX9x9/n5nZNqtuS5asYkm7K8lFbiorSzK2MQSCSQKBFEgAGQglIcSEEEQoNqRYgVCS0BIgFhAM4UcHBRtwLNtqq+ImF0lbJFm2mpvaztaZ8/vj3pFG693Vlpm9s7uf1/PMM3O/t50ZZO7OnHvOtwFYYGaTgBdH4g84IiIiIjKymJlV1Cyb4+mOCbnVrI5nKqqWuKfbG1qaHjzZ2zEKJTl/+fne0VaLxRKWKP1Navem516yTfXSyywWn4mxveu6Yxvu9GRNLWZm7t5WuWDlzzMdba9OVi8tSzVs3QJw6vF7U8CWMQtvLCufu6g2Wb00Uzq1etuxDXeOqL/llYA9t/5Wwn4Q+L+uCVgAdz9uZr8GPgT8aT6CExERkWHhZ8AnzOwPw2U3s1nAPwM/jC6s7i2dupQ9L+5h66GtA07CPn7wBCfbOhlbnuDSC8fnOcLoZFsRZ1sBL5269HQS9u2Xvb3g58+2Il4waQHjy0bO5yqjz+n5YJcuxWIjp125RMvdX4g6BhERERGRXMl518wkk5ninnY40z64omqp45m9LY3bi2YqjcoFK8sz7S2LMJth8ZIHU/Vb6rvdbv6KSvdMiWf8svLp8z8BYBbLJk87KuYuKo2VVjxYUbX0amBH864NrZULVt6V7my7vqJq8XWtTTvuz944eWrnPW3AlsoFK0vaD9UvTVYvdUuUbu9adSsjV3+TsLXAml7W/xx458DDERERkWHog8APgKNABXAfQRvizcBHI4yrW0unLuW/n/zv0wm/gdgctiKunXMe8ZjlK7TIdW0FvHRqMJflg0cepCPTQUmspKDnz/5vkj2vyHCVqgv+W6pcplbE0ndm9qO+buvuv1/IWERGgifn/3bUIQwLS3b/MuoQhoXJL3b7O728xNyoAxCRAqhccO0U72idBYDF/KyEa7zkYKqhbkekAZ7DhJVvtvZD9UsdZpnFniqdVvP93ipSMx2tVwLjzOyRrtu5Z/bG4iWXNO/a8HBF1eLTP5I079rQmqxecl8m49dUVC25vnzWwvtbn9qZzlnfAdRVLlgZ8462JRVVi8072rblbiMjU3+TsNOB3krETwHTBh6OiIiIDDfufhx4pZmtBC4HxgAPuvtvoo2se4umLiJmMfad2MeR5iNcUHlBv4+xqXHkzQd7vO04u17YBZxJgtZMrGFC2QSOtR3j8ece58opVxY0hrpDQfXg0mlKwsrwlWlro+WhhwDNByv9djzqAERERERkdKqcv2JiprNtDoBZ/PTd5u5pjyXKnm9p2r5jOE6LkaxastDhItxfjJUlf9385P3Het2+unYesUQD6c53W9w+kx13zxiAxWIZ93QMwCz+WLJqycJU47adAKmGbS2V81fUpTvaro6VlF9bOX/FtubdG5tzjx9WwNaFieFFFVWLE7GS8h1hklZGoP4mYZ8F5gN7e1i/AHhJ/2wREREZ+dx9A7Ah6jjOZVzpOBZMWsATzz/B1sNb+Z2q3+nX/h3pDNv3BR0hr5k7cpKw249sx3Fmj5vN5ORkAGIWY8nUJfx6/6/ZenhrQZOwh04d4ulTTxO3OFdPubpg5xEptJaHHsbb24lPPp/SOXOiDkeGEXe/OeoYRERERGTkqpy/otIz6fm42+lq1mzCNR4/3tq046HhmGjtTrK6tsoz6QsxK7V44onUns3nbItcuWBlCZ6ZaDAnA0+0NGxr6bqNxRIpT3dMAkg11J1K1tQmc9c3797YXDl/xYOZjtar6Wy/PFlTuztVX/eSaUXCCtvtZmYVVUuvrqhaXBIrKX+4edeG1kG8bSlC/U3C/oagreBdXVeYmYXrirLqRURERArDzN7fwyoHWoEG4H53L5oWK7VTa3ni+SfYdnhbv5Owjz59jFR7monJEhZMHVugCIdethVwdj7YrKVTl55Owr7r8ncV7PzZVsiXnncpY0rHFOw8IoWW2hpUdFfWLiP4iiQiIiIiIjI0ymctLIuVJi92T8eyCdbTCddYImXxxMMjeT7SZNWSaZ5JzyYWr7BY/HhL49YNfU0sZ9pblnpn+yaLJz4cS5R+I3dddk5Yiyf2eGfHH5xeEYs3Jqtr56Ua6vZkh5p3b2xOVtc+5J650pw5yXnLy1N7Nh3s7pxhbDsAkjXLrkhWLSm3ROljXStoZfjqbxL2H4AdZlYHfBbYHY4vIJgPbh6wKm/RiYiIyHDwAWAykAReDMcmAimCqQqmAE1mdoO7H4gmRFj36CG27g3aCB/umALA/zbewyNP9e/v2hdTHZRd0Mrk85Ks2bY973FG5Z799wBn5oPNyrYmfujIQ/xT3T8V7PwPHnmw2/OLFLuOo0d58dvfJpMKbpI+tX49AMlatdWWgTOzvQQ3M3XL3TXpnoiIiMgoVTa1OhEfN+UST7cn4Oz2wbHyMe3pE88+1na4oTO6CIde5SXXjfW21GVusbjFrdPiJXX9SWQmq5bMsXjigMdiN3om09RSX/dsd9s179qQKZ9zdczMzN09tXvTcxVVi6u7bpdqqDtVecl1j2Ramxea2cTk/OVlqd2beuowG+xTv+URgGRN7eXJqiUVlih9snn3xhN9fQ9SnPqVhHX3RjN7BbAW+B5nvhQa8ATwSndvyGuEIiIiUuz+DngX8E53bwQws2rgP4D/BDYS/N3weeCNUQVZt/d5vrV5f7AQSzKmJkFb7ARN7b/q34ESUDoJDjl8d1f+44xSwhIvSYLOGT+HC5IXcCR1hO/u+m7BY7jmwmsKfg6RfHr+a1/nxW9/++xBMyqv0b9lGZQvdFkuAa4CXg185iVbi4iIiMiIUrlgZQz3+Z7uTEJONavFLTFhamc6dXxX61M726KNMnrlsxbGY6UVSzDLAMTiiQPNuzfu788xJqx8s7n7tFi8ZAsdrW+MxRM/zl1vZlY+d9HpRLfFYk+Xz1lUDdQHy/EDlfNXXNT1vM1P3Heycv6KxzIdbZdZrLQtWb304lTD1ifPFU+qvu5RgGTVkksqqhaPs1h8T3ctjWV46G8lLO6+HbjMzK4CqgkSsHvc/eE8xyYiIiLDwz8Ab8gmYAHcvcHM/gb4obvPNbO/BX4YWYTAdfMmM76i5PTywba/43D74wM6VnkizsIZ44nHRlar0SsmX8Gk8klnjZkZX7jhC6w/sB7vuSgrLy6svPB05a3IcNHy8MMAjHvNayi5aBYA5RdfTOmMGRFGJcOdu/9rd+Nm9hfA4iEOR0REREQKwMysorq22jOdY4PlM+2DzWJu8ZI9qYY6taXtQbJ66VWWKC1xrMOcdKpx28MDOU77ofplsdLyLd7R9nKcg6n6urMqViuql55vscTzOUPNeOb0PEqp+rpnkjW1tcBLkr/NuzeeSNbUPk5nxyXEE08nq5delWrY+lBf4ko1bnsCIFmzbEFF1eJ5Fos3pnqo0JXi1e8kbJa7PwT06R/LYJjZWODvgdcTtDN8CPgrd9/Ww/bXA/d2s2qaux8uUJgiIiKj2TS6/5siAUwNXx8EIp1A9caLL+DGiy/IGZkP/EFPm0uOy86/jMvOvyzqMESKTqa9ndZdQUn85A/cQunMmRFHJKPAL4HbgZujDkREREREujd79bqSfWtu6oAw0Vqz7CIymfPg7IrWiqqlWCzelKrfUh9lvMNNsmrJfPfMRMyOm8XKLVHyaPOuDa0DOlZN7XTiiSMA7j4vFovd/5KNnOmpPZse7fVAFns2OW/5Bak9m450XZWqrzuerKl9wtMdF8dKKpqS1UuXpRq2bulrjKn6LbsAklVLqivmXFVlidJ9qfo65bqGiQElYc3scz2scqAVaAB+6u75KJH+OnAZ8BaCH3D/FPiNmV3i7s/0st98ILdf9tE8xCIiIiIvdS/wH2b2zvAmLcKOGV8B/i/cZiHQ69wXIiLDTdvu3dDRQXzCBEpU+SpD442AWpGJiIiIFKmLPvzzOzpfPLRq2tu+8LNjG/77kxVVS81iiQPNex7YF3Vsw11y3jUzvbN9uiVKD1smDWYnUvV152zv2xMzs/LZV81p2fvghoqqJTe4+4stDVsfe+mWbu6e6e1YqT2bm5LVS5cBL0nCQpCIrbz4Zbu8vWW+lSUfr6hafF1r04773b3PLcdSjdsagIbkvGvmJquW1BKLPZ2qr+stRyZFYKCVsFcBVwNxYHc4Ng9IA7uAPwc+a2Yr3f2JgQZnZhXAG4Dfc/fsHQi3mdnvAO8FPtbL7kfd/dhAzy0iIiJ99g7g28AOM+sIxxLAPeE6gFPAByOITUSkYFp27gSgfOFCzEZWe3KJlpk9BGf1gDeC7hKTCb5vi4iIiEiRmb16XYlZbFXJpOkAv3vBH3zyjdmKWBm4ZE3tJDKZ+Z7uPEg8niaTHtOfStKelM9dtDieHL+tfNbCuCVK51jMdna3nXvmrC97ZrFO987ybjY9XnnxyyY0P3n/se6O0/zk/ccq56/Y4+0tF8dKyjdVVC2+oXzWwvtan9qZ7k/cqT2bm4CmZNWSiyqqFi+zWPxgqr7uqf4cQ4ZObID7/RT4DXChuy9y90XADODXwHeB6cD9wOcHGV+CINHbtZS8BVh5jn0fNrNDZvZrM1vR24ZmVmZm47IPIm6XKCIiMlxYkHUoBX4XuJigv+8fAJe4+2+5e7aly73u/qvoIhURyb/WR4Pv6BUL1a5b8u4nBN+7s48fAZ8ELnP3/4wwLhERERHpQZhwXRsurlUCdnDGLLyxrGLu4uW4X4DFjlpJ6dTE2PMfSnVbrdo/yZrayWaxU6d23tNmJeXXOtYaKynf0d22ZrGzqlXTp154gkTppGRN7eTc8VTD1ie9vWV+b+dt3r3xRcwavLP9qtKpNffGSsqvrZy/onIg7yHVuG1/S+P2LRZLZCqqFi9L1tTOGchxpLAGWgn7IeCV7n663a+7Hzez24Bfufu/mtmngEH92OruJ81sM/BxM3uSoJT7TcA1BC2Pu3MIeA+wHSgD3gmsN7Nad3+wh30+Atw6mFhFRERGKSO4Jl/q7rs50yFDRGTEa3nsTCWsSD65+yejjkFERERE+m/fmptunr163buUgB04M7PyuYsW456xROluMukaYvG9qT2bGvN1Dk+na1qatm+qXLCyBM9cZPBk864NvbYczmo73NBZUbX4qKfTM4Fnzw4+1la5YGV5b3PUpurrXqi8+NrG9kP1i1KN29Ynq5Zck6yp3Z2qrxvQtCPNuzc+DTxduWDltGTVklos9lyqoS5vn5UMzkArYccDU7oZnwyMC18fI6iMGay3EPzA+wzQBryfoNq22/8g3H23u/+Hu+9w903u/nZgE/CBXs5xO8F7yj40oZOIiEgfhHNi1APnRR2LiMhQSp9qpr2xCYAKJWHzzszWm9kXcpaTZvZDMzthZm5mE6KLrvByOzV1eYw1s3x8zxYRERGRAlECduCS1UsvK59zdS3uj5jFErhPTjVs3ZLas6nbuVYHoqJqyRWx8sqdAOm25qVAS6yk/PF8HLulcesj3tl+5bm2a37ygedJlOxLVi1ZnGrcthlnTnLe8gsHc+7mXRsOpRq31Vmi9GRF1eJlyeraeYM5nuTHYNoR/5eZvd7MZoSP1wPfIGibBLAU2DPYAN290d2vA8YAM919KVACNPXjMFuB6l7O0ebuJ7IP4OSgghYRERldVgOfMTP14xSRUaP18cfBncS0aSTOPz/qcIadVatW8brXve6ssR/84AeYWauZdTeH+NuAa4HlwDTgeO5KM/tqmJy9pTARD7ljwIvdPI4BLWa238w+aWYD/U4vIiIiIlI0klVL5lTMXbQMswMWLzlm8cRVVlK2LVW/ZVdez1NTOx53b37ivpNjFt5YZliVJUoPNO/e2NzTPl3nhO2NuzvuXrlg5Tn/Tk/t3vQciZJ9yeraRamGuh14ZmJy/vJBtxRu3vXA0ZbG7VuspPT5ZNWS2mT10osHe0wZuIF+YXs3cA/wPWB/+PheOPaecJtdBK2A88Ldm939kJlNBF5FkAjuqysJ2hSLiIhI/n2L4OarR8ysxcxeyH1EHZyISCG0PpadD1ZVsPnw9a9/nT/5kz8BeK+7f7abTaqAJ939MXc/7O6n52UKbwheBhwcmmiHxCqC9/NPwOvCxz8RdIh6L/CfBF2iVkcSnYiIiIhIHiTnLb+gYu6iZcRi7bHSZBPuF1ui5NlUfV1dX9sD94enOy9radr+KEBn87FFlVe95i3nv+aWTbNXr7ujp326zgl7LlZSts072pb0ZdvU7k3PWaL0QLK6dlGqfsvjQDxfSdPmJx94PtW4rc5Kyg9VVC1elqxaoi+vERjQnLDufgr4MzP7ADA3HG4Kx7PbPDz48MDMXkXQjng3QTXrZwgSvHeE628Hprv7W8PlW4C9wONAOUEi+OXAb+UjHhEREXmJW6IOQERkqLU8mp0PVk0ABuvTn/40t956K9/73vd4/etf/5IfP8xsPXBd+NqB+9z9+nB5OvDvBDfqrhuyoAvvbcAH3f37OWM/N7OdwLvd/UYzewr4KEFyVkRERERk2Kicv6Iy09G60OIlzyfGTd6ePvXCYk93PJ9q2LqlUOdMVi+92ErKdgevl1RYvHRe+fSLXxErHwOwqr9z+Vq8pLNi7qLSlqYd7bnjzbs2ZJI1tZiZ5d482pPmXQ8cTc5bbhVVi69uadz+YHLeNTMr5i66sqVpx8P9fIvdH//J+48BWyovuW5sRdXiZWaxVKph66P5OLac24CSsFnufipb4ZKbgM2z8QRzts4AXgB+CHzU3bP/MUwDZuVsXwp8FpgOpIBHgVe4+70Fik9ERGRUc/dvRh2DiMhQa92ZrYS9POJIhrcPf/jDfPnLX+YXv/gFN954Y0+b/T6wBrgsfN0OELbi/TbwGXd/3KzPXcKGg+Wc6TKV6yHgmvD1Bs7+LiwiIiIiUtQqF6yMeUdbrUOqpWnHlmT10ss6Tz53VWvTjrq+JCwHcd5y3CtTuzc9CZDJ+JUx63wyUTlhLUEXmrXdJWDNLFZRtfQlcZnFM+nUsSfjyfGXEfyNfpZY+ZiHK6prrwIe7Et8qT2bjiTnLSdZvfSqVMPWh5I1ta0VVYuXtTRuz1tSuvmJ+04CW8ZcekOyYs5Vy4jF21ubdjxUyM9dBtiO2MxiZvYJMztO2I7YzI6Z2cfzPSeNu3/f3avcvczdp7n7+9z9eM76Vdm7oMPlT7t7tbtXuPt57n6DErAiIiJDw8zKzWxc7iPqmERE8q3z+efpOHgQzCi/7NKowxm2fvnLX/LpT3+an/70p70lYHH3FwhusG0PWxFnW91/GOgE/q3w0Q65A8A7uhl/R7gO4DyCeWJFRERERIpexdxFV2Y6WpdaSVmdxUueTVYvXYbZgZbG7dsKnQjMtLcsSjVu2w5BFa5ZbLaVlG3bt+amm4HS8PklkvOWz8R4puu4d7btiY+ZVOOZdLf5sFM772nzTGdpf2JM7dl0BLNDyeraK1P1dc/Gyiofr6hafJ3l+W7TU4/fm2rZ+9CWeHL84+VzF9Umq5YsnrDyzSPqjtZiMtCE6T8C7yOYf+aq8PF3wF8Cf5+f0ERERGQ4MLNKM/uimR0Fmgl+EM59iIiMKC1hFWzpnDnEx4yJOJphpD0Fz+6GVJBDvfzyy5k9eza33norp071r7GSmS0C/gpYNULv3P4b4ANm9oiZfT18PEwwBcAHw22WAP8TUXwiIiIiIn2SrK6dVzHn6mtiJeX18eT4h7yjrRajLNWwdUuqvu74uY8wyPNXLak2izVll9MdrQshszc752xvLYg93TGppWHrc13HW/Y9nPJ0R3lv540lyp5I1izr1127qfq6w5YoOZKsrr2y+Yn7TsZKyjdVVC2+oXzWwnh/jtMXp3be09bSuH2LlZQ90nZoT22yaklt5YKVeS2ylIEnYd8GvNPdv+Luj4aPLwN/RlC6LSIiIqPHpwnmX38v0EYwH/utwEHgrf05kJl9xMy2mdlJMztqZj8xs/l5j1hEZBBadz4GQMXChRFHMkxk0vB//wifqYIvLYXPVPPu87cxb9YU1q9fzzPPPMOrX/1qTp482Z+jXgtMAZ4ys04z6wQuAj5rZvsK8C6GlLv/DFgA/BKYFD5+CSxw91+E23zF3f86uihFRERERHqWrKmdXjF30TKMF1v2PrjZM51VmZaTl7Q0bd+S2r1p71DEECYvJ6catx0CSFbXjjG4KFZSvqOvxxjoTZ/NuzeeAO/3XbvNuzYcIhY/mqxeennzrg0dpVNr7o2VlF9bOX9F5UDi6MP5OsJk7LZMR+uiZNWS2kIkfUergSZhJwG7uhnfFa4TERGR0eN3gD939x8StIV8wN3/gaBLxp/081jXAV8ClgGvBEqAX5lZQf7QFBEZiJadjwJQriRs36y/He7/DHSkgmVPs7TyGT4yeycXzZrFfffdx+HDh3n1q1+NmY3t41G/DVwOXJnzOAh8BnhVnt9BJNx9r7uvdvffDx8fcfd9UcclIiIiItKbyvkrJlbMXbQMKG1p2rEFqEhWLbnGYonGVMPWIZ2D1BJltaXTak7Pq5rJdF6G+77mXRt6rH7t87Et5ph5b+2CLV66Nzl/eXV/j53as+kgZs8nq5defmzDnZ5q3LbeO9svT9bUFiz/1rxrQ6alcfu20mk1W2Ml5VdVVC1eVrlgZUmhzjdaJAa43yME7Yjf32X8fcCjg4pIREREhptJQLatywnO3JC1AfhKfw7k7q/OXTazVcBRYBFw/6CiFJERqePIUU6tX4+nO4fsnK2PBF95Ki4foiRsewqe/Bm09atStDh0tsHGLwBn/84SN+eyscfhmR3MnLmY9evXc8MNNwDcbWav7uZIZ3H354Hnc8fMrAM47O678xZ/hMzsWuDdwFzgD9z9GTN7C7DX3TdEG52IiIiIyNkqF6wsybS3LAI72dK0Y0vl/BWVFVWLl1ksfjDVuG3zUMeTrKmdZYmSg8c23OkAlfNXjDOLXRgrLfvfvh7DPdNjgtU9YxaLN1ZU11YD9d1t07zrgaMVVYvnAg39jT9VX/dM5fwVlqypvTxVX/doqnHb5mR17aLkvOXlqT2bDvb3eH0Vfl7bzczK5y66qqJqcWmspPzh5l0bWgt1zpFsoEnYvwXWmdkrgOx/PNcAM4HX5CMwERERGTaagDnAUwRdMf4Q2EpQIXtskMceHz6/0N1KMysDynKG+lpBJSIjxOFbb+XU+vVDfl4rKaFswYKhOdnW/4Df3DY05xpqh3fCjMXMmDGD9evXM2PGjPOBuzn7/9tHHTN7A0G173eAqznzeYwn6DSh790iIiIiUhTCZN1igNa9D9YBJKuXXoVZuqVx+5be9y5gTHOuntnStGNjdizT2XYJcLA/yUSzWK9Vu6n6uuMVVYurej1GvOSZ5LxrZqb2bD7Q1/NmNe/e+HRy3jWWrF56Waph62OphrodyeqllyWrlpSlGrcVtKVzWLH8IECyZtkVyaol5ZYofax598bmQp53pBlQEtbd7zOzecBfEMxTA/Aj4D+BjwEP5Cc8ERERGQbuAK4A7gPWAD83s/cRtBIe8Fx1ZhYDvgBsdPfHetjsIwTzz4rIKNXyePB/D5UrVhAb0+/pdgZs7MtvIFZaOjQnO/hQ8DztCpg4e2jOmS+dbbDnrp7Xj7vw9Mvp06fj7vO628zdbznXqdx9dr/jK14fA97j7t8ysz/OGd8YrhMRERERiVyyaskl5XOuHof7gy1NO9orF6yc7emOC2LlYx4+tfOetqjiKp+7qDYx9vy67HLl/BUTsdgFht2Tz/OECehet0nt2XwgWbWkFuh3Evb0/vOuoWLu4ktbmrY/nmrY+liyakl1snrpxamGrU8O5Jj9jqF+yyMAyaolCyuqFldaLP5kqr7u+FCce7gbaCUs7n4Q+GjumJldAbwDeNcg4xIREZFhwt0/n/P6N2a2gKB9cIO7D2aagi8BlwEre9nmduBzOctjgacHcU4RGUbSx4+TfvY5AKb/678SHzNCp49+dk/w/PJPQM0roo1lIP7nrbDrF+DpM2MWhzEXQNWN0cVV3ObTfRv+48CEoQ1FRERERORsyXnXzCbdOc0SpbtSuzc+UTl/xbhk1ZKrrKTsqVR9Xd25j1DA2Gpqp1os8fzJh+86PWdNprNtPnAk1bj9VN5O5DRVzLumyjvbz71touT5ygXXTmne9cDRgZwqtWfzgWRN7azTidjGbQ3JedfMrJi76MqWph0PD+SYA4qjcdtOgGTNsksrqhZfbLH4nlR9Xbfd6yQQizoAERERGd7M7K1hW2AA3H2/u/8I2GVmbx3gMb8IvBa4wd17TKq6e5u7n8g+gGE4YaKIDFRbYzAddWLq1JGbgE13wvPh9EGTuy0SLX6/8wWYfnW4EE6pNGYK/Mn/g/iA7wse6Q4D1d2Mr+TMPOwiIiIiIkMqWVM7uWLuomWkOztTjds2l0y+6FiypnaJZ9IzU43b6pp3bTgUdYye7qxK1W85PUdrsqZ2kmETLRbvd9Vob3PCtjRtP+ad7eP6cpzU7k0N3tk2t7/nP+sY9XVPmdmpZNWSSyBIzFo88UxF1eJlgznuwGLZ8nhL4/Ytnk5PqZi7aFmypnbyUMcwXCgJKyIiIoN1B2fmbs01NlzXZxb4IvB64OXuXtD5LURkeGtvagSgrKrXKXiGtxf3QqYDSpIwbkbU0QxMchK849fw9rvhpn+BN30PbtkJUy+LOrJi9jXgX82sFnDgQjP7E+CzwFcijUxERERERp0xl96QrJi7aBnOxJamHVtSjdueTs5fXt1+qH5pvHLiQ6n6LY9HHSNAxdxFi+IV4x7KHfNMeh7wwkDa555rTljczWLxF5Pzl5/fh8Odqpy/ok9J256kGrftB5or5i5eAJCqr3s2Vlb5eEXV4uvMrMeEcaG0NG3f1dK0YwvYhIqqxcuSNbVThzqGYqfbjkVERGSwjOAH4q5mELRN7I8vAW8Gfg84aWbZP96Ou3vLwEMUkZGorSFIwpZWDeqG4uL27O7g+fx5EBvG99CawaxlwUP6Yg3BTdP3AEmC1sRtwGeAr0cYl4iIiIiMIpULVsYy7S1LcW9r2fvgFoDKi689zzvaaixR1tTcuKkh6hizkjW1k8BaTz1+b+r02Pzl5xs21krLH+zv8cqmVicS4y/o7HUjM0/V1+2tqFp8NfBcb5uGc7nWAoNq15xq3LY/WVM7p2Lu4gUtTdt3NT9x38nKBSs3lc9ZdH35rIX3tz61M33uo+RXWHlcn5x3zdyKqsXLLBY/kKqve2ao4yhG/UrCmtmPzrHJhIGHIiIiIsOJmT1EkHx14B4zy/3DNA7MAe7q52HfGz6v7zJ+M7C2/1GKyEjWlq2EnTuCK2Gf3RU8T54fbRwypNzdgX80s88QtCUeAzwBvBvYC+gOcxEREREpqGT10ss9k07GSiu2Nu/akKlcsDLm6Y4lWPxYqmHrlqjj68rTnRe3NO3YeNZYZ3s1cDL15APP9/d4ifEXzM10tOzLV3wAxBPtYxbeWHZq5z1tgzlMqr5ub7JqyZxkzbIFqfotu5p3beiYsPLN69sO7r6ucv6Kbc27NzbnK+R+xbVncxPQlKxaclGYjD2Yqq97KopYikV/K2HPVc1yHPjWAGMRERGR4eUn4fOVwN3AqZx17cA+4If9OaC7D3nrFBEZvtrDOWHLqkdwEva5PcHz+cN0Pljpl3CO9duAVxJWvrr7T8zsZuDHQBr4fHQRioiIiMhIl6xaUg1MtrLkY1N+929b9625KZOsWbYAz4yzROm25l0bMlHH2FWyasnCWFnlWS2Rk/OWX4DFKiyWeGIgx8x0to1paXrwZH4iDLTUb3m4onrpUgZZDQuQaty2NznvmrnJ6tp5qYa6Pcc23OnA+mTVkmuSNbW7U/V1Lww+4gHHth/Yn5x3zcwwGXskVV83Kqcc61cS1t1vLlQgIiIiMry4+ycBzGwf8D/u3hptRCIymmRSKTqeCboblY7kOWFPV8IuiDYOGSqfIqh2/Q2wHPh/ZnYHsAz4IPD/3H3I24uJiIiIyMiXnLf8Qu9sm2XxxL5UfV3D7NXr7si0nlo17S2fXQf+1lTD1l1Rx9idZHXtGCwWb37y/mO5455unwO0pPZsOjKQ455zPlgI5oQ1s/K5i/p0THf3ZNUSKhesjOUjmZ3as7kpWV1blU3EAqQat22uqFp8dXLe8vLUnk0HB3uOQcZ3ADiQrFoyraJq8TKz+LOphrrGKGMaasN4UiEREREpBu7+TSVgRWSote0NbqKNT5pEYuLEiKMpkEwGnqsPXqsd8WjxB8Bb3f2NwG8RtPdPAFe4+/eGMgFrZmvN7CdDdT4RERERiUbl/BXjKuYuWka6o6ylaceWVH3d4dmr15V4Z8eqTEcbZdMX3DTlDZ/Ia0VoPnmm84pUQ93DuWPJmtqpWDxh8dJBtMK1cyZhLZ54pqJm2QyLxTsr5i4q7dNRS8q2ebqjb1nbPgiTmulkzbKa7FhL4/YHyXROSlYtmZOv8wxGqnHboZbG7VssUXoyWbWkNlldO2paPSkJKyIiIiIiw057Y3Y+2LkRR1JAxw9ARwpiJTCxKL47S+HNAHYAuPtjBC2JPx/OESsiIiIikjdjr3x1Ilm15BrvbL+opWnHllTjttPtYvetuanDEiVrE2PPA1i7b81NHdFF2rNk1ZL5ZrH6ruOeSc82z3QMrhL03FNmperrDnu6Y3KmLfW4xRKX9OWoYQVszMzyNiVXqqGukUzaw1bS4djWx4B4snrpxfk6z2A173rgaKpxWx3x+AsVc66urZi7eMS3fFISVkREREREhp22cD7YEd2KODsf7HnVEO/XTDIyfMUJ5lXP6uTsOdcjYWb7zOyWLmMPm9ltOctuZu80sx+bWcrM6s3sd89x3DIzG5d9AGML8gZERERE5DQzs4qqxVd3nnj26tJpNVtSjdt2drfdvjU33QyUhs9FJ6w8nZBq3HY0dzw5b/mFFi+BWOzQYI7vnulTktQsbq1P7Ux7uqPPSVWLlzxSUV17+cCje6lU47YGLBZLVtdWnTUWi5+qmLvoynyea7BSuzc917L3wTrgcMWcq2uTVUsWRh1ToSgJKyIiIiIiw057U1gJO5KTsM/uDp4nj5pOTQIGrDWzH5nZj4By4KvZ5ZzxYnUr8H3gcuB/ge+Y2aRetv8IcDzn8XTBIxQREREZxZLVSy+umLt4Wayk/ImWph1bj224s9eOK8VaARta0tK0fWvXQe9sm+HpDlL1dYNoRdzHOWEHqHnXhlbPdFbk+7jhvLDx5LxrTreMSu3ZfMDiiWcqqhYvy/f5Bqulafuxlr0P1lmidH/F3EXLKqqWXBF1TPmmJKyIiIiIiAw7bQ1BEra0agS3I352V/A8ecR3aJIzvgkc5UxS8r+Bg5ydqDweWXTnttbdv+vuDcDfAWOApb1sfzswPucxo/AhioiIiIw+yZraWRVzFy3D7Eiqcdvm5l0bWqOOaTCSVUvmWKL0qa7TdiSrlszA4hmLxY8M/iznnhMWwD09oGRtLFG2uxCtglMNdXvIZBLJ+ctPz2mTqq97NlZW+XjF3EUvy2cb5Hxp3r3xREvTji3x8jH1FVWLlyWrl15VjHEOhHpaiYiIyKCY2ed6WOVAK9AA/NTdXxi6qERkJPP2dtqfCm5qLquuPsfWw1i2HfH5qoQdLdy9KFu99cOj2Rfu3mxmJ4ApPW3s7m0E894CMEJ+ZxEREREpGsn5y8/3jrZqiycOtjTt2BJ1PPlQuWBlzD1zYcuezRu7rnN8Bpm0pxof3Nvdvv1z7jlhuzIz65oY7knz7o0vJmtqC/JlL9VQtyc5b/n8ZE3tnFR93V6A5ifuO1m5YOXm8jmLri+ftfD+1qd2pgtx7sE49fi9KWDLmIU3lpXPXVSbrFrSWTqtZse5KraLmSphRUREZLCuAt4BvAu4Lnz8WTh2I/A5oMHMLoksQhEZUdr374d0mlhlJYkpPeZ3hjf3nErY+dHGIgIZglbJuUq62a5ruzpHvzuIiIiIDLnKBSvLK+YuXk66c3JL044tg23NW0wy7a3LYqUVm7uOJ2tqZxGLt1gs/nw+ztPnOWFjQa2jJUqaKuYu7td8OVZSvj+3dXA+pfZs2g1Wlpx3zezsWPOuDR1lF85bb4nSayvnr6gsxHnz4dTOe9paGrdvsZKyR9oO7alNVi2prVyw8vT3itmr13X3XaQo6cuQiIiIDNZPgd8AF7r7IndfRNBO8NfAd4HpwP3A56MLUURGkrbGJgBKq6tGbuXcqaPQehwsBueN4GpfGS6eBaZlF8xsHDCn581FREREJAoTVr7ZkjW1td7ZcWnr3h2bUw1bn4w6pnxK1tRON7NDzbs2ZLqu80z6QstkylON2xoGe56yqdUVsURZS582NnshOX/5+an6uuOeSY/tz3man7jvMJl0we4sTtVv2UUmXZGbiD224U5vadqx3jvbL0/W1E4q1LnzoXnXho4wGbvNO9sXV1QtXnbRh366FmifvXrdHVHH1xdKwoqIiMhgfQj4uLufyA64+3HgNuBv3T0FfApYFE14IjLStDUG36nL5vbrJuPh5bndwfOEi6CkItpYROD/gLeY2bVmtpBg7tqia18mIiIiMpola2ovbz9UvyxeOXFHqqFuR1/b4g4XZmae7pydatz2klbDyXnXzLZY4gSx+PF8nCsxYercdMuJpr5s21K/Zb93ts8a8Mli8cPJmtrpA97/HFINW58kk65I1tSeFWOqcdtmT3fOSc5bfmGhzp0vzbs2ZFINW7dOvOHtOzpPPve2cHjVcKiIVRJWREREBms83c/3NhkYF74+BpQOVUAiMrK1h5WwZVUF6dpUHJ4Nk7CTF0Qbh4xmMaAzfH07cB/wC2Ad8BOgMZqwRERERCRXct41cyvmLlpmscS+VOO2zScfvqvz3HsNP+Vzrl4Sr5ywvbt1nu6Y6pnOcan6LbvycS7vbC9tfWpnW5+2HWSyO7Vn8z4ymRmDOcY5z9Gw9Umwscl518zMHW9p2rGDTOekZNWSYdHl5uDX/7yjZMLUteHi2n1rbuo6HUrRKfokrJmNNbMvmNl+M2sxs01mtuQc+1xvZg+aWZuZNZjZqiEKV0REZDT6KfBfZvZ6M5sRPl4PfIPgR1qApcCeqAIUkZGlrSlsR1w1githTydh50Ubh4xmU4DDAO5+wt3/2N3Hu/ssd/+mu1/p7rdlN3Z3c/ef5B7A3Se4+9qhDFpERERktEjW1E6tmLtoGenOlpamHVuad288ce69hqdk1ZIpYCdO7bznJYnRZE3tHEuUPW8WO5Wv8/V1Ptgss/jg5smJxV+ovPja8wZ1jHNI1W95HPdxXROxqYatjwHxZPXSiwt5/nzZt+amm4HS8LnoFX0SFvg68ErgLcBC4FfAb8ys2/JsM5tDcGfuvcCVwBeAr5vZq4YiWBERkVHo3cA9wPeA/eHje+HYe8JtdgHvjCQ6ERlRPJ2mvSlbCTuSk7DhDdznz482Dhl1zGyimb0WuJ5gzncRERERKSKVl1w3Nlm15BoymTEtTTu2pBq3HYo6pkJzz9S0NG3vtsrVOzumeHvLpDCZmBdmsYFVt1rsxeT85ef3d7dU/ZZ672yvHtA5+3eex/HMxJckYhu3NRCLn6qYu+jKQseQD8OhAjYrEXUAvTGzCuANwO+5+/3h8G1m9jvAe4GPdbPbe4C97v7BcPlJM1sJfAC4u9Axi4iIjDbufgr4MzP7AJDtDdoUjme3eTiK2ESk8Dqfe46Wx/L2Xfec0i+8iLe3Y2VllEwv2LQ50XsubB6gdsQy9P4LWAJ8lqDbhYiIiIgUgfJZC+OxkvKlmDWnGrdtjjqeoZKsrr3SYolHelhXZaUVR8h0jutu/cBZ/5KwYeVsS9P2fRVzrr4KeG4AJ21JVteOSTXU5a2itzup+rpHkzW1lydrajOp+rpnTo/v2XwgWVPbWlG1eFlL4/YthYxhNCnqJCxBfHGgtct4C7Cyh32u4aV3695NUBHbLTMrA8pyhsb2K0oRERHB3U+Z2QvZ11HHIyJDY/9b3kr73r1Dft7SOXOweHzIzzskWo7BqSPB6/NrIg1FRh93f33UMYiIiIhIYPbqdSX71tzUkaxeelWspLy0dFrNlmMb7hzUHKTDSeXFL5vgmXSmpWl7t78zuacnW2fGU43b6vJ7Zu9Xe2HHM4M9Y6q+7tFk1ZJaIM/vpYdzVS+9PDlvuaf2bDqYM/5s5SXXtVbMXfSy1r0PPjDY+W6lyJOw7n7SzDYDHzezJ4EjwJsIEq0NPew2Ndwu1xFgnJlVuHtLN/t8BLg1T2GLiIiMKmYWI+hO8UFgTDh2kqCC5h/dB/+HqIgUp0xLy+kEbPlll4ENbhqcvrJYjElvf/uQnCsSxw8Ez8nzoDzPN3SLiIiIiMiwMHv1ujs6Tzy3auqf/ss68D9INW7rLrcxomXami9tadqxsbt1yZplNRYvOUAmPSXf5x3AnLCnKuevGNe8e+MJYrGBfzGOl3RWLlhZ0rxrQ8Hb7aYatmYTsaT2bDqYTfg3P3HfycoFKzeXz1l0ffmshfe3PrUzXehYRrKiTsKG3kLQDukZIA08CHwXWJTHc9wOfC5neSzwdB6PLyIiMpL9I/AOYDWQ/cN4JXAbUA58NJqwRKTQOg4dBiCWTDL7/30fG6Ik7Ih3POwINW4Et1sWEREREZEezV69riTdempVrKyC8hkX31T+xts6o45pqFXMXXwpZt3OAwvgmc7zgPNam3bkvXK0v3PCtjRubaiorr2SIH81YKVTZj/YfqRxKUNQDQvZRGztlTP+4lufS4w9749mr163dt+am25u3rWhY8LKN69vO7j7usr5K7Y1797YPBTxjESxqAM4F3dvdPfrCCprZrr7UqAEaOphl8PABV3GLgBO9FAFi7u3ufuJ7AM4mafwRURERoO3Ae9096+4+6Ph48vAnwGrog1NRAqp8/AhABLTpikBm0/ZStjxM6ONQ0REREREIrFvzU0d8fIxa2NllQBr9625qeCVkcVkzKU3JMHLWxq3P9/d+mR17TwrKd8LpAvTMrd/c8K6e8YSJSUAFi/trJi7qHQgZ822mp6w8s1D9gV7yhs/8bh3tv9R+DGumr16XUk2lpamHeu9s/3yZE3tpKGKZ6Qp+iRslrs3u/shM5sIvAr4aQ+bbgZu7DL2ynBcRERE8m8S0N2dibvCdSIyQnUcCpKwJdOmRRzJCHMirIQdr0pYEREREZHRat+am24GSsPnUSWdOn51S9OOHT1u4JmJ3t5yUdm0edsLE0H/5oSdvXrdHRNe9tYts1evuyPTevJxiyUuGeiZ45UTd7Qfqs9nJ9he7VtzU0fJxGlrwxurX5LwTzVu2+zpzjnJecsvHKqYRpKiT8Ka2avM7NVmNsfMXgncS/Cj7h3h+tvN7Fs5u3wVmGtmnzazBWb258AfAp8f8uBFRERGh0eA93Uz/r5wnYiMUNl2xCXTpkYcyQhzPJwZZfyMaOMQEREREZFIjbYKWAjneo3FG3tcX7VkPvGSeovFT1eO5lPF7KsmWqL0RF+3DytHV4VVy6umvnlNzD094NzbyYfv6iQWjw90/4E4V8K/pWnHDjKdk5JVS+YMZVwjQdEnYYHxwJcIEq/fAjYAr3L37P/5TANmZTd2973ATQTVr48AHyRokXj3UAYtIiIyivwt8HYze8LMvhE+niBoRfyhaEMTkULqyGlHLHmkOWFFRERERGQUGnvlqxNk0uenGrcd6mkbz6QnkOmstkRpQapgraRsVjp1fF9ftw8T5WvjyfGQrST1/lXSdhUrrdiZrKm9fDDH6K9zJfxTDVsfA+LJ6qUXD1FII0LRJ2Hd/fvuXuXuZe4+zd3f5+7Hc9avcvfru+yz3t2vCvepcve1Qx23iIjIaOHu9wHzgB8DE8LHj4D57v5AdJGJSKF1HgzbEU9VEjavTqgSVkRERERERp/OE8/Wlk6r2dLT+mTVkkustHwXEGvetSFTiBg8k6b1qZ3p/uyT79bRpx6/N0UmU5GPY+VTqnFbA7H4qYq5i66MOpbhouiTsCIiIlL83P2gu3/U3d8QPj4GxMzsP6OOTUQKp+Ow2hHnXSYNJw4Gr5WEFRERERGRUSI575rZZrGne2sx7Pg4OjtqLFHa83yxg2RmA8qbda0ktXCS1QGLxeqT85bPH9QxCiC1Z/MBiyeeqahavCzqWIYDJWFFRESkUM4D3hF1ECJSGO6ek4RVJWzenDoKmU6wGIxRcltEREREREa+CSvfbN7ZPjPVuG1/T9skq5YsNIs/4Z5JNO/aULC5ct0zg0ueArGyysbyOYuqB3OMVH3dC3h6wmBjKYRUfd2zsUTZExVzF71s0MnmEU5JWBERERER6bfM8eN4KgVAYqqShXlzIpwPduw0iCeijUVERERERGQItB3aU+ud7Zt628bxSvCqWGnFw4WMxSzWYyVuXzU/ef8x8HGDjiVe8lTlgpWzB3ucQmjevfFErLRic/nsq64rn7UwHnU8xUpJWBERERER6bdsFWx80iRi5eURRzOCHD8QPKsVsYiIiIiIjALJqiXTDHuut3lYkzW1l8cSZTtxL23etaG1kPG4e0Hmmh2I5l0bDnm644Ko4+hJ864NHWXTF9xnidJrK+evqIw6nmKkJKyIiIiIiPRbx8FDAJSoCja/joeVsOOmRxuHiIiIiIjIEPBMuirVuK2h140ymQrPdFZZovSxQsdjFiu29rpHKxesLNo5gI5tuNNbmnas9872y5M1tZOijqfYqL+ViIiIDIiZ/egcm0wYijhEJBodh4MkbELzweZXth2xKmFFRERERGSEq7z42lqLx3f0tk2yeunlsfIxO9OtJy9P1dc1Fz4qz08S1mzQbY0BUvV1e5NVS2qBQ/k4XqGkGrdtrpi7aFFy3vLy1J5NB6OOp1ioElZEREQG6vg5HvuBb0UWnYgUVOehoB1xiZKw+aV2xCIiIiIiMgrM/Mvv3DnxhrdvmfLG277c0zZmZu6ZZLrl5ByLxZ8cyvgGy2LxF5Pzl5+fl4PFS45Vzl8xMS/HKqCWph07yHROSlYtmRN1LMVCSVgREREZEHe/uS+PqOMUkcLoOBS2I56mdsR5pXbEIiIiIiIyws1eva4E9zclxk0BWDV79bqS7rarqFpyZayk/GE8MyZVX3e80HEFSd/8zAmbqq/b653ts/JyrD2bdnumc14+jlVoqYatjwHxZPXSi6OOpRgoCSsiIiIiIv12uh2x5oTNr+NPB8+qhBURERERkRFq35qbOuJjJq4NF9fuW3NTR9dtwirYMu/smGWJkvqhiCs5f8XUWEnZ0aE41wC0jLn0hmTUQfRFqnFbA7H4qYq5i66MOpaoKQkrIiIiIiL91nkwWwl7YcSRjCCdbdAcft9XElZEREREREawfWtuuhkoDZ9foqJq6dW4P4hnJqbq614Yipi8s31qavfGw0Nxrv5K1dc9mmlrvizqOPoqtWfzAYsnnqmoWrws6liipCSsiIiIiIj0i6fTdBwNkoVqR5xHJw4Gz4lySJ4XbSwiIiIiIiIF1l0FLMCElW828BKLJWZaSfneoYzJ3X0oz9cvsXimfNbCeNRh9FWqvu7ZWKLsiYq5i15mZhZ1PFFQElZERERERPql87nnobMT4nESkydHHc7IkW1FPG46jM7vpyIiIiIiIrQfbrzaEqU73NOTm3c9MGTtgfM1H2yWWayzbGp1Il/HK50yZ0esrHJxvo43FJp3bzwRK63YXD77quuGUwI5X5SEFRERERGRfuk8FFRsJqZMwRJ5+z4pJ54JnsdPjzYOERERERGRiExY+WbDMwlPd8yweOlTQ3lus1he74bNtLc8HquceGm+jndsw51OJs1wqypt3rWho2z6gvssUXpt5fwVlVHHM5SUhBURERERkX7pOBxMkVMyVa2I8ypbCTt+ZrRxiIiIiIiIRKTt0J7FmY7W7Z5JX5Das+ng0J7d85rcbH1qZ5pMOq95OCspe7CiuvaqfB5zKBzbcKe3NO1Y753tlydraidFHc9QURJWRERERET6peNQmISdNi3iSEaY3HbEIiIiIiIio0zlgpUxw2Kx8jEXWrzkmajjyZfZq9eV5OtYzbs2dAB5O95QSzVu2+zpzjnJmtpR8cVXvcNERERERKRfOrLtiKepEjav1I5YBICphx5k3Ilk1GEUtUPTl0Qdgsio8tzEqqhDGBbGRR2AiAx73tG2xErKtnlH25JU47a6IT9/nueEBZjwsrfeBvzu7NXr1u5bc9PN+TimlZQ9kZy//LLU7k2P5eN4Q62laceOZPXSy5JVS0pTjdv2Rh1PIakSVkRERERE+qXzdCXshRFHMsIczyZhZ0Qbh4iIiIiIyBArn7Uwjpl7JjOVWOxoFDHke07Y2avXlZRMuvB3w8VV+aqIbX7ivpNk0sN6btVUw9bHgHiyeunFUcdSSErCioiIiIhIv3QcOgRAiSph8+t0O2IlYUVEREREZHSJlZQvLp1avc3T7bNS9XURVUfmd07YfWtu6oiVVa4NF9fuW3NTR94OHos3JmuW1eTteBFINW5rIBY/VTF30ZVRx1IoSsKKiIiIiEi/dBwOKmETU5WEzZu2k9B2PHitdsQiIiIiIjKKVC5YWQKk2w/VTzaLPxtFDEElbizv7YjDFsSl+WpFnJXavek5MulJ+TxmFFJ7Nh+weOKZiqrFy6KOpRCUhBURERERkT7LtLeTfu45AEouVDvivMm2Ii4fD2Vjo41FRERERERkCHlH29Wl02p2OD431VDXGEUM8eT4mZ7u3F+IY+e1AjaHJUqfqbz42lmFOPZQStXXPRtLlD1RMXfRy8wsr9XIUVMSVkRERIqGmb3MzH5uZgfNzM3sdVHHJCJn6wyrYK2sjPiECdEGM5KoFbGIiIiIiIxClQtWlmCxzo7nD0yyeOmLUcWR6Wgb39K0/VhU5x+I5t0bn/bO9mlRx5EPzbs3noiVVmwun33VdeWzFsajjidflIQVERGRYlIJPAL8RdSBiEj3Og4FSdiSqVMZYTeoRutEmIRVK2IRERERERlFMh2ti1oatz7o7a3VqT2bdkcVh5kNz3yZxZ5Nzlt+QdRh5EPzrg0dZdMX3Gfxkmsr56+ojDqefEhEHYCIiIhIlrv/EvglkPfkTvrYMTKpVF6PKTIate3eBUDiwhFxs+0Z7nDimeA5CkeDz5XxqoQVEREREZHRoXLBynKzWFtF9dJxYKeijmc4Su3Z3JSsXroUOBJ1LPlwbMOdDqyvmLt4ebKmdleqvu6FqGMajKJOwppZHLgN+FNgKnAQWAv8g3v3v46Y2fXAvd2smubuhwsRp4iIiETDzMqAspyhHidSfPbf/p0X77yz8EGJjBIlU0dYEvYHb4fHfxR1FDBOlbAiIiIiIjI6ZDpar2xt2lFXPndRbUvj9i1RxuLumSjPPxgWLzlZefHLJjQ/ef+xqGPJl5am7Zsq5i5alKyprZjyhk8cLdS8uoVW1ElY4MPAe4G3AY8Di4E7gOPAv51j3/nAiZzlo4UIUERERCL1EeDWvmxoJQmsrOzcG4rIOcUqKhj36ldFHUZ+7d8YPMdLIaouVBUTYd6rozm3iIiIiIjIEBpz6Q1JsJbkvOVJz3SqddkgNO/e+GSyprYWqIs6lnxqadqx48K3f+nH6eZjr5u9et3afWtuujnqmPqr2JOwy4Gfuvu6cHmfmb0JWNqHfY+6+7GCRSYiIpF79tln+cQnPsG6des4cuQIEydO5MiRI3cDn3L3jWbmwOvd/Sf9Oa6Z7QO+4O5fyBmbD3wVuAQYT9Cd4U7gk+4+LO/EGiFuBz6XszwWeLq7DS/4yEe44CMfGZKgRGQYajkWPP/lDpgwK9JQRERERERERrpM66mFLY3b6pJVS2pTjdsiTx4O2zlhs5y2ygUry5t3bWiNOpR8mb16XUnplNmvCxdXzV697l3DrSK22P9RbQJuNLN5AGZ2BbCScK64c3jYzA6Z2a/NbEVvG5pZmZmNyz7opZWhiIgUjze84Q089NBDfPOb32TPnj387Gc/A1gPnFeA03UA3wJ+i6Dbwi3AnwGfLMC5pI/cvc3dT2QfwMmoYxKRYaijBdJtwevyCZGGIiIiIiIiMtJVzl9RSSzWUrlgZTlm7VHHMxK0NG59xDvbL486jnwKE65rw8W1wy0BC8WfhF0DfA/YZWYdwEMElUnf6WWfQ8B7gDeEjwPAejO7upd9PkLQ4jj76LaCRkREisexY8d44IEH+Od//mduuOEGLrroIpYuXYq73+7uPwurWQF+bGZuZvvMLGlmd5lZRzh2ysy2mdkrssc1s/XARcDnw20cwN2b3P0Od3/E3fe7+8+A7wDXDvFbFxGRfGt5MXi2OJTpfkwREREREZFC8s72y1L1dY96R9sVLY3bHo46Hhjec8ICuLsTi1O5YGWx5/36JWxBXDocWxFD8Sdh/xD4E+DNwNUEc8P+jZm9racd3H23u/+Hu+9w903u/naCitoP9HKe2wlaS2YfM/L1BkREpDDGjBlDIpHg7W9/O21tbafHzeyNZtYK/Fc49GtgGrCE4DqyFPgn4HrgKuAu4OdmNsvMvgpcBxwDPhHuN62785tZNfBq4L68v7lRzMzGmNmVZnZlODQnXFZvUBEpnGwr4ooJYBZlJCIiIiIiIiNa5fwV4zBrrlywsgRIu7tHHdNIYbH4ds+kF0cdR74NxwrYrGJPwn4GWOPu33P3ne7+beDzBJWr/bEVqO5ppVoZiogMP4lEgpUrV3LgwAEmTJjAihUreM1rXgPBPK3vdfdPhZu2ufthd38WqAJ2uvut7n6fu9e7+8eBRuBjwDKCuV4dOBnudzj3vGa2KUzy1gMPECRrJX8WE3S+eChc/lz4+lM97iEiMlitx4JntSIWEREREREpqExn2yWphq2PZTpaF5VOq9kRdTwA5bMWllmiJB11HIPVvGtDhkwaM91dXCyKPQmbBLqWgKfpf9xXErQpFhGRYa7h6EkeqH+WIydaueiii3jVq17Fz372MyorK7nrrrsA4gRJ1LOEbYY/CLwsbDPcbGbHzOwUcDHwJoLuC+e6s+qPCLozvBm4CfibvL05wd3Xu7t181gVdWwiMoLlVsKKiIiIiIhIQVRe/LIJhp2oXLAyZpgf23BnUVTBxivGzfW2VFPUceRDrGLsIxVVS6+IOg4JJKIO4Bx+DnzUzJ4CHidoG/nXnGkxiZndDkx397eGy7cAe8Pty4F3Ai8HfmtIIxcRkbw6fLyVv7jzQXbsD+btixlMHreYC47/H7/5zW/YvHkzv/71r3nFK15xB/BJzkzanvX7BHON/y7QCnwceDh8/SCww90fP9eNYu5+IHz5hJnFgf80s8+6+7C/W05EZNRSJayIiIiIiEjBZdpTC1oat29JVi9daiVl26KOJyvT2V7asu/hVNRx5MOpnfe0JWuWlUUdhwSKPQn7l8DfA18GphC0iPwPzm5JOA3InSeuFPgsMB1IAY8Cr3D3e4ciYBERyT935213bKXh6KnTYxmHIxWzaWq5kJ9/+tPcc889vPzlLwd4AnhddlfAwmO8YGYpYAzwpbDFPWZ2K8FNOw/nnLKToKL2XGJASfisJKyIyHB1uhJ2YqRhiIiIiIiIjFSV81dMtFji2ISVbzbMrHnXhq5dUCVPLBZ/snLBykubd214POpYRruiTsK6+0nglvDR0zaruix/Gvh0IeMSEZFzc3c60k5HOkNHOkN7Z4b2dOb0WHtnJlwXLqczdHRm6Mz4WeMd6Qx7jpxk9+Fupus2o3LhKynddRcf/vCHef/73w/wt8BPwy1agZlmNhVoC8dagN83s58DCwjmGW/tcuQXCNoWf49gTtnnzCzbqnhneKzFwO3A/7j7sJ0cXkREgJagy4LaEYuIiIiIiBSGd7bPSzVuq6u8+NraeOXEopgLNsvMin3qzn5p3r3xRLJm2Zio45AiT8KKiMi5ZTJOezpDW2eQ2GzrTJ9OeLZ3nnm0dVnOJj6DfXISpTnr2nKXO4PEaHu669hLx7Nj+RQjw83xu3hb/FdMsRfZ5bP4Uufv8WsWk6KUHTt28O53vxvgm8Bt4W6NBN0SDgDPAD8Jx1LAJoJkailQBrzfzN5HUAGbAeaG25YRVNN2Ah8G5oXL+4EvAp/P6xsVEZGhp3bEIiIiIiIiBVN58bXnYbEXL/rwz++YsPJPVpWcN3MtcHPUcY1kFi/Zm5x3zdzUns0jYq7b4UpJWBGRQXB32sIkZltHOnydprUjc/p1sC7n9VnbhuMdL02i5i6fef3SRGu+k52FYgYl8Rhl8RgliRglcSMRi1GWiFESj5GIGyXxGKXxGCWJYF1pIlh+IdXOa/f/M2+K/x8QzAe7kCa+Vvo5Ptj+Lj7xxCOceP4wN9xwA42NjSs5c317HrjH3W8JYrAvAK3u/vJw+TyCtva57ga+Ddzh7ruzg+7+P8D/FO4TEhGRyJxuRzwhyihERERERERGJG9vrZ7yB7c9mGlrXpUYPxVg1ezV6961b81NRdFdzt1HXGvk5l0PHE1WL50NKAkbISVhRWTEyGSc1jAB2tqRpqUjTWtHmBDtSJ9e15azTfa5rTP7HCREu26b+9yWs09bZ/Fdn0sTQaKzNJHziL/0dVmX5ZKu2/Q0njNWErezxoKHnd42ET+TZI3HbMDvKfNcI7Ev/t9ZY3Fz3OHv4t9hfNntjJ85k/Xr1zNz5swpwN1m9upzHdfdnydI1J5mZh3A4dwErIiIjHCqhBURERERESmIZE3tZCz23L41N3XMXr1uLbAKWFssCdgRLRY/VDl/xYzm3RufjjqU0UpJWBEpqGylaEt7kBQ9kxhN09Ke6WYsSHbmjp1JqAbjbR3dbBNWiEbJDMoTccpLYpQl4pSVBInI8pL4mYRnInZ6myAR2mVdzrZd15V1GctNopaVxIMK0rhhNvBkZ8F1tgfz7rW8GPzgnX3dcixnOfv6GLQeh9ZjxJqf7/ZwZnBevBmeb4ApC5gxYwbA9cC9BBWtZUPwrkREZLhTJayIiIiIiEhBeCZd1dK4fQvAvjU33VxMFbBZI21O2KzUns0HkjXLagElYSOiJKzIKJZNkKayCdL2ztOJ0VR75+nkZ6o9mxxNn942mzANkqgZWnOSrNlts8seQbfcIDEZJDXLS7KJz/B1SZyyRO7rs5+zSdTcbcvCY5SVxF6yfTaxWvQJ0HzKpIOEaeoFaHnhzPNLxsKkajbZ2tFcmHhKK0+/dPdnCOZtfYlsW+LeuPvsfIUlIiLDRMuLwXPFxGjjEBERERERGUGS85ZfYPGSo7ljxZaAHfFiseeT85efn9q96bmoQxmNlIQVKWK5SdJUeyct7emchGn6rORpqtt1Z/bJTaDmVqUOZYK0JG5UlATJ0IrSOOWJ4LkiTHxmx8pPr4ud3qas5Mx22bEzydLs8bJJ1PigWt+OOplMUHmaegFSz+c8ngufc8fD163HgYH+47Gg0qhiYtD2sWJisFw+oefneAl87eXQ0XL2eS0O06+GCTMH+u5FRETUjlhERERERKQQ0h2zU43b6qIO41xG4pywWandmxqS1UuXzl697rgS4ENPSViRQcpk/Kxq0VRH5+nXzW2dZ61r6QjGzmzbJYHanlt5GuybGaIkaWk8SIImTydFw9elOa9zEqjJkjPrKsL12eRpdr+KkqByNFmaoDwRzA8qQ8A9+DG5+Tlofjbn8XzwnHouWJd6/syzpwd2rrLxkJwIFZMgOamb54lnEq7ZpGv5BIgN4N/C7/8n/L9VwfszguRxxQT4vS8NLHYREREIritqRywiIiIiIpJXlQtWTiMWO3ruLaNVNv3iMfExEwvUvq84TLj+5o94uvN1s1evW7tvzU03Rx3PaKIkrIwK6YyfVUmarRJNnV7OSYy2n0mCpk4nStPd7J9t2Ts0N8mUJmIkw+RneZgsTZYkgtc5CdPgOXE6MZpNhmaTqME2wfry0tjp16ocLXLpziB5eupo8Gh+Fpqzr587s9wcJlgzA7ipqXQsVJ4HyfMgeX6QRE1ml3Ne5yZY4yX5f689ufh34C93wMN3woln4ILL4Io/VutIEREZnI7UmeumKmFFRERERETywjvaZg2HKth45YQ56eNHn4w6jkKZvXpdSen5s14XLq4qxjl5RzIlYaUoZNvuZlvmnl0VmpP8zKkc7SmhelbCNDxee+fQJEqTpTnJ0JLE6aTomcRokPTMHUvmVJ8mSxNnJU1zk6iqIh2B3IM56E4dgZOHwwTrkfBx9EyS9dTRoFq1v+1/y8YHSdXKycEjeR5Unh++Pj9MuJ4fjCXPg0RZQd5mXk2cDTf8XdRRiIjISJKtgo0lzppjXERERERERAYmOW/5hcQTR6KOoy+8sz3edrihM+o4CmXfmps6Zq9etxZYBaxVAnZoKQkrfdJdkvTsOUY7z55v9Ky2umeSp11b8+YmTYei7a4ZYRvdM8nPZOmZ5Gd2ubwkTmXpS5OoZ1ecJk6/rixNUF4Sw0zVpAJk0kFl6snDYXL1MJw80s3zkf5VrFosSJqOmRImU6eEryefea6cfCbROhySqiIiIlHLzgdbMTH4Y1FEREREREQGxdPts1oat2+JOo4+sdgQTQgYnX1rbrpZFbDRUBJ2BEhnnNaOICnakn1uz5x53ZGTMG3P3SYnadpx5nV2n2yiNbt+qOYmLYnb6arQbEI0t9Xu6YRpSZxkWeKsxOmZqtLE2VWp4bHKEkqUyiBkK1dPHAwTrNnnQ2c/nzoC/ZnLvWIijLkgSKSOmRo+Tzkzlk22Js+DWLxw709ERGQ0ankxeFYrYpEBM7O1wAR3f13EoYiIiIhIxJLzrplpsfjBqOPoq1hp+ajIkykBG41R8Y8rCp3pDK1h5eiZR+as52wytLUzQ1s2AdoZJFBbO9O0nl4O9mnpcqxs4nSoWu1m5SZJz5pvtJtEaEU452jueHk385Pmtt0tUdtdiUK6I0iinjgUJFdP5DxOHjqTeE239e14FgsrVC+AsdNg7AVBgvX089QzSVZVrIqIiEQn2464YkKUUYgMiTBZ+rZuVtW4e8MQhyMiIiIiI5CnO6YPlyrY2avX3THhZW9bNXv1urX71tx0c9TxyMijJGw/fernT7Dv+ebTydC2zjOJ1bbOMwnWzqEqG+2iNBE7k+AsiVOWk/AsL4mdnpc0WD6T+HzJc/Z1l+VyJUllOOpsP5NYPf4MnMg+Dp55PnWUPs+5mjwPxl4YJFLHTQuSquOmhcnWqcFy5WSI6/9iRUREil62HbEqYWX0uAvo+gPTs1EEIiIiIiIjS7KmdpbF4geijqMvZq9eVwKsipUlAVapXa8UgjIE/VS393keP3iiX/uUJWKUZ5OgYfKzrCROeTieTZCWh+tyt60ojVOeiFNeevZ2uUnU8kRQbVqWiBGLqdWujDKZdND+9/gzcPwAHH86SKyefn4Gmo/27VixkiCZOm56kFAdd2H4PC1IumYTrapcFRERGTlUCSujT5u7H+46aGbXAZ8BrgBeAL4JfMzdO8P1bwRuBaqBFPAQ8Hvu3pxzjFuB9wFlwJ3A+929vbBvR0RERESKRiYzLdW4rS7qMPpi35qbOmavXrc2nhy/ClirBKwUgpKw/fS+G6o52dpJWUmMskTX5GksSJjmJEtL40qMigxK28kgoXrswJkka/Zx4umgijXTee7jxMuCpOq46TB+evCcXc4+J8+DmCq9RURERhVVwopgZtOB/wXWAm8FFgBfA1qB28xsGvBd4G+BHwNjgWuB3C+7N4bbXw/MBu4Angc+2st5ywgStllj8/B2RERERCQCyXnXzCZeMiyqYLP2rbnpZlXASiEpCdtPv71wWtQhiIwc7tD8HBx/6kyS9aznp6D1+LmPY/GzE6zjZ8C4GWeSreNnBAlW0w0RIiIi0kXLi8FzxcRo4xAZOq81s1M5y78E9gAHgPe5uwO7zOxC4J/N7FPANILfD37k7vvD/XZ2OW478HZ3TwGPm9kngM+Y2cfdPdNDLB8hqK4VERERkeEu3XnBcKmCzaUErBSSkrAiUjiZTNAq+NhTYWJ1f5BcPfZUOPY0dLac+zjlE4JE6viZ4fOMs5fHXKD5V0VERGRg1I5YRp97gffmLDcDXwI2hwnYrI3AGGAG8AhwD7DTzO4GfgX8wN1fzNn+kTABm7U53H8msJ/u3Q58Lmd5LPB0v9+RiIiIiEQqOe+aucRiPf3NJzJqKWshIgPnDqeOhsnVp4LnF/fnJFkPQPpcU0BZMM/qhJlnkqqnX4fL5eOG5O2IiIjIKKR2xDL6NLt7Q+6AnaNjjLunzeyVwHLgt4C/BP7RzGrdfe9AA3H3NqCtr3GIiIiISHHydMeUlsbtW6KOQ6TYKAkrIj1zD1r0nU6u5jxnE62drb0fw2JBa+AJs4Lk6oRZwWP8zGB53AxIlA7N+xERERHpSpWwIgBPAm8wM8uphl0BnCSsTA3HNwIbwxbF+4HXc6aS9Qozq3D3bKubZcApgjbHIiIiIjJCJatrq2IlFU1RxyFSjJSEFRntOlqCZOqL++HFfWGidd+ZZGvbid73txiMvRAmXgQTLgoTrRedSbaOuxDiJUPxTkRERET6T5WwIgBfBm4B/t3MvgjMBz4JfM7dM2ZWC9xI0Ib4KFALTCZI3maVAt8ws38AZof7f7GX+WBFREREZASwRMkFzbseaIw6DpFipCSsyEiXycCpw2FiNfcRJltPHT73McZcECRWTydaZ515PW66KllFRERk+DpdCTsx0jBEouTuz5jZa4DPEMz/+gLwDeAfwk1OAC8jSNSOI6iC/aC7/zLnMPcA9cD9QBnwXeC2IQhfRERERCIy/Z1f/dF5v/1Xr5+9et3afWtuujnqeESKjZKwIiNB26mcCtauj/2Qbut1d8rGnUmyTpwdPE4nXWdBSUWB34CIiIhIBLJTL4DaEcuo4O6rell3H7C0h3VPAq/u43FvHWB4IiIiIjKMzF69riQx6cLXWywOsGr26nXv2rfmpo6o4xIpJkrCigwHmQycOgIv7j2TXH0h53Xz0d73t/iZNsGT5nRJtM4OKj/MCvwmRERERIpM+ynwdPBa7YhFRERERET6bN+amzpmr163FlgFrFUCVuSllIQVKRadbWGL4L05SdacRGtna+/7l08IEqynE63ZZOtFMG4GxPWfu4iIiMhZsq2I46Xq/CEiIiIiItJP+9bcdLMqYEV6VtRZGTOLE8wh86fAVOAgsBb4B3f3Xva7HvgccClwINx+bUGDFTmXbLu7F/eenWB9YV/w+sRBoMd/1kE16/gZOZWsOUnWbDWriIiIiPRd67HguXyCuoKIiIiIiIgMgBKwIj0r6iQs8GHgvcDbgMeBxcAdwHHg37rbwczmAOuArwJ/AtwIfN3MDrn73UMRtIximTSceCanVXBOwvWFfdB2vPf9S8cEydVJs7skWmcHc7PGSwr9DkRERERGj2wlrOaDFREREREREZE8K/Yk7HLgp+6+LlzeZ2ZvApb2ss97gL3u/sFw+UkzWwl8AOg2CWtmZUBZztDYwYUtI1p7qkuCNef1sacgc44bf8ZMPdMuOLdt8KQ5kDxPVRgiIiIiQ6XlxeBZHUVEREREREREJM+KPQm7CXiXmc1z9z1mdgWwEvjrXva5BvhNl7G7gS/0ss9HgFsHE6iMIO7Q/Gw31azh61NHet8/VhK2CM5JrmYTrhMugtLkELwJERERETmn3HbEIiIiIiIiIiJ5VOxJ2DXAOGCXmaWBOPBRd/9OL/tMBbpmyY4A48yswt1butnndoI5ZLPGAk8PPGwpep1tcOzAmXlZX9yXk2jdBx3Nve9fPr77JOvEOTDuQojFC/4WRERERGSQ1I5YRERERERERAqk2JOwf0gwr+ubCeaEvRL4gpkddPdv5usk7t4GtGWXTe1ghz93SD0PL+4PE63ZBOv+INl64hnAezmAwbjpYWJ19tnJ1omzITlpKN6FiIiIiBSSKmFFREREREREpECKPQn7GWCNu38vXN5pZhcRtA/uKQl7GLigy9gFwIkeqmBluOpoDeZgfXEfHNt/poo1+2g/1fv+JckzSdWuidYJMyFR1vv+IiIiIjK8qRJWRERERERERAqk2JOwSSDTZSwNxHrZZzPwmi5jrwzHZTjJpOHkoaB69dj+sKp135mE68lD5z7G2Gk9JFpnQ+VkUNWziIiIyOilSlgRERERERERKZBiT8L+HPiomT1F0I74KuCvgf/KbmBmtwPT3f2t4dBXgfeZ2afD7V5O0Nb4pqEMXPogt2XwsX1nJ1uP7Q/mbM109H6M0jFnJ1gnzoYJF4XPs6CkvNDvQkRERESGq5YXg+eKidHGISIiIiIiIiIjTrEnYf8S+Hvgy8AU4CDwH8CncraZBszKLrj7XjO7Cfg88FfA08A73f3uoQpaQu7BD1unE6tPvfTR0dz7MWIJGD/jTGJ14kXB60lzYMLsYG5WVbOKiIiIyECoHbGIiIiIiIiIFEhRJ2Hd/SRwS/joaZtV3YytJ6ialUJyh+Zng4rV49nE6oHg+Xj4fK55WbGwZfBFQeXqhIvOJFonXgRjL4R4Uf8zFREREZHhSu2IRURERERERKRAlN2SnqU7gnlXjx0IkqrHD5x5nX3ubD33ccZMDROsM88kWk8/z4REWeHfi4iIiIhIV6qEFREREREREZECURJ2tMq2Cj7xDBx/uvvHyYPgmXMcyGBsNsk6C8bPPPN6wkVBK2HNyyoiIiIixSaTUSWsiIiIiIiIiBSMkrAjVdspOHEQTjwNx585O9l64plg7FzzsQLES4NE6vgZMD6sZh0/I0y2zoRxMyBRWvj3IyIiIiKST+0nz9xwqEpYEREREREREckzJWGHG3doOwEnDgXJ1BMHw8czOc/PQOvxvh0veT6Mnx4kVcdNPzvJOn4mVE6GWKyw70lEREREZKhlWxEnyqGkItJQRERERERERGTkURK2mKQ74NRROHk4aAV84lCX5/B1XypYAcrGwbgLg+Rqtpp13PScpOuF+sFJRESKkpn9BfAhYCrwCPCX7r412qhEZERRK2IRERERERERKSAlYYdCJg3Nz4bJ1cNw6vCZ1ycPhY/DQQIW79sxyyeECdYLYey0MwnWcdOCFsHjLoTycYV8VyIiIgVhZn8EfA54D1AH3ALcbWbz3f1olLGJyAiSrYRVK2IRERERERERKQAlYQejPQXNR+HkETiV8zh5+Ozn5mfPzDd1LrEEjJkKY6cGCdWxF4avswnW6cFyaWVh35uIiEh0/hr4mrvfAWBm7wFuAt4OrIkyMBEZQVQJKyIiIiIiIiIFpCRsf33/bXB4Z1C12n6y7/tZLJhfdezUoHJ17NQg2TpuWrA85oKgejV5vuZgFRGRUcvMSoFFwO3ZMXfPmNlvgGu62b4MKMsZGtvjwe//F3jsR/kLVkSGt2wSVpWwIiIiIiIiIlIASsL21/ED8ELjmeVEeZBAHXMBjJkSPI+d+tLnyskQi0cXt4iIyPBwPhAHjnQZPwIs6Gb7jwC39unIJw/D0ccHFZyIjECTu/u/FhERERERERGRwVEStr9+6x+D1sLZpGvZWDCLOioREZHR6naC+WOzxgJPd7vl0nfBgpuGIiYRGS4S5TBjSdRRiIiIiIiIiMgIpCRsf130kk6IIiIikj/PAWnggi7jFwCHu27s7m1AW3bZersxavK84CEiIiIiIiIiIiJSYJp8VERERIqGu7cDO4Abs2NmFguXN0cVl4iIiIiIiIiIiEh/qBK2FydOnIg6BBER6afx48ePA066u0cdiwzY54Bvmtl2YCtwC1AJ3NHXA+gaLiIyPOk6LgAnUy1Rh1D0Tp48GXUIRS/l6ahDGBb0b6lvjEzUIQwLZma6houIiJxhui6+lJlNp6f55EREZDgY7+7Kwg1jZvY+4EPAVOBh4P3uXteH/XQNFxEZ/nQdH6V0HRcRGfZ0DRcREcmhJGw3LJhQ7kKgu9sBxxJ8KZzRw3oZHH2+hafPuLD0+RZWXz9fVdCMUrqGR06fcWHp8y0sfb6F1Z/PV9fxUeoc1/Eo6P8X+kafU9/oczo3fUZ9U8yfk67hIiIiOdSOuBvhHwvPdLcu+E4IBH9U6M6uPNPnW3j6jAtLn29h6fOVc9E1PFr6jAtLn29h6fMtLH2+0he9XcejoH+3faPPqW/0OZ2bPqO+0eckIiIyfMSiDkBEREREREREREREREREZCRRElZEREREREREREREREREJI+UhO2/NuCT4bPknz7fwtNnXFj6fAtLn68Mhv79FJ4+48LS51tY+nwLS5+vDEf6d9s3+pz6Rp/Tuekz6ht9TiIiIsOEaa50EREREREREREREREREZH8USWsiIiIiIiIiIiIiIiIiEgeKQkrIiIiIiIiIiIiIiIiIpJHSsKKiIiIiIiIiIiIiIiIiOSRkrAiIiIiIiIiIiIiIiIiInmkJKyIiIiIiIiIiIiIiIiISB4pCdtPZvYXZrbPzFrNrM7MlkYd00hhZi8zs5+b2UEzczN7XdQxjRRm9hEz22ZmJ83sqJn9xMzmRx3XSGJm7zWzR83sRPjYbGa/HXVcI5GZrQ7/P+ILUcciw4uu4YWja3jh6BpeeLqGDy1dx0VERERERGS0UBK2H8zsj4DPAZ8ErgYeAe42symRBjZyVBJ8pn8RdSAj0HXAl4BlwCuBEuBXZlYZaVQjy9PAamARsBj4P+CnZnZppFGNMGa2BHg38GjUscjwomt4wekaXji6hheeruFDRNdxERERERERGU3M3aOOYdgwszpgm7u/L1yOAQeAf3f3NZEGN8KYmQOvd/efRB3LSGRmk4GjwHXufn/U8YxUZvYC8CF3/0bUsYwEZjYGeBD4c+BjwMPufkukQcmwoWv40NE1vLB0DR8auobnn67jMhyFN7z8IVANHAK+6+7PRxuVjARmlgAudPenoo4lSmb2QeAH7r4/6liGKzO7GFjn7nOjjkVEREReSpWwfWRmpQR3x/8mO+bumXD5mqjiEhmg8eHzC5FGMUKZWdzM/pigMmxz1PGMIF8i+HL5m3NuKZJD13AZYXQNLyBdwwtK13Epemb2hJlNCl/PBB4DPk/QieCTwBNmNifCECNnZkvNLJ6z/Fozu8/MnjGz7Wb21ijjG0YuBfZGHUQR+AzQaGa/NrM/Cv9ul/4pBS6KOggRERHpXiLqAIaR84E4cKTL+BFgwdCHIzIwYfXXF4CN7v5YxOGMKGa2kOAH23LgFEEl2BPRRjUyhD+IXw0siToWGZZ0DZcRQdfwwtE1vLB0HZdhZAFnfie5HTgIXOnux8Nq7h8D/wi8OaL4isFmYBpw1Mx+B/gJ8N/A/wBXAd8ws5Pu/uPoQpRh5p3A64BvAyfM7L+Br+tvnYCZfe4cm0wekkBERERkQJSEFRl9vgRcBqyMOpARaDdwJUGV0huBb5rZdfoRd3DCKoR/BV7p7q1RxyMiEiFdwwtH1/AC0XVchrFrgPe4+3EAdz9lZrcC34s2rMhZzuu/BT7t7h85vdJsbzg+qpOwZvbgOTapGJJAhof/dfe1ZjYFWAXcDPylme0AvgZ8z91PRhlgxP4KeBg40cP6MUMXioiIiPSXkrB99xyQBi7oMn4BcHjowxHpPzP7IvBa4GXu/nTU8Yw07t4ONISLO8xsCcEXpndHF9WIsAiYAjxodvo3nzjwMjN7H1Dm7umogpNhQddwGfZ0DS8sXcMLStdxGW48fC4nmAc21zOo6izXPOCWLmM/BD409KEUnUsIEvY9tRyeRvD5ScjdjwKfBj5tZtcC7yBoB/55RneisQH4vLv/d3crzexKYMeQRiQiIiJ9piRsH7l7e3gX3o0E7XayLeFuBL4YYWgi52TBL17/DrweuN7dNffM0IgBZVEHMQLcAyzsMnYHsAv4Z/1wK+eia7gMZ7qGR0bX8PzRdVyGm3vMrBMYB8wnmBc26yLg+UiiKi6XmNlUoIXg/y+70m9Nwb+bOnf/Sncrw8TZnw1pRMXJux10fwB4wMzeD/zR0IZUdLYT3NDUbRKW4DO0HtaJiIhIxPSHcf98jqA12XZgK8Edn5UEPyLIIIVz7FTnDM0Jv5i84O5PRRPViPElgnmLfg84GX5hBjju7i3RhTVymNntwC+Bp4CxBJ/39cCrIgxrRAhbT501H5CZNQPPa54g6QddwwtI1/CC0jW8wHQNLyxdx2WY+WSX5VNdln8HeGCIYilm93Am6bMC2Jaz7iqC/z8d7TYSJPF7chK4f4hiKWa9Jg/d/QRBS+LR7IP0cmOYuz9C9zdDiIiISBEw925vOpMehC2zPgRMJZiT4f3uXhdpUCOEmV0P3NvNqm+6+6ohDWaEMbOe/kO/2d3XDmUsI5WZfYOgqm4acBx4lKC649eRBjZCmdl64GF3vyXiUGQY0TW8cHQNLxxdwwtP1/Chp+u4yPBlZhd1GTrl7s/nrH8rgLt/a0gDExEREREpQkrCioiIiIiIiIhIn5hZgqBrwN3ufiTqeIqZmc0CDniXH9/C6QZmqmOIiIiIyMimJKyIiIiIiIiIjHpmthP4PrDW3Q9EHU8xM7MUcLG77486lmJmZmlgmrsf7TJ+HnDU3ePRRFYczOxq4MXsnPdm9hbgPcAsYD/wRXf/XoQhRsrMXqSHeXO7cvdJBQ5HREREBkBzBoiIiIiIiIiIwKXAXwF7zewuM3tDWPUpL7UVuDLqIIYBo/sk2higdYhjKUZ3AFUAZvZO4D+A7cA/Esw1/DUze3t04UXuFuAD4eMfwrG7gdvCx93h2N8PcVwiIiLSR6qEFREREREREZFRz8wywAxgKfB24LeBF4FvAd9w9ycjDK+omNkfArcDnwd2AM2569390SjiKhZm9rnw5V8BXwNSOavjQC2QdvcVQx1bMcmtqDazB4GvuPvXcta/Gfiou18aWZBFwsx+CNzr7l/sMv4+4BXu/rpIAhMREZFeKQkrIiIiIiIiIqNemISdmm0da2bTgFXAzQTVenXA1939vyILskiEn1VXTlj5qTa7dm/48jpgM9Ces7od2Af8i7vXD3FoRcXMngNe5e47zOwI8Fvu/kjO+ipgp7snIwuySJjZKeBKd2/oMl4NPOzuY6KJTERERHqjJKyIiIiIiIiIjHo9zd8ZrrseeAfweiU7wMwu6m295ooNmNkdwF+5+4moYylGZvZtoM3d32lm3wd2u/vHc9Z/BHiTu18eWZBFwsz2A//m7p/tMv5B4P3u3ut/kyIiIhINJWFFREREREREZNTrWgnbwzbjlFCTgTKzccDLgV3uvivqeKJmZhcCG4GnCOaCfS9Be+sngfnAMoIbH/43siCLhJmtAr4O/JKgKh+CttavBv7M3ddGE5mIiIj0RklYERERERERERn1wqrF97v7yahjKXZm9rs9rHKgFWhw971DGFJRCqs773f3L5pZBfAIMJugbfMfu/sPo4yvGJjZBGA18DvAXCAGHCJIzn7e3bdHF11xMbNa4P3AxeHQkwTVsXU97yUiIiJRUhJWZAQLW2bdC0x092PD5dgiIiKi67iIiBSvsGo4OwdsrtPzwgIbgNe5+4tDHF7RMLPDBHOePmJmbwY+CVwBvA14l7tfFWmAIiIiIlJQsagDEBlJzGytmXn4aDezBjP7hJkloo5tMMxsvZl9ocvwJmAacHzoIxIREck/XcdFRATAzJrM7LxuxieYWVMUMRWhVwLbwufx4eOVBG1SXwu8DDgP+JeoAiwS44EXwtevBn7o7ilgHVATWVQy7JjZa8zsVd2Mv8rMfjuKmEREROTchvUPSiJF6i7gZqAMeA3wJaADuD3KoPLN3duBw1HHISIikme6jouIyGwg3s14GTB9aEMpWv9KUMm5KWfsHjNrBf7T3S81s1uA/4okuuJxALjGzF4gSML+cTg+kaBt86hlZj8CVrn7ifB1j9z994corGK2hqBtc1cWrvvl0IYjIiIifaFKWJH8a3P3w+6+392/AvwG+F0zu97MtppZs5kdM7ONZnZRdicz+z0ze9DMWsM7r2/NVt6Y2eywKufKnO0nhGPX54y9xsz2mFmLmd1L8OPBWczsDWb2uJm1mdk+M/tgb2/GzNYC1wF/lVMdNDt8Px7O34KZrQrf12vNbLeZpczsB2aWNLO3hed60cz+zcziOccvM7N/MbNnws+mLvc9iYiIDDFdx3UdF5FRysx+187Mdfqq7HL4eD3wcWBfdBEWlSrgRDfjJwjm9QSoB84fsoiK0xeA7wBPAweB9eH4y4Cd0YRUNI4TtK3Ovu7tIUHl9BPdjO8Cqoc4FhEREekjVcKKFF4LQRumnwBfA94ElAJLCb9wmNm1wLeA9wMPEHyh/c9w/0/25SRmNhP4EUHFzn8Ci4HPdtlmEfB94Dbgf4DlwJfN7Hl3X9vDof8KmAc8BnwiHHuWbn4YBpLhe/hjYGwYz4+BYwTVRHOBHwIbw/MDfBG4JNznIPB64C4zW+ju9X157yIiIgWk67iu4yIyevwk5/U3u6zrIEjA9nrzyyiyA/iMmb3V3Z8FMLPJwKcJ2hRDkDQ6EFF8RcHdv2xmW4GZwK/dPROuagI+Fl1k0XP3m7t7LT06TvC32L4u49VA85BHIyIiIn2iJKxIgZiZATcCrwK+DVwN/MLdG8NNnszZ/FZgjbtnv+g3mdnHCb7A9unHW+C9QKO7Z38U2G1mC4EP52zz18A97v734fIeM7sE+BCwtruDuvtxM2sHUu5+um1h8PZeogR4b/Y9mtkPgLcAF7j7KeCJsLLnBuB/zGwWQcvHWe5+MDzGv5jZq8Pxv+vjexcREckrXcd1HReR0cfdYwBmthdY7O7PRxxSMXsH8FPgaTPLJlpnAo3A68LlMcA/DH1oxcXdtwPbu4ytiyicomRmb3L37/aw7jPu/qGhjqkI/RT4gpm9PudvtWqCm/Z+FmlkIiIi0iMlYUXy77Vmdorgh8wYcCdwC8F/b3eb2a8JWht+390PhftcAawws4/mHCcOlJtZso/nvRio6zK2uZttftplbCNwS9hacDlnzyPybnf/Th/PD8EPvI05y0eAfeEPt7ljU8LXCwne554uPwaXAfrBQ0REoqDr+Bm6jovIqGNmJQRVipPQ/5f1yN13hzcC/RZBxwWA3QQ3KX2MYL7Yn0QUXtEws8/1sMoJ5oRtAH7q7i8MXVRF6Stmdszdz5rX1Mw+T9BtQ0lY+FvgLmCXmT0djs0E7gf+JrKoREREpFdKwork370E1SztwEF37wzHbzazfwNeDfwR8A9m9kp330Jwh/CtBG3/umoFsi2Lcn/hLClA7NuBK3OWj/Rz/44uy97DWHY+6jFAGlgUPuc6hYiIyNDTdfwMXcdFZNRx9w4zuzzqOIaDsLXuXeEDADO7gqBK9l1RxVVkriLophEnSFJDkLROE8zl+efAZ81spbt3N9/naPEnwHfN7LXuvgHAzP4d+H2CDhyjXtjdZDnwSoIbAFuAR9z9gWgjExERkd4oCSuSf83u3tDdCnd/CHgIuN3MNgNvBrYADwLze9rPzJ4NX04L94ezf2SF4I7j3+0ytqybbVZ0GVsB7HH3NMEf8d3F0E7wpTHfHgqPO0VfHEREpEjoOt53uo6LyEj13wSJxNVRByLD3k+BF4Cb3f0EgJmNB74ObCCYb/5O4PMEUyCMSu6+zsz+HPiZmb2S4L+/3wNucPc90UYXLTO7BjjP3X/h7g78ysymEUx5kTSznwB/6e5tUcYpIiIi3VMSVmQImNkcgjuBfwYcBOYDNcC3wk0+BfzCzJ4CfkBQMXMFcJm7f8zdW8xsC7A6nJ9oCi+dW+erwAfN7DMEX+gWAau6bPNZYFs4T93/ANcA7yO4+7Y3+4BaM5tNUNmSl1ZJ7r7HzL4DfMvMPkjwY+5kgjn4HtU8OSIiUgx0He+eruMiMoIlgLeb2SuAHUBz7kp3/+tIopLh6EPAK7MJWDhd0Xgb8Ct3/1cz+xTwq6gCLBbufqeZTSCYauFZ4LqebnAbZT4BrAd+AWBmCwmS998kuEHvQwR/n94WTXgiIiLSGyVhRYZGClgAvA04DzgEfAn4DwB3v9vMXkvwx/WHCVr/7SL4ETbr7cA3CH4E2E0wH8jpL2ru/pSZvYHgDtq/BLYCfwf8V842D5rZHxL8WPzxMI5PuPvac8T/LwR/4D8BVABz+vsB9OJmgjmDPgtMB54jqCr6RR7PISIiMhi6jvdM13ERGYkuI+hyAGfmO80aO8SxyPA2nuDmq66thicD48LXx4DSIYypKPQyX+6zBP/9/Xl2zvlRfuPDlQR/92X9MbDV3f8MwMwOEFTF3jbkkYmIiMg5WdDJQkRERERERERk9DKzD7j753tZPxa4y927toYfNcysu/nPc00gqGAsRBv8YSfsGHEN8EFgWzi8hOAGqU3u/hYz+2Pgb9x9cURhRsLM7u3rtu4+aueFNbNWoMbdD4TLG4Bfuvs/hsuzgZ3urhtEREREipAqYUVERERERERE4J/M7Hl3/1bXFWZWCfySoCPCaHa8D+tf8vmNYu8m6HLxPYLf4IygY8Y3gWx15y7gnZFEF6HRnFjtpyMEXUwOmFkpcDVwa876sQT/pkRERKQIqRJWREREREREREY9M3sj8G3gj9z9ZznjY4C7CNrKXu/uByMKUYap8N/Q3HCxyd1PRRlPsTCz/zr3Vri7v6PgwRQpM/sKcAXBlBevI5ge40J3bw/X/wlwi7sviSxIERER6ZEqYUVERERERERk1HP3H5jZBOC7ZnaTu6/PqYC9gKDNrhKwck7nSi7mzHX69iEJqHitAvYDDxFUCctLfRz4EXAfcAp4WzYBG3o78KsoAhMREZFzUyWsiIiIiIiIiEjIzP4W+Cjwe8CngOkECdinIw1Mhg0zy9CH5KK7v37IgipCZvYl4E0En9UdwH+7+wvRRlWczGw8cMrd013GJ4Xj7d3vKSIiIlFSElZEREREREREJIeZrQE+BOwjaEF8INqIZDhRcrHvzKwM+H2Cis7lwDrgG8CvXD9aioiIyDCnJKyIiIiIiIiIjHpm9qMuQ68BHgGeyR10998fsqBk2FJysf/M7CKCFsVvJZhC7VLNnysiIiLDmeaEFRERERERERGB412WvxtJFDIiuHsbwb+h7+YkF78MJMxMycXuZQAnaOEcjzgWERERkUFTElZERERERERERj13vznqGGTEUnKxB10qhlcCvwDeB9zl7pkoYxMREREZrFjUAYiIiIiIiIiIiIwkZlZmZm8ys18De4CFBMnFWaqCDZjZl4FDwGqC5OtMd/8Dd/9fJWBFRERkJNCcsCIiIiIiIiIiInkSJhf/GDgA/BfwHXd/Ltqoio+ZZYCngIcIKoW7pXmYRUREZLhSElZERERERERERCRPlFzsGzNbSy+fT5ZahYuIiMhwpTlhRURERERERERE8udb9CG5ONq5+6qoYxAREREpJFXCioiIiIiIiIiIiIiIiIjkUSzqAERERERERERERERERERERhIlYUVERERERERERERERERE8khJWBERERERERERERERERGRPFISVkREREREREREREREREQkj5SEFRERERERERERERERERHJIyVhRUREREREREREQmbm53jcFnFsr4vq/CIiIiLSd4moAxARERERERERESki03Je/xHwKWB+ztip/hzMzErdvT0fgYmIiIjI8KFKWBERERERERERkZC7H84+gOPB0OnlSuA7ZnbEzE6Z2TYze0Xu/ma2z8w+bmbfMrMTwH+G439mZgfMLGVmPzazvzazY132/T0ze9DMWs2sycxuNbNE9rjhZj8OK2L3ISIiIiJFS0lYERERERERERGRvhkD/C9wI3AVcBfwczOb1WW7vwEeCbf5ezNbAXwV+FfgSuDXwEdzdzCza4FvhdtcArwbWJWz3ZLw+WaCat0liIiIiEjRMnePOgYREREREREREZGiY2argC+4+4RetnkM+Kq7fzFc3gc85O6vz9nme8AYd39tzth/A6/NHtvMfgPc4+6352zzp8Cn3f3CcNmB17v7T/L0FkVERESkQFQJKyIiIiIiIiIi0gdmNsbM/sXMnjSzY2Z2CrgY6FoJu73L8nxga5exrstXAJ8I2xyfCo/9NWCamSXz9R5EREREZGgkog5ARERERERERERkmPgX4JUE7YYbgBbgB0Bpl+2aB3DsMcCtwI+6Wdc6gOOJiIiISISUhBUREREREREREembFcBad/8xBJWxwOw+7Lebl87h2nX5QWC+uzf0cpwOIN63UEVEREQkSkrCioiIiIiIiIiI9E098Ptm9nPAgb+nb9N9/Ttwv5n9NfBz4OXAb4fHyPoU8Asze4qgujZD0KL4Mnf/WLjNPuBGM9sItLn7i4N/SyIiIiJSCJoTVkREREREREREpG/+GngR2ESQTL2boIK1V+6+EXhPuP8jwKuBz5PTZtjd7wZeC/wWsA3YAnwA2J9zqA8StEM+ADw06HcjIiIiIgVj7n7urURERERERERERCRvzOxrwAJ3vzbqWEREREQk/9SOWEREREREREREpMDM7G+AXwPNBK2I3wb8eaRBiYiIiEjBqBJWRERERERERESkwMzs+8D1wFigCfh3d/9qpEGJiIiISMEoCSsiIiIiIiIiIiIiIiIikkexqAMQERERERERERERERERERlJlIQVEREREREREREREREREckjJWFFRERERERERERERERERPJISVgRERERERERERERERERkTxSElZEREREREREREREREREJI+UhBURERERERERERERERERySMlYUVERERERERERERERERE8khJWBERERERERERERERERGRPPr/Uch59k3KbdUAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 2400x1200 with 12 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "branch=branches[branchname]\n",
    "# Draw dynamic network with notch layout\n",
    "layout1=layout.notch(nframe=nframe,dpi=dpi)\n",
    "pts,fig,panels,animate_ka=layout1.draw(\n",
    "\td0,branch,\n",
    "\t# Set genes to annotate\n",
    "\tbcde_tfs=tfs_ann,e_targets=target_ann,f_tfs=tfs_subnet,\n",
    "\t# Custom legend location for long cell type name\n",
    "\ta_ka={'scatterka':{'legend_loc':(0.6,1)}},\n",
    "\t# Custom configurations for color range\n",
    "\te_ka={'lim':[-0.02,0.02]},\n",
    ")\n",
    "ca=panel.animate_generic(pts,fig,panels)\n",
    "anim=ca.animate(**animate_ka)\n",
    "\n",
    "# Saving animation\n",
    "# Option 1: Visualize animation within jupyter. This will make the notebook large especially with many frames\n",
    "# from IPython.display import HTML\n",
    "# HTML(anim.to_jshtml())\n",
    "\n",
    "# Option 2: Save animation to mp4 file\n",
    "# If you receive a codec related error, try a different codec, e.g. with `codec='h264'`. You can list available codecs with `ffmpeg -codecs`.\n",
    "w=matplotlib.animation.writers['ffmpeg_file'](fps=fps,codec='mpeg4')\n",
    "w.frame_format='jpeg'\n",
    "fo=f'../output/animation-{branchname}.mp4'\n",
    "anim.save(fo,writer=w,dpi='figure')\n",
    "display(FileLink(fo))\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Hair-Shaft-Medulla branch\n",
    "##### Branch dependent customizations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "branchname='Hair-Shaft-Medulla'\n",
    "# Select TFs for each row's dynamic subnetwork graph\n",
    "tfs_subnet=[\n",
    "\t['Lhx2'],\n",
    "\t['Runx1'],\n",
    "]\n",
    "# Select TFs for each row's other plots\n",
    "tfs_ann=[\n",
    "\t['Lhx2','Dlx2','Sox4'],\n",
    "\t['Runx1','Prdm1'],\n",
    "]\n",
    "# Select genes to annotate as targets in all rows\n",
    "target_ann=['Mgst1','Mki67','Krt31','Scd1','Rnaset2b']\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "##### Branch independent lines"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<a href='../output/animation-Hair-Shaft-Medulla.mp4' target='_blank'>../output/animation-Hair-Shaft-Medulla.mp4</a><br>"
      ],
      "text/plain": [
       "../output/animation-Hair-Shaft-Medulla.mp4"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAB2sAAAP9CAYAAACjU4ZMAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjYuMCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy89olMNAAAACXBIWXMAAA9hAAAPYQGoP6dpAAEAAElEQVR4nOzdeXxUd73/8dc5Z/bJZLJASChLKF0ylLSUKm1trTVaxRvx1u0WtUrdrsGodftd0eu1Q3vVVOtubK5aNYpa16pIpVfNrVtbqVLaUCYtUAIFEkK2yezLOef3xzkThpBAAmH/PB+PPMic5Xu+Z9Isnfd8Pl/FNE2EEEIIIYQQQgghhBBCCCGEEEKcWurpnoAQQgghhBBCCCGEEEIIIYQQQpyPJKwVQgghhBBCCCGEEEIIIYQQQojTQMJaIYQQQgghhBBCCCGEEEIIIYQ4DSSsFUIIIYQQQgghhBBCCCGEEEKI00DCWiGEEEIIIYQQQgghhBBCCCGEOA0krBVCCCGEEEIIIYQQQgghhBBCiNNAwlohhBBCCCGEEEIIIYQQQgghhDgNJKwVQgghhBBCCCGEEEIIIYQQQojTQMJaIYQQQgghhBBCCCGEEEKc0xRLqaIoyumeixBCFHOc7gkIIYQQQkyRebonIIQQ4rjIi2JCfocLIcTZSX6Hi3NFAIhGo9HTPQ9xntMNk027BumLpakKeFi2oAJNlR+1Z5BT/sWQsFYIIYQQQgghhBBCCCGEEEKIk2zj1h7Wrt9GTzQ9uq0m6OGOFYtYvrjmNM5MnE7SBlkIIYQQQgghhBBCCCGEEEKIk2jj1h5Wr9t8WFAL0BtNs3rdZjZu7TlNMxOnm4S1QgghhBBCCCGEEEIIIYQQQpwkumGydv22cdcFKWxbu34buiErh5yPJKwVQgghhBBCCCGEEEIIIYQQ4iTZtGvwiIraYibQE02zadfgqZuUOGNIWCuEEEIIIYQQQgghhBBCCCHESdIXmzioPZ7jxLlFwlohhBBCCCGEEEIIIYQQQgghjpNumDy6c4Bfb9nHozsHjmhnXBXwTGqcyR4nTkx3dzeKorBly5bTPRVAwlohhBBCCCGEEEIIIYQQQgghjsvGrT1cf3cHb/rWY9x+/xbe9K3HuP7uDjZu7Rk9ZtmCCmqCHpQJxlCAmqCHZQsqTsmcz2a33XYbiqKgKAoul4uLLrqIO++8k3w+f8rnoijKDYqirFcUZb+iKKaiKDcfzzgS1gohhBBCCCGEEEIIIYQQQggxRRu39rB63eYj1qPtjaZZvW7zaGCrqQp3rFgEcERgW3h8x4pFaOpEca4otnz5cnp6eti+fTsf+chHCIfDfP7znz/iuGw2e7Kn4geeBJpPZBAJa4UQQgghhBBCCCGEEEIIIYSYAt0wWbt+G+Y4+wrb1q7fNtoSefniGu69dSnVwcNbHVcHPdx761KWL645uRM+h7jdbqqrq5k/fz6rV6/m5S9/Ob/5zW+47bbbuPnmm/n0pz/N7NmzufTSSwHYtGkTV155JR6Phxe84AU88cQTh4338MMPoygKDz30EIqiPKEoSkpRlA5FUaoURXmVoigRRVFGFEX5kaIovsJ5pmn+zjTNT5qm+cCJ3I/jRE4WQgghhBBCCCGEEEIIIYQQ4nyzadfgERW1xUygJ5pm065Brl1YCViB7U2Lqtm0a5C+WJqqgNX6WCpqT4zX62VgYACAP/7xj5SWlvL73/8egHg8zqtf/Wpuuukm1q1bx65du7j99tvHHSccDgO8D0gCP7U/MsCbgRLgAeD9wN3TOX8Ja4UQQgghhBBCCCGEEEIIIYSYgr7YxEHt0Y7TVGU0vBUnxjRN/vjHP/LQQw/x/ve/n4MHD+L3+/n2t7+Ny+UC4Jvf/CaGYXDffffh8Xi47LLL2Lt3L6tXrz5ivP/+7//mZS972d8AFEW5D/gssNA0zefsbT8HXso0h7XSBlkIIYQQQgghhBBCCCGEEEKIKagKeI590BSOExPTDZNHdw7w6y37OBjL8Nvf/paSkhI8Hg+vetWruOWWWwpVsdTX148GtQCRSITLL78cj+fQ1+Haa68d9zqXX3558cMDQLIQ1BZtq5qu+yqQylohhBBCCCGEEEIIIYQQQgghpmDZggpqgh56o+lx161VsNajXbag4lRP7ZyycWsPa9dvG2053f/sQUoWXMEXv/I1Xr54DrNnz8bhOBR3+v3+476W0+ksfmgCuTGHmJyEQliprBVCCCGEEEIIIYQQQgghhBBiCjRV4Y4ViwArmC1WeHzHikWyHu0J2Li1h9XrNh+xNnBWcXHXn4fYNuI8LKgdTygU4qmnniKdPjTGY489dlLme7wkrBVCCCGEEEIIIYQQQgghhBBiipYvruHeW5dSHTy81XF10MO9ty5l+eKa0zSzs59umKxdv23cquWCteu3oRtHOwLe/OY3oygK7373u9m2bRsPPvgg99xzz7TMUVGUEkVRliiKssTetMB+PG8q40gbZCGEEEIIIYQQQgghhBBCCCGOw/LFNdy0qJpNuwbpi6WpClitj6Wi9sRs2jV4REVtMRPoiabZtGuQaxdWTnhcSUkJ69evp6mpiSuvvJJFixZx99138/rXv346pvkC4P+KHn/R/rcduG2ygyimefTEWQghhBDiDCN/vAghxNlJXqkQ8jtcCCHOTvI7XJwTFEUpBaLRaJTS0tLTPR0hxDH8ess+br9/yzGP+8rKJfzrkgum89Kn/PeetEEWQgghhBBCCCGEEEIIIYQQQpwxqgKeYx80hePOZBLWCiGEEEIIIYQQQgghhBDirKEoyhpFUUxFUb58uucizh+6YfLozgF+vWUfj+4cOOZaqeLELFtQQU3QM2GZqwLUBK2W02c7WbNWCCGEEEIIIYQQQgghhBBnBUVRXgi8B3jqdM9FnD82bu1h7fpth62hWhP0cMeKRSxfXHMaZ3bu0lSFO1YsYvW6zSgcvqZKIcC9Y8Wic2JtYKmsFUIIIYQQQgghhBBCCCHEGU9RlBLgh8C7gaHTPB1xnti4tYfV6zYfFtQC9EbTrF63mY1be6b1elLBe8jyxTXce+tSqoOHtzquDnq499al50xQrpjm+ftFFkIIIcRZSf54EUKIs9PZ/3ZncaLkd7gQQpyd5He4OGMoitIODJqm+SFFUR4Gtpim+cFJnlsKRKPRKKWlpSdxluJcohsm19/dcURQW6BgBYd//VjDtFR4SgXv+HTDZNOuQfpiaaoCVuvjk1hRe8p/70kbZCHEWad2zYaHgS3dLY0fnOZxbwO+3N3SWHaC41QDPwBeBOROdLzpULtmgwm8trul8VeTODYM3Nzd0rjkJE9LCCGEEEIIIYQQQohJURRlJbAUeOEkj3cD7qJNgZMxL3Fu27RrcMKgFqx3I/ZE02zaNci1CytP6FqFCt6x73AsVPCeS5WkU6Wpygk/v2cyCWuFEKdM7ZoN3wPKulsabx6z/Ubg/4Dy7pbG4UkM9TogdxzXfwlwB7AE8AD7gEeAd3e3NGanOt5RfAiosa8TPY77O25HCVprOAdaw9Su2dCNFah/+TRPRQghhBBCCCGEEEKcIoqizAW+AtxkmubEydnhPo71WqAQx60vNrn/3CZ73ER0w2Tt+m3jtqIxsUo9167fxk2Lqs+JNVrF4WTNWiHOAPXt9fLTdQq6WxoHu1saYxPtr12zwTXOtkXARuAfwA1APfB+IAto0zzFhcA/u1sat3e3NPZN89jHpbulsbe7pTFzuudxvMb7mgohhBBCCCGEEEKI88ZVQBWwWVGUvKIoeeAlwAfsx+O9vvdZIFj0MeeUzVacM6oCnmMfNIXjJjKVCl5x7pHKWiFOs/r2ehXw17fXZztXdZ62MC1SF3IARqgrYpyuOQDUrtlQCXwdK1AtB3YCn+luafxx0TEPU9QG2a62vA+4GLgZ+CVw25ihXwH0drc0/kfRtp1YAe7YObwS+DIwF/gr8PbulsYee98Lgc8AVwJOYAvwoe6Wxs1Fc5lvf/42oB1YZQ89VLtmA0B7d0vj2PkVru0G7gTejPUH6PPAZ7tbGu8br01z7ZoNNwMPdLc0Kvb+O+zthTdhvb27pfF7Y9sg167ZMAf4PPBKrHYwEaC5u6Xx7xPM613AR4AFQDfw1e6Wxm+Md6x9vAp8FPh3+3k8APxPd0vjp+399VjvhrwWSAK/AD7c3dIYt/d/DygDHgeagUztmg27sZ7bL9Wu2fAlgO6WRsU+/nqsP8BfAPQDDwAf725pTNhfh28AV3a3NG63j/8G0AAs7W5pTE50H0IIIYQQQgghhBDijPBHrOKLYt8FuoC7TdPUx55gmmYGGH29VVGkXkZM3bIFFdQEPfRG0+NWvRbWrF22oOKErnOqKnjFmUkqa4U4/UwgDxzxB8WJsEPgyRzniNSFVMDL4Ws4nC4e4J9AI7AY+Cbwg9o1G5Yd47yPAk9ihah3jbO/F6ipXbPhhmOM47PHeitWYDwPuKdofwArgL0euAbYDjxYu2ZDYc2LF2IFwD/Faj18O/B6e9+lRdsm8n3gTcAHgBDwHiB+jDkX/AT4AvC0fZ0ae9thatdsKAH+BFwAvAa4AvgcE/xOqF2z4S1YAfJ/2nP6BHBX7ZoNq8Y73vZZYA3W12IRVvh8wB7PDzyE1Zb5hcAbgZdjhfTFXob1nN0EvBqr/fVe4FNF90ftmg0LsZ7zXwCXA7dgfX2+DtDd0vh94EHgh7VrNjhq12xoBN4FvEWCWiGEEEIIIYQQQogzn2maMdM0txZ/AAlgwP5ciJNCUxXuWLEIsILZYoXHd6xYdMKtiU9VBa84M0llrRCnWeeqThNITeWc+vZ6DTDscyfa7ztWtW59e70T8Pzbxx2Zn342n2GaA+MJvLp2zYax4eNom5LulsZ9HB6Ofs2udP03YNNRxu3obmn8wlH2/wyrivRPtWs29AKPYb0j7/vdLY0jRcc5gabulsadALVrNnwdKxwszK+jeNDaNRv+HRjGarvy2+6WxoO1azZkgFR3S2OvfUyhN0Xf0dasrV2z4RL7Pm/qbmn8g735uaPc02G6WxpT9nObL1x7Am8GZgIv7G5pLMxtx1GOXwt8pLul8Zf24112W+n3YAXXY+8jgBVIv6+7pbGwfydWlXLh+h7gbd0tjQn7nPcB62vXbPhYd0vjAfu4BPCu4vWEa9ds0IHYmPv7OPDDonVst9eu2fABrK/16u6WxrQ916eAr2KFvuHulsZ/HuWehRBCCCGEEEIIIYQQguWLa7j31qWsXb/tsFbF1UEPd6xYxPLFNSd8jVNVwSvOTBLWCnGK1bfXu7ECwWTnqs4ptxyub6/XrkmlSkHJtDZ1pAGa2xrGjmNwlGrd1qYODTC4ljzWmq35U9j++P+A1WO2XQ2sA6hds0HDqtz8N6zKTxdWxe+xKiD/Ufikds2GNuDWwuPulsaS7pZGHXh77ZoNn8Rqf3u1fZ2P1a7ZsKzQ5hhIFoJaWw9WO+LC2LOA/wZutLdrWNW4845148XsatX/Kdr0KqxKUR2r6vVkWgI8URTUTsiugl0I3Fe7ZsO3inY5gOgEp4WwvmZ/PMr+JwtBre1vWJW9l2JX4AKdxUHtUVwBXG4/pwWKPd4CINLd0jhUu2bDO7Eqeh8BWiYxrhBCCCGEEEIIIYQ4Q5mmeePpnoM4fyxfXMNNi6rZtGuQvliaqoAVnJ5oRW1BoYJ39brNKHBYYDudFbzizCRhrRDHIxxUsAK6POHoEZWr9e31SqHq1W5HbHau6jTr2+sLP2cLH0dV317vLU/UKLc8tcYAcs1tDXrnrj3Gb/0+dV4+r/3Fal2sArHPv+/XvsisR/yPz3sw0bmqMwmkbvvv/6fUt9c7mh79ioEVKuoLdv3W6Zwx+5LB0qDyjsc+t/tD975qosDtZEl0tzQeVsVpr59a8P+wqjI/CHRiVVd+GSu0Peq4RZ9/isOrc0fZlbs/wGqt/F/As0AT9lqvQG7MKSaHd7hoByrtOe7GWvfi0UnMb6zfAMXrw+7DagV8NAZHdttwTvG6MLVK7hL733dz+Hxh4krsKVWKH0Xi2IcA1hz/B6tqdqw9RZ/fgDXnGsAPxE5odkIIIYQQQgghhBBCiPOGpipcu7ByWsbSDfOI4PdUVPCKM5OEtUIcv3HfwlJoLbz826H0F/r6TS6ongHo9e31I6W67ro6nUl88b07rWrBcNCqcA1HzTFjuLGCOf9FA0tc9ueD9e31XhbMK20aGs5dtIf8Jc/+RH28viFT317vrpt3jc+XC+hV+bxBOKi29j7gKplR5g+kK3J9/t3kldzsg7695qB6UbA0W3KNajoOZDDSt71/o/awXx/pbmnMn8wnawquA37d3dJYqLRVgUuAbZMdoLulsQ/om8RxQ7VrNvRgBXdTmd97u1saH7TnNxeYcYxzCtWhxe2eY4wJC2vXbOjECt9fAvyBIx0EArVrNviLqlKXjHMtjaN7CnhX7ZoNFceqru1uaTxQu2bDfuDC7pbGHx5j3ILtWIHty4Bvj7M/Atw25j6uw/rv/JljjD3e/W0GFo19E0Cx2jUbXgR8DFgB3I21nu3R1twVQgghhBBCCCGEEEKIabdxa88RgWxNUSB7Mit4xZlJwlohjocVro5dd7XAAFiUyVTEVEVbEktXR5xuPeNRsjckk6UvTmXUT39tvvaSeDJ5PQSAZH17farQEtmuvnUBxpxcLruAA5la96NKY/nnjDbmqW7DUDKK6t5hep3z+v4R++OCv3kCqRmOPcFIJumJKiujqWBXrj67L7BdU01H4ILoJQMO3ZVLOeLVwXjVPCivzDq8STe+HSMwK68YTiBtt0Z2AMnmtoZjVv2eRNuBN9jh2hDwYWAWUwhrx1O7ZsN7sILNB7DWT/UAbwMuA94/xfm9tXbNhn8ApcDnOXYl6W6sCt1X167Z8CDWerZH/PfT3dLYXbtmQzvwHXvN1SeB+UBVd0vjT7EqW5PAZ2rXbPgqVivn28YOAyyoXbNhCbAXa33XsdXfP8ZqAf2r2jUbPo7V6vlKYH93S+Oj48z/DuCrtWs2RIGNWC2OXwCUd7c0fhGgds2GPwIPdLc0fr27pTFdu2bD3cDnatdsyGK1OJ4JXNbd0ngf8EOsdXDba9dsCNv7vgb8oGi92ol0AzfUrtlwP5DpbmnsxwpfH7PXF/42VkXuIqy1f99nr6H7A+Cr3S2Nv6tds2Ev8Hjtmg3ru1saf36M6wkhhBBCCCGEEEIIIcS02Li1h9XrNh/RdrM3mmb1us3ce+tSli+umbYKXnF2UE/3BIQ4BzneHB1RLsrm2K9p+gX9ZK/sNpIVGR0N9Bym5/pkuqI2r88CPD8NlGiAv769XgPo3LWHF6TSye/0HEj/bm+P+V/6r9yN5Z9zA97OXXtitf3X9D+VvVrbMPOlygPLr/W/OlEz59V9sy68sD90ZUX0gouezV/m35xsdKu6wwhkKrRrd/1rYGfFloCJWR3Mz9T8RmCvG2+HA8fTDtStflPdCaSxKoXPhLfn/DdWpeRDwMNAL/CraRh3E1a73Dbgaax1Ya8Bbu5uaZzKGrHvBMrtOf4Aq/XuUat47dbLd2Ctk3oAq6pzIquBnwPfALqAb2FX/tpVsLcC/4LVIvpNQHjM+b/AClT/D6sS903jzCcLvMKe94P2WGuYoK1xd0vjt4F3AW+3j/0TVki8q+iwhRxeYXwX8AXgTqxK2p9gr/3b3dKYBF4JVACP2/f7R+B9EzwnxT4F1GIF7gft8Z7Cqka+BPgL8IR93f32OV/BCnA/YR/faX/+P7VrNlwwiWsKIYQQQgghhBBCCCEEumHy6M4Bfr1lH4/uHEA3Jl/3pBsma9dvG3d9xMK2teu3TWlMcW5QTFO+6EJMmdW+GMLRI8Ktd7RdNPfiTC5Ym8+nvhsM5P0xXHfvixqzPGhpVyrlNs10mYmKFZAa9QvmpQDXPX39uVcmkm6sqtrMpvgtSY8SL73Y86dsf/7CGTHTp328cuDg4u7KqpTvEiPuK48pru4587MDc/zp2foOZ0VlwpndVxW7ZMSbCTgy7sSIU3dlHKarLK/k9JySKSvJVPQH9cqBnRVPpH5/6fdynas608Vzb23qcGKtgZo6zdW1QghxNPLzSQghzk5nwhsDxeklv8OFEOLsJL/DxTlBUZRSIBqNRiktLT3d0xFnoWO1Lz6WR3cO8KZvPXbM43787muksvb0OuW/96QNshDHxwtAOJgBqF8wT31FPGF84eCA+XqfT8M0U6hK7K3RGJVJI1s24q+KOD1Vhpo8+MJsbA9qVgfU4Xy174tP3KxcVfLLXIUjWWKPm+92OFIP6Tc6KlLVxl9i79JfXXZnBnDdkBqa6VPctUPJZ/q3zdp/wMNV8dLhsriSckTrHOXRhCs3lM2Vl7h0T7kv7g3mNYOYMzpUnZo/EHcORn1GyUGgxFB1L0B9e32mc1Vn8QsmKtZ6oAryQooQQgghhBBCCCGEEEIIMen2xUfTF0sfdf9UjxPnDmmDLMTxSdsf7t0Oh2dpOu29LpUuB/yNyeRwYyo11JhIJt4ai2uvSKZwGfGEbuSf7zVqDqwf+i99X3ZRBsjmDYc21/VkuVcZ8Q/nZ5Xsz1zav83lHFwxd7brp0s+6x/09qQB/bfDn+pf5Hms590MDNV4Zu3pqdbU2R5nzUv2vqTXk3I/g3tGarbO8LOz/ryna8bf+zxqtMvt2u11mdmZaEbuydkdg71lzw3tK3vWC2RqohcNA4kxQS3NbQ0ZIN7c1mCc4udTCCGEEEIIIYQQQgghhDjjTFf74qqAZ1LXm+xx4twhYa0QUxEOaoSDJYARuX+20ftEQK3YraX6NU1X0hcEn0gu9/UR0IdVNddeWlLzS78v6HCb8bI5qfTF/gOxC8yYOsPZraWMYBqonOHaq8x3/7PfqaZSz6Wv1h9PrPQs+kR/vi6T0crzuvaXBT8zAB/gIRzVO/rvyI3kX5I23Z50SaasImP+o7R+17AxK+n1OBRV17IzA3Mzzkt6S/bM7y7f6dlZFjGinr78rPh87YkL/mAOew64gIqSXFmgc1XnuIGstD8WQgghhBBCCCGEEEIIISybdg0e1vp4LBPoiabZtGvwqOMsW1BBTdAzYY9dBaut8rIFFcc9V3F2kjbIQkwkHHRhr99KOFoINv1xRalUTXPgghcNoGe1Ms2tO94UHTFHjNnZpzy4Hwl4cm9OxFM7nM58xqWkrs5ksuW9yvBD6kUVu6scmVdWrYtelM4oQAIwvVo8CSgaWW0of4F6//vu8315OKHtyznVpRc+7Pmr95Lc06nleYALXNuc+8qfd+Z8+3r9Qy8yTH2pu6/yObeWqe/vNfVE7UiqzGckiWmx7NP+2JM57WBFaW4Gi7pflB329mWrYwvzQACIn46nVAghhBBCCCGEEEIIIYQ4m0xX+2JNVbhjxSJWr9t8xDqEhQD3jhWL0FRZKvx8I2GtEEc3+lMxuraspNfp8EYV1YlC+bJ5GT2fxelwEVicUdxP+Z+JBY1IYmk6N6vcMLWVscTeWXre5et1eJ/7y2wqqhcpjgGvHip91r17ayUzFsfy/llZA6vCXY0bs/weNW4O5BekHHE1VavknXm3WfoCzzeTN37pc2mAF5b8dOQXZZW5ocTLzZqh67zPVD/lH1jQk7uy5yJlVrzWkczN6a8q/55xac7MdVQO5A46HFEg83TNX7KdqzqN1qYOA6t9s35ank0hhBBCCCGEEEIIIYQQ4iwyne2Lly+u4d5bl7J2/bbDqnWrgx7uWLHomOveinOThLXi/BUOalgthtOEo7kj90ezQNY+Vul2OWdkFbU0qyixWflcFvA4XAwBJTP1vFGTV2M3pTIuIAek1O/PUDLzkl7PwkTGraZ0T4YDizKPaJkhx0znLMWfSJQM+Rkcidw/Wzswc6m3b+HF+bSvNNrc1pCF5zMAX/rKvBlPut357wGtTR0eeECd24sxF9wZLZkb8O3t76582lx08No8YGya91sznilLzU5R9vlnawZuW9Q14smVOBft8bkjdSFz9iVvyu+ffb0BeFubOrL2GrWnTjj4PaCMcPRm+/NV9p48sBf4GfApwkW/pcLBlwB3AEsAD7APeAR4t/01EkIIIYQQQgghhBBCCCFOiqFEFlWBiZakVbDC1sm2L16+uIabFlWzadcgfbE0VQHrXKmoPX9JWCvOdyaMuy74IeGgB9Cr83o0oejO2qyhqxop3VBdmmo4o6o6/DP3xcGnlNqSma7/cy/JZuNAStGMknxaLXWXGOmLbz4wcnH4u4lIXUhzznXG+kpDjjSVZhW/SgHu0thuv7JXMa686m++xOr3XmAaDJX8z8Hh75QF44C58fa71Ct8/UEDLdeZfHV6oe8hv3vmj0c+/OE9icI0I3UhRyiyzN+1YHFZ1O8tyWlZHbq6r3vudcHLd5c6oqW/jtc9++PE/tnXp4ByINfa1KECWnNbw5Fh9amxEXg7Vrvpq4B2rK/HxwAIBxfZx3wN+ACQAi4GXg9op366QgghhBBCCCGEEEIIIc4XG7f20PyjzccIEabevlhTFa5dWHlikxPnDAlrxflMBdyASTioE44e+nkbDroBA6vi05NLKf5AQvWU+3VXYtDt2p28PB6Ym3bOdj2d8RuUzY/OU5/yVeUf8pfmZ+cHU1WG4SmpySjxXncUzNHq1dDK/UpKD8R3910QKA1uL9mvabnQyv2Dw7sG48POrFPNZvV8RtUUxXQD/PSzeR1QOm68znOF7zfUeR9OdSZf7U16uyuf8ajlH2iv3925qjPe2tShXDH/8pIF2j/KXIO51J74NR49G8u6L/d5I7Me9ea1qxLdQTX5wNUOT+jA/fmXPLcya9+fB9Bamzr05rYGgyL17fUOwNG5qnNyDfmPT4ZwtNf+/HnCwT8AN1EIa+EVQC/h6H8UnbMTK8AVQgghhBBCCCGEEEIIIU4K3TBZu37bUYNaVYGvv+lKaV8sToh6uicgxGnkAUqwAttRQ2+tLh953n2BnlV8gAsgl9BUPa05u3DHHvZ4U75Z+4NeNeZUwJU1/TMu14fVf81vzf7S64393u/TsgnFq3nyDkUlp2gmgEo4qORN5wwjZVTpDkeq36Vc0Dm8pO7h3TdX/ePCBWXPV1zq/Xnmy7ls1LmzdG6mz656dQHusqFnM08mXzP4s4F70gf9e/J/LVH6FPTE2oMDWmtTh/bo/F/5f35NqW/bkrJc4oKLMn1lWWfn3AFvxpnM7C/bPrxldsfgwIyrDcCIzHo0DyStdstkgPTYoNbmBJz17fWn5udEOLgYeBGF1tOWXqCGcPCGUzIHIYQQQgghhBBCCCGEEALYtGvwsHVlx2OYUO53H/UYIY5FKmvFuSscdAL5wypmDxcH0ljVs357DVvFVer0G1k1qzpMEygDDFD2OTyGvyxgzuyekc27nk+lvf2mSqUZ97lj6XJjn3lNd1z/yCMljpDTyYH5boevVDdnXzOEw2NqQI5w1HzqvW/KpwccXsOnkvJdMuyN9bgzSjT4uF+J6kZloqvi/4wvXF3ubHr0+4ZbiXl3XHVL5qJ//kRf+uRXjFBXRG9t6vAknCPqn6ueS36kJ+GZlVXKvqfEDVfe59xS9ay+PTcnf0N8iWe49MDAHy7eNACYnas6h1qbOryAY1HfdYnmtgajsFJsc1uDDugTPD9pQOlc1TlekDtdXk04GMf6WVSoZn5f0f6fAa8E/kQ42As8BvwR+D7h6MhJnJcQQgghhBBCCCGEEEKI81hfbHJNJyd7nBATkbBWnJusoNaDVaWZGf+YqAnkUh+o9CiKWeUK6IrqIO8py8U0J8NAJdbarkO+GTkHkJ6XJzUvn9cO7i+J92cqjOqKPs1BWq1w7h9KeFzuJb3e3OZqrzMfMNRraxLKwny+0GpZIRxMP2o8EJ0X+0M6EEjoi7RnBmZ5nkJzGUaL9xr9DTteY2699na3O+/VPN4n8ze6H6yYUbl7ZP8/tRxgtDZ1pACzdngxt2z5hF4S/Oigw3HQ4VKS2eUHFqZeYbR5tumLS/p4Pr8vWj8wIzFH3Vv2jL++vT4ZuvBaDEV3PFO1iWY6J/UUdq7qPPZ6vifu/4DVgB/4EFa4/ovRveGoDrydcPCTQANwNfAJ4GOEg8sIR3tO8vyEEEIIIYQQQgghhBBCnIeqAp5pPU6IiUhYK85VeayQNnfY1nDQlY1rSvKA25lLaomZv9hrxvd5NN+sdEDXIT2sZfWMVuLwGoa3Ip8YVBSfxzTzPvBhVX2mAd/+Jarrr73/mbnJ/IH/Ev5q/jP+Ok9lcI9yyWv/EUu6XK64qrpnZPKKff0UVstx5YX++/OPz1upAtkbW74yGoS+oanDDbhueva2dNI5ki/JBbweTzwbdBzId5VcrfdVXeXEmoMCZHy5Us+6/m86sMJod8yRS/UR4EVqZ5aKzlRr7wP5Fz/3xqphT5+Wd2QTv7/kezqQu3BgCXaVba65rSF/kr8Gk5EgHN0BQDj4DuBJwsF3Eo7ed9hR4eg+4AfADwgH/wt4FmgC7ji10xVCCCGEEEIIIYQQQghxrtENk027BumLpakKeFi2oIJlCyqoCXrojabHrWpSgOqgdezRxtFU5ZTdhzg7SVgrzk1W1Wz28G1BNZ9RKuI97kAm6ky5A3mFcDA58xfRROr9M7Y73XizI4orn1Fn5mIO3+5qJf8nnzdbncuzKJt1OE3Stfl8/H993tQuF+XDs36Tq0zvceQMd/ax+FuNcu15o9bzj5Il2awHGAEyhKNGpC6kOLy6e9bS4WBp6b5CQGrNLxxUW3sfUAF3lfNZ97+Z30/ODD+pr//APWhKPla/YJ75jZdVleipaidZdKx1ZAsBcBCrIrXsoMPU73K+Prqs55oRNVelAEowM9MMZmbmgfKFj1452NzWkG1t6nBhfd8bWIH2mSMcNQgHPwN8kXDwR4SjqQmOGyIc7MG6dyGEEEIIIYQQQgghhBDiuG3c2sPa9dsOW5+2JujhjhWLuGPFIlav24zC4W0oC/HrHSsWjYaxRxtn+eKak38j4qwlYa04u1jryvqwgtDssQ4vljjgcudzVBk5M5UedMSq6kcAfISDmrcSDxDzz8wbrhJ9KJ/SMnXZvLbH4cikFSXeWl5WsnfQy8/DW41XQPrOr9UO/2xeZ/Y/d+1xxfVK84X++3NL/L8pxQpTCz+zCz+v1XxKcw3t8PsqZzyZfPGF38xWu541jP98rlx1YtY4nzZj+gz3Eu0Bz0xXdyXh4MCKClyAq0y/wHw6uCd4WXamWe1/Rt+SWOHVcVcA0TfPeF8irlcYvxm60+HN+V0L+i8PGPkyh2pVFOtVji6zRB10Jc0yb29uUbS1qcPEasmcb25rGL819On3M+DzQDNwD+Hge4AlwAPATqzW1m8DLgPef5rmKIQQQgghhBBCCCGEEOIcsHFrD6vXbT6icrY3mmb1us3ce+tS7r116REhbPWYEHYy40hgKyYiYa0425hYVaHGVE90+PN+kprurM4MzViUSgIah38P+ACcXnOm05tPAbtfkUwZgCez1607t+qOyLqQK7Ryv/kpyH7q/d16a1OHDrgXef8371JTOSCJVbHqSw04Hd11IYDM0/OUxL4Kr+cq1fRf69iYT/U7laGoz6u6jMwNc7+Zzg4pzkSv1/irsshzVVnE7zVNBTA+2T/o1ZS0x6sujBs43XPdW6LdmavjgOOpxKu8gdTeTMVAZ7LMVHUUZgaGN47sq10eMzQt6FVjJZd4H47FjarY67/2vnykLuTYVvc2vbf66imF3KdUOJonHPw68B+Eg/cCm4DrgTZgNhAHngZuJhz90+mbqBBCCCGEEEIIIYQQQoizmW6YrF2/bdwWxyZWNdba9dv468cauGlR9YTtjSc7zk2LqtFUhdtuu4329nYAHA4Hc+bM4Y1vfCN33nknHs+pW//2m9/8Jj/60Y/YvHkzsViMoaEhysrKTtn1xSES1oqzSzhqAImxm//+kff4dZx+B9mBZV/4pk44WPhvW69fMM/16b5+9TUlRtBdYuSBAOCy96vAAawQcJaeVTL5rLnfXYIGZHMpxaE5cL5GjendB9z63JcPqPa5eaxg1gCMbalXGC/9Uku8talD9arDnhXldzpKnXvS9vhOQBnxweAer57wpNLOEt3s2xIcDi5I6rMv3ONOORzGpuS1zkh+tjPuHHAvzx4AUNyGqfe50pmnPeWJ4NA1/oQxQ7WvG9j//HznrAN9xpz8n1xd8y8rcee1qpJUv3fRtu/uHJ63WOsrqxvpz/97KmHMiF9h3atnUdf3ldf/6uP6SfjKTE44etu4nx9+TAvQYj96AnjrSZ6VEEIIIYQQQgghhBBCiPPMpl2Dh1XLjmUCPdE0m3YNcu3CSq5dWDkt4wAsX76c7373u+RyOf75z3+yatUqFEXh7rvvPpFbmpJkMsny5ctZvnw5H//4x0/ZdcWR1NM9ASGmi1cdcVzi/bOPcNAFlADBLW6X+4p0xjs/lZsLVGMFvXEKa8YeWtfWAejxXrdzsKs0b+TJ5VKKdrCztGT/pqD+3O+qciWzUz6HRy+xj1UBmqtfazRXvzbT3NZgALw48C3HFb71pTF9huYu1c1QVyQBKJftMR1veMSMXbyXuKci5/VW5CpCK/enkwdd5oEnSpXkQVfsQs+IvthMey9MzDCez1yR2ZZ8eXJ9wD9014zKA7d/+ouxhDFjEGu9WgAt5wrkRsrm+Q7OuMKzq6o/s3+mLzYSnJ90ODJKjfK0/2Lvo9mEMUPjUDCdsj+EEEIIIYQQQgghhBBCiPNaX2zigHUyx+mGyaM7B9jQuX/K47jdbqqrq5k7dy4333wzL3/5y/n9738PQG1tLV/+8pcPO3fJkiWEw+HRx4qi8O1vf5vXvva1+Hw+Lr74Yn7zm9+M7r/zzjuZPXs2AwMDo9saGxt56UtfimFYjUs/+MEPsmbNGq655ppJzV+cPFJZK85u4aAKuK8OkMYKMt1YQWo6ccClVe3ymR+fk9Crco58rlofcHqNODCCtfZpoQ2y/so5s9U7D/bnrqzMqg5vXlUdZFWHqZTOS7lVh6lUXJLIZkYcTlUji7UmbIE1TjgYJxw1L/c/aKSNkrhHjacIRwsVrEkOrWOrukp0B6AlDrjcubiD4V0+46IVvf6hvGdmWbY8l8rOT22ONWj+PTtL3vWbbfoXujpT9e31Hq6FzlWd6damDrc30eutjG/TA7U5dX9mnrfUvOxgyhfbvG/uCxjQr1KrXZF8PDszbt9fHiDUFTl9FbVCCCGEEEIIIYQQQgghxBmkKjC5lsPjHbdxa88R69ge7/W2bt3KI488wvz58yc9FsDatWv53Oc+x+c//3m+9rWv8Za3vIXdu3dTUVHBf/7nf7Jx40be9a538cADD9Da2sojjzzCk08+iapKHeeZRsJacbZTsQNXwtEs4aABeIHsvkfKzRmXxS6cp+UCubhzj5FV9+E1HFjBaaE9cRpgv9Ph7Ha6nFf7s5rLbySBGGCUVGf7sUJdzTczFwXyhKP5outnAI1w1Apjw9G8B+KEgx7CQTfhaDLUFbHephIOKlhVrrnYXnc2NegqK6lJD8+oj6U0J2WVyp7e5zJXx1/8+a8kMx/5d83PNo1Da/NqhQu2XXt79iVby/ovdnkq5vuVjGrWpQK5CmcgV+H99aKvxkY8/c7XdX4kmXbG/R2L7s49/N6HJKQVQgghhBBCCCGEEEIIIYosW1BBTdBDbzQ97nqzClAdtNanLbZxaw+r120e95yjGUocqgP77W9/S0lJCfl8nkwmg6qqfP3rX5/SeLfddhtvetObAPjMZz7DV7/6VTZt2sTy5cvRNI1169axZMkS1qxZw1e/+lW+/e1vM2/evCnOWpwKEtaKs1s4miccTNhr2RYep3MpxVEZijvcpTmH6sDprU2bDo9RDuT7t/ozvppMtatEz3WUeQwPpusV8cTITYmEgRWmRkfDV8i0NnUUAuFcc1uDOeb6OjBeGKrYH2O3OQBTdZppI6c4nCX5rLciZwLJjLsvP993bw7uYlngJ06WobOM4upcpb69XgNcfwsl/HuHFjh1RddcFZuSL36uNuPExRX7Gyqjnn7noLcnun3GZnfcOeyrb6/Pd67qLLR7pnA/zW0NWYQQQgghhBBCCCGEEEKI85CmKtyxYhGr121GgSPCVxP4r8YQmnropX7dMFm7ftuUg1qAT/56Kxnd5GAsw403vpS2tntJJBJ86UtfwuFw8PrXv35K411++eWjn/v9fkpLS+nr6xvdduGFF3LPPffwnve8h1tuuYU3v/nNxzFrcSpIrbM4aw2/rbpk8M01Aaw2xF67chXC0Vyi1+PX3Ealw6v3uUrznQ6PkQMqAb+RV0tje72ukW5f/r6y0sB9wdLAF5qfMypMMw1EATfhoD9SF3JG6kKFClz3dYH73Hbb5cOFg9qR26IpwtHEmG0GkCAcTfvvPahXXzWSrL5qxAe4tiRenfuGe6nzM5XlTvvoLJCpXzBPr2+vd2MFvR7AB+TyWi7e79vfu7cskvXk/O7nyyKObbP+lqkdXhyfP3RZYu5Inb50/8uTS/e/Iv+Fb+XNH7/x667Wpo5CjwUX4G5t6jhy3me5+vb6G+vb68369vqyKZzzvfr2+l8VPX64vr3+yydhekIIIYQQQgghhBBCCCHOIMsX13DvrUupDo7foviuDRE2bu0Zfbxp1+CUWh8XG0zk+NBPtvCnZw+yeX+SHWk/V1xxBd/5znf4+9//zn333QeAqqqY5uFxcC6XO2I8p9N52GNFUUbXoy3485//jKZpdHd3k8/nEWcmCWvF2SccVAgH1VxSC6hOsxIrfHRQqGQNBxV/VSbhKc8m3UFD0VwmRh63fUyq4tKE6q/KJNJDzv4rM5nEDalUyg56ffaHkhnRCuGoG8jOc/0zs8T/W6f9uHguDsBHOOia3NyjRn17vVLfXu9fM7PSidXm2NWTrZvt2vdmz8V7SNtjmYSj2X/tWeB8Z2/pjBWxuAfIXRNz5WY/+Xnjzh1z1Xd0vdTd0PUyytLV/f2BvUpvYJerua0hWZaZOQJQlq7Kv3RrVWJuPx5f6oAfcLY2dShYrZvTzW0Np7w9sh2MmvXt9W3j7Gu1933vVM9LCCGEEEIIIYQQQgghxPlp+eIa/qtx0bj7eqNpVq/bPBrY9sWOL6gdK503RsdVVZVPfOITfPKTnySVSjFz5kx6eg4FxCMjI+zatWvK1/jJT37CL3/5Sx5++GH27NnDXXfdNS1zF9NPwlpxNvLEe1ylyYOuvKFjDO/yOtMjqqPvycCMncsucg4+6/OZJg5PmZ4GEj3/KM0e3Fqi5NJKFPA5vEaupCbbM3vj7syaweHEO6OxpN32OIcVnubcXxyMY61nm2luazBXVPx3HqvCdmwwqwN5xm+FPMoOaIvbIisbSvwGhdbIJoOXPfd88h3/zAaycc2DvUbtioOz/VcMz3Wkchd43jWQKXn33y+Z+8b9f5izP3tplZrsrbt2W3rG+362Lq+Yav+zMx8fAfCmDjqc2RHXkLs38Mzc0lmPXfV+VzC6MwokmtsaTPvjyLfhnDrPAyvr2+u9hQ317fUe4M3AntM2KyGEEEIIIYQQQgghhBDnHd0wuWvDtnH3mfbHJx7o5C/PHGT7gfi0XdcE1vyyk79t7+d1r38DmqbR2tpKQ0MDP/jBD/jLX/5CZ2cnq1atQtOm1ihz7969rF69mrvvvpvrr7+e7373u3zmM5/hscceGz2mt7eXLVu2sGPHDgA6Ozv55+YneOifO/j1ln08unMA3Tiehs9iqmTNWnFWidSFHCUXlGuKitvhMZwuf05LD7ndyV6P39BxlM5P5pMHXc7BZ/2efEpVA3OTGX9VxpMadpHocfeULUhXAGVAlnDQCeQyUYfhDgc1wtEM4WAJ4CEczIdWcmgtXOvnZgYrsD3ECnlTk5i63x4j0bmq0wSsn+jhoBdQXlVxTy7yv7NTCa/bURmK6629D6ha0+9mXuSp9M7MjQz/oSKavW3vpa6EWWmUj+Tyz41cHkhqw07D83QOML/3yc/n4PMAXPv3sNpfvkjpftEyJeNya3urZmbe/sMfGTeOmVB9e70L0DtXdZ7qCtvNwELgdcAP7W2vwwpqR98eVN9erwIfA/4dqAaeBe7qXNX586Jj/gX4MjAXeAxoL75QfXt9GLi5c1XnkqJtHwQ+2Lmqs3Yyk61vr38rcDtwKZAAOuzz+456ohBCCCGEEEIIIYQQQogz3mRaGw8mcrz1u5um/drDyRxvue/v1AQ9vOINb+Nzn/sc27dvZ9euXbz61a8mGAxy1113Tamy1jRNbrvtNpYtW8b73vc+AF75yleyevVqbr31VrZs2UJJSQltbW2sXbt29LwbbrgBgMp/+SAl9S8HoCbo4Y4Vi1i+uGYa71qMJWGtONs44vu8TkD3VqUTJTV5lzto+Eb2umOeYF4rW5jyD+/0GLHnvV53Rcbt8OkVioN8xUXJPc89OCtbtmD/QcCLXSUb63Fpekrz6jllwGcFqCmsCtvDvzesUDbB8bMqWa2A2A2kCUfz1tq2wRxWZa6aT2nZb6d+puyo2FxRlppVYyZv6IsddGstW2uyL7yqI/dcRbV7VjZdoiejlc/7Lx0arL54T+jn3zFgNNx0f/xCRV+0d2+mOv6W9O/rfpbqrtiq/LS9Re1c1TnarN6u8nXb102ewH0dr+8Ab+dQWPsO4LvAjUXHfBy4FWgCtgM3AOvq2+sPdq7q/FN9e/1c4JdAK/BN4AXAF07CXJ3AfwHPAFXAF4HvAf9yEq4lhBBCCCGEEEIIIcRJoyiKBtwGvAzrta7Dum+aptlwGqYlxGk1Xa2NJ2tG44eO2NYbTdPruY7vd7yfYDDI/ffff9j+VatWHfZ47Jq2AMPDw6Of/+EPfzhi/1e/+lW++tWvjj4Oh8OEw2EANm7tYfW6zYwdtdAG+t5bl0pgexJJWCvONhnA5QrkytwBPZtLqZgGnlxCc2R1xVmaJ6m4zTwoSd+MXK+3Ij9s5HFm4uoQ4IvcPzsb6opECQc1QHN6jBI9rWbzSc0KLMNRHdAJBw2sCtuUHdQeEg6qWIFvlnD0qO2E7fa++c5VnRmAf7RU+YO6UXFxLjfIaHVtNB+pCylOf97/vP9aFdAzWlI/EOg+WP/c7hE1ny0bCczn8fgtiqKYlcaM/myP/0DiifLu55684P/M9vb/9HWu6kxitU52tC+/1HjNtvc7NHCXpmcYWEGxaa9X6wVynW2dufr2+hRW2+fTYR3w2fr2+vn24+uAldhhbX17vRv4BPDyzlWdj9rHPFffXn898B7gT8BqYGfnqs6P2PufqW+vr8eqxp02nas6v1P08Ln69voPAI/Xt9eXdK7qnL6eF0IIIYQQQgghhBBCnHxfwQprNwBb4YhsRojzTlXAM63jqQpMtXuwibVm4tr127hpUTWaqhzrlGmjGyZr128b94fB6ZzX+UTCWnH2CAfVOdd7Av0Rv8MVyLtKZqfLUgeduZHnPf5sUnX6KvNGos+ZU0wlpfnyyUSvJ122MJlVVVxY68+qWOvLjoaynnBwyFOeVwCTcNAH5AlHs1jBpwtIR+pCGuAB0qGuSN6ejQootWs2aIDW3dKYLTouE+qK5Ovb61V3zufMOJNK4bp3V5QZN8cSeTDzFxfdWmjlfiW2z+1wKdty2yB/2cHr4wCv+3WD+bPXtuQHKxe7BzOUamQTcS03+GzJpvigb5eO9T2sAHSu6sx94dZFSkNnr3qg6lee7Re+bCRS9ZjPmw14LjtwXdy+n0I1LZ2rOvOcJp2rOg/Wt9dvwPrDUAE2dK7q7K9vry8cchHgA35ftA2se3jC/jwE/H3M0I8yzerb668CwsAVQDmH3m04Dxh/IQMhhBBCCCGEEEIIIc5MK4F/M03zweM5WVEUHagxTbNvzPZKoM80zaktrCnEGWDZggpqgh56o+njevfCe2+8kEq/m4oSN9WlHq6aX84/dw/RG01x14YIQ4nspMY1gZ5omk27Brl2YeVxzOT4HKsN9Oma1/lEwlpxykXqQipghroiU/65F5iTNpL9Tt3IKt2Kw/T3PRnwZEecJH1VqYyRRTm4LztjUcysvhLHyB6vvu+vFW5PZbZEdZALdUUSkbqQGqkLBQE91BWJE46aff+5VKty7nQZOUXrj5S4HG+eTcUlo6HmkXO01rGNAbBmgw/QatdsyP3O2qtih6dNj35FHXEPKHk1l8fuUPDSZCp7WTabvDiXNwFu+eYljm1ut9kJRuCCTDrA9mxzuOGwEPWND6zJtjZ1qM5sLOPKjCSbfvjWZGuTP/HNO+4ygWzR86r8C7gPzLzQF4g/nwok48ZFB5fqLsPjXzhwpc+eW7a5rSEz1ef9JPkO8HX78+Yx+0rsfxuBfWP2TWX+BvbXo4hzvAPHU99e7wcesj/eAhzECmkfwgqOhRBCCCGEEEIIIYQ4m2SBHSdw/kRldW6KXqsU4myiqQp3rFjE6nWbj+v8S6tL+dclFxy2rRBqel0aq9dtxqoYm5xT3ZZ5stc71fM6n0hYK06pSF1IAfxYlaapKZ0cjhqEg2nVaVRmR1xa6qBDcwX1asVBqkebHx2KuxVtaCiVHnQ5PWU5Uv2OEkNX8yPd/hjgGrauncQK2fIAv3vF7aXOihv8l83U046eg4mBbQF3zbIhP9bPzTThqBkKk6fQsniM37nW0G+WKu+9aIRQS0SnEOJazNJMpQ4YhIMK4H+v1ZI4CvgJB51vKfHN6HJmYva8/FgB8REVr3XeP7odsYO+8if/mYnUfUZp7oqYrU0dDqzv4UxzW4MZ6oqYf33R6zPbF74+kPWUlQElCweWOrOOhEM1HAontubuybAR62thYoWfxbZhhbLzOld1/mmC8yPAa8Zsu2bM44NAdX17vdK5qrPwu3DJFOZYB1QCazpXdT4PUN9e/4IpnC+EEEIIIYQQQgghxJnkC8DtiqK8zxxv0csJKIryAftTE3iXoijFr5dqwA1A1/RNU4hTa/niGu69dSkf+dmTJDL6lM49Whvlwrhr1287avXqZMc7GSZ7vVM9r/OJhLXilAp1RUw7sHVH6kKZUFfEiNSFXIAS6opkCAedWK2IzUhdyGNvHw11+7f6TX91xpHJuFVTx3B4dDMzoplz8psCqkPpD14UD8T2ec3BZ0qipfOTHqdf1xIH3AfSAy5HsDZZUnFJQt/1vzMHwQqOy4IL1Wyq1DnwrBZbvHl/tr8ulC+7MAWgTbgerTVHD5AMqZgP+7wKzCypb69PjbYWDgeV5mpcQIZwVCd82Btn/MM7fWXJuCe++Eo9O8eZzHIooC0lHBwmHM21NnV4AKO5rSGb8u7x1Wgp1ZlPOmF0LAcQeLbyH+5/+9xPEi957haTy5sLa9BWAI6SbOnetKllDpbsdv146V35zlWdZ8xbXzpXder17fWhwudj9sXq2+vvAb5U316vAn8Fglhr2450rupsB9qAj9S3138e+DZwFVZb5WIPAzOB/6hvr/85sBx4FTAyyWnuwXpH4Pvr2+vbgMXAf03xVoUQQgghhBBCCCGEOFNcD7wUeJWiKE9jFZeMMk3zdROc9yH7XwVowl5qzZYFuu3tQpy1blpUTYlr66TDWgWoDnpYtqDiqMctX1zDTYuqeey5AZp/uJnh1PjRw2THm27HagN9uuZ1PlGPfYgQ0ygcVGdePmJUv3DYu+BVB7w7l13kxGpL69517cLKgS5/BVbLDADVWZJ3EA6W2AEp+YwajCXKZg6lq+dFB8oWOL261z8r7fMGMrNKq0eCvqpsqXdm1jPzihFHRV0sZeSUTHrARceNrfmD9debWdPjwloLVQ11Rcyy6M5M1cEnRrRkKgMQ6ooYgBcrTHZH6kKV9lq0xUys9roQjqbeP2tmwn5sFB2jPurxeD5dWV5S316vEI6ahKMJ7FYg2biW3jb8Mtdjez8Vzw+8Nm4Hw4PRbq++75Eyf+ftryv1qQM1QOl1bS91Peierf3Jtyi1d/YNxuNXfUz955U3uJb9/U4tpu3JZdVB/4UDS6rs+/JiVaQ+B+z+x7yNiZ7S5/Ix96CTM/DNGZ2rOkc6V3VOFJz+F3AX8HGsKtqNWG2Rd9nn7gFeD9wMPIn1x+AnxowfAd6L1Wb5SWAZcM8U5ncQKwB+I1a17xrgo5M9XwghhBBCCCGEEEKIM8ww8ADwJ6Afqwtg8ce4TNNcYJrmAvu8KwqP7Y9LTdN8pWmafz/50xdieuiGyaM7B/j1ln08unMA3TDZtGuQA7HJdfMu9AO/Y8UiNHWi7uCHaKrCdRfNoOX19Sgc2U98quNNp0Ib6OJ5nAnzOp8oU+h0IMQJGXnHLNXhNUpTQw6vkWVeZsThje3xRUC12gKDGlyQzM6+ejhmrwsL4aCKFUJmCUezQ2+t8eoud2l31wK/Z/CAq2z+SKr8wlQ6NeRyZIedSXcw50n2uXXvrHQe1ekf3lWSzA0S67ixVXErMdfc4UfS8574jQfrZ0yhJbKysPFAzhXQidw/Oxdaud8JaJH7Z6tAKdYfLUaoKzL+22nCQQdW4JwmHB39hrr1fy7xPOlxO4FE56pOo+h4X7zH7e7atkTpCSzNrniwZbRlSKQu5PJUZt2+6wLq31Jv9xmmmivzbXE/742q+4ffMFSWqVIAvarvn15/fJ+iVXkUh2v3zH71qsEBZXHOVLQSVE3Hatsce2L2H4LebMBVmarp/8XlX9CLWgELIcTZTH6WCSHE2Un+z17I73AhhDg7ye9wcU5QFKUUiEajUUpLS0/3dMRptHFrzxFtiWuCHl61uJrv/K17UmOU+Zy0vK6e5Ytrpu36d6xYdFzjTZczdV6nwSn/vSdhrTglInUhl+o03P6qjDc94lDcwewF+Yw6M33Quxt4vmR22lN+STxfUp2NRe6frWFVqTqxwlRqrh5KugN5rwnp4R1+Z7Tbl/NUZL0OXz5gGriyMedQLuZ0YFWuOlWXriUvvjiYzAZSFdufPPhc7b9ku2sbtebq1+qxfW7/0Ha/MxN16P5ZWZehK4NzrhvyDTzj9/Y9EUyFVu7P29dORu6fbWJV+joOzLwyceNffmQU3ZMCaKGV+zV7nknCUSvQDQc1wKxfMM/sXNVp2tW5HiAdWrlfzadV1/ZfVRfaH/sBJbRyf3w07A0H1fsOfM91kfvPc3zacCBr+vb+oHRmdFfFk87ZIxelX7D3VV5AvTS1Pr8w9+dAzKwa6h5ZGkw5S1391S8swXoRZJc9r0qstr/R5rYG+YYXQpwL5GeZEEKcneSFXiG/w4UQ4uwkv8PPQYqizAQutR8+Y5rmwUmep2F1onsZUMWY7p2maTZM4zSnlYS156dCxWxfLE1VwMNQIkvzjzYf8Ydp8TqGk1ET9PDXjzUcd7Xp2HktW1BxRlSunqnzOsVO+Q2fcW1RxTnLMHKqHtvnHQLMfEpLmiYj2G2BZ187lMyn1BlDO7wakHOW5HH68o5kn8cPKEPbvS69Kl9W5tKTJbPT3sC8ZGZ4uy+Qi2kzMzFnHybBsoviI7mEqueSjrJczBF/PrVkX/lwlw+YcWH3g0ONG7+QJIwauCCjeMpzqdjzHlBwVVyS1ICkkVOYecWICyiskWvYa+zm+ivrefqyd/mfburINrc1ZOz9LsAfuX92IrRyf6KoGlgDynY4nenOVZ0J+1gF6w8XBXA6PIYaWrm/sDav11mSc+gZBS0cTBOO5ghHjXRThzqsz0l6tWhyR/oGR9DZH9gf3JGt67vWjRX85p7xrki/vP1LSYAdr1mbdehZo3wwEsi4y80LKiPlzvhI6Y78S7JZT1kW6/t9/Gb4QgghhBBCCCGEEEKIc56iKH7ga8DbOBS06oqifB94v2mayWMM8RWssHYDsBV5M5Y4g41XKaoq4/9Ha2K9eK8oYEziv+qeaJpNuwa5dmHlcc1NU5XjPvdkOlPnda6TsFacEqGuSB6rihQAu9LUBBJOfz7/7C9qDH91OptPaxqQmr1sWFVdhrH3r9pgLqmVxWfpFV01DqV6WFMuHcmUeYJmJp9V0vmMNoShZp3+nBGcl3IPdXtLjJxZoVXkhl+z7OupyP2zc0A5oEXqQqq/usLvr87k+rYEVcDprczmD2wuM0JdEWNmOJij8AaacDTd2tSh0NTh4cbWHJD2qYMevzqow+ibw3KAorl1B6AQDubsylrXQU31/9nn8b62vT7fuWuPGVoJhKMxgE0f/Xd9WclP1KKWyYOVoYSiuU0XYNitn51+9VvG3tyV2b25K12AOS86MwlkLul/gdM+LwOorU0dztruBz3ZkjkVQ+V1qYt2PtDfP+Nyp5534MzojrL4jmSf5wU64Glt6jCb2xpGvw5CCCGEEEIIIYQQQojzyheBlwArgL/Z264Hvgp8AVh9jPNXAv9mmuaDJ22GQkyDjVt7WL3uyAraowWxJjCVZrR9sfSxDxJiEtRjHyLESTEC7HOX5VJzXjzoueg1vVqi15MycspIaOV+Mz3kyB98qtSRizuD/qqMx5czPJpuJrwDWj474kqlo47MrCXx4dnXDO9zzTB0Z4me9VXlkvm4I5bPqAO+qowBlIRW7jew1m/VAUV1GOW5lFqJ1do4nxpwpbBaLkM4mgHioxWyoPrj+9yXPvvjsoaHm7W3V71T/bcZ/89duIFQV8QAhmdeHssAzm0up6u+vd53f8Bv7HE4ov9wuws/1l35tOqM1IWU1qYO9+PxlSWtvQ9QNI5e/oOePOFo8lvbbtI2PfOqsu2p6/wXOLf6Zjp2JOz5DzW3NaTstW8NrKDWbX84FCPnrxzY6iwb6so/e8ktjJRcUNLfX5NKHXDud+RT/VjBchYwWps6tJPw9RRCCCGEEEIIIYQQQpz5Xg+80zTN35mmOWJ/PAi8G3jDJM7PAjtO6gyFOEG6YbJ2/bbjLvv+l8WzJnVcVcBznFcQ4nBSWStOCzvoTBMOKsl+pzq8y68AHodXN+I9LofDq+eSB50uRTUDrtJ82q2Z/aE/eROAGcO12+HLefNJh2E4HBXx0jmliVxwr/ufj6eTfZ4cEA/MGWbkeY8vM+w0AA3QQ10R/blrFw6lBtwBrDbCiVBXJAtAOOgE8kXVrjS3Nej/+7LVqaq+zYWAVh/nPnS7EjZzr/mvxuzoYKXHHMiCYj7pcY9gBavJ3X+c4cNamzZtj1NYm9bqfR6OmvXt9c53OmddsGe4RusdeWlfjf8Z74uD92U9d+4bGXNZHSt8dQN5VybqHAnUemf2bU74UweTg0AwujPgzCTKBv1Zc9j1S/eTZT8dWDL8NR3wYr1JI9ba1OEC8s1tDQZCCCGEEEIIIYQQQojzgQ84MM72PnvfsXwBuF1RlPeZ5lRqEIU4dTbtGjys9fFUXTmvnM17ovSOjD+GAlQHrfVchZgOEtaK0ybzoUqn5lHcvf8o0zLDzlLsRZuHn/MHVaeeWfDK/sRzG2cmszGnko1pOayw0+30Z+cZppLKJpUe/+x8zm1mR2bN2O4pyWQcQ9tLlFBXJBOpC7k5FEwmsFswX/jozlikLmRgtUbWAf764ds9892Xe+e6n0phhamjXvHHe7PY6+pS1MYZoLWpwwG4V1bOMyude7TZ8VozYcxQt7jiuZszmzJ/e8e20fVhs/eHcgB2C+I8QKQupIZW4rHnaFX/Kq5kzn2R/4B5Ve4q1y8zj8dvCT7V1GE0tzUUrxXhcqUH3LrDb+oOj5F1B41saWUyWRnqf169Rr9g35/8NfsfObh18btcGX3Il1bc7uu3X14Sn4lp33OutanDi/X9r46955MhUhc61h9ua0NdkbB9bBewAJgf6or0jjPWS4H/B1yN9TXuBn4HfDHUFdk3wfVrsP6QfAFwEfDVUFfkg8dzL0IIIYQQQgghhBBCnMUeBdYqivI20zTTAIqieIE77H3Hcj3wUuBViqI8jVVUMso0zddN83yFGKUbJpt2DdIXS1MVsMJSTVWOOO5E2xN/+sEuynzWaoTWuomHFK52x4pF415biOMhYa04LSJ1IdVTXlZacWnCVBQzrzqNuMOrux1e3espy2dzKTXlKtH1GZeNzNCcpheFPb2Pl48AHtVlDjucujNQk3WW1SaHZ7qeygCBA08EHFhr0+aw/kjIAE6gDCsMTdmXN+0PD5B4duBFTiOT8CgDz8fnTG7uCuALVV+jROrequzMXJOudO7Jlw5dlzf9G/VfVPemw6uih4WToa5IZswYTsAz0OXXK+sSOkDnqk6j9dGO6LA3YwJKTK8iZ3r81c5tamsTqea2BhNgwa4NmeDITm/F0DOZjhtb3QCvc3409Wz8RRcccF0ST7vKvTvm1+W/3PDfvWsebI4u3rXRYbr13NaZOLGCZw/W936GMX9MnUQ1RZ/fAtwJXFq0LQ4QqQtdjxXA/hxYBdxdPEikLvQe4BtAO1bLlm5gHvA24CPAhye4vhs4CPw38KETuhMhhBBCCCGEEEIIIc5etwMPAXsVRXnS3nYFVkHHKydx/jDwwLEOEuemyYalJ8PGrT2sXb/tsIrZmqCHO1YsYvnimsOOnVHiHnv6lA0nrZfOgz7n6OeFx29/0QJuWlR9wtcQokDCWnG6qOkhl2P/Yy4Aj6c8G1A0syTZ5xpJHHTFjLTDmHXlSKW7LFdu6kq0dE422/t4uQH4MkPufe6yXM7pS6K5SEXun+3wV6ejiV5PDiuIVEJdER0Y/tWKsH9m3+YLKqLbnZG6UBarLbFm/2sAXPb0dxRXdkSNZb0OIEc4qBatWzuhmt7H8g03/tIBuAlH480ANKTgc4cfGA4WwtHE7o5KdX7DgAaz84D+TPcLMk+UfdATbF7ni+qzdSBjaO44kK/zPpyL6TP6uzNXaVihczZSF1IWgDvhmeHMOvxprMCVnuTF+h71hd685lX2VFenIrOeUwD3W37SlG5tusQA1Oa2hgxAa1NHBsjZVb6nRHGFbKQuFAXM8apmgXcCPwL+BHyForA2UheaA3wVqyq2OHDtBv4cqQuVHeX63Vh/iBKpC73jeO9DCCGEEEIIIYQQQoizmWmaWxVFuRh4C1Bnb/4x8EPTNFMTnzl6/ttP5vwmoijKx4HXYc05BTwCfMw0zWdOx3zOR1MJS0/GtVev23zEGrS90TSr123m3luXjs5h49Yewr95+phjqgoYk2nkbZp88GUX871HuhlO5RhO5vjSH57l/sf3nJJ7F+cHCWvFKWFXo7qAbKgrUmjHO2JvUw1dSSkmeT2nZTCUGYBzeIdvOD2i+c28pib2ewNYVaGVQCYz7Ezv+1vl8D6rjW9ZoteTnPPiAW1wwF2a3eWN2uOz74IbDFcmGg3En0859TRACVZVbR47rC2N70nZY2cIB12Am3AwRThqhZmFbZAgHDXs+SfsfRWARjiYJhzN17fXOwA6V3UWB6EmYO74bZXTW5kNJHpdRmjl/lhr7wMs3PHLwOKhn/jU2X6lLz+SGIzXqMGR52aBEqUtOripqSMOlBbmCmgpT0XpUNklwZ7Z15vAgLXmbEPuf5s6ugGHphiYqgsOVc26AWdrU0eyua1Bt9eoPePWqY3UhQLAG7HaG3cBwUhd6MWhrshf7EPeiPXfy+fGOz/UFRk+FfMUQgghhBBCCCGEEOJsZppmEvjW6Z7HFL0EaAUex8o1PgP8r6Ioi0zTTJzWmZ0HphKWTjfdMFm7ftsR1wbrhXcFWLt+Gzctqub323rHnWexQh3w1990JT3RNHdtiBz1+sOpPF/+4/Yjtp+KexfnDwlrxanixAraDKwQ0W1/AIxkR5wAeVdpzp+NObKaN68NbveTSziy3oqcYhroWGvWmljBaiF0jWK16Ej/T52v5IIdasnc/WrsYoBwUGmuBqoZJBzNRepCqn19BUhf8KJBg3DQFeqKZu11bEt2d1Qas68dyh/YHFQKLZF7/hF0ak7D6ynPJUqPvK9hrAB5NuHgAAvmqfa84kXHZIFsLu5QTF1JlC1Mmq29D+gzHTscpupQa3sejM64xDD+MvKuvDrQb5aOdI+UDz2TsDv6qvYY5a1NHenYyz5qVO1rd7mVmclY6fwSe+woQMPDzQaQ3rroHSUX7gnywr1fMVof7XDa5zvsr4E+1S/cKbQS2B7qijwNEKkL3Y9VaVsIay8GRkJdkZ7TND8hhBBCCCGEEEIIIc46iqK8BvidaZo5+/MJmab5m2OMtQsmzsJM07zw+GZ5dKZpLh8zj9uAPuAq4M8n45rCMpWw9GS0RN60a/Cwat7x5tATTfPYcwMTzrNYdVE18K+37DvueZ2KexfnDwlrxamSw2p9m4vUhRxYwa0TqyWxF0iVzE5nyi9K6EPdHs3IqE4FJZiLO4dTA6441n+rOlYImrD/VUJdkWykLpQHXBf80acMlhCdnzMLP48V+zwTyIW6IoZ9LpG6kFI6L+3FqorNwWwF8CT73Pkdv65OAZ5IXcgMdUXywzv8Ofs8c5ywthAeu+z5Zb7We9BJOBjArsS1708NrdyfaO19wMDAeeuM1WbQ0Wsyg5FCBW/hr43Wpo4s4FwKmea2Br21qSNdeJ5K4t2BgYr57p6K57sXxvHa1yRSF1KipQu8GVfQKBveruiax4EV9FZihdlZDgW/Z6p3AOuKHq8D/hSpC70/1BWJceRa7uOK1IWKg/J1oa5I0/ROUwghhBBCCCGEEEKIs8qvgGqscPNXRznOxHod8mi+POaxE7gS6+XNzx/X7I5P0P53cKIDFEUpLhgCCJzUGZ2jJhuWbto1yLULK6f9+n2xia9drP2R7qPOs+CeN1zBdRfPAKAq4DmhuZ3sexfnDwlrxSkRWrnfMbTDZ9qtbjNYAWIea41ZP1AV3+/pL78oEQjUpHP9TwfielYrhLIA5ViBbz/WL9iUHb4S6ooYkbqQvuQ5M2NvNwEi989WVaeBkVPz3B/SACPUFTHtCltf35MBveqKWJJw1Ozo7TD9sb2ZiuFnsxfv/EXWvqZuj5/FCjuPFI6akbpQAoiHVu43O3ft8dnnGQD17fXqh4MB3h6N5QlHTZo6FEB9Jv0SfVnJT3KbYrfojzd1lADZ5raGbGtTRyG8Hr1ec1tDprWpwwA8ZZk5hkvL44vflGpua0gSDjpi73aXOEuC2UjdrfmsM6Dc8Lf/GAZ8qpHVtl/8bwkg3dzWkOMMFqkLLQKuAZZF6kJ3F+3SsCpuvwU8i9UaueYY1bVLij4fme65CiGEEEIIIYQQQghxNjFNUx3v8+Mc6yvjbVcUpRl4wYmMPVmKoqhYofHfTNPcepRDPw7ccSrmdC6bbFg62eOmarKB6v9uOzCp4/oTmdHPly2ooMzrZDh1Yi+fn6x7F+ePM73STpwLwkEF8Kguw4v1ZhMfVpVkFkhiBZsVQLmepze+35vPxV1OI6sFsN5QoNjHFNaOVTnUWh6AUFckF+qKJAFHpC4UsKt3MXKqhhUIzwS8kbqQF1BKZqcd/lkZD4e+BxwOPe10Z4bUUFfECHVF0oXQd8y9+AgHg4SDxe8w89rXKMgTjhaqaj1frCjXfv7IBWakLuRsbmvIAPFl93xTJxxNP55YqQFVHHonmOHMxs2qA4+77FAZADtsTS9++r7YNVseyjQ83Dza3jgbc/gx8ZTE9yfzTn/Sfm6Uufv+RHNbQ6y5rSHXaoXEZ7J3YrUruQIrbC18fNHeB/BzrP9m/mO8ASJ1oTKAUFdkR9FH38mctBBCCCGEEEIIIYQQZxNFUd5mV5yO3e5SFOVtJzD074DXn8D5U9EKLMYq8jiaz2K97lr4mHP0w8V4JhuWnmiV6kSWLaigJuhhul7gLp6npiq8/boF0zqmEMdDKmvFpNnhoRvIFKpaJyUcNTMfqkwPPev3lcxOK6kBZ07PaGaoK5K3x9RQ9RLFYZT2/jPYZ2Qcg/Z1HFjB7oj94cRqZzx+lavFsD9MrCC2sEZr2t7mAPJzbxgcsa+htzZ1qIBrxsBWdd7ejmO9gaGwdq6HcDAJ4CmfkUoPuUw7nI2POT4DqJftMT2AQTioNlejQjT1eEuVtrPmLSV1B+vNASXg+2LzhsCH2xpjD7/4LTFXdsT9zMW3eDuaOlLNbQ0G4aD3LQmf1s1cbbjsIq9q6oW3/2SMvNKfSzh44y8/agC0NrU6qvo2Jw497vAAztamjkRzW8Pkv26nSKQu5ATeCnwq1BXZOmbft4EPR+pCl4W6Ik9H6kIfAr4eqQuVAt8HurH+yHob1nP/kaNcZ4n9aQkw036cDXVFtk3vHQkhhBBCCCGEEEIIccb6LrARqyVysYC97/vHOe4bOEpL4umiKMrXgVcDN5imufdox5qmmcF6fbZw7kme3bmpEJb2RtPjrlGnYK0Du2xBxUm5vqYq3LFiEavXbZ7cOnkTmGie72u4iO8+sovh5NSra0/2vYvzh4S1Yio07LATu83vZLm/NJDzv+4CByo+36zMwcof78/CaHvkkeFdnu2o5iVOr1ER2+MYxGp5HOPQWquTCWrBanVcWJfWsM8vrFdLpC6khboihQA3BdAMtDZ1JHXNmQRMwkE/4Nzx26pkLu7IFwfTkftnZ8suSiTLLkwa3opcCWAueGU/gEI4mCUcPdRDAehc1akDeuSzoUKA7AHUP33oo74B38vLK8sfdQTMZz1Gvtp9ILM0BcRu/MsP07+4+TNmb/U1bqC8takjv8z/BkdgZo++f3Zdrqf66mTaO5PrAcJRcyboM+3r2RW03r6qpTpW1TJYXyud4/89drK9Bmtt3QfG7gh1RSKRulAEq7r2w6GuyDcidaFngY/ax3uxAtvfYlXhHs0TRZ9fBbwZ2A3UnuD8hRBCCCGEEEIIIYQ4W0yUd80Bosc8WVGeGHO+grUe7kzgvdMxwQmuqwBfA14L3Gia5q6TdS1xuKOFpYX4+44Vi9DUkxeGL19cw723LmXt+m2TWpd2PCbjz1NTFVpeV0/Tus1HnFN8v6fr3sX5QTHNMzW/EWeiSF1InVJVbUE4qKajWiATdXoP/DMYsytrk4SDHkB55hezfE5/bk7J7OyQK6Af6Pl7eb4QqkbqQh7AU7UkmsgltZKhZ0vSWAFuFlAKAW6kLlRok5wKdUXyx32T4WBFLqm6n/tdVdzIqblQV2T0p7/dXrnCU5nJLLhpII8VXDsHU7OuUlQzVu7uexKrXXOOcDRvt4A+9GFv+37f/7iGqn5XscU/rLw6pmvPlT+jPFTq6PvDOyNpu9K3UOEbAJSQ9w/OMm2/vvSe7/X/7HX3OBdvu49QV2Tct/q0NnU4y4ae9ZXGuvU9815hALkzfc1aIYSYIvnjRQghzk7yCoaQ3+FCCHF2kt/h54CikPUK4Gms1zULNGABsNE0zX87xjhj14A1gIPAw6Zpdk3fjI+47jewii/+FXimaFfUNM3UJMcoBaLRaJTS0tKTMMtz28atPUeEpTVBD3esWMTyxTWnZA66YbJp1yC/29rD9x/dPaVzy31O/vHJmyYMVo92f8Bpv3dxSp3y33tSWSsmLVIX0gBfpC6UmUSF6+HCUcMTDiZTB90JPaNVAEakLpSB2blQV0Q37g85cgllOJcwjPh+rwerqhbCQXfZwqAyvNOfLZ2fckR3+VxOfz6XSzgUrGBWsdehzWO1OtYZp+rXnjtFVbXj3Z8DMJz+qoSimTEjp8KhNsoFOpBJD7gzQF7PKT4jh5Ya1rRswo2/WvO6AnoQyBMODmOFrbPzWUwFZbcWDsYJR423Qaa+/dN9H3haDVy02+/62TVa9IDDUfgDqdACOtHc1jD009fdo9YpPyl3l+YNgMXb7tMAZ6QuZExwP4Yzn6jIO/ymlk9FdYfXwKpUFkIIIYQQQgghhBBCnJ9+Zf+7BHiIw5dzy2J1r/vFsQYxTXPtNM9rslbb/z48Zvvbge+d0pmcp5YvruGmRdVs2jVIXyxNVcBq/3sqq0o1VeHahZUAUw5rh5I5Nu0aHD1/rGPd3+m+d3Fuk7BWHMFeR9aDtZ5n8TusTA6tBzt14WiuPBx0DURKormEw4m19mthnde4kVOTI7t9Kla1rEE46ALcNS+M5mo27EkSDirpIWc+l3AYoa6IHqkLKVhVrEH732SoK2K1/g0H3fY1C22Jvfa/8R1XXewuqcl4q18QjRKOmvY9u7FC0lwu4XAA+uhYRUJdETNSFxoJrdzvSA85nNFu7wxfdSrhzw7up9+ldT89wzXn+oGob0bexH73ha6T3j2wuPS55LKZN/3gSyOFsTpXdeqDb55tGka/ds+Dztyc33cVQuYsoBfWl63fdp8ZxZUGV74S6Jt5ZbYk9rxx1Za/jAa1dvtjl32uEiuZN+jIJ9Ed3pHmtgZ597oQQgghhBBCCCGEEOexQsiqKEo38BPTNI+vl6w1hgbcDITsTU8DvzFNc8JCmRNlmqakYmeA4rD0dDrWOroT6Ysd/T/7o93fmXLv4twkYa0Yj4rV+kIt3mi3P04c96jhoANweypybjsQHcZutxHqiphYVau6fayGFZ7qWBWzEI6aMaslsl2VixHqimQidaF+wLTHKHDkM4riOLSAfAY7ZPZXZ9x6TqnseTxo1MCIXVFbhlV9migcG6kLKR03tmqAc8GuDekFux/UQl2RfGjlfg0o03OKw8grpplz6P5Z2YO5mOZJ9Zvs/kNVNrRyfx4rPB3RNAaeiL12ZtQxVxvzXKgVlzACxAhHR6uBm9saDj0Ph56bBNih7GXv8gP5qw5/dh329czmtoZsa1NHAip1CWqFEEIIIYQQQgghhBAFpmm2n8j5iqJcBDwIXMChdsQfB55XFKXRNM2dJzhFIY7paOvoHk1/LINumFIRK8446rEPEecbu5o2MeVWx2NE6kKKXaVrCUfzQCbR4y4EszDmv8FIXcgdqQsFI/fPdmVjWgZIFapfbWms6lEvVphLqCtijAlqefZXs8znNlSNPu64sVV9/oKXqJG6kDq80x8z8sqB4Z3+NOGgOueGAafq1BWstW51rJ/tJYBnlrPLEfL+wV+TfsIHeHe/+EINq+o4H3vek00edA+XzksPOb3G0Mz6RF8u4RgKdUWy2bgWSPS5qtNDDo1w1HzDjz7a987v39JTNMVCJS+FoLa1qcPf2tThO9pzaoevhbVyi7fn7OcmZz/OAy57TPnNI4QQQgghhBBCCCGEQFEUTVGUjyqKsklRlF5FUQaLPyYxxFeBncBc0zSXmqa5FJgH7LL3CXFKLF9cw723LiXoc076nLs2RLj+7g42bu059sFCnEJSWSvGZVfRnigP4Nh7U21iznVDzoNbS/L9W2e7sULFPFZYaUTqQjn78zxW62AHoOzcMKsc6AuFD1XzhroiObCCYI5cT3aUntZ0rHVdHR03thqe1EG3auTdWCF0ksKaDOGgU1Hw+GbmsvH9mm5fw4zUhfKA/vKROxUj6NdT3pw2gl+d/5fndMJBA0gOP+f3l8xOB7CqXvOAGlq5vzT/8TJ/vNebw0D3VOQ8hIN5QHk6eZPx8Mh7XUC2uZoUoIwJoo/aYtoOXZXmtobUmG1uIGsHtsUUTsNC2EIIIYQQQgghhBBCiDPWHcC7gC8A/w18GqjFamt85yTOfwlwjWmao8GuaZoDiqKsAf423ZMV4liGk2NfFj+63mia1es2c++tS1m+uOYkzUqIqZGwVpxMOcCcc92Qmo1r7kSvewZWe+VhIAn4gABW0OkECHVF0kA6UhcKYlW3jvturtDK/SpWsDlRYJvFagvsam5rSLY2dSSrDj6RB2tN3kIYHbl/tuqpyObTg65U8fq8oa5Iyj7W7y7LGZlhf6784nhw+G3VJWXfj8br2+tdn7gumXy1mTIOdgbcuZTqCsxNurxleUd6yOXPRh0x01B6Ki5JOrBC2OBs19PGDMdz5izndpNwNDV2wsUh7AS8gBb6jw1GSoXulsaE/Xw67eciU3xwc1vDca87Md1amzqO1YlibXNbQ9g+tgtYAMxvbmvoHWeslwL/D7ga6znpBn4HfLG5rWHfBNd/HbAaWIIVbj8NhJvbGh6a+t0IIYQQQgghhBBCCHHWegvwbtM0NyiKEgZ+bJrmTkVRngKu4djVsRms13THKsF6TVaIk0o3TDbtGqR3JM1dv316yuebWBVOa9dv46ZF1cfVErkwh75YmqqAh2ULKqS1sjghEtaKk8YOP/MAybdVZ9JDTid2sGhXr2awAluvfVzxL/MEsCPUFUlF6kLKYW2Ow0HFHkfBbvsbqQt5gUDZRYlEzQuimVBXNB+pCyWxglIaHm7W7XM89rZCkKmkB10moNvVuh57zEKr+3Rm2KmGuiK5PQ3za/Ipd/DF7fXbr0ml/FyadgRimaGeTWVuPaN5MCjLDOrJrqh3vzLo9FTEcQ/v9CtAJrRyv17u2G9cG2jnQK4u2NrUoTe3NUz1j5ccYKSKGkc3tzXkW5s6UoDe2tShlQ1vZ+mWLyuhrki+tanDg/U9njgD1q4tfovSLVjv0ru0aFscoLWp43qs/x5+DqwC7i4epLWp4z3AN4B24PVYQe084G3AR4APT3D9G4DfA5/AerPA24H1rU0dVze3NTxx/LclhBBCCCGEEEIIIcRZpRrotD+PA0H7898Cd03i/N8C31QU5Z3AJnvb1UAb8JtpnKcQR9i4tYe167fREz2xOiUT6Imm2bRrkGsXVp7wHGqCHu5YsUgqdcVxk7BWnBJl3+/N9NSFXFihbNLenAdGsELawxrLF4LeSF1I09y6/8C/zs3P+vXz1nnhqEk4mODwlsGmohmmb0bWE+91a8/XhVKFlsn2eGakLpTg0Jq3EA46QytRd/y2KpuLO3xACut7QgVUza0bl7z2QIZwNAegKGrOyFH6YHvc/ZvX6Mlr0mknYC5s7FOT/a7k8A6/7izJGL11mbLUAdNzQZe5HwO19q/PmYSDMYARvVrtyC1zbfHknM1TfKeZ3eY413zk9jxAa1OH150ZcgG5SF0oxY2tJpNfW/2kKq6QbW3qiALmeFWzwDuBHwF/Ar5CUVjb2tQxB+udfV9tbmv4UNE53cCfW5s6yo5y/Q+O2fSJ1qaOfwVWABLWCiGEEEIIIYQQQojzxV6swoo9WGvPvgLYDLyQMZ37JvABrEKKR7ELabBeU/0NcPt0T1aIgo1be1i9bvO0vuD9h229UwprJ5qDtFYWJ0rCWnFcjqh2PbTdBSihrkhmzHYvh9ZPVbHaF5scWqfVLDrWCeTt8TV3aa7cVZqPEw5mCEettsfh6GFr6hbaJxMOOnZuqPIAHnvd2UKlrF5c6QtwsDPgcZflnA63kczFR+eWB2YAnvJL4s8DXsLBNOFozsgpfZpb9+Qziq+hzZ9M40+xcr9hGoru8uv5uTcM5gHPVc9TnUkaruqlqaSnTDcIB1OARjiaWwzGq9dsSENeZRrYa9Y67Hlnco6SLKB86N2asm/G7RqQ7FzVeUYEtsfS2tQRAN6I9U68LiDY2tTx4ua2hr/Yh7wRq7X158Y7v7mtYXgK11Kx2rWM22ZbCCGEEEIIIYQQQohz1APAy4C/A18D1tlVsvOALx3rZNM0h4F/VRTlIiBkb46Yprnj5ExXCKvt8Nr126a9MumBLfv4ROOiSbUwPtocpqO1sji/TUtgJM4vkbqQBpTYwexYhVbHY2lYAW3W/pdQVyQX6ook7P3K9hqckbqQBytgLYyh5zNav9Orx0aD2qMJR/PZmCOFVSWrYoV7ZfZ8VXv+CkD/0wGz5+9l+dSASwOSHTe2Oh+/6j+cedUVAMqiz/kUe755gPkNA9nSuen+0jmZrOo0XIAWuX82239VrT33uyonoI487/aWpnFU5g1vqt/lTA87UoB7fzbk3/SR9xTuKbUy5sq2NnX47bB1QrVrNjhq12w42jGOwvPV3NaQe9PP3pcNdUUy+2Yoqv2cKzAaTp7pVgLbm9sanm5ua9CB+7EqbQsuBkaa2xp6puFaH8VaR+On0zCWEEIIIYQQQgghhBBnBdM015im+Rn7859gLR92L/AG0zTXTGGcHaZprrc/JKgVJ9WmXYMn3Pp4PIOJHJt2Ta6e51hzKG6tLMRUnQ0BjjjzmFiBqzHOviSH2hwTqQu5InUhH9YatMlQVyQFOCJ1IXfhmFBXJPuDl6r0lSnzBgIUeg7k7H36wk07EiXf7DusZbA9rpvxOQEt1BXR7Xmq9pxNu8LXbwe2SSOvFkJdBTBjgfl62lOx3VOR3Vl1RSzT2vtANnL/bC1SF3JF7p/tG3neG80lHXrFJQn/jMUjml39mwPykftnZxM9HiOXcPQZBruGd/nUgWd9gSf/dqX515G3Ox5P3OJqbepQulsajbm6BocqjcdVu2aDhrV+60T3CXZFLYdajgDQuaozA8Q7V3Ua9tq1/tamDu1o1zsDvANYV/R4HfBGu+IWDq0jfFStTR3xoo+2cfa/GbgD+Lfmtoa+aZi3EEIIIYQQQgghhBBnJdM0HzVN84umaa6fzPGKovxCUZSPjbP9PxRF+dn0z1AI6ItNf1B7tLF1w+TRnQP8ess+Ht05gG6Yk57DyZyrOHdJG2QxZaGuiBGpC2UAX6QulB67NiyHB2qFCk+K2iaXAs5IXai3sO1FEcMd8yo5V44RIGWvMevEal88XijsxArvxrZbVoAgVpB80J5LJtQVidv7jTHzNCJ1oXyoK2KG7MrfvTe9Q624NG4oFV6vZ/dBrWfWMlfNgU15wBl73puqfuFw1swrKcVhFgLl0Z++6dsrk0ZeUWP7vC5FxZGPO1wH1Qsr+9PzMjhwAH4g3tzWkBk793EYFFX2jqe5rcFkgrVvi9of5+3narzn8YzQ2tSxCLgGWNba1HF30S4Nq+L2W8CzWK2Ra45RXbuk6PORMddZCXwbeGNzW8MfpmPuQgghhBBCCCGEEEKcyRRFec1kjzVN8zfHOOQGIDzO9t8BH5nCtISYtKqAZ0rHqwoYk+yZPHbsjVt7WLt+22FVtDVBDy+YX3Zc4wkxGRLWnsUidSE/QFEr4ZNxDRWrsjNTCE3tbcXrzU4o1BVJR+pCmTHr245gtaD1R+pCiVBXxFzYSwrMQuVtodVyYb3Z8d6KkgQUu7rWgVW1a9ohbxS71TJWO+TRCtbCWrp2u2XdbsVcCIYJdUVysee9HiOnKN6QK5v2znSmPZU64MMKmTO9j5eZ9py03rqQXnxvux6qAhgJzE1lUwfdGuBzVCbmBmO7Dw6XXxID3K1NHdrny1JOwOhuaRw3aAXobmk0GSfQtStktea2hmxrU4cDUJvbGiYcp7mt4bC1es9Q7wT+DDSP2f52e9+3gJ8DLcB/AB8aO0BrU0dZc1vDcHNbw7htV1qbOt4EfAdY2dzWsGEa5y6EEEIIIYQQQgghxJnsV5M8zsQuvDmKEsYvHslhvX4qxLRbtqCCmqCH3mh63EBCASr8Lj7ZGKI66OWq+eU83j1I8w83M5zKjXOGdU510MOyBRWj2zZu7WH1us1HXKMnmmb9U73HnGfNmPGEmCwJa89uxwxLp4HD/shjVaFqWMFldmxIbLc7VsZuHxPUEuqKZCN1oQRWq2LT3pa0x1Cw1pkthLS6fU11vApe+3il6Fy3PW7aPu6IalL7uEJlbq61qcO9pPxSX8XQMzn7uplEryef6CXf3NWQgwYidb8rVLgm7flp9vPiKlQX2+P6geyc33enI3UhgGww/vyAceDvw8Pll+QBV0wxnYFswhlz+Qtr+AJgr02rdLc0HqsC1gU4Wps69ML92mvfZpvbGkw7wHUCabvy9ozW2tThBN4KfKq5rWHrmH3fBj7c2tRxWXNbw9OtTR0fAr7e2tRRCnwf6AbmAG8D4kzw7j279XE7cDvw99amjmp7V6q5rSF6Em5LCCGEEEIIIYQQQogzgmma07kcYidwC3DnmO0rgW3TeB0hRmmqwh0rFrF63eYj1sorVGp9+rWLWb64ZnT7dRfNoOX19axetxkmOOeOFYvQVOuRbpisXb/thAKXlS+cNzpeYcxNuwbpi6WpClhBbvF+IQpkzdqzWKgrkrIrV0/ad3eoK5LFqlotBKWG/aGPc3ih2vYwkbqQalfjFo+bKQS0Y2hYYaSz6JpuwDN2DFvWbnHsx3pXl4K1Ju6Ez4kd9CaAtB1yup5bsKIQxGKvdevCavPssUNoJ1YYaGCFyAkOVfy6InUhB1CGVQmbjtSFXO6yXKDqyqh55ZOP9DQ8/N1Mc1uDDqRu2Pzl3H2/b+G+3392bLWr56XPby7/zdUvP9abKNJYIaOOVTmcs+er2VW3Xqwg+Wz5/n4NUAk8MHZHc1tDBIhgVdfS3NbwDeAVwAX28V1YbY1HgHuOco1/x3pOWoGeoo+vTNdNCCGEEEIIIYQQQghxHrgL+C9FUdoVRVllf3wf+E97nxAnxfLFNdx761Kqg4e3Ga4Oerj31qWHBbXHc86mXYOHtT4+HrUzfKOfb9zaw/V3d/Cmbz3G7fdv4U3feozr7+5g49ajrfAnzleKaZ7xhXfiKIoqXTNYoZ0DrHa+0zS2EuqK5Iu2ubHWhE0W1oEdc47TnkO6UDUbqQsFsMLUQTv8PdZ1HTMui+nR3V5nLu5wY4eiY++puMoXq2WyhhWiVgDRCcLgw7Q2dXiwgme9ua1hNGi279O078WJFdI67GNNrJA4Y4e0XvvaXvseM5G6kDZj8UhpsDalu0r0OOFo8djqwRlXBGIlc9KNG+8ZbXP88ve0Od644+HgDfueTE1m7kX3oGC1Rc7b9+PECnPP9NbHQghxvOSPFyGEODvJW8iF/A4XQoizk/wOP4coivKpo+03TXNsxex4YzQCnwCWYBWTPAWsNU3zT9Mxx5NFUZRSIBqNRiktlY7NZ6vjqVadzDm/3rKP2+/fckJz+/G7r+HahZUTtlMuXHGicFmcMU757z1pg3z2K65mLcEK6rJYwe2J8mBVZ8aKtuXt8Sdq1esCAvZxOTv0BCtwHf+ccNCBFbqmCUezoa5InnDQV3JB2tH9vzML9zHe/9Qb9nXyWCGt296WYvx1Ew7T2tShOHIJZ1l0h+OWn3/osOC5sLZtYRy7qtdhj+/EasHs4lDL5gQQLwTboa6ITjgYBbTioNbeZ3Q0dRTGGQ1r//A/TflI3VeGi5+n4vVpa9dsKLReztrr2QJgtzouBLMZIC9BrRBCCCGEEEIIIYQQYhyvHfPYCSzAen1xJ0e2Nz6CaZobgA3TPzUhjk1TFa5dWDnt51QFPEfdfzTF698erZ2yaR+7dv02blpULS2RxSgJa89+KsDclwwQ7fb64vvdUSOnHRFU2lWoTiAzdg3Zo8gw5h0EdovgYXvMwhqxuVBXRI/UhdRZLxjWczGHGd3ldUTqQqZ9TSPUFYkxnnBQwQpqXVgha4HurcgZdptnP0eGxoV2xqn69nrlmoihfPhXhorVPnl4MjfX3NZgPvbCV+V8qT5npO6bjuIK4rHstW8Lwa1pz1mzdyft/YeH0VZIO1GoPW7lrP38FiusT5unDCfW85ln/DbUY4NbIYQQQgghhBBCCCGEGGWa5pVjt9kVp99jnGXKhDhfLFtQQU3QQ280fdR2MBOtmVtY//bRnQNHbadsAj3RNJt2DU45dBbnrrNlTUsxDnst1RJATfQ53YqCR3WaKqBF6kIldkBbUGjnq4031nhCXZH8OK2HlUhdyGuPXQiAHfZjv57SnJWL4gnVaTqxgsYEEwSTAISjJlbV7XDk/tl60fYM4dGfaGkOD3LH8j0WUn32cZniHZG6kMNuVTyuYKw768wns0wQfo7HDlQz2KGpHdROSXNbg26vOXssGayWxob9ebK7pXHScxVCCCGEEEIIIYQQQoijMU1zBLgDWXNWnKN0w+TRnQP8ess+Ht05gG6YR2wDK3CFiXvgvueGBROuf3vTomoe3TnA7ya5Jm1f7MTWxxXnFlmz9iwWqQt5sH5upAGXohleU1cTWG/O8ACpQqWmXQWrjlO5OdVrjq4TW1ibFat6VLGvmQ11RfL2dnOyQabdLtmFFewqWOu/psasl+uyxzwsQK5vr3cBSueqzsOCWvucEoBQVyRuP1/68aznaz9/fqwq4oy9TbXnqWCF0hytarno+MzRqniPxW6N7AbSxevsCiHEeUT+eBFCiLOT9PgS8jtcCCHOTvI7/DygKMr1wHrTNMtP91xOFlmz9vy0cWsPa9dvO6zatcznBGA4eSgqqAl6RsPaNb/sPGxf4ZyW19Vz06LqI9a//f223iOucSyF9W3FGUnWrBWTF+qKpGE0SHSauporCiLj9j4nh6o/T7gi0253nBgTwlYA6VBXJBGpC2mRulAAK5A8rB2zXeGqTxBo6liVqoW1XN1YIXQxl/1vzh5PA3w/ta490Rq1hefICZRitVI+3vV8TawqYgOrqrhQXVsITx32czPRCxAKVjX7lL/RC19jIMeNrap9zUJQLoQQQgghhBBCCCGEEMekKMoHxm4CaoC3Ar879TMS4uR58Kn9vPdHTxyxfWwQC9AbTbN63Wb+/YYFRMfZH03mWL1uM/feupTli2tGt2/c2sPqdZun9K7Ecp+TZQsqpnDGidMN84iQWdbMPXNIWHuGs6tBjaOEkQUGY9YqtcNMD1Y4OeWaejvgdHN4hW6hlXLarhQNYoWoebuKNWvPxRwzlgOrqjTLmFbFYLVcLszfXhP2iGMYv52yOfZa44xbqNxN29efslBXxIzUhVJY1bV+rHC5uMrWxTGCU/s5HH/tXqC1qUMBtOa2hnzRYy+Qa7D+aHIDZnNbQ661qUOXqlohhBBCCCGEEEIIIcQUfWjMYwM4CLQDnz3WyYqifAq4xzTN5JjtXuD/maZ553RNVIgT8eBTPbzvx0cGtRMphAzf+suucQMHE+tF+rXrt3HTomo0VUE3TNau3zbl9jEnu93M2GB2KJHlrg2HV/4WKomLg2dx+khYe+ZzYP3CHDdkLGpvnLQf+7BaBafsKtg0U6moDQed+bTqcniMFMwe7wgv4I/UhQY4VFE6ghWEzsQKhofGaX+s2/dwzKrWUFckF6kLHbEW7DiPdewK4kmMmYnUhbJHa1M8iTEM+/nUsJ6H4n1ZJhkE218zD1bYWxywuwBXa1NHyg5sFftahefOxA60m9sajNamDqW5rUHaiAkhhBBCCCGEEEIIISbFNM0FJzjEHUAbRxbW+Ox9EtaK027j1h7e+6PNx3WucZRX3E2gJ5pm065Brl1YyaZdg1NqfVwwnMyNjjHdxmv7PJ5CJfHYSmFxekhYe+ZLHGO/C3BF6kKF9V0Pq1uf6vqsAxG/2zSUQGyvpxCM5gFvpC5UqJiN29sKFa0DWAGsglUNWwgYxwar41bLjrcWrH38tFeNFoJa+5pgBaaEuiKpKYyRsyuW3fZH8jjWwlU5FMIXh7WFz3UYDWTjViAbgULQHQ469mUXafBpV2tTR665rWH0p25rU4ejUJkrhBBCCCGEEEIIIYQQ00xh/MLAK4DBUzwXIY6gGybh3zx9Uq/RF0sf9u+JjHEijqygzYzb9nk841UKi9NHwtoz3CQqQQsBrW63JVYY0w55LDus9GEFpNmi7V7/bFfAyKqkh1yavbkQ0rqxAskkVkDpw6ou1YGYXeU6FKkLqYWg1W597MJqo3y0+xhtZRypC/nt6xTWmlWxwtxkoRXzRPc0mapZ+95L7Hkf9aePfW11TPUroa5IOlIXGgac9jFOe6zcmPM1rBbW5pjzD1v3t/D1aIB8cWANMEHlrLvSsVv1qYP5pFEx+py0NnW4AHdrU0e6ua3heNflFUIIIYQQQgghhBBCnKMURXmA8cNWE6t74g7gR6ZpPjPmvCEOvY77rKIoxWNoWK+5tp2USQthm8y6q1/v2EHvyHirLE6fqoDnsH9PZIzjNV4FrTLFvHVspbA4fSSsPYvZQaEDyNhrqsIx1nAtogBqcRjqLs86MJmhqGYCGAbSdtCYtYNXw16z1sehqtBc8fXGVMRq9ofKBK2Y7Xmb9jHFcyvcXwnWurgwwXqvdmWrszgAnYh9vRzjrAM8NijGCqMDkbrQ0NgQlcPX5U0yppLYDmp9TLBe8DjzVIrOM4v322Gu87EXftJM+mtcL/Tfkl4W+Alv/8Yb9EhdyB2pw23PrxDcS2WtEEIIIYQQQgghhBBiPFHgZqzXf/9pb1sKlAH/C9wCfExRlJeZpvm3ovM+iPXa43ew2h1Hi/ZlgW7TNB89ifMW57nxwsmx665u3NrDl/7w7EmbgwJUB62QGGDZggpqgh56o+lJr0M7dozjsXFrD6vXbT7imuZxLpo4HVW+4sRIWHsWsAM8J1YIpxdVajqxKlcNrCpZk2O3TS5U68btscuwK03LLkw64/tdmeSAawArPPUQDtrh72wNu10xVogZB+J2lag2wXUykbrQEcGhXZVrFIWiiv1BqCuSsI9RscJOBUhx9HV3C1Wyk/pRZFfGTvQek+LtGezvETsQztrr1nqxQugE1nOhceTauQbWc5UDaG3q0JrbGiYMrDn09SixNxeP5wTcl2QeUvZ6XqQ+nliZWvaFbxqtTR2OhXNvKp3ZvyXNoRbThedS1rIVQgghhBBCCCGEEEKM1Qv8CHifaZoGgKIoKvAVrGKZlVgVsncD1xdOMk2z3T52F/A30zSlYEScMhOFk8Xrrt60qJq167ed1HmYwGuuqBmt5tVUhTtWLGL1us2TelG+ED7csWLRcbcd1g2Tteu3TWsAcKJVvuLESVh7dnBgtSH2YQW2I3bY6ILD16WN1IUK4a2GFeBOWGlqB6Iu7BYXelrL6WmHjq6lgAsAR/+2kqiRUxSsd0eBVSVaCFkLlbmeSF0oM7ZS1ea1/y0OH1UOr8YdL2Auzzr8LhSl35WLw5jK1aJ7ULCC31RrU4ezw2oFnCy0D7b3u7FCU489d7c9h8MC1rHzCHVF8pG6UCGwdWKt3RvDDo6LqnSLA/TCuYW2IbQ2dbgBV2tTR2oS68mOVvC2NnWozW0NBpBzlebMy7wPBWYNRDyPud5ttjYRB7Td825KD1RelrzKOqUQ6hsc+noJIYQQQgghhBBCCCFEwTuB6wpBLYBpmoaiKF8DHjFN8xOKonwd+Mt4J5um+SdFURYqivJ2YCFwu2mafYqivArYY5rmyV0sVJx3jhZOFq+76nc5Dqu6PVm++eddXDmvfLSad/niGu69dekRVb9lPicAw8lDKxZWj6kEPh6bdg1O231OR5WvmB4S1p5mxWutFoJPrArO0VCvqEI1ACiFcwpBYfFYwAx7jBGsd0Jl7O3q2DVf7SrREaw2wzP6nw4Mc6ilbwKYebAzUKao5oB9SrJoLdkAVkVuFLvi124jbHCofTIUhY9F1x1bhXqEvOY2h8ovDeRVV/6CA48ZTLw4vQdwROpCCW5sVbFC2OI3sfjt5y1mb1cpamFsPzeF9sepccZ3YoW7MfvzQstmZ6QupI/TStmD9X1lYn1tstzYmucoraABWps6HICXG1tTzW0N+damDifgaW3qSDdbYXxu+G3ViYHkHCMTKDcBmtsaMq1NHdnhsotdrU0dZnNbQ661qcOYqIJXCCGEEEIIIYQQQghx3nMAdcDYXrF1WMUgYBWhjFu4pyjKS4DfAX8DbgD+E+jj/7N33+FxnWX+/99nepE06pZcx3HJjGOnOL3iCAIJIiSUpYYltEVZscAu2UW7C2TDFsQSFthdbcyP5UtLQllKIIgYCCKQhCRO4hQ7HsV1XCWrS9Pr8/vjecaSZUmWHdfkfl2XLmtmzjnzzJGimcxn7vuG89BB8NuP/5LFq9mRwsnS3NWWe56Zdpvj7c4HNnPdioaD1bHXr2zkuhUNh83TBY44Y/doHa+WxcejylccPxLWTmDmsbrRoeTkmaEA9okh6nG4Pwvwm9AvhQ707BzaihcA0254zNxmN6FgamIAawLcsQn75yKhsNs8JhUJheNTVICmzcxYn9nOjv77NgYkUFa1KlgpTGXrhHORYjz8tAH1jIea6QnHH//YyPjjLoW6WbNvYXIFsKOQieecZcWq4UhpXSkgOeH+veb+S/Nni2EdDE8Oh0sB7MGq2Ulrsszas5FQeN/k8zNh/5JS6OoB3JFQeGhyCG62caOrloda1zalpjjOVCbOwS2Yr4PnpfK7vZlLIXOpudyhq4jxpAbdC/Z2ub773t0pyhc6mLlltBBCCCGEEEIIIYQQ4tXre8A3Lcv6N+Apc93FwD8A3zWXXwNMVyHbDnxGKfUflmXFJlzfBXzsBKxXvMrNNpxMZE/O2+KlcHj9ziEuX1Jz8Hq7zTrkcslU170cx6tl8fGo8hXHj6WOdeLwK5BpIexiQlhrwk4XunrUbTaNTw5tI6Gwgyna4R7h/ix06JgvVWhOrLSdYT+H2S81VXhsKnTt4e5IzlTAOtEBXqnqdJjx2aZFdFBrQ1fVutCBbQr9d6fMbJM3l33ogLT0eGNmBq3dHCMxcU0T5tkWJ1QQ+xkPee3mfjNwsH3wxMdSCtBtZk1us55Sq2cbut3zEf9im/NSgf75Hjzf5lwWwt2Rw6qAJ+xX2i5t2iP70IHt8FQ/LzPX1g2MTQr+XeifzWzC22kdrMSFXDjyHVvt4CZ7JHSLGqg9zwISpn2yEEK8UsmLFyGEODPJx7WFPIcLIcSZSZ7DX0Esy7IDbehgdY65+gDwX8AXlVIFy7IWAkWl1N4p9o8Dq5RSO01Ye55SaodlWUGgWyl12g6/tCyrAhgdHR2loqLiVC9HzNLj2wd59zeeONXLOMzX3nU+N50/76Tfb6GouOqLXfSOpo/qxXWVz8l/v3s1A4nMcavyfQU76SdGKmsnMAHe5FmfRUrtbHU450FXrFaiQ8ucuc1rrrcBmSmqLae6P4UOKidfd6T98pFQ+GC16xS3FyfcljBrtpsvH+OVr3azdr9Zf9rsl0b/bpSjQ1ovUM14YLkPHeImIqFw3BynUNo2EgonTSWwDd2W2Yeuiu1nvDI1xfi83By6urUADMDBkLY0bzZrvi/dRzHcHcma0NrJhJ/ZhLbG+SkC3NLs39LPbMqfwVTn04TOByuZw92RZCQUTk338zJh7FSBrB1dGX3EUP4ISuci2/SH7xQjobB9oPa8ImCToFYIIYQQQgghhBBCCDEVpVQB+FfgX014iVJqbNI2u2c4xAjQCOycdP0F6PeNhTiuhhOTI5vTw/GqcD1adpvFHTeu4LZ7NhwyD3ImFvCFt67iymW1J3h14lhJWHsEpmVuDiASCo+gwzYXOvRzmK9S62AnOigc4gS1ojVhZCk0tUxr44ktip3oSlNlAk+nWWPCBKgJszYfOuwrBZcOdKhbba4rzXi1M15RPISuTC2gP22VMcfxmO+LHBpUK3RgWWrvW6qSLZ2j0lxXZdaSNpXMpdm7Dt+ctLI51NCC30VLLTUmBrBeQE14/F7GWwdP9Tcqh34xcVhr5lnwmn8Pzts9lrA13B1JHYegFsZ//0qhcyGsrz/k9650PmdTeSyEEEIIIYQQQgghhHjlsyzLAawBlgD3mevmAmNKqfgMuwL8APiiZVl/hn4P1mZZ1pXAXYy3URbiuCgUFZ//5eZTvYxDWOgWwqWZtKfC9SsbufuW1dz5wOZD5vlW+ZwoYCQ5HoE0SrvjM4KEtUfBBGx5IB8JhUvhpA8d3nrQQdnB/zJMZWm9uTiGrswshailoHKq1r+l6tDCxHa5Zk6tE10tm8PMRY2EwgXGq1vd6BC3AATQ4Wra7EO4O5KZ0Jq49MGLHOPBrGK8NXEOXQlbZr7PAP3mcZTaCNeax3jAPMbJ52vMbOdkvIVyzKyzyqxtDB06FhlvOe3AUsrpK+BwF53T/EhKrZpL58xu7nfKSlmznilbHc/Cse5XalutAFu4O5I7DkEtmFm9rWubjvShAAfSqkYIIYQQQgghhBBCCAFYlrUIWAcsRL9H+1v0+7WfNpdbjnCIfwA6gD3o92M3m3/vA/7lxKxavFqt3zlE79jpV4d0x40rTnkL4etXNnLdigbW7xyiL5Y+2NoYOOy6U71WcWQS1h6jCdWjCdOSOEcpZByvePQAdZjZpcCQ2bac8XbASSBmAr1KYMS0OZ44MzdrAj4L08Y43B1Jm0C2YO7Hi55Fm0YHnn50yOpEz9gttSVW5rYKxitQHea4FtDHeHviMvQTrdOsea+5zTbhVKTRoWnSrMMeCYUHzTHz6KrOormtFAx7zTGTE46VnnD+8kA/ynKM7fUWHe5ipp7DTZyNa0LwxHEKQg8yM2aLpsL6WPYvzZa1A4VIKJw/ljWa41ildZiQtmDm/xLujiSm2XXGFs9CCCGEEEIIIYQQQohXla8BTwPnAYMTrv8Z8I0j7ayUygIfsSzrn4GV6PeQn1VKbT0BaxWvcn2x0yuorfQ6aX/bqtOmStVus7h8Sc1h1091nTi9SVh7HJgwMg4HKzxLMuhPGIEOJRU6tMyjw1IHkDIhaiOwCOiOhMLDjLcWdqBDuQI6ZHWh58LCeGVsqaq2NL+1DB3Ejpn9nZFQuFSTn0MHr5XokLXf7DvI+IzYCsbbGmeBlMNbSOdT9nKzhpipLC5VbSbNtg7zOD1mnQV0lW8SHcZ6zNoqzVp3mX2K4e5IzlT8etBtnBORUNilcraGXM7WP8ufw/EOakvnozQf9liU5h2XgutjXaMbfW4nh8YFZqicPd7nRAghhBBCCCGEEEIIcUa7GrhCKZW1rEPeVowC82Z7EDPXdqbZtkK8bKdqLux0br0ieNoEteKVxVJKspyTxYSyperYGnQwOYIO4ZzoOa3d6ECzER30jqKD0Dp0a4phdAhbCulS6CfRDLrVsQIWowPcXejQtB4dGB7AzIZFB7ZFc9nJ+OzaNDpMTZtjqZpwzMqlrJrMsDuZGXVWoitFDwADQIPZrs88jtLc3FI1sdPch8cc32tuKwN6TShbCrIxt+XN43OgZ+gm0dXEx1Td+nKYAPngXNwTraOly2pd23TYf5Q/f9NnXQlfg2e4OpyYRetjIYR4pZMXL0IIcWaS3ltCnsOFEOLMJM/hryCWZQ0DVyqlNluWFQPOU0rtsCzrKuAnSqk5R9j/P6a5SaHfU94G/FwpNXRcF34cWJZVAYyOjo5SUVFxqpcjZqFQVFzZ3nXEVsh+l41E9sS/hW8Bd9+yWgLbV76T/rwnYe1JZsI/L+NVtm5z0xh6xqxCh5qN6AB0wFw/HwijQ8xtwH50EBpAh7ij5hh2dGVsHToIjprjW+ZYpY+ipNFVskXGq2MVukJ4LqVWxOB3+PIeh7fgy8YcvcWsfT6wFD3HYJ/ZbjDcHUmbx+fFVIBOmGvrMI8zjZ5TmzDblObx+s32pWNUmNvHwt2Romn1awt3R2JHebrPKB0tXRY6xM63rm1KTbrNif7ZpTBBfevapuKE2x3oYD7VurZJTRf6ngirvrOqDChufP9GabkshDhZ5MWLEEKcmeSNXiHP4UIIcWaS5/BXEMuyfgiMKqX+woS156LfB/45sFsp9YEj7P97YDX6feiXzNXL0e9ZdgNno5/zr1JKbT4xj+LYSFh7Zlq3qYeWezbMuM2HrgzyzceiJ3wtFtAQ8PDop5tkDuwr20n/4dqOvIk4nsLdkUK4OxIPd0cS4e7IINAD9KIDzBHG57jGGa9SLXGb61xAEB1yZtEtjEstj/1m27g5loUOSYvoKtUyYA76yVQxYZ4qkDJtcxPmOguw8klHLj3oPlDM2rPo1hY7zW1+TFveSChcWqcdXTl8cN1mtmwaXeFbmtWbNduU5tjmzXFs5hgTq21LM3FfDfJMaLnc0dJldbR0lWb8plvXNuXR59dnbrd1tHS50OfMDlgdLV0+wG/C35NBnpWEEEIIIYQQQgghhDgz3A5caVnWZnRxyH2Mt0D+9Cz2/znwEDBXKXWhUupCdKHRb4Hvm+P8EfjK8V+6eDW6fmUja29ZTaXPedhtlT4na29ZzetWNJyUtSigZzTN+p2nXeG4OMPJzNpTzISjpVAyYSpvy9FtjbOYaknz/fPocDdgvkpzadPm8hD6idWPDmE96NB2DN3auBQCpsw+Dn29stucxfJizt4bCYVL83S96IA0x3i46je3DZvLY2abRiAdCYUPmOPmOPwT0zbGq3gxj6s0HzduAl1MJe0ouvWwKl030zk0lbuFM30+q6mETU262kL/nNyM/zxyHHoenWa/uLnOA6iTVVm78f0bX9EVz0IIIYQQQgghhBBCvFIopfZYlnUe8E7gPHRxzzeBe5VSk9+bnMrfAtcppcYmHHPUsqx/An6jlPqaZVmfB35z/FcvXq2uX9nIdSsaeGLHIH/aPsC+4RRzKz1cuaSOy5bUANAY8NA7mj4prVz6YjO3ZRbiaElYe5oJd0cKkVA4g66STKODNzs6iBsEBsLdkT7TKrgSHeyWKmrT6EC21Cq3FNiWglGL8fbGPnSFbaPNXUg4PaoqlyZbzNg96Cfootkvz3j1a8JcTqDD2lL1awodLvvMfY6EuyO5SVWyLrP+AOMhcs48hoOVmRNmxM7qb6oJar1mPZnZ7HM6K1XDloLW1rVNxY6WrgTj84hpXduUnbBLBiiYitvSMZLon78QQgghhBBCCCGEEEIAYFmWE92q+E1KqXuBe4/hMAF0B8XJLY7r0IU5oDtIuo5xmUJMyW6ziKVz/HTDPnpGdVj6Pw/voDHg4bPNYd518UK+8tCWk7KWrQdiPL59kEsWV0s7ZHFcyMza01QkFLbC3REVCYU96OAU9FxXZW4PMN4GudTOuvR9NXqmbA06IC3NO3Wig9F96JDUBuDw5pN2l6rJjDqKYHnQs21L7XZrzPGS6BbIpbm4perOUttjm9mmiK68dTI+mzePDotL+xTC3ZGECXNdQNZU1JZmthbC3ZFZtT02+7jMuTnxE8RPMNPy2AHEj7Yy1pwLL5AvzQsWQohXKHnxIoQQZyZ5F0PIc7gQQpyZ5Dn8FcSyrH3A65RSkWPc/17gcuBTwFPm6ouBu4A/KaXeZ1nWu4DblVIXHY81Hy8ys/bMtm5TD7fds+G4v6Cs9DnxOOz0jh19tWxjwMMdN67g+pWNx3lV4hQ76c97Ull7AkRC4XJ0eHjMtfATWgBPd4wcOiitBPajq1vnAQ3ocHTIXJ9Df6Ipia5mLWO8sjUNWPmUw5tPkQXmosPdUtgKukKzdLwk+n+uC4y3Oy4Ft050WJwywWsBXfVZZLyVsoPxVr2Yyy6zTRb9H8DB+bUlJrDOThXGmvN0XCpqO1q6Su2DT0qFrqkKtk0KVgswXlnb0dJVOkfp1rVNswmj7YyH+0IIIYQQQgghhBBCCDFRB/Bpy7I+rJTKH3Hrw30UPY/2B4znC3ngO8Bfm8vdwIdf7kKFKCkUFXc+sPmog9obz23ggRd6Z9zmnRfN5++uD/PfXVv5ykNbj+r4vaNpbrtnA3ffsloCW/GySFh7nJkAzo8OEI9r4/JSta25qNAhrB0dcvrQP88hdLhXgW434UQ/WZbmyKbQVbCVZnsHui2yy2y/wxzTaR7DMLpKtx5dtVmakVuq1HWZYxfNGmymjXM5ehZtxqzdh674LFUG29DVtl7Gw1ofHBpQR0JhF1CFDnlP9GxUByc36HSjz9fBiulJLY5B/6zsgK2jpau0T2aq4NZUYsfP9Nm9QgghhBBCCCGEEEKIE+Zi4LXA6y3L2ogev3aQUuqt0+1oWZYdWA38HTqYPcvctEMpdbBIRyn13HFes3iVW79z6GDr49mygM6NMwe1AP/fH3dywcIqPvG65ZzdUM6dD2ye9X0pcz93PrCZ61Y0SEtkccxsR95EHI1wdySPrk4dPZ7HjYTCTqDMzHQFXSGbemERA7853yrNik0DO9FtjkfRIaoLPZvWYy6PoINPGzpQdaIDwDw6qKxDh4Ol+bR2dFg6H6hlfHatZfYNoEPOSnNfGXNZma9Se94ChwahHsZnGJQ+wZVBh7YT5c16Z9UWuaSjpcvZ0dJ1tHMREugw+2RJAcmZwlUT3sbNTFo74PAmDnjufdfX/aX5thNJUCuEEEIIIYQQQgghhJjBCPAT4Nfozoyjk76mpZQqAL8BKpVScaXUC+YrPtN+QrxcfbGjr4tTQHEW75YrdNhaKCquX9nIo59u4vsfuYyvvOM8PtscpnXNkiPu3zOaZv3OoaNeoxAlUll7AoS7I7kTcFiFDjtLf14GgeHfXWDznb9dlTEeaoIOcu2MV9EqdDjqNutLR0LhXegA1W+uL1XD1qND2Bg6kC21Mh6bcLzSMavM/mPoELc0M7dUlZs196cioXCcQ+cTFdFP/tlSG+CpzptpfXwsFbUus5ZZz2492hmxL9dsZ+yW1tW6tinX0dJVbDzwZKBoc7qd2VgKTu6ahRBCCCGEEEIIIYQQZy6l1Ade5iE2oStqdx6H5QgxK/XlnhN6/J7RNE/sGMRmWfSOpRmIZRhOZrFZoGbZfHlioFwoKtbvHKIvlqa+3MMli6ul6lbMyFJKsp4znam2daMra0shahEdxtahg9we9EzbInDA7OpFh5ledFBbCmLz6CB/BB38lsLXUlVvCh0Me812g+b6UvvnADoc3j+hDbLN3D7jLF9ThetCh7hq0m0OoDibkLOjpcsGWK1rmwpH2vZM8/Pmf3RnXAF3/5yLFPpn7gZyJ2vWrhBCnAbkxYsQQpyZ5N0JIc/hQghxZpLncHGQZVnXA18APgs8w+FtlMdO8P23An8LNADPA3+llFo/y30rgNHR0VEqKiqOuL04fRSKiqu+2HXUrZCPhs9lJ5k99jjh+x+5jMuX1LBuU89hrZQbAx7uuHGFzLU9c5z05z2prH0FCHdHChzaJngIDs57jaOrXgvoYM+F/rmn0cGsD13h6kb/j3M/OoT1mOOUWimn0cFtOfoX1c54m+S6CcdLokPeAOMVu5hjF4BCJBR2M0UYa5Tm4ComVMWasNcL5COhsEKHvtP+5Wx6uNWjjxE5mW2NS7N5i5MDaROoF49Hm+JYRdDpS/T63OmhYsZTnWVCy+lV31llAWx8/0Z5E0QIIYQQQgghhBBCCHG8/cr8+wsO/SCWZS7bD9vjOLEs653AfwAtwJPAJ4FfW5Z1tlKq70Tdrzj17DaLzzaH+cv7nj1h9/FygtrGgK6eXbeph9vu2XDYJxR7R9Pcds8G7r5ltQS2YkoS1r6Chbsj2Ugo3F8KCCOh8Ag6YC2gw1iH+bKjWxmXqnFzQCrcHSlGQuEMOjzNMd5quYh+8i0w/gScNN+Xth0B7Ca8TJtq2KSZvXtYGDtBDlCTWyJPWItCB8F5cx9TGitfaB+oOdfd1dKVPcnVtRaTPnVhglof+vHOuvr1Gx+833fWjl94q4ciiQteeDzd0dLlARwr9/4+48gnnRl3FRlPtWpd2zTx02s+829iikMKIYQQQgghhBBCCCHEy3HtKbzvvwG+oZT6FoBlWS1AM/BBoP0UrkucBFV+96lewrQ+2xwG9OzbqaqoFDo0uPOBzVy3okFaIovDSFj7CjexkrNU7WnCQxdQgQ71MkDGzJa1oStibZFQ2ItuqZxFh7sFdKgLuqUxjM+mLVXZ5tEVtsUJ+5UuE+6O5CKhMGa7Q0RCYQ86qJ0y0CzNtjWhbdF87wQKk1sjP33hp0sVtSf1r164OzJVSFpkPOw+TEdLlx19njKYNtSta5uKyrLZMu5Kti19q/dPLV2WJzVYLNhdatM5H843Pdw68tx5H8dsP9Erru2zEEIIIYQQQgghhBDi9KCU+sOpuF/LslzAhegWzKW1FC3Legi4fJp93Oj3ukvKT+gixQk1cSbs6abK72b9zqEZ2zQr9Gzc9TuHuHxJzclbnDgjSFj7KhTujhQioXAMHeyVQlqXaZucRLc7Bl1Fm0EHtHagHqhEt0pOo8Nep9nWhv59SgK16JB0AFOFa0JgC90K+JCq2Qkc6L9ZM1afhrsjMTgYOnuAXCQUzqOfeFPh7kixdW1THojN+qScQCYwn+mZxGL8/DnN5cxffPPN8Y6WsiI6ME9d8eTnnOgwW0GErpYuL+B46NoPp+b1PGaFuyO5je/feEzPWMG2TheQj7Y3H3EesBBCCCGEEEIIIYQQ4tXNsiwfsBBdFHSQUuqFE3SXtej3qA9Muv4AEJpmn78H7jhB6xEnWX2551QvYVpHEySfzqGzOHUkrH2VMpWo8QlXORivQi2gK2cH7j37uvSuijmOTz39fb9bFdzoCtEAOkDMoIPbFKZ61ATBw+Y4PvSc2QK6ylahWyMnSoGtCVzLzLEOVqVGQmEL01J5YtWsCX1hvHNA2hy/3Gx/sOr2TNG6tinf0dIVb3q4ld76C53uzGgemko3O4B8Xd+GLKC2Lnmrp6uly9m6timHaRk9r+cxJ+CIhMKFKc5VaT7wlBW3wbZOh9nGYY6XnGq7Sfu4gWK0vXm60F0IIYQQQgghhBBCCHGasCzr47PdVin1n0c4Vh3wLeCGaTY5YTNrj8EX0DNuS8qBvadoLeJlumRxNdV+J0OJ0+9t6doyNzZrdk0+T+fQWZw6EtaKUmBqxwSjkVD4ADpAVeuCl9ou63nRm3B6C+5svB8dugbQFa1DQDLcHSm19y0FhaXZtWl0+OdEB4I5dMBqN+2Ls+hAN4Cp4jVVvh50yFs67sRKW7+5nyKmktesOWf2n7LV8Omqo6XLAmhd26QiIewNfc+AflwFgLNfui/pS/UVqka2Vj19waeyY4GzSnOGc+hzkEGf0/zkVtCMV+sWmL49cmlmccYcc0bBtk4L/bMszTYWQgghhBBCCCGEEEKc3v56ltspYMawFvgquvvipcDDwFuAOcBngE8d0+pmZwD9nuScSdfPAXqn2kEplWHCe8vWLMM0cXqy2yz+5aaV/OV9z57qpRzmUz96js+96RwaAx56R9NTzq21gIaAh0sWV5/s5YkzwOR5l+JVyFRdlmbPEu6OqHB3JBbujmT/6/f/wVu2PlyszsZjjFexWuiK2l6ASChcbgLfEgv9hF363oYOZR3o6tkcOiC0meMNmn99Zp9SCJkurWmC0jqzHFoxmgLUhMrbE8pU/pa+t096/DPt54yEwg7zvXvptp/UeVP9ZR0tXZZ5LInSzN5IKGzN63nMWzWy1Vuwueze9IBz4a5fq6aHWzMdLV1WYHR7TX3v+mUvLXm7Z6rW0ia4TpRm/U4l2t6cBuLR9uZ0tL15xnm3q76zyioPt9nRP8PUTNsKIYQQQgghhBBCCCFOD0qpxbP8OmsWh2sC/kYp9TT6fdxdSql7gL9Dtx0+UY8hCzwDvLZ0nWVZNnP58RN1v+L08sZz5/LRaxaf6mUc5sBYhtb7NvDm8xqB8RamJaXLd9y4ArtNPjQgDieVtQKAyYFeaR5sJfgrSbqB4XB3JGNCylLL5DJ0wKqAlKmILVXPFs3tefRs21IQ6EeHfQV09W2pGvRgG2azloPrmXCfoANeF5AuhZoT9neZtUwbTh4P5tz4IqFwxqzVa9Y+44xc8zg86MecB5QrG8sWbY5c69omBQfbU2O+V6ZiuGAvZrPnRL5TamtsAa6cs8wfGNnq96YG/R0tXTnTGvkQU1TbHuYo5tS6AHd5uC218f0bpapWCCGEEEIIIYQQQohXHz/QZ74fBuqALcBGYPUJvu//AL5jWdbTwHrgk2Y93zrB9ytOA4WiYv3OIVbMDfDxpqX81++3oaYqYT1BLItp7680s/EXz/fQ8Z4L+OfOCD2j47NpGwIe7rhxBdevbDwpaxVnHglrxXRKFbFJdKvhUpioIqFwEj3QPcN4yDoP3cq41HJ3AB1I+s3lvPkKosPUXYzPL7CAsckLMBWoytyHm/FKXDvgj4TCxVLb43B3JBcJhWG8dfKJpBhvxQy6AviIH4cx564035dwdyQbhuy15vaOli6rFNpOYEO3iR4yYXku3B1RXS1dVtI350BFbE/vQM1KB+DpaOlSrWubXvbjN6G7E4iXfu5GKYg/o1pNCyGEEEIIIYQQQgghxlmWNR94M7AQXaBxkFLqb46w+0vA2UAUeB74qGVZUaAF6Dnea520th+ambmfBxqA54DrlVIHTuT9ilNv3aYe7nxg8yEB6Ml2pGBYAT2jaar8bh79dBPrdw7RF0tTX65bH0tFrZiJpU7mRw/EGSUSCluTwrrS9Q50RW1pFu2YuZwzl9PmqxoYQYer1eiK2oA5TNRsCzqALQdG0UGgyxyrnvE2yU50yFkw35eb7+NmHy96busJrao93krnuKOly44+h5mmh1tzgGVm8drRFcqJyfN4O1q6vOhzW/oZ2YDUdIGtOZYDyHat6QDwLt32E/bOW+NMe2tyZv9M08OtTvQ5jt1w812Y41tHUYErhBAnmrx4EUKIM5O8OyHkOVwIIc5M8hz+CmJZ1muBXwA7gBCwCV1gYwEblFJNR9j/FsChlPq2ZVkXAuvQ7/1mgVuVUj88gct/WSzLqgBGR0dHqaioONXLEbO0blMPt92z4Yx5Ifm1d53PTefPO9XLEC/PSX/ek8pacYhIKOxG/14kpwpq4eAs1LFIKOxlvCVxPBIK+82+KcaDXBs6uC0dq1RR60AHrk5zudIcawz9H4JCtxUuAGpiy+NIKJxFh7SlwLho9k9wglsgz8Z0IfcU25XaKWcvqFxGz5xLHb2Nl6fRrZIdkVA4YebYjk5zCCe6xXTcfE1sK32YtLvSmfA3em3FfB6gYizqceViztrB5wt75zcdnCPctaYjD6S/VJlS5vhOIBf62q3pYq6qmB14nQ9IR9ubM9PdlxBCCCGEEEIIIYQQ4rT0BeAupdQdlmXFgLeh2xrfiw5eZ2Tm05a+f8ayrEXo0He3UmrgBK1ZvEpl80X+4Wcbz5igFqC+3HOqlyDOQLZTvQBxWpryUwORUNgyc1dLFDokLP2tzJh9HejgdL/5N4+eX+BEV8AW0cHiiPnymGOkzDFSQC7cHUmYy/5IKOybcL8uc0wLXalbhw5uC0wjEgp7TXvfE8oE2P5J52k6Cr3mYtXIVseKl+4pzZ0ttYw+5Dmoo6XL29HS5Z5wVQJ9/uKta5vyrWubkq1rm6YNa59e/bdqy5K3q2fP/6S9dW1TYawiONJw4Knh5dt+MtT0cGvm6kdvV3P2PMSe9M75W7LROdH25tL83yS2dMGR9lbaPPsO/hyW3/XX8qlKIYQQQgghhBBCCCHOLGHgu+b7POBVSsWBzwGfPtLOlmV9zrKsg+8RKqWSSqkNQMKyrM+diAWLV6d1m3q47Au/YyiRO9VLAfTM2jnl7mlLLi2gMaBbHgtxtKSyVhzCVLBOVzHpQ//NiUdCYZvZLmeqP0ucgGdC0AqAmXNbhQ5rSwFuAR22OtFVtHnGP0BQEQmFk+HuSDoSCisgbwJQ/4RtiuZ7ha7ITZhANj+5ZfCE7U60Avoc2c1aUpPOz0Hh7kixo6UrBdibHm5NTrg+h24DPZmdCUF669qmUhvoWcm6K12m8XTO7F+ESNac1xpnPlVdceDR4XS5z9/n9lnfuPYd/r/wVnr/v3NvGls9sKHONuSqzjoWbfvFfzenlrZ/ptbmPlC56jurdm98/8ZTXs0shBBCCCGEEEIIIYSYlQTjc2p7gCXAi+Zy7Sz2vwNYCyQnXe8zt33+OKxRvMqdjq2PlYL3XLqIrz605WBr0JLSm/Z33LhCZtOKYyKVteJo5Dk0NPVODCJNQDrAhLa9JtTFbDcAjIW7I0V0kFtqkxxHP7mnTFhcMNe7IqFwFeMvHirQ4a4bXY1bOs4outVyadbqYR9CCHdHEuHuSDISCjvMzN1ZiYTCrqPZPtwdyYa7I0n03+cZ/yp3tHS50AG2r2tNh31i6+RIKOw01cATj5FoXds0+UXQwWN968/vK3vyotfPtNaMOcYhz3HmfuNAz4J4f/aswcjAnrLKkX5PIOgo5n3eXNqzv9pVXpcaq/3MH9d53/6Bu1yA07LHigDBtk5/sK2zbKbHKoQQQgghhBBCCCGEOC08AVxlvv8V8GXLsv4R+H/mtiOZnFOVnAcMHZcVile1QlFx5wObT6ugtiRY6+PuW1bTEDi0iWdDwMPdt6zm+pWNp2hl4kwnlbVi1ibNjc0xxXzUSdu4AHckFE6FuyN5E2KWlILa0hP4wXmr4e5IPhIKD5nbPYwHn0V0VajdbJ9EB8hZxlspJ8LdkWIp5JxidqyX8fbNMzLHcKPD48mVujOaoTp2shx6/ZOrb0tzfSe++LF1tHR5gEzr2qbJ67EqxnaVjVQu8//qDZ8aeOOvv3zI8Tpaukrn8LA1mdm5pRnBzqfmrHB58hk/Fra58YHMtX27bAmVV8sHYpltDY6zKj1/GsiPvWfY4d9xYOP7NxaDbZ3OWTxOIYQQQgghhBBCCCHEqfc3QKnw4g7z/TuBrea2KVmWNYx+D1EBWyzLmvi+q90cZ+2JWLB4dVm/c4ie0fQJObbLbvGXa5awfE45n/9lhN6xo7uf+nIPly+p4boVDazfOURfLE19uW59LBW14uWQsFYck3B3ZDZ/xUoh51RzVDOMh7GlsC9hAlIHUI4OFhPowLTBHC8B1Jjv3eb4yXB3JDbp+F7AFgmFE6XA1hy7YI6LaVPsQAe8h31QJ9wdUaZ9s4KDgacHyE0Rlh611rVNWXRQexjT/jkzaV0W+pwd9le/dW1T5v/e+szIaGCJr7/u/MOOVzmy1RX3z3Xlnf7SHNqJXIzPAXY2Rx/P/8+5b+nZWz5n5MrB/Y752dycEcuTq0nmdh1osOzbGhyuOueDvu92PJuNdITTUd3yWgghhBBCCCGEEEIIcZpTSu2Y8H0CaJnlrp9Evy/5/9Ah7+iE27JAVCn1+HFapngV64udmKAWIFtQfPV326j0Ofm3m1extS/GVx7aesT9LHT1bGkerd1mcfmSmhO2TvHqI2GtOGFM6+PUNLcVI6FwhvGgthRC+tCfwsqiw0MPOkgsM1/70FWxKbO913w/OYDMA/ZJYacN/SmvUoVu6ZNgR3oMREJh3/lVZ/PceR8v7T9jWNvR0uUFCiaQPSaTA2QTEE8OpUvtlB1Xbv1R0Z2LZ/T6IqVw2QsUmp77qnOoKmRd+fjPplpPGjPnNxIK284b2F784/98KBcJhVXSU+eqs1XtS1Wflb74uXtUYshd8VyNpzztHyqF5d5vNL0j85GuH0lgK4QQQgghhBBCCCHEK5RS6jsAlmXtBB5TSr3sYhYhplJf7jnyRi/TSDLHX963gbW3rGbtLau584HNM1bzKuCNK3U1rVTRihPBUup07PwtXm0iobBlKlkr0bNpY+g2x/PjvkaXrZD1+DKDFtCHnns7ZiplnZj2vmYWbqmC1oUOZB2AVWrPbFr+FqeqpD3C+vxAsWtNR6Z1bdNUlcIHmZDUjw5rpwyrj8S0Oy7OFPZGQmEnYOta02E5c3HHFY9/xmMv5mzAQLg7UpywjnzTw60KUOHuyKzD40go7EOfv7gJ191ANZAJd0eGIqGw44mGFf6fLF3j3FR7VhzIR9ub5UWaEOJkkBcvQghxZpJ3NIQ8hwshxJlJnsPPcJZlDQHLlVIDE9oZT0kpVX3yVnZyWZZVAYyOjo5SUVFxqpcjplEoKq76Yhe9o+kT/uKxMeDh0U83Abr98kObe/nZc/sYSoxPE7RZUFSH7nPHjStkPu0r20l/3pOwVpxyJhQstUz2oatfk+igMBjzNrizzvJ4zdjWIXQAm0RXyebQ7ZL96MraUVMd6kBXlGYZD2uPOKP2eOpo6bJa1zaV2icfbLdcuu5I+6IfU7F1bVPSXOc0lw/Oov3mtX8W2FtW5/nWOc39fzviVU0PtzrQgeyRqn4twNG6tikH8I83f8L+/OIbKs7N2plXsI82PdxqQ5+/BGCbeLzJYXewrXPiXGEHEI+2N0+evyuEEMebvHgRQogzk7zRK+Q5XAghzkzyHH6Gsyzr/cAPlFIZy7JuZeaw9jsnbWEnmYS1Z451m3q47Z4NwIl/Afn9j1x2SEvjQlGxfucQv93cy/97LHrY9qU/iHffsloC21euk/68ZzvZdyjERKYK1oYOBQvolryjpgLUBWT8qd541di2ODrAdKGrad2Mz1lV5nsPgAkX0+iwNmm+TqiOli57R0uXvXR5Uih7xHbLE5l9E5gW0n+4+j22wMg2P/oxH/SzJVenfxW8PFXaJ9wdyR0pqDU8gK+jpcsRCYXdV+97vqIs2ecatanS34MAUIUOX4uRUNhV2jHcHSlMrEqOtjcX0W2hFTpwlxfvQgghhBBCCCGEEEKcRpRS31FKZcz33zaXp/w61WsVAuD6lY3cfctqGgJHbolcekP6r1+3jNcsrz3q+5o8I9dus7hkcTUPbuqdcvvSm+N3PrCZQlE+iyiOD5lZK04pE/zFJ1zOTbg5AWADO6gYugWvDR3CFhkPbi1zjOQ0x3lZfzEntFXOztA+2WfWdNjs1ta1TRkgM9N9/N9b77LV9T9XMaf/mWS4O5KdGPbW9z/rzhQS1ver3mL7Ulun/WubnrDvnX9t/pf/+/FMJBSeaU3TyTIerrIodiB77dbO0Y92/dC0kWYESIe7I1lT9eyMhMKF0vzeKbjQPwOFrsg9bK6uEEIIIYQQQgghhBDi1LMsqwA0KqX6Jl1fA/QppexT7ynEyXX9ykaKRfjMzzcxlBif7je5LXHDhLbEj28f5A9bBo7qfqaakbt+59ARZ9j2jKZZv3PokKpcIY6VhLXitBXujuRMS2MH48FsGl1hakf/TUyiq2vjE0PLSChcauWbLQW3JnT1o+fbzhieTlK6fxUJhYvm/tOTwss0LyMUTnlr3Ql/g6fXujgb1mHqRNlN5VW8GKhzfKT7Ybczlw8s3PWbRCTUmkUHqckZglTg0LbMppWy2T6SB/Lh8e0cX7r5rmJFJmHdddE19Qv040pNd/xgW6cdHVArdJAuHyUSQgghhBBCCCGEEOL0NV1nPDeHvy8pxCmzblMPrfdtOOwN59Jkzw9eGeS6FQ1csrgau03/Wl+yuJpqv+uQcHcmjQEPlyw+fEzz5Grb6cx2OyGORMJacbobMf+WWu1m0S16LXSbYHu4OzIGBwNan9kmjw4PJ7/4sKNDV4vx6txkuDtSnG4BJjTGHNMx1XFL819NK2TVurZp2uNNJVa+MBMrXzhYOs6k+y8QChcuPtCtatOj2b1zr7HXjm4C3c64VGU8rY6WLhfg6WjpSpRm3kZCYT+TZvma2bqea1PO9CaVdeRsDh96Pu3YVMcNtnVaQBlQiLY3x4+0DiGEEEIIIYQQQgghxKlhWdbHzbcK+LBlWfEJN9uBa4DuWRzHD7QBrwXqmTRqUSl11nFZsHhVKxQVdz6wecrKIIV+c/7BTb38Y/OKg0Et6BbGN58/d8pZs1O548ZD9y+Zqtp2KrPdTogjkbBWnNYmzWBNmEA2i64MtXNoQOgDytHVoPFwd2RyO16n2d7GeOBaCncP+6iNuS83E6pzgZz5mridA/AMVy5Lcf4nfWZt087JDbZ1+gAr2t58sGWyCXdnCjsdtelRG5C5ruvrMTNH1hbujszmoztFsyY16TprwvqLrOnIAZ6LMg7vHPucsbPGekZnOmi0vVkF2zrTR1i3EEIIIYQQQgghhBDi1Ptr868FtHCw8x6g3xuNmuuP5H+B1wDfA3qQTnviBHg5bYivW9Ewq7D2r1+3nOtXNk552yWLq2kMeOgdTU/5C26h2y9PVZUrxLGQsFacUUwF7AgcrA6dGBR60OFrGvBHQuHYpPa9EwNXh9k2t3PRDc6uli5769qm1KS7swOV6Hm4R5rDalWNbLWC0V8VB2tWHhL8RkJha9Jc2Yntmr2AmkXoWjTrscz2U/ZxmFzZ29HSVWYuHwyGI6GwE93GWUVCYVvaXVU24KsvfKkyVXhNypG9JON0LCjY1Wxm4Ubbm4+mnbQQQgghhBBCCCGEEOIUUEotBrAs6/fAW5VSw8d4qBuAZqXUY8dtcUJM8nLaEF+4qIpqv5OhxGFNLA9qqHDzsaal095ut1ncceMKbrtng35DfsJtpTrc6apyhTgWEtaKM1mOQ9ts+NBzanPoKtraSCg8iP6UmNtclzIhZC4SCseB4s7Fb/I4cgn7kxe93nHp07+ZWMmbB0aZVEk7man+jUVCYdeCvb+3FW1OV0dLV6Hp4VY7+r8x573v+np2pHKZA0hG1zZPDIUnVwdPZ6rq2EN0tHSVKoUnVvYWOTQcdqLPUSYSCucA547FNzp6yxttgO0P3nzuO197w5Rtj4UQQgghhBBCCCGEEGc2pdS1L/MQw8DQ8ViLOHMUior1O4foi6WpL/ccMif2RDjWNsTrNvVw5wObpw1qSyv+pzefc8T1X7+ykbtvWc2dD2w+pMq3IeDhjhtXTFuVK8SxkLBWnMkcgM0EkBawFx3I5tCB5LyCZStEF71R1QxudA75duUfusCW+sr4/k6A1rVNqefOvdTrzo7VRELhsXB3JAVgKk/zHNoSZCa5vfNeY9+98DpH5fAWF3oeLkAh7ammPLbLXjuwyQZNB483cWYs6NbLk+fnmnbMuemqaUta1zapjpauUovo0nWT2zHngQz6HLkAd+XIltygvy6LPp9H3dJ4QrvozEyzf4UQQgghhBBCCCGEEKeWZVl24FamnznbdIRDfBb4vGVZ71dKTTsKTrxylALQiYFltd/Fv9y0kjeee/SB5eTg98JFVTyza/iQIPhIbYgBKn1OikVFoai3+O+ubXzloS0z3vfRBq3Xr2zkuhUNJzWoFq9OEtaKM1mpQtWHnt+ajYTCAFXotsXDCluuanDzHEd2xNe9nIEDlXgjoXDChIr20oHc2bEcYBusClXe9867ncNVZ8dMZawbHQRP2e43Egq70f8dJcPdETV20etzvXMuKYxULc+hK1oL4e5IgVDYwoSj8OUpX8SYObTuSCicLs3INfNka9CtmBNwsIKW1rVNhz1Pta5tmrEtsakqzphjZwA1t/cJj29oo20NN3kWxvrc0Dy7HhPjHPpxUWSa8ySEEEIIIYQQQgghhDgtfA0d1nYCmzj6mbOfApYAByzLijKpK6FSavXLX6I4Xazb1MNt92w47JdkKJHlL+/bwEf3Lubv37jiqI43Ofi1WVCccAeNJlB983mNfP2PO6c91kgyx3u/+SSVPidKKUZT+Wm3Bajxu/jD316Ly2GbcbvJ7DbrsLm4QhxvEtaKM1ZppmokFJ7YVtiGDmHnAA6HygeqYzucQHrxAcbO2V20oatuE5hWwZFQ2Lsj+KaKos2RCozt8ObtHmWOn4+Ewml0NeqUxsoW2LzpQZszn3RFQuFCBXgvX39nVq9tvBLWVOlmMJWrJnB1tK5tmvhipmDua3Ilb2HSdT50gBznCCKhsMfc/2EBbCm4jYTCxcpsovCxzb+z/KkDtilm7M7IhOSqFDALIYQQQgghhBBCCCFOW+8C3qGU+tUx7n//cVyLmIWT3YJ44v3e+cDmGdP8r/9xJ+fNr+SN58494vGmC36Lk67oHU3Tcs+GWa9zJDm7t6UHE1me2TUswas4LUlYK854E1vvhrsjKdO6uBz9++1Ah5rF8N6DVZ9WJBQuB9Lh7kguEgoXCzaH21bMuesGXojVDbyQMuFqqRrWb46ZYLytvZV1lnki533cERjbaYW2fN8FpIEsM8+4La3VBbgeuvZDjnk9f8qGuyO6Ane8Wrj0ePKRUHhwUniam7COw5i20MrM0nVyhE/HhbsjuY6WLidg96b6Ux/8zjuP9tN0SFArhBBCCCGEEEIIIcQZIQtsO9adlVJ3Hse1iCOYqhK18STNTF2/c+iQ+53OZ36+iTesbJwxQJ5N8Fty1G9OH4W+2PSP51SF4kKAhLXilcmOrrC1AA863EwAfnO5VKmaBQh3RzJPvO/ewfL4HjcwUgpGTWAL4wGr1xzXBiRduXhFY++T6f2Nl8fN9tMGlpFQuNRS2RkJhYuLF70xO1gdtqqGX3L2zrnYEYaYqbZ1tq5typqWyLZwdyQ9Mag121h1B57y/vxND6ubfvn5rAmVra41HQBWk14nkVC41PJ4xiA1EgpbrOkoujIjjqrhbruZQauOprpWCCGEEEIIIYQQQghxRvgy8AnLsj6mlJL3/05j01Wi9o6mue2eDdx9y+oTGtjOFGxONJTIsX7n0IwVq7MNfk+0+nLPlNefylBcCJg0PFyIV4hS5akLHcj2mX8z6N/5gLndAXou7GVP/YvNm+ofnhhQBts6rRtuvisd7o6MoStUA+iKXZ85Dgv3/o6kv5GuNR2ejpYue0dLl6s0U3YiUzWbRn8wqHLxrl85Lnr2y5ndC15b2LL0z+ho6XKY9brN9w7AYYLYg8pju+2Vg5vnelODiyqHX6owV7sBvy/R4wV8Y2ULkpgWz+Y4M1XhWkDFpU9+3lvf/5yjcmRbJVCxr+HKud/68/vKZnGuhRBCCCGEEEIIIYQQZ46rgPcC2y3LesCyrJ9O/JpqB8uyhizLqjXfD5vLU36dzAfySjZTJWrpujsf2Exhcg/h42i6YHMqRwp2Zxv8nkiNAV0tO1kpFJ8cJpdC8XWbek7WEsWrmIS14hXHBKMDwAFgPzqsbEi5qgLo0LZUKWuPhMI16BC2vr5vw+LHLn9LxYRDuQFfsK3TDowBo+Y6D7p6d2DPvDWJmoGNZXV9TzuXv/T9gCc16GGainVTeZtBt2XOA2Xz9z9K3unPMF7pm/lSZcrxzhv+ybon9PoEumVzmWltzMpN37BXj2zJlsd299YPvjBqDp0HcvZiLgNkL336N3nTAjlu1uIzlb0AREJhWyQUdk9YWqFo2Qvz93alavufTwCWNzVQ1dDzhP+oTrwQQgghhBBCCCGEEOJ0NwL8DPgD+j3U0UlfU/lrIGa+/6S5PN2XOA6OVImqgJ7RNOt3nrh8/JLF1VT7XbPa9kjB7tEEvyfKHTeuOKyt8ekQigsB0gZZvEKZCtksQCQUzg5WhQb3N15Ffd/TFd7MULk32Zd3FtKl0DVZsDljOWfZPJRym2BzbNWVLbmbdjzqu7g34kdX69rMMVPo/3ayGXelvzy+11M18tJY+dgunysXY9PKv8hPXIsJWgvmogp3R2Lm+nxZYn++dW3TxDbF2Sc+8evKvFXpvG7XU3l0sOww++e8maFccM9v+zEtniOhcMEEs/mw3j9vjm0D6F7+LigqV2jbD22RUDgb7o6kMfNyzTbO4cpljr1zX2NbuOe3Pl96MAMMV4++1FM9+tLY8fp5CCGEEEIIIYQQQgghTj2l1AeOYZ/vTPW9OHFmW4l6IitW7TaLf7lpJX9534YZt5uuYnWiSxZX0xjw0DuaPqEzaafz169bPmU746MJxWdq8yzEyyVhrTitRUJhB3p2a/ZYjxHujhQ7WrpGAVflyEvFvMNXVNAXiO3OA1WA117MjWFZO8oS+zxAJVD898fWlipwfeiA044OQ2PoENW1aPevSXnqEhXx3UkgVz/wvKPp4VZvJETabF9EV+Lm0S2Us+hPrxHujqQmrjPY1mkBzltz7mwwdiA3JzXsQoe0pWOkwt2RYiQULmScFYFY+QJ7rHxRPDz1w/YD7J97da5yqFvtWvA6l62QSZltMy+s/Kg7MLqjbNGe3xYyzgqV9lTlcw5/3+5517q2LXu71bq2afBYz7cQQgghhBBCCCGEEOL0ZVmWA1gDLAHuU0rFLMuaC4wppeKndHECmH0l6omuWH3juY18dO9ivv7HnVPebjF1xepkdpvFHTeu4LZ7NmDBjIHtkW4/Wg0Vbj7WtHTK206HUFwIAEtmiIvTWSQU9qFDz/jEebLH4bgWeg6t11xVbi7HAV/GFSjYijnLmU+m0BW1TiCbdZRZw5XLUnMGns0BNea2PvO9hW4VYqHn01ajq3BT6OeXOeg2yIPo8DVnqmIBuOf6j9Vuqqhx/uCsS4ai7c2ZSChsf27VX9pXRL7tdOWTDmAs3B1RkVDYtr/hsvL9jVdaY4GzEk0Ptyp0xW7BVMuWvqxwdyTz+KVvqkz45lTbivmx7WfdnMi6AwrwO7Jx+4XPfnlo29K3B+r6ny2b2/t4T9eaDvdee8H+/fJsDkhF25tLgTUAwbZON2CPtjcnEUKIU0devAghxJlp5ndwxKuBPIcLIcSZSZ7DX0Esy1oErAMWoke+LVdK7bAs62uAWynVckoXeAJZllUBjI6OjlJRUXHE7U+lQlFx1Re7pq1EtYCGgIdHP910xKD0ePjVC/v5zM83MZQYbxLZGPBwx40rpqxYnc66TT3c+cDmQ6pZbRZM7DJcOi7AP/1iM71jLy8oXXvL6mnX+Pj2Qd79jSeOeIzvf+Qyqax9dTnpz3tSWStOd2l06Hhc/6c+6/D7k966kE0VRirie/ai/1vIAdm83ZMfrjq72lbMFev7n42jQ9c8kNg/90qXsmzlI9lg0pfsd6XdlemKxL682deHfvMhgQ6Yfeiq2FJVbQId1lrm/lQkpGtdu9Z0FOY5a6xgjvwXH/mfPDTTtaYDwP3iig8VLnjhv5KlcxDujhTDMNrR0uVoeri1YI5dMMf3Mh5uFztaupwLa1Z5ysd2js4ZeD69demfOc2akvUDz1bYi9nyRbt/XfCkBkuhcaJ+58/K3ur1l/902ZoM4/N9SxyAM9jWmYq2N8sbLUIIIYQQQgghhBBCnFm+BjwNnIcuKin5GfCNU7IicZiZKlFLKdJsKlqPlzeeO5c3rGxk/c4h+mJp6st16+Ojvf/rVzZy3YqGQ45z4aIqntk1POVxr1vRwH93beUrD209pnXfsLKB61Y0THv7kdozl0LxI7V5FuLlkspa8aq08ZwLKlOe2rOxGCiP7+0D6tFh5wGA0YrFjY5cwuVP9e1D/00uB8ZGKxbbM64KX/3A86k9c19Tnnd47fP2PxJz5RNJdKtkK+mpLfQ2XFZRNhZ1V49sSTiKWYc5xnCpktbMsS03+8TD3ZF4R0uX/dL1/+zxJ3vdwIgJW111fRvyqzZ/0x7ujuQmPw5zLM/+OZeW7170+tRl6/85A1QAiXB3JP2zN3/e4cgl5uSc/kLP3KsGMLNzmx5utZLuGrsvM+hEh7vOcHdkCOD5VZf4iirv8ORTsckhuWnVbE2uuBVCiJNMXrwIIcSZSapyhDyHCyHEmUmew19BLMsaBK5QSr1kWVYMOM9U1gaBzUop36ld4YlzJlXWlkxViXosFa1nunWbemj7yUZGUlO+RT6jI52vdZt6uO0ePZd3qlD87hkqc8Ur1kl/3pOwVrwqRUJhT8pdVZV2V1WUx/f1OYqZ8jH/PGd36H37L3mm3QHUoduA9ABJ9AzYMbO7CndHig9f/R6nJzNSFhjbWYZun5wByjLO8syOYLPXlYt5bcXCgcW71xXC3ZHMhPu2meOVZtmmS/NrI6Gwx9yWCHdH0uY6NzrUTXet6XAAVuvapuSE47n2NV5VNVCzMnPeprWjZv9sac5v5/W3u5Vlq/Kl+uOR0PsSAE0Pt5ahK5bHIqGwy3yfAeho6Qo4coniR795Y+y4n3ghhDg+5MWLEEKcmeSNXiHP4UIIcWaS5/BXEMuyhoErlVKbJ4W1VwE/UUrNmWY/O3AOsFUplZp0mw9YCmxSSp22RR5nYlgLuiXyy61ofSV4bOsA7/3mk8e8/1+/bhkfa1o25bmTUFxMImGtECfLT27+gi8wuq2+dnBT0Z0dO/DkRf9gtxez9os23GUHaoER9Axav9klhm5tnEe3B7aA3P76S6oti0BgbOcBX6o/B+Tzdrcv5akJuLKxfnculgt3RwoAwbZO26U9L5bf9sL9VXNSw2l0AGxHV/UqoAw9b9YFxMLdkfSfLnuzu2pkqwOwPb/qNgZrVhbRc3Ad6EpZz9kv3Zd53e//9+D8W4Afve3L/rr+5z0NfU+NAhV5m4s/XvOVApA37ZNBB8yOrjUdCh1O58xxi61rm9KrvrPKCeQ3vn+jMlW15UA22t4sE9WFEKeSvHgRQogz06vvHSUxmTyHCyHEmUmew19BLMv6ITCqlPoLE9aeC/QDPwd2K6U+MM1+twIfAy5VShUm3eYAngC+qpS650Su/+U4U8NaoR1pju9sNFR4+Kc3Tx3ASiguJpCZtUKcLG+7/++Tz626pM+diwWA8kuf/rccOjS1AX1A1mxajp6di7kuj66KtcLdkfTDH/5VftGe3/orR7Z5u9Z0xAErGO2MnRX9VTpvdzv3zrumYkvzZ2zOfKLABa8berFmcTFns5fm1mZAz6EFiITCOfQfAg9g+81rb3PlKs+uTrurM3sr1vu7a78+/G93v5joaOkqVdvmATsop6nATZpZt2XVNlcgVr6guG/uVf5l23+Sq4jtxpmNFzfV/ci99u8deSD2oy/kXUDV2S/dV9x21luc9kK68MC5P4vtrHnetfY7KPQM3LT5ssy5sZ2gH4kQQgghhBBCCCGEEOLEuh1YZ1nWZvR7kPcBy4AB4N0z7Pch4K7JQS2AUipvWda/o8Pc0zasFWe2ocEBqp/7Lk89+CCF5DA2TxmuusUErnw3nvkrZnWM3rE0t92zYcrWxnabxeVLagD4wQ9+wBVL381NN93E/ffff7wfihCHkbBWvCp1tHS5AOdV+pPdcXQIW5e1ubx2VbTsKq8AJ7rSNG3+9Zt/LbN9PhIKWwuDzcm6gY0DQ9VhT/XQ5sqUp7ZoL+RcQCzjrioOV56drxyKlHvTQ4UH77+9iA5o9wPFUsWtaY3MhNbHKmf3OssS+zK99RcnQGW2N1jOvsqDQWkWXf2ae/SKtxVqhjbb0BW6FqDm7fm9q67/WSvpbxx7ZPEa109r/6IY3vOH+C1/+rQ7d/4cV85xvjPlbYj11e7L+xL7y+v6nkmMlS1yuLPD1W+Mbyx+6zqVSXqsHDqYzUZCYSuqA+WRE/uTEUIIIYQQQgghhBBCnChKqT2WZZ0HvBM4D93p75vAvZPbG09yNrp6djpPAeHjtlAhJnnb295GJpsl+LbbSXvqKCSGSe96nmLq6Kf53fnAZq5b0TBl5Ww0GuX222/n6quvPh7LFmJWpA2yeFUylanOlS/+bzLun+vcP/cq++WPf9Yf8zbUO1TWV5bqG0UHs7vNLr683e3cftbNmfl7H7ZsqmB7/LI7Mys3fcNRtDl9tQPPF2LlC105Z1lhKLDMY0H27O3/lwdUxlmedOdiE4NeB7rFcRYgEgo7CpajYagqVNh47m39rWub8j9+S7vPkUuVOfPxwbOiv/IChXf8vSP9oy/kHUA+3B1RZl8nuvo1A2S71nRYgK3p4VYnumXzyH+d9zb/5uqgLRpoHHvw/tvZsvTtgZ6Gy1TopfvSWVe5L+mpc9cOvZiKl83LetJDvjn9GzJAfMJ92IAK9Bzcg7NyhRDiFJIXL0IIcWaSHmJCnsOFEOLMJM/hrxCWZTmBbuBNSqnIUe6bAC5XSr0wze3nAo8rpfxT3X46eDW1QT5RLX1PVavgkZERqqqq6Lj35/z7C/Ypt8mP9TH026+T3vU8WBbexRdSfd1HsfuryA3uoefbn6Tmhr/Cv2INAB+e28u/fvpjPPPMM6xYoStzC4UC11xzDR/84Ad55JFHGBkZkcraVydpgyzEydC6tikDZKCJjpYuAPv2s24aqh7uztsyIx50lap7b8MVgS2h98abHm5NZ1wBvz2fckbC7xt15hIW4No39yrLkxlWZfG9mazDV+7KjFY2HngiZmFl0W1EKt25WA+Q1PeHDz3vNmeCVjeQzTs89ow7YEf/N5nPOcrylSNb7XX9z3vRFb2xH30hbzfHLFX7Yo7lMtd5vckDtpRvjq1rTUe86eHWDMAHNncy5K7IveHpPxRpb6arpWtk8c5fuub0bwgULIdjpHJZrmY4oq56/KdJs87JlLmf4lTn0syyJdreLG+8CCGEEEIIIYQQQghxmlJK5SzL8hzj7luBK4Apw1rgKrONOMXWberhzgc20zOaPnhdY8DDHTdOPav1VB93NsrKyigrK+PBzl+gGt+E5XAecrtSRfp+8i/YXB7mvKcdigWGfns3/T//Ig3vacdZs4Cqaz/I4G/uxj1/BWDji3d/ii9+8YsHg1qAz3/+89TX1/OhD32IRx555IQ+JiEmkrBWvOq1rm3KdLR05W7q/JciMBwJhV2Aq4DdlfFU2Zdv+WEFkPCn+jINB9Zb9f3P5p++qK0A2C944b8ZDixNlSV7HIOVy1XWXWkLjO3IVcT31AHZHJa/YPcu9hSS29ABcAEd2pYB9rjX535yWar42hfie7yZYQc6dOX8jR0Fs60f3YrYgQ5lLfRcXQD2NV5p2fMpf+XojnJPdmSoZiiS2uubQ+vapiLoD8dFQuFkWS5dBOho6bIDZb7kAUd/zaqyuH9uevHuX48xTRALYCpsx2Y4hQGzxqGjPPVCCCGEEEIIIYQQQoiTqwP4tGVZH1ZK5Y9iv/uAf7Es60+Tq2tNW+XPA/9+HNcpjsG6TT3cds+Gw9qZ9I5OP6v1VB53thwOB9/+9rf58w98iFTqe7jmLMG9YCX+8DW46heTjj5Prj/KvJZv4qioA6Cm+W/o+eZfkunZgrtxOeWrm0nteJqBB76MZXdw7rmr+au/+quD9/Hoo4/yzW9+k+eee+6EPQ4hpiNhrRBgws2D8oDbTiEd3P0bj72Ycx2ouyAfK1uQX7TnIZz5pLN1bVMeyEdClFeNbnMA+Yrk/gPbF785M1B7bvyCF/7bCdjzDr8t76rI29LK4Sqm7ObYnoLN4TtQe0HxpYV+3445/bakZ3PxxvWbnU0PtwKRbLg7UoiEwv3ocNaNDm4D5l8F0NHSZXH2e1iy7SeqYHd5gfxNv/x8doqHV0SHqUWzb2GsIpjJuCtzQ1Wh4uLdv/aa4+aO5pyZ4NflrSCfsp2adjjBts5ywBltb5agWAghhBBCCCGEEEKII7sYeC3wesuyNjKhMARAKfXWafb7CnAD8IxlWQ+h2ykDhIDXAY+ZbcQpUigq7nxg85RzJxT6jeaZZrWe7OPO9r5LbZfnnr+Gb657mk987Qdk9neT3vEMY0/+hJobPk4xk8ReUXcwqAVw1S7E5vaTG9yDu3E5ADU3fIJ93/goNpuNnzz2KyxLrzcWi/G+972Pb3zjG9TW1h7XxyDEbEhYK8Qk4e5IMRIKpwFlL+Z8QGbHWTdZ1YObvX2154/tXnhdsaulyw+kmiCLDkJzz57/ySL6xU0RGARCrnxizFbIp1wq7RqqXO5x5hKO8sS+vVlneX7vgtfaYtVl6ZzzJ5kb1ys7kO8663W88IaPBVJ1l+YSazpSrWubimZmrAMYAebFvXOKHS1dBbNcK+/w7/en+soARyQUdoa7IzmASChsAVVmfdlIKJxo0sFvMtwdKWJaKa+77je+itiuQiQU9gLp0qzaWfACVR8b8yZa1zYdEpZ2tHR5AGfr2qajn+5+dOYCnmBb50i0vXna6uDTTbCt0wPko+3NR/PpRSGEEEIIIYQQQgghXq4R4CdHu5Npofx64K+B9wDXoHO6LcA/Al9VSh1VMYg4vtbvHDqkRfFkCugZTbN+5xCXL6k55cc9kqnaLlf7nXgXX4B38QVw5bsZfPA/GXn0Xioufssh+7599Xx+vGHvYcfM9e1E5dJYdjt9B3qZP28uANu3bycajXLjjTce3LZY1G83OxwOXnrpJZYsWXLcHpsQk0lYK8QUJgSeo0Dh8if/yZZ2Vrg9ubGa4erQYMpXX/qIkAV4k546R13/c2pfYcBdNrotATQALjuqxq7SjgK2XQrLU7RsDsDpzQwXLnjua0VnIVUAnOi2yEPdjRfXz4v3+edgDS7a+avAc6s+nXKbGbeAM+vw2Yt2J4GRbfaGA+udB+ovSu4KXl9YEn0gYY5RFgmF4+gKXgc6UM2gg1m7ua98R0uXDf08mmfJW1zz93TZq0a2Fsx+OYBv/fl9ZUXLaf/Qd/5sdJrTlES3aT7k41LBts7qSzyO8teknamOlq7EpKrl42034DjDglo78AFgGPjBKV6OEEIIIYQQQgghhHgVUUp94GXsm0O3OpZ2x6ehvtj0geqxbHeijzuT6douDyUO/TyAs2YBya1P4KxZQGGsn/xYP7UNc3nLBfNY6hzhtkwCZ81CAAqpGEPrvsp7P/oJ/IU4733ve9mwYQNer5dQKMTGjRsPOfZnPvMZYrEYX/va11iwYMFxe2xCTEXCWiFmUAptgWIkFB4G7Cs3/79C+Kd/mwaIhCgAMV+6X42QIlZMuGsK2Yq8zZlJuWszrsyozV1MWnaKoxWx3fuchRTAAsBlgtqRIY+j2OerIjTU76gZ2ZpYtffp8rpMwlu0e6rt+XRyxEP2sZXEHf5PjtSM2QcbDzxpv/C5rziA7LyexzJmfV500Frc13ilcuRTNeWxaNaXHjoAFErVspFQuHDDzXe53xYv+M7K23NAzJ0ZTvuTvQpQpcfb0dJlNYzuDObtHjvw/FTnpnVtU7GjpasPE9Z2tHR5AahEbXblx16TdsZOcFBLtL05dSKPfySRUNgBVIe7I32z3Sfa3lwItnXuQn+KUQghhBBCCCGEEEKIk8qyLAewBlgC3KeUilmWNRcYU0rFp9nng8C9SqnMVLeLU6++3HNctzvRx53OVG2XC6kx+u9vp+zc63DVBbFcXnK92xhd/xO8Sy/FEzwfZ12QgQfuovDaj/D2f95ErOvrrLrocr7+uffRF0vzn//QytzwUr79n18kk8lwwQUXcPvtt9PR0YHH42HlypWHrKOyshLgsOuFOBEkrBVilsLdkXgkFLbC3RHV0dJlta5tKoWbOYAw8B9vuDW7fGSvu7fuIptNFSyfw5XPFAvDZYn9HksVavbMvSbX0PuE3VnMOtCVsPH1S6r8Wyvnc2DXouQ7X/xxBab6FYgBqZgPRr3Ylu3/X4/fenOxYizqQFfFjqH/G3aiq2cLQHnlyLaGA3Xn24YDSzP7578m2bq26eDz2g0336V8ubRrgxrMXHag15vy1dsvfuaLWaAaGEK3daZ1bZP6/p+9OFa0ORyPXPH22tqhF2Ph7shhL8TMsUvH9wNWtL25v3R7sK3TuvBAd2XW5hjbWLe0CkhH25unfMF3hroQODcSCj8Y7o4c3ldjGtH25l+dwDUdN8G2Tj+wFNgdbW8ePtXrEUIIIYQQQgghhBAvj2VZi4B1wEL0yLTfot+H/LS53DLNrt8Afgn0mePsB65QSkVP8JLFLF2yuJrGgIfe0fSU82UtoCHg4ZLF1afFcaczVdtlm9OLe+5yYk/dT26kF4p5XIE6ys99AxWXvwPLsqh/22cY+u3XOXBfG1gW3sUXMnLZRxlNZRl94Xc8+ceHePbZZ3E4HDgcDu655x6uuuoq3vSmN3HDDTccl7ULcawkrBXiKIS7I+o3r73NFXSW1fzszY+NvOUXn00BdLR0OQD1N7/+dr6jpUsBPl98f7xqZIur4PB6Fux+KGArpPwJV1VurOKsfM1Idw8QKGCLD9edO5b2r/J5YhlSfdtUwe7eX5Y6MDhcuazeXsg67c7G1KU7Ew5vzl/nTz0+4k4PxdFVsIUnLnmjd7RisX+gZtXA6ue/5gDcnvRAxpfsy5TH95TN7X3cDU0H56JG25uLXauvLAayCWcicDYJfyPoYDiODnwpBdKL9vz2QM+cS52Vo9tc6Lm3dLR02QGX2dYLeICR1rVNBaaoFL0++kRlQ2Jw2Z7yOQl0mDxg7uswwbbOGiAXbW8ee9k/qJMnij4PA9NtEGzrdAO2U10FfIzs6BfpzlO9ECGEEEIIIYQQQghxXHwNeBo4DxiccP3P0IHsdKxJl8sx7xmK04PdZnHHjSu47Z4NWHBIsFr64d1x4wrstsk/ylNz3OlM1U7Zcjipes2t8JpbD173lXecR32Fh9Z7NzCSyuGoqKf+bZ89dD/gzgc28+in38ef//mfH3LbJZdcQjabnXYd3/72t1/GoxDi6EhYK8RRshVzRUvlc7ZirjDhai96tmzC/JtNls0t2Io5C1Qi5/T5cPoc5ckemzM7OvDU6r+NrXzxf1NFy1H/2o19NltxHRTzxUdCb8kHig5164//Rq277mN2eyFrV+ANJArerEulXlr2rnRgbEdd5cjWZBgyGWe5zZ5P+atGXso/deGnkys3fWPYmxkqNAw8a2fCf9/fet+9zqS/sQDY5ix6s9vRtyH1VMM5ydG6Cz3/vuoNZTudxZFoe3M+EgqXA85IKJyrAm/VyNY0MBzujqhgW6f9XXZXWdHCvyhvd6EDvBHMc3Lr2vFQuGR135bRjN3Zu75hhQNdtTsIEGzrdEbbmw8OGAi2dVpArdnmjAlrw92RA8CBI2z2TuCsYFvnGPrxPwHsBWqj7c27TvASX5Zoe/NYsK3zqWh781QfmhNCCCGEEEIIIYQQZ56r0RWxWcs6JFyLAvNOyYrEcXP9ykbuvmU1dz6w+ZDq1IaAhztuXMH1KxtPq+NOZbbtlBsCeirfSCo37TYK6BlNs37nEJcvqTkeyxPihJCwVoij9Lrf/28+EgrHAAWfL12dwnyoqHVtk4qEwuyde7W1Zfm7LHs+5c47/U53ajBZm9hsyzk8jvKxXTW99RcX6wZeyHnjvVlfZrAm4XR7e1UsZ0sOOyKhsLV9TUfPOS/+r1UxurPcnR0rs1HMjQXOysd8c1VgZFv5H656d8KX6isM5l2ZWPl8YuULqzat/HD8/Of/i10LX+/qq79w6EPf/rNM5/WfcpYVM0vCQ9vssfKFe7ec/a7c7+ZfkBuyqbr5BSvQkFfDO51Fgm2dtk8susS6ftf6YsxV7hz1zy0OzVtjDdSeyw1tnTag8qf+rLe+aCu6knBWrD91+VP/DGCDCKCrcrvWdLjR4fVoa9cPi8G2zgGgEhiItjerYFtnGVAVbOscjLY3JwHM9bvQQfes/EPrb5fGbCrwX//1+meOw4/1RNoG3IieVfwHYBhYBiwLtnXeM7Ft9OnoWILaYFunC7CfodXEQgghhBBCCCGEEK9kNnQ3tcnmo9shT2fiOLSpLovTxPUrG7luRQPrdw7RF0tTX65bFL/cytcTddzJjqbt8i9f2D+rY05VrSvE6UTCWiGOUiQUttCtgIuYebWlqtJIKFxq/eGo79tgbVn+LrzJPl/V8Jaks5CKAVl3Pu4tj++uTvoabIM15yTL4nvj5Zl+VZFL5Z6qnhe7fecfi93L322fu/+xcgvi3uxIFh3wJd7yi8/l/3jF27NjLl/DC40XeeJO/1gitdOylQftSyEdK1+Uyzu8RVd2rHrZlh/Me/D1j25/JFBuq09mXOU2m8udi1ev2rjWCqX37/1F+C1py7945Kp8ZeJzv/6k85eLL5+zuX6V5Z53syeZj83fZ88mar3BnU+4s37A48umsisHt+cuPdBdPjeV6F3Z/5IPaABykVC4ANQDRU9qML3TX1n+gD9b9aW2zuHQYFQtGdu/IFreEDCBrBOo4tA2K0Tbm4/qGXO/o7ggD5Wr/u5XL2789zfOuG8kFK4HgkAk3B2Z6UXncRdtb/5TsK3zfGAR8J/olsm1zPzi97QQbOtchF7rC9H25lywrbMBCABbo+3NMwXrlYAz2NbZc4TthBBCCCGEEEIIIcTJ9Rvgk8BfmMvKsqwy4E7gVzPsZwFbLMsq5WdlwLOWZR3y3o9S6vgMLp14x5YVBD4LNKHfj9wP3AP8q1Jq+j62r2J2m3VCKkmPx3ELRTVj4Hs0bZdnW4U72+2EOFUkrBXiKIW7IyoSCieY8DxhAlwL8JmrEq58AiAZGN3hSnuq3M5EKjVcuYyqka252qHI2GguSax8gass2esyx8o+9PWWfCT0tZqx/Y84hyvPtuz5dB7dGngk3B3JREJhp7tsviPjb3COeipsEa+3WEut7crBPe6AdcB7oOHSlDc95Kgc2erNOso9CW992cVDUWvY7c2nq0MJZ3o45Y8d8JbnCt73dq9Tuxa+3t9Xf2EBKISHdqWD8bFyt693Tq+/PrDP781XRH/mv0plUra5K62/2fDDWo/N54yXLSyWJ0aGHMVsEjjwbxe/b6w+k6m7bv/mGpfdpzpqbL0ZR9aesFG1YOyA5yuP/He6zxNI3BN+Q/HF2rOUeTwDmKA72NbpNI9/ERCLtjf3HelnEGzrLLd7iNghs+WLzemgrvxVM1SBFsz9TmxdXfq5XQqkw92R5474w59+PecC+Wh78+ZpNvn/AE+0vTlutl8AXA9cFGzr/BaQLd12MpjQdTXwSLS9eabQuICeaayCbZ0eYCn6PNqZuQp6FF1ZK0GtEEIIIYQQQgghxOnlU8CvLcvaDHiA+9Bd4AaAd8+w3wdOwtqmE0JXBH8U3cVuJXq+rh+4/RSuSxyldZt6Dmul3DhFK+XZtl0+mipcIU5nEtYKcQzC3ZHJIZQXHWBlABXujiiAMKiOFvz7FlzLOS9+05Py1lUC6aJlT+Wc5V5nLmEr2Fx2IBXujiTMsQoV8b1OVzY+6MmOFNEBcH7CbYnF2XjkQOXC2A/nnGNf4K0sfPy5f6rPuModw1XLi3mby+tNDaRtnsJIrCI4eMGO+x1ZV4D+2vOL3aH39TU93MojV37RXzX4oteWz1orXvxmYLB6hRpY8hbX4u2/sNXv6kw0+Br256vmj5038JJVnYkV3rzzsWHAMVoeKE/66mOe7LCF/hRb8pF55zkuSTsC68vC9jp75YGVBcv1R1feBuTtqjACFOvTo72/XXSJD3BF25tH0bN9CbZ1eoFG8/gWA/FgW+fAVCFfsK3Tj56b0QPMK1i4C/BCsK3Trk81CWDnFPu5o92RQSZV8k6QxwTHMzHB7mpgINwdmTxr9gJznCnD2mh7cx4oBbUOYDmQBrYAC4Frg22dncAKYC76U4xrgMeA7ejg84hrPAoLgCXAtmBb5zXo2cORaHvzkAm+Qb+WOeAo5P15u6MMXVGbB3aU1hJs6ywHktH25kNC8Gh7c+Y4rlUIIYQQQgghhBBCHCdKqb2WZZ0HvBM4D10h+03gXqXUtCOtlFLfOUlLnOq+1wHrJly1w7Kss4HbkLD2jLFuUw+33bPhsFC1dzTNbfds4O5bVh8W2B6p7fLRVOEKcTqTsFaI4yMPFMPdkYNtNyKhsBuANR0Fey6lQGUz7spU2l2Vji66wdlwYL1z3r4/jrryiQygIqGwF/0JsRhgebIjc9DhbwJId7R0+dyXfn6OTeWHP/jdd494Q2HbkuHX+jOucjfFQmGg+pxM1l1VMVy5PF03tCmeVlQGo78q2FQxaalCYaDmnDw6+M2uiHzHkXFV5O2FTM1oRdDvTQ0Ou9LDVSlvrauAvb882ZtcTa+tv5pcdQ/5G26+y7YwZ8t/cPsz/Sv2PDxSntjnASqA1Lcfuzf/8KoPjPlddb2bnQV/oEhZXcFK9tuVOxqYO3zDzXdZ6Jm+VebxTZRD/x0aBZ4GcjNUYyp0NWcR2A04zKzbIpA0X4cItnWuQM/H3WRC4om3Obn5rrJoe/PTs/oJ608aLjOPe3JY+wumqDQNtnXWoQPmFyY8rkuAGuCH6KB2GTqAHkGfCxv6heYSYA9wLrAg2Na59mhbRQfbOhuBm4EXo+3Nfwy2dVai5+f+Hh0Un4sOoN8F7A62df4GuBB9rvc3xPuHL+7Z7NlYv+wP0cDc59DneNAcuwzdWroPODDhPo9U5SyEEEIIIYQQQgghTiLLsjYAr1VKDVuW9TngLqXUvcC9p3hpL0cAGDrVixCzUygq7nxg85TVrwodrN75wGauW9FwWBh7pLbLs63CFeJ0JmGtEMfBxJB2glJ739HLnvq8x50d82xb+rbkvnnXeMpHdzq9qYGiCWoJd0fykVDYwfiHf2zoat0yIBnujhQ3vON/LFsxV+VJD/ufW3VJ1g32msGNFRl3ld2ZT7t86aE4kHTnxsg6y6yRwFJfT8Oljh1n3ex15JPEyxcodNjmqBnabAG24YolhYLNG/MnegaS3jp/0bJVYFn5MQ/W00sc/nj50hocvh5gxKss7yMLL7b/Knhx2c2bf9ngiW33Lx7dizs7Nu/Sbb+seraycezh4Ln2moKV+9wf/2OosmgVv37ZJ2of91kULAbRgaSKhMKuG26+K894+GoDAtH25h0zneNoe3MS3eZk8vUKOGzfYFtnFbpqF3QV62RLgcZgW+fT0fbmsUn7noWuHF5vqmIJd0dSkVB4HTp4nryG4WmWvRI4C9gH9AfbOkuVwZvQQexV6ID6O8G2zjejg9v/AS4HfomurL0SPQ+4ItjWeSmwJdre3DNhrecCPdH25v4p7r/RHKsu2Nb5GHAt8A5zP2ejg9kA+netEliFnsecBrYMO/39z9Yvdw16K/vM7VsBW7CtU5nHFQD6gm2di83j8gDzzffTVTILIYQQQgghhBBCiJMrjG4ZPAzcAaxlisKH2bAsaximzdzS6Pfvvq2U+taxLXVWa1gK/BVHqKq1LMsNuCdcVX6i1iRmtn7n0CFB6mQK6BlNs37n0DHNxJ1NFa4QpzMJa4U4cRIArWubVCQ0lgdUxl2lgHx5fHexauQl0IGuKxIKx4GMmYdrAfmEpy5TxJrvywzGIqFwbjWU9desGhgtDxZHKpYEUr66giMbT2LZVCVbrNqBF4rz9z4cv/SpX2cjoXCiaHcmvOnBqqwq5EYrl7qcmTHrvI3/Uw+MbFny1tycA09XVo1tz1aNbU8C+Z2L3+RKVC3jycXVfuWM1u91lR1wqmBmQXVF+sH7by/0VIWTPzz3nRVjvgpPn7/K4ytUu+2FbM3C+B57TS6ZijTMVzGbSrY+89340tF9SwYrl59z+eD+XXuc83fvcql0tL25GAmFKzI2RyA0tMvXXb0oFm1v3h9s69zLeJtn4GCrYM9Mc1yDbZ0uoA7om6ZF8Bj6xWE/UAy2ddagq5b96EreHvRM1sQU+1agw0nnxLXdcPNddUANen5rNNreHAX43jU3uz57xUdcE9cbCYWt689/+3PrgpftjLY395tK1HcA+9EvEpuA7zL+4nYfOpxPR9ubD7Z1CbZ1PgVcAbzX7FcWbOtcB7SgK3OTQAT40YR9POjg9DngB+g5sxeg2y8vQM8HrjDHswG9Zh0L0QF6GRDMuH3OvW7fBvRzxSp0Ne6foStyY2a/c9Eh8G/M7TZzboUQQgghhBBCCCHE6eE54FuWZT2KLmK83bKsKd93U0p9/gjH+jzwj8CDwHpz3SXA9UAHetTZ3ZZlOZRS35jpQJZltQOfPsL9hZVS3RP2mYduifx/Rzo+8PfocFqcYn2x2TUMnO12U5lNFa4QpysJa4U4QUpza833OSAX1hczkVCrH3D31ZxXLE/stXvTg564f659w3lXpr26NfBYX/0FBVc2ZkV8q1NUne8Od38vVTe4cTRv9+QPzLkwmPLVF/IO/1jFyDZfzD8vEIjvTi3Y+7t4R8saJ2s6nO70kO3sLT9Ie1MD6Scv/Vz/sm0/dtiKuYUHai8o37vgtYHB6nOyS7f/bNQCf+3QpuHRyqVbt9jzDXEHrqUjUfvle2xLM1Z/equtb2R/8PK6K4YO1Lxpx2Ppl8qrtvdYqdjueeeo5cOBxfERn211/9bowrG9ifYdfyg7b2B7GeAs2lxjGyuqbbtcyoq2NxeDbZ2+lVfd5rtj/T3JpLcWxqtds+iq0cKE0LXOXLfb3O6dIritAGrRraIHSleaNrwh9AzhUXRAeS36k3PPois+d0Xbm/ehWw9P5Xl06+AcQCQU9gPznTd+YUHO7lwMnI+uMv3qx97xDx+srpx/+fxY38ZgW+e90fbmUkXppZ947sfhTzz34/tpbwYdCj+Bbht8BVCMtjcfrMiOtjc/E2zrHEFXw0YnXJ8OtnU+gm7rUpp9+x7gMrPJN4HNJuD+M3P8GvTs2wfQIe0idPXsZegAugzdEjuBbuGcRrdd9qMrayvMNtXApejq3B3mmBeaYzkd+eyi2uTwwkFPxcKcy1tufg5dwJZgW6fzOM/ZFUIIIYQQQgghhBDH5lbgTuBN6A/s38Ck4glDocPYmVwFfEYptXbilZZlfRR4vVLqbZZlvQB8HDhSmPpl4NtH2OZgRz3Lsuaix3v9CfiLI+wH8AXgPyZcLgf2zmI/cZzVl3uO63bTKRSVVNeKM5KEtUKcGtm83eMaCwTL+uZcmKzrfy47VhGszNvd+Ya+p/15u6fwxt98bfTCv/1V5vW7N8xZkthR1Vd3/ognM0TTH7+b/uYt9+ZynoDlS/apjDtQNVaxsDzun5vduuQtNsDtS/R6sq4y+8ZzPlxQdpcDcGw+54M5ez41WHB4ARxpT3W8Z+6VVsXoDldgdFtZ08OtsS/dfNcOf7HWVR2ra6gtFMuWxXq87oqGubvKF4z4dz896iqkRyOOYK3lLvPhnZt60RkYWRjvGwp3R8aaz7u4fkP98oUPzb9w9+v2PrM57a158bEKn/e1u9Z733/Ls+XMv2DuptolFfdf/W9735mxOckcrGitQrflTQfbOl+Mtjen0C1ZcugQsR6oDLZ17jGtkEsGzTYHWxgH2zpLFbHz0KFnDl3pmUJX2O5AB5Qztug1rZUnBo0rgOUX9G3Jr288Z665DyvY1nlh5dxV160c3JmMOz0Z4I3Bts510fbm/kFPxejztUvqvrnyxjcOtXU+jQ44VwH/F21v/p5Zr83cX2me7TvMWr864TEFgG3R9uY+c3mReXzfBx4unRNTafxGdFC9Cf2JySZ0xe4L6Irai9FBbB4dCDvQ7aC/hQ50s+hK3TS6StaLDsTj5vIyc/uv0aHue+zFYtylCs/ndJWuC11pOwR0znSOhRBCCCGEEEIIIcTJoZR6CXgXgGVZRfT82r5jPNwbmLoa9nfo8BXgV0D7LNbVj37P7ohMRe3vgWeADyilikfYBaVUBl3oUTrGbO5KnACXLK6mMeChdzQ9ZQ9tCz1j9pLF1cd8H+s29Rw2t7ZR5taKM4SEtUKcJB0tXQ50VWO6tTuSi4TCIwPVKz3Jsrm5vvoLC+jQMQfKl3eWEwmF7e3+WveAJ2A1FiwrVn2Oa6gqnPrZjf/krykW3MqyUvvnr0lQLPR6kr1pf2YgW/CUO5Z0f89RObrdFS+ba990zoczBVW0wi/da5vTv4GuNR0JV2bEc9GGL+U8mRH7kL8xnnB67Ae85Y7PXP43ZUAyYSP7TP3ZQ/9XOW/fj359l9fm9C5YlBixxwNnWTXDL8W6q18zeuHI8JyVezae58nFd5zTs2lPsK3Td0PoDXPqksOehxeszt8TfoP7c0/dW/vJl56o2+52Vuzw+BXQ5y1izS3Y/EB+bUXa+6W2zqXNCWf8GXd+Xq9DWcBm0NWkmMpbU21aYNLcWROojpYuB9s67egwsYj+dF1gwubD6BmrbmCz2fdobAZGn2oIfwgdRgK8Ftg94qnofXTeeV9CB7jvBWqCbZ3f4vrPjaCDzYvRLYP3of/mTnxV+Of6Z8695vLPmBASB9s6fcB/AblgW2dLtL05F21v3hVs63wIHaR6gm2dOfTv1Z+hX9xWocPc7ejfqUb0HNnvmvXNRb9IfQGYY/YdAhSFQsBRzI/lHc4olu0n5hxegw5h64Buc/zrgPPyDtfCfZWNP0MH4pehq41XmuO+YNYghBBCCCGEEEIIIU4hy7I2oAPaYXSF7bSjx2ZhCLgR+Mqk6280t4Hu3hZ7GfdxCBPUPgzsQs+prSsFr0qp3uN1P+LEsdss7rhxBbfdswGLQ4cel94svePGFcdcBbtuUw+33bPhsCC4dzTNbfds4O5bVktgK05rEtYKcRJ0tHQ50S9SCuhgq5D01vtrhl6sqRnanFm242cDXWs6FFDRM/fqbG3/cwnANj8xYJufGOjN2z1lLqxi1chWd8VY1J/y1hTLxnZZoW3/537+nL9wOAqZpC/VNzZYfU6ucmwH/dXnePMOb2XV8JZo3cDz2Tn9G7wb55xfNn/37ypGqpYn9jVcWaXyI7G9JLx5y17xw7Pfvm3U7fcCFRf0bfF+dOPP84l5r7c/delnC8u2/qhYtDvKlOXI7fPVeF+qWugNFOxzX9ez2TFnOFoA1C2RdZ59/jr18ILVO27a+ofKYV+gblPVgnn+QsKZKls54PJUOHwFXE4sOzpMHY7ZlBNYNWxTsbNTufjisc2FK3s22aGZlnd91rEueFnVuf3bxh58bG05EAt3Rw5+Wi4SCtsmXjaqzblNRdubE5hZtMG2zrPRgWklOtzdhK4gLQW8yrRpPhcdCO+c2L43EgoHgLFwd2SLauv8EfAkEERXrS5Ah6EfAf4TeARd3VoVbW+OBts6f1p6vNH25gz6k38TlVobAxBtb+4OtnX6J9xuR1faNqJbEf/RXB9GV8TehG5fvB8dkObQL5RHzOMYCrZ1vgV4J7rly8+APcB56Arcfehq40uBRG1qpO+coeje3YHGr+8MzF2PDoCT6NB3Lrp9cw4d3hbN/QyiW007zOVhc35tCCGEEEIIIYQQQojTQRj93uQw8DngbvR7Psfin9Ezaa9lfGbtxej3xFrM5euAPxzzag93Hfq9sKUc3sZYymVPstm0Gp5qm+tXNnL3LasPq35teJnVr4Wi4s4HNk9ZsavQvyB3PrCZ61Y0HLJOaZksTicS1gpxctjRQW2ydW1TAcBeSBcKdnfemU8WAWz5jMedGW4oj+3uX7xrXd2B2gsK8bK5w29ad1cuEgrHlFL+hLvGNeKtcfXXrmxYoBirGd3qmrv/kXJlc9i9yYHMQM0q+/Orbkv74vvwJ/tsZ+34Rc6XGcw/U7esKmO3zXcn9xSK1Wf3PbbkdbVxm62mft/vdwdie/jsk9/J2VXRv7PxMs+j884rOhWZ4cqzfRUj2+y+RG82MBYdcRQz7pi33v+mHU+q/b5AakylHjsrOxQH6t770kPzd5TPqd1Stajs0t4XAwf8VdV3n/vW5xNOr/3KpIq9frBv7hyCO/+vohgHcl+qTLnQYeq+5935xJJUr2PlWE/57xZcuOLATX8Vv2Zol+vJhhXZykzcg56t6gB6ASKhcCWwKBIKR8PdkdEJ53gIHSg2BNs664GnTGg7Zm4rfT/xGfcSoBhs63wKXX0aBHYE2zp/C3jCg9Gbbp57nu+a/c//Dh1+PgM8E2zrXID+dOAf0ZWpGXTl6VbgKcyL3Wh78/6Zfimi7c2/mHg52Na5HPhQsK3zd+jZsN3oMPVZYKfZxga8hH4x/DfombTno+fs9qBb2iSB75rK3OvQrYxz6Irbf0G/KA8A29AB8rNA0V/MBvp9Vct3BubWocPYXUCDub8ydLDspFioXDmwY3e/r2r4QFltYX6s7w++bGLdlprF5ea+NzLLFjZCCCGEEEIIIYQQ4oR7DviWZVmPot8bu92yrCmra5VSM86sVUp9w7KszcDHgLeaq18CXqOU+pPZ5svT7X8slFLf5sizbcVJMJtWw0fa5roVDcc1JF2/c+iQ+5pMAT2jadbvHOLyJTWzfhxCnEwS1gpxErSubUp3tHRlWtc2HfyAz/kvPJl6rKWrF1BvevDfi0/c8r0KioU5SU+NbaR8kTfjKpsT983d1dHSFbnCFfBnXYGavY1XO3c7Fd5CIZCbd43aO+81li95IFUW31ftTQ34F+3+bcKRGZtXtDlqvZnhlC8z6Acc8+L9mZR9TNVmE0N7Q7cWbaoQsyvl8TW8Lndd5GN7AE/W4SvPZdK216Qco699+uGRjpYumz+xtzzjrsptWXbR0IK9D1f0155blataGqtIvJSMOZw2dJDnWnvOm4uV6dHElUN7Bj3K4VgQ67PqUqOjW6oX2ceGnwvUD+yJrxndnv2/86/Nqm3fXHRl48rqx+aduxN4MWGjscfjT6XnhLxbqhdVVGYTo2/f+vBgIBMf/eP882NV6bE564KXnpVp6ywDRh/UlaZpTEVqsK3TFm1vLkbbmwvBts6t6MB2LlAebOvc+uD9t28B+sLdkcLEn0kkFPZXXn9HYsRTnjOVtRvQFap2dJVqQ6+vKj7iKZtqxm0OwFHMNzzwi7b/vOHmuy4B/NH25meZ1Po3qNe92Kz5FuDH0fbmjeY2D/AeIPK3I95na8qtuYN2dS06oL7ErGcLOjQtrWE5uqr3SeAu4Dfov+Wl9jIu83NZgm7VHAd+iq4sTpnLNsADfCPa3rxp0novQgfA28x2g+gQOwf8AKAqk1h5Uf/WmkFPxZYHyq6qunrvc4MJl7d+R+X8nrzdGUFX2B63VjdCCCGEEEIIIYQQ4mW5Fd3++E3o7OoGJnR7m0ABM4a1AEqpx4DHjuP6xBlgNq2GgVm1Iy6FpsdDX2z6oHaq7aRlsjgdWUod7ehGIcSJ0NHS5XCmh+fnPFVUH3gu4MzF69KeyrwnF+8+56XvufbUX7YsXr6gopAZ2p23FYqVBaqLNrfdmY33BcZ2eFy52PBQzTkD5fE9Pns2uciTG8vaKO5FzxiNH6hbnfUne4j5549EVtzqd2VGU1l3wLso+itV1/9C9UjgLHt5Ym/ckxkeW/3cY2Nmxq4PWDRiFe3DdrV1Wd9mx/k7Om3PBaoL1+x7vgFdnTl885v+zVmXGrG/L10zVDu4qbInvrlqa9X8ke3z12x9TUItq08cGLtjQd0AcP6Kge3Bimyyb19Z/cY9FXMsoOILj661LFTZ/zunOfqbr982NPG8BNs6G9Fte3NAb7S9+UVzvQ1dKVoJRKLtzaXwth6oBxYC1W+IPpl4047Hxqozsa3VmdjeG26+y1uRiV/4uSe/XX/OUHTHDTff9SxQg27x2xNtb34q2NZ5pTlvvzXtiyeu5xzzbfk//+kbjRf1vVT1zhvutMbc/nS0vfneYFvnFeiw+P5oe3M+2NbZZNafBl4PfA+4H7jKnN9PAZveGncxZC++9g+efEBZ/BYdlIbR1bpuoA/4BLqS9w7gxWh7870An3rr7R9/qXrBR3YE5n1j85ff/p/Bts5rgL80P5/16MC3EtgUbW/+Q7Ct80PAZQvHer7w9a4vXwHsCndHHjGP7wrGZ9Seg66SdaGD3s3m/uvnx/rGkk7PziFPxfzgyP7hlNM9/4C/5gA6KPaYbYcBJp9DIY4DefEihBBnJukpJuQ5XAghzkzyHP4KYllWEWhQSvW9jGPYgZvR710BvAj8QilVmHan04BlWRXA6OjoKBUVFad6OWeUQlFx1Re7pq1gtYA5FW7Aonds+m0aAh4e/XTTcW03/Pj2Qd79jSeOuN33P3IZlyyuPuLjOBFrFGeck/7Dl8paIU4TTQ+32jOuigOPXfGF8qHaFQV/8kDKmY2VK8tWuXfOpfmBqrDXnR1b6M6lXD3zLutp3PXQ9sqRrRVJd1192hWoHKsIpraEb8lVjO4cs+dTA7ZCNt144MlA7eAL54A9mXZVbk166z3Vw5HGZVt/lPYm+/L99atHK2K7XZ7MoEuxJD/gbxxeNro9GwmFrZW151XsrAxmHz7rWlcW5dzoKTYEEpGyivie3DXxPTuA4oivPv29ZWuKl/Ru7r19ww/KRsrmu+5fco1rS+2lvbsqGvpvS7pc2HA8UzXX71C5irxFVaQ6aGuMDzRctX/j6nMGtvc/2bjyN+cPbBvpnXNx9U3W2a6Oli7blypTdnRYOIQOLSOYNsgAwbbOavS82Ly53R9s66w7K2fbd27RfuVeR7FiyK6GgNh+f429a8HqFYFsovadW3//ayAfd3qtvWV1B84Ziu5HV71ejA4Yo8G2zgp0a2AP4Ai2dZ6HDkqXoytGFwHZaHvzzyOh288BFq0a3P74Y3PPLc25daLDzdLM1g3olsIH0O2SO4GL0J9S9KDn6Lq6XYV6X9HapCz2Ax1AOfAG4AV0hewS4Gx0a+Sd5lgANCSHnJ5CdsCTz/7eXNWDroiNo+fovoBuAV3a54fAA1/v+vIIOkTOTvhVXGzWvxn9pOQCnkDPt30tsBJ4em95/S70m23LopVzM+ZxDpt1Vpl9q82/M7aDFkIIIYQQQgghhBAnh1LKduStpmdZ1lLgV+giipfM1X8P7LEsq1kptX3ancUZazathnvHZq7XmKod8fFwyeJqGgMeekfTU34ysBTAXrK4+phaJgtxMkhYK8Tpw+HOjtns+XTRl+qzwpHvHtjXeEXvaGCpb7RyaSHjLH9pYXRd2pEbnedJ7C/rqz3XnnOVp72JAz15u33/4wuvTNtyw+4yipmxquUObI66uK+uUuXz2ZrR7mzl6FY1VL0iC5anLNGTsOeSVsZVmY2XLRjyJg5UJly+5S/WnzO/18n2q3c92pf21i5UntrRxzz5/VWpwfJ83pfaEWj0dtevSoX6NqqMs4zd816T7517uXOLFSu4ivlMIL63du7wluoEC+J3Pfo/1dsr5ju+fOXH4+ekst6bevsbNnn4XWO8/5z93qrlvlyywaYKlj+X8gCpzeFbC+jqzxF04FmODhLdQGFSdWYOHTA2mO1yQHXaUqkrs84RnyoMPWHPjwCZjXVL+3cE5q54/a6nBoHCg/fffjawMdwd0RW8Ovh1oitHlwKBaHvzI0A62NY5F9MSGViNDlx/jg5YCXdHXgRevBcItnVeEmzrDAOPR9ub/4C+zkK3M77wvP6t56Uc7sCWqoUvolsE96OD30WAd7Or8EfgO8D70PNqfxdtb/6SOU4Z+gXwregWMw9iqlYBOs5/W8ZWLO68MOu6tqOl6/K/xfvDL1WmvgjcBlRH25v7gm2d1wJzgm2dz0Xbm7uBOO3NAD+e9Hu4DrCi7c0DwbbOveggO4UObVeY9T5kbneh2z4X0e2jc3c88f+eSTrc5V+66L159JxgqZ4QQgghhBBCCCGEOIUsy3oz8KBSKme+n5ZS6hdHONx/ot8PukwpNWSOXwPcY25rPg5LFqeZ2bYaPtnHArDbLO64cQW33bMBi0PfjCyVR95x4wrsNuuoWyYLcbJIWCvE6SMJUHB4EsDIYM05rkT5gsy8/Y+6RyvOcvXOvSyX9tX1LRx4tj+TGtlvt+x5X+KA350bzRZyWcpiOyqXD+0+t+hpsIEtmXZX5jK+huL++VcV0p4qd0ViX0V5bM/uQCyaBbIPXvUFok5qwnar3J6Lq6LlilmOcm+ZcnmBQNVwd2bUP9/5Z309TvfABk8RlVTeBucLi5rKNjScu2Rx/wuLFvY/l/673vU7ArFdAANFy1b4U+NKdyAVryzAgkBmbHiIxI6rt/22ftHIvooPxncvi1Y0ZH+87NrtL9Us3lGRie//zz98zYduWbwHGG5d25Ru1UHprmh7czrY1jkMEGzrXACUAVuj7c0xIBJs6xxDtzDeB3j3O5S3oWB7jJ7flA/UndW0rWpBHKhIuHy5ny17Te6ssf2e1+15xvpl8HL/DW2drwNGrt39TM+gt2LghbplT6DbBh98Jo62N+8PtnXG0BWpm4CxaHvzQOl2U3VbD/wOCAFXAnET8m4HrkbPqVXn92+dV7DsmS1VC19nftZtwPXAh9FtijegK2nPBy4Dbg22df4VOki+BF2Fuwpd9fvvQE+wrfNL6LD650Wb7fpBe/GmIurnNix7tL15d7Ct8xnAFWzrPAuoNcedG2zrvM7c31PmuMVoe3PcPOaD83mj7c37gm2d+6PtzSrY1rkI+APwOLq1DdH25izwnHm8C4DMZb2bfU80rJhblk2OvnnHo6Pv6/5NnPbm1Gz/IxBCCCGEEEIIIYQQx9396GKEPvP9dBT6A/kzeQ0TgloApdSgZVltyBzbV6z6cs9peayS61c2cvctq7nzgc2HVM42BDzcceOKgzNoZ3vfJ2KNQsxEwlohThPh7ogCCIPqaOlSsfKFDsCat++PMSzlAGxZd2W6d+6V6ZRvrn3J9p/6ypP7AVJ+4Ibun6YyDr+9r86rKke2psri+ys2LmnI7WiYN1A+VlaVy5U5806fDR3+Ve8eeXx/xl1hV7E+tzc9kAjEokOZwFl1AVwuwF4e36dqhjYvuHrElXKPRQY31C0Lkt9T3eOp2RIvn1/myifTlr23Lme3W1fHdr3w+FlvuPDphlW5Z+sbEtfs2eDbUrWw/8k5Kw6Musv3nLv3T4MjFWfZBmrP9Z418Pyev3vmvr4bbr6rCqi7bs8zuXOGoqnWtU1FJgSlD95/eyZy/+3VD0LsP89726XbA/Pqe/3Vxep0zA88DfCDX90x8vOzrrhmzOUPD3gqH3ly7srh1rVN2V9e+q/zE5YqU5bVvb1yfj/6xWjDly989/yE0/vS/FjfdXPiAzcfKKvtibu8PZaixlHIv5S3O4YsxTXBts488Ey0vbnPBMMAO6b4sTWaLyc6cD0X3Xp4CabaFx3M2h4MXl705tMPooPl89EhbRY9k/bX+kfPHuCTwN+hWx6PAH505aoNHUzPMdv1AW9Eh8VfBT63w1lc9OXK9OPR9uYhgGh784+DbZ3vAD6Hnj37XcDuymfPffeWh67ZWrng/D/NXTUf2BVs6/xhtL35YLVuSbS9ufRhtEWAF3jOhLQT1aArbwcAx9z4QH/c5WvI2BwVQME8DiGEEEIIIYQQQghxCkxsffxy2yCjixrKp7i+jENHbYkzjGVZ/OxnP+Pmm28+7LbZtBouzaw9MHbkdsQnwvUrG7luRQPrdw7RF0tTX67va+Ls2aNpmSzEySRhrRCnoda1TaqjpSu1bOuPrDn9G3JrHrk3DdDRQm6kOmQLjGz3DVWH7eXJ/QPh7khywq4RgEgobMvb3UscCw54fLl1seBeK1awOzwZT62/YDkTdpXLfHTTL7L/dvEtveftfdyqzsScgDPr9G1158YUUEy7KpJJb/0cG4V4Y3KotzwTX5C1OwOv2fV4tjIbL/PE97n/tOiKyqTDlR11+dIxT7WvLldw3/HM/bt+OT8U+4/V78pXp8cKD/z8784DhrqXvbNnzsDzCxl4fizcHSnOu/3Bwj5HMV20rDhQGwmFE+HuSCLY1ukGGj549hsy1wwPLPKmBvYtHd3ndhZyRVuxcFbRsl1w9u0/LWYc7ue/bXfOcRQKr0nZXKmrejY++sPvfjoFsGR0/9Ylo/t7P/Lav/OHB3d+OGt3btpeOT8FrOiuWpi9et/zC6/e90L/j89ueiRaPieZszuX5O2O+W9OuBb02ItvfcGVP5CxUQH8MNjWGUCHsC9E25tHSyc62Na5HHgeeDja3pwx634dOmR9Al216gUeBWqtYp6lI/vO3+evXZS3O1PoF7B9wG/QYed3gAFTTXw/4DHh6XCwrfN5dGD7hLm/jWYNi9Btm9PBts7ngDXmWP874XfiMXR4vAzYG21v/mEkFI4A73tg8RWZP81d9QK6pfLE36OpPIVuBb0q2NY5MqktdRboj7Y3FyP332758+kisPj+pdfYPMX8S/86fs4C6Ere/mh7c/EI9yeEEEIIIYQQQgghTj+/BP4/y7I+BKw3110KrAWO1EJZnGK33norIyMj3H///Ue132xaDX/qNXP5XsddPPPr31AY68fmDeBbfhmVV9+C3e0HxtsRnyh2mzXjrNmjaZksxMlkKSXjBIU4HUVCYQfgA7Lh7ogJa7t8mNmggNW6tikFEGzrtAEVQCZqWs5GQmHniwut4jm7lQco7Jl3TbmtmK+rHNlW8Kf6EuZu7Oh5pCn0bNgU+tNxnpSnxtfTcFlVNPjGva70kCM//OziTS6VrvUuiV0f+dEKf2xvYFdZfbm3kM3MSw1tyNmctf3V5yy2W9bgPQuWb8nYnOrvnrmvzK6KQQUM+uq7u2uC/gt6X9i+/sovg65I3df0cKtCV4fuD3dH0sG2Tn9wtGdpc882+8pkehGwLbj711u/vWLNG5+ds2y1lXeP1aVGdi8d3uPwFnPzFo32pIe8gX1fXf3Ox7J2pyva3rwrEgovABo/cfXHqlMO95+D9eyemmX4WAABAABJREFUQMNvgYuA5+9+6N+H/295U0PXwosG0RWjWwBvOGu/3KlYE3EVhnMWT6MrUZcAF3vymR1Zm4OizX4ZOpxcCjwSbW++v/QzC7Z1fheYjw5io8DDwE5gxRt3/KlqUaznop8uXdN7wF/zr8AQOrg8DzgHuLvUitgcay6QjrY3DwXbOp3oytWFwI3Aj6LtzdHJvzPBts5zgFS0vfmwKuBgW+dCxqt+0w/ef/tLQK5U0W22CQIrVmfsj//0K9cPR0LhOnOfL4S7IzkTti4ChqPtzXsm34f5vVsCOG96078NZR0ue7S9ucccu/Q76gH6JKwVL5O8eBFCiDOTvOMh5DlcCCHOTPIc/gphWZYNuBV4KxBEPzfvBH4MfE/NIiywLKsSXXRwI/o9StBFYb8AblVKjU6z6ylnWVYFMDo6OkpFRcWpXs4pMVNYO1Nlbcm6TT2HtRpuNK2G5zPIHXfcwapr38zPozb279vD0K87cNUFOffWzx/SjvhUm+lxnC5rFKfUSX/ek8paIU5fBXQoWHrRQ+vappmqH4vm62DQe85uZQOS4e5IpqulK7tw90OpeT1/cqEDs/noakqHuZ+MuS8P0OhND9rPinbuaV53V+L+G++Y60sOJodchZG5/VvmuFP9zzopZhpy+WstmHfAHXAHMnF/zdiO1LA7sH9n+TWqzBFoXLv6VseVgwP7FgxuXP77+oXhH5392mjuDTWuhZ6vVL9m+7sK1anGXLg7ko6EwokfLH/thRs+9LVK6pZu/8cnv90/PzlYsW3O6v13L79i3ubV19muSa4dSlRltyyPlvfVJ6zqXeVz1qSc7sAuX90zVblEzWt3rV8ayKX8D1/wr71z9GNJD3oDfYOeQASbbRTYjQ487be97u+S4cFonyufnZ91uFbaioWr61KjwW5vIKVs9j6gH/ijaQG8bfUnvz/055sf/OvN1cHFvwlemgHOQlfF7iud/EgoXLXsmo8/trV64dXo+bV29AxcAPaW139pa9WCWw/4a7yAc8Js2H3Bts51E8NLE86+DxgE/jfa3pwDcsG2zpz5meWn+gWItje/ONX1wbZOt5lhawEtQCHcHfkLc9sqYDnwK2DO3LztslUZx9kdLV3fb9LtoxcC0WBb55D5HSkD6oJtnfumCVz7Ew63PetwlQHlwbbOuPkd8wIHou3NI1OtUQghhBBCCCGEEEKcOJZlWehA9Y3obnEb0YFEGPg2OsC9+UjHUUqNADdZlrUM/R4YQEQpte24L1qcdAMDA7zlLW/h17/+NfPmzePLX/4yb37zmwH4/Oc/z9q1a3nu+RfYPmbRF0vzxb++FRd5Xv/p32GzNfKTn/wEgM8WFet3DvGz1TV89R8/zsOfuga3y3nY/c0mID4RZtMyWYiTScJaIU5DHS1dPtZ02FrXNsUnXOdHV9PGQbc6BlS4O6JMaDY26TB29CzVLJBpXdtUjIRaS7cVgAQ69BtBf4oui6529fa7K8bK8ulybyFbGwmFU2frYDLedfZ1tSOFTEXC7oh78/hT/ob9Gbsr9nS5z798bKCqMZvvj1np4tnDe6vHair9PTWhUd++/yvzZEfm1SXL92TsrrEl6g+FUUfG/qPz2/dtfP/Ggx9dUuBxFfNeQO2onNc7Pzk4/FeXv6cW/YIx/MeKd75k///Zu+/4qqv7j+Ovc2/2DjvMsBPgAiJLZSiKgoirjroaO53V1lrFtlatbcXRVq22/qq1Rm2r1i0oiiKCgOwRIGGHERICZO/k3vP743uDIawACTch7+fjkUdyv/NzcwMnyTvnc4L2Ved07L4rMXzfwtDqyoL80KgLoqvKupXb0AFtK4rtlZu/2hjuq+6VnJGeDuTunzZzEE7r4GX+51sI7Gtflj+4X8GOoQ8sfXXO9yb9Nq9NRfG1A/dtLRtqa575NHF0Cc66G/v9n+e4f0ObLbGdlwErPksctQ243f+5G+oPUL0futyJV275quPT0ddsqgwOzQE+AOLwz0Rd075PJfB/ODNyN/pf0zDg/F8Svoo6wW/m9CnVidNmzq3/mmZOn7IZeO5YXz91JU6b2RuY6g+EMxKnzZzBwf/3n4vzjfVsYMk5FUGFHXyufv7P11pge3JGej7TZg5y+7x971z1dkV0dfnuH335v8POjE3OSC9KnDYzCOgGtMP5WtuN8/Wl2bQiIiIiIiIiIoFxMzAOON9a+2XdHcaYCcD7xpjvWWtfbcjFrLWbgE2NXqUE1COPPMITTzzBk08+yV//+lduuOEGtm/fTps2bfj1r3/NrFmzuOUnP+a9997j+eefJ33VMlavXo3LdfBSyD/8wfdJTU098Lh7t64MHjyY6667jptvvvmQ4xuiurqa3/zmN3z88cds3bqV2NhYLrjgAqZPn07nzp2P+3rHapkscioprBVpniyHtgg7EHSlJyUbnBmOtaHrQZIz0muAvPSkZFdyRnrdgKwcJ2SsnZkZC3TZHNvFt7Jdn+pLMhfFBXmry9/te25kz6Kc8LE5G014VUFHoDwvNLpiU3x3b5fSfRXtKou7Afvb563dDBTv8/YeGeu1Ueviu8WlxbTfZ/EWJmctdA3ft33jrpDgHp2rS3e2qyzaUOMOqrzvv+HtYmvKXJFVxKQ/llyVnJFelZyRbq9LSl7wnZ93jXHn2n4zh3zfVdMvJBvKdwNfAlHUxFcNycqN71P49Y7vr//E+9QZ167bGpfQMdxb1TuyusJXFRTcKcTWZANu/+dnE5c/1SmhZO+YXoXZ+xZ0GbwXJzzd37aicOm4rNXF2ZHt8oDVpcFhn1UEh7gzIzptB84CqjKnTylKT0oOBgb7MP0Xdxqw/rXkSRlAm94Fu5ZnRbQbVxESluA/3vXzsXdWhHmryyuDQ+cDKzKnT6lKnDbzDmAszl8rnpk5fcoCnAC0VldgBOBNnDazEPBlTp9SBpA5fcriBn6tHEsfnDV35/iv+169/euBbKDEP5M4w/+GfwnkfP9x+aHeqvwz924KaV9ecNSWOJnTp9QkTpuZDrTH+RpzZ06fsq9Rno2IiIiIiIiIiJyI64A/1g9qAay1c4wx04EbcJYFO4gx5s8NvYm19p6TqlIC6uabb+a6664D4I9//CPPPvssS5YsYdKkSbjdbl5//XWGDh3KtGnTePbZZ3nppZfo3r37Ya81YcIENmzYwJVXXsnNN9/MrFmzuPvuu3n77bf58MMPCQo6vniqrKyMFStW8OCDDzJkyBDy8/O5++67ufTSS1m2bNlJP3eRQFJYK9IM1a5FW08NEP78rXOq78hIr0lPSq7CCWuPqF5QS3JGurfOOXvSk5LzgIiCkMiI6Jrydl7jKg/H7uuXvyO8Y7VtUx7aJgxj1odX5ncPr65I6J+/vcBr3JGFwRGFsdVlGT8df7f5wbqZvQfv29y/wh3mLo5p54qsLOwe4q3J2B8WkZ/ndnfc137YgIyQdqVF3qK+g3I3DVycMCDzii3z9+OsX1qenpQcvqxD/+jiLmd0vXLhjrj/9a7pWmp83m01e9Le/Wi6d0WHft3mdj1j96+XvrYZZw3fdkDYT1e/G1UZFLy8Kij0fVdlaeQl2xZ2dVvbDmdWbClOSLh8wP7MwS7sWJy1N3xA14w2iWnfWThrWeK0mROA0PLgsPkLOw++CCc4HQvseOH873Yc7zzOftFz+TkZbRK/06tg1yeD927pszc89ozSoND4vUHBZV6Xex+QuyW+WxJOG+ldn7x/b02iM6N2K9AZJ1j3JE6buahu6+An48pDutW4tnsq3fPKty5/sDJ7Y7Qxlzzj/8vExrIN+AxnDd1DZE6f8kVDLpI5fUoWkJWe9KCLBqw15p8dnIvTOrn+rG8RERERERERETm1BgP3HWX/J8BdR9h3RgPvofXpmxmvvx1xbavfY61KPHjw4AMfR0ZGEhMTQ25u7oFtvXr14qmnnuKWW27h2muv5frrrz/sdaqrq1mxYgWjR4/mT3/6E8HBwQwbNozRo0dz/vnn88orr/CjH/3ooHNeffVVbr/9dlauXEnfvn0BuP3225kzZw4rVqwgNjaW2bNnH3TOc889x8iRI9mxYwfdu3c/5PmqtbG0FAprRVoOixM2Wn8L5MrkjPST+gYoOSO9GihMhsL0pOQynNbJ9rysVblVQZFBNa7QjiHVxflA2Jp2veIXJHhi4yuLc+OqSrNG7UkPu3LLvMSlHZOCuhfn5LWtLCkdk7W6x+7INgMqQsLCNsZ1z46sLumTULI3Pysihr4526qv2Pp1Wf/87TE4gXG6//2wopCIoNKgsG4dshOie4Z70+IrN0cEFe7qNrvbme2KQiL6h9VUlfxuZMqe3y5J3b1i6M/aGG9JTZ9NL/l+vfR1A8QDBZMvf2rJiJz083Ii2lT2z9tesrJjv+LM6VMqp09+97WFnT19npj/fOJ/+09ss75Nj36efVvbJ05jDjAc6IDTpnc7UNEnf8ea0TnrN4/PWl0MFM7vPLg0s8MQfO6gmis3fLpla1TMBVXuYPeEnSv2Dd27KevX59zyUbU7uNRfx7wHF//rvF2R7WKjqsqqS0Ii8oGintWuqJ7V7jEhMBPYWecl6L21aGenr/95+wf4vBf4t91hjJkFXG+tzeckZU6fshF/2+XGUP8PAI5x7xr87aRFRERERERERCSg2gB7jrJ/D87vtw5hrT2vSSqSJjVrbTaPfLSe7MIDK9FRujabvrFHDi+Dgw9eV9YYg8938K8D582bh9vtJjMzk5qamkNmyBYXFzN79myCgoJ47733DrrmhAkTGDJkCO++++4hYe33vvc9ZsyYwQ033MDChQv59NNPeemll1i0aBERERGHrbewsBBjDHFxcYd9vgmxYTw0dQCTBiUc8TmLNAfH3xhcRALijhcm1NzxwoSSO16Y4MWZqRnZmNdPzkgvTM5Iz0vOSM8HXCE1pZURVXkb3LamHMib2+3M9dvjumxe1TFp7ag96blA8JDcTYVVrmD35thuPmBbtK9ypRu7O8Rbk5dYnG3jKopqOu1dvmVR2y5bisOi9pyds25528qSfaWukMglHfqG7w+J8mRGdeiTEd+jfG9YzDKfDWNk1u4+o7KWFScWbM/856BLcjfGdf20b8HO1TdlfBo7+fKnileFmcFbYnd5Xj/PxO52vn2MyuzA8O/m/+bKNR06j98Z0/F7c7sNmxDvdU9+/tY5fS7euSru8QX/l+vZvy1u6pb57iF7NwcPz83o+cn790YCC3HaAHfH+tqFV1eMG5e1ZuCkzMXmuSFXmsmXP9XmjyO/d9bW8Ihqr8u9eFBZyceVQaG/KQsO/6gsOMxsievS68OPHiiMqygOwgnTz1zScUD/7Mh2Lpe1y4G2wMhOXteu/tXu9HUh3imJ02beVOfTPmv3P28/H5+3/je8E4H/NObrKyIiIiIiIiIirZqbb5dGOxwvmtx12pi1NpvbXl9xUHAJUF7lZdWuQmatzT6h67755pu8++67zJ07lx07dvDoo48etL+oqIgLL7wQl8vF6NGjCQsLO+QaSUlJZGZmHvb6//d//0d2djZ33XUXP/zhD3n44Yc588wzD3tsRUUF999/P9dddx0Ld5Qe9vnmFFZw2+srTvj5ipwq+s9XpGWqAnzpScluwPjXqG1M1TgzIsuTnZbLBfcv/0/1K/9+0KYnJYfjtPYNaVNVsntAXma7uKriyC0xCTX/HDQ1947V786pCAnv4Pb5fAs6e7bkRLY1t6R9sCWmqmxyTlisfbfP+OheBbuHJZTtn/hx9xEVNUEhRWXBYa7ciLjtI/ZkzO5VlJ3gsr7oJ868rmrg/q0lk3csKzgrZ91KoAyILC/f4U0o31oaU4orrCLY/dKAidtGl3wc0bWgcsjgmvU5S9uftfuCHUtLhhQXdixOmFCZE9Olqlveptyc8LiavRHxw7fFJuzrVZS9HijOnD7l68RpM7Piygsnx1aWBPUoypmzJyKu91+HfMe1qkPfc4EuQObkLV9kjtu1JrtjcXbw8/P/szpx2sz87TGdatqXF+RdvnXBwKfmP3fDyvb9it/sf/4TnyaOSv80cVR65vQp5YnTZu4HsheF1aSNqQjesTPINxk48Kdk2x+/pCdw/mE+/25gkjGmbyO3RBYRERERERERkdbJAK8YYyqPsD/0VBYjTcfrszzy0foj9qS2laXc938f0v77Iw+0CG7btu0xr7tr1y5uu+02Hn/8ccaMGcO//vUvLrnkEiZPnszo0aPJLyhk7LnnU1JWStKQ4VRXV5OTkwNA+/btcbvdzv2txZjDz+6Nj4/nn//8JxdddBFnn30206ZNO+xx1dXVXHPNNVhree75v3Hx35cd9vlanC/8Rz5az8QBndQSWZothbUiLVByRnoFQHpScgzgTk9KLjjZlsj1rm+B4jqPq+rsrgTycELHyglZKwuAFfeMvbMovW1izC/H3r63Y2mee+i+zfHdinLCuhfv6RlXUdRlb1jcRRltelSGer05Yd5Kb41xFfncQSbMW73u+ozPimcmnjXm5UFTQ6/cPG/Dxq5x51TFhl4YVeZaMcKZxVuSnJFekQllP75u6WeTN26sAswvx/yoW0Lp/jPjtrcrHrRj3//O553CyZefVR5TVdZ3R2jI8F0h1aOze19Qs7vvmP/8ZvEreb0Ksrb+9ptXCrIj2zL58qc6R/ziHc8l25d1XdW+T1FUdUVFWvs+633G1aPKFTT8nKw1vTuW5W/5qusZu5Z1TE4oCI2OGZa7sTrZ+RzklIREPFUSEmG3R3cYuTOq4xllwaHlId7qiMzpU1YkTpsZlDht5uVAzsAqd99wn7nw5eiKX2ZOn/JhvU9172O8FH0AhbUiIiIiIiIiInKyUhtwzKtNXoU0uSXb8g6ZYVpXxY400p67leHPfbvthz/84VGvaa3l5ptvZuTIkdx5550AXHTRRdx2223ceOONPPmfWfz6xQ9IX70cgO2bMgBISHDaD2/bto3ExEQA0tPT6dmz5xHvVdtmOTs7m9LSUqKjow/aXxvUbt++nTlz5pCxv+aoz9cC2YUVLNmWx1m9jx1KiwSCwlqRlq0ccDVmUAuQnpQcjNNmuTg5I91bd59/zdKC9KTkKJx1LGqA8ku2LYiNqK6IXt4pKXd09rrdl22e54qrLgsGKiuNuyI/1LvMHd65Xbug8D2D8776z/Pj+3TqvX/j2D5Z2C+7DutSEBIV27lkr2t92x411W02Zw4o2xDfr8i79j9JE/dNzvxm0LJxl4fsC4+192xf8g3QA7BXbZprg73VYe3LC7OAXckZ6TUxDz4xLtTsv2Zw9p62HWpYuyc0zJVUWTC2ICz6s0H523usb5PY6b9JF4wClnUu3WdiqksH9CnM6rgjusM3+WEx41zWtzvYl99leO4Gt2f/1phzctaWvZp00cb4ypJOo3PWd0+cNjMXuBbIBObfev59OWE1lQ9EVZeXL3vmhiz/p8kA4UDYsMqgYqDkvIrgw631uuUYL8XmE3j58KR6XGkpaQ1eW7axBOq+IiIiIiIiIiJydNba7we6Bjk1couPHFy2m/Jz2k35OQDPfHcolw3tcmDfSy+9dMjxBQUFBz7+/PPPD9n/7LPPcvFPHuC211dg2ybR4/4ZAOyb+Rd8laV0uPI3/PyCvqwuCCZ7y35Ktq0iLS2Nn//854etb+HChTz++ON89NFH3H///dx5552kpn77dwa1Qe2mTZv48ssvadu2Lbk7sw57rfqO9nkRCTSFtSItWHJGenUTXToYCMH5P8J7hGPcfPt/SHBCaV5w76IsX1xVib1r9Ts+wAfsAryh1hvfuzjntdDCPf3LIjqVnLN6afmmSwe0yQsPr8qIP6NtkSsyMSuq3cKLty8uS8rf0Slhfe7OnPj0mmrC2r6ZeGXP+QmeTlXukP3ZUW0T/jXg4n0vfv5EUEx1mbdDWX7vbbEJ3fLColcnlOWFe1I9pSahfW/vdm90z6LC7vvCY3cT1Pmtc7NWdmxXUTQQ6BJWUxnTrTi3wLNv667b0j4waW16Lu5dlJ28Ly/aLqssCRpctDhxxllVu2YOj/3qvA+KtrarKNr2yOJ/LQH6A3twwum9OG2ihwDDKoJC52Q8deWB4DVz+pRq4L8Az9865xsg7I4XJpTX/wRaazcaY2bhrFHrrrPLZ4JD5/uqKg6ZVZuelBwN9ATW+oPzg3hSPVOAHp5Uz8tpKWmN8h2IJ9UTDQwFsnFC/HVAGHAZsNb/FgqM9qR6dqalpGk2sIiIiIiIiIiISAB0iD50ndiTOe5ojtZy2dZU4y3J58n3FuEtLaBi2wqKvvkfI8dfwPe+971Dji8uLuamm27irrvuYvLkyXTt2pURI0YwdepUrrrqKqqrq7nqqqtYsWIFM2bMwOv1kpOTg7uiEOutxriDD1NF4z5fkaaisFZEDpGckV6WnpRcWX9Wbb1jCtOTkotwZpGSnL+d5Pzt0UAHIDc5I30HQHpScjxO4OudOOcfq/zbwkeBq8pdtjk9dnfwpnj30LyKaM8b/S/YFFdRXDEob1t+dYcl/XyVCb33BLWp9Bl311Bvxfqvup5ZXOMOir19wj1rX//095V/GnZt18LQqOphezKmXLRjWd4rT23PWDggO6b9vpDcPRHxlbui2i96o/8F61PSZ1UAXYGPehVlRz+w7N8unHV5x3vytqXvieODL8a7Ss5ev4rBuysS45ZR06aYXckZ6WvrPOWl4EynBWYAJE6bGWdC9sZEJv5tqCd1WhjOWr7paSlpu2pPuuOFCRVAxY0PDbw+s6PpVBhp/p6WklY3uL0e+A8wqc62z2x15fVH+NSfDZwBlHL4mblFQAF1Qvb0pOQeQPA1DwSF47Re/jgtJa3qMOceSRTQDojFCWuzgbZANyAOiAGW4KwrfEgoLSIiIiIiIiIiIqfGyJ5tSIgNI6ew4rAhqgE6xYYxsmebk77X0VouV2xbzq7nbwKXG1dYFCHtexJ//k/Y4zmf2em5TBqUcNDxd999N5GRkfzxj38EwOPx8Mc//pFbbrmFs846i+rqaj780FllbujQoQedm/yjpyhvm9Tkz1ekqRhrG7V7qoi0YulJyeFANLD/aEFvelKyAdrgBI7ezOiO49La9qp6cdClmdVBwbHfXzvT3aFma68vepzB1rBBW87avXZMqLd617v9zvsCiMAJB7f/fPkbiVtjO/faGx5besm+vcndC7aXxOenF85IPLvnJ4mjzF2r31mfnL/DAB1xAsY1QCHQe2134rrsIzG+jGW33OnO75dlv7cnDjeWbU+84qvEaQE9Pz0p2SRnpNv0pGQXYOu2nE5PSu762gRXyEejXBOALJxAOC0tJe2bes83dvZQ87v13U3QgoGuXxxuxqsxpm/bi9qmRPSNyNvx1x1/9qR6hgH70lLSdtS7VjzOLN9lyRnpNQ18XX4AhF/zQNBGoC/wWlpKWvExTjuIJ9UTgfMHPpcCG4CVQAJOkBuUlpKWVu/4HkB5Wkpa7vHcR6SB9M2LiEjLZAJdgAScxnARkZZJY7icFowxMUBhYWEhMTExgS6nSc1am81tr68ADv4GrPYf899vHHZIWHoiPliVxd1vrDquc2rD06/vn4Db1Tj/vZyq5yutxikf9zSzVkQaTXJGejkNmFnpDzz3H9iQlLwusXhP8H2zXt6ZOG3mniu3fDU8yPpMfEnQV0P2vZ+3ql2f+I3x3WJwZou2AaLvW/bvsvN2rTyXnfiq3eGfLjj7jxXpCRUVYxY+kP+Vx1UVV5E2wmdsB5wAtQfwGc7s0B7lwfRc2s8V+/r5UVsv3PZY5v89d0eP4lDarOxFaEmkycdpYexdNiR58s4unP/CzwY8cSuclR9JpSfVM/utx2piXp7o6jiyO98Zs86X+9Eo12s4baOjcFol13++hSQlvzpxld3zwhPrDvunZtbaTZ5Uz2Yg3JPqiQIuxAmAX6t3rXzgm8Nc4mg+xfn/fifwdVpKWrkn1ROP0355WVpK2tZjXSAtJa3Mk+px46yjuzctJa0G2OlJ9QQDYzypnrH+emtbRY/HeY1n1r2OJ9UTizMD1+IE+yVpKWlN1c5bRERERERERESk1Zk0KIG/3ziMRz5af9DM106xYTw0dUCjBZcn0lrYAtmFFSzZlsdZvds2Sh2n6vmKNBXNrBWRY0qcNjMY8GVOn3LE2bKNxT/rtidOuLjJP6s1FqiZfPlTNUASUPbJ+/fuAQYB8cC2GnfYbs+6lQXpSckjZ4ygy8r4YUnB+X223bn8fW+Ur8oDfAyE48wszf5kWHhZlPvqoZFV8elDVz8zLKstZ1cEs6dvDvOB94ALd7aJPrckMmji9g5Bz5yzbm/6S1P6XLanTezK6S8ut89d4hpeGEG38Wtt8Z547nlznPtioDgtJe3t52+d4wY8yempNQl7lsQDC48207juc6+duetJ9fQFCtJS0vb6H7vTUtKOeg1PqicOKK0bfvrD0YtxQtlNdbZ3BC4BvklLSVt3zBfm8PeLBqbitGXehTPruRBIB4pxXpu9ODOoS3BaZAcBOThrGvfGCXc346yTXA24+DbEbdDMYWmV9M2LiEjLpFk5ojFcRKRl0hgup4XWNLO2ltdnWbItj9ziCjpEO62AG2s2a+31xzw+54gtl4/mzvP60LdjVKPW1dTPV1oNzawVkeYlcdpMgzMjtRrYdwpuGYQT1hXVBpfJGemF4KwXmzhtZgFOi+IiYKG/LXCPIG9F0kU/+VtWyDm38sg3L+/vEbfviz+MvHRblO+tIKDC/xYELALyLlxZ3Xdntz12T4euWS4oKIpgW1ANVThrvg7ZHc9NRZEmk5Ce83vtD17xw5/nrzpzR9LohOKIiO3tl6/slG93x5Sajh0KbLcx60nskO+NTr3AtQIgunhHl6rg6LP3R1W22xFrIodssxk4oeUR+UPq29OTkouTM9JfBbYDYQCeVM94YIQn1fOvtJS0AzOSPamekUBFWkraGk+qpw1wM5DhSfUsAa4BvgZycdaXPeg70LSUtD2eVM8rxwqA6/OkesKACcAOnIC1M5CBE4aPwFkv1wfk4wS2YTizoWOATjjh7D6cAS/H/7pE46x/W+g/LxJndm7J8dQmIiIiIiIiIiIi33K7TKPNXj3S9R+aOoDbXl+B4fj+Mu+5Lzcf+DihkWbANvXzFWkqrkAXICLNW+b0KRYnNCs9FfdLzkivxlkTdccRDonFWYO29vh8YCuwu8bljlzftqcpDoko8uzfUfnGJw/vwwkEy3ECy2CgMDkjPctta1Ym7vj05T9NfCLtmgeCsiqDzdf9s5kD5AH9drY3YVs6FkUP3LJiXVjJYh8wNT9k/uq2e2YG//ZG98C3xrm/euzv6169bE76H4CYsett8UvPejcCjFj++MAhaX+L+teErVtfPd9V8b17g4IBPKmeYZ5Uz/AjPG+LE1YW+jddCtziSfVE4gTI+UCV/zpdPKmeEJxWw+f4jy8BNvk/Fwbn/3eTlpKWA/wjLSVtef17Hm9Q62dx2lH7cELat4CXcdbRzUtLSZuHE7YOB3LTUtJWA6tx1gteDaSnpaRV4rSNrsQJk8P954ThhLT7OEVfbyIiIiIiIiIiInIwr8+yaMt+PliVxaIt+/H6jhzD1rYg7hR7/C2Ra+UUVnDb6yuYtTb7hK8h0pKpDbKItCiJ02a2A7yZ06fk192enpSc7DUm5sNeY1ZfsWV+F5zZwOnJGelF6UnJHXBCxvzkjPSa9KRkF3AR0OU/403o2h6m86WLbdjoDXYGTlibnBvDWJ/BtC8OHrK9Y+d9f7y28vOY4twJY9bb/Z3z7I7EPfzlvG/Si/33dgMh/jV7SU9KTgI6/Pgu93osiT+Y7Vvxg/fX+zypnp8A7rSUtL97Uj2dcMLWz3HC5IuAFWkpads9qZ74hH32dx0LrbdtEfc+/fS6mvSk5CuBTrfe4f44L8Z8F6d18AU4a/L+IC0lLfd4P5eeVM8goAvwRVpKWo0n1dMeGAssSktJy/akejoD3rSUtEPW4T3MtQbgBKy7gEk4YfEXwECctXL34MyuLcZpl3yzf5/BCZpfxVnzFpzQV4OTHI2+PkREWib1HxON4SIiLZPGcDkttMY2yCdi1trsQ9Z+bcjM17otiDP3lfH05xuBhn8DaHDWmP36/glqXSyBpjbIIiJHkzl9ypFaMe//ptPAiDf7ne++Ysv8XTihYHHtvvxIhs0fZLpdk+pZmZaR7nt9SnJMXAmXDNtsbVCN3R9dRuXqniZyyDYbCaR3KCIBGLEiMciX1bFqcI+coKJ13U3s1k42+8pFxAB3pycl/wmgNJRrMzuanGSYBZCckZ4BZLyYlBy+truZ/Mlw872/pHqm48xCdXlSPUFAj8777aiceAb7XGYtMLJtoZ38wO0Dg0POMNnRFXb0pd/4VnbdT7wn1ZN/u8f0DKuic3kw0Tiziwf630KBOJwZqgfxpHpGAMOAN9NS0grSk5KH44S7M5Iz0mtw2hJ3wZnlWgNE4cxAjvJfYjxO++u3j/aaeFI9CTgza9NwxpV4/zVqr2399/oeTmi7C7iUKjsYnw0BU04wcbjNaziziGvbKIuIiIiIiIiIiMgpMmttNre9vuKQgLV25uvfbxx2xMC2fgvi/p2iDgl9j8YC2YUVLNmWp1bG0uoorBWR08Lky58qAHq2LS/sgxM85taueQu4l/YzfTd1NqE47ZX3rejj2tB9j2/lsC3su/QbXO+MdfXMiaf9kG02F0gA0iuCGJgTV15R4a7yJu72VpWGula5fcaC3YQTgo4EMtZ3ZeCiZM58JNWz+cVnaorfOccV+eRf1239w7WuIAt998YaF866rMlAIuB++c81X5WFYr8cbDqE1tDlf2NcC4Zttlf23W3bburM+qhyPumdw5aUXwRdCMT/3yRXQc9cusSV0rM8nGhgCM7M1e04rZ4BSE9KHuc1tLnhPncuLhOH02rY3nHvwB+c28N4Bm23G3DaQdcAXwLBaSlpFQBpKWnbPKmeV/1tigEW4LQ8PpYSnNbFBThrzwb739r57xPqfxsI9MAJbttiCcOHxYUXSz+c9tYZ/ppFRERERERERETkFPH6LI98tP6wM2EtzlTDRz5az8QBnRo083XSoAQmDuh0YLbtpj3FPPfllmOel1vcsHBX5HSisFZETguZ06dUJU6bue3+Za93BAbgBIgAJGekV/3hoYEfretu2uOfsfnnZ9atSk9KzsMJD01MmV0YXkkBzqzOQcC6/dHMjKpiShDeHSaCXRcv9bXruYf2FUF8HFbDSpy2v1ck5hI2Y5RriPHZ6TOHm/LCCIo8qZ7p9HIVAUsB232Pb3ROvPleVYjZEVJtZy7pZ4LPSbdLJ6600ZXBTIio8IV9NMo80TuH4A3dXCveeqzmJqB9aJXdWBliap74lzctpIpEA/bOO4JmAj/AaZv8Yb1PRUJ2PD2Aa4FVaSlpj3tSPVEl4YxK62HsoO32P7Xtmv1r1no9qZ7RQGlaSlpanaCWtJS0I60bfJC0lLRiYD6AJ9VTCnyNEyK7gBFAT2AwzkznmUBfIJMQasDkYcw8nBbIacD+tJS0mobcV0RERERERERERBrHkm15R50FeyIzX+vOtl20ZX+DwtoO0Se+9q1IS6WwVkROG5nTp+SmJ90bhxMUZtXd9/oj6+q2Ra7VBael8OypS+xYnJmvBY9f5Ypf3tfV/63Har5uW0J1SDVjNiXQd28sG6uCGbyqJ7cmZ/FwbBnbCiPos7kzE8Irbb51uUrX9DSV4VUs+sEsb8jMUa64PfHm9aAa++vQasaN2Gjdq3qx5Ief+a7b1p6+t93h3lQZxM4ffuZrv6q3ubIw0oS+MMVVCoRndOXtVye4hleGmGFAzn0/cLfvttdOyI01g4EPgSeBBE+qZxJO+LnprcdqvgLWzxzpmhNdRsz4NK83/bHkkLSM9JKz/j7o2YwujH3vnKA2aU6rYQA8qZ4Q4Eog3JPqeSAtJa2kzr4wnKB1c1pKWrUn1ROKE77uBfalpaTtr3Ps2bEl9szJ6XbnZ8NMtNdtRgO9cNozTwWGAluAeTj3fxdjioEyYCOQjdM+WQtSiIiIiIiIiIiInGINndF6ojNfR/ZsQ0JsGDmFFYedvVu7Zu3Inm1O6PoiLZnCWhE53WwCXMkZ6Q1p37sMCEnOSK9MT0pei9NOuGNetCnqmG8jgMKwaj4BruqXjbt3Dm9/MZTy6iDTOXKLPQfY9fvvuoO67rNBvXLsghX9qMqLMZde8o1vS1oif42osJ1+OMu35r2zXVuKw83WxD2+t3rnYFzWTgz1mvgabH8MSZ8MM9lVwXwcUcEZxtq88jDzwW9vCvod1k6MKPd17rLXLtjU1dW9IIIOXhftIsptvMvyp5IIcwXOH7XtB+yr57mSBuy0Qwsj+fL+t73flIeYy147z3XFH+HNkghTDIQDkZ5UTzjgTUtJq0pLSavypHpmAx2Azp5UDzgh6x6c9WqHAjWeVE8W8Kj/8UfAPk+q55M6ge0lnfMY0X2Pnd+20PTLbUM4TmvknjgtoCtx2i77cMLaeUB7IBKoANw4rZMjcdopi4iIiIiIiIiIyCnS0BmtJzrz1e0yPDR1ALe+vuKQfbWzNx6aOqBBLZZFTjcKa0XktOJfp7YhQS3JGenVOIEkyRnpOThh7drHk5IjgcuAs5Mz0uemJyXfawBPevo+D6xNT0rujLPuatH2jmZep3xbOnKTnff2OC4auM2aLvttclpP4/UZiorDiKwKYlRejMnKaeteDew2PvYba0dUhJmfuHx0KY6wkTVu+saUW198KYt2t6VbRTBJFgitxF0WYroDfctDoFOe3Tduve0YXsVN/5jsXokTOJ8JXPvxcBOa2d5mp/cwAzO6uFcM3Gl3Le9tQj5K9UzEWavXBVyFszbsPOANT6onCGdmbj4wBYjDCUyLgadw1tjt7j+/DU6w+pX/Gu1wgmKAZ3e243tvjHddURhJb2AXThjbCWfm7CacNXbjgXL/9fenpaT5al8PT6onD61XKyIiIiIiIiIicsqdqpmvcRHBFJRVH7QtNiKY6Vd6mDQo4aSuLdJSKawVkVbBk+qJAyrTUtLKG3B4OU7L3hyA5Iz0fXV3Jmek7wZ2A6Q5LYFnAdyX6nn/ykW++M55dHhxsuu9vBhz7faO9jyMKXUO5VLgy9V9XBaIxGt3WUNFcYTZWR1MUmy53VIcxsr73vZdvKyPaTtzlOsv+XFmSD6cBeRXhhBWHuzL67fLhn86zDUAZ8bqqzjBaSkQXx1kOnQoIC+7nblncZL5BieMvgYnOO2G03a4GMi46E+DOgwu9P0qo5tJ7LHHPLepmwnGWcN3CU64GoEzy/Ys/8ff4MyA3Q38Fog/4x+D8sOr2UiU+Tq61E7YF8NgMG6cwHcJTnC+EtiJM6t2PU5b5QMhba0GvjYiIiIiIiIiIiLSyGpnvt72+goMHBTYNsbM11lrs7nt9RWHDYIL64W3Iq2NsfZw/zRERE4fnlRPMDAWKEpLSVt2vOenJyW3x2mtvKcBx3YGIpMz0jd5Uj2dcWayVgHvAINwZsIGAcm3zfBODa22rqevCFqKE2q6jc+a5/7ujZo5wjXm45GuN/3H3uY/bwJQFV1mtxRHmLNxws8JwHnABcFVNqhDoe27L8ZYl2V4eSghGFP7nU4FToCa638cOXSj7/MLV/m6Lu5v2rYpIvezM10FpRGmHJ/dm7TDttkfjW9vW1cYMAqnRbELWIXTzvhewBVa7gsPraK4ONosT9hv23kNcXvizH6CzIvAWuBaYCHwsb8Gm5aSpoFHTpa+hkREWib1MxON4SIiLZPGcDktGGNigMLCwkJiYmICXU7AeX2WJdvyyC2uoEO0M2PW7TLMWpvNIx+tJ7vw27VpE2LDeGjqgBOe+er1WcY8Puega9bXJjKYBy8ZSKeYb2sRCZBT/sWnsFZEWgVPqicRKElLSdt3rGPrS09KvgIITs5If6sxa0pPSj7ff91Z9ff515WtAK4A+gEvAnfiBKc7cVoKlwOzgV/gBLcvAHe3LbKTztxok1f2NmF74w04gW8VUGR8Nh+f7WFdhjYlrE3a7ovMiWNLeI3psqErFcFesn3QA8uAGjfWG+JaCnhw1rutxgmVi3Fm9dbgtTFBNeyrCTV/A2pcPrvF5zLzcNar7QCMBtLSUtK2NubnTlo9ffMiItIy6bctojFcRKRl0hgupwWFtd86ViB7pCD3RC3asp/rXvymwcefbDgscpIU1oqINDfpScldcWbW7jjV9/akeiJw1oadghO67gS2pKWkpfn3u3HW183GWVf2x7EldtDIjbbL6p6mODfe9PGfXwEsiij3xVcbzqgOMtW48Lm9uLzB5hsgovseX+zATBu9tA8x++JMPODF7ZqL0z65O07gmw3EACH+WsJx2h3PAT7BWZN2S+3sWU+qJxZnRrMGG2lM+noSEWmZ9Ite0RguItIyaQyX04LCWseR2hHX/kP/+43DGj0k/WBVFne/sarBxzdlLSINoLBWREQO5g9kU4DstJS0T45xbD+cdWl7ABuA83Fmwn4NVBmfvSGsikvLQ4nonmNHDNhh2dyFv2/u6nbHF/nObF9AxO42+ErDTFdrKMNtMoB1OOvidsEJaYtxQtv/An2A3sBm4H9pKWmlTfE5EKlH37yIiLRM+kWvaAwXEWmZNIbLaUFh7bHbERugU2wYX98/oVHbEB/vzNqmrEWkAU75F1zQqb6hiIgcn7SUNC/wcv3tnlSPCxgCbEtLSSvwby4AQtNS0hb4H79Z75xXysN4C5jcvsh6+2XbuHHr+eBX3yc/P8YVmh/DdpwQth3OersX4bRb/gtwLpAAlAAfpaWkLfWkepYBiYDbf5yIiIiIiIiIiIg0Q0u25R113VgLZBdWsGRbHmf1btto9x3Zsw0JsWHkFFY0+K/3mqoWkeZIYa2ISMvVDSdMTQNm+rdNBWI9qZ5n/CHvQfzbioG3fjxt4NtBPpMwde76rF+lenoBS4H0tJS07PSk5PD/jnO9/t45rq8BFxAHdAJS01LSVte5XjmQ3oTPUURERERERERERBpBbvGRg9oTOa6h3C7DQ1MHcNvrKzAcX7uVxq5FpDlSWCsi0nLtBGYDW+ts+wqIOFxQW8uT6ukKxJDsmvBNMm3/nOp53X+NPkBBelJyJHD2dfN8ub/7x7rP/OfEAPlARhM9FxEREREREREREWlCHaLDGvW44zFpUAJ/v3EYj3y0/qize09FLSLNjcJaEZEWKi0lzQcsr7dtcwNOnQi0Aapx1rctAoKBUUCv5Iz0F9KTkhfhtFSuvW4RsKxxKhcREREREREREZFT7VjtiGvXiR3Zs02T3H/SoAQmDujEkm155BSW8+jMdPJLqwJSi0hzorBWRKT1+RwnnG0H9McJbKuBLKAQIDkjfUvAqhMREREREREREZFGd7R2xMb//qGpA3C7zGHObrwaategDQ9xB7QWkebCFegCRETk1EpLSduZlpK2Fajxv9m0lDQLvAHMCGhxIiIiIiIiIiIi0mRq2xF3ij24vXCn2DD+fuMwJg1KaJW1iASSsfZ4lnIWERERCTh98yIi0jLpT+JFY7iISMukMVxOC8aYGKCwsLCQmJiYQJcTcF6fZcm2PHKLK+gQ7bQbDtQs1uZUiwgBGPcU1oqIiEhLo29eRERaJv22RTSGi4i0TBrD5bSgsFZEGuiUj3tqgywi0gp4Uj0xnlRPj0DXISIiIiIiIiIicqKMMaHGmFXGGGuMGRroekREGoPCWhGR1uFc4FJPqicuwHWIiIiIiIiIiIicqCeA3YEuQkSkMSmsFRFpHZYD3wCFgS5ERERERERERETkeBljJgMXAvcGuhYRkcakNWtFRESkpdE3LyIiLZPWuxON4SIiLZPGcAk4Y0xHnMkIlwP7gG3AGdbaVcdxDa1ZKyINccrHvaBTfUMREREREREREREREZGGMMYY4BXgBWvtMmNMYgPPCwVC62yKbvzqREROntogi4iIiIiIiIiIiIjIKWWMmW6Mscd4SwJ+ihO0Pnact3gAZ0mw2rddjfsMREQah9ogi4iISEujb15ERFomtVAUjeEiIi2TxnBpEsaY9kDbYxy2FXgLmMrB30u4AS/wb2ttyhGuf7iZtbvUBllEjuGUj3sKa0VERKSl0TcvIiItk37RKxrDRURaJo3hElDGmO5A3XS1M/ApcBWw2FrboBmzWrNWRBpIa9aKiIiIiIiIiIiIiIgAWGt31H1sjCnxf7iloUGtiEhzpjVrRUREREREREREREREREQCQDNrRURERERERERERESkRbDWZqL23CJyGtHMWhERERERERERERERERGRAFBYKyIiIiIiIiIiIiIiIiISAAprRUREREREREREREREREQCQGGtiIiIiIiIiIiIiIiIiEgAKKwVEREREREREREREREREQkAhbUiIiIiIiIiIiIiIiIiIgGgsFZEREREREREREREREREJAAU1oqIiIiIiIiIiIiIiIiIBIDCWhERERERERERERERERGRAFBYKyIiIiIiIiIiIiIiIiISAAprRUREREREREREREREREQCQGGtiIiIiIiIiIiIiIiIiEgAKKwVEREREREREREREREREQkAhbUiIiIiIiIiIiIiIiIiIgGgsFZEREREREREREREREREJAAU1oqIiIiIiIiIiIiIiIiIBEBQoAsQERERERERERERERE5FYqKigJdgog0Y7GxsTFAsbXWnqp7mlN4LxERERERERERERERkVPOGNMF2BXoOkSkRYi11p6yv+xQWCsiIiIiIiIiIiIiIqc1Y4wBOgPFTXSLaJwwuGsT3iMQTtfnBXpuLdWpeG6ndGat2iCLiIiIiIiIiIiIiMhpzR+8ZDXV9Z0sGHBCntOm1/Lp+rxAz62lOh2fmyvQBYiIiIiIiIiIiIiIiIiItEYKa0VEREREREREREREREREAkBhrYiIiIiIiIiIiIiIyMmpBB7xvz+dnK7PC/TcWqrT7rmZU7g+roiIiIiIiIiIiIiIiIiI+GlmrYiIiIiIiIiIiIiIiIhIACisFREREREREREREREREREJAIW1IiIiIiIiIiIiIiIiIiIBoLBWRERERERERERERERERCQAFNaKiIiIiIiIiIiIiIg0MmNMqDFmlTHGGmOGBrqek2WMSTTG/NMYs80YU26M2WKMecQYExLo2k6EMeYOY0ymMabCGLPYGDMy0DWdLGPMA8aYpcaYYmNMrjHmfWNM/0DX1diMMdP8/66eDnQtjUFhrYiIiIiIiIiIiIiISON7Atgd6CIaURJOrnQLMBD4OXAr8MdAFnUijDHXAn8GHgGGAauBT40xHQJa2MkbDzwPjAYmAsHAZ8aYyIBW1YiMMSNwvgbXBLqWxmKstYGuQURERERERERERERE5LRhjJmMEwZ+B1gHnGGtXRXQopqAMeaXwG3W2l6BruV4GGMWA0uttXf6H7uAncBfrbXTA1pcIzLGtAdygfHW2nmBrudkGWOigBXA7cBvgFXW2p8FtKhGoJm1IiIiIiIiIiIiIiIijcQY0xF4EbgJKAtwOU0tFsgLdBHHw9+2+Uzg89pt1lqf//FZgaqricT637eo1+gongdmWms/P+aRLUhQoAsQERERERERERERERE5HRhjDPAK8IK1dpkxJjGwFTUdY0wf4KfAvYGu5Ti1A9zAnnrb9+C0ej4t+GcLPw0ssNauDXA5J80Y812cltUjAl1LY9PMWhERERERERERERERkaMwxkw3xthjvCXhhJfRwGMBLrnBjuO51T2nCzAL+J+19sXAVC7H8DwwCPhuoAs5WcaYbsAzwA3W2opA19PYtGatiIiIiIiIiIiIiIjIUfjX/mx7jMO2Am8BU4G64Ysb8AL/ttamNE2FJ66hz81aW+U/vjMwF/gGuNnfQrjF8LdBLgOusta+X2d7KhBnrb0sULU1FmPMc8BlwDhr7bZA13OyjDGXA+/h/Duq5cb5d+YDQq213sOc2iIorBUREREREREREREREWkExpjuQEydTZ2BT4GrgMXW2l0BKayR+GfUfgksB25sqQGZMWYxsMRa+1P/YxewA3jOWjs9oMWdBH8b7r8CVwDnWms3BbikRmGMiQZ61Nv8LyADeLylt3nWmrUiIiIiIiIiIiIiIiKNwFq7o+5jY0yJ/8Mtp0lQOxfYjrNObXsnGwRrbU7gKjshfwZSjTHLgCXAz4BInACwJXseuB5nVm2xMaaTf3uhtbY8cGWdHGttMXBQIGuMKQX2t/SgFhTWioiIiIiIiIiIiIiIyLFNBPr43+oHz+bUl3PirLVv+ts//w7oBKwCJllr9wS0sJN3m//93Hrbvw+8ckorkQZTG2QRERERERERERERERERkQBwBboAEREREREREREREREREZHWSGGtiIiIiIiIiIiIiIiIiEgAKKwVEREREREREREREREREQkAhbUiIiIiIiIiIiIiIiIiIgGgsFZEREREREREREREREREJAAU1oqIiIiIiIiIiIiIiIiIBIDCWhERERERERERERERERGRAFBYKyIiIiIiIiIiIiIiIiISAAprRUREREREREREREREWhFjjDXGXF7ncZIx5htjTIUxZtWRtjU3xpibjTEFx3lOpjHmZ01TUfNV/zWX5kNhrYiIiIiIiIiIiIiISAtnjHnFH8hZY0y1MWaPMWa2MeYHxpj6eVAC8Emdx48ApUB/4PyjbAuYI4SsbwL9AlBOS1T/NZdmQmGtiIiIiIiIiIiIiIjI6WEWTiiXCEwGvgSeAWYYY4JqD7LW5lhrK+uc1xv42lq73Vq7/yjbjosxJuREzmsoa225tTa3Ke9xKhhH0LGPPHGHec2lmVBYKyIiIiIiIiIiIiIicnqo9IdyWdbaFdbaPwKX4QS3N9ceVLclrjHGAmcCv/Vvf/hw2/zHdjPGvGWMKTDG5BljPjDGJNa57ivGmPeNMb82xuwGNhznefcaY7KNMfuNMc8bY4L9++cCPYC/1M4e9m8/qA2yMaa3/9p7jDElxpilxpgLjveTaIz5kTEm3d8COsMYc3udfS8bY9YYY0L9j0OMMSuNMa/6Hyf6a/yuMWah/xprjTHj61zjXP8xk40xy4FKYIwxxmWMecAYs80YU26MWW2MuarOefHGmH8bY/b6928yxny/Th3P+T9/FcaY7caYBw73mvsfe4wxc/zX2W+M+YcxJqqhr4k0HoW1IiIiIiIiIiIiIiIipylr7RxgNXDlEQ5JANYBf/J//NThtvlDuk+BYmAscA5QAsyqN4P2fJzWyROBS47jvPNwZvOeB6TghMs3+/ddCewCfuuvJ+EIzyUK+Nhfwxk4M40/MsZ0P8LxhzDG3AD8Dvg1kAz8CnjUGJPiP+QuIBKY7n/8ByAOuLPepZ7E+fydASzy19G23jHTgWn++6wBHgC+B9wKDAT+ArxeJ+h9FBiAE74nA7cB++rUdSlwDc7n/wYg8wjPMRLnNckHRgBXAxcAz9U79GiviTSSJp1SLSIiIiIiIiIiIiIiIgGXAQw+3A5rbY4xpgYosdbm+DeX1N9mjLkRZxLgj6y1tTNbvw8UAOcCn/nPLfUfU3Wc5+UDd1prvUCGMWYmTuj6orU2zxjjBYrr1Hi457IaJ5iu9aAx5gqcELN+EHkkjwC/sNa+63+8zRgzALgFSLXWlvif01fGmGLgZ8B51tqietd5zlr7jv/53gZMAn4IPFHnmN9aa2f7jwnFCYYvsNYu8u/faowZ47/3V0B3YKW1dpl/f2ada3UHNuG0rrbA9qM8x+uBMOB71tpS//3vxAmU77fW7vEfd8TX5CjXluOksFZEREREREREREREROT0ZgB7ktcYAvQBio0xdbeH4cy+rJVWG9Qe53nr/KFgrWzAczwF+tv4PgxMwZl9GwSE4wSZDTk/0l/TP40xdQPJIKCw9oG1dpEx5ingQeBxa+3Xh7ncojrH1xhjluHMhq1rWZ2P+wARwOx6n6cQYKX/478D7xhjhuGE3O9baxf6970CzAY2GGNmATOstZ9xeMnA6tqg1m8BTqjeH6gNa0/6NZFjU1grIiIiIiIiIiIiIiJyeksGtp3kNaKA5TjtdevbW+fj0nr7Gnpedb19luNfzvMpnPbL9wKbgXLgbZzAsyFq12z9MbC43r4DoaUxxoXTztmLE7KeqLqfq9p7TwGy6h1XCWCt/cQY0wO4GOd5fmGMed5ae6+1doUxpidOi+QLgLeMMZ9ba6/ixDXGayLHoLBWRERERERERERERETkNGWMmYAzG/IvJ3mpFcC1QO5hWv42xXn1VQHuYxxzDvCKtfY9ODDTNrGhN7DW7jHG7AZ6WWv/fZRDfwkkAeOBT40x37fW/qveMaOBef46goAzOXor5vU4oWx3a+1XR6lxL5AKpBpj5uOsjXuvf18R8CbwpjHmbZx1gdtYa/PqXSYduNkYE1lndu05gA/YcJQapQko/RYRERERERERERERETk9hBpjOhljuhhjhhljfgV8AMwAXj3Ja/8b2Ad8YIwZa4zpaYw51xjzrDGmaxOcV18mMM7/3Nod4ZhNwJXGmKHGmCHAfzj+LOwh4AFjzF3GmH7GGI8x5vvGmHsAjDFnAL/DWYN3AXAP8Iwxple969xhjLnCGJMEPA/EAy8f6abW2mKcmcF/McakGGN6+1/DnxpjUvz3/p0x5jJjTB9jzEDgEpzgFWPMPcaY64wxScaYfsDVQA7O2sD1/RuowAl8BxljzgP+CrxWZ71aOUUU1oqIiIiIiIiIiIiIiJweJuGsK5oJzALOA+4CLqu39uhxs9aWAeOAHcC7OCHhP3HWnj3ijNkTPe8wfoszS3YLB7dPruseIB9YCHwEfIozs7fBrLUvAT8Cvg+kAV8BNwPbjDFhwOs4s3c/8h//D+BL4DVjTN2Zv9P8b6uBMcCl1tp9x7j9g8CjwAM4n6dZOG2Ra1tYVwGPAWtwZu16ge/69xUD9+Gsg7sU53N1sbXWd5jnWAZcBLTxH/s28AVw5zHqkyZgrD3Z9aRFRERERERERERERERExBiTiBOunmGtXRXYaqQl0MxaEREREREREREREREREZEAUFgrIiIiIiIiIiIiIiIiIhIAaoMsIiIiIiIiIiIiIiIiIhIAmlkrIiIiIiIiIiIiIiIiIhIACmtFRERERERERERERETkpBlj5hpjng50HQ1hjLnZGFMQ6DpEFNaKiIiIiIiIiIiIiIhIs+QPVe1h3n5U55gQY8x9xpjVxpgyY8w+Y8wCY8z3jTHBR7j0m0C/RqjPGGN+YoxZbIwpMcYUGGOWGWN+ZoyJ8B/zcJ26a4wxmcaYvxhjovz7E/37vMaYLvWun+A/xxpjEk+2Xml+ggJdgIiIiIiIiIiIiIiIiMhRFAH9620rBCeoBT4FhgAPAgv8x48G7gVWAqvqX9BaWw6UN0JtrwFXAr8H7gT2+mv5GZAJvO8/bh1wAU42dw7wMhAB3FLnWlnA94DH6mxL8W/v3gi1SjOkmbUiIiIiIiIiIiIiIiLS6Iwx8caYV40x+f4Zr58YY/rWO+bHxpid/v3vGWPuOUx7Ymutzan3Vhu0/gwYB5xvrX3eWrvKWrvVWvsfYBSw6Qi1HdQG2T/7dZUx5ib/zNdCY8wbxpjoozy/a4AbgOustX+01i611mZaaz8AJgBf1jm8xl/3Lmvtm8C/gUvrXTIV+H69bd/3b5fTlMJaERERERERERERERERaQqvAMNxQsmzAAN8XNua2BhzDvAC8AwwFJgN/Po473ED8Lm1dmX9Hdbaamtt6XFcqzdwOXCJ/208MO0Y997gD2fr39taawuPcm45EFJv24dAvDFmDID/fTzwUUOfgLQ8CmtFRERERERERERERESkUfln0F4K/MhaO99auxon3OyCE4gC/BT4xFr7lLV2o7X2b8Anh7lcrH892Nq3nDr7+gIZjVS2C7jZWrvWWjsfp8Xx+Uc5vi+w4XhvYow5E7gemFNvVzXwOvAD/+Mf+B9XH+89pOXQmrUiIiIiIiIiIiIiIiLS2JKBGmBx7QZr7X5jzAb/PnDWoX2v3nlLcGa11lUMDKvz2FfnY9Mo1ToyrbXFdR5nAx2Ocvzx3NtjjCkB3DgzamfirHFb38vAQmPMr4CrcWYkK887jbW6F9cYY4BooNhaawNdj4iIiDScxnEREZGWSWO4iIiIiJwkn7V28xH2bQSSGuk+9WewWo7epfZ47r0BZ6ZxDbDbWlt1uIOstWnGmAzgv0C6tXatMWZoA+8hLVCrC2txfjgsLCw8WptwERFpxhrzL+Wk5dE4LiLScmkMb92igcJVK1ewdcuRfscmIiLNVa/efTSOi8iJSMfJoUYBCwGMMW1xZtOu9x+zARhR77z6j4/lP8AfjTFn1F+31r82bshxrlt7vPd+wxhzWf11a/1/sBhTZ93aqqMEzvW9DPwNuK3xSpXmSmvWioiIiIiIiIiIiIiISKOy1m4CPgBeNMaMMcYMwVl/Ncu/HeCvwMXGmHuMMX2NMbcAk3FmtDbU08AC4AtjzB3GmCHGmF7GmGuAb3DWlW0qbwFvAv81xvzKGDPcGNPDGHMJ8Dlw3gle90WgPfBSI9UpzZjCWhEREREREREREREREWkK3weWAzOARTgddy621lYDWGsXALcC9wCrgUnAX4CKht7AWlsJTASeAG7BCWiXAncBzwJrG+m5HO7eFrgep/7Lga+ANcDDOIH0pyd43Rpr7T5rbU3jVCrNmWltS8UYY2Lwt0+MiYkJdDkNYq2lxtZQ7a2m2ldNja+GGl8NXuvF6/M6760Xn/VhsdR9TY0xGAxu48btcuMyLoJMEMHuYIJdzluIO4QgV2vsiC0iLZRaL7ViLXEcbw7Ka8qp8lYRGxob6FJEpHXTGN6K1Y7hq1auIDo6OtDliIjIcVIbZBE5lYwxLwJJ1tqxga5F5FRQQtdIrLWU15RTWl166FtNKWXVZZTXlFNWXUZZTdmBx+U15ZR7y6moqaCipoJKb+W3770VVHmrqPJWYY9rxv/xCzJBhAaFEuYOIzwonIjgCCKDI4kIiiAqJIqo4CiiQ6KJCYkhNjT2wFubsDa0CWtDXGicAl8REZFmqKKmghs+voGs4ixev/h1+sY3ZecfEREREREREZHjY4y5F5gNlOK0QE4Bbg9oUSKnkNK1OvaV7yO/Ip+S6hKKq4opriqmpKqE4uribx9Xl1BSVeK8r/NxaXUpPus7ZbUGmSDcLveBGbNu48yaBTAYjDFYaw+EvF7rxefzUWNr8Pq8VPuqDwqAa2wNNdU1lFaf2BrbBkNcaBztItrRPrw97cLb0TGiI50iO9EpshMdIzrSOaoz0SH6C2oREZFT6e+r/86m/E0APLTwIV6b/BpulzvAVYmIiIiIiIiIHDASuA+IBrYCd1lrtVartBoKa+t4cMGDfJ319Uldw2CICo46MDM1MjjS+TjIeR8RFHFg5mp4UDjhQeGEBYUd+DjUHUqoO5SwoDBC3CGEukMJcYUQ4g5x2ha7gwkyQRhzcp1HrLV4rZcqbxXVvmpnZq/Xmd1bXlNOWU0Z5dXlB0Lp2uC6qKqIgsoCCisLKawsJL8yn/yKfCzW+bgy/8AvhA8nOjiazlGd6RLVhW7R3Q689YjtQaeITvrlsYhIC2WMmQY8Bjxjrf3ZUY67GngUSAQ2Afdbaz8+FTW2Ruv2ryN1XSoAIa4Q0val8Xr666QMTAlwZSIiIiIiIiIiDmvtNYGuQSSQFNbW0SasDfGh8USHRBMVEkV0cPSBj2vbANe+jwyOPHBMZEgkkUGRRIdEEx4UftJB6qlgjCHIBB1oXXwya9h5fV4KKgvYX7GfvWV72Vu+l71le9lTtsd5K91DTmkO+ZX5FFcXsyF/AxvyNxxynRBXCN1jupMYk0jP2J70iutFr1jnLSwo7ITrExGRpmWMGQHcAqw5xnFnA/8FHgBmANcD7xtjhllr1zZ5oa1Mtbea3y74LV7rZVLiJEYljOKRRY/w15V/5bxu59E9pnugSxQREREREREREWn1jLVNuxZqc2OMiQEKCwsLiYmJCXQ5rUpZdRnZpdlklWSxq3gXO4t3sqt4FzuKd7CzeCfVvurDnucyLrpFd6NvXF/6xPehf3x/+sf3p0t0lwOtn0WkVWn+fxHTihhjooAVOOuI/AZYdaSZtcaYN4FIa+0ldbZ94z/n1gbeT+N4A72w+gWeX/U8caFxfHD5B8SHxvPjz37M4pzFDO84nH9e9E+NoyJyqmkMb8Vqx/BVK1cQHa3lcUREWppevftoHBcREWkiCmulWfD6vGSXZpNZlMm2wm1sLdzK1oKtbCncQmFl4WHPiQiKoH+b/iS3SSa5bTLJbZLpHdf7wGxhETlt6QfEZsQYkwrkWWt/boyZy9HD2h3An621T9fZ9ghwubV2yBHOCQVC62yKBnZpHD+6zfmbuXrG1dT4apg+djpTek0BYGfxTr7z4XcorynnzI5nEhUcddzXHth2ILcOufWonUQqvZX8ZflfOLvz2YzrOu6En4eInHY0hrdiCmtFRFo2hbUiIiJNR6mWNAtul5uu0V3pGt2VMV3GHNhurWV/xX425W9y3go2sSFvA1sKtlBWU8bK3JWszF154PhQdyhJbZIY2HYgg9oNYlC7QfSI6aGZQyIiTcAY811gGDCigad0AvbU27bHv/1IHgAeOv7qWi+f9fHoN49S46thfNfxXNzz4gP7ukV34+5hdzN9yXSW71l+Qtf/atdX9IztyaSek454zMdbP+bf6f/m3U3v8sFlH5AQlXBC9xIRERERERERETndKayVZs0YQ7vwdrQLb8dZnc86sL3GV0NmYSbpeemk56WTkZdB+v50SqpLWL13Nav3rj5wbHRINIPaDmJw+8EMaT+Ewe0Hn9QavSIiAsaYbsAzwERrbUUT3uox4M91HkcDu5rwfi3eB5s/YEXuCsKDwvn1qF8fMgP2+qTrSYhMoKCy4LivvTJ3Je9vfp8nlj7BmC5jiAo5/Mzcr3Z9BUB5TTnTl0znmQnPHPe9REREREREREREWgOFtdIiBbmC6BPfhz7xfZjaeyrgzCTaUbSDdfvXsXbfWtbtX8f6/espripmUfYiFmUvOnB+YkwiQzsM5YwOZzC0w1B6xvQ8ajtHERE5xJlAB2BFnf8/3cA4Y8ydQKi11lvvnBygY71tHf3bD8taWwlU1j7W/9VHV1BRwJ+XO9n2bUNuO+yMVmMME7pPOKHrT+k1hRV7VrCjeAfPr3qe+0fef8gxVd4qFu5eeODxnJ1z+GrnV4zvNv6E7ikiIiIiIiIiInI605q1clqr9lWzOX8za/auYc2+Nazeu5rtRdsPOS4uNI4zOpzBsA7DGNZxGMltkwl2BQegYhFpAKV1zYAxJhroUW/zv4AM4HFr7drDnPMmEGGtnVpn20JgjbX21gbeV+P4UTy88GHe2fQOfeL68NbUt5pkLFu4eyG3zL4Fl3HxxpQ3SG6bfPD+rIXc8vkttA9vz5ReU3hl3St0ierCe5e9R3hQeKPXIyItisbwVkxr1oqItGxas1ZERKTpaGatnNaCXcEkt00muW0y13ItAPkV+azZu4aVuStZtXcVa/etpaCygC93fsmXO78EIDwonMHtB3NmxzMZ3nE4g9sPJtQdGsinIiLSrFhri4GDAlljTCmwvzaoNca8CmRZax/wH/IM8JUx5hfATOC7wHDgJ6es8NPYqtxVvLPpHQAeHP1gk/3R0dmdz2ZS4iRmZc7i99/8ntcufu2gteHnZc0DYFzXcdw25DZmZc4iqySLF9e8yF3D7mqSmkRERERERERERFoqhbXS6sSHxTO+2/gD7RirvdWsz1vPyj0rWZ67nJW5KymsLGRx9mIWZy8GIMQVwuD2gxnRaQQjOo1QeCsi0jDdAV/tA2vtQmPM9cDvgT8Cm4DLDzcL97gtT4Wv/3Lg4T9CfXwQYjlS/5AYC0+WuejmO/iPwxcGWZ6IgKqIthAcBkD/+P48Mf6J4wo/P9n2Cf/N+C+PnP0IPWN7Nvi8HUU7uG/efRRVFTkbKgqhoujIJxgXRLaHoBDyK/IBuLzP5QzrOKzB9zwRvxzxS+ZnzWfNvjW8s+kdru53NQDWWubunAs4YW1EcATTRk7jZ1/+jJfXvsyszFkNun6/+H48Oe5Jgt3qciEiIiIiIiIiIqc3tUEWqcdnfWwp2MKKPStYtmcZy/YsY1/5voOOCXGFMKTDEEZ2GsmohFEMajtIv1AWOXXUeqkVO+I4vuBZmP0gAOXGMKZ7V6pcR/9SGVdWzvN79h54XGEMl3ZNIDvo0L9lu2/Efdw04KYG1bi/fD+XvHcJJdUljOg0gn9e+M8GrbVrreW2z29jwe4FDbrP4bQJa8N7l71Hm7A2J3yNhvp3+r+ZvmQ68aHxzLhyBjEhMWwt2MplH1xGsCuYr7/7NRHBEVhr+dmXP2POzjnHdf17h99LysCUJqpeRAJEY3grpjbIIiItm9ogi4iINB2FtSLHYK0lsyiTZXuWsTR7KUv3LD0kvA0PCmdYh2GMShjFqIRRJLVJOqglpIg0Kv2A2IodcRwvyobCnQDM35/G7aufpmNoPE8NOnQp3KLqUu5Oe54a6+WFIT/nnLaDAPi/bR/x3Lb36eT18cSeXMzoO1jStjN/XflXokOimXnFTOLD4o9Z4yOLHuHtjW8fePz0uU9zfo/zj3nevF3zuOOLOwhyBfHMec8Q8+XjsGsJdB0JZ9xw6Ak1lfD5w1BdBuc+AL0n0D2m+ykJagFqfDVc9eFVbCncwk0DbuK+Effxr7X/4s/L/8w5nc/hhYkvHDi2ylvFhrwNeK33mNddtmcZz6x4hujgaGZcOeOUPR8ROSU0hrdiCmtFRFo2hbUiIiJNR22QRY7BGEPP2J70jO3J1f2uxlrLtqJtLM1eypKcJSzNWUp+ZT4Ldi84MBsqNjSWkZ1GMjphNKMTRtMtuluDZlWJiMgJiklw3oCF2c7642O6T2DooOsPe/h1Nfm8tv41nsj8gLcH30ReeR7/nPcpAPf0uIQzdrwAi/+F56fL+CzzMzbkb+D5Vc/zm9G/OWoZG/I28O6mdwGnDfC8XfN4atlTjO06lhB3yBHPq/ZV8+TSJwG4KfkmxpVXwpavwRUMFz8LbXsf/sTiPPjid7DwRTjzFgiJPGp9jSnIFcR9I+7jls9v4b/p/+Xqflfz1a6vAOe51xXiDsHT3tOg63raefgs8zPS89J5fuXzPHjWg41eu4iIiIiIiIiISHOhqX8ix8kYQ6/YXlybdC1/OvdPzL12Lu9c+g73jbiP8V3HExkcSWFlIbO3z+bRbx5lyntTmPTOJB5a+BCzts0iryIv0E9BROS0tnD3QgDO7nz2EY+5dcitxIfGs7VwK29teItnVjxDeU05Q9sPZdK5f4COg6CiAPfcx7l/5P0A/G/j/9iUv+mI17TW8vjSx/FZH5MSJ/HkuCdpH96eXSW7eG39a0et+Y2MN8gsyqRNWBt+PPBm+PRXzo5Rtxw5qAUYfQfEdYfi3bDgmaPeoymc3eVsxnUdR42t4XeLfseq3FXAoWHt8XC73Nw34j4A3t70NhvyNjRGqSIiIiIiIiIiIs2SwlqRk+QyLvrF9+OmATfx3PnPMf+783lt8mvcOfROhnccTpAriN2lu3l307v8ct4vGf/meK756Br+vPzPfJP9DZXeykA/BRGR00ZOaQ5bC7fiMi5GJYw64nExITHcecadADy74lk+2voRAPePvB/jDoJJjzkHLnuZEa4oLuh+AT7r44mlT3CkJSS+2PEFS3OWEuoO5edn/pyI4Ah+dubPAPjHmn8c0kK/Vn5FPn9f/XcAfnrGT4le/Sbs2wgR7WD8fUd/wsFhMPFR5+MFz0DBzqMf3wTuHX4vQSaIZXuW4bVe+sT1oWt015O65vBOw5nYYyI+6+PJpU8e8XMuIiIiIiIiIiLS0mnNWpEmVlZdxorcFXyz+xsWZS9iY/7Gg/aHucM4s+OZnNX5LM7ufDZ94vqoZbLI0ekfSCt2rHH8nY3v8PCihxnSfgivX/z6Ua9V46vh6o+uZnPBZgAu7X0pfxjzh28PeOMGyJgB7ZPYmTCAy4qXU41lYlBbIsyhK0ksqskn11bxk8E/4adn/BQAn/Vx48c3krYvjUFtB9E77tBZsttylrOmdBf9XZG8GXUG7oyZUFkIlzwNw79/7E+KtfDKFNi+ABKGOLOCj8UYOOMm6D766MflbYVl/4Kz74Ko9kc87PElj/N6uvP5/v6g73PP4Nvhq+lQkvvtQWFxcO40CGvY919ZJVlc+t6lVPmqmNhjIhFBEcc8JyokituG3EZsaOxRj5u1bRZfZ319xP1tw9ty+9DbCXWHNqhWEWkwjeGtmNasFRFp2bRmrYiISNNRWCtyiu0r38fi7MUs3L2QRbsXsbd870H7O4R3OBDcntX5LOLD4gNUqUizpR8QW7FjjeO/mPsLPtv+GbcNuY3bh95+zOt9k/0NP/7sx4QHhTPjihl0iOjw7c68bfD8SPBWAfB0fCz/jDt6CNjB6+Ojqz4lIubbmaWrcldx0yc3HbOWl7P3MKLC322h4yC4ZR643Mc8D4DsNfB/44Dj+L4uqhPctRJCjhCCWgsvnQ9Zy2HAZXDNq0e8VGFlIVPfm0p+ZT6vTX6NoRmfw5xHDz1w9B0w6Y8NLvHZFc/yYtqLDT4e4Op+V/Pbs357xP3bi7Zz2fuX4bXeo17np2f8lJ8M/slx3VtEjkljeCumsFZEpGVTWCsiItJ0FNaKBJC1ls0Fm1m0exELsxeyPGc5Fd6KA/sNhgFtB3B257MZ02UMnvYegl3BAaxYpFnQD4it2NHGca/Py7g3x1FUVeQEhh2GNuiaX2d9TXxYPAPbDjx05/ZFsHMxANXWy3tFGynxVR++tt0rGbdvB71H3A4XHhxUzt81n00Fh1nvdvWbkLuOXjE9Obf/d5xtLrcTjsZ1b1D9B2z5ErJXN+zYpS9B4U44/yEYe8/hj0n/CN688dvHP54DXc484iW3FmxlZ/FOxrf1wDNDoLIIhv8A4npA6V5Y9By4Q+CnKyCuW4PKrPZW897m9yipLjnmscVVxbyU9hJu4+b9y94nMTbxsMf98qtfMitzFoPbD+b87ucfsn93yW7e3PAmUcFRfHLlJ8SFxTWoVhFpEI3hrZjCWhGRlk1hrYiISNNRWCvSjFR6K1mxZwULdy9k4e6Fh7RMjgqOYnTCaM7pcg7ndD6HhKiEAFUqElD6AbEVO9o4vnrvam78+EaiQ6KZd+08glyHtipuUhs/g/9cDUFhTiAZ2+Xox2cthxcnAAZuWwAdDxMWN5XVb8B7t0BYLNy9GsLrdXHw1sDfz3LWzg2Lg4oC6DkeUj489rU/+w0s/Kt/dvB8cLmcWbqpUyFzPgy9AS7/W1M8K+744g7m7ZrHRYkX8dT4pw7Zv27/Or4747sYDP+b+j/6t+l/yDE+6+PaGdeSkZdByoAU7h1xb5PUKtJKaQxvxRTWioi0bAprRUREmo4r0AU0J4/OWM+kp+dx+7+X8+SnGby9fBcrd+RTXHH4GTQijS3UHcpZnc/iF8N/wTuXvsMXV3/B78/5PZMTJxMXGkdJdQmf7/icRxY9woXvXMhl71/GE0ufYOHuhVR6KwNdvohIQC3cvRCA0QmjT31QC9B3IvQ4B2oqnPVaj+XzR5z3Q757aoNaAM/V0GEgVBTC1385dP/q/zpBbXgbuHkmuIJh21fO7N2jKcyCxf9wPj7/ISeoBWeN3Ase/vbauemN9lTquuuMuzAYPs38lHX71x2y/9kVzwIwpdeUwwa1AC7j4u5hdwPw34z/kl2S3SS1ioiIiIiIiIiIAATgN5nNV3p2ERk5xWTkFB+yLyE2jD4doujfMZr+naJJ6hRD345RhAU3cC05kRPQIaIDl/W5jMv6XIbX52X9/vUs2L2ABVkLWLNvDVsLt7K1cCuvrX+NMHcYIzqNYEyXMYztMpZuMQ1rMSkicrpYmOWEtWd3PjswBdQGkv+cCCtfh7N+Cu37Hf7YLXOc8NMdAuc+cErLBJxWyxc8BP+5Bhb/H4y85duZwNXlMPcx5+Oxv4BOg2DED2HxC/D5w84MW9cR/t5v7mPgrXRC674TD97XdTgkT3XaK3/xKFz3n0Z/Wv3b9GdKrynM2DqDp5c/zYsXfrvebe168UGuIO4YesdRr3NO53MY3nE4y/Ys42+r/8aj5xxm/V0REREREREREZFGoDbIdezMK2Nzbglb9pawbV8pW/eWsmVvCbnFh5+x6HYZerePZGDnWAZ2jmFQl1gGdYklKlQZuDS9wspCFmcv5uusr1mQtYDc8tyD9veI6cHYLmMZ02UMwzsNJ9QdGqBKRRqdWi+1Ykcax4uqihj3xji81stn3/kssG3i/3s9bJgJyZfCta8dut/ngxfPg+xVMOo2mNyAWbhNwVr412TYsQiGpcClzqxTFv7VaWUc0xV+uhyCw6BkLzw7FKpK4OpXYOAVh15v70b42yiwPvjhbOg28sSOOUm7incx9f2p1Phq+MfEf3BW57Ow1nL9zOtZu38t1yddzwOjjh2Q17bVdhkX7176Lr3jejd6rSKtkMbwVkxtkEVEWja1QRYREWk6CmsboLCsms17i9m4p4QNOcXO255i8kqrDnN96NM+iiHd4hjWPZ5hPeLo2yEat0vfz0jTsdayMX+jE9zuXsDKPSupsTUH9oe5wxiZMJKxXcYytutYukQdYx1FkeZN/6G2Ykcax2dvn809c++hZ2xPPry8AeuqNqXcdPj72U4gGdH20P3WB+X5EBINd6+CyHanvMQDdiyGly8EDES0cbZVFIKvBi57Hs648dtj5053Zs66QyD0ML9kr66A6lLoP+Xos2Y/uBNWvgaJY+HmGY36dGo9tvgx/pPxH0LdoUQEReDDR2FlIeFB4Xx85ce0C2/Y5/zuOXczZ+ccwoPCCXOHARARHMGT457E095z0LErc1cybd40ymvKT6jmiT0m8uBZD57QuSItiMbwVkxhrYhIy6awVkREpOkorD1B1lpyiytZt7uQtVlFrM0qJC2rkOzCikOOjQoNYliPeEb1bMPInm0Y3DWW0CC1T5amU1JVwuLsxczPms/8rPnklh0867ZXbC/GdhnLuK7jOKPDGQS7gwNUqcgJ0Q+IrdiRxvEnlj7Ba+tf44bkG5g2clrgCqw14+ew7OWjH3PBwzDm56eknKP6382w7r2Dt3UaDD+Z67RLrlVZDM+PhqJdR75WULhzXoekIx9TsBOeHuR8fN+2b0PiRrS/fD9XfHAF+ZX5B22/64y7+PHgHzf4OlsLtnLtjGup8B78/d2wDsN4ZdIrGOP8d2St5YaPbyBtX9pJ1f3a5NcY2mHoSV1DpJnTGN6KKawVEWnZFNaKiIg0HYW1jSy3uII1OwtZtbOAFTvyWbWzgLIq70HHhAa5GJ4Yz9m923FOn3YM6hxDkPsIa781B9ZCTSXUlPvfV/jfV4K3GnzV4K1yZuH4vM6b9Tozh2rPB2fasXH539zgDnJm57iCISgUgsKcVotB4RASCcERR14TTxqs7qzb+VnzWZW7Cq/99msyMjiSszuffWDWbUNnG4kEkH5AbMWONI5ba9lUsIlwd3jzWLPb54O8rc7YeDhBoRCf6IyNgeatgfxtzvhdKz7RGZPrqyyBwqOEtZHtIfIws4nre24E7NsI3/0vJF183CU3RHFV8UF/rBTiDqFrVNcDAWtD5Vfkk1eRd+CaP/z0h1T5qvjnhf9kZILTxvnrrK+57fPbCHOH8fJFLxMRHHFc93gx7UVmbp3JOZ3P4YWJLxzXuSItTDP4T08CRWGtiEjLprBWRESk6SisbWI1Xh8ZOcUsy8xjSWYeS7blsa/k4PbJMWFBjO3bnvH92jOuX3s6xR7ml6PHw1qoLoOKIqgscmbCVBb5Hxd/+1ZV7PzStarUWYOuqvbjUuf8qjLnfXU5EKCvk+BIp9ViWAyExkBYrDMDJ7yN8z6yHUR2gCj/W3QCBIcHptYWoqiqiIW7FzJ/13y+zvr6wC+gaw1sO5BxXccxvut4ktsm4zIKzKXZ0Q+IrdipHseliXx4F6xIhbPvggsfDXQ1x+UP3/yBNza8wYhOI3j5opex1nLjJzeyZu8abhpwE/eNuO+4r7mzeCdT35uK13r598X/ZnD7wU1QuUizoDG8FVNYKyLSsimsFRERaToKa08xay2bc0tYuGU/CzbvY9HW/RRXHDzrZmhCGJP6RHBeYhj9Yn2YykInaK0o/DZ0PeTjuscUOzNbm4RxwlB3iDMryB1aZ4ZskNMu0bj9710c/PsY68y2rZ156635dlZuTZV/xm7FyYfD4fEQ3Rliu0Bcd4jt5rxv0xPa9HICXwHAZ32s37+eebvmMW/XPNbtX3fQ/nbh7RjXdRzjuo7jrISzjnumkEgT0Q+IrVigx3FpJKv+C+/fCl1HwI8+D3Q1xyWnNIeL372Yal81/7roX1T5qrhl9i2EukOZ9Z1ZJ9yh4jdf/4YPtnzA2C5j+dsFf2vkqkWaDY3hrZjCWhGRlk1hrYiISNNRWNtYamezVvpnqFYWf/u+sqTODNfib2e6VhbjqyikvDifqtJCTGUR4b5SQs0RWiYeL+Pyz0aNgZDa2anREBLlvK99C4l0toVE+T+O/LYNcXC4/72/PbE7uOlbNlrrBLa1M34PzAougvICKM+Dsjwo2w+l+6A0F0pyoWSPE/YeS0RbaNMb2vWD9v3875Mgrkerb7u8t2wvX2d9zbxd81i4eyFlNWUH9oW4QhiRMIJzu57L+K7jSYhKCGCl0srpB8RWTGHtaSI/E54Z4vyh17SdENKy/hjo0UWP8tbGtxjVaRRVvipW5q7kxuQbuX/k/Sd8zR1FO5j6/lR81scbU95gYLuBjVixSLOhMbwVU1grItKyKawVERFpOgpr69q+EAp2+NsAl/tbAddtCexvEVxVdnDb4NqAthFbBfswlNhwioig2IZTRCSV7khi4tqS0KEjHdp3wIT5g9iwWAiNrdMq2P8+JLJ5rIV3qlgLFQVQlA3Fu5019Qp2QMFOKNgOeducYPdIQqKc0LbjQEgYDAlDocOAFvcL5MZS5a1i2Z5lzNs1j7k755JVknXQ/v7x/RnfbTzndTuPAW0HqF2ynEqt6D82qU9h7WnCWvhzMhRnQ8oM6Dk20BUdl+ySbC5+72Jq/GsSh7hC+OQ7n9AhosNJXfdX83/FR1s/4tyu5/LX8//aGKWKNDcaw1sxhbUiIi2bwloREZGmo7C2rv9eBxs+Ptk7+GeuRtWZxRrlhKeh/pmttY9rQ9WDPo4+MBO2wmtZsHkfH6fl8Hn6HgrLqw/cpXNsGFOHdObKYV3p30k/6DZYZbET2u7fDPs2wb4NsHcD7NvotGOuz7icwLbLMOhyJnQZDh2SnTbPrYi1lq2FW5m7cy7zds1j1d5V+KzvwP524e0Y39UJbkcljCIs6CTXXRY5Ov2A2IoprD2N/O/7sO5dOO/XMP7413kNtIcXPsw7m94B4Lqk6/jVqF+d9DUzCzO57IPL8Fkf53Y7F7dxvt8Y1G4QPxz0Q0xr+iM8OV3pi7gVU1grItKyKawVERFpOgpr65o7HXYu9rf99bcArm0HHBIBwZFO0FrbNjg4wv+43rYmaKVb7fWxYPM+PlqdzWfrciiu/LZV8qAuMXxnWFcuG9qFNpEhjX7vVsFbDfu3QO46yFkLOWsgezWU7j302NAY6Docup/lvHUd4bSJbkXyK/L5Outr5u6cy4LdCyitLj2wLzwonLM7n8253Zx2yfFh8YErVE5X+gGxFVNYexpZ8iJ8fC/0ngA3vRfoao5bVkkWU9+bisu4mHHFDDpFdmqU69bOrq0vdVIqwzoOa5R7iASQxvBWTGGtiEjLprBWRESk6SisbYEqqr3M3ZDLeyuzmJORS7XXeQ1D3C4uHNiR60Z256xebXG59D3USbHWac+YtQKylkPWMufjqpKDj3OHOoFt4hjoNd752B0cmJoDoMpbxbKcZXy580vm7ppLTmnOgX0u4+KMDmdwXrfzmNB9At2iuwWwUjmN6D+3ZsAYcxtwG5Do37QO+J219pMjHH8z8K96myuttcf11y6nwzgufjlp8MIY54/d7t8O7qBAV3Tc0ven4zIu+rfp32jXLKkq4fMdn1Pl7/gxd+dc5mfN59xu5/LXCWqNLC2exvBWTGGtiEjLprBWRESk6SisbeHySqv4cFUWb6/YxdqsogPbe7SN4IZR3blmeDfiIjTbttH4vLBnnTMDe8ciyFwAJTkHHxMS5QS3vc+HvhOhTc/A1BoA1loy8jL4cueXfLnzSzLyMg7a3ze+L+d3P58J3SaQ1CZJ7RzlROkLpxkwxkwFvMAmnNckBfglcIa1dt1hjr8ZeAaom2pZa+2e47zvaTWOt2o+LzzeEyoL4SdzofMZga6oWdpWuI3L3r8Mi+XDyz+kZ2zr+b5CTksaw1sxhbUiIi2bwloREZGmo7D2NLI2q5A3lu7g/ZW7KfG3SQ4NcnHZ0M5876xEBnWJDXCFpyFrIW8rZM6HbfNg61wo23/wMW37Qt8Lof8k6H52i5w5dKJ2l+x2gtsdX7JszzK81ntgX5eoLpzX7TzO734+Z3Q4A3crWwdYTop+QGymjDF5wC+ttf88zL6bgaettXEneY/TdhxvlV6/CjbPhknTYfRtga6m2bprzl18ufNLrup3FQ+d9VCgyxE5GRrDA8AYY4ErrLXvB7gOhbUiIi2YwloREZGmo7D2NFRWVcOHq3bz6qLtrM/+drbt6F5t+NGYXkxI6qAWyU3F53PWu936JWz+wpl96/t2fWHC46HvRZA0Bfpc4KyF3EoUVhYyb9c8vtjxBQuyFlDhrTiwr01YmwPB7aiEUYS4NRtcjkr/gTUzxhg3cDWQijOzdv1hjrkZeAnIAlzACuBXh5uFe4x7nfbjeKsy/0/wxe8g+VK49rVAV9NsrdizgpRZKYS4Qvj0qk9pF94u0CWJnCiN4U3EGPMKEGetvfww+04qrDXGtAEeAS4EugN7gfeBB621hcdxHYW1IiItmMJaERGRphPQsNYYkwn0OMyuv1lr7zjCOVcDj+KskbcJuN9a+/Fx3LPV/JLXWsvy7fmkLtrOx2nZeH3Oa92rXSQ/GtuLK4d1ISxYsxmbVEUhbPkSNn4KG2dBed63+4Ijod+FMOByZ+ZtKwpuy2vKWbh7IXN2zGHuzrkUVX37RwVRwVGM7zaeid0nck6XcwgLOq7lLKV10A+IzYQxxgMsAsKAEuD6I43JxpizgL7AGiAWuBcYBwy01u46yj1CgdA6m6KBXa1hHG8Vti+Ef02GyPZw7yZQe/zDstZy48c3smbfGm4ZfAt3nnFnoEsSOVH6R95EmjisHYQT1r4CrMf5Gf4FYI219qrjuI7CWhGRFkxhrYiISNMJdFjbHqibFg4CZgPnWWvnHub4s4F5wAPADOB64H5gmLV2bQPv2WrC2rp2F5STuiiT/yzeQXGFM9OzfXQoPxzTkxtGdSc6LDjAFbYC3hpnrdsNH0P6h1Cw49t9wZGQfAkMugp6nwfu1vN6VPuqWb5nOZ9v/5wvdnzBvvJ9B/aFB4UztstYJiZOZFyXcUQEt55AW45KPyA2E8aYEJwZNrHAVcCPgPGHm1l7mHODgXTgv9baB49y3MPAIX1fW9s4ftqqroDp3cBbBT9dAW17B7qiZmv29tncM/ceYkNj+ew7n2lMlJZKY3gTaUBY+2NgCnARTpeLX1hrP/Tv/y1wK+Cx1u73b5sJRADnW2t9h7nm1cDrQKS1tqb+/iPUqLBWRKQFU1grIiLSdJpVG2RjzNPAJUBfe5jCjDFv4vwweEmdbd8Aq6y1tzbwHq0yrK1VWlnDG0t38tL8rWQXOm1oo8OC+ME5PfnBOT2JjWg9IWFAWQu7V8L692Hd+1Cw/dt9EW2d0Hbo9ZAwpFXNMvJZH6v3rmb29tl8vv1zskuzD+wLdYcypssYJvaYyLndziUyODKAlUqAtZ5/FC2MMeZzYIu19pYGHv8/oMZae91RjtHM2tPdy5OcZQPiekBoA17TmM7wnRchLPbg7Rs+ga8ed/446mRFdYDvvAQRbU7+Wo3E6/My9f2p7CzeSbfobkQEHRrWuoyLq/tfzdX9rg5AhSINojG8iTQgrN0F3AcsBX4K/ADoYa3N8y9nMB/YY629whhzB/B7YIi1dkf96/mv+SPgMWtt+6PUdNgxXGGtiEjLpLBWRESk6TSbsNY/O2c38Gdr7R+PcMwO//6n62x7BLjcWjvkCOfol7yHUVXj44NVWbzw1Ra27C0FIDo0iJvPSeSHY3oSF6E1Q08Za2HXMkj7H6x7F0r3fruvwwAYegMMuQ4i2wauxgCw1rJ+/3pmb5/N7O2z2VH87e+JQlwhnNPlHCb2mMh53c4jKiQqgJVKAOgHxGbKGDMH2GGtvbkBx7qBdcDH1tp7juMerfqPrk5L8/8MXzxyfOdM/B2cc/e3j30+eHbowX/8dLLOfQDOndZ412sE72x8h4cXPXzUYyKCIvji6i80NkpzpTG8iTQgrP19bScLY0wkzvIFk621s/zbegGrgL8BdwE/stb+5wj3agcsB1631v76KDU9zGG6YyisFRFpmRTWioiINJ3mFNZeA/wH6G6t3X2EY6qAFGvtf+tsux14yFrb8QjnPIzaJx6Rz2f5ZG0Oz36xiQ17igEntP3R2F78YEyi2iOfat4a2PolrPoPZMwEb6Wz3RXstEkelgI9x4PLFdg6TzFrLRvzN/Jp5qfM3j6bzKLMA/tqg9tJiZMY3228Zty2DvoBsRkwxjwGfALswPlDqNqlCS6y1s42xrwKZFlrH/Af/1vgG2AzEAf8ErgcOLMhbZPr3Fdh7enGWwNZy6C67NjHbl8I8550ZuHeterb8XDTbPj3Vc5s26teBnMS42TWcpjze4hOgJ+lNaulCay1rNm3htLq0sPuf2zxY2QWZfLrUb/mu0nfPcXViTSIxvAm0oCw9hpr7f/qbCsEfmqtfbXOtp8A/we8aa097H8i/nF4NpAHXGqtrT5KTZpZKyJyGlFYKyIi0nSaU1j7KVBlrZ16lGNOJKzVzNoG8Pksn63P4enPN5GR44S28RHB3Dq+NylnJxIW7D7GFaTRlefD2ndhxauQverb7W16w4gfwdDrIDw+YOUFirWWTQWb+CzzMz7N/PSg4DbUHcq4ruO4KPEixnUdR3hQeOAKlaakHxCbAWPMP4HzgQSgEFgDPG6tne3fPxfIrJ1la4z5C3Al0AnIx5mR8xtr7crjvK/C2tasqgz+lASVhXDD29B3orP939fApk9h9B0w6bANWhqupgr+MsDpdHF1Kgy8/KTLPlX+nf5vpi+ZTu/Y3rx32XuYVrSUgrQY+qJsIg0Ia6+w1r5fZ1sB8DNr7St1tr0OfBdYBoypvxatMSYa+BQoAy6x1lYcZ41as1ZEpAVTWCsiItJ0mkVYa4zpAWwFrrTWfnCU4467DfJhrqFf8h6Fz2eZmZbNX2ZvZOs+Z9ZGQmwYP5/Yj+8M64rbpe/LAiJ7tRParnkLKoucbUHhMPhqGHUbdBwQ2PoCpO6M28+2f8b2om/bX4YHhXNu13OZ1HMSY7qMIcSt1t6nEf1H1IppHBdmPQDf/A36TYLr34T8THhmKGDhpyugbe+Tv8cXj8L8pyBxLNw84+Svd4oUVxVz/v/Op7ymnJcvepkRnUYEuiSR+jSGN5GTDWuNMdcC/wIuBN4CXrTWPlTn+BicoLYSuNha24B2CIfUobBWRKQFU1grIiLSdJpLWPswcAvQrf5f79Y77k0gou7sW2PMQmCNtfbWBt5Lv+RtgBqvj/dWZvH055vIKigHoF/HKKZNTuK8/h00UyNQKksg7S1Y8hLkrvt2e69zYfTt0Gdiq2uRXMtaS0ZeBrMyZ/Fp5qdklWQd2BcdHM2E7hOY3HMyoxJGEeQKCmCl0gj0H1ArpnFc2LcZnjsTMHD3alj2Mix4GnpPgJvea5x7FOyEZwaD9cEdS6B9/8a57inwyKJHeHvj21yUeBFPjX8q0OWI1KcxvIn4w9oewM/r7dqPs2TBEcNaY0xXnA4ZD1lr/2qMuQiYAYy11n7jH3s/AyKAK4C6vdj3Wmu9DaxRYa2ISAumsFZERKTpBDysNca4gG3Af6210+rtq7/e3dnAV8A0YCZOi6ZfAcOstWsbeD/9kvc4VFR7eW3Rdp77cjOF5c5yRGP6tOM3lyST1Emfv4CxFnYsgsUvQPpHzi+TAdr1g7PvgsHXQFDo0a9xGrPWsnbfWmZlzmJW5ixyy3IP7GsT1oaJPSZycc+LGdphKK6TWddQAkU/ILZiGscFgFcvd9Z4H307rHkTyvbDd/8DSVMa7x5v3AAZM2DkLXDxE4133Sa2IW8DV310FUEmiM+u+oz2Ee0DXZJIXRrDm4g/rE05zK5/Aj/kCGEtkIqzBm0NMNn6f0FgjHkWuBgYCgwHvjzCrXtaazMbWKPCWhGRFkxhrYiISNNpDmHthTjtlPpbazfW2zeXOuvd+bddDfweSAQ2AfdZaz8+jvvpl7wnoLCsmr/N3cy/FmRS5fXhMnDtiG7cM7E/7aNbbyjYLORvhyX/cNok17ZIjk6A0bfB8B9AaOv+RYjP+liZu5JPtn3C7O2zyavIO7CvU2QnJveczMU9L6Z/fH/NGG859EK1YhrHBYD0GfDmDd8+junqzLJ1N2LnhC1z4LUrIDQG7kmH0KjGu3YT+94n32Nl7kpuH3o7tw25LdDliNSlMbwVU1grItKyKawVERFpOgEPa081/ZL35OzMK2P6rAxmrskGIDo0iLsv6EvK2YkEuzVDMaAqimD5K846fsXO60NYnBPajroFwuMDWV2zUOOrYUn2EmZum8mcHXMoqS45sK9XbC+m9JrC5J6T6RbdLYBVSgPoB8RWTOO4AOCtgWeGQNEu5/GEB2HcvY17D58PnhsOeVvgvF9Dv4uc7SFR0KYXNOM/8Jm5dSbT5k+jQ3gH/nr+XzGH+W8zLCiMxJhE/aGSnGr6gmvFFNaKiLRsCmtFRESajsJaOSHLMvN45KP1pGUVAtCnQxQPTx3ImL7tAlyZUFPlrGv79dOwf5OzLSQaRv4Yzv4pRLQJaHnNRaW3kvm75vPxto/5audXVPmqDuwb3H4wU3pO4aLEi2gb3jaAVcoR6AfEVkzjuBww70mY83twBcM96yGqQ+PfY9Hz8OmvDt3+nX+C56rGv18jqfJWMfHtiQd1kzicaSOncUPyDUc9RqSRaQxvxRTWioi0bAprRUREmo7CWjlhPp/lf8t38sSsDewvdYKuKZ4EfnNJMgmx4QGuTvB5Yf0HMP9PsMe/pHNItDPT9qzbNdO2juKqYr7Y8QUfb/2YxTmL8fnXAHYbN6M7j+aSXpcwodsEIoIjAlyp+OkHxFZM47gcUJYH/7sZep0LY+9pmntUFMJ/vgv5mc7jmnIoz4duo+GHnzbNPRvJe5ve44XVL1Bjaw7ZV+OrIa8ij8SYRD68/EPNrpVTSV9srZjCWhGRlk1hrYiISNNRWCsnrbC8mr/M3sirizLxWYgIcfOzC/ry/XN6qjVyc2AtbPgYvnwM9qQ520Jj4ZyfwujbISQysPU1M3vL9jIrcxYzt85k3f51B7aHB4UzofsELul1CaMTRhPkasR1EeV46QfEVkzjuARUcQ78ORmsD366Atr2DnRFJ6Ssuoxz3zqX8ppyXpv8GkM7DA10SdJ6aAxvxRTWioi0bAprRUREmo7CWmk063cX8eAHa1m+PR+ApE7R/OEKD2f20AzOZsHng4wZMPcxyF3vbIvsAOPvg2EpEBQS2PqaoW2F2/h428fM3DqTncU7D2xvE9aGi3tezCW9LmFA2wGakXTq6RPeimkcl4D799Ww6TMYey+c/2Cgqzlhv/7613y45UOu6ncVD531UKDLkdZDY3grprBWRKRlU1grIiLSdBTWSqPy+Sxvr9jFYx+nk19WjTFw/cju3Dcpidjw4ECXJ+CEtuvehTmPftvWMT4RLngYBlwOCh4PYa1lzb41zNgyg08zPyW/Mv/AvsSYRC7pdQlTek2ha3TXAFbZquiLtBXTOC4Bt+49p/1yTBf4WRq43IGu6IQszVnKDz79AVHBUXx5zZeEBYUFuiRpHTSGt2IKa0VEWjaFtSIiIk1HYa00ibzSKh77OJ3/Ld8FQLuoUH532UAmD+qkWYjNRU0VrHwVvnoCSvY427qOhIv+AN1GBra2ZqzaV82i3YuYsXUGX+74kgpvxYF9wzoMY0qvKVyUeBGxobEBrPK0p/9EWjGN4xJw1RXwp/5QUQA3vQe9JwS6ohPisz4ufvdiskqymD52OlN6TQl0SdI6aAxvxRTWioi0bAprRUREmo7CWmlSi7bs59fvp7F1bykAEwd05NHLBtEpVrM3mo3KElj4V1j4LFSXOdsGfQcueATiugW2tmautLqUL3Z8wYwtM1icsxif9QEQ5ApiXJdxXNL7EsZ3HU+IWy2mG5l+QGzFNI5LszDzF7D0JfBcDd95KdDVnLC/rfobf1/9d85KOIt/XPiPQJcjrYPG8FZMYa2ISMumsFZERKTpKKyVJldZ4+X5OZv529wt1Pgs0aFBTLs4ietHdtcs2+akKBu+/AOsfB2wEBQOY34GZ98FIRGBrq7Zyy3L5ZNtn/DRlo/YkL/hwPbokGguSryIS3pdwhkdzsBlXAGs8rSh/zhaMY3j0ixkLYcXJ0BQGNy7EcJaZjeFXcW7mPzuZAyGz676jE6RnQJdkpz+NIa3YgprRURaNoW1IiIiTUdhrZwyGTlFTHsnjVU7CwA4u3dbpl85mO5tFQQ2K9mrYdYDsH2B8zimK0x6DJKnaj3bBtqYv5EZW2cwc+tMcstyD2zvEtWFi3tezCW9L6FXbK8AVtji6QuxFdM4Ls2CtfC3s2BvOkx9Bs68OdAVnbAffPoDluYs5a4z7uLHg38c6HLk9KcxvBVTWCsi0rIprBUREWk6CmvllPL6LK8szOTJTzOoqPYRHuzm/kn9+d5Zibhc+p6v2bAW1n8Anz0IhTucbb3Ph4ufhLa9A1tbC+L1eVm2Zxkzts5g9vbZlFaXHtg3sO1ALul1CZN6TqJdeLsAVtki6T+LVkzjuDQbC56F2Q9Ct1Hww88CXc0J+2DzB/xmwW/oEdODjy7/SF1PpKnpC6wVU1grItKyKawVERFpOgprJSC27y/lvrfXsHhbHgAje7bhyasG06NtZIArk4NUlcHXf4YFz4C3CtwhcM7PYOz/s3ff4VHUaxvHv8+mAkmA0HsTBAQpKgoWwIq9d0VEsdfjUV89ltM9ejx67P0o2LFhBexKEZQqiBSlS+8QIKQ87x+zwRhDSEKS2ST357rmyu7M7My9iEx2n3l+v5sgQfMOl8S27G18teQr3p//PuN+GUeO5wAQZ3H0btqbE9qeQP8W/amZoE7zYtAHxGpM13GJGZtXwgMdwXPh+ulQt3XYiUpla9ZW+g3vx7bsbbxy3Ct0bdA17EhStekaXo2pWCsiUrmpWCsiIlJ+VKyV0OTmOi9/u5h7PvqRrTtyqJEQx23HdeSCA1upyzbWrP0ZProZfv4seJ7eDk54ENr2DTdXJbV221pGLRzFh/M/ZMaaGTvX14ivwZEtj+SEtifQq0kv4iPxIaaMafoHohrTdVxiytCTYMFXcORfgnneK6lbvrqFkQtHMmifQdy0/01hx5GqTdfwakzFWhGRyk3FWhERkfKjYq2Ebsm6rdz85nQmzA+6bPu0q8d9Z+xL87rqMIwpeUMjj7wVtqwI1nU7D475B9RMDzdbJbZw48Kd89su3bJ05/p6yfU4ts2xnND2BDrX66xhKX9LfxjVmK7jElMmPQ8f3ABNusPlX4WdptQ+W/QZN3x5A01qNWH06aN1zZHypL9c1ZiKtSIilZuKtSIiIuVHxVqJCbm5zosTFvGvkbPZlpVDSlI8d53YmTP3a64vDGPN9o3w6V9g0v8Ah1oN4Lj7YZ9Twk5Wqbk701dP54P5HzB64Wg2ZG7Yua11WmuOa3scJ7Q5gRZpLcILGTv0j0I1puu4xJSMNXB/B/AcuG4qpLcNO1GpbM/eTt/X+7I1eysvH/cy+zbYN+xIUnXpGl6NqVgrIlK5qVgrIiJSflSslZiycE0GN70xncmL1gNwZKeG/PO0rjRM1fyoMWfJt/DuNbBmTvC804lB0Ta1cbi5qoCsnCzGLxvPB/M/4IslX5CZk7lz27719+W4tsdxTOtjqF+jfogpQ6UPiNWYruMSc4adAvO/gCPuhkP/EHaaUrv161v5aMFHDOw8kJsPuDnsOFJ16RpejalYKyJSualYKyIiUn5UrJWYk5PrPDNmPg98PJcdObmk10rkn6d2ZUAXFQFjTnYmfP1vGPsg5GZDch04/j/Q5XRQR3SZ2LJjC58t/oyPFnzEhOUTyPVcACIW4aAmB3Fcm+M4ouURpCSmhJy0QukvVzWm67jEnMlD4f3roPG+cMWYsNOU2ueLP+f6L66nca3GjD59NBGLhB1JqiZdw6sxFWtFRCo3FWtFRETKj4q1ErNmr9jEja9P58flmwA4c7/m3HViZ1KTE0JOJr+zYgaMuApWfB8873QSHP8ApDQIN1cVs2bbGkYvHM2H8z9kxpoZO9cnRhI5rPlhHNvmWA5rfhjJ8VW+E10fEKsxXccl5mxdB//eKxgK+dopUK9d2IlKJTMnk76v9yUjK4MXj32R7g27hx1JqiZdw6sxFWtFRCo3FWtFRETKj26Zl5jVsXEaI67uwxV922EGb0xeyrEPjeG7hevCjiYFNe4KQz6HfrdDJB5+fA8ePxB+/CDsZFVK/Rr1Ob/T+bxy/Ct8eOqHXN39atrUbsOO3B18uvhTbvrqJvoN78ftY27n66Vfk5WbFXZkqcLM7Eoz+97MNkWXb8zs2N285kwzm21m281shpkdV1F5RcpNzXRo2y94/MM7oUbZE0lxSfRv0R+A0QtHh5xGRERERERERKT6UGetVArfLljHH4ZPY+n6bUQMruzXjuuP6EBivO43iDnLp8M7V8KqH4LnPS6AAf+CJN09Xx7cnTnr5/DRgo8YtWAUyzOW79xWO6k2R7Y8kmNaH8MBjQ8gPhIfYtIypbt5Y4CZnQjkAPMI/ptcBNwM9HD3HwrZvw/wNXAb8AFwHnAr0NPdZ5bgvLqOS+yZ8iK8dw006gpXjg07Tal9ueRLrv38WhrWbMgnZ3yioZClPOgaXo2ps1ZEpHJTZ62IiEj5UbFWKo3N27P4y/uzeHPyUgC6NqvNf8/pTrsG1WquzsohOxO++AeMexhwqNsaTn0aWh4YdrIqzd2Zvno6IxeMZPTC0azdvnbntvTk9J2F2/0a7UdcJC7EpHtMHxBjlJmtA2529+cK2fY6UMvdT8i3bgIwzd2vKME5in0dz8kt+99xDIhEivdXMDfXiaXfsiIGpvnEy8fWdXB/+2D+9qu/hXp7BestUqnmcN+Rs4O+r/dlS9YWnj/meXo07LFHxzOzYhd8c3Jzin3ciEX0d7ny0n+4akzFWhGRyk3FWhERkfKjYq1UOh/NWM7t78xgw9YskhMi3HlCZ87r1VJf2sWihWPhnStg45LgC+u+/weH3gRxVabDM2bl5OYwaeUkRi0cxaeLPmVD5oad2+ol1+PIVkHhtmfDnpWxcKv/2WOMmcUBZwJDCTprZxWyz2LgAXf/b751fwFOcfduRRw7CUjKtyoVWLq76/jNb0znjejNPWUpLmLccXwnLj64TZH7PTd2Af/86MdyKRiXVut6NXnv2kNI09zv5eOlM+CnT367rm4buOxLqFEnjESl8qexf+K9n98rk2MlRBL4xyH/4Ng2RY6QzkNTHuK5Gc9R3NsbejTswfPHPF8Zr1+ia3i1pmKtiEjlpmKtiIhI+dHYZlLpHNe1CaNvOIxD9qrP9qxc/vTOTIYMm8zaLZlhR5OCWh8CV46Dfc8Gz4Uv/wlDT4ANS8JOVuXFReI4sMmB3N37bj4/63OePPJJTmt/GmmJaazdvpbX57zO4NGDOfyNw/nbN3/jm2XfkJ2bHXZsqWTMrKuZbQEygSeBUwsr1EY1BlYWWLcyur4otwEb8y27rcCuy9jBW1PKvlALQbfu0PELi9zH3Xlh/IKYKtQCLFy7lc9+LPifQMrMAZeCFSgerl8Ac0aGk6eUzuxwJgmRsinoZ+Vm8drs14reJyeLV2e/WuxCLcDUVVOZsWbGnsYTEREREREREYkJ6qyVSis31/nfuAXcN2oOO3JyaZCaxH/O7MZhHRqEHU0KM/11+PAPsGMLJNeGkx6FzieFnaraycrNYsKyCXyy6BM+W/wZm3Zs2rmtTlId+rfoz5GtjuSgJgeRGJcYYtIi6W7eGGFmiUBLoDZwBnAp0HcXnbU7gIvc/dV8664C7nb3RkWco8Sdte9MXcqNr0+nY+NUXrvsoFK8s8Jty8rhsPu+ICvH+fymvrTdxTD8P63azJEPfE1ifISvb+5PckL498Y9/uXPPP31fE7s1pRHzt2zoW2lCDu2Qvb24PG4/8K4h6DzKXDW0DBTldj27O1k5uzZTXCrtq7itPdOI2IRvj77a2on1S50v4nLJ3Lpx5eSnpzOuye/u9uRUv7yzV/4ZNEnDOk6hOt6XrdHGSUUuoZXY+qsFRGp3NRZKyIiUn40FqlUWpGIcemhbenTrj7XvzaVeau2MPB/33LpIW24ecDeJMVraLyY0u1saHEAvHUp/DIZhl8IvS6Do/8O8Um7f72UiYRIAoc2P5RDmx/Knb3v5Lvl3/Hxoo/5fPHnrM9czzs/vcM7P71DrYRaHNb8MI5seSSHNDuEmgk1Kzzr6tWrueuuu/jwww9ZuXIldevWpVu3bnz88ccHu/u4sjyXmZ0DvAq86+6nlOWxqzJ33wH8FH062cwOAK4HLi9k9xVAwaJso+j6os6RSdC5CxRvztXPZ68G4IhODalTs+xuOqgDHNS2HmPmreHz2at2Waz97MdVAPRuW4/GtZPL7Px74ph9GvP01/P5as4qsnJySYgLv4BcJSXWDBaATicHxdqfP4ecLIirPMNPJ8cnkxy/Z393ayfVpn3d9sxbP4+xv4zl+LbHF7rfV0u/AuCw5odRJ7nObo/bv0V/Pln0CV8v/VrFWhERERERERGpEvRNnVR6nZum8f61h3DhQa0AeHbsAk59bDw/rdoScjL5nfS2MHg09Il+ufrt0/DcUbD253BzVVMJkQT6NOvDn/v8mc/P+pxnj36Ws/c+mwY1GpCRlcHIBSO56aubOPS1Q7nms2t4e97brN22tsLynX766UydOpWhQ4cyd+5c3nvvPfr16wdQryzPY2atgfuBMWV53Goqwm+7YPP7BjiiwLqjouvLTFZOLl/NCYqlh3fcZcNuqR3esSHwa0G2MJ/NDrYd0alhmZ+/tLq3qEN6rUQ2bc9m8qL1YcepHpr2gFoNIHMTLC7Tv+aVRt/mfYFfC7KF+Xrp17/Zd3cOaXYIEYswZ/0cVmQUea+HiIiIiIiIiEiloGKtVAnJCXH87ZQuPDNwf+rWTGDW8k2c8MgYXv12MdVtqO+YF5cAR/8NzhsONdJh+XR4qi/MejfsZNVafCSeA5scyB0H3cGnZ37KS8e9xMX7XEyL1BbsyN3BV0u/4u7xd9N/eH8GjhzI8zOfZ8HGBeWWZ8OGDYwZM4Z7772X/v3706pVK3r16sVtt92Gu78HYGYtzexdM9tiZpvMbLiZNYpu62hmW83svLxjmtlZZrbNzDrnWxcHvAzcDcwvtzcUI8xsvpn9rthtZnXMrETv38zuMbPDzKx1dO7ae4B+BH+emNmw6Lo8DwEDzOym6H+fPwP7A4+W9v0UZvKi9Wzank16rUS6t6hTlocGfi3WfrdwHRu3Zf1u+4atO3YWQ/vvHTvF2riI0S86TcDns3ddaJYyFIlA+6ODx3NHh5slJIc1PwyAsb+MLXRe9IUbF7Jo0yLiI/H0btq7WMesm1yXbg26Ab8WekVEREREREREKjMVa6VKOapzI0bfcBiHtq/P9qxcbnt7Ble9PIUNW3eEHU0K6nAMXDEWWvaGHZth+EAYdXswVKSEKmIRujXoxh/2/wMfnvohb5/0Ntd0v4ZO6Z1wnKmrpvLA5Ac4acRJnPjOidz/3f18t+K7Qr+IL62UlBRSUlIYMWIEmZm/nzfRzCLAu0A60JegQ7Mt8DqAu88G/gg8Hi3qNgeeBG4tMJ/qXcAqd3+uzMLHttZAYWPEJwHNSnishsAwYA7wGXAAcIy7fxLd3hJokrezu48HzgMuA6YTzHF7irvPLOF5i5RXiOzXoQFxkbKfUqlVvVq0a1CL7FxnzLzVv9v+1dzV5OQ6ezdKpUV6xQ8fXpTDO+V1Ba8MOUk10uGY4OfcUeHmCMm+9feldlJtNu/YzLRV0363Pa/jdv9G+1MroVaxj5tXBC6qY1dEREREREREpLJQsVaqnIZpyQy9uBe3H9eRhDhj5MwVHPvQGCbOr7jhW6WYajeDi96HPtcGzyc8Bi+cAJuWhZtLdjIz2tdtz+XdLmf4icP55IxPuP3A2+nTtA/xkXgWblrI0FlDGTx6MH1f78stX9/CB/M/YMP2DXt03vj4eF544QWGDh1KnTp1OPjgg7n99tv5/vvv83Y5AugKnOfuk919IjAQ6BudNxV3fxwYC7wEvAB8BzyS770dAlwCDNmjsJWAmZ1kZidFnx6T9zy6nArcCSwsyTHd/RJ3b+3uSe7e0N2PzFeoxd37ufugAq95w933jr6mi7t/tIdv7XfyCpGHl+MQxEd0CoZX/ryQoZDzisXlef7SOrR9A+Ijxs+rM1i0NiPsONVD2/4QSYC1P8Gan3a/fxUTF4nj0GaHAoV3wZZ0COQ8eftPXD6Rbdnb9jCliIiIiIiIiEi4VKyVKikSMS47rB1vX3kwberXYvnG7Zz7zAQe+HgO2Tm5YceT/OIS4Oi/w9kvQVIaLJkATx0GC8eFnUwK0bhWY87teC5PHfUUY84ew/197+fEtidSJ6kOm3ZsYuSCkdw25jb6Du/LwJEDefr7p/lx7Y+lGo789NNPZ9myZbz33nsMGDCAL7/8kp49e2Jmg4BOwBJ3X5K3f7RjdkN0W57BwL5AT2CQR4OYWSrwIjDE3deU8o+jMhkRXRwYmu/5COA1gs7km8IIVpYWrsng59UZxEeMQ9s3KLfz5A2F/GW0izZPdk4uX81d/Zt9YkntGgkc0Dod0FDIFSY5DVofHDyeVz2HQs4rrBYs1m7esZkpK6f8Zp/i2qvOXjSp1YTMnEy+W/Fd2QQVEREREREREQmJirVSpXVtXpsPrj2EM/drTq7Dw5//xNlPT2DJuq1hR5OCOp0Il30JjbpAxmoYeiJMeAI053DMSklM4ZjWx/DPQ//Jl2d9ydABQ7mkyyW0r9ueXM9l6qqpPDL1Ec764CwOf+Nw7hh7B6MWjGJj5sZCj7dh+waG/jCUv33zN16Y+QLrtq8jOTmZo446ijvvvJPx48czaNAggL+UIGY3oFZ0aZJvfTuCIYHfN7NsM8sm6Mw9Kfq8Xcn/RGKXu0fcPQIsBhrmPY8uSdFu1w/Czrmn8gqQB7ROp3aNhHI7z36t6pKWHM+6jB1MW7Jh5/qpSzawYWsWdWom0KMc5sstC0dEO35VrK1AHQYEP6vpUMh9mvUhzuL4eePPLNm88/4axi8bT7Zn06Z2G1qktSjRMc3s16GQl2goZBERERERERGp3FSslSqvVlI8/z6zGw+f24PUpHgmL1rPcQ+N4YPvNdRuzKnXDi75GLqeCZ4Do/4P3h4CO1Rcj3VxkTh6NurJDfvdwNsnvc3Hp3/MnQfdSf8W/akRX4M129bw7s/vcvPXN3PY64dx/ofn89i0x5i6airZudnMWD2DAW8P4D+T/sPb897mgckPcOxbx/5ujsPOnTtDUHj9EWhhZju/4TezzkAdYFb0eTrB8Mf/iP582cxqRHefTTCMcvd8y3vAF9HHv1YUqhB3b1OVO4nzCpBHlPMQxAlxEQ7r0CB6zl/nf/3sx1/ny42Pi81fsfpHO34nzF/Llsyym2daitD+6ODnovGwvfCbVaqytMQ0ejTsAfy2uzbv8WHNDivVcfO6cb9a+lWpRm8QEREREREREYkV8WEHEKkoJ3VrSo8WdbjutalMXbyBa16Zypi5a7j7pM7UTNT/CjEjsRac9gw02w9G/wlmvAGr58A5r0CdknXeSHiapDThrL3P4qy9z2JHzg6mrJrCuF/GMfaXsfy04Se+X/M936/5nienP0mt+FrkeA6ZOZk4TrYHBaStO7Zy1cireLz749ROq82kSZO47777AN4FPgVmEBRgbyC4nj0OfOXuk6IxniQouv4dSAKmAvcDV7v7dmBm/sxmtgHA3X+zvqoxsyMI5vxtSIGbttx9cCihysCWzGwmLgjmJq+IIYiP6NSQD75fzmc/ruLmYzoCvxZuD4/OaRuL2tavRet6NVm4ditj561mQJcmu3+R7Jl67aBee1g7D37+HPY5NexEFa5v875MWjmJr5d+zfmdzicnN4cxS8cE21qUbAjkPL2a9KJGfA1Wbl3J3PVz2Tt977KMLCIiIiIiIiJSYay63YluZmnAxo0bN5KWlhZ2HAlBVk4uD306j8e+/Al3aNugFo+e25POTfX3IeYsHAfDB8LWNVCzPpw17Ne5/6TSWpGxgvHLxvPNsm+YsHwCGzI3FLn/yodWkjEngxYtWnDmmWfy97//vaa7bzOzlsAjBIXHXGAUcK27rzSzgQTF2x7uPg/AzHoBY4GT3X1kwfOY2QtAHXc/pezebWwxs7uBu4BJwHKCOWx3cveYryLt6jo+auZyrnhpCq3r1eTLm/uXe451GTvY/++fkOuQmhzc8LN5ezZxEWPKHUdRu2b5DcO8p/76/iz+N24BifERkuIL7wA+rUcz/nJylwpOVoWN/hN88yjEJUF8ctkeu25LuOgDqFGnbI9bUjnZ8MqZsHTy7zbNj49wcsNUAFJznVwgI2Kk5jpfrdhEaf9vubZuTb6skUByJIGEuLL9c923wb48esSjxEd0Q18Zs7ADSHjyruHTpk4hNTU17DgiIlJCbdvtpeu4iIhIOVGxVqqt8T+v4cbXp7FyUyaJcRFuP64jF/VpjZl+94wpG5bA6+fD8ukQiYdj74UDLg07lZSRXM9l+Jzh/GPiP3a5z2NHPLZzbsIo/U9aSma2HLjF3V8MO0tp7eo6/vcPZvHs2AUMPrgNd53YuUKyXDp0Ep/+uPI36wbs05gnL9yvQs5fWt8v3cCpj48nJ3fXvwNGDKbdfTRpybFbdK5Ulk+Hp/sHQ/yXh1Ofgm7nlM+xi2vJt/DcUYVucuCCJo34PjnpN+vP3rSZO9auL/UpP61ZgxsbNSj163fn5eNeZt8G+5bb8aspXcOrMRVrRUQqNxVrRUREyk/oxVozawbcCxwL1AR+Ai7ON4xlwf37EcwpWFATd19RjPOpWCs7rcvYwS1vTufT6DyDR3ZqyH1ndCO9VmLIyeQ3dmyF966FmW8Gzw+4FAb8C+JURKgKNu3YRL/X+5GVm/W7bfGReL448wvqJNfJv1ofEEvJzNYCvdz957CzlNauruPuzk+rtpCcEEeL9JoVkiUn11mybuvO9uSIQYu6NYlEYv+v6PqMHWzY9vv/5wAGPf8ti9Zu5bmL9ueIGB7SudLZug62lb4wWahvHoNJz0H3C+CUx8r22CX19f3w+d+COXoH/Ot3m7Nys1m2ddXO53GROJrVaLhnN8lt+oU1L51MRnwSXPYlJNTY7UuK41/f/ouxv4zl+p7Xc2lX3SBWxmL/H0gpNyrWiohUbirWioiIlJ9Qx/Uys7rAOILi67HAaqA9UJxvsvYGNuV7vmpXO4rsSnqtRJ4ZuD9Dxy/knx/N5tMfV3HsQ1/z4Nnd6dOuftjxJE9iTTj9WWi0D3z2V/juWVgzF84cCjXTw04neygtMY0h+w7h8WmPYxiO7/x58T4XFyzUyp55FjgP+FvYQcqamdG+UcV+8RsXMVrXr1Wh5ywrdWslUncXNyb1aVefRWsXM/7ntSrWlqWa6WV/zdr7uKBYu+ArcIcwRwdZ8HXws/3RwTy9BSQArSjjeWXT21K/ZmPqb14GG5ZDu7IZAv2QZocw9pexfLv8WxVrRURERERERKTchT0J063AEne/ON+6BcV87Sp331D2kaS6MTMGHdyGXm3qce2rU/h5dQbnPzuRa/rvxfVHtCc+rvD5/KSCmcGhf4AGHeHtIcGXws8eAee+Dg06hJ1O9tAV+15BgxoNeOGHF1iyeQnNUpoxaJ9BnNnhzLCjVTXJwGVmdiTwPfCb1kp3/0MoqSSm9GlXj1e/DYq1EuNaHhRMEbBxCaxfCOltwsmRtR2WTAwetzms6H3Lkllwvu9fC34vKKNi7YGNDwRg6qqp7MjZQWKcRlwRKUsZnorlqrO2KCmRzWFHkCoil7iwI1QKEcppqgoRERGRYgq7CnUSMMnM3jCzVWY21cyGFPO108xsuZl9YmYH72onM0sys7S8BdCnQilU56ZpvH/tIZy9fwvc4ZHPf+LspyewdP3WsKNJfh2Pg0s+hjotYd18eO5ImP9l2KlkD5kZZ3Q4gw9O/YDpA6fz0WkfcdbeZ2kO6bK3LzANyAW6AD3yLd1DSyUx5aC29QD4cfkm1mfsCDmNFCkpBZrtHzxeOCa8HEu/g+ztkNII6lfwDVR5xeEyfP/t6rQjPTmd7Tnb+X7192V2XBERERERERGRwoRdrG0LXAnMA44BngAeNrOLinjNcuAK4PTosgT40sx67mL/24CN+ZalZRNdqqKaifHce8a+PHxuD1KT4pm8aD3HPTSGkTOWhx1N8mu0Dwz5AlocCNs3wkunw+QXwk4lEvPcvX8Ry+Fh55PY0CA1iQ6NUgCYMF/dtTEvr1iZNwxxGPIKpa0PrfihmNscGvz8ZQps31T0vsVkZju7a79b8V2ZHFNEREREREREZFfCLtZGgCnufru7T3X3p4FnCIqxhXL3Oe7+lLtPdvfx7j4YGA/cuIuX3APUzrc0L9u3IFXRSd2a8uF1h9KtRR02bc/mypencPs7M9iepaFxYkat+jDwPeh6JuRmw/vXw8d3QG5u2MlERCq9vHnbNRRyJZBXrFzwdTBvbRjyCsUVOQRynjotoW5r8BxY/E2ZHfaAJgcAMHHFxDI7poiIiIiIiIhIYcIu1i4HZhVY9yPQsoTH+RbYq7AN7p7p7pvyFkCTv0ixtKxXkzev6M0VfdsB8MrExZz06FjmrNBfoZiRkAynPQP9bg+ej38E3hgIOzR0tUhhzOwLM/t8V0vY+SR29G4XDIX8jTprY1/zXhCXBFtWwpp5FX/+HRnBMMgQTrE2/3nLsLs4r7N2+urpbMveVmbHFREREREREREpKOxi7Thg7wLrOgCLSnic7gSFX5EylRAX4f+O7ciLl/SiQWoSc1du4aRHx/LyxEV4WN0r8ltm0O9WOO1ZiEuEH9+HoSfCllVhJxOJRdOA6fmWWUAi0BOYEV4siTUHtamHGfy0agurNm0PO44UJSEZWgaFRRZ8VfHnX/xNMMJF7RZBh2sY2vQNfpZhsbZFagsa12pMdm42U1dNLbPjioiIiIiIiIgUFHax9kHgIDO73cz2MrPzgMuAx/J2MLN7zGxYvuc3mNnJ0f27mNl/gcPzv0akrB3avgEjrz+Uvh0akJmdy5/emclVL09h49assKNJnn3PhIHvQo268MskePYIWD0n7FQiMcXdbyywXOPuhwD/BfQPmuxUu2YC+zRNA9RdWymEOW/tgjG/Zqjo+WrztI4OBb1iBmxdVyaHNDN6Ne4FaN5aERERERERESlfoRZr3f074FTgXGAmcCdwg7u/nG+3Jvx2WORE4D8EHUBfAd2AI939swoJLdVW/ZQknh90AHcc34mEOGPkzBUc9/AYJi0smy8FpQy06gOXfAp128CGxfDc0bBofNipRCqDl4DBYYeQ2JI3b+03mrc29rWOFmsXjq34udvDnK82T2ojqL834LBoXJkdNq9Y++3yb8vsmCIiIiIiIiIiBYXdWYu7f+DuXd092d07ufszBbYPcvd++Z7f5+57uXsNd6/n7v3d/YsKDy7VUiRiXHpoW966sg+t69Xklw3bOPvpCTz6+TxycjUsckyovxdc+ik0PwC2b4BhJ8PMt8NOJRLregMa61Z+o3fbYN7a8SrWxr5mPSGhFmxbB6t+qLjzbt8Iy6cFj/O6W8NSDt3FecXaH9b+wJYdW8rsuCIiIiIiIiIi+YVerBWpjPZtXocPrjuUU3s0IyfXuf/juVzw7ERWal6/2FCrPgx8DzqeADk74M2LYdzDoHmGpZozs7cLLO+Y2QTgeeCpsPNJbDmgTTpxEWPxuq0sXb817DhSlLiEYHQJqNihkBeNB8+F9HZQu1nFnbcw5VCsbZLShJapLcnxHKasmlJmxxURERERERERyc+8mhUvzCwN2Lhx40bS0tLCjiNVwFuTl3LnuzPZuiOH9FqJ3H/mvhzesVHYsQQgNwdG3QbfRmtQB14Jx/wTIrpPpZILaVLEys/Mni+wKhdYDXzu7h+HEKnEdB2vWKc+Po6pizdU+Hn3bV6bN6/oQ2K8/r0utnEPwSd3hXPu/QbBiQ+Fc+48W9fBfW2Bsv1s8+d66byVlvKbdc2ysnl12QrqFmfI6Y4nwDkv736/6kPX8Gos7xo+bso8UlJSw44T01Iim8OOIFVELnFhR6gUIuSEHaFSaNtuL13HRUREyom+ARPZQ6fv15wPrj2EfZqmsS5jB4NfmMRf359FZrZ+2Q9dJA6OvReO/nvwfOITQZdtljqgpXpy94sLLJe4+/9VlkKtVLzTezYP5bzfL93IjF82hnLuSqvTiZCYsvv9ylokAbqcUfHnLahmOux9bJkf9viMDOIL3Nz6S0I842okF+8Asz+AzSvLPJeIiIiIiIiIVB3qrBUpI5nZOfxr5GyeH7cQgC7N0njk3J60qV8r3GASmPEmvHMF5GZBq0OCLpcadcJOJaWju3n3kJntB3SKPv3B3aeW4hi3AacBHYFtwHjgVnefU8RrBhEMuZxfprsXs+qh63gYNm7LIiunGB2EZeSPb0znyzmrueP4Tlx6aNsKO2+VkLUNMit4btWEGpAUQpG4MO6wdW2ZT3uwNXsb27KDG72e+HEor89/j3PbncLt3a8r+oUvHA9r5sA5r0DH48s0UyWma3g1ps7a4lNnrZQVddYWjzpri0edtSIiIuUnPuwAIlVFUnwcd5+4D4fsVZ8/vjGdmb9s4oSHx/C3U7pwWkidSZJP1zOgVgN47XxYNBaePw4ueAvSmoSdTKTCmFlD4DWgH7AhurqOmX0BnOPuq0twuL7AY8B3BL9P/BP42Mw6u3tGEa/bBOyd73n1umusEqpdI6FCz3dA63S+nLOaaUs2VOh5q4SEGsFSXZkF89aXsZrRBWD/5ofy+vz3mLHhJ0hpUPQLW/QKirVLJ6lYKyIiIiIiIiK7pGGQRcrYEZ0aMfL6wziwTToZO3L4w/Dp/OH1aWzJzA47mrTtC4NHQkpjWPUDPHcUrJ4bdiqRivQIkArs4+7p7p4OdAHSgIdLciB3H+DuL7j7D+4+HRgEtAT22/1LfUW+ReODym/0aFEHIJS5ckV2p2uDrgDMXj+bzJzMonduvn/wc+l35ZxKKiszizezgWbWKOwsIiIiIiIiEh4Va0XKQePaybwy5CD+cFQHIgZvT/2FEx8Zy0zNvxe+xl3hko+h3l6wcQn875ig40WkehgAXOXuP+atcPdZwNXAnk72WDv6c91u9ksxs0VmtsTM3jWzfYra2cySzCwtbyEoNksVtm+LOpjBLxu2sWqz5hiX2NK0VlPqJdcjOzebH9f+WPTOzQ8Ifi6bCrkaXlF+z92zgSeBYk8HICIiIiIiIlWPirUi5SQuYlx3RHtev7w3TWsns2BNBqc9Pp7/jV1AdZsrOubUbQWDR0Oz/WDbOhh6Isz7JOxUIhUhAmQVsj6LPfidwMwiwH+Bce4+s4hd5wCDgZOBC6LnHG9mRY0VfxuwMd+ytLQ5pXJISYqnQ8OgJj9N3bUSY8xsZ3ft96u/L3rnBh0hMQV2bIHVsysgnVRS3wLdww4hIiIiIiIi4VGxVqScHdA6nY+uP5SjOzdiR04uf/1gFkOGTWJ9xo6wo1VvterDwPdgryMhayu8eg58PzzsVCLl7XPgITNrmrfCzJoBDwKf7cFxHyMYTvmconZy92/cfZi7T3P3r4DTgNXA5UW87B6Crt28RZOAVwM9WtYBYKrmrZUY1K1BNwBmrJlR9I6ROGjaI3isUTxk1x4HHjCza8yst5ntm38JO5yIiIiIiIiUPxVrRSpAnZqJPHXhfvz15H1IjIvw6Y+rOPahMUycvzbsaNVbUgqc+xp0PQtys+HtITDhibBTiZSnawjmp11oZj+b2c/Agui6a0tzQDN7FDgB6O/uJep6dfcsYCqwVxH7ZLr7prwF2FyanFK5dI/OW6vOWolFXesXs7MWNG+tFMdrQBuCuePHAdMIro15P0VERERERKSKiw87gEh1YWYM7N2a/VrV5dpXpjJ/TQbnPjOB64/owDWH70VcxMKOWD3FJcCpT0HNejDxCRj1f5CxGg6/E0z/TaRqcfclZtYTOBLoGF39o7t/WtJjmZkBjwCnAv3cfUEpjhEHdAU+KulrpWrr0bIuANOXbiAn13WNlJjSpX4XDGNZxjLWbFtD/Rr1d71z3ry1v0yumHBSGbUJO4CIiIiIiIiES521IhVsn6a1ef/aQzi9Z3NyHR78dC7nPzuBlZu2hx2t+opEYMA9cMRdwfMx/4EPboDcnFBjiZQHD3zi7o9ElxIXaqMeI5h39jxgs5k1ji418nYws2Fmdk++53eZ2dFm1jZaNH4JaAU8uwdvSaqgvRqmUCsxjq07cpi7Us3UEltqJdSiXZ12QDG6a5tFO2tX/QjbN5VzMqmM3H1RUUvY+URERERERKT8lahYa2b/K85SXmFFqopaSfH856xuPHh2N2olxjFh/jqOfWgMX8xZFXa06ssMDr0JTnwILAKTX4A3BkF2ZtjJRMqMmT1sZtcVsv4aM/tvCQ93JcEcsl8Cy/MtZ+fbpyXQJN/zusAzwI8E3bRpQB93n1XCc0sVFxcxuuUNhax5ayUG5c1bu9tibWojqN0ScFg2pfyDSaVkZu3M7BEz+zS6PGxm7cLOJSIiIiIiIhWjpJ21g4D+QB2CL1x3tYhIMZzaoznvX3sInZuksS5jBxc//x3//OhHdmTnhh2t+tpvEJz5AsQlwo/vwctnQqa6uqTKOJ1gPryCxgNnlORA7m67WF7It08/dx+U7/mN7t7K3ZPcvbG7H+/umo9PCtWjZR0Api5eH24QkULs22BfAGasmbH7nXfOWzupHBNJZWVmxwCzgF7A99HlQOAHMzsqzGwiIiIiIiJSMUparH2CoIumDfAFcIm7n1pwKfOUIlVY2wYpvH1VHy7q3QqAp7+ez1lPfcOSdVtDTlaNdT4Zzn8TElNgwVcw9ETIWBt2KpGyUA/YWMj6TUARky6KVLzuLYL7/9RZK7Goa/2uAMxcM5Oc3U2boGKtFO1fwIPufqC7/yG6HAj8F7g33GgiIiIiIiJSEUpUrHX3qwmGM7wPOBFYYmbDzewYM7PyCChSHSQnxPGXk7vw5AX7kZYcz7QlGzju4TGMmrk87GjVV9u+cNH7UCMdlk2F5wfAxqVhpxLZUz8BAwpZfywwv4KziBSpe3QY5HmrtrBpe1a4YUQKaFu7LbUSarE1eys/b/y56J2bHxD8/GUSuJd/OKlsOgHPFbL+f0DnCs4iIiIiIiIiIShpZy3ununur7r7UQQfHn8AHgcWmllKWQcUqU4GdGnMh9cdSvcWddi8PZsrXprCXe/OZHvWbjo2pHw06wmDR0FaM1gzF/43ANbMCzuVyJ54ALjPzP5iZn2jy1+JdvWEnE3kNxqkJtG8bg3c4fslhTWEi4QnLhJHl3pdgGLMW9t4X4gkQMZq2LCoAtJJJbMa6F7I+u7AqgpNIiIiIiIiIqGI38PX5wIOGBC353FEpEV6Td64ojf3j57DU1/PZ9g3i5i8aD2PndeT1vVrhR2v+mmwNwweDS+eAmt/Cgq2F7wFTbuHnUykxNz9f2aWBPwJuDO6eiFwpbsPCy2YyC70aFmXpeu38fSY+Xy7IBiOvmmdGpx9QAs0qIuEbd8G+zJxxUTemfcOKzJWFL1zs7aweQWMvBxq1N2zE8cnQ5NuEJewZ8cpQkpCCmfufSa1EvS7ZwV4BnjazNoSzCEPcDBwK8FNViIiIiIiIlLFmZdwKK7ol7ynAYOBQ4APgOeBUe6eW+YJy5iZpQEbN27cSFpaWthxRIr0xexV/GH4NNZvzSIlKZ57TuvKid2ahh2retqyGl4+HZZPh8RUOO81aH1I2KmqK1VoyoCZNQC2ufuWsLOUhK7j1csL4xbw5/dn/W79a5cdxEFt64WQSORXXy/9mqs/uzrsGOXmxv1uZHCXwWV9WF3DC4hOJ3QDcBOQ94v+MuDfwMNe0g/sMSzvGj5uyjxSUlLDjhPTUiKbw44gVUSu+iqKJYJGMyuOtu320nVcRESknJSoWGtmjwPnAEsI5tB52d3XlFO2cqEveaWyWb5xG9e9OpXvFq4H4PwDW3LnCZ1JTtCHrgq3fRO8ei4sGht0tZw5FPYubPpPKWf6gFiN6TpevWzdkc2TX81n49YdAIz9aQ0/r87gjuM7cemhbUNOJ9Vdrufy/MznWbl15e533rEVVs6AnOw9O+nWtUGHboO9oW3fPTvWLsxbP49JKydxTOtjuL/v/WV9eF3Di2BmqQDuXiUrdSrWFp+KtVJWVKwtHhVri0fFWhERkfJT0mJtLrAYmEow/HGh3P20PY9WPvQlr1RG2Tm5PPjpXB774mcAOjdJ4/HzNSxyKLK2wRsXw9yRYHFwyhPQ7eywU1U3+oBYjek6Xr3999O5/PfTeZyxX3PuP7Nb2HFEKt6sd2H4QGjaAy77slxOMfaXsVz56ZW0rd2Wd095t6wPr2t4AWb2OXCau28osD4NGOHuh4cSrByoWFt8KtZKWVGxtnhUrC0eFWtFRETKT6SE+w8DvgA2ABuLWESkDMXHRbj5mI4MHdyL9FqJzFq+iRMeGcsH3y8LO1r1k1ADzn4R9j0bPAfeuQwmPh12KhGRaqFj46BAP3vFppCTiISkUZfg56ofIbd8vlhuX6c9AIs2LSIzJ7NcziG/0Q9ILGR9MnBoxUYRERERERGRMMSXZGd3H1ROOUSkGPp2aMCH1x2yc1jka16ZyncL1nH78Z1IitcdsxUmLgFOeRKS68C3T8HIm2Hbeuh7C5huNBURKS8dGwedWPNWbiE7J5f4uJLedyhSydVtDfE1IHsbrFsA9fcq81M0rNmQ2km12Zi5kfkb5tOpXqcyP4eAme2b72lnM2uc73kcMAD4pWJTiYiIiIiISBhK/A2XmbU2syFmdrWZ7VMeoURk15rUrsGrQw7iyn7tABj6zSLOfPIblqzbGnKyaiYSgWPvhX63Bc+//CeMug1yc8PNJVIEMxtoZkmFrE80s4FhZBIpiZbpNamREEdmdi4L1+q6J9VQJA4adgwer5xZLqcws53dtfM2zCuXcwgA0/h1eqHPo8/zlsnAHcBfQ0kmIiIiIiIiFapExVoz6w/8ADwFPAJMNbMLyiOYiOxafFyEWwd05H+D9qd2jQS+X7qR4x8ewyezVoYdrXoxg37/B8feFzyf+ASMuBJyssLNJbJrzwO1C1mfGt0mEtMiEaNDtLt2zgrN5yfVVKPo/bKrZpXbKTrU7QDA3HVzy+0cQhugHcE8vr2iz/OWZkCau/8vvHgiIiIiIiJSUUraWfs34BOCD4/1gGeA+8o6lIgUz+EdG/HhdYfQvUUdNm3PZsiwSdwz8keyc9TdWaEOvBxOfRosDr5/DV6/ELK2hZ1KpDBG0MFTUHM057xUEh0b5RVrNW+tVFMNo8XalT+U2yna11VnbXlz90XuvtDdI+4+Kfo8b1nu7uUzKbGIiIiIiIjEnBLNWQt0Afq4+3IAM7sZuNzM6rn72jJPJyK71bxuTYZf3pt7Rv7I8+MW8tRX85m6eAOPntuDhmnJYcerPrqdDclp8MYgmDsSXjoDzn01WCcSMjPLG2bRgc/MLDvf5jiCLp5RYWQTKamOTYJi7Y/qrJXqqlEFFmvXq1hbUcysM9ASSMy/3t3fCyeRiIiIiIiIVJSSFmvTgDV5T9x9q5ltIxhSUcVakZAkxke4+8R92L9VOre+9T3fLljHcQ+P5ZFze9C7Xb2w41Ufex8LF7wNr54Di8bC0BPg/LcgpUHYyURGRH92B0YDW/Jt2wEsBN6q0EQipbS3hkGW6i6vWLt+IWRugaSUMj9F3py1q7etZv329dRNrlvm55CAmbUF3gG6EtxUZdFNeSNhxIWRS0RERERERCpOSYu1AMeYWf6hEiPAEWbWJW+F7v4VCcfx+zahU5NUrnp5CrNXbOaC5yZyyzF7c9lhbTGz3R9A9lzrg2HQB/DiabB8OvzvGBg4Auq0DDuZVGPu/hcAM1sIvO7u28NNJFJ6HRsHIxYsXreVjMxsaiWV5tdZkUqsVn2o1RAyVsHq2dB8/zI/Rc2EmjRPac7SLUuZt34evZr0KvNzyE4PAQuAI6I/exFMOfQf4I8h5hIREREREZEKUtI5awGGEnTo5C01gKfyPX+nLIKJSOm0bZDCO1cdzGk9mpGT69wzcjZXvDSZTduzwo5WfTTpBoNHQ+2WsO5neO4YWDU77FQiuPtQINnMLjWze8wsHcDMeppZs5DjiRRLeq1EGqYmATBnpbprpZqqyKGQSzlv7erVq7nyyitp2bIlSUlJNG7cGDMbbWYHl2VOADM7x8zczEaU9bErQG/gLndfA+QCue4+FrgNeDjUZCIiIiIiIlIhSlSsdfdIMRYN0yQSshqJcfznrG78/ZQuJMZFGP3DSk5+dJyGjKxI9feCS0ZDg46weRk8PwCWTgo7lVRzZrYvMBe4laBbp05002nAPSHFEikxDYUs1V5esXbVrHI7RYe6HQCYu35uqV5/+umnM3XqVIYOHcrcuXN57733AL4k6BotM2bWGrgfGFOWx61AcUDeP2ZrgKbRx4uAvUNJJCIiIiIiIhWqNJ21IlIJmBkXHNSK4Vf0pmntZBasyeCUx8bx/vRlYUerPtKawsUjodn+sG09DD0Rfvo07FRSvT0IvODu7YH8QyF/BBwWTiSRkuvUJBgKefbyTSEnEQlJw87Bz4rorF1f8s7aDRs2MGbMGO6991769+9Pq1at6NWrF+5+T96UOWbW0szeNbMtZrbJzIabWaPoto5mttXMzss7ppmdZWbbzKxzvnVxwMvA3cD8PXrD4ZkJdIs+ngjcEu0+votyfE/RTuRTyuv4IiIiIiIiUnwlKtaa2X5m9oWZpRWyrXZ0W7fCXisi4ejeog7vX3sIB+9Vj21ZOVz76lT+9sEssnJyw45WPdRMh4HvQrvDIWsrvHIOzHgz7FRSfe1PMHVBQb8AjSs4i0ip7d0o6Kydrc5aqa7yD4PsXi6nyCvW/rThJ3K9ZL83pqSkkJKSwogRI8jMzPzddjOLAO8C6UBf4CigLfA6gLvPJhgB4vFoUbc58CRwq7vnbye+C1jl7s+V8O3Fkr/z6+fyu4A2BF3CxwHXlfRgZvZCtBDrZpZlZivN7BMzGxz9cy8xM0sws3vNbIaZZZjZMjMbZmZNd/9qERERERER2Z2Sfli7Cfjc3X/XxuDuG4FPgJvLIpiIlJ16KUkMG3wgV/VrB8BzYxdwwbMTWbPl91+eSTlISoFzX4cup0NuFrx1KUx8OuxUUj1lAr+74QroAKyu4CwipbZzGOSVm/FyKlSJxLQGe4NFYNs62LyiXE7RMrUlSXFJbMvextLNS0v02vj4eF544QWGDh1KnTp1OPjgg7n99tvzhuMHOALoCpzn7pPdfSIwEOhrZgcAuPvjwFjgJeAF4DvgkbxzmNkhwCXAkD17p+Fy99Hu/nb08U/u3hGoDzR0989LedhRQBOgNXAs8AXwEPCBmcWX4ng1gZ7A36I/TyMYovm9UuYTERERERGRfEparD2Q4A7oXXkf6FOSA5pZMzN7yczWRoe1mmFm++/mNf3MbIqZZZrZT2Y2qCTnFKmO4iLGLQM68uQF+5GSFM/EBes48ZGxTFuyIexo1UN8Ipz2LPS6DHAYeTN8/vdy64YR2YX3gLvMLCH63M2sJXAv8FZJDmRmt5nZd2a22cxWmdkIM9vt3HpmdqaZzTaz7dFr/nElfxtS3e3VMIW4iLFhaxYrN+nGI6mGEmpAvb2Cx6vKZyjk+Eg8bWu3BYo3FLK7szFzIztydgDBnLXLli3jvffeY8CAAXz55ZcAU6Kf3ToBS9x9Sb7XzwI2RLflGQzsS1AgHOTRuzPMLBV4ERji7mv27J3GHndf53t2J0qmu69w91/cfYq7/xM4maBwO6jgzmY2MDocdft86x6PXq9ruvtGdz/K3Ye7+xx3nwBcA+wX/T1CRERERERE9kBJi7XNgKLGm9tCcAdvsZhZXWAckEXwwbEzQffu+iJe0wb4kODu4O7Af4FnzeyY4p5XpDob0KUxI67uQ9sGtVi+cTtnPfkNr3+3OOxY1UMkAsfeB/3/FDz/+t/w/nWQkx1uLqlObgJSgFVADeAr4CeCa/ufSnisvsBjwEEEw1cmAB+bWa1dvcDM+gCvAs8BPYARwAgz61LCc0s1l5wQR5v6wV+12Ss0b61UUzvnrZ1V9H57IG8o5Lkb5ha536gFozj+neM55LVDOPCVA/nT2D+xMXMjycnJHHXUUdx5552MHz8egg7Zv5QgQjegVnTJ/zmzHUHX6Ptmlm1m2QSduSdFn7crwTkqnJm9XdylrM4Z7dKdTtAVW3DbMIL56182s3gzOx64FDjf3bfu4pC1AScosBfKzJLMLC1vAVL38G2IiIiIiIhUSSUt1q4mGO5oVzoCJbmz+VaCO6ovdvdv3X2Bu3/s7j8X8ZorgAXufpO7/+jujwJvAjeW4Lwi1dpeDVN59+qDOapzI3bk5HLrWzO4c8RMzWNbEcyg7y1wwn+D4QunDIPhAyFrW9jJpBrI64wBTiSYB+9R4Dh37+vuGSU81gB3f8Hdf3D36QSdOi2B/Yp42fXAKHf/d/QaficwhaA7R6REOjbWvLVSzeWft7acdKjbASi6s3b0wtHc/PXNO4dKzs7N5sP5H3LJ6EvIyc0puPssgsLrj0ALM2uRt8HMOgN1ovtgZukExd1/RH++bGY1orvPJhhGuXu+5T1+vaF3Z8dujNpYgqUszSYochfmcoKC+MMEN1X92d0nF7ajmSUTjMrxamFTJOVzG799LyUbT1tERERERKSaKOl8NZ8SdN6MKrjBzCy67dMSHO8kYLSZvUHQofML8Li7P1PEa3oXco7RBB22IlJMqckJPHXBfjz6xU888MlcXpywiDkrN/P4+T2pn5IUdryqb/+LoWa9YP7aOR/Ci6fCua9CjbphJ5NqwN3HEswDWJZqR3+uK2Kf3sADBdaNBk4p4yxSDXRsnMoH3y/n67mraZleM+w4lU7NxDj6tKtPYnxJ792UmJFXrC2nYZDh187amWtm8vHCj3+33d2597t7g8f8OmpvjucwZ/0c/vryXzm92+mkpqYyadIkgFsIptX5FJhBUIC9geBz6ePAV+4+KXqYJwmKrn8HkoCpwP3A1e6+HZiZP4uZbYhm+s36WOTuF4d0agMKHV7Z3deb2SUE1+XxwL8KPUAwlcLw6LGu3M357uG31/1UVLAVERERERH5nZIWa/8OTDazicB/gDnR9R0JhlbsQCFz4BShLcEHvAeAfwIHAA+b2Q53H7qL1zQGVhZYtxJIM7Ma7v6b9jQzSyL4cJ9HQy+JREUixnVHtKdzkzRueH0a3y5Yx0mPjOWpC/ena/Pauz+A7JnOJ0HNt+HV82DxN/C/AXDBW1C7edjJpIoys+t2scmB7QRDIn/t7r9rhdrNcSMEN02N282X5Lu6hjcu4ti6jkuhOjZOA2D8z2sZ//PakNNUTrcf15HLDovp0WKlKHnDIK+eA7k5EIkr81PkddYuz1jOTV/dVKLXmhsjJo7goWsfIisrixYtWgA8A/zT3d3MTgYeAb4GcgluCL4WgjlUgeOAHu6eDWSb2QXAWDP7wN1Hls07rHY6AQuK2H4YkEPQYVuLAlMg5SvUtgIO301XLe6eCeycWDy4v1tEREREREQKKlGx1t1/NrMjCYaheo1f78o1guGqjnL3n0pwyAgwyd1vjz6fGp237gpgV8XakroNuLuMjiVSJR3ZuREjrj6Yy4ZNYv6aDM54cjz/PrMbJ3VrGna0qq/1ITB4JLx0OqyeDc8eBRe+DQ07hZ1MqqYbgQZATX6dH74usJVg3vmGwHwz6+/uJRlC8jGgC3BIGWbNo+u4FOrQDvU5rUczlm7QMPIltWrTdhau3cr3S8t6hFWpUHVaQlwi5OyAjUuhbqsyP0X9GvW5qttVTFwxsdDt7s7UVVN/01W7k8FNV97EwEcH5l97Z77XLgZO3sVxhwHDCqz7FkjcVVZ3H7TLNxLDzGwBu+h2BXD3tmV0nsMJho5+cBfb+xBMU3QiwRDHjwIX5dueV6htD/R3d90lIyIiIiIiUkZK2llLdFiqLmbWA9iLoFA7192nleL8y4nOSZTPj8DpRbxmBdCowLpGwKaCXbVRGnpJpBj2apjCiGsO5vpXp/LFnNVc9+pU5qzYxE1H7U0korvgy1WjfeCST4KC7Zo58L9j4JxXgkKuSNm6HbgMuDRvfngz2wt4CngaGEdwM9aDwBnFOaCZPQqcABzm7ru7vu7qGr6iiNfoOi6FSoqP44Gzu4cdo1IaNXMFV7w0mcXrtoYdRfZEJA7qtIK182D9gnIp1gJc2f1KrixitNu/T/g7b8x9g1zP3bnOMOIicRzX9rhyyVTF/LfA8wSgBzAA+Hcpj5lkZo2BOILr7ACCm58+oEARHMDMUoEXgYfdfaSZLQW+M7P33f3NaKH2TaAnwTU/Lnp8gHXuvqOUOUVERERERISgs7VU3H2qu7/h7sNLWaiF4EvhvQus6wAsKuI13wBHFFh3VHR9YTkz3X1T3kKBoZxE5FdpyQk8e9EBXN43uIH/sS9+5rIXJ7ElMzvkZNVAnRYweBS0OAi2bwzmsJ35dtippOr5O3BjXqEWIDoixh+Be6LF1luAg3d3IAs8CpxKMBRiUcMq5inRNTyaT9dxkTLWun4wx++itSrWVnrpbYKf6+aHFuGGnjewb/19AYizOAwjPhLPfYfdR/0a9UPLVVm4+0MFlvvd/XzgLn7/Wbm4BhDcGL2QYHjp/sB1wMm7mOrgISCD4KYu3H1G9PFTZtYMaAacBDQHpkWPnbf0KWVGERERERERiSpxZy2AmT2wi03557x7193X7eZQDwLjzex2giGVehF0/FyW71z3AM3cPW/8rCeBa8zsPuB/wOHAWcDxpXkvIvJbcRHjtmM70bFxKre+NYNPf1zF6Y+P59mL9qdFes2w41VtNdNh4Ah4ewj8+D68eTFsWgZ9rgk7mVQdTSj82h/Pr/PGLqN488I+BpxHMITl5nwdNhvzRrows2HAL+5+W3TbQ8BXZnYT8CFwDrA/+a77IlL+Wkav5xu3ZbFxaxa1ayaEnEhKLT06Qu664twvUz5SElMYeuxQJiyfwPRV06mdVJsBbQaQnpweWqYqYiTB6BIXl+RF0eGgBxVjP8v3eHAh2x/gtyNbaKgdERERERGRclLaztoewCUEX672jS5DouuOIPhQ95OZdS7qIO7+HUFHzrnATII5jG5w95fz7dYEaJnvNQsICrNHAdOBmwiGcxxdyvciIoU4tUdzhl/emwapScxZuZmTHxvHdwt3d/+F7LGEGnDmUDjwiuD5x3+CkbdCbmFNECIl9gVBl0yPvBXRx08An0dXdQWK863/lUBt4Et+22Fzdr59WhJcxwFw9/EEBd7LCK7hZwCnuPvM0r0dESmNmonxNEhNAmDRuoyQ08geqRvtrF0fXrEWIGIR+jTtw5Xdr+S8TuepUFs2zgD0y7eIiIiIiEg1UKrOWuBdgg+OF0eHJMTMagPPAmOBZ4BXCDpnjynqQO7+AcHcObvaPqiQdV8SFIxFpBx1b1GH9645mCHDJjHzl02c98wE/nFqV87av0XY0aq2SBwM+BfUbg4f3wETn4SNS+G0ZyBR3c2yRy4hmJNuspllRdfFA59FtwFsIbgRqkj5O3KK2KdfIeveAN4oZl4RKSet0muyenMmC9duZd/mdcKOI6W1cxjkhaHGkNIzs6kEI1TtXEUw2kUD4KpQQomIiIiIiEiFKm2x9mbgqLxCLYC7bzSzPwMfu/tDZvZX4OMyyCgiIWpSuwbDL+/NH9+YzkczVnDLm9/z8+ot3HpMRyIRjYZWbsygz7WQ1gzeuQJmfwAvHA/nvQ4pDcNOJ5WQmRmQSDDnXEt+nQdvjrvPydvP3b8IIZ6IVLCW9WoyadF6Fq9VZ22llr+z1j34/UEqmxEFnucCq4Ev3X12xccRERERERGRilbaYm1toCEwq8D6BkBa9PEGgi+FRaSSq5kYz6Pn9uTBBnN55POfeOqr+Sxck8GDZ3enZmJp/xmRYulyGqQ1hVfPhWVT4Nkj4Pw3ocHeu3+tyG8ZwZzy+0SLs3N2s7+IVGGt69UCYNHarSEnkT1StxVgsGMLZKzWDV2VkLv/JewMIiIiIiIiEq7Szln7LvA/MzvVzJpHl1OB5/j1zuBewNwyyCgiMSASMW46em8ePLsbiXERRv+wkrOe+oaVm7aHHa3qa3kQXPpp0D2zYTE8exT8rOZHKRl3zwXmAfXCziIi4WtVLxhWf9E6FWsrtfikYNoEgHXhzlsrpWNmabtYUs1MNz+LiIiIiIhUA6Ut1l5OML/da8Ci6PJadN0V0X1mA5fuaUARiS2n9mjOK0MOJL1WIjN/2cQpj43jx+Wbdv9C2TP12sGln0HL3pC5EV46HSa/EHYqqXz+D/i3mXUJO4iIhKtlelCsXazO2sqvbuvg53oVayupDcD6QpYNwDYzW2RmfzGz0n52FxERERERkRhXqg987r7F3YcQdOf0iC713P0yd8+I7jPN3aeVWVIRiRn7t05nxFUH065BLZZv3M4ZT4znyzmrwo5V9dWqBwPfha5ngefA+9fDx3dAbk7YyaTyGEYw8sV0M9tmZuvyL2GHE5GK0yo6DPKKTdvZnqXrSKWWHp23Vp21ldUgYBnwT+CU6PJP4BfgSuBp4DqCG65ERERERESkCtqjySbdfUvel7vuvqVsIolIZdCyXk3evvJgrnhpMt/MX8slQyfxl5P24YKDWoUdrWqLT4LTng46bb+8B8Y/Amt+gtOfgaTUsNNJ7Lsh7AAiEhvq1kwgNSmezZnZLF63lQ6NdA2ptOpGi7XqrK2sLgJucvfh+da9b2YzgMvd/QgzWwz8iaCIKyIiIiIiIlVMqTprzSxiZneZ2UaiwyCb2QYzu1PDM4lUH7VrJjB0cC/O2K85ObnOHSNmcs/IH8nN9bCjVW1m0O//4PTnIC4J5o6E546B9YvCTiYxzt2HFrWEnU9EKo6Z0ap+dN5aDYVcue3srJ0fbg4prT7A1ELWTwV6Rx+PBVpWWCIRERERERGpUKUtrP4DuIZgKKa8YZBvB64F/lY20USkMkiMj/DvM/blxiM7APDUV/O5/vVpZGZrSMVy1/UMuPgjSGkEq36AZw6HRd+EnUoqCTNLNrO0/EvYmUSkYrVKD4ZCXrQ2I+QkskfqahjkSm4JcEkh6y+JboNg+qH1FZZIREREREREKlRpi7UXAZe6+xPu/n10eRwYQjDnjohUI2bG9Ue25/4zuxEfMd6fvowLn/2WDVt3hB2t6mu+Pwz5HBp3ha1rYOiJMPmFsFNJjDKzWmb2qJmtAjIIvvjNv4hINdKyXtBZu3idOmsrtbzO2q1rIHNzuFmkNP4I3Ghm083s2egyjWDqgpui+xwAvB5SPhERERERESlnpS3WpgOzC1k/O7pNRKqhM/ZrztDBvUhNiufbhes448lv+GXDtrBjVX21m8Pg0dD5FMjNgvevhw//CDlZYSeT2HMfcDhwJZAJXArcDSwDBoaYS0RC0Co9KNYu1DDIlVtybahZL3is7tpKx93fAzoCIwk+S6dHH3d09w+i+zzh7n8IL6WIiIiIiIiUp9IWa6cTDINc0DXA96WPIyKV3cF71eeNK3vTOC2Zn1Zt4bTHxzFr2aawY1V9ibXgzBfg8DuC5989Ay+eChlrQo0lMedE4Cp3fwvIBsa4+98JpjI4P9RkIlLhdnbWahjkyi9vKOT1KtZWRu6+wN3/z91Piy63ufvCsHOJiIiIiIhIxShtsfYWYLCZzTKz56LLLIIhkP9YZulEpFLq2DiNt6/qQ4dGKazclMlZT33DuJ9UNCx3ZnDYzXDOq5CYCgvHwFN9YdnUsJNJ7EgH5kcfb+LX0TDGAoeFkkhEQtOqXjBn7dL128jOyQ05jeyRvKGQ180vej+JSWZ2qJm9ZGbjzaxZdN2FZnZI2NlERERERESk/JWqWOvuXwEdgHeAOtHlbWAf4MIyyiYilVjTOjV444o+HNgmnS2Z2Qx6/lven74s7FjVQ8fj4NJPod5esGkpPHcMTH057FQSG+YD0W/0mQ2cFX18IrAhjEAiEp4mackkxkfIznWWb9wedhzZE3mdtRoGudIxs9OB0cA2oCeQFN1Um2DkCxEREREREaniSttZi7svc/c/ufvp0eUOoC5wSdnFE5HKrHaNBIZd0ovjuzYhK8e57rWpvDBOXyJWiIYdYcjn0OFYyMmEd6+CD/4A2ZlhJ5NwPQ90iz7+F3C1mW0HHgT+HVoqEQlFJGK0qFsDgEWat7ZyS28b/NQwyJXRHcAV7j4EyMq3fhxB8VZERERERESquFIXa0VEiiMpPo6Hz+3BRb1b4Q5/fn8W942ajbuHHa3qS64N57wC/W4HDCY9B88fCxuWhJ1MQuLuD7r7w9HHnwIdgfOAHu7+UKjhRCQUeUMhL9S8tZXbzmGQF4YaQ0plb+DrQtZvJBjBSkRERERERKo4FWtFpNzFRYw/n7QPNx+zNwCPf/kzt771vebHqwiRCPS7Fc4bDsl14JfJ8NRh8NNnYSeTEJjZQDPLG14Rd1/k7m8Ds81sYIjRRCQkLdNrArB4nTprK7W8YZA3LYXsHeFmkZJaAexVyPpD+HWeeREREREREanCVKwVkQphZlzdfy/uPb0rEYPhk5Zy1ctT2J6VE3a06qHD0XD519CkO2xbBy+dDl/8E3L151/NPE8wB15BqdFtIlLNtKoXFGsXqbO2cktpCAm1wHNhw+Kw00jJPAM8ZGYHAg40NbPzgf8AT4SaTERERERERCpEfEl2NrO3d7NLndJHEZHq4OwDWlKnZiLXvjqVj2etZNDz3/LMwP1JTU4IO1rVV7cVDB4No26FyS/AV/fCovFw+rOQ2jjsdFIxjOCL4IKaEwy3KCLVTOvoMMias7aSM4O6rWHVD7BuPtQvrFFTYtS/CG6i/gyoSTAkcibBXPLPhphLREREREREKkiJirXs/ovcjcCwUmYRkWrimH0aM/TiXgwZNokJ89dxztMTGDq4F/VTknb/YtkzCclw4kPQ6hB4/3pYOAaePBROexra9Q87nZQTM5tKUKR14DMzy863OQ5oA4wKI5uIhKtlvV+HQR79w4rd7m/AAa3TqVsrsZyTSYmltwmKtesXhJ1ESsDdHfiHmf2bYDjkFGAWcDmwANAddSIiIiIiIlVciYq17n5xeQURkeqld7t6vHbZQVz0v2/5YdkmznryG1689ECa1akRdrTqYd8zoWl3GH5R8MXui6fCITdC/9shTl3OVdCI6M/uwGhgS75tO4CFwFsVmkhEYkLzujWIGGzdkcPlL04u1msObJPO65f3LudkUmLp0Xlr16lYWxlE55D/M3AU0U5adx9hZhcD7wA5wIPhJRQREREREZGKUtLOWhGRMtOlWW3euKI3Fz73LfPXZHDmE+N58dIDadcgJexo1UP99jDkMxj1f8GwyGMfCDptT382GEpRqgx3/wuAmS0EXnf37eEmEpFYkRQfxy0DOvJxMbpqM7Nz+WHZJuas3FwByaTE0poFPzcvDzeHFNdfCbpnPwX6AG+Y2fPAQcBNwBvunhNiPhEREREREakgKtaKSKjaNkjhjSt6c8FzE5m/OoOznvyGoYN70aVZ7bCjVQ8JNYJhkdv2h/evg6XfBcMiH/9A0H0rVYq7Dy2rY5nZYcDNwH5AE+BUdx9RxP79gC8K2dTE3XdfJRKRcnNF33Zc0bfdbvdbn7GDHn/7hA1bs8jMziEpPq4C0kmxpTQKfm5ZGW4OKa4zgYHu/p6ZdQG+J/h83i06NLKIiIiIiIhUE5GwA4iINK1Tgzcu702XZmmszdjBuc9MYNLCdWHHql72OQWuGAstDoLMTfD2pfDmJbBtQ9jJJHbVAqYDV5fwdXsTFHfzllVlnEtEykmdmgkkxgUfH1Zvzgw5jfxOanRq0826/6WSaA5MBnD3mQRDIT+oQq2IiIiIiEj1o2KtiMSEeilJvDLkIHq1Tmfz9mwufO5bxs5bE3as6qVOSxj0IfT/E1gczHwTnjgYFowJO5nEIHcf6e53uPs7JXzpKndfkW/JLZeAIlLmzIwGqUkArFKxNvbk76xVva8yiCOYNz5PNr+dU15ERERERESqCRVrRSRmpCUnMHRwLw7r0IBtWTkMfuG7Ys2hJ2UoLh763gKXfAzpbWHTUhh6Aoy6DbK2hZ1OqoZpZrbczD4xs4N3t7OZJZlZWt4CpFZARhHZhZ3F2k0q1sacvM7arK2QqXmFKwEDXjCzt83sbSAZeDLveb71IiIiIiIiUsWpWCsiMaVGYhzPDNyPAfs0ZkdOLle+PIV3p/0Sdqzqp/n+cPkY6HlR8HzC48Fctksnh5tLKrPlwBXA6dFlCfClmfXczetuAzbmW5aWZ0gRKVrDaLF29ebtISeR30msBYnR+1k0b21lMJRgKoC869tLwDJ+e83bGFo6ERERERERqTDxYQcQESkoKT6OR8/rwS1vfc/bU37hhtenkZmVy1kHtAg7WvWSlAInPQwdT4D3roW18+C5o+Dg66HvrZCQHHZCKSEze2AXmxzYDvwEvOvuZT5ptLvPAebkWzXezNoBNwIXFvHSe4D8uVNRwVYkNA3TNAxyTEttBGs3B/PW1m8fdhopgrtfHHYGERERERERiQ3qrBWRmBQfF+H+M7px3oEtcYdb3vqeoeMXhh2reupwNFz1DXQ5AzwHxj4ATx0GS74LO5mUXA/gEuAyoG90GRJddwRBUfQnM+tcQXm+BfYqagd3z3T3TXkLoLE9RULUMDW4UUfDIMeolOhQyOqsFREREREREak0VKwVkZgViRj/OKULgw9uA8Dd7/3AU1/9HHKqaqpmOpzxHJz9EtRqCGvmBF22o26HHRlhp5Piexf4FGjq7vu5+35Ac+AT4FWgGfA18GAF5elOMDyyiFQSO4dB3qJibUxKbRT83Lwi3BwiIiIiIiIiUmwq1opITDMz7jyhE9f0D5rv7hk5m4c/mxdyqmqs04lw9UTodh7gMOExeOwgmPdJ2MmkeG4G7ox2qALg7huBPwO3uPtW4K/Afrs7kJmlmFl3M+seXdUm+rxldPs9ZjYs3/43mNnJZraXmXUxs/8ChwOPldF7E5EK8OswyJqzNibt7KxVsVZERERERESkslCxVkRinpnxx2P25o9HdwDggU/mcv/oObh7yMmqqZrpcOoTcP6bULslbFwML58Bb14CW1aFnU6KVhtoWMj6BkBa9PEGILEYx9ofmBpdIBhCeSpBsRegCdAy3/6JwH+AGcBXQDfgSHf/rPjxRSRsGgY5xu3srNUwyCIiIiIiIiKVhYq1IlJpXHN4e24/riMAj37xE/eMnK2CbZjaHxXMZdv7GrAIzHwTHt0fvnsWcnMqLIaZuZmdUmEnrNzeBf5nZqeaWfPocirwHDAiuk8vYO7uDuTuX7q7FbIMim4f5O798u1/n7vv5e413L2eu/d39y/K+g2KSPnKGwZ5zZZMcnJ1DY456qwVERERERERqXRUrBWRSuWyw9rxl5P2AeDpr+fzl/dnqWAbpqQUOOYfMORzaNINtm+ED2+CZ4+AXybv0aEHDRqEmWFmJCQk0KhRI4466ijMbLCZler6ZWYJZnavmc0wswwzW2Zmw8ys6R6FrTwuBz4DXgMWRZfXouuuiO4zG7g0lHQiEvPqpSQRMch1WJuh7tqYo85aERERERERkUpHxVoRqXQu6tOaf57aFYAXxi/kzndnkqvunnA17QFDvoDj7oekNFg2FZ45At67DjLWlPqwAwYMYPny5SxcuJCRI0fSv39/gIeAD8wsvhSHrAn0BP4W/XkasDfwXqlDViLuvsXdhwD1gB7RpZ67X+buGdF9prn7tBBjikgMi4sY9VKi89ZqKOTYo85aERERERERkUpHxVoRqZTOO7Al952xL2bw0oTF3P7ODBVswxaJg15D4JpJ0PUswGHKUHi4J0x4EnKyS3zIpKQkGjduTLNmzejZsye33347wMnAscCggvub2UAz22Jm7fOte9zMZptZTXff6O5Huftwd5/j7hOAa4D9zKxlweNVVe6+BVgHrIs+FhEptryhkFdvVrE25uR11m7fCFnbws0iIiIiIiIiIsVSmq6kMmNmfwbuLrB6jrt33MX+g4DnC6zOdPfksk8nIrHurP1bkBBn3DR8Oq99t4TsXOfe0/clLmJhR6veUhvB6c/A/oNh5M2wYgaMuhUm/S8YMrn9UXt0eHf/3MymE3TFPltg2zAzOwF42cz6AMcQDOnb29237uKQtQEHNuxRsEogOnz0HcBNQEp03WbgP8A/3D03xHgiUkk0iBZrV23eHnIS+Z3kOhCXBDmZsGUl1G0ddiKR36llm0nRbeMiFaLuS/8MO0KlsPGCW8OOICIiItVcLHxE+gFokm85ZDf7byqwf6tyTSciMe3UHs357zk9iIsYb05eys1vTCdHHbaxoVVvuOwrOOFBqJEOa+bAy2fAi6fByll7evTZQOtdbLuc4PrwMPAc8Gd3L3QCXTNLBu4FXnX3TXsaqhL4B0En8f/x6zDItwPXEgwNLSKyW3mdtRoGOQaZad5aERERERERkUomFoq12e6+It+yu8kNvcD++hZCpJo7qVtTHjk3KNi+PfUXbho+jewcNQjGhEhc0GF73VTofQ1EEuDnz+DJg+Hda2DjL7/dPzcXMrcEP4tmBN2wv+Pu64FLgCuBn4F/FXoAswRgePRYV5bkbVViFwGXuvsT7v59dHkcGEIhw0qLiBSmYWowqM0qDYMcmzRvrYiIiIiIiEilEgvF2vZmtszM5pvZy8WYMzDFzBaZ2RIze9fM9ilqZzNLMrO0vAVILbvoIhIrjuvahEfP7UF8xBgxbRk3Dp+ugm0sqVEnGAL5mm+h00nguTD1RXikJ3xyN2xdC2P/C/e3h3uawb/bcULtOUQKr8cCdAIWFHHGw4Acgg7bWgU35ivUtgKOqiZdtQDpBF3JBc2ObhMR2a2GaRoGOaaps1ZERERERESkUgm7WDuRoJNnAEFXUxtgjJntqqA6BxgMnAxcQJB/vJk1L+IctwEb8y1LyyS5iMScY7s24dHzehIfMd6fvozrX1eHbcxJbwtnvwiXfAIt+0D2dhj3X/hPR/j0btgaHVxh2zpOrzOLi5vN+90hzOxwoCvwVmGniM5VeytwIrAFeLTA9rxCbXvgSHdfW2bvL/ZNJxgGuaBrottERHZr5zDI6qyNTeqsFREREREREalU4sM8ubuPzPf0ezObCCwCziKYZ7Dg/t8A3+Q9N7PxwI8E8xPeuYvT3AM8kO95KirYilRZA7o05vHze3L1K1P48Pvl4PDfc7qTEBf2vSnyGy16wcUfwdzR8PEdsLawoiwcV/8XVs2fQVZSOitXrmTUqFEA7wIfAMN+/xpLBV4EHnb3kWa2FPjOzN539zejhdo3gZ7ACUCcmUW/1Wadu+8ol/cbO24BPjSzI/n1etobaAEcF1oqEalUGuQNg6w5a2OTOmtFREREREREKpWYql64+wZgLrBXMffPAqYWtb+7Z7r7prwF2FwWWUUkdh29T2OeOH8/EuKMD2cs5/rXppKlDtvYYwZ7D4ABhU4pC0CcwcUDetK6dWsGDBjAF198AXAdcLK75xTykoeADOB2AHefEX38lJk1A5oBJwHNgWnA8nxLn7J6a7HK3b8COgDvAHWiy9vA3u4+JrxkIlKZ5HXWrt6cifsuh6uXsKizVkRERERERKRSCbWztiAzSwHaEXRFFWf/OIKhMD8qz1wiUvkc2bkRT5y/H1e+PJmPZqzAfSoPn9tDHbaxqFa9Ijd/+Nl4aHFA/lXP53/i7pbv8eCCr3f3B/jtCAtWcJ/qxN2XAX/Kv87MmpvZ0+5+WUixRKQSaRAt1u7IyWXjtizq1EwMOZH8Rmq0WKvOWhEREREREZFKIdSqhZndb2Z9zax1dI7Bd4Ac4NXo9mFmdk++/e8ys6PNrK2Z9QReAloBz4aRX0Ri25GdG/HkBfuRGBdh5MwVXPeqOmxjUpPuUL8DWNxv11sc1G0DzfYLJVY1Uw+4JOwQIlI5JCfEUbtGAqB5a2NSSnQYZHXWioiIiIiIiFQKYbeYNScozM4BhgNrgYPcfXV0e0ugSb796wLPEMxT+xGQBvRx91kVllhEKpUjOjXiyQt7qmAby8zgzKFQo27wPBIUAEiuDWe/CJGwL1UiIlJQ3lDImrc2BuV11masgZzscLOIiIiIiIiIyG6FOgyyu5+zm+39Cjy/EbixPDOJSNVzeMegYHvFi1N2Fmw1JHKMadQZbvgefhgBa+dBelvY5zRISgk7mYiIFKJhWhLzVm1h1ebtYUeRgmrWD0an8BzIWAVpTcNOtEtm5sCp7j4i7CwiIiIiIiIiYVGlQkSqhbyCrTpsY1hiLehxPhz5Z+g5UIVaEZEY1jA1GdAwyDEpEoGUhsHjzeEMhTxo0CDMDDMjISGBRo0acdRRR2Fmg82sVJ9BzSzBzO41sxlmlmFmy6LT5sRuNVpERERERESkGFSsFZFq4/COjXjqwl/nsL3+NRVspXows7eLWoAHw84oIpWLhkGOcTvnrV0ZWoQBAwawfPlyFi5cyMiRI+nfvz/AQ8AHZlaaEZ5qAj2Bv0V/ngbsDbxXVplFREREREREwqBirYhUK/07NtzZYfvRDBVspdrYuJtlETAstHQiUuk0yCvWahjk2JQ3b21InbUASUlJNG7cmGbNmtGzZ09uv/12gJOBY4FBBfc3s4FmtsXM2udb97iZzTazmu6+0d2Pcvfh7j7H3ScA1wD7mVnLCnpbIiIiIiIiImUu1DlrRUTCkH8O249mrACm8tA5msNWqi53vzjsDCJStTRM0zDIMS0GOmsL4+6fm9l0gq7YZwtsG2ZmJwAvm1kf4BjgUqC3u2/dxSFrAw5sKL/UIiIiIiIiIuVLlQkRqZYO79iIJy74tcP2htemqcNWRESkmPKGQV6tYm1sioHO2iLMBlrvYtvlQBPgYeA54M/uPrmwHc0sGbgXeNXdN5VDThEREREREZEKoWKtiFRbR3T6tWD74YzlKtiKiIgU069z1moY5JgUo521UUbQDfs77r4euAS4EvgZ+FehBzBLAIZHj3Vl+cQUERERERERqRgq1opItaaCrUjpmNlhZva+mS0zMzezU4rxmn5mNsXMMs3sJzMbVP5JRaQ85A2DnLEjh4zM7JDTyO+E0Vm7ajZMeRF+/IAEyylqz07AgiK2HwbkEHTY1iq4MV+hthVwlLpqRUREREREpLJTsVZEqj0VbEVKpRYwHbi6ODubWRvgQ+ALoDvwX+BZMzumnPKJSDlKSYqnZmIcoHlrY1JKtFhbEZ212ZnwxiB4/EB47xp4/XwebD6SzrU2/G5XMzsc6Aq8VdihonPV3gqcCGwBHi2wPa9Q2x440t3XluVbEREREREREQlDfNgBRERiQV7B9sqXpvDhjOU4zkPn9CAhTve0iBTG3UcCIwHMrDgvuQJY4O43RZ//aGaHADcCo8slpIiUq4apSSxcu5VVm7bTpv7vGiAlTKn5hkHOzYVIOf4+8+U9MOvd36yqGcnijrbTWbloDtnxKaxcuZJRo0YBvAt8AAwreBgzSwVeBB5295FmthT4zszed/c3o4XaN4GewAlAnJlFq9Ksc/cd5fYeRURERERERMqRirUiIlH5C7YfzViB+1QePlcFW5Ey0hv4tMC60QQdtrtkZklAUr5VqWUbS0RKq2FqclCsVWdt7KnVMPiZmw1vXgxxCeVzHneYNQL8tyOSRAySIzncdEZXnp3m1K1bl27dugFcBwx198KGMHkIyABuDw7tM8zsduApM/sGSABOiu47rcBr+wNfls2bEhEREREREalYKtaKiORzRKdGPHlhT654cQojZ67guldVsBUpI42BguNxrgTSzKyGu2/bxetuA+4u12QiUir1UxMBWLtFxdqYE58IdVrBhkVBMTUEkbhEnrjnJp448jf/hD+f/4m7W77Hgwsew90fAB7It6pYQzmIiIiIiIiIVCYq1oqIFHB4x0Y8deF+XP7iZEbOXME1r0zhkXN7khivgq1ICO7ht1/UpwJLQ8oiIvmkJQfdmpu3Z4ecRAp17msw/4vyPYc7fHUvZG76/bbcLGi0T/meX0RERERERKQKULFWRKQQ/Ts25KmBQcF29A8rufqVKTx2ngq2IntgBdCowLpGwKYiumpx90xgZ9teMefHFZEKkJocfJTYnKlibUxq1DlYyltCDfjwD79dZ3FQuxl0OrH8zy8iIiIiIiJSyanqICKyC/33bsjTF+5HYnyET2at5KqXJ5OZnRN2LJHK6hvgiALrjoquF5FKKCVJnbUC7D8Yjv47JKX9uq5Vbxj0IcQn7fp1IiIiIiIiIgKoWCsiUqR+ezfk2YH7kxQf4dMfV3HVS1NUsBUBzCzFzLqbWffoqjbR5y2j2+8xs2H5XvIk0NbM7jOzjmZ2FXAW8GDFJheRsrKzs3Z7VshJJFRm0Oda+OM8uGIc3DgrKNTWaRl2MhEREREREZFKQcVaEZHdOKxDA569KCjYfjZ7FZcNm8z2LBVspdrbH5gaXSCYV3Yq8Nfo8ybAzm/q3X0BcDxBN+104CbgUncfXVGBRaRs/VqsVWetAAnJ0LhLMPyxiIiIiIiIiBSbirUiIsVwaPsGPD/oAJITInw1dzVDhk1i2w4VbKX6cvcv3d0KWQZFtw9y936FvKaHuye5ezt3fyGE6CJSRvKKtVs0Z62IiIiIiIiISKmpWCsiUkx99qrPCxf3omZiHGPmrWHwC9+xdYe+oBYRkeopNTlvzloNgywiIiIiIiIiUloq1oqIlMBBbesxbHAvUpLi+Wb+Wgb97zt1FImISLWUkhTtrNUwyCIiIiIiIiIipaZirYhICe3fOp1hl/QiNTmebxeu48LnJrJxm7qKRESketGctSIiIiIiIiIie07FWhGRUujZsi6vXHoQtWskMHXxBs5/dgLrM3aEHUtERKTC5A2DvGVHNrm5HnIaEREREREREZHKScVaEZFS6tq8Nq8OOYh6tRKZ+csmzn1mAqs3Z4YdS0REpELkdda6Q4bmcBcRERERERERKRUVa0VE9kDnpmm8dtlBNEhNYvaKzZz99Dcs37gt7FgiIiLlLik+QkKcARoKWURERERERESktFSsFRHZQ+0bpTL88t40rZ3M/NUZnPnkNyxeuzXsWCIiIuXKzH4dCjlTxVoRERERERERkdJQsVZEpAy0qV+L4Vf0pnW9mixdv40znxrPT6s2hx1LRESkXKUkBUMhb96eFXISEREREREREZHKScVaEZEy0rxuTYZf3psOjVJYuSmTs56awMxfNoYdS0REpNzkzVu7ScMgi4iIiIiIiIiUioq1IiJlqGFaMq9d1puuzWqzLmMH5z49gW8XrAs7loiISLnI66zdomKtiIiIiIiIiEipqFgrIlLG0msl8sqQA+nVJp3Nmdlc+NxEvpizKuxYIiIiZS5vztrNKtaKiIiIiIiIiJSKirUiIuUgNTmBYYN70X/vBmRm5zJk6CTem74s7FgiIiJlKi1Zc9aKiIiIiIiIiOwJFWtFRMpJckIcT124Pyd2a0p2rnP9a1MZOn5h2LFERETKTEq0WLslU521ImEyswZm9oSZLTazTDNbYWajzezgcjjXOWbmZjairI8tIiIiIiJSHcWHHUBEpCpLjI/w37O7U7dmAsO+WcTd7/3A2owd3Hhke8ws7HgiIiJ7JHVnZ62KtSIhewtIBC4C5gONgCOAemV5EjNrDdwPjCnL44qIiIiIiFRn6qwVESlncRHjLyftw41HdgDg4c/mcceImeTkesjJRERE9kxKkuasFQmbmdUBDgVudfcv3H2Ru3/r7ve4+3vRfVqa2btmtsXMNpnZcDNrFN3W0cy2mtl5+Y55lpltM7PO+dbFAS8DdxMUhEVERERERKQMqFgrIlIBzIzrj2zP307pghm8PHExV740me1ZOWFHExERKbVUzVkrEgu2RJdTzCyp4EYziwDvAulAX+AooC3wOoC7zwb+CDweLeo2B54kKP7Oyneou4BV7v5ceb4ZERERERGR6kbFWhGRCnThQa147LyeJMZH+HjWSs5/diIbtu4IO5aIiEipaBhkkfC5ezYwiGAI5A1mNs7M/mlm+0Z3OQLoCpzn7pPdfSIwEOhrZgdEj/E4MBZ4CXgB+A54JO8cZnYIcAkwpLi5zCzJzNLyFiB1z96piIiIiIhI1RRqsdbM/mxmXmCZvZvXnGlms81su5nNMLPjKiqviEhZOK5rE14c3Iu05HgmL1rP6U+MZ8m6rWHHEhERKbG8Yu2WTBVrRcLk7m8BTYGTgFFAP2CKmQ0COgFL3H1Jvv1nARui2/IMBvYFegKD3N0BzCwVeBEY4u5rShDrNmBjvmVpKd6aiIiIiIhIlRcLnbU/AE3yLYfsakcz6wO8CjwH9ABGACPMrEv5xxQRKTsHtq3Hm1f2oUntZH5encGpj4/n+6Ubwo4lIiJSIqnJeXPWahhkkbC5+3Z3/8Td/+bufQg6ZP9SgkN0A2pFlyb51rcDWgPvm1m2mWUTdOaeFH3ebhfHuweonW9pXpL3IyIiIiIiUl3EQrE2291X5FuKulP3emCUu//b3X909zuBKcA1FRNVRKTsdGiUyttX9aFj41TWbMnk7Kcm8PEPK8KOJSIiUmzqrBWJabMICq8/Ai3MrEXeBjPrDNSJ7oOZpRMUd/8R/fmymdWI7j6bYBjl7vmW94Avoo93duzm5+6Z7r4pbwE2l91bExERERERqTpioVjb3syWmdl8M3vZzFoWsW9v4NMC60ZH14uIVDpNatfgjSt6c1iHBmzLyuHylybzv7ELiI46JxLzzOxqM1sYnZ5gopn1KmLfQYVMf7C9IvOKSNlKSQqKtZs0Z61IaMysnpl9bmYXmNm+ZtbGzM4EbgHeJfgMPYOgANszeq0eBnzl7pOih3mSoOj6d+APQBxwP+zs2J2ZfyEYQnlz9PmOiny/IiIiIiIiVU3YxdqJwCBgAHAl0AYYE50TpzCNgZUF1q2Mri+UmSWZWVreAuzq2CIioUhNTuC5i/bn3F4tcYe/fjCLO9+dSVZObtjRRIpkZmcDDxAMsdgTmA6MNrOGRbxsE7+d/qBVeecUkfKTNwzyjuxcMrNzQk4jUm1tIfhsfSPwNTAT+BvwDHBNdO7Zk4H10e2fAvOBswHMbCBwHHChu2e7ewZwATDEzI6t4PciIiIiIiJS7cSHeXJ3H5nv6fdmNhFYBJxFMC9tWbgNuLuMjiUiUi4S4iL889QutK5Xk3+Nms1LExazYE0Gj5+3H7VrJoQdT2RX/gA84+7PA5jZFcDxwGDgX7t4jbu7xvsWqSLyOmsBtmzPJiklLsQ0ItWTu2cSfO69rYh9FhMUbAvbNoyg0zb/um+BxCKON6g0WUVEREREROT3wu6s/Q133wDMBfbaxS4rgEYF1jWKrt+Ve4Da+Zbme5ZSRKR8mBmX923HUxfsR83EOMb9tJZTHx/H/NVbwo4m8jtmlgjsR77pCdw9N/q8qOkJUsxskZktMbN3zWyfco4qIuUoLmLUSgwKtJs1FLKIiIiIiIiISInFVLHWzFKAdsDyXezyDXBEgXVHRdcXyt0z3X1T3gJsLpOwIiLl5Oh9GvPmFX1oWjuZ+WsyOPmxcXwxZ1XYsUQKqk8wn11JpieYQ9B1ezLB8IoRYLyZ7fJGKk1nIBL78oZC3pKpYq2IiIiIiIiISEmFWqw1s/vNrK+ZtTazPsA7QA7wanT7MDO7J99LHgIGmNlNZtbRzP4M7A88WtHZRUTKU+emaYy45mB6tqzD5u3ZDH7hOx7/8ieCKcdEKid3/8bdh7n7NHf/CjgNWA1cXsTLbgM25luWln9SESmJlORgKORN27NCTiIiIiIiIiIiUvmE3VnbnKAwOwcYDqwFDnL31dHtLYEmeTu7+3jgPOAyYDpwBnCKu8+syNAiIhWhYWoyr152EOf2aoE73DdqDte8OpUMdS5JbFhDcINVSacn2Mnds4Cp7Hr6A9B0BiIxLzVarNUwyCIiIiIiIiIiJRcf5snd/ZzdbO9XyLo3gDfKK5OISCxJio/jntP2pUuz2tz97g98+P1y5q3czBMX7Ee7Bilhx5NqzN13mNlkgukJRgCYWST6vFgjXphZHNAV+KiI82QCmfleU/rQIlIudg6DrGKtiIiIiIiIiEiJhd1ZKyIixXD+ga147bKDaJiaxNyVWzjpkbF8NGNX03uLVJgHgCFmdpGZdQKeAGoBz8PvpzMws7vM7Ggza2tmPYGXgFbAsyFkF5EykpqU11mrYZBFREREREREREpKxVoRkUpi/9bpfHDdIRzUNp2MHTlc9fIU/vr+LHZk54YdTaopd38d+CPwV2Aa0B0Y4O4ro7v8ZjoDoC7wDPAjQTdtGtDH3WdVUGQRKQcaBllEREREREREpPRUrBURqUQapibz0iUHcnnftgD8b9wCznxyPIvXbg05mVRX7v6ou7dy9yR3P9DdJ+bb1s/dB+V7fmO+fRu7+/HuPjWU4CJSZlKinbVbNKe6iIiIiIiIiEiJqVgrIlLJxMdFuO3YTjwzcH9q10hg+tKNHP/wGD74flnY0UREpBrKm7N2kzprRURERERERERKTMVaEZFK6qjOjfjo+kPZv1VdNmdmc80rU7nlzelkqLNJREQqUN4wyOqsFREREREREREpORVrRUQqsWZ1avDaZQdxdf92mMHwSUs57uExTFm8PuxoIiJSTaTsnLM2K+QkIiIiIiIiIiKVj4q1IiKVXHxchJuP6cirQw6iWZ0aLFq7lTOf/IYHPplLVk5u2PFERKSKS9tZrFVnrYiIiIiIiIhISalYKyJSRRzUth4fXX8oJ3dvSk6u8/Bn8zj50XHMWrYp7GgiIlKFpSQFc9ZuUbFWRERERERERKTEVKwVEalCatdI4KFzevDwuT2oUzOBWcs3cdKjY3no03nqshURkXKRqmGQRURERERERERKTcVaEZEq6KRuTfn4xsM4unMjsnOdBz+dy4mPjGWq5rIVEZEytrNYm6nOWhERERERERGRklKxVkSkimqYmsxTF+7HQ+d0p07NBGav2MxpT4zn7ndnqvtJRETKTEq0WLslM5vcXA85jYiIiIiIiIhI5aJirYhIFWZmnNy9GZ/9oS+n9WiGOwz9ZhFHPfA1701fhru+VBcRkT2TlhzMWesOGTvUXSsiIiIiIiIiUhIq1oqIVAP1UpJ44OzuvHTJgbSqV5MVm7Zz3atTOefpCcxesSnseCIiUoklxUdIiDMg6K4VEREREREREZHiU7FWRKQaOaR9fUbfcBh/OKoDyQkRJi5Yx/EPj+Wud2eydktm2PFERKQSMjNSkqLz1m5XsVZEREREREREpCRUrBURqWaSE+K47oj2fPqHvhzbpTE5uc6wbxbR799f8viXP7E9KyfsiCIiUsmkRodC1pzoIiIiIiIiIiIlo2KtiEg11bxuTZ64YD9eGXIg+zRNY3NmNveNmsPh93/J698tJjsnN+yIIiJSSaizVkRERERERESkdFSsFRGp5vq0q8/71xzCA2d1o0ntZJZt3M6tb83gyAe+4t1pv5CT62FHFBGRGJearGKtiIiIiIiIiEhpqFgrIiJEIsZpPZvzxR/7ccfxnUivlcjCtVu5/rVpHPPfr3l7ylKyYqjT1szczE4JO4eIiATyhkHekqlirRRN13ARERERERGR31KxVkREdkpOiOPSQ9vy9S39ufmYvUlLjuenVVv4w/Dp9L//S16asKjC5rQdNGgQp5xySrkc28zSzewRM5tjZtvMbLGZPWxmtcvlhCIiVdyvnbWas1Z0DRcREREREREpCRVrRUTkd1KS4rm6/16M+7/DuWXA3tSrlcjS9du4Y8RM+vzrc/7z8RxWbdoedsw90TS6/BHoAgwCBgDPhZhJRKTS0jDIUoF0DRcREREREZEqRcVaERHZpdTkBK7qtxdjbz2cv5y0D83q1GBdxg4e+fwnDr73c258fRqTFq7DPZR5beub2TtmttXM5pnZSXkbzOwuM1tmZvXyrfvQzL4ws4i7z3T30939fXf/2d0/B/4EnGhm8WG8GRGRyiwlScVaKRFdw0VERERERESiVKwVEZHdqpEYx0V9WvPVzf144vye7N+qLlk5zjtTf+GMJ79hwH/HMHT8QjZurdDhL+8GhgP7Ah8BL5tZenTbP4CFwLMAZnY10Ae4yN13NflubWCTu6vSICJSQnlz1qpYK8Wka7iIiIiIiIhIlIq1IiJSbPFxEY7t2oQ3r+zDu1cfzFn7Nyc5IcKclZu5+70fOOAfn3LVy5P5ZNZKsnJ29X1qmXnB3V9195+A24EUoBeAu+cAFwBHmNm/gH8DV7v74sIOZGb1gTuBp8s7tIhIVZQ3DPKWTM1ZK8Wia7iIiIiIiIhIlIaJEhGRUunWog7dWtThT8d3ZsTUX3j128XMXrGZj2as4KMZK6hbM4Fj9mnMcV2b0LtdPRLiyvz+oO/zHrh7hpltAhrmWzffzP4IPAW87u6vFHYQM0sDPgRmAX8u65AiItWB5qyVEtI1XERERERERCRKxVoREdkjtWskcFGf1lzUpzWzlm3inalLGTFtGas3Z/Lad0t47bsl1KmZwBEdG3FEp4Yc2r7+zuEy85u/egsvTVjMgjVbaFWvFhlxqcCGok5dsH3L+f2IEYcBOUBrM4svODyimaUCo4DNwKnurpawEooOT3kz0BiYDlzr7t8Wsf+ZwN+A1sA84FZ3/6gCoopIOVKxtnpbl7GDV79dzORF66lTM4H1iQ0wXcNFREREREREikXFWhERKTOdm6bRuWlnbh3QkW8XrOODGcsZPXMFazN28NaUpbw1ZSkJccYBrdM5pH19DtmrPvs0rc2YeasZMmwSuQ45uU5cZA259Y+mdeYnpc5iZmcDpwH9CObFu5Ngjry87WnAaCATOMndt+/Je6+Oon/GDwBXABOBG4DRZra3u68qZP8+wKvAbcAHwHnACDPr6e4zKyy4iJS5vJtwtmSqWFvdLF67ldOeGMe6jB3kOsRFjJz0fjTKTS31MXUNFxERERERkepExVoRESlz8XER+uxVnz571eevJ+3DtwvX8fmPq/h89irmr8lg/M9rGf/zWu5jDmnJ8WzPyiUrx3e+PifXAWNRo0P5bvJUEuJs57YePXq02N35zaw58AT/z959x8dVnfkf/5yZUbsjyXLvtmyNim1sDLaKbdmGQBY2TmdJgkMCpLBJICSb6nSSbH54N5teCAkBQwiEkB6cAKHZloskg42Ni5p7r2ozqjPn98eMjGxLsiRLGpXv+/Wa18w999xznxmXke5zz3PCszYLjDF3AE8bY/5prd0Uucj7HOAQXhcvOdIGcDKyXp5c2meAX1lrHwYwxnwMWAZ8CFjZRv9PAc9Ya78b2f6aMebNwN2EE74iMkAlxrXMrNXkxqHm26t3ctbfRCjyNR6MvDg+ah5Pry1mUvIb1TT0HS4iIiIiIiJyMSVrRUSkV3ncLhamjWJh2ii++taZ7DlZy5rSk6wvP03hntNUt1cy0xhCMQ5L3n079Qe2td7zjbYPaDnMGGAVUAT8FMBa+6wx5n7gMWPMXOBqIDdySPkFQ0wD9nXlPQ5FxphYYB5wX0ubtTZkjHkeWNDOYQsIz8Rt7VngnR2cJw6Ia9XU/alaItJrWsogn6ptZMbXnolyNNJXrLXUN4fa3hcKcuuK71G18cnWzfoOFxEREREREbmAkrUiItKnpo9OZProRO5YNI3mYIjfFh7gG3/b0W7/fz73L67JHNO66SORx3mstSmtNq9vY/89wD2RzZcBc2Ef6ZJRgBs4fkH7cSCrnWPGtdN/XAfn+RKXuLgvItE3JimeySMSOHimjromTWwU8Hg8fP3eb/KZf/td62Z9h4uIiIiIiIhcQMlaERGJGo/bxXvmT+Z/ntlNoPHii/sJMW7mp46IQmTSj9zH+bNxk4BDUYpFRNoR63Hx/GeWcqK6IdqhSB+76/FXef1w1bkyyC2CIcs1WWPaPkhEREREREREzlGyVkREoioh1s1Xls3gK39+HZeBkAW3gaCFFf+edW4dROl3TgFBYOwF7WOBY+0cc6yL/bHWNgDnsj/hCpki0h/FedxMHuFEOwzpY998+yze+8AmgtYSDNlzU17fOmc8V01OiWZoIiIiIiIiIgOCroCLiEjUvT93KhNSEvj1ur2Unahh+qhEPrJ4GtfNuDCvJ/2FtbbRGPMKcB3wFwBjjCuy/dN2DtsY2f/DVm1vjrSLiMgAdNWU4fztk4u4/+UKCvecYYQ3lvflTGZ5zhTdYCMiIiIiIiLSCUM2WVtdXR3tEEREpJV54+OZ954Z57W19X/1sGHDkoEaa629aKf0te8DjxhjNgNFwKcBL/AwgDHmUeCwtfZLkf4/AtYYYz4LrAbeB8wH7uzqifU9LiLSf0xw4NtvSQPSzrUF/LUX9dN3uADU1l78d0NEeoervjHaIQwINTU10Q5hQDDGGH2Hi4iI9A4z1L5jjTET0Vp3IiID3TBrrbJ1/YAx5m7g88A4YCtwj7W2MLLvZWCftfb2Vv1vBv4bSAXKgC9Ya//RhfPpe1xEZGDTd/gQpe9wEZEBT9/hIiIivWQoJmsNMAFo67a5JMK/PE5qZ790jj7HnqHP8fLpM+wZ/fFz1KycIUrf471On2HP0Od4+fQZ9oz++DnqO3yIusR3eLT0x38j/Y0+o87R59Q5+pwurT9/RvoOFxER6SVDrgxy5IeKw23ta7WmUo3uFOs+fY49Q5/j5dNn2DP0OUp/ou/x3qXPsGfoc7x8+gx7hj5H6U86+g6PFv0buTR9Rp2jz6lz9Dldmj4jERGRockV7QBERERERERERERERERERIYiJWtFRERERERERERERERERKJAydrzNQDfjDxL9+lz7Bn6HC+fPsOeoc9RBgr9Xb18+gx7hj7Hy6fPsGfocxTpmP6NXJo+o87R59Q5+pwuTZ+RiIjIEGS0LryIiIiIiIiIiIiIiIiISN/TzFoRERERERERERERERERkShQslZEREREREREREREREREJAqUrBURERERERERERERERERiQIla0VEREREREREREREREREokDJWhERERERERERERERERGRKFCyVkREREREREREREREREQkCpSsFRERERERERERERERERGJAiVrRURERERERERERERERESiQMlaEREREREREREREREREZEoULJWRERERERERERERERERCQKlKwVEREREREREREREREREYkCJWtFRERERERERERERERERKJAyVoRERERERERERERERERkShQslZEREREREREREREREREJAqUrBURERERERERERERERERiQIla0VEREREREREREREREREokDJWhERERERERERERERERGRKFCyVkREREREREREREREREQkCpSsFRERERERERERERERERGJAiVrRURERERERERERERERESiQMlaEREREREREREREREREZEoULJWRERERERERERERERERCQKlKwVEREREREREREREREREYkCJWtFRERERERERERERERERKJAyVoRERERERERERERERERkShQslZEREREREREREREREREJAqUrBURERERERERERERERERiQIla0VEREREREREREREREREokDJWhERERERERERERERERGRKFCyVkREREREREREREREREQkCpSsFRERERERERERERERERGJAiVrRURERERERERERERERESiQMlaEREREREREREREREREZEoULJWRERERERERERERERERCQKlKwVEREREREREREREREREYkCJWtFRERERERERERERERERKJAyVoRERERERERERERERERkShQslZEREREREREREREREREJAqUrBURERERERERERERERERiQIla0VEREREREREREREREREokDJWhERERERERERERERERGRKFCyVkREREREREREREREREQkCpSsFRGRfskYs8oYYyOPJmPMXmPM/xpj4vs4jjuNMS8bY6ojsaT05flFREREREREREREZPBSslZERPqzZ4DxwHTgv4D/BL7ZxzE4kTj+Xx+fV0REREREREREREQGOSVrRUSkP2uw1h6z1h601v4FeB54M4AxZp8x5tOtOxtjthpj7m21bY0xHzHG/NkYEzDGlBlj3t5q/9eNMUeMMSNbta02xrxkjHEBWGt/aK1dCWzqzTcqIiIiIiIiIiIiIkOPkrUiIjIgGGOuABYCjV089BvA74E5wD+A3xpjRkT2fQfYBzwYOcddkXPcZq0N9UDYIiIiIiIiIiIiIiLtGnLJWhOWbIwx0Y5FREQu6a3GmFpjTD2wHRgDfLeLY6yy1j5hrS0HvgwkAjkA1togcCtwnTFmZWTsu6y1B3rsHUiP0ve4iIjIwKTvcBERERERkbZ5oh1AFCQBVVVVVdGOQ0RELlBT38Tx6nrGJMdz2223cfjwYe6//36v3+/nBz/4AR6Px/Xggw/+sYvDbmt5Ya31G2OqCSd9W9r2GGM+BzwAPGmtfbxn3o30En2Pi4gMXErSDW1JQNWWLVupqNgT7VhkgBvx1PeiHcKAcObmz0Y7BBlE0tKm63tcRESklwzFZK2IiPQzDc1BvrN6F08UHaApaIlxG0YlX83oqhp8Ph8ADz30EFdeeVD1M64AAOvZSURBVCXGmA9ba38NhLj4om9MG8M3XbBtubiyxBIgCKQaYzzW2ubLf1ciIiIiIiIiIiIiIh0bcmWQRUSk//nKn7bz2Kb9NAUtAE1By9GE6RwYt+RcH5fLxZe//GWA/zbGJAAngfEt+40xycC0rp7bGPNe4N3ANcAU4GvdfyciIiIiIiIiIiIiIp2nZK2IiPQZay11jUFO1Taw/7SfnUeqeeb1o/zx1cOE7AWdjaEyOY1DZwPnmm6++WYIz4C9C3gR+IAxZrExZjbwSGRfpxljJgH3A1+01hYAdwBfNsbkteozzhgzF/BFmmYbY+YaY0Z05VwiIhI9x6vrWb3tKPVNXfqaEBERERERERHpdSqDLCIil2StJdAYpLq+iZr6Zmrqm6iub6a2vpma+mZqG1ram6ltaMbfEH6uqX/jdW1DM4HGIMGLsrIdMIaSYzVMGu4A4PF4AH4KfAFIJzyT9mmgivCM2E7PrDXGGGAVUBQZE2vts8aY+4HHjDFzrbW1wMeAb7Q6dG3k+Y7I8SIi0o/tP+3nPQ9s5Hh1A6OT4vjENWnckjOF+Bh3tEMTEREREREREcFY24WL5oNApExmVVVVFcnJydEOR0SkzzQHQ1TVNVFV10Rl5Lk68lwViGzXN1Fd1xx+bvW6pr65a0nWTnBi3XjjPMS4DEeq6tvt9/Qn87li4rDWTReuUytDiL7HRaQrDp0N8N4HNnG4sg6X4VwVh3HJ8dx1bRrvzZ5CrEfFhvqQvsOHsJbv8C1btpKUlBTtcGSAG/HU96IdwoBw5ubPRjsEGUTS0qbre1xERKSXaGatiMgAY62lpqGZs/5GzvgbqQw0cTbQyNlAE1WR57OBRqrqws+VgXAytqah+bLP7XEZkuI9JMXHkBTvITEu/Do53kNivIekeA/eOA9JceHnxMjDGxfenxjnwYl148R6cLve+D3vnT9bz/bDVeclhN0uQ9a4JGZNUEJORES67mhVHct/Vcjhyjqmj/Ly2EdyebnkJD99sYwjVfV87a87eGDtHj59fQbvumried9Lcnluv/12HnnkESBcFWPSpEncfPPNfPe734231rZ/h1YPM8bcCSwHrgaSgOHW2sq+Or+IiIiIiIhIZyhZKyISZY3NIc74Gzntb+BMJAHb+nE20Hq7icpAI82XMcs1Kc5DckIMKU4MwxLeeCS3PMeH9ydHXg9LiDmXnE2IcROuHtyzfv7+q7ntoSLKTtSea5sywuEXt87rlfOJiEj/UNcY5JGN+1hffopQD1f8qTjh51h1PVNGODz+0TzGDYtnee4Ubpo3kSeLD/KTF8s5dLaOzz31Gve/XM7nb8jkhlnj9L3TQ2688UYefvhhmpqaeOWVV7jtttsAvgl8sQ/DcIBnIo/7+vC8IiIiIiIiIp2mZK2ISA8LhixnA42crm3kdG0Dp/zh59O14YRs+DnS5m+kpr57M14TYtyM8MaS4sREnmMZ7sSQkhBDihPLsIQYhnvDr1vakuM9eNz9r9zjhJQEnv30EjbuOc2ek7VMHekl3zcKl2Y5iYgMSs3BEL/ffIgfvVDK8eqGXjvPxJQEHv9oLuOGxZ9ri/O4+eCCVG6eN5lHNu7jF2sqqDjp52OPvcrcySl88cYsFqSN7LWYhoq4uDjGjRsHwOTJk7n++uv505/+9Gbgi8aYfcAPrbU/bOlvjNkK/MVae29k2wIfBZYBNwCHgc9aa/8W2f91wuvKz7bWno60rSacoL3OWhtqGd8Yc03vvlsRERERERGR7lOyVkSkE5qC4dmvJ2saOFnbwKmaBk61JGNrw69PRV6f8TfS1YmvbpdhuBPLSG8sI7yxjEiMZYQTy3BvuK3lOcWJYaQ3jhQnhvgYd++82ShxuQyLfKNY5BsV7VBERKQXHams44MPFVEeqaYwMSWBO5dMJ8WJ6dHzuF2Gxb7RDGtn3IRYNx9bmsby3Cn8au0eHly3l60HK7nlV5uYPtpLbOTmprgYNx9fOp0brxjfo/ENJa+//jobNmwAaOziod8AvgB8Hvgk8FtjzFRr7RngO8CNwIPAu4wxdwELgSuttaEeC15ERERERESklylZKyJDVihkqaxr4kRNfTgJ2+pxqjaclD0ZScqe8Xf12iKMiCRYRybGMtIbx6jEWEYmxrVqj4vsiyU5PkazSEVEZEi49287KD9RywhvLHdf6+P9eVOI80TvBqTk+Bg++2+ZfGDBVH7yQjlPFB1gz0n/eX3uenwLv77NzTWZY6IU5cDz9NNPk5iYSHNzMw0NDbhcLoDvdnGYVdbaJwCMMV8G7gFygGestUFjzK3AVmPMysi+j1hrD/Tg2xARERERERHpdUrWisigU98U5GRNAydqGjgZScSeiCRhT1yQkO3K2q8uAyO8cYxOCideRyfGMSryelRiXKtHeHZsfyw3LCIiEk1rS0/y3M7juF2G392ZR8bYpGiHdM6YpHi+/c4r+MS1aVSceCNZ+/vNB/nba0f4xG9f5YmP5nHl5BQAXio5wc9fKud0bfs3dF0xcRif/bcMpo70diqG07UN/PiFMvac8vM/N81hQkrCZb2naLr22mu5//778fv9/OAHP8Dj8fDggw/+sYvDbGt5Ya31G2OqgTGt2vYYYz4HPAA8aa19vGeiFxEREREREek7StaKyIARaGzmeHUDJ6rrOV4Tfj7R+jnyurqLa8AOd2IYkxTP6KQ3ErFjkuIZlRROwobb4hjuxOLW7FcREZFuaWwO8c2/7wDgtgWp/SpR29r4YQmMH/ZGkjRn2gjOBhpZV3aKD60q5rs3z2HVhv2sLT15ybH2nPLzz9ePcvvCVO5+UzrDEtouydzQHGTV+n389MVyahrCP8fc8XAxT318AcnxPVseujdYa9m05wyvHjhLihNDk4nB6/Xi8/kAeOihh7jyyisxxnzYWvtrIARc+ENVW2+06cJTARfeDbcECAKpxhiPtbZrPwiKiIiIiIiIRJmStSISdfVNQY5X13O8uiHyHE6+nntdHU7E1jZ0/tpbrMfFmEjy9Y3n+PNej04KlyGO0QxYERGRXvfoxn1UnPQzKjGWT785PdrhdFqsx8X9t87jvQ9sZMeRaj60ajMAMW7DHYumcf2MsZg27uWqbwry4Lq9rCk9ya/W7eXJ4oPtzpQ9VdvIqdoGAK6YmMyJ6gZKjtfwicde5eE7solxu2hoDvLrgr2sKTnJrXlTeeuc8Zi2TnwZGptDPLpxH3/deoSmYOeWfR2VGMfZQCM7jlTjdhlCIYsZ8zamNr1wro/L5eLLX/4yy5cv/29jzOPASeDcIsDGmGRgWlfjNca8F3g3cA3we+BrhNe5FRERERERERkwlKwVkV4TCllO+Rs4XtXAsep6jlXXc7wq8nzu0UBV3YWTJtrnjXUzJjn+XBJ2bHI4ATsm+Y1k7JikeJITPD1+AVNELo8x5l4uvoheYq3NikI4ItKHTtTU88PnywD4wg1ZA2K2aGuJcR4eviOb/7h/IwfOBLhh1li+9O8zSB3VcXnjxemjeankBN9ZvYvyE7VUH6tpt++YpDg+f0MmN109iZ1Hq3nPAxspKD/Fl/+0nRtmjePbq3ey/3QAgMK9Z/jNpv3c+7ZZzJyQ3CPv8eWSE3zr6Z0Xrdd7aW+8p2BkeQmLi30TruNUbQOjEuMAuPnmm1m+fHkQuAt4EbjdGPN3oBL4FuHZsZ1mjJkE3A980VpbYIy5A3jaGPNPa+2mSJ9xwDjAFzlstjGmBjhgrT3TxTcqIiIiIiIi0iuUrBWRbmkOhjhe08CxqjqOVtVzrKr+3POx6vDz8er6Tq8JGx/jYlxyPGOS4hk7LJ6xLYnYSBJ2bHIcY5LjSYzTf1siA9wO4PpW2ypXKTJAVJys5WcvlnOipqHLxx6pqqO2oZkrJw3jP+ZN6oXoet+YpHhW35PPiZoG0kYndvq4azPHkO8bxWsHK2lobnu2qssYrpw8DCc2/HPOFROH8bPlV/ORRzfz1CuHeOqVQ5EY4rhh1jieeuUgRXvP8NafrOP9uVP57L9lkOLEdut9HTgd4FtP7+T5XccBGJUYy6euz2D6JRLRAE3BEB9etZmgveDnPWOwuPjr1iN8OD88Ydbj8QD8FPgCkE54Ju3TQBXhGbGdnllrwnfkrQKKImNirX3WGHM/8JgxZq61thb4GOffJLQ28nxH5HgRERERERGRqFPWQ0QuYq3ltL+RI5V1kUd9+Lkq/PpYVT0naurpTB7WZWB0JPE6NjmeccnxjBsWH9mOCydok+NJjtdMWJEhotlaeyzaQYhI5wUam/npi+X8at0emoKduwmrLS4D9759Fq4BvP57UnwMSd2YFRzjdjE/dUSXjrk2awzfescsvvLn14l1u/jw4mncda2PxDgPH7smjf/3j12s3naU32zaz+rtR1lxYxb/MW9Spz/f+qYgv1hTwc9frqCxOYTHZbh9YSr3XJ/e6ZnP/obmixO1ER63i9O15yf2rbUrgZWRzfddcMgjF/S96I1Ya1NabV7fxv57gHtabd8L3NtO+CIiIiIiIiL9gpK1IkNQy6zYQ2cCHDpbx+HKOg5Hno9Uhp/bm/nRWozbMDY5nvHD4hk3LCH8HNkeOyz8PDoxDo/WhBWRN6QbY44A9cBG4EvW2gPtdTbGxAFxrZqSejk+kSGtpr6JRzbs43h1OMlmsby46wRHquoBuDZzNO+YO7HNNVovZepIL3Mnp/RgtIPf+3OnMmvCMEYlxjJpuHOufWJKAj9bfjXvzz3FN/66g7ITtXzhj9v4bdEB5kwcdslxLZa1pac4cCZcVnlh2ki+9Y5Z+MZ07b9YJ9ZN6kiH/acDXJiybQ5ZrtSft4iIiIiIiMglKVkrMghZazlZ08DBswEOnAlw8EwdByOJ2UOVAY5WXro8sTHhUnvjhyUwMSWBCSnxjB/2xvP4lHhGeeMG9OwYEelzhcDtQAkwnnBpynXGmCuste0t5PglLl7nVkR6QShkueeJLbxUcvKifRNTEvjG22by5pljVQmjj3WU4F6YNop/fGoxq9bv44fPl/LawUpeO1jZ6bHHJsfxtbfOZNns8d36czXG8F9vzuBTv9t6XrvbZfCNTuS6rDFdHlNERERERERkqFGyVmSAag6GOFxZx77TAfaf9rP/dID9pwMcOOPnwJkA9U0dz4yNcRsmpCQwaXg4GTsxxWFi5PWk4QmMTY4n1qMZsSLSc6y1/2y1uc0YUwjsB94D/Lqdw+4Dvt9qOwk41DsRigxtP3y+lJdKThLncfGRxdNwu8I/B4xOjOU/5k0mIdYd5QilLTFuFx9dMp23XTmBP285TF1TsFPHjXBi+I/5k0mMu7xfCd8xdyIha/m/Z0s5XFmH22VYNns833jbTFVXEREREREREekEJWtF+rFQyHKsup69p/zsOVnLnlN+9p3ys+90gINnAh3OjnUZGD8sgckjEpg83GHKCIdJkdcThycwNiles2JFJKqstZXGmFLA10GfBuDcooea0SfSO57bcYwfv1gOwH3vns27r54U5Yikq8YNi+fj16RF5dzvumoS77hyIqf8DSTGeXBi9WumiIiIiIiISGfpt2iRfqChOcjeU37KT9RSccJPxclaKk7Wsuekv8PZEXEeF1NHOkwZ4WXqSIfUkQ5TRnqZMsJhYkqCZsaKSL9mjEkE0oDfRDsWkaGs4mQtn/n9awDcvjBViVrpFpfLMCYpPtphiIiIiIiIiAw4/SpZa4xZAnwemEd4Lbt3WWv/0mq/Ab4JfBRIAdYDH7fWlvV5sCLd0NAcZM9JP6XHayKPWsqO13DgTID2Jsl6XIYpIx2mj/KSOtLLtNFepo30kjrKy7hkzY4VkYHDGPN/wN8Jlz6eQPg7PQg8Ec24RIayTXtO89nfv0ZtQzM5qSP4yrIZ0Q5JRERERERERGRI6VfJWsALvAY8BPypjf1fAO4BbgP2At8GnjXGzLTW1vdZlCKXYK3lSFU9u45Us/tYNbuO1VB6rIY9p/wE28nKJsV7SB+TiG9MImmjI48xiUwankCM1vsSkcFhEuHE7EjgJFAA5FlrT0Y1KpEhyN/QzP8+s5tHNu4HYPKIBH76/qv0M4eIiIiIiIiISB/rV8laa+0/gX/CxWvSRWbVfhr4b2vtXyNtHwSOA+8EfteHoYqc0xQMUX6ilh1Hqtl5pJqdR6vYeaSa6vrmNvsnxXvIHJtE+tgkMsYmkj4m/Dw6KU5rMYrIoGatfV+0YxAZioIhy/8+u5uKE7Xn2nYdreFwZR0At+RM4ctvySIpPiZaIYqIiIiIiIiIDFn9Kll7CdOAccDzLQ3W2ipjTCGwACVrpQ80BUOUHq9h+6Eqth2uYsfhKnYdq6GxOXRR3xi3IW10IjPGJ5M5LomscUlkjktiXHK8krIiIiLSZ36xpoIH1uy5qH1iSgIrb5rN4vTRUYhKRERERERERERgYCVrx0Wej1/QfrzVvosYY+KAuFZNST0clwxS1lr2nvLz2qFKXjtYxWuHKtlxpLrNxGxSnIcZE5KZNSGZWROGMXN8Mr4xicR6VEpQREREomfLgbN8/1+lAHxsaRrTRjkAxMe4uW7GWBLjBtKvAyIiIiIiIiI9L3XF6ph9K5c1RTsOGbqGwtWZLwHfiHYQ0v9V1zex5UAlWw6cZcuBSrYerKSq7uL/n5PiPcyZNIwrJg5j9sRhXDFhGFNGOLhcmi0rIiIi/UdtQzOf+t1WgiHLW+eM54s3Zqq6h4iIiIiIiEgrkz6x6klP8uj3pK5YvWrfymV3RDseGZoGUrL2WOR5LHC0VftYYGsHx90HfL/VdhJwqEcjkwHHWsuBMwE27zvL5v1neXX/WUpP1GDt+f1iPS5mTxzGnEnDmDs5hTmTUkgd6ehCp4iIiPR73/jrDg6cCTAxJYHvvGu2fn4RERERERERaSV1xeoYG2x6T2Tz9tQVq+/UDFuJhoGUrN1LOGF7HZHkrDEmGcgF7m/vIGttA9DQsq2LVENTMGTZdbSa4n1nIo+znKxpuKjflBEOV09J4eqpw5k7OYWscckqZSwiIiIDzt9fO8IfXz2Ey8AP3juXYQkx0Q5JREREREREpF/Zt3JZ0/jbfvB34G3AKiVqJVr6VbLWGJMI+Fo1TTPGzAXOWGsPGGN+CHzVGFNGOHn7beAI8Jc+DlX6uaZgiNcPV1G49wyFe06zed9Zahqaz+sT4zbMnjiM+akjmDd1OFdPGc7opLh2RhQREREZGE7U1PO1v74OwN3X+siZNiLKEYmIiIiIiIj0T5XrHvvm2Pd86yYlaiWa+lWyFpgPvNRqu6V88SPA7cD/Al7gl0AKUADcaK2t77sQpT8KhiyvH65i457TbKw4zeZ9Z/A3Bs/rkxjnYd7U4eRMG0F26gjmTBpGfIw7ShGLiIiIhAVDFpfpmQow1lq+8ufXqQw0MWtCMp+8Lr0HIhQREREREREZnIxxGSVqJdr6VbLWWvsy0O5VKmutBb4eecgQZq2l7EQt68tPsaHiNJv2nKam/vyZs8MSYsiZNoLcaSPImz6SGeOTcbtUBltERET6j5dKTvD5p7Yxflg8P77lKqaN8l7WeH977Qj/2nkcj8vwfzdfSYxbyzmIiIiIiIiIiPRn/SpZK9KRI5V1FJSfYkP5KdZXnL5ozdmkeA+500ayIG0kedNHMGNcMi4lZ0VERKQfCoYsP3y+lJ+8WA7AqdoG3v6TAr578xxuvGJ8t8Y8WdPAN/62A4BPvimdGeOTeyxeERERERERkcEmbkJWvDtxeMOle4r0LiVrpd+qqW9i054zFJSdZF35Kfac9J+3Pz7GRXbqCBb5RrEwbSSzJgzTzFkRERHp907VNvCp321hfflpAG7JmUL5iRqK953lY4+9ykfyp/HFf8+6aFZsMGQp3Hua6rrmtoblyeIDVAaamDk+mU9cm9br70NEREREREQ6L3XF6hiV2+1f3N6UzGD1yR1dPU5/ltLTlKyVfiMYsmw/XMW60pOsKzvFqwfO0hyy5/a7DMyZlMLi9FEs8o3iqikpxHm05qyIiIgMHK8drORjj73C0ap6EmLcrLxpNu+YO5GmYIj/e7aEB9bu4cGCvWw7XMXP3381oxLjAKgKNHHP77awpvRkh+Or/LGIiIiIiEj/M/HOX/4hZsTEm1JXrF61b+WyO6Idj4TZYJOn4Vh523dEt2Pq5/68KtTgv01/ltKTTHgZ2KHDGJMMVFVVVZGcrNJw0Xa0qo61pSdZW3aK9eWnqAycfzPK1JEOi9NHke8bzYK0kQxLiIlSpCLSj2gK/RCm73EZyJ7afJCv/OV1GptDTB/t5YFb55E+Num8Ps/uOMZnf/8atQ3NjB8Wzy9unUdcjIs7H32FA2cCxMe4mDVhWJvjG+A98yfznuzJffBuRLpF3+FDWMt3+JYtW0lKSrpkf5GOjHjqe9EOYUA4c/Nnox2CDCJpadP1PS7STakrVseEmhoamyuPETt6KkCsZmX2D05a9vxARfHmzvZPXbE6punskUZPyjiMcYH+LKWHaGat9Kn6piCFe8+EE7SlJyk7UXve/qR4D4vSRrE4YxSLfaOZMtKJUqQiIiIiPaMpGOLbT+/k0Y37Abh+xlh+8N4rSYq/+Ca0G2aNI+2uRO78zWb2nPRz8wMbcRtDXVOQScMT+OUH5jNzgm5UEBERERERGSj2rVzWlLpi9SpPyrjb6w/u+Mex335Byb0Bat/KZU3jb/vB08a43gqsUqJWeoqStdKrrLVUnKxlTekp1pSepHDPaRqaQ+f2t5Q2XpIxmqUZo7hyUgoele0TERGRQeKsv5GP//YVNu05A8B/XZ/BJ9/kw+Vqf2KCb0wif71rEf/15Gs8v+s4AIvTR/Hj913FcG9sn8QtIiIiIiIiPWffymV3pK5YfWfNltXxju8PiwPlReuiHZMAxtXl0rNVBY9/c8zN975biVrpSUrWSo+rqmtiQ/kp1padZE3JSY5U1Z+3f1xyPEsyRrE0YwyLfCNJcXTRUURERAaf8hM1fPiRzew/HcAb6+aH77uKN88c26ljk+Jj+OUH5vF40QEamkPctmCqbmgTEREREREZwPatXNbEymVN3plLtzq+nMtK2KauWB2jZGEPsKEulXh30nNH44k5rc9eepqStXLZgiHLa4cqWVt6knVlp9h6sJJg6I0bUmI9LnKnjWBpxmiWZIwmfUwixmiZCxERERm81pSe5O7fvkpNQzOThifw4G3zyRrXtfLFLpfh1rypvRShiIiIiIiIRIN/55qay0nYpq5Y/XBz1fHbU1esXrVv5bI7eiPGocLaUNdm1oZCUwNlhZ1e41aks5SslW45dDZAQVl49mxB2Smq65vP25822svSjDEsyRhF7rSRJMS6oxSpiIiISN96vPAAX/vr6wRDlpzUEdx/69WMTIyLdlgiIiIiIiLST5xL2KZlLwlUFK/t7HGpK1bHALcbTyzB2rO3p65YfadmeXZP/OQrvK6EpLpoxyECStZKJ9XUN7Gx4jQF5acoKDvFnlP+8/Ynx3tY5BvF0ozRLM4YzcSUhChFKiIiIhIdoZDlu8+VcP/LFQDcdPUk7nv3bGI9Kl8sIiIiIiIi5/PvXFOTOPv6V7uSsN23cllT6orVq9ze4bc3ntj71zMvPpjCymUnezvWwciVkJQRqq/dFu04REDJWmlHY3OIrQcrKSg/xfryi0sbu12GuZNTWJI+msUZo5gzcZjWURMREZFBraa+iVcPVLJg+siLErD1TUE+/4dt/P21IwD81/UZ3HOdT0s/iIiIiIiISLtqtz9f242E7R2pK1bfeeShu5sc328WJs6+rrp2+wsNvR3roBMKueoPbA92truTsTDTeGJLezMkGbqUrBUgvO7sziPVbKg4xYaK0xTtPUNd0/n/T00b5WWRbySL00ezIG0kyfExUYpWREREpG8dOhvggw8Vseekn3zfKH7xgXkkxoV/lK6ub+LORzezac8ZPC7D/9w0h5vmTYpyxCIDmzEmFdgLXGWt3RrdaEREREREek83E7ZNAHUVRRsT0rKXAGt6NcjByIa6dnd1KJjoLy+s7qVoZIhTsnaICoUsu45VU7jnDBv3nKZwz+mL1p0dlRjLgrRR5PtGssg3iknDnShFKyIiIhI9pcdr+OCvizhWXQ9AQfkplv9qEw/dnk3IWm57qJhdR6tJjPPwwAfmscg3KsoRi0TX7bffziOPPAJATEwMU6ZM4YMf/CDf+MY3PNba5ksc3qOMMUuAzwPzgPHAu6y1f+nLGERERERELqU7CVsAa61NnH3dJic9b0GgbNPG3oxxyHO5VDpLeo2StUNEczDEjiPVFO87w6Y9Zyjed4aquvPXHU+K85A7fQQL0kaxyDeSzLFJKt0nIiIiQ9rmfWf40KpiquubSR+TyOdvyGTFn7az7VAVN/9iI8GQ5cCZAKMS41h1RzZXTBwW7ZBF+oUbb7yRhx9+mIaGBv7xj39w1113QThpel/rfsaYWGttYy+G4gVeAx4C/tSL5xERERERuSznEra+nMWB8qJ1nT/uhQZv1uIKJz13TqCsUGuw9gJvVn4MxvTpjacytChZO0j5G5rZerCSzfvOsnn/GV7dfxZ/4/lljb2xbuanjiBv+kgWpo1k1oRkrTsrIoOKSiiKyOVYU3qS//zNZuqbQlw9JYWHbs8mxYklbUwiH/x1EXtP+QGYMsLhNx/OYepIb5QjFuk/4uLiGDduHAAf//jH+fOf/8y//vWvtxtjMoEUoBi4C2gAphljcoAHgBnA68B3Wo9njLkGeAm4EVgJZAEbgfcRnjX7fWAi8DTwEWttAMBa+0/gn5Exeu39ioiIiIj0hEjCdovjy10aKC/sdGlj/+51J5zMhclOxoLUQOnGfV05Z+qK1TEtZZWHEmtDttN9Q8Erjcv9am/GI0ObkrWDgLWWQ2frePXAWV7Zf5ZXD5xl19Eaghf8X5Mc7yE7dQTZ08IJ2iuUnBWRfkwlFEUkmp55/RiffOJVmoKWazJHc//755EQ6wYgbXQif/rEQj75xBY8LsMP3zeXMUnxUY5YpH9LSEgAiI1sXgdUA28GMMYkEk6y/gu4FZgG/Kidoe4F7gYCwO8jjwZgOZAI/Bn4JPA/Pf8uRERERER6X+3252u9mYs2dzVhGyjZUO5kLpzrnbF4pH/XutOdOWbSxx/6vTt5zM2pK1av2rdy2R3dj3rgMa4upMdsyPh3bwz1XjQy1ClZOwCd9Tey7XAV2w5WsjXyOO2/uHLYxJQE5k0dTnbqcOanjiBzbJLKqovIgKISiiISDX/ZcpjPPvUawZBl2ezx/OC9c4n1nH+D29jkeH7/nwuiFKHIwGGt5YUXXuDZZ58FeBEYDfgJz35tBDDG3Am4gA9ba+uBHcaYScD9bQz5VWvt+shxvyb8M0GatXZPpO0PwLUoWSsiIiIiA5i/ZL2/mwnbrU563pKkuTduqNn6TIeTHVJXrI5xJ468ufnsEWJGTLw9dcXqO4fKDFsnPXcYxlRHOw6RFkrW9nNn/Y3sOFLN60eq2H64iu2HqjhwJnBRvxi3Yeb4ZK6eOpx5U4dz9ZThTEhJiELEIiI9RyUURaS3Hauq53vPlXCytgGAYMhSUH4Ka+GmqyfxPzfNViUSGdxCQXj2K3C6/I22hBRY8gUYndHtYZ9++mkSExNpamoiFAqxfPlyHn300XuBnwHbL7jJagawLZKobbGxnaFbr8F1HAi0JGpbteV0O3ARERERkX6iuwnbuvLCdQlp2dcQvgbWrn0rlzWlrli9ysQm3N5cc+rJQz+7bUgkasNMWl154dbO9HQyFo417piTvRyQDHFK1vYTwZDlwJkAJceq2Xm0hp1Hqtl1tJrDlXVt9p82ysvsicOYOzmFuVNSmDk+mfgYdx9HLSLSt1RCUUR60tGqOm755Sb2nb74Rrhb86bwrbdfoaokMviVPw+FbUxg3bMGbvs7jMm69BjHd0DBD2H/enBGcl2S4ei11/Cz+39BbGwsEyZMwOPx8Mgjj/gjNz35LyPi1heQ7AXbLW26w0JEREREBoVzCdv0vCWBsk1rO3OMtdYmzrq20PHl5AXKizZ11HffymV3pK5YfeeJP34r15jbjbW20+u4DmjWGmtt58oaB5sm+0uLN/dyRDLEKVnbx6y1HK2qp/R4DWXHayk9XkPp8RpKjtdQ39T2/w2pIx1mTRzGFROGMWdS+HmYE9PHkYuIRI9KKIpITztWVX8uUTtpeAL3vCmdlonz44clsMg3UjPpZWjY9ffwc8aNMOPtgIXCX8Cx7fDI2+D2p2F0ZvvHH3oFVv07hJrDs3Srj3DrCMvUGWPwpaVB5/4d7QI+YIyJbzW7Nu+y3peIiIiIyCARSdi+4qRlLwlUFHcqYVu746WANyt/v5OeNytQtmlHR333rVzW5P3LfYUJ6Xm5QIfJ3cHChjqsEC3S55Ss7SWBxmb2nQqw95SfvadqqTjpp/xELXtO1uJvDLZ5TJzHRcbYJGaMT2Lm+GRmThhG1vgkkuOVmBWRIaLmGJzdDylTAJVQFJHecayqnvf9cuO5RO3v7sxj0nAn2mGJ9L1QEEr+GX6d93GYfk34deZb4JG3w/HtsOqt8L7HIWls22P843MQbIJzN6VbjIElI07Aoc0wObszkTxOeOmCXxlj7gNSgc91+321Eqm84WvVNM0YMxc4Y6090BPnEBERERHpbecStl0oiezfXXDUm7koyclYMDlQuvHgJfo2OZkLj3szF03yl6w/1DNR91/G1bnUmDHGxE+fpzu5pdcpWdtN1lrO+Bs5eLaOA2cCHDwTYP9pP/tOBzhwOsCx6vp2j/W4DNNGeUkfm0j6mCQyxiaRNT6J1JFe3Cq1JyJDUUMt/P1T8PofCVcvNHxi9ATOXL+E7//0lyqhKCI9JhSy3LGq+Fyi9omPKlErQ9jBQgicgvgUmLrojXZnBNz2tzcStr++vstDN1uDp+y5TiVrrbW1xpi3Ab8AtgA7gS8Cf+zyiS82n/PX6vp+5PkR4PYeGF9EREREpE90Zw1bf8n6UseXO887Y0mNf9fayo76Bko27HXSc/NT8pcfrix4PGrlkFNXrI7Zt3JZv1g/NyEtO8u4Y0qiHYcMfkrWtqOhOciJ6gaOVNZxtKqeI1V1HK2s59DZAIcr6zh0to5AOzNkWwx3Ypg2ykvqKC9poxPxjUkkbXQiU0c6xLiVAxAROeevn4iUYWz5OdAyzznCV2fU4/P5OjqyNZVQFJFL+teu4+w6Wk1yvIcnPprH5BFK1MoQtuvp8HPGjeC+oJqPMwI++Ff444fgQGHbx1sLwbZvUvW4XOCOvajdWnt720PZTcDcC5pNq/0vt96OtK0CVl3Qdi/htenbPU5EREREZKDqTsI2UF74iuPLWerNyl/n313Q4Tqtxh2zofH4njzar1bXqyZ9YtWTnuTR70ldsXrVvpXL7ohGDK1ZG0oMlKyvjnYcMvgpWdvKff/cRUHZKY5V1XPa33jpA4BxyfFMHpHA5OEOU0d6SR3lMGWEQ+pIL8O9F1+cEBGRC1QegJ1/441EbZjbWHKGnYbTFTAyrTMjqYSiiHTIWssv1lQAcGveVCVqZWizFnZHkrUz3tp2H+/IcMK2I08sh9JnwF5wI6sNwsx3XH6cIiIiIiJynu4kbGPH+dY2HqtYAnTY37+7IORkLDjqzVw01V+yfn+PBNxJqStWx9jmxvdENm9PXbH6zmjPsDWYqM0wlqFFydpWDp2pY8eRN26SiHW7GDcsngkp8UxISWDCsAQmDk9g0vAEJqYkMCElgfgYdxQjFhEZ4IJNsO0pLkzUnudUWaeStSqhKCKXsnn/WbYcqCTW7eL2RanRDkckuo6/DpX7wRMPaW/q/jg3fAcOFUHgdHjdWuMOJ2qXroDRGT0Xr4iIiIiInHMuYZuetyRQtmntpfpXFjxuvTOXvuqk52YHygqLO+obKN24z0nPW+TNyj94qZm4PWnfymVNE+984I+hxrqbXLEJq3orUetkLhyF4cyl+iVMnxdrPHGdm9UncpmUrG3lQ/nTuGneRMYmxzN+WALDnRgi6yKKiEhPCTbBnjWw88+wezXUne24f8qUi5pUQlFEuuOBNXsAuGneRMYkxUc5GpEo2706/Jx2HcR6uz/OiGnwiUJ45eHwGrjOSJi7HKYt6Zk4RURERESkTZGE7StOWvaSQEXxJRO2/p1rarwzlhxx0vOyAmWbdnfUN3bs9A2Nx/csBNb3WMCdcPiX//kf3hmLc/271rWzFksPCAan1lUUvXqpbsYTN9u4XFt6LQ6RVpSsbWXe1OHRDkFEZHAKhcIXcLc/BTv+DHWtbl7zjg7PwvGfCM/IaWHcMCkbxs7s+3hFZNApP1HD87uOYwx8ZPH0aIcjEn0t69VmLbv8sbwjYUmPrDwgIiIiIiJdcC5h28mSyP5daw876XlZ3hlLJvp3rT3cXr/Kgsetk7HggJO5cFqgZMPeno26Y7axPnjpXpdzgpCx1l66vLENuvy7N/TZzGIZ2lzRDkBERAaxU+XwwrfhR3Pg4Rth86/DiVpnFMz/MNz2d/hsCXysACbOP//Y8VfCex6NTtwiMuj8cm14Vu2bZ4wlbXRilKMRibKz++D4djAuyPz3aEcjIiIiIiKXwV+y3m/cns2OL3dpZ/oHyjbtts0NE7wzlyZ12K9040GCzWOT5t44uCb9GZfWoZV+Z3D9IxMRkeirr4bX/whbHw+vYdciNglmvA1m/wdMWwruVl9BiaPhI/+CI1vhdHm4pOKEq0Gl6EWkm3YdrabsRC0ATc0h/rLlCAD/ufTSa2CLDAq1J2BvO5XQ9q0LP09dBM6IvotJRERERER6xbk1bDs5wzZQVljs+HKuTclf/nJlwePtJi/rKooLE9Ky84F1PRpwR9wxjfFTZsfVH9je0GfnvIB35tJxGNfJaJ1fhh4la0VE5PJZCweL4NVHYcefoCkQbjeu8Fp4c2+BzLdATELH40yYG37IkGCMWQHcB/zIWvvpKIcjg8iRyjre8bP1NDafX60oO3W4lr2QoeFUOTx4HdRXdtwv6619Eo50nzFmFXBbZLMZOAQ8BXzdWlvfh3HcCSwHrgaSgOHW2sq+Or+IiIiIXNq5hG163pJA2aZLrmFrPLFrGo+VLwHaTe5aa613xpI9TuZCX6BkQ3mPBtxeXC7X665Y5yrglb44X1tsU/3kQFlhcbTOL0OPkrUiItJ9DTWw7UkofghO7HijfWQ6XP0BmPNeSBoXvfik3zLGZAP/CWyLdiwy+Pyu6ACNzSHGJMWdK3kcF+Pic/+WGeXIRPpA4Aw8fnM4UZsyBVKmtt0vcQzMXd6noUm3PQPcAcQA84BHAAt8sQ9jcCJxPEP4RisRERER6YfOrWHbiYStf3dByDtjyWuOL3deoLyw3cRoZJ3b3MTZ1x2s3f5Cr8929e8uCDlp2dEttxcKqVSy9Ckla0VEpOtOlUHhA/DaE9AYLjOKJwFmvROuvg2m5KmEsbTLGJMI/Bb4KPDVKIcjg0xTMMTvig8C8LW3zuRtV06IckQifai5EX7/QTizB4ZNho+8EE7KykDXYK09Fnl90BjzPPBm4IvGmH3AD621P2zpbIzZCvzFWntvZNsS/s5dBtwAHAY+a639W2T/14GPAbOttacjbasJJ2ivs9aGWsY3xlzTi+9TRERERHrAuYRtWvaSQEVxxwnbXWsrvTMWn/RmLsrwl6wvba9fXXlhUZ+XQ+4tNtThRUtjjImfPk8XNqVPuaIdgIiIDBChEJQ9D4/dBD+dD8W/CidqR/rghvvgs7vgXb+AqQuUqJVL+Rmw2lr7fLQDkcHn+Z3HOVHTwKjEOG6YpZn9MgBZC80d3KweOAPVR9t+rP5MeD3a2CRY/qQStYOQMeYKYCHQ2MVDvwH8HpgD/AP4rTGmZcHi7wD7gAcj57grco7brLWhi4fqdKxxxpjklgfh8skiIiIi0gf8Jev9xhP7iuPLXXrJvrvWHbDYWG9W/vj2+lhrLS5PqZOxsG9KVrk9eLPyezx/5c3KH4/bc7yjPglp2VmuOO/unj63SEc0s1ZERDrW3ADbfg8bfwonW35OMZD575BzJ0y/RslZ6TRjzPsIr3eX3cn+cUBcqyZd6JUOPVa4H4D3Zk8i1qP7EmUA+vN/wu5/wPufCt8A1dqzXwl/H3fEuOA/HoKxs3ovRulrbzXG1BL+/T0OCAF3d3GMVdbaJwCMMV8G7gFygGestUFjzK3AVmPMysi+j1hrD1xm3F8inCQWERERkSg4t4atL3dpoLyw3XVpAQIlG153MhbkJc66tqp2x0uBNvuUbjjuZCyYnDjrWqe9Pj3FuNyv22DTFfTw8lm2qWFi3Z7NHa6Fa20oMbBzTU1PnlfkUnQFS0RE2lZfBeu+Bz+cDX+7O5yojU2CvE/APa/CLU9A2rVK1EqnGWMmAz8C3m+tre/kYV8Cqlo9DvVSeDII7DlZy/ry0xgD78ueEu1wRLqu6lD4BqnGmnA54+ojb+x79dE3ErUuT9uP+GHw1h9Cxr9FJXzpNS8Bc4FcwuvVPmyt/WMXxzh3kcta6weqgTGt2vYAnyO8Du7frLWPX2bMEF7bdlirx6QeGFNEREREusBfst5v3J7NTnrekkv1DZRu3BRqqM0zpv2LfYHSjZtDjYGrezbKi/l3F9QTCsX2+MDGZa21g3Y92tQVq2OiHYN0j2bWiojI+WpPQuH9UPQraKgOtyVNgLyPw7zbwheCRbpnHuELw6+2+rnfDSwxxtwNxFlrgxcccx/w/VbbSShhK+14vDA8CezazDFMHuFEORqRbnjtCSBy3cB/IpywvX01HHsdVn823P6mr8KSz0ctRIkKv7W2HMAY8yHgNWPMh621vyY8y/bCi2ltXaBpumDbcvHN20uAIJBqjPFYa5svJ2hrbQNwrqZ3B9f8RERERKQXnVvDNj1vSaBsU4dr2BpP3LpLrU1rYhJKnIwFMwOlG3f2eLBRljB9Xqxxx3R1yZF+IXXF6oeB21NXrF61b+WyO6Idj3SNZtaKiEhYzXF45svhmbTrvhdO1I7Ognf+Aj71Giy6R4lauVwvALMJzw5qeWwGfgvMbSNRi7W2wVpb3fIAVIZG2lTfFOSpV8J5/FvzNKtWBqBQCLY8Fn699Ivh79xDxfC3T8LvPwDBRsh6K+R/NrpxSlRF1pD9f8B/G2MSgJPAubXFImvDTuvquMaY9wLvBq4BpgBf64l4RURERKRndXfmpL9kvd+43K8kpM3vcA1b/+6CJhMTt9Px5c5tt8/Ol08SCsZ5Zy4deEtV2VCHdxAal2d27NjpPVp6uS+krlgd03hy/+2hpgYIJ2w1w3aAUbJWRGSoa0nS/mgObPoZNNfBhKvgvb+Fj2+EubeAp+erjsjQY62tsda+3voB+IHTkdcinRYMWX72Ujlf+fN2vvLn7dz9+Baq6pqYmJLA0owxlx5ApL85sAHO7gsvObDoU3DTQ4CBbU9C9WEYlQHvvB9c+hVOeIrwDNi7gBeBDxhjFhtjZhMuk3zRzU8dMcZMAu4HvmitLQDuAL5sjMlr1WecMWYu4Is0zTbGzDXGjLjsdyMiIiIinTL5nt8+EWpqaIzMoOwyf8l6vzs+qfiSCdtd604bT0ylk7nQ116fQHnRFttYN7c7cXSW8cRWdBRD9wZ1dVwC2eUylQWPD6gyySn5y82Jp+5d5Bk29jeumDiAVftWLruwso70c/pNX0RkqAqcgee+1ipJWw+TsuHWP8JHX4IZb9UFYRHpt555/RjffbaE3xYe4LeFB3h+13EAludOwe1SqU0ZgFpm1V7xboj1Qvr18KavhNtik8I3UcUnRy8+6Tci5Yl/CnwBWAmsAZ4GVgN/ASo6O1ZkPbJVQFFkTKy1zxJO3j5mjEmMdP0YsAX4VWR7bWT77Zf1ZkRERESkU1JXrI5xOynvC9acIug/2+2Zk7U7Xgq445OKHV/OpWbY7iMUdLxZi9u9G9oV593hZCyY3Z04OsNfsv4szU0pPTroJWbWDjRJc2/0NB4rvy7UVL/uwPdv+iAQqxLIA5MZxGsptylSFqqqqqqK5GRd7BCRIaihBjb+HDb8BBojFWUnZcM1X4K0N0H/X0+s3wcovUff49LiI49s5vldx1mSMZqrp6QAkBQfw615U4jzuKMbnEhX1VfD/2WEq1t8+HmYnB1uD4Vg19/CyxKMyYpujD1D3+FDWMt3+JYtW0lKGngV86R/GfHU96IdwoBw5maVzpeek5Y2Xd/jIlHWsiZpU+XRP5z91wP/E6go3tzdsRJnXeuEGvzZgfKiNR31c3w5i4wn9hX/7oL6NvdnLJzjio3fX/v6i1XdjaXD86dlz7+c99maMcYkpOVcHSgvfKWt/d6s/PFYG+svWb+/J87X27yZi7w22DS/rqJ4rR1qib5BSFOmRESGimATFP0KfjQXXv5/4UTtuNmw/Cn48L/Ad91ASNSKiHDW38ia0hMAfHXZDD59fQafvj6DD+dPU6JWBqYdfwonakdlwqT5b7S7XDDrnYMlUSsiIiIiIpchMmMy9vAvPnKziXNKEtLmL/Vm5cd3Z6zaHS8FXHHeYseX2+EM20B50fpQY/2CSEWWi/eXbtgWrKvptdm1PcnJyp+M4XB7+22wadIAStQOt8HmKwPlRWuUqB0cBmSy1hiTZIz5oTFmvzGmzhizwRiTHe24RET6JWthx1/gZ7nwj89B4BSMSIP/eBjuXAsZ/6YkrYgMKKu3H6UpaJk5PpmMsZqdJYPAq78JP191q76TRURERESkXS1rkfp3rqmpq9i8xgabrnTSc6d0Z6xwwta5ZMLW7R22ISEte2F7+43Ltd3x5czpTgyX5HKd8Gblj++JoWxT4+hAWeGxnhgrmrxZ+eNtsHl6oLxwQ7RjkZ7jiXYA3fQgcAXwAeAIcCvwvDFmprW23TsjRESGnMOvwDNfhoObwtve0eFyx1d/ENzdWtpCRCTq/ro1/OPeO6+aEOVIpEe89iS8sgpsqG/Pm3EDLP5Mx33O7IXnvgrZHw4vFdBdh1+F5++F5oaL99kQHN4MLg9c+b7un0NERERERIacQFlhoZO50Of4cq4KlBdt6erxtTteCiTOurY4IW3+0rqKzW2WRK7d/kKDk55b6vhy5gTKi7a1EUOVk7HAJs66dkTtjpfOdOd9tCdQVnjAScueDxztyXEvZIwxCb6cyxojdcXqmJZkem9xMhakYkPJ7ZVyloFrwCVrjTEJwE3AO6y1ayPN9xpj3gZ8HPhq1IITEekvqo/AC9+C154Ib8c4sPCT4UecZqFJ5xlj9gDZ1trTF7SnAK9aa6dHJTAZsg6eCVC87yzGwNuvnBjtcORyHdsOf70LQr36+2zbDm4KrwWb9Za29web4A93wJEtsG8dfGITJHfjBoGGGnjqNqg80HG/GW+DxDFdH19ERERERIa0QMmGcu+MJSkJ0+ddA2yo2/NKY1eO70zCNlBWeNLJWJDkZC6cFijZsPei/aUbtzvpuflAQbfeREeMq2fK/NpQu2WMEqbPn2Fi4nd3d+jJn3zsceOJuyV1xepVkXLVPc6buSgDjAmUFV6UMB9K+iIpHg0DLllLOGY3cOGC1nVA/oWdjTFxQFyrJmUpRGTwam6AjT+Dtf8HTf5w25W3wHVf794FZhFIJfy9e6E4QJky6XN/e+0IAAumj2TcsG4tzSP9RXMD/Ok/w4la3/Vw9W19d+6yZ2HLY/D3e2ByDnhHXdxn3ffCiVqA+qpwUvnWP3W9TPEzXwonaodNgRv+G2jjeJcHpi3u8tsQEREREREB8O9aW2mMWZOQlrPAm7nogL9k/aGuHN+phG3pxj2OL3eud8bikf5d605fuN/ExL/W3Rm+HeogydolHSR9LdYb2LmmpjvDpq5YHROsq7kldtRwgNtTV6y+s6eTiU7GgtnYUG2grPCiRPlQET9ltnvEdXf+KXbs9Lf3ZlI8WgZcstZaW2OM2Qh8zRizCzgO3AIsAMrbOORLwDf6MEQRkegoex7++QU4UxHenpQD/74SJs6LblwyIBlj3t5q8wZjTFWrbTdwHbCvT4OSIc9ay5+3tJRA1r0CA96L/w0ndoAzCt75C0gc3XfnTv+3cGniEzvh75+C9z52fhL28Cuw5n/Dr6/5EhT8ACpehOIHIeejnT/P7n/Alt8ABt51P6RedG+piIiIiIhIj7DWWmCD48vNcNLzrgyUbXqtK8e3JGwdX+7SQHlh2wnb8sKtTlr2Em9W/kb/7oLzEpL+nWtqnPS8Ric9d3SgrPDkZbyV8xnj985cmuTvZjK1t5146t7pI9/yqd8B7wNW9Xii1pc7D5f7aKCs8EhPjjtQeDMXTbLNjRPd3uE2Zkxqy/XKXkmKR5Mr2gF00wcI35J+GGgA7gGeANpa6Oo+YFirx6Q+ilFEpG9UHYLfvR9+e1M4UesdA+96AD78nBK1cjn+EnlY4JFW238Bfge8GfhsNAKToWvHkWrKT9QS53Fx4xXjoh2OXI5962HDT8Kv3/7jvk3UAsTEw7t/Ca4Y2P00bH38jX2NgfCMXxuEWe+CpV+E678Z3vfc1+BUW/eHtsF/KjxzF2DBXUrUioiIiIhInwiUF5Yal/uA48tZmjT3xi5N2Kvd8VLAFecUO77cpe31qduzeZ1taljU5rnLNu3A2qyuxtyRuori3bYhkNmTY7bmzcqPNy5Pl0pHt0jJX24wrpEHf3LrLUBsT8/2dHw5ebjd+wOlG4ZUojYlf7lxfLnzHF9OjrUhE6goLvTvWltkjGtVpEuPJ8WjbcDNrAWw1lYAS40xXiDZWnvUGPMksKeNvg2EE7oAmK6WLRMR6a+CTVD4C3jpvnDJY+OGvI+HLyrHJ0c7OhngrLUuAGPMXsJr1p6KckjSz60rO8lzO463u39YQgwfvyYNb1z3f/z8S2RW7fUzxpIcH9PtcSQKdv8DKl54Y7vkGcDC3Fsha1l0Yho3G679MrzwTfjnF8OzaY2BU6VwugwSx8Gy74fbcu6Ekn/A3jXw5zvhQ8+Bu4O/y9aGZ+z6T8KYmfCmr/Xd+xIRERERkSHPX7L+bEr+8rWNxysWebPyK/y7C4529thLzbC11trEWdcWOb6cvEB50aYL97vivK84GQvmB0o3br7c99FyPict+7LGMMaYBF9um2WQbVPDFbHj01/pzriNxytyQo2BQoCeTB4aY0zC9PmLMa5XAyUbantq3P7OyVg4llDzVADjid1y4eztfSuX3THYZtS2GJDJ2hbWWj/gN8YMB24AvhDlkERE+sahV8KzdY6/Ht6enAdv/T6MnRXduGTQsdZOi3YM0v8dOB3go49upr6prSInbzgbaOQ775rdrXMcq6rnd8UHAZVAHnCO74DffwBCzee3p0yBG++LTkwtFn0KSp+Fg5tg86/P3/fOn4EzIvza5YJ3/hx+vjCc1N30s/Cx7dnxp/CMXVdMuNpFjNZXFhERERGRvlVZ8LgFCpyMBTMjpYm3dfbYS61hW7vjpYCTsXCvk7nwikDJhtfb2FfjZCwcGyjd0P5d3X0oIT1vmg027W9vf+Sz6hInPXcExlVTf2B78PKiO583K9+VkJb9JuOJXXNhsnIwMsaYhLTs2UA8LvepQHlRUUf9B2OiFgZostYYcwPhMsglgA/4LrAbeDiacYmI9LqG2vAaf4W/ACwkDIc3fxvmvj98IVmkFxhjriO8Ru0YLlhCwVr7oagEJf2GtZYv/3k79U0hZk8cxrVZYy7qE2ho5sGCvfy28ADvmDuRnGkjunyOr/7ldWobmpk7OYU3tXEO6adCQfjbJ8OJ2sl5MD1SScu44Yp3R78ShMsN73kUtjwarljRYtxs8F1/ft9hk+CG78Df7g5XtZjxNhgx/eIxA2fCM3UBFn8Wxs/pvfhFREREREQuIVC6caeTuXCU48tZajyx6/y7Czq+0zriUgnbQOmG4056XrI3c9FUf8n6/RfsK0mYPm+xMeZEZC3dy+P2BOPG+TwNx8qbL925DaHQ8LqKzRdVZr0soVBmoKJwY08O6c3Kj7fNDQtjx6W/0J0E8kDizVw03DY3ZiSkZVvjid3m311QH+2YomlAJmsJrz17H+H1Z88AfwS+Yq0dlBl1EREAyp6Hpz8NVeGZZcx5L9zw/8A7KqphyeBmjPkG8HVgM3CU8Bq2Iuf86dXDFJSfItbj4se3XMW0Ud42+9XUN/Pk5oN86U/bWH3PYuJj3J0+xz9fP8bzu47jcRn+56Y5uF1a1mLAKPpVeCZqXDLc/DAkT4h2RBdLGgtLPt+5vlfdCtt/D3vXwt8/DR/8a7hMcmvPfS1c/nhUJiz+TI+HKyIiIiIi0lWBkg2nvFn562xTQ743a/Fu/+51Jzpz3LmSyGnZSwIVxWsvGrdsU5mTsXCed8aSKv+utZWt97m9KUUJ0+fnAIWXG3+ovnabO3n0HGDL5Y7VmjcrfzxuT5dn/zrpeem4Y3o0+eudsSTFNjfODpQXv9iT4/Y3TsaCmYSCiWAqAxXFl/13Y7AYkNOwrLW/t9amWWvjrLXjrbV3W2uroh2XiEivqDsLf/kE/PamcKI2ZQrc+kd49y+VqJW+8DHgdmttrrX2ndbad7V+RDs4ia5TtQ18e/VOAD59fXq7iVqAL79lBqMS46g46efnL5V3+hxVgSa+/tcdAHzimjQyxyVdXtDSdyoPwAvfCr++/t7+majtKmPgrT8ET3x4/drXnjh//56XYetjgIG3/wQ8cVEIUkRERERE5GL+3QWhQEXxWhtsGu2k53V6LbXaHS8FXPGJmx1f7tK29gdKN7xim+qvjJ8y+7y7smu3v9CAJ+a0d8aSy17LqP7A9iDB5s7f9d1JNhScGCjdeLArx6TkLzdYO7onSzx7sxaPsU31GYHyonU9NWZ/4s1c5HXSsnOdtOxcY1yHAuVFRYHywtJox9WfDMhkrYjIkFHyT/hZHmz9LWAg7xPwiU0Xl2YU6T2xwIZoByH907ef3klloIkZ45P56OI2ysG2MsyJ4ZtvD/8ueP+aCkqO1XTqHPf9cxenahuYPtrLJ671XXbM0keshac/A01+mLIA5t0R7Yh6zsg0uGZF+PWzX4bak+HXjYHwbFuA7A/DlNyohCciIiIiItKRQNmmHRjXKceXszglf3mnSlfV7ngp4IpzittL2MaO8611xcQvvuhcJRvKbVP9dGMuLEk0cDUeLctxJw7vcF3VrnAyFky2wabxl1qrdSBy0vPSnbTsXBtsmhaoKC4MVBQX+kvWV0c7rv5ooJZBFhEZ3Ooq4ZkVb8zYGemDd/wMpuRFNSwZkh4ElgPfjnYg0vf+9toRVvxxGw3NbS9nEwxZXAZWvns2Me5L3wP4ltnjuH7GWJ7fdZwbf7QWVyd+VwuGwpW3V757TpdKJ0uUvf5HKP8XuGPhbT8efOuqL7g7/B6PbYf/SwfjAizYECRNgOu+Ee0IRURERERE2hUo3XA8fsrsU41HyxZ7Zyze4d+17vSljuloDdvKgsetd+bSVxxfTs6FSUd34oiNCWnZucCmywrauKwxxvTIGriAMcYk+HK6lET2Zi4ajstdW7P1me6tnXsBJ3OhD4gNlG16rSfG6w8SZ18XFwpUz8UYi9uzJ1C2qSzaMQ0Eg+yqiYjIIFDxEty/MJKoNbDwk/CxAiVqJVrigc8YY9YYY35ijPl+60e0g5Pe9ZuN+wg0BgmGbJsPgI8tTePKySmdGs8Yw7ffOYuR3lispd1xLzzHHYtSyZk2orfepvS0hlp47qvh14s/B6MzohtPb3DHwNt/CjEO4SRtMJyoNW54248gPjnaEYqIiIiIiHSo/sD2YKCieK1tbpzopOdldeaY2h0vBdzxScUJafMvmmHr37mmBpfnkJOxYGbr9pqtzzTjcp9wMhZMvpx4jduzOyE9b8bljNFagi93pnF5dnblGNvcmBUo27SjJ87vZCyYibWhQOnGLsXQXzkZC1IdX05OqK52Rt2ezUWB8qKiQMmGU9GOa6DQzFoRkf6iMQD/+joU/yq8PXwavOsBlVGUaJsDbI28vuKCfT1yJ6P0T1V1Tbx6oBKAv9+dz5jki9fe9LgMIxO7tibn+GEJrF/xJqrqmjrV3+0yjPTGdukcEmXrvgc1R2F4Kiz6VLSj6T0T5sLnyqChVUnvWAfih0UtJBERERERka4KlBVuczIWTnB8OYvqKoo3XGrmakczbAOlG4446XnJ3sxFk/wl6w+90b5xj5Oem5+Sv/xQZcHj3bqe5C9Z73fSsp2uHufNyndhzEXntKFmJ1C2yd/ZcZy0bB8u196unr/NsdLzrjQu9+nWn9FAFD9lttsVmzAPa42JiTvgL9046Eo59xUla0VE+oMjW+CPH4XTkaoQ2R+FN38TYr3RjUuGPGvttdGOQaJjQ/kpgiFL2mgvsyf1bPIpPsatksaD1ekK2PjT8Osb7oOY+OjG09viEsMPERERERGRASp1xeqYQOmGI96s/JPx0+ct8WYu2uYvWX+2o2NaErZOWvaSQEXx2tb7AmWbdju+nBxv5qLq1uuTGnfMhsZjFQuADd0O1ri6nui11ge2otvnJFwyOX7avDF1ZYXllzMOgJOem2vcsXv9u9eduNyxosXJWDiBYNNkV0x80HhiN/t3F7S9flYrqStWx+xbuaxzd+4PQSqDLCISTaFgeAbSg9eHE7WJ4+DWP8Gy/1OiVkSiak3pSQCWZoyJciQyoDz7ZQg2Qtp1kPnv0Y5GREREREREOjDp7keftDbUmLpi9cP+3QVNdRWb19hg81QnPS/9UsfW7ngp4IpP3Oz4ci8qiRwoLyoKNTXM82bln8tB+XcXhHC5jjgZC1K7HbANdWmNWQAbbE4KlBVWtW7zZuXHG+Nq6OwYCdPn57hi44u7eu4LOb6cxa447+6BmKhNyV9unPS8qx1fTo4xJiZQUVwYqCjubKL2YaAx8ixtULJWRCRaqg7BI2+HF74FoWaY8Xb4xEbwXRftyETOMca8ZIx5sb1HtOOT3mGtfSNZmzk6ytHIgFH6HJQ+Ay4P3LgSTJd/hxYREREREZE+krpidYzbO/w9zacPE/RX3p66YnUMQKC8cCvG1Du+nDxjOv7FrnbHSwFXnFPcVsLWFRu/xjY1LGndFijduI9QcGLrJG6XuGMOXlayNyLUVD+7rqJ4e2f6emcsScHlqvPvLuj2rNCU/OXG8WW/yXhii2tff7Hq0kf0H0567mgnLTu38Vh5tnF7tgfKi4r8Jev3d+bYlPzlJnHWtdlNZw7fHmk69/dMzqdkrYhINOz6O9y/CPYXQGwivOPn8J5HwRkR7chELrQVeK3VYycQC1wNdOqHWhl4yk/UcrSqnjiPi9xp+n9JOqG5EZ5ZEX6d93EYnRHdeERERERERKRD+1YuazLGtSpm1GRsU/1TJ/7wzZyU/OUGIFC68aArIWlLwvT5i72Zi5I7GqclYZuQNv+8hK1/d0HIxCZsddJzs1u3x47zbbDNjYu6E3OgdMNxgs2junPshS61Nu+5fk31swJlhdu6e57keW+LbTxWdq3xxL3s311Q391x+pqTnjvHScvOxZIcqCguDJQXFXUmYW2MMU7GgtmOLyen8Vh5drCu+rWYERNXRXavUinktmnNWhGRvtRUFy4Rufmh8PaEq+GmB2FkWnTjEmmHtfa/2mo3xtwLaKHGQaplVm3e9JFaW1Y659VH4EwFeEfDki9EOxoRERERERHphH0rl92RumL1nYd+8eEmb9bDrsZj5Qu9mYv2+0vWH6rd/kIDsNbx5c5zMhacDZRu3NPeOC1r2CakzV9aV7F5TUu7f9faSidjwTEnPS8rULZpN0BlwePWyViwz8lYML2jMdvV1XVru1E6uYWTudCHy7O3u8d7Zy5Nsg2Bq+r2bH6ps8nhaPLOWJJiG+syMca64ryv1+54KdDZY52MhZmEmoclpGVjjGuHv3yjv9XuO1JXrL5Tidr2aWatiEhfOVkKv7rujUTtok/Bh55VolYGqseAD0U7COkdb6xXqxLI0gkNtbDmf8Kvl34R4ju86VpERERERET6kZYEmn93QShQXrTe2lBs6xLIgfLCV4CQk5ad29E4tTteCrjjk4qdtOwLSx8fBNxOxsIJ57WFgmO8WfldL4nb1eTrBcldJ2PhBOOOOXbJw4wxBJvHBko3HOlihAB4ZyweaRvrZgUqitf290St48uZ4aRl59rmxnEts2g7k6h1MhZMd9Lzcp30vFzj9hwPlBcVRcok+y/sq0Rtx5SsFRHpC1sfh18uhRM7wrOOPvBnePO3wBMb7chEumsBMGBKt0jnBRqbKdxzBoAlStZKZ2z8GfhPwojpMO/2aEcjIiIiIiIilyFQunGP8cS+kpCWvbAlwRoo3bjPFZ+4PSFt/lJv5iJve8fW7ngp4IpP3HxRwrZs0w5CzVNaH1tXUVwYamroMAHcJrfnjJO5sPulkG1wYiSB3KEEX26O8cQWdecUTsbCCba5aWqgvGhTd47vC4mzrnWctOxcJy0718TEHw1UFBe2zH7uiJOxYLLjy8lxfDk5xuWpDZRtKgyUbSr071pb2QdhD1oqgywi0psa/bD6c/Da4+HtaUvh3b+CpLHRjUukk4wxf7qwCRgPzAe+3fcRSW8r3HOGxmCIiSkJpI1u9/cvkTD/Kdjw4/DrN30V3F2/KVpERERERET6l8japOud9Lx0x5eT02qm5RrHl5PjpOeeDJQVtlkeOFISebPjy10aKC88VxI5UF60yfHlXJuSv/zlyoLHrbXWOhkL9zjpeemBsk1lnY0tULpxj+PLnQeculTf+Cmz3a74xNB5jZ0oo+zNXJSMcdV1Zo3WCzmZC6dh8AZKN73a1WP7gpOxYDqh4CiMqa/bs7moM7N+nYyFYwk1TwUw7pgj/vKN3UpiS/uUrBUR6S0nS+D3t8HJXWBccM2XYfFnwKX1H2VAqbpgOwSUAF+31j4XhXikl50rgZw5mkjFI5H2rf0uNNbC+Cth5ruiHY2IiIiIiIj0oEDZpjJvVn6848tZZDxxZf7d604EyouKnMyF05y07PmBiuLNbR3XsobthQlbd+KIdY1HyxYDawECpRuOOL6cnMTZ18VF1sjtnE6WQnZ7UzJDdTUlLdvGGJOQnnfp4YNNswPlRes7HU+Ek56XBQQDZZte7+qxvcmblR9jmxuvwlpjYuL3+juxVrA3c9FwG2xKxxhr3DEn/aVFStD2IiVrRUR6w7an4O+fgiY/JI6Fm34N0xZHOyqRLrPW3hHtGKR31TcF2XGkmpYbKV8qOQFovVrphLP7oPjX4dfXfxNcWmFFRERERERksPHvLqgH1jsZCzMdX+7kQHnhK4GSDXu9M5eeSkibv9QVE18Y6XOecwnbtOxrAhXFLwPUbH2m2Zu5aLuTseCqQOnGLQB1FcXFCWnZ+cC6no7dNtY7dfu2nlt7NSFjwSyXJ25nR8c4GQumG3fM/q6ey0nPnWNcnip/yfouH9tbnPTcKYRC4zGmMXacr7iy4PEOZ9F6Zy5Nsk0NM7Eha9wxlYFyJWj7ipK1IiI9qbkBnv0yFD8Y3k5dHE7UquyxDHDGmHnAjMjmDmvtlmjGIz3n7sdf5fldJ85r87gMC9NGRikiGTBe/h8INcH0ayDt2mhHIyIiIiIiIr0oULqhxJu5yOv4chbh9pQESjacAtY46bm5Tnru0UBZ4YELj6nd8VLAm5W/yfHlLA2UF60B8JesP+vNXJTsZC70BUo2lEfKIZc66XlZnVkzFQC3p8HxZScEyovrOux3YcnjYLNTW7Ih0E7v8MzbtOzx/vKNXZpV62QsmI9xH/SXrD/eleN6gzcr32WDTfMIhVy4XIcCFcWFHfVPnH1dXKiu5kqMy+By+QNlmzrsL71Dt7+LiPSUqkPw8FveSNQu/hx88K9K1MqAZowZY4x5ESgGfhx5vGKMecEYo6mXA9zWg5U8v+sELgPTRnnPPe5+k4+keK09Kh04sxe2PRl+/aavRzcWERERERER6RP+kvX+QHnRekKhsY4vdy5AoKywEJc71vHlXNXmMbsL6o0nttDx5SxtNc5+LI6TsXAsQKB0w3FCwcTEWdc6nYmjrmzT6+Ca2QNv6TwJvpz5roSkNks7t8fx5SwyxlURKN0Q1UStNyt/vOPLybFNDVeHGgKvBiqKCwNlhYfb6hs/ZbbbyViQ46Tn5Ybq/Vl1FcXFgbJNhYGSDf2qfPNQopm1IiI9Yc/L8IcPQeA0xA+Dd/8KMm6IdlQiPeEnQBIwy1q7C8AYMxN4hHDi9pYoxiaX6acvlgPwrqsm8b33XBnlaGRAKfgB2CCkXQeT5kU7GpF+wxjjAZYDz1pro35XvYiIiIhIbwiUbdrhzVyUnJCWne/yxO4IlGwo985YkpKQNn8p1m6s2/NKY+v+/t0F9d6s/MLWM2wDpRu2Ob7chd6s/Cr/7oL6QEXxZseXs5hOlEO21lonLbtT69a2cHzZCRjXReWaW3hnLk0C09jZtXNT8pebxqNlS40ntthfst7flVh6SmQm8FysjcXtOd5R2eKU/OWm8cTeuYSCMa74xJBxuTf7dxeE+jJeaZ+StSIil8NaWP8jeOGbYEMwbg689zcwPDXakYn0lBuB61sStQDW2p3GmLuA56IXllyunUeqeX7XcYyBT1ybFu1wZCCpOgRbHw+/XvqF6MYi0s9Ya5uNMb/gjaUDREREREQGJX/J+mqgwEnPneOk504OlBVuM8asTUjLWeDNXHTAX7L+0Hn9dxfUJ115Q9F5Cdvywg0J0+dfa4x52VprcXt2Oel5swJlm3b0dLzW2ivqK4ranTVrGwJXBCqKN3ZmrKS5N3qCtaeXuJNGrq3Z+kxzz0XZOU7mwlE0N6UlpGVbV0LSa+0lmI0xJsGXMxtr4zHGGnfMVn/pxqa+jlcuTWWQRUS6q6EGnroNnv9GOFE791b48HNK1Mpg4wLa+iGuCf0cMaD97KXwrNpls8eTNjoxytHIgLL+x+G1alMXw5S8aEcj0h8VAXOjHYSIiIiISF8IlBVuMy7PwYS07HwnY2FSoLxwgw02O0563kUlvGpee7bOeGILnbTsa1ra3N5hG+Knz18EECjZcAobig3Pcr0Elzvkzcrv0rUpa61tq91Jz51mPLEHOzNG4qxrnaD/bH5dxeaXLpWoTV2xukfXmHJ8OVc4vpwcQqHhgYriwkB5UVFbiVpv5qIZTnpeboIvZ74r1ikPlBcVBcoKi/27C5So7ad0kVVEpDtOV8CD18POv4IrBpZ9H97xU4hJiHZkIj3tReBHxpgJLQ3GmInAD4AXohaVXJbyEzX84/WjANz9Jl+Uo5EBpeY4vPpI+PWSz0c3FpH+6+fA940xdxtjFhhj5rR+RDs4EREREZGe5i9Zf7auorjABpunO+l5swLlhaXG5T7g+HKWJs298bwKr/7dBfUmJm5TS8K2dvsLDcYTs7sluRsoL9oSaghccq0m4/Zst8Hm2e3tjxvn8+Bydy45GQqNu3AmcFu8mYuGhxr8VwXKCl9uL/HbYurn/vxIsPZsY+qK1Q93Kob2z5nspGXnJkyflwNmXzjxuqnswn5OxoLpTnputuPLycHlPhoo21QYKCssrt3xUuByzi99Q8laEZGuKn0OfnktnNwNSePhjn9A9ofBdGmZBJGB4m4gGdhnjKkwxlQAeyNtn4xqZNJtP3upAmvh32aOJWtccrTDkYFk40+guR4m5cC0JdGORqS/+h0wjfDa7uuBrcCWVs8iIiIiMoj19GzKgSRQXrgVl+t4Qlp2PtAYO863Nlh7JteblT++db9zCVtfzlKIzKg1psbJWDAdwOWJ3eGk53Z4o6N/d0EToWC7n7Vn2NgZofqaEgAnPXeicccca6uf48udZ2LiLvlzujcrf7wNNfsC5UXrL9U38Yo3zW2uPvlBlzcF4Pbu/J1wMhZmOmnZuba5cVKgoriwbs8rRYHywtrz+yyY7Phycpz0vFzj8tQGygqLA+VFRf5dayu7ej6JLq1ZKyLSWdZCwffhhW8DFibnwnsehaRx0Y5MpNdYaw8aY64GrgeyIs27rLXPRzEsuQwHTgf469bDgGbVSsTqz0H5vzrXtyr8d4cln9dNSiLtmxbtAEREREQkOiZ9YtWTtrnxPZPuevTJQz/74PuiHU80BEo2nAIKHF/uvMbje/yB8qL1ji9nhpOeOzpQVritpZ9/d0G9Nyu/sGUN20Dpxj2OL3euk7lwVKBkwynHlzPZO2NJSoeJR+Nqd3arDTbH1h/Y3gBgQ8EJdRWFxRf28WYu8mJMs393QX1H78mbuWiqDTUPC5RdPMZ5/bIWjwk11vlcMXG7YkZMXAXcDqzat3JZp2b4erPy421Tw5UAxhNb6q8oLmmjz3jb3DgZY6zxxB3xl24s6szY0r8pWSsi0hmNfvjLJ2DnX8Lb826Hf/8ueGKjGZVIn4iUdflX5CED3MulJwhZyJ02gjmTUqIdjkTb4Veg+FddO2ZSDqS/uXfiERkErLX7ox2DiIiIiPS91BWrYzzJo98DEAxUvdc7c+kPbVN9sGW/8cTu8e9adzp6EfatQHnhK07GwrEJadn5rpi4zdaGEhPS5i91xcSv8+8uCEEbCdvywq0J0+ct9mblbwqUF22LzNAtaPckNmSMMeZSJYkNps39trlxTqCieGNHxzrpeekYl6t1ovlCKfnLTePxPXnA2bo9mzdEmu9IXbH6zs4kap3MhdMINo8G6uv2bC668P14ZyweaZsbp2OtweU+ESgvUoJ2kFGyVkTkUioPwBPL4fj28Pq0b/lfmP+haEcl0ieMMT8Gyq21P76g/W7AZ639dFQCk27bdqgKgNzpI6McifQLm34Rfp7xNlj4qUv3NwbGzNCsWpFLMMakAZ8GZkSadgI/stZWRC0oEREREelV+1Yua0pdsXoVcLvbGbbKv3PNppZ9xhiTkLEgzUnPnX7ugFDImjinxL9zTU0Uwu0TgdINx4HjTnpuNphKV0z8OtvUkO+k5+4KlBWehFYJ2/TcawJlhS/X7321IH7avGuAl1ye2G2OL3duoLxwa1vjm9iEcidjYTpQ2l4MxhiTkJ53UbuTsSAVt6fDdWqdjAWzcblqAyUb9rbbJ3Ohj2BwTKgxUFh/YHuw9b6OErVJc2/0BP2V8wgFwR2z78IErHfm0iTbWBf+fcLtOXOpWb0ysF12stYY81Bn+llrldkQkYFn/wZ48gMQOAXOKHjvYzB1QbSjEulLNwFvb6N9A7CC8IVoGUC2R5K1cyYOi3IkEnXVR2HHn8OvF38WJlwV3XhEBgljzA3A3wivUduyntUiYIcx5m3WWlWqEBERERmk9q1c1uZsyshMyfLWbcYY42QszHLScxOxhDC4MC5rXO7XL1WWd6AJlBUWe7Pyx9umhnwTE7fRBpsznLTs0YGK4p1wLmG7yUnLvsZa+7I3K3+j48tZFCmh3OydsXhkW7OS/bvWVjq+3LSOzp2QsWCWyxO386IdwebxHc2qddLzrsa4jgVKNhxpa7935tIk2xC4AnfM/kB54Ya2+rQ5bsaCyQSbJ2BczbHj0ooqCx4/N4vWm5Ufb4PNV2JDFpe7VjNoh46emFl7O7Af2ALoFnsRGTxeWQWrPwuhZhg3B973OKRMjnZUIn1tJFDVRns1MKqPY5HLFGhspuxE+Ibd2ZOUrB3yNv8aQk0wZYEStSI9ayXwA2vtitaNxpiVwP+gZQVEREREBrXOrk8aSeDuat3mzcp3Ye0sJz03/lyjcTUHq09ubzhW3tyzkfYt/+6Co8aYYwnpebnG7Tlpjas2IS07P258+vrKgsftuYRtpCSyk7Gw3EnPnRMoL9qWMH3eYmBdt04cCibU7ngp0LrJ8eVc5YpPfK29Q5z0vAW4XGWR9Xcv3u/LuQqMvVQJ5RberHyXbW66yoaa3cYTezRQUVzYsi9+ymy3Kz4xPMPWuBrqygsvKoMsg19PJGvvB24BpgEPA49Za8/0wLgiItERbIbnvgKFkdKQs94F7/g5xDrRjUskOsqBG4GfXtD+78CergxkjPk48HEgNdK0A/iWtfaflxmjdNKOI9WELIxNjmNscvylD5DBq6keNj8cfp37sejGIjL4zADe00b7Q6gihYiIiIh0ILKW6/bWbXHjfB73sDFXOOm5MQCEQhZ3TKNxuV5vWft1oIgkITd5MxdNwoYyQ3XVGxuPli32zli8w79r3ekL17B10vOSnYwFqcbl2eL4cucFygtfuXjQUJcmEXozF3nB2AsTuBCe7Rw/fd4SV5z31bbKUzvpuRNtMDjVFRP3mr9kvf9S53IyFo4l2JRqsSHb1PBqS5nkyBq3V2FDHlect9m43Jv9pRsH1J+l9KzLTtZaa+8yxnwGeDfwIeA+Y8xq4NfAc7oDQEQGlLqz8NQdsOel8Pa1X4Uln9PafDKUfR/4qTFmNPBipO064LN0/YLzIcKlk8sIV+O4DfirMeYqa+2OnglXOtKyXu3siSnRDUSi7/U/hEv8D5sMWW+NdjQig81JYC7h77vW5gIn+joYERERERnYIjNqt7Zu82blx9tQ8GonLRtcrvCFS+Py15Vt2jUQcjL+kvWHUvKXH248vicPYw7Y5saJTsbCUYHSDSVtJGyvNm5PlQ02BbxZi8f4d687/2dql+uoN3PRJH/J+nPrz8ZNyIr3JI9udHzZCbg855WUts2Nc9qaEevNynclpM2/xoZCBf6daxpb70uae6MnWHs2B5frWN2ejksehxO+8+cYiAdOtMyiNcYYJz13DhCHcVnj9rzm313QqVnYMvj1xMxarLUNwBPAE8aYqYRLI/8c8BhjZllra3viPCIivepUOTzxXjhdDjEOvOsBmNnWUp0iQ4e19iFjTBzwFeBrkeZ9wMettY92cay/X9D0lchs2zzCs2yll20/VAnAHJVAHtqshU2R6hHZHwF3j/xKICJv+BXwS2PMdMJrvEN4zdovEr4JSkRERETkskTWtN3cus3x5SYmpOfNc9JzDaGQDSdxTWVdeWF5f0zgRtZq3ejNXDTVhoJJ2ObTji9nUV1F8QZr7fkJW1/O0lBjXYGxdgEX3AAZKCs87Phy5xGeJACAJ3lUZnPl0R0ub8pV9WWbzn1OTnruFFyuwxfG4s3Kj7fNjQtix6W/1HoN2fAxeVlgh8WOS9t44b7zxshcNNwGm9Ljp12NKyZuW8u6w97MRTNsqDkxwZeDK9bZ0daMXpHeuDITAizhGTPuXhhfRKTn7XkZfv9BqK+C5ElwyxMwfk60oxLpF6y19wP3R2bX1vXETVjGGDdwM+AF2l3fI5IojmvVlHS55x7Kth2OzKxVsnZo21cAx7eHb0y6+oPRjkZkMPo2UEO4CsV9kbYjwL3Aj6MUk4iIiIgMcoHywlouSOB6MxcNT0jLudpJz3UBYAkZt+eUv2T9/mjE2BZ/yfr93qz8g7a5cYE17iPx0+ct8WYu2uYvWX/Wm5Vf6KTnXhM7zrem4UjpNcawyfHl5ATKi4o6GtM2N3kajpU3J6TN57xEdSg08cJZtd7MRck22HRloLzopQvah9tg8wwTm1Dm3/ny7vbO5aTnzSIUTMS4zrbE5WQu9DnpeSMBjMu9K1BWWN2Nj0aGkB5J1kYupLaUQc4HngbuBp6x1qrOtoj0b5sfgtWfAxuESTnwvt9C4phoRyXS71hrT17uGMaY2YSTs/FALfAua+3ODg75EvCNyz2vQE19E3tOhpdTmTNRydohbfND4ec57wVnRHRjERmEIheDfgD8wBiTFGm7aL0rEREREZHe5i9ZfxY4b51X78yl45z03GwsIQwuAOOOOeTfXXA0KkFybq3e9U7mwmmETGOoqWGqk543KlC2qcyblb+p8Vj5EldMXJENNc/BctablT++w3hbSkO34vhy5xpP7LbWbd6sxWMwxhcoL1rX0maMMQlp2dkYlz9Q3nbJY8eXm4gNzQIwnthd/rJN1d7MRVOd9LxcsCFc7r2Bsk3l3f08ZOi57GStMebnwPuAg8BDwC3W2lOXO66ISK8LNsNzX4HCSCnI2e+Bt/8EYuKjG5fI4FZCeM2+YcB/AI8YY5Z2kLC9j/NLRibRqqyNdN7rh8M3cU5MSWBkYtwlesugFTgDu58Ov55/R3RjERmkjDEvAu+21la2TtIaY5KBv1hr3xS96ERERERkqPPvXHMMONa6LZxozM0+Vz7ZWmNi4ir8u9ad7svYAiUb9ibNvfFgc83pPIzb4/hy8uoqigudzEWFtrkxF5enDELDQ411PmPMsXOzZo2p9s5YkuLftbayZSzj9vyeUPBmYwxAM8acwNrfGmO+bq2tdzIWTE66atmvEqZddUPqitWr9q1cdoeTsSA1ftq8Ca6EpFdqt7/QcGF8TnpeOqHgCIzxByqKC71Z+eNtsCnTSc8F4z4cKNtUeOExxpg7geXA1YSvaw231lZe2E+Gtp6YWfsx4ACwB1gKLI385T+PtfbdPXAuEZGeUV8Nf/gQlP8rvP2mr8Liz0Eb/3+JSM+x1jYCLXcWvmKMyQY+BfxnO/0bgHM/HLf1M4Z0zjatVysA2/8AwUYYNxvGXxntaEQGq2uA2Dba44HFfRuKiIiIiMilRcoin1ca2clYMN1Jz50OQChkAYwnttRfsr5XS/rWbH2mGShwMhf6MK5x8dOuXoy1W40nttA2N+Yad8wJa0OHEnw5OUAhQF15YXlCWs7VtMwiDoUsxpUAwWeAO2LGTM9rOrEH4BHAerPyf2lcnitiRk25AcAGm273ZuU/bNyefYHSjefNpk2cfV1cKFA9F8DExpdDzBnb1JDm+HJyjDvmeKCssPgSb8kBnok87rtEXxmieiJZ+yjhNWpFRAaGs/vhiffBiZ3gSYB3/QJmvTPaUYkMVS7OX5NWeonWqxUAtvwm/HzVB6Ibh8ggZIyZ02pzpjFmXKttN3AjcLhvoxIRERER6Z5A6cY9hCfpAZHywL7cTCctO+NcJ7cH43K/7t9dUN/j5y/ZUJ44+7qDofrabBsK5mFM+bmELdRZl7vBSc+dGCgrPGyttY4vt61hGhLS82KMy7218XjFPmPM8xjX22wwWFhXvulH7qSRRcnz354TDFT93b+7YK0xZqsx5i/W2nudjAWpdWWb9rriEr8VavBfDfbNGHPCFZ/4nWCg+oHIZ/J1Y8zHgNnW2tORttWEE7TXWWtD1tofRtqv6enPSAaPy07WWmtv74E4RET6xsFi+N0t4D8JiWPhlidg4rxoRyXSbxljPgg8GZnh2ro9FniftfbRLox1H/BPwhU5kgiXgLkGuKHHApZ2bT8UTtbOmZgS3UAkeo6+Bse2gTsWZt8c7WhEBqOthG9ktsCLbeyvAz7ZlwGJiIiIiPSUSMnh3a3bvFn5LhsKznLSst+4Ed/lDgVrTm1rOFbefLnnjJQiLnAyFmZaG5pJU8NoExMXTtg2NznWMiwlf/mRyoLHz5tQaEPBNzZCocn+0o0bjDFXgFmCcZ01LtezgPVvf2GjcXuWVW36w7mlPU1swsSE6fNycMccBwg1+D/mik/8bqi+5r+w9pOhupr/M8Y8Za09A3yH8E2ZDwLvMsbcBSwErrTWhi73/cvQ0RMzazHGpAJvJlzq6WVr7Y6eGFdEpEe9/kf488ch2ABjZ8Py38GwSdGOSqS/e5hwmZYTF7QnRfZ1OlkLjIn0Hw9UAduAG6y1/+qBOKUDlYFGDpwJADB7ombWDllbfht+zloGzojoxiIyOE0DDOHZBznAyVb7GoET1tpgWweKiIiIiAxE/t0FIWB767b4KbPd7qRRVzhp2TEAGJfF5W4yLtfrkf5dFijdUJI469qDwcbAQpob/w2Xu8A2N12JMe7Go2V5wEaMafZm5cfY5qY4ExM3iWBzJXBLXUXR9cYYDxAHNkSo+WM2FJwDxhVqqN1XtekPp5z03ImEQpMwLgfAuGMMxhW5gGJ/Gayr/j8AY8yXgXsI/7z/jLU2aIy5FdhqjFkZ2fcRa+2B7rxPGbouO1lrjLkWeBpIiDQ1G2M+ZK197HLHFhHpEdbCuv+DF/87vJ3x73DTgxCXGN24RAYGQ9vLHUwinHDtNGvth3skIumy7ZESyFNHOgxzYqIcjURFUz1sezL8+qpboxuLyCBlrW1Z48sV1UBERERERKKo/sD2IPBa67b4KbPjXHHeuU5a9hs/K7s9gbqyTbsiM3YvqXbHSwHgeSdjwUwbbH6LcbmLLXa6NSbRSc+dYtwx220oeKXxxBJqqj8GgMv9CqHg+z2jpr49WHnszdYGaxKmzj1i3J5CsM0mzjspYXp2nnG7R+BynQSsbaw7EijbVAhgjIHwZAMArLV+Y0w14QkJLW17jDGfAx4gXJ3u8e59cjKU9cTM2m8D/wI+DtQD/w38L6BkrYhEX3MD/P1T8NoT4e28u+Dfvg0ud3TjEunnjDFbeKOU4wvGmNala9yEZw89E43YpOu2RUoga1btEFbyD6ivhOSJMP3aaEcjMiQYY2YCUwhXoDrHWvu36EQkIiIiIhId9Qe2NwCvtm5zfLmJCdPnz3PSst9odLmr6soLyztK4AZKN+70zlx6MNRYdyNQjbWx1pJv3PyOUMhlQ40YlytkPDEjbHPj0fjp8xfjcm/wJI0qqt/76q/qD2zPjZt8RQjjSgA8xuNpNi73v/y7C5oi178uPHfTBduWi2/QXAIEgVRjjMdae9kloGVo6Ylk7RXAQmvtUQBjzOeB/zTGjGxZUFlEJCoCZ+DJW2H/ejBueMv/QvZHoh2VyEDxl8jzXOBZoLbVvkZgH/DHPo1IOnTwTIDyk7Vt7ltbGq7EOWeSkrWD2tFtUHu87X1Fvww/z12uG5ZEepkxZjrwZ2A24Qs5JrKr5aKP/hGKiIiIyJAXKC+sBTa3bvNmLhp+XgLXuCwu1+lA6cZ9rfv5d66pAZ5yMhbmhYKNU7DGhhrr32pc7sNAAsYVb0OhehMT78PlOkywKWBcbtxJI58L1p7+ug02vh0b2m8b/I2B0o2bAYwxyYQnJ3SJMea9wLuBa4DfA18DvtHVcWRo64lkbTJwbvFla23AGFMHDAOUrBWR6DhdAb+9Gc5UQGwSvGcV+K6PdlQiA4a19psAxph9hEu41Ec3IulIVV0Tb/nxOmrqO75xc/bElL4JSPrewSL49Zsv3W/u8t6PRUR+BOwFros85wAjge8Bn4tiXCIiIiIi/Zq/ZP1ZLkjgOhkLxzpp2fMxLosNhW+EdMccCZRuOBIo3bDJO2PJ7lBD3futMW/ChgqMy5Nog82HTUzcdBtsqjPWjrTGNQHjKvekjLs3WHPqpoYD2/OAF4DbjTF/ByqBbxGeHdtpxphJwP3AF621BcaYO4CnjTH/tNZuivQZB4wDfJHDZhtjaoAD1toz3fyoZJDpiWQtwA3GmNbr1rmA64wxV7Q09FSpJ2OMG7gXuJXwX/AjwCrgvztb21xEBrn9G+F3y6HuDAybDMufhLGzoh2VyIBkrX3EGJNijLkVSAO+a609Y4y5GjhurT0c5RAFeH7ncWrqm0mK85A6yttmH9+YRLJTh/dxZNJnXvtd+DlpPCSObbtPxo0wYnrfxSQydC0A3mStPWWMCQGhyIWbLwE/Bq6KbngiIiIiIgNHoHTDceC8MlJOxoLJTlr2/JZtYyi0mGrXsNE/cCWNmGT9tXs5vOu0wbhMTNxzgV3rTr/R1/wU+AKQTngm7dNAFeEZsZ2eWWvCC9quAoqAnwJYa581xtwPPGaMmWutrQU+xvkzbddGnu+IHC/SY8naR9poe6DVa0vPlXr6IuH1cW8DdgDzgYcJ/2P6cQ+dQ0QGqm1PwV8/AcFGmHA13PI7SGrnorWIXJIxZg7wPOHv2VTgV8AZwuVdpgAfjFpwcs4/th8F4MOLp/Hp6zOiHI30uVAQdv09/PodP1UlCZHocwM1kdengAlACbAfyIxWUCIiIiIig0WovvaE20lJsU31cRYbwlpjXDF1JmHUpMC+HcSNT58WO+mKn7gIHbPNzdck+HL2G+MKAST4cp8DniM8KeF/gf/FGOuKTahtPnv0rw1Hdrf8LI+11lx4bmttSqvNi34Bt9beA9zTavtewhMQRdp12claa+2FCyn3toXAX621qyPb+4wxtxAuLSUiQ5W1sOZ/4eX/F97Oeiu8+1cQ60Q3LpGB7wfAKmvtFyIlWlr8A3g8SjFJK9X1TawrC69IsWz2+ChHI1GxfwP4T0DCcJi2NNrRiAi8DlxJuARyIfAFY0wjcCewJ5qBiYiIiIj0V8YYEz993gjj8kzFWtNS8thiQ8a4z9t2xSc2NVceLW84Vl7nZCycQLBpMthATHz8Znfa3PnB6pP7QjZkrCERG8Ra62BtuTE0Whu0xrhsqxPvrysrrIybNCvJM3zCSFfi8FQnPa9n817WGhObUG+b6k43nth7Nlh9slGVYqW1nppZ25c2AHcaYzKstaXGmCuBfOAzUY5LRKKluRH+fg+89v/Zu+/wqM4z/ePfZ2bUzlBt01wwRhVj3EAFIVzj2BvSnGRTnGxiJ3F6dlM2CellNxvvpji7qbtppDlOr/ziFBeMAEmAsY0NamCaMcY2fUZ15v39cUZYCAECjebMSPfnuubSnDLn3BoLH2me8z7vz/zl2vfCCz4HoUzfSyIyKi3A/3B5oCfxpyOQgP1909N0J5KUTh1H6bTxQceRIGz6nf+1YgmE8wKNIiIA/DvQ15P+U/ht1VYCzwGvCSqUiIiIiEgQzCxUVLbwQtfbfQ7OJc1CR0erOpd0/j4hK5q9AMsr2J+IH3ykc8fGU84d65VUXeIVV0YJR54mFN4HNr5z+4ZPY6FKoBHcC8F+b9CJf/NkOUYBhJqx0JGO9sZNAF75otlFxVUXASQ7Dyf90CG/wBqOxDFriTXXJ4f5Hlj+tOLCyORzzyo8/+LSZHdnvldac4oXhY4b1QtA6j07bvdIftLCkQM9z+16rntPW8w5N6zMklnDLtaa2XzgS8DLnHOHBmybCPwOeJ9z7pHhnivlDmAC0GxmCfwWUx93zv30BPkKgIJ+q/QppshoEt8Hv3gjbFsJFoYXfREq3xJ0KpHRpAv/ujtQGfBMhrPIIPpaIL9Io2rHpmQCNv3Bf37xzcFmERHAn6eq3/N2oMLMzgL26+55ERERERktCs6tKAyPP6uERKLg6KhXl3QWitB/uaik2pFM7Ozcun7bcH8fLpo9P98sdAWAFXibYptWHPbKFs4DYi7Z22kW2u9I9iYP7/u7Rc+eCG6xs9Bd5pJTwT1m4fxNJHovxyXPLiqu+ieXTDS7nq4k4bzd8dbVuweezyupHgdcWlS8IGQWcljI6Nca2cKRrbGWVftPlTv1fXfgD354cjjvwYmYWaRw9vyJeWeff254/NmeV1ozeLH3zE/gQvlF8UTswHPJjkMHuva096b1+Kdp1tLledvuWNITZIZ0SsfI2g8C9w0s1AI45w6a2d+ADwFvSMO5AF4NvB64BX/O2suBr5rZbufcYHPnfpRjJ28WkdFi31b46avhuTbIHw//uAxKNU+fSJr9AfiUmb06tezMbCbwn8Cvg4sl4LdAfrDVb4GsYu0YtWON3wK5cCJcdFXQaUTkBJxz+4LOICIiIiJyKkdbEYfzZ5Ho8Yt9FnKDFWMj48/u6j38XFvX7ubOkc7V1+rYIvndHW0NTX1FX6+05kpwT+O4EOe6LC9vg+vpnhueMGWeWeivyURPwiUTSyycd6+Z7Ur2dFZZOG9PvK3poWhFXT3J0JW4ZBjclUWzF5xvZklCIbNw3sZYc31nvL3xCPDwid4rr3zRbK+katbAIi6ARfJiQOtwR+UOlXOuF7+bz3MjcXwzC+WfW+FFJkw5J1QYPdcrrTm2vjhwfl+z0yvMO2eWX9hNond/z/4n9/U8s73jRMX9WUuX/wC4ddbS5cu23bHkttP7TrJTOoq11fijXU/kj8Bb03CePl8E7nDO3Z1a3mhmF+IXZQcr1n4B+Eq/5fHArjTmEZEg7GyCn70W4s/BhPPh9b+AaXODTiUyGn0Q+BWwFygCVuC3P14DfDzAXALcu9lvgVw8JUrZtHFBx5EgPP47/2vFiyGSH2gUkbHMzH4z1H2dc68YySwiIiIiIgNFK+pCWGiW6+44i775WlMF2P7LRbMXYPlF+xKx/Q8PpRXxSPNKa+aSTIwjHHk6vmVt47HbqqsJhbfgyCfEXpK9Z8Wa65NeaU0SCMdaVu33SqqazeyIS/bOx0LjO7asa/BKq88rmj2/xvIK2uOta9aB//5YXmi+SybCmCWSvd3lXnGlP89QOBLvaGvYPLBwmFrecsLsJdXjCIUu80prjivkgktaOG/bUEblZotUW+UjqceIyJt8bn7elFmT86dcNCsycXrhca2iLWSh/MLwxEW33Jo3eQb4Bdu3jYYRtuko1p4HHD7J9iNAOod6eMDAOxESwKCTUzrnuvBbOAJglt6R3yISgMd+A799ByS6YMZl8LqfwwSNKBMZCc65g8ANZlYHXAqMAx5yzv092GQCsPzRPQAsmTdDv+OMRckkbO5rgfzyQKOICAeDDiAylrx41atPvZPw10Xrg46QE/4YemXQEXJCcXHQCURGtzNt6VpwbkVhZMKUUtfbnX9c8bW/UCiJJXd2bF33RLZPzXFMq+NI/uZYW8NxXV29kqrFFs57JNay6pBXUrXI8gofJ9TrF1dd0mFhAOLtTY8VFVdOMqM7mUxMLSpdeG1HW+P9wJNecWV50ez5JeHopA2x5vouYC1A4cx54VDhuCtIJkJYqNclEzuLZi+Y7xVX+icP5+2Mt65++lTfR2pU7obBtj0/Krd69rFbXBIL+XPlZnBUbrbo2b+7G3g69TihWUuXLwNuBZaNhkItpKdY+wxQDjxxgu0VwLNpOE+fPwIfN7Md+G2QrwA+AHw/jecQkWzkHNTfCfd+1l8ufxG84jtQoNFkIiPNOVcP1AedQ553uLOHB9v8aYNfdKluWBmTdjbAkaf9Fsizrwk6jciY5pwbFa23RERERCTzBrZ0NTMrKqk6BwtdSKL3mDbEwDHLkQlTunoPPZORVsQjLVpRN8P1dM20ULi7Y8vapsGKypPqbrHuPW3XWqRgday5vtMrrbkyVBDdkOw8Mm/gyNs+4cJxDyU6Ds8NmSXBhYuKq5Z0bGlaHt+ytsXMrKi48nKvuDLcsXXdeuecS40o7htxm0ei53LAsFBv/rTZG3qe2T7TK632K7cWMtfb/XDH1vXdp/O9DnlUbkn184MU+7UVtlD4iVwalZtu2+5YcttoGVHbJx3F2r/jt0G8Z+AG84d4fDy1T7q8F/g34JvAVGA38L/A59J4DhHJNokeWP4BeOhH/nL1O+HGz0MoHGwukVHOzP75BJsc0Am0Aw865wJvjTPW3Lt5L929SWZPiVI+bXzQceR0PNsOj/0ahvvPZvtq/2v5ErVAFhERERERyUGzli7PS8T235rsigPcGp2z+NtFxVW9Fs57Dngo1rpm1I+s9EqqLnEuOc5C4eNaHfcXrajLS/Z0XRXKK7g/1lyfjM65ahK43iOP3x8vKl4w4H0yF62oC8Wa65NHHr8/7pXX7se5g7jkZIx4UUnlq0KRgt+kWvtuiFbU5RVedGW1V1J1MN7etLnvKLHm+h5SI26jFXV53Xva/SJtKNyTP/Wih3ue3WGE8+YdbZkcCh+JtzU0D/c9OdWo3KKyhcVHR+X2FXFTrZYzPVduUEZToRbSU6z9d2C9mTUCXwZaUusr8Oe5K8MfjpwWzrnDwPtSDxEZCzoPwi/eCFsfAAvBTXdA9duDTiUyVrwfmII/DUHfHXuTgTj+VAdTga1mdq1zbmcwEcemvzyuFsg56w/vgR1r0ne8uTen71giMmxm9gT+TU2Dcs7NPtE2ERERERlbtt2xpGfW0uXLwtHJtwLLYptXnrBYOZoMbHUcb1l1XKvj/qLli6Iu0bOgc+u6+/pG3Ca7Oy7p2LK23itbeIGFwrv772+h0OM4NxfYCBBvWd3ulVQtJpzXbi55jkv0tiZ7u/8xWlH3+1hzfWeqKNvglVaf5RVXLiQc2RVvXXPM51ypfZr68h8t3Fqoq29UbrR80QSvpHr+0RyR/J2x5pV7h/t+9Zf6/ttPtL1o9pXjLVLgz5X7/Iv856GQmYW2jOVRudlq2MVa59wWM3sBsAy4m+f/KDVgE3CDc+6EPzgiIid1YAf89NXwzGbIi8Krvg/lNwWdSmQs+RjwNuCtzrktAGZWgt/V4v+AVfjX/zuBVwUVcqzpSSSpb/Nnmbh+zrSA08hpie+Dnam/veffBqFh/jo++UIovWH4uUQknb46YDkPf/qem4AvZjyNiIiIiGS10djS9US80urpJJMXWjivp6O9cdBWxwNFyxdNdoneOfH2phXPH6dmrkXymgFcomd6x5Z1a/u/xm+RXF3Yf51F8le53u6FhMJPWTgCzj2W7O15SbR80d/7ipfxtsZ9wBqvtHqmV1JVY+G8lsEKm6m2x00A0Yq6wqLiykqvpMov3G5peqjv+/JKqy/yiisXABCOYKHwI6mi74jp2PrQYU46V25diVdacxHwfBEX/EJuKHyEIY7KPdN5lmVw6RhZi3NuHXCJmV0BlOAXaludcw+n4/giMkY9+RD87LX+fHzjZ8AtP4cZlwWdSmSs+XfglX2FWgDnXLuZ/Svwa+fcbDP7MPDrwBKOQQ/vPMDhrl4me3nMO29i0HHkdGx9AFwSpl4ML/lq0GlEZAQ45/57sPVm9m5gQYbjiIiIiEgOGO1FL6+0Zi7JxDhCob0na3V83OvKaqfhEjPj7Y2r+9aNm3d9AcnEuHhbw+MnfXEyeUwhONZcn/TKFu7Av5kyioX2mvFoMpG42iurbYq3rj46Ojfe1rgD2OGVLbzYK6kqt0j++hMVWWPN9Z0MWri1zo72po3OuSdS20Kut+dSr7jSr8uFwgfjbQ1tQ30v0iFVRD7hOaPliyY4lzx2rtxQKFXQNYdLtMfbGg8OnGd5hGOPCWkp1vZxzm3gBBV7EZHTsvmP8OvbobcDps3zC7UTzws6lchYNIPBf1+IANNTz3cDmjQ1gx5sfQaAutIphENqgZxT2u/1vxZfF2wOEQnCn4EvAPowQ0RERERGvWhFXZ5L9FxJMolF8jfH2hpO2ur4uNfPWTwTl5wcb2s8ZtRssuPwgo6t61aDXwQORQpajnmhhZIF00si4ejk444Zb12z0yupqrJI/qMu0XOZ5RVuoruz17nElV7ZwsJ465qtA/bfZGZWVFK1wCutSXS0N2442Wjg/oXbcXOv9Y4WbkPhjo62hsf6D3CMzrlqkldcueBoMTQUfiLesvrZ03mP0i3mt6M+0ajckFdeVxydc1X5pKv+6VYL5xOZcM6tY2VU+EhLW7HWzL5ygk0O6MTvof1759y+dJ1TREYh52DN1+GvnwQclNwA//gDKFAdSCQg9wP/a2ZvTd2URaqTxreA+1L7zAOeCCjfmLQiVay9umxKwEnktDgH7X/3n5e8INgsIhKEVwH6e1hERERERrW+VseEwj0d7U1DanV83DHKa0tIJiPxtoZHBhz7IkLhHUePmUyMG1gEdnAgPP7sc7AQZmYDzx9vb2oqKq6s69iytt4rqbrWIvkrSCYSOFfuldYUDRyxm3r92tSo2WqvrHZ/vHX1sQXiQRx5/P44fSNuyxdF/YJvNWDxeFvD47HNDx4A1h393soWzvZKq/32xBZyyc4jGzp3bEz0bQ+67bBzLklqVO6spcuXAbcCy1SoTY90jqy9ArgSCAN9P6hlQAJoBt4FfNnM6pxzm9J4XhEZLRK98OcPwbrv+8uVb4Wb/hPCaW0CICKn5y3Aj4H1Ztb3y1cEuDe1DeAI8MEAso1Jzx7p4tFdBwG4qvScgNPIaXn6cTiyB/I8mLkw6DQiMkLMbAP+TctHV+F3o5iC/3exiIiIiMio4xVXXgyMJ5K3N97WOORWx8cdp2zhxUBnvK2huf/6SXW3GM6dG29bswr8tsKEQsfPrdrbvc8Ko1NJJjcXldbMAY6rR1kkr9krrrw4f0bpA9172q+JtzfdHy1f1JtM9JR7JdWF8fbG9QNfkxo12+CV157jlVTVEIrs6N86+WRiLatiwFqA6MVXj/dKqyv9IKFYvHXNJoDUqN6tAIUz54VDheMu9UqrIwCTr3nz0ryzz39FtrQdHkvzLGdKOisgv8e/S/g259whADObCHwXqAe+A9wF3AncmMbzisho0HkIfnkrbLkXMLjxP6DmnWBq7ykSFDMzIB94KTATKE9tanHOHb2D0Dl3fwDxxqz6Nr8jzsUzJjB1QmHAaeS09I2qnbUY8vTfTmQU+92A5STwDPCAc675+N1FRERERHLTca2O/Ta6Z8wrrbkMs33x1jU7B27r3tNeHR531tEisOvputLyCo4rqlo4r9f1dOfF2xtjXtlCb7DzxFtWP+uVVp/b89yuiaGCaKNXUl0bb29c7ZVW9wLlfSNvT/Ra4FmvtPoir7hyoUXyHz+d7zu2acVh+gq35YsmHC3cYof7CtSpEbUbwB9Rm+zufEXq5VnTdjgbMowm6SzWfgi4oa9QC+CcO2hmnwH+6pz7bzP7HPDXNJ5TREaDAzvgrtfA3k3+aKNXfhcqlgSdSkT8kUDtwNxUcfaULV5k5B1tgVyuFsg5Z0tqvlq1QBYZ1Zxznw06g4iIiIjISOrf6jh/WnHTgfq7TrvV8SDHrMTCO+Ktq58+bltZ7TRCkf2HH76nt2+dw1m8uf74kbVDFG9rfLRo9vzFHVvXr4xWLG73Smsui7c1PuKV1/aacxVeafX1Fs67P3aCc8TbGp8AnvBKqy/1iiujyZ7Opv5ti4ciVeRdCzDukusmHi3cOg7G2xtbwS+Kqu3w6JfOYu1EYCrHDymfAkxIPT+AP0JHRMT35Hq467UQ2wvjpsMtd8O5VwSdSkTw56IwszbgbFJzUkiwkknHg6li7VWlKtbmlK4jsH2N/7zk+mCziMiIMrMJJ9jkgC7nXHcm84iIiIiIpItXtvBiEr3DbnV83HFLqmstHNkca1m1f9Adkr0l8famVX2L0fJFE8xCR4Z7XgtFNngl1fPj7Y3roxV1nldeWxJvWd0eLV/0uOvtvtglu64aN/faptT8s4OKtzU+OqnuFut+emuVV1rTOXCe3aE68th9B3l+xO3kouIFlamMB+JtDWo7PMqF0nis3wPfN7Obzez81ONm4Hs83waqCmhN4zlFJJdt+j38YIlfqJ12Cdx+rwq1ItlnKfBFM7sk6CACm546xHOxbqL5YeZfODnoOHI6tq2EZA9MngVnzQ46jYiMrAPA/kEeB4AOM9tuZp81s3T+PS4iIiIiMiKiFXV5XnFltVdcWY1LPhnfsrYx3rL6iXQc28zMK668JlTgPXyiQq1XUnWF5Rc92n9dsrerPN7etPmUJ3BJS03zNah4e+MR4HC0om5GrLl+G8lkgVdWe26sZdV+K/AedS6ZSHQcvjw6Z/HZJzvNgfq7XLytodFC4XavuHKhV15bcspsJxFrWbW/Y8u6tR1b1q21SN4+r6Sqau+vPnvFcI8r2SudI2vfjj8f7d39jtsL/BB4f2q5GXhrGs8pIrnIOVj1Vfj7Z/zl0hfCq74PBeODTCUig/sR4AGPmFk30NF/o3PurEBSjVF9LZBrS84hP6LP+HNK33y1JS/QfOwio9+twOeBZUBTal0V8Cbg3/G7T/0r0AX8R+bjiYiIiIicmldWO41Ezyws1Js/ozQtrY77K5w5L1xUXHlteNxZD/Rvb9xftHzRBDCXmuf1KGNoNz5aKNJeVLawGH+ar0HF2xtbveLKqybV3bIn3tbwuFdaUx0tX3Qk1rLqkFdSuQ5HdbK7c5ZXWl0Yb2t88mTni7WsigFrvLLaaV5x5UJCoSfibY17hpL1hMfcvPI54DkAr7z2HK+kqgqAcGRfvGX1Cb8vyS1pK9Y6544At5vZ+4G+4QJbU+v79nk4XecTkRzV2w3L3w8bfuIvV78DXvh5CKfz3hERSaP3BR1AnreiJTVfbZlaIOcU56Dtb/5zzVcrMha8Cfigc+4X/db90cw2Am93zl1vZjuAj6NirYiIiIhkGa+kag7OTSAceSa+ZW3aWh33F62oKwzlFdbkTy+592RFYJfomde//TFAtHzR+YQju4dynljLqv1ecWXxqfYLjz97dfdTbTXAmnhbQ6NXWn3N+Mtvqo+3r+2IVtStItFzlcM6vfLaoqEUSFPz7j7tlVQXe8WVCy2S/2iqkDss8ZbVzwLPAkQvvmZKv8LtM+ka7SzBSHt1xDl3xMz29T1P9/FFJIfF98Ev3ui3grQQ3PSfUP22oFOJyEk4534YdIah+MMju/lJw/ZAM4QMXlc1k5ddft5J99vxXJwv/rWF972glOIp40667/JHn+LHDdtIpv5sWb/D7wikYm1AdjTCA1+AxGlON5lMwIHtEMqDWYtHJpuIZJNa4B2DrN8ALEw9rwdmZiyRiIiIiMhJRCvq8lyi50qSSQiFmofUYvhMz1W+aIJL9FwW37L2gZPt55XVllukoG3gepfoOTfe3tQ02GvO1OGH7+n1yhY+5ZVWXxRva3wif1rxiu49bdcC98Wa63uiFXX309t9nUu4bV7Zwnnx1jUbh3LceHvjFmCLV1J9uVdSVWiR/KZYc30yHZljmx54BngGIFqxeKoKt7ktbcXa1Hw7nwA+CIxLrTsMfBn4vHMuLT+AIpKjnm2Hu14N+7ZA/nj4xx9A6Q1BpxKR02BmhUB+/3XOuUMBxTnGnoMdND2xL+gYbHkmxksvO5eTTIfCNx9o54+P7CYvZHzlNZef9Hj/8f828+SBYzpPM2fGBC44y0tHXDldD34Rtt5/5q8vvhYKTl6gF5FRYSfwFvx53/t7S2obwNn489iKiIiIiATmmFbH04vT3up4oGjF4qku0TMr3t608uT71eWR7J0ca13dMnCbc8nTyxgKDWkuonjrmm1eceXCaEXdrlSBdrVXUl0bb29cHWuuT06qu+Xe7j1t12KRrV5pdWW8rXHtUCPE2xsfjlbUhVxvd5VXWh2PtzU+eupXDV2seeVeYC/4/02PFm5D4b3x1jXb0nkuGRnpHFn7eZ7/g7RvWHod8BmgEL/Fk4iMRU88CD//J+g8ABNnwi13w7S5QacSkSEwsyjwn8Cr8T9YHiic2USDu37ONM6fHFwBM+kc7//5wzxzuIud+zqYefaJs6zbvv+Yryey52AnTx7oIGTw1ddeQSRkGDB/1uR0RpehSiZhV+rG3Rv/AyacfAT1cUJhmFWX/lwiko3+Ffilmf0D0PcBzgKgAnhVarkS+HkA2UREREREMtLqeKBo+aLzXaL3nKGMinW93VUdW9auHrjeK62psFC49dRnc88XaC005OJux9Z1DYUXXVkHrIw113dG5yxu8coWXhFvXbPhQP1dzszuLyquvMryClu9kqrFpyo695caUdsQLV80wSupriUcfmokRsD2tWAGiFbUzfBKqysBLBTZG2tZFWxbOjmhdBZr3wS81Tn3h37rHjWzJ4FvomKtyNi0/oew/AOQ7IXzK+G1d8G4qUGnEpGh+y/gWuCdwI+BdwPnAW/n+BFDgSmeMu6ULYVH2ndXPsHDOw+wfse+ExZrD8S7ad/rzxKxY1+cvYc7mTq+cNB916eKuRXTJ/DSy84dmdAydM+2QOdByPOg6m0Qzgs6kYhkKefcH8ysAv9aWZZa/Wfg5c65bal9vhVQPBEREREZo6IVdXmup+tKAMsvaoltfnDEWh0P5JXXXkQy6cXbGx8+1b7R8kUXEgo/6Zw7rsjqkr3jO7asaz7Z6y2S15vs7ix8fjl/V7R80YVDKVQ655xXVtvqlVTNibc3bY5tXvlctHzROK+stjzeurollWlFUXFlnUXyN3sllddZpOCB02ltHGtZdQhY7ZXVnusVVy4kFGqPtzU+M9TXn45Yc/1TwFMAXlntuX2FWyy0J966ZufJXiuZFUrjsc4CBvtH0pzaJiJjSTIBf/k4/PGf/ULtJa+EN/1RhVqR3PMS4F3OuV8DvcBK59y/Ax8DXh9osiyz4EJ/xOu6bSceMfvQjmO3PXSS0bXrtvttnRdoJG122NHgfz1vvgq1InJKzrknnHNLnXOvSD0+2leoFRERERHJJK+sdlrR7PlVrrfn0vwZpU3xLWsbY5sfPJDB85eTTEbibQ2Pn2pfM7Nkb/cFg7XujVbUhSx06gZvFsnvBFfQtxzbtGKPc8kpQ82bGplaGC1fNAEg1rJqOy5p0fJF5/ft07FlbT2JnhKLFD7serquipYvig71+P3Oszu+Ze0asEmp9suD382fJvHW1bvjbY1r422Na81CziupqvJKqqr6f18SnHQWax8B3jPI+vcAae2/LSJZrusw3H0LrPm6v3zNR+GV34O8omBziciZOAvYmnp+iOdvwKoHrgokUZaanyrWrj9ZAXZAIfdkhd2+4/QdVwK2M9UV6oLqYHOISE4ws8Vm9hMzW21m56XW/ZOZqR+6iIiIiGSEV1pT4RVXVoOLdmxd3xRvb1w/0nPSHp+h+lJwXfG2hrah7F9UUlUVyisYtC2zS/TMT3bFHzrVMZLdnYctnDesubLi7U0bkj2dlx1dbmtodoneadHyRZP77dPgEt3Flle4ySV6L/PKa885o3O1NbTFt6xd4xK9F3ulNdVmNqQ5docj1rJqV7y9qSne3tTkXNL6CrdeWa1auwUkncXaDwNvNrNNZva91GMTcCv+nD0iMhbs3w7fuxFa74FIIbzq+3DNUhj5a4yIjIytwEWp5834c9eCP+L2QBCBslXfXLItTx/mUGfPoPv0FWCrL/Jr3ut3DF6sjXf38vjuQ/5xVazNDn3F2pk1weYQkaxnZq8E/gJ0AFcCfXf1T8TvTCEiIiIiMiLGX35TxCuurC6aPb/KIvl74lvWNsZb12w99SvTzyupno+Fnx1slOxgohWLp+LcoVhz/eAfqiSToc4dGxOnOk7i8LOHCYWHVawFCOUVPuKVVF/etxxvb1zvervnRSvqjrbbirc1rnW9XecRCm8jmZw5nFGq8baGhywceaiouLLaK62ZO8z4Qz9v65qdfYVbM8s7OuK2om5GpjJIGou1zrkV+PPx/BaYlHr8BpgL/FO6ziMiWWxHA3znOtj7OESnwq3L/fbHIpLLfgD03Ul4B/BuM+sE7gS+GFiqLDR1fCEzz/JwDjbsOHDc9p5Ekkd2+evfcXUxAI89eZDOnuP/znhk50ESScf0CYWcN0ldCQJ3ZC/sS/1te35lsFlEJBd8AniHc+52oP8HTavwi7ciIiIiImnlldVO84orqxNH9l+WP6O0qWPr+qZMtjo+Lk9JVY3l5W+Lt67ePdTXuN6u0nh706Dz6Hol1eMIhY8M5Tjde9o6SCbyj1mZ6B1qjKP8uWVdV7Ri8dF5/Tq2rlvpersW9x/9Gm9v2oBLnIPZMy7RM94rry057ZP1nbO5vife3tRgofBur6SqJlq+6MIzPdYZnb9l1fajI26TiQKvpKrKK62ujl589fRM5hiLIuk8mHNuN/Dx/uvM7DLgLcDb0nkuEckyD98Ff/wXSHTD9Hnwurthotrdi+Q659yd/Z7/3cwqgPlAu3NO0xwMsODCyezYF2f9tn1cXXbsdCiP7z5EZ0+SSV4eV5dNYcr4Ap453MWjuw5SddFZx+y7PjVf7fwLJ5OB7jdyKn2jaqfMgaJJgUYRkZxQDjw4yPqD+Dc1i4iIiIikhVdSNQfnJhCOPBPfsnbQ9sGZZGZWOHv+VaGC6EOxTSsOD/V1XknVpZZXeMI5bZ1LzOloX7d2iIdL4JLHTm4bjjztldWeezrFY4B4e9PmotnzF5vZMxd+5E8R51zPhPkvqS8qrqwDVh7dr63xUa9s4cWEwnHAvLKF8+Ktazaezrn6i7Ws2g80eGULL/BKqmstL78ltnnlc2d6vDORGhG9DcArr73IK63254Wy8LbUvL6SRulsgywiY1EyAX/9JPzunX6hds5L4M1/UaFWZJQwszeaWV/7Rpxz251zvwGazeyNAUbLSlf2zVs7SHvjo3PQzpxMKGQsOMkct5qvNsscbYGs+WpFZEj2AIPdTV/H8/PAi4iIiIickcKZ88JeaXW1V1xZbXmFTwXZ6ri/aEVdqKh4wbVhb2LD6RRqo+WLokA4XSOBnXPHzcsbb12zE5c47wwP2Tj9DV/6I9A9a+nyHxxa/8duC+c95pUtXDDgHJtIJvOBMC75jFdaPezWXH6L4sbVLtE7zSuuXNi/BXMmxVtWPxFva2yMtzU2got6xZXVXklV1ZnO0yvHU7FWRM5c5yH42etg9f/4y4v/Ff7xR5AfDTaXiKTTD/Dn2BtofGqb9LMgNW/thh0H6E0kj9nWN1q2r6A7/2ixdt8x+yWT7mixtu94ErAdqWLtBZqvVkSG5DvAf5tZNeCAc83s9cCXgW8FmkxEREREcpZXWj3FK66sDuUVXpE/rbgpvmVtY5CtjvsbN+/6gmRP17UWKXjgyMZ7u07ntS7Rc3m8vWnDibZ7ZbXnWih8eiM5w3k9kYlTC06946lNe/XnXGTS9CWpxVtnLV2eF2tZtd8stCdavmhO/33j7Y2tJBIObJJFCtq8kqpjWiafqXjrmk0dW9c1uGTiMq+ken46jjmMLFvjW9Y2xtubmoBJR+e4nbP47KAyjQZpbYMsImPIvif8Qu0zmyFSCC/7Bsx7VdCpRCT9DP+D5oHOx2/nKP2UTR3P+MIIhzt7ad5zmEvO8+vczjnWbUsVYI8r1u7HOXe03XH7M0c41NlLUV6YOTMmBPBdyDF6OuGph/3nF1QFGkVEcsYd+DdG3wt4+C2Ru/Dnev9ugLlEREREJAd5pTUVJBMTsdCz2dDqeKDoxVePd13xKzq3rrtvsFGtJ+OV1pRaJP/ko4KTveenCoND5hI9zxWcWzGDVBtfACx0Wtn6bLtjSc+spcuXAbcCy7bdsaQHINayapdXVlseLV90Yaxl1fa+/ePtjVu8soWzXG/3eaGi8U1FJVXXFc6c90Dnjo2JMzl/n9R7u27cvOsLioorq73SmgPxtobm4RxzuOItq9v7nnvltSVeafVsgFC+t+XI4/fvO/ErZaBhF2vN7Den2GXScM8hIlnmiQfhF2+Ejv0wbjq87i44b37QqUQkjcxsA36R1gH3mllvv81h4CLgntM85keBVwAVQAewGviIc64lLaGzQChkXDlzMitan2H99v1Hi7W79new93AXkZBx2QWTAJh77kQKIiH2x3vY+myM4injgOdbIF92wUTywmqCErinHvbb/EenwFmzg04jIjkg9SHK583si/jtkMcBm4C3A08A0wOMJyIiIiI5oHDmvHAor3ABgEXyW2MBF+VOJDpn8dmup6s0vmXtg6f72sKZ88KhfG9KrLm+Le3BkolDLtk7KV2H23bHkttmLV3+tr5CbZ946+oWr7TmyuicxUf6zykbb12zLVq+6Pxkx+Hy/Okl93Xvab8qWr5oXaxlVWy4WVIjlxuicxaf7RVXLiQc2RVvXbNzuMcdrmMKt6U1pV5pdTGARQrasmUEeDZLxyeAB0/x2A78KA3nEZGgOQdN34Efvdwv1J57BbztfhVqRUan3wG/xx9Z+5fU877H3fgfOL/hNI95NfANoAa4AcgD/mpmo6p3et+I2XX95qLtK8DOPW8ihXlhAPIjIS47f5K/fdvz+z4/AvesTMSVU9nR4H+9oBqC6zIkIjnAzArM7Atmts7MVgEvcs5tAuYCLcC/AHcGGlJEREREslq0YvHUvlbHllewNr5lbWOsZdX+U78y86IVdTNcT/eseHtTw5m8PpRfVJM/vXjNyfbxymrLLZx32sVcC0WSJBLhY9eF9w9njtWBhdo+8baGh1xPV0W0oq6w//pYy6pdmO3t3tN+eby9aYVL9MxL5xyvsc0rn4tvWbsGCHvFlQujc66alK5jD1e8raEt3ta4Nt7WuNYleqd5pdWVXml1ZTZlzDbDHlnrnLstHUFEJMv1dsOfPwzrU1NUzns1vPR/IK8o2FwiMiKcc58FMLNtwM+dc51pOOZN/ZfN7FZgLzAfv0XkqNDX5nj9tue7vaxLzUvbt63P/FmTadq2j3Xb9/Hqygv816X2nX+h5qvNCjv75qutDjaHiOSCz+HfzPR3oBb4pZn9AP8mpQ8Cv3TODav1mYiIiIiMTtne6nggr2zhLFxyQry9cf0Zvv4CQuGnDtTfdfLWxMneibH2prR0ZIs112/xyhYuAJ5Nx/H6i7c3rfJKKq8zs/v7t4KOtzXu8cpre72Sqqp4e1ODV1J9uVe2sCidI2HjrWu2Adu88tpLvNKaOcmuWNNwWy6nU7x19dH/fl5pTYVXUlUGYOG85ljLqkPBJcsumrNWRE7tyF6/7fGONYDBCz4Di/5FI4xExgDn3A9H8PATU19H1RwWl10wiXDI2H2wk3Xb9nFWNJ+mJwYv1vYtr922n63PHOFwZy/bnosDcOVMFWsD4RzsfwKSSX+5r1g7sya4TCKSK/4ReKNz7g9mdgnwKP7f3Jed7txdIiIiIjK6zVq6PG/PXUuTofyi+QAWzmvL1lbHA3mlNaVYKBRva3z0TF5vZlY4e8GFHVvW1p9sv2hF3fA6w5pl9Hfw8LizHywqrroKWNF/fbxl9bPR8kUJr7hyYXzL2jVeaU1FtHxRWaxlVWs6zx9vWf3YpLpbrHtPe6VXUt0db298OJ3HT4f+c+x6ZQsv9kqqxuGcWYG3KbZpxeEgswVNxVoRObndD8Pdr4dDu6BgArzyu1B2Y9CpRCTHmVkI+Cqwyjn32En2KwAK+q0aP8LRhi1aEGHOjPE89uQhXvXtY7v5DBwt21eQfeLZGNd9+fnf5cumjWOilzfyYeV4f3gvbPjxsevCBTDjsmDyiEguOR9YD+Cce8zMuoA7VagVERERkf5mfuBXP04cevYNk6998x8PNf765bHm+mTQmYbKK62ZSygUj7esfuJMj1FUXFlpkfx1p9rP9XZfZpH8DWd6Hpw7fqRRoveMD3cqhx++p3fcJdc9nBpF29R/W6xl1f7oxVc/VjR7/uKOretXemULZ3ml1ZeeacH7RFIjlZvGzb3W84orFxIKPxtva0j/nMBpEG9ds6nvuVdaM9crqYrinFkk/7FYy6rYrKXL807Ueno0SsectSIyWm38FXz/Rr9Qe3YJ3H6fCrUiki7fAC4BXnuK/T4KHOz32DXCudLiTQtncXY0nwmFkaOPV1x5HlMnHDN9CZOj+byu6oJj9jsrms+bamcFE1xgZ+rvqfxxUDgRCidB9dshUnDSl4mIAGGgu99yL3AkoCwiIiIikoVmLV2eF8ovekPeORdQMKPsJVNe/tHwqV+VHbySqissFD44nEJtdM7is4F4rLl+KNNt5cWa68+sWJfhUbV9jjx230FCkV1eac3cgdtim1YctlBoXdHs+dd0tDVsx8JPe6U1IzLn0pHH74/Ht6xdY+G8g15x5cJoRd2MkThPusTbGh6Ptzc1dWxd1+RccvaMN37lj4mOw92zli7/QdDZMsXG2k2+ZjYBOHjw4EEmTJgQdByR7JTohb9/GtZ83V8ufSG84jtQNCnQWCIp6r+d48zs68DLgKuccyf9Bf8EI2t36TouI+aLJRB7Bt5RD9PnBZ1GZLQZ1ddwM0sCfwa6UqteAtwHxPrv55x7RYajZYW+v8U3bHiY8eOzvlGGZLk3vi9t07yNan/98N6gI+SEP4ZeGXSEnPDaRaFRfR0XyaRUAepWYNm2O5bcFnCcIfFKqqoweyLe1vjMMI+zKN7etOpU+0XLF0WdS86Ot67ZeEbnKaudj0u6eFvDQ8esL62psFB490jPleqV1pRaONITa67fNnBbtKIuz/V2Lc6fXnp/zzPbx7ve7ss7tq5bOZIdebyyhbNJJqdbfuHGbG83PGvp8jyOvQk2fyyMsNXIWhE5Vnwf/PSVzxdq694Pr7tbhVoRGTbzfR24GbjuVIVaAOdcl3PuUN8DyOpfKCXHOQcd+/3nRWcFm0VEctEPgb083w3iJ8Buju0QcTCwdCIiIiKSFVIF2vwcKtQutrzC1mEXassWzrNw3qZT7wku0TO3o63hhNNmDeEINlgb5I72xhaX7C0/8+MOTbytoc0leidGKxZPHbgt1lzfY5GCB7qearsu0XEoZnkFa4qKF1w7/vKbRmza0njrmq3x9sbVrqdrtldas3DY8wGPoFRhdllqcdlYKNSC5qwVkf72PAZ33wIHtkOeBy//Jsy9OehUIhIgM/vKCTY5oBNoB37vnNs3hMN9A7gFf1TtYTObnlp/0DnXMeywIsPVdQiSqflrPBVrReT0OOdy4sM2EREREQleLhSgJtXdYt172q61SMHq2OYHh9K2+ITGzb3WI5nMj7Wv2T/U1wxrpGkyOehrnXPOK1uYkXaz8baGR7zSmoXj5l575Mjj98f7b4s11ycn1d1yX9fu1mssr2B1vGXV/UXFVVdFL776oZEc+Rpva3gkWlEXcr3dVV5JVTze3pTWOXPTZdsdS26btXT523Lh30m6ZG31XEQybOOv4Hs3+IXaSRfCW/6mQq2IAFwBvAV4G3B16nF7at31wFeAdjO7eAjHeicwEXgAeKrf4zVpTy1yJvpG1UYKIa8o2CwiIiIiIiIiARl/+U2Rrqfarkt2d64Y4vyyJ5XoPHJlvL1x/VD29UqrpxMKD7+Xfyj46YDjbQ1rkt3xhZPqbjlulO+B+rtc5xPrH0jED9ZEL76mKN7euML1dF082GjcdIo11yfj7U0Nll/0hFdSXRutqCseyfOdqbFUqAUVa0Uk0Qt/+Tj8+i3QE4fi6+BtD8D0S4JOJiLZ4ffA34FznXPznXPzgfOBvwE/A84DHgTuPNWBnHN2gseyEcwvMnTx1ABxtUAWERERERGRMWrc3Gu9xJF9izq3rruvc8fGxHCP55VUF4ci+duGur9LJi6It64Z8v4nOVJWzLVt4bz7u5/ecvVg25xzrmPLuhXJrtgV0fJFE+JtDY0umZgeLV904Ujnim1acTje3rjaJZMdXkl1rVdWO22kzyknpmKtyFgWexZ+cvOx89O+/ldq/Sgi/X0I+GRqvlgAnHMHgc8AH3bOxYHPAfODiSeSRh2pYq2ugyIiIiIiIjIGRcsXTU52xa6ItzetGFYb4r7j+XOjTou1rNqVhnin5wStkC0U6fJKKjPWTivWXJ+0vML1Xml19Yn2ibc3rXLJ3jnR8kWT462rH3XOFXplteWzli7PG+l88dbVu+PtjatxyQleSXXtuLnXeiN9TjmeirUiY9WT6+F/r4YnHoS8KPzjD+EFn8mK9hAiklUmAoO1X5kCTEg9PwDkZyqQyIiJp9ogF00ONoeIiIiIiIhIhnlltdNcoqc03t60Kl3HdL3dCy2S1zDkDKU1pRYKb0nLyc1OUGx2jxPKG8p0XmkT27TisOUVbvfKFs470T7xtsZGl+wtiVYsnhpvXd1y1vVv/bdE/GD3rKXLf5CJjPG2hrZ4e+PqZE9nuVdeWz1Y62YZOSrWioxF638I378JDu2Cs4rh9nth7suDTiUi2en3wPfN7GYzOz/1uBn4HvC71D5VQGtQAUXSRiNrRUREREREZAzyyhZegEtOj7c3NaXvmLXnYqG9seb65JBf5JKT422N+9KVYTCx5vokiZ6MFyJjm1bsAWJe2cLZJ9on3ta41vV2nTf+8n+YGZk47R/D3kSAWzMxwvZohtY1G8JFE9Z3P721xiuv1VyJGaJirchY0tMJf3gv/PGfIdEN5UvgbffD1DlBJxOR7PV24F7gbmB76nF3at07Uvs0A28NJJ1IOmnOWhERERERERljvLKFs0kmxsXbGh5J53Fdomd2vK2hbaj7T6q7xUZ718d465qtQPRk88PG25s2JGL7JvUe3PuL1Kpl2+5Y0pOZhL7DD9/TG29rWGOhyC6vpLo2WlE3K5PnH4siQQcQkQzZvx1+8UZ46mHA4LpPQN0HIKR7NkTkxJxzR4Dbzez9QN+df1tT6/v2eTiIbCJpp5G1IiIiIiIiMoZ4pTUVhEK98famzWk9bnHlgnDhuIdO5zXdT2+9wsKRDenMcUKhUGAtfuOtazZ6pdXV0Yuvjsc2rTg86D5tjY96pTWfCxWO+/iRjX9vz3TGPrHNDx4AVntlCy/wSqprCYdb4y2rnw0qz2imKo3IWND+d/i/q/1CbdFZ8E+/gav+VYVaERmyVHF2H7Cvf6FWZFTRyFoREREREREZI7zSmsswi8VbVqe1GBgtXzSZUKj7yOP3x0/rhS4ZiTXXp2cEaWq+WjMbvCgbCvcUzZ6fn5ZznYF4W2Oj6+5YEK2oO+EH9PG2hseTHYdDXnltSSazDZqldc3OeHvjasOmeCXVtePmXV8QdKbRRpUakdEsmYD7vwA/eRV07Idzr4S3PwjF1wWdTERyhJmFzOxTZnaQVBtkMztgZp80M/0eIaOLRtaKiIiIiIjIGOCVVM/HQnvirWt2pvvYyd7uufG2xkdPL09lEWad6cwRKvBi+dNLxw22LdlxeGMorzDQ+Vjzp5c84Hq7rz7ZPvH2xlYSCeeV1lRkKtfJxFpWbe7Y0rQm2XHkEq+0uvKExXA5bTn3IauZbTMzN8jjG0FnE8kqsefgp6+CFXcADha8Gd58D0y6IOhkIpJbPg+8B1gKXJF6fAx4L/BvAeYSST+NrBUREREREZFRziuprrVwZGu8dfXTaT92ac1ci+Q1n+7rnHOXdLQ3bUxnlp7ndu3OO2fmjMG2de7YmHAuGWh97ED9XS5UEG30SmsWnmy/eHvjFsw6vdKauZnKdjLOORdvb1xv4byHi0qqa6Lli+YEnWk0yLliLVAJzOj3uCG1/peBJRLJNrvWwf9eBVvug0gR3Py/8OI7IaLuBCJy2t4EvNU59y3n3KOpxzeB24Fbg40mkmYd+/2vRZODzSEiIiIiIiKSZmZmXknV1RaOPBJrWbU/3ccfN+/6AlwyeiZzmpqFzDnn0pmne0/bEZfonZjOY6bbkcfvj1s4b4tXWnPZyfaLt67ZZqHwQa+k6tJMZTuVWHN9T7ytYQ2h0F6vtGahV7ZQo8SGIeeKtc65Z5xze/oewIuBLcCKgKOJBM85aPg2fP8mOLQLziqG2++Fy14bdDIRyV1nAYPdEdmc2iYyevQVa9UGWUREREREREaRaEVdqKik6jqL5K+OtayKjcQ5EvFDlR1b1q493dd5ZbXTCEeeSWsY5+zCj/zp+5MWv75p1tLlP0jrsdMs1rxyr4UjB081N22sZdUuzPZ6JVVXZCrbUMQ2r3wu3tawxiwU8kqqaqLli3QH/BnIuWJtf2aWD7wB+H6677oQyTmdB+GXb4J7PgLJHpjzUnjbAzAtK7ojiEjuegS/DfJA70ltExkdEj3Qdch/rjbIIiIiIiIiMkpEK+oKXW/31fnTiu+LNdf3jMQ5vPLaiywc2X5GdZpk74XxltVPpDOP5RWEgVstHAG4ddbS5XmD7OWiFXVZUSOLNddvAwq9stpzT7ZfvK1xD+HITq+kqiozyYYu1rJqe7y9qcG55AyvpLo2WlE3yHsuJ5IVP4jD8HJgErDsRDuYWYGZTeh7AOMzE00kg556BP7vGtj0ewjlwU3/Ca/+ERROCDqZiOS+DwNvNrNNZva91GMTfgvkDwUbTSSN+kbVYlA0KcgkIiIiIiIiImkRLV80wfV2V8bbm+4/UH/XiAx4m1R3i5HoPT/eumbnSBz/TLiergSwLDJhKsCybXcsOa5I7Xq7Nrpk8pKMhzuBeMvqx8CdP+6S607aujnesvpZC+e1ecWVJ53rNijx1jWbOrY0rXGJ3iu9kur5QefJFblerH0L8Gfn3O6T7PNR4GC/x65MBBPJCOdg7ffguzfAvq0w8QJ48z1Q8w4wCzqdiIwCzrkVQBnwW/wbpCYBvwHKnXMrg0smkmbxff7XwokQCgebRURERERERGSYvPLac1xv95x4e9OIfn7T/VRbTbK7Y/WZvNYrqS4mHNma7kwA2+5YchuQn/p6nI6t67txiYKROPeZireuaUp2xa4Yf/lNkZPtF2tZtd8KvMeKZs9fnKlsp8M55+JtDY0WyXvcn8+2tjzoTNkuZ4u1ZnYh8ALgu6fY9QvAxH6P80c4mkhmdB6CX70Zln8AEl1QeiO8/UE4f0HQyURklHHO7XbOfdw598rU4xNAyMz+L+hsImnTkSrWar5aERERERERyXFeWe25JJMz41vWNo7oeUqrp2Oh5zp3bEycyeudS5wVb1n9bLpz9RlsRG22y59WvCJx5LmrTrVfbNOKwxYKrSuaPf8as+wcuRVrru+MtzWssVB4v1dSXRstX6T63AnkbLEWuA3YCyw/2U7OuS7n3KG+B3A4I+lERlJf2+PHfwMWhhv+DV53tz5gFpFMOhu/w4XI6NA3slbz1YqIiIiIiEgO88prLwJ3dryt4aGRPpdLJkvi7Y2tZ/JaMzOzUFYWGYN0oP4uZ5GC1V5pdd2p9o23r+0I5RetKipecO2kuluy9r2MNa/cG29vXO1weamireZvHCAni7VmFsIv1v7QOdcbdB6RjHEOmr4D330B7NsCE86D2/4Mi/4ZQjn5z1lERCQ7aGStiIiIiIiI5DivrLacZDISb12zccTPVVJ1RajAe+RMX19UXHVZqGj8Gb8+LSzksnFUaqy5vtMi+Zu9kurLh7Bvj0UKHuh6qu26wpnzsnpep3jL6ifi7Y2rnXOzvLLa2mhFnYoaKbn6RrwAmAl8P+ggIhnTcQB++Sb4f/8KiW4o+wd4Rz3MrA46mYiISO7TyFoRERERERHJYV7ZwnnguuJtDW0jfa5o+aIJWCgZ27RiGJ1MXf6Rjfd2pS/V6TMLbfbKF10cZIYTiW1e+ZyFI/uHMt9rrLk+WTCj9D6LFFw1bt71WTUP72DirasftVCowSUTVUMpSI8FOVmsdc791TlnzrkzGl4vknN2roX/XQybfg+hPLjxC/C6n2n0j4iISLpoZK2IiIiIiIjkKK+kej7Yc/HWNdsycb5kb/el8baGMx4VG62oK8RCI1aotUher5lFTrVfrGVVzCUTRSOVY7hiLau2gwt7ZQsvONW+B+rvcp1PrH8gGT9UPW7utV4m8g1HrLk+GW9d02DhSJtXUl3rldeWAMxaujwv6GxByMlirciYkUxC/Z3wg5vgwA6YdCG8+S+w8F2Qfd0ZRGQUMbPfnOwB3Bl0RpG06tjvfy2aHGwOERERERERkdPglVTVEA5vj7eu3p2R85XWVITyCoc1kM71dl/a0d74aLoyDWSR/M786aVZW4Q9HfHWNZuAadHyRaf8wMI55+Jb1j6Y6Dh8ea7MCxtrWRWLtzeuNgvFpr/hS39KdBzunrV0+Q+CzpVpKtaKZKvDT8NPXgF//wwke2HuK+AdK+H8+UEnE5Gx4eApHtuBHwWWTiTdjrZBVrFWREREREREst+5b/1mnldceZXlFz0eb1n9bCbOGa2oy8MlJ8WaV+4d7rGccy4dmQaT7O48HJk0fWjFykTvSMVIm3jrmnUu0XtptKJuSKNOO7auW+0SPRf3FXhzYbTqlJd/9NnC8+cssVAI4NZcyJxOKtaKZKO2v8G3amHr/RApgpd+DV71fSicGHQyERkjnHO3DeURdE6RtOkbWas2yCIiIiIiIpLlLvzIH39AMtE95eaPvWV488aenmRPV3XHlrWNwzmGV1o9hVB4RIvLicPPHrZwZNxQ9zfL/jaWHVuaHnS9XYuHmjXe3tTgEj2lF7z3Jz8Dsn606rY7lvQAy0IFUYBlqeUxQ8VakWzS2wX3fAx++iqIPwtT58LbHoAr36i2xyIiIiPp6MhaFWtFREREREQke81aujyv98DTt+ZNmYVF8t+YqRGIXtnCWRaO7Br2iFjnLoq3rtmapliD6t7T1uF6uwuHsq/lF7UXlS4sG8k86eCccxYpWFlUUr14qK+Z+qpPb3CJ3temFrN+tOq2O5bcBuSnvo4pKtaKZItnWuG7L4CGb/jLVW+H2++DqRXB5hIRERkLOlLFWo2sFRERERERkSy27Y4lPXmTZywzC0GGRiCamblE7/nx1jXbRvpcaZJwvT2RoewY2/zgAVwiN+Z3ba7vsVB4o1e2cMFQ9t92x5KeyIQpy1KLOTFaNRcyjgQVa0WC5hys+z7871Ww51F/RM/r7oYX/RfkDenmHxERERkO5zSyVkRERERERHJGpkcgFpVUVYXyCobV/hjAK6+9yCIF29IQ6aRGcj7coMVaVu03C+2Jli+aM5T9x/Jo1VyiYq1IkOL74OdvgD+9H3o7YPa18K41UP4PQScTEREZO7qPQDJ146ZG1orIKGJms8zMmdnlQWcRERERkfTK1AjEaMXiqWAHY831wz9fondKrHnl3jTESi8L5VRxN9ayapdzLhktX3ThUPYfq6NVc4mKtSJB2XIffHMhNP8JQnnwws/DG34D46cHnUxERGRs6RtVGy6APC/YLCIiKWa2LFVodWbWbWbtZvYpMxtSO7c0Z7nKzP5oZrtTeV6e6QwiIiIiEoxkT2dZvK2hebjHMTNLRx7xxVtXt7hk4uzonMVnB51Fhk/FWpFM6+mEez4KP74ZjuyBc8rg9nuh9j0Q0j9JERGRjOs/X63+dhSR7HIPMAMoBb4MfAb40MCdzCx/hHNEgUeAd4/weUREREQki3ilNZeFIvmPpuNYRcWV8yxNx0o3C0V2RCvqZgWd43TF2xoecj1dFdGKOs2nmONUGRLJpD2PwXeug4Zv+suVt8PbVsCMy4LNJSIiMpZpvloRyV5dzrk9zrntzrlvAX8HXpoadfs7M/u4me0GWgDMrMrMNphZp5mtA67ofzAzuyY1MvbG1H4dZnafmU01s38ws81mdsjM7jKzo60GnHN/ds59wjn320x+8yIiIiISnOjFV48HwrGWVYfSdMjCWHN9Z5qOlVax5pV7XTJxTtA5zkS8vWmVS/QsmlR3i+4+z2EZb58kMiYlk7Dm63Dfv0GiG6JT4GXfgLIbg04mIiIiHfv9r0WTg80hInJqHUBfm7PrgUPADQBmNg74E/A34A3ARcB/n+A4nwHeA8SBX6QeXcAtwDjgt8B7gf8806BmVgAU9Fs1/kyPJSIiIiKZ57o7Lo+3N61Mx7GKZs/Pt0h+ZudNNTu9eWgTvSMUZOSFo5NXdO9pvwpYEXQWOTMaWSsy0g7sgB+9FP72Sb9QW/4ieOcaFWpFRESyRV+x1lOxVkSyk/leANwI3JdaHQPe6px73Dn3OH6hNQS8JbXuT8AXT3DITzjnVjnnNgDfA64G3umc2+CcWwn8Crh2mLE/Chzs99g1zOOJiIiISIZ4JdVlFslvT9fxLBy5rKOt4eF0HU+Odfjhe3pDheMe9kqqqoLOImdGxVqRkeIcPHwXfGsRbFsJeVF4yf/Aa++CcVOCTiciIiJ91AZZRLLXi83sCNAJ/Bn4Of6oWICNzrnufvvOAR51zvVvLbfmBMftP1fY00DcObd1wLqpwwkOfAGY2O9x/jCPJyIiIiIZUDhzXhg4J9Zc/1Q6j+ucO72RrpkWCuV0G+Ejj913kFBkl1dee0nQWeT0qVgrMhJiz8LP3wC/eyd0HYLzq+AdK2H+m8By+v/5IiIio09HqljrqVgrIlnnfuByoBQocs69yTkXS22LnfBVp9a/BZ0bsNy3blifFzjnupxzh/oewOHhHE9EREREMsPyChfmTy8+0U1/p80rrz2HUHh/uo43UiwUeSpaviinbzCMt67eDXRGK+pmBZ1FTo+KtSLp1rwcvlkDzX+CUB5c/yl48z1wdnHQyURERGQwGlkrItkr5pxrd87tcM6dahKtzcClZlbYb13NCGYTERERkVHGK1t4gUXynjxQf1f6RsEmemfHW1anraXySIm1rNrlkr0zgs4xXPGW1e0u0TsxWrF4uJ1yJINUrBVJl44D8Nt3wN23QOwZmDIHbr8PFn8QQuGg04mIiMiJaGStiIwOd+GPiP2OmV1sZi8C/jUdBzazcWZ2uZldnlp1UWp5ZjqOLyIiIiLBMzNzid4L4y2rnwg6S2AslN2tmoco3tbwiEv0FI+be60XdBYZGhVrRdKh/V74Vi088jOwECx6H7x9Bcy4NOhkIiIicioaWSsio4Bz7gjwEmAesAH4PPCRNB1+QeqYG1LLX0k9/1yaji8iIiIiASsqrqwMexPWpvOYXnntRYQi29N5zCFzbkzPRxhva1iT7O6omVR3y5h+H3JFJOgAIjmt6zD89ROwfpm/fNZsePm3YWZ1oLFERETkNGhkrYhkIefcrae7zTnXgD/HbX/Wb/sD/ZdT65YBywas+wzwmZO9TkRERERGD6+89hwsFDuy8d6utB440Tsl3t7UlNZjypBZOPJA9572a4D7g84iJ6eRtSJnausK+Gbt84XaqrfBO+pVqBUREck18f3+V42sFRERERERkbEo0Tsn3tbweDoPaWa5d7Of2XNeee05QcdIl1hzfdLyi9Z5pdUqWmQ5FWtFTlfXYfjTB+BHL4WDO2DSTHjTn+BFX4T8aNDpRERE5HQkeqHroP9cI2tFRERERERkjPFKqy+1cN5j6T5uUUnVvFBBNO3HHUnxltVP0NszK+gc6RTbtOKw5RVu98pqNWdjFlOxVuR0bLnfH0277nv+8oI3wzvXwEWLg80lIpIDzOwqM/ujme02M2dmLw86kwidB55/XjgpqBQiIiIiIiIiGTdu7rUejrxYy6r9aT+4c4VHHr8/nvbjjrRQKPdGBJ9CbNOKPRYKxaIVdcVBZ5HBqVgrMhSdB+EP/ww/fvnzo2nf+Ad48Z1QMC7odCIiuSIKPAK8O+ggIkfFU/PVFkyEcCTYLCIiIiIiIiIZlOg4ND/e3rg+3ceNVtTlYaHedB9XzlysuX6LS/REoxV1M4LOIsdTsVbkVFrugW/UwEM/9Jcrb/dH086+OthcIiI5xjn3Z+fcJ5xzvw06i8hRHalirTc52BwiIiIiIiIiGeSV15ZYpGDLSBzb9XZfkT9t9oaROPaQmblAz5+F4m2Nj7pk4sLoxVePDzqLHEvDB0ROJPYc3PMR2PhLf/ms2fDSr8GsumBziYiMIWZWABT0W6VfJsWXTMDfPw37tw/vOEf2+l+LNF+tiIiIiIiIjA3RiroQid5p8famVSN1jgP1d+VmsdTCh6JzrpoU2/zggaCjjIR465oGr6Tq2mhF3YpYc30y6DziU7FWZCDn/ALtPUsh/hxYCBa+G675GOR7QacTERlrPgp8OugQkoV2rIHVX0vf8c66KH3HEhEREREREclirrd7oUXy14zEscfNvfYsLJT+OXAzpKNtTWtRac18YF3QWUZK/vSSB7r3tF8D3B90FvGpWCvS34Gd8Kf3Q/vf/OWpc/3RtOfPDzaXiMjY9QXgK/2WxwO7Asoi2eTI0/7Xs0ug+h3DO1Y4D8qXDD+TiIiIiIiISJaLli86n3Dk6ZEaVZnsjhfH2xrXjsSxM8E557yyhUHHGFEH6u9y4+Ze2+iV1iyMtzWMSNFeTo+KtSLgt1Js+j+499+gJwbhfLj6w1D7LxDJDzqdiMiY5ZzrArr6ls0swDSSVeKpuWanVEDV7cFmEREREREREckRLtFzUby9aeXIncBlx4c3FsqOHFnqyOP3x6MVi7d4pTWXxdsaHgk6z1gXCjqASOCeehS+e73f9rgnBhfUwDtWwVUfUqFWREQkW3Uc8L8WTQ40hoiIiIiIiEiu8IorF4QKoutH7PhlC2dZJH/nSB1f0ivWvHKvhSMHvfLakqCzjHUq1srY1R2Dv34C/u8a2L0BCibCi78Kt/0ZppQFnU5EZNQxs3FmdrmZXZ5adVFqeWaQuSRHdaRG1npnBZtDREREREREJAdEyxdNJhzpOvL4/fERO0kyMTXWXP/UiB3/dCQTZ/5as86iWZd76QuTvWLN9dtIJgu8stpzg84ylqlYK2NTy5/hG9Ww+mvgEjDnpfCeJlhwG4T0z0JEZIQsADakHuDPRbsB+FxgiSR39bVBLlKxVkRERERERORUkj2dl8Rb12wcqeObmWVNC+Rhyp8y63HLL7o46ByZEm9reJxk7/njLrluYtBZxipVpWRsObgL7n49/Oy1cHAnTJwJt/wCXvNjGD896HQiIqOac+4B55wN8rg16GySgzSyVkRERERERGRIvJKqS0IF3qaRPEdRac0lFsl/bCTPkSkH6u9yuOSoKDwPVby9qSnZFbti/OU3RYLOMhbpTZexIdEDDd+CB+7w56W1MNS+B67+CORHg04nIiIip0sja0VEREREREROKVpRVwh4sc0rnxvRE7lkYax1TWxEzyEjKn9a8Yrup7dcB9wbdJaxRiNrZfTbtgq+vRj+9km/UHtBDbxjJdzwORVqRUREcpVG1oqIiIiIiIicUrKna0HHlrVrR/Ic4y+/KYKFkyN5joyzkAs6QqYdqL/LWThvlVdaXRd0lrFGxVoZvQ7vgV/fDsteBM9sBu9seNk34bY/w7S5QacTERGR4dDIWhEREREREZGT8soWzrZwZLtzbkQLj4nY/issFNowkuc4bWbD+p4tFO4pmj0/P11xckWsub7TIvmbvdKaK4POMpaoWCujT6IHVn8dvrYANv4CMJh/G7xnHVzxegjpx15ERCSnJRPQedB/rpG1IiIiIiIiIseZVHeLuUTvufHWNTtH/GTOWay5flSNrE3EDz4Wyiu8JOgcQYhtXvmchcLPeWW15UFnGStUtZLRpf1e+NYi+OvHofswnLcAbr8PXvJVfZgrIiIyWnQcAFI3yBZNDjKJiIiIiIiISFbq3rNloevpXDPS54mWL5pMKHJwpM+TaZ07NiacS47ZGlqsZdV2MwtFyxedH3SWsWDM/qDJKLPvCfjZLfCTV8CzLeCdAy/9Grzlb3CeRuuLiIiMKh37/a8FEyCcF2wWERERERERkSwTraibgdkznTs2Jkb6XC7RUxpvXd0y0uc5XRaOJMwsMqyDJJNjbt7a/mItqzY7l5weLV+kO+VHmIq1kts6D8HfPg3fqIKW5WBhqHkXvHc9XPlGtTwWEREZjTr65qudFGgMERERERERkWyU7Okqjrc1tAWdI0gWye/Mn15aNKyDuKSlKU7OireuWeeSiXnRijrdLT+CVMmS3JRMwEM/gq/Nh1VfhUQ3zL4G3rkabvqCPrwVEREZzeJ9xVpNcSAiIiIiIiLSn1dac2Uokr8hE+eKzlk8k1BkVybOdbqS3Z2HI5OmTxjWQcJ5rnDmvHCaIuWsjvbGla63a7GZjfni9UhRsVZyz5b74X+vhj+8F2J74axieN3d8E+/g6kVQacTERGRkdY3slbz0YuIiIiIiIgcFZ1z1SQgEWtZFcvE+Vxv97R46+rdmTjX6UocfvawhSPjhnMM19u1MVQ0fl66MuUq55yzSMHKopLqxUFnGa1UrJXcsXcz/ORV8OOXw9MboXAivPDz8K4GKP8H0E0dIiIiY4NG1oqIiIiIiIgcJ9ndcUm8reGRjJ3Quaz9UL57T1uH6+0uHM4xOrau7yaZUPtfINZc32Oh8EavbOGCoLOMRjlZrDWz88zsJ2b2nJl1mNlGM9MPyGh1aDf8/j3wrVpo/xuEIlD9Tvjnh6H2PRDJDzqhiIiIZJJG1oqIiIiIiIgcwyupmhPKK2zN2PlKa+ZaftHmTJ3vdDnnel1vT2TYB7KQS0OcUSHWsmq/WWiPV7bw4qCzjDbD/0HNMDObDKwC7gf+AXgGKAX2B5lLRkDHAVj139DwLejt8NfNeQm84LNwdnGg0URERCRAGlkrIiIiIiIiclS0oi4PmBhrXpnJ4um42KYVhzN4PskCsZZVu7yy2vJo+aILYy2rtgedZ7TIuWIt8BFgp3Putn7rnggqjIyA7jg0/R/U3wmdB/x1MxfCDZ+DC6oCjSYiIiJZQCNrRURERERERI5yvd01HVvW1mfqfIUz54VDheMSmTpfkMxCFq2oC8Wa65NBZ8kW8dbVLV7ZwiuicxYfiW1e+VzQeUaDXGyD/FJgnZn90sz2mtkGM7v9RDubWYGZTeh7AOMzF1VOS283rP0u/M8V8PdP+4XaKRXw2rvgtj+rUCsiIiK+jlRDFY2sFRERERERkTHOK1s4y8J5O5xzGWvXGyocd4WFwg9l6nxBskj+484l1fZ3gHjrmg2ut3tOtKJuWPMCiy8Xi7WzgXcCbcCNwLeA/zGzN51g/48CB/s9dmUipJyGRA889CP42nxY/kE4sgcmzYSXfxveuRoqloBl7TzlIiIikmnxVLHWmxxsDhEREREREZEAmZm5RO/5GW9H65KWEyNNzYZdwD7y+P1xEomCdMQZbeJtjfUu0bNoUt0tKuAMUy62QQ4B65xzH0stbzCzS4B3AD8cZP8vAF/ptzweFWyzQ6IHHv0FPPhfsH+bv27cdFj8QZj/Jojo/38iIiIyiA7NWSsiIiIiIiJSVFxZbZH8xkyec9wl100EG1tz1VpODnzMiHB08oruvU9cBawIOksuy8Vi7VPApgHrNgOvHGxn51wX0NW3bBqhGbzebnj0blj55eeLtNEpUPd+WPBmyCsKNJ6IiIhkuXhfsVYja0VERERERGRs8spqp1k472Csub4nk+dNdh4pj7c3NWXynJK9Dj98T2+0fNEGr7SmOt7WkNEbB0aTXCzWrgLKB6wrAzI7zF9OX08HbPgJrPofOLjDX+edA7XvharbIT8abD4RERHJfj0d0NvhP/c0slZERERERETGJpfoKY1vWVsfdI6xwswsk/MC55JYy6pD0TlX7fLKay+ZevPHW7bdsSSjNxCMBrk4dPtOoMbMPmZmJWZ2C/A24BsB55IT6TwE9XfCV+fB//tXv1A7bhrc+B/wvo1Q9z4VakVERGRo+kbVhiJQMCHYLCIiIiIiIiIB8EqqLw9F8h/N9HmjcxbPtHDe7kyfN2ihgmhrUenCsqBzZLPY5gefnHz1rZ9JxA92z1q6/AdB58k1OVesdc6tBW4GXgc8BnwSeJ9z7qeBBpPjHdoNf/0k3DkX/v4ZiD0DE2fCi74E//IILHw35HtBpxQREZFc0tGvBbKmtxAREREREZExJnrx1ePBWaxl1aFMn9v19syItazalenzBu3IY/cdBDc+6BzZbNbS5Xl5Z533yrA3EeDWWUuX5wWdKZfkYhtknHN/Av4UdA45gT0bYc03YeMvIZka7X5OuT8n7bxXQVj/RkVEROQMHZ2vVi2QRUREREREZOxx3R2Xx9ubVgZ09mQw55Vst+2OJT2zli5fBtwKLFMr5NOTk8VayULJBLT+BRq+Cdv6XScuXAS1/wylL4RQzg3kFhERkWzTsd//qvlqRUREREREZIzxymrLLb+oLYhzR8sXzbFQpCWIc0tu2HbHkttmLV3+NhVqT5+KtTI88X2w4Sew7nuwf5u/zsJw8ctg4Xvg/PmBxhMREZFRpkMja0VERERERGTsGX/5TRGSibNjm1YHUzANRybGNq3YHMi5JWeoUHtmVKyV0+cc7H4I1n4fHvsV9Hb66wsnwfxboep2mHh+kAlFRERktOprg+xNDjaHiIiIiIiISAYljuxbmD+9pD6Ic1/4od8tm7T4DW+atXT5sm13LLktiAxnxDlL27HMtnnltRfFW1Y/kbZjiqSoWCtD13nQn4d2/TJ/Xto+0+dB5e0w7x8h3wssnoiIiIwBfW2QNbJWRERERERExoho+aILCUd2Hai/y2X63LOWLs8jFHlTZNIMgFvHapvbeMvqZ73S6osAFWsl7VSslZNLJmF7PWz4KWz6PfR2+OvDBX6r48q3wgVVYOm7QUVERETkhI6OrFWxVkREREREREY/M7PCi66c2bF1/cogzr/tjiU9s5YuXwbcCizLqUKtWcaL2yJnQsVaGdxzW+DRX8AjP4MD259ff045LLgNLn2NPiQVERGRzDs6Z63aIIuIiIiIiMjoVzh7fnXYm9gUZIZtdyy5bayOqBXJBBVr5XmHn4ZNv/OLtE+ue359wQS45BVw+Rvg/AUaRSsiIiLB6RtZqzbIIiIiIiIiMsp55bXnWChy+MjGe7uCzqJCrcjIUbF2rIs9C5v/AI/9BravApf011sYiq/zR9BWLNFctCIiIpIdOtQGWURERERERMaIRO+ceHtTIO2P5XgWKdgdLV90fqxl1a6gs8joomLtWHTwSWhe7hdp+xdoAc5bAPNeBZe8EsZNDS6jiIiIyGA0slZERERERETGAK+s9lLLK9wYdA55Xmzzg096ZQsXACrWSlqpWDsWOAdPPQKt90DLn+Gph4/dPuMymPsKmHszTL4wkIgiIiIip5RMQucB/7lG1oqIiIiIiMgoFS1fFCWZyIu1rj4QdBYRGXkq1o5WHQdg6wPQ/jdovxcOP9Vvo8EF1XDxS6HixSrQioiISG7oOvh8RxCNrBUREREREZHRykKV8fbGB4KOISKZoWLtaNHbDU+u8wu0Wx+AXevAJZ7fnuf5c9CW/wOUvlAtjkVERCT39LVAzh8Hkfxgs4iIiIiIiIiMgPNu/9/fnHXju2+etXT5sm13LLkt6Dw5zUKW9mMmky7tx5QxT8XaXNXbBbs3wLZ6/7GzEXrix+5zThmU3AAl18OFiyCvMJisIiIiIunQsd//qlG1IiIiIiIiMgrNWro8L3LWeTebGcCts5Yuf9u2O5b0BJ1L+gmFnotWLJ4aa165N+goMnqoWJsrYs/5I2d3NPiP3Q9Bb+ex+3jnwOyrYfY1/mPSzCCSioiIiIyMvpG13uRgc4iIiIiIiIiMgG13LOmZtXT5MuBWYJkKtcPk0j8KNt66ZptXWl0JqFgraaNibTbqjsOejf7I2SfX+0XafVuP38872x8xO2sxzKqDKRUQCmU+r4iIiEgmdKSKtRpZKyIiIiIiIqPUtjuW3KYRtWninM1aujxP76VkOxVrgxbfB08/5hdn92yEpx6FZzaDSx6/79mlcEE1zKyGmQvh7BKw9LdcFxEREclKfSNrizSyVkREREREREYvFRfTY+Ki130KeKnm/5Vsp2JtpnQehGda4Zlm/7F3Ezy9CY7sGXz/cdPg3Cvh3Cvg/Plw3nx9MCkiIqOCmb0b+BAwHXgEeK9zrinYVJIT+kbWehpZKyIiIiIiIiInNmvp8ry8s857aWoxvfP/Wijt7ZVlbFOxNp16u+DADr9l8XPt8Gzb819PVJQFmHQhzLgUpl8K0+fBjMthwoyMxRYREckUM3sN8BXgHUAj8D7gL2ZW7pzTXB9ych37/a9qgywiIiIiIiIiJzGS8/+ahQ5F51w1Kbb5wQPpOqaMbSrWno5kAg7v8QuyB3fCge2wfzvs3+Z/PbRr8PbFfcbPgCnlcE45TLsYpl4MU+dAwfiMfQsiIiIB+wDwHefcDwDM7B3AEuDNwB1BBpMcENfIWhEREREREREZmpGa/zfeurqtqLRmPrAunceVsUvF2v4O74EDO+Hwbjj0lP/14JNw6En/6+HdkOw9+THyx8FZF8FZs/05Zs8p9eeWPacUCidm5vsQERHJQmaWD8wHvtC3zjmXNLO/AwuHdfAje+HQ7uEFlOx3cKf/VSNrRURERERERGQIRmL+X+ec80qq3ayly/M0v7Ckg4q1/f32HbD1/pPvE4rAhHP91sUTL4DJs2Dyhf7yWbNh3FQwy0hcERGRHHMOEAaeHrD+aaBisBeYWQFQ0G/V4O0oHrkb/vbJNESUnKCRtSIiIiIiIiISoIm1r/kssGTW0uXLtt2x5Lag80huU7G2v8mpAuz4Gf6csePP9QuzE8+DCef7X8fPgFA46KQiIiJjxUeBT59yr/yof92W0W/STJhZE3QKERERERERERmjZi1dnpc/rXhJavHWkWi1LGOLirX9veS/g04gIiIymj0LJIBpA9ZPA/ac4DVfAL7Sb3k8sOu4vSrf4j9ERERERERERERG0LY7lvTMWrp8GXArsEyFWhmuUNABREREZGxwznUD64Hr+9aZWSi1vOYEr+lyzh3qewCHMxJWRERERERERETkBFKtj/PVAlnSQSNrRUREJJO+AvzQzNYBTcD7gCjwgyBDiYiIiIiIiIiInA6NqJV0GbPF2kOHDgUdQUREzsDEiRMnAIedcy7oLHL6nHM/N7MpwOeA6cDDwE3OuadP5zi6jouI5B5dwwXgyJEjQUeQUaC3JxZ0hJxwKBYPOkJOiIf0t8VQmE02XcNFRERGho21a6yZncdgc92JiEgumZhqiStjjK7jIiI5T9fwMUrXcBGRnKdruIiIyAgZi8VaA85l8DnvxuP/8Xj+CbbL0Oh9TA+9j8On9zA9svF91KicMUrX8RGn9zA99D4On97D9MjG91HX8DHqFNfwoGTjv5Fso/doaPQ+DY3ep1PL5vdI13AREZERMubaIKd+qXhysG3+346A/8uH7hQ7Q3of00Pv4/DpPUwPvY+STXQdH1l6D9ND7+Pw6T1MD72Pkk1Odg0Piv6NnJreo6HR+zQ0ep9OTe+RiIjI2BQKOoCIiIiIiIiIiIiIiIiIyFikYq2IiIiIiIiIiIiIiIiISABUrD1WF/DZ1Fc5c3of00Pv4/DpPUwPvY+SK/SzOnx6D9ND7+Pw6T1MD72PIienfyOnpvdoaPQ+DY3ep1PTeyQiIjIGmeaFFxERERERERERERERERHJPI2sFREREREREREREREREREJgIq1IiIiIiIiIiIiIiIiIiIBULFWRERERERERERERERERCQAKtaKiIiIiIiIiIiIiIiIiARAxdp+zOzdZrbNzDrNrNHMqoLOlEvM7KNmttbMDpvZXjP7nZmVB50rl5nZUjNzZvbVoLPkGjM7z8x+YmbPmVmHmW00swVB58oVZhY2s38zsydS798WM/ukmVnQ2UQGo2v48Ogann66hp85XcOHR9dwERERERERkdyiYm2Kmb0G+ArwWeBK4BHgL2Y2NdBgueVq4BtADXADkAf81cyigabKUWZWCbwdeDToLLnGzCYDq4Ae4B+Ai4EPAvuDzJVjPgK8E3gPMCe1/GHgvUGGEhmMruFpoWt4GukafuZ0DU8LXcNFREREREREcog554LOkBXMrBFY65x7T2o5BOwEvuacuyPQcDnKzKYAe4GrnXMPBp0nl5jZOOAh4F3AJ4CHnXPvCzRUDjGzO4BFzrnFQWfJVWb2J+Bp59xb+q37NdDhnHtDcMlEjqdrePrpGn7mdA0fHl3Dh0/XcJEzY2YR4Fzn3I6gswTJzD4I/Mo5tz3oLDJ6mdkcYLlzbnbQWbKJmV0PvB//ZiuAzcBXnXN/Dy6ViIiIZIJG1gJmlg/MB47+8uOcS6aWFwaVaxSYmPq6L9AUuekb+H+46BfyM/NSYJ2Z/TLVznODmd0edKgcsxq43szKAMzsMqAO+HOgqUQG0DV8xOgafuZ0DR8eXcOHT9dwkTMzF3gi6BBZ4IvAFjP7m5m9JvW7lpwGM7vAzL4fdI4slw9cGHSIbGJm7wLuAQ4D/516HAL+n5m9O8hsIiIiMvIiQQfIEucAYeDpAeufBioyHyf3pUY1fRVY5Zx7LOA4OcXMXovfxrMy6Cw5bDZ++7+vAP+B/17+j5l1O+d+GGiy3HEHMAFoNrME/v8jP+6c+2mwsUSOo2t4mukafuZ0DU8LXcOHT9dwERmutwIvB34MHDKznwDf1e8FQ3YW8CbgzUEHCYqZfeUUu0zJSJDc8jHg/c65r/db9z9mtiq17RvBxBIREZFMULFWRso3gEvw7+KXITKzC/DvnrzBOdcZdJ4cFgLWOec+llreYGaXAO8A9EHv0LwaeD1wC/A4cDnwVTPbrQ/LRUY9XcPPgK7haaNr+PDpGi4yCDN76BS7FGUkSG74f865ZWY2FbgVuA14r5mtB74D3O2cOxxkwCCZ2UtPsYta+8K/AA/jjwwdzLjMRckZk/BH1g70V+A/MxtFREREMk3FWt+zQAKYNmD9NGBP5uPkNjP7OvBi4Crn3K6g8+SY+cBU4CEz61sXBq4ys/cABc65RFDhcshTwKYB6zYDrwwgS676InCHc+7u1PJGM7sQ+Cj6sFyyi67haaRr+LDoGp4euoYPn67hIoO7GLibE7c6ngGUZS5O9nPO7QX+C/gvM1sMvAW4M/UYy8W23wEOsJPs4zITJWu1A3c6534y2EYzuxxYn9FE2e8PwM341/H+Xgb8KfNxREREJJNUrAWcc92pO0Svx/+lu68F4PXA10/yUunH/E8mv4b/y+U1zjnN93P67gXmDVj3A6AZ+E99yDtkq4DyAevKgO0BZMlVHpAcsC6B5jqXLKNreHroGp4Wuoanh67hw6druMjgHgManXPfGmxjqnikObJPUGR0zq0EVprZPwOvyWykrPMU8C7n3O8H26hCJADr8G9kG7RYy6mL3WNC6t9Tn03Ax83sGmBNal0NsAj4cmaTiYiISKapWPu8rwA/NLN1QBPwPiCK/yGbDM038NutvQw4bGbTU+sPOuc6gouVO1KtpI6ZB8jMYsBzmh/otNwJrDazjwG/AKqAt6UeMjR/xP9DcQd+C8UrgA8A3w80lcjgdA0fPl3Dh0nX8LTRNXz4dA0XGdxgN4P0dxh4MENZstlJC2jOuUP4rZDHsvX4hchBi7WoEAnwQaDgRBudc4+gm4gA3j9geT9+F4CL+607gD//8b9nKJOIiIgEwJwb651ZnpdqUfchYDr+3Br/7JxrDDRUDjGzE/0w3eacW5bJLKOJmT0APOyce1/AUXKKmb0Y+AJQit/q7CvOubH+ocKQmdl44N/wR9lNBXYDPwM+55zrDjKbyGB0DR8eXcNHhq7hZ0bX8OHRNVxEZGSlWkJHnXODzS+KmUWBBc65FZlNJiIiIiK5SsVaERERERERERkTzGwmsNMN+DAkNSXABc65HcEkExERERGRsUotR0RERERERERkrHgCmDLI+rNS28Y0M7vSzC7qt/xPZrbKzHaaWb2ZvTbIfJL9zGy/me0byiPorNnCzGaY2RvM7EVmlj9gW9TMPhVUNhEREckMFWtFREREREREZKww/DlFBxoHdGY4Szb6AVAMYGZvBf4XWAd8HlgLfMfM3hxcvOxgZm81sx+a2W2p5deY2WYz22pmnw06X8Dehz8X6/t5fp7VvwCfST3+klr3bxnOlZXMrBLYBHwD+BXwuJnN7bfLOODTQWQTERGRzFEbZBEREREREREZ1czsK6mn/wJ8B4j32xwGqoGEc25RprNlEzOLA3Occ9vN7CHgW/3nDTezW4CPO+fmnvAgo5yZvQ+/CPkXYCF+ke39wJ34P0sfBD7knPu/oDJmCzP7NXC/c+7rA9a/B3iBc+7lgQTLImb2N2An8FYgCvwn8GrgBufcBjObBux2zoUDjCkiIiIjLBJ0ABERERERERGREXZF6qsB84Duftu6gUeAL2U6VBaKA+cA24HzgKYB2xuBiwa+aIx5O/A259xdZnYF/nv0Dufc9wDM7EngncCYL9YCNwIfGWT9PcAdGc6SreYD73bOJYHDwLvMbAdwr5ndCGgebRERkTFAxVoRERERERERGdWcc9cCmNkPgH9xzh0KOFK2+jN+ofGtwArgVfiF7D6vBtoDyJVNLgTqAVIjHxNAQ7/tK1Dhv89zwMuALw9Y/7LUNvEV9l9wzt1hZr3AX4Ex33ZcRERkLFCxVkRERERERETGBOfcbf2XzWwCcB3Q7JxrDiZVVvkIsMrMVuDPVftBM7sG2AyUAzXAzYGlyw5x/Ha1fZ4BjgzYR5+3+T4NfDf1M9SYWlcN3ATcHlCmbPMYUAs82n+lc+5LZhYCfhZIKhEREcmoUNABREREREREREQywcx+kZovEzMrwi9I/gLYaGavDDRcFnDO7cZvGb0Gv6BmQBXwQmAXsMg59/+CS5gVmoFL+xaccxc457b3214BbMt0qGzknFsGLAIOAa9IPQ4BdaltAj/Cf4+O45z7L/yCt1ohi4iIjHLmnAs6g4icodTdqfcDk51zB3Ll2CIiImOdruEiIsEwsz3Ajc65R8zsFuCzwGXAm/DnIb3ipAeQMc/MFgEx59zDJ9j+LiDknPt6RoOJiIiISM7SyFqR02Bmy8zMpR7dZtZuZp8ys5xucWRmD5jZVwesXg3MAA5mPpGIiEh66RouIiIpE4F9qec3Ab92zsWB5UBpYKkkZzjnVp2oUJva/k0Van1m9iIzu3GQ9Tea2T8EkSlbmdl9ZjZpkPUTzOy+ACKJiIhIBuX0h1MiAbkHuA0oAF4EfAPoAb4QZKh0c851A3uCziEiIpJGuoaLiMhOYKGZ7cMv1r42tX4y0BlYqixgZr8BbnXOHUo9PyHn3CsyFCtrmdlWoNI599yA9ZOAh5xzswMJll3uAJYOst5S2/6c2ThZ7Rogf5D1hcDizEYRERGRTNPIWpHT1+Wc2+Oc2+6c+xbwd+ClZnaNmTWZWczMDpjZKjO7sO9FZvYyM3vIzDrNbKuZfbpvNI+ZzUqN9Lm83/6TUuuu6bfuRWbWamYdZnY/MGtgODN7pZk9bmZdZrbNzD54sm/GzJYBVwP/0m/E0azU9+P67uw0s1tT39eLzazFzOJm9isz88zsTalz7Tez/zGzcL/jF5jZl8zsydR709j/exIREckgXcN1DRcR+SrwU/z5V3cDD6TWXwVsDCZS1jgIuH7PT/YQ/1oeHmR9AXB+ZqNkrVJg0yDrm4GSDGfJSmZ2qZn1zYF8cd9y6nEF8BbgyQAjioiISAZoZK3I8HUAZwO/A74DvA7/bsgqUn/omtli4EfAPwMrgWLg/1Kv/+xQTmJmFwC/wR8F9H/AAuDLA/aZD/wC+Azwc6AW+KaZPeecW3aCQ/8LUAY8Bnwqte4ZBvkQGfBS38NrgfGpPL8FDuCPUJoN/BpYlTo/wNeBi1Ov2Q3cDNxjZvOcc21D+d5FRERGiK7huoaLyBjjnPummTUBFwB/c84lU5u2Ap8ILlnwnHO3DfZcjmVmL+23eKOZ9S9eh4HrgScymyprHcT/HWPbgPUlQCzjabLTw/i/dzpgsHbHHcB7MxlIREREMk/FWpEzZGaG/0fYjcCPgSuBPznntqR22dxv908Ddzjnfpha3mpmnwT+iyF+0Au8E9jinOsbZdNiZvOAj/Tb5wPAvc65f0stt5rZxcCHgGWDHdQ5d9DMuoG4c+5oy0T/2ztOHvDOvu/RzH4F/BMwzTl3BNiUGi10LfBzM5uJ325ypnNud+oYXzKzm1LrPzbE711ERCRtdA3XNVxExjbn3Dpg3YB1ywOKk5XM7HXOuZ+dYNsXnXMfynSmLPK7fs9/OGBbD35h8qTdMcaQ3wNfNbOb+/0OUoJ/09ofAk2WPS7Cbwu9Ff+GwWf6besG9jrnEkEEExERkcxRsVbk9L3YzI7gf+gZAu4C3of/7+kvZvY3/LaKv3DOPZV6zWXAIjP7eL/jhIFCM/OGeN45QOOAdWsG2ef3A9atAt6XamtYy7FzwrzdOffTIZ4f/A+Dt/RbfhrYlvqQt/+6qann8/C/z9YBHxwXAMfM6yMiIpIBuoY/T9dwERmTzOwrJ9jk8OesbQd+75zbl7lUWelbZnbAOXfMnKJmdid+x4UxW6x1zoUAzOwJYMHAOWvlGB8G7gGazWxXat0FwIPAvwaWKos457annmqqOhERkTFMxVqR03c//giZbmC3c643tf42M/sf4CbgNcC/m9kNzrkGYBz+yJzfDHK8TqCv9Vb/T0PzRiD7OuDyfstPn+brewYsuxOs6/sjYxyQAOanvvZ3BBERkczSNfx5uoaLyFh1BX5HhTDQklpXhv//umbgXcCXzazOOTfYXJtjxeuBn5nZi51z9QBm9jXgFfhdGMY0M8vDHwl5FrqJ6YRSXUBqgRvwb4DrAB5xzq0MNll2MrN/At6BP9p2oXNuu5m9H9jqnBt4U5+IiIiMIirWipy+mHOufbANzrkNwAbgC2a2BrgFaAAeAspP9Doz62tzMyP1ejj2A1nwWzK+dMC6mkH2WTRg3SKgNdU2pwP/TvGBuvE/rEi3DanjTtUfYyIikgV0DR86XcNFZLT6PbAPuM05dwjAzCYC3wXq8ecwvwu4E79d/pjknFtuZu8C/mBmNwBvAV4GXOucaw02XfCccz1mdmnQObKVmS0EznbO/ck554C/mtkM/CkkPDP7HfBe51xXkDmziZm9E/gc8FXg4zz/+91+/E4wKtaKiIiMYmqxIZIGZnaRmX3BzBaa2YVm9kKglOfnvPsc8EYz+7SZzTWzOWb2WjP7dwDnXAf+B8JLU9uuBv59wGm+DZSa2RfNrNzMbgFuHbDPl4HrzeyTZlZmZm8C3gN86RTfwjag2sxmmdk5ZpaW/zek/oj/KfAjM3tF6n2qMrOPmtmSdJxDRERkOHQNH5yu4SIyin0I+GRfoRb80X/AZ4APO+fi+P/vnx9MvOzhnLsL+AR+W/6XAFerUHuMn+AXseV4nwLm9i2Y2Tz8GyH+BtyB//P00WCiZa33Arc75z7PsV1N1uFPTyEiIiKjmEbWiqRHHKgA3gScDTwFfAP4XwDn3F/M7MX4f7B8BL/tYDP+3dt93gx8D1iP347rw8Bf+zY653aY2Svx7/B+L9AEfAz4fr99HjKzV+N/uPDJVI5POeeWnSL/l4AfApuAIvyWO+lyG/4f+F8GzgOexf9Q+09pPIeIiMiZ0jX8xHQNF5HRaCL+/NwDWxxPASaknh8A8jOYKSucZD7fZ/A7Tbyrbx5z59wHMpUri0WAN5vZC/B/B4j13zjG36PL8X+f6fNaoMk5dzuAme3EH2X7mYwny14X8XyXlv66gGiGs4iIiEiGmd+NRERERERERERkdDOznwILgQ8Ca1OrK/FvflntnPsnM3st8K/OuQUBxQyEmd0/1H2dc5q39uTv1/ix9vPTn5l1AqXOuZ2p5Xrgz6lRo5jZLGCjc258cCmzi5ltAj7qnPu9mR0GLnPObTWz9+K3bb8y4IgiIiIygjSyVkRERERERETGirfjdzq4G/8zEcPvmvBDoG8kZDPw1kDSBUgF2KExs/c75+480ftlZuOBezIcK9s8jT9SdKeZ5QNXAp/ut308/r87ed5XgG+YWSH+/5eqzOx1+O2ix9z/j0RERMYajawVERERERERkTHFzMYBs1OLW51zR4LMky3M7Pun3gvnnBuzc7WaWQfwdufcjwbZFgX+ApzjnKvIeLgsYWbfAi7Dn0Li5fjTTZzrnOtObX898D7nXGVgIbNQ6n35DFCcWrUb+LRz7nuBhRIREZGMULFWREREREREREa1IRYhcc69eaSzZDMzSwLb8efOtBPt55y7OWOhsoyZvQr4MfAa59wf+q0fhz+idipwjXNud0ARA2dm5wC/AeqAI8CbnHO/7bf9XqDBOffxgCJmNTPzgHHOub1BZxEREZHMULFWREREREREREY1FSGHxsy+AbwO/736AfAT59y+YFNlHzN7K/DfwBLn3AOpEbX3ANOBq8dyobY/M5sIHHHOJQasPyu1vjuYZNnHzIrwP6eNp5YvBG4GNjnn/hpoOBERERlxKtaKiIiIiIiIyKimIuTQmVkB8ArgzUAtsBz4HvBXpw+RjjKzDwMfB14GfA44D79QuyvQYJKTzOyvwG+cc982s0lAC9ANnAN8wDn3rSDziYiIyMhSsVZERERERERERj0VIU9fanTfrcAbgQgwV/P7Ps/M7gA+BGzDb328M9hEkqvM7Fn8Yv/jqZHb7wWuAF4JfM45NyfQgCIiIjKiIkEHEBEREREREREZac65LuBnwM/6FSG/CUTMTEXIwSUBh986OhxwlqxgZr8ZsKoHeBb4b7PnO2w7516RyVyS8zzgcOr5C/FH2SbNrAG4MLhYIiIikgmhoAOIiIiIiIiIiGSYipAnYGYFZvY6M/sb0ArMA94DzFRBG4CDAx4/AzYNsl7kdLQDLzezC4Abgb55aqcChwJLJSIiIhmhNsgiIiIiIiIiMuoNaINcB/wJf/7ae5xzySCzZQsz+ybwWmAn8H3gp865Z4NNJTL6mdmrgLvwbx651zn3wtT6jwJXOef+Ich8IiIiMrJUrBURERERERGRUU1FyKExsySwA9iAP/J4UGrxK5J+ZjYdmAE80ncDiZlVAYecc82BhhMREZERpWKtiIiIiIiIiIxqKkIOjZkt4yTvTx/n3G0jn0ZEREREZGyIBB1ARERERERERGSE/YghFCHHOufcrUFnEBmrzGwB8GpgJpDff9tYv5FERERktFOxVkRERERERERGNRUhRSSbmdlr8W8q+QvwQuCvQBkwDfhtgNFEREQkA0JBBxAREREREREREREZwz4GvN859xKgG/gXoAL4BX4LdxERERnFVKwVERERERERERERCU4xsDz1vBuIOucccCfwtsBSiYiISEaoWCsiIiIiIiIiIiISnP3A+NTzJ4FLUs8nAV4QgURERCRzNGetiIiIiIiIiIiISHAeBG4ANgK/BP7bzK5Lrbs3yGAiIiIy8szvqCEiIiIiIiIiIiIimWZmZwGFzrndZhYCPgzUAm3Avzvn9gcaUEREREaUirUiIiIiIiIiIiIiIiIiIgFQG2QRERERERERERGRAKVG1JYAU4FQ/23OuQcDCSUiIiIZoWKtiIiIiIiIiIiISEDMrAa4C7gQsAGbHRDOeCgRERHJGBVrRURERERERERERILzbWAdsAR4Cr9AKyIiImOE5qwVERERERERERERCYiZxYDLnHPtQWcRERGRzAudehcRERERERERERERGSGN+PPVioiIyBikNsgiIiIiIiIiIiIiwfka8GUzmw5sBHr6b3TOPRpIKhEREckItUEWERERERERERERCYiZJU+y2TnnwhkLIyIiIhmnkbUiIiIiIiIiIiIiwbnoJNuiGUshIiIigdDIWhEREREREREREZEsYmYFwLuBDzvnpgedR0REREZOKOgAIiIiIiIiIiIiImONmRWY2RfMbJ2ZrTazl6fW3wY8AbwfuDPIjCIiIjLyNLJWREREREREREREJMPM7D+BtwN/B2qBKcAPgBrgP4BfOucSwSUUERGRTNCctSIiIiIiIiIiIiKZ94/AG51zfzCzS4BH8T+vvcxphI2IiMiYoZG1IiIiIiIiIiIiIhlmZt3ARc65J1PLHUCVc25jsMlEREQkkzRnrYiIiIiIiIiIiEjmhYHufsu9wJGAsoiIiEhA1AZZREREREREREREJPMMWGZmXanlQuDbZhbrv5Nz7hUZTyYiIiIZo2KtiIiIiIiIiIiISOb9cMDyTwJJISIiIoHSnLUiIiIiIiIiIiIiIiIiIgHQnLUiIiIiIiIiIiIiIiIiIgFQsVZEREREREREREREREREJAAq1oqIiIiIiIiIiIiIiIiIBEDFWhERERERERERERERERGRAKhYKyIiIiIiIiIiIiIiIiISABVrRURERERERERkTDAzd4rHZwLO9vKgzi8iIiIiwYgEHUBERERERERERCRDZvR7/hrgc0B5v3VHTudgZpbvnOtORzARERERGZs0slZERERERERERMYE59yevgdw0F91dDkK/NTMnjazI2a21sxe0P/1ZrbNzD5pZj8ys0PA/6XW325mO80sbma/NbMPmNmBAa99mZk9ZGadZrbVzD5tZpG+46Z2+21qhO02RERERGRMULFWREREREREREQExgH/D7geuAK4B/ijmc0csN+/Ao+k9vk3M1sEfBv4b+By4G/Ax/u/wMwWAz9K7XMx8Hbg1n77Vaa+3oY/+rcSERERERkTzDkXdAYREREREREREZGMMrNbga865yadZJ/HgG87576eWt4GbHDO3dxvn7uBcc65F/db9xPgxX3HNrO/A/c6577Qb583AP/lnDs3teyAm51zv0vTtygiIiIiOUAja0VEREREREREZMwzs3Fm9iUz22xmB8zsCDAHGDiydt2A5XKgacC6gcuXAZ9KtVc+kjr2d4AZZual63sQERERkdwTCTqAiIiIiIiIiIhIFvgScAN+m+N2oAP4FZA/YL/YGRx7HPBp4DeDbOs8g+OJiIiIyCihYq2IiIiIiIiIiAgsApY5534L/khbYNYQXtfC8XPMDlx+CCh3zrWf5Dg9QHhoUUVERERktFCxVkRERERERET+f3v3q6IFEIVx+Dd278PgLQiiIAomb8AqNlejSS/AsEXwDqxrWWzCimAQsyD+uYBNBotj+FbYpAtfmP3wefLM4Ux+OWeA+lTdGWMcVLN62tm+ENuv3owx9qqD6lp166TGH0+qV2OMb22mdX+1WY18ec75+OTMl+r6GOOo+jnnPN7+SQAAnHf+rAUAAACA2quOq7dtQtfDNhOxfzXnPKrundz/WN2snnVqvfGc87C6Xd2o3lfvqgfV11OlHrZZw/y9+rD1awAA2AljzvnvUwAAAADAmYwxXlSX5pxXVvcCAMD5Zg0yAAAAAGxhjPGoel39aLMC+W51f2lTAADsBJO1AAAAALCFMcbL6mp1sfpc7c85ny9tCgCAnSCsBQAAAAAAAFjgwuoGAAAAAAAAAP5HwloAAAAAAACABYS1AAAAAAAAAAsIawEAAAAAAAAWENYCAAAAAAAALCCsBQAAAAAAAFhAWAsAAAAAAACwgLAWAAAAAAAAYAFhLQAAAAAAAMACvwGEJN+Xf5Xr6wAAAABJRU5ErkJggg==\n",
      "text/plain": [
       "<Figure size 2400x1200 with 12 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "branch=branches[branchname]\n",
    "# Draw dynamic network with notch layout\n",
    "layout1=layout.notch(nframe=nframe,dpi=dpi)\n",
    "pts,fig,panels,animate_ka=layout1.draw(\n",
    "\td0,branch,\n",
    "\t# Set genes to annotate\n",
    "\tbcde_tfs=tfs_ann,e_targets=target_ann,f_tfs=tfs_subnet,\n",
    "\t# Custom legend location for long cell type name\n",
    "\ta_ka={'scatterka':{'legend_loc':(0.6,1)}},\n",
    "\t# Custom configurations for color range\n",
    "\te_ka={'lim':[-0.02,0.02]},\n",
    ")\n",
    "ca=panel.animate_generic(pts,fig,panels)\n",
    "anim=ca.animate(**animate_ka)\n",
    "\n",
    "# Save animation to mp4 file\n",
    "# If you receive a codec related error, try a different codec, e.g. with `codec='h264'`. You can list available codecs with `ffmpeg -codecs`.\n",
    "w=matplotlib.animation.writers['ffmpeg_file'](fps=fps,codec='mpeg4')\n",
    "w.frame_format='jpeg'\n",
    "fo=f'../output/animation-{branchname}.mp4'\n",
    "anim.save(fo,writer=w,dpi='figure')\n",
    "display(FileLink(fo))\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Download the inferred dynamic networks to share with collaborators and community\n",
    "Others can load the networks and re-analyze/re-draw them with Dictys"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<a href='../output/dynamic.h5' target='_blank'>../output/dynamic.h5</a><br>"
      ],
      "text/plain": [
       "../output/dynamic.h5"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from IPython.display import FileLink\n",
    "FileLink('../output/dynamic.h5')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
